{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "15f5ff67-0ddc-43c6-b48a-2941be338093",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\Users\\shawn\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\transformers\\modeling_utils.py:4713: FutureWarning: `_is_quantized_training_enabled` is going to be deprecated in transformers 4.39.0. Please use `model.hf_quantizer.is_trainable` instead\n",
      "  warnings.warn(\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "27a54e20cbd7438fa2dc8c29fcdf6778",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Loading checkpoint shards:   0%|          | 0/5 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "import os\n",
    "os.environ[\"CUDA_VISIBLE_DEVICES\"] = \"1\"\n",
    "# os.environ[\"USE_FLASH_ATTENTION\"] = \"1\"\n",
    "\n",
    "from transformers import AutoModelForCausalLM, AutoTokenizer\n",
    "device = \"cuda\" # the device to load the model onto\n",
    "\n",
    "model = AutoModelForCausalLM.from_pretrained(r\"D:\\Local LLM\\models\\Qwen\\Qwen2.5\\Qwen2.5-32B-Instruct-GPTQ-Int4\",\n",
    "    device_map=\"auto\", torch_dtype=\"auto\",attn_implementation=\"flash_attention_2\",\n",
    ")\n",
    "tokenizer = AutoTokenizer.from_pretrained(r\"D:\\Local LLM\\models\\Qwen\\Qwen2.5\\Qwen2.5-32B-Instruct-GPTQ-Int4\")\n",
    "\n",
    "#Qwen1.5-72B-Chat-GPTQ-Int4\n",
    "#Qwen1.5-14B-Chat"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "34f38374-11ac-4236-8b83-7e19083eac9b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved combined search results to saccharomyces cerevisiae production gene expressionsaccharomyces cerevisiae production gene deletion_pubmed_search_results.xlsx\n"
     ]
    }
   ],
   "source": [
    "from Bio import Entrez\n",
    "import pandas as pd\n",
    "\n",
    "# Define your email to use with NCBI Entrez\n",
    "Entrez.email = \"xiao.zhengyang@wustl.edu\"\n",
    "\n",
    "def search_pubmed(keyword):\n",
    "    # Adjust the search term to focus on abstracts\n",
    "    search_term = f\"{keyword}[Abstract]\"\n",
    "    handle = Entrez.esearch(db=\"pubmed\", term=search_term, retmax=1000)\n",
    "    record = Entrez.read(handle)\n",
    "    handle.close()\n",
    "    # Get the list of Ids returned by the search\n",
    "    id_list = record[\"IdList\"]\n",
    "    return id_list\n",
    "\n",
    "def fetch_details(id_list):\n",
    "    ids = ','.join(id_list)\n",
    "    handle = Entrez.efetch(db=\"pubmed\", id=ids, retmode=\"xml\")\n",
    "    records = Entrez.read(handle)\n",
    "    handle.close()\n",
    "\n",
    "    # Create a list to hold our article details\n",
    "    articles = []\n",
    "\n",
    "    for pubmed_article in records['PubmedArticle']:\n",
    "        article = {}\n",
    "        article_data = pubmed_article['MedlineCitation']['Article']\n",
    "        article['Title'] = article_data.get('ArticleTitle')\n",
    "        \n",
    "        # Directly output the abstract\n",
    "        abstract_text = article_data.get('Abstract', {}).get('AbstractText', [])\n",
    "        if isinstance(abstract_text, list):\n",
    "            abstract_text = ' '.join(abstract_text)\n",
    "        article['Abstract'] = abstract_text\n",
    "\n",
    "        article['Journal'] = article_data.get('Journal', {}).get('Title')\n",
    "\n",
    "        articles.append(article)\n",
    "\n",
    "    return articles\n",
    "\n",
    "def perform_search_and_fetch(keyword):\n",
    "    id_list = search_pubmed(keyword)\n",
    "    return fetch_details(id_list)\n",
    "\n",
    "# Example usage: Performing two searches\n",
    "keyword1 = \"saccharomyces cerevisiae production gene expression\"\n",
    "keyword2 = \"saccharomyces cerevisiae production gene deletion\"\n",
    "keyword = keyword1 + keyword2\n",
    "# Fetch articles for both keywords\n",
    "articles1 = perform_search_and_fetch(keyword1)\n",
    "articles2 = perform_search_and_fetch(keyword2)\n",
    "\n",
    "# Convert both lists of articles to DataFrames\n",
    "df1 = pd.DataFrame(articles1)\n",
    "df2 = pd.DataFrame(articles2)\n",
    "\n",
    "# Add a column to differentiate the search terms in the final DataFrame\n",
    "df1['SearchTerm'] = keyword1\n",
    "df2['SearchTerm'] = keyword2\n",
    "\n",
    "# Concatenate the DataFrames\n",
    "combined_df = pd.concat([df1, df2], ignore_index=True)\n",
    "\n",
    "# Save the combined DataFrame to an Excel file\n",
    "excel_filename = keyword+\"_pubmed_search_results.xlsx\"\n",
    "combined_df.to_excel(excel_filename, index=False)\n",
    "\n",
    "print(f\"Saved combined search results to {excel_filename}\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "2b070d21-ec38-44c8-9119-a0ea88746198",
   "metadata": {
    "scrolled": true,
    "tags": []
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Response for Row 1:\n",
      "Answer to Question 2: (deletion of gene Acptp2,3, reduced sexual spore production by 4.4 times), (deletion of gene Acptp2,3, reduced asexual spore production by 4.6 times), (deletion of gene Acptp2,3, decreased response to sorbitol-induced osmotic stress), (deletion of gene Acptp2,3, slower growth in high concentration sucrose medium), (deletion of gene Acptp2,3, slower growth in hydrogen peroxide), (deletion of gene Acptp2,3, slower growth in Congo red), (deletion of gene Acptp2,3, slower growth in SDS), (deletion of gene Acptp2,3, deeper colony color), (deletion of gene Acptp2,3, altered expression levels of stress-related genes), (deletion of gene Acptp2,3, altered expression levels of pigments-related genes)\n",
      "Progress: 0.05% completed\n",
      "Response for Row 2:\n",
      "Answer to Question 2: (deletion of gene tri18, absence of type D trichothecene toxin epiroridin E), (deletion of gene tri18, reduction in pathogenicity of P. roridum towards pumpkin), (heterologous expression of tri3, toxin resistance in Saccharomyces cerevisiae), (heterologous expression of tri17KR, toxin resistance in Saccharomyces cerevisiae)\n",
      "Progress: 0.10% completed\n",
      "Response for Row 3:\n",
      "Answer to Question 2: (expression of genes related to the uptake of phosphate, improved wastewater treatment performance), (expression of genes related to the uptake of ammonium ions, improved wastewater treatment performance)\n",
      "Progress: 0.15% completed\n",
      "Response for Row 4:\n",
      "Answer to Question 2:  (overexpression of metabolism-related genes, altered metabolic pathways), (overexpression of stress response genes, enhanced stress tolerance), (overexpression of cell adhesion genes, increased cell attachment)\n",
      "Progress: 0.20% completed\n",
      "Response for Row 5:\n",
      "Answer to Question 2:  (overexpression of gene C3H15, lower chlorophyll content), (overexpression of gene C3H15, reduced photosynthetic efficiency), (overexpression of gene C3H15, accelerated senescence), (overexpression of gene C3H15, altered mobilization of copper to seeds), (overexpression of gene C3H15, altered mobilization of zinc to seeds)\n",
      "Progress: 0.25% completed\n",
      "Response for Row 6:\n",
      "Answer to Question 2: (deletion of gene zwf1, change in protein content), (deletion of gene zwf1, change in enzymatic activity), (deletion of gene gnd1, change in protein content), (deletion of gene gnd1, change in enzymatic activity), (deletion of gene ald6, change in protein content), (deletion of gene ald6, change in enzymatic activity)\n",
      "Progress: 0.30% completed\n",
      "Response for Row 7:\n",
      "Answer to Question 2: (overexpression of gene POS5Δ17, enhanced reduced nicotinamide adenine dinucleotide phosphate pool), (overexpression of gene POS5Δ17, elevated titer of vitamin A by almost 46%), (introduction of gene retinol dehydrogenase 12, achieved more than 90% retinol purity), (combinational expression of genes Mbblh and Ssbco, increased flux from β-carotene to vitamin A metabolism), (multi-copy integration of gene Mbblh, improved synthesis of vitamin A)\n",
      "Progress: 0.35% completed\n",
      "Response for Row 8:\n",
      "Answer to Question 2: (deletion of oxygen, enhanced fermentation rate), (deletion of oxygen, biomass yield increase), (overexpression of genes involved in lipid biosynthesis pathways, anaerobiosis), (overexpression of genes associated with thiamine metabolism, presence of oxygen), (overexpression of genes associated with methionine metabolism, presence of oxygen), (overexpression of genes associated with sulfur metabolism, presence of oxygen), (deletion of oxygen, decline in total levels of NAD(H)), (presence of oxygen, total NADP(H) levels increase)\n",
      "Progress: 0.40% completed\n",
      "Response for Row 9:\n",
      "Answer to Question 2: (expression of gene invertase, consequence increases extracellular hexose concentrations), (expression of gene invertase, consequence suppresses metabolic efficiency of competitors), (expression of gene invertase, consequence enhances hexose capture rate), (expression of gene invertase, consequence induces transporter expression)\n",
      "Progress: 0.45% completed\n",
      "Response for Row 10:\n",
      "Answer to Question 2: (overexpression of gene tHMGR, increased FPP production), (overexpression of gene ERG20, increased FPP production), (overexpression of gene GGPPS, increased FPP production), (overexpression of gene TS, increased taxadiene production), (overexpression of all mevalonate pathway genes, increased FPP availability), (increased FPP availability, enhanced taxadiene production)\n",
      "Progress: 0.50% completed\n",
      "Response for Row 11:\n",
      "Answer to Question 2: (Deletion of gene FgMDM33, consequence 1: defects in mycelial growth), (Deletion of gene FgMDM33, consequence 2: reduction in conidia production), (Deletion of gene FgMDM33, consequence 3: decreased virulence), (Deletion of gene FgMDM33, consequence 4: elongated mitochondria), (Deletion of gene FgMDM33, consequence 5: respiratory-deficient growth phenotype), (Deletion of gene FgMDM33, consequence 6: reduction in proteolysis of GFP-FgAtg8), (Deletion of gene FgMDM33, consequence 7: upregulation of FgAIF1 gene expression)\n",
      "Progress: 0.55% completed\n",
      "Response for Row 12:\n",
      "Answer to Question 2: (expression of gene YJR107W, consequence specific activity of 0.397 U/mg), (expression of gene YJR107W, consequence optimum temperature of 37 °C), (expression of gene YJR107W, consequence pH 6), (expression of gene LIP3, consequence no lipase activity), (expression of gene LIP3, consequence specific activity of 1.36 U/mg in bioreactor), (expression of gene YJR107W, consequence specific activity of 1.25 U/mg in bioreactor)\n",
      "Progress: 0.60% completed\n",
      "Response for Row 13:\n",
      "Answer to Question 2: (co-expression of α-amylase gene, enzyme production), (co-expression of α-amylase gene, SPR hydrolysis), (co-expression of α-amylase gene, glucose fermentation), (co-expression of glucoamylase gene, enzyme production), (co-expression of glucoamylase gene, SPR hydrolysis), (co-expression of glucoamylase gene, glucose fermentation), (expression of α-amylase gene, production of 33.03 g/L ethanol), (expression of glucoamylase gene, production of 28.37 g/L ethanol), (co-expression of α-amylase and glucoamylase genes, production of 42.22 g/L ethanol), (expression of glucoamylase gene, increased ethanol concentration to 50.55 g/L)\n",
      "Progress: 0.65% completed\n",
      "Response for Row 14:\n",
      "Answer to Question 2: (deletion of gene SSK1, consequence high antioxidant activity), (deletion of gene HIP1R, consequence high antioxidant activity), (overexpression of gene PAS13, consequence balanced profile), (overexpression of gene FRI751, consequence high polyphenol content)\n",
      "Progress: 0.70% completed\n",
      "Response for Row 15:\n",
      "Answer to Question 2: (yeast mixture administration, increased faecal consistency), (yeast mixture administration, increased abundance of Bifidobacterium), (yeast mixture administration, increased abundance of Coprococcus 2), (yeast mixture administration, decreased abundance of Clostridium Sensu Stricto 1), (yeast mixture administration, increased villous height), (yeast mixture administration, increased villous width), (yeast mixture administration, increased crypt depth), (yeast mixture administration, higher presence of PAS-positive mucins)\n",
      "Progress: 0.75% completed\n",
      "Response for Row 16:\n",
      "Answer to Question 2:  (asymmetric bulge at 3' end of miR-5p, reduced precision of the second cleavage), (asymmetric bulge at other sites of miR-5p, mainly affected the accumulation level of miR-5p), (many asymmetric bulges in miR-3p, strong negative impact on processing precision and accumulation level of miR-5p), (asymmetric bulges on middle of miR-5p, less impact on DCL1 cleavage efficiency and precision), (asymmetric bulges on miR-3p or near ends of miR-5p, strongly reduce DCL1 cleavage activity, precision or both), (interaction between 2-nt 3' overhang of pre-miRNA and 3'BP, important for precision of the second cleavage reaction)\n",
      "Progress: 0.80% completed\n",
      "Response for Row 17:\n",
      "Answer to Question 2: (deletion of gene PRS5, increase in RNA content), (deletion of gene PRS5, upregulation of genes sharing similar functions), (deletion of gene DBP8, increase in RNA content), (deletion of gene DBP8, upregulation of genes sharing similar functions), (upregulation of gene TAL1, increase in RNA content), (upregulation of gene PGM2, increase in RNA content), (upregulation of gene TAL1, activation of pentose phosphate pathway), (upregulation of gene PGM2, activation of pentose phosphate pathway), (deletion of gene PRS5, upregulation of CDK genes), (deletion of gene PRS5, increase in intracellular nucleotide levels), (deletion of gene DBP8, increase in intracellular nucleotide levels), (polygenic recombinant strain G03-△PRS5+△DBP8, increase in RNA content by 71.8%), (polygenic recombinant strain G03-△PRS5+△DBP8+PGM2, increase in RNA content by 80.1%), (polygenic recombinant strain G03-△PRS5+△DBP8, increase in maximum specific growth rate), (polygenic recombinant strain G03-△PRS5+△DBP8+PGM2, increase in maximum specific growth rate)\n",
      "Progress: 0.85% completed\n",
      "Response for Row 18:\n",
      "Answer to Question 2: (overexpression of gene MGD1, consequence increased monogalactosyldiacylglycerol production), (overexpression of gene GAT1, consequence increased monogalactosyldiacylglycerol production), (overexpression of gene ICT1, consequence increased monogalactosyldiacylglycerol production), (overexpression of gene PAH1, consequence increased monogalactosyldiacylglycerol production), (multi-copy integration of MGD1 expression cassette, consequence increased monogalactosyldiacylglycerol production)\n",
      "Progress: 0.90% completed\n",
      "Response for Row 19:\n",
      "Answer to Question 2: (deletion of gene tcb1, upregulation of glycolysis), (deletion of gene tcb1, upregulation of fermentation), (deletion of gene tcb1, upregulation of amino acid synthesis), (deletion of gene tcb1, upregulation of low-affinity glucose uptake), (deletion of gene tcb1, downregulation of tricarboxylic acid cycle), (deletion of gene tcb1, downregulation of respiration), (deletion of gene tcb1, downregulation of high-affinity glucose uptake), (deletion of gene tcb1, downregulation of amino acid uptake), (deletion of gene tcb1, increased growth), (deletion of gene tcb1, increased glucose consumption), (deletion of gene tcb1, increased CO2 production), (deletion of gene tcb1, increased ethanol generation)\n",
      "Progress: 0.95% completed\n",
      "Response for Row 20:\n",
      "Answer to Question 2: (expression of gene MET15, growth induction), (expression of gene FAR1, growth inhibition)\n",
      "Progress: 1.00% completed\n",
      "Response for Row 21:\n",
      "Answer to Question 2:  (expression of gene RgCuAO1, production of salidroside), (expression of gene RgPAR1, production of salidroside), (expression of gene RgUGT2, production of salidroside), (expression of gene RgTyDC2, production of salidroside)\n",
      "Progress: 1.05% completed\n",
      "Response for Row 22:\n",
      "Answer to Question 2: (deletion of gene med3, impaired S. cerevisiae growth under hyperosmotic stress), (deletion of gene med3, decreased intracellular NADPH content), (deletion of gene med3, increased oxidative stress), (deletion of gene med3, elevated levels of intracellular reactive oxygen species), (deletion of gene med3, impacted bud formation)\n",
      "Progress: 1.10% completed\n",
      "Response for Row 23:\n",
      "Answer to Question 2: (deletion of chaperone genes, sensitivity to canavanine), (canavanine exposure, attenuation of RTG-target-gene expression), (deletion of idh1, sensitivity to canavanine), (deletion of idh2, sensitivity to canavanine), (deletion of idh1, synthetic growth inhibition with chaperone mutants), (deletion of idh2, synthetic growth inhibition with chaperone mutants)\n",
      "Progress: 1.15% completed\n",
      "Response for Row 24:\n",
      "Answer to Question 2: (overexpression of gene ZWF1, increased xylitol production), (deletion of gene ALD6, increased xylitol production), (reduced expression of gene PGI1, improved xylitol production on glucose), (heterologous expression of gene GDP1, improved xylitol production on glucose), (expression of gene POS5, improved xylitol production on glucose), (overexpression of gene ZWF1, improved l-galactonate production), (deletion of gene ALD6, improved l-galactonate production)\n",
      "Progress: 1.20% completed\n",
      "Response for Row 25:\n",
      "Answer to Question 2: (deletion of natural promoters, limited transcriptional capacity), (overexpression using synthetic promoters, enhanced metabolic engineering requirements)\n",
      "Progress: 1.25% completed\n",
      "Response for Row 26:\n",
      "Answer to Question 2: (expression of Frs2v, cell growth inhibition), (expression of Frs2v, sensitivity to 2 mM 4-CP), (removal of Frs2v-expressing plasmid, after culturing in 4-CP-containing medium), (expression of Frs2v, scarless gene replacement)\n",
      "Progress: 1.30% completed\n",
      "Response for Row 27:\n",
      "Answer to Question 2: (expression of gene EhN3, high desaturation activity toward C18-ω6 PUFAs), (expression of gene EhN3, high desaturation activity toward C20-ω6 PUFAs), (expression of gene EhN3, utilization of galactolipids as substrates), (decreased growth temperature, increased expression of gene EhN3), (increased expression of gene EhN3, increased production of ω3-PUFAs), (increased expression of gene EhN3, increased production of OPA)\n",
      "Progress: 1.35% completed\n",
      "Response for Row 28:\n",
      "Answer to Question 2: (deletion of gene ulp2, increase in the levels of intron-containing pre-mRNA of ribosomal protein transcripts), (deletion of gene ulp2, improper splicing of ribosomal protein transcripts)\n",
      "Progress: 1.40% completed\n",
      "Response for Row 29:\n",
      "Answer to Question 2: (deletion of gene Esc2, consequence Ty1 transcription受影响), (deletion of gene Rad57, consequence Ty1 transcription受影响), (overexpression of gene GAGsi, consequence gene silencing), (overexpression of gene GAGsi, consequence silent chromatin formation)\n",
      "Progress: 1.45% completed\n",
      "Response for Row 30:\n",
      "Answer to Question 2: (upregulation of genes involved in maintaining cell wall integrity, consequence 1), (upregulation of genes involved in cell wall structural components, consequence 2), (upregulation of genes involved in the production of H₂S, consequence 3), (upregulation of HSP stress response genes, consequence 4), (downregulation of genes belonging to the FLO family, consequence 5)\n",
      "Progress: 1.50% completed\n",
      "Response for Row 31:\n",
      "Answer to Question 2: (expression of gene mleS, reduction in L-malic acid content), (expression of gene MAE1, reduction in L-malic acid content), (expression of gene mleA, reduction in L-malic acid content), (expression of gene MAE2, reduction in L-malic acid content), (expression of gene melP, reduction in L-malic acid content), (expression of gene MAE1 and mleS, improvement in flavor quality), (expression of gene MAE1 and MAE2, reduction in L-malic acid content)\n",
      "Progress: 1.55% completed\n",
      "Response for Row 32:\n",
      "Answer to Question 2: (deletion of gene HMX1, consequence: downregulation of 14 genes related to the cell wall and membrane system), (deletion of gene HMX1, consequence: decrease in acetyl-CoA levels by 1.02-fold), (deletion of gene HMX1, consequence: decrease in riboflavin levels by 1.44-fold), (deletion of gene HMX1, consequence: increase in pantothenic acid content)\n",
      "Progress: 1.60% completed\n",
      "Response for Row 33:\n",
      "Answer to Question 2: (no specific gene expression/deletion mentioned, general discussion on splicing regulation in yeast)\n",
      "Progress: 1.65% completed\n",
      "Response for Row 34:\n",
      "Answer to Question 2: (deletion of gene involved in nitrogen metabolism, consequence 1), (deletion of gene involved in pentose phosphate pathway, consequence 2), (deletion of gene involved in ethanol conversion, consequence 3), (deletion of gene involved in amino acid metabolism, consequence 4), (deletion of gene involved in fatty acid β-oxidation, consequence 5), (overexpression of gene involved in fatty acid precursor biosynthesis, consequence 6), (overexpression of gene involved in boosting carbon fluxes of acetyl-CoA, consequence 7), (overexpression of gene involved in boosting carbon fluxes of NADPH, consequence 8)\n",
      "Progress: 1.70% completed\n",
      "Response for Row 35:\n",
      "Answer to Question 2: (TORC1 inhibition, increased proteasome assembly chaperone translation), (TORC1 inhibition, activation of MAPK Mpk1), (TORC1 inhibition, relocalisation of assembly chaperone mRNA to patches of dense actin), (TORC1 inhibition, activation of cell wall integrity pathway), (TORC1 inhibition, activation through Wsc1, Wsc3 and Wsc4 sensor proteins), (Sch9 inhibition, necessary for driving protein expression)\n",
      "Progress: 1.75% completed\n",
      "Response for Row 36:\n",
      "Answer to Question 2: (deletion of gene transmembrane transport, consequence generalized downregulation), (overexpression of gene sulfur compounds, consequence upregulation), (overexpression of gene pentose phosphate pathway, consequence upregulation)\n",
      "Progress: 1.80% completed\n",
      "Response for Row 37:\n",
      "Answer to Question 2: (overexpression of key genes tHMG1-ERG20, increased valencene yield by 93.49%), (iterative integration of the valencene synthase gene VSm, increased valencene yield to 254.67 mg/L)\n",
      "Progress: 1.85% completed\n",
      "Response for Row 38:\n",
      "Answer to Question 2: (deletion of gene IRA2, increased glycolytic flux), (deletion of gene IMA1, increased glycolytic flux), (overexpression of gene MALX, increased sugar consumption), (overexpression of maltose-related genes, increased maltotriose consumption)\n",
      "Progress: 1.90% completed\n",
      "Response for Row 39:\n",
      "Answer to Question 2: (deletion of gene tre49304, improved cellulase production), (deletion of gene tre120391, improved cellulase production), (deletion of gene tre123541, improved cellulase production), (overexpression of regulatory factors, enhanced cellulase production), (RNA interference of specific genes, increased endoglucanase activity), (sub-genomic engineering, improved microbial performance)\n",
      "Progress: 1.95% completed\n",
      "Response for Row 40:\n",
      "Answer to Question 2: (deletion of gene not mentioned, sanguinarine cytotoxicity), (expression of reticuline oxidases, increased sanguinarine biosynthesis), (expression of flavoproteins, enhanced enzyme function), (expression of cytochrome P450 enzymes, enhanced enzyme function), (SIMTeGES implementation, decoupling cell growth from product synthesis), (engineered yeast strains, production of sanguinarine at 448.64 mg L-1), (engineered yeast strains, biosynthesis of fluorinated sanguinarine)\n",
      "Progress: 2.00% completed\n",
      "Response for Row 41:\n",
      "Answer to Question 2: (deletion of gene ade2, off-target effects), (deletion of gene ade2, heteroclones presence), (deletion of gene trp1, successful knockout), (expression of gene ade2, red color phenotype)\n",
      "Progress: 2.05% completed\n",
      "Response for Row 42:\n",
      "Answer to Question 2: (HMO1 overexpression, enhanced ER stress tolerance), (HMO1 overexpression, improved protein folding efficiency), (BIO5 deletion, enhanced ER stress tolerance), (BIO5 deletion, improved protein folding efficiency), (HMO1 overexpression and BIO5 deletion, 2.14-fold increase in α-amylase production), (HMO1 overexpression and BIO5 deletion, 2.04-fold increase in cell density)\n",
      "Progress: 2.10% completed\n",
      "Response for Row 43:\n",
      "Answer to Question 2: (deletion of gene related to amino acid metabolism, consequence 1: decrease in amino acid metabolites), (overexpression of gene related to nucleotide metabolism, consequence 1: increase in nucleotide metabolites), (deletion of gene related to fructose and mannose metabolism, consequence 1: decreased fructose and mannose metabolism), (deletion of gene related to galactose metabolism, consequence 1: decreased galactose metabolism), (deletion of gene related to carboxylic acid metabolism, consequence 1: decrease in carboxylic acids), (deletion of gene related to protein synthesis and metabolism, consequence 1: decreased protein synthesis and metabolism)\n",
      "Progress: 2.15% completed\n",
      "Response for Row 44:\n",
      "Answer to Question 2: (deletion of gene ScAlaX, induced Ala- and Thr-to-Ser misincorporation), (deletion of gene hAlaX, induced Ala- and Thr-to-Ser misincorporation), (overexpression of gene hAlaX, impeded the decoding efficiency of consecutive Ser codons), (deletion of gene ScAlaX, gain in geneticin resistance), (deletion of gene ScAlaX, sensitivity to cycloheximide)\n",
      "Progress: 2.20% completed\n",
      "Response for Row 45:\n",
      "Answer to Question 2: (expression of rate-limited enzymes, enhancement of precursor flux), (suppression of competitive pathway flux, mitigation of cytotoxicity), (optimization of substrate utilization, refinement of fermentation process)\n",
      "Progress: 2.25% completed\n",
      "Response for Row 46:\n",
      "Answer to Question 2: (co-expression of gene ARO10, yield of 2-phenylethanol increased), (co-expression of gene yahK, yield of 2-phenylethanol increased), (introduction of amino acid substitution ARO10 I544W, 2-phenylethanol yield increased 4.1-fold), (expression of gene ARO10 I544W, 2-phenylethanol yield of 2.5 g/L achieved), (expression of gene ARO10 I544W, yield from glucose of 0.16 g/g achieved)\n",
      "Progress: 2.30% completed\n",
      "Response for Row 47:\n",
      "Answer to Question 2: (gene expression of multiple genes in glycolysis, increased β-carotene production), (gene expression of multiple genes in mevalonate, increased β-carotene production), (gene expression of multiple genes in β-carotene synthesis, increased β-carotene production), (decreased expression of ERG9, increased β-carotene production)\n",
      "Progress: 2.35% completed\n",
      "Response for Row 48:\n",
      "Answer to Question 2: (deletion of gene candidate genes, consequence total carotenoid production improvement), (deletion of gene eight out of ten candidate genes, consequence total carotenoid production 1.7 times compared with the parental strain)\n",
      "Progress: 2.40% completed\n",
      "Response for Row 49:\n",
      "Answer to Question 2: (overexpression of gene PDR5, latent-stationary phase post-diauxic shift), (overexpression of gene PDR5, mitochondrial regulation), (overexpression of gene PDR5, lipidomic remodeling), (overexpression of gene CIS1, mitochondrial regulation), (overexpression of gene CIS1, lipidomic remodeling), (overexpression of gene YGR035C, mitochondrial regulation), (overexpression of gene YGR035C, lipidomic remodeling), (knockout of gene PDR5, altered post-diauxic shift phenotype), (rescue of gene PDR5, restored post-diauxic shift phenotype), (CBD treatment, fatty acid production), (CBD treatment, glycerophospholipid production), (CBD treatment, phosphate-salvage indicators)\n",
      "Progress: 2.45% completed\n",
      "Response for Row 50:\n",
      "Answer to Question 2: (deletion of gene ortholog, consequence 1: functional redundancy in gene-protein-reaction rule), (overexpression of gene ortholog, consequence 1: enhanced metabolic capabilities), (deletion of gene ortholog, consequence 2: altered oxidative and non-oxidative parts of the pentose phosphate pathway)\n",
      "Progress: 2.50% completed\n",
      "Response for Row 51:\n",
      "Answer to Question 2: (deletion of gene GAL80, delayed response to low-temperature triggers), (deletion of gene GAL80, weak response at 30°C), (overexpression of gene Gal4p, suboptimal induction), (overexpression of gene Gal3Cp, rapid stringent high-level induction)\n",
      "Progress: 2.55% completed\n",
      "Response for Row 52:\n",
      "Answer to Question 2: (deletion of gene MKS1, increase in glycerol production), (deletion of gene MKS1, reduction in ethanol production), (deletion of gene MKS1, decrease in acetic acid levels), (deletion of gene RTG2, defect in wine fermentation speed)\n",
      "Progress: 2.60% completed\n",
      "Response for Row 53:\n",
      "Answer to Question 2: (expression of gene Cppks1, detrimental effect on growth of transformed yeast cells), (expression of gene Cppks1, production of phleichrome), (expression of gene encoding PPTase from A. nidulans, enzymatic activation of Cppks1 protein product), (cultivation at 20 °C and 25 °C, changes in pigmentation), (cultivation at 30 °C, no changes in pigmentation), (presence of phleichrome in cell extracts, verified by LC/MS/MS)\n",
      "Progress: 2.65% completed\n",
      "Response for Row 54:\n",
      "Answer to Question 2: (deletion of gene PRS1-PRS5, consequence reduced PRPP production), (deletion of gene PRS1-PRS5, consequence affects cell wall integrity signalling pathway), (expression of gene Prs5, consequence increased expression of FKS2), (expression of gene Prs5, consequence increased expression of Rlm1)\n",
      "Progress: 2.70% completed\n",
      "Response for Row 55:\n",
      "Answer to Question 2: (no gene expression/deletion mentioned, consequence: detection of long disome footprints outside regions of non-random elongation stalls), (no gene expression/deletion mentioned, consequence: identification of footprints linked to translation initiation and protein biosynthesis rates), (no gene expression/deletion mentioned, consequence: application of STE to investigate rapid changes to mRNA translation in yeast undergoing glucose depletion)\n",
      "Progress: 2.75% completed\n",
      "Response for Row 56:\n",
      "Answer to Question 2: (deletion of genes involved in the glyoxylate cycle, consequence 1), (overexpression of genes involved in gluconeogenesis, consequence 1), (overexpression of genes involved in the pentose phosphate pathway, consequence 1), (overexpression of genes involved in protein folding, consequence 1), (overexpression of genes involved in meiosis, consequence 1), (deletion of genes related to the biosynthesis of vitamin B6, consequence 1), (deletion of genes related to the biosynthesis of thiamin, consequence 1), (deletion of genes related to the biosynthesis of purine, consequence 1), (overexpression of genes related to protein folding, consequence 2), (overexpression of genes related to toxin efflux, consequence 2), (overexpression of genes related to cell wall remodeling, consequence 2), (overexpression of genes connected to the pentose phosphate pathway, consequence 3), (overexpression of genes connected to gluconeogenesis, consequence 3), (overexpression of genes connected to fatty acid utilization, consequence 3), (overexpression of HSP42, consequence 4), (overexpression of HSP26, consequence 4)\n",
      "Progress: 2.80% completed\n",
      "Response for Row 57:\n",
      "Answer to Question 2: (no gene overexpression/deletion mentioned, consequence 1: accurate prediction of translation initiation rate in Saccharomyces cerevisiae), (no gene overexpression/deletion mentioned, consequence 2: optimization of nucleotide sequence of any gene to achieve a specific target initiation rate)\n",
      "Progress: 2.85% completed\n",
      "Response for Row 58:\n",
      "Answer to Question 2: (deletion of gene prp45(1-169), consequence chromatin remodellers and modifiers as enriched category), (deletion of gene prp45(1-169), consequence strong growth defect with htz1), (deletion of gene prp45(1-169), consequence disproportionate effect on low copy number transcripts of intron-containing genes), (deletion of gene srb2 intron, consequence partial repair of growth phenotypes), (deletion of gene prp45(1-330), consequence synthetic growth defect with htz1Δ at 16°C), (deletion of gene prp45(1-330), consequence enhancement of pre-mRNA hyper-accumulation with htz1Δ)\n",
      "Progress: 2.90% completed\n",
      "Response for Row 59:\n",
      "Answer to Question 2: (znf1 deletion, hypersensitivity to isobutanol), (znf1 deletion, hypersensitivity to acid inhibitors), (overexpression of gene ZNF1, enhanced cell survival), (isobutanol induction, activation of ATP1-3 expression), (isobutanol induction, activation of COX6 expression), (Znf1 activation, PPP genes upregulation), (Znf1 activation, valine biosynthetic genes upregulation), (Znf1 activation, maintenance of NADP/NADPH redox balance)\n",
      "Progress: 2.95% completed\n",
      "Response for Row 60:\n",
      "Answer to Question 2: (overexpression of gene EcCAX3, consequence tolerance to different metal stresses in yeast), (overexpression of gene EcCAX3, consequence better tolerance to higher concentration of different metal ions in Arabidopsis plants), (overexpression of gene EcCAX3, consequence abundance of metal transporters transcripts under metal stress conditions), (overexpression of gene EcCAX3, consequence abundance of cation exchanger transporter transcripts under metal stress conditions), (overexpression of gene EcCAX3, consequence higher accumulation of macro-elements under different metal stress), (overexpression of gene EcCAX3, consequence higher accumulation of micro-elements under different metal stress)\n",
      "Progress: 3.00% completed\n",
      "Response for Row 61:\n",
      "Answer to Question 2: (overexpression of HAC1, enhanced stress signal transduction), (overexpression of HAC1, expanded signal output range of UPRE2m), (use of UPRE2m, increased recombinant protein production), (use of UPRE2m, increased natural product synthesis)\n",
      "Progress: 3.05% completed\n",
      "Response for Row 62:\n",
      "Answer to Question 2: (overexpression of gene AoErg24A, increased ergosterol content), (overexpression of gene AoErg24B, decreased ergosterol content)\n",
      "Progress: 3.10% completed\n",
      "Response for Row 63:\n",
      "Answer to Question 2: (expression of gene TDH3, consequence 1), (expression of gene TDH3, consequence 2), (mutation in promoter of gene TDH3, consequence 3), (mutation disrupting binding sites for transcription factors, consequence 4), (mutation disrupting TATA box, consequence 5)\n",
      "Progress: 3.15% completed\n",
      "Response for Row 64:\n",
      "Answer to Question 2: (overexpression of gene AcFMO, catalyzes the synthesis of alliin by S-allyl-l-cysteine)\n",
      "Progress: 3.20% completed\n",
      "Response for Row 65:\n",
      "Answer to Question 2: (deletion of gene fol3, increased OA titer), (deletion of gene abz1, increased OA titer), (deletion of gene abz2, increased OA titer), (regulation of gene fol3 by malonyl-CoA, improved sustainable OA production), (regulation of gene abz1 by malonyl-CoA, improved sustainable OA production), (regulation of gene abz2 by malonyl-CoA, improved sustainable OA production), (regulation of gene acc1 by malonyl-CoA, improved sustainable OA production)\n",
      "Progress: 3.25% completed\n",
      "Response for Row 66:\n",
      "Answer to Question 2: (expression of gene ARO8, highest normalized gene expression in SC1, SC4, SC5 samples), (expression of gene ARO9, highest normalized gene expression in SC1, SC4, SC5 samples), (expression of gene ADH2, no significant difference among Mp strains), (expression of gene ADH5, no expression in Mp strains except MP6), (expression of gene ADH2, higher in Sc strains compared to Mp strains), (expression of gene ADH5, higher in Sc strains compared to Mp strains)\n",
      "Progress: 3.30% completed\n",
      "Response for Row 67:\n",
      "Answer to Question 2: (expression of tRNA-Leu variant, defects in fluorescent protein production), (expression of tRNA-Leu variant, confirmed mistranslation with Leu), (expression of tRNA-Leu anticodon variant, dramatic reductions in growth), (expression of tRNA-Ala anticodon variant, more tolerable than Leu mis-incorporation)\n",
      "Progress: 3.35% completed\n",
      "Response for Row 68:\n",
      "Answer to Question 2: (overexpression of gene Cox20, improved respiratory growth of COX2W56R strain), (overexpression of gene Oxa1, improved respiratory growth of COX2W56R strain), (overexpression of gene Pse1, improved respiratory growth of COX2W56R strain), (deletion of gene MGR2, altered Cox2W56R import), (overexpression of gene MGR2, altered Cox2W56R import)\n",
      "Progress: 3.40% completed\n",
      "Response for Row 69:\n",
      "Answer to Question 2: (overexpression of gene NtWRKY70b, consequence increased levels of reactive oxygen species (ROS)), (overexpression of gene NtWRKY70b, consequence decreased H2S content), (disruption of gene NtWRKY70b, consequence decreased accumulation of ROS), (disruption of gene NtWRKY70b, consequence elevated concentration of H2S), (overexpression of gene NtWRKY70b, consequence upregulated expression of senescence-associated genes), (overexpression of gene NtWRKY70b, consequence upregulated expression of ROS biosynthesis-related genes NtRbohD and NtRbohE), (overexpression of gene NtWRKY70b, consequence inhibited expression of H2S biosynthesis-related genes NtDCD and NtCYSC1), (NtWRKY70b, consequence activated transcript levels of NtRbohD), (NtWRKY70b, consequence activated transcript levels of NtPPH), (NtWRKY70b, consequence repressed expression of NtDCD), (NtWRKY70b, consequence repressed expression of NtCYCS1)\n",
      "Progress: 3.45% completed\n",
      "Response for Row 70:\n",
      "Answer to Question 2: (expression of gene EcOSC, production of skeleton products 1-3), (expression of gene CYP716s, production of minor oxidation products 4-9), (expression of gene CYP716E60, production of 6β-OH derivatives), (presence of compound 9, inhibition of NO production), (presence of compound 9, reduction of IL-1β production), (presence of compound 9, reduction of IL-6 production), (presence of compounds 1, 2, 8, 9, increase in HepG2 cell survival rate)\n",
      "Progress: 3.50% completed\n",
      "Response for Row 71:\n",
      "Answer to Question 2:  (expression of gene ycf1, consequence oxidative stress), (expression of gene ycf1, consequence mutagenic stress resistance), (overexpression of gene involved in oxidative stress, consequence changes in gene expression patterns), (overexpression of gene involved in mutagenic stress, consequence changes in gene expression patterns)\n",
      "Progress: 3.55% completed\n",
      "Response for Row 72:\n",
      "Answer to Question 2: (expression of sugar transporter-encoding genes, higher expression in LBCM31), (expression of genes involved in iron metabolism, higher in LBCM109), (expression of genes promoting accumulation of sphingolipids, higher in LBCM109), (expression of genes promoting accumulation of phospholipids, higher in LBCM109), (expression of genes promoting accumulation of ergosterol, higher in LBCM109)\n",
      "Progress: 3.60% completed\n",
      "Response for Row 73:\n",
      "Answer to Question 2: (heterologous expression of fsr1, bostrycoidin biosynthesis), (heterologous expression of fsr2, bostrycoidin biosynthesis), (heterologous expression of fsr3, bostrycoidin biosynthesis), (heterologous expression of 6MSAS, 6-methylsalicylic acid biosynthesis), (overexpression of TGL4, no effect on polyketide production), (overexpression of AOX2, no effect on polyketide production)\n",
      "Progress: 3.65% completed\n",
      "Response for Row 74:\n",
      "Answer to Question 2: (expression of genes responsible for ammonium transport in Chlamydomonas reinhardtii, increased inorganic nitrogen availability), (expression of genes responsible for CO2 enrichment mechanism in Chlamydomonas reinhardtii, not the main reason for growth enhancement), (expression of genes responsible for glycolysis in Saccharomyces cerevisiae, increased energy availability for Chlamydomonas reinhardtii growth), (expression of genes responsible for stress responses in Saccharomyces cerevisiae, improved growth conditions for Chlamydomonas reinhardtii)\n",
      "Progress: 3.70% completed\n",
      "Response for Row 75:\n",
      "Answer to Question 2: (levan treatment, increased yeast viability), (levan treatment, prevented accumulation of reactive oxygen species), (levan treatment, prevented accumulation of malondialdehyde), (levan treatment, prevented accumulation of protein carbonyl content), (levan treatment, ameliorated reduced mitochondrial membrane potential), (levan treatment, ameliorated loss of plasma membrane integrity), (levan treatment, ameliorated externalization of phosphatidylserine), (levan treatment, modulated expression of genes in TORC1/Sch9 pathway), (levan treatment, extended CLS of sod2Δ mutant), (levan treatment, extended CLS of pep4Δ mutant), (levan treatment, extended CLS of pkh2Δ mutant), (levan treatment, extended CLS of rim15Δ mutant), (levan treatment, extended CLS of atg1 mutant), (levan treatment, extended CLS of ras2Δ mutant), (levan treatment, did not affect CLS of tor1Δ mutant), (levan treatment, did not affect CLS of sch9Δ mutant)\n",
      "Progress: 3.75% completed\n",
      "Response for Row 76:\n",
      "Answer to Question 2: (overexpression of gene SmMYB113, increased flower drop rate), (overexpression of gene SmMYB113, reduced fruit yield), (activation of gene SmACS1 by SmMYB113, increased ethylene biosynthesis), (activation of gene SmACS8 by SmMYB113, increased ethylene biosynthesis), (activation of gene SmACO4 by SmMYB113, increased ethylene biosynthesis), (interaction between SmMYB113 and SmERF38, enhanced activation of SmACS1 promoter)\n",
      "Progress: 3.80% completed\n",
      "Response for Row 77:\n",
      "Answer to Question 2: (no specific gene expression/deletion mentioned, no direct consequences provided)\n",
      "Progress: 3.85% completed\n",
      "Response for Row 78:\n",
      "Answer to Question 2: (deletion of gene PFK1, higher oxidative tolerance), (deletion of gene PFK1, slower growth in lag and exponential growth stages), (deletion of gene PFK1, higher growth in stationary stage), (deletion of gene PFK1, decelerated glucose consumption rate), (deletion of gene PFK1, decreased expression of HXT2), (deletion of gene PFK1, decreased expression of HXT5), (deletion of gene PFK1, decreased expression of HXT6), (deletion of gene PFK1, increased expression of ZWF1), (deletion of gene PFK1, higher residual glucose in medium), (deletion of gene PFK1, lower ethanol yield in early stage), (deletion of gene PFK1, decreased expression of PDC5), (deletion of gene PFK1, decreased expression of PDC6), (deletion of gene PFK1, lower interspecies competitiveness), (deletion of gene PFK1, increased ethanol production efficiency in stationary stage)\n",
      "Progress: 3.90% completed\n",
      "Response for Row 79:\n",
      "Answer to Question 2: (expression of BmXyn11A xylanase, improved GX hydrolysis), (expression of XylA β-xylosidase, improved GX hydrolysis), (expression of Agu115 α-methyl-glucuronidase, no additional growth benefit), (co-expression of complementary enzymes, boosted yeast growth), (co-expression of complementary enzymes, increased ethanol fermentation), (external enzyme supplementation, boosted yeast growth), (external enzyme supplementation, increased ethanol fermentation)\n",
      "Progress: 3.95% completed\n",
      "Response for Row 80:\n",
      "Answer to Question 2: (overexpression of shikimic acid pathway genes, increased precursor supply), (overexpression of chorismic acid pathway genes, increased precursor supply), (enhanced expression of CIT, increased intracellular acetyl-CoA content), (enhanced expression of MAC1/3, increased intracellular acetyl-CoA content), (enhanced expression of CTP1, increased intracellular acetyl-CoA content), (enhanced expression of YHM2, increased intracellular acetyl-CoA content), (enhanced expression of RtME, increased intracellular acetyl-CoA content), (enhanced expression of MDH, increased intracellular acetyl-CoA content), (synthesis of trans-cinnamic acid by ZmPAL, enhanced chrysin production), (synthesis of chrysin through ScCPR-EbFNSI-1 and PcFNSI, increased chrysin concentration)\n",
      "Progress: 4.00% completed\n",
      "Response for Row 81:\n",
      "Answer to Question 2: (deletion of gene rgt1, consequence ght3 transcriptional change), (deletion of gene rgt1, consequence ght4 transcriptional change), (deletion of gene rgt1, consequence ght6 transcriptional change), (deletion of gene rgt1, consequence ght7 transcriptional change), (overexpression of gene rgt1, consequence ght3 transcriptional change), (overexpression of gene rgt1, consequence ght4 transcriptional change), (overexpression of gene rgt1, consequence ght6 transcriptional change), (overexpression of gene rgt1, consequence ght7 transcriptional change)\n",
      "Progress: 4.05% completed\n",
      "Response for Row 82:\n",
      "Answer to Question 2: (deletion of gene ZWF1, reduced [NADPH] level), (deletion of gene ZWF1, disturbed growth on glucose), (deletion of gene ZWF1, disturbed growth on xylose), (replacement of gene TDH3 with GDP1, increased xylose consumption rate after glucose depletion), (replacement of gene TDH3 with GDP1, increased ethanol yield), (modulation of expression of ZWF1, optimized competitive equilibrium between glucose-xylose co-metabolism), (use of EMP instead of PPP, reduced wasteful metabolic cycle), (use of EMP instead of PPP, reduced CO2 release)\n",
      "Progress: 4.10% completed\n",
      "Response for Row 83:\n",
      "Answer to Question 2: (yeast supplementation, higher final body weights), (yeast supplementation, weight gains), (yeast supplementation, specific growth rates), (garlic supplementation, higher final body weights), (garlic supplementation, weight gains), (garlic supplementation, specific growth rates), (yeast + garlic supplementation, elevated levels of reduced glutathione), (yeast + garlic supplementation, activities of catalase), (yeast + garlic supplementation, activities of superoxide dismutase), (yeast supplementation, structural enhancements in liver), (garlic supplementation, structural enhancements in liver), (yeast supplementation, structural enhancements in intestine), (garlic supplementation, structural enhancements in intestine), (yeast + garlic supplementation, increased expression of igf-1), (yeast + garlic supplementation, increased expression of ghr1), (yeast + garlic supplementation, increased expression of il-10), (yeast + garlic supplementation, increased expression of lyz), (yeast + garlic supplementation, increased expression of hep), (yeast + garlic supplementation, improved resistance to A. sobria infection), (yeast + garlic supplementation, improved survival rates), (yeast + garlic supplementation, lower mortality)\n",
      "Progress: 4.15% completed\n",
      "Response for Row 84:\n",
      "Answer to Question 2: (deletion of gene tsa1, growth defect), (mutation in gene cdc19, reduced activity of Pyk1), (deletion of gene tsa1, depletion of glycolytic intermediates), (deletion of gene tsa1, restriction of amino acid synthesis pathways), (deletion of gene tsa1, decreased activity of fructose-bisphosphate aldolase 1)\n",
      "Progress: 4.20% completed\n",
      "Response for Row 85:\n",
      "Answer to Question 2: (deletion of gene expression by limonene, growth inhibition of Candida spp.), (deletion of gene expression by limonene, potentiation of Fluconazole effect), (deletion of gene expression by limonene, reduction in morphological transition from yeast to hyphae)\n",
      "Progress: 4.25% completed\n",
      "Response for Row 86:\n",
      "Answer to Question 2: (deletion of gene HcSE1-2, low yield of cucurbitacin IIa), (deletion of gene HcOSC1-6, low yield of cucurbitacin IIa), (deletion of gene HcCYP87D20, low yield of cucurbitacin IIa), (deletion of gene HcCYP81Q59, low yield of cucurbitacin IIa), (deletion of gene HcAT1, low yield of cucurbitacin IIa), (expression of gene HcSE1-2, increased production of 11-carbonyl-20β-hydroxy-Cuol), (expression of gene HcOSC1-6, increased production of 11-carbonyl-20β-hydroxy-Cuol), (expression of gene HcCYP87D20, increased production of 11-carbonyl-20β-hydroxy-Cuol), (expression of gene HcCYP81Q59, increased production of 11-carbonyl-20β-hydroxy-Cuol), (expression of gene HcAT1, increased production of 11-carbonyl-20β-hydroxy-Cuol)\n",
      "Progress: 4.30% completed\n",
      "Response for Row 87:\n",
      "Answer to Question 2:  (silencing of gene GaKAN2, decrease in single-cell trichomes), (silencing of gene GaKAN2, reduction in internodal distance), (expression of gene GaKAN2, regulation of trichomes development through GaERS), (expression of gene GaKAN2, enrichment in biosynthesis of secondary metabolites), (expression of gene GaKAN2, enrichment in plant hormone signal transduction pathways)\n",
      "Progress: 4.35% completed\n",
      "Response for Row 88:\n",
      "Answer to Question 2: (deletion of gene ino2, increase in lycopene yield), (deletion of gene ino4, increase in lycopene yield), (overexpression of gene ino2, no promotion of lycopene accumulation), (deletion of gene ino2, decrease in intracellular free fatty acids)\n",
      "Progress: 4.40% completed\n",
      "Response for Row 89:\n",
      "Answer to Question 2: (glycerol addition, enhanced CK production), (glycerol addition, up-regulation of MVA pathway genes), (glycerol addition, up-regulation of UDPG synthesis pathway genes), (glycerol utilization, increased terpene synthesis pathway flux), (glycerol utilization, facilitated production of glycosyl donors)\n",
      "Progress: 4.45% completed\n",
      "Response for Row 90:\n",
      "Answer to Question 2: (deletion of gene atpD, consequence reduced acid production), (deletion of gene eno, consequence reduced acid production)\n",
      "Progress: 4.50% completed\n",
      "Response for Row 91:\n",
      "Answer to Question 2: (expression of gene PwNAC31, consequence upregulation of drought-responsive genes DREB2A and ERD1), (expression of gene PwNAC31, consequence enhancement of seed vigor and germination rates in atnac072 Arabidopsis mutants), (expression of gene PwNAC31, consequence restoration of hypersensitive phenotype under drought stress in atnac072 Arabidopsis mutants), (interaction of protein PwNAC31 with DREB2A, consequence regulation of ERD1 expression), (interaction of protein PwNAC31 with ABF3, consequence not specified)\n",
      "Progress: 4.55% completed\n",
      "Response for Row 92:\n",
      "Answer to Question 2: (deletion of Ca2+, higher expression of SOD1, GSH1, GPX2 genes), (availability of Ca2+, higher ethanol output), (availability of Ca2+, higher glycerol output), (availability of Ca2+, lower trehalose synthesis capacity), (availability of Ca2+, higher intracellular reactive oxygen species content), (availability of Ca2+, lower intracellular malondialdehyde content), (availability of Ca2+, higher superoxide dismutase activity), (availability of Ca2+, higher catalase activity), (availability of Ca2+, higher glutathione content), (availability of Ca2+, higher expression of antioxidant-related genes except SOD1 and CTT1)\n",
      "Progress: 4.60% completed\n",
      "Response for Row 93:\n",
      "Answer to Question 2:  (deletion of sterol 22-desaturase gene, increased campesterol production to 6.6 mg/L), (expression of sterol Δ7-reductase, increased campesterol production to 88.3 mg/L), (enhanced expression of sterol Δ7-reductase, increased campesterol titer to 438.28 mg/L in shake flask), (enhanced expression of sterol Δ7-reductase, increased campesterol titer to 1.44 g/L in 5 L bioreactor)\n",
      "Progress: 4.65% completed\n",
      "Response for Row 94:\n",
      "Answer to Question 2: (ZNF598 overexpression, aborts stalled translation of mitochondrial outer membrane-associated mRNAs), (ZNF598 overexpression, removes faulty translation products), (ZNF598 overexpression, improves mitochondrial health), (ZNF598 overexpression, improves tissue health)\n",
      "Progress: 4.70% completed\n",
      "Response for Row 95:\n",
      "Answer to Question 2: (deletion of Gal4 DNA binding, decrease in transcription), (expression of Gal4 DNA binding, increase in transcription)\n",
      "Progress: 4.75% completed\n",
      "Response for Row 96:\n",
      "Answer to Question 2: (deletion/mutation of gene SLC12A5, reduced cell surface expression of KCC2), (deletion/mutation of gene SLC12A5, decreased activity of KCC2), (expression of gene SLC12A5, increased stability of KCC2 in primary rat neurons), (expression of gene SLC12A5, increased glycosylation of KCC2 in ER and Golgi)\n",
      "Progress: 4.80% completed\n",
      "Response for Row 97:\n",
      "Answer to Question 2:  (overexpression of gene xylA, consequence xylose utilization), (overexpression of gene Xltr1p, consequence enhance xylose utilization), (overexpression of gene ACS, consequence promote xylose utilization and ethanol yield), (overexpression of gene ADH, consequence promote xylose utilization and ethanol yield), (overexpression of gene TKL, consequence promote xylose utilization and ethanol yield)\n",
      "Progress: 4.85% completed\n",
      "Response for Row 98:\n",
      "Answer to Question 2: (over-expression of gene TaSnRK1α, increase in starch content), (over-expression of gene TaSnRK1α, enlargement of starch granules), (over-expression of gene TaSnRK1α, restoration of starch accumulation), (over-expression of gene TaSnRK1α, increase in AGPase activity)\n",
      "Progress: 4.90% completed\n",
      "Response for Row 99:\n",
      "Answer to Question 2: (deletion of genes involved in protein biosynthesis, moderate downregulation), (expression of secretory pathway genes, regulation seemingly detrimental to protein secretion), (overexpression of storage lipids, preparation for long-term survival), (overexpression of storage carbohydrates, preparation for long-term survival)\n",
      "Progress: 4.95% completed\n",
      "Response for Row 100:\n",
      "Answer to Question 2: (expression of pathway genes, improved pathway flux), (expression of pathway genes, doubled production titers of astaxanthin)\n",
      "Progress: 5.00% completed\n",
      "Response for Row 101:\n",
      "Answer to Question 2: (expression of XYL1 from Scheffersomyces xylosifermentans, improved anaerobic growth on xylose), (expression of XYL genes from Scheffersomyces stipitis, no improved anaerobic growth on xylose), (high aeration, decrease in enzyme activity related to xylose metabolism), (high aeration, decrease in gene expression related to xylose metabolism), (oxygen levels, affect xylose metabolism balance)\n",
      "Progress: 5.05% completed\n",
      "Response for Row 102:\n",
      "Answer to Question 2: (over-expression of TaHXK7-1A, increased O₂·⁻ content), (over-expression of TaHXK7-1A, increased malondialdehyde content), (over-expression of TaHXK7-1A, up-regulated RBOHs genes), (over-expression of TaHXK7-1A, inhibited growth of Arabidopsis seedlings), (over-expression of TaHXK7-1A, increased ROS accumulation), (gene silencing of TaHXK7-1, decreased O₂·⁻ content), (gene silencing of TaHXK7-1, decreased malondialdehyde content), (gene silencing of TaHXK7-1, down-regulated RBOHs genes), (gene silencing of TaHXK7-1, improved drought resistance of wheat)\n",
      "Progress: 5.10% completed\n",
      "Response for Row 103:\n",
      "Answer to Question 2: (knockdown of gene Eo-eIF5A, inhibited +1 PRF of the η-tubulin gene), (expression of gene Eo-eIF5A<sup>H</sup>, promoted +1 PRF at the canonical AAA_UAA frameshifting site of Euplotes)\n",
      "Progress: 5.15% completed\n",
      "Response for Row 104:\n",
      "Answer to Question 2: (deletion of gene OCA6, increased secreted laccase activity), (deletion of gene PMT1, increased secreted laccase activity), (deletion of gene PMT2, increased secreted laccase activity), (deletion of gene SKI3, increased secreted laccase activity), (deletion of gene SCJ1, decreased secreted laccase activity), (overexpression of gene SCJ1, detrimental effect on laccase secretion)\n",
      "Progress: 5.20% completed\n",
      "Response for Row 105:\n",
      "Answer to Question 2: (deletion of gene CGB, hyper-susceptibility to Pseudomonas syringae), (deletion of gene CGB, loss of tRNA thiolation), (deletion of gene CTU2, loss of tRNA thiolation), (loss of tRNA thiolation, compromised transcriptome reprogramming during immune responses), (loss of tRNA thiolation, compromised proteome reprogramming during immune responses), (reduced translation of gene NPR1, compromised salicylic acid signaling)\n",
      "Progress: 5.25% completed\n",
      "Response for Row 106:\n",
      "Answer to Question 2: (methylation of histone H3 on lysine-4 residue, facilitation of transcriptional output of the genome), (methylation of histone H3 on lysine-4 residue, establishment of cellular identity)\n",
      "Progress: 5.30% completed\n",
      "Response for Row 107:\n",
      "Answer to Question 2: (deletion of gene RPG, consequence higher oxidant sensitivity), (deletion of gene RPG, consequence increased H2O2 concentrations), (deletion of gene RPG, consequence elevated catalase activity), (deletion of gene RPG, consequence higher SOD activities), (deletion of gene RPG, consequence increased transcription of CTT1), (deletion of gene RPG, consequence increased transcription of CTA1), (deletion of gene RPG, consequence enhanced ribosomes' ability to translate mRNAs associated with oxidative stress-related genes), (deletion of gene RPG, consequence cleavage of 25S rRNA), (deletion of gene RPG, consequence altered translation efficiency of CTT1 mRNA), (deletion of gene RPG, consequence altered translation efficiency of CTA1 mRNA)\n",
      "Progress: 5.35% completed\n",
      "Response for Row 108:\n",
      "Answer to Question 2: (expression of gene xylA, increased xylose utilization), (expression of homologous genes related to xylose utilization, increased xylose utilization), (increase in expression level of pathways associated with sugar metabolism, improved sugar utilization), (initial increase followed by decrease in genes related to glycerol lipid metabolism, optimized metabolic pathway)\n",
      "Progress: 5.40% completed\n",
      "Response for Row 109:\n",
      "Answer to Question 2: (deletion of genes, genome restructuring), (overexpression of BnaGPAT9-C1, increased erucic acid content)\n",
      "Progress: 5.45% completed\n",
      "Response for Row 110:\n",
      "Answer to Question 2: (depletion of gene ASF1, decrease in replicative histones at protein level), (depletion of gene ASF1, decrease in replicative histones at RNA level), (depletion of gene ASF1, increase in nascent/newly synthesized RNAs), (depletion of gene ASF1, 3' end processing defect of replicative histone RNAs), (depletion of gene ASF1, increase in pre-mRNAs/unprocessed transcripts), (depletion of gene ASF1, production defect of replicative histone RNAs)\n",
      "Progress: 5.50% completed\n",
      "Response for Row 111:\n",
      "Answer to Question 2: (no specific gene expression/deletion mentioned, general discussion on RNA production and adaptation in yeast Saccharomyces cerevisiae)\n",
      "Progress: 5.55% completed\n",
      "Response for Row 112:\n",
      "Answer to Question 2: (expression of AnanC4H, increase in lignin content), (expression of AnanC4H, increase in number of lignified cells in stems)\n",
      "Progress: 5.60% completed\n",
      "Response for Row 113:\n",
      "Answer to Question 2: (deletion of gene related to lactate metabolism, consequence differential gene expression), (deletion of gene related to iron uptake, consequence differential gene expression), (deletion of gene related to cell wall structure, consequence differential gene expression)\n",
      "Progress: 5.65% completed\n",
      "Response for Row 114:\n",
      "Answer to Question 2: (deletion of gene TCF25, mild negative effects on cell growth), (deletion of gene TCF25, post-translational protein degradation by the proteasome), (aging leads to reduced mRNA levels of RQC genes, reduced mRNA levels in specific tissues)\n",
      "Progress: 5.70% completed\n",
      "Response for Row 115:\n",
      "Answer to Question 2: (deletion of translation initiation, consequence NAC and Zuotin can still cross-link to each other), (presence of NAC, consequence central globular domain of NAC is modestly shifted), (presence of Hsp70, consequence Hsp70 can situate with peptide binding site at the tunnel exit), (presence of Zuotin, consequence J-domain of Zuotin is appropriately positioned to drive stabilization of nascent chain binding)\n",
      "Progress: 5.75% completed\n",
      "Response for Row 116:\n",
      "Answer to Question 2: (deletion of gene Zn2+, consequence oxidation of thioredoxins in vivo), (deletion of gene Zn2+, consequence induction of oxidative stress), (deletion of gene Zn2+, consequence interaction with multiple metals including Mn2+ and Cu2+), (expression of gene thiolutin, consequence direct inhibition of RNA polymerase II transcription initiation in vitro), (expression of gene thiolutin, consequence pause prone, defective elongation in vitro), (expression of gene thiolutin, consequence widespread Pol II occupancy changes in vivo), (expression of gene thiolutin, consequence Tor pathway inhibition in vivo), (expression of gene thiolutin, consequence stress induction in vivo)\n",
      "Progress: 5.80% completed\n",
      "Response for Row 117:\n",
      "Answer to Question 2: (deletion of gene Cncat, impact on H2O2-induced oxidative stress), (over-expression of gene CnCat, improvement in oxidative resistance), (over-expression of gene CnCat, reduction in intracellular ROS accumulation)\n",
      "Progress: 5.85% completed\n",
      "Response for Row 118:\n",
      "Answer to Question 2: (deletion of gene CRS1, reduced synthesis of CysSSH), (deletion of gene CRS1, reduced chronological aging), (deletion of gene CRS1, increased endoplasmic reticulum stress response), (deletion of gene CRS1, impaired mitochondrial bioenergetics)\n",
      "Progress: 5.90% completed\n",
      "Response for Row 119:\n",
      "Answer to Question 2: (deletion of gene Mex67, consequence Dbp5 recruitment to mRNA is abolished), (overexpression of gene Dbp5, consequence dominant-negative effect on tRNA export), (binding of gene Dbp5 to Gle1, consequence required for tRNA export), (interaction of gene Dbp5 with tRNA, consequence does not activate Dbp5 ATPase activity alone), (interaction of gene Dbp5 with Gle1 and tRNA, consequence fully activates Dbp5 ATPase activity)\n",
      "Progress: 5.95% completed\n",
      "Response for Row 120:\n",
      "Answer to Question 2: (expression of gene green fluorescent protein, consequence carbon-source dependent expression), (expression of gene green fluorescent protein, consequence inulin favorable carbon source), (expression of gene green fluorescent protein, consequence high correlation between in vivo expression and sucrose condition)\n",
      "Progress: 6.00% completed\n",
      "Response for Row 121:\n",
      "Answer to Question 2: (deletion of gene SAM1, altered chromosome stability), (deletion of gene SAM1, decreased AdoMet concentration), (deletion of gene SAM2, altered chromosome stability), (deletion of gene SAM2, increased AdoMet concentration), (deletion of gene SAM1, increased sensitivity or resistance to azoles), (deletion of gene SAM2, increased sensitivity or resistance to azoles), (deletion of gene SAM1, increased sensitivity or resistance to cisplatin), (deletion of gene SAM2, increased sensitivity or resistance to cisplatin), (deletion of gene SAM1, increased sensitivity or resistance to oxidative stress), (deletion of gene SAM2, increased sensitivity or resistance to oxidative stress), (deletion of gene SAM1, perturbations in arginine biosynthesis), (deletion of gene SAM2, perturbations in arginine biosynthesis), (deletion of gene SAM1, altered sensitivity to DNA synthesis inhibitors), (deletion of gene SAM2, altered sensitivity to DNA synthesis inhibitors), (deletion of gene SAM1, altered response to tamoxifen), (deletion of gene SAM2, altered response to tamoxifen)\n",
      "Progress: 6.05% completed\n",
      "Response for Row 122:\n",
      "Answer to Question 2: (deletion of RNA polymerase II activity, altered initiation efficiency), (deletion of RNA polymerase II activity, selective modulation of preference for initiating nucleotide), (overexpression of guanosine-5'-triphosphate, increased initiation efficiency at first five transcript positions), (overexpression of cytosine-5'-triphosphate, increased initiation efficiency at second position), (overexpression of uridine-5'-triphosphate, increased initiation efficiency at second position)\n",
      "Progress: 6.10% completed\n",
      "Response for Row 123:\n",
      "Answer to Question 2: (no specific gene expression/deletion mentioned, general review of meiotic gene expression control)\n",
      "Progress: 6.15% completed\n",
      "Response for Row 124:\n",
      "Answer to Question 2: (expression of gene PgfB, consequence heterologous production of nerol), (expression of gene homologous protein of PgfB, consequence heterologous production of nerol)\n",
      "Progress: 6.20% completed\n",
      "Response for Row 125:\n",
      "Answer to Question 2: (expression of gene HAP2, activation of hybrid promoter), (overexpression of promoter phy47, 2.93-fold higher GFP expression levels than GAP), (application of UAS<sub>E</sub> and URS<sub>D</sub> to promoter GAP, 34% increase in expression level), (application of UAS<sub>E</sub> and URS<sub>D</sub> to promoter ENO1, 18% decrease in expression level), (construction of artificial promoters with regulatory sequence, 61% expansion of expression range), (use of hybrid promoter variants, higher titers of pinene and rHuPH20 biosynthesis)\n",
      "Progress: 6.25% completed\n",
      "Response for Row 126:\n",
      "Answer to Question 2: (expression of gene Lyz-1, production of N-acetylmuramoyl-L-alanine amidase), (expression of gene Lyz-2, production of D-alanyl-D-alanine carboxypeptidase), (expression of gene Lyz1, antimicrobial activity against Micrococcus luteus), (expression of gene Lyz1, antimicrobial activity against Salmonella typhimurium), (expression of gene Lyz1, higher concentration required to control Saccharomyces cerevisiae compared to Aspergillus oryzae), (expression of gene Lyz1, lower concentration required to disintegrate Gram-positive bacteria cells compared to Gram-negative bacteria cells)\n",
      "Progress: 6.30% completed\n",
      "Response for Row 127:\n",
      "Answer to Question 2: (cloning of polypeptide-p gene, production of recombinant secretion protein), (transformation into Saccharomyces boulardii, reduction in blood glucose level)\n",
      "Progress: 6.35% completed\n",
      "Response for Row 128:\n",
      "Answer to Question 2: (expression of gene P450_cur, consequence increased 7β-hydroxylation of androstadienedione), (expression of gene P450_cur, consequence increased 7β-hydroxylation of dehydroepiandrosterone), (expression of gene P450_cur, consequence increased 7β-hydroxylation of androstenediol), (expression of gene P450_cur, consequence increased 7α-hydroxylation of 1-dehydrotestosterone), (expression of gene CPR, consequence enhanced functional activity of P450_cur)\n",
      "Progress: 6.40% completed\n",
      "Response for Row 129:\n",
      "Answer to Question 2: (deletion of gene CCL5, consequence downregulation in monocytes), (deletion of gene IL6, consequence downregulation in monocytes), (deletion of gene NFKB, consequence downregulation in neutrophils)\n",
      "Progress: 6.45% completed\n",
      "Response for Row 130:\n",
      "Answer to Question 2: (deletion of gene Rds2, consequence 1), (expression of gene Rds2, consequence 2), (deletion of gene Gsm1, consequence 3), (expression of gene Gsm1, consequence 4), (binding of protein Rds2, consequence 5), (binding of protein Gsm1, consequence 6), (cooperative binding of proteins Gsm1 and Rds2, consequence 7), (binding of protein Ert1, consequence 8)\n",
      "Progress: 6.50% completed\n",
      "Response for Row 131:\n",
      "Answer to Question 2: (deletion of gene GPA1, consequence slower cell-cycle progression), (overexpression of gene GPA1, consequence increased mating efficiency)\n",
      "Progress: 6.55% completed\n",
      "Response for Row 132:\n",
      "Answer to Question 2: ( deletion of gene THI3, decreased isoamyl alcohol production), ( deletion of gene THI3, lowered decarboxylase activity), ( deletion of gene THI3, reduced thiamine levels), ( deletion of gene THI3, reduced thiamine diphosphate (TPP) levels), ( deletion of gene THI3, decreased isoamyl alcohol production under low-thiamine conditions)\n",
      "Progress: 6.60% completed\n",
      "Response for Row 133:\n",
      "Answer to Question 2: (CRISPR-edited defects in PtrSWRA-PtrAREB1-2 pentamer, impaired drought tolerance), (elevated expression of PtrHox2, improved drought tolerance), (elevated expression of PtrHox46, improved drought tolerance), (elevated expression of PtrHox52, improved drought tolerance), (elevated expression of PtrHox2, improved growth), (elevated expression of PtrHox46, improved growth), (elevated expression of PtrHox52, improved growth)\n",
      "Progress: 6.65% completed\n",
      "Response for Row 134:\n",
      "Answer to Question 2: (expression of gene sHSP-HB8, consequence increased cell viability at 42°C), (expression of gene sHSP-HB8, consequence 17.2% increase in cell density), (expression of gene HSP12, consequence increased cell viability at 42°C), (expression of gene HSP12, consequence 19.8% increase in cell density), (expression of gene HSP26, consequence increased cell viability at 42°C), (expression of gene HSP30, consequence increased cell viability at 42°C), (expression of gene HSP42, consequence increased cell viability at 42°C), (expression of gene ibpa-MB4, consequence increased cell viability at 42°C), (expression of gene PK, consequence enhanced protein quality at higher temperatures), (expression of gene MDH, consequence enhanced protein quality at higher temperatures)\n",
      "Progress: 6.70% completed\n",
      "Response for Row 135:\n",
      "Answer to Question 2: (integration of 2μ sequence-based yeast episomal plasmid, increased nerolidol production), (integration of yeast integrative plasmid, increased diversity in sesquiterpene productivities), (increased gene copy number, increased nerolidol production), (increased gene copy number, increased valencene production), (sesquiterpene synthase expression level increase, increased nerolidol production), (sesquiterpene synthase expression level increase, increased valencene production)\n",
      "Progress: 6.75% completed\n",
      "Response for Row 136:\n",
      "Answer to Question 2: (gene upregulation associated with glycolysis, elevated levels of fermentation products), (gene upregulation associated with energy metabolism, elevated levels of fermentation products), (reduction in glycolytic activity, diminished ATP supply), (formation of flocs, protection against toxic chemicals), (variations in the nucleotide sequences of the flocculin genes and their regulators, distinct flocculation traits)\n",
      "Progress: 6.80% completed\n",
      "Response for Row 137:\n",
      "Answer to Question 2: (expression of gene udh, consequence increased mucic acid production), (expression of gene udh, consequence improved productivity up to 0.83 g/L-h), (expression of gene udh, consequence theoretical yield of 0.18 g/g using pectin-containing citrus peel waste), (utilization of xylose as co-substrate, consequence overcoming NAD+ dependency for pectin to mucic acid conversion)\n",
      "Progress: 6.85% completed\n",
      "Response for Row 138:\n",
      "Answer to Question 2: (deletion of TORC1, consequence 1: reduced cellular anabolism), (deletion of TORC1, consequence 2: reduced cellular proliferation), (overexpression of Sfp1, consequence 1: suppression of temperature-sensitive TORC1 mutant phenotype), (phosphorylation of Sfp1 by TORC1, consequence 1: protection of Sfp1 from proteasomal degradation), (positive regulation of genes involved in ribosome production by Sfp1, consequence 1: increased ribosome biosynthesis), (interaction of Ifh1 with Fhl1, consequence 1: regulation of Ifh1 nuclear localization in response to nutrient levels), (regulation of Ifh1 nuclear localization by TORC1 and Sfp1, consequence 1: modulation of ribosome biosynthesis genes expression)\n",
      "Progress: 6.90% completed\n",
      "Response for Row 139:\n",
      "Answer to Question 2: (deletion of mRNA by nuclear exosome-NNS pathway, consequence 1: suppression of mRNAs involved in protein production), (deletion of mRNA by nuclear exosome-NNS pathway, consequence 2: suppression of mRNAs involved in cellular organization), (deletion of mRNA by nuclear exosome-NNS pathway, consequence 3: suppression of mRNAs involved in metabolism)\n",
      "Progress: 6.95% completed\n",
      "Response for Row 140:\n",
      "Answer to Question 2: (overexpression of CALCOCO2, promotion of BVDV propagation), (knockdown of CALCOCO2, inhibition of BVDV replication), (expression of CALCOCO2, inhibition of IFN-I production), (expression of CALCOCO2, inhibition of IFN-stimulated gene production), (expression of CALCOCO2, reduction of IRF3 protein levels), (interaction of CALCOCO2 with IRF3, promotion of Npro-mediated ubiquitination degradation of IRF3)\n",
      "Progress: 7.00% completed\n",
      "Response for Row 141:\n",
      "Answer to Question 2: (deletion of gene not specified, accumulation of toxic pyrophosphate intermediates), (expression of MVA pathway enzymes in mitochondria, increased terpenoid production), (expression of MVA pathway enzymes in mitochondria, accumulation of mono- and pyrophosphate compounds), (mitigation of mono- and pyrophosphate compound accumulation, improved squalene titer)\n",
      "Progress: 7.05% completed\n",
      "Response for Row 142:\n",
      "Answer to Question 2:  (upregulation of gene SDS3, improved biocontrol efficacy), (upregulation of gene DAL81, improved biocontrol efficacy), (upregulation of gene DSE1, improved biocontrol efficacy), (upregulation of gene SNF5, improved biocontrol efficacy), (upregulation of gene SUN41, improved biocontrol efficacy), (upregulation of gene FLO8, improved biocontrol efficacy), (upregulation of gene HOP1, improved biocontrol efficacy)\n",
      "Progress: 7.10% completed\n",
      "Response for Row 143:\n",
      "Answer to Question 2: (deletion of gene eGFP, repression efficiency from 62% to 177%), (overexpression of gene eGFP, activation efficiency from 154% to 320%)\n",
      "Progress: 7.15% completed\n",
      "Response for Row 144:\n",
      "Answer to Question 2: (expression of Wolbachia effector proteins in S. cerevisiae, consequence 1), (expression of Wolbachia effector proteins in S. cerevisiae, consequence 2), (expression of CI-related proteins in S. cerevisiae, consequence 3)\n",
      "Progress: 7.20% completed\n",
      "Response for Row 145:\n",
      "Answer to Question 2: (expression of genes involved in pullulan synthesis, increased pullulan production), (expression of genes involved in melanin synthesis, increased melanin production), (expression of genes involved in polymalic acid synthesis, increased polymalic acid production)\n",
      "Progress: 7.25% completed\n",
      "Response for Row 146:\n",
      "Answer to Question 2: (overexpression of RAS2S225A, mislocalisation), (overexpression of RAS2S225A, constitutive activation), (overexpression of RAS2S225A, slow growth), (overexpression of RAS2S225A, loss of respiration), (overexpression of RAS2S225A, reduced stress response), (overexpression of RAS2S225A, quiescence), (overexpression of RAS2S225E, mislocalisation), (overexpression of RAS2S225E, constitutive activation), (overexpression of RAS2S225E, slow growth), (overexpression of RAS2S225E, loss of respiration), (overexpression of RAS2S225E, reduced stress response), (overexpression of RAS2S225E, quiescence), (deletion of CUP9, rescue of quiescence), (addition of peptone, rescue of quiescence)\n",
      "Progress: 7.30% completed\n",
      "Response for Row 147:\n",
      "Answer to Question 2: (deletion of gene Fpt1, reduction of RNAPIII eviction), (deletion of gene Fpt1, impairment of shutdown of ribosome biogenesis genes upon nutrient perturbation)\n",
      "Progress: 7.35% completed\n",
      "Response for Row 148:\n",
      "Answer to Question 2: (deletion of gene involved in Cd tolerance, consequence 1), (deletion of gene involved in Zn tolerance, consequence 2), (deletion of gene involved in Fe tolerance, consequence 3), (deletion of gene involved in Mn tolerance, consequence 4), (deletion of gene involved in B tolerance, consequence 5), (deletion of gene involved in Cu tolerance, consequence 6)\n",
      "Progress: 7.40% completed\n",
      "Response for Row 149:\n",
      "Answer to Question 2: (expression of GPCR signaling pathway, reshaped yeast galactose regulon), (expression of synthetic transcription factor, coupled GAL system with GPCR signaling pathway), (modulation of key components of GPCR signaling cascade, refabricated dynamic range, sensitivity, and response time of the GAL system), (expression of α factor, achieved quorum-sensing behaviors), (repurposing of GAL system, made responsive to heat shock)\n",
      "Progress: 7.45% completed\n",
      "Response for Row 150:\n",
      "Answer to Question 2: (S-adenosyl-l-methionine usage by S. cerevisiae, counteracts acid production of L. plantarum), (S. cerevisiae impacts gene expression, upregulates protein synthesis pathways), (S. cerevisiae impacts gene expression, upregulates cell growth pathways), (S. cerevisiae impacts gene expression, activates ribosome pathway), (S. cerevisiae impacts gene expression, activates SNARE interactions pathway), (S. cerevisiae impacts gene expression, activates basal transcription factors pathway), (S. cerevisiae impacts gene expression, activates MAPK signaling pathway)\n",
      "Progress: 7.50% completed\n",
      "Response for Row 151:\n",
      "Answer to Question 2: (deletion of subcellular localization signal of gene T16H, no effect on gene expression levels), (expression of peroxisome-localized T16H, highest transformation efficiency), (expression of T16H in different organelles, varying conversion rates from tabersonine to 16-hydroxy tabersonine)\n",
      "Progress: 7.55% completed\n",
      "Response for Row 152:\n",
      "Answer to Question 2: (expression of gene CENP-A, consequence reduced histone humanization efficiency), (expression of gene CENP-A, consequence toxic to yeast), (expression of gene CENP-A, consequence disrupted cell cycle progression), (expression of gene CENP-A, consequence disrupted kinetochore function in wild-type cells), (deletion of gene subunits of 3 conserved chromatin remodeling complexes, consequence suppressed CENP-A toxicity)\n",
      "Progress: 7.60% completed\n",
      "Response for Row 153:\n",
      "Answer to Question 2: (amplification of gene GAP1, higher RNA abundance), (amplification of gene GAP1, reduced effect at proteome level), (amplification of gene GAP1, involvement of post-transcriptional dosage compensation), (genes with different translational efficiency, enrichment for upstream open reading frames), (genes with different translational efficiency, enrichment for RNA binding sites for SSD1), (genes with lower protein expression efficiency, enrichment for genes encoding proteins in protein complexes)\n",
      "Progress: 7.65% completed\n",
      "Response for Row 154:\n",
      "Answer to Question 2: (deletion of interleukin-1β transcription, effect on distal gut inflammation), (deletion of interleukin-6 mRNA, effect on proximal gut inflammation), (no change in TNF-α mRNA, no effect on inflammation)\n",
      "Progress: 7.70% completed\n",
      "Response for Row 155:\n",
      "Answer to Question 2: (deletion of gene tHMG1, decrease in carotenoid productivity), (deletion of gene ERG8, decrease in carotenoid productivity), (overexpression of gene tHMG1, increase in carotenoid productivity), (overexpression of gene ERG8, increase in carotenoid productivity)\n",
      "Progress: 7.75% completed\n",
      "Response for Row 156:\n",
      "Answer to Question 2: (expression of gene BRCA2, consequence increased radiation resistance), (expression of gene BRCA2, consequence increased DSB repair frequency), (deletion of gene Rad52, consequence reduced DSB repair efficiency), (expression of gene BRCA2, consequence improved protection against ionizing radiation by at least 1.93-fold), (expression of gene BRCA2, consequence increased repair frequency by 6.1-fold)\n",
      "Progress: 7.80% completed\n",
      "Response for Row 157:\n",
      "Answer to Question 2: (deletion of gene involved in cell membrane permeability, consequence 1: reduction in permeability and integrity of cell membrane), (overexpression of gene involved in ergosterol synthesis, consequence 1: increase in ergosterol proportion in cell membrane), (overexpression of gene involved in unsaturated fatty acid synthesis, consequence 1: increase in unsaturated fatty acids proportion in cell membrane), (overexpression of DEGs in ergosterol synthesis pathway, consequence 1: increase in ergosterol synthesis), (overexpression of DEGs in linoleic acid synthesis pathway, consequence 1: increase in linoleic acid synthesis), (activation of cell wall integrity (CWI) pathway, consequence 1: enhanced cell wall integrity), (activation of high osmolarity glycerol mitogen-activated protein kinase (HOG-MAPK) pathway, consequence 1: enhanced osmotic stress response)\n",
      "Progress: 7.85% completed\n",
      "Response for Row 158:\n",
      "Answer to Question 2: (deletion of gene rer2, growth defect at higher temperature), (expression of DHDDS-like protein, production of medium-chain C55 product), (expression of DHDDS-like protein and NgBR homologue, production of longer-chain products)\n",
      "Progress: 7.90% completed\n",
      "Response for Row 159:\n",
      "Answer to Question 2: (hypomodification of tRNA core, increased translational errors), (modifications of tRNA anticodon loop, modulate accuracy), (tRNA core modifications, stabilize tRNA structure), (stabilizing tRNA structure, avoid degradation), (degradation of tRNA, increased at high growth temperatures)\n",
      "Progress: 7.95% completed\n",
      "Response for Row 160:\n",
      "Answer to Question 2:  (introduction of mysA, consequence 1), (introduction of mysB, consequence 1), (introduction of mysC, consequence 1), (introduction of mysD, consequence 1), (introduction of mysE, consequence 1), (overexpression of GND1, consequence 1), (overexpression of ZWF1, consequence 1), (introduction of xylose utilization pathway, consequence 1), (strengthening promoters, consequence 1), (increasing gene copies, consequence 1)\n",
      "Progress: 8.00% completed\n",
      "Response for Row 161:\n",
      "Answer to Question 2: (CRISPR-Cpf1 genome-editing system in Saccharomyces cerevisiae, plasmid curing efficiency 96 ± 4%), (transforming only one crRNA into S. cerevisiae, curing efficiency about 70%), (CRISPR-Cas editing technology, produces traceless products without foreign genes)\n",
      "Progress: 8.05% completed\n",
      "Response for Row 162:\n",
      "Answer to Question 2: (deletion of gene intron splicing efficiency, consequence reduced splicing), (overexpression of gene intron splicing efficiency, consequence increased use of cryptic splice sites), (mutation in gene intron splicing efficiency, consequence formation of RNA structures that pair N50 bases)\n",
      "Progress: 8.10% completed\n",
      "Response for Row 163:\n",
      "Answer to Question 2: (deletion of gene xxx, disruption at high efficiency), (expression of heterologous protein, integration from a safe chromosomal harbour site)\n",
      "Progress: 8.15% completed\n",
      "Response for Row 164:\n",
      "Answer to Question 2: (expression of foreign genes, impaired by endogenous factors), (expression of foreign genes, facilitated by promoter engineering), (expression of foreign genes, improved by codon optimization), (expression of foreign genes, enhanced by signal peptide fusion), (gene disruption, performed by CRISPR/Cas), (foreign gene insertion, performed by CRISPR/Cas), (site-directed mutation, performed by CRISPR/Cas)\n",
      "Progress: 8.20% completed\n",
      "Response for Row 165:\n",
      "Answer to Question 2: (expression of biosynthetic genes, production of Orsellinic acid), (expression of biosynthetic genes, production of Colletochlorins), (expression of biosynthetic genes, production of Colletorins)\n",
      "Progress: 8.25% completed\n",
      "Response for Row 166:\n",
      "Answer to Question 2: (overexpression of gene OsGSTZ4, conferred cellular Cd tolerance), (overexpression of gene OsGSTZ4, improved plant growth), (overexpression of gene OsGSTZ4, attenuated Cd-induced toxicity), (overexpression of gene OsGSTZ4, accumulated more Cd in roots), (induction of gene OsGSTZ4, CHH hypermethylation), (induction of gene OsGSTZ4, detection of 24-nt siRNAs), (loss of function of gene osdrm2, compromised CHH methylation), (defective in gene osrdr2i, compromised CHH methylation), (promoter deletion of gene OsGSTZ4, essential for proper expression)\n",
      "Progress: 8.30% completed\n",
      "Response for Row 167:\n",
      "Answer to Question 2: (deletion of gene GCN4, heightened mitochondrial activity), (deletion of gene GCN4, increase in reactive oxygen species (ROS) accumulation), (deletion of gene GCN4, reduced ethanol production), (deletion of gene GCN4, reduction in trehalose), (deletion of gene GCN4, disparities in free amino acids), (deletion of gene GCN4, disrupted cellular homeostasis), (deletion of gene GCN4, shortened lifespan)\n",
      "Progress: 8.35% completed\n",
      "Response for Row 168:\n",
      "Answer to Question 2: (deletion of inflammatory response genes, increased plasminogen expression), (deletion of inflammatory response genes, increased glutathione metabolism genes expression)\n",
      "Progress: 8.40% completed\n",
      "Response for Row 169:\n",
      "Answer to Question 2: (overexpression of gene ERG20, increase in cembratriene-ol titer by 1.84-fold), (overexpression of gene cbts*Δp, increase in cembratriene-ol titer by 1.84-fold), (overexpression of gene ERG20, increase in cembratriene-ol titer by 2.22-fold), (overexpression of gene cbts*Δp, increase in cembratriene-ol titer by 2.22-fold), (overexpression of gene ERG9, increase in cembratriene-ol titer by 1.80-fold), (overexpression of gene cbts*Δp, increase in cembratriene-ol titer by 26.1-fold), (overexpression of gene ggpps, increase in cembratriene-ol titer by 26.1-fold), (mitochondrial compartmentalized expression of gene cbts*Δp, increase in cembratriene-ol titer by 76.3-fold), (mitochondrial compartmentalized expression of gene ggpps, increase in cembratriene-ol titer by 76.3-fold)\n",
      "Progress: 8.45% completed\n",
      "Response for Row 170:\n",
      "Answer to Question 2: (deletion of gene ADE2, consequence 1), (expression of gene ErCas12a, consequence 2), (expression of gene ErCas12a, decreased maximum specific growth rate on glucose)\n",
      "Progress: 8.50% completed\n",
      "Response for Row 171:\n",
      "Answer to Question 2: (deletion of transcription factors, rapid destabilization of existing ribosomal protein transcripts), (deletion of transcription factors, gradual slowing of new transcript production)\n",
      "Progress: 8.55% completed\n",
      "Response for Row 172:\n",
      "Answer to Question 2: (expression of glucoamylase gene, enhanced raw starch conversion), (expression of glucoamylase gene, 36% higher ethanol concentration)\n",
      "Progress: 8.60% completed\n",
      "Response for Row 173:\n",
      "Answer to Question 2: (allotopic expression of mitochondrial genes, adaptation of codon usage), (allotopic expression of mitochondrial genes, inclusion of sequences encoding mitochondrial targeting signals), (allotopic expression of mitochondrial genes, consideration of average apparent free energy of membrane insertion), (allotopic expression of mitochondrial genes, consideration of final functional topology), (allotopic expression of mitochondrial genes, consideration of protein translocator TIM23 sorting mechanism)\n",
      "Progress: 8.65% completed\n",
      "Response for Row 174:\n",
      "Answer to Question 2:  (increase in expression of genes related to key components of membrane microdomains, improvement in acid tolerance), (knockout of gene ERG6, reduction in acid tolerance), (disruption of membrane microdomains, inhibition of the formation of the H+ ATPase complex)\n",
      "Progress: 8.70% completed\n",
      "Response for Row 175:\n",
      "Answer to Question 2: (deletion of gene Not4, disruption of ribosomal protein homeostasis), (overexpression of gene Not4, disruption of ribosomal protein homeostasis), (deletion of gene Not4, inhibition of 40S ribosomal autophagy), (overexpression of gene Not4, inhibition of 40S ribosomal autophagy), (interaction of gene Not4 with endolysosomal pathway, control of RP expression), (interaction of gene Not4 with endolysosomal pathway, inhibition of 40S ribosomal autophagy)\n",
      "Progress: 8.75% completed\n",
      "Response for Row 176:\n",
      "Answer to Question 2: (expression of gene TUB1, consequence cell fitness during microtubule stress), (expression of gene TUB1, consequence promotes α-tubulin expression), (expression of gene TUB1, consequence requires chromatin remodeling genes), (expression of gene TUB1, consequence requires genes involved in α/β-tubulin heterodimer assembly), (expression of gene TUB1, consequence requires genes involved in spindle assembly checkpoint)\n",
      "Progress: 8.80% completed\n",
      "Response for Row 177:\n",
      "Answer to Question 2: (expression of gene AG1IA_06082, decreased sensitivity to fungicides), (expression of gene AG1IA_08645, decreased sensitivity to fungicides), (expression of ABC transporter-encoding genes, increased MDR), (expression of MFS transporter-encoding genes, increased MDR), (overexpression of gene AG1IA_06082, high accumulation of SYP-14288), (overexpression of gene AG1IA_08645, high accumulation of SYP-14288)\n",
      "Progress: 8.85% completed\n",
      "Response for Row 178:\n",
      "Answer to Question 2: (deletion of gene in glycolysis pathway, lower expression in W303a), (overexpression of gene ARO9, increased phenylalanine production in CEN.PK2-1C), (overexpression of gene ARO7, increased phenylalanine production in CEN.PK2-1C), (overexpression of gene PHA2, increased phenylalanine production in CEN.PK2-1C), (overexpression of seven DEGs associated with phospholipid biosynthesis, increased phospholipids production in W303a)\n",
      "Progress: 8.90% completed\n",
      "Response for Row 179:\n",
      "Answer to Question 2: (expression of gene 4CL, consequence increased coumaroyl-CoA production), (expression of gene BAS, consequence increased raspberry ketone synthesis), (expression of gene 4CL and BAS, consequence production of over 30 μg/g raspberry ketone in N. benthamiana), (expression of gene 4CL and BAS, consequence production of up to 60 mg/L raspberry ketone in S. cerevisiae)\n",
      "Progress: 8.95% completed\n",
      "Response for Row 180:\n",
      "Answer to Question 2: (deletion of gene TOLLIP, consequence decrease in doxorubicin resistance), (deletion of gene TOLLIP, consequence increase in sensitivity to doxorubicin), (overexpression of gene TOLLIP, consequence increase in doxorubicin resistance)\n",
      "Progress: 9.00% completed\n",
      "Response for Row 181:\n",
      "Answer to Question 2: (deletion of nitrogen, consequence increased total carbohydrate percentage), (deletion of nitrogen, consequence increased lipid percentage), (deletion of nitrogen, consequence increased lipid productivity), (overexpression of delta-9 fatty acid desaturase gene, consequence increased oleic acid level), (overexpression of delta-9 fatty acid desaturase gene, consequence increased palmitoleic acid level)\n",
      "Progress: 9.05% completed\n",
      "Response for Row 182:\n",
      "Answer to Question 2: (deletion of TNFα gene, reduction of inflammation in non-stimulated colon cells), (deletion of TNFα gene, reduction of inflammation in lipopolysaccharide-stimulated colon cells)\n",
      "Progress: 9.10% completed\n",
      "Response for Row 183:\n",
      "Answer to Question 2: (deletion of gene FYV6, decreases splicing of reporter genes), (deletion of gene FYV6, accumulates first-step products), (deletion of gene FYV6, defect in exon ligation), (deletion of gene FYV6, changes 3' splice site selection in SUS1 transcript)\n",
      "Progress: 9.15% completed\n",
      "Response for Row 184:\n",
      "Answer to Question 2: (deletion of gene RAS/PKA, consequence diminished stress tolerance), (deletion of gene Snf1, consequence diminished stress tolerance), (deletion of gene HOG, consequence diminished stress tolerance), (overexpression of metabolic enzymes, consequence enhanced xylose fermentation)\n",
      "Progress: 9.20% completed\n",
      "Response for Row 185:\n",
      "Answer to Question 2: (deletion of gene GCNT3, downregulation of mucin type O-glycan biosynthesis), (deletion of gene GALNT5, downregulation of mucin type O-glycan biosynthesis), (deletion of gene B3GNT3, downregulation of mucin type O-glycan biosynthesis), (deletion of gene GALNT18, downregulation of mucin type O-glycan biosynthesis), (deletion of gene GALNT14, downregulation of mucin type O-glycan biosynthesis), (deletion of gene COL6A1, downregulation of cell and tissue structure genes), (deletion of gene COL1A1, downregulation of cell and tissue structure genes), (deletion of gene COL4A1, downregulation of cell and tissue structure genes), (deletion of gene COL1A2, downregulation of cell and tissue structure genes), (deletion of gene COL6A2, downregulation of cell and tissue structure genes), (deletion of gene LAMB2, downregulation of cell and tissue structure genes), (deletion of gene ITGA8, downregulation of cell and tissue structure genes), (deletion of gene ITGA2, downregulation of cell and tissue structure genes), (deletion of gene ITGA5, downregulation of cell and tissue structure genes), (deletion of gene CCL16, downregulation of chemokines), (deletion of gene CCL21, downregulation of chemokines), (deletion of gene CCL14, downregulation of chemokines), (deletion of gene CXCL12, downregulation of chemokines), (deletion of gene CXCL14, downregulation of chemokines), (overexpression of gene GSTA1, upregulation of glutathione metabolism), (overexpression of gene RRM2B, upregulation of glutathione metabolism), (deletion of gene GSTM1, downregulation of glutathione metabolism), (deletion of gene GPX8, downregulation of glutathione metabolism)\n",
      "Progress: 9.25% completed\n",
      "Response for Row 186:\n",
      "Answer to Question 2: (no gene expression/deletion mentioned, no specific consequences related to gene expression/deletion)\n",
      "Progress: 9.30% completed\n",
      "Response for Row 187:\n",
      "Answer to Question 2: (decrease in expression of ATF1 gene, decrease in enzyme synthesis), (transcriptional repression of ATF1 gene, decrease in acetate production), (expression of ATF1 gene, regulated by ROX1), (expression of ATF1 gene, regulated by RAP1)\n",
      "Progress: 9.35% completed\n",
      "Response for Row 188:\n",
      "Answer to Question 2: (expression of Pho89, control by Crz1), (deletion of TRPCa7-like, impaired expression of Pho89)\n",
      "Progress: 9.40% completed\n",
      "Response for Row 189:\n",
      "Answer to Question 2: (expression of SOD, consequence 1: increased antioxidant enzyme activity), (expression of CAT, consequence 1: increased antioxidant enzyme activity), (expression of GR, consequence 1: increased antioxidant enzyme activity), (decrease of ROS, consequence 1: reduced oxidative stress), (decrease of MDA, consequence 1: reduced oxidative stress), (reduction of oxidative stress, consequence 1: elevated ethanol fermentation efficiency)\n",
      "Progress: 9.45% completed\n",
      "Response for Row 190:\n",
      "Answer to Question 2: (deletion of introns, decrease in transcription), (introduction of introns, increase in transcription), (enhancement of gene expression, increased efficacy of recombinant expression cassettes)\n",
      "Progress: 9.50% completed\n",
      "Response for Row 191:\n",
      "Answer to Question 2: (deletion of gene variants, inactive genes), (expression of gene variants, unique pathway configuration), (supply of gRNAs, activation of gene expression), (activation of chromosomal repair pathway, self-assembly of pathway), (generation of pathway configurations, distinct color phenotypes), (generation of pathway configurations, carotenoid product profiles)\n",
      "Progress: 9.55% completed\n",
      "Response for Row 192:\n",
      "Answer to Question 2: (deletion of gene Mec1p, consequence downregulates efficiency of transcription termination), (deletion of gene Mec1p, consequence reduces efficiency of pre-mRNAs cleavage at the pA sites)\n",
      "Progress: 9.60% completed\n",
      "Response for Row 193:\n",
      "Answer to Question 2: (recombinant rhfFSH expression, stimulated E2 levels), (recombinant rhfFSH expression, upregulated aromatase gene expression), (recombinant rhfLH expression, stimulated E2 levels), (recombinant rhfLH expression, upregulated aromatase gene expression)\n",
      "Progress: 9.65% completed\n",
      "Response for Row 194:\n",
      "Answer to Question 2: (deletion of RQC components, activation of stress response), (deletion of RQC components, neurodegenerative diseases), (regulation of protein synthesis, prevention of aberrant or unnecessary proteins production)\n",
      "Progress: 9.70% completed\n",
      "Response for Row 195:\n",
      "Answer to Question 2: (overexpression of gene involved in trafficking of vesicles, increased α-amylase secretion), (deletion of gene involved in trafficking of vesicles, increased α-amylase secretion), (overexpression of gene involved in endosome to Golgi transport, increased α-amylase secretion), (deletion of gene involved in endosome to Golgi transport, increased α-amylase secretion), (overexpression of gene at phagophore assembly site, increased α-amylase secretion), (deletion of gene at phagophore assembly site, increased α-amylase secretion), (overexpression of gene involved in cell cycle, increased α-amylase secretion), (deletion of gene involved in cell cycle, increased α-amylase secretion), (overexpression of gene involved in energy supply, increased α-amylase secretion), (deletion of gene involved in energy supply, increased α-amylase secretion)\n",
      "Progress: 9.75% completed\n",
      "Response for Row 196:\n",
      "Answer to Question 2: (no expression/deletion of gene mentioned, improvement in prediction of protein abundance), (no expression/deletion of gene mentioned, improved correlation with CAI)\n",
      "Progress: 9.80% completed\n",
      "Response for Row 197:\n",
      "Answer to Question 2: (expression of multiple genes, efficient polycistronic gene expression), (expression of 13 biosynthetic genes, production of 1,090.41 ± 80.92 mg L<sup>-1</sup> squalene), (expression of 13 biosynthetic genes, production of 1.04 ± 0.02 mg L<sup>-1</sup> mogrol)\n",
      "Progress: 9.85% completed\n",
      "Response for Row 198:\n",
      "Answer to Question 2: (deletion of ribosomal proteins, consequence high-level cotranslational protein degradation), (overexpression of ribosomal proteins, consequence increased cotranslational folding efficiency), (fusion of ubiquitin to ribosomal proteins, consequence protection from cotranslational protein degradation)\n",
      "Progress: 9.90% completed\n",
      "Response for Row 199:\n",
      "Answer to Question 2: (expression of GmPHT2;2, higher phosphorus content in chloroplasts), (expression of GmPHT2;2, higher whole-plant phosphorus content), (expression of GmPHT2;1, lower phosphorus content compared to GmPHT2;2), (overexpression of GmPHT2, increased phosphorus utilization in yeast), (up-regulation of GmPHT2 by low-temperature stress, implication in low-temperature response), (expression of GmPHT2;1 and GmPHT2;2, cyclic pattern in response to light), (localization of GmPHT2 protein, chloroplast)\n",
      "Progress: 9.95% completed\n",
      "Response for Row 200:\n",
      "Answer to Question 2:  (overexpression of gene ClNAC100, induced by phosphate deficiency), (overexpression of gene ClNAC100, positively regulated by ABA), (overexpression of gene ClNAC100, negatively regulated by JA), (overexpression of gene ClNAC100, highly expressed in roots), (overexpression of gene ClNAC100, highly expressed in leaves)\n",
      "Progress: 10.00% completed\n",
      "Response for Row 201:\n",
      "Answer to Question 2: (deletion of gene Opi1, hypersensitivity to genotoxins), (deletion of gene Opi1, delayed G1-to-S-phase transition), (deletion of gene Opi1, decreased gamma-H2A levels), (deletion of gene Opi1, increased mitochondrial DNA instability upon MMS treatment), (activation of transcription factor Ino2-Ino4, responsible for genotoxin sensitivity in Opi1-deficient cells), (production of inositol pyrophosphates by Kcs1, counteracts Opi1 function during MMS-induced stress)\n",
      "Progress: 10.05% completed\n",
      "Response for Row 202:\n",
      "Answer to Question 2: (overexpression of gene IME4, elevation of mRNA levels of methylated genes in glycolysis), (overexpression of gene IME4, elevation of mRNA levels of methylated genes in acetyl-CoA synthesis), (overexpression of gene IME4, elevation of mRNA levels of methylated genes in shikimate/aromatic amino acid synthesis), (overexpression of gene IME4, induction of gene ACS1), (overexpression of gene IME4, induction of gene ADH2), (overexpression of gene IME4, increase in titers of isoprenoids), (overexpression of gene IME4, increase in titers of aromatic compounds)\n",
      "Progress: 10.10% completed\n",
      "Response for Row 203:\n",
      "Answer to Question 2:  (high gene expression difference, abiotic stress tolerance enhancement), (high gene expression difference, cell wall/carbohydrate synthesis enhancement), (biostimulant coating seed treatment, similar pathway enhancement), (biostimulant coating seed treatment, common genes activation), (abiotic stress tolerance enhancement, plant protection from abiotic stresses), (cell wall/carbohydrate synthesis enhancement, higher level of sugars in plant)\n",
      "Progress: 10.15% completed\n",
      "Response for Row 204:\n",
      "Answer to Question 2: (LPS injection, upregulation of some tissue inflammation-related genes), (LPS injection, downregulation of intestinal tight junction-related genes), (LPS injection, elevation of liver phospho-nuclear factor kappa B (p-NF-κB)), (LPS injection, elevation of phospho-inhibitory subunit of NF-κB (p-IκBα)), (LPS injection, elevation of phospho-c-Jun N-terminal kinase (p-JNK)), (LPS injection, elevation of Nuclear factor kappa-B (NF-κB)), (LPS injection, elevation of the inhibitory subunit of NF-κB (IκBα)), (maternal dietary supplementation with XPC, reduction of inflammatory factors in plasma and thymus), (maternal dietary supplementation with XPC, downregulation of IL-6 gene expression in thymus), (maternal dietary supplementation with XPC, downregulation of IL-10 gene expression in thymus), (maternal dietary supplementation with XPC, decrease of JNK protein expression in liver)\n",
      "Progress: 10.20% completed\n",
      "Response for Row 205:\n",
      "Answer to Question 2: (deletion of gene Saccharomyces cerevisiae terminator, consequence promoter strength increase), (deletion of gene Saccharomyces cerevisiae terminator, consequence terminator strength increase), (overexpression of gene TATA box-like sequence, consequence adequate execution of both functions), (regulation of gene UAS, consequence promoter-like strength increase), (regulation of gene spacer sequence, consequence terminator-like strength increase), (application of bifunctional elements, consequence improved pathway assembly efficiency), (application of bifunctional elements, consequence higher lycopene yield)\n",
      "Progress: 10.25% completed\n",
      "Response for Row 206:\n",
      "Answer to Question 2: (deletion of nitrogen source, consequence 1: activation of transcription by GATA family transcriptional activators), (deletion of nitrogen source, consequence 2: recruitment of SAGA chromatin remodeling complex), (inhibition of glutamine production, consequence 1: activation of transcription by GATA family transcriptional activators)\n",
      "Progress: 10.30% completed\n",
      "Response for Row 207:\n",
      "Answer to Question 2: (deletion of pyrimidine dimers by TC-NER, removal from transcribed strand of active genes), (deletion of pyrimidine dimers by GG-NER, removal from inactive genome), (UV irradiation, increase in RNA polymerase-I density at 5'-end of rRNA genes), (elongation and pausing/release of RNA polymerase-I at PDs, formation of discrete chromatin structures), (replacement of RNA polymerase-I by nucleosomes, participation of GG-NER in PD removal from TS), (density of RNA polymerase-I, faster removal of PDs in 5'-end than 3'-end of gene), (GG-NER, sole responsible for PD removal from non-transcribed strand of rRNA gene coding regions)\n",
      "Progress: 10.35% completed\n",
      "Response for Row 208:\n",
      "Answer to Question 2: (chromosomal deletion of gene STB5, enhances autophagy), (loss of gene Stb5, upregulation of ATG transcripts), (Stb5-mediated impact, effect on genes involved in NADPH production), (Stb5-mediated impact, effect on genes involved in pentose phosphate pathway)\n",
      "Progress: 10.40% completed\n",
      "Response for Row 209:\n",
      "Answer to Question 2: (osFAR1 overexpression, increased production of primary alcohol), (osFAR1 overexpression, enhanced amounts of C22:0 primary alcohol), (osFAR1 overexpression, enhanced amounts of C24:0 primary alcohol), (osFAR1 overexpression, enhanced amounts of C24:0 to C32:0 primary alcohols), (osFAR1 overexpression, increased content of C24:0 to C30:0 primary alcohols), (osFAR1 overexpression, reduced cuticle permeability), (osFAR1 overexpression, enhanced drought tolerance)\n",
      "Progress: 10.45% completed\n",
      "Response for Row 210:\n",
      "Answer to Question 2: (deletion of gene gRNAs, consequence increased guide RNA cutting efficiency), (expression of gene gRNAs, consequence integration efficiency of a gene cassette), (expression of gene gRNAs, consequence improved cellular fitness), (expression of gene gRNAs, consequence increased GFP expression level), (expression of five genes, consequence improved production of 5-aminolevulinic acid), (expression of five gene cassettes, consequence production of 9 g/L succinic acid)\n",
      "Progress: 10.50% completed\n",
      "Response for Row 211:\n",
      "Answer to Question 2: (deletion of gene PDR1, reduced yeast resistance to tyrosol), (deletion of gene PDR3, reduced yeast resistance to tyrosol), (expression of gene PDR5, contributed to yeast resistance to tyrosol), (tyrosol, inhibited efflux of rhodamine 6G), (tyrosol, induced MDR), (Pdr5-GFP levels, increased), (tyrosol, reduced yeast ability to accumulate Nile red), (tyrosol, inhibited cytostatic effect of clotrimazole)\n",
      "Progress: 10.55% completed\n",
      "Response for Row 212:\n",
      "Answer to Question 2:  (deletion of gene SAM1, altered chromosome stability), (deletion of gene SAM1, change in AdoMet concentrations), (deletion of gene SAM2, altered chromosome stability), (deletion of gene SAM2, change in AdoMet concentrations), (deletion of gene SAM1, sensitivity or resistance to azoles), (deletion of gene SAM2, sensitivity or resistance to azoles), (deletion of gene SAM1, sensitivity or resistance to cisplatin), (deletion of gene SAM2, sensitivity or resistance to cisplatin), (deletion of gene SAM1, sensitivity or resistance to oxidative stress), (deletion of gene SAM2, sensitivity or resistance to oxidative stress), (deletion of gene SAM1, perturbations in arginine biosynthesis), (deletion of gene SAM2, perturbations in arginine biosynthesis), (deletion of gene SAM1, sensitivity or resistance to DNA synthesis inhibitors), (deletion of gene SAM2, sensitivity or resistance to DNA synthesis inhibitors), (deletion of gene SAM1, sensitivity or resistance to tamoxifen), (deletion of gene SAM2, sensitivity or resistance to tamoxifen)\n",
      "Progress: 10.60% completed\n",
      "Response for Row 213:\n",
      "Answer to Question 2: (overexpression of gene PAA1, acetylation of different aralkylamines), (overexpression of gene HPA2, AANAT activity)\n",
      "Progress: 10.65% completed\n",
      "Response for Row 214:\n",
      "Answer to Question 2: (exogenous expression of long-chain human insulin growth factor-1, promotion of cell proliferation), (exogenous expression of platelet-derived growth factor-BB, promotion of cell proliferation), (exogenous expression of basic fibroblast growth factor, promotion of cell proliferation), (exogenous expression of epidermal growth factor, promotion of cell proliferation), (endogenous protease knockout, enhancement of cytokine stability), (genomic co-expression, increased yield of cytokines), (expression frame gene order optimization, improved cytokine production efficiency)\n",
      "Progress: 10.70% completed\n",
      "Response for Row 215:\n",
      "Answer to Question 2: (overexpression of gene YPR015C, increased resistance to furfural), (overexpression of gene YPR015C, improved enzyme reductase activity), (overexpression of gene YPR015C, accumulation of oxygen reactive species), (overexpression of gene YPR015C, involvement in amino acid biosynthesis), (overexpression of gene YPR015C, oxidative stress response), (overexpression of gene YPR015C, cell wall response), (overexpression of gene YPR015C, heat shock protein response), (overexpression of gene YPR015C, mitochondrial-associated protein response)\n",
      "Progress: 10.75% completed\n",
      "Response for Row 216:\n",
      "Answer to Question 2: (deletion of gene CgMCUR1, reduced acetate tolerance), (deletion of gene CgMCUR1, reduced H2O2 tolerance), (deletion of gene CgMCUR1, reduced high temperature tolerance), (expression of gene CgMCUR1, improved acetic acid tolerance), (expression of gene CgMCUR1, improved H2O2 tolerance), (expression of gene CgMCUR1, improved high temperature tolerance), (expression of gene CgMCUR1, enhanced intracellular proline accumulation), (overexpression of gene CgMCUR1, affected proline metabolism), (overexpression of gene CgMCUR1, reduced cellular lipid peroxidation), (overexpression of gene CgMCUR1, altered SFA to UFA ratio), (overexpression of gene CgMCUR1, increased ethanol production), (overexpression of gene CgMCUR1, increased conversion rate), (overexpression of gene CgMCUR1, improved ethanol yield in undetoxified cellulose hydrolysate), (overexpression of gene CgMCUR1, improved conversion rate in undetoxified cellulose hydrolysate)\n",
      "Progress: 10.80% completed\n",
      "Response for Row 217:\n",
      "Answer to Question 2: (codon pair usage patterns, modulation of translation efficiency), (codon pair usage patterns, protein folding), (codon pair usage patterns, mRNA decay), (dicodon-based measure, greater correlations with gene expression level), (dicodons associated with low adaptiveness, mediate strong translational inhibition in yeast)\n",
      "Progress: 10.85% completed\n",
      "Response for Row 218:\n",
      "Answer to Question 2: (overexpression of gene LcMT3, resistance to salt in E. coli cells), (overexpression of gene LcMT3, resistance to salt in yeast), (overexpression of gene LcMT3, higher germination rates in transgenic plants), (overexpression of gene LcMT3, longer roots in transgenic plants), (overexpression of gene LcMT3, reduced accumulation of malondialdehyde in transgenic plants), (overexpression of gene LcMT3, reduced relative conductivity in transgenic plants), (overexpression of gene LcMT3, reduced reactive oxygen species in transgenic plants), (overexpression of gene LcMT3, increased concentration of proline in transgenic plants), (overexpression of gene LcMT3, increased relative water content in transgenic plants), (overexpression of gene LcMT3, increased chlorophyll content in transgenic plants), (overexpression of gene LcMT3, increased activity of superoxide dismutase in transgenic plants), (overexpression of gene LcMT3, increased activity of peroxidase in transgenic plants), (overexpression of gene LcMT3, increased activity of catalase in transgenic plants), (overexpression of gene LcMT3, reduced Na+ accumulation in transgenic plants), (overexpression of gene LcMT3, lower Na+/K+ ratio in transgenic plants), (overexpression of gene LcMT3, regulatory effect on transporter proteins SOS), (overexpression of gene LcMT3, regulatory effect on transporter proteins NHX1)\n",
      "Progress: 10.90% completed\n",
      "Response for Row 219:\n",
      "Answer to Question 2: (expression of gene PfCOQ2, complementation of growth defect in coq2Δ mutant strain), (expression of gene PfCOQ2, production of ubiquinone UQ in coq2Δ mutant), (expression of gene PfCOQ2, detection of UQ when labeled with [ring-14C(U)]-p-aminobenzoic acid), (expression of gene PfCOQ2, detection of UQ when labeled with [ring-14C(U)]-4-hydroxybenzoic acid)\n",
      "Progress: 10.95% completed\n",
      "Response for Row 220:\n",
      "Answer to Question 2: (deletion of gene x, increased NPA secretion), (deletion of gene y, increased NPA secretion), (deletion of gene z, increased NPA secretion), (deletion of gene w, increased NPA secretion), (increased copy number of NPA expression cassette, increased NPA concentration in supernatant), (use of native and synthetic secretion signals, tuned NPA secretion)\n",
      "Progress: 11.00% completed\n",
      "Response for Row 221:\n",
      "Answer to Question 2: (deletion of gene GAL1p, consequence reduced toxic effect on host cell), (expression of gene IdeS, consequence production of protease IdeS), (expression of fused protein IdeS-GFP, consequence measurable fluorescence with flow cytometry)\n",
      "Progress: 11.05% completed\n",
      "Response for Row 222:\n",
      "Answer to Question 2: (loss of gene DPH, increased resistance to sordarin), (loss of gene DPH, increased -1 ribosomal frameshifting), (loss of gene DPH, increased ribosomal drop-off during elongation), (loss of gene DPH, premature termination at out-of-frame stop codons)\n",
      "Progress: 11.10% completed\n",
      "Response for Row 223:\n",
      "Answer to Question 2: (depletion of glucose, trigger environmental stress response), (recurring glucose shortages, increase ribosome portfolio)\n",
      "Progress: 11.15% completed\n",
      "Response for Row 224:\n",
      "Answer to Question 2: (expression of MCT1 gene, higher volatile fatty acid absorption by the rumen epithelium), (decrease in expression of Protozoa, due to antimicrobial effects of turmeric and thymol), (decrease in expression of Anaerobic fungi, due to antimicrobial effects of turmeric and thymol), (decrease in expression of Prevotellaceae_UCG-004, due to antimicrobial modulation), (decrease in expression of Bacteroidetes_BD2-2, due to antimicrobial modulation), (decrease in expression of Papillibacter, due to antimicrobial modulation), (decrease in expression of Schwartzia, due to antimicrobial modulation), (decrease in expression of Absconditabacteriales_SR1, due to antimicrobial modulation), (decrease in expression of Prevotellaceae_NK3B31_group, due to antimicrobial modulation), (decrease in expression of Clostridia_UCG-014, due to antimicrobial modulation), (decrease in expression of Fibrobacter succinogenes, due to antimicrobial modulation), (increase in expression of Selenomonas ruminantium, due to antimicrobial modulation)\n",
      "Progress: 11.20% completed\n",
      "Response for Row 225:\n",
      "Answer to Question 2: (overexpression of gene Gal4p, increased activity of native PGAL1 by 131.20%), (overexpression of gene Gal4p, increased activity of heterologous PSkGAL2 by 72.45%), (expression of gene KlLac9p, increased activity of PScGAL1 by 41.56%), (expression of gene KlLac9p, increased activity of PSkGAL2 by 100.63%), (optimized GAL expression system, increased production of β-carotene by 9.02-fold)\n",
      "Progress: 11.25% completed\n",
      "Response for Row 226:\n",
      "Answer to Question 2:  (upregulation of gene expression of key enzymes in fatty acid synthesis pathway, increase in POA titer)\n",
      "Progress: 11.30% completed\n",
      "Response for Row 227:\n",
      "Answer to Question 2: (deletion of gene RAD6, consequence H2BK123 ubiquitylation impairment), (mutation in gene RAD6, consequence similar transcriptome profile to H2B ubiquitylation site mutant), (interaction between Rtf1 subunit HMD and Rad6, consequence stimulation of H2BK123ub)\n",
      "Progress: 11.35% completed\n",
      "Response for Row 228:\n",
      "Answer to Question 2: (overexpression of gene OsDHN2, consequence improved Cd tolerance), (overexpression of gene OsDHN2, consequence reduced Cd concentration in yeast), (overexpression of gene OsDHN2, consequence increased expression levels of SOD1), (overexpression of gene OsDHN2, consequence increased expression levels of CTA1), (overexpression of gene OsDHN2, consequence increased expression levels of GSH1), (overexpression of gene OsDHN2, consequence increased expression levels of CTT1), (overexpression of gene OsDHN2, consequence increased antioxidant enzymatic activities)\n",
      "Progress: 11.40% completed\n",
      "Response for Row 229:\n",
      "Answer to Question 2: (overexpression of key genes, removal of limitations in protein secretion), (inappropriate gene expression, negative effects), (overexpression of ERO1, increased α-amylase production), (overexpression of SLY1, increased α-amylase production)\n",
      "Progress: 11.45% completed\n",
      "Response for Row 230:\n",
      "Answer to Question 2: (overexpression of gene CYP709C69, consequence diclofop-methyl resistance in plants), (overexpression of gene CYP709C69, consequence production of hydroxylated diclofop acid in yeast), (overexpression of gene CYP81A12/21, consequence production of hydroxylated diclofop acid), (overexpression of gene CYP709C69, consequence presumed clomazone-activating function), (overexpression of gene CYP81A12/21, consequence part of transcriptionally linked overexpression with CYP709C69)\n",
      "Progress: 11.50% completed\n",
      "Response for Row 231:\n",
      "Answer to Question 2:  (overexpression of gene GmC4H2, increased p-coumaric acid production), (overexpression of gene GmC4H14, increased p-coumaric acid production), (overexpression of gene GmC4H20, increased p-coumaric acid production), (deletion of gene GmC4H10, no change in p-coumaric acid production)\n",
      "Progress: 11.55% completed\n",
      "Response for Row 232:\n",
      "Answer to Question 2: (deletion of gene OsSMC5, consequence 1), (deletion of gene OsSMC6, consequence 1), (deletion of gene OsMMS21, consequence 1), (deletion of gene OsMMS21, consequence 2), (deletion of gene OsMMS21, consequence 3)\n",
      "Progress: 11.60% completed\n",
      "Response for Row 233:\n",
      "Answer to Question 2: (introduction of PRK, increased acetaldehyde production), (introduction of PRK, increased acetate production), (reduction of RuBisCO copy number, decreased acetaldehyde production), (reduction of RuBisCO copy number, decreased acetate production), (C-terminal fusion of 19-amino-acid tag to PRK, decreased acetaldehyde production), (C-terminal fusion of 19-amino-acid tag to PRK, decreased acetate production), (use of ANB1 promoter for PRK synthesis, decreased acetaldehyde production), (use of ANB1 promoter for PRK synthesis, decreased acetate production), (overexpression of PRK and RuBisCO, increased acetaldehyde production), (overexpression of PRK and RuBisCO, increased acetate production)\n",
      "Progress: 11.65% completed\n",
      "Response for Row 234:\n",
      "Answer to Question 2: (expression of plaA gene, increased production of plantaricin 423), (expression of munST4SA gene, increased production of mundticin ST4SA), (codon optimization of plaA gene, no significant increase in PlaX activity), (codon optimization of munST4SA gene, eight-fold increase in MunX activity against Listeria monocytogenes), (use of MFα1 secretion signal, superior secretion of bacteriocins compared to XYNSEC), (HPLC-purification of codon-optimized genes, yield of 20.9 mg/L MunX), (HPLC-purification of codon-optimized genes, yield of 18.4 mg/L PlaX), (production of PlaX, MIC of 1.18 µM against L. monocytogenes), (production of MunX, MIC of 108.52 nM against L. monocytogenes)\n",
      "Progress: 11.70% completed\n",
      "Response for Row 235:\n",
      "Answer to Question 2: (no gene overexpression/deletion mentioned, monitoring mitochondrial translation by labeling newly synthesized mitochondrial polypeptides with [S<sup>35</sup>]-methionine)\n",
      "Progress: 11.75% completed\n",
      "Response for Row 236:\n",
      "Answer to Question 2: (knocking out the endogenous bypass gene, increased d-pantothenic acid production), (adjusting the copy number of the pathway modules, increased d-pantothenic acid production), (balancing NADPH utilization, increased d-pantothenic acid production), (regulating the GAL inducible system, increased d-pantothenic acid production)\n",
      "Progress: 11.80% completed\n",
      "Response for Row 237:\n",
      "Answer to Question 2: (loss-of-function mutation in OsSPT4, severe reproductive growth defects), (loss-of-function mutation in OsSPT5-1, severe reproductive growth defects), (deletion of OsSPT5-2, no phenotypes), (mutation in OsSPT5-1, gibberellin-related dwarfed defects), (mutation in OsSPT5-1, weak root system), (mutation in OsSPT5-1, short life cycle), (interaction between OsSPT5-1 and APO2, regulation of rice shoot growth), (OsSPT5-1 overexpression, involvement in gibberellin pathway), (OsSPT5-1 overexpression, involvement in auxin pathway), (OsSPT5-1 overexpression, involvement in cytokinin pathway)\n",
      "Progress: 11.85% completed\n",
      "Response for Row 238:\n",
      "Answer to Question 2: (None, identification of motifs bound by BpERF2), (None, verification of motifs by yeast one hybrid), (None, verification of motifs by electrophoretic mobility shift assay), (None, verification of motifs by ChIP study)\n",
      "Progress: 11.90% completed\n",
      "Response for Row 239:\n",
      "Answer to Question 2: (heterologous expression of gene CYP51, demethylates obtusifoliol), (heterologous expression of gene CYP51, demethylates 4-desmethyl-24,25-dihydrolanosterol), (identification of sterol side-chain reductase, reduces Δ24,28 double bonds), (identification of sterol side-chain reductase, reduces Δ24,25 double bonds), (reduction of cycloartenol to cycloartanol, initiates cholesterogenesis)\n",
      "Progress: 11.95% completed\n",
      "Response for Row 240:\n",
      "Answer to Question 2: (expression of super-clogger ANT1 variant, neurodegeneration), (expression of super-clogger ANT1 variant, age-dependent dominant myopathy), (missense mutations in ADP/ATP translocase 1 (ANT1), protein accumulation along the protein import pathway), (mutations in ADP/ATP carrier 2 (Aac2), obstruction of general protein translocation into mitochondria), (obstruction of general protein translocation into mitochondria, impairment of mitochondrial respiration), (obstruction of general protein translocation into mitochondria, impairment of cytosolic proteostasis), (obstruction of general protein translocation into mitochondria, reduction in cell viability), (double mutant Aac2/ANT1, severe clogging at the translocase of the outer membrane (TOM) complex), (severe clogging at the translocase of the outer membrane (TOM) complex, extreme toxicity in yeast)\n",
      "Progress: 12.00% completed\n",
      "Response for Row 241:\n",
      "Answer to Question 2: (overexpression of gene SsWRKY1, higher antioxidant enzyme activity), (overexpression of gene SsWRKY1, higher proline content), (overexpression of gene SsWRKY1, lower malondialdehyde content), (overexpression of gene SsWRKY1, lower ROS accumulation), (overexpression of gene SsWRKY1, upregulation of six stress-related genes)\n",
      "Progress: 12.05% completed\n",
      "Response for Row 242:\n",
      "Answer to Question 2: (deletion of gene AQPF, reduced hydrogen peroxide transport), (deletion of gene AQPF, impaired phenotypic growth)\n",
      "Progress: 12.10% completed\n",
      "Response for Row 243:\n",
      "Answer to Question 2: (expression of gene ScFLO1, improved tolerance to d-lactic acid), (expression of gene ScFLO1, increased specific growth rate), (expression of gene LpDLDH, increased d-lactic acid production), (upregulation of lactate transport genes, increased tolerance to d-lactic acid), (upregulation of iron metabolism genes, increased tolerance to d-lactic acid)\n",
      "Progress: 12.15% completed\n",
      "Response for Row 244:\n",
      "Answer to Question 2: (expression of BfPMHA, increased tolerance to hypo-salinity), (expression of BfPMHA, higher expression level with greater low salinity stress), (overexpression of BfPMHA, enhanced yeast tolerance to NaCl stress), (overexpression of BfFBA, enhanced yeast tolerance to NaCl stress), (overexpression of BfGAPDH, enhanced yeast tolerance to NaCl stress), (overexpression of BfMnSOD, enhanced yeast tolerance to NaCl stress)\n",
      "Progress: 12.20% completed\n",
      "Response for Row 245:\n",
      "Answer to Question 2: (deletion of gene T7RNAP in yeast, failure to create a functional T7RNAP-based expression system), (overexpression of gene T7RNAP in yeast, potential increase in efficiency and processivity of yeast as an expression system)\n",
      "Progress: 12.25% completed\n",
      "Response for Row 246:\n",
      "Answer to Question 2: (expression of gene KatA, increased catalase activity), (expression of gene KatA, maximum yield 338.8 ± 9.6 U/mL), (expression of gene KatA, specific activity 14826.58 U/mg), (expression of gene KatA, optimal activity at 25 °C and pH 11.0), (expression of gene KatA, Km for hydrogen peroxide 10.9 ± 0.5 mM), (expression of gene KatA, kcat/Km 5788.1 ± 25.6 s^-1 mM^-1)\n",
      "Progress: 12.30% completed\n",
      "Response for Row 247:\n",
      "Answer to Question 2: (deletion of gene ssu1, increased intracellular selenomethionine content), (expression of gene ssu1, selenium efflux process), (mutation in gene fzf1, selenium tolerance), (selenite, induces expression of ssu1)\n",
      "Progress: 12.35% completed\n",
      "Response for Row 248:\n",
      "Answer to Question 2: (decarboxylase production by yeast, conversion of p-coumaric acid to vinylphenols), (decarboxylase production by lactic acid bacteria, conversion of ferulic acid to vinylphenols), (vinylphenol reductase action, production of 4-ethylphenol), (vinylphenol reductase action, production of 4-ethylguaiacol)\n",
      "Progress: 12.40% completed\n",
      "Response for Row 249:\n",
      "Answer to Question 2: (deletion of gene sky1, pathways involved in the regulation of gene expression are downregulated), (deletion of gene sky1, mitochondrial processes are upregulated), (deletion of gene sky2, metabolic processes are affected), (deletion of gene sky2, arginine-induced hyphae formation is impaired)\n",
      "Progress: 12.45% completed\n",
      "Response for Row 250:\n",
      "Answer to Question 2: (upregulation of genes encoding hexose monophosphate biosynthesis pathway, RNA accumulation), (upregulation of genes encoding sulfur-containing amino acid biosynthesis pathway, RNA accumulation)\n",
      "Progress: 12.50% completed\n",
      "Response for Row 251:\n",
      "Answer to Question 2: (overexpression of gene Xks1, growth inhibition), (overexpression of gene Xks1, decreased ATP levels), (decreased expression of gene Xks1, restored growth at high D-xylose concentrations), (decreased expression of gene Xks1, increased ATP levels), (decreased expression of gene Xks1, high rates of xylose metabolism)\n",
      "Progress: 12.55% completed\n",
      "Response for Row 252:\n",
      "Answer to Question 2: (deletion of gene ERG5, thermotolerant capability improvement), (deletion of gene ERG4, thermotolerant capability improvement), (deletion of gene ERG3, thermotolerant capability improvement), (deletion of gene ERG5, increase in ethanol production), (deletion of gene ERG4, increase in ethanol production), (deletion of gene ERG3, increase in ethanol production), (deletion of gene ERG5, up-regulation of gene ERG1), (deletion of gene ERG4, up-regulation of gene ERG1), (deletion of gene ERG3, up-regulation of gene ERG1), (deletion of gene ERG5, up-regulation of gene ERG11), (deletion of gene ERG4, up-regulation of gene ERG11), (deletion of gene ERG3, up-regulation of gene ERG11), (deletion of gene ERG5, down-regulation of gene ERG9), (deletion of gene ERG4, down-regulation of gene ERG9), (deletion of gene ERG3, down-regulation of gene ERG9), (deletion of gene ERG5, down-regulation of gene ERG26), (deletion of gene ERG4, down-regulation of gene ERG26), (deletion of gene ERG3, down-regulation of gene ERG26), (deletion of gene ERG5, improvement in cell viability), (deletion of gene ERG4, improvement in cell viability), (deletion of gene ERG3, improvement in cell viability)\n",
      "Progress: 12.60% completed\n",
      "Response for Row 253:\n",
      "Answer to Question 2: (expression of CHS, increase in anthocyanidin production), (expression of F3H, increase in anthocyanidin production), (expression of VmF3Hv1, best performance in anthocyanidin production), (expression of VuCHS, best performance in anthocyanidin production), (expression of flavonoid 3',5'-hydroxylase, substantial increase in total anthocyanidin production)\n",
      "Progress: 12.65% completed\n",
      "Response for Row 254:\n",
      "Answer to Question 2: (deletion of gene FgIDOA, abnormal conidial morphology), (deletion of gene FgIDOA, reduced mycelial growth), (deletion of gene FgIDOA, decreased virulence in wheat heads), (deletion of gene FgIDOA, reduced deoxynivalenol accumulation), (deletion of gene FgIDOA/B, NAD<sup>+</sup> auxotrophy), (deletion of gene FgIDOB, shifts toward alternative TRP degradation pathways to melatonin and indole derivatives)\n",
      "Progress: 12.70% completed\n",
      "Response for Row 255:\n",
      "Answer to Question 2: (deletion of gene BcTaf14, slow growth), (deletion of gene BcTaf14, abnormal colony morphology), (deletion of gene BcTaf14, reduced conidiation), (deletion of gene BcTaf14, abnormal conidial morphology), (deletion of gene BcTaf14, reduced virulence), (deletion of gene BcTaf14, altered responses to various stresses), (mutation of gene BcTaf14 G80 and W81, affected regulatory effect on mycelial growth), (mutation of gene BcTaf14 G80 and W81, affected regulatory effect on virulence), (mutation of gene BcTaf14 G80 and W81, did not affect production and morphology of conidia), (absence of ET domain in BcTaf14, unable to localize to nucleus), (expression of BcTaf14 without ET domain, defects not recovered to wild-type levels)\n",
      "Progress: 12.75% completed\n",
      "Response for Row 256:\n",
      "Answer to Question 2: (deletion of gene ipac2, viable virus formation), (deletion of gene ipac2, plaque formation similar to intact ipac2), (expression of gene GFP, recombinant replication-competent ILTV generation)\n",
      "Progress: 12.80% completed\n",
      "Response for Row 257:\n",
      "Answer to Question 2: (deletion of pro-growth mRNAs, controlled by Gcn2), (deletion of pro-growth mRNAs, controlled by Ccr4), (phosphorylation of eIF2α, leads to induction of ISR), (induction of ISR, results in translation of Gcn4), (translation of Gcn4, does not necessarily result in canonical Gcn4-dependent transcription)\n",
      "Progress: 12.85% completed\n",
      "Response for Row 258:\n",
      "Answer to Question 2: (overexpression of gene GhNRT2.1e, increased biomass), (overexpression of gene GhNRT2.1e, increased nitrogen accumulation), (overexpression of gene GhNRT2.1e, increased nitrate accumulation), (overexpression of gene GhNRT2.1e, increased nitrogen uptake), (overexpression of gene GhNRT2.1e, increased nitrogen utilization efficiency), (overexpression of gene GhNRT2.1e, increased nitrogen-metabolizing enzyme activity), (overexpression of gene GhNRT2.1e, increased amino acid content), (silencing of gene GhNRT2.1e, suppressed nitrate uptake), (silencing of gene GhNRT2.1e, suppressed nitrate accumulation), (silencing of gene GhNRT2.1e, hampered plant growth), (silencing of gene GhNRT2.1e, affected nitrogen metabolism processes), (silencing of gene GhNRT2.1e, reduced tolerance to low nitrate)\n",
      "Progress: 12.90% completed\n",
      "Response for Row 259:\n",
      "Answer to Question 2: (expression of gene Lg-FLO1, increased flocculation), (expression of gene RIM15, regulates flocculation), (nitrogen starvation, promotes flocculation), (amino acid starvation, promotes flocculation)\n",
      "Progress: 12.95% completed\n",
      "Response for Row 260:\n",
      "Answer to Question 2: (no gene expression/deletion mentioned, consequence: formation of the 3' end of a eukaryotic mRNA), (no gene expression/deletion mentioned, consequence: cleavage and polyadenylation specificity factor (CPSF) coordinates enzymatic activities), (no gene expression/deletion mentioned, consequence: regulation of cleavage site selection), (no gene expression/deletion mentioned, consequence: control of poly(A) tail length)\n",
      "Progress: 13.00% completed\n",
      "Response for Row 261:\n",
      "Answer to Question 2: (no specific gene expression/deletion mentioned, impact on mRNA stability and translational efficiency discussed), (ribosomes control mRNA stability, impact of codon optimality), (transfer RNAs affect mRNA stability, role as master gene regulators), (translation elongation rate triggers mRNA decay, consequence of codon optimality)\n",
      "Progress: 13.05% completed\n",
      "Response for Row 262:\n",
      "Answer to Question 2: (exogenous addition of 2-PE, promotion of ethanol production), (exogenous addition of 2-PE, reduction of negative impact of L. plantarum on S. cerevisiae), (exogenous addition of 2-PE, decrease in relative expression levels of ADH1), (exogenous addition of 2-PE, decrease in relative expression levels of MLS1), (exogenous addition of 2-PE, decrease in relative expression levels of CIT2), (exogenous addition of 2-PE, decrease in relative expression levels of IDH1), (exogenous addition of 2-PE, decrease in relative expression levels of CIT1), (exogenous addition of 2-PE, inhibition of glyoxylic and tricarboxylic acid cycles), (exogenous addition of 2-PE, promotion of ethanol accumulation)\n",
      "Progress: 13.10% completed\n",
      "Response for Row 263:\n",
      "Answer to Question 2: (heterologous expression of mlrA, reduction of microcystins by 83%), (expression of mlrA from plasmid, reduction in toxin content), (linearization of microcystin structure by mlrA, reduction in toxicity and stability)\n",
      "Progress: 13.15% completed\n",
      "Response for Row 264:\n",
      "Answer to Question 2: (deletion of gene DDGS, consequence no mycosporines production), (deletion of gene OMT, consequence no mycosporines production), (deletion of gene ATPG, consequence no mycosporines production), (deletion of gene ATPG, consequence accumulation of 4-deoxygadusol), (expression of gene DDGS, consequence production of 4-deoxygadusol), (expression of gene OMT, consequence production of 4-deoxygadusol), (expression of gene DDGS, consequence production of MG), (expression of gene OMT, consequence production of MG), (expression of gene ATPG, consequence production of MG), (integration of cluster DDGS, OMT, ATPG, consequence production of MG), (integration of cluster DDGS, OMT, ATPG, consequence production of mycosporine glutaminol glucoside), (deletion of gene MIG1, consequence upregulation of mycosporinogenesis), (deletion of gene CYC8, consequence upregulation of mycosporinogenesis), (deletion of gene OPI1, consequence upregulation of mycosporinogenesis), (deletion of gene ROX1, consequence downregulation of mycosporinogenesis), (deletion of gene SKN7, consequence downregulation of mycosporinogenesis), (deletion of gene TUP6, consequence no effect on mycosporinogenesis), (deletion of gene YAP6, consequence no effect on mycosporinogenesis)\n",
      "Progress: 13.20% completed\n",
      "Response for Row 265:\n",
      "Answer to Question 2: (RhPLATZ9-silenced, accelerated senescence), (RhPLATZ9-silenced, higher ROS contents), (RhWRKY33a-silenced, accelerated senescence), (RhWRKY33a-silenced, higher ROS contents), (overexpression of RhWRKY33a, delayed flower senescence), (overexpression of RhPLATZ9, delayed flower senescence), (overexpression of RhPLATZ9, lower ROS accumulation), (RhPLATZ9-silenced, upregulated RhRbohs), (RhPLATZ9, represses RhRbohD)\n",
      "Progress: 13.25% completed\n",
      "Response for Row 266:\n",
      "Answer to Question 2: (deletion of gene eEF3, increase in product from downstream ORF), (deletion of gene eEF3, enhancement of release of luciferase from polysome by eRF1 and eRF3)\n",
      "Progress: 13.30% completed\n",
      "Response for Row 267:\n",
      "Answer to Question 2: (expression of gene sensor array, consequence low background), (expression of gene sensor array, consequence 40- to 5000-fold dynamic range), (expression of gene heterologous pathway, consequence identification of bottlenecks in metabolic pathway)\n",
      "Progress: 13.35% completed\n",
      "Response for Row 268:\n",
      "Answer to Question 2: (deletion of ppe-miR393a, increased PpTIR1), (deletion of ppe-miR393b, increased PpTIR1), (overexpression of PpTIR1, increased interaction with PpIAA13), (overexpression of PpIAA13, increased binding to PpACS1 promoter), (overexpression of PpIAA13, increased transcription of PpACS1), (overexpression of PpACS1, increased ethylene biosynthesis), (suppression of ppe-miR393a, decreased ethylene production), (suppression of ppe-miR393b, decreased ethylene production), (overexpression of ppe-miR393a, decreased ethylene production), (overexpression of ppe-miR393b, decreased ethylene production), (overexpression of PpIAA13, increased fruit softening), (suppression of PpIAA13, decreased fruit softening)\n",
      "Progress: 13.40% completed\n",
      "Response for Row 269:\n",
      "Answer to Question 2: (deletion of gene Cdc73, consequence alteration of Paf1C-dependent histone modifications genome-wide), (depletion of Spt6, consequence dissociation of Paf1 from chromatin)\n",
      "Progress: 13.45% completed\n",
      "Response for Row 270:\n",
      "Answer to Question 2: (deletion of gene daf-2, consequence decreased ROS level), (deletion of gene daf-16, consequence decreased ROS level), (deletion of gene daf-2, consequence increased GSH content), (deletion of gene daf-16, consequence increased GSH content), (deletion of gene daf-2, consequence reduced intestinal damage), (deletion of gene daf-16, consequence reduced intestinal damage)\n",
      "Progress: 13.50% completed\n",
      "Response for Row 271:\n",
      "Answer to Question 2: (no gene overexpression/deletion mentioned, no specific gene product or metabolite consequences mentioned)\n",
      "Progress: 13.55% completed\n",
      "Response for Row 272:\n",
      "Answer to Question 2: (deletion of gene luxS, consequence 1), (deletion of gene luxS, consequence 2), (deletion of gene pfs, consequence 1), (deletion of gene pfs, consequence 2)\n",
      "Progress: 13.60% completed\n",
      "Response for Row 273:\n",
      "Answer to Question 2: (expression of α-amylase, improved product titers), (expression of α-amylase, diminished production yields without control), (expression of α-amylase, reduced efficiency without control)\n",
      "Progress: 13.65% completed\n",
      "Response for Row 274:\n",
      "Answer to Question 2: (deletion of gene ScNRK1, consequence enzyme activity in fermentation broth 14.75 IU/mL), (expression of gene ScNRK1, consequence specific enzyme activity 2252.59 IU/mg), (immobilization of ScNRK1, consequence increase in optimum temperature by 10°C), (immobilization of ScNRK1, consequence improvement in temperature stability), (immobilization of ScNRK1, consequence activity remains above 80% after four cycles)\n",
      "Progress: 13.70% completed\n",
      "Response for Row 275:\n",
      "Answer to Question 2: (expression of gene Cas9-Brex27, increased efficiency of quadruple integration), (expression of gene Cas9-Brex27, attracted Rad51 recombinase to double-strand breaks), (integration of four genes in mogrol biosynthetic pathway, achieved through CMI system)\n",
      "Progress: 13.75% completed\n",
      "Response for Row 276:\n",
      "Answer to Question 2: (deletion of gene MPC1, no significant impact on lactic acid production), (deletion of gene MPC1, negatively affected cell growth), (expression of gene VHb, detected in aerobic cultivation), (blocking cell respiration using hypoxia, suitable for producing lactic acid)\n",
      "Progress: 13.80% completed\n",
      "Response for Row 277:\n",
      "Answer to Question 2: (overexpression of surveillance systems, consequence: disposal of incomplete nascent chains), (deletion of proper mRNA processing, consequence: stalled translating ribosomes), (overexpression of co-translational quality control systems, consequence: elimination of harmful products from aberrant translation)\n",
      "Progress: 13.85% completed\n",
      "Response for Row 278:\n",
      "Answer to Question 2: (expression of gene MrBBS, increased (-)-α-bisabolol production), (deletion of gene ERG9, reduced competitive pathway), (fusion of genes ERG20 and MrBBS, increased (-)-α-bisabolol titer), (overexpression of gene PDR15, increased extracellular (-)-α-bisabolol production)\n",
      "Progress: 13.90% completed\n",
      "Response for Row 279:\n",
      "Answer to Question 2: (deletion of H2A.Z C terminus, consequence: reduced occupation level of H2A.Z), (deletion of H2A.Z C terminus, consequence: improper positioning of targeted nucleosomes), (overexpression of ZA protein, consequence: altered nucleosome positioning depending on the locus), (overexpression of ZA-rII', consequence: enhanced transcriptional activation through interaction with chromatin-remodeling entities and/or transcription modulators), (deletion of H2A.Z C terminus, consequence: disruption of gene regulation primarily in genes without H2A.Z nucleosome in the promoter), (interaction with SWR1-C, consequence: localization of H2A.Z derivatives at specific loci)\n",
      "Progress: 13.95% completed\n",
      "Response for Row 280:\n",
      "Answer to Question 2: (deletion of gene TrLYS9, lysine auxotrophy), (deletion of gene TrLYS9, impaired vegetative growth), (deletion of gene TrLYS9, impaired conidiation), (deletion of gene TrLYS9, impaired conidial germination), (deletion of gene TrLYS9, impaired lysine biosynthesis), (deletion of gene TrLYS9, impaired cellulase production)\n",
      "Progress: 14.00% completed\n",
      "Response for Row 281:\n",
      "Answer to Question 2: (no specific gene expression/deletion mentioned, no direct consequences provided)\n",
      "Progress: 14.05% completed\n",
      "Response for Row 282:\n",
      "Answer to Question 2: (depletion of Not1, mRNAs become soluble), (depletion of Not1, mitochondrial mRNAs become soluble), (depletion of Not4, mRNAs with lower non-optimal codon content become soluble), (depletion of Not4, mRNAs with higher expression become soluble), (depletion of Not1, impacts ribosome dwelling according to codon optimality for soluble mRNAs), (depletion of Not4, impacts ribosome dwelling according to codon optimality for soluble mRNAs)\n",
      "Progress: 14.10% completed\n",
      "Response for Row 283:\n",
      "Answer to Question 2: (overexpression of gene TpCHS1, consequence abnormal cell morphology), (overexpression of gene TpCHS1, consequence reduced growth rates), (overexpression of gene TpCHS1, consequence inhibition of the G2/M phase)\n",
      "Progress: 14.15% completed\n",
      "Response for Row 284:\n",
      "Answer to Question 2: (deletion of gene CDS6, decreased mitochondrial PG biosynthesis capacity), (deletion of gene CDS6, decreased mitochondrial CL biosynthesis capacity), (deletion of gene CDS6, severe growth deficiency), (deletion of gene CDS6, plant death), (deletion of gene CDS6, abnormal mitochondrial ultrastructure), (deletion of gene CDS6, missing cristae structures), (deletion of gene CDS6, impaired TAG degradation in lipid droplets), (deletion of gene CDS6, impaired starch degradation in chloroplasts), (deletion of gene CDS6, upregulation of genes involved in mitochondrial electron transport chain), (deletion of gene CDS6, altered polar glycerolipids content), (deletion of gene CDS6, loss of cell proliferation capacity), (deletion of gene CDS6, higher oxidase activity)\n",
      "Progress: 14.20% completed\n",
      "Response for Row 285:\n",
      "Answer to Question 2: (expression of CmPT2, recovery of yeast strain MB192 growth under LP conditions), (mutation of G84A, inability to grow normally under LP conditions), (mutation of G222A, inability to grow normally under LP conditions), (mutation of T239A, inability to grow normally under LP conditions), (mutation of Y242A, inability to grow normally under LP conditions), (mutation of N422A, inability to grow normally under LP conditions), (overexpression of CmPT2, increase in plant height), (overexpression of CmPT2, increase in effective panicle numbers), (overexpression of CmPT2, increase in branch numbers), (overexpression of CmPT2, increase in yield), (overexpression of Cmphts, decrease in plant height), (overexpression of Cmphts, decrease in effective panicle numbers), (overexpression of CmPT2, higher Pi content in roots under HP conditions), (overexpression of CmPT2, higher Pi content in roots under LP conditions), (overexpression of N422A, lower Pi content in leaves under HP conditions), (overexpression of N422A, lower Pi content in leaves under LP conditions), (overexpression of N422A, lower Pi content in roots under HP conditions), (overexpression of N422A, lower Pi content in roots under LP conditions), (overexpression of CmPT2, inhibition of PSR genes transcription under LP conditions), (overexpression of T239A, different expression pattern of phosphorus-related genes under LP conditions), (overexpression of Y242A, different expression pattern of phosphorus-related genes under LP conditions), (overexpression of N422A, different expression pattern of phosphorus-related genes under LP conditions)\n",
      "Progress: 14.25% completed\n",
      "Response for Row 286:\n",
      "Answer to Question 2: (deletion of gene Huura3, consequence 1), (deletion of gene Huhis3, consequence 1), (deletion of gene Huleu2, consequence 1), (deletion of gene Huade2, consequence 1)\n",
      "Progress: 14.30% completed\n",
      "Response for Row 287:\n",
      "Answer to Question 2: (deletion of APS2, delayed endocytosis), (overexpression of DID2, cell cycle arrest), (deletion of profilin, increased F-actin), (overexpression of coronin, favor assembly of actin filaments)\n",
      "Progress: 14.35% completed\n",
      "Response for Row 288:\n",
      "Answer to Question 2: (expression of gene LmMTP1, increased Zn tolerance), (expression of gene LmMTP1, increased Co tolerance), (expression of gene LmMTP1, increased Cd tolerance), (expression of gene LmMTP1, increased Zn content), (expression of gene LmMTP1, increased Co content), (expression of gene LmMTP1, decreased Cd content)\n",
      "Progress: 14.40% completed\n",
      "Response for Row 289:\n",
      "Answer to Question 2: (deletion of nitrogen, consequence 1: decreased fermentation efficiency), (deletion of nitrogen, consequence 2: altered aroma production), (overexpression of nitrogen, consequence 1: increased transcript levels of genes involved in ethyl and acetate esters production), (overexpression of nitrogen, consequence 2: up-regulation of BGL2 gene), (overexpression of nitrogen, consequence 3: up-regulation of EXG1 gene), (overexpression of nitrogen, consequence 4: increased terpenol concentration)\n",
      "Progress: 14.45% completed\n",
      "Response for Row 290:\n",
      "Answer to Question 2: (expression of eponemycin BGC, production of TMC-86A), (expression of eponemycin BGC, biosynthesis of 4,5-dehydro-l-leucine), (expression of TMC-86A BGC, production of TMC-86A), (expression of TMC-86A BGC, biosynthesis of 4,5-dehydro-l-leucine)\n",
      "Progress: 14.50% completed\n",
      "Response for Row 291:\n",
      "Answer to Question 2: (deletion of gene SUT125, down-regulation of PIL1), (deletion of gene PIL1, increased secretion of invertase), (deletion of gene LSP1, increased secretion of invertase), (deletion of gene SUR7, increased secretion of invertase), (deletion of gene PIL1, increased secretion of GFP), (deletion of gene PIL1, increased secretion of Pexelizumab scFv), (deletion of gene PIL1, up-regulation of sphingolipid biosynthesis), (deletion of gene PIL1, up-regulation of serine biosynthesis)\n",
      "Progress: 14.55% completed\n",
      "Response for Row 292:\n",
      "Answer to Question 2: (deletion of gene FoRSR1, reduction in mycelial dry weight), (deletion of gene FoRSR1, blunted and swollen hyphae), (deletion of gene FoRSR1, fewer and morphologically abnormal conidia), (deletion of gene FoRSR1, defective chlamydospore formation), (deletion of gene FoRSR1, reduced virulence), (deletion of gene FoRSR1, limited necrosis at wounding sites), (deletion of gene FoRSR1, affected pigmentation), (deletion of gene FoRSR1, affected autophagy), (deletion of gene FoRSR1, reduced production of fusaric acid), (deletion of gene FoRSR1, downregulation of genes involved in secondary metabolism)\n",
      "Progress: 14.60% completed\n",
      "Response for Row 293:\n",
      "Answer to Question 2: (deletion of gene NOD-like receptor signalling pathway, consequence decreased immune response)\n",
      "(deletion of gene influenza A, consequence decreased immune response)\n",
      "(deletion of gene cytokine-cytokine receptor interaction, consequence decreased immune response)\n",
      "(deletion of gene haematopoietic cell lineage, consequence decreased immune response)\n",
      "(deletion of gene African trypanosomiasis, consequence decreased immune response)\n",
      "(overexpression of gene selenoprotein W, consequence increased antioxidant activity)\n",
      "(overexpression of gene glutathione peroxidase 1, consequence increased antioxidant activity)\n",
      "(overexpression of gene glutathione S-transferase A1, consequence increased antioxidant activity)\n",
      "(overexpression of gene tumour necrosis factor, consequence increased inflammatory response)\n",
      "(overexpression of gene tumour necrosis factor superfamily member 10, consequence increased inflammatory response)\n",
      "(overexpression of gene tumour necrosis factor superfamily member 8, consequence increased inflammatory response)\n",
      "(overexpression of gene tumour necrosis factor superfamily member 13b, consequence increased inflammatory response)\n",
      "Progress: 14.65% completed\n",
      "Response for Row 294:\n",
      "Answer to Question 2: (deletion of gene ADE2, precision editing), (deletion of gene ADE2, large DNA region replacement)\n",
      "Progress: 14.70% completed\n",
      "Response for Row 295:\n",
      "Answer to Question 2: (transcription termination of gene, production of mature mRNA), (transcription termination of gene, export of mRNA from nucleus), (premature transcription termination of gene, altered gene expression), (transcription readthrough of gene, altered gene expression)\n",
      "Progress: 14.75% completed\n",
      "Response for Row 296:\n",
      "Answer to Question 2: (deletion of ubiquitination sites on eS10, failure to perform NGD), (deletion of ubiquitination sites on uS10, failure to perform NGD), (action of nuclease NONU-1, occurs after ZNF-598 activity), (involvement of ribosome rescue factors HBS-1/PELO-1, facilitates mRNA decay via NONU-1)\n",
      "Progress: 14.80% completed\n",
      "Response for Row 297:\n",
      "Answer to Question 2: (deletion of gene HXT2.4, improved co-assimilation of glucose and xylose), (expression of mutant HXT2.4, improved co-assimilation of glucose and xylose)\n",
      "Progress: 14.85% completed\n",
      "Response for Row 298:\n",
      "Answer to Question 2: (overexpression of gene PvAvh77, non-specific cell death in Vitis riparia), (overexpression of gene PvAvh77, non-specific cell death in Nicotiana benthamiana), (overexpression of gene PvAvh77, non-specific cell death in Nicotiana tabacum), (overexpression of gene PvAvh77, neutralization of host immunity in Vitis vinifera), (overexpression of gene PvAvh77, enhancement of P. viticola colonization in Vitis vinifera), (truncation of gene PvAvh77 to PvAvh77-M2, increased induction of cell death in Nicotiana benthamiana), (truncation of gene PvAvh77 to PvAvh77-M2, increased induction of cell death in Vitis riparia), (application of exogenous PvAvh77-M2, increased accumulation of salicylic acid in grapevine), (application of exogenous PvAvh77-M2, increased accumulation of H2O2 in grapevine), (application of exogenous PvAvh77-M2, reduced infection of P. viticola in grapevine)\n",
      "Progress: 14.90% completed\n",
      "Response for Row 299:\n",
      "Answer to Question 2: (expression of D-lactate dehydrogenase, increased D-lactic acid productivity), (expression of glycolytic genes, increased D-lactic acid productivity)\n",
      "Progress: 14.95% completed\n",
      "Response for Row 300:\n",
      "Answer to Question 2: (deletion of gene not specified, consequence improved targeted chromosomal integration), (expression of foreign gene in Saccharomyces cerevisiae, consequence in-frame expression of foreign protein), (use of CRISPR-Cas9, consequence 74% efficiency in targeted chromosomal integration), (use of 15 bp microhomology flanked arms, consequence 50% targeted knock-in), (optimization of CRISPR-Cas9 method, consequence no off-target effects)\n",
      "Progress: 15.00% completed\n",
      "Response for Row 301:\n",
      "Answer to Question 2: (expression of gene interleukin 15, consequence increased expression due to RAC feeding), (deletion of gene interleukin 8, consequence decreased expression due to RAC feeding), (expression of gene interleukin 6, consequence tended to decrease in SCFP29 and SCFP57 on day -13)\n",
      "Progress: 15.05% completed\n",
      "Response for Row 302:\n",
      "Answer to Question 2: (heterologous expression of gene EgrGH64, production of recombinant protein EgrGH64), (heterologous expression of gene EgrGH64, hydrolysis of laminarin), (heterologous expression of gene EgrGH64, hydrolysis of paramylon), (heterologous expression of gene EgrGH64, hydrolysis of ramified β-glucan), (heterologous expression of gene EgrGH64, production of laminaripentaose), (heterologous expression of gene EgrGH64, inhibition of fungal growth)\n",
      "Progress: 15.10% completed\n",
      "Response for Row 303:\n",
      "Answer to Question 2: (deletion of gene sod1, improvement in ATP supply), (deletion of gene sod1, 22.3% improvement in SAM content), (deletion of gene sod1, positive regulation of SAM synthesis), (deletion of gene sod1, enhancement of substrate assimilation leading to increased SAM production)\n",
      "Progress: 15.15% completed\n",
      "Response for Row 304:\n",
      "Answer to Question 2: (deletion of gene STSs, consequence reduced sesquiterpenoid production), (expression of gene PoSTS-06, consequence production of pleostene)\n",
      "Progress: 15.20% completed\n",
      "Response for Row 305:\n",
      "Answer to Question 2:  (expression of gene CsAMT2.2, consequence NH4+ uptake), (expression of gene CsAMT2.3, consequence NH4+ uptake), (expression of gene CsAMT2.2, consequence upregulation under high NH4+ treatment), (expression of gene CsAMT2.3, consequence upregulation under high NH4+ treatment), (expression of gene CsAMT2.2, consequence high expression under low NH4+ treatment), (expression of gene CsAMT2.3, consequence high expression under low NH4+ treatment), (expression of gene CsAMT2.3, consequence higher transport efficiency than CsAMT2.2)\n",
      "Progress: 15.25% completed\n",
      "Response for Row 306:\n",
      "Answer to Question 2: (overexpression of MdTCP17, inhibited AR formation), (overexpression of MdWOX11, promoted AR primordium formation), (interference of MdWOX11, inhibited AR primordium production), (overexpression of MdTCP17, reduced binding of MdWOX11 and MdLBD29 promoters), (coexpression of MdTCP17 and MdWOX11, reduced MdLBD29 expression)\n",
      "Progress: 15.30% completed\n",
      "Response for Row 307:\n",
      "Answer to Question 2: (deletion of gene not applicable, cell communication modulated by light-dependent production of α-factor pheromone), (expression of gene luciferase, activated upon constant blue light), (deletion of gene not applicable, response level varies with dark/blue-light transitions), (deletion of gene not applicable, amplitude of response tuned by modifying strain ratio), (deletion of gene not applicable, higher fold inductions in two-population system compared to autonomous strains)\n",
      "Progress: 15.35% completed\n",
      "Response for Row 308:\n",
      "Answer to Question 2: (deletion of gene mismatch, consequence fatty alcohol production compromised), (expression of gene MaFAR1, consequence improved fatty alcohol biosynthesis 1.62-fold), (enhancement of peroxisomal supply of acyl-CoA, consequence improved fatty alcohol production), (enhancement of peroxisomal supply of NADPH, consequence improved fatty alcohol production 2.52 times higher than the starting strain)\n",
      "Progress: 15.40% completed\n",
      "Response for Row 309:\n",
      "Answer to Question 2: (deletion of endogenous galactose metabolic pathway genes, improved production of 2'-FL), (deletion of endogenous galactose metabolic pathway genes, improved production of 3-FL), (expression of Kα, enhanced expression of pathway genes), (expression of Gal4p, enhanced expression of pathway genes), (enhanced expression of pathway genes, increased production of 2'-FL), (enhanced expression of pathway genes, increased production of 3-FL)\n",
      "Progress: 15.45% completed\n",
      "Response for Row 310:\n",
      "Answer to Question 2: (overexpression of gene TOMM40, consequence late onset Alzheimer's Disease), (overexpression of gene Tom40, consequence caspase-dependent cell death), (overexpression of gene Tom40, consequence degeneration of neuronal eye tissue), (overexpression of gene Tom40, consequence upregulation of DmHsp22), (overexpression of gene Tom40, consequence activation of regulated death pathways)\n",
      "Progress: 15.50% completed\n",
      "Response for Row 311:\n",
      "Answer to Question 2: (deletion of gene Fir1, impaired immune responses), (deletion of gene Fir1, reduced resistance to Pseudomonas syringae pv. tomato DC3000), (deletion of gene Fir1, decreased production of reactive oxygen species), (deletion of gene Fir1, reduced PATHOGENESIS-RELATED 1b (PR1b) gene expression), (deletion of gene Fir1, increased Pst DC3000 population growth), (deletion of gene Fir1, failure to close stomata upon infection), (deletion of gene Fir1, altered expression of jasmonic acid-related genes), (deletion of gene Fir1, deregulation of JA response), (deletion of gene Fir1, increased susceptibility to Pst DC3000 when inoculated by dipping), (deletion of gene Fir1, increased resistance to Botrytis cinerea following PTI activation)\n",
      "Progress: 15.55% completed\n",
      "Response for Row 312:\n",
      "Answer to Question 2: (deletion of gene Achog1, delayed conidia production), (deletion of gene Achog1, reduced conidia number), (deletion of gene Achog1, increased sensitivity to hyperosmolarity), (deletion of gene Achog1, increased sensitivity to oxidative stress), (deletion of gene Achog1, inhibited growth under alkaline stress), (deletion of gene Achog1, decreased pigmentation), (deletion of gene Achog1, suppressed expression of genes related to asexual development), (deletion of gene Achog1, suppressed expression of genes related to osmotic stress resistance), (deletion of gene Achog1, suppressed expression of genes related to oxidative stress resistance)\n",
      "Progress: 15.60% completed\n",
      "Response for Row 313:\n",
      "Answer to Question 2: (deletion of gene DBP1, reduced expression of MRP51), (deletion of gene DBP1, function reduction of MRP51), (insertion of selection cassette, repression of MRP51), (bidirectional promoter activity, production of divergent transcript), (divergent transcript, transcriptional interference of MRP51), (divergent transcript, translational repression of MRP51), (production of LUTI, repression of MRP51)\n",
      "Progress: 15.65% completed\n",
      "Response for Row 314:\n",
      "Answer to Question 2: (deletion of glucose, consequence reduced growth rate), (overexpression of PKA, consequence increased internal growth capacities), (overexpression of Snf1, consequence down-regulation of stress response)\n",
      "Progress: 15.70% completed\n",
      "Response for Row 315:\n",
      "Answer to Question 2:  (deletion of gene hap4, improved growth rate in glucose-maltose shift), (deletion of gene hap5, improved growth rate in glucose-maltose shift), (deletion of gene cin5, improved growth rate in glucose-maltose shift), (lower transcriptional activity of maltose metabolism genes, lower fermentation efficiency in QC18 strain), (higher expression levels of glucose transporters, longer lag phase in QC18 strain), (allelic variants of CIN5, activation of repressor of diauxic shift in QC18 strain)\n",
      "Progress: 15.75% completed\n",
      "Response for Row 316:\n",
      "Answer to Question 2: (expression of gene ScAMT2;1, consequence increased ammonium uptake by roots), (expression of gene ScAMT2;1, consequence increased nitrogen translocation to shoots), (expression of gene ScAMT2;1, consequence increased biomass production), (expression of gene ScAMT2;1, consequence K<sub>m</sub> of 90.17 µM), (expression of gene ScAMT2;1, consequence V<sub>max</sub> of 338.99 µmoles h<sup>-1</sup> g<sup>-1</sup> root DW)\n",
      "Progress: 15.80% completed\n",
      "Response for Row 317:\n",
      "Answer to Question 2: (deletion of TdGASA genes, impaired growth and development), (deletion of TdGASA genes, impaired flower induction), (deletion of TdGASA genes, impaired seed development), (overexpression of TdGASA1, enhanced salt tolerance), (overexpression of TdGASA4, enhanced osmotic tolerance), (overexpression of TdGASA14, enhanced oxidative tolerance), (overexpression of TdGASA19, enhanced heat tolerance)\n",
      "Progress: 15.85% completed\n",
      "Response for Row 318:\n",
      "Answer to Question 2: (expression of IL-1β, immunomodulation), (expression of IL-6, immunomodulation), (expression of INF-γ, immunomodulation)\n",
      "Progress: 15.90% completed\n",
      "Response for Row 319:\n",
      "Answer to Question 2: (deletion of gene FaSmi1, consequence mycelial growth defects), (deletion of gene FaSmi1, consequence asexual reproduction defects), (deletion of gene FaSmi1, consequence virulence reduction), (deletion of gene FaSmi1, consequence increased sensitivity to osmotic stresses), (deletion of gene FaSmi1, consequence increased tolerance to oxidative stresses), (deletion of gene FaSmi1, consequence increased tolerance to cell wall integrity inhibitors), (deletion of gene FaSmi1, consequence lower expression of antioxidant system-associated genes), (deletion of gene FaSmi1, consequence decreased DON content)\n",
      "Progress: 15.95% completed\n",
      "Response for Row 320:\n",
      "Answer to Question 2: (deletion of gene VdERG2, reduced ergosterol content), (deletion of gene VdERG2, decreased conidial germination rate), (deletion of gene VdERG2, decreased conidial yield), (deletion of gene VdERG2, abnormal conidia production), (deletion of gene VdERG2, decreased melanin production), (deletion of gene VdERG2, reduced microsclerotia production), (deletion of gene VdERG2, decreased expression of genes VaflM, Vayg1, VDH1, VdLAC, VdSCD, VT4HR), (deletion of gene VdERG2, increased sensitivity to congo red), (deletion of gene VdERG2, increased sensitivity to sodium dodecyl sulfate), (deletion of gene VdERG2, increased sensitivity to hydrogen peroxide), (deletion of gene VdERG2, weakened mycelium penetration ability), (deletion of gene VdERG2, impaired pathogenicity)\n",
      "Progress: 16.00% completed\n",
      "Response for Row 321:\n",
      "Answer to Question 2: (deletion of gene N/A, consequence Colocalization of enzymes increases pathway flux and synthesis of nonnative products), (expression of gene Pex15, consequence Enables display of pathway enzymes on the surface of the peroxisome), (expression of gene Cat2, consequence Converts acetyl-carnitine to acetyl-CoA upon export, increasing acetyl-CoA levels 3.7-fold), (expression of gene Acc1, consequence Converts acetyl-CoA to malonyl-CoA), (expression of gene 2-pyrone synthase, consequence Increases synthesis of the polyketide triacetic acid lactone by 21% over cytosolic expression), (overexpression of gene Pex11, consequence Increases peroxisomal membrane area, enhancing access to acetyl-CoA pool and synthesis of acetyl-CoA-based products)\n",
      "Progress: 16.05% completed\n",
      "Response for Row 322:\n",
      "Answer to Question 2: (deletion of gene cyclooxygenase-2, consequence decreased mRNA expression in dogs fed SCFP), (deletion of gene malondialdehyde, consequence decreased mRNA expression in dogs fed SCFP), (overexpression of gene cyclooxygenase-2, consequence increased mRNA expression in control dogs), (overexpression of gene malondialdehyde, consequence increased mRNA expression in control dogs)\n",
      "Progress: 16.10% completed\n",
      "Response for Row 323:\n",
      "Answer to Question 2: (deletion of MS2-MCP system, activation of nonsense-mediated mRNA decay pathway), (expression of improved MS2-MCP system, increase in efficiency of translation termination of tagged mRNAs)\n",
      "Progress: 16.15% completed\n",
      "Response for Row 324:\n",
      "Answer to Question 2: (deletion of gene Dot6, slower growth acclimation), (activation of gene Dot6, faster production of Ctt1 protein)\n",
      "Progress: 16.20% completed\n",
      "Response for Row 325:\n",
      "Answer to Question 2: (deletion of gene yil014c-a, improved ncAA incorporation efficiency), (deletion of gene alo1Δ, improved ncAA incorporation efficiency)\n",
      "Progress: 16.25% completed\n",
      "Response for Row 326:\n",
      "Answer to Question 2: (deletion of gene atg8, higher sensitivity to ER stress), (deletion of gene atg8, attenuation of UPR gene expression), (deletion of gene ire1, higher sensitivity to ER stress), (deletion of gene ire1, attenuation of UPR gene expression), (deletion of gene atg8, defect in ER-phagy), (deletion of gene ire1, defect in ER-phagy), (deletion of gene atg8, attenuated vacuolar fusion), (deletion of gene ire1, attenuated vacuolar fusion)\n",
      "Progress: 16.30% completed\n",
      "Response for Row 327:\n",
      "Answer to Question 2:  (overexpression of gene CYP79A2, consequence 2-fold increase in BGLS production), (overexpression of gene CYP83B1, consequence 2-fold increase in BGLS production), (overexpression of gene SOT16, consequence 1.7-fold increase in BGLS production), (introduction of gene APK1, consequence 1.7-fold increase in BGLS production), (overexpression of gene MET3, consequence 2.4-fold increase in BGLS production), (overexpression of gene MET14, consequence 12.81 μmol/L BGLS production)\n",
      "Progress: 16.35% completed\n",
      "Response for Row 328:\n",
      "Answer to Question 2: (deletion of gene GmIFN-γ, consequence strong signal not detected until 40 days post hatching), (expression of gene GmIFN-γ, consequence strong antivirus effect), (expression of gene GmIFN-γ, consequence high transcript levels of MHC class I chain-related protein A (MICA) gene), (expression of gene GmIFN-γ, consequence upregulation of zebrafish IFN-γ transcript)\n",
      "Progress: 16.40% completed\n",
      "Response for Row 329:\n",
      "Answer to Question 2: (overexpressing artemisinin biosynthesis pathway genes, improving artemisinin content), (blocking key genes in competitive pathways, improving artemisinin content), (regulating the expression of transcription factors related to artemisinin biosynthesis, improving artemisinin content)\n",
      "Progress: 16.45% completed\n",
      "Response for Row 330:\n",
      "Answer to Question 2: (overexpression of gene BUD1, increased surface-displayed α-amylase activity), (overexpression of gene CDC42, increased surface-displayed α-amylase activity), (overexpression of gene AXL1, increased surface-displayed α-amylase activity), (overexpression of gene BUD10, increased surface-displayed α-amylase activity), (overexpression of gene BUD1, enhanced secreted α-amylase activity), (overexpression of gene BUD3, enhanced secreted α-amylase activity), (overexpression of gene BUD4, enhanced secreted α-amylase activity), (overexpression of gene BUD7, enhanced secreted α-amylase activity), (overexpression of gene BUD10, enhanced secreted α-amylase activity), (overexpression of gene BUD1, increased surface-displayed α-amylase by 56%), (overexpression of gene BUD1, increased secreted α-amylase by 49%), (co-overexpression of gene BUD1 and CDC42, increased α-amylase surface display by 100%), (two genomic copies of gene BUD1, improved α-amylase secretion by 92%)\n",
      "Progress: 16.50% completed\n",
      "Response for Row 331:\n",
      "Answer to Question 2: (expression of Y. lipolytica ACC, higher accumulation of malonyl-CoA), (expression of Y. lipolytica ACC, decreased growth rate), (concomitant expression of homologous and heterologous ACC1 genes, elimination of growth defect), (concomitant expression of homologous and heterologous ACC1 genes, marginal reduction of malonyl-CoA accumulation)\n",
      "Progress: 16.55% completed\n",
      "Response for Row 332:\n",
      "Answer to Question 2: (expression of Hog1, activation of autophagy-related genes), (expression of Hog1, accumulation of lipid droplets), (expression of Hog1, trigger vacuole calcium release), (deletion of Hog1, shrinkage in the cell wall), (deletion of Hog1, condensation of the cytoplasmic part), (deletion of Hog1, high-level ROS production), (leptospiral LPS, Hog1 phosphorylation), (Hog1 phosphorylation, activation of autophagy-related genes)\n",
      "Progress: 16.60% completed\n",
      "Response for Row 333:\n",
      "Answer to Question 2: (decreased expression of gene IL6, consequence 1), (decreased expression of gene TNF, consequence 1), (decreased expression of gene TGFB, consequence 1), (decreased expression of gene CLDN3, consequence 1), (decreased expression of gene MUC2, consequence 1)\n",
      "Progress: 16.65% completed\n",
      "Response for Row 334:\n",
      "Answer to Question 2: (deletion of gene Gal1, consequence 1), (deletion of gene Gal1, consequence 2)\n",
      "Progress: 16.70% completed\n",
      "Response for Row 335:\n",
      "Answer to Question 2: (constitutive expression of HAC1i, growth retardation), (constitutive expression of HAC1i, frequent yield of fast-growing and low-UPR progeny), (weak ER stress induction, rescued growth of HAC1i cells), (expression of ER-located GFP, stable strong UPR activity), (expression of ER-located GFP, highly expanded ER), (expression of ER-located GFP, abundant production of triglycerides), (expression of ER-located GFP, abundant production of heterogenous carotenoids)\n",
      "Progress: 16.75% completed\n",
      "Response for Row 336:\n",
      "Answer to Question 2:  (deletion of gene ARS, consequence integration efficiency increase), (deletion of gene ARS, consequence cell fitness impact), (overexpression of gene β-carotene pathway, consequence increased β-carotene production)\n",
      "Progress: 16.80% completed\n",
      "Response for Row 337:\n",
      "Answer to Question 2: (expression of gene nrps33, accumulation of anthranilic acid), (expression of gene nrps33, affected transcription of tryptophan biosynthesis genes), (expression of gene nrps33, inhibition of Trp4 activity), (inhibition of Trp4 activity, accumulation of anthranilic acid), (accumulation of anthranilic acid, down-regulation of downstream tryptophan biosynthesis genes)\n",
      "Progress: 16.85% completed\n",
      "Response for Row 338:\n",
      "Answer to Question 2: (deletion of gene nhp6AΔBΔ, consequence SNR6 transcription restored), (deletion of gene nhp6AΔBΔ, consequence fitness restored), (deletion of gene nhp6AΔBΔ, consequence heat sensitivity reversed), (deletion of gene hmbAΔ, consequence ascospore formation affected), (deletion of gene hmbAΔ, consequence utilization of various carbon- and nitrogen-sources affected), (deletion of gene hmbAΔ, consequence radial growth rate affected), (deletion of gene hmbAΔ, consequence hypha elongation by polarized growth affected), (deletion of gene hmbAΔ, consequence sterigmatocystin production not complemented), (expression of gene HmbA, consequence utilization of various carbon- and nitrogen-sources influenced), (expression of gene HmbA, consequence stress tolerance influenced), (expression of gene HmbA, consequence secondary metabolism influenced), (expression of gene HmbA, consequence hyphae elongation influenced), (expression of gene HmbA, consequence maintenance of polarized growth influenced), (expression of gene HmbA, consequence normal expression of endochitinase chiA influenced)\n",
      "Progress: 16.90% completed\n",
      "Response for Row 339:\n",
      "Answer to Question 2: (introduction of t26AaLS1, increase in linalool production), (introduction of IUP, increase in IPP/DMAPP production), (integration of IDI1, increase in metabolic flux of IPP), (integration of ERG20<sup>F96W-N127W</sup>, increase in metabolic flux of IPP), (copy number variation of t26AaLS1, increase in linalool titer), (copy number variation of ERG20<sup>F96W-N127W</sup>, increase in linalool titer), (optimization of isoprenol and prenol concentrations, increase in linalool titer), (addition of Mg<sup>2+</sup>, increase in linalool titer)\n",
      "Progress: 16.95% completed\n",
      "Response for Row 340:\n",
      "Answer to Question 2: (expression of gene Ura1, supports synthesis of pyrimidines), (expression of gene from Alternaria alternata, supported growth of S. cerevisiae ura1Δ mutant), (expression of gene from Schizophyllum commune, supported growth of S. cerevisiae ura1Δ mutant), (expression of gene encoding dihydrouracil oxidases, shows uracil- and thymine-dependent oxygen consumption), (expression of gene encoding dihydrouracil oxidases, shows hydrogen-peroxide production), (expression of gene encoding dihydrouracil oxidases, catalyses reaction dihydrouracil + O₂ → uracil + H₂O₂), (expression of DHO, strains were highly sensitive to 5-fluorodihydrouracil), (expression of DHO, plasmids were readily lost during growth on 5F-dhu-containing media)\n",
      "Progress: 17.00% completed\n",
      "Response for Row 341:\n",
      "Answer to Question 2: (deletion of cytokines, consequence under-regulation in the lung), (overexpression of Saccharomyces cerevisiae boulardii CNCM I-1079, consequence improvement in piglet performance during lactation), (overexpression of Saccharomyces cerevisiae boulardii CNCM I-1079, consequence reduction in inflammatory status of the lungs), (deletion of IgA producing cells, consequence fewer cells in the lymph node)\n",
      "Progress: 17.05% completed\n",
      "Response for Row 342:\n",
      "Answer to Question 2: (no expression/deletion of gene mentioned, no specific consequences related to gene expression/deletion)\n",
      "Progress: 17.10% completed\n",
      "Response for Row 343:\n",
      "Answer to Question 2: (deletion of gene UBR1, consequence alters abundance of 36 proteins), (overexpression of gene UBR1, consequence increases UPS activity), (deletion of gene DOA10, consequence reduces recognition of cellular proteins), (overexpression of gene NTA1, consequence increases processing of cellular proteins), (deletion of gene UBC6, consequence reduces ubiquitination of cellular proteins)\n",
      "Progress: 17.15% completed\n",
      "Response for Row 344:\n",
      "Answer to Question 2: (deletion of gene ERG28, reduced total cholesterol levels), (deletion of gene ERG28, 60-75% reduction in the rate of cholesterol synthesis), (expression of gene ERG28, rescued reduction in the rate of cholesterol synthesis), (deletion of gene ERG28, impaired activation of SREBP-2)\n",
      "Progress: 17.20% completed\n",
      "Response for Row 345:\n",
      "Answer to Question 2: (deletion of gene SD38, reduced cell length), (deletion of gene SD38, affected lipid metabolism), (deletion of gene SD38, reduced content of ACC), (overexpression of VLCFA C24:0, increased expression of OsACS3), (overexpression of VLCFA C24:0, increased expression of OsACS4), (overexpression of VLCFA C24:0, increased expression of OsACO7), (overexpression of VLCFA C24:0, increased plant height), (exogenous application of ACC, increased plant height), (exogenous application of ethephon, increased plant height)\n",
      "Progress: 17.25% completed\n",
      "Response for Row 346:\n",
      "Answer to Question 2: (deletion of gene Acpbs2, delayed conidia production), (deletion of gene Acpbs2, reduced conidia number in hypertonic media), (deletion of gene Acpbs2, increased sensitivity to 3 M NaCl), (deletion of gene Acpbs2, increased sensitivity to 3 M sucrose), (deletion of gene Acpbs2, increased sensitivity to 3 M sorbitol), (deletion of gene Acpbs2, increased sensitivity to high oxidative stress), (deletion of gene Acpbs2, inhibited growth under alkaline-pH stress), (deletion of gene Acpbs2, lighter colony color), (deletion of gene Acpbs2, altered expression of genes related to conidial development), (deletion of gene Acpbs2, altered expression of genes related to osmotic stress), (deletion of gene Acpbs2, altered expression of genes related to oxidative stress), (deletion of gene Acpbs2, altered expression of genes related to carbon metabolism), (deletion of gene Acpbs2, disrupted interaction with Achog1)\n",
      "Progress: 17.30% completed\n",
      "Response for Row 347:\n",
      "Answer to Question 2: (recombinant expression of TV-AFB<sub>1</sub>D, AFB<sub>1</sub> degradation efficiency increased), (recombinant expression of TV-AFB<sub>1</sub>D, AFB<sub>1</sub> concentration decreased to 32.6 μg/ml), (expression of AOX1 in S. cerevisiae, AFB<sub>1</sub> content decreased to 3.4 μg/g), (expression of CaMV 35S promoter in S. cerevisiae, AFB<sub>1</sub> content decreased to 2.9 μg/g), (expression of TV-AFB<sub>1</sub>D in S. cerevisiae, ethanol production increased)\n",
      "Progress: 17.35% completed\n",
      "Response for Row 348:\n",
      "Answer to Question 2: (expression of gene Kluyveromyces marxianus TPI, constitutive expression in Pichia pastoris), (expression of gene Hansenula polymorpha PMA, constitutive expression in Pichia pastoris), (expression of gene Candida tropicalis ICL, inducible expression in Pichia pastoris), (expression of gene Saccharomyces cerevisiae CUP, inducible expression in Pichia pastoris), (expression of gene ScCUP, responsive to Cu<sup>2+</sup> in Pichia pastoris)\n",
      "Progress: 17.40% completed\n",
      "Response for Row 349:\n",
      "Answer to Question 2: (recombinant expression of egtA, EGT biosynthetic capacity), (recombinant expression of egtA, EGT production), (recombinant expression of egtA and egt2a, decrease in EGT levels)\n",
      "Progress: 17.45% completed\n",
      "Response for Row 350:\n",
      "Answer to Question 2: (expression of gene TOR1, increased growth signaling), (expression of gene MSN2, increased stress response), (expression of gene MSN4, increased stress response), (addition of morin, reduced gene expression of TOR1), (addition of morin, reduced gene expression of MSN2), (addition of morin, reduced gene expression of MSN4)\n",
      "Progress: 17.50% completed\n",
      "Response for Row 351:\n",
      "Answer to Question 2: (heterologous expression of BFTS, production of diterpene variediene), (heterologous expression of BFTS, production of neovariediene), (heterologous expression of BFTS, production of neoflexibilene), (heterologous expression of BFTS and VndE, production of seven norditerpenoids), (modification by VndE, multi-site hydroxylation by A. oryzae endogenous enzymes)\n",
      "Progress: 17.55% completed\n",
      "Response for Row 352:\n",
      "Answer to Question 2: (transcriptional device library, expression strength of 16-520%), (transcriptional device library, expression strength of 30-500%), (PAOX1-driven pathway, 5.5-fold production of dihydromonacolin L), (selected devices from cTRDL, 3.0-fold production of monacolin J)\n",
      "Progress: 17.60% completed\n",
      "Response for Row 353:\n",
      "Answer to Question 2:  (upregulation of gene expression, improvement in mucosal barrier function), (upregulation of gene expression, increase in goblet cells), (upregulation of gene expression, enhancement of tight junction), (upregulation of gene expression, increase in sIgA level), (decrease in pathogenic bacteria, stabilization of microbiome), (increase in beneficial bacteria, stabilization of microbiome), (promotion of anti-inflammatory response, suppression of inflammatory response)\n",
      "Progress: 17.65% completed\n",
      "Response for Row 354:\n",
      "Answer to Question 2: (no specific gene overexpression/deletion mentioned, no specific consequences mentioned)\n",
      "Progress: 17.70% completed\n",
      "Response for Row 355:\n",
      "Answer to Question 2: (expression of gene Cas9, consequence 1), (expression of gene Cas9, consequence 2), (expression of gene Cas9, consequence 3), (expression of gene Cas9, consequence 4)\n",
      "Progress: 17.75% completed\n",
      "Response for Row 356:\n",
      "Answer to Question 2: (deletion of gene catalase, consequence lower mRNA expression), (overexpression of gene superoxide dismutase, consequence higher serum concentrations)\n",
      "Progress: 17.80% completed\n",
      "Response for Row 357:\n",
      "Answer to Question 2:  (deletion of gene clp1, increased fungal background autophagy level), (deletion of gene clp1, reduced hyphal growth), (deletion of gene clp1, hyperbranching), (deletion of gene clp1, abnormal fungal morphology), (deletion of gene clp1, hindered appressorial glycogen metabolism), (deletion of gene clp1, hindered turgor production), (deletion of gene clp1, weakened plant infection), (deletion of gene clp1, decreased virulence)\n",
      "Progress: 17.85% completed\n",
      "Response for Row 358:\n",
      "Answer to Question 2: (no gene expression/deletion mentioned, focus on discussion of bacterial hosts and gene expression systems)\n",
      "Progress: 17.90% completed\n",
      "Response for Row 359:\n",
      "Answer to Question 2: (deletion of gene REG1, improved SAM production), (deletion of gene REG1, enhanced glucose utilization), (deletion of gene REG1, elevated glycolytic intermediates), (deletion of gene REG1, extended lifespan), (overexpression of gene SNF1, improved SAM production), (overexpression of gene SNF1, enhanced glucose utilization), (overexpression of gene SNF1, elevated glycolytic intermediates), (overexpression of gene SNF1, extended lifespan), (deletion of gene REG1, increased expression of genes involved in glucose transport and glycolysis), (overexpression of gene SNF1, increased expression of genes involved in glucose transport and glycolysis), (deletion of gene REG1, enhanced expression of ACS1), (deletion of gene REG1, enhanced expression of ALD6), (deletion of gene REG1, enhanced activity of ADH2), (overexpression of gene SNF1, enhanced expression of ACS1), (overexpression of gene SNF1, enhanced expression of ALD6), (overexpression of gene SNF1, enhanced activity of ADH2), (deletion of gene REG1, enhanced gene expressions involved in sulfur-containing amino acids), (overexpression of gene SNF1, enhanced gene expressions involved in sulfur-containing amino acids)\n",
      "Progress: 17.95% completed\n",
      "Response for Row 360:\n",
      "Answer to Question 2:  (overexpression of gene PgCBF3, consequence increased survival rate after cold treatment), (overexpression of gene PgCBF7, consequence increased survival rate after cold treatment), (overexpression of gene PgCBF3, consequence increased soluble sugar content), (overexpression of gene PgCBF7, consequence increased soluble sugar content), (overexpression of gene PgCBF3, consequence increased proline content), (overexpression of gene PgCBF7, consequence increased proline content), (overexpression of gene PgCBF3, consequence decreased electrolyte leakage), (overexpression of gene PgCBF7, consequence decreased electrolyte leakage), (overexpression of gene PgCBF3, consequence decreased malondialdehyde content), (overexpression of gene PgCBF7, consequence decreased malondialdehyde content), (overexpression of gene PgCBF3, consequence decreased reactive oxygen species production), (overexpression of gene PgCBF7, consequence decreased reactive oxygen species production), (overexpression of gene PgCBF3, consequence elevated catalase enzyme activity), (overexpression of gene PgCBF7, consequence elevated catalase enzyme activity), (overexpression of gene PgCBF3, consequence elevated peroxidase enzyme activity), (overexpression of gene PgCBF7, consequence elevated peroxidase enzyme activity), (overexpression of gene PgCBF3, consequence elevated superoxide dismutase enzyme activity), (overexpression of gene PgCBF7, consequence elevated superoxide dismutase enzyme activity), (overexpression of gene PgCBF3, consequence upregulated expression of AtCOR15A), (overexpression of gene PgCBF7, consequence upregulated expression of AtCOR15A), (overexpression of gene PgCBF3, consequence upregulated expression of AtCOR47), (overexpression of gene PgCBF7, consequence upregulated expression of AtCOR47), (overexpression of gene PgCBF3, consequence upregulated expression of AtRD29A), (overexpression of gene PgCBF7, consequence upregulated expression of AtRD29A), (overexpression of gene PgCBF3, consequence upregulated expression of AtKIN1), (overexpression of gene PgCBF7, consequence upregulated expression of AtKIN1)\n",
      "Progress: 18.00% completed\n",
      "Response for Row 361:\n",
      "Answer to Question 2: (deletion of gene MoDHX35, reduced tolerance to doxorubicin), (deletion of gene MoDHX35, reduced appressorium formation rate), (deletion of gene MoDHX35, attenuated virulence)\n",
      "Progress: 18.05% completed\n",
      "Response for Row 362:\n",
      "Answer to Question 2: (deletion of D14, activation of D14 to bind to downstream signaling partners), (overexpression of SL, induction of D14 interaction with SMXL7), (overexpression of SL, induction of D14 interaction with MAX2), (presence of MAX2, strengthening of SL-induced D14 interaction with SMXL7), (presence of SMXL7, weakening of SL-induced D14 interaction with MAX2)\n",
      "Progress: 18.10% completed\n",
      "Response for Row 363:\n",
      "Answer to Question 2: (deletion of gene RPD3, decrease in de novo pathway metabolites production), (deletion of gene HST1, increase in BNA gene expression), (deletion of gene RPD3, reduction in acetylation of H4K5 and H4K12), (deletion of gene HST1, increase in acetylation of H4K5 and H4K12), (deletion of gene RPD3, opposition to spreading of Hst1-dependent heterochromatin)\n",
      "Progress: 18.15% completed\n",
      "Response for Row 364:\n",
      "Answer to Question 2: (overexpression of gene Mit1, upregulates pAOX1), (overexpression of gene Mit1, increases VEGF production by 1.9 fold), (overexpression of gene Mit1, increases eGFP production by 2.2 fold), (overexpression of gene Mit1, duplicates AOX1 enzyme activity), (overexpression of gene Mit1, enhances yeast cells' capacity for methanol detoxification), (intracellular overexpression of gene Mit1, upregulates heterologous target gene eGFP production)\n",
      "Progress: 18.20% completed\n",
      "Response for Row 365:\n",
      "Answer to Question 2: (deletion of gene L7/uL30 major form, consequence 1: favoring translation of genes with short open reading frames), (overexpression of gene L7/uL30 minor form, consequence 1: increasing translation of long open reading frames), (overexpression of gene L7/uL30 minor form, consequence 2: promoting tolerance to drugs through increased translation of cell wall proteins)\n",
      "Progress: 18.25% completed\n",
      "Response for Row 366:\n",
      "Answer to Question 2: (deletion of gene Ltn1, consequence 1: impaired ubiquitination of substrates), (overexpression of gene Ago, consequence 1: enhanced RQC activity), (overexpression of gene Ago, consequence 2: increased elimination of defective nascent polypeptides), (overexpression of gene Ago, consequence 3: removal of unwanted protein products)\n",
      "Progress: 18.30% completed\n",
      "Response for Row 367:\n",
      "Answer to Question 2: (overexpression of eGFP, varied GFP signal), (overexpression of MaFAR1, fatty alcohol production)\n",
      "Progress: 18.35% completed\n",
      "Response for Row 368:\n",
      "Answer to Question 2: (upregulation of lamBDCA, increased adhesion capability), (downregulation of ari1, decreased growth), (downregulation of skg6, decreased growth), (downregulation of kex2/gas1, decreased growth), (downregulation of DNA to RNA to protein pathways, decreased growth), (downregulation of cell cycle, decreased proliferation), (downregulation of meiosis, decreased proliferation), (downregulation of MAPK signaling pathways, decreased proliferation)\n",
      "Progress: 18.40% completed\n",
      "Response for Row 369:\n",
      "Answer to Question 2: (expression of genes responsible for growth and ethanol stress, increased ethanol fermentation efficiency), (expression of genes responsible for oxidative stress, increased ethanol fermentation efficiency), (expression of genes responsible for acetic acid stress, increased ethanol fermentation efficiency), (expression of genes responsible for DNA repair, increased ethanol fermentation efficiency), (expression of genes responsible for pyruvate-to-tricarboxylic acid (TCA) pathway, increased ethanol fermentation efficiency), (expression of genes responsible for pyruvate-to-ethanol pathway, increased ethanol fermentation efficiency)\n",
      "Progress: 18.45% completed\n",
      "Response for Row 370:\n",
      "Answer to Question 2: (deletion of glycosylation site N15, lower secretion efficiency), (deletion of glycosylation site N70, lower secretion efficiency), (deletion of glycosylation sites N15 and N70, lower secretion efficiency), (preservation of glycosylation sites, higher thermal stability), (deletion of glycosylation sites N15 and N70, slightly higher proliferation efficacy)\n",
      "Progress: 18.50% completed\n",
      "Response for Row 371:\n",
      "Answer to Question 2: (deletion of gene GPD2, increase in ethanol content), (deletion of gene GPD2, decrease in glycerol content), (deletion of gene FPS1, increase in ethanol content), (deletion of gene FPS1, decrease in glycerol content), (deletion of gene ADH2, increase in ethanol content), (deletion of gene ADH2, decrease in glycerol content), (deletion of gene DLD3, increase in ethanol content), (deletion of gene DLD3, decrease in glycerol content), (deletion of gene GPD2, decrease in acetic acid content), (deletion of gene FPS1, decrease in acetic acid content), (deletion of gene ADH2, decrease in acetic acid content), (deletion of gene DLD3, decrease in acetic acid content), (deletion of gene GPD2, decrease in lactic acid content), (deletion of gene FPS1, decrease in lactic acid content), (deletion of gene ADH2, decrease in lactic acid content), (deletion of gene DLD3, decrease in lactic acid content)\n",
      "Progress: 18.55% completed\n",
      "Response for Row 372:\n",
      "Answer to Question 2: (integration of MCR-C, stable gene expression), (integration of MCR-C, higher 3-HP production 4.4 g/L under batch fermentation), (integration of MCR-C, higher 3-HP production 56.5 g/L under fed-batch fermentation), (plasmid loss, instability of the engineered strain), (toxic intermediate malonate semialdehyde, cause of plasmid loss)\n",
      "Progress: 18.60% completed\n",
      "Response for Row 373:\n",
      "Answer to Question 2: (no expression/deletion of gene mentioned, consequence: large intermolecular interfaces promote cotranslational assembly)\n",
      "Progress: 18.65% completed\n",
      "Response for Row 374:\n",
      "Answer to Question 2: (deletion of gene GPD2, increase in ethanol production), (deletion of gene FPS1, increase in ethanol production), (deletion of gene ADH2, increase in ethanol production), (deletion of gene GPD2, decrease in glycerol content), (deletion of gene FPS1, decrease in glycerol content), (deletion of gene ADH2, decrease in glycerol content), (deletion of gene GPD2, decrease in lactic acid content), (deletion of gene FPS1, decrease in lactic acid content), (deletion of gene ADH2, decrease in lactic acid content), (deletion of gene GPD2, decrease in acetic acid content), (deletion of gene FPS1, decrease in acetic acid content), (deletion of gene ADH2, decrease in acetic acid content), (deletion of gene GPD2, decrease in succinic acid content), (deletion of gene FPS1, decrease in succinic acid content), (deletion of gene ADH2, decrease in succinic acid content)\n",
      "Progress: 18.70% completed\n",
      "Response for Row 375:\n",
      "Answer to Question 2: (deletion of non-phytate phosphorus, consequence final body weight and total gain decrease), (overexpression of phytase, consequence final body weight and total gain increase), (overexpression of multi-strain probiotic, consequence final body weight and total gain increase), (overexpression of Saccharomyces cerevisiae, consequence final body weight and total gain increase), (overexpression of fumaric acid, consequence feed conversion ratio equivalent to positive control), (overexpression of phytase, consequence improved feed conversion ratio), (overexpression of multi-strain probiotic, consequence improved feed conversion ratio), (overexpression of phytase, consequence greater dressing percentage), (overexpression of multi-strain probiotic, consequence greater dressing percentage), (overexpression of phytase, consequence higher relative weights of spleen and bursa of Fabricius), (overexpression of multi-strain probiotic, consequence higher relative weights of spleen and bursa of Fabricius), (overexpression of Saccharomyces cerevisiae, consequence higher relative weights of spleen and bursa of Fabricius), (overexpression of fumaric acid, consequence higher relative weights of spleen and bursen of Fabricius), (overexpression of phytase, consequence improved crude protein digestibility), (overexpression of multi-strain probiotic, consequence improved crude protein digestibility), (overexpression of Saccharomyces cerevisiae, consequence improved crude protein digestibility), (overexpression of phytase, consequence improved crude fiber digestibility), (overexpression of multi-strain probiotic, consequence improved crude fiber digestibility), (overexpression of Saccharomyces cerevisiae, consequence improved crude fiber digestibility), (overexpression of phytase, consequence improved calcium digestibility), (overexpression of multi-strain probiotic, consequence improved calcium digestibility), (overexpression of Saccharomyces cerevisiae, consequence improved calcium digestibility), (overexpression of phytase, consequence improved phosphorus digestibility), (overexpression of multi-strain probiotic, consequence improved phosphorus digestibility), (overexpression of Saccharomyces cerevisiae, consequence improved phosphorus digestibility), (overexpression of phytase, consequence greater tibia breaking strength), (overexpression of multi-strain probiotic, consequence greater tibia breaking strength), (overexpression of Saccharomyces cerevisiae, consequence greater tibia breaking strength), (overexpression of phytase, consequence increased tibia ash content), (overexpression of multi-strain probiotic, consequence increased tibia ash content), (overexpression of Saccharomyces cerevisiae, consequence increased tibia ash content), (overexpression of phytase, consequence increased tibia calcium concentration), (overexpression of multi-strain probiotic, consequence increased tibia calcium concentration), (overexpression of Saccharomyces cerevisiae, consequence increased tibia calcium concentration), (overexpression of phytase, consequence increased tibia phosphorus concentration), (overexpression of multi-strain probiotic, consequence increased tibia phosphorus concentration), (overexpression of Saccharomyces cerevisiae, consequence increased tibia phosphorus concentration), (overexpression of phytase, consequence increased tibia cortical area thickness), (overexpression of multi-strain probiotic, consequence increased tibia cortical area thickness), (overexpression of Saccharomyces cerevisiae, consequence increased tibia cortical area thickness), (overexpression of phytase, consequence reduced serum total cholesterol), (overexpression of multi-strain probiotic, consequence reduced serum total cholesterol), (overexpression of Saccharomyces cerevisiae, consequence reduced serum total cholesterol), (overexpression of phytase, consequence reduced serum triglycerides), (overexpression of multi-strain probiotic, consequence reduced serum triglycerides), (overexpression of Saccharomyces cerevisiae, consequence reduced serum triglycerides), (overexpression of phytase, consequence reduced serum triiodothyronine), (overexpression of multi-strain probiotic, consequence reduced serum triiodothyronine), (overexpression of Saccharomyces cerevisiae, consequence reduced serum triiodothyronine), (overexpression of phytase, consequence reduced serum thyroxine), (overexpression of multi-strain probiotic, consequence reduced serum thyroxine), (overexpression of Saccharomyces cerevisiae, consequence reduced serum thyroxine), (overexpression of phytase, consequence reduced serum glucose), (overexpression of multi-strain probiotic, consequence reduced serum glucose), (overexpression of Saccharomyces cerevisiae, consequence reduced serum glucose), (overexpression of phytase, consequence reduced serum alkaline phosphatase), (overexpression of multi-strain probiotic, consequence reduced serum alkaline phosphatase), (overexpression of Saccharomyces cerevisiae, consequence reduced serum alkaline phosphatase), (overexpression of phytase, consequence increased serum calcium), (overexpression of multi-strain probiotic, consequence increased serum calcium), (overexpression of Saccharomyces cerevisiae, consequence increased serum calcium), (overexpression of phytase, consequence increased serum phosphorus), (overexpression of multi-strain probiotic, consequence increased serum phosphorus), (overexpression of Saccharomyces cerevisiae, consequence increased serum phosphorus), (overexpression of phytase, consequence enhanced villi height), (overexpression of multi-strain probiotic, consequence enhanced villi height), (overexpression of Saccharomyces cerevisiae, consequence enhanced villi height), (overexpression of fumaric acid, consequence enhanced villi height), (overexpression of phytase, consequence enhanced villi width), (overexpression of multi-strain probiotic, consequence enhanced villi width), (overexpression of Saccharomyces cerevisiae, consequence enhanced villi width), (overexpression of fumaric acid, consequence enhanced villi width), (overexpression of phytase, consequence enhanced villi absorptive surface area), (overexpression of multi-strain probiotic, consequence enhanced villi absorptive surface area), (overexpression of Saccharomyces cerevisiae, consequence enhanced villi absorptive surface area), (overexpression of fumaric acid, consequence enhanced villi absorptive surface area), (overexpression of multi-strain probiotic, consequence up-regulated NaP-IIb mRNA expression), (overexpression of fumaric acid, consequence up-regulated NaP-IIb mRNA expression)\n",
      "Progress: 18.75% completed\n",
      "Response for Row 376:\n",
      "Answer to Question 2: (deletion of gene OLI1, consequence 1: reduced ethanol production), (deletion of gene OLI1, consequence 2: altered porphyrin biosynthesis), (overexpression of gene CYC3, consequence 1: enhanced ethanol production), (overexpression of gene CYC3, consequence 2: increased oxidative phosphorylation), (overexpression of gene ADH5, consequence 1: enhanced ethanol production), (overexpression of gene PDA1, consequence 1: enhanced ethanol production), (regulation by Tup1, consequence 1: affects expression of CYC3), (regulation by Tup1, consequence 2: affects expression of ADH5)\n",
      "Progress: 18.80% completed\n",
      "Response for Row 377:\n",
      "Answer to Question 2: (eIF4A overexpression, accelerates eIF4E-mRNA association), (eIF4A overexpression, modulates eIF4F-mRNA interactions), (eIF4A overexpression, catalyzes ATP hydrolysis), (ATP hydrolysis, ejects eIF4E), (ATP hydrolysis, ejects eIF4E•eIF4G)\n",
      "Progress: 18.85% completed\n",
      "Response for Row 378:\n",
      "Answer to Question 2: (deletion of gene SOD, consequence 1), (deletion of gene GPx, consequence 1), (deletion of gene COX II, consequence 1), (deletion of gene TNF-α, consequence 1), (deletion of gene Casp-3, consequence 1), (deletion of gene Claudin-3, consequence 1)\n",
      "Progress: 18.90% completed\n",
      "Response for Row 379:\n",
      "Answer to Question 2: (overexpression of gene SsSWEET13c, consequence plant root elongation), (overexpression of gene SsSWEET13c, consequence increase fresh weight of Arabidopsis thaliana), (positive regulation of gene SsSWEET13c by SsbHLH34, consequence increased activity in morning), (negative regulation of gene SsSWEET13c by SsbZIP44, consequence decreased activity in morning), (positive regulation of gene SsSWEET13c by SsTFIIIA-a, consequence increased activity in morning), (positive regulation of gene SsSWEET13c by SsMYR2, consequence increased activity in morning), (positive regulation of gene SsSWEET13c by SsRAP2.4, consequence increased activity in morning), (positive regulation of gene SsSWEET13c by SsbHLH035, consequence increased activity in morning), (negative regulation of gene SsSWEET13c by SsABS5, consequence decreased activity in morning), (negative regulation of gene SsSWEET13c by SsTFIIIA-b, consequence decreased activity in morning), (negative regulation of gene SsSWEET13c by SsERF4, consequence decreased activity in morning), (positive regulation of gene SsSWEET13c by SsbZIP44, consequence increased sugar accumulation in sink tissues), (positive regulation of gene SsSWEET13c by SsTFIIIA-b, consequence increased sugar accumulation in sink tissues), (positive regulation of gene SsSWEET13c by SsbHLH34, consequence increased sugar accumulation in sink tissues), (positive regulation of gene SsSWEET13c by SsTFIIIA-a, consequence increased sugar accumulation in sink tissues), (negative regulation of gene SsSWEET13c by SsERF4, consequence decreased sugar accumulation in sink tissues), (negative regulation of gene SsSWEET13c by SsHB36, consequence decreased sugar accumulation in sink tissues), (negative regulation of gene SsSWEET13c by SsDEL1, consequence decreased sugar accumulation in sink tissues), (negative regulation of gene SsSWEET13c by SsABS5, consequence decreased sugar accumulation in sink tissues)\n",
      "Progress: 18.95% completed\n",
      "Response for Row 380:\n",
      "Answer to Question 2: (overexpression of gene ZmWRKY20, decrease in salt tolerance), (overexpression of gene ZmWRKY20, reprogramming of genes involved in defense response), (overexpression of gene ZmWRKY20, reprogramming of genes involved in oxidation-reduction response), (interaction of ZmWRKY20 and ZmWRKY115, repression of ZmbZIP111 expression), (repression of ZmbZIP111 expression, increase in sensitivity to salt stress)\n",
      "Progress: 19.00% completed\n",
      "Response for Row 381:\n",
      "Answer to Question 2: (deletion of gene gapA, decrease in glucose consumption by competing pathways), (deletion of gene budA, decrease in by-products pathways), (expression of gene gpd1, production of glycerol), (expression of gene gpp2, production of glycerol), (expression of CRISPRi system, inhibition of gapA), (expression of CRISPRi system, inhibition of budA), (overexpression of glycerol pathway genes, increase in glycerol production), (use of glucose as carbon source, production of 3-HP)\n",
      "Progress: 19.05% completed\n",
      "Response for Row 382:\n",
      "Answer to Question 2: (deletion of gene rimO, failure to activate starvation-induced BGCs), (deletion of gene rimO, down-regulation of genes involved in basic metabolism), (deletion of gene rimO, down-regulation of genes involved in membrane biogenesis), (deletion of gene rimO, down-regulation of genes involved in growth), (overexpression of gene rimO, increased resistance to oxidative stress), (overexpression of gene rimO, increased resistance to osmotic stress), (overexpression of gene rimO, insensitivity to glucose repression), (overexpression of gene rimO, overproduction of several SMs), (expression of gene rimO, activation of sterigmatocystin gene cluster), (expression of gene rimO, induction of aflR transcription), (expression of gene rimO, required for nuclear accumulation of AflR protein)\n",
      "Progress: 19.10% completed\n",
      "Response for Row 383:\n",
      "Answer to Question 2: (deletion/mutation of gene ZNHIT3, decrease in steady-state Hit1 protein levels), (deletion/mutation of gene ZNHIT3, significant reduction of box C/D snoRNA levels), (deletion/mutation of gene ZNHIT3, defects in rRNA processing), (deletion/mutation of gene ZNHIT3, altered cellular translation), (deletion/mutation of gene ZNHIT3, site-specific changes in the rRNA modification pattern)\n",
      "Progress: 19.15% completed\n",
      "Response for Row 384:\n",
      "Answer to Question 2: (deletion of gene XdWC2, similar carotenoid profile to ARTP mutant), (deletion of gene XdWC2, regulation of CrtI expression), (deletion of gene XdWC2, regulation of CrtS expression), (overexpression of gene CrtI, increase in 3-hydroxy-3', 4'-didehydro-β, ψ-carotene-4-one production), (interaction of XdWC2 with XdWC1, formation of WCC complex), (transcriptional increase of XdWC2, no significant change in carotenoid profile), (transcriptional increase of CrtS, no significant change in carotenoid profile)\n",
      "Progress: 19.20% completed\n",
      "Response for Row 385:\n",
      "Answer to Question 2: (deletion of protease gene, reduced protease activity), (deletion of sporulation gene, reduced sporulation), (deletion of autolysis-related gene, reduced autolysis), (overproduction of Sox, increased secretion of Sox), (overproduction of interleukin-1β, increased secretion of interleukin-1β)\n",
      "Progress: 19.25% completed\n",
      "Response for Row 386:\n",
      "Answer to Question 2: (expression of ACC1, increased MCFAEE levels), (expression of FAS1, increased MCFAEE levels), (expression of FAA2, increased MCFAEE levels), (expression of EEB1, increased MCFAEE levels), (deletion of POX1, decreased MCFAEE levels)\n",
      "Progress: 19.30% completed\n",
      "Response for Row 387:\n",
      "Answer to Question 2: (overexpression of gene OsNACL35, increased resistance to salinity stress), (deletion of gene OsNACL35, hypersensitivity to salt stress), (overexpression of gene OsNACL35, upregulation of gene OsDCD1), (overexpression of gene OsNACL35, increased H2S content)\n",
      "Progress: 19.35% completed\n",
      "Response for Row 388:\n",
      "Answer to Question 2: (deletion of ribosomal proteins, consequence reduced cell growth), (overexpression of ribosomal proteins, consequence increased cell growth), (deletion of ribosomal proteins, consequence decreased protein synthesis), (overexpression of ribosomal proteins, consequence increased protein synthesis), (deletion of rRNA, consequence impaired ribosome assembly), (overexpression of rRNA, consequence enhanced ribosome assembly), (deletion of ribosomal proteins, consequence altered protein expression profile), (overexpression of ribosomal proteins, consequence altered protein expression profile)\n",
      "Progress: 19.40% completed\n",
      "Response for Row 389:\n",
      "Answer to Question 2: (expression of Cas9, high-efficiency genomic integrations), (expression of Cas9, markerless transformations), (overexpression of gene of interest, supports localization studies), (overexpression of gene of interest, supports overexpression studies), (expression of green fluorescent protein, colocalization experiments), (expression of red fluorescent protein, colocalization experiments)\n",
      "Progress: 19.45% completed\n",
      "Response for Row 390:\n",
      "Answer to Question 2: (expression of ADH genes, increased alcoholic fermentation), (deletion of ACO2 gene expression, altered malate metabolism), (inhibition of tricarboxylic acid cycle enzymes, increased malate metabolism), (increased alcoholic fermentation, altered metabolism to overcome nitrosative stress)\n",
      "Progress: 19.50% completed\n",
      "Response for Row 391:\n",
      "Answer to Question 2: (deletion of gene expression levels, enhanced predictive capability of metabolic models), (overexpression of chromatin accessibility levels, altered metabolic pathways), (deletion of epigenetic information, limited metabolic modeling accuracy), (overexpression of RNA-seq data, improved condition-specific metabolic simulations), (overexpression of ATAC-seq data, enhanced metabolic predictions)\n",
      "Progress: 19.55% completed\n",
      "Response for Row 392:\n",
      "Answer to Question 2: (no gene overexpression/deletion mentioned, no direct consequence related to gene overexpression/deletion)\n",
      "Progress: 19.60% completed\n",
      "Response for Row 393:\n",
      "Answer to Question 2:  (deletion of gene CLN3, predominance of G2/M phase cells), (deletion of gene CLN3, increased biofilm formation), (deletion of gene CLN3, increased extracellular polysaccharide formation), (deletion of gene CLN3, increased expression of genes in the FLO gene family), (deletion of gene SIC1, predominance of G2/M phase cells), (deletion of gene SIC1, increased biofilm formation), (deletion of gene SIC1, increased extracellular polysaccharide formation), (deletion of gene SIC1, increased expression of genes in the FLO gene family), (deletion of gene ACE2, contrasting performance compared to CLN3 and SIC1 deletion)\n",
      "Progress: 19.65% completed\n",
      "Response for Row 394:\n",
      "Answer to Question 2: (deletion of gene SPT10, increase in total phospholipids), (deletion of gene SPT10, insufficient accumulation of key metabolites), (deletion of gene SPT10, massive endoplasmic reticulum degradation), (overexpression of gene DGK1, stable massive ER structure), (overexpression of gene DGK1, decrease in autophagy), (overexpression of gene DGK1, increase in MEV/ERG pathway activity), (heterologous expression of β-amyrin synthase, production of triterpene β-amyrin)\n",
      "Progress: 19.70% completed\n",
      "Response for Row 395:\n",
      "Answer to Question 2: (deletion of gene CYP2E1, consequence 1: reduced inflammation), (deletion of gene CYP2E1, consequence 2: improved fatty liver condition), (inhibition of gene MBOAT7, consequence 1: exacerbation of fatty liver), (inhibition of gene TMC4, consequence 1: exacerbation of fatty liver), (expression of gene CYP7A1, consequence 1: inhibition of lipid production), (fermentation by Lactobacillus plantarum, consequence 1: reduction in liver cholesterol), (fermentation by Lactobacillus plantarum, consequence 2: reduction in fat accumulation), (fermentation by Lactobacillus plantarum, consequence 3: promotion of short-chain fatty acid production), (fermentation by Lactobacillus plantarum, consequence 4: regulation and restoration of intestinal flora), (fermentation by Lactobacillus plantarum, consequence 5: decrease in endotoxin-bearing Proteobacteria levels)\n",
      "Progress: 19.75% completed\n",
      "Response for Row 396:\n",
      "Answer to Question 2: (expression of gene VDS, consequence valerena-4,7(11)-diene production), (overexpression of gene ERG8, consequence increased valerena-4,7(11)-diene production), (overexpression of gene ERG12, consequence increased valerena-4,7(11)-diene production), (overexpression of gene ERG19, consequence increased valerena-4,7(11)-diene production), (expression of gene LsGAO2, consequence valerenic acid production)\n",
      "Progress: 19.80% completed\n",
      "Response for Row 397:\n",
      "Answer to Question 2: (overexpression of gene HAA1, enhancement of carbohydrate metabolism pathways), (overexpression of gene HAA1, enhancement of amino acid metabolism pathways), (overexpression of gene HAA1, enhancement of transcription pathways), (overexpression of gene HAA1, enhancement of translation pathways), (overexpression of gene HAA1, enhancement of cofactors metabolism pathways), (overexpression of gene HAA1, enhancement of vitamins metabolism pathways), (overexpression of gene HAA1, involvement in oxidative phosphorylation), (overexpression of gene HAA1, involvement in purine metabolism), (overexpression of gene HAA1, involvement in vitamin B6 metabolism), (overexpression of gene HAA1, involvement in spliceosome), (overexpression of gene TYE7, enhancement of carbohydrate metabolism pathways), (overexpression of gene TYE7, enhancement of amino acid metabolism pathways), (overexpression of gene TYE7, enhancement of transcription pathways), (overexpression of gene TYE7, enhancement of translation pathways), (overexpression of gene TYE7, enhancement of cofactors metabolism pathways), (overexpression of gene TYE7, enhancement of vitamins metabolism pathways), (overexpression of gene TYE7, involvement in riboflavin metabolism), (overexpression of gene TYE7, involvement in spliceosome), (overexpression of gene HAA1, preference for enhancing oxidative phosphorylation), (overexpression of gene TYE7, tendency to upregulate glycolysis/gluconeogenesis), (overexpression of gene HAA1, rewiring global gene expressions), (overexpression of gene TYE7, rewiring global gene expressions), (overexpression of gene HAA1, positive perturbations of energy metabolism), (overexpression of gene TYE7, positive perturbations of energy metabolism), (overexpression of gene HAA1, positive perturbations of amino acid metabolism), (overexpression of gene TYE7, positive perturbations of amino acid metabolism), (overexpression of gene HAA1, strengthening of key cofactor metabolism), (overexpression of gene TYE7, strengthening of key cofactor metabolism), (overexpression of gene HAA1, improvement in transcriptional regulation), (overexpression of gene TYE7, improvement in transcriptional regulation), (overexpression of gene HAA1, improvement in translational regulation), (overexpression of gene TYE7, improvement in translational regulation)\n",
      "Progress: 19.85% completed\n",
      "Response for Row 398:\n",
      "Answer to Question 2: (deletion of gene EHD3, consequence malonyl-CoA hydrolase activity), (mutation of gene EHD3, consequence F121 to I121), (mutation of gene EHD3, consequence E124 to S124), (integration of mutant EHD3, consequence increased expression), (integration of ACC1, consequence increased expression), (regulation of key enzyme expression in malonyl-CoA pathway, consequence improved precursor supply), (inhibition of competing pathways, consequence increased malonate production)\n",
      "Progress: 19.90% completed\n",
      "Response for Row 399:\n",
      "Answer to Question 2: (deletion of gene n/a, consequence increased gene activation efficiency 34.9-fold), (deletion of gene n/a, consequence increased gene repression efficiency 95%), (expression of gene n/a, consequence improved metabolic pathway rewiring for 3-hydroxypropanoic acid production)\n",
      "Progress: 19.95% completed\n",
      "Response for Row 400:\n",
      "Answer to Question 2: (decrease of toxin production by lentil extract, reduced pro-inflammatory interleukin-8 production), (decrease of toxin production by lentil extract, modulated mucus-related gene induction), (reduction of ETEC adhesion by yeast product, reduced adhesion to mucin), (reduction of ETEC adhesion by yeast product, reduced adhesion to Caco-2/HT29-MTX cells), (modulation of toxin-related gene expression by both fiber-containing products, strengthened intestinal barrier function)\n",
      "Progress: 20.00% completed\n",
      "Response for Row 401:\n",
      "Answer to Question 2: (deletion of gene XsGAO1, consequence Germacrene A not converted to Germacrene A acid), (deletion of gene XsGAO2, consequence Germacrene A not converted to Germacrene A acid), (expression of gene XsGAO1, consequence Germacrene A converted to Germacrene A acid), (expression of gene XsGAO2, consequence Germacrene A converted to Germacrene A acid)\n",
      "Progress: 20.05% completed\n",
      "Response for Row 402:\n",
      "Answer to Question 2: (overexpression of gene ari1, higher tolerance to furfural), (overexpression of gene ari1, higher tolerance to HMF), (overexpression of gene ari1, increased growth rate in aldehyde-containing environment), (overexpression of gene ari1, increased ethanol production capacity in aldehyde-containing environment)\n",
      "Progress: 20.10% completed\n",
      "Response for Row 403:\n",
      "Answer to Question 2: (expression of gene TDH3, consequence constitutive promoter activity maintained at high temperature), (expression of gene TDH3, consequence constitutive promoter activity maintained in presence of ethanol), (expression of gene HSP26, consequence responds to high temperature), (expression of gene PCK1, consequence responds to gluconeogenic sources), (expression of gene XYL1, consequence responds to presence of xylose oligomers)\n",
      "Progress: 20.15% completed\n",
      "Response for Row 404:\n",
      "Answer to Question 2: (thi4 gene overexpression, TOL production increase), (thi4 gene overexpression, involvement in TOL production), (Trp overexpression, TOL production increase), (Phe overexpression, reduction in Trp's stimulatory effect on TOL production), (Trp overexpression, stimulation of Ehrlich pathway activity), (allantoin catabolic genes dal1, dal2, dal4 overexpression, role in TOL production), (transcriptional activator gene aro80 overexpression, role in TOL production)\n",
      "Progress: 20.20% completed\n",
      "Response for Row 405:\n",
      "Answer to Question 2: (amplification of heterologous gene copies, improvement in production of nerolidol), (amplification of heterologous gene copies, improvement in production of limonene), (amplification of heterologous gene copies, improvement in production of lycopene), (increasing gene dosage, overcoming metabolic bottlenecks)\n",
      "Progress: 20.25% completed\n",
      "Response for Row 406:\n",
      "Answer to Question 2: (expression of gene Erg1p, consequence squalene loss reduction), (expression of gene ERG1 under P<sub>THI2</sub>, consequence 20 times higher squalene production), (overexpression of rate-limiting enzyme, consequence increased squalene production), (deletion of non-essential competing pathway gene, consequence increased squalene production)\n",
      "Progress: 20.30% completed\n",
      "Response for Row 407:\n",
      "Answer to Question 2: (deletion of gene CiSHT1, reduced synthesis of tetracoumaroyl spermine), (deletion of gene CiSHT2, reduced initial N-acylation steps), (expression of gene CiSHT2, accumulation of partially substituted spermine), (coexpression of gene CiSHT2 and CiSHT1, synthesis and accumulation of fully substituted spermine), (availability of spermine, increased accumulation of spermine derivatives over spermidine derivatives)\n",
      "Progress: 20.35% completed\n",
      "Response for Row 408:\n",
      "Answer to Question 2: (deletion of gene EpSDR-5, reduction in jatrophanes), (deletion of gene EpSDR-5, increase in ingenanes), (deletion of gene Casbene Synthase, reduction in jatrophanes), (deletion of gene Casbene Synthase, reduction in ingenanes), (deletion of gene CYP71D365, reduction in jatrophanes), (deletion of gene CYP71D365, reduction in ingenanes), (expression of gene EpSDR-5, conversion of jolkinol C to jolkinol E)\n",
      "Progress: 20.40% completed\n",
      "Response for Row 409:\n",
      "Answer to Question 2: (decreased expression of proinflammatory cytokines, enhanced gut barrier), (increased expression of TUNEL-positive tumor epithelial cells, increased apoptosis), (decreased expression of oncocyte proliferation indicators, decreased tumor multiplicity)\n",
      "Progress: 20.45% completed\n",
      "Response for Row 410:\n",
      "Answer to Question 2: (expression of gene ldhA, D-lactic acid production), (expression of gene ldhA, PDLA accumulation), (expression of gene ldhA, D-lactic acid content in P(LA-3HB) copolymer)\n",
      "Progress: 20.50% completed\n",
      "Response for Row 411:\n",
      "Answer to Question 2:  (overexpression of gene OPI3, consequence 1), (overexpression of gene OPI3, consequence 2), (overexpression of gene EFM6, consequence 1), (overexpression of gene EFM6, consequence 2), (overexpression of gene YOL014W, consequence 1), (overexpression of gene YOL014W, consequence 2)\n",
      "Progress: 20.55% completed\n",
      "Response for Row 412:\n",
      "Answer to Question 2: (expression of LacZ gene, activation of β-galactosidase), (hydrolysis of X-gal, production of 5,5'-dibromo-4,4'-dichloro-indigo), (expression of LacZ gene, measurement of protein-protein interaction strength), (use of ONPG, detection of β-galactosidase activity)\n",
      "Progress: 20.60% completed\n",
      "Response for Row 413:\n",
      "Answer to Question 2: (no expression/deletion of gene mentioned, no consequence related to expression/deletion)\n",
      "Progress: 20.65% completed\n",
      "Response for Row 414:\n",
      "Answer to Question 2: (cloning and expression of gene CaHMGS, increased mevalonate biosynthesis in yeast), (overexpression of gene CaHMGS, higher expression level in roots of Centella asiatica), (deletion of HMGS in Saccharomyces cerevisiae strain YSC1021, reduced mevalonate biosynthesis), (expression of gene CaHMGS, enhanced accumulation of isoprenoids and triterpenoid saponins)\n",
      "Progress: 20.70% completed\n",
      "Response for Row 415:\n",
      "Answer to Question 2: (deletion of gene sorB, consequence sorbicillinoid production affected), (deletion of gene axl1, consequence no impact on yield), (deletion of gene axl2, consequence no impact on dry weight), (deletion of gene bud3, consequence no impact on pH), (deletion of gene bud4, consequence no impact on stress tolerance), (deletion of gene sorB, consequence arthrospores development differs from wild-type)\n",
      "Progress: 20.75% completed\n",
      "Response for Row 416:\n",
      "Answer to Question 2: (reconstitution of brassinolide biosynthesis pathway, production of brassinolide intermediates)\n",
      "Progress: 20.80% completed\n",
      "Response for Row 417:\n",
      "Answer to Question 2: (heterologous expression of fungal biosynthetic gene cluster, increase yields of natural products), (heterologous expression of fungal biosynthetic gene cluster, investigation of biosynthetic pathway)\n",
      "Progress: 20.85% completed\n",
      "Response for Row 418:\n",
      "Answer to Question 2: (deletion of gene Ccr4, increased CLB1 mRNA levels), (deletion of gene Ccr4, increased CLB2 mRNA levels), (deletion of gene Ccr4, increased CLB3 mRNA levels), (deletion of gene Ccr4, increased CLB4 mRNA levels), (deletion of gene Ccr4, increased CLB5 mRNA levels), (deletion of gene Ccr4, increased CLB6 mRNA levels), (deletion of gene Ccr4, increased Clb1 protein levels), (deletion of gene Ccr4, increased Clb4 protein levels), (deletion of gene Ccr4, increased Clb6 protein levels), (Ccr4-mediated regulation, destabilization of CLB6 mRNA through coding sequence), (Ccr4-mediated regulation, destabilization of CLB6 mRNA through 3' UTR)\n",
      "Progress: 20.90% completed\n",
      "Response for Row 419:\n",
      "Answer to Question 2: (deletion of multiple genes, increased β-carotene production), (overexpression of multiple genes, increased β-carotene production), (deletion of multiple genes, increased S-adenosyl-L-methionine production), (overexpression of multiple genes, increased S-adenosyl-L-methionine production)\n",
      "Progress: 20.95% completed\n",
      "Response for Row 420:\n",
      "Answer to Question 2: (expression of BGCs, yield improvement), (expression of BGCs, discovery of new-to-nature compounds), (host engineering of E. coli, production of secondary metabolites), (host engineering of S. cerevisiae, production of secondary metabolites), (engineering of non-model microbes, biosynthesis of secondary metabolites), (genetic manipulation in non-model microorganisms, optimization of phenotypes), (genetic manipulation in non-model microorganisms, improvement of yields of secondary metabolites)\n",
      "Progress: 21.00% completed\n",
      "Response for Row 421:\n",
      "Answer to Question 2: (deletion of gene T4-endonuclease V, consequence CPD formation and repair monitored), (overexpression of ncRNA transcription, consequence enhancement of NER in subtelomeric elements Y' and X), (deletion of gene activity in silenced heterochromatin, consequence slow repair of UV photoproducts)\n",
      "Progress: 21.05% completed\n",
      "Response for Row 422:\n",
      "Answer to Question 2: ( deletion of gene EHT1, consequence lower amounts of ethyl esters), ( deletion of gene EEB1, consequence lower amounts of ethyl esters), ( deletion of gene EEB1, consequence increased expression of ATF2), ( deletion of gene EEB1, consequence increased content of acetate esters), ( deletion of gene EEB1, consequence affected production of higher alcohols), ( deletion of gene EHT1, consequence affected production of higher alcohols)\n",
      "Progress: 21.10% completed\n",
      "Response for Row 423:\n",
      "Answer to Question 2: (overexpression of gene hchA, improved tolerance against salinity stress), (overexpression of gene hchA, improved tolerance against dicarbonyl stress), (overexpression of gene hchA, improved tolerance against oxidative stress), (overexpression of gene EcGLY-III, tolerance against salinity at seedling stage), (overexpression of gene EcGLY-III, tolerance against salinity at reproductive stage), (overexpression of gene EcGLY-III, increased glyoxalase activity), (overexpression of gene EcGLY-III, increased antioxidant enzyme activity), (overexpression of gene EcGLY-III, resistance to accumulation of methylglyoxal), (overexpression of gene EcGLY-III, resistance to accumulation of reactive oxygen species), (overexpression of gene EcGLY-III, increased anti-glycation activity), (overexpression of gene EcGLY-III, prevention of stress-induced senescence)\n",
      "Progress: 21.15% completed\n",
      "Response for Row 424:\n",
      "Answer to Question 2: (attenuation of OTC gene, reduced ornithine production), (overexpression of SPE1-S gene, increased ornithine production), (overexpression of SPE1-S gene, increased putrescine production), (overexpression of ArgB/C gene, increased ornithine accumulation), (overexpression of ArgB/C gene, increased putrescine production)\n",
      "Progress: 21.20% completed\n",
      "Response for Row 425:\n",
      "Answer to Question 2:  (overexpression of BbgCYP11A1, increased cholesterol processing), (overexpression of plant CPR from Anemarrhena asphodeloides, higher bufadienolides production)\n",
      "Progress: 21.25% completed\n",
      "Response for Row 426:\n",
      "Answer to Question 2:  (overexpression of gene spt23, lower thermal tolerance), (deletion of gene spt23, higher survival rate), (deletion of gene spt23, higher biomass), (deletion of gene spt23, higher ethanol production), (deletion of gene spt23, lower total unsaturated lipid acid contents), (deletion of gene spt23, lower LTR richness), (deletion of gene spt23, higher transposon activities)\n",
      "Progress: 21.30% completed\n",
      "Response for Row 427:\n",
      "Answer to Question 2: (deletion of gene POT1, consequence 1: increased genetic stability), (deletion of gene POT1, consequence 2: increased p-coumaric acid production), (overexpression of p-CA synthesis cassette, consequence 1: increased p-coumaric acid production), (overexpression of p-CA synthesis cassette, consequence 2: stable p-coumaric acid production in nonselective medium)\n",
      "Progress: 21.35% completed\n",
      "Response for Row 428:\n",
      "Answer to Question 2: (deletion of AAT gene in S. cerevisiae, decreased antifungal activity), (expression of GcAAT gene in S. cerevisiae AAT knockout strain, 32% decrease in dry biomass weight of Rhizoctonia solani), (supplementation with L-leucine in native host, induced expression of GcAAT gene), (supplementation with α-ketoisocaproate in native host, induced expression of GcAAT gene), (expression of GcAAT gene in S. cerevisiae AAT knockout strain, increased oxidative stress in R. solani), (expression of GcAAT gene in S. cerevisiae AAT knockout strain, distorted mycelial morphology in R. solani), (expression of GcAAT gene in S. cerevisiae AAT knockout strain, increased hyphal branching in R. solani), (feeding with L-leucine in S. cerevisiae AAT knockout strain expressing GcAAT, 21% increase in isoamyl acetate ester), (feeding with α-ketoisocaproate in S. cerevisiae AAT knockout strain expressing GcAAT, 48% increase in isoamyl acetate ester), (expression of GcAAT gene in S. cerevisiae AAT knockout strain, 58% higher antifungal activity against R. solani)\n",
      "Progress: 21.40% completed\n",
      "Response for Row 429:\n",
      "Answer to Question 2: (codon-optimized gene sequences of Ulp1 and Ulp2, cloned and expressed in E. coli), (expression of SpUlp1, higher affinity towards SUMO compared to ScUlp1), (purification of SpUlp1 catalytic domain, 83.33% recovery yield), (expression of SpUlp1, 90% cleavage of fusion proteins within 1 hour), (fed-batch cultivation, 3.5 g/L of SpUlp1SD production), (fed-batch cultivation, Y_P/X of 77.26 mg/g DCW), (fed-batch cultivation, volumetric productivity of 205.88 mg/L/h)\n",
      "Progress: 21.45% completed\n",
      "Response for Row 430:\n",
      "Answer to Question 2:  (expression of gene IaAO2, C-28 oxidation of α-amyrin), (expression of gene IaAO2, C-28 oxidation of β-amyrin), (expression of gene IaAO4, C-23 oxidation of ursolic acid), (expression of gene IaAO4, C-23 oxidation of oleanolic acid), (expression of gene IaAO5, C-24 oxidation of β-amyrin), (expression of genes IaAO1, IaAO4, IaCPR, production of C-23 hydroxyl ursane-type triterpenoid derivatives)\n",
      "Progress: 21.50% completed\n",
      "Response for Row 431:\n",
      "Answer to Question 2: (overexpression of gene ENA5, improved multiple stress-tolerance), (overexpression of gene ENA5, increased ethanol production), (regulation of expression of gene ASP3, improved multiple stress-tolerance), (regulation of expression of gene ASP3, increased ethanol production), (regulation of expression of gene YOL162W, improved multiple stress-tolerance), (regulation of expression of gene YOL162W, increased ethanol production), (regulation of expression of gene YOR012W, improved multiple stress-tolerance), (regulation of expression of gene YOR012W, increased ethanol production), (regulation of expression of gene Crz1p, improved multiple stress-tolerance), (regulation of expression of gene Crz1p, increased ethanol production), (regulation of expression of gene Tos8p, improved multiple stress-tolerance), (regulation of expression of gene Tos8p, increased ethanol production)\n",
      "Progress: 21.55% completed\n",
      "Response for Row 432:\n",
      "Answer to Question 2: (deletion of prephenate dehydratase, alleviation of byproducts formation), (tuning down expression of OPc4CL2, alleviation of byproducts formation)\n",
      "Progress: 21.60% completed\n",
      "Response for Row 433:\n",
      "Answer to Question 2: (inducible expression of enzymes, balance cellular growth and production), (overexpression of genes using optogenetics, improved fermentation control), (overexpression of genes using optogenetics, improved productivity)\n",
      "Progress: 21.65% completed\n",
      "Response for Row 434:\n",
      "Answer to Question 2:  (overexpression of gene LpCPC, increased tolerance to NaHCO3 stress), (overexpression of gene LpCPC, higher chlorophyll content), (overexpression of gene LpCPC, higher soluble phenol content), (overexpression of gene LpCPC, higher lignin content), (overexpression of gene LpCPC, lower relative conductivity), (overexpression of gene LpCPC, higher expression of NHX1), (overexpression of gene LpCPC, higher expression of SOS1), (overexpression of gene LpCPC, higher expression of CCR1), (overexpression of gene LpCPC, higher expression of CAD)\n",
      "Progress: 21.70% completed\n",
      "Response for Row 435:\n",
      "Answer to Question 2: (expression of eEmRFP, gene expression in yeast), (expression of eEmRFP, gene expression in mammalian cells), (expression of GFP-fused human synaptobrevin VAMP1, proof-of-concept in yeast), (expression of GFP-fused human synaptobrevin VAMP1, proof-of-concept in E. coli), (expression of GFP-fused human synaptobrevin VAMP1, proof-of-concept in HEK293 cells), (expression of wild-type CLuc luciferase, proof-of-concept in yeast), (expression of wild-type CLuc luciferase, proof-of-concept in E. coli), (expression of wild-type CLuc luciferase, proof-of-concept in HEK293 cells), (expression of codon-changed CLuc luciferase, proof-of-concept in yeast), (expression of codon-changed CLuc luciferase, proof-of-concept in E. coli), (expression of codon-changed CLuc luciferase, proof-of-concept in HEK293 cells)\n",
      "Progress: 21.75% completed\n",
      "Response for Row 436:\n",
      "Answer to Question 2:  (improved expression of ARV1, decreased membrane fluidity), (improved expression of ARV1, enhanced acid tolerance), (improved expression of GIT1, increased membrane integrity), (improved expression of GIT1, improved oxidative tolerance), (improved expression of NFT1, reduced membrane permeability), (improved expression of NFT1, enhanced osmotic tolerance), (engineered Med15 V76R/R84K, increased pyruvate production)\n",
      "Progress: 21.80% completed\n",
      "Response for Row 437:\n",
      "Answer to Question 2: (expression of gene CsupYPAQ, production of (Z)-11-hexadecenol), (expression of gene CsupFAR2, production of (Z)-11-hexadecenol), (expression of gene Atr∆11, production of (Z)-11-hexadecenal), (expression of gene ATF1, production of (Z)-11-hexadecenyl acetate)\n",
      "Progress: 21.85% completed\n",
      "Response for Row 438:\n",
      "Answer to Question 2: (deletion of gene RPL3, consequence degradation of RPL3 mRNA), (deletion of gene RPL4, consequence degradation of RPL4 mRNA), (overexpression of gene RPL3, consequence massive aggregation of Rpl3), (overexpression of gene RPL4, consequence massive aggregation of Rpl4), (deletion of gene TOM1, consequence perturbation of overall proteostasis)\n",
      "Progress: 21.90% completed\n",
      "Response for Row 439:\n",
      "Answer to Question 2: (deletion of gene SAE2, increased CRISPR/SpCas9-induced mutagenesis efficiency), (deletion of gene SAE2, increased CRISPR/AsCpf1-induced mutagenesis efficiency), (deletion of gene EXO1, increased CRISPR/SpCas9-induced mutagenesis efficiency), (deletion of gene EXO1, increased CRISPR/AsCpf1-induced mutagenesis efficiency), (deletion of gene FUN30, increased CRISPR/SpCas9-induced mutagenesis efficiency), (deletion of gene FUN30, increased CRISPR/AsCpf1-induced mutagenesis efficiency), (overexpression of gene MRE11-H125N, increased CRISPR/SpCas9-induced mutagenesis efficiency), (overexpression of gene MRE11-H125N, increased CRISPR/AsCpf1-induced mutagenesis efficiency), (deletion of gene SAE2, diversified CRISPR/SpCas9-induced mutations at URA3 locus), (deletion of gene SAE2, diversified CRISPR/SpCas9-induced mutations at ADE2 locus), (overexpression of gene MRE11-H125N, diversified CRISPR/SpCas9-induced mutations at URA3 locus), (overexpression of gene MRE11-H125N, diversified CRISPR/SpCas9-induced mutations at ADE2 locus), (modulation of gene FPS1 expression, improved ethanol/glycerol ratio), (modulation of gene GPD1 expression, improved ethanol/glycerol ratio)\n",
      "Progress: 21.95% completed\n",
      "Response for Row 440:\n",
      "Answer to Question 2: (deletion of gene for high concentration of vinblastine and vincristine, consequence low production in planta), (expression of gene through metabolic engineering in heterologous systems, consequence improved MIA biosynthesis), (expression of gene through yeast engineering, consequence alternative production source of phytomolecules), (expression of gene through yeast engineering, consequence discovery of missing plant pathway enzymes and genes)\n",
      "Progress: 22.00% completed\n",
      "Response for Row 441:\n",
      "Answer to Question 2: (deletion of AtGTR1, reduced glucosinolate transport), (deletion of AtGTR1, reduced phytohormone transport), (mutation of EXXEK motif in AtGTR1, impaired glucosinolate transport function), (phosphorylation of Thr105 in AtGTR1, regulation of protein expression), (T105D mutation in AtGTR1, abrogation of glucosinolate transport function)\n",
      "Progress: 22.05% completed\n",
      "Response for Row 442:\n",
      "Answer to Question 2:  (constitutive expression of gene GPH1, partial suppression of filamentation), (repression of gene CaCDC4, partial suppression of filamentation), (deletion of gene GPH1, reduction in germ tube formation), (deletion of gene GPH1, reduction in cell surface hydrophobicity), (deletion of gene GPH1, increase in fibronectin binding), (deletion of gene GPH1, increase in adhesion), (deletion of gene GPH1, similar capacity to form mature biofilm)\n",
      "Progress: 22.10% completed\n",
      "Response for Row 443:\n",
      "Answer to Question 2:  (expression of gene McBBE, consequence 1.12 mg/L yield of (S)-scoulerine), (deletion of N-terminal amino acids of gene McBBE, consequence decrease in (S)-scoulerine production), (integration of gene McBBE using CRISPR-Cas9, consequence no significant increase in (S)-scoulerine yield)\n",
      "Progress: 22.15% completed\n",
      "Response for Row 444:\n",
      "Answer to Question 2: (integrated expression of coTAL, improved caffeic acid production), (episomal expression of coC3H, improved caffeic acid production), (episomal expression of coCPR1, improved caffeic acid production), (delta integration of coC3H, increased caffeic acid production), (delta integration of coCPR1, increased caffeic acid production), (delta integration method, 50-fold increased caffeic acid production)\n",
      "Progress: 22.20% completed\n",
      "Response for Row 445:\n",
      "Answer to Question 2: (deletion of ERG20 promoter, increased GPP production), (replacement of ERG20 promoter with ERG1 promoter, increased GPP-derived linalool production), (replacement of ERG20 promoter with ERG1 promoter, increased NPP-derived nerol production)\n",
      "Progress: 22.25% completed\n",
      "Response for Row 446:\n",
      "Answer to Question 2: (deletion of gene VdAda1, consequence severe defects in the formation of conidia), (deletion of gene VdAda1, consequence severe defects in the formation of microsclerotia), (deletion of gene VdAda1, consequence defects in melanin biosynthesis), (deletion of gene VdAda1, consequence reduction in virulence), (deletion of gene VdAda1, consequence genome-wide alteration of the V. dahliae transcriptome), (deletion of gene VdAda1, consequence altered expression of genes encoding transcription factors), (deletion of gene VdAda1, consequence altered expression of genes encoding secreted proteins)\n",
      "Progress: 22.30% completed\n",
      "Response for Row 447:\n",
      "Answer to Question 2: (deletion of gene FvMed1, increased FB1 production)\n",
      "Progress: 22.35% completed\n",
      "Response for Row 448:\n",
      "Answer to Question 2: (expression of gene NDI1, resisted rotenone-induced changes in cell morphology), (expression of gene NDI1, resisted rotenone-induced loss of cell viability), (expression of gene NDI1, resisted rotenone-induced accumulation of α-synuclein), (expression of gene NDI1, resisted rotenone-induced accumulation of pS129 α-synuclein), (expression of gene NDI1, resisted rotenone-induced mitochondrial ROS production), (expression of gene NDI1, resisted rotenone-induced mitochondria-mediated apoptosis), (expression of gene NDI1, increased basal oxygen consumption), (expression of gene NDI1, increased maximal oxygen consumption), (expression of gene NDI1, increased mitochondrial coupling efficiency), (expression of gene NDI1, increased basal ATP content), (expression of gene NDI1, increased oligomycin-sensitive ATP content), (expression of gene NDI1, increased complex I activity), (expression of gene NDI1, increased number of viable dopaminergic neurons), (expression of gene NDI1, increased expression of tyrosine hydroxylase), (expression of gene NDI1, increased content of dopamine in right striatum)\n",
      "Progress: 22.40% completed\n",
      "Response for Row 449:\n",
      "Answer to Question 2: (deletion of gene Cbf1, consequence 1: accumulation of cenRNAs at all phases of the cell cycle), (deletion of gene Cbf1, consequence 2: increased chromosome mis-segregation)\n",
      "Progress: 22.45% completed\n",
      "Response for Row 450:\n",
      "Answer to Question 2: (expression of ZNF1, activation of xylose metabolism), (expression of ZNF1, activation of xylose reductase genes), (expression of ZNF1, activation of pentose phosphate shunt), (expression of ZNF1, activation of related pathways for xylose utilisation), (deletion of Bud21, increased xylitol production), (overexpression of ZNF1, increased xylitol production to 12.14 g/L), (overexpression of ZNF1, increased tolerance to lignocellulosic inhibitors), (overexpression of ZNF1, improved cell dry weight)\n",
      "Progress: 22.50% completed\n",
      "Response for Row 451:\n",
      "Answer to Question 2: (overexpression of gene XySC1, consequence: efficient consumption and metabolism of xylose under aerobic conditions)\n",
      "Progress: 22.55% completed\n",
      "Response for Row 452:\n",
      "Answer to Question 2: (overexpression of gene Itr1, increase in glucaric acid yield by 26%), (fusional expression of MIOX4 and Udh, increase in glucaric acid yield by 40%), (down-expression of gene ZWF1, improvement in glucaric acid production efficiency)\n",
      "Progress: 22.60% completed\n",
      "Response for Row 453:\n",
      "Answer to Question 2: (deletion of gene MoOPY2, consequence hyphal growth defects), (deletion of gene MoOPY2, consequence conidiation defects), (deletion of gene MoOPY2, consequence germ tube extension defects), (deletion of gene MoOPY2, consequence appressorium formation defects), (deletion of gene MoOPY2, consequence appressorium turgor generation defects), (deletion of gene MoOPY2, consequence invasive growth defects), (deletion of gene MoOPY2, consequence attenuated pathogenicity), (deletion of gene MoOPY2, consequence cumulative MoAtg8 lipidation), (deletion of gene MoOPY2, consequence rapid GFP-MoAtg8 degradation)\n",
      "Progress: 22.65% completed\n",
      "Response for Row 454:\n",
      "Answer to Question 2: (TOR signaling overexpression, regulates ribosome production), (TOR signaling overexpression, represses autophagy), (TOR signaling overexpression, represses cellular stress response), (PKA signaling overexpression, regulates ribosome production), (PKA signaling overexpression, represses autophagy), (PKA signaling overexpression, represses cellular stress response), (Sch9 overexpression, shares kinase consensus motif with PKA), (TORC1 overexpression, directly regulates PKA), (TOR and PKA signaling overexpression, targets shared transcription factors), (TORC1 overexpression, activates PP2A<sup>Sit4</sup>/PP2A branch)\n",
      "Progress: 22.70% completed\n",
      "Response for Row 455:\n",
      "Answer to Question 2:  (mutation of gene TDH3 UAS, enhanced promoter strength), (mutation of gene TDH3 UAS, increased transcriptional regulation dynamic range), (mutation of gene TDH3 UAS, improved β-carotene production)\n",
      "Progress: 22.75% completed\n",
      "Response for Row 456:\n",
      "Answer to Question 2: (expression of GDP-bound Gtr1p, resistance to hydrogen peroxide), (expression of GTP-bound Gtr1p, sensitivity to hydrogen peroxide), (deletion of Iml1p, sensitivity to hydrogen peroxide), (expression of GDP-bound Gtr1p, induction of autophagy), (expression of GTP-bound Gtr1p, decreased SNQ2 expression), (overexpression of SNQ2, rescue of oxidative stress sensitivity)\n",
      "Progress: 22.80% completed\n",
      "Response for Row 457:\n",
      "Answer to Question 2: (deletion of gene CAP1, reduced utilization of non-preferred amino acids), (deletion of gene END3, reduced utilization of non-preferred amino acids), (overexpression of gene CAP1, increased utilization of non-preferred amino acids), (overexpression of gene END3, increased utilization of non-preferred amino acids), (overexpression of gene CAP1, reduced accumulation of urea), (overexpression of gene END3, reduced accumulation of urea), (deletion of gene ATP12, reduced utilization of non-preferred amino acids), (deletion of gene MRPL22, reduced utilization of non-preferred amino acids), (deletion of gene MRP1, reduced utilization of non-preferred amino acids), (deletion of gene NAM9, reduced utilization of non-preferred amino acids), (overexpression of gene ATP12, increased utilization of non-preferred amino acids), (overexpression of gene MRPL22, increased utilization of non-preferred amino acids), (overexpression of gene MRP1, increased utilization of non-preferred amino acids), (overexpression of gene NAM9, increased utilization of non-preferred amino acids), (overexpression of gene ATP12, reduced accumulation of urea), (overexpression of gene MRPL22, reduced accumulation of urea), (overexpression of gene MRP1, reduced accumulation of urea), (overexpression of gene NAM9, reduced accumulation of urea)\n",
      "Progress: 22.85% completed\n",
      "Response for Row 458:\n",
      "Answer to Question 2: (deletion of gene Dxo1, consequence 1: impaired final 5' end processing of 25S rRNA), (overexpression of gene Dxo1, consequence 1: enhanced distributive exonuclease activity on endonuclease or decapping products)\n",
      "Progress: 22.90% completed\n",
      "Response for Row 459:\n",
      "Answer to Question 2:  (overexpression of gene AoErg10s, consequence growth speed change), (overexpression of gene AoErg10s, consequence sporulation change), (overexpression of gene AoErg10s, consequence fatty acid content change), (overexpression of gene AoErg10s, consequence ergosterol content change), (expression of gene AoErg10A, consequence recovery of S. cerevisiae erg10 weak mutant phenotype), (expression of gene AoErg10D, consequence recovery of S. cerevisiae erg10 weak mutant phenotype), (expression of gene AoErg10E, consequence recovery of S. cerevisiae erg10 weak mutant phenotype), (expression of gene AoErg10F, consequence recovery of S. cerevisiae erg10 weak mutant phenotype), (expression of gene AoErg10B ΔMTS, consequence recovery of S. cerevisiae erg10 weak mutant phenotype), (expression of gene AoErg10D, consequence recovery of S. cerevisiae pot1 mutant phenotype), (expression of gene AoErg10E, consequence recovery of S. cerevisiae pot1 mutant phenotype), (expression of gene AoErg10F, consequence recovery of S. cerevisiae pot1 mutant phenotype)\n",
      "Progress: 22.95% completed\n",
      "Response for Row 460:\n",
      "Answer to Question 2: (expression of gene amyA, increased amylolytic activity), (expression of gene glaA, increased amylolytic activity), (expression of gene amyA, production of more than 4 g/L ethanol), (expression of gene glaA, production of more than 4 g/L ethanol)\n",
      "Progress: 23.00% completed\n",
      "Response for Row 461:\n",
      "Answer to Question 2: (deletion of gene TUP1/CYC8, compromised tolerance to heavy metals exposure), (deletion of gene TUP1/CYC8, derepression of GSH1 expression), (deletion of gene TUP1/CYC8, altered endoplasmic reticulum homeostasis), (deletion of gene TUP1/CYC8, failure to activate unfolded protein response pathway), (deletion of gene TUP1/CYC8, impaired activation of KAR2 transcription)\n",
      "Progress: 23.05% completed\n",
      "Response for Row 462:\n",
      "Answer to Question 2: (deletion of gene whi2, consequence 1), (deletion of gene whi2, consequence 2), (deletion of gene whi2, consequence 3)\n",
      "Progress: 23.10% completed\n",
      "Response for Row 463:\n",
      "Answer to Question 2: (suppression of survivin gene expression, apoptosis in breast cancer cells)\n",
      "Progress: 23.15% completed\n",
      "Response for Row 464:\n",
      "Answer to Question 2: (deletion of ERG6 gene, increase in DSG synthesis), (deletion of ERG6 gene, reduction in by-product formation), (overexpression of cholesterol 22-hydroxylase, construction of DSG biosynthesis pathway), (optimization of CYP to CPR ratio, increase in DSG production), (overexpression of PDR5, contribution to DSG production), (overexpression of PRY1, contribution to DSG production)\n",
      "Progress: 23.20% completed\n",
      "Response for Row 465:\n",
      "Answer to Question 2: (overexpression of gene PK-w, consequence functionally active proteinase K), (overexpression of gene PK-w, consequence secretion into the culture medium), (overexpression of gene PK-w, consequence productivity of 25 μg/mL), (overexpression of gene PK-w, consequence high specific proteolytic activity of ~5000 U/mg)\n",
      "Progress: 23.25% completed\n",
      "Response for Row 466:\n",
      "Answer to Question 2: (expression of cinB, nearly complete loss of embryo viability), (expression of cinA, rescue of embryo viability), (expression of CidB, inhibition of growth), (expression of CidA, rescue of growth), (expression of CinB, dependence on nuclease active site for toxicity)\n",
      "Progress: 23.30% completed\n",
      "Response for Row 467:\n",
      "Answer to Question 2: (expression of gene D-limonene, consequence 1.48-fold production improvement in S. cerevisiae), (expression of gene ovalbumin, consequence 1.71-fold production improvement in B. subtilis)\n",
      "Progress: 23.35% completed\n",
      "Response for Row 468:\n",
      "Answer to Question 2: (overexpression of gene RAD52, enhanced HR efficiency), (overexpression of gene RAD59, enhanced HR efficiency), (overexpression of gene MRE11, enhanced HR efficiency), (overexpression of gene SAE2, enhanced HR efficiency), (enhanced HR efficiency, enabled single-locus integration of heterologous gene expression cassettes with ∼40 bp homology arms with efficiencies as high as 100%), (enhanced HR efficiency, enabled two-loci integration of heterologous gene expression cassettes with ∼40 bp homology arms with efficiencies as high as ∼98%), (enhanced HR efficiency, enabled three-loci integration of heterologous gene expression cassettes with ∼40 bp homology arms with efficiencies as high as ∼81%), (construction of β-carotene producing strain, demonstrated in a single step), (optimization of betaxanthin producing strain, demonstrated in a single step)\n",
      "Progress: 23.40% completed\n",
      "Response for Row 469:\n",
      "Answer to Question 2: (CBG exposure, reduces reactive oxygen species levels), (CBG exposure, inhibits Interleukin-1β release), (CBG exposure, inhibits Interleukin-6 release), (CBG exposure, inhibits Interleukin-8 release), (CBG exposure, inhibits tumor necrosis factor α release), (CBG application, improves transepidermal water loss), (CBG application, reduces redness appearance)\n",
      "Progress: 23.45% completed\n",
      "Response for Row 470:\n",
      "Answer to Question 2: (no expression/deletion of gene mentioned, discussion on conserved transcriptional regulators across species)\n",
      "Progress: 23.50% completed\n",
      "Response for Row 471:\n",
      "Answer to Question 2: (altered translational efficiency during ageing, decline in cellular proteostasis), (ageing, altered kinetics of translation elongation), (ribosome pausing at specific positions in aged yeast and worms, increased ribosome collisions), (increased ribosome collisions, trigger ribosome-associated quality control (RQC)), (aged yeast cells, impaired clearance and increased aggregation of RQC substrates), (long-lived yeast mutants, reduced age-dependent ribosome pausing), (extended lifespan, greater flux through the RQC pathway), (nascent polypeptides exhibiting age-dependent ribosome pausing in C. elegans, strongly enriched among age-dependent protein aggregates), (ageing, increased pausing and aggregation of many components of proteostasis), (proteostasis collapse, systemic decline during ageing)\n",
      "Progress: 23.55% completed\n",
      "Response for Row 472:\n",
      "Answer to Question 2: (translational control, shapes the proteome), (alternative 5' UTR isoforms, increased translation), (C-rich silencers, repress translation)\n",
      "Progress: 23.60% completed\n",
      "Response for Row 473:\n",
      "Answer to Question 2: (phosphorylation of Sen1, reduces efficiency of Sen1-mediated termination), (phosphorylation of Sen1, impairs termination at unannotated non-coding gene), (impairment of termination at unannotated non-coding gene, represses expression of Zap1), (repression of Zap1, causes expression pattern mimicking conditions of Zn excess)\n",
      "Progress: 23.65% completed\n",
      "Response for Row 474:\n",
      "Answer to Question 2: (deletion of gene ILV1, intracellular accumulation of isoleucine), (deletion of gene ILV1, extracellular overproduction of 2-methyl-1-butanol), (deletion of gene ILV1, increased production of 2-methylbutyl acetate)\n",
      "Progress: 23.70% completed\n",
      "Response for Row 475:\n",
      "Answer to Question 2: (no specific gene expression/deletion mentioned, general discussion on 2,3-BDO production using genetic engineering and metabolic engineering in Saccharomyces cerevisiae)\n",
      "Progress: 23.75% completed\n",
      "Response for Row 476:\n",
      "Answer to Question 2: (induction of anticancer immunity, improved abundance and functional orientation of effector components in tumor microenvironment), (induction of anticancer immunity, improved abundance and functional orientation of effector components in TDLN), (treatment with small size YCW NPs, anticancer efficiency in melanoma-challenged mice), (combination with PD-L1 blockade, enhanced anticancer efficiency)\n",
      "Progress: 23.80% completed\n",
      "Response for Row 477:\n",
      "Answer to Question 2: (no gene overexpression/deletion mentioned, SptF catalyzes several continuous oxidation reactions), (no gene overexpression/deletion mentioned, SptF exhibits extremely broad substrate specificity), (no gene overexpression/deletion mentioned, SptF efficiently produced unique cyclopropane-ring-fused 5/3/5/5/6/6 and 5/3/6/6/6 scaffolds from terretonins), (no gene overexpression/deletion mentioned, SptF hydroxylates steroids with different regiospecificities), (no gene overexpression/deletion mentioned, crystallographic and structure-based mutagenesis studies revealed the molecular basis of the enzyme reactions), (no gene overexpression/deletion mentioned, malleability of the loop region contributes to the remarkable substrate promiscuity)\n",
      "Progress: 23.85% completed\n",
      "Response for Row 478:\n",
      "Answer to Question 2: (deletion of gene PDC, consequence glucose uptake decreased), (expression of gene dCas9, consequence ethanol synthesis reduced), (expression of gene dCas9, consequence 2,3-butanediol production increased)\n",
      "Progress: 23.90% completed\n",
      "Response for Row 479:\n",
      "Answer to Question 2: (deletion of activation domains from dSpCas9, improved interaction with AcrIIs), (expression of AcrIIA4, high inhibition at low concentrations), (deletion of activation domains from dSpCas9, limited control of gene transcription regulation hindered), (expression of bare dSpCas9, best solution for building synthetic genetic networks in yeast)\n",
      "Progress: 23.95% completed\n",
      "Response for Row 480:\n",
      "Answer to Question 2: (no specific gene expression/deletion mentioned, no direct causal relationship extracted)\n",
      "Progress: 24.00% completed\n",
      "Response for Row 481:\n",
      "Answer to Question 2: (deletion of gene related to protein biosynthesis, consequence inhibition of protein biosynthesis), (overexpression of gene related to glycolysis, consequence activation of intracellular metabolic pathways of energy production), (overexpression of gene related to glycogen synthesis, consequence increase of glycogen), (deletion of gene related to energy consumption metabolism, consequence decrease of energy consumption metabolism), (overexpression of gene related to autophagy, consequence cells undergoing autophagy), (overexpression of gene related to MAPK signaling pathway, consequence activation of intracellular metabolic pathways), (overexpression of gene related to redox regulation, consequence induction of oxidative stress), (deletion of gene related to amino acid synthesis, consequence suppression of amino acid synthesis), (overexpression of gene related to sexual reproduction and spore formation, consequence induction of sexual reproduction and spore formation)\n",
      "Progress: 24.05% completed\n",
      "Response for Row 482:\n",
      "Answer to Question 2: (deletion of gene RIM21, improved flocculation capability), (deletion of gene RIM21, up-regulated expression of FLO5), (deletion of gene RIM21, up-regulated expression of Lg-FLO1), (deletion of gene RIM21, enhanced resistance to cell wall inhibitors)\n",
      "Progress: 24.10% completed\n",
      "Response for Row 483:\n",
      "Answer to Question 2: (overexpression of gene IZH1, down-regulation of ERG9 expression), (deletion of gene ERG9, diversion of metabolic flux to β-carotene pathway), (addition of oleic acid, improvement in LDs-triacylglycerol metabolism), (addition of oleic acid, increase in β-carotene content), (induction of extra constitutive β-carotene synthesis pathway, conversion of precursor farnesol to β-carotene), (engineering of LDs related genes, minor promotion on β-carotene accumulation)\n",
      "Progress: 24.15% completed\n",
      "Response for Row 484:\n",
      "Answer to Question 2: (deletion of gene SCW4, consequence increased secretion of hGH), (deletion of gene SCW4, consequence increased secretion of exendin-4), (deletion of gene SCW4, consequence increased secretion of hPTH), (modification of gene SCW4, consequence improved function as secretion fusion partner), (presence of O-glycosylation sites in Scw4p, consequence enhanced protein secretion), (presence of hydrophilic sequences in Scw4p, consequence enhanced protein secretion)\n",
      "Progress: 24.20% completed\n",
      "Response for Row 485:\n",
      "Answer to Question 2: (deletion of gene Spt6-tSH2:Rpb1 interface, changes in transcription start site selection), (deletion of gene Spt6-tSH2:Rpb1 interface, changes in Spt6 accumulation pattern at specific sites), (deletion of gene Spt6-tSH2:Rpb1 interface, diminished splicing efficiency), (deletion of gene Spt6-tSH2:Rpb1 interface, impaired chromatin-mediated repression), (deletion of gene Spt6-tSH2:Rpb1 interface, role in maintaining +1 nucleosomes at ribosomal protein genes)\n",
      "Progress: 24.25% completed\n",
      "Response for Row 486:\n",
      "Answer to Question 2: (deletion of CRE1/CreA, release from carbon catabolite repression), (overexpression of CRE1/CreA, repression of cellulolytic gene expression), (interaction of CRE1/CreA with Tup1-Cyc8, transcriptional repression), (interaction of PoCyc8 with PoCreA, transcriptional repression), (interaction of PoCyc8 with PoSet2, transcriptional repression), (methylation of H3K36 by PoSet2, repression of cellulolytic gene expression), (expression of TF PoCreA, positive correlation with di-methylation level of H3K36 in cellulolytic gene promoters)\n",
      "Progress: 24.30% completed\n",
      "Response for Row 487:\n",
      "Answer to Question 2: (deletion of gene ncRNA, consequence 1: more abundant reverse-transcribed DNA (RT-DNA)), (expression of gene retron operon, consequence 1: efficient reverse transcription), (expression of gene retron operon, consequence 2: more efficient genome editing), (expression of gene retron RT-DNA, consequence 1: precise editing of cultured human cells)\n",
      "Progress: 24.35% completed\n",
      "Response for Row 488:\n",
      "Answer to Question 2: (expression of gene RmproB, high protease activity of 1242.2 U/mL), (expression of gene RmproB, optimal activity at pH 2.5), (expression of gene RmproB, optimal activity at 40 °C), (expression of gene RmproB, stability within pH 1.5-6.5), (expression of gene RmproB, stability up to 45 °C), (expression of gene RmproB, broad substrate specificity), (expression of gene RmproB, milk-clotting activity of 3894.1 SU/mg), (expression of gene RmproB, cleavage of specific peptide bonds in κ-casein), (expression of gene RmproB, preparation of ACE-inhibitory peptides), (expression of gene RmproB, high ACE-inhibitory activity IC50 of 0.195 mg/mL)\n",
      "Progress: 24.40% completed\n",
      "Response for Row 489:\n",
      "Answer to Question 2: (expression of heterologous terpene synthases, increased isoprenoid production), (metabolic engineering of the mevalonate pathway, increased isoprenoid production), (metabolic engineering of the MEP pathway, increased isoprenoid production), (expression tuning of heterologous genes, improved isoprenoid production)\n",
      "Progress: 24.45% completed\n",
      "Response for Row 490:\n",
      "Answer to Question 2: (deletion of nucleosomes, increased GG-NER efficiency), (presence of nucleosomes, obstructed GG-NER), (overexpression of RNA polymerase-I, enhanced TC-NER), (dislodgement of RNA polymerase-I, partial replacement by nucleosomes)\n",
      "Progress: 24.50% completed\n",
      "Response for Row 491:\n",
      "Answer to Question 2: (expression of gene CYP5150L8, formation of HLDOA), (expression of gene CYP5139G1, C-28 oxidation of HLDOA), (expression of gene CYP5139G1, formation of DHLDOA), (adjustment of hygromycin dosage, manipulation of plasmid copy number pRS425-Hyg-CYP5150L8-iGLCPR), (adjustment of geneticin G418 dosage, manipulation of plasmid copy number pRS426-KanMx-CYP5139G1), (balanced expression of CYP5150L8 and CYP5139G1, improvement of DHLDOA production)\n",
      "Progress: 24.55% completed\n",
      "Response for Row 492:\n",
      "Answer to Question 2: (deletion of nonoptimal codons, decrease in transcript's half-life), (insertion of second nonoptimal cluster near 5' end, increase in mRNA's half-life), (5' end cluster suppression, reduction in ribosome queues), (introduction of two nonoptimal clusters, increase in mRNA half-life up to fourfold), (presence of two clusters, 5' end cluster determines mRNA half-life)\n",
      "Progress: 24.60% completed\n",
      "Response for Row 493:\n",
      "Answer to Question 2:  (expression of gene lgdA, consequence LGDH activity), (expression of gene lgdA, consequence facilitates bacterial catabolism of levoglucosan), (expression of gene sugar isomerase, consequence facilitates bacterial catabolism of levoglucosan), (expression of gene transport proteins, consequence facilitates bacterial catabolism of levoglucosan)\n",
      "Progress: 24.65% completed\n",
      "Response for Row 494:\n",
      "Answer to Question 2: (expression of L-A, provides proteins for expression), (expression of L-A, provides replication for both viruses), (expression of L-A, provides capsids for both viruses), (expression of M, encodes the toxin), (expression of L-A-1, associates with M1), (expression of L-A-2, associates with M2), (expression of L-A-28, associates with M28), (expression of L-A-lus, associates with Mlus)\n",
      "Progress: 24.70% completed\n",
      "Response for Row 495:\n",
      "Answer to Question 2: (deletion of gene Gcn5, consequence 1: decrease in histone acetylation), (deletion of gene Gcn5, consequence 2: affects gene expression and development), (deletion of gene chiffon, consequence 1: decrease in histone acetylation exceeding Ada2b mutation), (expression of chiffon gene product CHAT, consequence 1: regulates expression of a subset of developmental genes), (expression of chiffon gene product DDK, consequence 1: necessary for maternally driven DNA replication in the embryo)\n",
      "Progress: 24.75% completed\n",
      "Response for Row 496:\n",
      "Answer to Question 2: (expression of genes related to central carbon metabolism flux, consequence 1: minimizes proteome cost in Crabtree-positive yeasts), (expression of genes related to translation efficiency, consequence 1: lower ribosomal content and/or efficiency in Crabtree-positive yeasts), (expression of genes related to ATP yield, consequence 1: maximized ATP yield in Crabtree-negative yeasts), (expression of genes related to protein translation parameters, consequence 1: higher protein translation parameters in Crabtree-negative yeasts)\n",
      "Progress: 24.80% completed\n",
      "Response for Row 497:\n",
      "Answer to Question 2: (deletion of gene FDC1, avoid ferulic acid deviation), (expression of gene 4CL1, produce curcumin), (expression of gene FerA, produce curcumin), (expression of gene PKSs from Oryza sativa, produce curcumin), (expression of gene PKSs from C. longa, produce curcumin)\n",
      "Progress: 24.85% completed\n",
      "Response for Row 498:\n",
      "Answer to Question 2: (expression of gene BbGT, consequence improved water solubility of natural products), (expression of gene BbGT, consequence conversion of quercetin to five monoglucosylated products), (expression of gene BbGT, consequence conversion of quercetin to one diglucosylated product), (expression of gene BbGT, consequence production of quercetin-7-O-β-D-glucoside in E. coli), (expression of gene BbGT, consequence production of quercetin-7-O-β-D-glucoside in P. putida), (expression of gene BbGT, consequence production of quercetin-7-O-β-D-glucoside in P. pastoris), (expression of gene BbGT, consequence production of quercetin-3-O-β-D-glucoside in S. cerevisiae), (expression of gene BbGT, consequence stimulation by Ca<sup>2+</sup>), (expression of gene BbGT, consequence stimulation by Mg<sup>2+</sup>), (expression of gene BbGT, consequence stimulation by Mn<sup>2+</sup>), (expression of gene BbGT, consequence inhibition by Zn<sup>2+</sup>)\n",
      "Progress: 24.90% completed\n",
      "Response for Row 499:\n",
      "Answer to Question 2: (deletion of gene Gal80, consequence 1: stringent regulation of gene expression), (expression of gene Gal4, consequence 1: copper-induced transcriptional activation), (expression of gene Gal80, consequence 1: copper-inhibited repression), (expression of gene CUP1, consequence 1: native copper-inducible promoter activity), (expression of gene EGFP, consequence 1: 72-fold induction with CuIGR4 system), (expression of gene responsible for lycopene production, consequence 1: 33-fold increase in lycopene production)\n",
      "Progress: 24.95% completed\n",
      "Response for Row 500:\n",
      "Answer to Question 2: (gene conversion-like process of VLR genes, generation of distinct antigen receptors), (expression of anti-carbohydrate VLRBs, isolation using yeast surface display platform), (binding of anti-glycan VLRBs, enrichment by glycan microarrays), (sequencing of VLR antibody clone, expression as VLR-IgG-Fc fusion protein)\n",
      "Progress: 25.00% completed\n",
      "Response for Row 501:\n",
      "Answer to Question 2: (gene expression-based node addition, preserves resilience of PPI network), (gene expression-based node addition, increases resilience of PPI network)\n",
      "Progress: 25.05% completed\n",
      "Response for Row 502:\n",
      "Answer to Question 2:  (mutation of gene MpFADS6[Q409R], consequence abolished desaturation), (mutation of gene MpFADS6[M242P], consequence abolished desaturation), (mutation of gene MpFADS6[F419V], consequence significantly reduced catalytic activities), (mutation of gene MpFADS6[A374Q], consequence significantly reduced catalytic activities)\n",
      "Progress: 25.10% completed\n",
      "Response for Row 503:\n",
      "Answer to Question 2: (deletion of gene involved in glucose sensing, reduced glucose preference), (overexpression of gene involved in xylose sensing, improved xylose utilization), (modification of signaling pathway, enhanced xylose consumption rate)\n",
      "Progress: 25.15% completed\n",
      "Response for Row 504:\n",
      "Answer to Question 2: (depression of whole-genome duplicates, growth adaptation to acidic stress), (depression of whole-genome duplicates, reduction of growth rate), (depression of whole-genome duplicates, increase in biomass production), (overexpression of ROS response genes, adaptation to acidic stress)\n",
      "Progress: 25.20% completed\n",
      "Response for Row 505:\n",
      "Answer to Question 2: (deletion of gene Fot1, reduced oligopeptide utilization), (deletion of gene Fot2, reduced oligopeptide utilization), (overexpression of gene Fot1, increased oligopeptide utilization), (overexpression of gene Fot2, increased oligopeptide utilization), (overexpression of gene Fot1, improved fermentation efficiency), (overexpression of gene Fot2, improved fermentation efficiency), (overexpression of gene Fot1, increased production of desirable organoleptic compounds), (overexpression of gene Fot2, increased production of desirable organoleptic compounds)\n",
      "Progress: 25.25% completed\n",
      "Response for Row 506:\n",
      "Answer to Question 2: (deletion of gene lac3, reduced virulence), (deletion of gene CpHsp24, reduced virulence), (overexpression of gene CpHsp24, increased laccase activity), (interaction of protein CpHSP24, enhanced function of protein LAC3)\n",
      "Progress: 25.30% completed\n",
      "Response for Row 507:\n",
      "Answer to Question 2: (deletion of gene associated with ribosome function, consequence 1: reduced protein synthesis), (overexpression of gene related to energy metabolism, consequence 1: enhanced energy production), (overexpression of Hsp70/Hsp90 and related molecular chaperones, consequence 1: increased recognition and degradation of misfolded proteins), (overexpression of genes in ergosterol biosynthesis, consequence 1: increased tolerance to acetic acid stress)\n",
      "Progress: 25.35% completed\n",
      "Response for Row 508:\n",
      "Answer to Question 2: (deletion of gene MoRPD3, lethal effect), (depletion of gene MoRPD3, reduction in pathogenicity), (depletion of gene MoRPD3, delay in appressorium-mediated penetration), (depletion of gene MoRPD3, impaired invasive growth), (depletion of gene MoRPD3, smaller appressoria size), (association of gene MoRPD3, regulation through TOR kinase pathway)\n",
      "Progress: 25.40% completed\n",
      "Response for Row 509:\n",
      "Answer to Question 2: (deletion of gene xxx, consequence 1), (overexpression of gene xxx, consequence 2)...\n",
      "\n",
      "Note: The provided abstract does not specify particular genes being overexpressed or deleted, nor does it provide specific consequences of such actions. It discusses changes in metabolic pathways and gene expression levels but does not detail individual genes or their direct causal relationships. Therefore, I cannot provide specific gene-expression consequences pairs as requested without making assumptions beyond the information given.\n",
      "Progress: 25.45% completed\n",
      "Response for Row 510:\n",
      "Answer to Question 2: (gene deletion simulations, consequence 1), (gene expression data integration, consequence 2)\n",
      "Progress: 25.50% completed\n",
      "Response for Row 511:\n",
      "Answer to Question 2: (deletion of gene cat, consequence 1), (expression of gene cat, consequence 2)\n",
      "Progress: 25.55% completed\n",
      "Response for Row 512:\n",
      "Answer to Question 2: (deletion of gene Set1, reduced virulence), (deletion of gene Set1, higher cellular mitochondrial reactive oxygen species (ROS)), (deletion of gene Set1, increased H2O2 sensitivity), (deletion of gene Set1, prevents proper colony formation), (deletion of gene Set1, defective expression of mitochondrial protein genes), (deletion of gene Set1, disrupted mitochondrial membrane integrity)\n",
      "Progress: 25.60% completed\n",
      "Response for Row 513:\n",
      "Answer to Question 2: (inducible expression of CePAH1, up-regulation of diacylglycerol), (inducible expression of CePAH1, increase in oil content)\n",
      "Progress: 25.65% completed\n",
      "Response for Row 514:\n",
      "Answer to Question 2:  (supplementation with Saccharomyces cerevisiae fermentation products, reduced thoracic ultrasonography scores), (supplementation with Saccharomyces cerevisiae fermentation products, less severe lung pathology scores), (supplementation with Saccharomyces cerevisiae fermentation products, higher serum IL-6 concentrations), (supplementation with Saccharomyces cerevisiae fermentation products, lower concentrations of serum nonesterified fatty acids), (supplementation with Saccharomyces cerevisiae fermentation products, lower concentrations of beta-hydroxybutyric acid)\n",
      "Progress: 25.70% completed\n",
      "Response for Row 515:\n",
      "Answer to Question 2:  (overexpression of genes involved in pyruvate metabolism, increased ethanol yield), (overexpression of genes involved in glycolysis, increased ethanol yield), (overexpression of genes involved in pentose phosphate pathway, increased ethanol yield), (overexpression of genes involved in glucose transport, increased ethanol yield), (overexpression of genes involved in reducing power production, increased ethanol yield), (increase in intracellular glucose, increased ethanol yield), (increase in NADH content, increased ethanol yield), (decrease in acetate and its derivatives, increased ethanol yield)\n",
      "Progress: 25.75% completed\n",
      "Response for Row 516:\n",
      "Answer to Question 2: (deletion of gene for testosterone, consequence decreased sperm density and viability), (deletion of gene for FSH, consequence decreased sperm density and viability), (deletion of gene for LH, consequence decreased sperm density and viability), (overexpression of gene for anti-apoptotic proteins, consequence improved sperm parameters), (overexpression of gene for steroid synthetase, consequence improved sperm parameters)\n",
      "Progress: 25.80% completed\n",
      "Response for Row 517:\n",
      "Answer to Question 2: (deletion of 3' UTR sequence elements, consequence 1: reduced mRNA stability), (deletion of 3' UTR sequence elements, consequence 2: decreased translation), (overexpression of consensus 3' UTR efficiency element, consequence 1: increased protein expression), (binding of Puf proteins, consequence 1: increased expression), (mutation within efficiency element motifs, consequence 1: altered binding affinity for Hrp1 protein), (presence of consensus positioning element, consequence 1: minor effect on expression)\n",
      "Progress: 25.85% completed\n",
      "Response for Row 518:\n",
      "Answer to Question 2: (deletion of GAL1, consequence 1), (deletion of GAL80, consequence 2), (overexpression of gene with synthetic GAL promoters, consequence 3), (manipulation of UASs in GAL promoters, consequence 4), (replacement of core promoter in GAL promoters, consequence 5)\n",
      "Progress: 25.90% completed\n",
      "Response for Row 519:\n",
      "Answer to Question 2: (deletion of general stress response genes, improved robustness), (overexpression of oxidative stress response genes, enhanced tolerance to inhibitors), (overexpression of furan-aldehyde reductases, reduced toxicity of furans), (deletion of certain transporter genes, altered transport capabilities), (overexpression of specific drug:H+ antiporters, increased efflux of toxic compounds), (overexpression of thiamin biosynthesis genes, improved thiamin levels), (overexpression of biotin biosynthesis genes, improved biotin levels)\n",
      "Progress: 25.95% completed\n",
      "Response for Row 520:\n",
      "Answer to Question 2: (deletion of gene ssl2, consequence: upstream TSS shifts at the model promoter ADH1), (deletion of gene ssl2, consequence: downstream TSS shifts at the model promoter ADH1), (deletion of gene ssl2, consequence: altering TSS selection by increasing the distribution of TSSs used at individual promoters), (deletion of gene ssl2, consequence: altering TSS selection by narrowing the distribution of TSSs used at individual promoters)\n",
      "Progress: 26.00% completed\n",
      "Response for Row 521:\n",
      "Answer to Question 2: (deletion of gene BgtCDA1, inhibition of Bgt invasion), (overexpression of gene TiAP1, up-regulation of cell wall xylan metabolism), (overexpression of gene TiAP1, up-regulation of lignin biosynthesis-related genes), (overexpression of gene TiAP1, up-regulation of defence genes involved in signal transduction), (interaction of TiAP1, inactivation of BgtCDA1 deacetylation function), (inactivation of BgtCDA1 deacetylation function, exposure of chitin oligomers), (exposure of chitin oligomers, triggering of wheat immune response), (triggering of wheat immune response, inhibition of Bgt growth and penetration), (inhibition of Bgt growth and penetration, enhancement of wheat resistance to pathogens)\n",
      "Progress: 26.05% completed\n",
      "Response for Row 522:\n",
      "Answer to Question 2: (expression of gene HvHorcH, increased salt tolerance in yeast), (expression of gene HvHorcH, increased protein levels in response to salinity treatment), (expression of gene HvHorcH, increased salt tolerance in Arabidopsis thaliana)\n",
      "Progress: 26.10% completed\n",
      "Response for Row 523:\n",
      "Answer to Question 2: (deletion of eCIS-related genes, no effect on morphological differentiation), (deletion of eCIS-related genes, no effect on antibiotic production), (deletion of eCIS-related genes, increased susceptibility to fungal invasion), (SLP expression elevation, response to co-culture with fungi), (co-culture with Bacillus subtilis, decreased SLP expression), (co-culture with Bacillus subtilis, increased antibiotic production)\n",
      "Progress: 26.15% completed\n",
      "Response for Row 524:\n",
      "Answer to Question 2: (deletion of Cbp7, affects cytb mRNA stability), (deletion of Cbp7, impairs detection of Cytb protein), (deletion of Cbp8, affects cytb mRNA stability), (deletion of Cbp8, impairs detection of Cytb protein), (interaction of Cbp7/Cbp8, modulates Cytb synthesis), (interaction of Cbp7/Cbp8, modulates Cox1 synthesis)\n",
      "Progress: 26.20% completed\n",
      "Response for Row 525:\n",
      "Answer to Question 2: (no gene expression/deletion mentioned, no specific consequences related to gene expression/deletion)\n",
      "Progress: 26.25% completed\n",
      "Response for Row 526:\n",
      "Answer to Question 2: (expression of gene ScCNS1, increased cordycepin production), (expression of gene ScCNS2, increased cordycepin production), (up-regulation of gene ZWF1, enhanced pentose phosphate pathway activity), (up-regulation of gene PRS4, enhanced purine metabolism), (up-regulation of gene ADE4, enhanced purine metabolism), (addition of Cu²⁺, increased cordycepin yield), (addition of adenine, increased cordycepin yield)\n",
      "Progress: 26.30% completed\n",
      "Response for Row 527:\n",
      "Answer to Question 2: (deletion of gene for specific caffeoyl-CoA production, consequence MbA not enhanced in N. benthamiana), (expression of gene Cc4CL1, consequence activation of coumaric and caffeic acid towards flavanone biosynthesis in yeast), (expression of gene Cc4CL2, consequence activation of coumaric and caffeic acid towards flavanone biosynthesis in yeast)\n",
      "Progress: 26.35% completed\n",
      "Response for Row 528:\n",
      "Answer to Question 2: (deletion of gene KanMX, alteration in transcription efficiency of neighboring genes), (deletion of gene KanMX, impairment of mRNA translation of neighboring genes), (deletion of gene KanMX, changes in protein abundance of neighboring genes), (deletion of gene KanMX, shift of transcription start site in 'head-to-head' orientation), (deletion of gene KanMX, introduction of new uAUG codons in expanded 5' UTR in 'head-to-head' orientation), (deletion of gene KanMX, activation of alternative polyadenylation signals in 'tail-to-tail' arrangement), (deletion of gene KanMX, alteration of 3' UTR in 'tail-to-tail' arrangement)\n",
      "Progress: 26.40% completed\n",
      "Response for Row 529:\n",
      "Answer to Question 2: (expression of gene ZFN, consequence 1), (expression of gene ZFN, consequence 2), (expression of gene ZFN, consequence 3), (deletion of gene LYS2, consequence 4), (insertion at gene LYS2, consequence 5), (expression of gene ZFN, consequence 6)\n",
      "Progress: 26.45% completed\n",
      "Response for Row 530:\n",
      "Answer to Question 2: (optogenetic control of gene expression, precise control of cellular functions), (optogenetic switches in yeasts, protein production), (optogenetic switches in yeasts, metabolite production)\n",
      "Progress: 26.50% completed\n",
      "Response for Row 531:\n",
      "Answer to Question 2: (expression of gene product, consequence 1: fitness defect in specific mutant background), (expression of gene product, consequence 2: little effect on fitness in wild-type background), (overexpression of wild-type gene, consequence 3: sensitization of yeast to increased dosage), (overexpression of gene carrying dominant-negative missense mutation, consequence 4: model of protein-inhibitor interactions)\n",
      "Progress: 26.55% completed\n",
      "Response for Row 532:\n",
      "Answer to Question 2: (deletion of gene akr1, increased intracellular ROS levels), (deletion of gene akr1, cell death), (deletion of gene erg3, increased intracellular ROS levels), (deletion of gene erg3, cell death), (calcium, inhibited increased intracellular ROS levels), (calcium, inhibited cell death), (calcium, induced expression of gene hxt1), (calcium, induced expression of gene agp1), (deletion of gene akr1, decreased cytosolic calcium concentration), (deletion of gene erg3, decreased cytosolic calcium concentration), (calcium/calcineurin signalling pathway, induced expression of gene akr1), (calcium/calcineurin signalling pathway, induced expression of gene erg3)\n",
      "Progress: 26.60% completed\n",
      "Response for Row 533:\n",
      "Answer to Question 2: (deletion of U2AF1, alternative splicing), (overexpression of two introns, alternative RNA isoforms)\n",
      "Progress: 26.65% completed\n",
      "Response for Row 534:\n",
      "Answer to Question 2: (deletion of gene Pus4/TruB, consequence 1: greater proliferation rates), (deletion of gene Pus4/TruB, consequence 2: shortened lifespan), (deletion of gene Pus4/TruB, consequence 3: altered protein synthesis), (deletion of gene Pus4/TruB, consequence 4: differential synthesis of dozens of proteins)\n",
      "Progress: 26.70% completed\n",
      "Response for Row 535:\n",
      "Answer to Question 2: (cloning and heterologous expression of gene Aa7DR1, Δ7-reduction activity in yeast), (high expression of candidate genes in rhizome, accumulation of timosaponins in rhizomes)\n",
      "Progress: 26.75% completed\n",
      "Response for Row 536:\n",
      "Answer to Question 2: (deletion of gene expression for key xylose-metabolism enzymes, reduced D-xylose consumption), (deletion of gene expression for xylose reductase (XR), decreased specific activity of XR), (deletion of gene expression for xylitol dehydrogenase (XDH), decreased specific activity of XDH), (deletion of gene expression for xylulokinase (XK), decreased specific activity of XK), (presence of D-glucose, inhibition of D-xylose consumption), (presence of D-glucose, prevention of D-xylose-mediated induction of XR gene), (presence of D-glucose, prevention of D-xylose-mediated induction of XDH gene), (presence of D-glucose, prevention of D-xylose-mediated induction of XK gene)\n",
      "Progress: 26.80% completed\n",
      "Response for Row 537:\n",
      "Answer to Question 2: (deletion of BDF2, increased nuclear retention of BDF2 mRNA), (deletion of BDF2, mediated by RNase III-mediated decay (RMD)), (expression of Rnt1p, cleavage of BDF2 mRNA), (inhibition of mRNA export, hyperactivation of RMD)\n",
      "Progress: 26.85% completed\n",
      "Response for Row 538:\n",
      "Answer to Question 2: (deletion of gene NRRL3_09595, increased chitin deposition), (deletion of gene NRRL3_09595, sensitivity to SDS), (deletion of gene NRRL3_09595, production of unknown yellow pigment), (deletion of gene cwcA, constitutive activation of cell wall stress response pathway), (constitutive activation of cell wall stress response pathway, increased chitin content in cell wall)\n",
      "Progress: 26.90% completed\n",
      "Response for Row 539:\n",
      "Answer to Question 2: (expression of gene Haa1, consequence relocation to the nucleus), (expression of gene BM3R1, consequence control expression of a reporter gene), (expression of gene reporter gene, consequence detection of acetic acid production)\n",
      "Progress: 26.95% completed\n",
      "Response for Row 540:\n",
      "Answer to Question 2: (heterologous expression of VuNAC1, improved growth), (heterologous expression of VuNAC1, increased biomass), (heterologous expression of VuNAC1, extended lifespan), (heterologous expression of VuNAC1, enhanced fermentation efficiency), (heterologous expression of VuNAC1, altered cellular composition of biomolecules), (heterologous expression of VuNAC1, conferred tolerance to high salinity), (heterologous expression of VuNAC1, conferred tolerance to osmotic stress), (heterologous expression of VuNAC1, conferred tolerance to freezing), (heterologous expression of VuNAC1, conferred tolerance to aluminum toxicity), (heterologous expression of VuNAC1, reprogrammed synthesis of nucleotides), (heterologous expression of VuNAC1, reprogrammed synthesis of vitamin B complex), (heterologous expression of VuNAC1, reprogrammed synthesis of amino acids), (heterologous expression of VuNAC1, reprogrammed synthesis of antioxidants), (heterologous expression of VuNAC1, reprogrammed synthesis of flavonoids), (heterologous expression of VuNAC1, reprogrammed synthesis of energy currencies), (heterologous expression of VuNAC1, reprogrammed synthesis of cofactors), (heterologous expression of VuNAC1, improved survival during starvation), (heterologous expression of VuNAC1, improved survival during stress recovery)\n",
      "Progress: 27.00% completed\n",
      "Response for Row 541:\n",
      "Answer to Question 2: (deletion of gene TRM1, decreased induction of AOX1 promoter), (deletion of gene TRM2, decreased induction of AOX1 promoter), (deletion of gene MPP1, decreased induction of AOX1 promoter), (overexpression of gene TRM1, increased induction of AOX1 promoter), (overexpression of gene MPP1, increased induction of AOX1 promoter), (presence of glycerol, catabolite repression of AOX1 expression), (presence of glucose, catabolite repression of AOX1 expression)\n",
      "Progress: 27.05% completed\n",
      "Response for Row 542:\n",
      "Answer to Question 2: (deletion of S. uvarum chromosome I, loss of genetic material), (duplication of S. cerevisiae chromosome III, increased gene dosage), (duplication of S. uvarum chromosome VII-XVI, increased gene dosage), (loss of heterozygosity on S. cerevisiae chromosome I, reduced genetic diversity), (overexpression of unknown genes due to adaptation, increased sulfite resistance), (slight overexpression of unknown genes due to adaptation, slightly improved ethanol tolerance), (underexpression of unknown genes due to adaptation, decreased glucose and fructose fermentation capacity)\n",
      "Progress: 27.10% completed\n",
      "Response for Row 543:\n",
      "Answer to Question 2: (deletion of lanosterol synthase, production of euphol), (expression of EtOSC5, production of euphol), (expression of EtOSC5, production of tirucallol), (expression of EtOSC6, production of taraxasterol), (expression of EtOSC6, production of β-amyrin), (chromosomal integration of EtOSC5 and three other precursor genes, production of euphol)\n",
      "Progress: 27.15% completed\n",
      "Response for Row 544:\n",
      "Answer to Question 2: (deletion of Mat*a*1, consequence 1), (overexpression of transcripts for mating type specific genes, consequence 2), (deletion of translation of pheromones and receptors, consequence 3)\n",
      "Progress: 27.20% completed\n",
      "Response for Row 545:\n",
      "Answer to Question 2: (expression of gene ATF, consequence β-carotene production), (expression of gene ATF, consequence ∼4.8-fold higher β-carotene productivity compared to S. cerevisiae), (expression of gene ATF, consequence ∼59% of β-carotene productivity from farnesyl diphosphate in S. cerevisiae), (expression of gene ATF, consequence wide spectrum of inducible transcriptional outputs), (expression of gene ATF, consequence small growth defects)\n",
      "Progress: 27.25% completed\n",
      "Response for Row 546:\n",
      "Answer to Question 2: (expression of gene PFUR17_02294, production of lecanoric acid)\n",
      "Progress: 27.30% completed\n",
      "Response for Row 547:\n",
      "Answer to Question 2: (overexpression of gene patK, consequence 1: increased biosynthesis of 6-methyl salicylic acid), (overexpression of gene patG, consequence 1: increased conversion of 6-methyl salicylic acid to m-cresol), (deletion of gene ST, consequence 1: enhanced metabolic flux towards m-cresol production), (deletion of gene EM, consequence 1: improved metabolic flux towards m-cresol production), (promoter engineering, consequence 1: increased expression levels of patK and patG), (gene multiplication, consequence 1: increased copy number of patK and patG leading to higher enzyme activity), (fed-batch fermentation, consequence 1: increased m-cresol production to 2.03 g/L)\n",
      "Progress: 27.35% completed\n",
      "Response for Row 548:\n",
      "Answer to Question 2: (deletion of ALT1, alanine accumulation), (deletion of ALT1, repression of GAD1), (deletion of ALT1, repression of UGA1), (deletion of ALT1, repression of UGA2), (deletion of ALT1, growth impairment), (expression of ALT1, induction by alanine), (expression of ALT1, repression of GABA shunt), (deletion of UGA1, heat sensitivity), (deletion of UGA1, reduced NADPH pools), (deletion of ALT1, null expression of COX2), (deletion of ALT1, null expression of COX3), (deletion of ALT1, null expression of ATP6), (deletion of ALT1, decreased mitochondrial/nuclear DNA ratio), (deletion of ALT1, petite phenotype)\n",
      "Progress: 27.40% completed\n",
      "Response for Row 549:\n",
      "Answer to Question 2: (deletion of gene TE repression mechanism, consequence 1: increased TE propagation), (overexpression of gene Ty1_cer, consequence 1: higher translation efficiency)\n",
      "Progress: 27.45% completed\n",
      "Response for Row 550:\n",
      "Answer to Question 2: (deletion of A at position -132 nt upstream of Spt15, increased isobutanol tolerance), (insertion of G at position -65 nt upstream of Spt15, increased isobutanol tolerance), (overexpression of ILV3 gene, increased isobutanol production), (overexpression of ILV2 gene, increased isobutanol production), (overexpression of ARO10 gene, increased isobutanol production), (mutations of Spt15, increased expression of SPT15), (mutations of Spt15, downregulated expression of genes involved in TCA cycle), (mutations of Spt15, downregulated expression of genes involved in glyoxylic acid cycle), (mutations of Spt15, increased expression of genes related to cell stability)\n",
      "Progress: 27.50% completed\n",
      "Response for Row 551:\n",
      "Answer to Question 2: (expression of gene Bs3, consequence hypersensitive response), (expression of gene Bs3, consequence inhibition of yeast proliferation), (expression of gene Bs3, consequence increase of intracellular oxidation state), (mutation S211A in gene Bs3, consequence twofold increase in NADPH oxidase activity), (mutation S211A in gene Bs3, consequence increase of intracellular oxidation state), (expression of gene Bs3, consequence production of H2O2)\n",
      "Progress: 27.55% completed\n",
      "Response for Row 552:\n",
      "Answer to Question 2: (heterologous expression of orotidine 5'-monophosphate decarboxylases, weak cross-species expression), (pyrG promoters drive URA3 expression, efficient expression in yeast), (inefficient cross-species splicing of Penicillium introns, weak cross-species expression)\n",
      "Progress: 27.60% completed\n",
      "Response for Row 553:\n",
      "Answer to Question 2: (deletion of stabilizing H-bonds, partially functional variants), (elimination of conserved H-bond at N-latch position, non-functional Cin8 variant), (additional H-bond stabilization of N-latch position, nonfunctional variant), (replacement of N-latch asparagine with glycine, rescued functional defects)\n",
      "Progress: 27.65% completed\n",
      "Response for Row 554:\n",
      "Answer to Question 2:  (overexpression of Gas1, inhibition of glucan-elongating activity), (overexpression of Gas2, inhibition of glucan-elongating activity), (overexpression of Crh1, inhibition of chitin-glucan transglycosylase activity), (activation of cell-wall integrity pathway, transcriptional co-regulation), (activation of high-osmolarity glycerol signaling pathway, transcriptional co-regulation)\n",
      "Progress: 27.70% completed\n",
      "Response for Row 555:\n",
      "Answer to Question 2:  (up-regulation of genes involved in sugar fermentation, increased ethanol production), (up-regulation of genes involved in ethanol metabolization, enhanced growth in adverse environments), (up-regulation of genes involved in protective responses against abiotic stresses, production of high abundant proteins), (expression of AHP1, production of peptides with health-beneficial activities), (expression of TSA1, production of peptides with health-beneficial activities), (expression of HSP26, production of peptides with health-beneficial activities), (expression of SOD1, production of peptides with health-beneficial activities), (expression of HSP10, production of peptides with health-beneficial activities), (expression of UTR2, production of peptides with health-beneficial activities), (expression of ENO1/2, production of peptides with health-beneficial activities), (expression of TDH1/2/3, production of peptides with health-beneficial activities), (expression of ADH1/2, production of peptides with health-beneficial activities), (expression of FBA1, production of peptides with health-beneficial activities), (expression of PDC1, production of peptides with health-beneficial activities)\n",
      "Progress: 27.75% completed\n",
      "Response for Row 556:\n",
      "Answer to Question 2: (no specific gene expression/deletion mentioned, development of yETFL model for Saccharomyces cerevisiae), (no specific gene expression/deletion mentioned, prediction of maximum growth rate by yETFL), (no specific gene expression/deletion mentioned, prediction of essential genes by yETFL), (no specific gene expression/deletion mentioned, prediction of overflow metabolism phenotype by yETFL)\n",
      "Progress: 27.80% completed\n",
      "Response for Row 557:\n",
      "Answer to Question 2: (no gene expression/deletion mentioned, consequence 1: rapidly folding domains allow for efficient transmembrane integration with low-hydrophobicity thresholds), (no gene expression/deletion mentioned, consequence 2: sequences that are intrinsically disordered allow for efficient transmembrane integration with low-hydrophobicity thresholds), (no gene expression/deletion mentioned, consequence 3: very short sequences allow for efficient transmembrane integration with low-hydrophobicity thresholds), (no gene expression/deletion mentioned, consequence 4: sequences capable of binding chaperones with high affinity allow for efficient transmembrane integration with low-hydrophobicity thresholds), (no gene expression/deletion mentioned, consequence 5: long protein fragments interfere with membrane integration, requiring higher hydrophobicity), (no gene expression/deletion mentioned, consequence 6: folding-deficient mutant domains interfere with membrane integration, requiring higher hydrophobicity), (no gene expression/deletion mentioned, consequence 7: artificial sequences not binding chaperones interfere with membrane integration, requiring higher hydrophobicity)\n",
      "Progress: 27.85% completed\n",
      "Response for Row 558:\n",
      "Answer to Question 2: (deletion of gene CWI pathway, consequence elevated ROS production), (deletion of gene CWI pathway, consequence cell death), (deletion of gene VPS, consequence elevated ROS production), (deletion of gene VPS, consequence cell death), (deletion of gene calcium/calcineurin pathway, consequence elevated ROS production), (deletion of gene calcium/calcineurin pathway, consequence cell death), (deletion of gene PHO pathways, consequence elevated ROS production), (deletion of gene PHO pathways, consequence cell death)\n",
      "Progress: 27.90% completed\n",
      "Response for Row 559:\n",
      "Answer to Question 2:  (deletion of gene hsm3, increase in frequency of mutations induced by different mutagens), (deletion of gene hsm3, suppression of UV-induced mutagenesis by inactivation of Polη polymerase), (deletion of gene hsm3, significant role of Polη for mutagenesis at NBP sites), (deletion of gene hsm3, lower efficiency of RNR gene expression after UV irradiation compared to wild-type cells), (deletion of gene hif1, suppression of UV-induced mutagenesis by inactivation of Polη polymerase), (deletion of gene hif1, significant role of Polη for mutagenesis at NBP sites), (deletion of gene hif1, lower efficiency of RNR gene expression after UV irradiation compared to wild-type cells), (deletion of gene hat1, not specified in abstract), (increase in dNTP levels, suppression of hsm3-dependent mutagenesis), (increase in dNTP levels, suppression of hif1-dependent mutagenesis), (Polη, responsible for hsm3-dependent mutagenesis), (Polη, responsible for hif1-dependent mutagenesis)\n",
      "Progress: 27.95% completed\n",
      "Response for Row 560:\n",
      "Answer to Question 2: (deletion of gene eIF4G, increased retention of eIF4A), (deletion of gene eIF4G, increased efficiency of reinitiation on downstream initiation codons), (mutation affecting eIF4A-binding site, elimination of specificity with mutation in cis-acting feature), (mutation affecting integrity of eIF4G1-RNA2-binding domain, elimination of specificity with mutation in cis-acting feature)\n",
      "Progress: 28.00% completed\n",
      "Response for Row 561:\n",
      "Answer to Question 2: (deletion of gene Gal4p, consequence 1), (deletion of gene Gal80p, consequence 1), (expression of gene Q System, consequence 1), (expression of gene OptoQ-INVRT, consequence 1), (expression of gene OptoQ-AMP, consequence 1), (expression of gene OptoQ-AMP, consequence 2), (expression of gene OptoQ-INVRT, consequence 2), (expression of gene OptoQ-AMP, consequence 3), (expression of gene OptoQ-INVRT, consequence 3)\n",
      "Progress: 28.05% completed\n",
      "Response for Row 562:\n",
      "Answer to Question 2: (expression of gene tHMG1, consequence squalene production increase), (expression of gene GFP, consequence fluorescence intensity increase)\n",
      "Progress: 28.10% completed\n",
      "Response for Row 563:\n",
      "Answer to Question 2: (deletion of gene rpc128-1007, upregulation of mRNA expression), (deletion of gene rpc128-1007, derepression of general transcription factor Gcn4), (deletion of gene rpc128-1007, increase in expression of ribosome biogenesis genes), (deletion of gene rpc128-1007, downregulation of proteasome complex genes)\n",
      "Progress: 28.15% completed\n",
      "Response for Row 564:\n",
      "Answer to Question 2: (deletion of gene Gas1, increased lysozyme activity), (deletion of gene Gas1, increased Ig proportion), (deletion of gene Gas1, increased mcsfra expression), (deletion of gene Gas1, increased hepcidin expression), (overexpression of gene Gas1, enhanced disease resistance)\n",
      "Progress: 28.20% completed\n",
      "Response for Row 565:\n",
      "Answer to Question 2: (overexpression of gene ARE2, increased ergosterol content to 10 mg/g DCW), (overexpression of gene UPC2-1, increased ergosterol content to 16.7 mg/g DCW), (overexpression of gene ACC1, increased ergosterol content from 16.7 to 20.7 mg/g DCW)\n",
      "Progress: 28.25% completed\n",
      "Response for Row 566:\n",
      "Answer to Question 2: (deletion of gene related to glycosylation, consequence: improved functional expression of antibody fragments in S. cerevisiae), (expression of gene related to α-amylase secretion, consequence: enhanced secretion capacity in HA strain), (expression of gene related to amino acid metabolism, consequence: increased secretion of antibody fragments), (expression of gene related to protein synthesis, consequence: increased secretion of antibody fragments), (expression of gene related to cell cycle regulation, consequence: altered secretion efficiency of antibody fragments)\n",
      "Progress: 28.30% completed\n",
      "Response for Row 567:\n",
      "Answer to Question 2: (deletion of gene Ist2, consequence 1), (deletion of gene Ist2, consequence 2), (overexpression of gene Osh6, consequence 3), (overexpression of gene Osh7, consequence 4)\n",
      "Progress: 28.35% completed\n",
      "Response for Row 568:\n",
      "Answer to Question 2: (expression of gene Rap1, consequence RPGs expression depends upon it), (expression of gene Fhl1, consequence binds to RPG promoters), (expression of gene Ifh1, consequence binds to RPG promoters and its binding correlates with RPG expression), (expression of gene Sfp1, consequence activates RiBi and other growth-related genes), (expression of gene Hmo1, consequence stabilizes or re-configures Fhl1-Ifh1 binding at RPG promoters), (expression of gene Fpr1, consequence affects Fhl1-Ifh1 binding at RPG promoters), (down regulation of gene Ifh1, consequence RPG expression is reduced under stress conditions), (activation of RASTR, consequence controls RPG transcription through reversible condensation of Ifh1)\n",
      "Progress: 28.40% completed\n",
      "Response for Row 569:\n",
      "Answer to Question 2: (deletion of gene IMS, decrease in isomultiflorenol production), (deletion of gene CBS, decrease in cucurbitadienol production), (overexpression of gene IMS, increase in isomultiflorenol production), (overexpression of gene CBS, increase in cucurbitadienol production)\n",
      "Progress: 28.45% completed\n",
      "Response for Row 570:\n",
      "Answer to Question 2: (deletion of gene alsgt-2, decrease in total SG), (deletion of gene alsgt-2, almost complete loss of GL-B)\n",
      "Progress: 28.50% completed\n",
      "Response for Row 571:\n",
      "Answer to Question 2: (expression of gene ScARO10, increase in isoamyl alcohol titer), (expression of gene ScARO10 with weak promoter P_UAS1B4-EXPm, highest titer increase)\n",
      "Progress: 28.55% completed\n",
      "Response for Row 572:\n",
      "Answer to Question 2: (deletion of gene Gpa2, consequence regulating carbohydrate metabolism), (deletion of gene Asc1, consequence regulating amino acid metabolism), (deletion of gene Gpa2, consequence regulating purine metabolism), (deletion of gene Asc1, consequence regulating purine metabolism), (deletion of gene Gpa2, consequence determining amplitude of response to glucose addition), (deletion of gene Gpa2, consequence affecting cell growth), (deletion of gene Asc1, consequence affecting cell growth)\n",
      "Progress: 28.60% completed\n",
      "Response for Row 573:\n",
      "Answer to Question 2: (deletion of cardiolipin synthesis, precluded stimulation of protein import), (deletion of cardiolipin synthesis, compromised cellular fitness)\n",
      "Progress: 28.65% completed\n",
      "Response for Row 574:\n",
      "Answer to Question 2: (deletion of HAC1, promote nuclear accumulation of Hog1), (deletion of IRE1, increase cytosolic and bud neck localization of Slt2), (activation of UPR pathway, required for cadmium-induced phosphorylation of Hog1), (Slt2, required for activation of UPR pathway through Swi6 and Rlm1), (Hog1, regulate cellular process in absence of UPR pathway), (Slt2, regulate cellular process in absence of UPR pathway)\n",
      "Progress: 28.70% completed\n",
      "Response for Row 575:\n",
      "Answer to Question 2: (deletion of gene RAD52, enhanced homologous recombination efficiency), (deletion of gene RAD52, increased simultaneous assembly of multiple fragments), (disruption of multiple genes, efficient genome integration), (assembly of multiple genes, regulation of gene expression), (regulation of metabolic pathways, increased fatty alcohol production)\n",
      "Progress: 28.75% completed\n",
      "Response for Row 576:\n",
      "Answer to Question 2: (deletion of SNORA48, change in translational efficiency), (deletion of SNORA48, alteration in stoichiometric translation rates of sarcomere proteins), (mutation in translation machinery, protein length-dependent shift in translational efficiency)\n",
      "Progress: 28.80% completed\n",
      "Response for Row 577:\n",
      "Answer to Question 2: (deletion of N-linked glycosylation, consequence 1: improper protein folding), (deletion of N-linked glycosylation, consequence 2: premature degradation), (deletion of N-linked glycosylation, consequence 3: increased high-level expression yields), (deletion of N-linked glycosylation, consequence 4: reduced immunological consequences)\n",
      "Progress: 28.85% completed\n",
      "Response for Row 578:\n",
      "Answer to Question 2: (overexpression of OsFEX, enhanced fluoride tolerance in yeast), (overexpression of OsFEX, enhanced fluoride tolerance in Nicotiana benthamiana), (suppression of OsFEX, fluoride-hyperaccumulative nature of rice cultivar), (upregulation of OsFEX by gibberellic acid, increased OsFEX expression), (upregulation of OsFEX by melatonin, increased OsFEX expression), (suppression of OsFEX by abscisic acid, decreased OsFEX expression)\n",
      "Progress: 28.90% completed\n",
      "Response for Row 579:\n",
      "Answer to Question 2: (deletion of β-isopropylmalate dehydrogenase gene, affects leucine biosynthesis), (deletion of BCAA aminotransferase gene, affects BCAA biosynthesis), (overexpression of BCAA aminotransferase gene, increases BCAA biosynthesis), (regulation of leucine biosynthesis genes by LeuB, modulates response to leucine availability)\n",
      "Progress: 28.95% completed\n",
      "Response for Row 580:\n",
      "Answer to Question 2: (deletion of gene PLN1, consequence 1: increased efficiency of converting DD to PPD by 394%), (deletion of gene PLN1, consequence 2: increased conversion rate of DD from 17.4% to 86.0%), (overexpression of gene Pn3-29, consequence 1: reduced accumulation of PPD), (overexpression of gene Pn3-29, consequence 2: increased production of CK to 41.3 mg/L/OD)\n",
      "Progress: 29.00% completed\n",
      "Response for Row 581:\n",
      "Answer to Question 2: (depletion of gene Bdf1, consequence hypersensitivity of genes to depletion), (depletion of gene Bdf2, consequence hypersensitivity of genes to depletion), (depletion of gene Bdf1/2, consequence defects in transcription initiation), (depletion of gene Bdf1/2, consequence defects in early elongation), (depletion of gene Bdf1/2, consequence affect preinitiation complex formation), (depletion of gene Bdf1/2, consequence reduced recruitment of TFIID), (depletion of gene Bdf1/2, consequence reduced recruitment of Mediator), (depletion of gene Bdf1/2, consequence reduced recruitment of basal transcription factors)\n",
      "Progress: 29.05% completed\n",
      "Response for Row 582:\n",
      "Answer to Question 2: (deletion of gene DER1, consequence 1), (deletion of gene ERO1, consequence 1), (overexpression of heterologous proteins, consequence increased ER stress), (overexpression of heterologous proteins, consequence reduced protein secretion)\n",
      "Progress: 29.10% completed\n",
      "Response for Row 583:\n",
      "Answer to Question 2: (deletion of gene CLEC7A, decrease in IL-8 production), (deletion of gene CLEC7A, decrease in TNF expression), (overexpression of gene CLEC7A, increase in CD80 surface expression), (overexpression of gene CLEC7A, increase in CD86 surface expression), (overexpression of gene CLEC7A, increase in IL-8 production), (overexpression of gene CLEC7A, increase in TNF mRNA expression), (overexpression of gene CLEC7A, increase in IL1B mRNA expression), (overexpression of gene CLEC7A, increase in IL6 mRNA expression)\n",
      "Progress: 29.15% completed\n",
      "Response for Row 584:\n",
      "Answer to Question 2: (codon choice of genes, regulation of translation process timing), (codon choice of genes, impact on protein folding classes), (codon choice of genes, impact on functional classes involved in transcription process), (codon choice of genes, low impact on proteins associated with energy production and metabolism), (low average score k-mers, occurrence in coil or close to coil regions), (high average score k-mers, occurrence in alpha helices and close to coil regions), (significant codon patterns, enrichment at the 5' end of protein-coding sequences)\n",
      "Progress: 29.20% completed\n",
      "Response for Row 585:\n",
      "Answer to Question 2: (deletion of leptin, higher in control group), (deletion of growth hormone, higher in control group), (deletion of MMP3 gene expression, lower in treatment group), (overexpression of hormone receptors, higher in brain), (overexpression of plasma cholesterol, higher in control group), (overexpression of MDA, higher in control group), (overexpression of AMP kinase, higher in control group)\n",
      "Progress: 29.25% completed\n",
      "Response for Row 586:\n",
      "Answer to Question 2: (expression of gene SSU1, consequence 1), (haploinsufficiency of gene SSU1, consequence 2)\n",
      "Progress: 29.30% completed\n",
      "Response for Row 587:\n",
      "Answer to Question 2: (deletion of gene znf1, increased sensitivity to ethanol), (deletion of gene znf1, increased sensitivity to β-mercaptoethanol), (deletion of gene znf1, increased sensitivity to calcofluor white), (deletion of gene znf1, increased glycerol levels), (deletion of gene smi1, increased glycerol levels), (overexpression of gene znf1, increased ethanol tolerance), (overexpression of gene znf1, increased ethanol production), (activation of gene gup1 by znf1, increased glycerol production), (activation of gene gpp1 by znf1, increased glycerol production), (activation of gene gpp2 by znf1, increased glycerol production), (activation of gene gpd1 by znf1, increased glycerol production), (activation of gene gat1 by znf1, increased glycerol production), (activation of gene ole1 by znf1, increased fatty acid production), (activation of gene fks1 by znf1, increased cell wall biosynthesis), (activation of gene sed1 by znf1, increased cell wall biosynthesis), (activation of gene smi1 by znf1, increased cell wall biosynthesis), (activation of gene hsp30 by znf1, increased unfolded protein response), (activation of gene hsp104 by znf1, increased unfolded protein response), (activation of gene kar1 by znf1, increased unfolded protein response), (activation of gene lhs1 by znf1, increased unfolded protein response)\n",
      "Progress: 29.35% completed\n",
      "Response for Row 588:\n",
      "Answer to Question 2: (deletion of PDC genes, elimination of ethanol production), (overexpression of PCAD, minimized PCA production), (insertion of DHS dehydratase from P. anserina, biosynthesis of muconic acid), (insertion of PCA decarboxylase from K. pneumoniae, biosynthesis of muconic acid), (insertion of catechol 1,2-dioxygenase from C. albicans, biosynthesis of muconic acid), (increased yeast pitching rate, lowered biomass formation), (use of PPG, in situ product removal)\n",
      "Progress: 29.40% completed\n",
      "Response for Row 589:\n",
      "Answer to Question 2: (expression of recombinant proteins, increased production speed), (expression of recombinant proteins, ease of handling), (expression of insulin analogues using PURE IVTT system, correct insulin expression and folding), (expression of insulin analogues using PURE IVTT system, similar insulin receptor affinity to yeast-produced insulin)\n",
      "Progress: 29.45% completed\n",
      "Response for Row 590:\n",
      "Answer to Question 2:  (deletion of gene AoTgla, increase of total lipid contents), (deletion of gene AoTglb, increase of total lipid contents), (deletion of gene AoTgla, increase in TAG fraction), (deletion of gene AoTglb, increase in TAG fraction), (deletion of gene AoTgla, prolonged lipid-accumulating stage), (deletion of gene AoTglb, prolonged lipid-accumulating stage), (deletion of gene AoTgla, reduction in specific rate of lipid turnover), (deletion of gene AoTglb, reduction in specific rate of lipid turnover), (deletion of gene AoTgla, achievement of maximal lipid levels), (deletion of gene AoTglb, achievement of maximal lipid levels), (deletion of gene AoTgla, achievement of maximal TAG levels), (deletion of gene AoTglb, achievement of maximal TAG levels)\n",
      "Progress: 29.50% completed\n",
      "Response for Row 591:\n",
      "Answer to Question 2: (deletion of RTG pathway genes, activation of amino acid biosynthesis in U cells), (deletion of RTG pathway genes, repression of mitochondrial ribosome components expression in U cells), (deletion of RTG pathway genes, reduced mitochondrial translation in U cells)\n",
      "Progress: 29.55% completed\n",
      "Response for Row 592:\n",
      "Answer to Question 2: (deletion of gene laccase, consequence reduced lignin-degrading ability), (expression of gene laccase from Agrocybe pediades, consequence increased ability to oxidize aromatic compounds), (expression of mutated laccase variant, consequence improved activity at neutral pH), (expression of mutated laccase variant, consequence enhanced tolerance to inhibitors), (expression of mutated laccase variant, consequence ability to oxidize high-redox potential mediators and recalcitrant organic dyes), (expression of laccase with N-glycosylation sites, consequence increased secretion of enzyme), (expression of laccase with N-glycosylation sites, consequence altered catalytic activity)\n",
      "Progress: 29.60% completed\n",
      "Response for Row 593:\n",
      "Answer to Question 2: (deletion of functional mitochondria, cell stress adaptation), (expression of RTG-target genes, changes in nuclear gene expression), (expression of CIT2 gene, increased Cit2p-GFP protein product), (phosphorylation of Rtg1/3p, controls RTG-target gene transcription)\n",
      "Progress: 29.65% completed\n",
      "Response for Row 594:\n",
      "Answer to Question 2: (no gene overexpression/deletion mentioned, consequence: translation efficiency varies across different organisms due to differences in sequence features), (no gene overexpression/deletion mentioned, consequence: codon adaptation index shows high correlation in all analyzed organisms)\n",
      "Progress: 29.70% completed\n",
      "Response for Row 595:\n",
      "Answer to Question 2: (deletion of gene Gcn2, improved α-amylase production), (deletion of gene Gcn2, reduced intracellular H₂O₂ levels), (deletion of gene Gcn2, reduced Hac1 splicing ratio), (deletion of gene Gcn2, increased expression of antioxidants), (deletion of gene Gcn2, increased expression of PDI1)\n",
      "Progress: 29.75% completed\n",
      "Response for Row 596:\n",
      "Answer to Question 2: (deletion of gene function, tolerated in one genetic background), (deletion of gene function, lethal in another genetic background), (gain-of-function of modifier gene, suppression of fitness defects)\n",
      "Progress: 29.80% completed\n",
      "Response for Row 597:\n",
      "Answer to Question 2:  (deletion of gene MoWHI2, multiple appressoria formed by one conidium), (deletion of gene MoWHI2, pathogenicity significantly reduced), (deletion of gene Mopsr1, multiple appressoria formed by one conidium), (deletion of gene Mopsr1, pathogenicity significantly reduced), (deletion of gene MoWHI2, cAMP levels increasing), (deletion of gene Mopsr1, cAMP levels increasing), (deletion of gene MoWHI2, inappropriate activation of MoTor signalling pathway), (deletion of gene Mopsr1, inappropriate activation of MoTor signalling pathway), (increased expression of MoTOR, activation of MoTor signalling pathway), (increased expression of MoRS3, activation of MoTor signalling pathway)\n",
      "Progress: 29.85% completed\n",
      "Response for Row 598:\n",
      "Answer to Question 2: (deletion of polyasparagine domain in Azf1p, genes upregulated related to carbon metabolism), (deletion of polyasparagine domain in Azf1p, genes upregulated related to storage)\n",
      "Progress: 29.90% completed\n",
      "Response for Row 599:\n",
      "Answer to Question 2: (translation of uORFs, inhibition of main coding sequence translation), (translation of uORFs, premature dissociation of ribosomal units), (translation of uORFs, ribosome stalling), (translational arrest of primary coding sequences, during stress), (increased relative abundance of uORF-encoded peptides, during stress), (decreased translation of uORFs, facilitating CDS translation during stress), (up-regulation of translation, in a subset of genes during stress), (uORFs, act as translational switches in genes encoding proteins needed rapidly upon stress)\n",
      "Progress: 29.95% completed\n",
      "Response for Row 600:\n",
      "Answer to Question 2: (deletion of gene los1, increased extracellular level of anti-CD22 scFv), (deletion of gene los1, altered susceptibility to protein synthesis inhibitors), (deletion of gene los1, increased total proteins secreted)\n",
      "Progress: 30.00% completed\n",
      "Response for Row 601:\n",
      "Answer to Question 2: (deletion of 5caC, consequence increased Pol II translocation), (overexpression of 5caC, consequence Pol II pausing), (interaction of 5caC with fork loop 3, consequence compromised NTP binding), (interaction of 5caC with fork loop 2, consequence Pol II pausing), (presence of 5caC, consequence suppressed bridge helix kinking)\n",
      "Progress: 30.05% completed\n",
      "Response for Row 602:\n",
      "Answer to Question 2: (deletion of gene Pbp1, consequence slower growth on medium containing glycerol), (deletion of gene Pbp1, consequence slower growth on medium containing lactate), (deletion of gene Pbp1, consequence decreased expression of gene PCK1), (deletion of gene Pbp1, consequence decreased expression of gene FBP1), (deletion of gene Pbp1, consequence decreased expression of gene COX10), (deletion of gene Pbp1, consequence decreased expression of gene COX11), (deletion of gene Dcp1, consequence recovery of COX10 expression in pbp1Δ mutant), (deletion of gene Xrn1, consequence recovery of COX11 expression in pbp1Δ mutant)\n",
      "Progress: 30.10% completed\n",
      "Response for Row 603:\n",
      "Answer to Question 2: (expression of UPO genes, production of four active UPOs), (expression of UPO genes, one-step purification), (expression of UPO genes, application in enantioselective conversion), (expression of UPO genes, yield of 24 mg/L secreted UPO enzyme), (expression of UPO genes, preparative scale conversion of a phenethylamine derivative reaching 98.6 % ee)\n",
      "Progress: 30.15% completed\n",
      "Response for Row 604:\n",
      "Answer to Question 2: (overexpression of gene KCS1, increased octanoic acid titers), (overexpression of gene FSH2, increased octanoic acid titers)\n",
      "Progress: 30.20% completed\n",
      "Response for Row 605:\n",
      "Answer to Question 2: (no gene overexpression/deletion mentioned, consequence: impact on yeast life span), (no gene overexpression/deletion mentioned, consequence: involvement of TOR and cAMP/PKA pathways in response to stress factors and nutrients), (no gene overexpression/deletion mentioned, consequence: reductive metabolism of living cells clears undesirable carbonyl compounds), (no gene overexpression/deletion mentioned, consequence: improvement of product's shelf life through bottle conditioning)\n",
      "Progress: 30.25% completed\n",
      "Response for Row 606:\n",
      "Answer to Question 2: (deletion of gene YGP1, consequence 20-fold increase of fluorescence upon addition of 30 mM acetic acid), (overexpression of reporter gene, consequence significant drop of the fluorescence), (application of yeast pheromone-based signal amplification system, consequence compensation for drop of fluorescence)\n",
      "Progress: 30.30% completed\n",
      "Response for Row 607:\n",
      "Answer to Question 2: (deletion of intron in HAC1 mRNA, translational derepression of HAC1 mRNA), (translational derepression of HAC1 mRNA, production of Hac1 protein), (production of Hac1 protein, activation of expression of enzymes and chaperones), (activation of expression of enzymes and chaperones, increase in protein-folding capacity of the cell), (regulation by Vps34, regulation of HAC1 mRNA translation), (regulation by TOR, regulation of HAC1 mRNA translation), (regulation by GCN2, regulation of HAC1 mRNA translation)\n",
      "Progress: 30.35% completed\n",
      "Response for Row 608:\n",
      "Answer to Question 2: (deletion of gene HXK1, consequence 1), (deletion of gene HXK1, consequence 2), (deletion of gene HXT1, consequence 1), (deletion of gene HXT1, consequence 2), (deletion of gene HXT2, consequence 1), (deletion of gene HXT2, consequence 2), (deletion of gene HXT4, consequence 1), (deletion of gene HXT4, consequence 2), (deletion of gene PFK1, consequence 1), (deletion of gene PFK1, consequence 2), (overexpression of stress response genes, consequence 1), (overexpression of stress response genes, consequence 2)\n",
      "Progress: 30.40% completed\n",
      "Response for Row 609:\n",
      "Answer to Question 2: (deletion of gene associated with mitochondria, mitochondrial damage), (deletion of gene associated with mitochondria, ROS accumulation), (deletion of gene associated with mitochondria, ATP content changes), (deletion of oxidative phosphorylation-related gene, higher ROS production), (deletion of oxidative phosphorylation-related gene, reduced ATP levels)\n",
      "Progress: 30.45% completed\n",
      "Response for Row 610:\n",
      "Answer to Question 2: (expression of FRE5, up-regulated during iron limitation), (expression of FTR1, up-regulated during iron limitation), (expression of FET3, up-regulated during iron limitation), (overexpression of LcFTR1 I, high expression of intracellular transport of Fe3+), (overexpression of LcFTR1 II, moderate expression of intracellular transport of Fe3+), (overexpression of LcFTR1 IV, moderate expression of intracellular transport of Fe3+)\n",
      "Progress: 30.50% completed\n",
      "Response for Row 611:\n",
      "Answer to Question 2: (no specific gene overexpression/deletion mentioned, no direct consequences related to gene expression/deletion)\n",
      "Progress: 30.55% completed\n",
      "Response for Row 612:\n",
      "Answer to Question 2:  (expression of gene Talaromyces minioluteus dextranase, consequence increased extracellular yield of r-TmDEX), (expression of gene Talaromyces minioluteus dextranase, consequence no inhibition of cell growth), (expression of gene Talaromyces minioluteus dextranase, consequence volumetric productivity of 1706 U/L/h), (expression of gene Talaromyces minioluteus dextranase, consequence optimal activity at pH 4.5-5.5), (expression of gene Talaromyces minioluteus dextranase, consequence optimal activity at temperature 50-60 °C), (addition of sucrose, consequence retention of r-TmDEX activity after 1-h incubation at 50-60 °C and pH 5.5), (application of crude preparation of secreted r-TmDEX, consequence complete removal of bacterial dextran in deteriorated sugarcane juice)\n",
      "Progress: 30.60% completed\n",
      "Response for Row 613:\n",
      "Answer to Question 2:  (overexpression of gene not specified, increased protein synthesis in mitochondria), (deletion of gene not specified, decreased protein synthesis in mitochondria)\n",
      "Progress: 30.65% completed\n",
      "Response for Row 614:\n",
      "Answer to Question 2: (deletion of AHLs, consequence 1: major QS disruption), (deletion of AHLs, consequence 2: significant molecular changes), (deletion of AHLs, consequence 3: significant phenotypic changes), (deletion of AHLs, consequence 4: drastic changes in social interactions between C. violaceum and B. cereus), (deletion of AHLs, consequence 5: drastic changes in social interactions between C. violaceum and S. cerevisiae), (deletion of AHLs, consequence 6: drastic changes in social interactions between C. violaceum and murine macrophages), (deletion of AHLs, consequence 7: drastic changes in social interactions between C. violaceum and planarian Schmidtea mediterranea)\n",
      "Progress: 30.70% completed\n",
      "Response for Row 615:\n",
      "Answer to Question 2: (deletion of gene pdc1, ethanol conversion decreased), (deletion of gene pdc1, glycerol conversion increased), (deletion of gene pdc1, acetic acid concentration decreased), (deletion of gene pdc1, acetoin concentration increased), (deletion of gene pdc1, pyruvate decarboxylase activity decreased), (deletion of gene pdc5, ethanol conversion decreased), (deletion of gene pdc5, glycerol conversion increased), (deletion of gene pdc5, acetic acid concentration decreased), (deletion of gene pdc5, acetoin concentration increased), (deletion of gene pdc5, pyruvate decarboxylase activity decreased)\n",
      "Progress: 30.75% completed\n",
      "Response for Row 616:\n",
      "Answer to Question 2: (deletion of gene mbkZ, reduced production of red pigments metacycloprodigiosin and undecylprodigiosin), (expression of gene mbkZ, increased production of red pigments metacycloprodigiosin and undecylprodigiosin)\n",
      "Progress: 30.80% completed\n",
      "Response for Row 617:\n",
      "Answer to Question 2: (deletion of gene Gal80p, consequence Gal4p expression without interruption)\n",
      "Progress: 30.85% completed\n",
      "Response for Row 618:\n",
      "Answer to Question 2: (stop codon recognition by eRF1, termination of translation), (near-cognate tRNA outcompetes eRF1, stop codon readthrough), (readthrough, translation into mRNA 3'-UTR), (cis-acting elements modulation, termination efficiency), (cis-acting elements modulation, readthrough efficiency), (wild-type or temperature-sensitive eRF1 expression, ribosome profiling analysis), (stop codon identity, influence on readthrough efficiency), (nucleotide after stop codon, influence on readthrough efficiency), (codon in the P site, influence on readthrough efficiency), (3'-UTR length, influence on readthrough efficiency), (nts +5 to +9, mild effect on readthrough efficiency), (low readthrough genes, shorter 3'-UTRs in thermally inactivated eRF1 cells), (high readthrough genes, longer 3'-UTRs in thermally inactivated eRF1 cells), (new mRNA or peptide features identification, affecting translation termination and readthrough efficiency)\n",
      "Progress: 30.90% completed\n",
      "Response for Row 619:\n",
      "Answer to Question 2: (deletion of gene ARO8, reduced 2-PE content), (deletion of gene ARO8, reduced ethanol yield), (deletion of gene ARO8, decreased biofilm formation), (deletion of gene ARO8, reduced extracellular polysaccharides), (deletion of gene ARO8, reduced FLOp expression), (deletion of gene ARO9, reduced 2-PE content), (deletion of gene ARO9, reduced ethanol yield), (deletion of gene ARO9, decreased biofilm formation), (deletion of gene ARO9, reduced extracellular polysaccharides), (deletion of gene ARO9, reduced FLOp expression)\n",
      "Progress: 30.95% completed\n",
      "Response for Row 620:\n",
      "Answer to Question 2:  (expression of S. cerevisiae gene, linked to nitrogen metabolism), (expression of S. cerevisiae gene, linked to sulfur metabolism), (expression of S. cerevisiae gene, linked to thiamine metabolism)\n",
      "Progress: 31.00% completed\n",
      "Response for Row 621:\n",
      "Answer to Question 2: (deletion of gene, efficient CRISPR-mediated genome editing), (expression of gene pathway, β-carotene synthesis), (expression of gene pathway, violacein production)\n",
      "Progress: 31.05% completed\n",
      "Response for Row 622:\n",
      "Answer to Question 2: (R-loop accumulation, DNA double-strand breaks), (R-loop presence, genome instability)\n",
      "Progress: 31.10% completed\n",
      "Response for Row 623:\n",
      "Answer to Question 2: (deletion of gene involved in l-proline metabolism, consequence 1), (deletion of gene involved in l-arginine metabolism, consequence 2), (overexpression of gene in ubiquitin system, consequence 3), (overexpression of transcription factors, consequence 4)\n",
      "Progress: 31.15% completed\n",
      "Response for Row 624:\n",
      "Answer to Question 2: (optogenetic activation of gene expression, increased transcriptional response to blue light), (optogenetic activation of gene expression, 41-fold induction between dark and light conditions), (optogenetic activation of gene expression, efficient activation at low light duty cycles), (optogenetic activation of gene expression, strong homogeneous light-induction in bioreactors), (enzyme expression controlled by optogenetic circuits, improved production of lactic acid), (enzyme expression controlled by optogenetic circuits, improved production of isobutanol), (enzyme expression controlled by optogenetic circuits, improved production of naringenin)\n",
      "Progress: 31.20% completed\n",
      "Response for Row 625:\n",
      "Answer to Question 2: (deletion of asparaginase genes, consequence 1), (deletion of maltose utilization loci, consequence 2), (deletion of several genes from the tandem array of the DUP240 family, consequence 3), (loss of a stop codon in the FLO8 gene, consequence 4)\n",
      "Progress: 31.25% completed\n",
      "Response for Row 626:\n",
      "Answer to Question 2:  (overexpression of gene CvHSF30-2, consequence 1: confers tolerance to heat stress in Escherichia coli), (overexpression of gene CvHSF30-2, consequence 2: confers tolerance to heat stress in Saccharomyces cerevisiae), (deletion of gene HSF30-2, consequence 1: leads to heat sensitivity in tobacco), (deletion of gene HSF30-2, consequence 2: leads to heat sensitivity in Clematis), (overexpression of gene CvHSF30-2, consequence 3: elevates heat shock protein (HSP) expression), (negative regulation by gene CvHSFB2a, consequence 1: reduces heat shock protein (HSP) expression)\n",
      "Progress: 31.30% completed\n",
      "Response for Row 627:\n",
      "Answer to Question 2: (overexpression of gene RtFAD2, consequence increased LA contents), (overexpression of gene RtFAD2, consequence increased LA titers), (overexpression of gene RtFAD2, consequence increased lipid titer), (overexpression of gene RtFAD2, consequence increased lipid content), (overexpression of gene RtFAD1, consequence down-regulated in co-overexpression with RtFAD2)\n",
      "Progress: 31.35% completed\n",
      "Response for Row 628:\n",
      "Answer to Question 2:  (expression of gene Lbo_PPTQ, production of (Z)-11-tetradecenoic acid), (expression of gene Lbo_31670, chain shortening of (Z)-11-tetradecenoic acid to (Z)-9-dodecenoic acid), (expression of gene Lbo_49602, chain shortening of (Z)-11-tetradecenoic acid to (Z)-9-dodecenoic acid)\n",
      "Progress: 31.40% completed\n",
      "Response for Row 629:\n",
      "Answer to Question 2: (deletion of gene GSP2, increase in RNA content), (deletion of gene CTT1, increase in RNA content), (overexpression of gene HXT1, increase in RNA content)\n",
      "Progress: 31.45% completed\n",
      "Response for Row 630:\n",
      "Answer to Question 2: (deletion of gene BdFAR4, reduced content of C20:0 fatty alcohols), (deletion of gene BdFAR4, reduced content of C22:0 fatty alcohols), (overexpression of gene BdFAR4, accumulation of C20:0 fatty alcohols), (overexpression of gene BdFAR4, accumulation of C22:0 fatty alcohols), (transcriptional regulation by BdMYB41, increased BdFAR4 transcript levels)\n",
      "Progress: 31.50% completed\n",
      "Response for Row 631:\n",
      "Answer to Question 2: (no gene expression/deletion mentioned, no specific consequence related to gene expression/deletion)\n",
      "Progress: 31.55% completed\n",
      "Response for Row 632:\n",
      "Answer to Question 2: (overexpression of gene XYL1, faster xylose consumption), (overexpression of gene XYL2, faster xylose consumption), (overexpression of gene XKS, faster xylose consumption), (overexpression of gene XUT4, faster xylose consumption)\n",
      "Progress: 31.60% completed\n",
      "Response for Row 633:\n",
      "Answer to Question 2: (overexpression of gene CeDGAT2-2, consequence TAG biosynthesis restoration in yeast mutant H1246), (overexpression of gene CeDGAT2-2, consequence significant increase in oleic acid level in transgenic yeast), (overexpression of gene CeDGAT2-2, consequence dramatic enhancement of leafy oil by 7.15-fold in tobacco compared to wild-type), (overexpression of gene CeDGAT2-2, consequence dramatic enhancement of leafy oil by 1.7-fold in tobacco compared to plants expressing AtDGAT1), (overexpression of gene CeDGAT2-2, consequence increase in oleic acid from 5.1% to 31.33% in tobacco leaves), (overexpression of gene CeDGAT2-2, consequence reduction of saturated fatty acids in tobacco leaves), (overexpression of gene CeDGAT2-2, consequence small decline in starch content in tobacco), (overexpression of gene CeDGAT2-2, consequence no significant negative impact on photosynthesis in tobacco), (overexpression of gene CeDGAT2-2, consequence no significant negative impact on growth rates in tobacco), (overexpression of gene CeDGAT2-2, consequence no significant negative impact on seed germination in tobacco)\n",
      "Progress: 31.65% completed\n",
      "Response for Row 634:\n",
      "Answer to Question 2: (shuttle expression of GAL1/10 bidirectional promoter, catabolite repression in E. coli), (shuttle expression of multigene pathway, cinnamyl alcohol production)\n",
      "Progress: 31.70% completed\n",
      "Response for Row 635:\n",
      "Answer to Question 2: (expression of gene ads, enhanced artemisinin production), (expression of gene fps, enhanced artemisinin production)\n",
      "Progress: 31.75% completed\n",
      "Response for Row 636:\n",
      "Answer to Question 2: (gene expression of metabolic specialization genes, consequence 1: cellular adaptation to grape must environment), (gene expression of metabolic specialization genes, consequence 2: ethanol tolerance), (gene expression of unknown key genes, consequence 1: impact on industrial wine fermentation outcomes), (gene expression of unknown pathways, consequence 2: need for further research)\n",
      "Progress: 31.80% completed\n",
      "Response for Row 637:\n",
      "Answer to Question 2: (expression of gene psulω3, consequence eicosapentaenoic acid production), (expression of gene pmd17, consequence eicosapentaenoic acid production)\n",
      "Progress: 31.85% completed\n",
      "Response for Row 638:\n",
      "Answer to Question 2: (up-regulation of genes associated with one-carbon metabolism, improved cellular redox homeostasis), (down-regulation of genes related with DNA repair, reduced DNA repair activity), (down-regulation of genes related with stress responses, reduced stress response), (down-regulation of genes related with ion homeostasis, altered ion homeostasis), (down-regulation of genes associated with protein quality control systems, lower protein turnover)\n",
      "Progress: 31.90% completed\n",
      "Response for Row 639:\n",
      "Answer to Question 2: (expression of gene HXT7, suppression in presence of glucose), (expression of gene HXT7, derepression by xylose), (expression of gene XYL1, massive accumulation of xylitol), (expression of gene XYL2, massive accumulation of xylitol), (expression of gene XYL1-K270R, balanced xylose metabolism), (expression of gene XYL2, balanced xylose metabolism), (expression of pentose phosphate pathway genes, balanced xylose metabolism), (expression of XYL1-K270R and XYL2, ethanol production)\n",
      "Progress: 31.95% completed\n",
      "Response for Row 640:\n",
      "Answer to Question 2:  (overexpression of vtc2, increased VC accumulation), (fusion and expression of L-GalDH and L-GLDH, increased VC accumulation), (overexpression of vtc2, balanced gene expression and cell growth), (overexpression of vtc2, highest VC production from glucose)\n",
      "Progress: 32.00% completed\n",
      "Response for Row 641:\n",
      "Answer to Question 2: (expression of gene OVCA2, increased production of reactive oxygen species), (expression of gene OVCA2, decreased cell growth), (expression of gene OVCA2, disturbed mitochondrial morphology), (expression of gene OVCA2, reduced membrane potential), (expression of gene OVCA2, increased chromatin condensation), (expression of gene OVCA2, externalization of phosphatidylserine), (complementation of gene OVCA2 in fsh3Δ cells, reduced cell growth), (complementation of gene OVCA2 in fsh3Δ cells, increased apoptotic phenotypes)\n",
      "Progress: 32.05% completed\n",
      "Response for Row 642:\n",
      "Answer to Question 2: (overexpression of Haa1p, improved xylose consumption rates), (overexpression of Haa1p, enhanced ethanol yield), (overexpression of Tye7p, improved xylose consumption rates), (overexpression of Tye7p, enhanced ethanol yield), (co-overexpression of Haa1p and Tye7p, increased xylose consumption rate), (co-overexpression of Haa1p and Tye7p, increased ethanol yield)\n",
      "Progress: 32.10% completed\n",
      "Response for Row 643:\n",
      "Answer to Question 2: (deletion of gene PTH3, impaired respiratory growth), (deletion of gene PTH3, reduced protein expression of mtDNA-encoded genes), (presence of antibiotics, stalled mitoribosomes), (deletion of gene PTH4, less impairment in respiratory growth compared to PTH3 deletion)\n",
      "Progress: 32.15% completed\n",
      "Response for Row 644:\n",
      "Answer to Question 2: (overexpression of gene ChbZIP1, increased alkali resistance in Saccharomyces cerevisiae), (overexpression of gene ChbZIP1, increased alkali resistance in Arabidopsis), (overexpression of gene ChbZIP1, upregulation of gene GPX1), (overexpression of gene ChbZIP1, upregulation of gene DOX1), (overexpression of gene ChbZIP1, upregulation of gene CAT2), (overexpression of gene ChbZIP1, upregulation of gene EMB), (overexpression of gene ChbZIP1, enhanced antioxidant ability in Arabidopsis)\n",
      "Progress: 32.20% completed\n",
      "Response for Row 645:\n",
      "Answer to Question 2: (expression of gene TPI1, increase in plasmid viability), (expression of gene TPI1, increase in plasmid copy number), (overexpression of gene RAF1, higher copy number), (overexpression of gene REP2, higher copy number), (overexpression of gene TPI1, smaller cell-to-cell variation), (overexpression of gene TPI1, increased DHAA production)\n",
      "Progress: 32.25% completed\n",
      "Response for Row 646:\n",
      "Answer to Question 2: (no gene overexpression/deletion mentioned, biosensor development for small molecule detection), (no gene overexpression/deletion mentioned, biosensor application in regulating gene expression in yeast), (no gene overexpression/deletion mentioned, biosensor application in detecting metabolite production from multi-enzyme biosynthetic pathway)\n",
      "Progress: 32.30% completed\n",
      "Response for Row 647:\n",
      "Answer to Question 2: (expression of hexose transporters, emergence of glucose gradients), (expression of other genes regulated by glucose availability, quantified with intra-colony growth rates)\n",
      "Progress: 32.35% completed\n",
      "Response for Row 648:\n",
      "Answer to Question 2: (deletion of gene PpHCT, consequence 1: reduced production of caffeate derivatives), (deletion of gene PpHCT, consequence 2: impaired formation of an intact cuticle)\n",
      "Progress: 32.40% completed\n",
      "Response for Row 649:\n",
      "Answer to Question 2: (expression of gene D-xylose isomerase, consequence faster aerobic growth on D-xylose media), (expression of gene D-xylose isomerase, consequence 72% higher D-xylose isomerization rate), (expression of gene D-xylose isomerase, consequence 2.6 times higher specific activity), (expression of gene D-xylose isomerase, consequence 37% lower K<sub>M</sub> for D-xylose), (expression of gene D-xylose isomerase, consequence higher activity over a broader temperature range)\n",
      "Progress: 32.45% completed\n",
      "Response for Row 650:\n",
      "Answer to Question 2: (deletion of gene dugB, increase in GSH content), (deletion of gene dugC, increase in GSH content), (deletion of gene dugB, reduced conidium production), (deletion of gene dugC, reduced conidium production), (deletion of gene dugB, disturbed sexual development), (deletion of gene dugC, disturbed sexual development), (deletion of gene dugB, downregulation of steC), (deletion of gene dugC, downregulation of steC), (deletion of gene dugB, downregulation of sskB), (deletion of gene dugC, downregulation of sskB), (deletion of gene dugB, downregulation of hogA), (deletion of gene dugC, downregulation of hogA), (deletion of gene dugB, downregulation of mkkA), (deletion of gene dugC, downregulation of mkkA), (deletion of gene dugB, downregulation of flbA), (deletion of gene dugC, downregulation of flbA), (deletion of gene dugB, downregulation of flbC), (deletion of gene dugC, downregulation of flbC), (deletion of gene dugB, downregulation of flbE), (deletion of gene dugC, downregulation of flbE), (deletion of gene dugB, downregulation of nosA), (deletion of gene dugC, downregulation of nosA), (deletion of gene dugB, downregulation of rosA), (deletion of gene dugC, downregulation of rosA), (deletion of gene dugB, downregulation of nsdC), (deletion of gene dugC, downregulation of nsdC), (deletion of gene dugB, downregulation of nsdD), (deletion of gene dugC, downregulation of nsdD), (deletion of gene dugB, slowed depletion of GSH pools), (deletion of gene dugC, slowed depletion of GSH pools), (deletion of gene dugB, reduced accumulation of reactive oxygen species), (deletion of gene dugC, reduced accumulation of reactive oxygen species), (deletion of gene dugB, decreased autolytic cell wall degradation), (deletion of gene dugC, decreased autolytic cell wall degradation), (deletion of gene dugB, decreased enzyme secretion), (deletion of gene dugC, decreased enzyme secretion), (deletion of gene dugB, increased sterigmatocystin formation), (deletion of gene dugC, increased sterigmatocystin formation)\n",
      "Progress: 32.50% completed\n",
      "Response for Row 651:\n",
      "Answer to Question 2: (over-expression of ILV2, increased isobutanol titers), (over-expression of ILV3, increased isobutanol titers), (over-expression of ARO10, increased isobutanol titers), (mutation in multiple genes, improved isobutanol tolerance), (over-expression of CWP2, improved isobutanol tolerance), (over-expression of SRP4039, improved isobutanol tolerance)\n",
      "Progress: 32.55% completed\n",
      "Response for Row 652:\n",
      "Answer to Question 2: (stalled mRNA translation, production of incompletely synthesized proteins), (stalled translation of nonstop mRNA, nucleolar sequestration of defective proteins), (polylysine tracts production by translation of poly(A) tails, nucleolar sequestration), (nucleolar sequestration, increase in solubility of invading proteins), (nucleolar disruption, alteration of nucleoli dynamics), (nucleolar disruption, alteration of nucleoli morphology), (nucleolar disruption, alteration of resistance to stress in cell culture), (nucleolar disruption, alteration of resistance to stress in intact flies)\n",
      "Progress: 32.60% completed\n",
      "Response for Row 653:\n",
      "Answer to Question 2: (deletion of DDK, reduced phosphorylation of Mcm2-7 tails), (overexpression of DDK, increased phosphorylation of Mcm2-7 tails), (phosphorylation of Mcm2-7 tails, increased number of CtGs formed), (formation of CtGs, increased frequency of CMG formation)\n",
      "Progress: 32.65% completed\n",
      "Response for Row 654:\n",
      "Answer to Question 2: (knockdown of Trcyc8, impaired cellulase gene expression), (knockdown of Trtup1, impaired cellulase gene expression), (repression of Trtup1, defect in XYR1 occupancy), (repression of Trcyc8, defect in XYR1 occupancy), (repression of Trtup1, loss of histone H4 at cellulase gene promoters), (repression of Trcyc8, loss of histone H4 at cellulase gene promoters), (XYR1 overexpression, cannot overcome defects in Trtup1 repressed cells), (XYR1 overexpression, cannot overcome defects in Trcyc8 repressed cells)\n",
      "Progress: 32.70% completed\n",
      "Response for Row 655:\n",
      "Answer to Question 2: (overexpression of gene RPL40B, increase in octanoic acid titers by 40%)\n",
      "Progress: 32.75% completed\n",
      "Response for Row 656:\n",
      "Answer to Question 2: (deletion of survivin gene expression, decreased cell viability), (deletion of survivin gene expression, induced apoptosis)\n",
      "Progress: 32.80% completed\n",
      "Response for Row 657:\n",
      "Answer to Question 2: (deletion of gene CCP1, consequence increased pyruvate productivity), (overexpression of gene CCP1, consequence decreased pyruvate productivity), (overexpression of gene CCP1, consequence activated respiratory metabolism), (overexpression of gene CCP1, consequence increased reactive oxygen species)\n",
      "Progress: 32.85% completed\n",
      "Response for Row 658:\n",
      "Answer to Question 2: (no expression/deletion of gene mentioned, focus on Pichia pastoris as host for heterologous protein expression), (no expression/deletion of gene mentioned, high-cell density fermentation method for P. pastoris discussed), (improvements in gene-editing tools, strain improvement for P. pastoris)\n",
      "Progress: 32.90% completed\n",
      "Response for Row 659:\n",
      "Answer to Question 2: (expression of gene CaNES1, consequence differential transcription in different tissues of C. angulatus), (expression of gene CaNES2, consequence differential transcription in different tissues of C. angulatus), (expression of gene CaNES2, consequence production of trans-nerolidol 7.01 g/L in S. cerevisiae)\n",
      "Progress: 32.95% completed\n",
      "Response for Row 660:\n",
      "Answer to Question 2: (upregulation of SWEET gene, sucrose accumulation in M. truncatula root tissue), (upregulation of GspInv gene, increased sucrose utilization by w5), (upregulation of GspSUT1 gene, increased sucrose transport by w5), (heterologous expression of GspInv, invertase function confirmed), (heterologous expression of GspSUT1, sugar transporter function confirmed)\n",
      "Progress: 33.00% completed\n",
      "Response for Row 661:\n",
      "Answer to Question 2: (deletion of Hsf1p binding site, reduced copper-dependent gelatin induction), (overexpression of CUP1 promoter, increased gelatin expression), (induction by copper, dose-dependent gelatin expression)\n",
      "Progress: 33.05% completed\n",
      "Response for Row 662:\n",
      "Answer to Question 2: (deletion of gene MoMca1, reduced sporulation), (deletion of gene MoMca1, delay in conidial germination), (deletion of gene MoMca1, attenuation of disease severity), (deletion of gene MoMca2, reduced sporulation), (deletion of gene MoMca2, delay in conidial germination), (deletion of gene MoMca2, attenuation of disease severity), (deletion of gene MoMca1 and MoMca2, increased radial growth in presence of oxidative stress), (deletion of gene MoMca1 and MoMca2, increased accumulation of insoluble aggregates)\n",
      "Progress: 33.10% completed\n",
      "Response for Row 663:\n",
      "Answer to Question 2: (heterologous expression of fungal BGCs, possibility investigation), (heterologous expression of fungal BGCs, effectiveness evaluation)\n",
      "Progress: 33.15% completed\n",
      "Response for Row 664:\n",
      "Answer to Question 2: (deletion of gene GFP, consequence assessed by levels of GFP fluorescence), (deletion of gene GFP, consequence assessing growth rates of GFP integrated strains), (expression of gene synthetic gRNA-binding sites, consequence resulted in five different yeast strains harbouring 1-5 synthetic gRNA-binding sites in their genomes), (expression of gene 2-, 3-, 4-, or 5-genes, consequence demonstrated with efficiencies from 85% to 98%), (expression of gene 6-gene, consequence generating a yeast strain producing a sesquiterpene lactone, kauniolide)\n",
      "Progress: 33.20% completed\n",
      "Response for Row 665:\n",
      "Answer to Question 2: (deletion of gene AcOTAbZIP, loss of OTA), (deletion of gene AcOTAbZIP, loss of OTB), (deletion of gene AcOTAbZIP, loss of OTβ), (deletion of gene AcOTAbZIP, down-regulation of AcOTApks), (deletion of gene AcOTAbZIP, down-regulation of AcOTAnrps), (deletion of gene AcOTAbZIP, down-regulation of AcOTAp450), (deletion of gene AcOTAbZIP, down-regulation of AcOTAhal)\n",
      "Progress: 33.25% completed\n",
      "Response for Row 666:\n",
      "Answer to Question 2: (deletion of gene, anthranilate accumulation), (expression of anthranilic acid methyltransferase 1, methyl anthranilate production)\n",
      "Progress: 33.30% completed\n",
      "Response for Row 667:\n",
      "Answer to Question 2: (deletion of gene NRG1, consequence production of mannosyl-erythritol lipids), (deletion of gene NRG1, consequence inhibition of expression of siderophore cluster genes), (deletion of gene NRG1, consequence filamentous growth), (deletion of gene NRG1, consequence virulence), (deletion of gene NRG1, consequence oxidative stress), (deletion of gene NRG1, consequence alteration of 368 genes expression), (deletion of gene NRG1, consequence up-regulation of 205 genes), (deletion of gene NRG1, consequence down-regulation of 163 genes), (deletion of gene NRG1, consequence affects HOG1 for osmotic stress), (deletion of gene NRG1, consequence affects RIM101 for alkaline pH), (deletion of gene NRG1, consequence affects WCO1 encoding PAS domain), (deletion of gene NRG1, consequence glucose repression), (deletion of gene NRG1, consequence metabolic adaptation to adverse conditions), (deletion of gene NRG1, consequence cellular transport), (deletion of gene NRG1, consequence cell rescue), (deletion of gene NRG1, consequence defense), (deletion of gene NRG1, consequence interaction with acidic pH environment)\n",
      "Progress: 33.35% completed\n",
      "Response for Row 668:\n",
      "Answer to Question 2: (gene expression changes, inhibition of cell proliferation), (gene expression changes, cell wall damage), (gene expression changes, alteration of function and morphology of organelles), (gene expression changes, presence of oxidative stress bio-indicators), (gene expression changes, genotoxicity), (gene expression changes, cell death)\n",
      "Progress: 33.40% completed\n",
      "Response for Row 669:\n",
      "Answer to Question 2: (expression of gene β-carotene pathway, consequence increased β-carotene production), (expression of gene zeaxanthin pathway, consequence increased zeaxanthin production), (expression of gene astaxanthin pathway, consequence increased astaxanthin production)\n",
      "Progress: 33.45% completed\n",
      "Response for Row 670:\n",
      "Answer to Question 2: (overexpression of polytryptophan sequences, induction of Rqc2-independent RQC), (overexpression of polytryptophan sequences, requirement of Hel2-mediated ribosome ubiquitination), (overexpression of polytryptophan sequences, requirement of the RQT complex), (overexpression of polytryptophan sequences, inhibition of CAT-tailing by tandem CGA codons), (overexpression of polytryptophan sequences, induction of Hel2-mediated canonical RQC-coupled NGD), (overexpression of polytryptophan sequences, induction of RQC-uncoupled NGD outside the stalled ribosomes)\n",
      "Progress: 33.50% completed\n",
      "Response for Row 671:\n",
      "Answer to Question 2: (deletion of cap in mRNA, consequence 1: reduced translation efficiency), (expression of BBV-seq, consequence 1: increased translation efficiency on cap-free mRNA), (expression of BBV-seq, consequence 2: enhanced cap-dependent translation initiation), (interaction of BBV-seq with eIF4G1, consequence 1: promotion of cap-independent translation initiation)\n",
      "Progress: 33.55% completed\n",
      "Response for Row 672:\n",
      "Answer to Question 2: (expression of gene AnfH, active in electron donation to NifDK), (expression of gene AnfDK, exists as apo-AnfDK complex), (addition of FeMo-co, activates apo-AnfDK for acetylene reduction)\n",
      "Progress: 33.60% completed\n",
      "Response for Row 673:\n",
      "Answer to Question 2: (deletion of essential subunits of 26S proteasome, sensitivity to genotoxic stress), (deletion of essential subunits of 26S proteasome, sensitivity to proteotoxic stress), (deletion of essential subunits of 26S proteasome, sensitivity to high salinity), (deletion of essential subunits of 26S proteasome, sensitivity to osmolarity)\n",
      "Progress: 33.65% completed\n",
      "Response for Row 674:\n",
      "Answer to Question 2: (deletion of gene ADY2, reduced carotenoid accumulation), (deletion of gene HES1, reduced carotenoid accumulation), (deletion of gene CUP1, reduced carotenoid accumulation), (overexpression of gene ACE1, increased carotenoid yield), (addition of copper ions, increased carotenoid yield)\n",
      "Progress: 33.70% completed\n",
      "Response for Row 675:\n",
      "Answer to Question 2: (no gene overexpression/deletion mentioned, consequence 1: improvement of biofuels yield), (no gene overexpression/deletion mentioned, consequence 2: improvement of chemicals yield), (no gene overexpression/deletion mentioned, consequence 3: enabling real-time control through optogenetic circuits)\n",
      "Progress: 33.75% completed\n",
      "Response for Row 676:\n",
      "Answer to Question 2: (deletion of gene Atg16, fails to rescue autophagy-defective phenotype), (mutation in amphipathic α-helix of Atg16, fails to rescue autophagy-defective phenotype), (mutation in coiled-coil region of Atg16, fails to rescue autophagy-defective phenotype)\n",
      "Progress: 33.80% completed\n",
      "Response for Row 677:\n",
      "Answer to Question 2: (deletion of gene PMA1, reduction in ribosome sequestration at the ER membrane), (overexpression of cell-wall component genes, increased biosynthetic demands in the early secretory pathway)\n",
      "Progress: 33.85% completed\n",
      "Response for Row 678:\n",
      "Answer to Question 2: (deletion of gene Dbr1p, accumulation of intron RNA lariats), (deletion of gene Dbr1p, equivalent production of minus strand strong stop DNA), (deletion of gene Dbr1p, accumulation of branched Ty1 RNAs)\n",
      "Progress: 33.90% completed\n",
      "Response for Row 679:\n",
      "Answer to Question 2: (differential expression of ribosomal proteins, affecting relative proportion between ohnologs), (differential expression of cytosolic ribosomal proteins, reprogramming of translation apparatus), (differential expression of mitochondrial ribosomal proteins, reprogramming of translation apparatus), (differential expression of genes coding for products related to translation processes, affecting translation accuracy), (differential expression of genes coding for products related to rRNA post-transcriptional maturation, affecting translation accuracy), (differential expression of genes coding for products related to mRNA decapping, affecting translation accuracy), (differential expression of genes coding for products related to snoRNA synthesis, affecting translation accuracy)\n",
      "Progress: 33.95% completed\n",
      "Response for Row 680:\n",
      "Answer to Question 2: (deletion of kinase module in S. cerevisiae core-TFIIH, reduced dsDNA-stimulated ATPase activity), (deletion of kinase module in S. cerevisiae core-TFIIH, upstream shift in start-site observed in vitro)\n",
      "Progress: 34.00% completed\n",
      "Response for Row 681:\n",
      "Answer to Question 2: (deletion of butyrate-producing commensals, increased Enterobacteriaceae), (overexpression of inflammatory gene signatures, increased Enterobacteriaceae), (overexpression of extracellular matrix gene signatures, reduced levels of butyrate-producing commensals), (deletion of respiratory electron transport gene signature, increased inflammatory gene signatures), (overexpression of respiratory gene signatures, decreased inflammatory gene signatures), (deletion of extracellular matrix gene expression, presence of Aggregatibacter and Blautia bacteria)\n",
      "Progress: 34.05% completed\n",
      "Response for Row 682:\n",
      "Answer to Question 2: (adriamycin interaction with phosphofructokinase, impaired enzyme activity), (adriamycin interaction with enolase, impaired enzyme activity), (adriamycin causes dysregulation of hexokinase, impaired glycolysis), (adriamycin causes dysregulation of phosphoglycerate mutase, impaired glycolysis), (adriamycin causes dysregulation of enolase, impaired glycolysis), (adriamycin causes downregulation of pyruvate kinase, impaired glycolysis), (adriamycin causes downregulation of sod, increased oxidative stress), (adriamycin causes upregulation of bax, promotes apoptosis), (adriamycin causes upregulation of caspase, promotes apoptosis)\n",
      "Progress: 34.10% completed\n",
      "Response for Row 683:\n",
      "Answer to Question 2: (deletion of gene CYP719A37, consequence 1: no methylenedioxy bridge formation with ferulic acid), (deletion of gene CYP719A37, consequence 2: no methylenedioxy bridge formation with feruperine), (expression of gene CYP719A37, consequence 1: methylenedioxy bridge formation with feruperic acid), (expression of gene CYP719A37, consequence 2: production of piperic acid)\n",
      "Progress: 34.15% completed\n",
      "Response for Row 684:\n",
      "Answer to Question 2: (expression of tlr2, consequence elevation), (expression of il1r1, consequence elevation), (expression of irak4, consequence elevation), (expression of tollip2, consequence elevation), (expression of tnfα, consequence elevation), (expression of il10, consequence elevation), (expression of tgfβ, consequence elevation), (expression of il1β, consequence elevation), (expression of ifnγ, consequence elevation), (expression of il12, consequence elevation), (expression of gata3, consequence elevation)\n",
      "Progress: 34.20% completed\n",
      "Response for Row 685:\n",
      "Answer to Question 2: (overexpression of gene PDR3, improvement of lactic acid tolerance), (overexpression of gene PDR3, contribution to yeast stress tolerance)\n",
      "Progress: 34.25% completed\n",
      "Response for Row 686:\n",
      "Answer to Question 2:  (deletion of gene MSS2, impaired growth upon chitosan treatment), (deletion of gene MSS2, slow growth on glycerol as sole carbon source), (deletion of gene MSS2, impaired ATP production), (deletion of gene MSS2, impaired invasive ability in vitro), (deletion of gene MSS2, impaired invasive ability ex vivo), (deletion of gene MSS2, impaired biofilm formation), (deletion of gene MSS2, reduced virulence), (repression of gene MSS2, inhibited by chitosan), (repression of gene MSS2, leads to repression of mitochondrial function)\n",
      "Progress: 34.30% completed\n",
      "Response for Row 687:\n",
      "Answer to Question 2: (SNF1 overexpression, enhanced cell tolerance in freezing), (SNF1 overexpression, enhanced fermentation capacity in freezing), (SNF1 overexpression, upregulated proteasome), (SNF1 overexpression, altered carbon source metabolism), (SNF1 overexpression, altered protectant molecule metabolism), (SNF1 overexpression, changed cell membrane components), (SNF1 overexpression, increased leucine level before freezing), (SNF1 overexpression, increased proline level before freezing), (SNF1 overexpression, increased serine level before freezing), (SNF1 overexpression, increased isoleucine level before freezing), (SNF1 overexpression, increased arginine level before freezing), (SNF1 overexpression, increased homocitrulline level before freezing), (SNF1 overexpression, increased glycerol level before freezing), (SNF1 overexpression, increased palmitic acid level before freezing), (SNF1 overexpression, increased lysophosphatidylcholine level before freezing), (SNF1 overexpression, increased lysophosphatidylethanolamine level before freezing), (SNF1 overexpression, maintained high proline level after freezing), (SNF1 overexpression, maintained high lysine level after freezing), (SNF1 overexpression, maintained high glycerol level after freezing), (SNF1 overexpression, increased lysophosphatidylcholine level after freezing), (SNF1 overexpression, increased lysophosphatidylethanolamine level after freezing)\n",
      "Progress: 34.35% completed\n",
      "Response for Row 688:\n",
      "Answer to Question 2: (expression of gene chimeric cellobiohydrolase I, consequence glycosylation differences), (expression of gene chimeric cellobiohydrolase I, consequence surface charge differences), (expression of gene chimeric cellobiohydrolase I, consequence thermal stability differences), (expression of gene chimeric cellobiohydrolase I, consequence proteolytic stability differences), (expression of gene chimeric cellobiohydrolase I, consequence biomass digestion efficacy differences), (expression of gene chimeric cellobiohydrolase I in L. starkeyi, consequence inferior cellulase production), (expression of gene chimeric cellobiohydrolase I in S. cerevisiae, consequence lowest activity against dilute-acid-pretreated corn stover), (expression of gene chimeric cellobiohydrolase I in Y. lipolytica, consequence lower glycosylation extent), (expression of gene chimeric cellobiohydrolase I in Y. lipolytica, consequence better protease stability), (expression of gene chimeric cellobiohydrolase I in Y. lipolytica, consequence higher activity against dilute-acid-pretreated corn stover)\n",
      "Progress: 34.40% completed\n",
      "Response for Row 689:\n",
      "Answer to Question 2: (deletion of gene Gbp2, consequence 1: promotion of degradation of faulty pre-mRNAs), (deletion of gene Hrb1, consequence 1: promotion of degradation of faulty pre-mRNAs), (expression of gene Gbp2, consequence 1: function in nonsense mediated decay of spliced premature termination codon-containing mRNAs), (expression of gene Hrb1, consequence 1: function in nonsense mediated decay of spliced premature termination codon-containing mRNAs), (expression of gene Gbp2, consequence 2: help to transmit the Upf1-mediated PTC recognition to the transcripts ends), (expression of gene Hrb1, consequence 2: help to transmit the Upf1-mediated PTC recognition to the transcripts ends), (expression of gene Gbp2, consequence 3: promote translation repression of spliced transcripts that contain a PTC), (expression of gene Hrb1, consequence 3: promote translation repression of spliced transcripts that contain a PTC), (expression of gene Gbp2, consequence 4: facilitate degradation of RNA), (expression of gene Hrb1, consequence 4: facilitate degradation of RNA)\n",
      "Progress: 34.45% completed\n",
      "Response for Row 690:\n",
      "Answer to Question 2: (no expression/deletion of gene mentioned, consequence: regulation of translation elongation ensures correct folding of proteins), (no expression/deletion of gene mentioned, consequence: detection of widespread ribosome collisions), (no expression/deletion of gene mentioned, consequence: collisions occur more frequently in gap regions between α-helices), (no expression/deletion of gene mentioned, consequence: paused or collided ribosomes are associated with specific chaperones)\n",
      "Progress: 34.50% completed\n",
      "Response for Row 691:\n",
      "Answer to Question 2: (deletion of gene AGP1, consequence 1), (deletion of gene AGP1, reduced higher alcohol formation), (deletion of gene GDH1, consequence 1), (deletion of gene GDH1, reduced higher alcohol formation), (deletion of gene THR6, consequence 1), (deletion of gene THR6, reduced higher alcohol formation)\n",
      "Progress: 34.55% completed\n",
      "Response for Row 692:\n",
      "Answer to Question 2: (deletion of gene HTZ1, consequence increased tyrosine biosynthesis), (deletion of gene HTZ1, consequence increased p-coumaric acid titer), (random mutagenesis, consequence increased tyrosine production), (random mutagenesis, consequence increased p-coumaric acid titer)\n",
      "Progress: 34.60% completed\n",
      "Response for Row 693:\n",
      "Answer to Question 2: (deletion of gene Rpn4, consequence 1), (overexpression of gene RPN4, consequence 2)\n",
      "Progress: 34.65% completed\n",
      "Response for Row 694:\n",
      "Answer to Question 2: (deletion of gene MSN2, decrease in liamocin yield by 92.28%), (complementation of gene MSN2, restoration of liamocin synthesis), (knockout of gene BCY1, decrease in liamocin yield by 96.73%), (deletion of gene HOG1, decrease in liamocin biosynthesis by 98.09%), (deletion of gene HOG1, reduction in Msn2 and liamocin synthesis), (high glucose and low nitrogen conditions, positive regulation of genes related to liamocin biosynthesis by Msn2)\n",
      "Progress: 34.70% completed\n",
      "Response for Row 695:\n",
      "Answer to Question 2: (deletion of gene expression, consequence 1: inability to balance the glycolytic pathway through the glycogen/trehalose shunt), (expression of new isoforms of glycolytic enzymes, consequence 1: increase in the amount of adenosine triphosphate), (expression of new isoforms of glycolytic enzymes, consequence 2: increase in efficiency of product formation), (expression of new isoforms of glycolytic enzymes, consequence 3: maintenance of glycolytic balance)\n",
      "Progress: 34.75% completed\n",
      "Response for Row 696:\n",
      "Answer to Question 2: (expression of gene speD, increased spermidine titer), (expression of gene speE, increased spermidine titer), (co-expression of gene speD and speE, spermidine titer of 105.2 mg/L), (fermentation medium optimization, spermidine titer increased to 227.4 mg/L), (consumption of S-adenosylmethionine, accumulation of spermidine)\n",
      "Progress: 34.80% completed\n",
      "Response for Row 697:\n",
      "Answer to Question 2:  (expression of SLC28A3, increased liver mRNA), (expression of SLC29A1, increased liver mRNA), (expression of SLC29A2, increased liver mRNA), (expression of PC, increased liver mRNA), (expression of PCK1, increased liver mRNA), (expression of FBP1, increased liver mRNA), (expression of SREBP1c, increased liver mRNA), (expression of HSL, increased liver mRNA), (expression of CYP7a1, increased liver mRNA), (deletion of EAAT2, decreased placental gene expression), (deletion of EAAT3, decreased placental gene expression), (deletion of LAT1, decreased placental gene expression), (deletion of PAT1, decreased placental gene expression), (deletion of PPARγ, decreased placental protein expression), (deletion of AKT, decreased placental protein expression), (deletion of phosphorylated-AKT, decreased placental protein expression), (deletion of phosphorylated-mTOR, decreased placental protein expression), (deletion of Raptor, decreased placental protein expression)\n",
      "Progress: 34.85% completed\n",
      "Response for Row 698:\n",
      "Answer to Question 2: (deletion of gene Yas1p, consequence 1), (deletion of gene Yas2p, consequence 1), (deletion of gene Yas3p, consequence 1)\n",
      "Progress: 34.90% completed\n",
      "Response for Row 699:\n",
      "Answer to Question 2: (deletion of gene BDH1, consequence 1: significant enhancement of acetoin synthesis), (overexpression of gene BDH2, consequence 1: significant enhancement of acetoin synthesis), (deletion of gene BDH1, consequence 2: 2.6-fold increase of TTMP production), (overexpression of gene ALS, consequence 1: enhancement of diacetyl synthesis), (replacement of promoter ILV2 with PGK1p, consequence 1: no further increase in diacetyl, acetoin, and TTMP production)\n",
      "Progress: 34.95% completed\n",
      "Response for Row 700:\n",
      "Answer to Question 2: (mutations in splicing factors, altered gene expression), (carcinogenic mutations, faithful selection of key intronic regions), (splicing modulators, molecular mechanism of splicing), (pre-mRNA cleavage, chemical details by quantum-classical molecular dynamics)\n",
      "Progress: 35.00% completed\n",
      "Response for Row 701:\n",
      "Answer to Question 2: (deletion of gene ERAD components, consequence 1), (overexpression of gene ERAD components, consequence 2)\n",
      "Progress: 35.05% completed\n",
      "Response for Row 702:\n",
      "Answer to Question 2: (deletion of gene fab1, affects growth), (deletion of gene fab1, affects conidia development), (deletion of gene fab1, affects sclerotial development), (deletion of gene fab1, affects cellular acidification), (deletion of gene fab1, affects metal ion homeostasis), (deletion of gene fab1, affects aflatoxin production), (deletion of gene fab1, affects pathogenicity), (deletion of gene fab1, leads to vacuolar dysregulation), (deletion of gene fab1, impairs aflatoxin synthesis)\n",
      "Progress: 35.10% completed\n",
      "Response for Row 703:\n",
      "Answer to Question 2: (deletion of gene Spt4/5, consequence reduced association with elongation complexes), (overexpression of gene Gal4-VP16, consequence increased binding of RNApII to template DNA), (binding of gene Spt4/5, consequence association of mRNA binding protein Hek2)\n",
      "Progress: 35.15% completed\n",
      "Response for Row 704:\n",
      "Answer to Question 2: (expression of gene TaCslA12, production of wheat-like linear mannan), (expression of gene TaCslA12, lack of glucose residues in mannan), (expression of gene TaCslA12, moderate acetylation of mannan)\n",
      "Progress: 35.20% completed\n",
      "Response for Row 705:\n",
      "Answer to Question 2: (deletion of gene CCL5, consequence suppressed expression), (deletion of gene CXCL16, consequence suppressed expression), (deletion of gene IL8, consequence suppressed expression), (deletion of gene IL1B, consequence tended to decrease expression)\n",
      "Progress: 35.25% completed\n",
      "Response for Row 706:\n",
      "Answer to Question 2:  (overexpression of gene mtRNAP, consequence promoter melting), (overexpression of gene mtRNAP, consequence template alignment), (overexpression of gene mtRNAP, consequence DNA scrunching), (overexpression of gene mtRNAP, consequence abortive synthesis), (overexpression of gene mtRNAP, consequence transition into elongation), (interaction with transcription factor MTF1, consequence base-specific interactions with flipped non-template nucleotides \"AAGT\"), (formation of initiation complex IC, consequence positioning of RNA and NTP analog UTPαS), (formation of initiation complex IC, consequence NT scrunching into an NT loop), (stabilization by MTF1 C-tail, consequence stabilization of the scrunched NT loop), (dynamic equilibrium between IC and PmIC states, consequence frequent scrunching/unscrunching transitions), (steric clashes of the inflating NT loop and growing RNA:DNA with the C-tail, consequence abortive synthesis), (steric clashes of the inflating NT loop and growing RNA:DNA with the C-tail, consequence transition into elongation)\n",
      "Progress: 35.30% completed\n",
      "Response for Row 707:\n",
      "Answer to Question 2: (deletion of gene RME1, abolishes chlamydosporulation), (overexpression of gene RME1, bypasses the requirement for chlamydosporulation cues and regulators), (activation of gene expression by Rme1, leads to upregulation of genes during chlamydosporulation)\n",
      "Progress: 35.35% completed\n",
      "Response for Row 708:\n",
      "Answer to Question 2: (expression of genes related to amino acid metabolic processes, increased 2,3-BDO tolerance)\n",
      "Progress: 35.40% completed\n",
      "Response for Row 709:\n",
      "Answer to Question 2: (introduction of synonymous mutations in genes, improved allocation of ribosomes), (introduction of synonymous mutations in genes, decreased traffic jams during translation), (introduction of synonymous mutations in genes, increased resources availability in the cell), (introduction of synonymous mutations in genes, improved growth-rate)\n",
      "Progress: 35.45% completed\n",
      "Response for Row 710:\n",
      "Answer to Question 2:  (expression of gene CYP716A249, consequence production of oleanolic acid), (expression of gene CYP716A249, consequence oxidation of C-28 position of β-amyrin)\n",
      "Progress: 35.50% completed\n",
      "Response for Row 711:\n",
      "Answer to Question 2: (expression of gene S. cerevisiae terminator, consequence tunable range of 17-fold in P. pastoris), (expression of gene S. cerevisiae terminator, consequence maintenance of function in P. pastoris), (expression of gene reporter gene, consequence successful tuning of protein expression levels), (expression of gene betaxanthin pathway, consequence successful tuning of protein expression levels), (expression of gene mRNA abundance, consequence increase in protein expression levels), (stability of mRNA 3'-UTR secondary structure, consequence reflection of protein expression levels)\n",
      "Progress: 35.55% completed\n",
      "Response for Row 712:\n",
      "Answer to Question 2: (no specific gene expression/deletion mentioned, consequence mitochondrial function modulation), (no specific gene expression/deletion mentioned, consequence cellular homeostasis maintenance), (no specific gene expression/deletion mentioned, consequence bioenergetic production modulation)\n",
      "Progress: 35.60% completed\n",
      "Response for Row 713:\n",
      "Answer to Question 2: (no specific gene overexpression/deletion mentioned, general discussion on ribosome heterogeneity and its role in translation specificity), (no specific gene overexpression/deletion mentioned, failure in ribosome-mediated regulation of protein synthesis can play a crucial role in disease development)\n",
      "Progress: 35.65% completed\n",
      "Response for Row 714:\n",
      "Answer to Question 2: (expression of gene FgSULT1, increased solubility of unnatural natural product sulfates), (expression of gene FgSULT1, modification of benzenediol lactones), (expression of gene FgSULT1, modification of anthraquinone)\n",
      "Progress: 35.70% completed\n",
      "Response for Row 715:\n",
      "Answer to Question 2: (expression of gene GAL1, consequence 132.9 ± 22.6-fold induction in darkness), (expression of metabolic enzymes, consequence boosts production of lactic acid by more than 50%), (expression of metabolic enzymes, consequence boosts production of isobutanol by more than 15%)\n",
      "Progress: 35.75% completed\n",
      "Response for Row 716:\n",
      "Answer to Question 2: (deletion of gene coq2, inhibition of CoQ biosynthesis by Bz), (overexpression of gene ppt1, overcoming inhibition of CoQ biosynthesis by Bz)\n",
      "Progress: 35.80% completed\n",
      "Response for Row 717:\n",
      "Answer to Question 2: (deletion of gene, consequence 1), (expression of gene, consequence 2)\n",
      "Progress: 35.85% completed\n",
      "Response for Row 718:\n",
      "Answer to Question 2: (deletion of gene Gal4, insufficient inducibility), (overexpression of gene Gal4 mutants, improved transcriptional regulation), (overexpression of gene Gal4<sup>T406A/V413A</sup>, increased transcription of P<sub>GAL</sub>-driven lycopene pathway genes), (overexpression of gene Gal4<sup>T406A/V413A</sup>, 48% higher lycopene accumulation)\n",
      "Progress: 35.90% completed\n",
      "Response for Row 719:\n",
      "Answer to Question 2: (gene overexpression in brewer's yeast strains, adaptation to brewing environment), (gene overexpression in baker's yeast, adaptation to osmotic stress), (gene overexpression in yeast used for ethanol production, adaptation to toxic stress)\n",
      "Progress: 35.95% completed\n",
      "Response for Row 720:\n",
      "Answer to Question 2: (deletion of gene Cldn2, consequence increased gene expression in PH mice), (deletion of gene Cldn3, consequence increased gene expression in PH mice), (deletion of gene Cldn8, consequence increased gene expression in PH mice), (deletion of gene Ocln, consequence increased gene expression in PH mice)\n",
      "Progress: 36.00% completed\n",
      "Response for Row 721:\n",
      "Answer to Question 2: (no expression/deletion of gene mentioned, development of RiboDiPA tool for analyzing ribosome occupancy patterns), (no expression/deletion of gene mentioned, RiboDiPA identifies genes with significant differences in ribosome occupancy patterns)\n",
      "Progress: 36.05% completed\n",
      "Response for Row 722:\n",
      "Answer to Question 2: (deletion of gene mpkB, reduction in conidia production), (deletion of gene mpkB, vesicular dome-immaturity in the conidiophore head), (deletion of gene mpkB, decreased phialide numbers), (deletion of gene mpkB, very short stalks), (deletion of gene mpkB, inability to produce sclerotia), (deletion of gene mpkB, normal production of aflatoxin B1)\n",
      "Progress: 36.10% completed\n",
      "Response for Row 723:\n",
      "Answer to Question 2: (deletion of gene pab1, decrease in translation termination efficiency), (proximity of nonsense codon to mRNA 3' end, increase in premature termination products), (proximity of nonsense codon to mRNA 3' end, decrease in readthrough efficiencies), (lengthening of 3' UTR, increase in readthrough efficiencies for PTC), (inefficient termination, trigger for nonsense-mediated mRNA decay)\n",
      "Progress: 36.15% completed\n",
      "Response for Row 724:\n",
      "Answer to Question 2:  (deletion of gene GDH1, total higher alcohol concentration decreased by 27.31%), (deletion of gene CAN1, total content of higher alcohols increased by 211.44%), (deletion of gene GAT1, total content of higher alcohols increased by 28.36%)\n",
      "Progress: 36.20% completed\n",
      "Response for Row 725:\n",
      "Answer to Question 2: (deletion of gene SOR1, increased xylitol yield), (deletion of gene SOR2, increased xylitol yield), (deletion of gene XKS1, increased xylitol yield), (deletion of gene XKS1, weakened SOR1 expression), (deletion of gene XKS1, weakened SOR2 expression), (overexpression of gene XYL1, reduced xylose to xylitol)\n",
      "Progress: 36.25% completed\n",
      "Response for Row 726:\n",
      "Answer to Question 2: (ectopic expression of CSyGTs, complement soyasaponin biosynthesis), (CSyGT mutants, do not accumulate soyasaponin), (expression of CSyGTs, produce glycyrrhizin de novo)\n",
      "Progress: 36.30% completed\n",
      "Response for Row 727:\n",
      "Answer to Question 2: (mutated XR, higher transcript levels of XYL1), (mutated XR, higher transcript levels of XYL2), (mutated XR, higher transcript levels of XKS1), (mutated XR, improved xylose consumption rate), (mutated XR, improved ethanol yield), (mutated XR, upregulated genes related to protein biosynthesis in diploid strains), (mutated XR, downregulated genes involved in glyoxylate shunt in diploid strains), (mutated XR, improved biomass yield in diploid strains), (haploid vs diploid, downregulated genes involved in glycolysis in diploid strains), (haploid vs diploid, downregulated genes involved in MAPK signaling pathway in diploid strains), (haploid vs diploid, upregulated oxidative stress related TFs in diploid strains)\n",
      "Progress: 36.35% completed\n",
      "Response for Row 728:\n",
      "Answer to Question 2: (deletion of tyrosine feedback inhibition, increased naringenin production), (downregulation of competitive pathway, increased naringenin production), (multi-copy integration of flavonoid pathway genes, increased naringenin production), (increased copy numbers of (2S)-naringenin metabolic pathway genes, increased 4CL transcription), (increased copy numbers of (2S)-naringenin metabolic pathway genes, increased CHS transcription), (increased 4CL transcription, enhanced naringenin production), (increased CHS transcription, enhanced naringenin production)\n",
      "Progress: 36.40% completed\n",
      "Response for Row 729:\n",
      "Answer to Question 2: (expression of nuclear receptors, regulation of cytokine genes), (overexpression of TFs, modulation of cytokine production), (deletion of specific TFs, alteration in cytokine gene promoters interaction)\n",
      "Progress: 36.45% completed\n",
      "Response for Row 730:\n",
      "Answer to Question 2: (up-regulation of Sho1, activation of MAPK pathway), (up-regulation of Sho1, filamentous growth), (up-regulation of fatty acid synthesis genes, lipid droplets accumulation), (up-regulation of fatty acid degradation genes, lipid mobilization)\n",
      "Progress: 36.50% completed\n",
      "Response for Row 731:\n",
      "Answer to Question 2: (overexpression of gene Cwh41p, increase in protein production), (upregulation of glycogen metabolism, increase in α-amylase productivity), (upregulation of trehalose metabolism, increase in α-amylase productivity), (improved folding precision, reduced energy expenditure), (regulation of N-glycans, improved folding precision)\n",
      "Progress: 36.55% completed\n",
      "Response for Row 732:\n",
      "Answer to Question 2: (deletion of gene MTQ2, severe growth impairment), (deletion of gene MTQ2, impaired nucleolar release of pre-60S), (deletion of gene MTQ2, reduced production of mature 5.8S rRNA), (deletion of gene MTQ2, reduced production of mature 25S rRNA)\n",
      "Progress: 36.60% completed\n",
      "Response for Row 733:\n",
      "Answer to Question 2: (overexpression of gene SSA1, increased vector yields), (overexpression of gene SSE1, increased vector yields), (overexpression of gene SSE2, increased vector yields), (overexpression of gene CCP1, increased vector yields), (overexpression of gene GTT1, increased vector yields), (overexpression of gene RVB2, increased vector yields), (overexpression of gene HSP40, increased vector yields), (overexpression of gene HSC70, increased vector yields), (overexpression of heat shock proteins, activation of unfolded protein response), (overexpression of stress protein markers, activation of unfolded protein response), (overexpression of proteins involved in oxidative stress response, reduction in reactive oxygen species effects)\n",
      "Progress: 36.65% completed\n",
      "Response for Row 734:\n",
      "Answer to Question 2: (deletion of gene Nrc, consequence 1), (deletion of gene Nrc, consequence 2)\n",
      "Progress: 36.70% completed\n",
      "Response for Row 735:\n",
      "Answer to Question 2: (no expression/deletion of gene mentioned, discussion on method for identifying translation initiation sites), (no expression/deletion of gene mentioned, highlights challenges in studying non-canonical protein isoforms), (no expression/deletion of gene mentioned, supports expanded use of empirical protein-coding region identification)\n",
      "Progress: 36.75% completed\n",
      "Response for Row 736:\n",
      "Answer to Question 2: (deletion of gene HIM1, consequence reduces efficiency of induction of RNR genes after UV irradiation), (deletion of gene RAD30, consequence suppresses him1-dependent UV mutagenesis), (overexpression of gene Polη, consequence increases him1-dependent mutagenesis at NBP sites), (increase in dNTP levels, consequence suppresses him1-dependent mutagenesis)\n",
      "Progress: 36.80% completed\n",
      "Response for Row 737:\n",
      "Answer to Question 2: (expression of gene SlCHS, nitrogen-carbon bond formation), (expression of gene SlCHS, production of hydroxycinnamic acid amide), (expression of gene SlMT2, methylation of 3-hydroxyanthranilic acid), (expression of gene SlCHS, condensation of 3-HAA methyl ester and p-coumaroyl-CoA), (expression of gene SlCHS, formation of amide bond)\n",
      "Progress: 36.85% completed\n",
      "Response for Row 738:\n",
      "Answer to Question 2: (deletion of gene Matα:57-70, consequence highest G-CSF titre of 39.4 ± 1.4 mg/L), (deletion of gene Matα:30-43, consequence completely diminished G-CSF secretion), (deletion of gene Matα:47-49, consequence dampened the secretion of WT-GCSF), (addition of deletion Matα:57-70 to Matα:30-43, consequence partially restored G-CSF secretion by 60%), (addition of deletion Matα:57-70 to Matα:47-49, consequence partially restored G-CSF secretion by 60%), (deletion of gene Matα:57-70 and Matα:47-49, consequence stimulatory effect on G-CSF secretion with CO-GCSF)\n",
      "Progress: 36.90% completed\n",
      "Response for Row 739:\n",
      "Answer to Question 2: (optogenetic control of gene expression, activation of target gene), (optogenetic control of protein subcellular localization, reconstitution of protein activity), (optogenetic control of protein sequestration, protein oligomerization), (FUN-LOV system, precise activation of target gene)\n",
      "Progress: 36.95% completed\n",
      "Response for Row 740:\n",
      "Answer to Question 2: (deletion of HMGR1, increase in total carotenoids to 32 mg/gDCW), (expression of carotenogenic genes CrtE, CrtYB, CrtI, increase in total carotenoids to 16 mg/gDCW), (expression of PhCCD1, production of β-ionone up to 33 mg/L)\n",
      "Progress: 37.00% completed\n",
      "Response for Row 741:\n",
      "Answer to Question 2: (deletion of gene Apn1, consequence oxidative stress), (deletion of gene Apn2, consequence oxidative stress), (deletion of gene Rad27, consequence oxidative stress), (deletion of gene Ntg1, consequence DNA replication stress), (deletion of gene Ntg2, consequence DNA replication stress)\n",
      "Progress: 37.05% completed\n",
      "Response for Row 742:\n",
      "Answer to Question 2: (co-expression of gene Tfu_0875, AA production), (co-expression of gene Tfu_2399, AA production), (co-expression of gene Tfu_0067, AA production), (co-expression of gene Tfu_1647, AA production), (co-expression of gene Tfu_2576, AA production)\n",
      "Progress: 37.10% completed\n",
      "Response for Row 743:\n",
      "Answer to Question 2: (heterologous expression of HaWS, 57 kDa protein confirmed by immunoblotting), (heterologous expression of HaWS, accumulation of C-32 to C-36 WE), (heterologous expression of HaWS, synthesis of specific WE of desired lengths)\n",
      "Progress: 37.15% completed\n",
      "Response for Row 744:\n",
      "Answer to Question 2: (ectopic expression of gene OsDGAT1-1, restoration of TAG synthesis), (ectopic expression of gene OsDGAT1-1, restoration of lipid body formation), (ectopic expression of gene OsDGAT1-2, restoration of TAG synthesis), (ectopic expression of gene OsDGAT1-2, restoration of lipid body formation), (deletion/mutation in N-terminal region of OsDGAT1-1, reduced lipid accumulation), (alteration in N-terminal region of OsDGAT1-1, reduced functionality), (expression of gene OsDGAT1-1, accumulation of triacylglycerol), (expression of gene OsDGAT1-2, accumulation of triacylglycerol)\n",
      "Progress: 37.20% completed\n",
      "Response for Row 745:\n",
      "Answer to Question 2: (no expression/deletion of gene specified, complexity of optimal control problems), (no expression/deletion of gene specified, prediction of temporal profiles of enzyme activation), (no expression/deletion of gene specified, prediction of metabolite concentrations)\n",
      "Progress: 37.25% completed\n",
      "Response for Row 746:\n",
      "Answer to Question 2: (heterologous expression of FvPPT, production of core scaffold compound of bostrycoidin), (heterologous expression of fsr1, production of core scaffold compound of bostrycoidin), (heterologous expression of fsr2, production of core scaffold compound of bostrycoidin), (heterologous expression of fsr3, production of core scaffold compound of bostrycoidin)\n",
      "Progress: 37.30% completed\n",
      "Response for Row 747:\n",
      "Answer to Question 2: (heterologous genes from photosynthetic organisms, enhancement of precursors supply), (proper truncation of the N-terminal transit peptide, improvement in tocotrienols production), (cold-shock-triggered temperature control system, efficient control of two-stage fermentation), (engineered yeast, production of 320 mg/L tocotrienols)\n",
      "Progress: 37.35% completed\n",
      "Response for Row 748:\n",
      "Answer to Question 2: (overexpression of gene ScHac1p, improved GOX secretion capacity), (overexpression of gene ScHac1p, increased overall enzyme production), (overexpression of gene ScHac1p, reduced cell growth), (overexpression of gene PpHac1p, improved GOX secretion capacity), (overexpression of gene PpHac1p, increased overall enzyme production), (overexpression of gene PpHac1p, reduced cell growth), (overexpression of gene TrHac1p, improved GOX secretion capacity), (overexpression of gene TrHac1p, increased overall enzyme production), (overexpression of gene TrHac1p, reduced cell growth), (overexpression of gene HsXbp1, improved GOX secretion capacity), (overexpression of gene HsXbp1, increased overall enzyme production), (overexpression of gene ScHac1p, increased oxidative stress), (overexpression of gene PpHac1p, increased oxidative stress), (overexpression of gene TrHac1p, increased oxidative stress), (overexpression of gene ScHac1p, loss of GOX gene), (overexpression of gene PpHac1p, loss of GOX gene)\n",
      "Progress: 37.40% completed\n",
      "Response for Row 749:\n",
      "Answer to Question 2: (disruption of gene MTI1, consequence haploid and stably heterothallic), (disruption of gene MTI1, consequence inability to self-diploidize), (mutation in gene MTI1, consequence formation of truncated product), (mutation in gene MTI1, consequence heterothallism from original homothallism)\n",
      "Progress: 37.45% completed\n",
      "Response for Row 750:\n",
      "Answer to Question 2: (expression of norcoclaurine synthase, toxicity in Saccharomyces cerevisiae), (cytosolic expression of norcoclaurine synthase, restriction of (S)-reticuline production), (targeting of norcoclaurine synthase to peroxisome, alleviation of toxicity), (targeting of norcoclaurine synthase to peroxisome, promotion of increased (S)-reticuline titer), (expression of engineered transcription factors, mimicking oleate response), (expression of engineered transcription factors, creation of larger peroxisomes), (larger peroxisomes, increase in benzylisoquinoline alkaloid titer)\n",
      "Progress: 37.50% completed\n",
      "Response for Row 751:\n",
      "Answer to Question 2: (upregulation of mevalonate biosynthetic gene, enhancement of astragaloside production), (addition of yeast extract, stimulation of astragaloside content), (pretreatment with yeast extract, increase in astragaloside levels)\n",
      "Progress: 37.55% completed\n",
      "Response for Row 752:\n",
      "Answer to Question 2: (deletion of gene DCL4, no 22nt TBSV-derived small RNA detected), (deletion of key RNAi factors, TBSV-BS3Ng undergoes non-canonical RNAi defensive reaction)\n",
      "Progress: 37.60% completed\n",
      "Response for Row 753:\n",
      "Answer to Question 2: (deletion of I-SceI site, cells survive cleavage), (integration of antibiotic marker-free cassette, no need for plasmid curing), (expression of λ-red proteins, facilitates genome integration), (cleavage by I-SceI, kills cells with intact I-SceI site)\n",
      "Progress: 37.65% completed\n",
      "Response for Row 754:\n",
      "Answer to Question 2: (up-regulation of genes involved in ROS production, promotion of ROS accumulation), (down-regulation of catalase, promotion of ROS accumulation)\n",
      "Progress: 37.70% completed\n",
      "Response for Row 755:\n",
      "Answer to Question 2: (deletion of templating base, conformation change of DNA substrate), (incorporation of nucleotide, conversion of pyrophosphate to monophosphates), (incorporation of nucleotide, breaking of hydrogen bonds between nucleotide and arginine side chain), (incorporation of nucleotide, prevention of potentially mutagenic processive synthesis), (incorporation of nucleotide, facilitation of dissociation of DNA product from enzyme)\n",
      "Progress: 37.75% completed\n",
      "Response for Row 756:\n",
      "Answer to Question 2: (deletion of gene not mentioned, consequence not applicable), (expression of gene 2-pyrone synthase, consequence increase in specific TAL titer), (expression of gene EGFP, consequence decrease in expression with increasing temperature), (expression of gene SSA3, consequence increase in expression above 40 °C), (expression of gene ADH1, consequence increase in expression above 40 °C)\n",
      "Progress: 37.80% completed\n",
      "Response for Row 757:\n",
      "Answer to Question 2: (deletion of gene YALI0D20196g, abolished citrate production), (expression of gene YALI0D20196p, enhanced citrate production 5.2-fold), (expression of gene YALI0D20196p, facilitated citrate export in S. cerevisiae), (expression of gene YALI0D20196p, complemented ability to secrete citrate in A. niger)\n",
      "Progress: 37.85% completed\n",
      "Response for Row 758:\n",
      "Answer to Question 2: (deletion of gene AOX, consequence strong inhibition of growth), (deletion of gene AOX, consequence respiratory capacity decrease), (expression of gene AOX, consequence no effect on growth), (expression of gene AOX, consequence minor effect on growth)\n",
      "Progress: 37.90% completed\n",
      "Response for Row 759:\n",
      "Answer to Question 2: (deletion of gene PsSOD2, reduced Pst virulence), (deletion of gene PsSOD2, restricted hyphal development), (deletion of gene PsSOD2, increased ROS accumulation), (expression of gene PsSOD2, suppressed flg22-elicited ROS production)\n",
      "Progress: 37.95% completed\n",
      "Response for Row 760:\n",
      "Answer to Question 2: (deletion of gene Put6, consequence 1: perturbs mitochondrial proline homeostasis), (deletion of gene Put6, consequence 2: defects in proline utilization), (deletion of gene Put7, consequence 1: perturbs mitochondrial proline homeostasis), (deletion of gene Put7, consequence 2: defects in proline utilization)\n",
      "Progress: 38.00% completed\n",
      "Response for Row 761:\n",
      "Answer to Question 2: (cohesin binding, formation of loops of up to 50 kb), (cohesin binding, inter-sister connections every ~35 kb)\n",
      "Progress: 38.05% completed\n",
      "Response for Row 762:\n",
      "Answer to Question 2: (expression of gene HXT7, increased D-LA production), (expression of gene HXK2, increased D-LA production), (expression of gene PGI1, increased D-LA production), (expression of gene PFK1, increased D-LA production), (expression of gene PFK2, increased D-LA production), (expression of gene FBA1, increased D-LA production), (expression of gene TPI1, increased D-LA production), (expression of gene TDH3, increased D-LA production), (expression of gene PGK1, increased D-LA production), (expression of gene GPM1, increased D-LA production), (expression of gene ENO2, increased D-LA production), (expression of gene PYK2, increased D-LA production), (expression of gene D-LDH, increased D-LA production)\n",
      "Progress: 38.10% completed\n",
      "Response for Row 763:\n",
      "Answer to Question 2: (deletion of oxygen, consequence 1: redirection of gene expression towards genes involved in central carbon metabolism), (deletion of oxygen, consequence 2: redirection of gene expression towards genes involved in lipid metabolism), (deletion of oxygen, consequence 3: redirection of gene expression towards genes involved in remodeling of the cell wall), (deletion of oxygen, consequence 4: redirection of gene expression towards genes involved in autophagy), (induction of genes involved in lactate generation, consequence 5: redirection of metabolic flux towards lactate)\n",
      "Progress: 38.15% completed\n",
      "Response for Row 764:\n",
      "Answer to Question 2: (overexpression of GAL1, enhanced galactose utilization rate), (overexpression of GAL7, enhanced galactose utilization rate), (overexpression of GAL10, enhanced galactose utilization rate), (overexpression of PGM2, twofold increase in galactose utilization rate), (overexpression of PGM2, 1.4-fold increase in ethanol production), (overexpression of PGM2, increased glucose-6-phosphate levels), (overexpression of PGM2, increased fructose-6-phosphate levels), (overexpression of PGM2, enhanced glycolytic flux)\n",
      "Progress: 38.20% completed\n",
      "Response for Row 765:\n",
      "Answer to Question 2: (deletion of adhesion ability, consequence 1), (deletion of auto-aggregation, consequence 2), (deletion of exopolysaccharide production, consequence 3), (alteration of biofilm-related gene expression, consequence 4)\n",
      "Progress: 38.25% completed\n",
      "Response for Row 766:\n",
      "Answer to Question 2: (deletion of TBP, repression of interaction with spurious DNA), (overexpression of SAGA, stimulation of transcription), (presence of SAGA, efficient TBP delivery onto gene promoters)\n",
      "Progress: 38.30% completed\n",
      "Response for Row 767:\n",
      "Answer to Question 2:  (overrepresentation of GCN codons, depress translation), (disrupting conformance to (GCN)n, neutral or positive effects on translation), (interaction surface CAR, modulates mRNA-ribosome interaction), (modulation of mRNA-ribosome interaction, regulates protein translation)\n",
      "Progress: 38.35% completed\n",
      "Response for Row 768:\n",
      "Answer to Question 2: (expression of crt1 gene, high-affinity cellobiose transporter function), (expression of crt1 gene, lactose transport capability), (expression of RUT-C30 variant of crt1 gene, full restoration of cellulase induction), (expression of cellobiose transporters from other fungal species, full restoration of cellulase induction), (expression of QM6a variant of crt1 gene, partial complementation of cellulase induction)\n",
      "Progress: 38.40% completed\n",
      "Response for Row 769:\n",
      "Answer to Question 2: (deletion of gene TPS2, decline in stress response capacity), (deletion of gene TPS2, decline in translational rate)\n",
      "Progress: 38.45% completed\n",
      "Response for Row 770:\n",
      "Answer to Question 2: (gene overexpression, reduced gene-space), (gene overexpression, improved sample classification accuracy), (gene deletion, reduced gene-space), (gene deletion, improved sample classification accuracy)\n",
      "Progress: 38.50% completed\n",
      "Response for Row 771:\n",
      "Answer to Question 2: (deletion of gene PIP1, consequence interaction with PCNA and Rad6-Rad18减弱), (deletion of gene PIP2, consequence maintaining architecture and dynamics of multi-protein complex减弱)\n",
      "Progress: 38.55% completed\n",
      "Response for Row 772:\n",
      "Answer to Question 2: (deletion of gene SMO4, consequence nucleolar hypertrophy), (deletion of gene SMO4, consequence nucleolar disorganization), (deletion of gene SMO4, consequence overaccumulation of 5.8S rRNA precursors), (deletion of gene SMO4, consequence overaccumulation of 18S rRNA precursors), (deletion of gene SMO4, consequence imbalanced 40S:60S ribosome subunit ratio)\n",
      "Progress: 38.60% completed\n",
      "Response for Row 773:\n",
      "Answer to Question 2: (deletion of nuclear genes, respiratory growth impairment), (deletion of twelve genes, viability loss in absence of mtDNA), (overexpression of ARG8^m, selection for maintenance of mtDNA), (deletion of up to 176 nuclear genes, impairment in expression of mitochondria-encoded proteins)\n",
      "Progress: 38.65% completed\n",
      "Response for Row 774:\n",
      "Answer to Question 2: (deletion of gene CIT2, no effect on β-amyrin production), (deletion of gene MLS1, increase in β-amyrin production by 1.85 times), (weakened expression of gene PGI1, increase in β-amyrin yield by 3.75 times)\n",
      "Progress: 38.70% completed\n",
      "Response for Row 775:\n",
      "Answer to Question 2: (deletion of gene product, consequence 1), (deletion of gene product, consequence 2), (transcriptional repression by Tet-OFF promoter, consequence 3), (proteolytic elimination of target protein by AID system, consequence 4)\n",
      "Progress: 38.75% completed\n",
      "Response for Row 776:\n",
      "Answer to Question 2: (expression of dCas9, alteration of gene expression up to ~2.5-fold), (expression of dCas9-VP64 fusion, alteration of gene expression up to ~2.5-fold), (expression of dCas9-VPR fusion, alteration of gene expression up to ~2.5-fold), (expression of dCas9-Mxi1 fusion, alteration of gene expression up to ~2.5-fold)\n",
      "Progress: 38.80% completed\n",
      "Response for Row 777:\n",
      "Answer to Question 2: (no specific gene expression/deletion mentioned, cellular damage accumulation), (no specific gene expression/deletion mentioned, decline in protein quality control), (no specific gene expression/deletion mentioned, maintenance of cellular homeostasis in germline), (no specific gene expression/deletion mentioned, elimination of age-induced damage in budding yeast)\n",
      "Progress: 38.85% completed\n",
      "Response for Row 778:\n",
      "Answer to Question 2: (expression of TLR2b, upregulated by Y), (expression of TLR2b, upregulated by YCW), (expression of TLR2b, upregulated by YCW + E), (expression of TLR2b, upregulated by PY/N), (expression of TLR4, downregulated by E), (expression of TLR4, downregulated by YCW), (expression of TLR4, downregulated by YCW + E), (expression of TLR4, upregulated by DDGS), (expression of TLR4, upregulated by PY/N), (expression of IL-4, upregulated by YCW), (expression of IL-4, upregulated by PY/N), (expression of IL-10, upregulated by YCW), (expression of IL-10, upregulated by PY/N), (expression of IFN-γ, upregulated by YCW), (expression of IFN-γ, upregulated by PY/N), (expression of IL-12, upregulated by YCW), (expression of IL-12, upregulated by PY/N), (expression of IL-6, upregulated by YCW), (expression of IL-6, upregulated by DDGS), (expression of IL-6, upregulated by PY/N)\n",
      "Progress: 38.90% completed\n",
      "Response for Row 779:\n",
      "Answer to Question 2: (deletion of gene xxx, consequence 1), (overexpression of gene xxx, consequence 2)\n",
      "Progress: 38.95% completed\n",
      "Response for Row 780:\n",
      "Answer to Question 2: (overexpression of gene PMA1, decrease in CPC production), (overexpression of gene PMA1, shift in beta-lactam intermediates content), (overexpression of gene PMA1, decrease in cef genes expression), (overexpression of gene PMA1, decrease in ATP level)\n",
      "Progress: 39.00% completed\n",
      "Response for Row 781:\n",
      "Answer to Question 2: (deletion of IL-1β expression, reduced cytokine response), (deletion of TNF-α expression, reduced cytokine response), (overexpression of IL-1β, enhanced cytokine response), (overexpression of TNF-α, enhanced cytokine response)\n",
      "Progress: 39.05% completed\n",
      "Response for Row 782:\n",
      "Answer to Question 2: (deletion of anticodon loop modifications, synthetic growth defects), (deletion of anticodon loop modifications, translational inefficiency of certain mRNAs), (deletion of anticodon loop modifications, disturbed protein homeostasis), (deletion of anticodon loop modifications, accumulation of protein aggregates), (deletion of anticodon loop modifications, transcriptional changes resembling cellular response to nutrient starvation)\n",
      "Progress: 39.10% completed\n",
      "Response for Row 783:\n",
      "Answer to Question 2: (deletion of gene not specified, consequence elevated reactive oxygen species (ROS) levels), (activation of gene not specified, consequence ROS defense mechanisms)\n",
      "Progress: 39.15% completed\n",
      "Response for Row 784:\n",
      "Answer to Question 2: (no gene overexpression/deletion mentioned, consequence: regulation of mitochondrial transcription through dynamic conformational changes), (no gene overexpression/deletion mentioned, consequence: stalled initiation complexes remain in dynamic scrunching and unscrunching states), (no gene overexpression/deletion mentioned, consequence: abrupt transition to an unbent conformation at position +8)\n",
      "Progress: 39.20% completed\n",
      "Response for Row 785:\n",
      "Answer to Question 2: (heterologous expression of gene PaFAD2, conversion of oleic acid to linoleic acid), (heterologous expression of gene PaFAD2, conversion of palmitoleic acid to 9,12-hexadecadienoic acid)\n",
      "Progress: 39.25% completed\n",
      "Response for Row 786:\n",
      "Answer to Question 2: (deletion of gene ADE2, consequence 1), (activation of gene mCherry, consequence 2), (repression of gene mVenus, consequence 3), (overexpression of gene CRISPR-ARE, consequence 4)\n",
      "Progress: 39.30% completed\n",
      "Response for Row 787:\n",
      "Answer to Question 2: (no gene overexpression/deletion mentioned, method to measure pairing of DNA loci during homologous recombination), (no gene overexpression/deletion mentioned, visualization of DNA at break site and homologous DNA repair template)\n",
      "Progress: 39.35% completed\n",
      "Response for Row 788:\n",
      "Answer to Question 2: (amplification of gene(s) of interest, high gene-expression levels), (amplification of gene(s) of interest, accurate gene dosage), (amplification of gene(s) of interest, stable propagation in selection-free media)\n",
      "Progress: 39.40% completed\n",
      "Response for Row 789:\n",
      "Answer to Question 2: (overexpression of gene PdGLX1, enhanced growth in E. coli), (overexpression of gene PdGLX1, enhanced MG detoxification ability in E. coli), (overexpression of gene PdGLX1, complemented loss-of-function MG hypersensitive GLO1 in yeast), (overexpression of gene PdGLX1, promoted growth in yeast), (overexpression of gene PdGLX1, reduced accumulation of ROS in yeast)\n",
      "Progress: 39.45% completed\n",
      "Response for Row 790:\n",
      "Answer to Question 2: (no gene overexpression/deletion mentioned, consequence: study of eukaryotic termination mechanisms), (no gene overexpression/deletion mentioned, consequence: identification of potential therapeutics that modulate termination)\n",
      "Progress: 39.50% completed\n",
      "Response for Row 791:\n",
      "Answer to Question 2: (cloning and expression of EGT biosynthetic genes from Grifola frondosa, increased EGT production in Saccharomyces cerevisiae EC1118), (daily addition of 1% glycerol to culture medium, increased EGT production to 20.61 mg/L)\n",
      "Progress: 39.55% completed\n",
      "Response for Row 792:\n",
      "Answer to Question 2: (deletion of metabolic flux imbalances, improvement in industry-scale production of high-value chemicals), (expression of multiple genes using COMPASS, balanced expression in biochemical pathways), (expression of pathway genes controlled by inducible artificial transcription factors, enhanced gene expression balance), (integration of genes using CRISPR/Cas9, one-step multilocus integration)\n",
      "Progress: 39.60% completed\n",
      "Response for Row 793:\n",
      "Answer to Question 2: (no gene overexpression/deletion mentioned, development of AssemblX toolkit for designing and building large DNA constructs), (no gene overexpression/deletion mentioned, allows assembly of up to 25 functional units), (no gene overexpression/deletion mentioned, uses overlap-based, scar-free, and sequence-independent cloning methods), (no gene overexpression/deletion mentioned, relies on rare cutting homing endonucleases and computationally optimized overlap sequences)\n",
      "Progress: 39.65% completed\n",
      "Response for Row 794:\n",
      "Answer to Question 2: (expression of gene ATF1, consequence n-butyl acetate production), (expression of gene ATF1, consequence direct production from glucose), (expression of gene ATF1, consequence elimination of lipase addition)\n",
      "Progress: 39.70% completed\n",
      "Response for Row 795:\n",
      "Answer to Question 2: (deletion of gene Cpl1, increased adhesive and invasive growth), (deletion of gene Cpl1, accumulation of intracellular labile zinc ion), (accumulation of intracellular labile zinc ion, increased glucose uptake), (accumulation of intracellular labile zinc ion, higher level of mitochondrial membrane potential), (accumulation of intracellular labile zinc ion, increased ATP production), (accumulation of intracellular labile zinc ion, increased ROS production), (increased ROS production, activation of Snf1), (activation of Snf1, increased invasive growth)\n",
      "Progress: 39.75% completed\n",
      "Response for Row 796:\n",
      "Answer to Question 2: (deletion of feedback inhibition of ARG5,6, consequence intracellular Orn accumulation), (mutation in ARG5,6 gene, consequence increased Orn production), (mutation in ARG5,6 gene, consequence increased Pro production)\n",
      "Progress: 39.80% completed\n",
      "Response for Row 797:\n",
      "Answer to Question 2: (Deletion of gene FgLsb6, consequence 1: defects in vegetative growth), (Deletion of gene FgLsb6, consequence 2: decreased deoxynivalenol (DON) production), (Deletion of gene FgLsb6, consequence 3: reduced pathogenicity), (Deletion of gene FgLsb6, consequence 4: increased resistance to osmotic stress), (Deletion of gene FgLsb6, consequence 5: increased resistance to oxidative stress), (Deletion of gene FgLsb6, consequence 6: increased resistance to cell wall stress), (Deletion of gene FgLsb6, consequence 7: defective generation of PI4P on endosomes), (Deletion of gene FgLsb6, consequence 8: defective endocytosis)\n",
      "Progress: 39.85% completed\n",
      "Response for Row 798:\n",
      "Answer to Question 2: (expression of gene GH from Caulobacter crescentus, increased cellobiose conversion to glucose), (expression of gene GH, enhanced growth on cellobiose), (upregulation of secretion-related proteins, aided release of GHs), (upregulation of transport-related proteins, facilitated cellobiose entry into the periplasm), (upregulation of outer membrane-related proteins, promoted surface display of GHs)\n",
      "Progress: 39.90% completed\n",
      "Response for Row 799:\n",
      "Answer to Question 2: (expression of α-amylase gene, improved transcript levels), (expression of α-amylase gene, increased protein concentrations), (expression of α-amylase gene, higher amylase activities), (expression of α-amylase gene and glucoamylase gene, improved fermentation vigour), (expression of α-amylase gene and glucoamylase gene, 97% of theoretical ethanol yield), (expression of α-amylase gene and glucoamylase gene, 100% conversion of available carbon to products)\n",
      "Progress: 39.95% completed\n",
      "Response for Row 800:\n",
      "Answer to Question 2: (deletion of A503 from Bp BteA, enhanced cytotoxic activity on mammalian HeLa cells), (deletion of A503 from Bp BteA, increased toxicity to Saccharomyces cerevisiae cells), (insertion of A503 into Bb BteA, decreased cytotoxicity to yeast), (insertion of A503 into Bb BteA, decreased cytotoxicity to HeLa cells), (expression of Bp BteAΔA503, increased virulence of B. pertussis B1917 in mouse model), (expression of Bp BteAΔA503, reduced LD50 in mouse model), (expression of Bp BteAΔA503, less inflammatory pathology in infected mouse lungs at sublethal infectious doses)\n",
      "Progress: 40.00% completed\n",
      "Response for Row 801:\n",
      "Answer to Question 2: (deletion of gene PGI1, higher ethanol production), (deletion of gene PGI1, accumulation of intracellular NADPH), (deletion of gene PGI1, co-utilization of glucose and xylose more effectively), (deletion of gene PGI1, release of glucose repression on xylose utilization), (deletion of gene ZWF1, decreased ethanol productivity), (activity of PPP, regulation of NADPH/NADP+ ratio), (NADPH/NADP+ ratio, regulation of expression of NDE1), (NADPH/NADP+ ratio, regulation of expression of NDE2), (NADPH/NADP+ ratio, regulation of expression of GLR1), (NADPH/NADP+ ratio, regulation of expression of GDP1)\n",
      "Progress: 40.05% completed\n",
      "Response for Row 802:\n",
      "Answer to Question 2: (deletion of gene expression in Streptococcus thermophilus, consequence: differential gene expression in amino acid metabolism), (deletion of gene expression in Streptococcus thermophilus, consequence: differential gene expression in citrate metabolism), (deletion of gene expression in Streptococcus thermophilus, consequence: differential gene expression in urease metabolism), (deletion of gene expression in Streptococcus thermophilus, consequence: differential gene expression in purine metabolism), (deletion of gene expression in Streptococcus thermophilus, consequence: differential gene expression in pyrimidine metabolism), (overexpression of peptides in growth media, consequence: modulation of quorum sensing system expression)\n",
      "Progress: 40.10% completed\n",
      "Response for Row 803:\n",
      "Answer to Question 2: (deletion of gene FLO400, consequence prevents pseudohyphae formation), (deletion of gene FLO5-1, consequence prevents pseudohyphae formation), (deletion of gene FLO400, consequence hampers FLO11 expression), (deletion of gene FLO5-1, consequence hampers FLO11 expression), (expression of gene FLO400, consequence leads to induction of FLO11), (expression of gene FLO5-1, consequence leads to induction of FLO11), (chromatin modulation, consequence regulates expression of FLO400 and FLO5-1)\n",
      "Progress: 40.15% completed\n",
      "Response for Row 804:\n",
      "Answer to Question 2: (mitochondrial volume fraction increase, increase in mRNA localization to mitochondria), (mitochondrial mRNA localization increase, increase in protein production), (ribosome stalling, enhance protein synthesis by increasing mRNA localization to mitochondria)\n",
      "Progress: 40.20% completed\n",
      "Response for Row 805:\n",
      "Answer to Question 2: (deletion of gene xxx, failure in gene editing with previous methods), (expression of sgRNA, high efficiency CRISPR/Cas9 gene editing), (deletion of gene xxx, successful gene editing with new tRNA-sgRNA fusions), (integration of gene xxx, successful with new tRNA-sgRNA fusions)\n",
      "Progress: 40.25% completed\n",
      "Response for Row 806:\n",
      "Answer to Question 2: (deletion of miR365, improvement of PTOA symptoms), (engulfment of NPs-YCWP by macrophages, regulation of gene expression), (protection of miR365 antagomir/AAT by YCWP, prevention of degradation in gastrointestinal tract)\n",
      "Progress: 40.30% completed\n",
      "Response for Row 807:\n",
      "Answer to Question 2: (deletion of gene ZIP3, tolerance to toxic concentrations of manganese), (deletion of gene ZIP3, imbalanced expression of intracellular metal transporters), (deletion of gene ZIP3, higher sensitivity to reactive oxygen species), (deletion of gene ZIP3, alteration in the expression of ROS-detoxifying enzyme-coding genes), (deletion of gene ZIP3, decreased virulence), (deletion of gene ZIP3, decreased melanization), (deletion of gene ZIP3, decreased secretion of glucuronoxylomannan), (deletion of gene ZIP3, altered extracellular vesicle dimensions profile)\n",
      "Progress: 40.35% completed\n",
      "Response for Row 808:\n",
      "Answer to Question 2: (deletion of gene atsf1, consequence leaf variegation), (deletion of gene atsf1, consequence reduced accumulation of mature plastid ribosomal RNA), (mutation in gene atsf1, consequence hypersensitivity to coldness), (mutation in gene atsf1, consequence low splicing activity for introns with suboptimal BPS sequences), (deletion of gene atsf1, consequence upregulation of many genes), (deletion of gene atsf1, consequence downregulation of many genes), (deletion of gene atsf1, consequence insufficient intron splicing)\n",
      "Progress: 40.40% completed\n",
      "Response for Row 809:\n",
      "Answer to Question 2: (deletion of gene, increased fitness burden), (expression of gene, decreased fitness burden), (expression of gene, increased vanillin-β-glucoside production), (expression of gene, increased total vanillin-β-glucoside pathway metabolite accumulation)\n",
      "Progress: 40.45% completed\n",
      "Response for Row 810:\n",
      "Answer to Question 2: (overexpression of gene Calb, increased secretory production of Candida antarctica lipase B), (use of DHAS TT terminator, inducible CALB expression levels similar to AOX1 terminator), (use of DHAS TT terminator, threefold higher constitutive expression of CALB under GAP promoter), (enhanced transcript stability by correct choice of transcription terminators, increased transcription rate)\n",
      "Progress: 40.50% completed\n",
      "Response for Row 811:\n",
      "Answer to Question 2:  (overexpression of gene FHL1, consequence RNA content increased by 14.94%), (overexpression of gene SSF2, consequence RNA content increased by 24.4%), (deletion of gene HRP1, consequence RNA content increased by 19.36%), (overexpression of gene IFH1, consequence no significant change in RNA production), (overexpression of gene SSF1, consequence no significant change in RNA production), (overexpression of gene FHL1, consequence part of complex key to RP gene transcription), (overexpression of gene IFH1, consequence part of complex key to RP gene transcription), (overexpression of gene SSF2, consequence important role in ribosome biosynthesis), (deletion of gene HRP1, consequence removes negative regulator of cell growth), (overexpression of gene FHL1, overexpression of gene IFH1, overexpression of gene SSF2, deletion of gene HRP1, consequence RNA content increased by 38.8%), (overexpression of gene FHL1, overexpression of gene IFH1, overexpression of gene SSF2, deletion of gene HRP1, consequence increased cell size), (overexpression of gene FHL1, overexpression of gene IFH1, overexpression of gene SSF2, deletion of gene HRP1, consequence increased growth rate)\n",
      "Progress: 40.55% completed\n",
      "Response for Row 812:\n",
      "Answer to Question 2: (deletion of canonical start codon usage, production of alternative N-terminally extended protein isoforms), (overexpression of non-AUG start codon usage, increased during meiosis), (decrease in eIF5A, induction of non-AUG initiation)\n",
      "Progress: 40.60% completed\n",
      "Response for Row 813:\n",
      "Answer to Question 2: (deletion of gene PIGR, no difference in expression), (deletion of gene CD79A, no difference in expression), (overexpression of gene PIGR, higher secretory IgA concentrations in ileum), (overexpression of gene CD79A, higher secretory IgA concentrations in colon), (overexpression of gene PIGR, greater number of IgA-producing plasma cells in ileum), (overexpression of gene CD79A, greater number of IgA-producing plasma cells in colon)\n",
      "Progress: 40.65% completed\n",
      "Response for Row 814:\n",
      "Answer to Question 2: (expression of sidD gene, increase under iron limitation and presence of 600-800 μM heavy metal)\n",
      "Progress: 40.70% completed\n",
      "Response for Row 815:\n",
      "Answer to Question 2: (expression of gene btrC, production of 2-DOI), (expression of gene btrC, reduction of 2-DOI into scyllo-quercitol), (expression of gene btrC, reduction of 2-DOI into (-)-vibo-quercitol), (deletion of gene pgi1, inactive Pgi1), (deletion of gene zwf1, increased production of carbocyclic compounds), (deletion of gene pgi1, deletion of gene zwf1, trace production of compounds), (expression of gene btrC, assimilation of 2-DOI by S. cerevisiae)\n",
      "Progress: 40.75% completed\n",
      "Response for Row 816:\n",
      "Answer to Question 2:  (expression of gene SRS pathway, consequence H2S production), (expression of gene Sulfur metabolism, consequence H2S production), (expression of gene Glycine, serine and threonine metabolism, consequence H2S production), (expression of gene Biosynthesis of amino acids, consequence H2S production), (overexpression of gene sulfate reduction sequence (SRS) pathway, consequence increased H2S production), (deletion of gene sulfate reduction sequence (SRS) pathway, consequence decreased H2S production)\n",
      "Progress: 40.80% completed\n",
      "Response for Row 817:\n",
      "Answer to Question 2: (overexpression of gene HAC1, improvement in xylanase production), (integration of gene xynB, constitutive secretory expression of xylanase), (overexpression of gene HAC1, increase in yield of xylanase to 381 U/mL)\n",
      "Progress: 40.85% completed\n",
      "Response for Row 818:\n",
      "Answer to Question 2: (deletion of gene RIM15, enhanced biofilm formation), (deletion of gene RIM15, improved fermentation ability), (deletion of gene RIM15, increased expression of adhesion protein FLO11), (deletion of gene RIM15, synthesis of trehalose)\n",
      "Progress: 40.90% completed\n",
      "Response for Row 819:\n",
      "Answer to Question 2: (deletion of gene GT1, impaired zinc uptake activity), (deletion of gene ZRT1, impaired zinc uptake activity), (zinc deficiency, upregulation of gene ZRT1), (block of Gt1 function, upregulation of gene ZRT1)\n",
      "Progress: 40.95% completed\n",
      "Response for Row 820:\n",
      "Answer to Question 2: (deletion of transaminase gene, consequence: potential loss of transamination activity), (overexpression of transgene, consequence: increased biomass up to 60% wet cell weight by volume)\n",
      "Progress: 41.00% completed\n",
      "Response for Row 821:\n",
      "Answer to Question 2: (overexpression of a key protein, increase in alcoholic fermentation rate), (modification of a specific domain or amino acid, increase in ethanol tolerance)\n",
      "Progress: 41.05% completed\n",
      "Response for Row 822:\n",
      "Answer to Question 2: (heterologous expression of NDH, increased cellular sensitivity to furfural), (overexpression of GLR1, enhanced furfural tolerance), (overexpression of OYE2, enhanced furfural tolerance), (overexpression of ZWF1, enhanced furfural tolerance), (overexpression of IDP1, enhanced furfural tolerance), (expression of PNTB, little impact on furfural tolerance), (expression of POS5, little impact on furfural tolerance)\n",
      "Progress: 41.10% completed\n",
      "Response for Row 823:\n",
      "Answer to Question 2: (expression of Rad51-NTD, high-level production of native Rad51), (expression of Rad51-NTD, increased half-life of Rad51), (phosphorylation by Mec1/Tel1, antagonizes proteasomal degradation pathway)\n",
      "Progress: 41.15% completed\n",
      "Response for Row 824:\n",
      "Answer to Question 2: (upregulation of acid-responsive genes involved in ergosterol synthesis, increased acid tolerance), (upregulation of acid-responsive genes involved in ferrous iron uptake, increased acid tolerance), (upregulation of genes for fatty acid ethyl ester synthesis, enhanced aroma compound formation), (upregulation of genes for aromatic amino acid biosynthesis, enhanced aroma compound formation)\n",
      "Progress: 41.20% completed\n",
      "Response for Row 825:\n",
      "Answer to Question 2: (deletion of gene ARO10, reduction in benzyl alcohol production)\n",
      "Progress: 41.25% completed\n",
      "Response for Row 826:\n",
      "Answer to Question 2: (deletion of gene RPH1, consequence alleviates cell slow growth), (deletion of gene RPH1, consequence derepression of rRNA transcription), (deletion of gene RPH1, consequence derepression of RPG transcription), (phosphorylation of gene RPH1, consequence more resistance to rapamycin treatment), (phosphorylation of gene RPH1, consequence decreased association with ribosome-related genes), (phosphorylation of gene RPH1, consequence faster cell growth)\n",
      "Progress: 41.30% completed\n",
      "Response for Row 827:\n",
      "Answer to Question 2: (deletion of DH1 domain, complete loss of activity in the biosynthesis of VLCPUFAs), (mutation of DH1 domain, complete loss of activity in the biosynthesis of VLCPUFAs), (deletion of DH2 domain, production of a small amount of DPA), (mutation of DH2 domain, production of a small amount of DPA), (substitution of DH1 with DH2, complete loss of activity in the biosynthesis of VLCPUFAs), (expression of mutated DH2, production of a small amount of DPA), (expression of DH1, essential for the biosynthesis of VLCPUFAs), (expression of DH2, required for the biosynthesis of DHA)\n",
      "Progress: 41.35% completed\n",
      "Response for Row 828:\n",
      "Answer to Question 2: (overexpression of genes from the NA strain, higher growth efficiency under nitrogen-limited conditions), (overexpression of ECM38 allele, higher growth efficiency under nitrogen-limited conditions)\n",
      "Progress: 41.40% completed\n",
      "Response for Row 829:\n",
      "Answer to Question 2: (expression of gene PvFAD2, consequence increase in LA production), (expression of gene PvFAD3, consequence increase in ALA production)\n",
      "Progress: 41.45% completed\n",
      "Response for Row 830:\n",
      "Answer to Question 2: (deletion of SSU1, confirmation of role in sulphite tolerance)\n",
      "Progress: 41.50% completed\n",
      "Response for Row 831:\n",
      "Answer to Question 2: (deletion of gene ChrR1, increase in carotenoid production), (overexpression of gene crtE1, no production of geranyl pyrophosphate), (overexpression of gene crtE2, no production of geranyl pyrophosphate), (expression of gene GPPS, production of geraniol), (expression of gene GES, production of geraniol), (expression of gene ads, production of amorphadiene)\n",
      "Progress: 41.55% completed\n",
      "Response for Row 832:\n",
      "Answer to Question 2: (deletion of gene segments, alteration of chemical product formation), (overexpression of mutated genes, identification of important determinants for successful mutagenesis)\n",
      "Progress: 41.60% completed\n",
      "Response for Row 833:\n",
      "Answer to Question 2: (no gene overexpression/deletion mentioned, no specific consequences related to gene overexpression/deletion mentioned)\n",
      "Progress: 41.65% completed\n",
      "Response for Row 834:\n",
      "Answer to Question 2: (overexpression of gene SUT2, promotes expression of AOX1), (overexpression of gene SUT2, increases ergosterol content), (expression of gene SUT2, induced by methanol), (expression of gene SUT2, repressed by glycerol), (P_SUT2-driven expression of gene MXR1, elevates expression level of Mxr1), (P_SUT2-driven expression of gene MXR1, high activity of P_AOX1), (P_AOX1-driven expression of gene enhanced green fluorescent protein, improved by 18%)\n",
      "Progress: 41.70% completed\n",
      "Response for Row 835:\n",
      "Answer to Question 2: (genetic encoding of biosensors, dynamic response), (genetic encoding of biosensors, sensitivity), (genetic encoding of biosensors, operational range), (regulatory architecture modification, dynamic assembly), (protein interaction alteration, regulatory function), (genetic element modification, interaction efficiency)\n",
      "Progress: 41.75% completed\n",
      "Response for Row 836:\n",
      "Answer to Question 2: (deletion of gene TRF5, increases RNA abundance for mRNAs specifically showing high Trf5 binding)\n",
      "Progress: 41.80% completed\n",
      "Response for Row 837:\n",
      "Answer to Question 2: (deletion of gene CgHHF1, MMS resistance), (deletion of gene CgHHF2, MMS resistance), (deletion of gene CgHHF1, higher rate of homologous recombination), (deletion of gene CgHHF2, higher rate of homologous recombination), (reduction in histone H4 protein levels, stress resistance)\n",
      "Progress: 41.85% completed\n",
      "Response for Row 838:\n",
      "Answer to Question 2: (fusion of FAS subunits, increased FAS activity), (fusion of FAS subunits, improved fatty acid production)\n",
      "Progress: 41.90% completed\n",
      "Response for Row 839:\n",
      "Answer to Question 2: (deletion of gene whi2, consequence competitive superiority), (deletion of gene psr1psr2, consequence competitive superiority), (deletion of gene whi2, consequence production of diffusible compound), (deletion of gene psr1psr2, consequence production of diffusible compound), (deletion of gene whi2, consequence resistance to diffusible compound), (deletion of gene psr1psr2, consequence resistance to diffusible compound)\n",
      "Progress: 41.95% completed\n",
      "Response for Row 840:\n",
      "Answer to Question 2: (no expression/deletion of gene mentioned, focus on expression systems for membrane proteins)\n",
      "Progress: 42.00% completed\n",
      "Response for Row 841:\n",
      "Answer to Question 2: (deletion of gene YJL175W, reduction in transcription of genes encoding highly expressed, secreted proteins), (deletion of gene YJL175W, overall reduction in translation)\n",
      "Progress: 42.05% completed\n",
      "Response for Row 842:\n",
      "Answer to Question 2: (up-regulation of genes required for ergosterol biosynthesis, increased levels of ergosteryl esters), (aeration with microbubbles, increased O2 transfer to the propagation cultures)\n",
      "Progress: 42.10% completed\n",
      "Response for Row 843:\n",
      "Answer to Question 2: (expression of gene HXK2S14A, increased xylose consumption rate), (expression of gene HXK2S14A, increased ethanol production rate), (expression of gene HXK2S14A, increased ethanol yield), (deletion of gene MIG1, decreased xylose utilization), (deletion of gene MIG1, eliminated positive effect of Hxk2p)\n",
      "Progress: 42.15% completed\n",
      "Response for Row 844:\n",
      "Answer to Question 2: (expression of MET5, decrease in CFU values), (expression of MET10, decrease in CFU values), (overexpression of sulfite reductase, decrease in CFU values), (decrease in selenium concentration, negative effect on cell population), (increase in sulfite concentration, positive effect on cell population), (production of selenium sulfide nanoparticles, drop in S. cerevisiae cell population)\n",
      "Progress: 42.20% completed\n",
      "Response for Row 845:\n",
      "Answer to Question 2: (deletion of gene Bud27, alteration of phosphorylation kinetics of ribosomal protein S6), (deletion of gene Bud27, impact on homeostasis of ribosome biogenesis process), (deletion of gene Bud27, modulation of transcription mediated by RNA polymerase I), (deletion of gene Bud27, influence on transcription of RNA polymerase III), (deletion of gene Bud27, regulation of RNA polymerase II-dependent transcription of ribosomal proteins), (deletion of gene Bud27, regulation of ribosome biogenesis regulon genes)\n",
      "Progress: 42.25% completed\n",
      "Response for Row 846:\n",
      "Answer to Question 2: (deletion of gene XRN1, impaired gene silencing), (deletion of gene XRN1, affecting the composition of siRNA species), (deletion of gene XRN1, influencing the efficiency of siRNA loading into Argonaute), (deletion of gene XRN1, degradation of cleaved passenger strand), (deletion of gene XRN1, degradation of sliced target RNA)\n",
      "Progress: 42.30% completed\n",
      "Response for Row 847:\n",
      "Answer to Question 2: (osw2 Δ, enhanced catalytic activity), (osw2 Δ, excellent reusability), (osw2 Δ, thermostability), (osw2 Δ, tolerance to various harsh conditions)\n",
      "Progress: 42.35% completed\n",
      "Response for Row 848:\n",
      "Answer to Question 2: (deletion of uORF3, consequence 1: up-regulation of HNT1 translation), (overexpression of uORF3, consequence 1: down-regulation of HNT1 translation)\n",
      "Progress: 42.40% completed\n",
      "Response for Row 849:\n",
      "Answer to Question 2: (expression of SbDGAT1-1, restored TAG accumulation capability), (expression of SbDGAT1-2, restored TAG accumulation capability), (deletion of N-terminal region in SbDGAT1-1, decreased TAG accumulation), (deletion of N-terminal region in SbDGAT1-1, decreased FA accumulation)\n",
      "Progress: 42.45% completed\n",
      "Response for Row 850:\n",
      "Answer to Question 2: (expression of gene TDH1, increased promoter strength), (expression of gene PGK1, increased promoter strength), (expression of gene INO1, increased promoter strength), (expression of gene SED1, increased promoter strength), (expression of gene CCW12, increased promoter strength), (expression of gene TEF1, weaker promoter strength compared to 15 other promoters), (expression of multiple genes, optimized biosynthesis pathway of (2S)-naringenin)\n",
      "Progress: 42.50% completed\n",
      "Response for Row 851:\n",
      "Answer to Question 2: (deletion of glucose, consequence central metabolites drastically deplete), (deletion of glucose, consequence intracellular AMP-to-ATP ratio strongly increases), (activation of bulk autophagy, consequence frees amino acids), (activation of β-oxidation, consequence lipid degradation), (activation of bulk autophagy, consequence ensures long-term survival during starvation), (activation of β-oxidation, consequence ensures long-term survival during starvation)\n",
      "Progress: 42.55% completed\n",
      "Response for Row 852:\n",
      "Answer to Question 2: (deletion of gene isc10, reduced efficiency of spore differentiation), (overexpression of gene ssp2 before MII, reduced efficiency of spore differentiation), (deletion of gene ama1, Isc10 not degraded), (activation of gene smk1 by ssp2, triggered by degradation of Isc10)\n",
      "Progress: 42.60% completed\n",
      "Response for Row 853:\n",
      "Answer to Question 2: (expression of gene CYP505D13, enhanced ST-3 production), (expression of gene CYP505D13, increased ST-3 titer to 78.61 mg/L), (expression of gene CYP505D13, improved bioproduction efficiency 6.43 times)\n",
      "Progress: 42.65% completed\n",
      "Response for Row 854:\n",
      "Answer to Question 2: (deletion of gene Aim23p, quantitative imbalance of mitochondrial translation products), (deletion of gene Aim23p, increased amount of Pet111p), (interaction of Aim23p with Pet111p, ensures proper COX2 mRNA translation)\n",
      "Progress: 42.70% completed\n",
      "Response for Row 855:\n",
      "Answer to Question 2: (deletion of gene Vta1, consequence colorless microsclerotia), (deletion of gene Vta1, consequence no melanization), (activation of gene Vta1, consequence melanin production), (activation of gene Vta1, consequence transcription of PKS1), (activation of gene Vta1, consequence transcription of LAC1)\n",
      "Progress: 42.75% completed\n",
      "Response for Row 856:\n",
      "Answer to Question 2: (deletion of DNA polymerase alpha/primase, disruption of Okazaki fragment initiation), (deletion of DNA polymerase alpha/primase, disruption of replication origin firing), (deletion of Ctf4, impairment of lagging-strand initiation), (severe depletion of DNA polymerase alpha/primase, activation of the checkpoint essential for viability)\n",
      "Progress: 42.80% completed\n",
      "Response for Row 857:\n",
      "Answer to Question 2: (deletion of gene products, consequence old daughters grow more slowly), (deletion of gene products, consequence old daughters age), (deletion of gene products, consequence old daughters have lower fluorescence levels)\n",
      "Progress: 42.85% completed\n",
      "Response for Row 858:\n",
      "Answer to Question 2: (expression of envelope domain III of dengue virus serotype-1, increased secretory expression), (expression of envelope domain III of dengue virus serotype-1, improved screening using immunoassay method)\n",
      "Progress: 42.90% completed\n",
      "Response for Row 859:\n",
      "Answer to Question 2: (deletion of Ded1p function, represses translation of housekeeping mRNAs), (deletion of Ded1p function, promotes translation of stress mRNAs)\n",
      "Progress: 42.95% completed\n",
      "Response for Row 860:\n",
      "Answer to Question 2:  (expression of piggyBac transposase, long-term gene expression), (mutation of piggyBac transposase, higher transpose activity), (expression of bz-hyPBase, high-efficiency editing in CHO cells), (expression of bz-hyPBase, high-efficiency editing in T cells)\n",
      "Progress: 43.00% completed\n",
      "Response for Row 861:\n",
      "Answer to Question 2: (deletion of gene SET1, increase in serine-2 phosphorylation within the C-terminal domain of RNA polymerase II), (deletion of gene SET1, increase in recruitment of the cleavage/polyadenylation complex), (deletion of gene SET2, increase in serine-2 phosphorylation within the C-terminal domain of RNA polymerase II), (deletion of gene SET2, increase in recruitment of the cleavage/polyadenylation complex), (chemical inhibition of TOR signaling, Set1-dependent alternative polyadenylation), (chemical inhibition of TOR signaling, Set2-dependent alternative polyadenylation), (expression of gene SET1, decrease in efficiency of using single pA sites), (expression of gene SET2, decrease in efficiency of using single pA sites), (expression of gene SET1, control nucleosome occupancy around pA sites), (expression of gene SET2, control nucleosome occupancy around pA sites)\n",
      "Progress: 43.05% completed\n",
      "Response for Row 862:\n",
      "Answer to Question 2: (expression of gene involved in oxidative stress, consequence: enhanced oxidative stress response in blastospores), (expression of gene involved in amino acid metabolism, consequence: increased amino acid metabolism in blastospores), (expression of gene involved in respiration processes, consequence: enhanced respiration in blastospores), (expression of gene involved in transmembrane transport, consequence: improved transmembrane transport in blastospores), (expression of gene involved in production of secondary metabolites, consequence: increased production of secondary metabolites in blastospores), (expression of gene involved in growth, consequence: enhanced growth in hyphae), (expression of gene involved in metabolism, consequence: increased metabolism in hyphae), (expression of gene involved in cell wall re-organization, consequence: altered cell wall structure in hyphae)\n",
      "Progress: 43.10% completed\n",
      "Response for Row 863:\n",
      "Answer to Question 2: (deletion of Fld1, LD biogenesis impaired), (deletion of Nem1, LD biogenesis impaired), (overexpression of Lro1, increased LD formation), (overexpression of Dga1, increased LD formation), (deletion of Fld1, reduced recruitment of TAG-synthases), (deletion of Nem1, reduced recruitment of TAG-synthases), (presence of Fld1, increased DAG production at ER subdomains), (presence of Nem1, increased DAG production at ER subdomains)\n",
      "Progress: 43.15% completed\n",
      "Response for Row 864:\n",
      "Answer to Question 2: (gene expression of stress tolerance genes, consequence 1), (gene expression of growth genes, consequence 2), (gene expression of metabolic genes, consequence 3)\n",
      "Progress: 43.20% completed\n",
      "Response for Row 865:\n",
      "Answer to Question 2: (expression of gene Cys3, upregulation), (expression of gene Cys4, upregulation), (expression of gene MET17, upregulation), (expression of gene IRC7, upregulation), (expression of gene IRC7, increased 4-MMP release)\n",
      "Progress: 43.25% completed\n",
      "Response for Row 866:\n",
      "Answer to Question 2: (deletion of gene AtSFC1, reduced seed germination), (deletion of gene AtSFC1, impaired radicle growth), (deletion of gene AtSFC1, reduced respiration rate)\n",
      "Progress: 43.30% completed\n",
      "Response for Row 867:\n",
      "Answer to Question 2: (expression of gene SNZ2, increased intracellular vitamin B6 and thiamine), (expression of gene THI6, increased intracellular vitamin B6 and thiamine), (expression of gene THI21, increased intracellular vitamin B6 and thiamine), (expression of gene THI80, increased intracellular vitamin B6 and thiamine), (proanthocyanidins treatment, increased cell viability), (proanthocyanidins treatment, increased unsaturated fatty acids content), (proanthocyanidins treatment, decreased saturated fatty acids content)\n",
      "Progress: 43.35% completed\n",
      "Response for Row 868:\n",
      "Answer to Question 2: (upregulation of 17 ribosomal proteins, activation of reserve translational capacity)\n",
      "Progress: 43.40% completed\n",
      "Response for Row 869:\n",
      "Answer to Question 2: (heterologous expression of sugar transporters, utilization of cellodextrins), (intracellular expressed β-glucosidase, utilization of cellodextrins), (expression of cellodextrin transporters from Aspergillus niger, transport of cellobiose with low affinity), (expression of Ls120451 from Lipomyces starkeyi, efficient growth on cellobiose), (expression of Ls120451, growth on glucose), (expression of Ls120451, growth on lactose), (expression of Ls120451, no growth on cellotriose), (expression of Ls120451, no growth on cellotetraose), (expression of CdtG from Penicillium oxalicum, consumption of cellobiose at concentrations below 1.0 g/L more efficiently than Ls120451), (expression of Ls120451, consumption of cellobiose at higher concentrations more efficiently than CdtG), (expression of Ls120451, production of up to 30% more ethanol than CdtG)\n",
      "Progress: 43.45% completed\n",
      "Response for Row 870:\n",
      "Answer to Question 2: (deletion of gene glycolysis, consequence repression of glycolytic pathway), (overexpression of gene FLO11, consequence enhanced cell adhesion), (deletion of gene ion transporters, consequence reduced ion transportation), (overexpression of gene oxidative carbon metabolism, consequence increased metabolic activity), (overexpression of gene high affinity transport of sugars, consequence enhanced sugar uptake), (overexpression of gene glycerol utilization, consequence increased glycerol breakdown), (overexpression of gene sulfur metabolism, consequence enhanced sulfur compound processing), (overexpression of gene protein quality control and recycling, consequence improved protein maintenance), (overexpression of gene cell wall biogenesis, consequence strengthened cell wall), (overexpression of gene apoptosis, consequence regulated cell death)\n",
      "Progress: 43.50% completed\n",
      "Response for Row 871:\n",
      "Answer to Question 2: (conformational fluctuation of host soluble prion protein, regulates efficiency of cross-species prion transmission), (specific conformation of short disordered segment, governs interspecies prion transmissibility), (critical residues within segment, alter dynamics and conformation of soluble prion proteins), (methylene group difference between asparagine and glutamine, changes short segment structure), (short segment structure change, modulates cross-seeding activity)\n",
      "Progress: 43.55% completed\n",
      "Response for Row 872:\n",
      "Answer to Question 2: (overexpression of synapsin-1, improved viability of PC12 cells by up to 71.17%), (downregulation of septin-5, improved viability of PC12 cells by up to 67.00%), (overexpression of alcohol dehydrogenase-6, led to 22.38- and 12.33-fold decreases in the production of MDA and H₂O₂, respectively), (overexpression of alcohol dehydrogenase-6, led to 7.15-, 13.93-, 12.08-, and 8.11-fold improvements in the activity of catalase, glutathione reductase, superoxide dismutase, and glutathione peroxidase, respectively)\n",
      "Progress: 43.60% completed\n",
      "Response for Row 873:\n",
      "Answer to Question 2:  (addition of xylose, transcriptional activation of target genes), (addition of xylose, activation of mcherry), (addition of xylose, activation of xylose utilization gene)\n",
      "Progress: 43.65% completed\n",
      "Response for Row 874:\n",
      "Answer to Question 2: (deletion of gene GID4, accumulation of Mdh2), (deletion of gene GID4, high malate production), (deletion of gene GID1, high malate production), (deletion of gene GID2, high malate production), (deletion of gene GID3, high malate production), (deletion of gene GID5, high malate production), (deletion of gene GID8, high malate production), (deletion of gene GID9, high malate production)\n",
      "Progress: 43.70% completed\n",
      "Response for Row 875:\n",
      "Answer to Question 2: (co-adaptation index increase, positive correlation with protein abundance), (co-adaptation index increase, positive correlation with translation rate), (high co-AI, positive correlation with mRNA secondary structure stability), (high co-AI, positive correlation with mRNA half-life), (highly expressed proteins encoded by high co-AI genes, assisted by molecular chaperones to attain proper functional conformation), (high co-AI, prevention of protein accumulation by molecular chaperones)\n",
      "Progress: 43.75% completed\n",
      "Response for Row 876:\n",
      "Answer to Question 2: (expression of gene ADH4, increased response to MMP), (overexpression of gene ADH4, increased methionol production in yeast strain S4), (induced expression of gene ADH4 in E. coli, preparation of 42 kDa Adh4p), (catalysis by Adh4p, conversion of MMP to methionol)\n",
      "Progress: 43.80% completed\n",
      "Response for Row 877:\n",
      "Answer to Question 2:  (overexpression of genes associated with potassium synthesis, increased tolerance to acetic acid), (overexpression of genes associated with iron synthesis, increased tolerance to acetic acid), (overexpression of genes associated with zinc synthesis, increased tolerance to acetic acid), (overexpression of genes associated with glutathione synthesis, increased tolerance to acetic acid), (upregulation of glycolysis, provision of nicotinamide adenine dinucleotide), (upregulation of TCA cycle, provision of nicotinamide adenine dinucleotide), (upregulation of chaperones, enhanced protein stability), (upregulation of coenzyme synthesis, improved metabolic function), (upregulation of energy production pathways, increased cellular energy), (upregulation of energy transformation pathways, enhanced stress response)\n",
      "Progress: 43.85% completed\n",
      "Response for Row 878:\n",
      "Answer to Question 2: (deletion of gene not applicable, expression of Braz in S. cerevisiae, higher solubility), (deletion of gene not applicable, expression of Braz in S. cerevisiae, increased safety (GRAS)), (deletion of gene not applicable, expression of Braz in E. coli, higher yield of GST-Braz), (deletion of gene not applicable, expression of Braz in E. coli, lower solubility compared to S. cerevisiae)\n",
      "Progress: 43.90% completed\n",
      "Response for Row 879:\n",
      "Answer to Question 2: (no gene expression/deletion mentioned, review of NR biosynthesis mechanisms)\n",
      "Progress: 43.95% completed\n",
      "Response for Row 880:\n",
      "Answer to Question 2: (expression of gene rimA, increase in rimocidin production), (expression of gene rimB, increase in rimocidin production), (expression of gene rimC, increase in rimocidin production), (expression of gene rimD, increase in rimocidin production), (expression of gene rimE, increase in rimocidin production), (expression of gene rimF, increase in rimocidin production), (expression of gene rimG, increase in rimocidin production), (expression of gene rimH, increase in rimocidin production), (expression of gene rimI, increase in rimocidin production), (expression of gene rimJ, increase in rimocidin production), (expression of gene rimK, increase in rimocidin production)\n",
      "Progress: 44.00% completed\n",
      "Response for Row 881:\n",
      "Answer to Question 2:  (over-expression of gene DUR3, increased utilization of urea), (over-expression of gene DUR3, reduced urea concentration by 92.0%), (over-expression of gene DUR3, reduced EC concentration by 58.5%)\n",
      "Progress: 44.05% completed\n",
      "Response for Row 882:\n",
      "Answer to Question 2: (no specific gene expression/deletion mentioned, synchronization of protein synthesis with cell division), (no specific gene expression/deletion mentioned, synchronization of glycolysis with cell division), (no specific gene expression/deletion mentioned, synchronization of amino acid metabolism with cell division), (no specific gene expression/deletion mentioned, increase in mitochondrial volume)\n",
      "Progress: 44.10% completed\n",
      "Response for Row 883:\n",
      "Answer to Question 2: (deletion of gene not mentioned, consequence not applicable), (overexpression of xylose reductase gene from Neurospora crassa controlled by PCgcwp1, xylitol production increased by 204%), (overexpression of xylose reductase gene from Neurospora crassa controlled by PCgaac, medium-high xylitol production at 42°C), (overexpression of xylose reductase gene from Neurospora crassa controlled by PCgpot1, medium-high xylitol production at 42°C), (overexpression of xylose reductase gene from Neurospora crassa controlled by PCghsp12, modest xylitol production at 42°C), (overexpression of xylose reductase gene from Neurospora crassa controlled by PCgatp1, modest xylitol production at 42°C), (overexpression of xylose reductase gene from Neurospora crassa controlled by PCgino1, weaker xylitol production at 42°C), (overexpression of xylose reductase gene from Neurospora crassa controlled by PCgscl, weaker xylitol production at 42°C)\n",
      "Progress: 44.15% completed\n",
      "Response for Row 884:\n",
      "Answer to Question 2: (mutations in gene KARS, affect cytosolic LysRS level), (mutations in gene KARS, affect cytosolic tRNA aminoacylation in vitro), (mutation c.517T>C in gene KARS, deficient in charging mitochondrial tRNA<sup>Lys</sup> in vitro), (mutation c.517T>C in gene KARS, deficient in charging mitochondrial tRNA<sup>Lys</sup> in vivo)\n",
      "Progress: 44.20% completed\n",
      "Response for Row 885:\n",
      "Answer to Question 2: (down-regulation of nuclear factor-kappa B, reduction in neutrophil infiltration), (down-regulation of nuclear factor-kappa B, reduction in production of proinflammatory cytokines), (up-regulation of Zonna occludens-1, increase in intestinal barrier integrity), (up-regulation of Zonna occludens-2, increase in intestinal barrier integrity), (up-regulation of Claudin-1, increase in intestinal barrier integrity), (up-regulation of Occludin, increase in intestinal barrier integrity), (up-regulation of Mucin-1, increase in intestinal barrier integrity), (up-regulation of Mucin-2, increase in intestinal barrier integrity)\n",
      "Progress: 44.25% completed\n",
      "Response for Row 886:\n",
      "Answer to Question 2: (gene expression regulation by promoters, improved activity of displayed proteins), (gene expression regulation by promoters, improved specificity of displayed proteins), (gene expression regulation by promoters, improved stability of displayed proteins), (secretion signals affecting secretion efficiency, improved activity of displayed proteins), (secretion signals affecting secretion efficiency, improved specificity of displayed proteins), (secretion signals affecting secretion efficiency, improved stability of displayed proteins), (length of anchor proteins affecting accessibility, improved activity of displayed proteins), (length of anchor proteins affecting accessibility, improved specificity of displayed proteins), (length of anchor proteins affecting accessibility, improved stability of displayed proteins)\n",
      "Progress: 44.30% completed\n",
      "Response for Row 887:\n",
      "Answer to Question 2: (recombinant expression of CGA-N12 in Pichia pastoris GS115, direct secretion into culture media), (recombinant expression of CGA-N12 in Pichia pastoris GS115, enhanced expression effect), (expression of CGA-N12, recognition and excision by endogenous protease Kex2), (expression of CGA-N12, recognition and excision by endogenous protease Ste13), (expression of CGA-N12, release of short CGA-N12 peptides from tandem repeat sequences), (expression of CGA-N12, yield of up to 30 mg/L of yeast culture)\n",
      "Progress: 44.35% completed\n",
      "Response for Row 888:\n",
      "Answer to Question 2: (loss of specificity between ncRNA-mRNA interaction profiles, repression of ncRNA affinities with mRNAs)\n",
      "Progress: 44.40% completed\n",
      "Response for Row 889:\n",
      "Answer to Question 2: (heterologous gene expression, improving ginsenoside production)\n",
      "Progress: 44.45% completed\n",
      "Response for Row 890:\n",
      "Answer to Question 2: (deletion of gene RPN4, increased sensitivity to ethanol), (deletion of gene RPN4, no increase in autophagy activity upon ethanol stress), (deregulation of PRE1 leading to Rpn4 accumulation, increased PRB1 expression upon ethanol stress), (stabilization of Rpn4, contribution to PRB1 upregulation), (repression of Rpn4 binding sites in PRB1 promoter, inhibition of PRB1 induction and hypersensitivity to ethanol)\n",
      "Progress: 44.50% completed\n",
      "Response for Row 891:\n",
      "Answer to Question 2: (heterologous expression of Clostridium pathway, increased butanol production), (modification of amino acid uptake pathways, increased butanol production)\n",
      "Progress: 44.55% completed\n",
      "Response for Row 892:\n",
      "Answer to Question 2: (expression of COX1, restricted synthesis by nuclear-encoded translational activator), (expression of ATP synthase subunits, regulated by F1 module), (expression of Cox6, part of high-molecular-weight complexes affecting ratio of COX and ATP synthase), (expression of Atp9, part of high-molecular-weight complexes affecting ratio of COX and ATP synthase)\n",
      "Progress: 44.60% completed\n",
      "Response for Row 893:\n",
      "Answer to Question 2: (expression of ShmA, production of preshimalactone), (expression of ShmA and ShmB, production of shimalactones A and B)\n",
      "Progress: 44.65% completed\n",
      "Response for Row 894:\n",
      "Answer to Question 2: (expression of gene lipoxygenase, consequence (R)-(-)-1-octen-3-ol biosynthesis), (expression of gene peroxide lyase, consequence (R)-(-)-1-octen-3-ol biosynthesis)\n",
      "Progress: 44.70% completed\n",
      "Response for Row 895:\n",
      "Answer to Question 2: (deletion of gene Sen1, consequence 1: increased pervasive transcription), (expression of gene Sen1, consequence 1: termination of non-coding transcription), (interaction between Sen1 and CTD, consequence 1: enhanced release of paused RNAPII from DNA)\n",
      "Progress: 44.75% completed\n",
      "Response for Row 896:\n",
      "Answer to Question 2: (deletion of gene AaBRE1, loss of H2B monoubiquitination), (deletion of gene AaBRE1, decrease in H3K4me3 enrichment), (deletion of gene AaBRE1, suppression of genes encoding C2H2 zinc-finger family transcription factors), (deletion of gene AaBRE1, suppression of genes encoding cell wall-degrading enzymes), (deletion of gene AaBRE1, suppression of genes encoding chitin-binding proteins), (deletion of gene AaBRE1, reduction in hyphal growth), (deletion of gene AaBRE1, reduction in conidial formation), (deletion of gene AaBRE1, reduction in pathogenicity)\n",
      "Progress: 44.80% completed\n",
      "Response for Row 897:\n",
      "Answer to Question 2: (idle design of gene regulation, minimized crosstalk), (excess use of regulation, high crosstalk levels), (positive regulation of frequently expressed genes, increased crosstalk), (negative regulation of infrequently expressed genes, increased crosstalk), (default unregulated state of genes in idle design, frequently required activity state)\n",
      "Progress: 44.85% completed\n",
      "Response for Row 898:\n",
      "Answer to Question 2: (expression of CD gene, enhanced cytotoxicity), (expression of NTR gene, enhanced cytotoxicity), (expression of NTR gene, suppression of tumor growth), (expression of NTR gene, prolonged survival), (administration of 5-fluorocytosine, enhanced cytotoxicity), (administration of CB1954, suppression of tumor growth), (administration of CB1954, prolonged survival)\n",
      "Progress: 44.90% completed\n",
      "Response for Row 899:\n",
      "Answer to Question 2: (deletion of gene Aoglo3, growth defects), (deletion of gene Aoglo3, increase in hyphal septum), (deletion of gene Aoglo3, decrease in sporulation capacity), (deletion of gene Aoglo3, change in spore shape), (deletion of gene Aoglo3, decrease in spore germination rate), (deletion of gene Aoglo3, transcriptional repression of sporulation-related genes), (deletion of gene Aoglo3, sensitivity to chemical stressors), (deletion of gene Aoglo3, involvement in endocytosis), (deletion of gene Aoglo3, increase in myelin figures), (deletion of gene Aoglo3, increase in autophagosomes), (deletion of gene Aoglo3, affect production of mycelial traps), (deletion of gene Aoglo3, affect production of serine proteases)\n",
      "Progress: 44.95% completed\n",
      "Response for Row 900:\n",
      "Answer to Question 2: (expression of gene TBRT, consequence enhanced activities of Bap2 and Tat1), (expression of gene TBRT, consequence improved growth), (deletion of gene TBRT, consequence decreased transcription of TAT1), (activity of TOR signaling pathway, consequence repressed expression of BAP2 and TAT1), (activity of TOR signaling pathway, consequence induced expression of TBRT)\n",
      "Progress: 45.00% completed\n",
      "Response for Row 901:\n",
      "Answer to Question 2: (co-expression of gene SchTPS1, production of (-)-rotundone), (co-expression of gene VvSTO2, production of (-)-rotundone)\n",
      "Progress: 45.05% completed\n",
      "Response for Row 902:\n",
      "Answer to Question 2:  (overexpression of gene TtASR1, increase in E. coli viability under heat stress), (overexpression of gene TtASR1, increase in E. coli viability under cold stress), (overexpression of gene TtASR1, increase in Saccharomyces cerevisiae tolerance under salt stress), (overexpression of gene TtASR1, increase in Saccharomyces cerevisiae tolerance under osmotic stress), (upregulation of gene TtASR1 by ABA, response to abiotic stress), (upregulation of gene TtASR1 by PEG, response to abiotic stress), (upregulation of gene TtASR1 by salt stress in Mahmoudi roots, associated with salt tolerance), (downregulation of gene TtASR1 by salt stress in Azizi roots, associated with salt sensitivity)\n",
      "Progress: 45.10% completed\n",
      "Response for Row 903:\n",
      "Answer to Question 2: (deletion of sirtuins, affected relevant phenotypes in K. lactis), (deletion of sirtuins, no effect on phenotypes in S. cerevisiae)\n",
      "Progress: 45.15% completed\n",
      "Response for Row 904:\n",
      "Answer to Question 2: (deletion of gene Maf1, consequence 1: sequesters Pol III elements involved in transcription initiation), (deletion of gene Maf1, consequence 2: binds the mobile C34 winged helix 2 domain, sealing off the active site), (deletion of gene Maf1, consequence 3: Maf1 binding site overlaps with that of TFIIIB in the preinitiation complex)\n",
      "Progress: 45.20% completed\n",
      "Response for Row 905:\n",
      "Answer to Question 2: (deletion of gene involved in metabolism, increased sensitivity to SDS), (deletion of gene involved in cell cycle, increased sensitivity to SDS), (deletion of gene involved in DNA processing, increased sensitivity to SDS), (deletion of gene involved in cellular transport, increased sensitivity to SDS), (deletion of gene involved in transport facilities, increased sensitivity to SDS), (deletion of gene involved in transport routes, increased sensitivity to SDS), (deletion of gene involved in transcription, increased sensitivity to SDS), (deletion of gene involved in protein with binding function or cofactor requirement, increased sensitivity to SDS), (deletion of gene, increased intracellular ROS levels), (up-regulation of antioxidant defense genes, response to oxidative damage)\n",
      "Progress: 45.25% completed\n",
      "Response for Row 906:\n",
      "Answer to Question 2: (expression of gene DES1, production of oleic acid), (expression of gene DES2, production of 16:1<sup>Δ11cis</sup>), (expression of gene DES2, production of vaccenic acid)\n",
      "Progress: 45.30% completed\n",
      "Response for Row 907:\n",
      "Answer to Question 2: (deletion of gene PWP2, increased CCM titer), (deletion of gene EST2, increased CCM titer), (deletion of gene ATG1, increased CCM titer), (deletion of gene DIT1, increased CCM titer), (deletion of gene CDC15, increased CCM titer), (deletion of gene CTS2, increased CCM titer), (deletion of gene MNE1, increased CCM titer), (duplication of gene encoding dehydroshikimate dehydratase, increased CCM titer), (duplication of gene encoding catechol 1,2-dioxygenase, increased CCM titer), (overexpression of gene encoding PCA decarboxylase, increased CCM titer), (overexpression of gene encoding AROM protein without shikimate dehydrogenase domain, increased CCM titer), (restoration of URA3 prototrophy, increased CCM titer)\n",
      "Progress: 45.35% completed\n",
      "Response for Row 908:\n",
      "Answer to Question 2: (deletion of gene tg, consequence high levels of squalene), (deletion of gene tg, consequence high levels of ergosterol), (deletion of gene tg, consequence upregulation of MEV pathway genes), (overexpression of gene DGK1, consequence elevation in squalene levels), (overexpression of gene DGK1, consequence elevation in ergosterol levels), (overexpression of gene DGK1, consequence increase in transcript levels of MEV pathway genes), (overexpression of gene β-Amyrin synthase, consequence production of β-Amyrin)\n",
      "Progress: 45.40% completed\n",
      "Response for Row 909:\n",
      "Answer to Question 2: (expression of recombinant proteins, appropriate folding), (expression of recombinant proteins, secretion to external environment), (expression of recombinant proteins, easy purification), (expression of subunit vaccines, affects medical biotechnology market)\n",
      "Progress: 45.45% completed\n",
      "Response for Row 910:\n",
      "Answer to Question 2: (expression of gene Xltr1p, consequence increased d-xylose uptake), (expression of gene N326F^Xltr1p, consequence high efficiency d-xylose transport), (expression of gene N326F^Xltr1p, consequence reduced glucose utilization), (expression of gene F300A^Xltr1p, consequence increased glucose utilization), (expression of gene F300A^Xltr1p, consequence loss of d-xylose transport activity)\n",
      "Progress: 45.50% completed\n",
      "Response for Row 911:\n",
      "Answer to Question 2: (deletion of gene NUP170, lower amount of SAGA complex bound to target genes), (deletion of gene NUP170, levels of SAGA complex decreased), (deletion of gene NUP170, levels of SLIK complexes increased), (deletion of gene NUP170, increased nuclear activity of Pep4), (increased nuclear activity of Pep4, production of the SLIK complex), (deletion of gene NUP170, increased nuclear entry of Pep4), (increased nuclear entry of Pep4, increase in the sieving limits of the NPC diffusion channel)\n",
      "Progress: 45.55% completed\n",
      "Response for Row 912:\n",
      "Answer to Question 2: (overexpression of gene LEU4, increase in isopentanol to isobutanol product ratio), (overexpression of gene LEU1, increase in isopentanol to isobutanol product ratio), (overexpression of gene LEU2, increase in isopentanol to isobutanol product ratio), (mitochondrial localization of LEU4, reduction in intermediate loss), (mitochondrial localization of LEU1, reduction in intermediate loss), (mitochondrial localization of LEU2, reduction in intermediate loss), (reduction in intermediate loss, boost in isopentanol production), (increase in isopentanol production, isopentanol accounts for 86% of total BCHAs produced), (increase in isopentanol production, highest isopentanol titer reported for Saccharomyces cerevisiae)\n",
      "Progress: 45.60% completed\n",
      "Response for Row 913:\n",
      "Answer to Question 2: (expression of gene TMEM165, abrogates Ca2+-induced growth defects), (expression of gene TMEM165, abrogates Mn2+-induced growth defects), (expression of gene TMEM165, prevents excessive Mn2+ accumulation), (expression of gene TMEM165, corrects glycosylation defects), (expression of gene TMEM165, mediates Ca2+ influx), (expression of gene TMEM165, mediates Mn2+ influx), (mutation E108G in gene TMEM165, reduces TMEM165 activity)\n",
      "Progress: 45.65% completed\n",
      "Response for Row 914:\n",
      "Answer to Question 2: (HAC1 overexpression, enhanced transcriptional activity of GAL1 promoter), (HAC1 overexpression, increased nerolidol titer), (HAC1 overexpression, increased cell vitality), (HAC1 overexpression, metabolic regulation of monocarboxylic acid metabolic processes), (HAC1 overexpression, metabolic regulation of cellular amino acid biosynthetic process)\n",
      "Progress: 45.70% completed\n",
      "Response for Row 915:\n",
      "Answer to Question 2: (deletion of gene ERG20, lethal phenotype), (deletion of gene ERG20, successful construction with high efficiency), (deletion of gene ERG20, improvement in mutation efficiency), (deletion of gene ERG20, simplification of genetic manipulation procedure), (regulation of mutant gene expression, improvement in genome integration efficiency), (optimization of integration modules, improvement in genome integration efficiency)\n",
      "Progress: 45.75% completed\n",
      "Response for Row 916:\n",
      "Answer to Question 2: (introduction of ScADH1, increase in ethanol production from glucose by 57%), (overexpression of GLT-1, increase in ethanol production from glucose by 131%), (overexpression of CDT-1/CDT-2, increase in ethanol production from cellobiose by 200%), (turning down expression of pyc gene, increase in ethanol production by 23%)\n",
      "Progress: 45.80% completed\n",
      "Response for Row 917:\n",
      "Answer to Question 2: (gene knockout data, observe gene expression levels at steady-state), (gene knockout data, determine non-existent regulation), (gene knockout data, reduce number of regulation required for inference)\n",
      "Progress: 45.85% completed\n",
      "Response for Row 918:\n",
      "Answer to Question 2: (deletion of gene nif in non-diazotrophs, inability to fix nitrogen), (expression of gene nif from Klebsiella oxytoca in Escherichia coli, enhanced nitrogen fixation), (expression of gene nif from Azotobacter vinelandii in Saccharomyces cerevisiae, enhanced nitrogen fixation), (expression of gene nif from Paenibacillus polymyxa in tobacco plant, enhanced nitrogen fixation)\n",
      "Progress: 45.90% completed\n",
      "Response for Row 919:\n",
      "Answer to Question 2: (deletion of gene Morgs, consequence sRNA production alteration), (overexpression of gene milR236, consequence delayed appressorium formation), (overexpression of gene milR236, consequence virulence attenuation), (deletion of gene Mohat1, consequence delayed appressorium formation), (deletion of gene Mohat1, consequence virulence attenuation), (binding of transcription factor MoMsn2 to promoter of milR236, consequence suppression of MoHAT1 transcription), (suppression of MoHAT1 transcription, consequence reduced appressorium formation), (suppression of MoHAT1 transcription, consequence reduced virulence)\n",
      "Progress: 45.95% completed\n",
      "Response for Row 920:\n",
      "Answer to Question 2: (expression of CrtZ from Pantoea ananatis, increased violaxanthin yield), (expression of truncated ZEP from Haematococcus lacustris, increased violaxanthin yield), (co-expression of truncated ferredoxin-3 and truncated root ferredoxin oxidoreductase-1, 2.2-fold increase in violaxanthin yield), (increasing gene copy number of carotenogenic genes, reached 7.3 mg/g DCW violaxanthin yield)\n",
      "Progress: 46.00% completed\n",
      "Response for Row 921:\n",
      "Answer to Question 2: (overexpression of gene Candida tropicalis XR, increased xylose consumption), (overexpression of gene Candida tropicalis XR, increased xylitol yield), (overexpression of gene S. cerevisiae Gre3p aldose reductase, increased xylose consumption), (overexpression of gene S. cerevisiae Gre3p aldose reductase, increased xylitol yield), (overexpression of gene NADH-preferring C. tropicalis XR mutant, decreased xylose consumption), (overexpression of gene NADH-preferring C. tropicalis XR mutant, lower xylitol yield), (replacement of ZWF1 gene promoter with GPD promoter, increased xylitol yield), (deletion of gene XYL2, no increase in xylitol yield), (deletion of gene SOR1, no increase in xylitol yield)\n",
      "Progress: 46.05% completed\n",
      "Response for Row 922:\n",
      "Answer to Question 2:  (expression of gene ORF131, consequence activation of immune-related genes CXCa), (expression of gene ORF131, consequence activation of immune-related genes CXCR1), (expression of gene ORF131, consequence activation of immune-related genes IL-1β), (expression of gene ORF131, consequence activation of immune-related genes TNF-α), (expression of gene ORF131, consequence activation of immune-related genes INF-a1), (expression of gene ORF131, consequence activation of immune-related genes Mx-1), (expression of gene ORF131, consequence activation of immune-related genes IgM), (expression of gene ORF131, consequence activation of immune-related genes IgT1), (expression of gene ORF131, consequence production of specific serum IgM), (expression of gene ORF131, consequence relative percent of survival ranging from 53.33% to 66.67%)\n",
      "Progress: 46.10% completed\n",
      "Response for Row 923:\n",
      "Answer to Question 2: (deletion of gene YHB1, consequence down-regulation), (deletion of gene PRB1, consequence clipping of Histone H3 N-terminal tails)\n",
      "Progress: 46.15% completed\n",
      "Response for Row 924:\n",
      "Answer to Question 2: (deletion of SAGA, consequence 1), (deletion of SAGA, reduced TBP delivery to gene promoters), (deletion of SAGA, impaired transcription factor IIA efficiency), (deletion of SAGA, decreased gene expression regulation)\n",
      "Progress: 46.20% completed\n",
      "Response for Row 925:\n",
      "Answer to Question 2: (co-translational interaction of β-subunit with nascent α-subunit, initiates assembly), (exchange of two amino acids in emerging interface, prevents assembly), (co-translational interaction of subunits at other sites, initiates assembly), (formation of dimeric subunit, orchestrates complex formation), (dimeric subunit, acts as platform for post-translational phosphopantetheinylation)\n",
      "Progress: 46.25% completed\n",
      "Response for Row 926:\n",
      "Answer to Question 2: (kinetic proofreading of activator-DNA recognition, higher dissociation energies imply longer DNA dwell times of the activator), (kinetic proofreading of activator-DNA recognition, lower activator concentrations entail longer pauses between binding events), (kinetic proofreading of activator-DNA recognition, activated transcription occurs in stochastic bursts)\n",
      "Progress: 46.30% completed\n",
      "Response for Row 927:\n",
      "Answer to Question 2: (deletion of gene gsy1, consequence sporulation inefficiency), (deletion of gene gsy2, consequence sporulation inefficiency), (deletion of gene tps1, consequence sporulation inefficiency), (deletion of gene cln3, consequence increased cell size), (deletion of gene cln3, consequence increased mitochondria stores), (deletion of gene cln3, consequence increased lipid stores), (deletion of gene cln3, consequence partial rescue of meiosis progression), (deletion of gene cln3, consequence partial rescue of spore ascus formation)\n",
      "Progress: 46.35% completed\n",
      "Response for Row 928:\n",
      "Answer to Question 2:  (expression of gene elovl4, consequence elongation of C18-22 polyunsaturated fatty acids), (expression of gene elovl4, consequence production of C24 products), (expression of gene elovl4, consequence production of C26 products), (expression of gene elovl4, consequence elongation of saturated fatty acids), (expression of gene elovl4, consequence production of 28:0 product), (expression of gene elovl4, consequence production of 30:0 product)\n",
      "Progress: 46.40% completed\n",
      "Response for Row 929:\n",
      "Answer to Question 2: (no gene overexpression/deletion mentioned, no specific consequences related to gene overexpression/deletion)\n",
      "Progress: 46.45% completed\n",
      "Response for Row 930:\n",
      "Answer to Question 2: (expression of zeaxanthin epoxidase, increased integration efficiency), (expression of zeaxanthin epoxidase, minimal experimental effort)\n",
      "Progress: 46.50% completed\n",
      "Response for Row 931:\n",
      "Answer to Question 2: (deletion of ISC machinery components, impaired Fe/S cluster synthesis), (deletion of ISC machinery components, disrupted Fe/S cluster trafficking), (deletion of ISC machinery components, inhibited Fe/S cluster insertion into target apoproteins), (deletion of Hsp70 chaperone, impaired Fe/S cluster trafficking), (deletion of Hsp70 chaperone, disrupted Fe/S cluster insertion into target apoproteins)\n",
      "Progress: 46.55% completed\n",
      "Response for Row 932:\n",
      "Answer to Question 2: (deletion of gene REV1, no significant increase in meiotic mutagenicity), (deletion of gene REV3, no significant increase in meiotic mutagenicity), (deletion of gene RAD30, no significant increase in meiotic mutagenicity), (deletion of gene POL4, no significant increase in meiotic mutagenicity), (expression of gene CAN1, increased mutation frequency in meiosis compared to mitosis), (meiotic recombination, introduction of new mutations at DNA sequence level)\n",
      "Progress: 46.60% completed\n",
      "Response for Row 933:\n",
      "Answer to Question 2: (deletion of gene PP_2426, decrease in vanillyl alcohol yield), (expression of gene FEZ21_09870, conversion of vanillin to vanillyl alcohol), (expression of gene PP_2426, conversion of vanillin to vanillyl alcohol), (expression of gene PP_3839, oxidation of coniferyl alcohol to coniferyl aldehyde)\n",
      "Progress: 46.65% completed\n",
      "Response for Row 934:\n",
      "Answer to Question 2: (expression of gene RNAPII, consequence RNA polymerase II elongation rates measured in vivo), (transient inhibition of gene CDK9, consequence early elongation inhibited), (introduction of 4-thiouridine (4SU) labeling, consequence specific capture of new transcripts), (use of TT<sub>chem</sub>-seq, consequence high-resolution profiles of transcribed regions achieved)\n",
      "Progress: 46.70% completed\n",
      "Response for Row 935:\n",
      "Answer to Question 2: (deletion of gene eIF1, consequence increased translation of numerous upstream open reading frames (uORFs) initiating with near-cognate codons (NCCs)), (deletion of gene eIF1, consequence increased translation of numerous upstream open reading frames (uORFs) initiating with AUGs in poor context), (deletion of gene eIF1, consequence reduced translation of the downstream main coding sequences (CDS)), (deletion of gene eIF1, consequence elevated initiation at certain NCCs initiating amino-terminal extensions of GRS1 product), (deletion of gene eIF1, consequence elevated initiation at certain NCCs initiating amino-terminal extensions of ALA1 product), (deletion of gene eIF1, consequence elevated initiation at main CDS AUG codons with especially poor context), (deletion of gene eIF1, consequence altered ratios of protein isoforms translated from near-cognate versus AUG start codons)\n",
      "Progress: 46.75% completed\n",
      "Response for Row 936:\n",
      "Answer to Question 2: (deletion of gene warA, sensitivity to sorbic acid), (deletion of gene warA, sensitivity to propionic acid), (deletion of gene warA, sensitivity to benzoic acid), (overexpression of gene An14g03570, resistance to weak acids), (deletion of gene pdrA, reduced resistance to sorbic acid)\n",
      "Progress: 46.80% completed\n",
      "Response for Row 937:\n",
      "Answer to Question 2: (reduced expression of duplicated genes, reduced protein-to-mRNA abundance ratio), (reduced expression of dosage sensitive genes, reduced translational efficiency), (translationally repressed duplicated genes, inhibited at the level of transcription)\n",
      "Progress: 46.85% completed\n",
      "Response for Row 938:\n",
      "Answer to Question 2: (deletion of gene xxx, increased cobalt sensitivity), (deletion of gene xxx, increased intracellular cobalt accumulation), (deletion of gene xxx, increased intracellular ROS levels), (deletion of gene xxx, decreased expression of oxidative stress scavenging genes), (deletion of gene xxx, sensitivity to tunicamycin), (deletion of gene xxx, induction of unfolded protein response (UPR)), (deletion of gene xxx, activation of Ire1)\n",
      "Progress: 46.90% completed\n",
      "Response for Row 939:\n",
      "Answer to Question 2: (deletion of gene PSK1, unstable EMO), (deletion of gene PSK1, disappearance of fluctuations in EMO), (deletion of gene PSK1, reduced amplitude of EMO), (deletion of gene PSK1, downregulation of glycogen synthesis), (deletion of gene PSK1, downregulation of glycogen breakdown), (deletion of gene PSK1, decrease in glucose concentrations), (deletion of gene PSK1, decrease in NADH levels)\n",
      "Progress: 46.95% completed\n",
      "Response for Row 940:\n",
      "Answer to Question 2: (deletion of gene atg22, delays acetic acid-induced programmed cell death), (deletion of gene atg22, maintains cell wall integrity), (deletion of gene atg22, protects cytomembrane integrity), (deletion of gene atg22, changes cytomembrane phospholipids), (deletion of gene atg22, changes sterols), (deletion of gene atg22, changes fatty acids), (deletion of gene atg22, increases intracellular amino acids), (overexpression of gene atg22, enhances acetic acid-induced programmed cell death), (deletion of gene atg22, upregulates heat shock protein family expression), (deletion of gene atg22, upregulates cell wall integrity genes expression), (deletion of gene atg22, upregulates autophagy genes expression)\n",
      "Progress: 47.00% completed\n",
      "Response for Row 941:\n",
      "Answer to Question 2: (deletion of gene PbPAT14, consequence dwarf yellowing phenotype), (deletion of gene PbPAT14, consequence shorter, thinner stems), (deletion of gene PbPAT14, consequence reduction in cell number), (deletion of gene PbPAT14, consequence higher level of endogenous abscisic acid (ABA)), (deletion of gene PbPAT14, consequence higher transcript level of ABA pathway genes)\n",
      "Progress: 47.05% completed\n",
      "Response for Row 942:\n",
      "Answer to Question 2: (deletion of Cbs1, permits cytochrome b synthesis), (engagement of Cbp3-Cbp6 in assembly of cytochrome b, represses translation of cytochrome b-encoding mRNA by Cbs1)\n",
      "Progress: 47.10% completed\n",
      "Response for Row 943:\n",
      "Answer to Question 2: (deletion of gene HbSUT5, decrease in Suc uptake efficiency), (deletion of gene HbSUT5, decrease in Suc level in latex cytosol), (overexpression of gene HbSUT5, increase in Suc sink strength in laticifers), (deletion of gene HbSUT5, decrease in transcript levels in Hevea latex and bark under ethephon treatment)\n",
      "Progress: 47.15% completed\n",
      "Response for Row 944:\n",
      "Answer to Question 2: (deletion of gene SRS domain superfamily, consequence reduced parasite interaction with host), (overexpression of gene GPI biosynthetic enzymes, consequence increased parasite survival and pathogenicity)\n",
      "Progress: 47.20% completed\n",
      "Response for Row 945:\n",
      "Answer to Question 2: (no expression/deletion of gene mentioned, no specific gene overexpression/deletion or consequences described)\n",
      "Progress: 47.25% completed\n",
      "Response for Row 946:\n",
      "Answer to Question 2:  (overexpression of gene rsp5<sup>A401E</sup>, consequence growth restoration by RA), (inhibition of enzyme calcineurin, consequence suppressed β-hexosaminidase release), (inhibition of enzyme GSK-3β, consequence suppressed glucose production)\n",
      "Progress: 47.30% completed\n",
      "Response for Row 947:\n",
      "Answer to Question 2: (deletion of gene major paralog, consequence increased expression of minor paralog under stress), (overexpression of gene major paralog, consequence dominant ribosome production under normal conditions)\n",
      "Progress: 47.35% completed\n",
      "Response for Row 948:\n",
      "Answer to Question 2: (deletion of a 23.8 kb genomic region, increased p-CA production), (deletion of a 23.8 kb genomic region, upregulation of ENO2), (deletion of a 23.8 kb genomic region, upregulation of TKL1), (deletion of a 23.8 kb genomic region, upregulation of ARO1), (deletion of a 23.8 kb genomic region, upregulation of ARO2), (deletion of a 23.8 kb genomic region, upregulation of ARO4), (deletion of a 23.8 kb genomic region, upregulation of ARO7), (deletion of a 23.8 kb genomic region, upregulation of TYR1), (deletion of a 23.8 kb genomic region, change in AVT6 transcription), (deletion of a 23.8 kb genomic region, change in VBA5 transcription), (deletion of a 23.8 kb genomic region, change in THI72 transcription), (deletion of a 23.8 kb genomic region, change in ARO80 transcription)\n",
      "Progress: 47.40% completed\n",
      "Response for Row 949:\n",
      "Answer to Question 2: (over-expression of human wtp53, induction of apoptosis), (over-expression of human wtp53, targeting mitochondria), (over-expression of mitochondrial HXK-2, enhanced cell survival), (over-expression of mitochondrial HXK-2, accumulation of G6PDH), (over-expression of mitochondrial HXK-2, accumulation of 6PGDH), (over-expression of mitochondrial HXK-2, increased NADPH production), (over-expression of mitochondrial HXK-2, scavenging of ROS), (over-expression of mitochondrial HXK-2, resistance to apoptosis), (deletion of hxk2, defective growth), (deletion of hxk2, lack of G6PDH accumulation), (deletion of hxk2, lack of 6PGDH accumulation)\n",
      "Progress: 47.45% completed\n",
      "Response for Row 950:\n",
      "Answer to Question 2: (no expression/deletion of gene mentioned, no direct consequences related to gene expression/deletion mentioned)\n",
      "Progress: 47.50% completed\n",
      "Response for Row 951:\n",
      "Answer to Question 2: (deregulation of phospholipid metabolism, production of oleoylethanolamide), (engineering of subcellular trafficking, increase in oleoylethanolamide production), (screening of biosynthetic enzymes, enhancement in oleoylethanolamide yield)\n",
      "Progress: 47.55% completed\n",
      "Response for Row 952:\n",
      "Answer to Question 2: (deletion of gene AT-rich downstream DNA, consequence Pol I pausing), (deletion of gene AT-rich downstream DNA, consequence inhibition of Pol I nucleolytic cleavage activity), (overexpression of gene GC-rich downstream DNA, consequence decrease in Pol I occupancy), (overexpression of gene AT-rich downstream DNA, consequence increase in Pol I occupancy)\n",
      "Progress: 47.60% completed\n",
      "Response for Row 953:\n",
      "Answer to Question 2: (expression of gene BaFAD2, desaturation of oleic acid into linoleic acid), (expression of gene BaFAD2, desaturation of palmitoleic acid into hexadecadienoic acid), (expression of gene BaFAD3, conversion of linoleic acid to α-linolenic acid), (expression of gene BaD6D-1, conversion of linoleic acid to γ-linolenic acid), (expression of gene BaD6D-1, conversion of α-linolenic acid to stearidonic acid)\n",
      "Progress: 47.65% completed\n",
      "Response for Row 954:\n",
      "Answer to Question 2: (deletion of gene erg1, functional complementation by TwSQE6), (deletion of gene erg1, functional complementation by TwSQE7), (expression of gene TwOSC4, product cycloartenol), (expression of gene TwOSC6, product cycloartenol), (expression of gene TwOSC8, product β-amyrin)\n",
      "Progress: 47.70% completed\n",
      "Response for Row 955:\n",
      "Answer to Question 2: (deletion of gene HEM25, lower ethanol production), (expression of gene HEM25, higher ethanol production)\n",
      "Progress: 47.75% completed\n",
      "Response for Row 956:\n",
      "Answer to Question 2: (expression of gene MdSTP13a, uptake of hexose), (expression of gene MdSTP13a, uptake of Suc), (expression of gene MdSTP13a, essential for pollen tube growth on Glc), (expression of gene MdSTP13a, essential for pollen tube growth on Suc), (activation of gene MdSTP13a, by MdMYB39L), (repression of gene MdSTP13a, blocks sorbitol-modulated pollen tube growth)\n",
      "Progress: 47.80% completed\n",
      "Response for Row 957:\n",
      "Answer to Question 2: (deletion of orgA, abolishment of diffusible pigment biosynthesis), (deletion of orgB, abolishment of diffusible pigment biosynthesis), (deletion of orgC, abolishment of diffusible pigment biosynthesis), (enhanced expression of org, production of thailandene A-C), (insertion upstream of σ54-dependent transcription factor, enhanced expression of org)\n",
      "Progress: 47.85% completed\n",
      "Response for Row 958:\n",
      "Answer to Question 2: (deletion of gene uORFs, consequence increased translational efficiency of main ORF at 37 °C), (deletion of gene uORFs, consequence decreased translational efficiency of main ORF at 20 °C), (overexpression of gene uORFs starting with AUG, consequence increased translational efficiency at 37 °C), (overexpression of gene uORFs starting with AUG, consequence decreased translational efficiency at 20 °C), (overexpression of gene uORFs starting with NCCs, consequence variable translational efficiency based on mRNA-specific features), (alteration in translation of specific uORFs by temperature, consequence regulation of gene expression)\n",
      "Progress: 47.90% completed\n",
      "Response for Row 959:\n",
      "Answer to Question 2: (expression of gene gRNA, consequence rapid and efficient multiplexed refactoring of natural product BGCs in yeast), (expression of gene gRNA, consequence simultaneous replacement of up to four promoter sites with nearly 100% efficiency), (expression of gene gRNA, consequence simultaneous replacement of a total of six promoter sites with 68% efficiency), (expression of gene gRNA, consequence simultaneous replacement of a total of eight promoter sites with 32% efficiency)\n",
      "Progress: 47.95% completed\n",
      "Response for Row 960:\n",
      "Answer to Question 2: (expression of gene mtg, consequence extracellular activity of 16.1 U/mg), (replacement of native signal peptide with SP_sacB, consequence increased extracellular activity), (replacement of pro-peptide with S. hygroscopicus pro-peptide, consequence increased extracellular activity), (introduction of modified VMA intein, consequence simplified activation process), (placement of cleavage site between pro-peptide and core domain, consequence activation process at 18 °C), (promoter replacement with PyvyD, consequence extracellular enzymatic activity reached 2.6 U/mg)\n",
      "Progress: 48.00% completed\n",
      "Response for Row 961:\n",
      "Answer to Question 2: (no gene overexpression/deletion mentioned, consequence: resistance to copper), (no gene overexpression/deletion mentioned, consequence: resistance to SO₂)\n",
      "Progress: 48.05% completed\n",
      "Response for Row 962:\n",
      "Answer to Question 2: (no specific gene expression/deletion mentioned, general discussion on potential for CRISPR/Cas9 system use)\n",
      "Progress: 48.10% completed\n",
      "Response for Row 963:\n",
      "Answer to Question 2: (no gene overexpression/deletion mentioned, no specific consequences related to gene overexpression/deletion)\n",
      "Progress: 48.15% completed\n",
      "Response for Row 964:\n",
      "Answer to Question 2:  (stable integration of tabgl1 gene, secretion of TaBgl1), (secretion of TaBgl1, hydrolysis of cellobiose), (hydrolysis of cellobiose, production of ethanol), (stable integration of tabgl1 gene, high osmotolerance), (stable integration of tabgl1 gene, resistance to various concentrations of ethanol), (stable integration of tabgl1 gene, resistance to furfural), (stable integration of tabgl1 gene, resistance to high temperatures)\n",
      "Progress: 48.20% completed\n",
      "Response for Row 965:\n",
      "Answer to Question 2: (deletion of gRNA plasmid, increased efficiency of sequential gene targeting), (overexpression of gene xxx, increased production of patchoulol), (disruption of gene xxx, increased production of patchoulol), (replacement of promoter, increased production of patchoulol)\n",
      "Progress: 48.25% completed\n",
      "Response for Row 966:\n",
      "Answer to Question 2: (deletion of gene a1S, effect on mating-type gene expression at MATa locus), (deletion of gene a1S, effect on mating-type gene expression at MATα locus)\n",
      "Progress: 48.30% completed\n",
      "Response for Row 967:\n",
      "Answer to Question 2: (overexpression of gene POS5, lower biomass-substrate yields), (overexpression of gene POS5, 2-fold increase in product specific productivity), (overexpression of gene POS5, no significant effect on growth physiology when targeted to mitochondria), (overexpression of gene POS5, no significant effect on recombinant protein production when targeted to mitochondria), (overproduction of recombinant protein, increased energy demand), (overproduction of recombinant protein, endoplasmic reticulum stress), (overproduction of recombinant protein, oxidative stress), (overproduction of recombinant protein, deviation from optimal NAD(P)H regeneration balance)\n",
      "Progress: 48.35% completed\n",
      "Response for Row 968:\n",
      "Answer to Question 2: (expression of gene COX20, reduced levels of reactive oxygen species), (expression of gene COX20, reduction in apoptotic cells), (expression of gene COX20, reduction in necrotic cells), (overexpression of gene COX20, induced respiratory growth in Δimp1), (overexpression of gene COX20, induced respiratory growth in Δcox18), (presence of acetic acid, induces oxidative stress), (presence of acetic acid, induces programmed cell death)\n",
      "Progress: 48.40% completed\n",
      "Response for Row 969:\n",
      "Answer to Question 2: (depletion of gene Pol5, affects processing of pre-rRNAs for large subunit), (depletion of gene Pol5, affects processing of pre-rRNAs for small subunit), (depletion of gene Pol5, impairs recruitment of ribosomal proteins forming polypeptide exit tunnel in LSU), (depletion of gene Pol5, impairs release of 5' ETS fragments from early pre-40S particles)\n",
      "Progress: 48.45% completed\n",
      "Response for Row 970:\n",
      "Answer to Question 2: (deletion of gene FAF1, consequence 1), (deletion of gene SDA1, consequence 2), (deletion of gene DAN1, consequence 3), (deletion of gene TIR1, consequence 4)\n",
      "Progress: 48.50% completed\n",
      "Response for Row 971:\n",
      "Answer to Question 2: (overexpression of genes involved in energy conservation and production pathways, caffeine hyperresistance), (overexpression of genes involved in pleiotropic drug resistance, caffeine hyperresistance), (single nucleotide polymorphism in PDR1, potential role in caffeine hyperresistance), (single nucleotide polymorphism in PDR5, potential role in caffeine hyperresistance), (single nucleotide polymorphism in RIM8, potential role in caffeine hyperresistance)\n",
      "Progress: 48.55% completed\n",
      "Response for Row 972:\n",
      "Answer to Question 2: (PEX34 overexpression, stress to endoplasmic reticulum), (PEX34 overexpression, stress to mitochondria), (PEX34 overexpression, enhanced acetate utilization in agc1∆ yeast), (PEX34 overexpression, redirection of acetyl-CoA from synthetic reactions for amino acids), (Yhm2p requirement, PEX34 stimulated growth on acetate medium), (PEX34 overexpression, increased activity of redox shuttle involving mitochondrial citrate export), (PEX34 overexpression, increased formation of products from the glyoxylate shunt), (PEX34 overexpression, enhanced utilization of acetyl-CoA), (PEX34 overexpression, promotion of alternative mitochondrial redox shuttle activity)\n",
      "Progress: 48.60% completed\n",
      "Response for Row 973:\n",
      "Answer to Question 2: (deletion of gene erg9, increase in valencene yield), (deletion of gene rox1, increase in valencene yield), (overexpression of FPP upstream genes in MVA pathway, increase in valencene yield), (introduction of valencene synthase, biosynthesis of valencene), (use of P<sub>HXT7</sub>-VS-T<sub>TPI1</sub> cassette, improvement in valencene production)\n",
      "Progress: 48.65% completed\n",
      "Response for Row 974:\n",
      "Answer to Question 2: (deletion of gene HNT1, decreased protein production), (overexpression of gene Zap1, shift from SLT HNT1 RNA to LLT HNT1 RNA), (overexpression of gene Zap1, increased RNA of HNT1), (expression of gene LLT HNT1, dependent on Zap1)\n",
      "Progress: 48.70% completed\n",
      "Response for Row 975:\n",
      "Answer to Question 2: (decrease in aromatic amino acid gene expression, decrease in aromatic amino acid production), (up-regulation of the glyoxylate pathway, increase in p-coumaric acid production)\n",
      "Progress: 48.75% completed\n",
      "Response for Row 976:\n",
      "Answer to Question 2: (integration of tHMG1, increased terpenoid production), (integration of UPC2-1, increased terpenoid production), (knockdown of ERG9, reduced squalene synthesis), (mutation in ERG20 to mERG20, increased geranylgeranyl diphosphate synthesis), (expression of BTS1-ERG20, increased GGOH production), (expression of ERG20-mERG20, highest GGOH production), (expression of mERG20-ERG20, increased GGOH production), (integration of nor-CPS, produced nor-copalyl diphosphate), (integration of ent-CPS, produced ent-copalyl diphosphate), (integration of syn-CPS, produced syn-copalyl diphosphate)\n",
      "Progress: 48.80% completed\n",
      "Response for Row 977:\n",
      "Answer to Question 2: (deletion of gene Cln3, consequence 1: prevents irreversible commitment to the cell division cycle), (overexpression of gene Cln3, consequence 1: increases Cln3 concentration leading to passage through Start), (overexpression of gene Cln3, consequence 2: observed in both daughter and mother cells across growth conditions)\n",
      "Progress: 48.85% completed\n",
      "Response for Row 978:\n",
      "Answer to Question 2: (deletion of gene mkt1, consequence defective dikaryotic hyphal production), (deletion of gene mkt1, consequence reduced pheromone gene (MFα1) expression), (deletion of gene mkt1, consequence attenuated virulence), (deletion of gene pbp1, consequence reduced pheromone gene (MFα1) expression)\n",
      "Progress: 48.90% completed\n",
      "Response for Row 979:\n",
      "Answer to Question 2: (deletion of nuclear and mitochondrial gene products, deficits in energy production), (deletion of nuclear and mitochondrial gene products, defects in mitochondrial functions)\n",
      "Progress: 48.95% completed\n",
      "Response for Row 980:\n",
      "Answer to Question 2: (deletion of C20-hydroxylation by-products, increase in hydrocortisone productivity), (overexpression of ClCDR4 gene, increase in hydrocortisone productivity), (protein engineering of CYP5311B2 to R126D/Y398F variant, increase in hydrocortisone productivity)\n",
      "Progress: 49.00% completed\n",
      "Response for Row 981:\n",
      "Answer to Question 2: (deletion of gene VdSho1, reduced penetration), (deletion of gene VdSho1, defective melanin biosynthesis), (deletion of gene Vst50, defective penetration), (deletion of gene Vst50, defective melanin production), (deletion of gene Vst7, defective penetration), (deletion of gene Vst7, defective melanin production), (deletion of gene Vst11, defective penetration), (deletion of gene Vst11, defective melanin production), (overexpression of gene Vst50, restored penetration), (overexpression of gene Vst50, restored melanin production)\n",
      "Progress: 49.05% completed\n",
      "Response for Row 982:\n",
      "Answer to Question 2: (overexpression of gene Hub1, enhances splicing efficiency), (overexpression of gene Hub1, relaxes constraints on splice-site usage), (overexpression of gene Hub1, relaxes constraints on branch-site usage), (overexpression of gene Hub1, promotes efficient splicing of introns with non-canonical splice-sites), (impairment of metallothionein-mediated defense system, increases importance of Hub1 for cadmium tolerance)\n",
      "Progress: 49.10% completed\n",
      "Response for Row 983:\n",
      "Answer to Question 2: (no specific gene expression/deletion mentioned, discussion on CRISPR technology improvements), (no specific gene expression/deletion mentioned, exploration of gene editing and transcriptional regulation at genome scale), (no specific gene expression/deletion mentioned, enabling programmable transcriptional regulation using endonuclease-null Cas enzymes)\n",
      "Progress: 49.15% completed\n",
      "Response for Row 984:\n",
      "Answer to Question 2: (co-expression of cellulase and xylanase, increased cellulase activity), (co-expression of cellulase and xylanase, increased xylanase activity), (co-expression of cellulase and xylanase, enhanced ethanol production), (co-expression of cellulase and xylanase, improved efficiency in using partly delignified corn stover)\n",
      "Progress: 49.20% completed\n",
      "Response for Row 985:\n",
      "Answer to Question 2: (low pH, increased maintenance-energy requirements), (low pH, increased death rates), (high CO<sub>2</sub>, affected genome-wide transcriptional responses to low pH), (low pH and high CO<sub>2</sub>, interference with low-pH signaling), (common MAPK signaling pathways, involved in low-pH and high-CO<sub>2</sub> signaling), (cell wall integrity pathway, activated by low-pH and high-CO<sub>2</sub>), (high-osmolarity glycerol pathway, activated by low-pH and high-CO<sub>2</sub>), (calcineurin pathway, activated by low-pH and high-CO<sub>2</sub>)\n",
      "Progress: 49.25% completed\n",
      "Response for Row 986:\n",
      "Answer to Question 2: (deletion of actin crosslinkers, softer Arp2/3-branched actin networks), (deletion of actin crosslinkers, decreased efficiency of endocytosis), (loss of actin crosslinking, limiting the transmission of the force), (failure of endocytosis, increased lifetime of endocytic patches), (increased number of patches, reduced pool of polymerizable actin), (reduced pool of polymerizable actin, slowing down actin assembly), (slowing down actin assembly, further impairing endocytosis)\n",
      "Progress: 49.30% completed\n",
      "Response for Row 987:\n",
      "Answer to Question 2: (deletion of gene uS3/Rps3, consequence 1: reduced stop codon readthrough), (deletion of gene uS3/Rps3, consequence 2: increased stop codon readthrough), (mutation of residue K108E in uS3/Rps3, consequence 1: reduced stop codon readthrough), (mutation of residue R116D in uS3/Rps3, consequence 1: increased stop codon readthrough), (combination of R116D mutation with readthrough-reducing mutations in eIF3, consequence 1: suppressed readthrough phenotypes), (combination of K108E mutation with readthrough-reducing mutations in eIF3, consequence 1: exacerbated readthrough phenotypes)\n",
      "Progress: 49.35% completed\n",
      "Response for Row 988:\n",
      "Answer to Question 2: (deletion of gene CYP706A3, increase in sesquiterpene emissions), (deletion of gene CYP706A3, increase in monoterpene emissions), (overexpression of gene CYP706A3, decrease in sesquiterpene emissions), (overexpression of gene CYP706A3, decrease in monoterpene emissions), (overexpression of gene CYP706A3, generation of terpene oxides), (combined expression of genes TPS11 and CYP706A3, suppression of volatile emissions), (combined expression of genes TPS11 and CYP706A3, generation of soluble sesquiterpene oxides), (expression of gene CYP706A3, avoidance of feeding by Plutella xylostella larvae), (expression of gene CYP706A3, modulation of floral microbiome)\n",
      "Progress: 49.40% completed\n",
      "Response for Row 989:\n",
      "Answer to Question 2: (no gene overexpression/deletion mentioned, consequence: development of platform for simultaneous soluble secretion and cell surface display of protein in yeast Saccharomyces cerevisiae using F2A peptide)\n",
      "Progress: 49.45% completed\n",
      "Response for Row 990:\n",
      "Answer to Question 2: (deletion of gene transcription factors, consequence ethanol yields on biomass change), (deletion of gene transcription factors, consequence involvement in cell cycling), (deletion of gene transcription factors, consequence involvement in chromatin remodeling), (deletion of gene transcription factors, consequence involvement in transcription), (deletion of gene transcription factors, consequence involvement in stress response), (deletion of gene transcription factors, consequence involvement in protein synthesis), (deletion of gene transcription factors, consequence involvement in lipid synthesis)\n",
      "Progress: 49.50% completed\n",
      "Response for Row 991:\n",
      "Answer to Question 2: (no expression/deletion of gene mentioned, no specific consequence mentioned)\n",
      "Progress: 49.55% completed\n",
      "Response for Row 992:\n",
      "Answer to Question 2: (deletion of gene Pop2, increased ERCs formation), (deletion of gene Pop2, accumulation of transcripts from E-pro), (deletion of gene Pop2, reduced cohesin amounts), (deletion of gene Pop2, reduced condensin amounts), (deletion of gene Pop2, decreased rRNA amount), (deletion of gene Ccr4, similar phenotypes to Pop2 deletion), (deletion of gene Not4, similar phenotypes to Pop2 deletion)\n",
      "Progress: 49.60% completed\n",
      "Response for Row 993:\n",
      "Answer to Question 2: (deletion of FgBMH1, consequence 1: effect on gene expression), (deletion of FgBMH1, consequence 2: effect on DON production), (deletion of FgBMH1, consequence 3: effect on mycelial growth), (deletion of FgBMH2, consequence 1: effect on gene expression), (deletion of FgBMH2, consequence 2: effect on DON production), (deletion of FgBMH2, consequence 3: effect on mycelial growth), (deletion of FgBMH2, consequence 4: regulation of reproduction), (deletion of FgBMH2, consequence 5: regulation of virulence), (deletion of FgBMH2, consequence 6: regulation of glutamine-induced AreA repression)\n",
      "Progress: 49.65% completed\n",
      "Response for Row 994:\n",
      "Answer to Question 2: (expression of gene XYL1, higher ethanol yield), (expression of gene XYL2, higher ethanol yield), (expression of gene XKS1, higher ethanol yield), (expression of gene TAL1, higher ethanol yield), (expression of gene PYK1, higher ethanol yield), (expression of gene MGT05196, higher ethanol yield), (constitutive promoters use, higher ethanol yield), (increased copies of six-gene cluster, higher ethanol yield)\n",
      "Progress: 49.70% completed\n",
      "Response for Row 995:\n",
      "Answer to Question 2: (deletion of gene BGS13, elevated levels of secretion for a variety of reporter proteins), (deletion of gene BGS13, reduced protein kinase C activity), (deletion of gene BGS13, different location in the cell compared to the wt protein), (deletion of gene BGS13, inherent structural problems in the cell wall), (deletion of gene BGS13, increased release of normally secreted, endogenous proteins), (deletion of gene BGS13, increased release of endoplasmic reticulum-resident chaperone proteins)\n",
      "Progress: 49.75% completed\n",
      "Response for Row 996:\n",
      "Answer to Question 2: (deletion of gene HEM1, consequence 5-aminolevulinic acid auxotrophy), (expression of gene α-ketoglutarate dehydrogenase complex subunits, consequence increased αKGDH activity in cytosolic fraction), (expression of gene lipoic-acid scavenging enzyme, consequence enabled cytosolic lipoylation of αKGDH complex), (expression of gene bacterial ALA synthase, consequence synthesis of 5-aminolevulinic acid from cytosolic succinyl-CoA)\n",
      "Progress: 49.80% completed\n",
      "Response for Row 997:\n",
      "Answer to Question 2: (deletion of gene involved in chromatin remodeling, increased 3-HP production), (overexpression of gene involved in malonyl-CoA precursor supply, increased malonyl-CoA levels), (overexpression of gene involved in cofactor supply, increased malonyl-CoA levels), (deletion of gene, enhanced endogenous metabolic fluxes towards acetyl-CoA), (deletion of gene, enhanced endogenous metabolic fluxes towards malonyl-CoA)\n",
      "Progress: 49.85% completed\n",
      "Response for Row 998:\n",
      "Answer to Question 2: (deletion of gene IL-8, consequence upregulation in beef steers fed CON diet on day 21), (expression of gene TLR1, consequence increased expression with PROB supplementation), (expression of gene TLR2, consequence increased expression with PROB supplementation), (expression of gene TLR6, consequence increased expression with PROB supplementation), (expression of gene STAT6, consequence increased expression with PROB supplementation), (expression of gene ICAM1, consequence increased expression with PROB supplementation), (expression of gene RORC, consequence increased expression with PROB supplementation), (expression of gene TBX21, consequence increased expression with PROB supplementation), (expression of gene CXCR3, consequence increased expression with PROB supplementation), (expression of gene TNF, consequence increased expression with PROB supplementation), (expression of gene CASP1, consequence increased expression with PROB supplementation), (expression of metabolite 5-methylcytosine, consequence increased concentration with PROB supplementation), (expression of metabolite indoleacrylic acid, consequence increased concentration with PROB supplementation)\n",
      "Progress: 49.90% completed\n",
      "Response for Row 999:\n",
      "Answer to Question 2: (expression of gene CaOSC, consequence lupeol biosynthesis), (expression of gene CaOSC, consequence synthesis of lupeol as a single product), (expression of gene CaOSC, consequence increase in lupeol biosynthesis after induction through fermentation process)\n",
      "Progress: 49.95% completed\n",
      "Response for Row 1000:\n",
      "Answer to Question 2: (overexpression of gene QDR3, improved tolerance to glutaric acid), (overexpression of gene QDR3, improved tolerance to adipic acid), (overexpression of gene QDR3, improved tolerance to pimelic acid), (overexpression of gene QDR3, improved tolerance to muconic acid), (overexpression of gene QDR3, improved tolerance to glutaconic acid), (overexpression of gene QDR3, improved production of muconic acid)\n",
      "Progress: 50.00% completed\n",
      "Response for Row 1001:\n",
      "Answer to Question 2: (deletion of gene Acptp2,3, reduced sexual spore production by 4.4 times), (deletion of gene Acptp2,3, reduced asexual spore production by 4.6 times), (deletion of gene Acptp2,3, increased sensitivity to sorbitol-induced osmotic stress), (deletion of gene Acptp2,3, slower growth in high concentration sucrose medium), (deletion of gene Acptp2,3, slower growth on media containing hydrogen peroxide), (deletion of gene Acptp2,3, slower growth on media containing Congo red), (deletion of gene Acptp2,3, slower growth on media containing SDS), (deletion of gene Acptp2,3, deeper colony color)\n",
      "Progress: 50.05% completed\n",
      "Response for Row 1002:\n",
      "Answer to Question 2: (deletion of gene LAC9, consequence growth impairment on lactose), (deletion of gene LAC9, consequence growth impairment on galactose), (overexpression of gene LAC9, consequence enhanced lactose metabolism), (overexpression of gene LAC9, consequence enhanced galactose metabolism)\n",
      "Progress: 50.10% completed\n",
      "Response for Row 1003:\n",
      "Answer to Question 2: (deletion of gene PRS5, increase in intracellular nucleotide levels), (deletion of gene DBP8, increase in intracellular nucleotide levels), (deletion of gene PRS5, upregulation of genes sharing similar functions), (deletion of gene DBP8, upregulation of genes sharing similar functions), (upregulation of gene TAL1, increase in RNA content), (upregulation of gene PGM2, increase in RNA content), (upregulation of genes encoding CDK, observed in G03-△PRS5 strain), (upregulation of gene TAL1, influence through pentose phosphate pathway), (upregulation of gene PGM2, influence through pentose phosphate pathway), (polygenic recombinant strains G03-△PRS5+△DBP8, increase in RNA content by 71.8%), (polygenic recombinant strains G03-△PRS5+△DBP8+PGM2, increase in RNA content by 80.1%), (polygenic recombinant strains, increase in maximum specific growth rate)\n",
      "Progress: 50.15% completed\n",
      "Response for Row 1004:\n",
      "Answer to Question 2: (deletion of gene Tcb1, upregulation of glycolysis), (deletion of gene Tcb1, upregulation of fermentation), (deletion of gene Tcb1, upregulation of amino acid synthesis), (deletion of gene Tcb1, upregulation of low-affinity glucose uptake), (deletion of gene Tcb1, downregulation of tricarboxylic acid cycle), (deletion of gene Tcb1, downregulation of respiration), (deletion of gene Tcb1, downregulation of high-affinity glucose uptake), (deletion of gene Tcb1, downregulation of amino acid uptake), (deletion of gene Tcb1, increased growth), (deletion of gene Tcb1, increased glucose consumption), (deletion of gene Tcb1, increased CO2 production), (deletion of gene Tcb1, increased ethanol generation)\n",
      "Progress: 50.20% completed\n",
      "Response for Row 1005:\n",
      "Answer to Question 2: (deletion of gene med3, impaired S. cerevisiae growth under hyperosmotic stress), (deletion of gene med3, decrease in intracellular NADPH content), (deletion of gene med3, increased oxidative stress), (deletion of gene med3, elevated levels of intracellular reactive oxygen species), (deletion of gene med3, impact on bud formation)\n",
      "Progress: 50.25% completed\n",
      "Response for Row 1006:\n",
      "Answer to Question 2: (overexpression of gene ZWF1, increased xylitol production), (deletion of gene ALD6, increased xylitol production), (reduced expression of gene PGI1, synergistic effect with ZWF1 overexpression on glucose), (heterologous expression of gene GDP1, synergistic effect with ZWF1 overexpression on glucose), (expression of gene POS5, improved xylitol production on glucose but not with enhanced ZWF1 activity)\n",
      "Progress: 50.30% completed\n",
      "Response for Row 1007:\n",
      "Answer to Question 2: (deletion of gene irc10, increase in GC frequencies), (deletion of gene irc22, increase in GC frequencies), (deletion of gene irc23, increase in GC frequencies), (deletion of gene irc24, increase in GC frequencies), (deletion of gene irc19, reduction in GC frequencies), (deletion of gene irc21, reduction in GC frequencies), (deletion of gene irc19, reduction in SSA frequency), (deletion of gene irc19, reduction in deletion products), (deletion of gene irc19, reduction in gene conversion products), (deletion of gene irc19, synergistic interaction with rad52Δ in SSA)\n",
      "Progress: 50.35% completed\n",
      "Response for Row 1008:\n",
      "Answer to Question 2: (HMO1 overexpression, enhanced ER stress tolerance), (HMO1 overexpression, improved protein folding efficiency), (BIO5 deletion, enhanced ER stress tolerance), (BIO5 deletion, improved protein folding efficiency), (HMO1 overexpression, 2.14-fold increase in α-amylase production), (BIO5 deletion, 2.04-fold increase in cell density)\n",
      "Progress: 50.40% completed\n",
      "Response for Row 1009:\n",
      "Answer to Question 2: (deletion of gene MKS1, increase in glycerol production), (deletion of gene MKS1, reduction in ethanol production), (deletion of gene MKS1, decrease in acetic acid levels), (deletion of gene RTG2, defect in wine fermentation speed)\n",
      "Progress: 50.45% completed\n",
      "Response for Row 1010:\n",
      "Answer to Question 2: (deletion of gene GAL80, efficient β-carotene synthesis from sucrose), (deletion of gene GAL80, achieved β-carotene titers up to 727.8 ± 68.0 mg/L), (use of agricultural by-products, yielded β-carotene titer of 354.9 ± 8.2 mg/L)\n",
      "Progress: 50.50% completed\n",
      "Response for Row 1011:\n",
      "Answer to Question 2: (znf1 deletion, enhanced sensitivity to isobutanol), (znf1 deletion, enhanced sensitivity to acid inhibitors), (overexpression of ZNF1, enhanced cell survival), (Znf1 activation, ATP1-3 expression), (Znf1 activation, COX6 expression), (Znf1 activation, PPP genes expression), (Znf1 activation, valine biosynthetic genes upregulation), (isobutanol induction, ATP1-3 activation), (isobutanol induction, COX6 activation)\n",
      "Progress: 50.55% completed\n",
      "Response for Row 1012:\n",
      "Answer to Question 2: (deletion of gene FAA1, increased free fatty acid concentrations), (deletion of gene FAA4, increased free fatty acid concentrations), (deletion of gene POX1, increased free fatty acid concentrations), (deletion of gene ADO1, enhanced intracellular S-adenosylmethionine levels), (overexpression of gene SAM2, enhanced intracellular S-adenosylmethionine levels), (expression of gene DmJHAMT, enhanced FAME production)\n",
      "Progress: 50.60% completed\n",
      "Response for Row 1013:\n",
      "Answer to Question 2: (deletion of gene fol3, higher OA titer), (deletion of gene abz1, higher OA titer), (deletion of gene abz2, higher OA titer), (dynamic regulation of gene fol3, higher OA titer), (dynamic regulation of gene abz1, higher OA titer), (dynamic regulation of gene abz2, higher OA titer), (dynamic regulation of gene acc1, higher OA titer)\n",
      "Progress: 50.65% completed\n",
      "Response for Row 1014:\n",
      "Answer to Question 2: (deletion of gene PFK1, higher oxidative tolerance), (deletion of gene PFK1, slower growth in lag and exponential growth stages under 2% and 5% glucose), (deletion of gene PFK1, higher growth in stationary stage under 2% and 5% glucose), (deletion of gene PFK1, decelerated glucose consumption rate during lag growth phase), (deletion of gene PFK1, decreased expression of HXT2), (deletion of gene PFK1, decreased expression of HXT5), (deletion of gene PFK1, decreased expression of HXT6), (deletion of gene PFK1, increased expression of ZWF1), (deletion of gene PFK1, higher amount of residual glucose in medium), (deletion of gene PFK1, lower ethanol yield in early stage), (deletion of gene PFK1, decreased expression of PDC5), (deletion of gene PFK1, decreased expression of PDC6), (deletion of gene PFK1, reduced interspecies competitiveness), (deletion of gene PFK1, increased growth of co-cultured E. coli), (deletion of gene PFK1, increased ethanol production efficiency at stationary stage to antagonize E. coli)\n",
      "Progress: 50.70% completed\n",
      "Response for Row 1015:\n",
      "Answer to Question 2: (deletion of gene ino2, increase in lycopene yield), (deletion of gene ino4, increase in lycopene yield), (overexpression of gene ino2, no promotion of lycopene accumulation), (deletion of gene ino2, decrease in intracellular free fatty acids)\n",
      "Progress: 50.75% completed\n",
      "Response for Row 1016:\n",
      "Answer to Question 2: (deletion of gene OCA6, increased secreted laccase activity), (deletion of gene PMT1, increased secreted laccase activity), (deletion of gene PMT2, increased secreted laccase activity), (deletion of gene SKI3, increased secreted laccase activity), (deletion of gene SCJ1, decreased secreted laccase activity), (expression of gene SCJ1, restored laccase secretion), (overexpression of gene SCJ1, detrimental effect on laccase secretion)\n",
      "Progress: 50.80% completed\n",
      "Response for Row 1017:\n",
      "Answer to Question 2:  (overexpression of six target genes, enhanced ethanol production), (knockout of seven target genes, enhanced ethanol production), (knockout of SDH subunit gene, increased ethanol yield), (dual-gene deletion of GPD and SDH, significant enhancements in ethanol production)\n",
      "Progress: 50.85% completed\n",
      "Response for Row 1018:\n",
      "Answer to Question 2: (deletion of gene not specified, consequence efficient electron transport chain), (deletion of gene not specified, consequence key growth factor synthesis), (expression of gene not specified, consequence high lactate production)\n",
      "Progress: 50.90% completed\n",
      "Response for Row 1019:\n",
      "Answer to Question 2: (deletion of gene Cncat, significant impact on H2O2-induced oxidative stress), (over-expression of gene CnCat, improved oxidative resistance), (over-expression of gene CnCat, reduction in intracellular ROS accumulation)\n",
      "Progress: 50.95% completed\n",
      "Response for Row 1020:\n",
      "Answer to Question 2: (deletion of gene SAM1, altered chromosome stability), (deletion of gene SAM1, changes in AdoMet concentrations), (deletion of gene SAM2, altered chromosome stability), (deletion of gene SAM2, changes in AdoMet concentrations), (deletion of gene SAM1, increased sensitivity or resistance to azoles), (deletion of gene SAM2, increased sensitivity or resistance to azoles), (deletion of gene SAM1, increased sensitivity or resistance to cisplatin), (deletion of gene SAM2, increased sensitivity or resistance to cisplatin), (deletion of gene SAM1, increased sensitivity or resistance to oxidative stress), (deletion of gene SAM2, increased sensitivity or resistance to oxidative stress), (deletion of gene SAM1, perturbations in arginine biosynthesis), (deletion of gene SAM2, perturbations in arginine biosynthesis), (deletion of gene SAM1, changes in response to DNA synthesis inhibitors), (deletion of gene SAM2, changes in response to DNA synthesis inhibitors), (deletion of gene SAM1, changes in response to tamoxifen), (deletion of gene SAM2, changes in response to tamoxifen)\n",
      "Progress: 51.00% completed\n",
      "Response for Row 1021:\n",
      "Answer to Question 2: (deletion of oxidoreductases, improvement in vanillin accumulation), (overexpression of coenzyme-A free pathways, enhancement in vanillin production), (optimization of NADPH supply, increase in vanillin productivity), (optimization of S-adenosylmethionine supply, increase in vanillin productivity), (metabolic reconfiguration of yeast central metabolism, enhancement in vanillin productivity)\n",
      "Progress: 51.05% completed\n",
      "Response for Row 1022:\n",
      "Answer to Question 2: (deletion of gene met17, inability to grow on sulfate as sole sulfur source), (deletion of gene met17, ability to grow on sulfate at high cell densities), (accumulation of hydrogen sulfide, enables growth of met17Δ on sulfate), (induction of gene hsu1, response to sulfur starvation), (induction of gene hsu1, response to high sulfide concentrations), (gene hsu1, enables assimilation of hydrogen sulfide), (low efficiency of sulfide assimilation in met17Δ, explains density-dependent growth on sulfate)\n",
      "Progress: 51.10% completed\n",
      "Response for Row 1023:\n",
      "Answer to Question 2: (deletion of gene SIW14, increase in cells with abnormal endoplasmic reticulum), (deletion of gene SIW14, increase in accumulation of reactive oxygen species), (involvement of gene OCA1, protein phosphorylation pathway), (involvement of gene SIW14, protein phosphorylation pathway)\n",
      "Progress: 51.15% completed\n",
      "Response for Row 1024:\n",
      "Answer to Question 2: (deletion of gene PfVP1, delayed ring stage development), (deletion of gene PfVP1, blockage of ring-to-trophozoite transition), (deletion of gene PfVP1, onset of parasite death)\n",
      "Progress: 51.20% completed\n",
      "Response for Row 1025:\n",
      "Answer to Question 2: (overexpression of RAS2S225A, mislocalisation), (overexpression of RAS2S225A, constitutive activation), (overexpression of RAS2S225A, slow growth), (overexpression of RAS2S225A, loss of respiration), (overexpression of RAS2S225A, reduced stress response), (overexpression of RAS2S225A, state of quiescence), (overexpression of RAS2S225E, mislocalisation), (overexpression of RAS2S225E, constitutive activation), (overexpression of RAS2S225E, slow growth), (overexpression of RAS2S225E, loss of respiration), (overexpression of RAS2S225E, reduced stress response), (overexpression of RAS2S225E, state of quiescence), (deletion of CUP9, rescue of quiescent cell fate), (addition of peptone, rescue of quiescent cell fate)\n",
      "Progress: 51.25% completed\n",
      "Response for Row 1026:\n",
      "Answer to Question 2:  (overexpression of gene OsGSTZ4, conferred cellular Cd tolerance), (overexpression of gene OsGSTZ4, improved plant growth), (overexpression of gene OsGSTZ4, attenuated Cd-induced toxicity), (overexpression of gene OsGSTZ4, accumulated more Cd in roots), (induction of gene OsGSTZ4, CHH hypermethylation), (induction of gene OsGSTZ4, detection of 24-nt siRNAs), (loss of function for CHH methylation in osdrm2, compromised CHH methylation at the promoter and transcript level of OsGSTZ4), (defective in RNA-dependent RNA polymerase 2 in osrdr2i, compromised CHH methylation at the promoter and transcript level of OsGSTZ4)\n",
      "Progress: 51.30% completed\n",
      "Response for Row 1027:\n",
      "Answer to Question 2: (deletion of gene GCN4, heightened mitochondrial activity), (deletion of gene GCN4, increase in reactive oxygen species (ROS) accumulation), (deletion of gene GCN4, reduced ethanol production), (deletion of gene GCN4, reduction in trehalose), (deletion of gene GCN4, disparities in free amino acids), (deletion of gene GCN4, disrupted cellular homeostasis), (deletion of gene GCN4, shortened lifespan)\n",
      "Progress: 51.35% completed\n",
      "Response for Row 1028:\n",
      "Answer to Question 2: (deletion of gene CAR1, reduces accumulation of ethyl carbamate), (deletion of gene CAR1, causes strong growth inhibition on medium with arginine as only nitrogen source)\n",
      "Progress: 51.40% completed\n",
      "Response for Row 1029:\n",
      "Answer to Question 2: (deletion of gene FYV6, decreases splicing in vivo of reporter genes), (deletion of gene FYV6, accumulates first-step products in vitro), (deletion of gene FYV6, defect in exon ligation), (deletion of gene FYV6, changes 3' splice site selection in SUS1 transcript)\n",
      "Progress: 51.45% completed\n",
      "Response for Row 1030:\n",
      "Answer to Question 2: (deletion of gene pdc, subdue ethanol formation), (introduction of gene l-lactate dehydrogenase from Weizmannia coagulans 2-6, redirect ethanol fermentation pathway to l-LA production), (introduction of gene L-ldhA from Bos taurus, redirect ethanol fermentation pathway to l-LA production), (deletion of gene dld(chr3), increase optical purity of l-LA)\n",
      "Progress: 51.50% completed\n",
      "Response for Row 1031:\n",
      "Answer to Question 2: (deletion of gene BUD21, improvement in growth), (deletion of gene BUD21, enhancement in substrate utilization), (deletion of gene BUD21, increase in ethanol productivity), (deletion of gene BUD21, no confirmation of positive effect on aerobic growth in non-engineered strains), (deletion of gene BUD21, no confirmation of positive effect on xylose utilization in non-engineered strains)\n",
      "Progress: 51.55% completed\n",
      "Response for Row 1032:\n",
      "Answer to Question 2: (chromosomal deletion of gene STB5, enhances autophagy), (loss of gene Stb5, upregulation of select ATG transcripts under nutrient-replete conditions), (Stb5-mediated impact, effect on genes involved in NADPH production), (Stb5-mediated impact, effect on genes involved in pentose phosphate pathway)\n",
      "Progress: 51.60% completed\n",
      "Response for Row 1033:\n",
      "Answer to Question 2: (deletion of gene PDR1, reduced yeast resistance to tyrosol), (deletion of gene PDR3, reduced yeast resistance to tyrosol), (expression of gene PDR5, contributed to yeast resistance to tyrosol), (tyrosol, inhibited efflux of rhodamine 6G), (tyrosol, induced MDR), (tyrosol, increased Pdr5-GFP levels), (tyrosol, reduced yeast ability to accumulate Nile red), (tyrosol, inhibited cytostatic effect of clotrimazole)\n",
      "Progress: 51.65% completed\n",
      "Response for Row 1034:\n",
      "Answer to Question 2:  (deletion of gene SAM1, altered chromosome stability), (deletion of gene SAM1, changes in AdoMet concentrations), (deletion of gene SAM2, altered chromosome stability), (deletion of gene SAM2, changes in AdoMet concentrations), (deletion of gene SAM1, changes in sensitivity or resistance to azoles), (deletion of gene SAM1, changes in sensitivity or resistance to cisplatin), (deletion of gene SAM1, changes in sensitivity or resistance to oxidative stress), (deletion of gene SAM1, perturbations in arginine biosynthesis), (deletion of gene SAM1, changes in sensitivity or resistance to DNA synthesis inhibitors), (deletion of gene SAM1, changes in sensitivity or resistance to tamoxifen), (deletion of gene SAM2, changes in sensitivity or resistance to azoles), (deletion of gene SAM2, changes in sensitivity or resistance to cisplatin), (deletion of gene SAM2, changes in sensitivity or resistance to oxidative stress), (deletion of gene SAM2, perturbations in arginine biosynthesis), (deletion of gene SAM2, changes in sensitivity or resistance to DNA synthesis inhibitors), (deletion of gene SAM2, changes in sensitivity or resistance to tamoxifen)\n",
      "Progress: 51.70% completed\n",
      "Response for Row 1035:\n",
      "Answer to Question 2: (deletion of gene MoCBP7, consequence 1), (deletion of gene MoCBP7, consequence 2), (deletion of gene MoCBP7, consequence 3), (deletion of gene MoCBP7, consequence 4), (deletion of gene MoCBP7, consequence 5)\n",
      "Progress: 51.75% completed\n",
      "Response for Row 1036:\n",
      "Answer to Question 2: (deletion of gene x, increased protein secretion), (deletion of gene y, increased protein secretion), (deletion of gene z, increased protein secretion), (deletion of gene w, increased protein secretion), (combination of gene deletions x,y,z,w, 5045 mg/L secretory NPA)\n",
      "Progress: 51.80% completed\n",
      "Response for Row 1037:\n",
      "Answer to Question 2: (deletion of gene involved in calcium metabolism, resistance to aluminium), (deletion of gene involved in calcium metabolism, resistance to gallium), (deletion of gene involved in calcium metabolism, resistance to indium), (deletion of gene involved in Ire1/Hac1-mediated protection, resistance to aluminium), (deletion of gene involved in Ire1/Hac1-mediated protection, resistance to gallium), (deletion of gene involved in Ire1/Hac1-mediated protection, resistance to indium), (deletion of gene involved in vesicle-mediated transport, resistance to aluminium), (deletion of gene involved in autophagy, resistance to aluminium), (deletion of gene involved in protein folding, resistance to gallium), (deletion of gene involved in phospholipid metabolism, resistance to gallium), (deletion of gene involved in chorismate metabolic processes, resistance to indium)\n",
      "Progress: 51.85% completed\n",
      "Response for Row 1038:\n",
      "Answer to Question 2: (deletion of gene Hsp26, decreased production of ethyl acetate), (deletion of gene Hsp78, decreased production of ethyl acetate), (deletion of gene Hsp82, decreased production of ethyl acetate), (overexpression of gene HSP30, increased transcriptional abundance of ATF1 gene), (overexpression of gene HSP30, increased production of ethyl acetate), (overexpression of gene Hsf1, alleviated unfolded protein response), (overexpression of gene Hsf1, reprogramed pattern of ATF1 gene transcript levels), (overexpression of gene Hsf1, increased production of ethyl acetate), (overexpression of gene Hsf1 and Hsps, increased production of ethyl acetate), (strong promoter TDH3p, decreased production of ethyl acetate), (strong promoter TDH3p, decreased transcription level of ATF1), (strong promoter TDH3p, activated unfolded protein reaction)\n",
      "Progress: 51.90% completed\n",
      "Response for Row 1039:\n",
      "Answer to Question 2: (deletion of gene DFG5, improved BGL1 secretion), (deletion of gene YPK1, improved BGL1 secretion), (deletion of gene FYV5, improved BGL1 secretion), (deletion of gene CCW12, improved BGL1 secretion), (deletion of gene KRE1, improved BGL1 secretion), (double deletion of genes FYV5 and CCW12, increased secreted BGL1 activity by 6.13-fold), (double deletion of genes FYV5 and CCW12, increased surface-displayed BGL1 activity by 7.99-fold)\n",
      "Progress: 51.95% completed\n",
      "Response for Row 1040:\n",
      "Answer to Question 2: (deletion of gene ssu1, increased intracellular selenomethionine content), (expression of gene ssu1, selenium efflux process), (mutation in gene fzf1, selenium tolerance), (selenite induction, expression of gene ssu1)\n",
      "Progress: 52.00% completed\n",
      "Response for Row 1041:\n",
      "Answer to Question 2: (deletion of gene ERG5, improvement of thermotolerance), (deletion of gene ERG4, improvement of thermotolerance), (deletion of gene ERG3, improvement of thermotolerance), (deletion of gene ERG5, increase in ethanol production), (deletion of gene ERG4, increase in ethanol production), (deletion of gene ERG3, increase in ethanol production), (deletion of gene ERG5, up-regulation of gene ERG1), (deletion of gene ERG4, up-regulation of gene ERG1), (deletion of gene ERG3, up-regulation of gene ERG1), (deletion of gene ERG5, up-regulation of gene ERG11), (deletion of gene ERG4, up-regulation of gene ERG11), (deletion of gene ERG3, up-regulation of gene ERG11), (deletion of gene ERG5, down-regulation of gene ERG9), (deletion of gene ERG4, down-regulation of gene ERG9), (deletion of gene ERG3, down-regulation of gene ERG9), (deletion of gene ERG5, down-regulation of gene ERG26), (deletion of gene ERG4, down-regulation of gene ERG26), (deletion of gene ERG3, down-regulation of gene ERG26)\n",
      "Progress: 52.05% completed\n",
      "Response for Row 1042:\n",
      "Answer to Question 2: (deletion of gene BcTaf14, slow growth), (deletion of gene BcTaf14, abnormal colony morphology), (deletion of gene BcTaf14, reduced conidiation), (deletion of gene BcTaf14, abnormal conidial morphology), (deletion of gene BcTaf14, reduced virulence), (deletion of gene BcTaf14, altered responses to various stresses), (mutation of gene BcTaf14 G80 and W81, affected regulatory effect on mycelial growth), (mutation of gene BcTaf14 G80 and W81, affected regulatory effect on virulence), (mutation of gene BcTaf14 G80 and W81, did not affect production and morphology of conidia), (deletion of ET domain in BcTaf14, unable to localize to nucleus)\n",
      "Progress: 52.10% completed\n",
      "Response for Row 1043:\n",
      "Answer to Question 2:  (deletion of gene DDGS;OMT;ATPG, consequence not producing mycosporines), (deletion of gene ATPG, consequence accumulating intermediate 4-deoxygadusol), (expression of gene DDGS;OMT, consequence producing 4-deoxygadusol), (expression of gene DDGS;OMT;ATPG, consequence producing MG), (integration of gene DDGS;OMT;ATPG, consequence producing MG and mycosporine glutaminol glucoside), (deletion of gene MIG1, consequence upregulation of mycosporinogenesis), (deletion of gene CYC8, consequence upregulation of mycosporinogenesis), (deletion of gene OPI1, consequence upregulation of mycosporinogenesis), (deletion of gene ROX1, consequence downregulation of mycosporinogenesis), (deletion of gene SKN7, consequence downregulation of mycosporinogenesis), (deletion of gene TUP6, consequence no effect on mycosporinogenesis), (deletion of gene YAP6, consequence no effect on mycosporinogenesis)\n",
      "Progress: 52.15% completed\n",
      "Response for Row 1044:\n",
      "Answer to Question 2: (deletion of gene QDR3, reduced support for Lactobacillus fermentum growth), (deletion of gene QDR1, reduced lactic acid production), (deletion of gene QDR2, reduced lactic acid production), (deletion of gene QDR3, reduced lactic acid production), (deletion of gene QDR1, no significant reduction in ethanol production), (deletion of gene QDR2, no significant reduction in ethanol production), (deletion of gene QDR3, no significant reduction in ethanol production)\n",
      "Progress: 52.20% completed\n",
      "Response for Row 1045:\n",
      "Answer to Question 2: (deletion of gene MPC1, no significant impact on lactic acid production), (deletion of gene MPC1, negatively affected cell growth), (expression of gene VHb, detected in aerobic cultivation), (blocking cell respiration using hypoxia, suitable for producing lactic acid)\n",
      "Progress: 52.25% completed\n",
      "Response for Row 1046:\n",
      "Answer to Question 2: (deletion of gene Huura3, consequence 1), (deletion of gene Huhis3, consequence 1), (deletion of gene Huleu2, consequence 1), (deletion of gene Huade2, consequence 1)\n",
      "Progress: 52.30% completed\n",
      "Response for Row 1047:\n",
      "Answer to Question 2: (deletion of gene SUT125, down-regulation of PIL1), (deletion of gene PIL1, increased secretion of invertase), (deletion of gene LSP1, increased secretion of invertase), (deletion of gene SUR7, increased secretion of invertase), (deletion of gene PIL1, increased secretion of GFP), (deletion of gene PIL1, increased secretion of Pexelizumab scFv peptide), (deletion of gene PIL1, up-regulation of sphingolipid biosynthesis), (deletion of gene PIL1, redistribution of sphingolipid in the plasma membrane)\n",
      "Progress: 52.35% completed\n",
      "Response for Row 1048:\n",
      "Answer to Question 2: (deletion of gene FoRSR1, reduction in mycelial dry weight), (deletion of gene FoRSR1, blunted and swollen hyphae), (deletion of gene FoRSR1, fewer and morphologically abnormal conidia), (deletion of gene FoRSR1, defective chlamydospore formation), (deletion of gene FoRSR1, reduced virulence), (deletion of gene FoRSR1, limited necrosis at wounding sites), (deletion of gene FoRSR1, affected pigmentation), (deletion of gene FoRSR1, affected autophagy), (deletion of gene FoRSR1, reduced production of fusaric acid), (deletion of gene FoRSR1, downregulation of genes involved in secondary metabolism)\n",
      "Progress: 52.40% completed\n",
      "Response for Row 1049:\n",
      "Answer to Question 2: (deletion of gene SSE1, consequence severe synthetic growth defects), (deletion of gene PRE9, consequence severe synthetic growth defects), (deletion of gene PBA3, consequence severe synthetic growth defects), (deletion of gene UMP1, consequence severe synthetic growth defects), (deletion of gene SSE1, consequence high levels of ubiquitin-conjugates), (deletion of gene PRE9, consequence high levels of ubiquitin-conjugates), (deletion of gene PBA3, consequence high levels of ubiquitin-conjugates), (deletion of gene UMP1, consequence high levels of ubiquitin-conjugates), (deletion of gene SSE1, consequence accumulation of distinct small complexes with α subunits), (deletion of gene PRE9, consequence accumulation of distinct small complexes with α subunits), (deletion of gene PBA3, consequence accumulation of distinct small complexes with α subunits), (deletion of gene UMP1, consequence accumulation of distinct small complexes with α subunits)\n",
      "Progress: 52.45% completed\n",
      "Response for Row 1050:\n",
      "Answer to Question 2: (deletion of gene ADE2, precision editing), (deletion of gene ADE2, large DNA region replacement)\n",
      "Progress: 52.50% completed\n",
      "Response for Row 1051:\n",
      "Answer to Question 2: (deletion of gene MKS1, increased IA titer), (deletion of gene HXK2, increased IA titer), (deletion of gene GSF2, increased IA titer), (overexpression of gene cadA, increased IA titer), (overexpression of gene Dz.ada, increased IA titer), (overexpression of gene Mt.acs, increased IA titer), (overexpression of gene aac2, increased IA titer), (overexpression of gene related to glucose derepression, increased IA titer), (overexpression of acetyl-CoA synthesis pathway, increased IA titer)\n",
      "Progress: 52.55% completed\n",
      "Response for Row 1052:\n",
      "Answer to Question 2: (deletion of gene sod1, improved ATP supply), (deletion of gene sod1, 22.3% improvement in SAM content), (deletion of gene sod1, positive regulation of SAM synthesis), (deletion of gene sod1, enhancement of substrate assimilation leading to increased SAM production)\n",
      "Progress: 52.60% completed\n",
      "Response for Row 1053:\n",
      "Answer to Question 2: (deletion of gene pho84, increased vanillin tolerance), (deletion of gene lam3, increased vanillin tolerance), (deletion of genes involved in phosphate metabolism, increased vanillin tolerance), (deletion of genes involved in inositol polyphosphate metabolism, increased vanillin tolerance), (deletion of genes involved in sterol transport, increased vanillin tolerance), (activation of phosphate signaling pathway, decreased vanillin tolerance), (reduction of sterol transport from plasma membrane, increased vanillin tolerance), (vanillin, inhibits growth in Saccharomyces cerevisiae), (pho84Δ, promotes rapid ethanol production), (lam3Δ, promotes rapid ethanol production)\n",
      "Progress: 52.65% completed\n",
      "Response for Row 1054:\n",
      "Answer to Question 2: (deletion of endogenous galactose metabolic pathway genes, improved production of 2'-FL), (deletion of endogenous galactose metabolic pathway genes, improved production of 3-FL), (expression of Kα in mating type \"a\" yeast, reduced growth interference), (enhanced expression of pathway genes by Gal4p, increased production of 2'-FL), (enhanced expression of pathway genes by Gal4p, increased production of 3-FL)\n",
      "Progress: 52.70% completed\n",
      "Response for Row 1055:\n",
      "Answer to Question 2: (deletion of gene Achog1, delay in conidia production), (deletion of gene Achog1, reduction in conidia number), (deletion of gene Achog1, increased sensitivity to hyperosmolarity), (deletion of gene Achog1, increased sensitivity to high oxidative stress), (deletion of gene Achog1, inhibited growth under alkaline stress), (deletion of gene Achog1, decreased pigmentation), (deletion of gene Achog1, suppression of genes related to asexual development), (deletion of gene Achog1, suppression of genes related to osmotic stress resistance), (deletion of gene Achog1, suppression of genes related to oxidative stress resistance)\n",
      "Progress: 52.75% completed\n",
      "Response for Row 1056:\n",
      "Answer to Question 2: (deletion of gene Met17p, consequence organosulfur auxotrophy), (expression of gene HSU1, consequence substitution for Met17p function), (expression of gene HSU1, consequence reassimilation of hydrosulfide ions), (expression of gene HSU1, consequence formation of homocysteine)\n",
      "Progress: 52.80% completed\n",
      "Response for Row 1057:\n",
      "Answer to Question 2: (deletion of gene FaSmi1, consequence mycelial growth defects), (deletion of gene FaSmi1, consequence asexual reproduction defects), (deletion of gene FaSmi1, consequence virulence reduction), (deletion of gene FaSmi1, consequence increased sensitivity to osmotic stresses), (deletion of gene FaSmi1, consequence increased tolerance to oxidative stresses), (deletion of gene FaSmi1, consequence increased tolerance to cell wall integrity inhibitors), (deletion of gene FaSmi1, consequence lower expression of antioxidant system-associated genes), (deletion of gene FaSmi1, consequence decreased DON content)\n",
      "Progress: 52.85% completed\n",
      "Response for Row 1058:\n",
      "Answer to Question 2: (deletion of gene VdERG2, reduced ergosterol content), (deletion of gene VdERG2, decreased conidial germination rate), (deletion of gene VdERG2, decreased conidial yield), (deletion of gene VdERG2, abnormal conidia production), (deletion of gene VdERG2, decreased melanin production), (deletion of gene VdERG2, decreased microsclerotia production), (deletion of gene VdERG2, decreased expression of VaflM), (deletion of gene VdERG2, decreased expression of Vayg1), (deletion of gene VdERG2, decreased expression of VDH1), (deletion of gene VdERG2, decreased expression of VdLAC), (deletion of gene VdERG2, decreased expression of VdSCD), (deletion of gene VdERG2, decreased expression of VT4HR), (deletion of gene VdERG2, increased sensitivity to congo red), (deletion of gene VdERG2, increased sensitivity to sodium dodecyl sulfate), (deletion of gene VdERG2, increased sensitivity to hydrogen peroxide), (deletion of gene VdERG2, weakened penetration ability of mycelium), (deletion of gene VdERG2, impaired growth of mycelium), (deletion of gene VdERG2, impaired pathogenicity)\n",
      "Progress: 52.90% completed\n",
      "Response for Row 1059:\n",
      "Answer to Question 2: (deletion of gene MoGap1, sporulation reduction), (deletion of gene MoGap1, increased sensitivity to cell wall stress), (deletion of gene MoGap1, increased sensitivity to carbon source stress), (deletion of gene MoGap1, decreased TOR activity), (deletion of gene MoGap1, improved level of autophagy)\n",
      "Progress: 52.95% completed\n",
      "Response for Row 1060:\n",
      "Answer to Question 2: (overexpression of gene BUD1, increased surface-displayed α-amylase activity), (overexpression of gene BUD1, increased secreted α-amylase activity), (overexpression of gene CDC42, increased surface-displayed α-amylase activity), (overexpression of gene AXL1, increased surface-displayed α-amylase activity), (overexpression of gene BUD10, increased surface-displayed α-amylase activity), (overexpression of gene BUD3, increased secreted α-amylase activity), (overexpression of gene BUD4, increased secreted α-amylase activity), (overexpression of gene BUD7, increased secreted α-amylase activity), (overexpression of gene BUD10, increased secreted α-amylase activity), (overexpression of gene BUD1, increased surface-displayed α-amylase expression by 56%), (overexpression of gene BUD1, increased secreted α-amylase expression by 49%), (co-overexpression of gene BUD1 and CDC42, increased α-amylase surface display by 100%), (two genomic copies of gene BUD1, improved α-amylase secretion by 92%)\n",
      "Progress: 53.00% completed\n",
      "Response for Row 1061:\n",
      "Answer to Question 2: (upregulation of genes associated with the pentose phosphate pathway, cell death), (upregulation of genes associated with cell wall organization or biogenesis, cell death), (upregulation of genes associated with glutathione metabolism, cell death), (upregulation of genes associated with protein refolding, cell death), (upregulation of genes associated with phosphatidylcholine biosynthesis, cell death), (upregulation of genes associated with AdoMet synthesis, cell death), (inactivation or structure alteration of YNL194Cp, cell death)\n",
      "Progress: 53.05% completed\n",
      "Response for Row 1062:\n",
      "Answer to Question 2: (deletion of gene Acpbs2, delayed conidia production), (deletion of gene Acpbs2, reduced conidia number), (deletion of gene Acpbs2, increased sensitivity to 3 M NaCl), (deletion of gene Acpbs2, increased sensitivity to 3 M sucrose), (deletion of gene Acpbs2, increased sensitivity to 3 M sorbitol), (deletion of gene Acpbs2, increased sensitivity to high oxidative stress), (deletion of gene Acpbs2, inhibited growth under alkaline-pH stress), (deletion of gene Acpbs2, lighter colony color), (deletion of gene Acpbs2, altered expression of genes related to conidial development), (deletion of gene Acpbs2, altered expression of genes related to osmotic stress), (deletion of gene Acpbs2, altered expression of genes related to oxidative stress), (deletion of gene Acpbs2, altered expression of genes related to carbon metabolism)\n",
      "Progress: 53.10% completed\n",
      "Response for Row 1063:\n",
      "Answer to Question 2: (expression of gene Kluyveromyces marxianus TPI, constitutive expression), (expression of gene Hansenula polymorpha PMA, constitutive expression), (expression of gene Candida tropicalis ICL, inducible expression), (expression of gene Saccharomyces cerevisiae CUP, inducible expression), (expression of gene HpPMA, highest promoter activity), (elevating Cu<sup>2+</sup>, increased expression of reporter gene with ScCUP)\n",
      "Progress: 53.15% completed\n",
      "Response for Row 1064:\n",
      "Answer to Question 2: (deletion of gene FgELO2, serious defects in vegetative growth), (deletion of gene FgELO2, disrupted conidiation), (deletion of gene FgELO2, reduced ergosterol content), (deletion of gene FgELO2, disrupted ER-localization of 14-α-demethylase FgCyp51s), (deletion of gene FgELO2, elevated sensitivity to tebuconazole), (deletion of gene FgELO2, decreased phosphorylation of high osmolarity glycerol (HOG) pathway), (deletion of gene FgELO2, increased sensitivity to osmotic stress), (deletion of gene FgELO2, reduced fungal virulence), (deletion of gene FgELO2, inhibited deoxynivalenol production), (deletion of gene FgELO2, inhibited formation of infection structures)\n",
      "Progress: 53.20% completed\n",
      "Response for Row 1065:\n",
      "Answer to Question 2: (deletion of gene REG1, improved SAM production), (overexpression of gene SNF1, improved SAM production), (deletion of gene REG1, enhanced glucose utilization), (overexpression of gene SNF1, enhanced glucose utilization), (deletion of gene REG1, elevated levels of glycolytic intermediates), (overexpression of gene SNF1, elevated levels of glycolytic intermediates), (deletion of gene REG1, increased expression of ACS1), (overexpression of gene SNF1, increased expression of ACS1), (deletion of gene REG1, increased expression of ALD6), (overexpression of gene SNF1, increased expression of ALD6), (deletion of gene REG1, enhanced ADH2 activity), (overexpression of gene SNF1, enhanced ADH2 activity), (deletion of gene REG1, enhanced sulfur-containing amino acids biosynthesis), (overexpression of gene SNF1, enhanced sulfur-containing amino acids biosynthesis), (deletion of gene REG1, extended yeast lifespan), (overexpression of gene SNF1, extended yeast lifespan)\n",
      "Progress: 53.25% completed\n",
      "Response for Row 1066:\n",
      "Answer to Question 2: (deletion of gene gpd1, improved D-LA titer), (deletion of gene gpd2, improved D-LA titer), (deletion of gene adh1, improved D-LA titer), (deletion of gene gpd1, growth impairment), (deletion of gene gpd2, growth impairment), (deletion of gene adh1, growth impairment), (expression of gene DLDH, increased D-LA production)\n",
      "Progress: 53.30% completed\n",
      "Response for Row 1067:\n",
      "Answer to Question 2: (deletion of gene MoDHX35, reduced tolerance to doxorubicin), (deletion of gene MoDHX35, reduced appressorium formation rate), (deletion of gene MoDHX35, attenuated virulence)\n",
      "Progress: 53.35% completed\n",
      "Response for Row 1068:\n",
      "Answer to Question 2: (deletion of gene RPD3, decreases in production of de novo pathway metabolites), (deletion of gene RPD3, reduced acetylation of H4K5 and H4K12), (deletion of gene HST1, increases acetylation of H4K5 and H4K12), (deletion of gene HST1, increases BNA gene expression), (deletion of gene RPD3, increases BNA gene expression), (deletion of gene HST1, decreases repression of de novo NAD<sup>+</sup> synthesis)\n",
      "Progress: 53.40% completed\n",
      "Response for Row 1069:\n",
      "Answer to Question 2: (deletion of gene MoSCS2, reduction in growth), (deletion of gene MoSCS2, reduction in conidiation), (deletion of gene MoSCS2, abnormal conidial morphology), (deletion of gene MoSCS2, abnormal septum formation), (deletion of gene MoSCS2, delayed appressorium formation), (deletion of gene MoSCS2, inability to form huge turgor pressure), (deletion of gene MoSCS2, inhibition of invasive hyphae expansion), (deletion of gene MoSCS2, decline in pathogenicity)\n",
      "Progress: 53.45% completed\n",
      "Response for Row 1070:\n",
      "Answer to Question 2: (deletion of gene GPD2, increase in ethanol content), (deletion of gene GPD2, decrease in glycerol content), (deletion of gene FPS1, increase in ethanol content), (deletion of gene FPS1, decrease in glycerol content), (deletion of gene ADH2, increase in ethanol content), (deletion of gene ADH2, decrease in glycerol content), (deletion of gene DLD3, increase in ethanol content), (deletion of gene DLD3, decrease in glycerol content), (deletion of gene GPD2, decrease in acetic acid content), (deletion of gene FPS1, decrease in acetic acid content), (deletion of gene ADH2, decrease in acetic acid content), (deletion of gene DLD3, decrease in acetic acid content), (deletion of gene GPD2, decrease in lactic acid content), (deletion of gene FPS1, decrease in lactic acid content), (deletion of gene ADH2, decrease in lactic acid content), (deletion of gene DLD3, decrease in lactic acid content)\n",
      "Progress: 53.50% completed\n",
      "Response for Row 1071:\n",
      "Answer to Question 2: (deletion of gene HXS1, slower rate of glucose consumption), (deletion of gene HXS1, slower rate of xylose consumption), (deletion of gene HXS1, 4 times less ethanol production), (overexpression of gene HXS1, 10% increase of ethanol production from glucose), (overexpression of gene HXS1, more than 2 times increase of ethanol production from xylose), (overexpression of gene AZF1, 10% more ethanol production in glucose medium), (overexpression of gene AZF1, 2.4 times more ethanol production in xylose medium), (deletion of gene AZF1, 1.5 times drop in ethanol accumulation in xylose medium), (deletion of gene AZF1, 1.8 times drop in ethanol accumulation in glucose medium)\n",
      "Progress: 53.55% completed\n",
      "Response for Row 1072:\n",
      "Answer to Question 2: (deletion of gene fet3, inability to grow in low-iron conditions), (deletion of gene arn1-4, inability to grow in low-iron conditions), (expression of gene sib1+, production of ferrichrome), (expression of gene sib2+, production of ferrichrome), (deletion of gene sib1+, defect in ferrichrome production), (deletion of gene sib2+, defect in ferrichrome production), (deletion of gene sib3+, severe growth defect in iron-poor media), (deletion of gene sib3+, inability to promote ferrichrome-dependent growth of S. cerevisiae)\n",
      "Progress: 53.60% completed\n",
      "Response for Row 1073:\n",
      "Answer to Question 2: (deletion of gene las21, impaired yeast growth on medium with cell wall stressors), (deletion of gene ecm33, impaired yeast growth on medium with cell wall stressors), (overexpression of gene ECM33, improved specific yield of p-coumaric acid), (deletion of gene ECM33, improved specific yield of p-coumaric acid), (overexpression of gene ECM33, improved specific yield of β-carotene), (deletion of gene ECM33, improved specific yield of β-carotene), (overexpression of gene ECM33, decreased chitin deposition), (deletion of gene ecm33, increased chitin deposition)\n",
      "Progress: 53.65% completed\n",
      "Response for Row 1074:\n",
      "Answer to Question 2: (deletion of gene GPD2, increase in ethanol production), (deletion of gene FPS1, increase in ethanol production), (deletion of gene ADH2, increase in ethanol production), (deletion of gene GPD2, decrease in glycerol content), (deletion of gene FPS1, decrease in glycerol content), (deletion of gene ADH2, decrease in glycerol content), (deletion of gene GPD2, decrease in lactic acid content), (deletion of gene FPS1, decrease in lactic acid content), (deletion of gene ADH2, decrease in lactic acid content), (deletion of gene GPD2, decrease in acetic acid content), (deletion of gene FPS1, decrease in acetic acid content), (deletion of gene ADH2, decrease in acetic acid content), (deletion of gene GPD2, decrease in succinic acid content), (deletion of gene FPS1, decrease in succinic acid content), (deletion of gene ADH2, decrease in succinic acid content)\n",
      "Progress: 53.70% completed\n",
      "Response for Row 1075:\n",
      "Answer to Question 2: (deletion of gene BAT2, decline in yields of 2-methyl-butyraldehyde), (deletion of gene BAT2, decline in yields of 3-methyl-butyraldehyde), (deletion of gene BAT2, decline in yields of 2-methyl-1-butanol), (deletion of gene BAT2, decline in yields of 3-methyl-1-butanol), (deletion of gene BAT2, increase in yields of 2-methylbutyl-acetate), (deletion of gene BAT2, increase in yields of 3-methylbutyl-acetate), (overexpression of Ile, increase in yields of 2-methyl-butyraldehyde), (overexpression of Ile, increase in yields of 2-methyl-1-butanol), (overexpression of Ile, increase in yields of 2-methylbutyl-acetate), (overexpression of Leu, increase in yields of 3-methyl-butyraldehyde), (overexpression of Leu, increase in yields of 3-methyl-1-butanol), (overexpression of Leu, increase in yields of 3-methylbutyl-acetate)\n",
      "Progress: 53.75% completed\n",
      "Response for Row 1076:\n",
      "Answer to Question 2: (deletion of gene xxx, beneficial for surviving some toxins), (deletion of gene xxx, detrimental for others)\n",
      "Progress: 53.80% completed\n",
      "Response for Row 1077:\n",
      "Answer to Question 2: (deletion of gene Gem1, consequence 1: decreased viability), (deletion of gene Gem1, consequence 2: increased mitochondrial H2O2 production), (deletion of gene Gem1, consequence 3: increased ER stress), (deletion of gene Gem1, consequence 4: increased growth in presence of mutant alpha-synuclein), (deletion of gene Gem1, consequence 5: reduced ER stress when expressing A53T alpha-synuclein), (deletion of gene Gem1, consequence 6: increased ability to deal with oxidative stress)\n",
      "Progress: 53.85% completed\n",
      "Response for Row 1078:\n",
      "Answer to Question 2: (deletion of protease gene, reduced protease activity), (deletion of sporulation gene, reduced sporulation), (deletion of autolysis-related gene, reduced autolysis), (overexpression of Sox, production of reactive hydrogen peroxide), (overexpression of Sox, undesired cross-linking of functional proteins), (overexpression of interleukin-1β, pro-inflammatory cytokine production)\n",
      "Progress: 53.90% completed\n",
      "Response for Row 1079:\n",
      "Answer to Question 2: (gene deletion of unknown gene, improved α-amylase secretion), (point mutation in unknown gene, improved α-amylase secretion)\n",
      "Progress: 53.95% completed\n",
      "Response for Row 1080:\n",
      "Answer to Question 2: (deletion of gene Aqr1, no impact at mild acetic acid stress), (deletion of gene Tpo2, decreased growth at severe acetic acid stress), (deletion of gene Tpo3, decreased growth at severe acetic acid stress), (deletion of gene Tpo2, crucial role in acetic acid tolerance), (deletion of gene AQR1, least effect on growth and acetate efflux), (deletion of gene Tpo2 and Tpo3, enhanced severe growth defects at 20 mM acetic acid), (deletion of gene Tpo2 and Tpo3, reduced rate of acetate secretion), (overexpression of gene TPO2, restored tolerance in ∆tpo2∆tpo3∆ strain), (overexpression of gene TPO3, restored tolerance in ∆tpo2∆tpo3∆ strain), (deletion of gene Tpo2 and Tpo3, accumulation of acetate intracellularly), (activation of gene ethanol metabolism, under acetic acid stress conditions), (reduction of gene acetic acid production, under acetic acid stress conditions)\n",
      "Progress: 54.00% completed\n",
      "Response for Row 1081:\n",
      "Answer to Question 2:  (deletion of gene ATF1, increase of 2-PE production by 114% in PK-2C strain), (deletion of gene ATF1, decrease of 2-PE yield from 3.50 to 0.83 g/L in CWY-132 strain), (deletion of gene ALD2, increase of 2-PE yield from 0.21 to 1.20 g/L in PK-2C strain), (deletion of gene ALD2, decrease of 2-PE yield from 3.50 to 3.02 g/L in CWY-132 strain), (deletion of gene ALD3, decrease of 2-PE yield from 3.50 to 2.93 g/L in CWY-132 strain), (deletion of genes ALD2 and ALD3, decrease of 2-PE yield to 1.65 g/L in CWY-132 strain), (supply of NADH, increase of 2-PE production in S. cerevisiae), (supply of GSH, increase of 2-PE production in S. cerevisiae)\n",
      "Progress: 54.05% completed\n",
      "Response for Row 1082:\n",
      "Answer to Question 2: (deletion of gene SPT10, increase in total phospholipids), (deletion of gene SPT10, insufficient accumulation of key metabolites), (deletion of gene SPT10, massive endoplasmic reticulum degradation), (overexpression of gene DGK1, stable massive ER structure), (overexpression of gene DGK1, decrease in autophagy), (overexpression of gene DGK1, increase in MEV/ERG pathway activity), (heterologous expression of β-amyrin synthase, production of triterpene β-amyrin)\n",
      "Progress: 54.10% completed\n",
      "Response for Row 1083:\n",
      "Answer to Question 2: (deletion of gene GAL80, increased hydroxytyrosol yield), (overexpression of gene paHD, improved catalytic performance for hydroxytyrosol production), (overexpression of gene aro4K229L, enhanced hydroxytyrosol precursor supply), (overexpression of gene aro7G141S, enhanced hydroxytyrosol precursor supply), (overexpression of gene AAS, enhanced tyrosine metabolic pathway for hydroxytyrosol production), (overexpression of gene Bbxfpkopt, redistributed metabolic flux for increased hydroxytyrosol production)\n",
      "Progress: 54.15% completed\n",
      "Response for Row 1084:\n",
      "Answer to Question 2: (expression of gene ERG1, consequence squalene loss reduction), (overexpression of rate-limiting enzyme, consequence increased squalene production), (deletion of non-essential competing pathway gene, consequence increased squalene production)\n",
      "Progress: 54.20% completed\n",
      "Response for Row 1085:\n",
      "Answer to Question 2: (deletion of gene CiSHT2, partially substituted spermine accumulation), (deletion of gene CiSHT1, CiSHT2 catalyzes first N-acylation steps), (expression of gene CiSHT2, partially substituted spermine accumulation), (coexpression of gene CiSHT2 and CiSHT1, synthesis and accumulation of fully substituted spermine), (CiSHT2 catalyzes, N5,N10-dicoumaroyl spermine formation), (CiSHT1 uses, N5,N10-dicoumaroyl spermine to synthesize tetracoumaroyl spermine)\n",
      "Progress: 54.25% completed\n",
      "Response for Row 1086:\n",
      "Answer to Question 2: (deletion of gene GAL7, involvement in HR repair pathway), (deletion of gene GAL7, involvement in NHEJ repair pathway), (deletion of gene YMR130W, involvement in HR repair pathway), (deletion of gene YMR130W, involvement in NHEJ repair pathway), (deletion of gene YHI9, involvement in HR repair pathway), (deletion of gene YHI9, involvement in NHEJ repair pathway), (deletion of gene GAL7, enhanced sensitivity to DNA damage-inducing drugs), (deletion of gene YMR130W, enhanced sensitivity to DNA damage-inducing drugs), (deletion of gene YHI9, enhanced sensitivity to DNA damage-inducing drugs), (deletion of gene GAL7, interaction with SIR2), (deletion of gene GAL7, interaction with SIR3), (deletion of gene GAL7, interaction with SIR4), (deletion of gene YMR130W, interaction with SIR2), (deletion of gene YMR130W, interaction with SIR3), (deletion of gene YMR130W, interaction with SIR4), (deletion of gene YHI9, interaction with SIR2), (deletion of gene YHI9, interaction with SIR3), (deletion of gene YHI9, interaction with SIR4)\n",
      "Progress: 54.30% completed\n",
      "Response for Row 1087:\n",
      "Answer to Question 2: (deletion of gene SNF1, affects exponential growth), (deletion of gene SNF1, affects mitochondrial respiration), (deletion of gene SNF1, affects transcript levels of hexose transporters), (deletion of gene HXK2, affects exponential growth), (deletion of gene HXK2, affects mitochondrial respiration), (deletion of gene HXK2, affects transcript levels of hexose transporters), (deletion of gene SNF1, correlates positively with Vmax of hexose transporters), (deletion of gene SNF1, correlates negatively with mitochondrial respiration), (deletion of gene HXK2, correlates positively with Vmax of hexose transporters), (deletion of gene HXK2, correlates negatively with mitochondrial respiration), (deletion of gene SNF1/HXK2/MIG1, most affects HXT2 gene transcript levels)\n",
      "Progress: 54.35% completed\n",
      "Response for Row 1088:\n",
      "Answer to Question 2: ( deletion of gene EHT1, consequence lower amounts of ethyl esters), ( deletion of gene EEB1, consequence lower amounts of ethyl esters), ( deletion of gene EEB1, consequence increased expression of ATF2), ( deletion of gene EEB1, consequence increased content of acetate esters), ( deletion of gene EEB1, consequence affected production of higher alcohols), ( deletion of gene EHT1 and EEB1, consequence distinct ethyl ester profiles)\n",
      "Progress: 54.40% completed\n",
      "Response for Row 1089:\n",
      "Answer to Question 2:  (overexpression of gene spt23, lower thermal tolerance), (deletion of gene spt23, higher survival rate), (deletion of gene spt23, higher biomass), (deletion of gene spt23, higher ethanol production), (deletion of gene spt23, lower unsaturated lipid acid content), (deletion of gene spt23, lower LTR richness), (deletion of gene spt23, higher transposon activity)\n",
      "Progress: 54.45% completed\n",
      "Response for Row 1090:\n",
      "Answer to Question 2: (loss of exons in DDS-encoding gene, lack of dammarane-type saponin production), (overexpression of PgDDS gene, recovery of dammarane-type saponins accumulation), (tandem duplication of AeCYP72As, synthesis of oleanane-type saponins), (tandem duplication of AeCSLMs, synthesis of oleanane-type saponins), (tandem duplication of AeUGT73s, synthesis of oleanane-type saponins and aralosides)\n",
      "Progress: 54.50% completed\n",
      "Response for Row 1091:\n",
      "Answer to Question 2: (deletion of gene VdAda1, consequence severe defects in the formation of conidia), (deletion of gene VdAda1, consequence severe defects in the formation of microsclerotia), (deletion of gene VdAda1, consequence defects in melanin biosynthesis), (deletion of gene VdAda1, consequence reduction in virulence), (deletion of gene VdAda1, consequence genome-wide alteration of the V. dahliae transcriptome), (deletion of gene VdAda1, consequence regulation of transcription factors expression), (deletion of gene VdAda1, consequence regulation of secreted proteins expression)\n",
      "Progress: 54.55% completed\n",
      "Response for Row 1092:\n",
      "Answer to Question 2: (deletion of gene FvMed1, increased FB1 production)\n",
      "Progress: 54.60% completed\n",
      "Response for Row 1093:\n",
      "Answer to Question 2: (deletion of gene Cbf1, consequence increased chromosome mis-segregation), (deletion of gene Cbf1, consequence accumulation of cenRNAs at all phases of the cell cycle)\n",
      "Progress: 54.65% completed\n",
      "Response for Row 1094:\n",
      "Answer to Question 2: (deletion of gene GAL80, elimination of galactose supplementation requirement), (deletion of gene GAL80, comparable artemisinic acid production), (mutation of gene URA3, affected growth), (mutation of gene URA3, reduced artemisinic acid production), (restoration of gene ura3, normal growth), (restoration of gene ura3, comparable artemisinic acid yield)\n",
      "Progress: 54.70% completed\n",
      "Response for Row 1095:\n",
      "Answer to Question 2: (deletion of gene MoOPY2, consequence hyphal growth defects), (deletion of gene MoOPY2, consequence conidiation defects), (deletion of gene MoOPY2, consequence germ tube extension defects), (deletion of gene MoOPY2, consequence appressorium formation defects), (deletion of gene MoOPY2, consequence appressorium turgor generation defects), (deletion of gene MoOPY2, consequence invasive growth defects), (deletion of gene MoOPY2, consequence attenuated pathogenicity), (deletion of gene MoOPY2, consequence cumulative MoAtg8 lipidation), (deletion of gene MoOPY2, consequence rapid GFP-MoAtg8 degradation)\n",
      "Progress: 54.75% completed\n",
      "Response for Row 1096:\n",
      "Answer to Question 2: (deletion of gene ATG32, increased fermentation performance), (deletion of gene ATG32, no significant differences in CO2 production), (deletion of gene ATG32, no significant differences in final ethanol titers), (deletion of gene ATG32, no significant differences in cell viability)\n",
      "Progress: 54.80% completed\n",
      "Response for Row 1097:\n",
      "Answer to Question 2: (deletion of gene involved in endocytosis, disruption of vesicle-mediated transport), (deletion of gene involved in vesicle-mediated transport, disruption of glycosylation), (deletion of gene encoding transferases, induction of cytotoxicity), (deletion of gene encoding proteases, induction of cytotoxicity), (deletion of gene encoding chaperones, disruption of metabolic processes)\n",
      "Progress: 54.85% completed\n",
      "Response for Row 1098:\n",
      "Answer to Question 2: (deletion of gene RIM21, improved flocculation capability), (deletion of gene RIM21, up-regulated expression of FLO5), (deletion of gene RIM21, up-regulated expression of Lg-FLO1), (deletion of gene RIM21, enhanced resistance to cell wall inhibitors)\n",
      "Progress: 54.90% completed\n",
      "Response for Row 1099:\n",
      "Answer to Question 2: (deletion of gene FgPEX22-like, consequence 1: impaired sexual and asexual reproduction), (deletion of gene FgPEX22-like, consequence 2: reduced fatty acid utilization), (deletion of gene FgPEX22-like, consequence 3: decreased pathogenicity), (deletion of gene FgPEX22-like, consequence 4: reduced production of deoxynivalenol), (deletion of gene FgPEX22-like, consequence 5: increased production of lipid droplets), (deletion of gene FgPEX22-like, consequence 6: decreased elimination of reactive oxygen species), (deletion of gene FgPEX22-like, consequence 7: disrupted biogenesis of Woronin bodies)\n",
      "Progress: 54.95% completed\n",
      "Response for Row 1100:\n",
      "Answer to Question 2: (overexpression of gene VlAbcG1a, increased tolerance to β-pinene), (deletion of gene VlAbcG1a homologous gene, increased sensitivity to certain monoterpenes), (overexpression of gene VlAbcG1a, increased virulence in tomato plants)\n",
      "Progress: 55.00% completed\n",
      "Response for Row 1101:\n",
      "Answer to Question 2: (deletion of gene ADA2, increase in efficiency of AMT), (deletion of gene ADA2, increase in double-strand DNA breaks)\n",
      "Progress: 55.05% completed\n",
      "Response for Row 1102:\n",
      "Answer to Question 2: (deletion of gene MoRPD3, lethal effect), (depletion of gene MoRPD3, reduction in pathogenicity), (depletion of gene MoRPD3, delay in appressorium-mediated penetration), (depletion of gene MoRPD3, impaired invasive growth), (depletion of gene MoRPD3, smaller appressoria size), (association of gene MoRPD3 with TOR kinase pathway, dependency of appressorium formation on hydrophilic surfaces), (association of gene MoRPD3 with TOR kinase pathway, dependency of appressorium formation on TOR's inhibition)\n",
      "Progress: 55.10% completed\n",
      "Response for Row 1103:\n",
      "Answer to Question 2: (gene deletion simulations, metabolic network models enable analysis), (integration of gene expression data, metabolic network models require GPR rules), (reconstruction of GPRs, manual and time-consuming process), (automating GPR reconstruction, GPRuler tool proposed), (GPRuler, reconstructs GPRs from organism name or existing model), (GPRuler performance, evaluated with small-scale and genome-scale metabolic models), (GPRuler, reproduces original GPR rules with high accuracy), (GPRuler, reveals more accurate than original models in some cases), (GPRuler, enables study of context-specific metabolic networks)\n",
      "Progress: 55.15% completed\n",
      "Response for Row 1104:\n",
      "Answer to Question 2: (deletion of open reading frame, consequence complete deletion), (insertion of gene fragments to essential genes, consequence median efficiency 52.1%), (introduction of single amino acid substitutions, consequence median efficiency 79.2%)\n",
      "Progress: 55.20% completed\n",
      "Response for Row 1105:\n",
      "Answer to Question 2: (deletion of gene Set1, reduced virulence of C. albicans), (deletion of gene Set1, increased cellular mitochondrial reactive oxygen species (ROS)), (deletion of gene Set1, increased H2O2 sensitivity), (deletion of gene Set1, prevents proper colony formation when interacting with macrophage), (deletion of gene Set1, defect in expression of mitochondrial protein genes), (deletion of gene Set1, attenuated virulence in vivo)\n",
      "Progress: 55.25% completed\n",
      "Response for Row 1106:\n",
      "Answer to Question 2: (deletion of eCIS-related genes, no effect on morphological differentiation), (deletion of eCIS-related genes, no effect on antibiotic production), (deletion of eCIS-related genes, increased susceptibility to fungal invasion), (SLP expression, elevated upon co-culture with fungi), (co-culture with Bacillus subtilis, decreased SLP expression), (co-culture with Bacillus subtilis, increased antibiotic production)\n",
      "Progress: 55.30% completed\n",
      "Response for Row 1107:\n",
      "Answer to Question 2: (deletion of gene KanMX, alteration in transcription efficiency of neighboring genes), (deletion of gene KanMX, impairment of mRNA translation of neighboring genes), (deletion of gene KanMX, change in protein abundance of neighboring genes), (deletion of gene KanMX, shift of transcription start site of neighboring genes in head-to-head orientation), (deletion of gene KanMX, introduction of new uAUG codons in 5' UTR of neighboring genes in head-to-head orientation), (deletion of gene KanMX, activation of alternative polyadenylation signals in neighboring genes in tail-to-tail arrangement), (deletion of gene KanMX, alteration in 3' UTR of neighboring genes in tail-to-tail arrangement)\n",
      "Progress: 55.35% completed\n",
      "Response for Row 1108:\n",
      "Answer to Question 2: (expression of gene ZFN, consequence 1), (expression of gene ZFN, consequence 2), (expression of gene ZFN, consequence 3), (deletion of gene LYS2, consequence 1), (insertion at gene LYS2, consequence 2)\n",
      "Progress: 55.40% completed\n",
      "Response for Row 1109:\n",
      "Answer to Question 2: (deletion of gene HOG1, increased rates of aerobic and anaerobic xylose fermentation), (deletion of gene ISU1, increased rates of aerobic and anaerobic xylose fermentation), (deletion of gene GRE3, increased rates of aerobic and anaerobic xylose fermentation), (deletion of gene IRA2, increased rates of aerobic and anaerobic xylose fermentation), (duplications of genes encoding engineered xylose metabolism enzymes, increased rates of aerobic and anaerobic xylose fermentation), (duplications of gene TKL1, increased rates of aerobic and anaerobic xylose fermentation)\n",
      "Progress: 55.45% completed\n",
      "Response for Row 1110:\n",
      "Answer to Question 2: (deletion of gene Akr1, increased intracellular ROS levels), (deletion of gene Akr1, increased cell death), (deletion of gene Erg3, increased intracellular ROS levels), (deletion of gene Erg3, increased cell death), (addition of calcium, inhibited increased intracellular ROS levels), (addition of calcium, inhibited cell death), (calcium induced expression of gene HXT1, maintained intracellular ROS levels), (calcium induced expression of gene AGP1, maintained intracellular ROS levels), (deletion of gene akr1, decreased cytosolic calcium concentration), (deletion of gene erg3, decreased cytosolic calcium concentration), (calcium/calcineurin signalling pathway induced expression of gene AKR1, helped yeast cells survive under high calcium concentrations), (calcium/calcineurin signalling pathway induced expression of gene ERG3, helped yeast cells survive under high calcium concentrations)\n",
      "Progress: 55.50% completed\n",
      "Response for Row 1111:\n",
      "Answer to Question 2: (deletion of gene NRRL3_09595, increased cell wall chitin), (deletion of gene NRRL3_09595, sensitivity to SDS), (deletion of gene NRRL3_09595, production of unknown yellow pigment), (deletion of gene cwcA, increased chitin content), (deletion of gene cwcA, constitutive activation of cell wall stress response pathway)\n",
      "Progress: 55.55% completed\n",
      "Response for Row 1112:\n",
      "Answer to Question 2: (overexpression of gene CAB1 W331R, level of CoA nucleotides increased 15-fold), (overexpression of wild-type CAB1, level of CoA nucleotides increased fourfold), (supplementation with pantothenate, level of CoA nucleotides increased 2.3-fold), (overexpression of gene FEN2, minor increase of CoA nucleotides), (deletion of gene PCD1, minor increase of CoA nucleotides)\n",
      "Progress: 55.60% completed\n",
      "Response for Row 1113:\n",
      "Answer to Question 2: (deletion of gene TRM1, decreased induction of AOX1 promoter), (deletion of gene TRM2, decreased induction of AOX1 promoter), (deletion of gene MPP1, decreased induction of AOX1 promoter), (overexpression of gene TRM1, increased induction of AOX1 promoter), (overexpression of gene MPP1, increased induction of AOX1 promoter), (presence of glycerol, catabolite repression of AOX1 expression), (presence of glucose, catabolite repression of AOX1 expression)\n",
      "Progress: 55.65% completed\n",
      "Response for Row 1114:\n",
      "Answer to Question 2: (deletion of A at position -132 nt upstream of Spt15, increased isobutanol tolerance), (insertion of G at position -65 nt upstream of Spt15, increased isobutanol tolerance), (synonymous mutation at position 315 nt downstream of Spt15, increased isobutanol tolerance), (overexpression of ILV3 gene, increased isobutanol production), (overexpression of ILV2 gene, increased isobutanol production), (overexpression of ARO10 gene, increased isobutanol production), (increased expression of SPT15, increased isobutanol tolerance), (downregulation of genes involved in TCA cycle, increased isobutanol tolerance), (downregulation of genes involved in glyoxylic acid cycle, increased isobutanol tolerance), (increased expression of genes related to cell stability, increased isobutanol tolerance)\n",
      "Progress: 55.70% completed\n",
      "Response for Row 1115:\n",
      "Answer to Question 2: (deletion of gene ADH2, reduced production of acetaldehyde), (deletion of gene THI3, reduced production of fusel alcohols), (deletion of gene ADH2 and THI3, reduced yields of acetaldehyde), (deletion of gene ADH2 and THI3, reduced yields of isoamyl alcohol), (deletion of gene ADH2 and THI3, reduced yields of iso-butanol)\n",
      "Progress: 55.75% completed\n",
      "Response for Row 1116:\n",
      "Answer to Question 2: (deletion of gene FpHsp104, severe defects in heat shock resistance), (deletion of gene FpHsp104, defects in conidia production), (deletion of gene FpHsp104, defects in conidia morphology), (deletion of gene FpHsp104, defects in conidia germination), (deletion of gene FpHsp104, reduced transcript levels of FpFluG), (deletion of gene FpHsp104, reduced transcript levels of FpVosA), (deletion of gene FpHsp104, reduced transcript levels of FpWetA), (deletion of gene FpHsp104, reduced transcript levels of FpAbaA), (deletion of gene FpHsp104, attenuated virulence)\n",
      "Progress: 55.80% completed\n",
      "Response for Row 1117:\n",
      "Answer to Question 2: (deletion of gene MBP1, decrease in final ethanol concentration under oxygen-limited conditions), (deletion of gene MBP1, decrease in respiration intensity under aerobic conditions), (deletion of gene MBP1, decrease in respiration intensity under oxygen-limited conditions), (deletion of gene MBP1, pseudohyphal growth), (deletion of gene MBP1, increase in cell volume), (deletion of gene MBP1, decrease in the fraction of budded cells)\n",
      "Progress: 55.85% completed\n",
      "Response for Row 1118:\n",
      "Answer to Question 2: (overexpression of gene Dga1p, increased TAG accumulation), (deletion of gene ACC1, increased TAG accumulation), (deletion of genes involved in glycogen synthesis, increased TAG accumulation), (deletion of genes involved in steryl ester synthesis, increased TAG accumulation), (deletion of genes involved in TAG hydrolysis, increased TAG accumulation), (deletion of gene CKB1, increased TAG accumulation)\n",
      "Progress: 55.90% completed\n",
      "Response for Row 1119:\n",
      "Answer to Question 2: (deletion of gene Gpa2, consequence regulating carbohydrate metabolism), (deletion of gene Asc1, consequence regulating amino acid metabolism), (deletion of gene Gpa2, consequence regulating purine metabolism), (deletion of gene Asc1, consequence regulating purine metabolism), (deletion of gene Gpa2, consequence determining amplitude of response to glucose addition), (deletion of gene Gpa2, consequence affecting cell growth), (deletion of gene Asc1, consequence affecting cell growth)\n",
      "Progress: 55.95% completed\n",
      "Response for Row 1120:\n",
      "Answer to Question 2: (deletion of HAC1, promotion of nuclear accumulation of Hog1), (deletion of IRE1, promotion of nuclear accumulation of Hog1), (deletion of HAC1, increase in cytosolic localization of Slt2), (deletion of IRE1, increase in cytosolic localization of Slt2), (deletion of HAC1, increase in bud neck localization of Slt2), (deletion of IRE1, increase in bud neck localization of Slt2)\n",
      "Progress: 56.00% completed\n",
      "Response for Row 1121:\n",
      "Answer to Question 2: (overexpression of gene ERG, increased squalene accumulation), (overexpression of gene HMG, increased squalene accumulation), (overexpression of gene DGA1, increased lipid production), (deletion of gene PEX10, increased lipid production), (deletion of gene URE2, increased lipid accumulation), (deletion of gene URE2, increased squalene content), (deletion of gene PEX10, deletion of gene URE2, positive synergistic effect on squalene production)\n",
      "Progress: 56.05% completed\n",
      "Response for Row 1122:\n",
      "Answer to Question 2: (deletion of PDC genes, elimination of ethanol production), (overexpression of PCAD, minimized PCA production), (insertion of DHS dehydratase from P. anserina, biosynthesis of muconic acid), (insertion of PCA decarboxylase from K. pneumoniae, biosynthesis of muconic acid), (insertion of CDO from C. albicans, biosynthesis of muconic acid), (increased yeast pitching rate, reduced biomass formation), (use of PPG, in situ product removal)\n",
      "Progress: 56.10% completed\n",
      "Response for Row 1123:\n",
      "Answer to Question 2:  (deletion of gene AoTgla, increase of total lipid contents), (deletion of gene AoTglb, increase of total lipid contents), (deletion of gene AoTgla, increase in TAG fraction), (deletion of gene AoTglb, increase in TAG fraction), (deletion of gene AoTgla, prolonged lipid-accumulating stage), (deletion of gene AoTglb, prolonged lipid-accumulating stage), (deletion of gene AoTgla, reduction in specific rate of lipid turnover), (deletion of gene AoTglb, reduction in specific rate of lipid turnover), (deletion of gene AoTgla, achievement of maximal lipid levels), (deletion of gene AoTglb, achievement of maximal lipid levels), (deletion of gene AoTgla, achievement of maximal TAG levels), (deletion of gene AoTglb, achievement of maximal TAG levels)\n",
      "Progress: 56.15% completed\n",
      "Response for Row 1124:\n",
      "Answer to Question 2: (deletion of gene GAT1, consequence lower availability of FAN), (deletion of gene GAT1, consequence lower yield of higher alcohols), (deletion of gene GAT1, consequence downregulation of GAP1), (deletion of gene GAT1, consequence downregulation of ARO9), (deletion of gene GAT1, consequence downregulation of ARO10)\n",
      "Progress: 56.20% completed\n",
      "Response for Row 1125:\n",
      "Answer to Question 2: (deletion of a single pivotal gene, extensive effects induced)\n",
      "Progress: 56.25% completed\n",
      "Response for Row 1126:\n",
      "Answer to Question 2: (deletion of polyasparagine domain in Azf1p, genes upregulated related to carbon metabolism), (deletion of polyasparagine domain in Azf1p, genes upregulated related to carbon storage)\n",
      "Progress: 56.30% completed\n",
      "Response for Row 1127:\n",
      "Answer to Question 2: (deletion of gene los1, extends replicative lifespan), (deletion of gene los1, increases extracellular level of anti-CD22 scFv), (deletion of gene los1, alters susceptibility to protein synthesis inhibitors)\n",
      "Progress: 56.35% completed\n",
      "Response for Row 1128:\n",
      "Answer to Question 2: (deletion of gene Pbp1, consequence slower growth on medium containing glycerol), (deletion of gene Pbp1, consequence slower growth on medium containing lactate), (deletion of gene Pbp1, consequence decreased expression of gene PCK1), (deletion of gene Pbp1, consequence decreased expression of gene FBP1), (deletion of gene Pbp1, consequence decreased expression of gene COX10), (deletion of gene Pbp1, consequence decreased expression of gene COX11), (deletion of gene Dcp1, consequence recovery of COX10 expression in pbp1Δ mutant), (deletion of gene Xrn1, consequence recovery of COX11 expression in pbp1Δ mutant)\n",
      "Progress: 56.40% completed\n",
      "Response for Row 1129:\n",
      "Answer to Question 2: (deletion of gene MIG1, no significant influence on ethanol production from xylose), (deletion of gene MIG1, no significant influence on ethanol production from glucose), (deletion of both genes MIG1 and MIG2, reduced ethanol production from xylose), (deletion of both genes MIG1 and MIG2, reduced ethanol production from glucose), (deletion of gene HAP4-A, increased ethanol production from xylose), (deletion of gene TUP1, increased ethanol production from xylose), (overexpression of gene HAP4-A, reduced ethanol production from xylose), (overexpression of gene TUP1, reduced ethanol production from xylose)\n",
      "Progress: 56.45% completed\n",
      "Response for Row 1130:\n",
      "Answer to Question 2: (deletion of gene IRC7, consequence 1), (deletion of gene IRC7, consequence 2), (deletion of gene IRC7, consequence 3)\n",
      "Progress: 56.50% completed\n",
      "Response for Row 1131:\n",
      "Answer to Question 2: (deletion of gene associated with mitochondria, mitochondrial damage), (deletion of gene associated with mitochondria, ROS accumulation), (deletion of gene associated with mitochondria, ATP content changes), (deletion of oxidative phosphorylation-related gene, higher ROS production), (deletion of oxidative phosphorylation-related gene, reduced ATP levels)\n",
      "Progress: 56.55% completed\n",
      "Response for Row 1132:\n",
      "Answer to Question 2: (deletion of gene KGD1, improved tolerance to adipic acid), (deletion of gene KGD1, improved tolerance to catechol), (disruption of ergosterol biosynthesis, adipic acid hypersensitivity), (disrupting protein trafficking, adipic acid hypersensitivity), (disrupting vacuolar transport, adipic acid hypersensitivity), (adipic acid, disrupts Membrane Compartment of Can1 (MCC)), (adipic acid, impacts endocytosis), (rapid internalization of Can1, vacuolar degradation)\n",
      "Progress: 56.60% completed\n",
      "Response for Row 1133:\n",
      "Answer to Question 2: (deletion of gene pdc1, ethanol conversion decreased by 30.19%), (deletion of gene pdc1, glycerol conversion increased by 40.005%), (deletion of gene pdc1, acetic acid concentration decreased by 43.54%), (deletion of gene pdc1, acetoin concentration increased by 12.79 times), (deletion of gene pdc1, pyruvate decarboxylase activity decreased by 40.91%), (deletion of gene pdc5, ethanol conversion decreased by 30.19%), (deletion of gene pdc5, glycerol conversion increased by 40.005%), (deletion of gene pdc5, acetic acid concentration decreased by 43.54%), (deletion of gene pdc5, acetoin concentration increased by 12.79 times), (deletion of gene pdc5, pyruvate decarboxylase activity decreased by 40.91%), (deletion of gene pdc1 and pdc5, production of 2,3-BD increased)\n",
      "Progress: 56.65% completed\n",
      "Response for Row 1134:\n",
      "Answer to Question 2: (deletion of gene ATF1, elevated citronellol titer), (overexpression of gene Erg20<sup>F96W</sup>, increased GPP), (overexpression of gene CrIS, produced S-citronellol)\n",
      "Progress: 56.70% completed\n",
      "Response for Row 1135:\n",
      "Answer to Question 2: (deletion of gene GPD1, minimizes glycerol production), (deletion of gene GPD2, minimizes glycerol production), (deletion of gene DLD1, prevents D-lactic acid conversion to pyruvate), (overexpression of gene LpDLDH, increases D-lactic acid production), (downregulation of gene ADH1, minimizes ethanol production)\n",
      "Progress: 56.75% completed\n",
      "Response for Row 1136:\n",
      "Answer to Question 2: (deletion of gene ARO8, reduced 2-PE content), (deletion of gene ARO8, reduced ethanol yield), (deletion of gene ARO8, decreased biofilm formation), (deletion of gene ARO8, reduced extracellular polysaccharides), (deletion of gene ARO8, reduced FLOp expression), (deletion of gene ARO9, reduced 2-PE content), (deletion of gene ARO9, reduced ethanol yield), (deletion of gene ARO9, decreased biofilm formation), (deletion of gene ARO9, reduced extracellular polysaccharides), (deletion of gene ARO9, reduced FLOp expression)\n",
      "Progress: 56.80% completed\n",
      "Response for Row 1137:\n",
      "Answer to Question 2: (deletion of gene xxx, accumulation of intracellular manganese), (deletion of gene xxx, manganese sensitivity), (addition of CaCl2, inhibition of manganese sensitivity), (regulation by calcium/calcineurin signaling pathway, interaction between manganese and calcium), (function of Pmr1, transport of Ca2+ and Mn2+)\n",
      "Progress: 56.85% completed\n",
      "Response for Row 1138:\n",
      "Answer to Question 2: (deletion of gene GSP2, increase in RNA content), (deletion of gene CTT1, increase in RNA content), (overexpression of gene HXT1, increase in RNA content)\n",
      "Progress: 56.90% completed\n",
      "Response for Row 1139:\n",
      "Answer to Question 2: (deletion of gene PaPRO1, sufficient production of PaE without degradation fragments), (deletion of gene PaPRO2, sufficient production of PaE without degradation fragments), (overexpression of gene PaE, increased production of PaE), (deletion of gene PaPRO2, maintenance of PaE activity after cultivation)\n",
      "Progress: 56.95% completed\n",
      "Response for Row 1140:\n",
      "Answer to Question 2: (deletion of gene adh1, consumption of geraniol), (deletion of gene adh1, production of geranic acid), (deletion of gene adh1, production of citronellic acid), (deletion of gene adh1, production of citronellol), (deletion of gene adh3, consumption of geraniol), (deletion of gene adh3, production of geranic acid), (deletion of gene adh3, production of citronellic acid), (deletion of gene adh3, production of citronellol), (deletion of gene adh4, consumption of geraniol), (deletion of gene adh4, production of geranic acid), (deletion of gene adh4, production of citronellic acid), (deletion of gene adh4, production of citronellol), (deletion of gene sfa1, consumption of geraniol), (deletion of gene sfa1, production of geranic acid), (deletion of gene sfa1, production of citronellic acid), (deletion of gene sfa1, production of citronellol)\n",
      "Progress: 57.00% completed\n",
      "Response for Row 1141:\n",
      "Answer to Question 2: (deletion of gene GID4, high malate production), (deletion of gene GID2, high malate production)\n",
      "Progress: 57.05% completed\n",
      "Response for Row 1142:\n",
      "Answer to Question 2: (deletion of gene PTH3, impaired respiratory growth), (deletion of gene PTH3, reduced protein expression of mtDNA-encoded genes), (presence of antibiotics, stalled mitoribosomes), (deletion of gene PTH4, less impairment of respiratory growth compared to PTH3), (tetracycline presence, reduced protein expression due to PTH3 deletion)\n",
      "Progress: 57.10% completed\n",
      "Response for Row 1143:\n",
      "Answer to Question 2: (deletion of gene dugB, increased GSH content), (deletion of gene dugB, reduced conidium production), (deletion of gene dugB, disturbed sexual development), (deletion of gene dugC, increased GSH content), (deletion of gene dugC, reduced conidium production), (deletion of gene dugC, disturbed sexual development), (deletion of gene dugB and dugC, downregulated steC), (deletion of gene dugB and dugC, downregulated sskB), (deletion of gene dugB and dugC, downregulated hogA), (deletion of gene dugB and dugC, downregulated mkkA), (deletion of gene dugB and dugC, downregulated flbA), (deletion of gene dugB and dugC, downregulated flbC), (deletion of gene dugB and dugC, downregulated flbE), (deletion of gene dugB and dugC, downregulated nosA), (deletion of gene dugB and dugC, downregulated rosA), (deletion of gene dugB and dugC, downregulated nsdC), (deletion of gene dugB and dugC, downregulated nsdD), (deletion of gene dugB and dugC, slowed GSH depletion during carbon starvation), (deletion of gene dugB and dugC, reduced reactive oxygen species accumulation), (deletion of gene dugB and dugC, decreased autolytic cell wall degradation), (deletion of gene dugB and dugC, decreased enzyme secretion), (deletion of gene dugB and dugC, increased sterigmatocystin formation)\n",
      "Progress: 57.15% completed\n",
      "Response for Row 1144:\n",
      "Answer to Question 2: (deletion of competing pathway genes, increased carbon flux towards desired product), (overexpression of pathway enzymes, increased pathway efficiency), (colocalization of enzymes, increased reaction likelihood along pathway)\n",
      "Progress: 57.20% completed\n",
      "Response for Row 1145:\n",
      "Answer to Question 2:  (over-expression of ILV2, increased isobutanol titers), (over-expression of ILV3, increased isobutanol titers), (over-expression of ARO10, increased isobutanol titers), (mutations in 59 genes, improved cell growth), (mutations in 59 genes, enhanced basal transcription factors), (mutations in 59 genes, improved cell integrity signaling), (mutations in 59 genes, enhanced translation initiation and elongation), (mutations in 59 genes, improved ribosome assembly and function), (mutations in 59 genes, enhanced oxidative stress response), (over-expression of CWP2, improved tolerance toward isobutanol), (over-expression of SRP4039, improved tolerance toward isobutanol)\n",
      "Progress: 57.25% completed\n",
      "Response for Row 1146:\n",
      "Answer to Question 2: (deletion of gene CLB5, reduced efficiency of replication initiation in rDNA), (deletion of gene CLB5, arrested forks more stably at the RFB), (deletion of gene CLB5, led to homologous recombination-dependent rDNA instability), (deletion of gene FOB1, did not completely suppress rDNA instability in clb5Δ)\n",
      "Progress: 57.30% completed\n",
      "Response for Row 1147:\n",
      "Answer to Question 2: (degradation of Erg20p, redirection of metabolic flux toward monoterpene production), (depletion of hexokinase-2, lifting of glucose repression), (depletion of hexokinase-2, boosting of sesquiterpene nerolidol production to 3.5 g L^-1), (depletion of Acc1p, growth arrest), (depletion of Acc1p, no diminishment of production capacity in nerolidol-producing yeast)\n",
      "Progress: 57.35% completed\n",
      "Response for Row 1148:\n",
      "Answer to Question 2: (deletion of gene MoMca1, reduced sporulation), (deletion of gene MoMca1, delay in conidial germination), (deletion of gene MoMca1, attenuation of disease severity), (deletion of gene MoMca2, reduced sporulation), (deletion of gene MoMca2, delay in conidial germination), (deletion of gene MoMca2, attenuation of disease severity), (deletion of gene MoMca1 and MoMca2, increased radial growth in the presence of oxidative stress), (deletion of gene MoMca1 and MoMca2, increased accumulation of insoluble aggregates)\n",
      "Progress: 57.40% completed\n",
      "Response for Row 1149:\n",
      "Answer to Question 2: (deletion of gene AcOTAbZIP, loss of OTA), (deletion of gene AcOTAbZIP, loss of OTB), (deletion of gene AcOTAbZIP, loss of OTβ), (deletion of gene AcOTAbZIP, down-regulation of AcOTApks), (deletion of gene AcOTAbZIP, down-regulation of AcOTAnrps), (deletion of gene AcOTAbZIP, down-regulation of AcOTAp450), (deletion of gene AcOTAbZIP, down-regulation of AcOTAhal)\n",
      "Progress: 57.45% completed\n",
      "Response for Row 1150:\n",
      "Answer to Question 2: (deletion of essential subunits of 26S proteasome, sensitivity to genotoxic stress), (deletion of essential subunits of 26S proteasome, sensitivity to proteotoxic stress), (deletion of essential subunits of 26S proteasome, sensitivity to high salinity), (deletion of essential subunits of 26S proteasome, sensitivity to high osmolarity)\n",
      "Progress: 57.50% completed\n",
      "Response for Row 1151:\n",
      "Answer to Question 2: (deletion of gene kcs1, increased SAM by 83.3%), (deletion of gene arg82, increased SAM by 31.8%), (deletion of gene kcs1, improved transcription levels of various glycolytic genes), (deletion of gene kcs1, enhanced activities of relative enzymes), (deletion of gene kcs1, enhanced levels of glycolytic intermediates), (deletion of gene kcs1, enhanced levels of ATP), (deletion of gene kcs1 in Ymls1ΔGAPmK, increased SAM to 2.89 g L^-1 in a 250-mL flask), (deletion of gene kcs1 in Ymls1ΔGAPmK, increased SAM to 8.86 g L^-1 in a 10-L fermentation tank)\n",
      "Progress: 57.55% completed\n",
      "Response for Row 1152:\n",
      "Answer to Question 2: (deletion of gene MSS2, impaired growth upon chitosan treatment), (deletion of gene MSS2, slow growth on glycerol), (deletion of gene MSS2, impaired ATP production), (deletion of gene MSS2, impaired invasive ability), (deletion of gene MSS2, impaired biofilm formation), (deletion of gene MSS2, reduced virulence)\n",
      "Progress: 57.60% completed\n",
      "Response for Row 1153:\n",
      "Answer to Question 2: (deletion of gene fab1, affects growth), (deletion of gene fab1, affects conidia development), (deletion of gene fab1, affects sclerotial development), (deletion of gene fab1, affects cellular acidification), (deletion of gene fab1, affects metal ion homeostasis), (deletion of gene fab1, affects aflatoxin production), (deletion of gene fab1, affects pathogenicity), (deletion of gene fab1, leads to impaired aflatoxin synthesis)\n",
      "Progress: 57.65% completed\n",
      "Response for Row 1154:\n",
      "Answer to Question 2: (deletion of gene ade2, brown colony phenotype), (expression of gene ScRad52, improved HR efficiency), (expression of gene ScRad52, gene targeting efficiencies up to 95%), (deletion of gene ku70, traditionally used strategy for improving HR efficiency)\n",
      "Progress: 57.70% completed\n",
      "Response for Row 1155:\n",
      "Answer to Question 2: (deletion of gene RME1, consequence abolishes chlamydosporulation), (overexpression of gene RME1, consequence bypasses requirement for chlamydosporulation cues and regulators)\n",
      "Progress: 57.75% completed\n",
      "Response for Row 1156:\n",
      "Answer to Question 2: (overexpression of gene ppt1, overcome inhibition of CoQ biosynthesis by Bz), (deletion of gene ppt1, inhibited CoQ biosynthesis by Bz), (addition of benzoic acid, inhibited CoQ biosynthesis), (addition of p-aminobenzoate, counteracted inhibition of CoQ biosynthesis by Bz), (addition of p-hydroxybenzoate, counteracted inhibition of CoQ biosynthesis by Bz)\n",
      "Progress: 57.80% completed\n",
      "Response for Row 1157:\n",
      "Answer to Question 2: (deletion of gene mpkB, reduction in conidia production), (deletion of gene mpkB, vesicular dome-immaturity in the conidiophore head), (deletion of gene mpkB, decreased phialide numbers), (deletion of gene mpkB, very short stalks), (deletion of gene mpkB, inability to produce sclerotia), (deletion of gene mpkB, normal production of aflatoxin B1)\n",
      "Progress: 57.85% completed\n",
      "Response for Row 1158:\n",
      "Answer to Question 2: (deletion of gene GDH1, total higher alcohol concentration decreased by 27.31%), (deletion of gene CAN1, total content of higher alcohols increased by 211.44%), (deletion of gene GAT1, total content of higher alcohols increased by 28.36%)\n",
      "Progress: 57.90% completed\n",
      "Response for Row 1159:\n",
      "Answer to Question 2: (deletion of gene SOR1, increased xylitol yield), (deletion of gene SOR2, increased xylitol yield), (deletion of gene XKS1, increased xylitol yield), (deletion of gene XKS1, weakened SOR1 expression), (deletion of gene XKS1, weakened SOR2 expression), (deletion of gene XKS1, increased xylitol accumulation)\n",
      "Progress: 57.95% completed\n",
      "Response for Row 1160:\n",
      "Answer to Question 2: (deletion of gene MTQ2, severe growth impairment), (deletion of gene MTQ2, impaired nucleolar release of pre-60S), (deletion of gene MTQ2, reduced production of mature 5.8S rRNA), (deletion of gene MTQ2, reduced production of mature 25S rRNA)\n",
      "Progress: 58.00% completed\n",
      "Response for Row 1161:\n",
      "Answer to Question 2: (deletion of HXT gene, chimeric HXT sequences), (deletion of HXT gene, interstitial deletions), (cleavage of HXT gene, gene conversions), (cleavage of HXT gene, deletion products), (impairment of SSA pathway, reduced survival frequency), (disruption of Rad51-dependent homologous recombination, modest reduction in survival frequency)\n",
      "Progress: 58.05% completed\n",
      "Response for Row 1162:\n",
      "Answer to Question 2: (deletion of gene Matα:Δ57-70, consequence increased G-CSF secretion), (deletion of gene Matα:Δ57-70, consequence highest G-CSF titre of 39.4 ± 1.4 mg/L), (deletion of gene Matα:Δ30-43, consequence diminished G-CSF secretion), (deletion of gene Matα:Δ47-49, consequence dampened G-CSF secretion), (deletion of gene Matα:Δ57-70;47-49, consequence stimulatory effect on G-CSF secretion), (codon optimization of gene GCSF, consequence overcoming control by signal peptide segments), (deletion of gene Matα:Δ57-70;30-43, consequence partial restoration of G-CSF secretion)\n",
      "Progress: 58.10% completed\n",
      "Response for Row 1163:\n",
      "Answer to Question 2: (overexpression of gene MAL62, increase in freezing tolerance), (overexpression of gene MAL62, increase in UDPG-dependent trehalose synthesis), (abolition of UDPG-dependent trehalose synthase activity, failure to promote intracellular trehalose synthesis upon MAL62 overexpression), (lack of trehalose synthesis, slight reduction in cell viability during prefermentation freezing with MAL62 overexpression), (MAL62 overexpression, increase in intracellular glycerol concentration), (MAL62 overexpression, elevated transcription of glycerol synthesis pathway members)\n",
      "Progress: 58.15% completed\n",
      "Response for Row 1164:\n",
      "Answer to Question 2: (deletion of gene TSA1, diminishes growth in medium with glucose), (deletion of gene TSA1, diminishes growth in medium with sucrose), (deletion of gene TSA1, diminishes growth in molasses), (deletion of gene TSA1, lowers trehalose accumulation in early growth stages), (deletion of gene TSA1, increases trehalose accumulation at end of batch growth), (deletion of gene TSA1, increases glycogen accumulation at entry of stationary phase), (deletion of gene TSA1, reduces fermentative capacity in grape juice), (deletion of gene TSA1, decreases acetic acid production), (deletion of gene TSA1, decreases acetaldehyde production)\n",
      "Progress: 58.20% completed\n",
      "Response for Row 1165:\n",
      "Answer to Question 2: (deletion of gene TPK1, reduces efficiency of NHEJ repair of breaks with overhangs), (deletion of gene TPK1, reduces efficiency of NHEJ repair of blunt ends), (deletion of gene TPK1, defects in the repair of chromosomal breaks induced by HO-site specific endonuclease)\n",
      "Progress: 58.25% completed\n",
      "Response for Row 1166:\n",
      "Answer to Question 2: (deletion of gene HOG1, consequence high sensitivity to high osmolarity), (deletion of gene HOG1, consequence reduction in trap formation), (deletion of gene HOG1, consequence reduction in predation efficiency), (deletion of gene HOG1, consequence required for conidiation), (deletion of gene MSB2, consequence high sensitivity to high osmolarity), (deletion of gene MSB2, consequence reduction in trap formation), (deletion of gene MSB2, consequence reduction in predation efficiency)\n",
      "Progress: 58.30% completed\n",
      "Response for Row 1167:\n",
      "Answer to Question 2: (deletion of gene AOX, consequence strong inhibition of growth), (deletion of gene AOX, consequence respiratory capacity reduced), (presence of gene AOX, consequence no effect of inhibitors on growth), (presence of gene AOX, consequence minor effect of inhibitors on growth)\n",
      "Progress: 58.35% completed\n",
      "Response for Row 1168:\n",
      "Answer to Question 2: (deletion of nuclear genes, consequence respiratory growth impairment), (deletion of twelve genes, consequence viability loss in absence of mtDNA), (upregulation of 176 nuclear genes, consequence expression of mitochondria-encoded proteins)\n",
      "Progress: 58.40% completed\n",
      "Response for Row 1169:\n",
      "Answer to Question 2: (deletion of gene CIT2, no effect on β-amyrin production), (deletion of gene MLS1, increased β-amyrin production by 1.85 times), (weakened expression of gene PGI1, increased β-amyrin yield by 3.75 times)\n",
      "Progress: 58.45% completed\n",
      "Response for Row 1170:\n",
      "Answer to Question 2: (deletion of gene TPO1, MCFOH production improvement), (expression of modified MmCAR, increased MCFOH production)\n",
      "Progress: 58.50% completed\n",
      "Response for Row 1171:\n",
      "Answer to Question 2: (deletion of gene MCK1, enrichment in all processes tested), (deletion of gene RIM11, enrichment in all processes tested), (deletion of gene MRK1, enrichment in all processes tested), (deletion of gene YGK3, enrichment in all processes tested), (deletion of gene MCK1, reflects presence of significant stresses), (deletion of gene RIM11, reflects presence of significant stresses), (deletion of gene MRK1, reflects presence of significant stresses), (deletion of gene YGK3, reflects presence of significant stresses), (deletion of gene MCK1, affected by feeding scheme), (deletion of gene RIM11, affected by feeding scheme), (deletion of gene MRK1, affected by feeding scheme), (deletion of gene YGK3, affected by feeding scheme)\n",
      "Progress: 58.55% completed\n",
      "Response for Row 1172:\n",
      "Answer to Question 2: (deletion of gene Cpl1, increased adhesive and invasive growth), (deletion of gene Cpl1, accumulation of intracellular labile zinc ion), (accumulation of intracellular labile zinc ion, increased glucose uptake), (accumulation of intracellular labile zinc ion, higher level of mitochondrial membrane potential), (accumulation of intracellular labile zinc ion, increased ATP production), (accumulation of intracellular labile zinc ion, increased ROS production), (increased ROS production, activation of Snf1), (activation of Snf1, invasive growth)\n",
      "Progress: 58.60% completed\n",
      "Response for Row 1173:\n",
      "Answer to Question 2: (ΔFglsb6 deletion, defects in vegetative growth), (ΔFglsb6 deletion, decreased deoxynivalenol (DON) production), (ΔFglsb6 deletion, reduced pathogenicity), (ΔFglsb6 deletion, increased resistance to osmotic stress), (ΔFglsb6 deletion, increased resistance to oxidative stress), (ΔFglsb6 deletion, increased resistance to cell wall stresses), (ΔFglsb6 deletion, defective generation of PI4P on endosomes), (ΔFglsb6 deletion, defective endocytosis)\n",
      "Progress: 58.65% completed\n",
      "Response for Row 1174:\n",
      "Answer to Question 2: (deletion of gene UvATG8, consequence vegetative growth defect), (deletion of gene UvATG8, consequence conidiation defect), (deletion of gene UvATG8, consequence oxidative stress adaptation defect), (deletion of gene UvATG8, consequence hyperosmotic stress adaptation defect), (deletion of gene UvATG8, consequence cell wall stress adaptation defect), (deletion of gene UvATG8, consequence production of toxic compounds defect), (deletion of gene UvATG8, consequence reduction in virulence), (deletion of gene UvATG8, consequence reduced conidiation), (deletion of gene UvATG8, consequence reduced secondary spore formation)\n",
      "Progress: 58.70% completed\n",
      "Response for Row 1175:\n",
      "Answer to Question 2: (deletion of A503 from Bp BteA, enhanced cytotoxic activity on mammalian HeLa cells), (deletion of A503 from Bp BteA, increased toxicity to Saccharomyces cerevisiae cells), (insertion of A503 into Bb BteA, decreased cytotoxicity to yeast cells), (insertion of A503 into Bb BteA, decreased cytotoxicity to HeLa cells), (expression of Bp BteAΔA503, increased virulence of B. pertussis B1917 in mouse model), (expression of Bp BteAΔA503, reduced LD50 in mouse model), (expression of Bp BteAΔA503, yielded less inflammatory pathology in infected mouse lungs at sublethal infectious doses)\n",
      "Progress: 58.75% completed\n",
      "Response for Row 1176:\n",
      "Answer to Question 2: (deletion of gene FLO400, consequence prevents pseudohyphae formation), (deletion of gene FLO5-1, consequence prevents pseudohyphae formation), (deletion of gene FLO400, consequence hampers FLO11 expression), (deletion of gene FLO5-1, consequence hampers FLO11 expression), (expression of gene FLO400, consequence leads to induction of FLO11), (expression of gene FLO5-1, consequence leads to induction of FLO11)\n",
      "Progress: 58.80% completed\n",
      "Response for Row 1177:\n",
      "Answer to Question 2: (deletion of gene xxx, improved gene editing efficiency), (expression of sgRNA, enhanced chromosomal gene integration), (expression of sgRNA, successful gene deletion)\n",
      "Progress: 58.85% completed\n",
      "Response for Row 1178:\n",
      "Answer to Question 2:  (overexpression of gene FHL1, consequence RNA content increased by 14.94%), (overexpression of gene SSF2, consequence RNA content increased by 24.4%), (deletion of gene HRP1, consequence RNA content increased by 19.36%), (overexpression of gene IFH1, consequence no significant change in RNA production), (overexpression of gene SSF1, consequence no significant change in RNA production), (overexpression of gene FHL1, consequence forms complex with Ifh1p to play a key role in transcription of ribosomal protein gene), (overexpression of gene IFH1, consequence forms complex with Fhl1p to play a key role in transcription of ribosomal protein gene), (overexpression of gene SSF2, consequence plays an important role in ribosome biosynthesis), (deletion of gene HRP1, consequence eliminates negative regulator of cell growth), (overexpression of gene FHL1, overexpression of gene IFH1, overexpression of gene SSF2, deletion of gene HRP1, consequence RNA content increased by 38.8%), (overexpression of gene FHL1, overexpression of gene IFH1, overexpression of gene SSF2, deletion of gene HRP1, consequence cell size increased), (overexpression of gene FHL1, overexpression of gene IFH1, overexpression of gene SSF2, deletion of gene HRP1, consequence growth rate increased)\n",
      "Progress: 58.90% completed\n",
      "Response for Row 1179:\n",
      "Answer to Question 2: (deletion of gene RIM15, hampered cell cycle arrest), (deletion of gene RIM15, enhanced biofilm formation), (deletion of gene RIM15, improved fermentation ability), (deletion of gene RIM15, increased expression level of adhesion protein FLO11), (deletion of gene RIM15, increased synthesis of trehalose)\n",
      "Progress: 58.95% completed\n",
      "Response for Row 1180:\n",
      "Answer to Question 2: (deletion of gene ARO10, reduced production of benzyl alcohol)\n",
      "Progress: 59.00% completed\n",
      "Response for Row 1181:\n",
      "Answer to Question 2: (deletion of gene GPD1, consequence 1), (deletion of gene FUM1, consequence 1), (overexpression of gene PYC2, consequence 1)\n",
      "Progress: 59.05% completed\n",
      "Response for Row 1182:\n",
      "Answer to Question 2: (deletion of gene RPH1, alleviates cell slow growth), (deletion of gene RPH1, derepression of rRNA transcription), (deletion of gene RPH1, derepression of RPG transcription), (phosphorylation of Rph1, resistance to rapamycin treatment), (phosphorylation of Rph1, decreased association with ribosome-related genes), (phosphorylation of Rph1, faster cell growth)\n",
      "Progress: 59.10% completed\n",
      "Response for Row 1183:\n",
      "Answer to Question 2: (deletion of DH1 domain, complete loss of activity in the biosynthesis of VLCPUFAs), (mutation of DH1 domain, complete loss of activity in the biosynthesis of VLCPUFAs), (deletion of DH2 domain, production of a small amount of DPA), (mutation of DH2 domain, production of a small amount of DPA), (DH1 domain, essential for the biosynthesis of VLCPUFAs), (DH2 domain, required for the biosynthesis of DHA)\n",
      "Progress: 59.15% completed\n",
      "Response for Row 1184:\n",
      "Answer to Question 2: (deletion of SSU1, reduced sulphite tolerance)\n",
      "Progress: 59.20% completed\n",
      "Response for Row 1185:\n",
      "Answer to Question 2: (overexpression of gene SUT2, promotes expression of AOX1), (overexpression of gene SUT2, increases ergosterol content), (expression of gene MXR1, elevated expression level of Mxr1), (expression of gene MXR1, high activity of P<sub>AOX1</sub>), (P<sub>AOX1</sub> expression, improved production of heterologous proteins by 18%)\n",
      "Progress: 59.25% completed\n",
      "Response for Row 1186:\n",
      "Answer to Question 2: (TRF5 deletion, increases RNA abundance for mRNAs specifically showing high Trf5 binding)\n",
      "Progress: 59.30% completed\n",
      "Response for Row 1187:\n",
      "Answer to Question 2: (deletion of gene cwcA, increase in cell wall chitin), (deletion of gene cwcA, sensitivity to SDS), (deletion of gene cwcA, production of unknown yellow pigment), (deletion of gene cwcA, constitutive activation of cell wall stress response pathway)\n",
      "Progress: 59.35% completed\n",
      "Response for Row 1188:\n",
      "Answer to Question 2: (expression of HXK2S14A, increased xylose consumption rate), (expression of HXK2S14A, increased ethanol production rate), (expression of HXK2S14A, increased ethanol yield), (deletion of MIG1, decreased xylose utilization), (deletion of MIG1, eliminated positive effect of Hxk2p)\n",
      "Progress: 59.40% completed\n",
      "Response for Row 1189:\n",
      "Answer to Question 2: (deletion of gene XRN1, impaired gene silencing in N. castellii), (deletion of gene XRN1, affecting the composition of siRNA species in the cell), (deletion of gene XRN1, influencing the efficiency of siRNA loading into Argonaute), (deletion of gene XRN1, degradation of cleaved passenger strand), (deletion of gene XRN1, degradation of sliced target RNA)\n",
      "Progress: 59.45% completed\n",
      "Response for Row 1190:\n",
      "Answer to Question 2: (expression of SbDGAT1-1, restored TAG accumulation capability), (expression of SbDGAT1-2, restored TAG accumulation capability), (deletion of N-terminal region in SbDGAT1-1, decreased TAG accumulation), (deletion of N-terminal region in SbDGAT1-1, decreased FA 16:0 accumulation), (deletion of N-terminal region in SbDGAT1-1, decreased FA 18:0 accumulation)\n",
      "Progress: 59.50% completed\n",
      "Response for Row 1191:\n",
      "Answer to Question 2: (deletion of gene thiamine transporter, consequence growth rate decrease), (deletion of gene thiamine transporter, consequence growth efficiency decrease), (deletion of gene thiamine transporter, consequence CO2 production decrease)\n",
      "Progress: 59.55% completed\n",
      "Response for Row 1192:\n",
      "Answer to Question 2: (deletion of gene Vta1, consequence colorless microsclerotia), (deletion of gene Vta1, consequence no melanization), (activation by Vta1, consequence transcription of PKS1), (activation by Vta1, consequence transcription of LAC1), (requirement of Vta1, consequence melanin production), (requirement of Vta1, consequence survival of microsclerotia)\n",
      "Progress: 59.60% completed\n",
      "Response for Row 1193:\n",
      "Answer to Question 2: (deletion of gene YPS1-5, consequence highly sensitive to α-factor pheromone), (deletion of gene YPS1-5, consequence mates with reduced efficiency)\n",
      "Progress: 59.65% completed\n",
      "Response for Row 1194:\n",
      "Answer to Question 2: (deletion of gene SET1, increase in serine-2 phosphorylation within the C-terminal domain of RNA polymerase II), (deletion of gene SET1, increase in recruitment of the cleavage/polyadenylation complex), (deletion of gene SET2, increase in serine-2 phosphorylation within the C-terminal domain of RNA polymerase II), (deletion of gene SET2, increase in recruitment of the cleavage/polyadenylation complex), (chemical inhibition of TOR signaling, Set1-dependent alternative polyadenylation), (chemical inhibition of TOR signaling, Set2-dependent alternative polyadenylation), (overexpression of gene SET1, decrease in efficiency of using single pA sites), (overexpression of gene SET2, decrease in efficiency of using single pA sites), (overexpression of gene SET1, control nucleosome occupancy around pA sites), (overexpression of gene SET2, control nucleosome occupancy around pA sites)\n",
      "Progress: 59.70% completed\n",
      "Response for Row 1195:\n",
      "Answer to Question 2: (deletion of gene tps1, elimination of yeast ability to grow on glucose as the sole carbon source in Saccharomyces cerevisiae), (deletion of gene tps1, no growth failure in glucose, galactose, or fructose in Kluyveromyces lactis), (deletion of gene tps1, no effect on yeast growth rate in Kluyveromyces lactis), (deletion of gene tps1, absence of trehalose in the metabolism of Kluyveromyces lactis)\n",
      "Progress: 59.75% completed\n",
      "Response for Row 1196:\n",
      "Answer to Question 2: (deletion of gene ECM33, consequence increased squalene production), (deletion of gene ECM33, consequence suppressed cell lysis), (deletion of gene ECM33, consequence improved stress tolerance), (activation of CWI pathway, consequence enhanced squalene production efficiency), (activation of CWI pathway, consequence restored membrane rigidity)\n",
      "Progress: 59.80% completed\n",
      "Response for Row 1197:\n",
      "Answer to Question 2: (deletion of gene MHY1, consequence 1), (deletion of gene MHY1, consequence 2), (deletion of gene MHY1, consequence 3)\n",
      "Progress: 59.85% completed\n",
      "Response for Row 1198:\n",
      "Answer to Question 2: (deletion of gene pex3, decrease of ethanol production from xylose), (deletion of gene cta1, impairment of ethanol production from xylose), (overexpression of gene pex34, activation of xylose alcoholic fermentation)\n",
      "Progress: 59.90% completed\n",
      "Response for Row 1199:\n",
      "Answer to Question 2: (integration of gene ARO10*, consequence YMGA* produced 0.14 mM tyrosol from 2% glucose without expensive media supplements), (deletion of gene feaB, consequence increased tyrosol production), (deletion of gene pheA, consequence increased tyrosol production), (deletion of gene tyrB, consequence increased tyrosol production), (deletion of gene tyrR, consequence increased tyrosol production), (increasing copy number of gene ARO10* to five copies, consequence increased tyrosol production to 10.92 mM), (optimizing fermentation conditions, consequence increased tyrosol production to 28 mM)\n",
      "Progress: 59.95% completed\n",
      "Response for Row 1200:\n",
      "Answer to Question 2: (deletion of gene RPN4, increased sensitivity to ethanol), (deletion of gene RPN4, no increase in autophagy activity upon ethanol stress), (deregulation of gene PRE1 leading to Rpn4 accumulation, increased PRB1 expression upon ethanol stress), (stabilization of Rpn4, contribution to PRB1 upregulation), (repression of Rpn4 binding sites in PRB1 promoter, inhibition of PRB1 induction and YPL hypersensitivity to ethanol)\n",
      "Progress: 60.00% completed\n",
      "Response for Row 1201:\n",
      "Answer to Question 2: (deletion of gene AaBRE1, loss of H2B monoubiquitination), (deletion of gene AaBRE1, decrease in H3K4me3 enrichment), (deletion of gene AaBRE1, suppression of transcription of genes encoding C2H2 zinc-finger family transcription factors), (deletion of gene AaBRE1, suppression of transcription of genes encoding cell wall-degrading enzymes), (deletion of gene AaBRE1, suppression of transcription of genes encoding chitin-binding proteins), (deletion of gene AaBRE1, reduced hyphal growth), (deletion of gene AaBRE1, reduced conidial formation), (deletion of gene AaBRE1, reduced pathogenicity)\n",
      "Progress: 60.05% completed\n",
      "Response for Row 1202:\n",
      "Answer to Question 2: (deletion of gene Aoglo3, growth defects), (deletion of gene Aoglo3, increase in hyphal septum), (deletion of gene Aoglo3, decrease in sporulation capacity), (deletion of gene Aoglo3, change in spore shape), (deletion of gene Aoglo3, decrease in spore germination rate), (deletion of gene Aoglo3, transcriptional repression of fluG), (deletion of gene Aoglo3, transcriptional repression of rodA), (deletion of gene Aoglo3, transcriptional repression of abaA), (deletion of gene Aoglo3, transcriptional repression of medA), (deletion of gene Aoglo3, transcriptional repression of lreA), (deletion of gene Aoglo3, sensitivity to Congo red), (deletion of gene Aoglo3, sensitivity to NaCl), (deletion of gene Aoglo3, sensitivity to sorbitol), (deletion of gene Aoglo3, involvement in endocytosis), (deletion of gene Aoglo3, increase in myelin figures), (deletion of gene Aoglo3, increase in autophagosomes), (deletion of gene Aoglo3, affect production of mycelial traps), (deletion of gene Aoglo3, affect production of serine proteases)\n",
      "Progress: 60.10% completed\n",
      "Response for Row 1203:\n",
      "Answer to Question 2: (deletion of sirtuins, affected relevant phenotypes in K. lactis), (deletion of sirtuins, no effect on phenotypes in S. cerevisiae)\n",
      "Progress: 60.15% completed\n",
      "Response for Row 1204:\n",
      "Answer to Question 2: (deletion of gene xxx, increased sensitivity to SDS), (deletion of gene xxx, elevated intracellular ROS levels), (overexpression of antioxidant defense genes, up-regulation due to SDS stress)\n",
      "Progress: 60.20% completed\n",
      "Response for Row 1205:\n",
      "Answer to Question 2: (overexpression of gene Candida tropicalis XR, increased xylose consumption), (overexpression of gene Candida tropicalis XR, increased xylitol yield), (overexpression of gene S. cerevisiae Gre3p aldose reductase, increased xylose consumption), (overexpression of gene S. cerevisiae Gre3p aldose reductase, increased xylitol yield), (overexpression of gene C. tropicalis XR mutant, decreased xylose consumption), (overexpression of gene C. tropicalis XR mutant, lower xylitol yield), (replacement of ZWF1 promoter with GPD promoter, increased xylitol yield), (deletion of gene XYL2, no increase in xylitol yield), (deletion of gene SOR1, no increase in xylitol yield)\n",
      "Progress: 60.25% completed\n",
      "Response for Row 1206:\n",
      "Answer to Question 2: (deletion of gene YHB1, consequence down-regulation), (deletion of gene PRB1, consequence clipping Histone H3 N-terminal tails)\n",
      "Progress: 60.30% completed\n",
      "Response for Row 1207:\n",
      "Answer to Question 2: (deletion of gene gsy1, inefficient sporulation), (deletion of gene gsy2, inefficient sporulation), (deletion of gene tps1, inefficient sporulation), (deletion of gene gsy1, partial compensation by trehalose), (deletion of gene gsy2, partial compensation by trehalose), (deletion of gene tps1, partial compensation by glycogen), (deletion of gene cln3, increase in cell size), (deletion of gene cln3, increase in mitochondria), (deletion of gene cln3, increase in lipid stores), (deletion of gene cln3, rescue of meiosis progression), (deletion of gene cln3, rescue of spore ascus formation), (deletion of gene gsy1, increased dependency on mitochondrial activity), (deletion of gene gsy2, increased dependency on mitochondrial activity), (deletion of gene tps1, increased dependency on mitochondrial activity), (deletion of gene gsy1, increased dependency on lipid stores), (deletion of gene gsy2, increased dependency on lipid stores), (deletion of gene tps1, increased dependency on lipid stores)\n",
      "Progress: 60.35% completed\n",
      "Response for Row 1208:\n",
      "Answer to Question 2: (deletion of gene PP_2426, decrease in vanillyl alcohol yield), (expression of gene FEZ21_09870, conversion of vanillin to vanillyl alcohol), (expression of gene PP_2426, conversion of vanillin to vanillyl alcohol), (expression of gene PP_3839, oxidation of coniferyl alcohol to coniferyl aldehyde)\n",
      "Progress: 60.40% completed\n",
      "Response for Row 1209:\n",
      "Answer to Question 2: (deletion of gene warA, sensitivity to sorbic acid), (deletion of gene warA, sensitivity to propionic acid), (deletion of gene warA, sensitivity to benzoic acid), (overexpression of gene An14g03570, resistance to weak acids), (deletion of gene pdrA, reduced resistance to sorbic acid)\n",
      "Progress: 60.45% completed\n",
      "Response for Row 1210:\n",
      "Answer to Question 2: (deletion of gene, increased cobalt sensitivity), (deletion of gene, increased intracellular cobalt accumulation), (deletion of gene, increased intracellular ROS levels), (deletion of gene, decreased expression of oxidative stress scavenging genes), (deletion of gene, sensitivity to tunicamycin), (deletion of gene, induction of unfolded protein response), (deletion of gene, activation of serine/threonine kinase Ire1), (deletion of gene, activation of endoribonuclease Ire1)\n",
      "Progress: 60.50% completed\n",
      "Response for Row 1211:\n",
      "Answer to Question 2: (deletion of gene xxx, high efficiency of gene alteration), (introduction of site-specific polymorphism, high efficiency of gene alteration), (insertion of tag, high efficiency of gene alteration), (cleavage within GFP, replacing GFP tag with other sequences)\n",
      "Progress: 60.55% completed\n",
      "Response for Row 1212:\n",
      "Answer to Question 2: (deletion of gene PSK1, unstable EMO), (deletion of gene PSK1, disappearance of fluctuations in EMO), (deletion of gene PSK1, reduced amplitude of EMO), (deletion of gene PSK1, downregulation of glycogen synthesis), (deletion of gene PSK1, downregulation of glycogen breakdown), (deletion of gene PSK1, decrease in glucose concentrations), (deletion of gene PSK1, decrease in NADH levels)\n",
      "Progress: 60.60% completed\n",
      "Response for Row 1213:\n",
      "Answer to Question 2: (deletion of gene atg22, delays acetic acid-induced programmed cell death), (deletion of gene atg22, maintains cell wall integrity), (deletion of gene atg22, protects cytomembrane integrity), (deletion of gene atg22, changes cytomembrane phospholipids), (deletion of gene atg22, changes sterols), (deletion of gene atg22, changes fatty acids), (deletion of gene atg22, increases intracellular amino acids), (overexpression of gene atg22, enhances acetic acid-induced programmed cell death), (deletion of gene atg22, upregulates heat shock protein family expression), (deletion of gene atg22, upregulates cell wall integrity related genes expression), (deletion of gene atg22, upregulates autophagy related genes expression)\n",
      "Progress: 60.65% completed\n",
      "Response for Row 1214:\n",
      "Answer to Question 2: (deletion of a 23.8 kb genomic region, increased p-CA production), (deletion of a 23.8 kb genomic region, upregulation of ENO2), (deletion of a 23.8 kb genomic region, upregulation of TKL1), (deletion of a 23.8 kb genomic region, upregulation of ARO1), (deletion of a 23.8 kb genomic region, upregulation of ARO2), (deletion of a 23.8 kb genomic region, upregulation of ARO4), (deletion of a 23.8 kb genomic region, upregulation of ARO7), (deletion of a 23.8 kb genomic region, upregulation of TYR1), (deletion of a 23.8 kb genomic region, upregulation of AVT6), (deletion of a 23.8 kb genomic region, upregulation of VBA5), (deletion of a 23.8 kb genomic region, upregulation of THI72), (deletion of a 23.8 kb genomic region, upregulation of ARO80)\n",
      "Progress: 60.70% completed\n",
      "Response for Row 1215:\n",
      "Answer to Question 2: (over-expression of p53, induction of apoptosis), (over-expression of p53, targeting of mitochondria), (over-expression of p53, accumulation of PPP enzymes), (over-expression of p53, enhanced mitochondrial NADPH production), (over-expression of HXK-2, enhanced association with mitochondria), (over-expression of HXK-2, accumulation of G6PDH), (over-expression of HXK-2, accumulation of 6PGDH), (over-expression of HXK-2, increased NADPH), (over-expression of HXK-2, scavenging of ROS), (deletion of hxk2, defective growth), (deletion of hxk2, lack of G6PDH accumulation), (deletion of hxk2, lack of 6PGDH accumulation)\n",
      "Progress: 60.75% completed\n",
      "Response for Row 1216:\n",
      "Answer to Question 2: (deletion of gene HEM25, lower ethanol production), (expression of gene HEM25, higher ethanol production)\n",
      "Progress: 60.80% completed\n",
      "Response for Row 1217:\n",
      "Answer to Question 2: (deletion of gene a1S, effect on mating-type gene expression at MATa locus), (deletion of gene a1S, effect on mating-type gene expression at MATα locus)\n",
      "Progress: 60.85% completed\n",
      "Response for Row 1218:\n",
      "Answer to Question 2: (deletion of gene GND1, hypersensitivity to isobutanol), (deletion of gene ZWF1, hypersensitivity to isobutanol), (deletion of gene GLN3, increased tolerance to branched-chain alcohols), (upregulation of gene GLN3, induces nitrogen starvation response), (upregulation of gene GCN4, induces nitrogen starvation response), (disruption of nitrogen starvation response by deletion of GLN3, enhances tolerance to isobutanol), (disruption of nitrogen starvation response by deletion of GLN3, boosts isobutanol production 4.9-fold)\n",
      "Progress: 60.90% completed\n",
      "Response for Row 1219:\n",
      "Answer to Question 2: (deletion of gene rpe1, consequence 1), (deletion of gene pda1, consequence 1), (deletion of gene adh3, consequence 1), (deletion of gene adh1, consequence 1), (deletion of gene zwf1, increased l-PAC formation), (deletion of gene pdc1, consequence 1)\n",
      "Progress: 60.95% completed\n",
      "Response for Row 1220:\n",
      "Answer to Question 2:  (deletion of gene FpDEP1, retarded growth on PDA plates), (deletion of gene FpDEP1, reduced aerial hyphae formation), (deletion of gene FpDEP1, no typical leaf lesions), (deletion of gene FpDEP1, limited expansion capability of coleoptiles), (deletion of gene FpDEP1, massive reactive oxygen species (ROS) accumulation), (deletion of gene FpDEP1, limited expansion to neighboring cells), (deletion of gene FpDEP1, sensitivity to H2O2), (deletion of gene FpDEP1, sensitivity to SDS), (deletion of gene FpDEP1, delayed endocytosis), (expression of gene FpDEP1-GFP, complementation of mutant phenotypes), (expression of gene FpDEP1-GFP, localization to the nucleus in spores and mycelia)\n",
      "Progress: 61.00% completed\n",
      "Response for Row 1221:\n",
      "Answer to Question 2: (deletion of gene mkt1, consequence 1), (deletion of gene mkt1, consequence 2), (deletion of gene mkt1, consequence 3), (deletion of gene pbp1, consequence 1), (deletion of gene pbp1, consequence 2)\n",
      "Progress: 61.05% completed\n",
      "Response for Row 1222:\n",
      "Answer to Question 2: (deletion of gene VdSho1, reduced penetration), (deletion of gene VdSho1, defective melanin biosynthesis), (deletion of gene Vst50, defective penetration), (deletion of gene Vst50, defective melanin production), (deletion of gene Vst7, defective penetration), (deletion of gene Vst7, defective melanin production), (deletion of gene Vst11, defective penetration), (deletion of gene Vst11, defective melanin production), (overexpression of gene Vst50, restores penetration), (overexpression of gene Vst50, restores melanin production)\n",
      "Progress: 61.10% completed\n",
      "Response for Row 1223:\n",
      "Answer to Question 2: (prime editing of HBB, correction of sickle cell disease cause), (prime editing of HEXA, correction of Tay-Sachs disease cause), (prime editing of PRNP, installation of protective transversion)\n",
      "Progress: 61.15% completed\n",
      "Response for Row 1224:\n",
      "Answer to Question 2: (deletion of gene transcription factors, consequence reduced ethanol yields on biomass), (deletion of gene transcription factors, consequence involvement in cell cycling), (deletion of gene transcription factors, consequence involvement in chromatin remodeling), (deletion of gene transcription factors, consequence involvement in transcription), (deletion of gene transcription factors, consequence involvement in stress response), (deletion of gene transcription factors, consequence involvement in protein synthesis), (deletion of gene transcription factors, consequence involvement in lipid synthesis)\n",
      "Progress: 61.20% completed\n",
      "Response for Row 1225:\n",
      "Answer to Question 2: (deletion of gene Pop2, increased ERCs formation), (deletion of gene Pop2, accumulation of transcripts from E-pro), (deletion of gene Pop2, reduced cohesin amounts), (deletion of gene Pop2, reduced condensin amounts), (deletion of gene Pop2, decreased rRNA amount), (deletion of gene Ccr4, similar unstable rDNA phenotype), (deletion of gene Not4, similar unstable rDNA phenotype)\n",
      "Progress: 61.25% completed\n",
      "Response for Row 1226:\n",
      "Answer to Question 2: (deletion of chromosomal regions, enhanced by CRISPR/Cas9), (replacement of chromosomal regions, enhanced by CRISPR/Cas9), (integration of CRISPR/Cas9, activates homologous recombination), (multiple chromosomal regions manipulation, enabled by CRISPR-PCD), (multiple chromosomal regions manipulation, enabled by CRISPR-PCRep)\n",
      "Progress: 61.30% completed\n",
      "Response for Row 1227:\n",
      "Answer to Question 2: (deletion of gene sucC, increase in SAM titer), (expression of gene SAM2, increase in SAM titer), (expression of gene metA, improvement in SAM production), (expression of gene metB, improvement in SAM production), (expression of gene mccA, improvement in SAM production)\n",
      "Progress: 61.35% completed\n",
      "Response for Row 1228:\n",
      "Answer to Question 2: (expression of gene ldhA, production of lactic acid), (deletion of gene JEN1 function, shift in product formation towards ethanol), (deletion of gene ADY2 function, shift in product formation towards ethanol), (upregulation of gene JEN1, production of lactic acid), (upregulation of gene ADY2, production of lactic acid)\n",
      "Progress: 61.40% completed\n",
      "Response for Row 1229:\n",
      "Answer to Question 2: (deletion of gene PHO13, increased arabinose consumption rate), (deletion of gene PHO13, increased specific ethanol productivity), (deletion of gene PHO13, activation of gene TAL1), (activation of gene TAL1, depletion of sedoheptulose), (deletion of gene PHO13, upregulation of pentose phosphate pathway), (upregulation of pentose phosphate pathway, trehalose accumulation), (deletion of gene PHO13, downregulation of TCA cycle)\n",
      "Progress: 61.45% completed\n",
      "Response for Row 1230:\n",
      "Answer to Question 2: (deletion of gene xxx, high efficiency gene knockout), (deletion of gene xxx, simultaneous deletion of two different genes), (expression of Cas9, mediated by inducible promoter), (expression of gRNA, mediated by constitutive promoter)\n",
      "Progress: 61.50% completed\n",
      "Response for Row 1231:\n",
      "Answer to Question 2: (deletion of gene GPD1, consequence 1), (deletion of gene GPD1, redox imbalance resolved), (deletion of gene GPD1, growth defect), (deletion of gene GPD2, consequence 1), (deletion of gene GPD2, redox imbalance resolved), (deletion of gene GPD2, growth defect), (overexpression of NADH oxidase, redox imbalance resolved), (introduction of Candida tropicalis PDC1, growth defect resolved), (introduction of Candida tropicalis PDC1, minimized ethanol production), (deletion of GPD1 and GPD2, glycerol production eliminated), (engineered strain BD5_T2 nox_dGPD1,2_CtPDC1, 99.4 g/L 2,3-BD produced), (fed-batch fermentation, 108.6 g/L 2,3-BD produced), (fed-batch fermentation, negligible glycerol production)\n",
      "Progress: 61.55% completed\n",
      "Response for Row 1232:\n",
      "Answer to Question 2: (deletion of gene gal80, consequence artemisinic acid production without galactose induction)\n",
      "Progress: 61.60% completed\n",
      "Response for Row 1233:\n",
      "Answer to Question 2: (deletion of gene TPK2, increased cAMP levels), (deletion of gene TPK2, decreased PKA activity), (deletion of gene BCY1, increased cAMP levels), (deletion of gene BCY1, decreased PKA activity), (deletion of gene TPK1, increased cAMP levels), (deletion of gene TPK1, decreased PKA activity), (deletion of gene TPK3, increased cAMP levels), (deletion of gene TPK3, decreased PKA activity), (overexpression of cAMP, improved heat resistance), (overexpression of cAMP, no influence on growth), (overexpression of cAMP, no influence on fermentation properties)\n",
      "Progress: 61.65% completed\n",
      "Response for Row 1234:\n",
      "Answer to Question 2: (MAL62 overexpression, enhanced maltose metabolism), (MAL62 overexpression, trehalose synthesis before fermentation), (MAL62 overexpression, similar effect on cell viability after freezing as TPS1 overexpression), (NTH1 deletion, strengthened freezing tolerance with MAL62 overexpression), (NTH1 deletion, strengthened freezing tolerance with TPS1 overexpression), (NTH1 deletion, improved leavening ability with MAL62 overexpression), (NTH1 deletion, improved leavening ability with TPS1 overexpression)\n",
      "Progress: 61.70% completed\n",
      "Response for Row 1235:\n",
      "Answer to Question 2: (overexpression of gene Ilv2, increase of isobutanol production), (overexpression of gene Ilv5, increase of isobutanol production), (overexpression of gene Ilv3, increase of isobutanol production), (deletion of gene ILV2, increase of isobutanol production), (deletion of gene BDH1, increase of isobutanol production), (deletion of gene BDH2, increase of isobutanol production), (deletion of gene LEU4, increase of isobutanol production), (deletion of gene LEU9, increase of isobutanol production), (deletion of gene ECM31, increase of isobutanol production), (deletion of gene ILV1, increase of isobutanol production), (deletion of gene ADH1, increase of glycerol formation), (deletion of gene GPD1, increase of isobutanol production), (deletion of gene GPD2, increase of isobutanol production), (deletion of gene ALD6, increase of isobutanol production)\n",
      "Progress: 61.75% completed\n",
      "Response for Row 1236:\n",
      "Answer to Question 2: (overexpression of gene New1, rescues inviability of yeast strain lacking eEF3), (loss of gene New1, leads to ribosome queuing upstream of 3'-terminal lysine and arginine codons)\n",
      "Progress: 61.80% completed\n",
      "Response for Row 1237:\n",
      "Answer to Question 2: (deletion of gene Ded1, reduction in translation efficiency of many mRNAs), (deletion of gene Dbp1, reduction in translation efficiency of many mRNAs), (overexpression of gene Dbp1, improvement in translation of Ded1-hyperdependent mRNAs), (deletion of gene Ded1, increased dependence on Dbp1 for translation), (deletion of gene Dbp1, increased dependence on Ded1 for translation), (expression of gene Dbp1, acceleration of 48S PIC assembly on structured 5'-UTRs), (deletion of gene Ded1, decreased PIC attachment and scanning on many mRNAs), (deletion of gene Dbp1, decreased PIC attachment and scanning on many mRNAs)\n",
      "Progress: 61.85% completed\n",
      "Response for Row 1238:\n",
      "Answer to Question 2: (deletion of gene FvSec4, no impairment in expression of key conidiation-related genes), (deletion of gene FvSec4, no defect in sexual development), (deletion of gene FvSec4, defect in corn stalk rot virulence), (deletion of gene FvSec4, alteration of fumonisin B1 production), (deletion of gene FvSec4, higher sensitivity to oxidative stress), (deletion of gene FvSec4, higher sensitivity to cell wall stress agents), (deletion of gene FvSec4, defects in carbon utilization)\n",
      "Progress: 61.90% completed\n",
      "Response for Row 1239:\n",
      "Answer to Question 2: (expression of gene LI1035, consequence inhibits yeast growth), (expression of gene LI1035, consequence interacts with components of the MAPK pathway), (expression of gene LI1035, consequence inhibits MAPK signaling cascades in yeast), (expression of gene LI1035, consequence inhibits MAPK signaling cascades in mammalian cells), (expression of gene LI1035, consequence regulates actin organization in yeast), (expression of gene LI1035, consequence regulates actin organization in mammalian cells)\n",
      "Progress: 61.95% completed\n",
      "Response for Row 1240:\n",
      "Answer to Question 2: (deletion of gene ATG2, failure to increase chronological lifespan), (deletion of gene ATG32, failure to increase chronological lifespan), (overexpression of gene ATG32, increase in autophagy), (overexpression of gene SOD1, participation in antiaging effect of CuB), (overexpression of gene SOD2, participation in antiaging effect of CuB), (overexpression of gene UTH1, participation in antiaging effect of CuB), (overexpression of gene SKN7, participation in antiaging effect of CuB), (increase in Sod activity, increased survival rate under oxidative stress), (decrease in ROS, reduced oxidative damage), (decrease in MDA production, reduced lipid peroxidation)\n",
      "Progress: 62.00% completed\n",
      "Response for Row 1241:\n",
      "Answer to Question 2: (expression of mfsG, consequence 1: increased efflux ability of fungitoxic compounds), (deletion of mfsG, consequence 1: decreased efflux ability), (deletion of mfsG, consequence 2: increased accumulation of isothiocyanates), (deletion of mfsG, consequence 3: increased sensitivity to isothiocyanates), (deletion of mfsG, consequence 4: reduced virulence towards plants containing glucosinolates)\n",
      "Progress: 62.05% completed\n",
      "Response for Row 1242:\n",
      "Answer to Question 2: (deletion of various gene targets, greater than 97% gene disruption efficiency), (deletion of double gene, disruption efficiency of 90%), (deletion of triple gene, disruption efficiency of 47%)\n",
      "Progress: 62.10% completed\n",
      "Response for Row 1243:\n",
      "Answer to Question 2: (deletion of gene TPS1, consequence decreased trehalose), (deletion of gene TPS1, consequence decreased Val), (deletion of gene TPS1, consequence decreased Thr), (deletion of gene TPS1, consequence decreased Lys), (deletion of gene TPS1, consequence decreased Asp), (deletion of gene TPS1, consequence decreased His), (deletion of gene TPS1, consequence decreased Trp), (deletion of gene TPS1, consequence decreased malonate), (deletion of gene TPS1, consequence decreased citrate), (deletion of gene TPS1, consequence decreased uridine), (deletion of gene TPS1, consequence decreased guanosine), (deletion of gene TPS1, consequence decreased inosine), (deletion of gene TPS1, consequence decreased AMP), (deletion of gene TPS1, consequence decreased C10:0), (deletion of gene TPS1, consequence decreased C16:1), (deletion of gene TPS2, consequence decreased trehalose), (deletion of gene TPS2, consequence decreased Val), (deletion of gene TPS2, consequence decreased Thr), (deletion of gene TPS2, consequence decreased Lys), (deletion of gene TPS2, consequence decreased Asp), (deletion of gene TPS2, consequence decreased His), (deletion of gene TPS2, consequence decreased Trp), (deletion of gene TPS2, consequence decreased malonate), (deletion of gene TPS2, consequence decreased citrate), (deletion of gene TPS2, consequence decreased uridine), (deletion of gene TPS2, consequence decreased guanosine), (deletion of gene TPS2, consequence decreased inosine), (deletion of gene TPS2, consequence decreased AMP), (deletion of gene TPS2, consequence decreased C10:0), (deletion of gene TPS2, consequence decreased C16:1), (deletion of gene TPS1, consequence inhibited glycolysis), (deletion of gene TPS1, consequence inhibited tricarboxylic acid cycle), (deletion of gene TPS1, consequence promoted phosphopentose pathway), (deletion of gene TPS1, consequence promoted nucleotide synthesis), (deletion of gene TPS2, consequence inhibited glycolysis), (deletion of gene TPS2, consequence inhibited tricarboxylic acid cycle), (deletion of gene TPS2, consequence promoted phosphopentose pathway), (deletion of gene TPS2, consequence promoted nucleotide synthesis), (deletion of gene TPS1, consequence reduced acetyl-CoA-mediated mycotoxin biosynthesis), (deletion of gene TPS2, consequence reduced acetyl-CoA-mediated mycotoxin biosynthesis), (deletion of gene TPS2, consequence increased fatty acids), (deletion of gene TPS2, consequence impaired trehalose biosynthesis), (deletion of gene TPS2, consequence trehalose 6-phosphate accumulation)\n",
      "Progress: 62.15% completed\n",
      "Response for Row 1244:\n",
      "Answer to Question 2: (deletion of gene BAT2, higher alcohol reduction), (overexpression of gene ATF1, ester improvement)\n",
      "Progress: 62.20% completed\n",
      "Response for Row 1245:\n",
      "Answer to Question 2: (deletion of pCS, change in fluorescence intensity), (expression of pCS, drives EGFP expression), (heat stress, increases fluorescence intensity), (deletion of pCS between -500 bp and -400 bp, alters heat response), (mutation of pCS between -490 bp and -500 bp, affects heat-inducible element)\n",
      "Progress: 62.25% completed\n",
      "Response for Row 1246:\n",
      "Answer to Question 2: (deletion of gene FRA2, upregulation of iron-sulfur metabolism), (expression of gene xylB, part of Weimberg pathway), (expression of gene xylD, part of Weimberg pathway), (expression of gene xylX, part of Weimberg pathway), (expression of gene xylA, not functional in S. cerevisiae), (replacement of gene xylA with ksaD, higher in vitro activity observed), (loss of gene XylB, minimizing xylonate accumulation), (introduction of gene xylD, part of lower Weimberg pathway), (introduction of gene xylX, part of lower Weimberg pathway), (introduction of gene ksaD, part of lower Weimberg pathway), (expression of Weimberg pathway, detection of intermediates and biomass generation from xylose)\n",
      "Progress: 62.30% completed\n",
      "Response for Row 1247:\n",
      "Answer to Question 2: (deletion of gene pcr4401, consequence albino phenotype), (heterologous expression of gene pcr4401, consequence melanin precursor YWA1 synthase), (heterologous expression of uncharacterized PKS gene from Aspergillus, consequence biosynthetic products elucidated), (heterologous expression of two uncharacterized PKS genes from Penicillium strains, consequence biosynthetic products elucidated)\n",
      "Progress: 62.35% completed\n",
      "Response for Row 1248:\n",
      "Answer to Question 2: (deletion of gene ira2, consequence specific xylose consumption rate increase), (deletion of gene ira2, consequence ethanol production rate increase), (deletion of gene ira2, consequence biomass titre decrease), (deletion of gene isu1, consequence specific xylose consumption rate increase), (deletion of gene isu1, consequence ethanol production rate increase), (deletion of gene isu1, consequence biomass titre decrease), (deletion of gene hog1, consequence xylose uptake improvement), (deletion of gene ira2, consequence signalling of high glucose), (deletion of gene isu1, consequence signalling of high glucose), (deletion of gene ira2, consequence signalling of low glucose), (deletion of gene isu1, consequence signalling of low glucose)\n",
      "Progress: 62.40% completed\n",
      "Response for Row 1249:\n",
      "Answer to Question 2: (deletion of gene FgPEX1, reduced mycelial growth), (deletion of gene FgPEX1, reduced conidiation), (deletion of gene FgPEX1, production of perithecia reduced), (deletion of gene FgPEX1, shortage of acetyl-CoA), (deletion of gene FgPEX1, reduced deoxynivalenol production), (deletion of gene FgPEX1, inhibited virulence), (deletion of gene FgPEX1, increased accumulation of lipid droplets), (deletion of gene FgPEX1, increased endogenous reactive oxygen species), (deletion of gene FgPEX1, involvement in cell wall integrity maintenance), (deletion of gene FgPEX1, involvement in Woronin bodies maintenance), (deletion of gene FgPEX10, reduced mycelial growth), (deletion of gene FgPEX10, reduced conidiation), (deletion of gene FgPEX10, production of perithecia reduced), (deletion of gene FgPEX10, shortage of acetyl-CoA), (deletion of gene FgPEX10, reduced deoxynivalenol production), (deletion of gene FgPEX10, inhibited virulence), (deletion of gene FgPEX10, increased accumulation of lipid droplets), (deletion of gene FgPEX10, increased endogenous reactive oxygen species), (deletion of gene FgPEX10, involvement in cell wall integrity maintenance), (deletion of gene FgPEX10, involvement in Woronin bodies maintenance)\n",
      "Progress: 62.45% completed\n",
      "Response for Row 1250:\n",
      "Answer to Question 2: (deletion of gene SIR2, increased rDNA origin efficiency), (deletion of gene SIR2, relocalization of Mcm2-7 helicases)\n",
      "Progress: 62.50% completed\n",
      "Response for Row 1251:\n",
      "Answer to Question 2: (SNF1 deletion, decreased glycolytic flux), (SNF1 deletion, increased NAD(P)H concentration), (SNF1 deletion, enhanced HXK2 gene transcription), (SNF1 deletion, decreased mitochondrial respiration), (SNF1 deletion, increased mitochondrial respiration)\n",
      "Progress: 62.55% completed\n",
      "Response for Row 1252:\n",
      "Answer to Question 2: (deletion of gene tir1, reduction in higher alcohol production by 23.5%), (deletion of gene gap1, reduction in higher alcohol production by 19.66%)\n",
      "Progress: 62.60% completed\n",
      "Response for Row 1253:\n",
      "Answer to Question 2: (overexpression of LmSAP, increased tolerance to salt and osmotic stresses), (overexpression of LmSAP, increased SOD activity), (overexpression of LmSAP, increased CAT activity), (overexpression of LmSAP, increased POD activity), (overexpression of LmSAP, decreased MDA levels), (overexpression of LmSAPΔAN1, increased tolerance to salt and osmotic stresses), (overexpression of LmSAPΔAN1, increased SOD activity), (overexpression of LmSAPΔAN1, increased CAT activity), (overexpression of LmSAPΔAN1, increased POD activity), (overexpression of LmSAPΔAN1, decreased MDA levels), (deletion of A20 domain in LmSAP, reduced tolerance to salt and osmotic stresses), (deletion of A20 domain in LmSAP, reduced SOD activity), (deletion of A20 domain in LmSAP, reduced CAT activity), (deletion of A20 domain in LmSAP, reduced POD activity), (deletion of A20 domain in LmSAP, increased MDA levels), (deletion of both A20 and AN1 domains in LmSAP, reduced tolerance to salt and osmotic stresses), (deletion of both A20 and AN1 domains in LmSAP, reduced SOD activity), (deletion of both A20 and AN1 domains in LmSAP, reduced CAT activity), (deletion of both A20 and AN1 domains in LmSAP, reduced POD activity), (deletion of both A20 and AN1 domains in LmSAP, increased MDA levels)\n",
      "Progress: 62.65% completed\n",
      "Response for Row 1254:\n",
      "Answer to Question 2: (deletion of gene OSW2, improved activity of encapsulated AI), (deletion of gene OSW2, no damage to thermostability of AI), (deletion of gene OSW2, no damage to pH tolerance of AI), (deletion of gene OSW2, no damage to resistance toward SDS of AI), (deletion of gene OSW2, no damage to resistance toward proteinase treatments of AI), (expression of gene AI, production of l-ribulose from l-arabinose), (expression of gene AI, 25% conversion rate of l-arabinose to l-ribulose)\n",
      "Progress: 62.70% completed\n",
      "Response for Row 1255:\n",
      "Answer to Question 2: (deletion of linker 2, increase in gene expression of weak and medium-strength terminators), (deletion of linker 2, decrease in gene expression of strong terminators), (reduction of stem length, increase in gene expression of weak and medium-strength terminators), (reduction of stem length, decrease in gene expression of strong terminators)\n",
      "Progress: 62.75% completed\n",
      "Response for Row 1256:\n",
      "Answer to Question 2: (deletion of gene pmr1, consequence less susceptibility to growth inhibition from artemisinin and its derivatives), (deletion of gene pmr1, consequence elevated basal ROS levels), (deletion of gene pmr1, consequence no enhancement of ROS accumulation upon artemisinin exposure), (deletion of gene pmr1, consequence accumulation of excess manganese ions)\n",
      "Progress: 62.80% completed\n",
      "Response for Row 1257:\n",
      "Answer to Question 2: (gene up-regulation, increased metabolic reaction activity), (gene down-regulation, decreased metabolic reaction activity), (introduction of heterologous genes, altered metabolic pathways), (gene deletion, altered metabolic pathways)\n",
      "Progress: 62.85% completed\n",
      "Response for Row 1258:\n",
      "Answer to Question 2: (overexpression of gene STB5, decreased growth rate), (overexpression of gene STB5, increased free fatty acid production), (overexpression of gene STB5, wide-ranging effects on the transcriptome), (overexpression of gene STB5, reduced flux through the oxidative PPP), (overexpression of gene STB5, induction of target genes GND1/2, TAL1, ALD6, YEF1)\n",
      "Progress: 62.90% completed\n",
      "Response for Row 1259:\n",
      "Answer to Question 2: (deletion of gene mpkB, increase of DHN-melanin production), (deletion of gene mpkA, normalization of DHN-melanin production), (deletion of gene gpaA, increase of DHN-melanin production), (deletion of gene gprM, increase of DHN-melanin production)\n",
      "Progress: 62.95% completed\n",
      "Response for Row 1260:\n",
      "Answer to Question 2: (deletion of gene URA3, successful targeted disruption), (deletion of gene CAR2, editing efficiencies reached roughly 50%)\n",
      "Progress: 63.00% completed\n",
      "Response for Row 1261:\n",
      "Answer to Question 2: (deletion of gene tptA, decreased resistance to iron chelator BPS), (deletion of gene tptA, severe growth defects), (deletion of gene tptA, decreased expression of hapX), (overexpression of gene hapX, rescued growth defect), (overexpression of gene hapX, increased siderophore production), (deletion of gene tptA, attenuation of virulence in murine model)\n",
      "Progress: 63.05% completed\n",
      "Response for Row 1262:\n",
      "Answer to Question 2: (deletion of gene YER161C (SPT2), increased alkali tolerance)\n",
      "Progress: 63.10% completed\n",
      "Response for Row 1263:\n",
      "Answer to Question 2: (deletion of gene IRA2, promotes xylose growth and metabolism), (deletion of gene BCY1, decouples growth from metabolism to enable robust fermentation without division)\n",
      "Progress: 63.15% completed\n",
      "Response for Row 1264:\n",
      "Answer to Question 2: (deletion of gene xxx, 87% efficiency), (deletion of gene xxx, 60% efficiency), (deletion of gene xxx, 23% efficiency), (deletion of eight genes, 30-fold increase in free fatty acid production)\n",
      "Progress: 63.20% completed\n",
      "Response for Row 1265:\n",
      "Answer to Question 2: (deletion of gene YPL062W, upregulation of pyruvate dehydrogenase bypass), (deletion of gene YPL062W, upregulation of mevalonate pathway), (deletion of gene YPL062W, upregulation of heterologous expression of GAL promoter-regulated genes), (deletion of gene YPL062W, upregulation of energy metabolism), (deletion of gene YPL062W, upregulation of membrane composition synthesis), (deletion of gene YPL062W, increased production of carotenoids), (deletion of gene YPL062W, increased production of all major terpenoid classes), (deletion of gene YPL062W, increased expression of ALD6 negatively correlated to terpenoid productivity)\n",
      "Progress: 63.25% completed\n",
      "Response for Row 1266:\n",
      "Answer to Question 2: (deletion of gene TRX1, reduced growth), (deletion of gene TRX1, impaired sugar consumption), (deletion of gene TRX1, reduction in ribosomal proteins), (deletion of gene TRX1, reduction in translation elongation factors), (deletion of gene TRX1, reduction in glycolysis enzymes), (deletion of gene TRX1, reduction in amino acid biosynthesis enzymes), (deletion of gene TRX1, increase in proteogenic amino acids), (deletion of gene TRX1, increase in phospholipids), (deletion of gene TRX1, increase in sphingolipids), (deletion of gene TRX1, higher fatty acid desaturase Ole1p content), (deletion of gene TRX1, low glycolytic activity), (deletion of gene TRX1, downregulation of all hexokinases), (deletion of gene TRX1, early aggregation of pyruvate kinase Cdc19p), (deletion of gene TRX2, reduced growth), (deletion of gene TRX2, impaired sugar consumption), (deletion of gene TRX2, reduction in ribosomal proteins), (deletion of gene TRX2, reduction in translation elongation factors), (deletion of gene TRX2, reduction in glycolysis enzymes), (deletion of gene TRX2, reduction in amino acid biosynthesis enzymes), (deletion of gene TRX2, increase in proteogenic amino acids), (deletion of gene TRX2, increase in phospholipids), (deletion of gene TRX2, increase in sphingolipids), (deletion of gene TRX2, higher fatty acid desaturase Ole1p content), (deletion of gene TRX2, low glycolytic activity), (deletion of gene TRX2, downregulation of all hexokinases), (deletion of gene TRX2, early aggregation of pyruvate kinase Cdc19p)\n",
      "Progress: 63.30% completed\n",
      "Response for Row 1267:\n",
      "Answer to Question 2: (deletion of gene VPS21, consequence severe defect in the AP-3 pathway), (deletion of gene YPT7, consequence severe defect in the AP-3 pathway), (deletion of gene VPS21, consequence defect in the CPY pathway), (deletion of gene YPT7, consequence defect in the CPY pathway), (activation of gene YPT7, consequence mediates vacuolar fusion), (deletion of gene VPS21, consequence cargos trafficked via the VPS or endocytic pathway accumulate beside nucleus), (deletion of gene YPT7, consequence cargos trafficked via the VPS or endocytic pathway accumulate beside nucleus), (deletion of gene VPS21, consequence cargo trafficked via the AP-3 pathway disperse in the cytosol), (deletion of gene YPT7, consequence cargo trafficked via the AP-3 pathway disperse in the cytosol)\n",
      "Progress: 63.35% completed\n",
      "Response for Row 1268:\n",
      "Answer to Question 2: (deletion of gene IRC7, reduced thiol release), (mutation of gene IRC7, reduced cysteine-S-conjugate β-lyase activity), (variation of gene IRC7, modulated formation of volatile sulfur compounds)\n",
      "Progress: 63.40% completed\n",
      "Response for Row 1269:\n",
      "Answer to Question 2: (deletion of gene HapB, consequence 1), (tagging of gene HapB, consequence 2)\n",
      "Progress: 63.45% completed\n",
      "Response for Row 1270:\n",
      "Answer to Question 2: (deletion of gene rtfA, consequence 1), (deletion of gene rtfA, consequence 2), (deletion of gene rtfA, consequence 3), (deletion of gene rtfA, consequence 4), (deletion of gene rtfA, consequence 5), (deletion of gene rtfA, consequence 6), (deletion of gene rtfA, consequence 7), (deletion of gene rtfA, consequence 8), (deletion of gene rtfA, consequence 9), (deletion of gene rtfA, consequence 10) \n",
      "\n",
      "- consequence 1: impaired normal pathogenicity in plants\n",
      "- consequence 2: impaired pathogenicity in animals\n",
      "- consequence 3: reduced adhesion to surfaces\n",
      "- consequence 4: decreased protease activity\n",
      "- consequence 5: decreased lipase activity\n",
      "- consequence 6: altered cell wall composition\n",
      "- consequence 7: compromised cell wall integrity\n",
      "- consequence 8: reduced tolerance to oxidative stress\n",
      "- consequence 9: decreased production of aflatoxin B1\n",
      "- consequence 10: decreased production of aflatrem, leporins, aspirochlorine, ditryptophenaline, and aflavinines\n",
      "Progress: 63.50% completed\n",
      "Response for Row 1271:\n",
      "Answer to Question 2: (deletion of gene ptn1, reduction in mating efficiency), (deletion of gene ptn1, reduction in virulence), (deletion of gene ptn1, reduction in tumor formation), (deletion of gene ptn1, reduction in mature teliospores), (deletion of gene ptn1, reduction in spore germination)\n",
      "Progress: 63.55% completed\n",
      "Response for Row 1272:\n",
      "Answer to Question 2: (deletion of gene gpd1, consequence increase in ethanol), (deletion of gene gpd2, consequence increase in ethanol), (deletion of gene gpd1, consequence reduction in glycerol), (deletion of gene gpd2, consequence reduction in glycerol), (deletion of gene ssk1, consequence improved ethanol yields), (deletion of gene smp1, consequence improved ethanol yields), (deletion of gene hot1, consequence tested but no osmosensitivity), (deletion of gene ssk1, consequence decrease in glycerol yield), (deletion of gene smp1, consequence decrease in glycerol yield), (deletion of gene ssk1, consequence decrease in acetic acid yields), (deletion of gene smp1, consequence decrease in acetic acid yields)\n",
      "Progress: 63.60% completed\n",
      "Response for Row 1273:\n",
      "Answer to Question 2: (overexpression of key genes associated with fatty acid synthesis, increased lycopene accumulation), (overexpression of OLE1, modulation of TAG fatty acyl composition), (deletion of FLD1, regulation of lipid-droplet size), (modulation of TAG fatty acyl composition, increased lycopene accumulation)\n",
      "Progress: 63.65% completed\n",
      "Response for Row 1274:\n",
      "Answer to Question 2: (deletion of gene products functioning in gene expression, deviation from gamma distribution of cell size), (deletion of gene products functioning in transcription or transcription-linked processes, deviation from gamma distribution of cell size), (altered activity substitution in Rpo21p/Rpb1, increased cell size), (extreme altered activity Pol II substitutions, deviation from expected gamma distribution of cell size)\n",
      "Progress: 63.70% completed\n",
      "Response for Row 1275:\n",
      "Answer to Question 2: (deletion of gene PFK-2, reduced glycolytic flux), (deletion of gene PFK-2, eliminated pathway to produce fructose-2,6-bisphosphate), (expression of gene MTH1-ΔT, lowered glucose uptake rate), (down-regulation of gene PFK-1, reduced glucose metabolism), (down-regulation of gene Pyk1, reduced glucose metabolism), (combined strategies, improved N-acetylglucosamine production)\n",
      "Progress: 63.75% completed\n",
      "Response for Row 1276:\n",
      "Answer to Question 2: (deletion of gene STL1, decreased growth in high salt concentrations), (deletion of gene STL1, disturbed intracellular pH homeostasis), (expression of gene STL1, improved growth under osmotic stress), (expression of gene SkSTL1 in Saccharomyces cerevisiae, complemented osmosensitivity of hog1Δ stl1Δ mutant)\n",
      "Progress: 63.80% completed\n",
      "Response for Row 1277:\n",
      "Answer to Question 2: (deletion of gene FET3, sensitivity to EGCG), (deletion of gene FTR1, sensitivity to EGCG), (presence of EGCG, induction of Cu(I) production), (surplus of Cu(II) and EGCG, synergistic deleterious effect on wild type cells)\n",
      "Progress: 63.85% completed\n",
      "Response for Row 1278:\n",
      "Answer to Question 2: (deletion of gene aro7, consequence increased mandelic acid production), (expression of gene hmaS, consequence production of hydroxymandelic acid), (expression of gene pheAfbr, consequence increased mandelic acid production), (expression of gene aro7 and pha2, consequence compartmentalization of mandelic acid biosynthesis pathway)\n",
      "Progress: 63.90% completed\n",
      "Response for Row 1279:\n",
      "Answer to Question 2: (deletion of gene FLO5, consequence 1), (deletion of gene FLO5, re-organization of the cell wall), (deletion of gene FLO5, finding different adhesion ways), (expression of gene FLO5, up-regulation of genes involved in stress response), (expression of gene FLO5, higher alcohol production)\n",
      "Progress: 63.95% completed\n",
      "Response for Row 1280:\n",
      "Answer to Question 2: (deletion of gene RIM15, improved alcoholic fermentation), (deletion of gene CDC55, abolished high fermentation performance), (enhanced TORC1 signaling, increased initial fermentation rate), (elevated PP2A^B55δ activity, high fermentation performance)\n",
      "Progress: 64.00% completed\n",
      "Response for Row 1281:\n",
      "Answer to Question 2: (deletion of gene, genome modification), (integration of gene TAL, biosynthesis of resveratrol), (integration of gene 4CL, biosynthesis of resveratrol), (integration of gene STS, biosynthesis of resveratrol), (fusion expression of gene TAL, increased resveratrol production), (fusion expression of gene 4CL, increased resveratrol production), (fusion expression of gene STS, increased resveratrol production), (integration of gene HSA, biosynthesis of human serum albumin), (integration of gene cadA, biosynthesis of cadaverine)\n",
      "Progress: 64.05% completed\n",
      "Response for Row 1282:\n",
      "Answer to Question 2: (deletion of gene GcnE, abrogated conidiation), (deletion of gene GcnE, impeded vegetative growth of hyphae), (deletion of gene GcnE, increase of glutamine synthetase activity), (deletion of gene gbpA, affected conidiogenesis), (deletion of gene glnA, affected glutamine biosynthesis)\n",
      "Progress: 64.10% completed\n",
      "Response for Row 1283:\n",
      "Answer to Question 2: (deletion of gene FZO1, reduces mitophagy), (deletion of gene FZO1, activates retrograde response), (activation of retrograde response, extends replicative lifespan), (deletion of gene FZO1, accumulates Atg32-Atg11-Dnm1 complexes), (accumulation of Atg32-Atg11-Dnm1 complexes, engages mitophagy receptor Atg32)\n",
      "Progress: 64.15% completed\n",
      "Response for Row 1284:\n",
      "Answer to Question 2: (deletion of gene YALI0E25135g, decrease in resistance to hyperosmotic stress), (deletion of gene YALI0E25135g, negative effect on erythritol production), (deletion of gene YALI0E25135g, morphological defects), (deletion of gene YALI0E25135g, resistance to Congo red), (deletion of gene YALI0E25135g, resistance to calcofluor white)\n",
      "Progress: 64.20% completed\n",
      "Response for Row 1285:\n",
      "Answer to Question 2: (Deletion of gene aro8, decrease in methionol production by 44% in BY4743), (Deletion of gene aro8, decrease in methionol production by 92% in F15), (Deletion of gene aro9, increase in methionol production by 46% in F15), (Deletion of gene aro10, no effect on methionol production), (Deletion of genes aro8 and aro9, increase in methionol production in F15 compared to single Δaro8), (Deletion of genes aro8, aro9, and aro10, increase in methionol production in F15 compared to single Δaro8)\n",
      "Progress: 64.25% completed\n",
      "Response for Row 1286:\n",
      "Answer to Question 2: (deletion of gene ADH2, improved ethanol yield by up to 74.7%)\n",
      "Progress: 64.30% completed\n",
      "Response for Row 1287:\n",
      "Answer to Question 2: (deletion of gene IAH1, consequence 1: decrease in esterase activity), (deletion of gene IAH1, consequence 2: increase in relative abundance of acetate and ethyl esters), (deletion of gene TIP1, consequence 1: increase in ethyl esters), (deletion of gene IAH1 and TIP1, consequence 1: aroma profile comparable to wild type levels)\n",
      "Progress: 64.35% completed\n",
      "Response for Row 1288:\n",
      "Answer to Question 2: (deletion of gene Abp1p, consequence resistance to wBm0076 expression), (expression of gene wBm0076, consequence disruption of actin cytoskeleton dynamics), (expression of gene wBm0076, consequence colocalization with yeast cortical actin patches), (expression of gene wBm0152, consequence disruption of endosome:vacuole cargo trafficking)\n",
      "Progress: 64.40% completed\n",
      "Response for Row 1289:\n",
      "Answer to Question 2: (deletion of 10 amino acids in HPV 33 L1 protein, soluble expression), (deletion of 15 amino acids in HPV 52 L1 protein, soluble expression), (deletion of 10 amino acids in HPV 58 L1 protein, soluble expression)\n",
      "Progress: 64.45% completed\n",
      "Response for Row 1290:\n",
      "Answer to Question 2: (deletion of gene YNL010W, consequence 1: inability to hydrolyze sorbitol-6-phosphate), (deletion of gene YNL010W, consequence 2: inability to hydrolyze ribitol-5-phosphate), (deletion of gene YNL010W, consequence 3: inability to hydrolyze (d)-glycerol-3-phosphate), (expression of gene PYP1, consequence 1: increased yeast growth rate), (expression of gene PYP1, consequence 2: maintenance of Pgi activity in presence of environmental sugar alcohols), (expression of gene PYP1, consequence 3: prevention of inhibition of glycolysis by sugar alcohol phosphates)\n",
      "Progress: 64.50% completed\n",
      "Response for Row 1291:\n",
      "Answer to Question 2: (deletion of gene YER182W, improvement of PDV biosynthesis)\n",
      "Progress: 64.55% completed\n",
      "Response for Row 1292:\n",
      "Answer to Question 2: (deletion of gene MIG1, consequence 1), (deletion of gene MIG1, consequence 2), (deletion of gene MIG1, consequence 3), (deletion of gene MIG1, consequence 4), (deletion of gene FLO11, consequence 5)\n",
      "\n",
      "- Consequence 1: Weaker biofilm formation ability on fibers.\n",
      "- Consequence 2: Weaker biofilm formation ability on plastic.\n",
      "- Consequence 3: Down-regulation of adhesion genes FLO1, FLO5, and FLO9.\n",
      "- Consequence 4: Reduced flocculation ability and slightly invasive growth ability.\n",
      "- Consequence 5: Decreased ethanol tolerance and reduced biofilm formation during fermentation.\n",
      "Progress: 64.60% completed\n",
      "Response for Row 1293:\n",
      "Answer to Question 2: (overexpression of gene LsFAD2, accumulation of linoleic acid), (overexpression of gene LsFAD2, reduction in oleic acid levels), (overexpression of gene LsFAD3, production of α-linolenic acid), (deletion of gene LsFAD2, accumulation of oleic acid), (deletion of gene LsFAD3, accumulation of linoleic acid)\n",
      "Progress: 64.65% completed\n",
      "Response for Row 1294:\n",
      "Answer to Question 2: (deletion of gene spermidine synthase, severe growth defect), (deletion of gene spermidine synthase, hypersensitivity to furans), (deletion of gene spermidine synthase, hypersensitivity to acetic acid), (overexpression of gene spermidine synthase, increased tolerance to furans), (overexpression of gene spermidine synthase, increased tolerance to acetic acid)\n",
      "Progress: 64.70% completed\n",
      "Response for Row 1295:\n",
      "Answer to Question 2: (deletion of gene CSS1, increase in astaxanthin yield), (expression of gene CrtZ from Agrobacterium aurantiacum, increase in astaxanthin yield), (expression of gene CrtZ from Agrobacterium aurantiacum, increase in astaxanthin ratio), (ARTP mutagenesis, increase in astaxanthin yield)\n",
      "Progress: 64.75% completed\n",
      "Response for Row 1296:\n",
      "Answer to Question 2: (deletion of gene ATF1, lower ethyl acetate production), (overexpression of gene EAT1, increased ethyl acetate production), (mutation eat1K179fs, increased ethyl acetate production), (mutation snf8E148*, decreased ethyl acetate production), (presence of eat1K179fs and ATF1, higher acetate esters production), (introduction of snf8E148* with ATF1, decreased ethyl acetate levels), (snf8E148* mutation, affects other aroma compounds), (snf8E148* mutation, affects growth), (snf8E148* mutation, affects fermentation)\n",
      "Progress: 64.80% completed\n",
      "Response for Row 1297:\n",
      "Answer to Question 2: (deletion of gene Mrx15, consequence 1: disruption of cotranslational protein insertion), (deletion of gene Mrx15, consequence 2: altered interaction with the large ribosomal subunit), (overexpression of gene Mrx15, consequence 1: enhanced membrane protein biogenesis), (overexpression of gene Mrx15, consequence 2: increased contact with mitochondrial translation products)\n",
      "Progress: 64.85% completed\n",
      "Response for Row 1298:\n",
      "Answer to Question 2:  (overexpression of gene ERG pathway genes, increased doubling time), (overexpression of gene ERG pathway genes, respiratory deficiencies on glycerol media), (overexpression of gene ERG pathway genes, cell wall insufficiencies on Congo red media), (overexpression of gene ERG pathway genes, disrupted ion homeostasis under iron starvation conditions), (overexpression of gene ERG pathway genes, disrupted ion homeostasis under calcium starvation conditions), (deletion of gene ERG pathway genes, altered susceptibility to fluconazole), (deletion of gene ERG pathway genes, altered susceptibility to fenpropimorph), (deletion of gene ERG pathway genes, altered susceptibility to lovastatin), (deletion of gene ERG pathway genes, altered susceptibility to nystatin), (deletion of gene ERG pathway genes, altered susceptibility to amphotericin B), (deletion of gene ERG pathway genes, altered susceptibility to terbinafine)\n",
      "Progress: 64.90% completed\n",
      "Response for Row 1299:\n",
      "Answer to Question 2: (gene transformation cassettes, increased transformation efficiency), (TDH1p natMX, five times greater efficiency), (transformation cassettes encoding fluorescent proteins, discrimination between subpopulations of transformed B. bruxellensis cells)\n",
      "Progress: 64.95% completed\n",
      "Response for Row 1300:\n",
      "Answer to Question 2: (deletion of gene ADH1, reduced ethanol production), (deletion of gene ADH2, reduced ethanol production), (deletion of gene ADH3, reduced ethanol production), (deletion of gene ADH4, reduced ethanol production), (deletion of gene ADH5, reduced ethanol production), (deletion of gene ADH6, reduced ethanol production)\n",
      "Progress: 65.00% completed\n",
      "Response for Row 1301:\n",
      "Answer to Question 2: (overexpression of gene ERG13, enhanced metabolic flux through MVA pathway), (overexpression of gene tHMGR, enhanced metabolic flux through MVA pathway), (deletion of gene ROX1, enhancement of metabolic flux through MVA pathway), (overexpression of lupeol synthase, increased yield of pentacyclic triterpenes), (overexpression of lupeol synthase, production of lupeol), (overexpression of lupeol synthase, production of β-amyrin)\n",
      "Progress: 65.05% completed\n",
      "Response for Row 1302:\n",
      "Answer to Question 2: (overexpression of ATF1, increased ethyl acetate production), (deletion of BAT2, increased ethyl acetate production), (overexpression of ATF1, increased AATase activity), (PGK1p library use, precise control of ATF1 expression), (PGK1p library use, regulation of acetate esters production)\n",
      "Progress: 65.10% completed\n",
      "Response for Row 1303:\n",
      "Answer to Question 2: (expression of pATOM36, rescues growth defects in yeast cells lacking Mim1 and/or Mim2), (expression of pATOM36, rescues mitochondrial biogenesis defects in yeast cells lacking Mim1 and/or Mim2), (expression of pATOM36, rescues morphology defects in yeast cells lacking Mim1 and/or Mim2), (co-expression of Mim1 and Mim2, restores assembly defects of MOM proteins in trypanosomes ablated for pATOM36), (co-expression of Mim1 and Mim2, restores insertion defects of MOM proteins in trypanosomes ablated for pATOM36)\n",
      "Progress: 65.15% completed\n",
      "Response for Row 1304:\n",
      "Answer to Question 2: (overexpression of gene ACC1**, consequence 129 mg∙gCDW^-1 of TAGs), (overexpression of gene PAH1, consequence 129 mg∙gCDW^-1 of TAGs), (overexpression of gene DGA1, consequence 129 mg∙gCDW^-1 of TAGs), (deletion of gene TGL3, consequence 218 mg∙gCDW^-1 of TAGs), (deletion of gene TGL4, consequence 218 mg∙gCDW^-1 of TAGs), (deletion of gene TGL5, consequence 218 mg∙gCDW^-1 of TAGs), (deletion of gene ARE1, consequence 218 mg∙gCDW^-1 of TAGs), (deletion of gene POX1, consequence no change in TAGs levels), (deletion of gene GUT2, consequence no change in TAGs levels), (deletion of gene PXA1, consequence 254 mg∙gCDW^-1 of TAGs)\n",
      "Progress: 65.20% completed\n",
      "Response for Row 1305:\n",
      "Answer to Question 2: (expression of gene MAL73L, enhanced maltose fermentation), (expression of gene MAL73L, increased growth rate on maltose), (deletion of gene MAL73, decreased maltose fermentation ability)\n",
      "Progress: 65.25% completed\n",
      "Response for Row 1306:\n",
      "Answer to Question 2: (deletion of gene Mig1p, improved tolerance towards acetic acid), (deletion of gene Mig1p, improved tolerance towards formic acid), (deletion of gene Mig1p, improved tolerance towards levulinic acid), (deletion of gene Mig1p, reduced acetic acid production), (deletion of gene Mig1p, reduced glycerol production), (deletion of gene Mig1p, increased acetate consumption), (deletion of gene Mig1p, increased formate consumption)\n",
      "Progress: 65.30% completed\n",
      "Response for Row 1307:\n",
      "Answer to Question 2: (deletion of gene YEL013W, increase in carotenoid yield)\n",
      "Progress: 65.35% completed\n",
      "Response for Row 1308:\n",
      "Answer to Question 2: (deletion of gene SGS1, consequence 1), (deletion of gene SGS1, consequence 2), (deletion of smORFs, consequence 3)\n",
      "Progress: 65.40% completed\n",
      "Response for Row 1309:\n",
      "Answer to Question 2: (deletion of gene involved in ergosterol biosynthesis, increased MEHP sensitivity), (deletion of gene involved in tryptophan biosynthesis, increased MEHP sensitivity), (deletion of gene regulating ion homeostasis, increased MEHP sensitivity), (deletion of gene involved in protein sorting, increased MEHP sensitivity), (deletion of gene involved in cell wall integrity, increased MEHP sensitivity), (deletion of gene involved in intracellular pH homeostasis, increased MEHP sensitivity)\n",
      "Progress: 65.45% completed\n",
      "Response for Row 1310:\n",
      "Answer to Question 2: (deletion of gene ALE1, reduction in efficiency of palmitoleic acid introduction to HHPC), (deletion of gene ALE1, reduction in molecular species containing acyl residues in HHPC)\n",
      "Progress: 65.50% completed\n",
      "Response for Row 1311:\n",
      "Answer to Question 2: (deletion of gene ZRT2, reduced kidney fungal burden), (deletion of gene ZRT2, growth inhibition by neutrophil extracellular traps), (overexpression of gene ZRT1, supports growth at pH 7 and above), (deletion of gene Zrc1, affects zinc tolerance), (deletion of gene Zrc1, impacts virulence), (deletion of gene Zrc1, impacts liver colonization)\n",
      "Progress: 65.55% completed\n",
      "Response for Row 1312:\n",
      "Answer to Question 2: (deletion of gene RTS1, no synthetic rescue of temperature-sensitivity in ORC2 mutants), (deletion of gene RTS1, additive phenotype indicating separate biological processes from ORC2), (deletion of gene RTS1, increased temperature-sensitivity in ts-replication factor strains)\n",
      "Progress: 65.60% completed\n",
      "Response for Row 1313:\n",
      "Answer to Question 2: (deletion of gene(s) in BGC, increased pathway productivity), (deletion of gene(s) in BGC, production of new bottromycin-related metabolites)\n",
      "Progress: 65.65% completed\n",
      "Response for Row 1314:\n",
      "Answer to Question 2: (deletion of gene fragment, change in gene structure), (mutation of gene fragment, change in gene structure), (recombination of gene fragment, change in gene structure)\n",
      "Progress: 65.70% completed\n",
      "Response for Row 1315:\n",
      "Answer to Question 2: (deletion of RPN4, impaired model dough fermentation), (deletion of PDR1, impaired model dough fermentation), (deletion of PDR3, impaired model dough fermentation), (overexpression of genes involved in oxidative phosphorylation, decrease in glycolytic capacity), (low mRNA levels of proteasome complex component genes, accumulation of ubiquitinated proteins)\n",
      "Progress: 65.75% completed\n",
      "Response for Row 1316:\n",
      "Answer to Question 2: (deletion of gene OSH6, growth defects on n-alkanes of 10-16 carbons), (deletion of gene OSH6, production of functional cytochrome P450 not observed), (deletion of gene OSH6, transcription of ALK1 observed), (deletion of gene OSH6, translation of ALK1 transcript observed), (deletion of gene OSH6, phospholipid composition altered), (deletion of gene OSH6, ratio of phosphatidylserine reduced)\n",
      "Progress: 65.80% completed\n",
      "Response for Row 1317:\n",
      "Answer to Question 2: (deletion of gene Tr69957, affected fungal growth), (deletion of gene Tr69957, affected biomass accumulation), (deletion of gene Tr69957, affected sugar uptake), (deletion of gene Tr69957, affected gene expression of cellobiohydrolases), (deletion of gene Tr69957, affected gene expression of β-glucosidases), (deletion of gene Tr69957, affected gene expression of xylanases)\n",
      "Progress: 65.85% completed\n",
      "Response for Row 1318:\n",
      "Answer to Question 2: (expression of gene SOD2, consequence high tolerance to oxidative stress), (expression of gene CTT1, consequence high tolerance to oxidative stress), (removal of gene Msn2, consequence significant decrease in CTT1-expression), (removal of gene Yap1, consequence significant decrease in CTT1-expression), (abundance of Msn2, consequence high expression of ROS-degrading enzymes)\n",
      "Progress: 65.90% completed\n",
      "Response for Row 1319:\n",
      "Answer to Question 2: (deletion of gene hst3, active glycolysis), (deletion of gene hst4, active glycolysis), (deletion of gene sir2, active glycolysis), (deletion of gene fbp1, active glycolysis), (deletion of gene hst3, high glucose consumption), (deletion of gene hst4, high glucose consumption), (deletion of gene sir2, high glucose consumption), (deletion of gene fbp1, high glucose consumption), (deletion of gene hst3, active ethanol productivity), (deletion of gene hst4, active ethanol productivity), (deletion of gene sir2, active ethanol productivity), (deletion of gene fbp1, active ethanol productivity), (deletion of gene hst3, accumulation of glycolytic metabolites), (deletion of gene hst4, accumulation of glycolytic metabolites), (deletion of gene sir2, accumulation of glycolytic metabolites), (deletion of gene fbp1, accumulation of glycolytic metabolites), (deletion of gene hst3, accumulation of secondary metabolites), (deletion of gene hst4, accumulation of secondary metabolites), (deletion of gene sir2, accumulation of secondary metabolites), (deletion of gene fbp1, accumulation of secondary metabolites), (deletion of gene hst3, synthesis of nucleotides through PP pathway), (deletion of gene hst4, synthesis of nucleotides through PP pathway), (deletion of gene sir2, synthesis of nucleotides through PP pathway), (deletion of gene fbp1, synthesis of nucleotides through PP pathway)\n",
      "Progress: 65.95% completed\n",
      "Response for Row 1320:\n",
      "Answer to Question 2: (overexpression of gene tHMG1, consequence 20-fold higher squalene accumulation), (overexpression of gene DGA1, consequence enhanced lipid biosynthesis), (deletion of gene POX1, consequence reduced lipid β-oxidation), (deletion of gene PXA2, consequence reduced lipid β-oxidation), (simultaneous overexpression of gene tHMG1 and DGA1, consequence over 250-fold higher squalene accumulation)\n",
      "Progress: 66.00% completed\n",
      "Response for Row 1321:\n",
      "Answer to Question 2: (deletion of genes involved in transmembrane and membrane transport, sensitivity to ZnONPs), (deletion of genes involved in cellular ion homeostasis, sensitivity to ZnONPs), (deletion of genes involved in cell wall organization or biogenesis, sensitivity to ZnONPs), (deletion of genes involved in transcription and RNA processing, sensitivity to AgNPs), (deletion of genes involved in cellular respiration, sensitivity to AgNPs), (deletion of genes involved in endocytosis and vesicular transport, sensitivity to AgNPs)\n",
      "Progress: 66.05% completed\n",
      "Response for Row 1322:\n",
      "Answer to Question 2: (deletion of STR3, divert accumulated thiols to GSH biosynthesis), (overexpression of CYS3, divert accumulated thiols to GSH biosynthesis), (overexpression of CYS4, divert accumulated thiols to GSH biosynthesis), (up-regulation of PDI, reduce unfolded protein response), (accumulation of homocysteine, promote cysteine synthesis), (accumulation of cystathionine, promote cysteine synthesis), (higher concentration of glutamic acid, contribute to GSH synthesis), (higher concentration of glycine, contribute to GSH synthesis), (higher concentration of cysteine, contribute to GSH synthesis)\n",
      "Progress: 66.10% completed\n",
      "Response for Row 1323:\n",
      "Answer to Question 2: (deletion of gene GAP1, reduction in extracellular urea), (deletion of gene CAN1, reduction in extracellular urea), (overexpression of gene Gln3p<sub>1-653</sub>, reduction in extracellular urea), (deletion of gene ALP1, reduction in extracellular urea)\n",
      "Progress: 66.15% completed\n",
      "Response for Row 1324:\n",
      "Answer to Question 2: (overexpression of tdcB, increase in 1-propanol production), (enhancement of threonine biosynthesis from aspartate by thrA, thrB and thrC, increase in 1-propanol production), (deletion of GLY1 gene, increase in 1-propanol production), (construction of an artificial 2 KB biosynthetic pathway from pyruvate via citramalate (cimA), increase in 1-propanol production)\n",
      "Progress: 66.20% completed\n",
      "Response for Row 1325:\n",
      "Answer to Question 2: (deletion of gene involved in signaling cascades, affects lipid accumulation), (deletion of gene involved in gene expression, affects lipid accumulation), (deletion of gene involved in protein modification, affects lipid accumulation), (deletion of gene involved in vesicular trafficking, affects lipid accumulation), (deletion of gene involved in autophagy, affects lipid accumulation), (deletion of gene involved in amino acid synthesis, affects lipid accumulation), (deletion of gene involved in tRNA modification, affects lipid accumulation)\n",
      "Progress: 66.25% completed\n",
      "Response for Row 1326:\n",
      "Answer to Question 2: (deletion of Nup mutants, reduction in Ty1 mobility), (deletion of Nup mutants, impairment in Ty1 insertion upstream of tRNA genes), (deletion of components of the NPC nuclear basket, mis-targeting of Ty1 elements to the ends of chromosomes)\n",
      "Progress: 66.30% completed\n",
      "Response for Row 1327:\n",
      "Answer to Question 2: (deletion of gene MFA2, reduction in fermentation duration by 55%), (deletion of gene UBC13, reduction in fermentation duration), (deletion of gene MMS2, reduction in fermentation duration), (deletion of gene UBP7, reduction in fermentation duration), (deletion of gene UBI4, reduction in fermentation duration), (deletion of gene BRO1, reduction in fermentation duration), (deletion of gene TPK2, reduction in fermentation duration), (deletion of gene EAR1, reduction in fermentation duration), (deletion of gene MRP17, reduction in fermentation duration), (deletion of gene MVB12, reduction in fermentation duration), (deletion of gene AAT2, reduction in fermentation duration)\n",
      "Progress: 66.35% completed\n",
      "Response for Row 1328:\n",
      "Answer to Question 2:  (overexpression of gene MIG1, consequence enhanced growth), (overexpression of gene MIG1, consequence decreased xylitol accumulation), (overexpression of gene MIG1, consequence increased biomass accumulation), (overexpression of gene MIG1, consequence improved activities of XR), (overexpression of gene MIG1, consequence improved activities of XDH), (overexpression of gene MIG1, consequence improved activities of XK)\n",
      "Progress: 66.40% completed\n",
      "Response for Row 1329:\n",
      "Answer to Question 2: (deletion of fatty acid activation genes, redirected fatty acid metabolism), (deletion of fatty acid activation genes, reduced feedback regulation), (deletion of main storage lipid formation genes, constrained lipid metabolic network), (deletion of phosphatidate phosphatase genes, fatty acid flux directed toward phospholipids), (deletion of phosphatidate phosphatase genes, increases of phospholipids by up to 2.8-fold), (deletion of phosphatidate phosphatase genes, increases of FFAs by up to 40-fold), (deletion of phospholipase genes PLB1 and PLB2, 46% decrease in FFA levels), (deletion of phospholipase genes PLB1 and PLB2, 105% increase in phospholipid levels)\n",
      "Progress: 66.45% completed\n",
      "Response for Row 1330:\n",
      "Answer to Question 2: (deletion of gene NYn43, faster fermentation rate)\n",
      "Progress: 66.50% completed\n",
      "Response for Row 1331:\n",
      "Answer to Question 2: (deletion of gene MRI1, reduction in DMTS-P1), (deletion of gene MRI1, suppression of DMTS formation), (deletion of gene MRI1, no change in sake brewing properties)\n",
      "Progress: 66.55% completed\n",
      "Response for Row 1332:\n",
      "Answer to Question 2: (deletion of gene TcGS, increased NH4+ toxicity), (overexpression of gene TcGS, reduced NH4+ toxicity), (deletion of gene TcGS, impaired escape from parasitophorous vacuole), (overexpression of gene TcGS, enhanced escape from parasitophorous vacuole), (inhibition of GS activity by Methionine sulfoximine, reduced parasite survival under excess ammonium conditions)\n",
      "Progress: 66.60% completed\n",
      "Response for Row 1333:\n",
      "Answer to Question 2: (deletion of gene ILV1, consequence n-propanol concentration decreased), (deletion of gene ILV1, consequence active amyl alcohol concentration decreased), (deletion of gene ILV1, consequence 2-phenylethanol concentration decreased), (deletion of gene ILV1, consequence isobutanol concentration increased), (deletion of gene ILV1, consequence isoamyl alcohol concentration increased), (deletion of gene LEU1, consequence n-propanol concentration increased), (deletion of gene LEU1, consequence isoamyl alcohol concentration decreased), (deletion of gene LEU1, consequence 2-phenylethanol concentration decreased), (deletion of gene LEU1, consequence isobutanol concentration increased), (deletion of gene LEU2, consequence n-propanol concentration increased), (deletion of gene LEU2, consequence isoamyl alcohol concentration decreased), (deletion of gene LEU2, consequence 2-phenylethanol concentration decreased), (deletion of gene LEU2, consequence isobutanol concentration increased)\n",
      "Progress: 66.65% completed\n",
      "Response for Row 1334:\n",
      "Answer to Question 2: (deletion of gene Ndi1p, consequence 1: abolishment of sporulation), (deletion of gene Ndi1p, consequence 2: failure of premeiotic DNA replication), (deletion of gene Ndi1p, consequence 3: insufficient IME1 expression), (overexpression of gene RIM101, consequence 1: inhibition of Smp1p), (overexpression of gene RIM101, consequence 2: promotion of IME1 expression), (overexpression of gene RIM101, consequence 3: promotion of meiosis initiation)\n",
      "Progress: 66.70% completed\n",
      "Response for Row 1335:\n",
      "Answer to Question 2: (deletion of gene recD, consequence 1: drastically decreased the length of resection), (deletion of gene recD, consequence 2: allowed for recombination with short ectopic plasmid homologies), (deletion of gene recD, consequence 3: significantly increased the efficiency of horizontal gene transfer between strains)\n",
      "Progress: 66.75% completed\n",
      "Response for Row 1336:\n",
      "Answer to Question 2: (deletion of shear stress genes, consequence 1: cellular redox responses), (deletion of shear stress genes, consequence 2: apoptosis)\n",
      "Progress: 66.80% completed\n",
      "Response for Row 1337:\n",
      "Answer to Question 2: (deletion of gene ECM33, shortest fermentation), (deletion of gene ECM33, up-regulation of SLT2), (deletion of gene ECM33, up-regulation of HOG1), (deletion of gene ECM33, up-regulation of CHS3), (deletion of gene ECM33, alteration in transcript abundance of GLN1), (deletion of gene ECM33, alteration in transcript abundance of GLT1), (deletion of gene ECM33, alteration in transcript abundance of GDH1), (deletion of gene ECM33, alteration in transcript abundance of GDH2)\n",
      "Progress: 66.85% completed\n",
      "Response for Row 1338:\n",
      "Answer to Question 2: (deletion of gene FUS3, consequence increased yeast lifespan), (deletion of gene KSS1, consequence increased yeast lifespan), (deletion of gene HOG1, consequence increased yeast lifespan), (deletion of gene MAPK and TOR1, consequence greater lifespan increase than tor1Δ alone), (endogenous expression of non-activatable form of Fus3, consequence similar effects as deletion of MAPK and TOR1), (deletion of gene FUS3 under nitrogen starvation, consequence decreased CLS)\n",
      "Progress: 66.90% completed\n",
      "Response for Row 1339:\n",
      "Answer to Question 2: (deletion of gene Hxt, consequence increased D-xylose affinity), (expression of gene Hxt2C505P, consequence increased D-xylose affinity), (mutation of gene Hxt2, consequence increased D-xylose affinity)\n",
      "Progress: 66.95% completed\n",
      "Response for Row 1340:\n",
      "Answer to Question 2: (deletion of secondary metabolite gene clusters, improved mithramycin A production)\n",
      "Progress: 67.00% completed\n",
      "Response for Row 1341:\n",
      "Answer to Question 2: (deletion of gene, increased β-carotene production), (overexpression of gene, improved endoglucanase display)\n",
      "Progress: 67.05% completed\n",
      "Response for Row 1342:\n",
      "Answer to Question 2: (deletion of gene FAA1, increased production of medium-chain α, ω-DCAs), (deletion of gene FAA4, increased production of medium-chain α, ω-DCAs), (deletion of gene FAA1, enabled production of C14 α, ω-DCAs), (deletion of gene FAA4, enabled production of C16 α, ω-DCAs), (deletion of gene POX1, no significant impact on production of α, ω-dicarboxylic acids), (overexpression of thioesterase genes, no significant impact on production of α, ω-dicarboxylic acids)\n",
      "Progress: 67.10% completed\n",
      "Response for Row 1343:\n",
      "Answer to Question 2: (deletion of gene FAA1, consequence 1), (deletion of gene FAA4, consequence 1), (deletion of gene POX1, consequence 1), (deletion of gene TES1, consequence 1), (regulation of gene OLE1, consequence 1), (regulation of gene HMG1, consequence 1), (regulation of gene ACS1, consequence 1)\n",
      "Progress: 67.15% completed\n",
      "Response for Row 1344:\n",
      "Answer to Question 2: (deletion of 25 genes encoding transport proteins, lower specific growth rate), (deletion of 25 genes encoding transport proteins, lower biomass-specific lactate production rate), (expression of Lactobacillus casei lactate dehydrogenase (LcLDH), increased lactate production), (deletion of 25 transporter genes, unaffected lactate export rates at fixed specific growth rate)\n",
      "Progress: 67.20% completed\n",
      "Response for Row 1345:\n",
      "Answer to Question 2: (heterologous overexpression of mcr gene, 3-HP production), (heterologous overexpression of mcr gene, increase in 3-HP titer), (increase in promoter strength and mcr gene copy number, improvement in 3-HP titer), (overexpression of mcr gene, decrease in acetyl-CoA pool size), (overexpression of mcr gene, rapid degradation of 3-HP in stationary phase), (overexpression of mcr gene, accumulation of intracellular 3-hydroxypropionyl-CoA), (deletion of META1_4251 gene, reduction in 3-HP degradation rate)\n",
      "Progress: 67.25% completed\n",
      "Response for Row 1346:\n",
      "Answer to Question 2: (deletion of endogenous gene products, repair of existing mutations), (expression of multiple sgRNAs, multiplexing with a single sgRNA), (installation of artificial Cas9 target sequences, reduction/elimination in possible off-target effects)\n",
      "Progress: 67.30% completed\n",
      "Response for Row 1347:\n",
      "Answer to Question 2: (deletion of gene FgPLD1, consequence hyphal growth affected), (deletion of gene FgPLD1, consequence conidiation affected), (deletion of gene FgPLD1, consequence perithecium formation affected), (deletion of gene FgPLD1, consequence reduced deoxynivalenol production), (deletion of gene FgPLD1, consequence reduced virulence in flowering wheat heads), (deletion of gene FgPLD1, consequence reduced virulence in corn silks)\n",
      "Progress: 67.35% completed\n",
      "Response for Row 1348:\n",
      "Answer to Question 2: (expression of gene xylB, consequence D-xylose dehydrogenase activity), (expression of gene xylD, consequence D-xylonate dehydratase activity), (expression of gene yagE, consequence glycolaldehyde production), (expression of gene yjhH, consequence glycolaldehyde production), (expression of gene aldA, consequence glycolic acid production), (expression of gene ldhL, consequence lactic acid production), (expression of gene xylB and xylD, consequence 29 mg/L 2K3DXA production), (expression of gene xylB and xylD, consequence 9 g/L D-xylonic acid accumulation), (FRA2 deletion, consequence improved dehydratase activity), (targeting xylD expression to mitochondria, consequence improved dehydratase activity), (expression of gene aldA and ldhL, consequence 1 g/L glycolic acid accumulation), (endogenous aldo-keto reductase activity, consequence ethylene glycol production)\n",
      "Progress: 67.40% completed\n",
      "Response for Row 1349:\n",
      "Answer to Question 2: (deletion of gene BAT1, increase in isobutanol production by 14.2-fold), (deletion of gene BAT1, additional 2.1-fold increase in isobutanol production when biosynthetic pathway is compartmentalized in mitochondria), (addition of valine, inhibition of isobutanol production), (removal of valine, increase in isobutanol production), (interaction between valine and Ilv6p, affects isobutanol production), (overexpression of gene BAT1, influences isobutanol production), (overexpression of gene BAT2, influences isobutanol production), (addition of valine, boost in 2-methyl-1-butanol production)\n",
      "Progress: 67.45% completed\n",
      "Response for Row 1350:\n",
      "Answer to Question 2: (deletion of gene HUR1-A, reduced efficiency of NHEJ in overhang end plasmid repair assays), (deletion of gene HUR1-A, reduced efficiency of NHEJ in blunt end plasmid repair assays), (deletion of gene HUR1-A, reduced efficiency of chromosomal repair), (deletion of gene HUR1-A, reduced efficiency of HR repair)\n",
      "Progress: 67.50% completed\n",
      "Response for Row 1351:\n",
      "Answer to Question 2: (deletion of gene ECM22, mating defects), (deletion of gene UPC2, mating defects), (deletion of gene PRM4, reduced mating efficiency), (overexpression of gene STE12, increased PRM1 levels), (deletion of gene STE12, lowered PRM1 expression), (deletion of gene ECM22 and UPC2, abolished PRM1 expression), (mutation of Ste12-binding sites in PRM1 promoter, abolished PRM1 expression)\n",
      "Progress: 67.55% completed\n",
      "Response for Row 1352:\n",
      "Answer to Question 2: (ectopic expression of Saccharomyces cerevisiae MATα2, suppresses sporulation), (ectopic expression of Saccharomyces cerevisiae MATα2, inhibits riboflavin overproduction), (ectopic expression of Saccharomyces cerevisiae MATα2, downregulates AgSOK2), (deletion of AgSOK2, reduces riboflavin production), (deletion of AgSOK2, blocks sporulation), (AgSok2, promotes sporulation), (AgSok2, does not control AgIME1 transcription), (AgSok2, targets AgIME2), (AgSok2, targets AgNDT80)\n",
      "Progress: 67.60% completed\n",
      "Response for Row 1353:\n",
      "Answer to Question 2: (deletion of CAN1 gene, effectiveness in Saccharomyces cerevisiae), (introduction of single base mutation A128C into rpsL gene, effectiveness in Escherichia coli), (deletion of gene using RMGE-bceSIIM, higher counter-selection efficiency in Escherichia coli), (replacement of gene using RMGE-bceSIIM, higher counter-selection efficiency in Escherichia coli), (deletion of gene using RMGE-mcrA, higher counter-selection efficiency in Bacillus subtilis), (replacement of gene using RMGE-mcrA, higher counter-selection efficiency in Bacillus subtilis)\n",
      "Progress: 67.65% completed\n",
      "Response for Row 1354:\n",
      "Answer to Question 2: (deletion of gene ctpxa1, weakened expression of ctpxa1), (deletion of gene ctpxa1, increased dodecanedioic acid production)\n",
      "Progress: 67.70% completed\n",
      "Response for Row 1355:\n",
      "Answer to Question 2: (gene expression of biomass hydrolyzing enzymes, efficient biomass utilization), (gene expression of transporters, enhanced sugar input), (gene expression of transporters, improved product output)\n",
      "Progress: 67.75% completed\n",
      "Response for Row 1356:\n",
      "Answer to Question 2: (deletion of gene PEX22, increased malate production), (deletion of gene PEX22, DMS sensitivity), (deletion of genes encoding peroxins, increased malate production), (mislocalization of Mdh3p, increased malate production)\n",
      "Progress: 67.80% completed\n",
      "Response for Row 1357:\n",
      "Answer to Question 2: (deletion of gene FgOXP1, decreased virulence on wheat), (deletion of gene FgOXP1, decreased deoxynivalenol production), (deletion of gene FgOXP2, decreased virulence on wheat), (deletion of gene FgOXP2, decreased deoxynivalenol production), (deletion of gene FgOXP2, defective conidiation), (deletion of gene FgOXP2, defective sexual reproduction), (deletion of gene FgOXP1 and FgOXP2, severe defects in conidiation), (deletion of gene FgOXP1 and FgOXP2, severe defects in DON production), (deletion of gene FgOXP1 and FgOXP2, severe defects in virulence on plants)\n",
      "Progress: 67.85% completed\n",
      "Response for Row 1358:\n",
      "Answer to Question 2: (deletion of gene PYK1, reduced ethanol production), (deletion of gene PFK1, reduced ethanol production), (deletion of gene PFK2, reduced ethanol production), (expression of gene HuPYK1, increased ethanol production), (expression of gene HuPFK1, increased ethanol production), (expression of gene HuPFK2, increased ethanol production)\n",
      "Progress: 67.90% completed\n",
      "Response for Row 1359:\n",
      "Answer to Question 2: (expression of G-protein-coupled receptor, detection of serotonin), (overproduction of serotonin, increased production of hydroxystrictosidine)\n",
      "Progress: 67.95% completed\n",
      "Response for Row 1360:\n",
      "Answer to Question 2: (deletion of gene LIG4, enhanced probability of obtaining correctly modified transformants), (replacement of ORF in gene ADE2 by SAT1 selective marker, adenine auxotrophy), (introduction of premature stop codon in gene ADE2, adenine auxotrophy), (replacement of ORF in gene MET15 by SAT1 selective marker, methionine auxotrophy), (introduction of premature stop codon in gene MET15, methionine auxotrophy), (insertion of triple HA-tag sequence in gene SOK2, verified in western blot), (deletion of gene LIG4, increased CRISPR-Cas9 mediated targeting efficiency)\n",
      "Progress: 68.00% completed\n",
      "Response for Row 1361:\n",
      "Answer to Question 2: (Deletion of gene lst4, reduced H2S production), (Deletion of gene lst7, reduced H2S production), (Deletion of gene agp1, reduced H2S production), (Deletion of gene gnp1, reduced H2S production), (Deletion of gene mup1, reduced H2S production), (Deletion of gene stp1, reduced H2S production), (Deletion of gene dal81, reduced H2S production)\n",
      "Progress: 68.05% completed\n",
      "Response for Row 1362:\n",
      "Answer to Question 2: (deletion of gene nuo1, down-regulation of nuclear-encoded CI subunit proteins), (deletion of gene nuo1, decreased phospholipid synthesis), (deletion of gene nuo1, decreased ergosterol synthesis), (deletion of gene nuo1, decreased cell wall mannan synthesis), (deletion of gene nuo1, increased CIV proteins), (deletion of gene nuo1, increased AOX2), (deletion of gene nuo1, participation in nucleotide synthesis), (deletion of gene nuo1, participation in ribosomal biogenesis), (deletion of gene nuo2, down-regulation of nuclear-encoded CI subunit proteins), (deletion of gene nuo2, decreased phospholipid synthesis), (deletion of gene nuo2, decreased ergosterol synthesis), (deletion of gene nuo2, decreased cell wall mannan synthesis), (deletion of gene nuo2, increased CIV proteins), (deletion of gene nuo2, increased AOX2), (deletion of gene nuo2, involvement in vesicle trafficking), (deletion of gene goa1, down-regulation of nuclear-encoded CI subunit proteins), (deletion of gene goa1, decreased phospholipid synthesis), (deletion of gene goa1, decreased ergosterol synthesis), (deletion of gene goa1, decreased cell wall mannan synthesis), (deletion of gene goa1, increased CIV proteins), (deletion of gene goa1, increased AOX2), (deletion of gene goa1, regulation of membrane transporter proteins), (deletion of gene goa1, regulation of ROS removal), (deletion of gene goa1, regulation of substrates trafficking between peroxisomes and mitochondria), (deletion of gene nuo1, nuo2, goa1, avirulence phenotype in murine model of invasive disease)\n",
      "Progress: 68.10% completed\n",
      "Response for Row 1363:\n",
      "Answer to Question 2: (expression of gene Lm. ldhA, increased d-LA production), (deletion of gene ADH1, reduced ethanol production), (deletion of gene ADH2, reduced ethanol production), (deletion of gene ADH3, reduced ethanol production), (deletion of gene ADH4, reduced ethanol production), (deletion of gene ADH5, reduced ethanol production), (deletion of gene GPD1, reduced glycerol production), (deletion of gene GPD2, reduced glycerol production), (deletion of gene DLD1, reduced d-LA degradation), (mutation of gene SUR1 I245S, increased LA tolerance), (mutation of gene SUR1 I245S, increased d-LA production), (loss-of-function mutation of gene ERF2, increased d-LA production), (deletion of gene PDC1, increased d-LA production), (additional integration of gene Lm. ldhA, increased d-LA production)\n",
      "Progress: 68.15% completed\n",
      "Response for Row 1364:\n",
      "Answer to Question 2: (deletion of gene nox-1, failure to induce hepatic cell death), (overexpression of gene TG2, enhancement of cellular TG activity), (overexpression of gene TG2, induction of hepatic cell death), (production of metabolite ROS, enhancement of nuclear TG activity), (deletion of gene TG2, inhibition of cell death), (administration of product N-acetyl cysteine, inhibition of enhanced TG activity), (administration of product corn peptide fraction, inhibition of hepatic ROS and TG activities)\n",
      "Progress: 68.20% completed\n",
      "Response for Row 1365:\n",
      "Answer to Question 2:  (deletion of gene bgl2, consequence 15% reduction in β-(1,6)-branching on β-(1,3)-glucan), (deletion of gene gas1, consequence 70% reduction in β-(1,6)-branching on β-(1,3)-glucan), (deletion of gene gas1, deletion of gene bgl2, consequence drastically sick phenotype), (expression of gene ScGas1p, consequence introduction of β-(1,6)-branching on β-(1,3)-oligomers), (expression of gene Gel4p, consequence dual β-(1,3)-glucan elongating and branching activity), (presence of CBM43, consequence required for dual β-(1,3)-glucan elongating and branching activity)\n",
      "Progress: 68.25% completed\n",
      "Response for Row 1366:\n",
      "Answer to Question 2: (deletion of gene Ylsdh5, accumulation of by-product acetate), (deletion of gene Ylach, elimination of acetic acid formation), (deletion of gene Ylach, improvement of succinic acid production), (deletion of gene Ylach, improvement of cell growth), (overexpression of gene ScPCK, improvement of succinic acid titer), (overexpression of gene YlSCS2, improvement of succinic acid titer)\n",
      "Progress: 68.30% completed\n",
      "Response for Row 1367:\n",
      "Answer to Question 2: (deletion of gene ADY2, improved growth under acetic acid stress), (deletion of gene ADY2, improved growth under ethanol stress), (deletion of gene ADY2, improved growth under hydrogen peroxide stress), (deletion of gene ADY2, increased ethanol production by 14.7%), (deletion of gene ADY2, decreased intracellular acetic acid), (deletion of gene ADY2, decreased ROS accumulation), (deletion of gene ADY2, decreased plasma membrane permeability)\n",
      "Progress: 68.35% completed\n",
      "Response for Row 1368:\n",
      "Answer to Question 2: (deletion of gene DGA1, increased fatty alcohol production), (deletion of gene HFD1, increased fatty alcohol production), (deletion of gene ADH6, increased fatty alcohol production), (overexpression of gene FAR, increased fatty alcohol production), (overexpression of mutant acetyl-CoA carboxylase, increased fatty alcohol production), (limiting gene GDH1 expression, increased fatty alcohol production), (overexpression of gene OLE1, increased fatty alcohol production)\n",
      "Progress: 68.40% completed\n",
      "Response for Row 1369:\n",
      "Answer to Question 2: (overexpression of gene BIO1, growth in biotin-free medium), (overexpression of gene BIO1, increased specific growth rate), (deletion of gene TPO1, increased specific growth rate), (deletion of gene PDR12, increased specific growth rate)\n",
      "Progress: 68.45% completed\n",
      "Response for Row 1370:\n",
      "Answer to Question 2: (deletion of gene yro2, consequence reduced DIV secretion), (deletion of gene mrh1, consequence reduced DIV secretion), (deletion of gene hxt5, consequence increased isobutanol production), (deletion of 18 genes, consequence slightly increased intermediates secretion), (deletion of 18 genes, consequence decreased isobutanol production)\n",
      "Progress: 68.50% completed\n",
      "Response for Row 1371:\n",
      "Answer to Question 2: (deletion of gene Pub1, decreased efficiency of nonsense readthrough at a majority but not all tetranucleotide stop signals)\n",
      "Progress: 68.55% completed\n",
      "Response for Row 1372:\n",
      "Answer to Question 2: (deletion of gene Dhh1, decrease in expression of Ste12), (deletion of gene Dhh1, severe mating defects), (mutation in ATPase A motif of Dhh1, reduced mating efficiency), (mutation in ATPase B motif of Dhh1, reduced mating efficiency), (C-terminus deletion of Dhh1, reduced mating efficiency), (C-terminus deletion of Dhh1, reduced Ste12 protein expression), (deletion of gene Dhh1, dispensable for growth at 37°C), (deletion of gene Dhh1, role in regulating Ste12 protein expression), (deletion of gene Dhh1, role in mating processes), (P-body accumulation induced by α-mating factor, requires ATPase domain of Dhh1), (P-body accumulation induced by α-mating factor, requires RNA-binding domain of Dhh1), (P-body accumulation induced by α-mating factor, requires Q/P-rich C-terminal domains of Dhh1)\n",
      "Progress: 68.60% completed\n",
      "Response for Row 1373:\n",
      "Answer to Question 2: (deletion of gene YAP1, narrowed response to LoaOOH), (deletion of gene YAP1, decreased expression of Tsa1p), (deletion of gene YAP1, increased expression of Ahp1p), (deletion of gene YAP1, increased expression of Hsp31p), (deletion of gene OYE3, altered expression of Sod1p), (deletion of gene OYE3, altered expression of Hsp60p), (deletion of gene OYE3, altered expression of Ssa1p), (deletion of gene OYE3, altered expression of Sse1p), (deletion of gene OYE3, altered expression of Pst2p), (deletion of gene OYE3, altered expression of Tpm1p), (deletion of gene OYE2, altered expression of Sod1p), (deletion of gene OYE2, altered expression of Hsp60p), (deletion of gene OYE2, altered expression of Ssa1p), (deletion of gene OYE2, altered expression of Sse1p), (deletion of gene OYE2, altered expression of Pst2p), (deletion of gene OYE2, altered expression of Tpm1p)\n",
      "Progress: 68.65% completed\n",
      "Response for Row 1374:\n",
      "Answer to Question 2: (removal of gene Mep1, growth dependent on extracellular NH3 concentrations), (removal of gene Mep2, growth dependent on extracellular NH3 concentrations), (removal of gene Mep3, growth dependent on extracellular NH3 concentrations), (removal of gene Mep1, higher intracellular NHX concentration), (removal of gene Mep2, higher intracellular NHX concentration), (removal of gene Mep3, higher intracellular NHX concentration), (removal of gene Mep1, up-regulation of vacuolar proteases), (removal of gene Mep2, up-regulation of vacuolar proteases), (removal of gene Mep3, up-regulation of vacuolar proteases), (removal of gene Mep1, increased stress response genes), (removal of gene Mep2, increased stress response genes), (removal of gene Mep3, increased stress response genes), (removal of gene Mep1, higher turnover rates of biomass components), (removal of gene Mep2, higher turnover rates of biomass components), (removal of gene Mep3, higher turnover rates of biomass components)\n",
      "Progress: 68.70% completed\n",
      "Response for Row 1375:\n",
      "Answer to Question 2: (gene deletion and replacement, increased flexibility for refactoring pathways)\n",
      "Progress: 68.75% completed\n",
      "Response for Row 1376:\n",
      "Answer to Question 2: (deletion of gene Crz1, intermediate phenotypes between wild-type and calcineurin mutants), (deletion of calcineurin docking domain, inability of Crz1 to translocate into the nucleus under thermal stress), (deletion of gene cna1, down-regulation of 102 genes regulated in a calcineurin-Crz1-dependent manner at 37°C), (deletion of gene crz1, down-regulation of 102 genes regulated in a calcineurin-Crz1-dependent manner at 37°C), (deletion of gene Crz1, regulation of 59 genes independently of calcineurin), (activation of calcineurin, increased Crz1 binding to target gene promoters upon thermal stress), (deletion of gene calcineurin, impact on mRNA expression/stability through factors beyond Crz1)\n",
      "Progress: 68.80% completed\n",
      "Response for Row 1377:\n",
      "Answer to Question 2: (deletion of gene hexokinase, consequence increased d-xylose utilization in high d-glucose), (mutation in gene CYC8, consequence altered expression of 95 genes), (mutation in gene CYC8, consequence upregulation of HXT10), (mutation in gene CYC8, consequence upregulation of HXT13), (mutation in gene CYC8, consequence upregulation of HXT15), (mutation in gene CYC8, consequence upregulation of HXT16), (mutation in gene CYC8, consequence increased d-xylose uptake in presence of d-glucose), (mutation in gene CYC8, consequence elevated Vmax for d-glucose transport), (mutation in gene CYC8, consequence elevated Vmax for d-xylose transport)\n",
      "Progress: 68.85% completed\n",
      "Response for Row 1378:\n",
      "Answer to Question 2: (deletion of gene xxx, identification of genes involved in biological processes), (overexpression of gene xxx, probing genetic networks), (deletion of gene xxx, identification of multiple naturally occurring allelic variants), (overexpression of gene xxx, contribution to polygenic phenotypes), (deletion of gene xxx, limitation on use of modern genetic techniques), (overexpression of gene xxx, production of volatile aroma compounds)\n",
      "Progress: 68.90% completed\n",
      "Response for Row 1379:\n",
      "Answer to Question 2: (deletion of gene Hmo1p, reduced repair efficiency), (deletion of gene Hmo1p, reduced repair accuracy), (inhibition of DNA end resection, restored repair efficiency), (deletion of gene Ku, reduced repair efficiency), (inhibition of resection, restored repair efficiency), (excessive DNA resection, inhibitory for A-EJ)\n",
      "Progress: 68.95% completed\n",
      "Response for Row 1380:\n",
      "Answer to Question 2: (deletion of gene in O1A O-antigen gene cluster, consequence: altered substrate preferences of Wzx flippases), (replacement of gene in O1A O-antigen gene cluster, consequence: altered substrate preferences of Wzx flippases)\n",
      "Progress: 69.00% completed\n",
      "Response for Row 1381:\n",
      "Answer to Question 2: (deletion of gene FAA1, increase in FFA levels), (deletion of gene FAA4, increase in FFA levels), (dynamic expression of gene FAA1, regulation of FFA and acyl-CoA pools), (expression of gene MmCAR, production of fatty alcohols from FFAs), (expression of gene Adh5, production of fatty alcohols from FFAs), (expression of fatty acyl-CoA reductase from Marinobacter aquaeolei, production of fatty alcohols from acyl-CoA), (expression of gene FAA1 under HXT1 promoter, increased accumulation of fatty alcohols), (expression of gene FAA1 under HXT1 promoter, decrease in FFA levels)\n",
      "Progress: 69.05% completed\n",
      "Response for Row 1382:\n",
      "Answer to Question 2: (deletion of gene YjgB, increase in protoilludene production), (overexpression of gene IDI, enhancement of protoilludene production to 67.4mg/L), (overexpression of gene Fpr, increase in protoilludene production to 318.8mg/L), (overexpression of gene FldA, increase in protoilludene production to 318.8mg/L), (introduction of gene tPos5p, increase in protoilludene production to 512.7mg/L)\n",
      "Progress: 69.10% completed\n",
      "Response for Row 1383:\n",
      "Answer to Question 2: (deletion of gene spt3, increased expression of Fps1), (deletion of gene srb5, increased expression of Fps1), (deletion of gene gcn5, increased expression of Fps1), (expression of gene GCN4, increased expression of Fps1), (phosphorylation of eIF2α, increased expression of Fps1), (deletion of uORF1, increased expression of Fps1), (deletion of uORF1 and uORF2, increased expression of Fps1)\n",
      "Progress: 69.15% completed\n",
      "Response for Row 1384:\n",
      "Answer to Question 2: (no expression/deletion of gene mentioned, no specific consequences related to expression/deletion)\n",
      "Progress: 69.20% completed\n",
      "Response for Row 1385:\n",
      "Answer to Question 2:  (overexpression of gene Saccharomyces cerevisiae, consequence increased protein phosphorylation), (deletion of gene Saccharomyces cerevisiae, consequence decreased protein phosphorylation), (overexpression of gene Candida albicans, consequence increased protein phosphorylation), (deletion of gene Candida albicans, consequence decreased protein phosphorylation)\n",
      "Progress: 69.25% completed\n",
      "Response for Row 1386:\n",
      "Answer to Question 2: (deletion of gene CAT8, diminished growth on glycerol), (deletion of gene CAT8, diminished growth on ethanol), (deletion of gene CAT8, diminished respiration on xylose), (deletion of gene CAT8, activated transcription of gene XYL3), (deletion of gene CAT8, activated transcription of gene DAS1), (deletion of gene CAT8, activated transcription of gene RPE1), (deletion of gene CAT8, increase in activity of enzymes involved in xylose metabolism), (deletion of gene CAT8, increase in ethanol production from xylose), (overexpression of gene CAT8, drop in ethanol production from xylose)\n",
      "Progress: 69.30% completed\n",
      "Response for Row 1387:\n",
      "Answer to Question 2: (deletion of gene CAN1, increased translation of proteins that facilitate cellular response to stress)\n",
      "Progress: 69.35% completed\n",
      "Response for Row 1388:\n",
      "Answer to Question 2: (deletion of gene UPF1, consequence 1), (deletion of gene UPF1, consequence 2), (deletion of gene UPF2, consequence 1), (deletion of gene UPF2, consequence 2), (deletion of gene UPF3, consequence 1), (deletion of gene UPF3, consequence 2)\n",
      "Progress: 69.40% completed\n",
      "Response for Row 1389:\n",
      "Answer to Question 2: (atf1 overexpression, higher production of acetate esters), (atf1 overexpression, lower production of higher alcohols), (iah1 deletion, increased production of isobutyl acetate), (iah1 deletion, increased production of isoamyl acetate), (bat2 deletion, decline in production of higher alcohols), (atf1 overexpression, reduced hydrolysis of acetate esters), (atf1 overexpression and bat2 deletion interaction, decline in production of higher alcohols)\n",
      "Progress: 69.45% completed\n",
      "Response for Row 1390:\n",
      "Answer to Question 2: (deletion of gene ILV2, reduction in diacetyl content), (overexpression of gene BDH2, reduction in diacetyl content), (overexpression of gene ILV5, reduction in diacetyl content)\n",
      "Progress: 69.50% completed\n",
      "Response for Row 1391:\n",
      "Answer to Question 2: (deletion of gene SCH9, deregulated NCE103 expression on RNA level), (deletion of gene SCH9, deregulated NCE103 expression on protein level), (expression of gene NCE103, acceleration of CO2 conversion to bicarbonate), (activation of gene ScCst6, regulation of NCE103 expression), (activation of gene ScSch9, control of ScCst6/Rca1 activity), (deletion of gene SCH9 homologue, impaired CO2-dependent regulation of NCE103 expression)\n",
      "Progress: 69.55% completed\n",
      "Response for Row 1392:\n",
      "Answer to Question 2: (deletion of gene mel1, elimination of α-galactosidase reactions), (deletion of gene mel1, elimination of melibiose degradation), (deletion of gene mel1, elimination of galactose by-product formation), (integration of gene fsy1, contribution to fructose transport), (integration of gene ffzi1, contribution to fructose transport), (overexpression of gene gcr1, enhancement of fructose by-product biopurification rate)\n",
      "Progress: 69.60% completed\n",
      "Response for Row 1393:\n",
      "Answer to Question 2: (deletion of gene MAL11, no growth on sucrose), (deletion of gene MAL21, no growth on sucrose), (deletion of gene MAL31, no growth on sucrose), (deletion of gene MPH2, no growth on sucrose), (deletion of gene MPH3, no growth on sucrose), (deletion of gene SUC2, growth on sucrose), (deletion of gene MAL12, growth on sucrose), (deletion of gene MAL22, growth on sucrose), (deletion of gene MAL32, growth on sucrose), (upregulation of gene IMA1, increased sucrose-hydrolysis activity), (upregulation of gene IMA5, increased sucrose-hydrolysis activity), (deletion of gene IMA1, no sucrose hydrolysis), (deletion of gene IMA2, no sucrose hydrolysis), (deletion of gene IMA3, no sucrose hydrolysis), (deletion of gene IMA4, no sucrose hydrolysis), (deletion of gene IMA5, no sucrose hydrolysis)\n",
      "Progress: 69.65% completed\n",
      "Response for Row 1394:\n",
      "Answer to Question 2: (deletion of gene, increased SAM accumulation), (overexpression of gene, increased SAM accumulation), (mutation of gene, increased SAM accumulation)\n",
      "Progress: 69.70% completed\n",
      "Response for Row 1395:\n",
      "Answer to Question 2: (deletion of gene hrdA, increased levels of mutant MsdS in intracellular fraction), (overexpression of mutant MsdS, secreted to medium), (overexpression of mutant MsdS, underwent hyperglycosylation), (overexpression of mutant MsdS, induced UPR), (deletion of gene α-amylase, alleviated UPR induction by mutant MsdS overexpression)\n",
      "Progress: 69.75% completed\n",
      "Response for Row 1396:\n",
      "Answer to Question 2: (SCH9 deletion, shortens chronological life span), (SCH9 deletion, increases glycerol production), (SCH9 deletion, upregulates amino acid biosynthesis pathways), (SCH9 deletion, increases glycerol-3-P dehydrogenase Gpd1p protein level), (SCH9 deletion, downregulates proteinogenic amino acids), (SCH9 deletion, increases lipids such as ergosterol), (GCN4 overexpression, increases glycerol production), (GCN4 overexpression, downregulates glycolysis)\n",
      "Progress: 69.80% completed\n",
      "Response for Row 1397:\n",
      "Answer to Question 2: (deletion of gene VPS30, consequence 1: increased production of recombinant antibody 2-fold), (deletion of gene VPS30, consequence 2: increased expression of endogenous yeast acid phosphatase 1.7-fold), (deletion of gene TAR1, consequence 1: mildly affected secretion levels), (deletion of gene TAR1, consequence 2: increased expression of endogenous yeast acid phosphatase 2.4-fold)\n",
      "Progress: 69.85% completed\n",
      "Response for Row 1398:\n",
      "Answer to Question 2: (deletion of gene glo1, consequence significant production of H2S), (deletion of gene glo1, consequence inefficient sulfate assimilation), (deletion of gene glo1, consequence normal growth on cysteine), (overexpression of gene glo1, consequence dominant negative phenotype), (deletion of gene glo1, consequence reduced expression of Met17p)\n",
      "Progress: 69.90% completed\n",
      "Response for Row 1399:\n",
      "Answer to Question 2: (deletion of NC domain, abrogates RNA-based inhibition of Gag-membrane binding)\n",
      "Progress: 69.95% completed\n",
      "Response for Row 1400:\n",
      "Answer to Question 2: (deletion of gene Set2, transcription initiation of antisense RNAs), (deletion of gene Set2, production of Set2-repressed antisense transcripts (SRATs)), (addition of H3K36 methyl mark by Set2, suppression of SRATs)\n",
      "Progress: 70.00% completed\n",
      "Response for Row 1401:\n",
      "Answer to Question 2: (deletion of gene VPS10, consequence Vps10p is the dominant sorting receptor for PrA Golgi-to-vacuolar transportation), (overexpression of gene VPS10, consequence PrA activity in the broth lowered by 76.96%), (overexpression of gene VPS10, consequence increased growth rate), (overexpression of gene VPS10, consequence higher biomass), (overexpression of gene VPS10, consequence faster fermentation speed), (overexpression of gene VPS10, consequence thorough wort fermentation), (overexpression of gene VPS10, consequence 1.41-fold higher ethanol production), (deletion of gene SEC5, consequence PrA activity in the broth lowered by 32.39%)\n",
      "Progress: 70.05% completed\n",
      "Response for Row 1402:\n",
      "Answer to Question 2: (deletion of 28 amino acids in the hydrophilic loop connecting TM8 and TM9 of Acr3, no effect on Acr3 folding), (deletion of 28 amino acids in the hydrophilic loop connecting TM8 and TM9 of Acr3, no effect on Acr3 trafficking), (deletion of 28 amino acids in the hydrophilic loop connecting TM8 and TM9 of Acr3, no effect on substrate binding), (deletion of 28 amino acids in the hydrophilic loop connecting TM8 and TM9 of Acr3, no effect on transport activity)\n",
      "Progress: 70.10% completed\n",
      "Response for Row 1403:\n",
      "Answer to Question 2: (deletion of transcription factor, improved ethanol production), (deletion of transcription factor, increased xylose import rates), (deletion of transcription factor, improved xylose integration into central carbon metabolism)\n",
      "Progress: 70.15% completed\n",
      "Response for Row 1404:\n",
      "Answer to Question 2: (deletion of gene UBP3, exacerbated karyotype-specific phenotypes), (deletion of gene UBP3, increased global stresses of aneuploid cells), (deletion of gene UBP3, enhanced oxidative stress), (deletion of gene UBP3, increased proteotoxic stress), (deletion of gene UBP3, impaired proteasome function), (deletion of gene USP10, detrimental to the fitness of human cells upon chromosome missegregation), (deletion of gene USP10, autophagy inhibition)\n",
      "Progress: 70.20% completed\n",
      "Response for Row 1405:\n",
      "Answer to Question 2: (deletion of gene Sef1, consequence full growth under iron limitation conditions), (overexpression of gene Aft1, consequence activation under iron starvation conditions), (overexpression of gene Cth2, consequence degradation of mRNAs encoding iron-requiring enzymes)\n",
      "Progress: 70.25% completed\n",
      "Response for Row 1406:\n",
      "Answer to Question 2: (deletion of GPD2, no effect on acetate consumption), (overexpression of AADH, increased acetate consumption), (overexpression of ACS, increased acetate consumption), (expression of XR and XDH, redox imbalance), (coupling of NADH-dependent acetate reduction pathway, efficient xylose fermentation), (coupling of NADH-dependent acetate reduction pathway, in situ detoxification of acetate)\n",
      "Progress: 70.30% completed\n",
      "Response for Row 1407:\n",
      "Answer to Question 2: (deletion of gene PRE3, impaired growth on glucose), (deletion of gene IDI1, impaired growth on glucose), (deletion of gene MRPL37, impaired growth on glucose), (deletion of gene NOC3, impaired growth on glucose), (deletion of gene NOP56, impaired growth on glucose), (deletion of gene RPL17, impaired growth on glucose)\n",
      "Progress: 70.35% completed\n",
      "Response for Row 1408:\n",
      "Answer to Question 2: (deletion of gene OCH1, severe defects in cell growth), (deletion of gene OCH1, defects in morphogenesis), (deletion of gene OCH1, response to environmental challenges), (overexpression of gene RHO1, partial recovery of growth defect), (overexpression of gene RHO1, enhanced glucan layer), (overexpression of gene RHO1, recovered mannoprotein layer), (overexpression of gene FKS2, complementation of cell wall structure), (overexpression of gene RHO1, improved temperature tolerance), (overexpression of gene RHO1, improved osmotic potential), (overexpression of gene RHO1, improved drug sensitivity), (overexpression of gene RHO1, increased N-glycan site occupancy), (overexpression of gene RHO1, increased amount of secreted glycoproteins)\n",
      "Progress: 70.40% completed\n",
      "Response for Row 1409:\n",
      "Answer to Question 2: (deletion of introns from RPS25A, RPS26A, RPS26B, and RPS24A promoters, decreased luciferase expression), (expression of intron-containing promoters RPS25A, RPS26A, RPS26B, and RPS24A, higher expression than TDH3 promoter), (insertion of RPS25A intron into TDH3 promoter, 50-fold higher expression than TDH3p alone), (insertion of RPS25A intron into GAL10 promoter, enhanced expression), (insertion of RPS25A intron into CUP1 promoter, enhanced expression), (RPS25A intron in chimeric promoter, enhances expression of some recombinant proteins)\n",
      "Progress: 70.45% completed\n",
      "Response for Row 1410:\n",
      "Answer to Question 2: (deletion of gene involved in mitochondrial functions, resistance to valinomycin), (deletion of gene involved in mitochondrial functions, resistance to nigericin), (deletion of gene involved in ribosome biogenesis, hypersensitivity to valinomycin), (deletion of gene involved in ribosome biogenesis, hypersensitivity to nigericin), (deletion of gene involved in telomere maintenance, resistance to valinomycin), (deletion of gene involved in telomere maintenance, resistance to nigericin), (deletion of gene involved in vacuolar biogenesis, hypersensitivity to valinomycin), (deletion of gene involved in vacuolar biogenesis, hypersensitivity to nigericin), (deletion of gene involved in stress response, resistance to valinomycin), (deletion of gene involved in stress response, resistance to nigericin)\n",
      "Progress: 70.50% completed\n",
      "Response for Row 1411:\n",
      "Answer to Question 2: (deletion of gene GSF2, improved LA tolerance), (deletion of gene GSF2, improved LA production), (deletion of gene GSF2, improved glucose uptake), (deletion of gene GSF2, derepression of glucose-repressed genes), (deletion of gene GSF2, increased respiratory pathway activity), (deletion of gene GSF2, more efficient ATP generation), (deletion of gene GSF2, more efficient NAD+ generation), (deletion of gene MIG1, improved D-LA production), (deletion of gene HXK2, improved D-LA production)\n",
      "Progress: 70.55% completed\n",
      "Response for Row 1412:\n",
      "Answer to Question 2: (deletion of gene KU70, enhanced double-crossover homologous recombination rate), (deletion of gene KU70, increased precise homologous recombination rate), (deletion of 11 target genes encoding alcohol dehydrogenase and alcohol oxidase, facilitated by enhanced gene deletion efficiency), (expression of Cre recombinase, removal of selection marker gene)\n",
      "Progress: 70.60% completed\n",
      "Response for Row 1413:\n",
      "Answer to Question 2: (deletion of gene HXK2, reduced acetic acid production), (deletion of gene PYK1, reduced acetic acid production), (deletion of gene REG1, reduced acetic acid production), (deletion of gene PDE2, reduced acetic acid production), (deletion of gene PDC1, reduced acetic acid production), (deletion of gene REG1, altered glycerol production), (deletion of gene PDC1, altered glycerol production), (deletion of gene REG1, altered pyruvate derived metabolites production), (deletion of gene PDC1, altered pyruvate derived metabolites production)\n",
      "Progress: 70.65% completed\n",
      "Response for Row 1414:\n",
      "Answer to Question 2: (deletion of gene CYP61, decreased ergosterol biosynthesis), (deletion of gene CYP61, increased astaxanthin production)\n",
      "Progress: 70.70% completed\n",
      "Response for Row 1415:\n",
      "Answer to Question 2: (expression/deletion of gene xxx, consequence 1), (expression/deletion of gene xxx, consequence 2)\n",
      "Progress: 70.75% completed\n",
      "Response for Row 1416:\n",
      "Answer to Question 2: (deletion of gene Tom1, consequence 1: reduced ubiquitination of multiple RPs), (deletion of gene Tom1, consequence 2: exceptional accumulation of detergent-insoluble proteins including multiple RPs), (deletion of gene Tom1, consequence 3: hypersensitivity to imbalances in production of RPs and rRNA), (overproduction of gene Rpl26, consequence 1: fails to assemble into ribosomes), (overproduction of gene Rpl26, consequence 2: degraded in the nucleus/nucleolus by a ubiquitin-proteasome system)\n",
      "Progress: 70.80% completed\n",
      "Response for Row 1417:\n",
      "Answer to Question 2: (deletion of gene GLC3, improvement of SAM accumulation), (deletion of gene SPE2, improvement of SAM accumulation), (deletion of gene ERG4, no obvious effect on SAM production), (deletion of gene ERG6, no obvious effect on SAM production), (overexpression of gene SAM2, 12.47g/L SAM production), (overexpression of gene SAM2, 81.0% increase in SAM production compared to parent strain)\n",
      "Progress: 70.85% completed\n",
      "Response for Row 1418:\n",
      "Answer to Question 2: (deletion of nuclease activity in CRISPR/Cas9, reduction in toxicity), (expression of Target-AID complex, induction of specific point mutation at cytidines), (use of uracil DNA glycosylase inhibitor, suppression of indel formation), (expression of Target-AID complex, improvement in mutagenesis efficiency)\n",
      "Progress: 70.90% completed\n",
      "Response for Row 1419:\n",
      "Answer to Question 2: (deletion of gene EXG1, enhancement of flavonoid glucoside production), (over-expression of phosphoglucomutase, increase in uridine diphosphate glucose synthesis), (over-expression of UTP-glucose-1-phosphate uridylyltransferase, increase in uridine diphosphate glucose synthesis), (expression of gene SbGT34, synthesis of flavonoid glucosides), (deletion of gene SPR1, reduction in glucosidic bond hydrolysis), (deletion of gene YIR007W, reduction in glucosidic bond hydrolysis)\n",
      "Progress: 70.95% completed\n",
      "Response for Row 1420:\n",
      "Answer to Question 2:  (deletion of gene TORC1, consequence decreases inhibition of SNF1 by TORC1), (deletion of gene PKA, consequence mitigates PKA pathway), (no specific gene deletion, consequence coordinates unknown processes), (deletion of gene PKA, consequence diminishes PKA inhibition on SNF1), (overexpression of gene Rim15, consequence intensifies Rim15 activity), (deletion of gene Sch9, consequence inhibits Sch9 activated by PKH1/2)\n",
      "Progress: 71.00% completed\n",
      "Response for Row 1421:\n",
      "Answer to Question 2: (deletion of gene PRO1, defect for ribophagy), (deletion of gene PRO1, sensitivity to various stresses), (expression of inactive γ-Glutamyl kinase, defect for ribophagy)\n",
      "Progress: 71.05% completed\n",
      "Response for Row 1422:\n",
      "Answer to Question 2: (gene overexpression, re-routing carbon flux towards desired product), (gene deletion, growth-product coupling)\n",
      "Progress: 71.10% completed\n",
      "Response for Row 1423:\n",
      "Answer to Question 2: (expression of gene pfldh, l-lactic acid formation from glucose), (expression of gene pfldh, l-lactic acid formation from xylose), (deletion of gene pdc5, reduced ethanol formation), (deletion of gene pdc5, reduced xylitol formation)\n",
      "Progress: 71.15% completed\n",
      "Response for Row 1424:\n",
      "Answer to Question 2: (deletion of gene MoYcp4, consequence growth rate reduced), (deletion of gene MoYcp4, consequence conidial production increased), (deletion of gene MoYcp4, consequence appressorial turgor abnormal), (deletion of gene MoYcp4, consequence ability to infect rice and barley reduced), (deletion of gene MoYcp4, consequence pathogenicity decreased)\n",
      "Progress: 71.20% completed\n",
      "Response for Row 1425:\n",
      "Answer to Question 2: (deletion of gene BcMtg2, defect in spore production), (deletion of gene BcMtg2, defect in conidial germination), (deletion of gene BcMtg2, defect in sclerotial formation), (deletion of gene BcMtg2, increased sensitivity to environmental stresses), (deletion of gene BcMtg2, increased sensitivity to osmotic stress), (deletion of gene BcMtg2, increased sensitivity to oxidative stress), (deletion of gene BcMtg2, increased sensitivity to Congo red), (deletion of gene BcMtg2, decreased virulence on host plant tissues), (deletion of gene BcMtg2, minor glycerol response to osmosis stress)\n",
      "Progress: 71.25% completed\n",
      "Response for Row 1426:\n",
      "Answer to Question 2: (overexpression of gene ACC1, increase in ethyl caproate production), (deletion of gene OPI1, increase in ethyl caproate production), (overexpression of gene FAS1, increase in ethyl caproate production), (overexpression of gene FAS1, increase in acetyl-CoA content), (overexpression of gene FAS1, increase in ethyl acetate content), (overexpression of gene FAS2, increase in ethyl caproate production), (overexpression of gene FAS1, increase in FAEEs content)\n",
      "Progress: 71.30% completed\n",
      "Response for Row 1427:\n",
      "Answer to Question 2: (deletion of gene, high gene deletion efficiency)\n",
      "Progress: 71.35% completed\n",
      "Response for Row 1428:\n",
      "Answer to Question 2: (deletion of WHEP domain, reduced catalytic efficiency), (deletion of WHEP domain, increased thermal stability), (deletion of WHEP domain, decreased structural flexibility)\n",
      "Progress: 71.40% completed\n",
      "Response for Row 1429:\n",
      "Answer to Question 2: (deletion of gene ADH1, elimination of ethanol production), (deletion of gene ADH2, elimination of ethanol production), (deletion of gene ADH3, elimination of ethanol production), (deletion of gene ADH4, elimination of ethanol production), (deletion of gene ADH5, elimination of ethanol production), (deletion of gene GPD1, elimination of glycerol production), (deletion of gene GPD2, elimination of glycerol production), (overexpression of gene AlsS, increase in acetoin production), (overexpression of gene AlsD, increase in acetoin production), (deletion of gene BDH1, reduction in conversion of acetoin to 2,3-butanediol), (overexpression of gene NoxE, relief of cofactor imbalance), (overexpression of gene NoxE, increase in acetoin production)\n",
      "Progress: 71.45% completed\n",
      "Response for Row 1430:\n",
      "Answer to Question 2: (deletion of gene end3, consequence internalization of AgNPs not observed), (deletion of genes related to oxidative stress defense, consequence no change in toxicity pattern), (deletion of genes related to cell wall/membrane integrity, consequence no change in toxicity pattern)\n",
      "Progress: 71.50% completed\n",
      "Response for Row 1431:\n",
      "Answer to Question 2: (deletion of gene xxx, consequence 1), (deletion of gene xxx, consequence 2)\n",
      "Progress: 71.55% completed\n",
      "Response for Row 1432:\n",
      "Answer to Question 2: (deletion of gene Irc3, loss of mitochondrial genome integrity), (deletion of gene Irc3, accumulation of irregular branched molecules), (overexpression of gene RecG, complementation of Irc3 deletion)\n",
      "Progress: 71.60% completed\n",
      "Response for Row 1433:\n",
      "Answer to Question 2: (deletion of gene F508, consequence functional expression defect of CFTR at apical plasma membrane), (deletion of gene F508, consequence degradation of misfolded CFTR at ER), (deletion of gene F670, consequence folding and trafficking defects of Yor1), (silencing of gene RPL12, consequence increased folding efficiency of ΔF508-CFTR), (silencing of gene RPL12, consequence increased conformational stability of ΔF508-CFTR), (silencing of gene RPL12, consequence augmented PM density and function of ΔF508-CFTR), (combination of RPL12 knockdown and VX-809, consequence restoration of mutant function to ~50% of wild-type channel)\n",
      "Progress: 71.65% completed\n",
      "Response for Row 1434:\n",
      "Answer to Question 2: (deletion of gene OCH1, improved extracellular activities of heterologous proteins), (deletion of gene MNN9, improved extracellular activities of heterologous proteins), (deletion of gene OCH1, up-regulated genes in the secretory pathway), (deletion of gene MNN9, up-regulated genes in the secretory pathway), (deletion of gene OCH1, affected cell wall integrity), (deletion of gene MNN9, affected cell wall integrity)\n",
      "Progress: 71.70% completed\n",
      "Response for Row 1435:\n",
      "Answer to Question 2: (deletion of multiple genes, redirection of metabolic fluxes), (chromosomal integration of expression cassettes, redirection of metabolic fluxes), (expression of beta-carotene biosynthetic pathway, increased beta-carotene production)\n",
      "Progress: 71.75% completed\n",
      "Response for Row 1436:\n",
      "Answer to Question 2: (deletion of ALD6, increase in glycerol yield), (deletion of ALD6, enhancement of acetate consumption), (expression of NAD(+)-dependent 6-phosphogluconate dehydrogenase, increase in cytosolic NADH formation), (expression of NAD(+)-dependent 6-phosphogluconate dehydrogenase, increase in glycerol yield), (replacement of glycerol formation by acetate reduction pathway, increase in acetate consumption), (combined expression of NAD(+)-dependent 6-phosphogluconate dehydrogenase and deletion of ALD6, increase in ethanol yield on glucose)\n",
      "Progress: 71.80% completed\n",
      "Response for Row 1437:\n",
      "Answer to Question 2: (deletion of gene ASC1, compromised translation of specific functional groups), (deletion of gene ASC1, cellular phenotypes consistent with gene-specific translation defects), (depletion of eIF4G, mimics the translational defects of ASC1 mutants)\n",
      "Progress: 71.85% completed\n",
      "Response for Row 1438:\n",
      "Answer to Question 2: (deletion of gene MoYVH1, consequence 1: significant reductions in vegetative growth), (deletion of gene MoYVH1, consequence 2: conidial production reduction), (deletion of gene MoYVH1, consequence 3: virulence reduction), (deletion of gene MoYVH1, consequence 4: defects in cell-wall integrity), (deletion of gene MoYVH1, consequence 5: hyposensitivity to exogenous osmotic stress), (deletion of gene MoYVH1, consequence 6: defects in appressorium function), (deletion of gene MoYVH1, consequence 7: invasive hyphae growth defects), (overexpression of gene MoPDEH, consequence 1: suppression of conidia formation defects), (overexpression of gene MoPDEH, consequence 2: suppression of pathogenicity defects)\n",
      "Progress: 71.90% completed\n",
      "Response for Row 1439:\n",
      "Answer to Question 2: (overexpression of gene pbcA, consequence PD production), (deletion of gene pbcA, consequence abolished PD production), (overexpression of gene cps/ks, consequence ent-kaurene production), (overexpression of gene gamma-terpinene synthase, consequence gamma-terpinene production)\n",
      "Progress: 71.95% completed\n",
      "Response for Row 1440:\n",
      "Answer to Question 2: (deletion of gene Sgs1, consequence 1), (deletion of gene Mph1, consequence 1), (deletion of gene Sgs1, consequence 2), (deletion of gene Mph1, consequence 2), (overexpression of gene Rad51, consequence 1)\n",
      "Progress: 72.00% completed\n",
      "Response for Row 1441:\n",
      "Answer to Question 2: (deletion of gene metJ, increase in SAM availability), (overexpression of gene metA, increase in SAM availability), (overexpression of gene cysE, increase in SAM availability), (overexpression of gene mtn, increase in SAM regeneration), (overexpression of gene luxS, increase in SAM regeneration), (supplementation with methionine, increase in vanillate titers)\n",
      "Progress: 72.05% completed\n",
      "Response for Row 1442:\n",
      "Answer to Question 2: (deletion of gene JJJ1, shorter lag phase), (deletion of gene JJJ1, increased specific ethanol production rate), (deletion of gene JJJ1, higher long chain fatty acid content), (deletion of gene JJJ1, higher trehalose content), (deletion of gene JJJ1, higher catalase content)\n",
      "Progress: 72.10% completed\n",
      "Response for Row 1443:\n",
      "Answer to Question 2: (deletion of gene CgTRK1, consequence affects growth at low potassium concentrations), (deletion of gene CgTRK1, consequence reduces tolerance to toxic alkali-metal cations), (deletion of gene CgTRK1, consequence reduces tolerance to cationic drugs), (deletion of gene CgTRK1, consequence alters membrane potential), (deletion of gene CgTRK1, consequence alters intracellular pH)\n",
      "Progress: 72.15% completed\n",
      "Response for Row 1444:\n",
      "Answer to Question 2: (overexpression of gene MAL62, increased trehalose content), (overexpression of gene MAL62, increased cell viability after freezing), (deletion of gene NTH1, strengthened freezing tolerance), (deletion of gene NTH1, improved leavening ability after freezing-thaw stress)\n",
      "Progress: 72.20% completed\n",
      "Response for Row 1445:\n",
      "Answer to Question 2: (deletion of gene rtfA, reduced conidial production), (deletion of gene rtfA, decreased sclerotial production), (deletion of gene rtfA, reduced production of aflatoxin B1), (deletion of gene rtfA, altered production of secondary metabolites), (deletion of gene rtfA, controls expression of veA), (deletion of gene rtfA, controls expression of laeA)\n",
      "Progress: 72.25% completed\n",
      "Response for Row 1446:\n",
      "Answer to Question 2: (deletion of gene horA, reduced coenzyme Q content), (deletion of gene horA, impaired response to oxidative stress), (deletion of gene horA, impaired response to reductive stress), (deletion of gene horA, delay in germination), (deletion of gene horA, accumulation of NADH), (deletion of gene horA, increased resistance against antifungal drugs), (deletion of gene horA, attenuated virulence in murine infection models)\n",
      "Progress: 72.30% completed\n",
      "Response for Row 1447:\n",
      "Answer to Question 2: (deletion of gene MNN1, elimination of interfering structures), (expression of UDP-N-acetylglucosamine transporter from Kluyveromyces lactis, increase in complex-type glycan abundance), (deletion of gene MNN1, improvement in glycan pattern homogeneity), (expression of UDP-N-acetylglucosamine transporter from Kluyveromyces lactis, improvement in complex-type glycosylation efficiency)\n",
      "Progress: 72.35% completed\n",
      "Response for Row 1448:\n",
      "Answer to Question 2:  (deletion of gene NIK1, insensitivity to dicarboximide and phenylpyrrole fungicides), (deletion of gene HOG1, insensitivity to dicarboximide and phenylpyrrole fungicides), (deletion of gene NIK1, increased sensitivity to hyperosmotic stress), (deletion of gene HOG1, increased sensitivity to hyperosmotic stress), (deletion of gene HOG1, increased tolerance to 6-methoxy-2-benzoxazolinone), (deletion of gene HOG1, reduced production of pycnidia)\n",
      "Progress: 72.40% completed\n",
      "Response for Row 1449:\n",
      "Answer to Question 2: (overexpression of gene ARO10, increased 2-PE production), (overexpression of gene ADH1, increased 2-PE production), (deletion of gene aro8, increased 2-PE production), (deletion of gene aat2, increased 2-PE production)\n",
      "Progress: 72.45% completed\n",
      "Response for Row 1450:\n",
      "Answer to Question 2: (deletion of gene ACE2, increased ethanol yields), (deletion of gene SWI5, increased ethanol yields), (overexpression of gene ACE2, reduced ethanol yields), (overexpression of gene SWI5, reduced ethanol yields), (deletion of gene CTS1, increased ethanol yields), (deletion of gene RPS4a, increased ethanol yields), (deletion of gene SIC1, increased ethanol yields), (deletion of gene EGT2, increased ethanol yields), (deletion of gene DSE2, increased ethanol yields), (deletion of gene SCP160, increased ethanol yields), (overexpression of gene CTS1, reduced ethanol yields), (overexpression of gene RPS4a, reduced ethanol yields), (deletion of gene MCR1, decreased ethanol yields), (deletion of gene HO, decreased ethanol yields)\n",
      "Progress: 72.50% completed\n",
      "Response for Row 1451:\n",
      "Answer to Question 2: (deletion of gene PEF1, reduction in virulence), (deletion of gene PEF1, delay in penetration), (deletion of gene PEF1, delay in invasive growth), (expression of gene PEF1, peroxisomal fission), (deletion of gene PEF1, formation of stringy linked immature peroxisomes), (efficient utilization of fatty acids, increased expression of PEF1), (deletion of gene PEF1, reduced intracellular production of ROS), (deletion of gene PEF1, induced ROS accumulation in host cells)\n",
      "Progress: 72.55% completed\n",
      "Response for Row 1452:\n",
      "Answer to Question 2: (deletion of gene RIM15, consequence increased fermentation rate), (deletion of gene RIM15, consequence utilization of available glucose instead of its storage), (deletion of gene RIM15, consequence decreased fungicide resistance), (deletion of gene RIM15, consequence reduced synthesis and transport of complex sugars into the membrane)\n",
      "Progress: 72.60% completed\n",
      "Response for Row 1453:\n",
      "Answer to Question 2: (overexpression of gene coaA, increased CoA synthesis), (deletion of genes ADH1-5, reduced ethanol formation), (expression of gene adhE(A267T/E568K), converted acetaldehyde into acetyl-CoA), (deletion of gene ADH6, decreased ethanol formation), (deletion of gene GPD2, decreased glycerol formation), (supplementation with pantothenate, increased n-butanol production)\n",
      "Progress: 72.65% completed\n",
      "Response for Row 1454:\n",
      "Answer to Question 2:  (deletion of gene PDH, consequence 1: predominant role in linking glycolysis and TCA cycle can be replaced by cytosolic PDH bypass and Cit2), (deletion of gene Ach1, consequence 1: strongly impaired growth), (deletion of gene Ach1, consequence 2: high incidence of respiratory deficiency), (overexpression of gene AGP2, consequence 1: enables carnitine shuttle to efficiently link glycolysis and TCA cycle), (overexpression of gene HNM1, consequence 1: enables carnitine shuttle to efficiently link glycolysis and TCA cycle), (overexpression of gene YAT2, consequence 1: enables carnitine shuttle to efficiently link glycolysis and TCA cycle), (overexpression of gene YAT1, consequence 1: enables carnitine shuttle to efficiently link glycolysis and TCA cycle), (overexpression of gene CRC1, consequence 1: enables carnitine shuttle to efficiently link glycolysis and TCA cycle), (overexpression of gene CAT2, consequence 1: enables carnitine shuttle to efficiently link glycolysis and TCA cycle)\n",
      "Progress: 72.70% completed\n",
      "Response for Row 1455:\n",
      "Answer to Question 2: (deletion of gene OPI1, increase in growth rate), (deletion of gene OPI1, alteration in cell membrane composition), (deletion of gene OPI1, alteration in cell wall composition), (deletion of gene OPI1, reduction in protein aggregation), (exposure to lactic acid, decrease in phosphatidylcholine), (exposure to lactic acid, rearrangement in lipid hydrocarbon tails), (exposure to lactic acid, rearrangement in cell wall carbohydrates), (exposure to lactic acid, transient protein aggregation)\n",
      "Progress: 72.75% completed\n",
      "Response for Row 1456:\n",
      "Answer to Question 2: (deletion of gene SSK1, elevated sensitivity to oxidants), (deletion of gene SSK1, failure to detoxify H2O2 effectively), (deletion of gene SSK1, minor necrosis on susceptible citrus leaves), (deletion of gene SSK1, resistance to dicarboximide fungicides), (deletion of gene SSK1, resistance to phenylpyrrole fungicides), (deletion of gene SSK1, impaired germ tube production from conidia), (deletion of gene SSK1, involvement in multi-drug resistance)\n",
      "Progress: 72.80% completed\n",
      "Response for Row 1457:\n",
      "Answer to Question 2: (deletion of gene XsTPS2, consequence none of the XsTPS2 product, guaia-4,6-diene, can be detected)\n",
      "Progress: 72.85% completed\n",
      "Response for Row 1458:\n",
      "Answer to Question 2: (deletion of gene RTT109, enhanced resistance to acetic acid), (deletion of gene RTT109, improved growth compared to wild-type), (deletion of gene RTT109, shortened lag phase), (deletion of gene RTT109, advanced glucose consumption), (deletion of gene RTT109, increased ethanol production rate), (deletion of gene RTT109, elevated transcription levels of HSP12), (deletion of gene RTT109, elevated transcription levels of CTT1), (deletion of gene RTT109, elevated transcription levels of GSH1), (deletion of gene RTT109, increased activities of antioxidant enzymes), (deletion of gene RTT109, improved resistance to oxidative stress), (deletion of gene RTT109, improved acetic acid tolerance)\n",
      "Progress: 72.90% completed\n",
      "Response for Row 1459:\n",
      "Answer to Question 2: (deletion of gene PUB1, increase in Gpd1p activity), (deletion of gene PUB1, higher intracellular glycerol concentration), (deletion of gene PUB1, greater tolerance to osmotic stress), (deletion of gene PUB1, enhancement of nicotinamidase activity)\n",
      "Progress: 72.95% completed\n",
      "Response for Row 1460:\n",
      "Answer to Question 2: (deletion of gene CA_C1502, consequence C. acetobutylicum Δcac1502Δupp strain is marker-less), (deletion of gene upp, consequence C. acetobutylicum Δcac1502Δupp strain is marker-less), (deletion of gene CA_C3535, consequence improves efficiency of method), (deletion of gene ctfAB, consequence strains no longer produce acetone), (deletion of gene ldhA, consequence strains no longer produce l-lactate)\n",
      "Progress: 73.00% completed\n",
      "Response for Row 1461:\n",
      "Answer to Question 2: ( deletion of gene CAR1, consequence 1), ( deletion of gene CAR1, reduced arginase activity), ( deletion of gene CAR1, decreased urea production), ( deletion of gene CAR1, decreased ethyl carbamate production)\n",
      "Progress: 73.05% completed\n",
      "Response for Row 1462:\n",
      "Answer to Question 2: (deletion of upstream components of CWI pathway, activation of Slt2 by heat stress), (interaction of Ste11 and Mkk1 mediated by Nst1, connection of high-osmolarity glycerol and pheromone pathways to CWI pathway in response to heat and pheromone), (activation of Ser(407) and Thr(411) residues of Mkk1, activation by MAPK pathways)\n",
      "Progress: 73.10% completed\n",
      "Response for Row 1463:\n",
      "Answer to Question 2: (maf1 deletion, increased pyruvate kinase activity), (maf1 deletion, enhanced ethanol production), (maf1 deletion, elevated lipid content), (maf1 deletion, accumulation of smaller lipid droplets)\n",
      "Progress: 73.15% completed\n",
      "Response for Row 1464:\n",
      "Answer to Question 2: (deletion of gene lys5, consequence 1), (expression of gene EppA, consequence 1), (expression of gene EppA, consequence 2)\n",
      "Progress: 73.20% completed\n",
      "Response for Row 1465:\n",
      "Answer to Question 2: (deletion of gene PHO13, enhancement of ethanol production from xylose), (overexpression of gene XK, increase in ethanol yield), (deletion of gene GRE3, increase in ethanol yield), (deletion of gene PHO13, upregulation of GND1 expression), (deletion of gene PHO13, upregulation of SOL3 expression), (deletion of gene PHO13, upregulation of TAL1 expression), (deletion of gene PHO13, upregulation of RKI1 expression), (deletion of gene PHO13, upregulation of TKL1 expression), (deletion of gene PHO13, upregulation of NDE1 expression), (deletion of gene PHO13, upregulation of ACO1 expression), (deletion of gene PHO13, upregulation of ACO2 expression), (deletion of gene PHO13, upregulation of SDH2 expression), (deletion of gene PHO13, upregulation of IDH1 expression), (deletion of gene PHO13, upregulation of IDH2 expression), (deletion of gene PHO13, upregulation of ATP7 expression), (deletion of gene PHO13, upregulation of ATP19 expression), (deletion of gene PHO13, upregulation of SDH4 expression), (deletion of gene PHO13, upregulation of SDH3 expression), (deletion of gene PHO13, upregulation of CMC2 expression), (deletion of gene PHO13, upregulation of ATP15 expression), (deletion of gene PHO13, change in expression levels of 125 cell cycle genes)\n",
      "Progress: 73.25% completed\n",
      "Response for Row 1466:\n",
      "Answer to Question 2: (deletion of gene lag1, consequence elevated free fatty acids), (deletion of gene lag1, consequence elevated sphingoid bases), (deletion of gene lac1, consequence elevated free fatty acids), (deletion of gene lac1, consequence elevated sphingoid bases), (deletion of gene ypc1, consequence small amounts of complex sphingolipids), (deletion of gene ydc1, consequence small amounts of complex sphingolipids), (deletion of gene yjr116w, consequence oxygen-dependent hypersensitivity to Cu2+), (deletion of gene yjr116w, consequence increased mitochondrial production of reactive oxygen species), (deletion of gene yjr116w, consequence mitochondrially orchestrated programmed cell death in presence of copper), (deletion of gene ypr114w, consequence general copper hypersensitivity), (deletion of gene ypr114w, consequence fragmented vacuoles), (deletion of gene ypr114w, consequence produces less ROS than wild type), (deletion of gene ypr114w, consequence increased chronological life span)\n",
      "Progress: 73.30% completed\n",
      "Response for Row 1467:\n",
      "Answer to Question 2: (deletion of genes, stabilization of numerous mRNAs), (deletion of genes, induction of selected lncRNAs), (activation of HOG signaling pathway, stabilization of numerous mRNAs), (activation of HOG signaling pathway, induction of selected lncRNAs)\n",
      "Progress: 73.35% completed\n",
      "Response for Row 1468:\n",
      "Answer to Question 2: (gene overexpression, increase in titer of SBCFAs), (gene deletion, improve production of SBCFAs), (overexpression of PDR12, increase secretion of SBCFAs)\n",
      "Progress: 73.40% completed\n",
      "Response for Row 1469:\n",
      "Answer to Question 2: (deletion of gene maf1, general activation of RNAP III), (deletion of gene maf1, decrease in N2,N2-dimethyl G26 (m(2)2G26) modification efficiency), (deletion of gene maf1, antisuppression phenotype), (over-expression of gene Trm1, reversal of maf1-antisuppression), (mutation in RNAP III, decrease in tRNA transcription), (mutation in RNAP III, increase in m(2)2G26 efficiency), (treatment with rapamycin, decrease in tRNA synthesis), (treatment with rapamycin, increase in m(2)2G26 modification)\n",
      "Progress: 73.45% completed\n",
      "Response for Row 1470:\n",
      "Answer to Question 2: (deletion of gene Zap1p, reduced expression of ETR1), (reduced expression of ETR1, reduced mitochondrial fatty acid biosynthesis), (reduced expression of ETR1, reduction in lipoic acid content), (deletion of gene ETR1, accumulation of TAG), (introduction of ETR1 in zap1Δ strain, rescue of TAG level), (compromised mitochondrial fatty acid biosynthesis, reduction in lipoic acid), (reduction in lipoic acid, loss of mitochondrial function), (defect in mitochondria, slowed ATP production), (slowed ATP production, carbon channeled towards lipid biosynthesis), (carbon channeled towards lipid biosynthesis, stored as TAG)\n",
      "Progress: 73.50% completed\n",
      "Response for Row 1471:\n",
      "Answer to Question 2:  (deletion of gene xxx, increased sensitivity to 5-fluorouracil), (deletion of gene xxx, increased sensitivity to nystatin), (deletion of gene xxx, altered transcription profile similar to human HepG2 cells)\n",
      "Progress: 73.55% completed\n",
      "Response for Row 1472:\n",
      "Answer to Question 2: (expression of gene GYP8, increased acetic acid tolerance), (expression of gene WSC4, increased acetic acid tolerance), (expression of gene PMT1, increased acetic acid tolerance), (expression of gene KTR7, increased acetic acid tolerance), (expression of gene RKR1, increased acetic acid tolerance), (expression of gene TIF3, increased acetic acid tolerance), (expression of gene ILV3, increased acetic acid tolerance), (expression of gene MSN4, increased acetic acid tolerance), (expression of gene ZBAI_02295, increased acetic acid tolerance), (expression of gene ZbMSN4, increased acetic acid tolerance), (expression of gene ZbTIF3, increased acetic acid tolerance)\n",
      "Progress: 73.60% completed\n",
      "Response for Row 1473:\n",
      "Answer to Question 2: (deletion of gene SNF1, consequence 1: mimics the addition of glucose), (deletion of gene SNF1, consequence 2: prevents glucose-induced mRNA decay), (deletion of gene VTS1, consequence 1: does not slow the rate of glucose-induced mRNA decay), (expression of gene SNF1, consequence 1: influences mRNA stability by altering the recruitment activity of the transcription factor Adr1)\n",
      "Progress: 73.65% completed\n",
      "Response for Row 1474:\n",
      "Answer to Question 2: (knockdown of gene SSB1, increased lactic acid tolerance), (knockdown of gene SSB1, increased cell growth), (knockdown of gene SSB1, increased glucose consumption), (knockdown of gene SSB1, increased L-lactic acid production)\n",
      "Progress: 73.70% completed\n",
      "Response for Row 1475:\n",
      "Answer to Question 2: (deletion of mitochondrial gene, enhanced sensitivity to sampangine), (expression of sampangine, increased oxygen consumption rates), (expression of sampangine, induction of reactive oxygen species), (expression of sampangine, alteration in heme biosynthesis), (expression of sampangine, redox cycling leading to superoxide production), (high concentration of sampangine, neurotoxicity)\n",
      "Progress: 73.75% completed\n",
      "Response for Row 1476:\n",
      "Answer to Question 2: (overexpression of genes of alkaline phosphatase, reduced biomass accumulation), (overexpression of genes of apyrase, reduced biomass accumulation), (overexpression of enzymes involved in futile cycles, reduced biomass accumulation), (overexpression of XYL1, improved xylose fermentation), (overexpression of XYL2, improved xylose fermentation), (overexpression of XYL3, improved xylose fermentation), (deletion of CAT8, improved xylose fermentation), (overexpression of DAS1, increased ethanol yield from xylose), (overexpression of TAL2, increased ethanol yield from xylose), (deletion of ATG13, increased ethanol yield from xylose)\n",
      "Progress: 73.80% completed\n",
      "Response for Row 1477:\n",
      "Answer to Question 2: (deletion of gene sdh1, consequence shorter life span), (deletion of gene sdh2, consequence shorter life span), (deletion of gene sdh4, consequence shorter life span), (deletion of gene cor1, consequence shorter life span), (deletion of gene cyt1, consequence shorter life span), (deletion of gene qcr7, consequence shorter life span), (deletion of gene qcr8, consequence shorter life span), (deletion of gene rip1, consequence shorter life span), (deletion of gene cox6, consequence shorter life span), (deletion of gene cox7, consequence shorter life span), (deletion of gene cox9, consequence shorter life span), (deletion of gene atp4, consequence shorter life span), (deletion of gene atp7, consequence shorter life span), (deletion of gene atp17, consequence shorter life span), (deletion of gene complex II, consequence higher superoxide levels), (deletion of gene complex IV, consequence higher superoxide levels), (deletion of gene complex V, consequence lower ATP levels)\n",
      "Progress: 73.85% completed\n",
      "Response for Row 1478:\n",
      "Answer to Question 2: (expression of gene ldhA, D-LA production), (deletion of gene DLD1, inhibition of D-LA consumption), (deletion of gene JEN1, inhibition of D-LA consumption), (deletion of gene PDC1, reduction of ethanol production), (deletion of gene ADH1, reduction of ethanol production), (deletion of gene GPD1, elimination of glycerol production), (deletion of gene GPD2, elimination of glycerol production), (overexpression of gene HAA1, enhancement of LA tolerance), (expression of gene ldhA, increased D-LA production up to 48.9 g/L), (expression of gene ldhA, increased D-LA production up to 112.0 g/L)\n",
      "Progress: 73.90% completed\n",
      "Response for Row 1479:\n",
      "Answer to Question 2: (deletion of gene glpO, affected production of H2O2)\n",
      "Progress: 73.95% completed\n",
      "Response for Row 1480:\n",
      "Answer to Question 2: (deletion of gene PYC1, increased pyruvate production), (deletion of gene PYC2, increased pyruvate production), (deletion of gene PYC3, increased pyruvate production), (mutation of gene TUP1, enabled mannitol assimilation), (mutation of gene CYC8, enabled mannitol assimilation)\n",
      "Progress: 74.00% completed\n",
      "Response for Row 1481:\n",
      "Answer to Question 2: (deletion of Puf4, decrease in gene expression of mitochondrial complex-related mRNAs), (deletion of Puf3, increase in gene expression of mitochondrial complex-related mRNAs), (expansion of Puf4 targets, inclusion of TCA cycle genes), (expansion of Puf3 targets, inclusion of ETC complex I genes)\n",
      "Progress: 74.05% completed\n",
      "Response for Row 1482:\n",
      "Answer to Question 2: (deletion of gene involved in mitochondrial translation, altered extracellular ATP levels), (deletion of gene involved in vesicle sorting in the late endosome, altered extracellular ATP levels), (deletion of gene involved in amino acid signaling, altered ATP efflux), (deletion of gene involved in mitochondrial retrograde signaling, altered ATP efflux)\n",
      "Progress: 74.10% completed\n",
      "Response for Row 1483:\n",
      "Answer to Question 2: (deletion of gene protein kinase, consequence growth rate alteration), (deletion of gene phosphatase, consequence lag time duration change), (deletion of gene protein kinase, consequence growth efficiency change), (deletion of gene phosphatase, consequence stress-induced diauxic shift delay)\n",
      "Progress: 74.15% completed\n",
      "Response for Row 1484:\n",
      "Answer to Question 2: (deletion of gene JAC1, consequence lethal in S. pombe), (deletion of gene JAC1, consequence lethal in S. cerevisiae), (mutation in gene JAC1, consequence lethal in S. pombe), (mutation in gene JAC1, consequence non-lethal in S. cerevisiae), (coevolution of gene Ssq1, consequence increased efficiency of interaction with Jac1), (sequence divergence of gene Jac1, consequence higher in species encoding Ssq1)\n",
      "Progress: 74.20% completed\n",
      "Response for Row 1485:\n",
      "Answer to Question 2: (deletion of gene xxx, fitness loss due to mistranslation mitigated), (overexpression of hexose transporter genes, increased glucose uptake), (acceleration of ubiquitin-proteasome-mediated protein degradation, reduction of toxic protein aggregation), (increase in protein synthesis, mitigation of deleterious effects of mistranslation), (duplication of genes, large phenotypic leaps enabling rapid adaptation)\n",
      "Progress: 74.25% completed\n",
      "Response for Row 1486:\n",
      "Answer to Question 2: (deletion of gene flo8, consequence adhesion or flocculation in haploids), (deletion of gene flo8, consequence pseudohyphal growth in diploids), (deletion of gene somA, consequence slow growth), (deletion of gene somA, consequence block in asexual development), (deletion of gene somA, consequence inability to form further differentiated structures beyond aerial hyphae), (deletion of gene ptaB, consequence similar phenotype to somA deletion), (expression of gene somA, consequence biofilm formation), (expression of gene somA, consequence regulation of transcription factors controlling conidiation or adhesion), (deletion of gene somA, consequence reduced pathogenicity)\n",
      "Progress: 74.30% completed\n",
      "Response for Row 1487:\n",
      "Answer to Question 2: (deletion of gene GLT1, negative effect on scavenging activity of glutamate against ROS accumulation), (overexpression of gene GLT1, lower viability in glutamine medium)\n",
      "Progress: 74.35% completed\n",
      "Response for Row 1488:\n",
      "Answer to Question 2: (deletion of gene AQP8, consequence 1), (deletion of gene AQP8, consequence 2), (deletion of gene AQP8, consequence 3), (deletion of gene AQP8, consequence 4)\n",
      "Progress: 74.40% completed\n",
      "Response for Row 1489:\n",
      "Answer to Question 2: (deletion of gene RIM15, impaired glucose-anabolic pathways), (deletion of gene RIM15, impaired synthesis of UDP-glucose), (deletion of gene RIM15, impaired synthesis of trehalose), (deletion of gene RIM15, impaired synthesis of glycogen), (deletion of gene RIM15, impaired accumulation of cell wall β-glucans), (deletion of gene RIM15, increased fermentation rate), (deletion of gene RIM15, impaired transcriptional induction of PGM2), (deletion of gene RIM15, impaired transcriptional induction of UGP1), (downregulation of UDPG synthesis pathway, accelerated alcoholic fermentation)\n",
      "Progress: 74.45% completed\n",
      "Response for Row 1490:\n",
      "Answer to Question 2: (deletion of gene essential genes, consequence reduce genome size), (expression of gene significant pathway genes, consequence improve vanillin production), (expression of gene significant pathway genes, consequence improve acetate production)\n",
      "Progress: 74.50% completed\n",
      "Response for Row 1491:\n",
      "Answer to Question 2: (overexpression of gene dhaB1B2B3, production of 3-HP), (overexpression of gene dhaR1R2, production of 3-HP), (overexpression of gene aldhH, production of 3-HP), (overexpression of gene GPD1, production of glycerol), (overexpression of gene GPP2, production of glycerol), (deletion of gene ptsG, relief of carbon catabolite repression), (overexpression of gene xylR, simultaneous utilization of glucose and xylose), (overexpression of gene xylR, production of glycerol at 0.48 g/g yield), (overexpression of gene xylR, production of glycerol at 0.35 g/L-h productivity), (expression of genes dhaB1B2B3, dhaR1R2, aldhH, GPD1, GPP2, ptsG deletion, xylR overexpression, production of 3-HP at 29.4 g/L), (expression of genes dhaB1B2B3, dhaR1R2, aldhH, GPD1, GPP2, ptsG deletion, xylR overexpression, production of 3-HP at 0.54 g/L-h productivity), (expression of genes dhaB1B2B3, dhaR1R2, aldhH, GPD1, GPP2, ptsG deletion, xylR overexpression, production of 3-HP at 0.36 g/g yield)\n",
      "Progress: 74.55% completed\n",
      "Response for Row 1492:\n",
      "Answer to Question 2: (deletion of gene HAP4, increased maximum concentration of ethanol), (deletion of gene HAP4, increased production rate of ethanol), (deletion of gene HAP4, increased yield of ethanol), (deletion of gene HAP4, capability to produce ethanol from xylose under aerobic conditions), (deletion of gene HAP4, improved ethanol yield from detoxified hydrolysate of wood chips)\n",
      "Progress: 74.60% completed\n",
      "Response for Row 1493:\n",
      "Answer to Question 2: (depletion of gene Aur1, accumulation of ceramides), (overexpression of gene YPC1, rescue of cells under Aur1 depletion), (overexpression of gene YPC1, occurrence of ceramide hydrolysis), (depletion of gene Aur1, requirement for vesicle-mediated transport between Golgi apparatus, endosomes, and vacuole), (acute repression of gene Aur1 by AbA, activation of vacuolar acidification), (acute repression of gene Aur1 by AbA, induction of cell wall integrity pathway)\n",
      "Progress: 74.65% completed\n",
      "Response for Row 1494:\n",
      "Answer to Question 2: (overexpression of gene Hxt7, increased glucose uptake rate), (overexpression of gene Hxt2, increased glucose uptake rate), (overexpression of gene Hxt4, increased glucose uptake rate), (deletion of gene STD1, differential effect on glucose uptake rate), (deletion of gene MTH1, differential effect on glucose uptake rate), (overexpression of gene GCR1, increased transcription levels of HXT1), (overexpression of gene GCR1, increased transcription levels of ribosomal protein genes), (overexpression of gene GCR1, increased ethanol production rate), (overexpression of gene GCR1, improved lactic acid production), (overexpression of gene GCR1, increased productivity of lactic acid), (overexpression of gene GCR1, increased titers of lactic acid)\n",
      "Progress: 74.70% completed\n",
      "Response for Row 1495:\n",
      "Answer to Question 2: (deletion of gene Slt2, differential effect of Rohitukine on cell viability), (deletion of gene Slt2, induction of apoptosis), (deletion of gene Hog1, differential effect of Rohitukine on cell viability), (deletion of gene Hog1, induction of apoptosis), (overexpression of gene p53, triggered apoptosis in A549 cells), (overexpression of gene caspase9, triggered apoptosis in A549 cells), (down regulation of gene Bcl-2, triggered apoptosis in A549 cells), (Rohitukine exposure, generation of reactive oxygen species), (Rohitukine exposure, induction of apoptosis)\n",
      "Progress: 74.75% completed\n",
      "Response for Row 1496:\n",
      "Answer to Question 2: (deletion of FAA2, increase in hexanoic acid levels), (deletion of FAA2, increase in octanoic acid levels), (deletion of PXA1, increase in hexanoic acid levels), (deletion of PXA1, increase in octanoic acid levels), (deletion of POX1, increase in hexanoic acid levels), (deletion of POX1, increase in octanoic acid levels), (deletion of ANT1, increase in hexanoic acid levels), (deletion of ANT1, increase in octanoic acid levels), (deletion of PEX11, increase in hexanoic acid levels), (deletion of PEX11, increase in octanoic acid levels), (optimization of hFAS, increase in hexanoic acid levels), (optimization of hFAS, increase in octanoic acid levels), (optimization of hFAS, increase in decanoic acid levels), (optimization of hFAS, increase in total short chain fatty acid levels)\n",
      "Progress: 74.80% completed\n",
      "Response for Row 1497:\n",
      "Answer to Question 2: (deletion of gene PDR1, consequence hypersensitivity to xenobiotics), (deletion of gene PDR3, consequence hypersensitivity to xenobiotics), (deletion of gene PDR1, consequence unaltered oxidative stress response), (deletion of gene PDR3, consequence unaltered oxidative stress response), (deletion of gene PDR1, consequence unaltered general stress response), (deletion of gene PDR3, consequence unaltered general stress response), (deletion of gene PDR1, consequence unaltered unfolded stress response), (deletion of gene PDR3, consequence unaltered unfolded stress response), (deletion of gene PDR1, consequence unaltered calcium signaling pathways), (deletion of gene PDR3, consequence unaltered calcium signaling pathways), (deletion of gene PDR1, consequence increased bioactivity of compounds), (deletion of gene PDR3, consequence increased bioactivity of compounds)\n",
      "Progress: 74.85% completed\n",
      "Response for Row 1498:\n",
      "Answer to Question 2: (deletion of genes involved in proline synthesis, weakened strain tolerance against FAP stress), (deletion of genes involved in myo-inositol synthesis, weakened strain tolerance against FAP stress), (addition of proline in medium, protective effect on cell growth under FAP stress), (addition of myo-inositol in medium, protective effect on cell growth under FAP stress), (overexpression of key gene PRO1, significantly increased FAP tolerance), (overexpression of key gene INO1, significantly increased FAP tolerance), (increase of proline content, decrease in intracellular level of reactive oxygen species under FAP condition)\n",
      "Progress: 74.90% completed\n",
      "Response for Row 1499:\n",
      "Answer to Question 2: (deletion of gene GntA, complete absence of zwitterionic GIPC), (expression of gene GntA, production of N-acetylhexosamine-IPC), (expression of gene GntA, generation of glycolipid product resistant to saponification and digestion by phosphatidylinositol-phospholipase C)\n",
      "Progress: 74.95% completed\n",
      "Response for Row 1500:\n",
      "Answer to Question 2: (deletion of genes encoding proteins homologous to those implicated in the Saccharomyces cerevisiae secretion apparatus, altered cellulolytic activity), (deletion of genes homologous to those known to differ between the Trichoderma reesei hyper-secreting strain RUT-C30 and its ancestral wild-type strain, altered cellulolytic activity), (deletion of genes encoding proteins identified in the secretome of N. crassa when cultured on plant biomass, altered cellulolytic activity), (deletion of genes encoding proteins predicted to traverse the secretory pathway, altered cellulolytic activity), (deletion of genes encoding proteins that act in transcription, altered cellulolytic activity), (deletion of genes encoding proteins that act in protein synthesis, altered cellulolytic activity), (deletion of genes encoding proteins that act in intracellular trafficking, altered cellulolytic activity), (deletion of genes encoding fungal-specific proteins of undetermined function, altered cellulolytic activity), (deletion of genes peripherally linked to protein processing or secretion, altered cellulolytic activity), (deletion of genes involved in the SREBP pathway, hyper-production phenotype), (deletion of SREBP pathway components in T. reesei, hyper-production phenotype), (deletion of a predicted E3 ligase, altered protein production under cellulolytic conditions), (deletion of SREBP transcription factor, altered protein production under cellulolytic conditions)\n",
      "Progress: 75.00% completed\n",
      "Response for Row 1501:\n",
      "Answer to Question 2: (deletion of gene FPS1, reducing glycerol export/formation), (expression of gene gapN, improving redox balance), (overexpression of gene UTR1, improving redox balance), (combined strategy, increasing ethanol yield by 4.6%)\n",
      "Progress: 75.05% completed\n",
      "Response for Row 1502:\n",
      "Answer to Question 2: (expression of gene CC1225, production of D-gluconate), (expression of gene CC1225, production of D-sorbitol), (expression of gene CC1225, conversion of D-xylose to xylitol), (expression of gene CC1225, production of D-xylonate), (deletion of gene Gre3p, no effect on xylitol production in Cc aaor expressing strain), (deletion of gene Gre3p, decrease in xylitol production in Zm gfor expressing strain), (expression of gene CC1225 and xylC, increase in D-xylonate production), (expression of gene CC1225 and xylC, increase in D-xylitol production)\n",
      "Progress: 75.10% completed\n",
      "Response for Row 1503:\n",
      "Answer to Question 2: (deletion of gene filC, accumulation of filipin III derivatives), (deletion of gene filD, accumulation of filipin III derivatives), (disruption of genes in the genomic region, identification of biosynthetic cluster for the 28-membered ring of the polyene macrolide filipin), (cytochrome P450 monooxygenase encoded by filC, catalyses specific hydroxylations at C26), (cytochrome P450 monooxygenase encoded by filD, catalyses specific hydroxylations at C1'), (formation of filipin III from filipin I through filipin II, alternative route), (formation of filipin III from filipin I through 1'-hydroxyfilipin I, alternative route), (1'-hydroxyfilipin and filipin II, enhanced antifungal bioactivity)\n",
      "Progress: 75.15% completed\n",
      "Response for Row 1504:\n",
      "Answer to Question 2: (over-expression of glutathione synthases, increased glutathione production), (deletion of glutathione reductase, increased GSSG content), (over-expression of ERV1, increased GSSG content by 2.9-fold), (over-expression of ERV1, decreased GSH content), (over-expression of ERO1, increased GSSG content by 2.0-fold), (over-expression of ERO1, did not decrease GSH content), (over-expression of Pdi1, enhanced increase in GSSG content due to ERO1 over-expression)\n",
      "Progress: 75.20% completed\n",
      "Response for Row 1505:\n",
      "Answer to Question 2: (deletion of gene ROX1, increase in recombinant fungal α-amylase yield), (deletion of gene ROX1, increase in productivity of recombinant proteins), (deletion of gene ROX1, improvement in human insulin precursor production), (deletion of gene ROX1, improvement in yeast endogenous enzyme invertase production), (upregulation of gene UPC2, effect on protein production)\n",
      "Progress: 75.25% completed\n",
      "Response for Row 1506:\n",
      "Answer to Question 2: (deletion of RMT genes, decrease in ηMMA), (deletion of RMT genes, decrease in δMMA), (deletion of RMT genes, increase in sDMA), (deletion of RMT genes, decrease in aDMA)\n",
      "Progress: 75.30% completed\n",
      "Response for Row 1507:\n",
      "Answer to Question 2: (deletion of gene HTM1, increase in antibody production), (deletion of gene HRD1, decrease in antibody production), (deletion of gene HRD3, decrease in antibody production), (deletion of gene UBC7, decrease in antibody production), (deletion of gene YOS9, stronger decrease in antibody production), (deletion of gene IRE1, decreased production in wild-type cells), (deletion of gene IRE1, reversed effects in combination with Δhtm1), (deletion of gene IRE1, reversed effects in combination with Δyos9), (deletion of genes YOS9 and IRE1, delayed IgG clearance from cells), (deletion of genes HTM1 and IRE1, delayed IgG clearance from cells)\n",
      "Progress: 75.35% completed\n",
      "Response for Row 1508:\n",
      "Answer to Question 2: (KlGCR1 deletion, diminishes glucose consumption), (KlGCR1 deletion, increases resistance to oxidative stress), (KlGCR1 deletion, increases glucose 6P dehydrogenase activity), (KlGCR1 deletion, increases NADPH/NADP(+) ratio), (KlGCR1 deletion, increases GSH/GSSG ratio), (KlGCR1 deletion, decreases ethanol production), (KlGCR1 deletion, transcriptional regulation of KlZWF1), (KlGCR1 deletion, increased flux of glucose through the pentose phosphate pathway), (KlGCR1 deletion, decreases mitochondrial respiration)\n",
      "Progress: 75.40% completed\n",
      "Response for Row 1509:\n",
      "Answer to Question 2: (deletion of gene SUB1, increased sporulation efficiency), (deletion of gene SUB1, increased middle sporulation gene transcript levels), (SUB1 downregulation, induction of middle sporulation gene expression), (SUB1 deletion, genetic interaction with HOS2)\n",
      "Progress: 75.45% completed\n",
      "Response for Row 1510:\n",
      "Answer to Question 2: (deletion of gene erg5, redox imbalance), (deletion of gene erg5, higher ratio of cytosolic free NADH/NAD+), (deletion of gene erg5, more glycerol accumulation), (deletion of gene erg5, more ethanol accumulation), (expression of NOX, alleviated redox imbalance), (expression of AOX1, alleviated redox imbalance), (alleviation of redox imbalance, increased production of 7-dehydrocholesterol by 74.4%), (alleviation of redox imbalance, reduced ratio of free NADH/NAD+ by 78.0%), (alleviation of redox imbalance, reduced glycerol concentration by 50.7%), (alleviation of redox imbalance, reduced ethanol concentration by 7.9%)\n",
      "Progress: 75.50% completed\n",
      "Response for Row 1511:\n",
      "Answer to Question 2: (deletion of gene xxx, delay in vacuolar rupture), (deletion of gene xxx, alternative form of PCD), (deletion of gene xxx, catastrophic consequences for underdeveloped spores)\n",
      "Progress: 75.55% completed\n",
      "Response for Row 1512:\n",
      "Answer to Question 2: (deletion of gene FAS2, consequence Native FAS activity eliminated), (expression of gene acpS + acpP, consequence Enzyme activity confirmed), (expression of gene fabB, consequence Enzyme activity confirmed), (expression of gene fabD, consequence Enzyme activity confirmed), (expression of gene fabG, consequence Enzyme activity confirmed), (expression of gene fabH, consequence Enzyme activity confirmed), (expression of gene fabI, consequence Enzyme activity confirmed), (expression of gene fabZ, consequence Enzyme activity confirmed), (expression of gene tesA, consequence Increased total fatty acid titer), (expression of gene fatB, consequence Increased total fatty acid titer 1.7-fold), (expression of gene fatB, consequence Shifted fatty acid profile towards C14 production), (expression of gene fatB, consequence Increased C14 production from <1% to more than 30%), (expression of gene fatB, consequence Reduced C18 production from 39% to 8%)\n",
      "Progress: 75.60% completed\n",
      "Response for Row 1513:\n",
      "Answer to Question 2: (deletion of gene dga1, improvement in lipid accumulation), (deletion of gene dga2, improvement in lipid accumulation), (deletion of gene lro1, improvement in lipid accumulation), (deletion of gene are1, improvement in lipid accumulation), (overexpression of gene dga2, improvement in lipid accumulation), (deletion of gene mfe1, further increase in lipid accumulation), (expression of gene suc2, establishment of sucrose utilization), (expression of gene suc2, increase in lipid storage), (expression of gene suc2, increase in biomass yield)\n",
      "Progress: 75.65% completed\n",
      "Response for Row 1514:\n",
      "Answer to Question 2: (deletion of gene gpd1, consequence 1), (deletion of gene gpd2, consequence 1), (deletion of gene gpd1, increased GPD2 promoter activity), (deletion of gene gpd2, increased GPD2 promoter activity), (overexpression of gene gpd2, hyperactive GPD2 promoter), (excess oxygen, inactive GPD2 promoter), (continuous cultivation at 0.3 h^-1, inactive GPD2 promoter)\n",
      "Progress: 75.70% completed\n",
      "Response for Row 1515:\n",
      "Answer to Question 2: (expression of gene FLO1, consequence strong flocculation), (expression of gene FLO1, consequence better cell growth under acetic acid stress), (expression of gene FLO1, consequence higher ethanol production rate under acetic acid stress), (expression of gene FLO1c, consequence strong flocculation), (expression of gene FLO1c, consequence better cell growth under acetic acid stress), (expression of gene FLO1c, consequence higher ethanol production rate under acetic acid stress)\n",
      "Progress: 75.75% completed\n",
      "Response for Row 1516:\n",
      "Answer to Question 2: (deletion of PTR2 gene in Saccharomyces cerevisiae, inability to uptake dipeptides), (expression of BghPTR2 gene, high during appressorial germ tube formation), (expression of BghPTR2 gene, low in haustoria), (expression of BghPTR2 gene, high during conidiation and secondary infection), (BghPTR2 gene expression, importance for initial infection establishment), (BghPTR2 gene expression, suggested role in internal transport of nutrient reserves), (BghPTR2 gene expression, suggested role in uptake of breakdown products from plant surface)\n",
      "Progress: 75.80% completed\n",
      "Response for Row 1517:\n",
      "Answer to Question 2: (deletion of gene LIG4, increase in correct gene targeting), (deletion of gene LIG4, absence of clear side effects)\n",
      "Progress: 75.85% completed\n",
      "Response for Row 1518:\n",
      "Answer to Question 2: (overexpression of gene ldhA, high lactate dehydrogenase activity), (overexpression of gene ldhA, production of lactic acid from glucose), (overexpression of gene ldhA, production of lactic acid from xylose), (substrate-dependent product formation, ethanol as major product from glucose), (substrate-dependent product formation, lactic acid as minor product from glucose), (substrate-dependent product formation, lactic acid as exclusive product from xylose under oxygen-limited conditions)\n",
      "Progress: 75.90% completed\n",
      "Response for Row 1519:\n",
      "Answer to Question 2: (deletion of gene ADH3, impact on organic acid profiles), (deletion of gene AAD6, impact on organic acid profiles), (deletion of gene SER33, impact on organic acid profiles), (deletion of gene ICL1, impact on organic acid profiles), (deletion of gene GLY1, impact on organic acid profiles), (deletion of gene SFC1, impact on organic acid profiles), (deletion of gene SER1, impact on organic acid profiles), (deletion of gene KGD1, impact on organic acid profiles), (deletion of gene AGX1, impact on organic acid profiles), (deletion of gene OSM1, impact on organic acid profiles), (deletion of gene GPD2, impact on organic acid profiles)\n",
      "Progress: 75.95% completed\n",
      "Response for Row 1520:\n",
      "Answer to Question 2: (deletion of gene LOA1, consequence 1), (deletion of gene LOA1, consequence 2), (deletion of gene TGL3, consequence 1), (deletion of gene TGL3, consequence 2)\n",
      "Progress: 76.00% completed\n",
      "Response for Row 1521:\n",
      "Answer to Question 2: (deletion of functional domain in CrtYB, lycopene cyclase function inactivated), (coexpression of CrtYB11M, increased lycopene production), (coexpression of CrtE, increased lycopene production), (coexpression of CrtI, increased lycopene production), (coexpression of tHMG1, increased lycopene production), (enhanced catalytic performance of CrtE, increased FPP competitiveness), (increased copy number of Crt genes, increased lycopene production), (fed-batch fermentation, increased lycopene accumulation), (fed-batch fermentation, reduced squalene production)\n",
      "Progress: 76.05% completed\n",
      "Response for Row 1522:\n",
      "Answer to Question 2: (deletion of gene pyruvate dehydrogenase complex, improved isobutanol production), (deletion of gene isobutyrate biosynthetic pathway, reduced carbon outflow competing with isobutanol biosynthesis), (deletion of gene pantothenate biosynthetic pathway, reduced carbon outflow competing with isobutanol biosynthesis), (deletion of gene isoleucine biosynthetic pathway, reduced carbon outflow competing with isobutanol biosynthesis), (deletion of gene ILV1, increased isobutanol productivity), (combination of ILV1 deletion and activation of two steps of the Ehrlich pathway, increased isobutanol productivity), (combination of ILV1 deletion and transhydrogenase-like shunt, increased isobutanol productivity)\n",
      "Progress: 76.10% completed\n",
      "Response for Row 1523:\n",
      "Answer to Question 2: (deletion of gene rox1, increase in carotenoid production), (deletion of gene rox1, increase in sclareol production), (deletion of gene dos2, increase in carotenoid production), (deletion of gene dos2, increase in sclareol production), (deletion of gene yer134c, increase in carotenoid production), (deletion of gene yer134c, increase in sclareol production), (deletion of gene vba5, increase in carotenoid production), (deletion of gene vba5, increase in sclareol production), (deletion of gene ynr063w, increase in carotenoid production), (deletion of gene ynr063w, increase in sclareol production), (deletion of gene ygr259c, increase in carotenoid production), (deletion of gene ygr259c, increase in sclareol production)\n",
      "Progress: 76.15% completed\n",
      "Response for Row 1524:\n",
      "Answer to Question 2: (deletion of gene ε-COP, no detectable changes in vegetative growth), (deletion of gene ε-COP, no detectable changes in asexual development), (deletion of gene ε-COP, decrease in production of cleistothecium), (over-expression of gene ε-COP, no rescue of thermo-sensitive growth defect in α-COP mutant)\n",
      "Progress: 76.20% completed\n",
      "Response for Row 1525:\n",
      "Answer to Question 2: (deletion of gene FaTBCA, reduced mycelial growth), (deletion of gene FaTBCA, abnormal conidiation), (deletion of gene FaTBCA, increased sensitivity to low temperatures), (deletion of gene FaTBCA, loss of growth ability at 4°C), (deletion of gene FaTBCA, blebbing phenotypes in hyphae at 4°C), (deletion of gene FaTBCA, enlarged nodes containing several nuclei at 4°C), (deletion of gene FaTBCA, decreased virulence on wheat head), (deletion of gene FaTBCA, decreased virulence on non-host tomato)\n",
      "Progress: 76.25% completed\n",
      "Response for Row 1526:\n",
      "Answer to Question 2: (deletion of gene FgVAM7, reduces vegetative growth), (deletion of gene FgVAM7, reduces conidiation), (deletion of gene FgVAM7, reduces conidial germination), (deletion of gene FgVAM7, reduces sexual reproduction), (deletion of gene FgVAM7, reduces virulence), (deletion of gene FgVAM7, defect in vacuolar maintenance), (deletion of gene FgVAM7, delayed endocytosis), (deletion of gene FgVAM7, insensitive to salt stress), (deletion of gene FgVAM7, insensitive to osmotic stress), (deletion of gene FgVAM7, hypersensitive to cell wall stressors), (deletion of gene FgVAM7, reduces expression of gene TRI5), (deletion of gene FgVAM7, reduces expression of gene TRI6), (deletion of gene FgVAM7, reduces expression of gene TRI101), (deletion of gene FgVAM7, reduces DON production)\n",
      "Progress: 76.30% completed\n",
      "Response for Row 1527:\n",
      "Answer to Question 2: (deletion of target mRNA sequences, impact on detection and interpretation of biological functions), (modification of target mRNA sequences, improvement in signal and reproducibility in single-RNA detection), (synonymous modification to FRET biosensors, elimination of aberrant truncation products), (correct expression of full-length biosensors, significant difference in biological interpretations)\n",
      "Progress: 76.35% completed\n",
      "Response for Row 1528:\n",
      "Answer to Question 2: (deletion of gene ACO1, decrease in succinic acid levels), (deletion of gene ACO1, deletion of gene ICL1, decrease in succinic acid levels), (deletion of gene IDH1, deletion of gene IDP1, decrease in succinic acid production), (deletion of gene IDH1, deletion of gene IDP1, affect fermentation rate), (deletion of gene SDH1, deletion of gene SDH2, increase in succinic acid accumulation), (deletion of gene FRD1, deletion of gene OSM1, no effect on fermentation rate), (deletion of gene FRD1, deletion of gene OSM1, no effect on succinic acid production)\n",
      "Progress: 76.40% completed\n",
      "Response for Row 1529:\n",
      "Answer to Question 2: (deletion of gene GAL3, consequence PARP-1 protein level unchanged), (deletion of gene HHO1, consequence increased nuclear localization of PARP-1 after UV radiation), (deletion of gene HUL4, consequence PARP-1 protein level unchanged), (deletion of gene OTU1, consequence PARP-1 protein level unchanged), (deletion of gene POM152, consequence increased nuclear localization of PARP-1 after UV radiation), (deletion of gene SNT1, consequence increased nuclear localization of PARP-1), (expression of gene PARP-1, consequence inhibited yeast growth), (expression of gene PARP-1, consequence reduced UV-induced homologous recombination), (expression of gene PARP-1, consequence produced PAR products)\n",
      "Progress: 76.45% completed\n",
      "Response for Row 1530:\n",
      "Answer to Question 2: (deletion of gene Fg06297, reduced virulence), (deletion of gene Fg03333, reduced virulence), (deletion of gene Fg03826, reduced virulence), (deletion of gene Fg07932, reduced virulence), (deletion of gene Fg10516, negative regulation of MAPK signaling), (deletion of gene Fg03333, negative regulation of MAPK signaling), (deletion of gene Fg12867, negative regulation of MAPK signaling), (deletion of gene FgCdc14, involved in ribosome biogenesis)\n",
      "Progress: 76.50% completed\n",
      "Response for Row 1531:\n",
      "Answer to Question 2: (deletion of gene PdMit1, slow radial growth), (deletion of gene PdMit1, decreased conidia production), (deletion of gene PdMit1, delayed conidial germination), (deletion of gene PdMit1, hypersensitivity to Ca(2+)), (deletion of gene PdMit1, inhibition of vacuole fusion upon Ca(2+) treatment), (deletion of gene PdMit1, significant inhibition of Ca(2+)-ATPase genes upregulation upon CaCl2 treatment)\n",
      "Progress: 76.55% completed\n",
      "Response for Row 1532:\n",
      "Answer to Question 2: (deletion of gene SIN3, weak UV-sensitivity), (deletion of gene SIN3, decrease in spontaneous levels), (deletion of gene SIN3, decrease in UV-induced levels), (deletion of gene SIN3, malfunction of ribonucleotide reductase activity regulation), (malfunction of ribonucleotide reductase activity regulation, decrease in dNTP pool), (deletion of gene SIN3, inaccurate error-prone damage bypass postreplication repair pathway), (inaccurate error-prone damage bypass postreplication repair pathway, reduction in the incidence of mutations)\n",
      "Progress: 76.60% completed\n",
      "Response for Row 1533:\n",
      "Answer to Question 2: (deletion of gene QDR3, improved acetic acid tolerance), (deletion of gene IKS1, improved acetic acid tolerance), (deletion of gene QDR3, increased glucose consumption rate), (deletion of gene QDR3, increased ethanol productivity)\n",
      "Progress: 76.65% completed\n",
      "Response for Row 1534:\n",
      "Answer to Question 2: (no expression/deletion of gene, protein synthesis impacted by iterated AAA codons), (no expression/deletion of gene, ribosome pausing on consecutive AAA-lysines), (no expression/deletion of gene, ribosome sliding on homopolymeric A sequence), (no expression/deletion of gene, premature translation termination on out of frame stop codons), (no expression/deletion of gene, mRNAs targeted for Nonsense-Mediated-Decay (NMD))\n",
      "Progress: 76.70% completed\n",
      "Response for Row 1535:\n",
      "Answer to Question 2: (deletion of gene SSA1, increase in intracellular protein content), (deletion of gene SSA1, increase in cell size), (deletion of gene SSB1, increase in intracellular protein content), (deletion of gene SSB1, increase in cell size)\n",
      "Progress: 76.75% completed\n",
      "Response for Row 1536:\n",
      "Answer to Question 2: (deletion of gene RPO41, increase in synthesis of abortive transcripts), (deletion of gene RPO41, increase in probability of polymerase disengagement from template), (deletion of gene RPO41, increase in amount of slippage products from homopolymeric template), (deletion of gene RPO41, decrease in binding affinity for mitochondrial transcription factor Mtf1)\n",
      "Progress: 76.80% completed\n",
      "Response for Row 1537:\n",
      "Answer to Question 2: (deletion of gene BAT2, decrease in higher alcohols production), (deletion of gene BAT2, decrease in isobutanol production), (deletion of gene BAT2, decrease in isoamyl alcohol production)\n",
      "Progress: 76.85% completed\n",
      "Response for Row 1538:\n",
      "Answer to Question 2: (deletion of gene Os3378, reduced transposition activity), (fusion of fluorescent protein to gene Os3378, enhanced transposition activity), (substitution in gene Os3378, altered transposition activity)\n",
      "Progress: 76.90% completed\n",
      "Response for Row 1539:\n",
      "Answer to Question 2: (induction of Aro80, activation of ARO9), (induction of Aro80, activation of ARO10), (expression of alsS, increase in acetoin production), (expression of alsD, increase in acetoin production)\n",
      "Progress: 76.95% completed\n",
      "Response for Row 1540:\n",
      "Answer to Question 2: (deletion of gene RPE1, simultaneous utilization of xylose and glucose), (deletion of gene RPE1, xylose consumption at the beginning of glucose consumption), (deletion of gene RPE1, xylose consumption ratio of about 1:10 with glucose)\n",
      "Progress: 77.00% completed\n",
      "Response for Row 1541:\n",
      "Answer to Question 2: (deletion of gene HFD1, consequence 1), (expression of gene redox system, consequence 1), (deletion of gene HFD1, production of tridecane), (deletion of gene HFD1, production of pentadecane), (deletion of gene HFD1, production of heptadecane)\n",
      "Progress: 77.05% completed\n",
      "Response for Row 1542:\n",
      "Answer to Question 2: (deletion of ACA tRNA, modest efficiency decrease)\n",
      "Progress: 77.10% completed\n",
      "Response for Row 1543:\n",
      "Answer to Question 2: (deletion of gene V1 sub-complex of yeast V-ATPase, consequence cysteine degradation impairment), (deletion of gene associated with vacuole acidification, consequence cysteine degradation impairment), (deletion of gene involved in translation, consequence cysteine catabolism impairment), (deletion of gene involved in RNA processing, consequence cysteine catabolism impairment), (deletion of gene involved in folate-derived one-carbon metabolism, consequence cysteine catabolism impairment), (deletion of gene involved in mitochondrial iron-sulfur homeostasis, consequence cysteine catabolism impairment)\n",
      "Progress: 77.15% completed\n",
      "Response for Row 1544:\n",
      "Answer to Question 2: (deletion of gene xxx, improved vanillin production), (deletion of gene xxx, increased growth rate)\n",
      "Progress: 77.20% completed\n",
      "Response for Row 1545:\n",
      "Answer to Question 2: (expression of GhPAS2, enhanced VLCFA biosynthesis), (expression of GhPAS2, restored viability to yeast psh1-deletion strain), (application of acetochlor, suppressed fiber cell protrusion), (overexpression of proteins involved in carboxylic acid metabolism, enhanced in wild-type ovules), (overexpression of proteins involved in small-molecule metabolic processes, enhanced in wild-type ovules), (overexpression of proteins involved in hormone regulation, enhanced in wild-type ovules), (overexpression of proteins involved in lipid metabolism, enhanced in wild-type ovules)\n",
      "Progress: 77.25% completed\n",
      "Response for Row 1546:\n",
      "Answer to Question 2: (elevation of Hsp104, increased under heat stress in stationary growth phase), (loss of mitochondrial DNA, inhibited induction of Hsp104 synthesis under heat stress), (petite mutation, suppressed activity of antioxidant enzymes), (petite mutation, lower thermotolerance), (petite mutation, inhibited production of reactive oxygen species), (disruption of mitochondrial functional state, suppressed expression level of yeast nuclear genes upon transitioning to stationary growth phase)\n",
      "Progress: 77.30% completed\n",
      "Response for Row 1547:\n",
      "Answer to Question 2: (deletion of gene COX9, higher ethanol production rates by 37%), (deletion of gene COX9, slight growth disadvantages), (deletion of gene QCR9, higher ethanol production rates by 27%), (deletion of gene QCR9, slight growth disadvantages), (deletion of gene QCR6, improved ethanol production rate by 24%), (deletion of gene QCR6, no growth disadvantage)\n",
      "Progress: 77.35% completed\n",
      "Response for Row 1548:\n",
      "Answer to Question 2: (deletion of gene cluster for red-pigmented undecylprodiginine complex, demonstration of efficacy and efficiency of system), (expression of I-SceI gene, production of yeast I-SceI homing endonuclease), (production of yeast I-SceI homing endonuclease, creation of double strand breaks at unique 18 base pair recognition sequence), (homologous recombination, survival of genomes)\n",
      "Progress: 77.40% completed\n",
      "Response for Row 1549:\n",
      "Answer to Question 2: (deletion of gene, increased production of amorphadiene), (deletion of gene, yield of amorphadiene produced 8-10-fold greater compared to the wild type)\n",
      "Progress: 77.45% completed\n",
      "Response for Row 1550:\n",
      "Answer to Question 2: (deletion of gene Hsp82, reduction in cell viability upon MMS treatment), (deletion of gene Hsp82, reduction in cell viability upon UV treatment), (deletion of gene Hsp82, decrease in gene-targeting efficiency), (deletion of gene Hsp82, decrease in endogenous levels of Rad51), (deletion of gene Hsp82, decrease in endogenous levels of Rad52), (inhibition of gene Hsp82 by 17-AAG, proteasomal degradation of Rad51), (mutation in ATPase domain of Hsp82, decrease in Rad51 protein levels), (mutation in ATPase domain of Hsp82, decrease in Rad52 protein levels), (charged linker deletion in Hsp82, abrogation of HR activity), (charged linker deletion in Hsp82, decrease in Rad51 focus formation)\n",
      "Progress: 77.50% completed\n",
      "Response for Row 1551:\n",
      "Answer to Question 2:  (deletion of gene vsl1, no obvious defect in vacuolar protein transport), (deletion of gene vsl1, extreme defects in vacuolar protein transport when combined with fsv1 deletion), (deletion of gene vsl1, vacuolar morphology defect when combined with fsv1 deletion), (overexpression of gene fsv1, mislocalization of GFP-Vsl1p), (overexpression of gene VAM7, suppression of ZnCl2 sensitivity in vsl1Δ cells), (overexpression of gene VAM7, suppression of vacuolar morphology defect in vsl1Δfsv1Δ cells)\n",
      "Progress: 77.55% completed\n",
      "Response for Row 1552:\n",
      "Answer to Question 2: (deletion of gene Ran GTPase, SG defects), (deletion of gene Ran GTPase, enhanced heat-induced mutation rate), (deletion of gene Ran GTPase, increased stress-induced mRNA accumulation), (deletion of gene Ran GTPase, decreased stress-repressed mRNA levels)\n",
      "Progress: 77.60% completed\n",
      "Response for Row 1553:\n",
      "Answer to Question 2: (deletion of gene PDC1, consequence increased L-lactic acid production), (deletion of gene CYB2, consequence increased L-lactic acid production), (deletion of gene GPD1, consequence increased L-lactic acid production), (expression of gene LDH, consequence increased L-lactic acid production), (deletion of gene NDE1, consequence improved intracellular redox balance), (deletion of gene NDE2, consequence improved intracellular redox balance)\n",
      "Progress: 77.65% completed\n",
      "Response for Row 1554:\n",
      "Answer to Question 2: (deletion of gene ADH1, acetaldehyde accumulation), (deletion of gene ADH1, increased HEMF production)\n",
      "Progress: 77.70% completed\n",
      "Response for Row 1555:\n",
      "Answer to Question 2: (deletion of gene SAM2, higher resistance to lactic acid), (deletion of gene SAM2, higher lactic acid production), (overexpression of gene SAM2, detrimental effect)\n",
      "Progress: 77.75% completed\n",
      "Response for Row 1556:\n",
      "Answer to Question 2: (deletion of gene PpATT1, improved cellular fitness levels), (deletion of gene PpATT1, improved thermal tolerance levels), (deletion of gene PpATT1, reduced cell lysis defects), (deletion of gene PpATT1, improved fermentation robustness), (deletion of gene PpATT1, increased product yields)\n",
      "Progress: 77.80% completed\n",
      "Response for Row 1557:\n",
      "Answer to Question 2: (deletion of gene BAR1, higher degree of synchronization with α-factor), (deletion of gene BAR1, ability to synchronize at high cell density), (presence of gene BAR1, need for higher concentration of α-factor), (presence of gene BAR1, transient synchrony)\n",
      "Progress: 77.85% completed\n",
      "Response for Row 1558:\n",
      "Answer to Question 2: (deletion of gene DFG5, enhanced heat tolerance), (deletion of gene DFG5, lowered level of reactive oxygen species), (deletion of gene DFG5, decreased membrane permeability), (deletion of gene DFG5, activation of mitogen-activated protein kinases Hog1), (deletion of gene DFG5, activation of mitogen-activated protein kinases Slt2)\n",
      "Progress: 77.90% completed\n",
      "Response for Row 1559:\n",
      "Answer to Question 2: (deletion of gene involved in MAPK-signaling pathways, sensitive growth response to HMF), (deletion of gene involved in PI-signaling pathways, sensitive growth response to HMF), (overexpression of genes involved in MAPK-signaling pathways, increased tolerance to HMF), (overexpression of genes involved in PI-signaling pathways, increased tolerance to HMF), (mutation in genes involved in purine metabolism, unknown consequence), (mutation in genes involved in pyrimidine metabolism, unknown consequence)\n",
      "Progress: 77.95% completed\n",
      "Response for Row 1560:\n",
      "Answer to Question 2: (overexpression of gene HAA1, improved cell growth), (overexpression of gene HAA1, improved ethanol production from xylose), (deletion of gene PHO13, further improvement in yeast growth), (deletion of gene PHO13, improved ability to ferment xylose)\n",
      "Progress: 78.00% completed\n",
      "Response for Row 1561:\n",
      "Answer to Question 2: (deletion of gene ALD6, elimination of competing pathways), (deletion of gene BAT1, elimination of competing pathways), (overexpression of gene Leu3Δ601, increased transcription of endogenous genes in valine and leucine biosynthetic pathways), (overexpression of genes ILV2, ILV3, ILV5, ARO10, and ADH2, 376.9 mg/L isobutanol production from 100 g/L glucose), (overexpression of gene LEU2, increased 3-methyl-1-butanol synthesis), (overexpression of gene LEU4 (D578Y), increased 3-methyl-1-butanol synthesis), (overexpression of gene LEU4 (D578Y), 765.7 mg/L 3-methyl-1-butanol production)\n",
      "Progress: 78.05% completed\n",
      "Response for Row 1562:\n",
      "Answer to Question 2: (overexpression of gene Dga1∆Np, increased POA content), (overexpression of gene Dga1∆Np, increased dry cell weight), (high concentration of methionine, increased POA production), (low temperatures (20-25 °C), increased POA content), (low temperatures (20-25 °C), increased dry cell weight), (overexpression of gene Ole1p, increased POA content in wild-type strain), (genomic Ole1p in dga1 mutant, sufficient for optimal POA production)\n",
      "Progress: 78.10% completed\n",
      "Response for Row 1563:\n",
      "Answer to Question 2: (deletion of gene ypk1, increased PC deacylation), (deletion of gene ypk1, increased glycerophosphocholine production), (deletion of gene plb1, curtails increased PC deacylation in ypk1Δ mutant), (deletion of gene ypk1, elevated Plb1 protein levels), (deletion of gene ypk1, elevated PLB1 mRNA levels), (deletion of gene plb1 in ypk1Δ mutant, exacerbated slowed growth), (deletion of gene plb1 in ypk1Δ mutant, exacerbated sensitivity to cell wall perturbation), (inhibition of sphingolipid biosynthesis, increased PLB1 expression), (deletion of gene ypk1, increased PLB1 expression), (control of Crz1, regulation of PLB1 expression upon inhibition of sphingolipid synthesis), (control of Crz1, regulation of PLB1 expression upon deletion of gene ypk1)\n",
      "Progress: 78.15% completed\n",
      "Response for Row 1564:\n",
      "Answer to Question 2: (deletion of gene EFM3, increases sensitivity to antibiotics that target translation), (deletion of gene EFM3, decreases translational fidelity), (deletion of gene EFM2, increases sensitivity to antibiotics that target translation), (deletion of gene EFM2, decreases translational fidelity)\n",
      "Progress: 78.20% completed\n",
      "Response for Row 1565:\n",
      "Answer to Question 2: (deletion of gene CgVPS15, impairment of phagosomal maturation), (deletion of gene CgVPS15, hyperadherence to epithelial cells), (deletion of gene CgVPS15, susceptibility to antimicrobial arsenal), (deletion of gene CgVPS15, susceptibility to diverse stresses), (deletion of gene CgVPS15, virulence attenuation in mice), (deletion of gene CgVPS34, impairment of phagosomal maturation), (deletion of gene CgVPS34, hyperadherence to epithelial cells), (deletion of gene CgVPS34, susceptibility to antimicrobial arsenal), (deletion of gene CgVPS34, susceptibility to diverse stresses), (deletion of gene CgVPS34, virulence attenuation in mice)\n",
      "Progress: 78.25% completed\n",
      "Response for Row 1566:\n",
      "Answer to Question 2: (deletion of gene MrKu70, increased gene disruption efficiency), (deletion of gene MrKu70, 93% frequency of disrupting gene Cag8), (deletion of gene MrKu70, no difference in development compared to wild-type), (deletion of gene MrKu70, no difference in pathogenicity compared to wild-type), (deletion of gene MrKu70, no difference in tolerance to various abiotic stresses compared to wild-type)\n",
      "Progress: 78.30% completed\n",
      "Response for Row 1567:\n",
      "Answer to Question 2: (deletion of gene xxx, accurate targeting into yeast genome), (mutation of gene xxx, integration into yeast genome), (fusion of gene xxx, construction at N-terminus), (fusion of gene xxx, construction at C-terminus)\n",
      "Progress: 78.35% completed\n",
      "Response for Row 1568:\n",
      "Answer to Question 2: (deletion of gene Mrlig4, no apparent defects in vegetative growth), (deletion of gene Mrlig4, no apparent defects in colony phenotype), (deletion of gene Mrlig4, no apparent defects in microscopic morphology), (deletion of gene Mrlig4, no apparent defects in spore yield), (deletion of gene Mrlig4, no apparent defects in production of Monascus pigments), (deletion of gene Mrlig4, no apparent defects in production of citrinin), (deletion of gene Mrlig4, increased gene targeting of ku70), (deletion of gene Mrlig4, increased gene targeting of triA), (deletion of gene Mrlig4, increased sensitivity to methyl methanesulfonate)\n",
      "Progress: 78.40% completed\n",
      "Response for Row 1569:\n",
      "Answer to Question 2: (deletion of histone deacetylase, higher RIND-EDSB levels), (deletion of endonucleases, higher RIND-EDSB levels), (deletion of topoisomerase, higher RIND-EDSB levels), (deletion of DNA repair regulators, higher RIND-EDSB levels), (deletion of high-mobility group box proteins, lower RIND-EDSB levels), (deletion of Sir2, lower RIND-EDSB levels)\n",
      "Progress: 78.45% completed\n",
      "Response for Row 1570:\n",
      "Answer to Question 2: (deletion of gene FMP21, decreased respiratory growth rate), (deletion of gene FMP21, increased thermosensitivity), (overexpression of gene FMP21, improved thermotolerance)\n",
      "Progress: 78.50% completed\n",
      "Response for Row 1571:\n",
      "Answer to Question 2: (deletion of gene involved in mitochondrial protein biosynthesis process, complete mtDNA loss), (deletion of gene involved in nucleic acid metabolism, complete mtDNA loss), (deletion of gene involved in oxidative phosphorylation, complete mtDNA loss), (deletion of gene involved in bud-site selection, complete mtDNA loss), (deletion of gene involved in mitochondrial intermembrane space protein import, complete mtDNA loss), (deletion of gene involved in assembly of cytochrome-c oxidase, complete mtDNA loss), (deletion of gene involved in vacuolar protein sorting, complete mtDNA loss), (deletion of gene involved in protein-nucleus import, complete mtDNA loss), (deletion of gene involved in calcium-mediated signaling, complete mtDNA loss), (deletion of gene involved in heme biosynthesis, complete mtDNA loss), (deletion of gene involved in iron homeostasis, complete mtDNA loss)\n",
      "Progress: 78.55% completed\n",
      "Response for Row 1572:\n",
      "Answer to Question 2: (deletion of gene FgILV1, consequence inability to grow on minimal medium), (deletion of gene FgILV1, consequence inability to grow on fructose gelatin agar lacking Ile), (deletion of gene FgILV1, consequence exogenous supplementation of Ile rescues growth defect), (deletion of gene FgILV1, consequence exogenous supplementation of 2-KB rescues growth defect), (deletion of gene FgILV1, consequence exogenous supplementation of Met rescues growth defect), (deletion of gene FgILV1, consequence defects in conidial formation), (deletion of gene FgILV1, consequence defects in conidial germination), (deletion of gene FgILV1, consequence decreased virulence on wheat heads), (deletion of gene FgILV1, consequence low level of deoxynivalenol (DON) production in wheat kernels)\n",
      "Progress: 78.60% completed\n",
      "Response for Row 1573:\n",
      "Answer to Question 2: (introduction of ldhL gene, increased lactic acid production), (deletion of xylose reductase gene, increased lactic acid production), (deletion of xylitol dehydrogenase gene, increased lactic acid production), (overexpression of xylulokinase gene, increased lactic acid production), (introduction of xylose isomerase gene, increased lactic acid production)\n",
      "Progress: 78.65% completed\n",
      "Response for Row 1574:\n",
      "Answer to Question 2: (deletion of gene YDR307W, improved tolerance to ethanol), (deletion of gene YHL042W, improved tolerance to ethanol), (deletion of gene YDR307W, transcriptional re-programming of metabolism), (deletion of gene YHL042W, transcriptional re-programming of metabolism), (deletion of gene YDR307W, mis-perception of nutritional environment), (deletion of gene YHL042W, mis-perception of nutritional environment), (deletion of gene YDR307W, excess amount of glucose perception), (deletion of gene YHL042W, deficiency of methionine or sulfur perception), (mutation leading to increase in ribosome biogenesis, improvement of ethanol tolerance), (modulation of chronological life span, contribution to ethanol tolerance)\n",
      "Progress: 78.70% completed\n",
      "Response for Row 1575:\n",
      "Answer to Question 2: (deletion of hexose transporter in Saccharomyces cerevisiae, rescued by ApST3), (deletion of hexose transporter in Saccharomyces cerevisiae, rescued by ApST4), (overexpression of ApST4 in yeast, transports glucose), (overexpression of ApST4 in yeast, transports fructose)\n",
      "Progress: 78.75% completed\n",
      "Response for Row 1576:\n",
      "Answer to Question 2: (deletion of gene xxx, increased sensitivity to trichothecin), (deletion of gene xxx, higher reactive oxygen species levels), (deletion of gene xxx, reduced cytotoxicity with antioxidant treatment), (deletion of gene xxx, reduced cytotoxicity with rapamycin treatment), (deletion of gene xxx, impaired mitophagy leading to increased sensitivity)\n",
      "Progress: 78.80% completed\n",
      "Response for Row 1577:\n",
      "Answer to Question 2: (deletion of trehalase gene, accumulation of trehalose), (deletion of trehalase gene, stress tolerance to acetic acid)\n",
      "Progress: 78.85% completed\n",
      "Response for Row 1578:\n",
      "Answer to Question 2: (deletion of gene involved in cell wall remodeling, improved acetic acid tolerance), (deletion of gene involved in membrane transport, improved acetic acid tolerance), (deletion of gene involved in energy metabolism, improved acetic acid tolerance), (deletion of gene involved in amino acid biosynthesis and transport, improved acetic acid tolerance), (deletion of gene involved in global transcription regulation, improved acetic acid tolerance)\n",
      "Progress: 78.90% completed\n",
      "Response for Row 1579:\n",
      "Answer to Question 2: (deletion of gene MIG1, consequence 1), (deletion of gene MIG1, promotes maltose metabolism), (deletion of gene MIG1, enhances leavening ability), (deletion of gene SSN6, consequence 1), (deletion of gene SSN6, promotes maltose metabolism), (deletion of gene SSN6, enhances leavening ability), (deletion of gene TUP1, consequence 1), (deletion of gene TUP1, negative for glucose derepression), (deletion of gene TUP1, hinders maltose metabolism)\n",
      "Progress: 78.95% completed\n",
      "Response for Row 1580:\n",
      "Answer to Question 2: (deletion of gene in phenylalanine biosynthesis pathway, synthesis of phenylalanine and phenylpyruvate as by-products), (deletion of gene in phenylalanine biosynthesis pathway, non-enzymatic rearrangement of chorismate to phenylpyruvate), (deletion of gene in phenylalanine biosynthesis pathway, enzymatic transamination of phenylpyruvate to form phenylalanine)\n",
      "Progress: 79.00% completed\n",
      "Response for Row 1581:\n",
      "Answer to Question 2: (SOD2 deletion, increased lifespan), (cryptotanshinone treatment, lifespan extension), (cryptotanshinone treatment, activation of Tor1), (cryptotanshinone treatment, activation of Sch9), (cryptotranshinone treatment, activation of Gcn2)\n",
      "Progress: 79.05% completed\n",
      "Response for Row 1582:\n",
      "Answer to Question 2: (deletion of gene PHO13, improved xylose fermentation), (deletion of gene PHO13, improved tolerance to formic acid), (deletion of gene PHO13, improved tolerance to acetic acid), (deletion of gene PHO13, improved tolerance to levulinic acid), (overexpression of gene TAL1, improved xylose fermentation), (overexpression of gene TAL1, improved tolerance to formic acid), (overexpression of gene TAL1, improved tolerance to acetic acid), (overexpression of gene TAL1, improved tolerance to levulinic acid), (deletion of gene PHO13 and overexpression of gene TAL1, improved ethanol production), (deletion of gene PHO13 and overexpression of gene TAL1, reduced xylitol accumulation)\n",
      "Progress: 79.10% completed\n",
      "Response for Row 1583:\n",
      "Answer to Question 2: (None, None)\n",
      "Progress: 79.15% completed\n",
      "Response for Row 1584:\n",
      "Answer to Question 2: (deletion of gene SIR2, reduces FLO11 expression), (deletion of gene SIR2, reduces adhesion of cells to themselves), (deletion of gene SIR2, reduces adhesion of cells to agar), (deletion of gene GCN5, reduces FLO11 expression), (deletion of gene GCN5, reduces adhesion of cells to themselves), (deletion of gene GCN5, reduces adhesion of cells to agar), (overexpression of gene FLO5, increases flocculation), (deletion of gene SAS2, enhances flocculation), (deletion of gene SAS2, enhances maltose fermentation)\n",
      "Progress: 79.20% completed\n",
      "Response for Row 1585:\n",
      "Answer to Question 2: (deletion of gene ARO8a, no change in aroma production), (deletion of gene ARO8b, impaired aroma production), (deletion of gene ARO10, impaired aroma production), (deletion of gene ARO80, impaired aroma production), (overexpression of gene ARO80, increased VOC production)\n",
      "Progress: 79.25% completed\n",
      "Response for Row 1586:\n",
      "Answer to Question 2: (deletion of gene dps1, sensitivity to oxidative stress), (deletion of gene dps1, production of sulfide), (deletion of gene dps1, requirement of antioxidant for growth on minimal medium), (deletion of gene dps1, failure to survive at stationary phase), (deletion of gene dlp1, sensitivity to oxidative stress), (deletion of gene dlp1, production of sulfide), (deletion of gene dlp1, requirement of antioxidant for growth on minimal medium), (deletion of gene dlp1, failure to survive at stationary phase), (deletion of gene ppt1, sensitivity to oxidative stress), (deletion of gene ppt1, production of sulfide), (deletion of gene ppt1, requirement of antioxidant for growth on minimal medium), (deletion of gene ppt1, failure to survive at stationary phase), (deletion of gene coq3, sensitivity to oxidative stress), (deletion of gene coq3, production of sulfide), (deletion of gene coq3, requirement of antioxidant for growth on minimal medium), (deletion of gene coq3, failure to survive at stationary phase), (deletion of gene coq4, sensitivity to oxidative stress), (deletion of gene coq4, production of sulfide), (deletion of gene coq4, requirement of antioxidant for growth on minimal medium), (deletion of gene coq4, failure to survive at stationary phase), (deletion of gene coq5, sensitivity to oxidative stress), (deletion of gene coq5, production of sulfide), (deletion of gene coq5, requirement of antioxidant for growth on minimal medium), (deletion of gene coq5, failure to survive at stationary phase), (deletion of gene coq6, sensitivity to oxidative stress), (deletion of gene coq6, production of sulfide), (deletion of gene coq6, requirement of antioxidant for growth on minimal medium), (deletion of gene coq6, failure to survive at stationary phase), (deletion of gene coq7, sensitivity to oxidative stress), (deletion of gene coq7, production of sulfide), (deletion of gene coq7, requirement of antioxidant for growth on minimal medium), (deletion of gene coq7, failure to survive at stationary phase), (deletion of gene coq8, sensitivity to oxidative stress), (deletion of gene coq8, production of sulfide), (deletion of gene coq8, requirement of antioxidant for growth on minimal medium), (deletion of gene coq8, failure to survive at stationary phase), (deletion of gene coq9, sensitivity to oxidative stress), (deletion of gene coq9, production of sulfide), (deletion of gene coq9, requirement of antioxidant for growth on minimal medium), (deletion of gene coq9, failure to survive at stationary phase), (expression of human COQ3, recovery of CoQ10 production), (expression of human COQ7, recovery of CoQ10 production), (expression of Arabidopsis thaliana COQ6, recovery of CoQ10 production)\n",
      "Progress: 79.30% completed\n",
      "Response for Row 1587:\n",
      "Answer to Question 2: (deletion of endogenous prenyltransferase genes, insufficient (+)-valencene production), (heterologous expression of FPP synthase gene ispA, (+)-valencene production), (heterologous expression of ERG20, (+)-valencene production), (heterologous expression of valencene synthase from Nootka cypress, improved (+)-valencene titers)\n",
      "Progress: 79.35% completed\n",
      "Response for Row 1588:\n",
      "Answer to Question 2: (deletion of gene SIZ1, higher furfural tolerance), (deletion of gene SIZ1, rapid furfural reduction to furfuryl alcohol), (deletion of gene SIZ1, higher ethanol productivity in presence of furfural), (deletion of gene SIZ1, tolerance towards oxidative stress)\n",
      "Progress: 79.40% completed\n",
      "Response for Row 1589:\n",
      "Answer to Question 2: (deletion of gene FLX1, consequence 1), (deletion of gene FLX1, consequence 2), (deletion of gene FLX1, consequence 3), (deletion of gene FLX1, consequence 4)\n",
      "Progress: 79.45% completed\n",
      "Response for Row 1590:\n",
      "Answer to Question 2: (deletion of insert in Δ12 desaturase, abolished activity), (reintroduction of two histidine residues in Δ12 desaturase, partially restored activity), (deletion of gene not specified, ruled out role in sex pheromone synthesis for Δ5 and Δ9/C16:0 desaturases)\n",
      "Progress: 79.50% completed\n",
      "Response for Row 1591:\n",
      "Answer to Question 2: (deletion of gene MoAND1, impaired nuclear distribution along the hyphae), (deletion of gene MoAND1, ∼65% decrease in conidia production), (deletion of gene MoAND1, abnormalities in septation pattern), (deletion of gene MoAND1, abnormalities in nuclear distribution), (deletion of gene MoAND1, inability to germinate), (deletion of gene MoAND1, defective programmed nuclear migration), (deletion of gene MoAND1, defective programmed nuclear positioning), (deletion of gene MoAND1, delay in penetration of host plants), (deletion of gene MoAND1, decreased virulence)\n",
      "Progress: 79.55% completed\n",
      "Response for Row 1592:\n",
      "Answer to Question 2: (deletion of gene xxx, indicates essentiality under experimental conditions), (mutation of gene xxx, reveals function not observable in wild-type gene in vivo), (mutation of gene xxx, reveals function not observable in purified gene product in vitro)\n",
      "Progress: 79.60% completed\n",
      "Response for Row 1593:\n",
      "Answer to Question 2: (deletion of Snf1 regulation on Acc1, enhanced activity of Acc1), (mutation of Ser659 in Acc1, increased total fatty acid content), (mutation of Ser1157 in Acc1, increased total fatty acid content), (enhanced activity of Acc1, increased supply of malonyl-CoA), (increased supply of malonyl-CoA, increased production of fatty acid ethyl esters), (increased supply of malonyl-CoA, increased production of 3-hydroxypropionic acid)\n",
      "Progress: 79.65% completed\n",
      "Response for Row 1594:\n",
      "Answer to Question 2: (deletion of gene FAA1, accumulation of free fatty acids), (deletion of gene FAA4, accumulation of free fatty acids), (expression of gene Acot5s, accumulation of free fatty acids), (expression of gene Acot5s, higher unsaturated fatty acid ratio), (deletion of gene FAA1, growth inhibition), (deletion of gene FAA4, growth inhibition), (expression of gene Acot5s, restored growth), (deletion of gene FAA1, increased extracellular total fatty acids), (deletion of gene FAA4, increased extracellular total fatty acids), (expression of gene Acot5s, increased extracellular total fatty acids)\n",
      "Progress: 79.70% completed\n",
      "Response for Row 1595:\n",
      "Answer to Question 2: (deletion of gene Cdh1, decreased origin activity), (deletion of gene Sic1, decreased origin activity), (deletion of gene Cdh1 and Sic1, compromised origin redundancy), (CDK deregulation, decreased number of active origins), (CDK deregulation, decreased number of dormant origins), (origin shortage, increased chromosome instability)\n",
      "Progress: 79.75% completed\n",
      "Response for Row 1596:\n",
      "Answer to Question 2: (deletion of beta-oxidation pathway, increased fatty acid production), (deletion of acyl-CoA synthetases, increased fatty acid production), (overexpression of thioesterases, increased fatty acid production), (overexpression of ACC1, no significant increase in fatty acid production), (deletion of ADH1, redirected metabolic flux to fatty acid synthesis), (identification of ACC1 phosphorylation sites, potential regulation of ACC1 activity)\n",
      "Progress: 79.80% completed\n",
      "Response for Row 1597:\n",
      "Answer to Question 2: (deletion of gene RAD52, stable maintenance of integrated cassette), (overexpression of gene ACB1, 40% increase in FAEE production), (overexpression of gene gapN, 40% increase in FAEE production)\n",
      "Progress: 79.85% completed\n",
      "Response for Row 1598:\n",
      "Answer to Question 2: (deletion of gene xxx, increased vanillin production)\n",
      "Progress: 79.90% completed\n",
      "Response for Row 1599:\n",
      "Answer to Question 2: (overexpression of alsS, conversion of pyruvate into 2,3-BDO), (overexpression of alsD, conversion of pyruvate into 2,3-BDO), (deletion of PDC1, elimination of pyruvate decarboxylation reaction), (deletion of PDC5, elimination of pyruvate decarboxylation reaction), (introduction of cellobiose utilization pathway, production of 2,3-BDO from cellobiose), (production of 2,3-BDO, titer of 5.29 g/l), (production of 2,3-BDO, productivity of 0.22 g/l h), (production of 2,3-BDO, yield of 0.29 g 2,3-BDO/g cellobiose)\n",
      "Progress: 79.95% completed\n",
      "Response for Row 1600:\n",
      "Answer to Question 2: (deletion of gene ARO8, upregulation of ARO10), (deletion of gene ARO8, production of phenylethanol), (deletion of gene ARO8, changes in intracellular concentrations of amino acids), (combination of aro8Δ mutation with other mutations, stimulation of phenylethanol production)\n",
      "Progress: 80.00% completed\n",
      "Response for Row 1601:\n",
      "Answer to Question 2: (deletion of neutral lipid biosynthesis genes in S. cerevisiae, rescue of cell growth by CtDGAT2b), (overexpression of CtDGAT2b in S. cerevisiae, increased triacylglycerol production), (presence of CtDGAT2b in S. cerevisiae, incorporation of free fatty acids into triacylglycerol), (mutation in CtDGAT2a/b, altered efficiency in triacylglycerol production), (overexpression of CtDGAT2b in C. tropicalis, enhanced production of storage lipid enriched with saturated fatty acids)\n",
      "Progress: 80.05% completed\n",
      "Response for Row 1602:\n",
      "Answer to Question 2: (deletion of gene ARO9, no effect on growth with aromatic amino acids), (overexpression of gene ARO8, initialization of histidine degradation), (overexpression of gene ARO8, supports growth on histidine as sole nitrogen source)\n",
      "Progress: 80.10% completed\n",
      "Response for Row 1603:\n",
      "Answer to Question 2: (deletion of gene tpsA, lower internal trehalose contents), (deletion of gene tppA, lower internal trehalose contents), (deletion of gene tppB, lower internal trehalose contents), (deletion of gene tppA, reduced sporulation), (deletion of gene tppA, abnormal conidiophores), (deletion of gene tppA, accumulated levels of trehalose-6-phosphate), (deletion of gene tppB, conidia more susceptible to thermal stress)\n",
      "Progress: 80.15% completed\n",
      "Response for Row 1604:\n",
      "Answer to Question 2: (deletion of gene SUB1, increased splicing efficiency)\n",
      "Progress: 80.20% completed\n",
      "Response for Row 1605:\n",
      "Answer to Question 2: (overexpression of gene PDR12, increased tolerance to sorbic acid), (overexpression of gene PDR12, increased tolerance to propionic acid), (overexpression of gene PDR12, increased tolerance to levulinic acid), (deletion of gene PDR12, increased tolerance to acetic acid), (deletion of gene PDR12, increased tolerance to formic acid), (deletion of gene PDR12, rapid recovery in acetic acid), (deletion of gene CMK1, different response to weak organic acids when combined with deletion of PDR12)\n",
      "Progress: 80.25% completed\n",
      "Response for Row 1606:\n",
      "Answer to Question 2: (deletion of gene TEL1, increased sensitivity to camptothecin), (deletion of gene TEL1, synthetic cytotoxic interactions with 14 other gene deletions), (deletion of gene TEL1, telomere defects), (deletion of gene TEL1, conserved synthetic cytotoxic interaction in Caenorhabditis elegans)\n",
      "Progress: 80.30% completed\n",
      "Response for Row 1607:\n",
      "Answer to Question 2: (deletion of gene eco1, reduced ribosomal DNA (rDNA) transcription), (deletion of gene eco1, transcriptional signature of starvation), (deletion of gene fob1, rescues rRNA production), (deletion of gene fob1, partially rescues transcription genome-wide), (deletion of gene fob1, corrects genome-wide replication defects), (deletion of gene fob1, corrects nucleolar structure), (deletion of gene fob1, corrects rDNA segregation)\n",
      "Progress: 80.35% completed\n",
      "Response for Row 1608:\n",
      "Answer to Question 2: (gdh-null mutant, viable on all nitrogen sources tested), (gdh-null mutant, GS/GOGAT as primary alternative pathway on ammonia and glutamine), (gdh-null mutant, growth on proline requires both PUT and GS/GOGAT pathways), (gdh-null mutant, growth on glucose-urea media elicited decrease in GOGAT activity), (gdh-null mutant, growth on glucose-urea media elicited increase in P5CDH activity)\n",
      "Progress: 80.40% completed\n",
      "Response for Row 1609:\n",
      "Answer to Question 2: (deletion of gene fumarate reductase, consequence increased succinate yield), (deletion of gene pyruvate formate lyase, consequence increased succinate yield), (deletion of gene citrate lyase, consequence increased succinate yield), (deletion of gene β-galactosidase, consequence increased succinate yield), (deletion of gene aconitase, consequence increased succinate yield)\n",
      "Progress: 80.45% completed\n",
      "Response for Row 1610:\n",
      "Answer to Question 2: (deletion of gene ACA13, reduced Ca(2+) export), (deletion of gene ACA13, defects in compatible pollen germination), (deletion of gene ACA13, defects in seed production)\n",
      "Progress: 80.50% completed\n",
      "Response for Row 1611:\n",
      "Answer to Question 2: (evolution of orthogonal ribosomes, increased efficiency of unnatural amino acid incorporation), (manipulating the genome, enhanced efficiency of unnatural amino acid incorporation), (evolution of orthogonal quadruplet decoding ribosomes, increased number of distinct amino acids that can be site-specifically encoded), (discovery of mutually orthogonal synthetase/tRNA pairs, increased number of distinct amino acids that can be site-specifically encoded), (development and application of the pyrrolysyl-tRNA synthetase/tRNA pair, aided incorporation of unnatural amino acids in eukaryotic cells and animals), (combining chemoselective reactions with encoded amino acids, facilitated installation of posttranslational modifications), (combining chemoselective reactions with encoded amino acids, facilitated rapid derivatization with diverse fluorophores for imaging)\n",
      "Progress: 80.55% completed\n",
      "Response for Row 1612:\n",
      "Answer to Question 2: (deletion of gene involved in nitrate utilization, consequence inability to utilize nitrate)\n",
      "Progress: 80.60% completed\n",
      "Response for Row 1613:\n",
      "Answer to Question 2: (deletion of gene TPA1, modulates termination efficiency up to ∼10-fold), (deletion of gene TPA1, affects translational accuracy in a stop codon context-dependent manner), (hydroxylation of gene RPS23, increases translational accuracy), (hydroxylation of gene RPS23, decreases translational accuracy), (hydroxylation of gene RPS23, determines viability as a consequence of nonsense codon suppression)\n",
      "Progress: 80.65% completed\n",
      "Response for Row 1614:\n",
      "Answer to Question 2: (deletion of gene OLA1, affects stop codon bypass), (deletion of gene BSC2, affects stop codon bypass), (deletion of gene YNL040W, affects stop codon bypass)\n",
      "Progress: 80.70% completed\n",
      "Response for Row 1615:\n",
      "Answer to Question 2: (deletion of PKMT gene, altered stoichiometry of lysine methylation), (deletion of PKMT gene, processive methylation on lysine residues), (deletion of PKMT gene, selective methylation by specific PKMTs), (deletion of PKMT gene, broad range of stoichiometries of lysine methylation), (deletion of PKMT gene, specialized roles in regulation of non-histone protein activity), (deletion of PKMT gene, new relationships between SEE1 and EFM1 in EF1α methylation), (deletion of PKMT gene, reinterpretation of EFM1's direct PKMT activity)\n",
      "Progress: 80.75% completed\n",
      "Response for Row 1616:\n",
      "Answer to Question 2: (deletion of gene GPD1, succinate titer improved to 8.09±0.28g/L)\n",
      "Progress: 80.80% completed\n",
      "Response for Row 1617:\n",
      "Answer to Question 2: (deletion of gene FgIlv5, failure to grow under BCAA-limited conditions), (deletion of gene FgIlv5, decrease in aerial hyphae formation), (deletion of gene FgIlv5, decrease in red pigmentation), (deletion of gene FgIlv5, blockage of conidia formation), (deletion of gene FgIlv5, increased sensitivity to osmotic stress), (deletion of gene FgIlv5, increased sensitivity to oxidative stress), (deletion of gene FgIlv5, reduced virulence on wheat heads), (deletion of gene FgIlv5, low deoxynivalenol production in wheat kernels)\n",
      "Progress: 80.85% completed\n",
      "Response for Row 1618:\n",
      "Answer to Question 2: (deletion of gene gpi10, consequence Man-III addition defect), (deletion of gene smp3, consequence Man-IV addition defect), (expression of gene Gpi10/PigB from T. cruzi, consequence complements gpi10Δ mutation), (expression of gene Gpi10/PigB from T. cruzi, consequence complements smp3Δ mutation), (expression of gene Gpi10/PigB from T. congolense, consequence complements gpi10Δ mutation), (expression of gene Gpi10/PigB from T. congolense, consequence complements smp3Δ mutation), (expression of gene Gpi10/PigB from P. falciparum, consequence complements gpi10Δ mutation), (expression of gene Gpi10/PigB from P. falciparum, consequence complements smp3Δ mutation), (expression of gene PIGB from T. vivax, consequence complements gpi10Δ mutation), (expression of gene PIGB from T. brucei, consequence complements gpi10Δ mutation), (expression of gene T. cruzi PIGB, consequence complements gpi10Δ/smp3Δ double mutant)\n",
      "Progress: 80.90% completed\n",
      "Response for Row 1619:\n",
      "Answer to Question 2: (deletion of gene HLJ1, consequence 1), (deletion of gene HLJ1, consequence 2), (deletion of gene TEF2, consequence 1), (deletion of gene TEF2, consequence 2)\n",
      "Progress: 80.95% completed\n",
      "Response for Row 1620:\n",
      "Answer to Question 2: (deletion of gene Not4, consequence 1: impaired cellular protein solubility), (deletion of gene Not4, consequence 2: disruption of co-translational quality control), (deletion of gene Not4, consequence 3: impaired proteasome assembly), (overexpression of gene Not4, consequence 1: enhanced clearance of aberrant proteins via the proteasome)\n",
      "Progress: 81.00% completed\n",
      "Response for Row 1621:\n",
      "Answer to Question 2: (deletion of gene pox1, growth defect), (expression of gene pox2, increased ratio of MCFAs), (expression of gene crot, unblocked β-oxidation pathway), (expression of gene crot, normal growth rate), (deletion of gene pox1, expression of gene pox2, increased fatty acid accumulation), (deletion of gene pox1, expression of gene pox2, increased total fatty acid production), (deletion of gene pox1, expression of gene pox2, increased intracellular MCFAs), (deletion of gene pox1, expression of gene pox2, increased MCFAs in culture medium), (deletion of gene pox1, expression of gene pox2, expression of gene crot, increased total fatty acid production), (deletion of gene pox1, expression of gene pox2, expression of gene crot, increased intracellular MCFAs), (deletion of gene pox1, expression of gene pox2, expression of gene crot, increased MCFAs in culture medium)\n",
      "Progress: 81.05% completed\n",
      "Response for Row 1622:\n",
      "Answer to Question 2: (deletion of tRNA gene, no appreciable phenotype in rich medium), (deletion of tRNA gene, additional phenotypes under challenging conditions), (deletion of tRNA gene, robustness facilitated by backup compensation), (deletion of tRNA gene with identical anticodon, different contribution to cellular fitness), (deletion of single-copy tRNA family gene, stress response), (deletion of multi-copy tRNA family gene, increased expression of translation machinery)\n",
      "Progress: 81.10% completed\n",
      "Response for Row 1623:\n",
      "Answer to Question 2: (deletion of gene AIM44, defects in contractile ring closure), (deletion of gene AIM44, delay in Hof1p phosphorylation), (deletion of gene AIM44, altered Hof1p localization), (overexpression of gene DBF2, rescues cytokinesis defect)\n",
      "Progress: 81.15% completed\n",
      "Response for Row 1624:\n",
      "Answer to Question 2: (deletion of gene, marked decline in the rate of growth), (deletion of gene, efficiency of growth remains nearly unaffected), (affecting membrane integrity, decrease in efficiency of growth), (affecting ion homeostasis, decrease in efficiency of growth)\n",
      "Progress: 81.20% completed\n",
      "Response for Row 1625:\n",
      "Answer to Question 2: (adh1Δ, accumulation of 120 mg/L 1-butanol), (overexpression of pathway enzymes, highest reported 1-butanol titer of 242.8 mg/L), (elimination of competing pathways, highest reported 1-butanol titer of 242.8 mg/L)\n",
      "Progress: 81.25% completed\n",
      "Response for Row 1626:\n",
      "Answer to Question 2: (deletion of gene xxx, hypersensitivity to H2O2), (deletion of gene xxx, impaired oxidative stress tolerance), (deletion of essential gene xxx, enrichment in nucleus localization), (deletion of essential gene xxx, enrichment in nucleolus localization), (deletion of essential gene xxx, over-representation in rRNA synthesis), (deletion of essential gene xxx, over-representation in tRNA synthesis)\n",
      "Progress: 81.30% completed\n",
      "Response for Row 1627:\n",
      "Answer to Question 2: (deletion of COQ genes, respiratory deficiency), (deletion of COQ genes, decreased levels of Coq4), (deletion of COQ genes, decreased levels of Coq6), (deletion of COQ genes, decreased levels of Coq7), (deletion of COQ genes, decreased levels of Coq9), (over-expression of Coq8, restoration of steady-state levels of sensitive Coq polypeptides), (over-expression of Coq8, promotion of synthesis of late-stage Q-intermediates), (over-expression of Coq8, affects association of several Coq polypeptides in high molecular mass complexes), (supplementation with exogenous Q6, increased steady-state levels of Coq4), (supplementation with exogenous Q6, increased steady-state levels of Coq7), (supplementation with exogenous Q6, increased steady-state levels of Coq9), (supplementation with exogenous Q6, promotion of formation of late-stage Q-intermediates)\n",
      "Progress: 81.35% completed\n",
      "Response for Row 1628:\n",
      "Answer to Question 2: (disruption of gene MoMSN2, defects in aerial hyphal growth), (disruption of gene MoMSN2, reduction in conidial production), (disruption of gene MoMSN2, impaired infection of host plants), (disruption of gene MoMSN2, reduced expression of genes involved in conidiophore formation), (disruption of gene MoMSN2, impaired appressorium formation), (disruption of gene MoMSN2, impaired penetration), (disruption of gene MoMSN2, sensitivity to common osmotic stressors), (disruption of gene MoMSN2, modulation of genes participating in conidiation), (disruption of gene MoMSN2, modulation of genes participating in infection), (disruption of gene MoMSN2, modulation of genes participating in cell-wall integrity), (disruption of gene MoMSN2, modulation of genes participating in stress response)\n",
      "Progress: 81.40% completed\n",
      "Response for Row 1629:\n",
      "Answer to Question 2: (deletion of gene figA, consequence retardant hyphal growth), (deletion of gene figA, consequence sharp reduction of conidial production), (deletion of gene figA, consequence impairment of homothallic mating), (deletion of gene figA, consequence requirement of HACS midA for heterothallic mating), (deletion of gene figA, consequence decreased expression of brlA), (deletion of gene figA, consequence decreased expression of nsdD)\n",
      "Progress: 81.45% completed\n",
      "Response for Row 1630:\n",
      "Answer to Question 2: (deletion of gene involved in cell wall biosynthesis, compromised cell surface integrity), (deletion of gene involved in calcium homeostasis, increased susceptibility to stress conditions), (deletion of gene involved in nutritional response, impaired growth under nutrient-limited conditions), (deletion of gene involved in stress response, increased sensitivity to harsh phagosomal environment), (deletion of gene involved in protein glycosylation, altered recognition by macrophages), (deletion of gene involved in iron homeostasis, reduced iron uptake)\n",
      "Progress: 81.50% completed\n",
      "Response for Row 1631:\n",
      "Answer to Question 2: (deletion of gene Sir2p, consequence 1), (overexpression of gene Sirt1, consequence 2), (overexpression of gene Sirt1, restoration of transcriptional silencing at rDNA), (overexpression of gene Sirt1, repression of extrachromosomal rDNA circles formation), (overexpression of gene Sirt1, decrease in histone acetylation at H3K9), (overexpression of gene Sirt1, decrease in histone acetylation at H4K16)\n",
      "Progress: 81.55% completed\n",
      "Response for Row 1632:\n",
      "Answer to Question 2: (deletion of gene Gln3, activation of NCR-sensitive transcription by Gat1), (activation of Gln3 and Gat1, activation of GAT1 transcription), (production of Gat1 isoforms IsoA and IsoB, low level production in rich and poor nitrogen conditions), (placement of Myc(13) tag after Gat1 Ser-233, increased production of Gat1 isoforms IsoC-F in proline medium compared to glutamine medium), (requirement of Gln3 and UASGATA promoter elements, production of Gat1 isoforms IsoA, IsoC, and IsoD), (regulated production of wild type versus truncated Gat1 proteins, failure to be regulated in parallel), (sequence(s) downstream of Gat1 leucine 61, responsible for nonparallel production)\n",
      "Progress: 81.60% completed\n",
      "Response for Row 1633:\n",
      "Answer to Question 2: (deletion of gene Ppq1, consequence 1: up-regulation of mating signaling), (overexpression of gene Ppq1, consequence 1: down-regulation of mating signaling), (dephosphorylation by Ppq1, consequence 1: targeting at or upstream of the terminal MAP kinase Fus3)\n",
      "Progress: 81.65% completed\n",
      "Response for Row 1634:\n",
      "Answer to Question 2: (deletion of gene SGR810-815, consequence 1), (deletion of gene SGR810-815, consequence 2)\n",
      "Progress: 81.70% completed\n",
      "Response for Row 1635:\n",
      "Answer to Question 2: (deletion of gene LPD1, increased isobutanol production), (overexpression of gene pyruvate carboxylase, increased isobutanol production), (overexpression of gene malate dehydrogenase, increased isobutanol production), (overexpression of gene malic enzyme, increased isobutanol production), (deletion of gene LPD1, increased pyruvate supply for isobutanol biosynthesis), (activation of transhydrogenase-like shunt, increased isobutanol levels)\n",
      "Progress: 81.75% completed\n",
      "Response for Row 1636:\n",
      "Answer to Question 2: (deletion of gene FgFim, reduced rate of mycelial growth), (deletion of gene FgFim, reduced conidiation), (deletion of gene FgFim, delayed conidium germination), (deletion of gene FgFim, irregularly shaped hyphae), (deletion of gene FgFim, lack of sexual reproduction), (deletion of gene FgFim, decreased resistance to JS399-19), (deletion of gene FgFim, increased sensitivity to metal cations), (deletion of gene FgFim, increased sensitivity to osmotic stress), (deletion of gene FgFim, increased sensitivity to oxidative stress), (deletion of gene FgFim, increased sensitivity to cell membrane damage), (deletion of gene FgFim, increased sensitivity to cell wall damage), (deletion of gene FgFim, impaired virulence on flowering wheat heads), (deletion of gene FgFim, reduced production of deoxynivalenol), (deletion of gene FgFim, lower expression of Cyp51 genes)\n",
      "Progress: 81.80% completed\n",
      "Response for Row 1637:\n",
      "Answer to Question 2: (comK overexpression, fast preparation of competent cell), (exogenous endonuclease I-SceI overexpression, double-strand break repair), (upp gene deletion, counter-selection marker function), (foreign dsDNA integration, enhancement by double-crossover homologous recombination), (DSB repair, stimulation of second intramolecular homologous recombination), (second intramolecular homologous recombination, enhancement of resolution frequency)\n",
      "Progress: 81.85% completed\n",
      "Response for Row 1638:\n",
      "Answer to Question 2: (deletion of gene ARV1, decreased neutral lipid synthesis), (deletion of gene ARV1, increased fatty acid sensitivity), (deletion of gene ARV1, lipoapoptosis), (overexpression of gene ARV1, increased triglyceride mass), (overexpression of gene ARV1, increased lipid droplet number), (overexpression of gene ARV1, up-regulation of DGAT1), (overexpression of gene ARV1, up-regulation of CD36), (transcriptional regulation of PPARα by ARV1, up-regulation of DGAT1), (transcriptional regulation of PPARα by ARV1, up-regulation of CD36)\n",
      "Progress: 81.90% completed\n",
      "Response for Row 1639:\n",
      "Answer to Question 2: (deletion of methionine, yeast chronological lifespan extension), (increase of glutamic acid, yeast chronological lifespan extension), (restriction of glucose, yeast chronological lifespan extension), (deletion of methionine, low acetic acid production), (increase of glutamic acid, low acetic acid production), (gene deletion, distinct mechanisms of yeast longevity)\n",
      "Progress: 81.95% completed\n",
      "Response for Row 1640:\n",
      "Answer to Question 2: (expression of gene CmINV, complementation of growth on sucrose), (expression of gene CmINV, production of short-chain fructooligosaccharides), (deletion of gene suc2, inability to grow on sucrose), (expression of gene CmINV, increased activity by 50 mM LiCl)\n",
      "Progress: 82.00% completed\n",
      "Response for Row 1641:\n",
      "Answer to Question 2: (deletion of gene Tat1p, interference with aromatic and branched-chain amino acids transport), (deletion of gene Gap1p, interference with aromatic and branched-chain amino acids transport), (deletion of genes involved in biosynthesis and transport of aromatic and branched-chain amino acids, increased susceptibility to eugenol), (eugenol exposure, leakage of cell contents), (eugenol exposure, amino acid starvation)\n",
      "Progress: 82.05% completed\n",
      "Response for Row 1642:\n",
      "Answer to Question 2: (deletion of gene xxx, reduced statistical power without biological replicates), (deletion of gene xxx, 96% power to detect a two-fold change with 6 million reads per condition)\n",
      "Progress: 82.10% completed\n",
      "Response for Row 1643:\n",
      "Answer to Question 2: (deletion of gene FUI1, consequence uridine transport unaffected), (deletion of gene DUR3, consequence normal growth on urea), (deletion of gene FUR4, consequence normal growth on uracil)\n",
      "Progress: 82.15% completed\n",
      "Response for Row 1644:\n",
      "Answer to Question 2: (deletion of gene hemF, lethal phenotype), (deletion of gene hemF, impaired growth), (deletion of gene hemF, cellular porphyrin accumulation), (deletion of gene hemH, lethal phenotype), (deletion of gene hemH, impaired growth), (deletion of gene hemH, cellular porphyrin accumulation), (overexpression of gene hemH, increased cellular heme content), (overexpression of gene hemA, no increase in cellular heme content), (overexpression of gene hemF, increased cellular porphyrin intermediates), (overexpression of gene hemH, increased cellular porphyrin intermediates), (overexpression of gene hemA, increased cellular porphyrin intermediates)\n",
      "Progress: 82.20% completed\n",
      "Response for Row 1645:\n",
      "Answer to Question 2: (deletion of gene HpFAD2, absence of C18:2 and αC18:3), (deletion of gene HpFAD3, absence of αC18:3), (heterologous expression of gene HpFAD2, presence of C18:2 and αC18:3), (heterologous expression of gene HpFAD3, presence of C18:2 and αC18:3), (low oxygen conditions, up-regulation of HpFAD2), (low oxygen conditions, up-regulation of HpFAD3), (excess of C18:1, repression of HpFAD3 transcription), (excess of C18:2, repression of HpFAD3 transcription), (excess of C18:3, repression of HpFAD3 transcription)\n",
      "Progress: 82.25% completed\n",
      "Response for Row 1646:\n",
      "Answer to Question 2: (deletion of gene Mcm1, consequence competitive interference between paralogs), (overexpression of gene Mcm1, consequence increased molecular complexity of gene regulatory network)\n",
      "Progress: 82.30% completed\n",
      "Response for Row 1647:\n",
      "Answer to Question 2: (deletion of gene hxk2, consequence mitochondrial dysfunction), (deletion of gene hxk2, consequence accumulation of ROS), (deletion of gene hxk2, consequence cell death), (deletion of gene hxk2, consequence increase of apoptotic cells), (deletion of gene hxk2, consequence DNA fragmentation), (deletion of gene hxk2, consequence externalization of phosphatidylserine)\n",
      "Progress: 82.35% completed\n",
      "Response for Row 1648:\n",
      "Answer to Question 2: (deletion of gene PEX3, increased production of HBsAg), (deletion of gene PEX3, production of HBsAg nearly 8-9% of the total cell protein)\n",
      "Progress: 82.40% completed\n",
      "Response for Row 1649:\n",
      "Answer to Question 2:  (deletion of gene ole1, consequence 1), (expression of gene fD9desA, consequence 1), (expression of gene fD9desB, consequence 1), (expression of gene fD9desC, consequence 1), (expression of gene fD9desD, consequence 1)\n",
      "Progress: 82.45% completed\n",
      "Response for Row 1650:\n",
      "Answer to Question 2: (deletion of gene Prodh, hypovirulence), (deletion of gene Prodh, lower level of sporulation), (deletion of gene P5Cdh, hypovirulence), (supplementation of exogenous proline, increase in intracellular level of proline), (supplementation of exogenous proline, increase in activity of Prodh), (supplementation of exogenous proline, increase in activity of P5Cdh), (cells infected with hypovirus, down-regulation of Prodh), (cells infected with hypovirus, down-regulation of P5Cdh), (disruption of genes involved in conversion of arginine to ornithine, no effect on virulence), (disruption of genes involved in conversion of ornithine and glutamate to P5C, no effect on virulence), (disruption of genes involved in conversion of P5C to proline, no effect on virulence), (deletion of gene pro1, reduction in asexual sporulation), (deletion of gene pro2, reduction in asexual sporulation)\n",
      "Progress: 82.50% completed\n",
      "Response for Row 1651:\n",
      "Answer to Question 2: (deletion of gene Med18, consequence activation of transcription at the CYC1 promoter is impaired), (deletion of gene Med20, consequence activation of transcription at the CYC1 promoter is impaired), (deletion of gene Med19, consequence activation of transcription at the CYC1 promoter is impaired), (deletion of Mediator components, consequence normal levels of recruitment and activity of TFIIH at the CYC1 promoter are reduced), (deletion of Mediator components, consequence activation by the same activator at a different gene is unaffected), (overexpression of Mediator components, consequence triggering of inactive polymerase at CYC1 into a productively elongating form is enhanced)\n",
      "Progress: 82.55% completed\n",
      "Response for Row 1652:\n",
      "Answer to Question 2: (deletion of gene Aoso, effects on formation and localization of stress granules in response to heat stress)\n",
      "Progress: 82.60% completed\n",
      "Response for Row 1653:\n",
      "Answer to Question 2: (deletion of gene xxx, formation of mutagenesis cassette), (deletion of gene xxx, self-excision of mutagenesis cassette), (deletion of gene xxx, promotion of homologous recombination), (deletion of gene xxx, seamless deletion via single transformation), (deletion of gene xxx, dependency on length of inverted repeat), (deletion of gene xxx, improvement of deletion efficiency with increase of inverted repeat length)\n",
      "Progress: 82.65% completed\n",
      "Response for Row 1654:\n",
      "Answer to Question 2: (deletion of entire paralog set, severe reduction of dityrosine fluorescence), (deletion of genes in network, reduced dityrosine assembly), (presence of gene products, involvement of lipid droplets in outer spore wall formation)\n",
      "Progress: 82.70% completed\n",
      "Response for Row 1655:\n",
      "Answer to Question 2: (deletion of gene Pdc, consequence 1: rapid glucose consumption), (deletion of gene Pdc, consequence 2: no ethanol production), (mutation in gene MTH1, consequence 1: rapid glucose consumption), (overexpression of gene alsS, consequence 1: efficient 2,3-BD production), (overexpression of gene alsD, consequence 1: efficient 2,3-BD production), (overexpression of gene BDH1, consequence 1: efficient 2,3-BD production), (overexpression of gene alsS, consequence 2: faster glucose consumption), (overexpression of gene alsD, consequence 2: faster glucose consumption), (overexpression of gene BDH1, consequence 2: faster glucose consumption)\n",
      "Progress: 82.75% completed\n",
      "Response for Row 1656:\n",
      "Answer to Question 2: (deletion of RTG2, AA-PCD), (deletion of RTG3, AA-PCD), (up-regulation of peroxisomal isoform of citrate synthase, resistance to AA-PCD), (up-regulation of isocitrate dehydrogenase isoform 1, resistance to AA-PCD), (deletion of MKS1, no effect on AA-PCD), (deletion of HAP4, no effect on AA-PCD), (activation of RTG pathway, resistance to AA-PCD), (decrease in ROS production, resistance to AA-PCD), (decrease in cytochrome c release, resistance to AA-PCD)\n",
      "Progress: 82.80% completed\n",
      "Response for Row 1657:\n",
      "Answer to Question 2: (rad52 deletion, decreased efficiency of GRC), (dnl4 deletion, increased frequency of GRC in C. glabrata), (dnl4 deletion, minimal increase in GRC frequency in S. cerevisiae)\n",
      "Progress: 82.85% completed\n",
      "Response for Row 1658:\n",
      "Answer to Question 2: (expression of gene um06350.1, consequence xylan degradation), (expression of gene um06350.1, consequence production of xylotriose)\n",
      "Progress: 82.90% completed\n",
      "Response for Row 1659:\n",
      "Answer to Question 2: (deletion of gene ALD3, increased 2-PE production), (overexpression of gene ARO9, increased phenylpyruvate formation), (overexpression of gene ARO10, increased phenylacetaldehyde formation), (overexpression of gene ARO80, increased transcription of ARO9 and ARO10), (use of two-phase fermentation method, increased 2-PE extraction)\n",
      "Progress: 82.95% completed\n",
      "Response for Row 1660:\n",
      "Answer to Question 2: (deletion of gene YJR142W, lowered oxythiamin resistance), (overexpression of gene YJR142W, raised oxythiamin resistance), (expression of plant counterpart gene, restored oxythiamin resistance to YJR142W deletant), (expression of bacterial counterpart gene, restored oxythiamin resistance to YJR142W deletant)\n",
      "Progress: 83.00% completed\n",
      "Response for Row 1661:\n",
      "Answer to Question 2: (deletion of gene KCS1, consequence inositol auxotrophy), (deletion of gene KCS1, consequence decreased intracellular inositol), (deletion of gene KCS1, consequence decreased phosphatidylinositol), (deletion of gene KCS1, consequence decreased transcription of INO1), (deletion of gene OPI1, consequence does not fully rescue INO1 expression in kcs1Δ), (expression of gene KCS1, consequence increased Kcs1 protein levels in response to inositol depletion), (Kcs1-catalyzed production of inositol pyrophosphates from inositol pentakisphosphate, consequence optimal INO1 transcription)\n",
      "Progress: 83.05% completed\n",
      "Response for Row 1662:\n",
      "Answer to Question 2: (over-expression of the glutathione peroxidase 3 (GPX3) gene, increased GSSG content), (over-expression of the GSH1/GSH2 genes, increased total glutathione content), (deleting the glutathione reductase (GLR1) gene, yielded 7.3-fold more GSSG), (deleting the glutathione reductase (GLR1) gene, enhanced total glutathione content up to 1.6-fold)\n",
      "Progress: 83.10% completed\n",
      "Response for Row 1663:\n",
      "Answer to Question 2: (deletion of insertion peptide in GlyRS1, impaired catalytic efficiency in vitro), (deletion of insertion peptide in GlyRS1, 2- to 3-fold increase in Km), (deletion of insertion peptide in GlyRS1, 5- to 8-fold decrease in kcat), (mutation of insertion peptide in GlyRS1, impaired catalytic efficiency in vitro), (GRS2 codon optimization, functional gene conversion), (deletion of insertion peptide in GlyRS1, dispensable for protein stability in vivo), (deletion of insertion peptide in GlyRS1, dispensable for rescue activity of GRS1 at 30°C in vivo)\n",
      "Progress: 83.15% completed\n",
      "Response for Row 1664:\n",
      "Answer to Question 2: (deletion of gene SFP1, decrease in Sup35p amount), (overexpression of gene SFP1, increase in termination efficiency), (overexpression of gene SFP1, increase in Sup35p amount)\n",
      "Progress: 83.20% completed\n",
      "Response for Row 1665:\n",
      "Answer to Question 2: (deletion of gene KRE6, facilitation of mogroside III E production)\n",
      "Progress: 83.25% completed\n",
      "Response for Row 1666:\n",
      "Answer to Question 2: (deletion of gene tps1, consequence lower ethanol yields), (deletion of gene tdh3, consequence lower ethanol yields), (upregulation of gene tps1, consequence decrease in ethanol production), (upregulation of gene tps1, consequence increase in trehalose biosynthesis)\n",
      "Progress: 83.30% completed\n",
      "Response for Row 1667:\n",
      "Answer to Question 2:  (deletion of gene meu1, decrease in DMTS-P1 production), (deletion of gene mri1, decrease in DMTS-P1 production), (deletion of gene mde1, decrease in DMTS-P1 production), (deletion of gene mri1, decrease in DMTS producing potential), (deletion of gene mde1, decrease in DMTS producing potential), (deletion of gene spe2, increase in DMTS-P1 content), (deletion of gene spe2, decrease in DMTS producing potential), (deletion of gene spe2, deletion of gene mri1, further decrease in DMTS producing potential)\n",
      "Progress: 83.35% completed\n",
      "Response for Row 1668:\n",
      "Answer to Question 2: (deletion of gene TPS1, impaired ethanol production), (deletion of gene TPS1, diminished plasma membrane H⁺-ATPase activation), (deletion of gene TPS1, depletion of nearly all adenine nucleotides), (deletion of gene TPS1, conversion of adenine nucleotides into inosine), (deletion of gene AMD1, prevention of inosine formation), (expression of Tps1 protein from Yarrowia lipolytica, 90%-reduced T6P content), (mutation in gene TPS2, hyperaccumulation of T6P), (hyperaccumulation of T6P, increase in cytosolic pH), (hyperaccumulation of T6P, reduction in growth rates on non-fermentable carbon sources)\n",
      "Progress: 83.40% completed\n",
      "Response for Row 1669:\n",
      "Answer to Question 2: (deletion of gene Pot1p, consequence 1: impaired fatty acid utilisation), (deletion of gene Fox3p, consequence 1: partial impairment in fatty acid utilisation at elevated temperatures), (deletion of gene Pot13p, consequence 1: no detectable phenotype), (deletion of multiple genes Pot1p, Fox3p, Pot13p, consequence 1: no significant attenuation in virulence)\n",
      "Progress: 83.45% completed\n",
      "Response for Row 1670:\n",
      "Answer to Question 2: (deletion of gene UGA1, GABA growth defect), (deletion of gene UGA1, thermosensitivity), (deletion of gene UGA1, heat-induced production of reactive oxygen species), (expression of gene AtGABA-T, GABA growth), (expression of gene AtGABA-T, thermotolerance), (expression of gene AtGABA-T, limiting production of ROS), (expression of gene ScGABA-TKG, GABA growth), (expression of gene ScGABA-TKG, thermotolerance), (expression of gene ScGABA-TKG, limiting production of ROS)\n",
      "Progress: 83.50% completed\n",
      "Response for Row 1671:\n",
      "Answer to Question 2: (deletion of gene involved in electron transport chain, attenuation of oxidative stress), (deletion of gene involved in electron transport chain, partial amelioration of cell death)\n",
      "Progress: 83.55% completed\n",
      "Response for Row 1672:\n",
      "Answer to Question 2: (deletion of gene xxx, increased DNA content), (deletion of gene xxx, defect in mitosis), (deletion of gene xxx, defect in cytokinesis), (deletion of gene xxx, defect in cell separation)\n",
      "Progress: 83.60% completed\n",
      "Response for Row 1673:\n",
      "Answer to Question 2: (overexpression of gene SPT15-M2, enhanced ethanol tolerance), (overexpression of gene SPT15-M2, tolerance to hyperosmotic stress), (overexpression of gene SPT15-M2, improved ethanol production from VHG fermentations), (overexpression of gene SPT15-M2, constitutive and sustained activation of Hog1), (overexpression of gene SPT15-M2, reduced accumulation of ROS), (overexpression of gene SPT15-M3, enhanced ethanol tolerance), (overexpression of gene SPT15-M3, tolerance to hyperosmotic stress), (overexpression of gene SPT15-M3, improved ethanol production from VHG fermentations), (overexpression of gene SPT15-M3, constitutive and sustained activation of Hog1), (overexpression of gene SPT15-M3, reduced accumulation of ROS), (deletion of gene GPH1, sensitivity to 50% glucose), (deletion of gene HSP12, sensitivity to 50% glucose), (deletion of gene AIM17, sensitivity to 50% glucose), (deletion of gene SSA4, sensitivity to 50% glucose), (deletion of gene USV1, sensitivity to 50% glucose), (deletion of gene IGD1, sensitivity to 50% glucose)\n",
      "Progress: 83.65% completed\n",
      "Response for Row 1674:\n",
      "Answer to Question 2: (deletion of gene AAF1, reduced acetic acid levels in wine by up to 39.2%), (deletion of gene AAF1, did not increase acetaldehyde levels), (deletion of gene ALD6, reduced acetic acid levels), (deletion of gene ALD6, increased acetaldehyde levels by 41.4%), (overexpression of gene AAF1, increased acetic acid levels in wine in an Ald6p-dependent manner), (overexpression of gene AAF1, produced significantly more acetic acid in wine in an ald4Δald6Δ strain)\n",
      "Progress: 83.70% completed\n",
      "Response for Row 1675:\n",
      "Answer to Question 2: (deletion of gene SIT4, decreased pyruvate decarboxylase activity), (deletion of gene SIT4, reduced fermentative capacity), (deletion of gene SIT4, hyperphosphorylation of Pdc1p in logarithmic phase), (phosphorylation of Pdc1p, increased K(m) for TPP), (phosphorylation of Pdc1p, increased K(m) for pyruvate), (phosphorylation of Pdc1p, decreased V(max))\n",
      "Progress: 83.75% completed\n",
      "Response for Row 1676:\n",
      "Answer to Question 2: (deletion of gene pdc1, consequence 1), (deletion of gene pdc5, consequence 1), (deletion of gene pdc6, consequence 1), (deletion of gene aro10, consequence 1), (deletion of gene thi3, consequence 1), (expression of gene LgScAro10, consequence 1), (expression of gene LgSeubAro10, consequence 1), (induction of gene LgScAro10 by phenylalanine, consequence 1), (induction of gene LgSeubARO10 by phenylalanine, consequence 1), (higher basal expression of gene LgSeubARO10 during growth with ammonia, consequence 1), (deletion of gene LgSeubARO80, consequence 1)\n",
      "Progress: 83.80% completed\n",
      "Response for Row 1677:\n",
      "Answer to Question 2: (deletion of enzymes essential for sporulation, impact on metabolic pathways), (overexpression of metabolic genes required for sporulation, enhancement of ATP production in early phase), (overexpression of metabolic genes required for sporulation, increase in carbohydrate breakdown in middle and late stages)\n",
      "Progress: 83.85% completed\n",
      "Response for Row 1678:\n",
      "Answer to Question 2: (deletion of gene RRN10, severe growth defects), (deletion of gene RRN10, reduced RNA content), (overexpression of gene RPL40A, restored growth defects), (overexpression of gene RPL40A, increased RNA content), (overexpression of gene RPL40B, restored growth defects), (overexpression of gene RPL40B, increased RNA content)\n",
      "Progress: 83.90% completed\n",
      "Response for Row 1679:\n",
      "Answer to Question 2: (deletion of ribosome biogenesis-related genes, increased L-lactate production)\n",
      "Progress: 83.95% completed\n",
      "Response for Row 1680:\n",
      "Answer to Question 2: (deletion of gene PDC, detrimental to cell growth), (introduction of gene LDH, efficient production of lactate), (exclusive expression of gene PDC, metabolic flow towards ethanol fermentation), (exclusive expression of gene LDH, metabolic flow towards lactate fermentation)\n",
      "Progress: 84.00% completed\n",
      "Response for Row 1681:\n",
      "Answer to Question 2: (overexpression of gene SPBC16A3.08c, extended chronological lifespan), (overexpression of gene SPBC16A3.08c, decreased mating efficiency), (overexpression of gene SPBC16A3.08c, high temperature-sensitive growth), (overexpression of gene SPBC16A3.08c, low temperature-sensitive growth), (deletion of gene oga1 (+), caffeine-sensitive phenotype), (deletion of gene oga1 (+), canavanine-resistant phenotype)\n",
      "Progress: 84.05% completed\n",
      "Response for Row 1682:\n",
      "Answer to Question 2: (deletion of gene xxx, lethal phenotype), (deletion of gene xxx and gene yyy, synthetic lethality), (disruption of metabolic pathways, few examples of synthetic lethality), (formation of protein-protein interactions, synthetic lethality), (genes involved in essential pairs, recent duplicates), (functional similarity, preservation of essential protein complexes)\n",
      "Progress: 84.10% completed\n",
      "Response for Row 1683:\n",
      "Answer to Question 2: (gene deletion, improved prediction of intra-cellular fluxes), (gene deletion, coupling cellular biological objective with product flux)\n",
      "Progress: 84.15% completed\n",
      "Response for Row 1684:\n",
      "Answer to Question 2: (overexpression of gene Dga1∆Np, increased lipid content), (deletion of gene dga1, increased lipid content), (overexpression of gene Dga1∆Np, dependent on glucose concentration), (overexpression of gene Dga1∆Np, dependent on nitrogen limitation), (overexpression of gene Dga1∆Np, dependent on active leucine biosynthesis), (deletion of 3' terminal region of dga1 ORF, increased lipid content), (overexpression of gene Esa1p, reverted lipid content to wild-type level), (deletion of gene dga1, decreased ESA1 mRNA expression)\n",
      "Progress: 84.20% completed\n",
      "Response for Row 1685:\n",
      "Answer to Question 2: (deletion of FAD-dependent monooxygenase, altered oxidation state of intermediates), (deletion of cytochrome P450 oxygenase 1, altered oxidation state of intermediates), (deletion of cytochrome P450 oxygenase 2, altered oxidation state of intermediates), (heterologous expression of FAD-dependent monooxygenase, formation of intermediate compounds), (heterologous expression of cytochrome P450 oxygenase 1, formation of intermediate compounds), (heterologous expression of cytochrome P450 oxygenase 2, formation of intermediate compounds)\n",
      "Progress: 84.25% completed\n",
      "Response for Row 1686:\n",
      "Answer to Question 2: (deletion of gene mdh1, failure to produce succinate), (deletion of gene oac1, failure to produce succinate), (deletion of gene dic1, production of 0.02 C-mol/C-mol glucose succinate), (deletion of gene dic1, reductive TCA cycle activity)\n",
      "Progress: 84.30% completed\n",
      "Response for Row 1687:\n",
      "Answer to Question 2: (overproduction of gene DIN7, enhanced Mhr1-dependent mtDNA replication), (overproduction of gene DIN7, increased number of residual DSBs at ori5 in HS-ρ(-) cells), (overproduction of gene DIN7, increased deletion mutagenesis at the ori5 region in ρ(+) cells), (simultaneous overproduction of gene Mhr1, suppressed enhanced mtDNA replication), (simultaneous overproduction of gene Mhr1, suppressed increased residual DSBs at ori5), (simultaneous overproduction of gene Mhr1, suppressed increased deletion mutagenesis at the ori5 region), (relative activity levels of Din7 and Mhr1, modulated preference for replication versus homologous recombination to repair DSBs at ori5)\n",
      "Progress: 84.35% completed\n",
      "Response for Row 1688:\n",
      "Answer to Question 2: (deletion of gene RAD50, consequence 1), (deletion of gene RAD51, consequence 1), (deletion of gene RAD52, consequence 1), (deletion of gene RAD54, consequence 1), (deletion of gene RAD55, consequence 1), (deletion of gene RAD57, consequence 1), (deletion of gene MRE11, consequence 1), (deletion of gene XRS2, consequence 1), (deletion of 73 additional genes, consequence 1)\n",
      "Progress: 84.40% completed\n",
      "Response for Row 1689:\n",
      "Answer to Question 2: (expression of HIV-1 Gag protein, release of Gag virus-like particles), (expression of HIV-1 Tat, no support for transcription), (replacement of HIV-1 promoter U3 with yeast glyceraldehyde-3-phosphate dehydrogenase gene promoter, restoration of gag mRNA transcription), (expression of HIV-1 Rev, no support for protein synthesis), (deletion of R-U5 and SL region, restoration of Gag protein expression), (co-expression of HIV-1 genome with R-U5-SL region and Gag protein, transcription and incorporation of HIV-1 genomic RNA into Gag VLPs), (deletion of R-U5 to gag gene, failure of HIV-1 RNA packaging into Gag VLPs)\n",
      "Progress: 84.45% completed\n",
      "Response for Row 1690:\n",
      "Answer to Question 2: (deletion of gene Dga1p, reduced TG synthesis), (deletion of gene Lro1p, major reduction in TG synthesis), (deletion of gene Are2p, absence of SE synthesis), (deletion of gene Dga1p, preference for polyunsaturated fatty acids), (deletion of gene Lro1p, low fatty acid substrate specificity), (deletion of gene Are2p, preference for certain fatty acids in SE), (deletion of gene Dga1p, Lro1p, insufficient SE for lipid droplet formation), (deletion of gene Are2p, Lro1p, Dga1p, insufficient TG for lipid droplet biogenesis)\n",
      "Progress: 84.50% completed\n",
      "Response for Row 1691:\n",
      "Answer to Question 2: (deletion of gene SSK1, reduced glycerol production), (deletion of gene SSK1, increased ethanol yield), (expression of gene ssk1(E330N…K356N), reduced glycerol/ethanol ratio), (expression of gene ssk1(E330N…K356N), compromised volumetric productivity), (expression of gene ssk1(E330N…K356N), compromised osmotolerance)\n",
      "Progress: 84.55% completed\n",
      "Response for Row 1692:\n",
      "Answer to Question 2: (deletion of mannosyltransferases, consequence 1: obtaining more than 90% homogeneity in glycan structures)\n",
      "Progress: 84.60% completed\n",
      "Response for Row 1693:\n",
      "Answer to Question 2: (deletion of gene FPS1, consequence decreased xylitol yield), (deletion of gene FPS1, consequence increased ethanol yield), (deletion of gene FPS1, consequence increased intracellular xylitol concentration), (deletion of gene FPS1, consequence decreased xylose consumption rate under anaerobic conditions)\n",
      "Progress: 84.65% completed\n",
      "Response for Row 1694:\n",
      "Answer to Question 2: (deletion of PHA synthase gene, increased 3HA production), (deletion of β-oxidation pathway gene, increased 3HA production), (expression of tesB gene, increased 3HTD production), (expression of tesB gene, increased 3HDD production), (expression of PTE1 gene, increased 3HD production)\n",
      "Progress: 84.70% completed\n",
      "Response for Row 1695:\n",
      "Answer to Question 2: (deletion of amino acids 57-70 in α-mating factor secretion signal, increased secretion of horseradish peroxidase), (deletion of amino acids 57-70 in α-mating factor secretion signal, increased secretion of lipase)\n",
      "Progress: 84.75% completed\n",
      "Response for Row 1696:\n",
      "Answer to Question 2: (deletion of gene PPR1, drastic growth defects), (deletion of gene PPR1, increased microconidia production), (deletion of gene PPR2, elevated fumonisin production), (deletion of gene PPR1, abnormally swollen cells formation), (deletion of gene PPR2, early hyphal branching during conidia germination)\n",
      "Progress: 84.80% completed\n",
      "Response for Row 1697:\n",
      "Answer to Question 2: (deletion of gene xxx, impact on small RNAs), (temperature-sensitive mutation of gene xxx, impact on small RNAs)\n",
      "Progress: 84.85% completed\n",
      "Response for Row 1698:\n",
      "Answer to Question 2: (deletion of gene PilA, consequence viability unaffected), (deletion of gene PilA, consequence germination unaffected), (deletion of gene PilB, consequence viability unaffected), (deletion of gene PilB, consequence germination unaffected), (deletion of gene SurG, consequence viability unaffected), (deletion of gene SurG, consequence germination unaffected), (deletion of genes PilA, PilB, SurG, consequence viability unaffected), (deletion of genes PilA, PilB, SurG, consequence germination unaffected), (presence of myriocin, consequence reduced number of PilA-GFP foci), (presence of myriocin, consequence altered distribution of PilA-GFP foci)\n",
      "Progress: 84.90% completed\n",
      "Response for Row 1699:\n",
      "Answer to Question 2: (deletion of gene PHO3, consequence cAPase-negative phenotype), (deletion of gene PHO3, consequence genetic instability of cAPase activity), (deletion of gene PHO3, consequence generation of chimeric PHO5/3 fusion genes)\n",
      "Progress: 84.95% completed\n",
      "Response for Row 1700:\n",
      "Answer to Question 2: (deletion of gene CnPlc1, reduced IP(3) content), (deletion of gene CnPlc1, impaired thermotolerance), (deletion of gene CnPlc1, compromised cell walls), (deletion of gene CnPlc1, reduced capsule production), (deletion of gene CnPlc1, reduced melanization), (deletion of gene CnPlc1, defective cell separation), (deletion of gene CnPlc1, inability to form mating filaments), (deletion of gene CnArg1, increased IP(3) content), (deletion of gene CnArg1, impaired thermotolerance), (deletion of gene CnArg1, compromised cell walls), (deletion of gene CnArg1, reduced capsule production), (deletion of gene CnArg1, reduced melanization), (deletion of gene CnArg1, defective cell separation), (deletion of gene CnArg1, inability to form mating filaments), (deletion of gene CnArg1, enlarged vacuoles), (overexpression of gene CnPlc1, increased PIP(2) hydrolysis), (overexpression of gene CnPlc1, increased IP(3) production), (overexpression of gene CnArg1, increased complex inositol polyphosphates synthesis), (overexpression of gene CnArg1, essential for cellular homeostasis), (overexpression of gene CnArg1, essential for virulence)\n",
      "Progress: 85.00% completed\n",
      "Response for Row 1701:\n",
      "Answer to Question 2: (overexpression of gene ERR2, complement growth defect in double-null strain), (overexpression of gene ERR3, complement growth defect in double-null strain)\n",
      "Progress: 85.05% completed\n",
      "Response for Row 1702:\n",
      "Answer to Question 2: (deletion of gene Sdh3p, increase in succinate production), (deletion of gene Ser3p/Ser33p, reduction in serine formation from glycolysis), (deletion of gene Sdh3p, decrease in succinate depletion by succinate dehydrogenase complex), (deletion of gene Ser3p/Ser33p, redirection of carbon flux towards succinate production)\n",
      "Progress: 85.10% completed\n",
      "Response for Row 1703:\n",
      "Answer to Question 2: (expression of gene LAC4, abolished activity of acid/neutral trehalase), (expression of gene LAC12, abolished activity of acid/neutral trehalase), (deletion of gene MIG1, relieved glucose repression), (expression of gene LAC4, no physiological burden in cells), (expression of gene LAC12, no physiological burden in cells), (expression of gene LAC4, increased galactose consumption), (expression of gene LAC12, increased galactose consumption), (expression of gene LAC4, specific lactose uptake rate 0.98 g lactose g CDW(-1) h(-1)), (expression of gene LAC12, specific lactose uptake rate 2.50 g lactose g CDW(-1) h(-1)), (expression of gene LAC4, ethanol production 24.3 g lactose g CDW(-1) h(-1)), (expression of gene LAC12, ethanol production 23.4 g lactose g CDW(-1) h(-1))\n",
      "Progress: 85.15% completed\n",
      "Response for Row 1704:\n",
      "Answer to Question 2: (deletion of gene VPS34, reduced mRNA production from long or G+C-rich coding sequences), (deletion of gene VPS34, reduced rate of RNA polymerase II progression through lacZ CDS), (deletion of gene VPS15, reduced mRNA production from long or G+C-rich coding sequences), (deletion of gene VPS15, reduced rate of RNA polymerase II progression through lacZ CDS), (deletion of gene VPS15, reduced NuA4 occupancy in certain transcribed CDS), (deletion of gene VPS34, reduced NuA4 occupancy in certain transcribed CDS), (deletion of gene VPS15, impaired localization of the induced GAL1 gene to the nuclear periphery), (deletion of gene VPS34, impaired localization of the induced GAL1 gene to the nuclear periphery)\n",
      "Progress: 85.20% completed\n",
      "Response for Row 1705:\n",
      "Answer to Question 2: (deletion of gene FgMETB, inability to grow on fructose gelatin agar medium containing SO(4)(2-) as sole sulfur source), (deletion of gene FgMETB, more than 90% conidia unable to germinate in 2% sucrose solution within 6 or 12 h of incubation), (deletion of gene FgMETB, rescue of mycelial growth and spore germination by supplementation of 1 mM Met or 0.5 mg/ml homocysteine), (deletion of gene FgMETB, decreased virulence on wheat heads), (deletion of gene FgMETB, low level of deoxynivalenol (DON) production in wheat kernels), (deletion of gene FgMETB, increased sensitivity to the sterol demethylation inhibitor tebuconazole)\n",
      "Progress: 85.25% completed\n",
      "Response for Row 1706:\n",
      "Answer to Question 2: (deletion of gene SLI1, almost complete loss of tri- and tetraacetyl phytosphingosines), (deletion of gene ATF2, 15-fold increase in triacetyl phytosphingosine)\n",
      "Progress: 85.30% completed\n",
      "Response for Row 1707:\n",
      "Answer to Question 2: (deletion of gene FUM1, FA production), (deletion of gene FUM1, slightly lower growth of S. cerevisiae), (introduction of gene RoPYC, increased FA production to 1134±48 mg L(-1)), (introduction of gene SFC1, increased FA production to 1675±52 mg L(-1))\n",
      "Progress: 85.35% completed\n",
      "Response for Row 1708:\n",
      "Answer to Question 2: (Deletion of gene dse2, enhanced resistance to lactic acid), (Deletion of gene scw11, enhanced resistance to lactic acid), (Deletion of gene eaf3, enhanced resistance to lactic acid), (Deletion of gene sed1, enhanced resistance to lactic acid), (Deletion of gene dse2, shortened lag phase in 6% lactic acid), (Deletion of gene scw11, shortened lag phase in 6% lactic acid), (Deletion of gene eaf3, shortened lag phase in 6% lactic acid), (Deletion of gene sed1, shortened lag phase in 6% lactic acid), (Introduction of gene LDH, increased lactic acid productivity), (Quadruple deletion of genes dse2, scw11, eaf3, sed1, 27% increase in lactic acid productivity)\n",
      "Progress: 85.40% completed\n",
      "Response for Row 1709:\n",
      "Answer to Question 2: (overexpression of gene SIR2, extends CLS), (overexpression of gene SIR2, reduces acetate production), (overexpression of gene HST3, shortens CLS), (overexpression of gene HST3, increases ethanol level), (overexpression of gene HST3, reduces acetic acid production), (overexpression of gene SOD2, moderately extends CLS), (deletion of gene NGR1, decreases CLS), (deletion of gene UTH4, decreases CLS), (deletion of gene PUF3, increases CLS), (deletion of gene PUB1, increases CLS), (deletion of gene PUB1, increases glycerol production), (deletion of gene PUB1, blocks stress granule formation), (deletion of gene YCA1, shortens CLS), (deletion of gene AIF1, shortens CLS)\n",
      "Progress: 85.45% completed\n",
      "Response for Row 1710:\n",
      "Answer to Question 2: (overexpression of optimized, cytosolically localized valine biosynthesis pathway, complemented valine auxotrophy), (overexpression of xylose isomerase XylA, enabled growth on D-xylose), (overexpression of transaldolase Tal1, enabled growth on D-xylose), (overexpression of xylulokinase Xks1, enabled growth on D-xylose), (ilv2,3,5 triple deletion, valine auxotrophy), (overexpression of ketoacid decarboxylase Aro10, enabled fermentation to isobutanol), (overexpression of alcohol dehydrogenase Adh2, enabled fermentation to isobutanol)\n",
      "Progress: 85.50% completed\n",
      "Response for Row 1711:\n",
      "Answer to Question 2: (deletion of gene ubc7, increased sesquiterpene production), (deletion of gene ssm4, increased sesquiterpene production), (deletion of gene doa10, increased sesquiterpene production), (deletion of gene pho86, increased sesquiterpene production), (deletion of gene ubc7, Hmg1p stabilization), (deletion of gene ssm4, Hmg1p stabilization), (deletion of gene doa10, Hmg1p stabilization), (deletion of gene pho86, Hmg1p stabilization), (deletion of gene ubc7, Hmg2p stabilization), (deletion of gene ssm4, Hmg2p stabilization), (deletion of gene doa10, Hmg2p stabilization), (deletion of gene pho86, Hmg2p stabilization)\n",
      "Progress: 85.55% completed\n",
      "Response for Row 1712:\n",
      "Answer to Question 2: (deletion of gene SsSOD1, increased sensitivity to heavy metal toxicity), (deletion of gene SsSOD1, increased sensitivity to oxidative stress), (deletion of gene SsSOD1, reduced ability to detoxify superoxide in infected leaves), (deletion of gene SsSOD1, reduced expression levels of sspg1), (deletion of gene SsSOD1, reduced expression levels of sspg3), (deletion of gene SsSOD1, reduced virulence)\n",
      "Progress: 85.60% completed\n",
      "Response for Row 1713:\n",
      "Answer to Question 2: (deletion of gene EXO1, increase in mutation rates), (deletion of gene EXO1, reduction in efficiency of MMR of replication errors generated by Pol α), (deletion of gene EXO1, implication that 5' ends serve as entry points for 5' excision of replication errors made by Pol α), (deletion of gene EXO1, mutation rates in Pol α mutator strain are 5- to 25-fold lower compared to an msh2Δ strain completely lacking MMR)\n",
      "Progress: 85.65% completed\n",
      "Response for Row 1714:\n",
      "Answer to Question 2: (deletion of gene isc1, higher resistance to acetic acid), (deletion of gene isc1, lower levels of some phytoceramide species), (deletion of gene isc1, lower levels of ROS production), (deletion of gene isc1, reduced mitochondrial alterations), (deletion of gene lag1, higher resistance to acetic acid), (deletion of gene lag1, lower levels of some phytoceramide species), (deletion of gene lag1, lower levels of ROS production), (deletion of gene lag1, reduced mitochondrial alterations)\n",
      "Progress: 85.70% completed\n",
      "Response for Row 1715:\n",
      "Answer to Question 2: (deletion of gene alg3, consequence 1), (deletion of gene alg3, consequence 2), (deletion of gene alg11, consequence 1), (deletion of gene alg11, consequence 2), (expression of artificial flippase, consequence 3), (expression of protozoan oligosaccharyltransferase, consequence 4), (expression of human N-acetylglucosaminyltransferases I, consequence 5), (expression of human N-acetylglucosaminyltransferases II, consequence 6)\n",
      "Progress: 85.75% completed\n",
      "Response for Row 1716:\n",
      "Answer to Question 2: (deletion of gene RAD1, loss of LLR activity), (deletion of gene CSM3, loss of LLR activity), (deletion of gene RAD1, exacerbation of loop repair failure when combined with CSM3 deletion), (deletion of gene CSM3, exacerbation of loop repair failure when combined with RAD1 deletion), (expression of gene CSM3, control of minisatellite length expansions), (expression of gene TOF1, control of minisatellite length expansions), (expression of gene MRC1, control of minisatellite contractions)\n",
      "Progress: 85.80% completed\n",
      "Response for Row 1717:\n",
      "Answer to Question 2: (deletion of gene GAL1, graded response), (deletion of gene GAL3, graded response), (overexpression of gene GAL1, inactivation of remaining feedback loop), (overexpression of gene GAL3, inactivation of remaining feedback loop)\n",
      "Progress: 85.85% completed\n",
      "Response for Row 1718:\n",
      "Answer to Question 2: (deletion of chaperone genes, altered [PSI(+)] induction efficiency), (deletion of chaperone genes, changed Rnq1p aggregate morphology), (deletion of chaperone genes, modified Rnq1p aggregate size), (deletion of chaperone genes, variant dominance alteration)\n",
      "Progress: 85.90% completed\n",
      "Response for Row 1719:\n",
      "Answer to Question 2: (deletion/insertion changes, effect on multiple genes), (over 180 SNPs, more than one-quarter nonsynonymous mutations), (altered expression of approximately 900 genes, similar alterations in all three evolved strains), (genes with a role in oxidative metabolism, altered expression), (decreased ethanol production, switch from fermentation to respiration)\n",
      "Progress: 85.95% completed\n",
      "Response for Row 1720:\n",
      "Answer to Question 2: (deletion of gene pfh1, sensitive to growth under aerobic conditions), (deletion of gene pfh1, increased levels of total iron), (deletion of gene pfh1, hallmarks of oxidative stress such as protein carbonylation), (deletion of gene pfh1, decreased aconitase activity), (deletion of gene pfh1, lower levels of oxygen consumption), (deletion of gene pfh1, gene products up-regulated upon iron depletion in wild-type cells are constitutively up-regulated), (deletion of gene pfh1, gene products down-regulated upon iron depletion in wild-type cells are constitutively down-regulated), (deletion of gene pfh1, decrease of cytosolic available iron), (deletion of gene pfh1, activation of Grx4)\n",
      "Progress: 86.00% completed\n",
      "Response for Row 1721:\n",
      "Answer to Question 2: (deletion of gene BAT1, decrease in higher metabolite formation), (deletion of gene BAT2, decrease in higher metabolite formation), (deletion of gene AAD6, decrease in higher alcohol production), (deletion of gene HOM2, decrease in higher alcohol production), (deletion of gene PAD1, decrease in higher alcohol production), (deletion of gene PRO2, decrease in higher alcohol production), (deletion of gene SPE1, decrease in higher alcohol production), (deletion of gene THI3, decrease in higher alcohol production), (double-deletion of genes BAT2 and HOM2, greater decrease in isobutanol production), (double-deletion of genes BAT2 and HOM2, greater decrease in isoamyl alcohol production), (double-deletion of genes BAT2 and HOM2, greater decrease in isobutyric acid production), (double-deletion of genes BAT2 and HOM2, greater decrease in isovaleric acid production), (triple-deletion of genes BAT2, THI3, and AAD6, greatest impact on end metabolite production), (triple-deletion of genes BAT2, THI3, and HOM2, greatest impact on end metabolite production), (triple-deletion of genes HOM2, PRO2, and AAD6, greater effect on isoamyl alcohol levels), (triple-deletion of genes HOM2, PRO2, and AAD6, greater effect on isovaleric acid levels)\n",
      "Progress: 86.05% completed\n",
      "Response for Row 1722:\n",
      "Answer to Question 2: (deletion of gene PDR8, consequence nerolidol production), (deletion of gene ABZ1, consequence 2-phenyl ethanol production), (deletion of gene PLB2, consequence ethyl esters production)\n",
      "Progress: 86.10% completed\n",
      "Response for Row 1723:\n",
      "Answer to Question 2: (gene deletion, generation of product derivatives), (gene deletion, creation of artificial gene clusters), (gene expression, heterologous expression), (gene expression, introduction of single or multiple point mutations)\n",
      "Progress: 86.15% completed\n",
      "Response for Row 1724:\n",
      "Answer to Question 2:  (expression of gene Pi-xylA, improved xylose utilization), (overexpression of gene XKS1, improved xylose utilization), (overexpression of gene RPE1, improved xylose utilization), (overexpression of gene RKI1, improved xylose utilization), (overexpression of gene TAL1, improved xylose utilization), (overexpression of gene TKL1, improved xylose utilization), (deletion of gene GRE3, poor xylose utilization initially), (deletion of gene COX4, poor xylose utilization initially), (deletion of gene PHO13, decreased xylose growth in respiratory deficiency strain), (increased activity of xylose isomerase, improved xylose utilization), (upregulation of enzymes involved in glycolysis, improved xylose utilization), (upregulation of enzymes involved in glutamate synthesis, improved xylose utilization), (downregulation of trehalose synthesis, improved xylose utilization), (downregulation of glycogen synthesis, improved xylose utilization)\n",
      "Progress: 86.20% completed\n",
      "Response for Row 1725:\n",
      "Answer to Question 2: (deletion of gene, maximizing productivity), (deletion of gene, minimizing raw material costs), (deletion of gene, altering product rates), (deletion of gene, altering product costs), (deletion of gene, altering product yields), (deletion of gene, altering product purity)\n",
      "Progress: 86.25% completed\n",
      "Response for Row 1726:\n",
      "Answer to Question 2: (deletion of gene alg3, production of predominantly Man(3)GlcNAc(2) protein-linked N-glycans)\n",
      "Progress: 86.30% completed\n",
      "Response for Row 1727:\n",
      "Answer to Question 2: (deletion of gene Hrp3, nucleosome perturbation across the chromosome), (deletion of gene Hrp3, production of antisense transcripts), (deletion of gene Hrp3, destabilization of the DNA-histone interaction), (deletion of gene Hrp3, cell cycle-dependent alleviation of heterochromatin silencing)\n",
      "Progress: 86.35% completed\n",
      "Response for Row 1728:\n",
      "Answer to Question 2: (deletion of gene otg1(+), complete loss of galactosylation), (deletion of gene otg2(+), complete loss of galactosylation), (deletion of gene otg3(+), complete loss of galactosylation), (expression of gene otg2(+), Gal transfer activity toward pyridylaminated Man(9)GlcNAc(2) oligosaccharide), (expression of gene otg2(+), Gal transfer activity toward pyridylaminated Manα1,2-Manα1,2-Man oligosaccharide), (expression of gene otg3(+), Gal transfer activity toward pyridylaminated Man(9)GlcNAc(2) oligosaccharide), (expression of gene otg2(+), generation of α1,3-linkage), (expression of gene otg3(+), generation of α1,3-linkage)\n",
      "Progress: 86.40% completed\n",
      "Response for Row 1729:\n",
      "Answer to Question 2: (deletion of segment from rat CCT, increase in lipid-independent V(max)), (deletion of segment from rat CCT, weakening of membrane binding affinity)\n",
      "Progress: 86.45% completed\n",
      "Response for Row 1730:\n",
      "Answer to Question 2: (deletion of gene MFG1, consequence 1), (deletion of gene MFG1, consequence 2)\n",
      "Progress: 86.50% completed\n",
      "Response for Row 1731:\n",
      "Answer to Question 2: (deletion of gene xxx, shift in half-cell glutathione redox potential by 75-10 mV)\n",
      "Progress: 86.55% completed\n",
      "Response for Row 1732:\n",
      "Answer to Question 2: (deletion of gene ZrTRK1, growth at low K(+) concentrations impaired), (deletion of gene ZrTRK1, decreased potassium content in cells), (deletion of gene ZrTRK1, plasma membrane hyperpolarization), (deletion of gene ZrTRK1, increased plasma membrane depolarization during potassium starvation), (deletion of gene ZrTRK1, reduced effect on ZrPma1 activity), (expression of gene ZrTRK1, complementation of S. cerevisiae trk1Δ trk2Δ cell phenotypes)\n",
      "Progress: 86.60% completed\n",
      "Response for Row 1733:\n",
      "Answer to Question 2: (deletion of gene tpi, consequence 1), (overexpression of gene tpi2, consequence 2)\n",
      "Progress: 86.65% completed\n",
      "Response for Row 1734:\n",
      "Answer to Question 2: (deletion of gene tRNALeu(GAG), no phenotypic consequence), (deletion of gene tRNALeu(UAG), lethal effect), (deletion of three genes for tRNALeu(UAG), inability to compensate decoding capacity by another tRNALeu isoacceptor)\n",
      "Progress: 86.70% completed\n",
      "Response for Row 1735:\n",
      "Answer to Question 2: (deletion of gene NEO1, consequence haploinsufficiency in presence of tripelennamine), (overexpression of autophagy-related genes, consequence hypersensitivity to tripelennamine)\n",
      "Progress: 86.75% completed\n",
      "Response for Row 1736:\n",
      "Answer to Question 2: (expression of MsrB, catalytic efficiency towards free Met-O >2000-fold lower than yeast fRMsr), (expression of MsrA, k(cat) values 40-500-fold greater than corresponding MsrBs), (expression of yeast MsrA, ratio of free to peptide Msr activity 1:0.8), (expression of MsrBs and MsrAs, apparent in vivo free Msr activities in yeast cells)\n",
      "Progress: 86.80% completed\n",
      "Response for Row 1737:\n",
      "Answer to Question 2: (deletion of gene CSF1, unable to complete fermentation), (overexpression of gene CSF1, decreased lag phase), (overexpression of gene CSF1, increased fermentation rate), (overexpression of gene CSF1, reached higher populations than control strain), (deletion of gene HSP104, unable to complete fermentation), (overexpression of gene HSP104, decreased lag phase), (overexpression of gene HSP104, increased fermentation rate), (overexpression of gene HSP104, reached higher populations than control strain), (deletion of gene TIR2, unable to complete fermentation), (overexpression of gene TIR2, decreased lag phase), (overexpression of gene TIR2, increased fermentation rate), (overexpression of gene TIR2, reached higher populations than control strain)\n",
      "Progress: 86.85% completed\n",
      "Response for Row 1738:\n",
      "Answer to Question 2: (deletion of gene SGS1, increased gap-repair efficiency)\n",
      "Progress: 86.90% completed\n",
      "Response for Row 1739:\n",
      "Answer to Question 2: (deletion of gene veA, consequence biosynthesis of sterigmatocystin), (deletion of gene veA, consequence inability to produce toxin), (mutation in gene rtfA, consequence regain capability to produce toxin), (deletion of gene rtfA, consequence restore mycotoxin production), (deletion of gene rtfA, consequence decrease in penicillin production), (deletion of gene rtfA, consequence decrease in growth rate), (deletion of gene rtfA, consequence reduction in conidiation), (deletion of gene rtfA, consequence loss of sexual development)\n",
      "Progress: 86.95% completed\n",
      "Response for Row 1740:\n",
      "Answer to Question 2: (deletion of gene SsMNT1, inability to synthesize specific oligosaccharides), (expression of gene SsMNT1, addition of up to two mannose residues to acceptor molecules), (expression of gene SsMNT1, involvement in N-linked glycosylation pathway), (expression of gene SsMNT1, involvement in O-linked glycosylation pathway), (deletion of gene SsMNT1, disruption in glycosylation pathways)\n",
      "Progress: 87.00% completed\n",
      "Response for Row 1741:\n",
      "Answer to Question 2: (deletion of CCR4-NOT complex, compromised aneuploid viability), (deletion of poly(A)-specific nuclease, compromised aneuploid viability), (deletion of genes required for homologous recombination repair, compromised aneuploid viability)\n",
      "Progress: 87.05% completed\n",
      "Response for Row 1742:\n",
      "Answer to Question 2: (gene overexpression, improved xylose assimilation), (gene overexpression, increased glutathione production), (gene overexpression, enhanced butanol production), (gene deletion, optimized metabolic pathways)\n",
      "Progress: 87.10% completed\n",
      "Response for Row 1743:\n",
      "Answer to Question 2: (Deletion of gene AMT1, reduced vegetative growth), (Deletion of gene AMT1, increased sensitivity to oxidative stress), (Deletion of gene AMT1, increased sensitivity to membrane stress), (Deletion of gene AMT1, reduced DON mycotoxin production), (Deletion of gene AMT1, reduced virulence on flowering wheat heads), (Deletion of gene AMT1, impaired nuclear export of FgHrp1), (Deletion of gene AMT1, normal nuclear export of FgNab2), (Deletion of gene AMT1, Δamt1 Δamt2 double mutants show similar defects), (Deletion of gene AMT2, similar defects to Δamt1 mutant when deleted together)\n",
      "Progress: 87.15% completed\n",
      "Response for Row 1744:\n",
      "Answer to Question 2: (deletion of gene related to sulfur metabolism, increase in production of 3MH and 3MHA), (deletion of gene related to sulfur metabolism, decrease in production of 3MH and 3MHA), (deletion of gene related to nitrogen metabolism, increase in production of 3MH and 3MHA), (deletion of gene related to nitrogen metabolism, decrease in production of 3MH and 3MHA), (addition of urea, increase in production of 3MH and 3MHA), (addition of di-ammonium phosphate, increase in production of 3MH and 3MHA), (addition of cysteine, increase in production of 3MH and 3MHA), (addition of S-ethyl-L-cysteine, increase in production of 3MH and 3MHA)\n",
      "Progress: 87.20% completed\n",
      "Response for Row 1745:\n",
      "Answer to Question 2: (deletion of gene DBP2, accumulation of noncoding transcripts), (deletion of gene DBP2, inefficient 3' end formation), (deletion of gene DBP2, appearance of aberrant transcriptional initiation products), (deletion of gene DBP2, synthetic lethal with deletion of RRP6)\n",
      "Progress: 87.25% completed\n",
      "Response for Row 1746:\n",
      "Answer to Question 2: (deletion of gene THI2, enhanced pyruvate yield), (deletion of gene THI2, decreased ethanol titre), (deletion of gene THI3, enhanced pyruvate yield), (addition of thiamine, increased pyruvate concentration)\n",
      "Progress: 87.30% completed\n",
      "Response for Row 1747:\n",
      "Answer to Question 2: (deletion of gene ady2, consequence spore formation regulation affected), (deletion of gene nce103, consequence spore formation regulation affected), (overexpression of gene ras, consequence increased PKA activation), (overexpression of gene Cyr1, consequence increased PKA activation), (deletion of gene Gpa2, consequence reduced PKA activation), (deletion of gene Gpr1, consequence reduced PKA activation), (overexpression of gene Ady1, consequence proper localization of Mpc70 and Spo74), (overexpression of gene Ady1, consequence initiation of spore formation)\n",
      "Progress: 87.35% completed\n",
      "Response for Row 1748:\n",
      "Answer to Question 2: (deletion of gene ADH1, increased 2,3-butanediol production), (deletion of gene ADH3, increased 2,3-butanediol production), (deletion of gene ADH5, increased 2,3-butanediol production), (deletion of gene ADH1, overproduction of glycerol), (deletion of gene ADH3, overproduction of glycerol), (deletion of gene ADH5, overproduction of glycerol), (deletion of gene GPD2, reduced glycerol production), (deletion of gene GPD2, reduced 2,3-butanediol production), (deletion of gene ALD6, reduced acetate production), (introduction of 2,3-butanediol biosynthetic pathway from Bacillus subtilis, increased titer and yield), (introduction of 2,3-butanediol biosynthetic pathway from E. aerogenes, increased titer and yield)\n",
      "Progress: 87.40% completed\n",
      "Response for Row 1749:\n",
      "Answer to Question 2: (deletion of gene SPCC1450.16c, increased TG content by 1.7-fold), (deletion of gene SPAC1786.01c, increased TG content by 1.7-fold), (deletion of gene SPAC1A6.05c, increased TG content by 1.7-fold), (triple deletion of genes SPCC1450.16c, SPAC1786.01c, SPAC1A6.05c, increased TG content to 2.7-fold), (triple deletion mutant, growth sensitive to cerulenin), (addition of oleic acid, restored growth defect by cerulenin), (introduction of CpFAH12 gene, increased ricinoleic acid composition by 1.1-fold), (triple deletion mutant, increased total fatty acid content by 1.6-fold), (triple deletion mutant, increased ricinoleic acid production by 1.8-fold)\n",
      "Progress: 87.45% completed\n",
      "Response for Row 1750:\n",
      "Answer to Question 2: (deletion of gene FgOS-2, reduced DON production in planta), (deletion of gene FgOS-2, reduced ZEA production in planta), (deletion of gene FgOS-2, increased DON production in vitro), (deletion of gene FgOS-2, similar ZEA production in vitro as wild type), (deletion of gene FgOS-2, reduced pathogenicity towards maize), (deletion of gene FgOS-2, reduced pathogenicity towards wheat), (deletion of gene FgOS-2, inability to reach rachis node in wheat), (deletion of gene FgOS-2, increased resistance to fludioxonil), (deletion of gene FgOS-2, reduced growth on NaCl-supplemented agar), (deletion of gene FgOS-2, impaired conidial germination on osmotic media), (deletion of gene FgOS-2, swollen germ tubes containing multiple nuclei), (deletion of gene FgOS-2, failure to produce perithecia), (deletion of gene FgOS-2, failure to produce ascospores), (deletion of gene FgOS-2, modulation of catalase transcription and activity), (deletion of gene FgOS-2, regulation of noxC gene), (deletion of gene FgOS-2, regulation of atf1 gene)\n",
      "Progress: 87.50% completed\n",
      "Response for Row 1751:\n",
      "Answer to Question 2: (overexpression of gene TAL1, increase in flux from pentose phosphate pathway into glycolytic pathway), (overexpression of gene TKL1, increase in flux from pentose phosphate pathway into glycolytic pathway), (deletion of gene TAL1, decrease in growth on xylose), (deletion of gene TAL1, decrease in xylose-fermenting ability), (deletion of gene TKL1, decrease in growth on xylose), (deletion of gene TKL1, decrease in xylose-fermenting ability), (deletion of gene TKL1, requirement for glucose utilization), (deletion of gene NQM1, slight influence on growth on xylose), (deletion of gene NQM1, slight influence on xylose-fermenting ability), (deletion of gene TKL2, slight influence on growth on xylose), (deletion of gene TKL2, slight influence on xylose-fermenting ability), (overexpression of gene TKL1, impairment in rate of xylose consumption), (overexpression of gene TKL1, impairment in ethanol production), (overexpression of gene TKL2, impairment in rate of xylose consumption), (overexpression of gene TKL2, impairment in ethanol production)\n",
      "Progress: 87.55% completed\n",
      "Response for Row 1752:\n",
      "Answer to Question 2: (deletion of gene yar1, accumulation of 20S pre-rRNA), (deletion of gene yar1, 40S export defect), (deletion of gene rps3, accumulation of 20S pre-rRNA), (deletion of gene rps3, 40S export defect), (overexpression of gene rps3, suppression of yar1 deletion defects), (deletion of gene yar1, enhancement of rps3 mutation phenotypes)\n",
      "Progress: 87.60% completed\n",
      "Response for Row 1753:\n",
      "Answer to Question 2: (deletion of gene HO, consequence 1), (deletion of gene Sc-HO, consequence 2)\n",
      "Progress: 87.65% completed\n",
      "Response for Row 1754:\n",
      "Answer to Question 2: (deletion of HDS1 domain, increased cytoplasmic localization of Sec7), (deletion of Arf1, increased cytoplasmic localization of Sec7), (interaction of Sec7 with activated Arf1, localization of Sec7 to TGN)\n",
      "Progress: 87.70% completed\n",
      "Response for Row 1755:\n",
      "Answer to Question 2: (overexpression of gene ILV5, quicker process of mitochondrial degeneration), (overexpression of gene ILV5, altered intracellular amino acid profile), (overexpression of gene ILV5, laxer nitrogen catabolite repression regulation), (overexpression of gene ILV5, improved fermentation activity in the first hours of fermentation)\n",
      "Progress: 87.75% completed\n",
      "Response for Row 1756:\n",
      "Answer to Question 2: (deletion of gene RIM15, defective stress response), (deletion of gene RIM15, decreased synthesis of trehalose), (deletion of gene RIM15, decreased synthesis of glycogen), (deletion of gene RIM15, impaired G1 arrest), (expression of gene RIM15, suppression of defective stress response), (loss of function of gene RIM15, improved fermentation rate), (loss of function of gene IGO1, improved fermentation rate), (loss of function of gene IGO2, improved fermentation rate)\n",
      "Progress: 87.80% completed\n",
      "Response for Row 1757:\n",
      "Answer to Question 2: (deletion of gene STF2, promotion of reactive oxygen species production), (deletion of gene STF2, induction of apoptotic cell death), (overexpression of gene STF2, reduction in ROS production)\n",
      "Progress: 87.85% completed\n",
      "Response for Row 1758:\n",
      "Answer to Question 2: (deletion of gene Fus3, reduced mating efficiency)\n",
      "Progress: 87.90% completed\n",
      "Response for Row 1759:\n",
      "Answer to Question 2: (deletion of gene Sey1, reduced frequency of Chr4 disomy), (deletion of gene Glo3, reduced frequency of Chr4 disomy), (deletion of gene Gcs2, reduced frequency of Chr4 disomy), (disruption of Sey1, elongated cytoplasmic ER), (disruption of Glo3, reduced reticulation in ER), (disruption of Gcs2, reduced reticulation in ER), (deletion of Sey1, expanded ER cisternae)\n",
      "Progress: 87.95% completed\n",
      "Response for Row 1760:\n",
      "Answer to Question 2: (deletion of gene MHO1, consequence double-deleted cells lacking MHO1 and PLC1 proliferate for up to ten generations), (deletion of gene MHO1, consequence no effect on invasion induced by nutrient deprivation), (overexpression of gene MHO1, consequence blocks the invasive ability of yeast cells)\n",
      "Progress: 88.00% completed\n",
      "Response for Row 1761:\n",
      "Answer to Question 2: (deletion of multiple genes, combinatorial loss of mini-chromosomes), (deletion of 12 genes, synthetic lethality), (loss of mini-chromosomes, genome-wide amplifications), (loss of mini-chromosomes, amplification of mitochondrial genome), (loss of mini-chromosomes, amplification of genomic regions not in mini-chromosomes)\n",
      "Progress: 88.05% completed\n",
      "Response for Row 1762:\n",
      "Answer to Question 2: (deletion of gene ribosomal protein, consequence slow growth), (deletion of gene ribosomal protein, consequence resistance to ER stress), (deletion of gene ribosomal protein, consequence enhanced tunicamycin resistance), (deletion of gene ribosomal protein, consequence independent of Hac1), (deletion of gene ribosomal protein, consequence reduced translation)\n",
      "Progress: 88.10% completed\n",
      "Response for Row 1763:\n",
      "Answer to Question 2: (loss of gene YKL225W, increased ethanol tolerance), (loss of gene PAU16, increased ethanol tolerance), (loss of gene YKL223W, increased ethanol tolerance), (loss of gene YKL222C, increased ethanol tolerance), (loss of gene MCH2, increased ethanol tolerance), (loss of gene FRE2, increased ethanol tolerance), (loss of gene COS9, increased ethanol tolerance), (loss of gene SRY1, increased ethanol tolerance), (loss of gene JEN1, increased ethanol tolerance), (loss of gene URA1, increased ethanol tolerance), (loss of gene YHL050C, increased ethanol tolerance), (loss of gene YKL050W-A, increased ethanol tolerance), (loss of gene YHL049C, increased ethanol tolerance), (loss of gene YHL048C-A, increased ethanol tolerance), (loss of gene COS8, increased ethanol tolerance), (loss of gene YHLComega1, increased ethanol tolerance), (loss of gene ARN2, increased ethanol tolerance), (loss of gene YHL046W-A, increased ethanol tolerance), (loss of gene PAU13, increased ethanol tolerance), (loss of gene YHL045W, increased ethanol tolerance), (loss of gene YHL044W, increased ethanol tolerance), (loss of gene ECM34, increased ethanol tolerance), (loss of gene YHL042W, increased ethanol tolerance), (loss of gene YHL041W, increased ethanol tolerance), (loss of gene ARN1, increased ethanol tolerance)\n",
      "Progress: 88.15% completed\n",
      "Response for Row 1764:\n",
      "Answer to Question 2: (deletion of gene pre9, decrease in boundary function), (deletion of gene ump1, decrease in boundary function)\n",
      "Progress: 88.20% completed\n",
      "Response for Row 1765:\n",
      "Answer to Question 2: (deletion of gene PHO13, improved ethanol production from xylose), (deletion of gene PHO13, increased ethanol yield in medium with 30 mM acetic acid), (deletion of gene PHO13, fourfold higher specific ethanol productivity in presence of 90 mM furfural), (deletion of gene PHO13, greater than fourfold higher xylose consumption rate from rice straw hydrolysate)\n",
      "Progress: 88.25% completed\n",
      "Response for Row 1766:\n",
      "Answer to Question 2: (deletion of SIZ2, consequence increased gene silencing), (overexpression of SIZ2, consequence reduced binding of Sir2 to telomeres), (mutation of ESC1, consequence enhanced antagonism to gene silencing by SIZ2 overexpression)\n",
      "Progress: 88.30% completed\n",
      "Response for Row 1767:\n",
      "Answer to Question 2: (deletion of gene DPP1, increase in α-santalene titer), (overexpression of gene HMGR, increase in α-santalene productivity), (deletion of gene LPP1, increase in α-santalene productivity), (modulation of expression of gene ERG9, diversion of carbon flux from sterol synthesis towards α-santalene), (use of HXT1 promoter to control ERG9 expression, improvement in α-santalene productivity by 3.4 fold)\n",
      "Progress: 88.35% completed\n",
      "Response for Row 1768:\n",
      "Answer to Question 2: (overexpression of gene RDL2, increased tolerance to oxidative stress), (deletion of gene RDL2, increased reactive oxygen species production), (deletion of gene RDL2, loss of viability at end of fermentation), (upregulation of adenylate cyclase gene, higher PKA activity), (removal of δ element in promoter, increased life span), (higher expression of glycolytic genes, faster growth), (higher level of protein degradation by autophagy, fast growth)\n",
      "Progress: 88.40% completed\n",
      "Response for Row 1769:\n",
      "Answer to Question 2: (deletion of essential gene, affects large range of other essential genes), (deletion of non-essential gene, interrupts other non-essential genes), (deletion of essential genes, halts essential metabolites production)\n",
      "Progress: 88.45% completed\n",
      "Response for Row 1770:\n",
      "Answer to Question 2: (deletion of gene GTR1, affected acetate production), (deletion of gene GTR2, affected acetate production), (deletion of gene LIP5, affected acetate production), (deletion of gene LSM1, affected acetate production), (deletion of gene PHO85, affected acetate production), (deletion of gene PLM2, affected acetate production), (deletion of gene RTG1, affected acetate production), (deletion of gene RTG2, affected acetate production), (deletion of gene UBP3, affected acetate production), (deletion of gene EMI5, affected succinate production), (deletion of gene SDH1, affected succinate production), (deletion of gene SDH2, affected succinate production), (deletion of gene SDH4, affected succinate production), (deletion of gene TCM62, affected succinate production), (deletion of gene YDR379C-A, affected succinate production), (overexpression of gene LIP5, affected acetate production in S. cerevisiae), (overexpression of gene EMI5, affected acetate production in S. pastorianus), (overexpression of gene LIP5, affected acetate production in S. pastorianus), (overexpression of gene RTG2, affected acetate production in S. pastorianus), (overexpression of gene UBP3, affected acetate production in S. pastorianus), (overexpression of gene EMI5, affected citrate production in S. pastorianus), (overexpression of gene LIP5, affected citrate production in S. pastorianus), (overexpression of gene RTG2, affected citrate production in S. pastorianus), (overexpression of gene UBP3, affected citrate production in S. pastorianus), (overexpression of gene EMI5, affected lactate production in S. pastorianus), (overexpression of gene LIP5, affected lactate production in S. pastorianus), (overexpression of gene RTG2, affected lactate production in S. pastorianus), (overexpression of gene UBP3, affected lactate production in S. pastorianus), (overexpression of gene EMI5, affected succinate production in S. pastorianus), (overexpression of gene LIP5, affected succinate production in S. pastorianus), (overexpression of gene RTG2, affected succinate production in S. pastorianus), (overexpression of gene UBP3, affected succinate production in S. pastorianus)\n",
      "Progress: 88.50% completed\n",
      "Response for Row 1771:\n",
      "Answer to Question 2: (deletion of gene sulE, loss of de-esterification ability), (deletion of gene sulE, increased sensitivity to herbicides), (expression of gene sulE, ability to de-esterify sulfonylurea herbicides), (expression of gene sulE, increased resistance to herbicides)\n",
      "Progress: 88.55% completed\n",
      "Response for Row 1772:\n",
      "Answer to Question 2: (deletion of gene YBR246W, accumulation of diphthine)\n",
      "Progress: 88.60% completed\n",
      "Response for Row 1773:\n",
      "Answer to Question 2: (deletion of gene HpYPS1, minor influence on sensitivity to cell wall stress), (deletion of gene HpYPS7, minor influence on sensitivity to cell wall stress), (expression of gene HpYPS1, restoration of growth defects in S. cerevisiae yps1-deletion strains), (expression of gene HpYPS7, limited complementation effect on S. cerevisiae yps7-deletion strain), (expression of gene HpYPS1, induction by sodium dodecyl sulphate), (expression of gene HpYPS1, induction by tunicamycin), (deletion of gene HpYPS1, elimination of aberrant proteolytic cleavage of recombinant proteins)\n",
      "Progress: 88.65% completed\n",
      "Response for Row 1774:\n",
      "Answer to Question 2: (deletion of gene sla1(+), irregularities in tRNA processing), (deletion of gene sla1(+), altered distribution of pre-tRNA intermediates), (deletion of gene sla1(+), increased mRNAs from amino acid metabolism genes), (deletion of gene sla1(+), slow growth in Edinburgh minimal medium), (deletion of gene sla1(+), sensitivity to rapamycin), (deletion of gene sla1(+), deficiency of leucine uptake), (deletion of gene sla1(+), hypersensitivity to NH4), (deletion of gene sla1(+), genetic links to the TOR pathway), (overexpression of gene los1(+), suppression of reduction in pre-tRNA levels), (overexpression of gene los1(+), suppression of AAM gene up-regulation), (overexpression of gene los1(+), suppression of slow growth of sla1-Δ cells)\n",
      "Progress: 88.70% completed\n",
      "Response for Row 1775:\n",
      "Answer to Question 2: (deletion of ladB, growth arrest on galactitol), (deletion of ladB, significantly slower growth on D-galactose with D-xylose), (deletion of xyrA, reduced growth on D-galactose), (overexpression of ladB in Saccharomyces cerevisiae, production of L-xylo-3-hexulose from galactitol)\n",
      "Progress: 88.75% completed\n",
      "Response for Row 1776:\n",
      "Answer to Question 2: ( deletion of gene nde1, consequence shorter life span), ( deletion of gene tcm62, consequence shorter life span), ( deletion of gene rip1, consequence shorter life span), ( deletion of gene cyt1, consequence shorter life span), ( deletion of gene qrc8, consequence shorter life span), ( deletion of gene pet117, consequence shorter life span), ( deletion of gene cox11, consequence shorter life span), ( deletion of gene atp11, consequence shorter life span), ( deletion of gene fmc1, consequence shorter life span), ( deletion of gene nde1, consequence lower respiration rate), ( deletion of gene fmc1, consequence lower respiration rate), ( deletion of gene nde1, consequence higher intrinsic superoxide radicals), ( deletion of gene fmc1, consequence higher intrinsic superoxide radicals)\n",
      "Progress: 88.80% completed\n",
      "Response for Row 1777:\n",
      "Answer to Question 2: (deletion of gene YAP1, increased resistance to PAF26), (deletion of gene YAP1, disrupted induction of NO), (deletion of gene ARG1, reduced basal NO levels), (presence of PAF26, induced intracellular production of NO), (presence of PAF26, blocked NO production in Δarg1 mutant), (presence of l-NAME, restored yeast growth in presence of PAF26)\n",
      "Progress: 88.85% completed\n",
      "Response for Row 1778:\n",
      "Answer to Question 2: (cloning and transformation of 4CL1 gene, resveratrol production), (cloning and transformation of STS gene, resveratrol production), (deletion of PAD1 gene, no enhancement of resveratrol production)\n",
      "Progress: 88.90% completed\n",
      "Response for Row 1779:\n",
      "Answer to Question 2: (deletion of gene Cin8 loop 8 insert, consequence reduced midzone-directed motility), (deletion of gene Cin8 loop 8 insert, consequence affected ionic strength-dependent directional switching), (deletion of gene Cin8 loop 8 insert, consequence reduced efficiency to support spindle elongation)\n",
      "Progress: 88.95% completed\n",
      "Response for Row 1780:\n",
      "Answer to Question 2: (deletion of gene ADH1, production of glycerol), (deletion of gene ADH1, production of acetaldehyde), (expression of gene ADH1, efficient reduction of acetaldehyde to ethanol), (expression of gene ADH1, oxidation of produced or added ethanol), (expression of gene ADH2, production of ethanol from glucose), (expression of gene ADH3, production of ethanol from glucose), (deletion of gene ADH4, poor growth on glucose), (deletion of gene ADH5, poor growth on glucose)\n",
      "Progress: 89.00% completed\n",
      "Response for Row 1781:\n",
      "Answer to Question 2: (Deletion of gene msnA, Retarded colony growth), (Deletion of gene msnA, Increased conidiation), (Deletion of gene msnA, Slightly higher aflatoxin production), (Deletion of gene msnA, Elevated kojic acid production), (Deletion of gene msnA, Up-regulation of superoxide dismutase), (Deletion of gene msnA, Up-regulation of catalase), (Deletion of gene msnA, Up-regulation of cytochrome c peroxidase), (Deletion of gene msnA, Higher levels of reactive oxygen species (ROS))\n",
      "Progress: 89.05% completed\n",
      "Response for Row 1782:\n",
      "Answer to Question 2: (deletion of gene KNR4, increased release of mannoproteins during fermentation), (deletion of gene KNR4, autolysis continues after aging for 78 days), (deletion of gene KNR4, moderate flocculence), (deletion of gene KNR4, changes in expression of Flo1p-regulated genes), (deletion of gene KNR4, overexpression of FKS1)\n",
      "Progress: 89.10% completed\n",
      "Response for Row 1783:\n",
      "Answer to Question 2: (deletion of gene PPT1, consequence constitutive hyperphosphorylation of Hsf1p), (deletion of gene PPT1, consequence enhanced fermentation ability), (overexpression of gene PPT1, consequence recovery of HSE-mediated stress response in sake yeast)\n",
      "Progress: 89.15% completed\n",
      "Response for Row 1784:\n",
      "Answer to Question 2: (deletion of gene ADH1, decrease in ethanol production), (deletion of gene ADH1, upregulation of other isozyme genes), (deletion of all known ADH isozyme genes, disruption of ethanol production)\n",
      "Progress: 89.20% completed\n",
      "Response for Row 1785:\n",
      "Answer to Question 2: (insertional mutagenesis of gene CaGIT1, abolished ability to use GroPIns as phosphate source at acidic pH), (insertional mutagenesis of gene CaGIT1, abolished transport of [(3)H]GroPIns at acidic and physiological pHs), (reintegration of gene GIT1, restored ability to use GroPIns as phosphate source), (reintegration of gene GIT1, restored transport of [(3)H]GroPIns), (growth under low-phosphate conditions, increased label uptake from [(3)H]GroPIns), (growth under low-phosphate conditions, increased label uptake from [(3)H]GroPCho), (deletion of gene CaPHO4, prevented utilization of GroPIns as phosphate source), (deletion of gene CaPHO4, did not affect utilization of GroPCho as phosphate source)\n",
      "Progress: 89.25% completed\n",
      "Response for Row 1786:\n",
      "Answer to Question 2: (deletion of gene MNL2, diminished degradation efficiency of misfolded CPY(*))\n",
      "Progress: 89.30% completed\n",
      "Response for Row 1787:\n",
      "Answer to Question 2: (deletion of AGT1, reversion of growth characteristics), (expression of AGT1, improved sucrose-transport capacity), (expression of SUC2 with modified promoter and 5' coding sequences, predominant cytosolic localization of invertase), (overexpression of AGT1, elevated sucrose-transport capacity)\n",
      "Progress: 89.35% completed\n",
      "Response for Row 1788:\n",
      "Answer to Question 2: (deletion of gene yfh1, consequence Iron homoeostasis was improved), (deletion of gene yfh1, consequence Soluble/usable mitochondrial iron was increased), (deletion of gene yfh1, consequence Accumulation of insoluble/non-usable iron within mitochondria was prevented), (deletion of gene yfh1, consequence Cytochromes were returned to normal), (deletion of gene yfh1, consequence Haem synthesis was restored), (deletion of gene yfh1, consequence Fe-S cluster enzyme activities were improved), (deletion of gene yfh1, consequence Efficiency of new Fe-S cluster synthesis was markedly increased)\n",
      "Progress: 89.40% completed\n",
      "Response for Row 1789:\n",
      "Answer to Question 2: (deletion of gene RNT1, increased expression of Snf3p-Rgt2p-associated transcription factors), (deletion of gene RNT1, promoter-specific transcription of glucose sensing genes)\n",
      "Progress: 89.45% completed\n",
      "Response for Row 1790:\n",
      "Answer to Question 2: (deletion of gene GPD1, consequence reduced glycerol production), (deletion of gene GDH1, consequence reduced glycerol production), (deletion of gene GPD1 and GDH1, consequence lowest glycerol concentration of 2.31 g/L), (deletion of gene GPD1 and GDH1, consequence 46.4% lower glycerol concentration than wild-type strain), (deletion of gene GPD1 and GDH1, consequence slight improvement in ethanol yield to 0.414 g/g)\n",
      "Progress: 89.50% completed\n",
      "Response for Row 1791:\n",
      "Answer to Question 2: (gene deletion, reduced cell growth), (introduction of E. coli genes, improved higher alcohol productivity)\n",
      "Progress: 89.55% completed\n",
      "Response for Row 1792:\n",
      "Answer to Question 2: (deletion of gene Pdr1p, consequence 1), (overexpression of gene Pdr5p, consequence 1), (overexpression of gene Snq2p, consequence 1)\n",
      "Progress: 89.60% completed\n",
      "Response for Row 1793:\n",
      "Answer to Question 2: (deletion of gene PIL1, consequence 1: endocytic patches carrying Abp1 and Las17 persisted much longer), (deletion of gene PIL1, consequence 2: scission efficiency was greatly affected), (deletion of gene PIL1, consequence 3: frequency of formation of endocytic sites carrying Rvs161- and Rvs167-GFP was affected), (deletion of gene PIL1, consequence 4: mistargeting of synaptojanins Sjl1 and Sjl2 to the cytoplasm), (deletion of gene PIL1, consequence 5: severe motility defect of Abp1-GFP during its internalization in a codeletion with SJL2)\n",
      "Progress: 89.65% completed\n",
      "Response for Row 1794:\n",
      "Answer to Question 2: (deletion of gene Sc-ILV6, reduction of diacetyl concentration in green beer)\n",
      "Progress: 89.70% completed\n",
      "Response for Row 1795:\n",
      "Answer to Question 2: (deletion of gene HSP104, protein degradation), (deletion of gene URA3, protein degradation), (deletion of gene GFP, protein degradation), (expression of gene Flp recombinase, deletion efficiency of 96%), (basal activity of GAL1 promoter, pre-excision frequency of 2%), (spontaneous recombination between FRT sites, pre-excision frequency of 2%)\n",
      "Progress: 89.75% completed\n",
      "Response for Row 1796:\n",
      "Answer to Question 2: (induction of gene GTS1, cell aggregation), (induction of gene GTS1, protein aggregation), (induction of gene GTS1, ROS production), (induction of gene GTS1, apoptosis), (deletion of gene POR1, altered ROS production profile), (deletion of gene YHB1, altered apoptosis profile)\n",
      "Progress: 89.80% completed\n",
      "Response for Row 1797:\n",
      "Answer to Question 2: (deletion of gene efflux pumps, consequence hypersensitivity against several compounds), (deletion of gene transcription factors, consequence hypersensitivity against several compounds), (mutation of gene RME1, consequence hypersensitivity against several compounds)\n",
      "Progress: 89.85% completed\n",
      "Response for Row 1798:\n",
      "Answer to Question 2: (decreased expression of gene frataxin, oxidative damage of mitochondria), (decreased expression of gene frataxin, iron accumulation in mitochondria), (decreased expression of gene frataxin, iron-sulfur cluster defects), (decreased expression of gene frataxin, high sensitivity to oxidative stress), (treatment with idebenone, improvement in neurological functions), (treatment with rapamycin, reduction of ROS production), (treatment with rapamycin, decrease in mitochondrial mass)\n",
      "Progress: 89.90% completed\n",
      "Response for Row 1799:\n",
      "Answer to Question 2: (deletion of gene STB3, partial bypass of growth defects), (deletion of gene DOT6, partial bypass of growth defects), (deletion of gene TOD6, partial bypass of growth defects), (phosphorylation of Stb3 by Sch9, inhibition of Stb3), (phosphorylation of Dot6 by Sch9, inhibition of Dot6), (phosphorylation of Tod6 by Sch9, inhibition of Tod6), (dephosphorylation of Stb3, enabling recruitment of RPD3L), (dephosphorylation of Dot6, enabling recruitment of RPD3L), (dephosphorylation of Tod6, enabling recruitment of RPD3L), (recruitment of RPD3L, repression of Ribi/RP gene promoters), (Sch9 regulation, promotion of ribosome biogenesis), (Sch9 regulation, promotion of ribosomal protein gene expression)\n",
      "Progress: 89.95% completed\n",
      "Response for Row 1800:\n",
      "Answer to Question 2: (upregulation of Pdr5p, diclofenac tolerance), (deletion of PKC pathway components, increased diclofenac toxicity), (incubation with calcofluor white, decreased diclofenac toxicity), (downregulation of ribosome biogenesis genes, effect on diclofenac tolerance), (downregulation of zinc-responsive genes, effect on diclofenac tolerance), (deletion of Zap1p, increased diclofenac toxicity), (addition of zinc chelator 1,10-phenanthroline, increased diclofenac toxicity)\n",
      "Progress: 90.00% completed\n",
      "Response for Row 1801:\n",
      "Answer to Question 2: (deletion of gene GPD1, decreased fermentative ability), (deletion of gene GPD2, decreased fermentative ability), (deletion of gene GPD1, increased pyruvate production), (deletion of gene GPD2, increased pyruvate production), (deletion of gene GPD1, decreased acetate production), (deletion of gene GPD2, decreased acetate production), (deletion of gene GPD1, decreased succinate production), (deletion of gene GPD2, decreased succinate production), (expression of gene SOR1, increased isoamyl alcohol production), (expression of gene srlD, increased isoamyl alcohol production), (expression of gene SOR1, increased 2-phenyl alcohol production), (expression of gene srlD, increased 2-phenyl alcohol production), (deletion of gene GPD1, reduced ethyl acetate concentration), (deletion of gene GPD2, reduced ethyl acetate concentration), (deletion of gene GPD1, increased isobutyric acid), (deletion of gene GPD2, increased isobutyric acid)\n",
      "Progress: 90.05% completed\n",
      "Response for Row 1802:\n",
      "Answer to Question 2: (gene overexpression, improved product yield by 2~4 folds), (gene knockout, improved product yield by 2~4 folds)\n",
      "Progress: 90.10% completed\n",
      "Response for Row 1803:\n",
      "Answer to Question 2: (deletion of gene SHB17, decrease in sedoheptulose-1,7-bisphosphate hydrolysis), (deletion of gene SHB17, disruption in riboneogenesis pathway), (overexpression of gene SHB17, increase in flux through SHB17), (overexpression of gene SHB17, increase in ribose-5-phosphate production)\n",
      "Progress: 90.15% completed\n",
      "Response for Row 1804:\n",
      "Answer to Question 2: (deletion of gene CHD1, abolition of PHO5 gene expression)\n",
      "Progress: 90.20% completed\n",
      "Response for Row 1805:\n",
      "Answer to Question 2: (deletion of gene bug1, reduced efficiency of [PSI+] induction), (deletion of gene bem1, reduced efficiency of [PSI+] induction), (deletion of gene arf1, reduced efficiency of [PSI+] induction), (deletion of gene hog1, reduced efficiency of [PSI+] induction), (deletion of gene las17, inhibited [PSI+] induction and ring formation), (deletion of gene vps5, inhibited [PSI+] induction and ring formation), (deletion of gene sac6, inhibited [PSI+] induction and ring formation)\n",
      "Progress: 90.25% completed\n",
      "Response for Row 1806:\n",
      "Answer to Question 2: (deletion of gene Hub1, consequence cannot use certain non-canonical 5' splice sites), (deletion of gene Hub1, consequence defective in alternative SRC1 splicing)\n",
      "Progress: 90.30% completed\n",
      "Response for Row 1807:\n",
      "Answer to Question 2: (deletion of gene chc1, higher levels of intracellular glutathione), (deletion of gene cst6, higher levels of intracellular glutathione), (deletion of gene ddc1, higher levels of intracellular glutathione), (deletion of gene def1, higher levels of intracellular glutathione), (deletion of gene pep12, higher levels of intracellular glutathione), (deletion of gene rts1, higher levels of intracellular glutathione), (deletion of gene ubp6, higher levels of intracellular glutathione), (deletion of gene yih1, higher levels of intracellular glutathione), (overexpression of gene DEF1, higher production of glutathione), (overexpression of gene CYS4, higher production of glutathione), (overexpression of gene GSH1, higher production of glutathione), (combination of overexpression of GSH1 and deletion of one of the eight genes, multiplier effect on activation of glutathione synthesis), (deletion of def1, higher levels of intracellular methionine), (deletion of pep12, higher levels of intracellular methionine), (deletion of ubp6, higher levels of intracellular methionine), (overexpression of DEF1, higher levels of intracellular methionine), (deletion of def1, higher levels of oxidized glutathione), (deletion of pep12, higher levels of oxidized glutathione), (deletion of ubp6, higher levels of oxidized glutathione), (overexpression of DEF1, higher levels of oxidized glutathione), (overexpression of GSH1, increased glutathione production in Candida utilis), (overexpression of CYS4, increased glutathione production in Candida utilis), (overexpression of DEF1, increased glutathione production in Candida utilis)\n",
      "Progress: 90.35% completed\n",
      "Response for Row 1808:\n",
      "Answer to Question 2: (deletion of gene YFR017C, reduction in ability to accumulate glycogen during aerobic growth), (deletion of gene YFR017C, reduction in ability to accumulate glycogen during fermentation), (expression of gene YFR017C, inhibition of Gdb1p activity), (expression of gene YFR017C, enhancement of glucose storage as glycogen)\n",
      "Progress: 90.40% completed\n",
      "Response for Row 1809:\n",
      "Answer to Question 2: (expression of gene mvaK1, synthesis of β-carotene), (expression of gene mvaK1, complementation of Erg12 function), (expression of carotenogenic genes of Xanthophyllomyces dendrorhous, synthesis of β-carotene), (deletion of gene erg12, requirement for MvaK1 function), (expression of gene mvaK1, high plasmid stability), (expression of gene mvaK1, stable carotenoid production)\n",
      "Progress: 90.45% completed\n",
      "Response for Row 1810:\n",
      "Answer to Question 2: (deletion of gene IRC7, reduced 4MMP production), (overexpression of gene IRC7, increased 4MMP production), (overexpression of gene IRC7, increased 3MH production), (overexpression of gene IRC7, increased 3MHA production)\n",
      "Progress: 90.50% completed\n",
      "Response for Row 1811:\n",
      "Answer to Question 2: ( deletion of gene BAR1, failure to display a distinct sterol-rich domain at the hyphal tip), ( deletion of gene BAR1, non-pathogenic when inoculated onto wheat heads), ( deletion of gene BAR1, severe perturbation in in vitro growth), ( deletion of gene BAR1, incapability of producing perithecia), ( deletion of gene BAR1, production of macroconidia only in the presence of NaCl), ( deletion of gene BAR1, necessity for the production of glucosylceramides), ( deletion of glucosylceramide synthase gene, resistance to heat stable antifungal factor (HSAF))\n",
      "Progress: 90.55% completed\n",
      "Response for Row 1812:\n",
      "Answer to Question 2: (expression of gene Pc22g24830, production of oxalic acid), (deletion of gene Pc22g28430, elimination of oxalate production), (deletion of gene Pc22g28430, improvement of ad-6-APA yields)\n",
      "Progress: 90.60% completed\n",
      "Response for Row 1813:\n",
      "Answer to Question 2: (deletion of gene xxx, impact on higher alcohol production), (deletion of gene yyy, impact on higher alcohol production), (deletion of gene zzz, impact on higher alcohol production)\n",
      "Progress: 90.65% completed\n",
      "Response for Row 1814:\n",
      "Answer to Question 2: (concerted evolution, rate and dynamics of rDNA repeats studied), (mutation in spacer of rDNA unit, monitored fate without selectable marker), (duplication and deletion events, occur frequently), (large deletions, increase homogenization efficiency)\n",
      "Progress: 90.70% completed\n",
      "Response for Row 1815:\n",
      "Answer to Question 2: (deletion of gene YKU70, affects telomere capping), (mutation in gene CDC13, affects telomere capping), (deletion of gene in NMD pathway, suppresses cdc13-1 mutation), (deletion of gene in NMD pathway, enhances yku70Δ mutation), (increased levels of Stn1, affects growth of cells with cdc13-1 mutation), (increased levels of Stn1, affects growth of cells with yku70Δ mutation)\n",
      "Progress: 90.75% completed\n",
      "Response for Row 1816:\n",
      "Answer to Question 2: (deletion of gene RPL12A, slow-growth phenotype), (deletion of gene RPL12B, slow-growth phenotype), (deletion of gene RPL12A, suppression of PHO84 transcription), (deletion of gene RPL12B, suppression of PHO84 transcription), (deletion of gene RPL12A, decrease in phosphate transport activity), (deletion of gene RPL12B, decrease in phosphate transport activity), (deletion of gene RPL12A, decrease in acidic phosphatase activity), (deletion of gene RPL12B, decrease in acidic phosphatase activity), (addition of gene RPL12A, lessening of suppression of PHO84 transcription), (addition of gene RPL12B, lessening of suppression of PHO84 transcription), (addition of gene RPL12A, increase in phosphate transport activity), (addition of gene RPL12B, increase in phosphate transport activity), (addition of gene RPL12A, increase in acidic phosphatase activity), (addition of gene RPL12B, increase in acidic phosphatase activity)\n",
      "Progress: 90.80% completed\n",
      "Response for Row 1817:\n",
      "Answer to Question 2: (deletion of gene MATE2, disappearance of leaf anthocyanin pigmentation), (deletion of gene MATE2, pale flower color), (deletion of gene MATE2, drastic decreases in the levels of various flavonoids), (deletion of gene MATE2, increase in seed proanthocyanidin biosynthesis), (malonylation of flavonoid glucosides, increased affinity for uptake by MATE2), (malonylation of flavonoid glucosides, increased transport efficiency for uptake by MATE2)\n",
      "Progress: 90.85% completed\n",
      "Response for Row 1818:\n",
      "Answer to Question 2: (deletion of gene CnSEC14-1, consequence CnPlb1 secretion is dependent on CnSec14-1p), (deletion of gene CnSEC14-1, consequence hypovirulent in mice), (deletion of gene CnSEC14-1, consequence did not disseminate to the CNS by day 14 post infection), (deletion of gene CnSEC14-1, consequence reduced macrophage expulsion of live cells), (deletion of gene CnSEC14-2, consequence did not abrogate CnPlb1 secretion or virulence), (deletion of gene CnSFH5, consequence did not abrogate CnPlb1 secretion or virulence), (reconstitution of gene CnSEC14-1 with CnSEC14-1, consequence restored CnPlb1 secretion and virulence), (reconstitution of gene CnSEC14-1 with CnSEC14-2, consequence restored CnPlb1 secretion and virulence), (expression of gene CnPlb1, consequence essential for dissemination to the CNS), (expression of gene CnPlb1, consequence essential for vomocytosis)\n",
      "Progress: 90.90% completed\n",
      "Response for Row 1819:\n",
      "Answer to Question 2: (overexpression of gene YCL056c/Pex34p, increased numbers of peroxisomes), (deletion of gene PEX34, reduced peroxisome division), (interaction of Pex34p, Pex11p), (interaction of Pex34p, Pex25p), (interaction of Pex34p, Pex27p)\n",
      "Progress: 90.95% completed\n",
      "Response for Row 1820:\n",
      "Answer to Question 2: (deletion of gene SPT15, enhanced ethanol tolerance), (deletion of gene SPT15, increased ethanol production), (deletion of gene SPT15, improved survival upon ethanol shock), (deletion of gene SPT15, enhanced ethanol yield), (deletion of gene SPT15, increased ethanol productivity)\n",
      "Progress: 91.00% completed\n",
      "Response for Row 1821:\n",
      "Answer to Question 2: (deletion of gene FPG1, consequence foamless strain), (overexpression of gene FPG1, consequence foam hyper-producing strain)\n",
      "Progress: 91.05% completed\n",
      "Response for Row 1822:\n",
      "Answer to Question 2: (deletion of Pho8, increase in internal concentration of NmR), (activation of PHO pathway, increase in release rate of NmR), (activation of PHO pathway, increase in internal concentration of NmR), (deletion of Fun26, change in size of vacuolar and cytosolic NmR pools), (depletion of NaMN, activation of PHO pathway), (activation of PHO pathway, mediation of NMN catabolism), (mediation of NMN catabolism, contribution to NmR salvage), (mediation of NMN catabolism, contribution to phosphate acquisition)\n",
      "Progress: 91.10% completed\n",
      "Response for Row 1823:\n",
      "Answer to Question 2: (induction of gene GTS1, consequence heat tolerance), (co-expression of gene SSA1, consequence inhibition of Gts1p nuclear import), (co-expression of gene SSA2, consequence inhibition of Gts1p nuclear import), (deletion of gene SSA1 and SSA2, consequence enhancement of Gts1p-mediated heat tolerance), (induction of gene GTS1, consequence increased production of Ssa1p and Ssa2p)\n",
      "Progress: 91.15% completed\n",
      "Response for Row 1824:\n",
      "Answer to Question 2: (deletion of gene hacA, consequence UPR-deficient strain), (deletion of gene hacA, consequence reduced virulence in Alternaria brassicicola), (overexpression of gene hacA, consequence constitutively activated UPR strain), (activation of gene HAC1, consequence increased translation of Hac1p), (splicing of HAC1 mRNA, consequence relief of translational block in HAC1 mRNA), (stress-induced splicing of unconventional intron, consequence activation of UPR pathway), (expression of UPR target genes, consequence regulation of folding, yield, and delivery of secretory proteins)\n",
      "Progress: 91.20% completed\n",
      "Response for Row 1825:\n",
      "Answer to Question 2: (deletion of gene RPA49, disappearance of nucleolar structure), (deletion of gene RPA49, decrease in Pol I loading rate per gene), (deletion of gene RPA49, decreased contact between adjacent Pol I complexes), (deletion of gene RPA49, restoration of nucleolar assembly by decreasing ribosomal gene copy number)\n",
      "Progress: 91.25% completed\n",
      "Response for Row 1826:\n",
      "Answer to Question 2: (deletion of gene VMA1, five- to sevenfold elevated riboflavin production), (deletion of gene VMA1, twofold decreased iron cell content), (deletion of gene FRA1, 1.8-2.2-fold more iron accumulation), (deletion of gene FRA1, five- to sevenfold more riboflavin production), (deletion of gene VMA1, inability to grow at 37°C), (deletion of gene FES1, inability to grow at 37°C)\n",
      "Progress: 91.30% completed\n",
      "Response for Row 1827:\n",
      "Answer to Question 2: (SAK1 overexpression, upregulation of glucose-repressed genes), (HAP4 overexpression, upregulation of glucose-repressed genes), (SAK1 overexpression, reduced ethanol formation), (HAP4 overexpression, reduced ethanol formation), (SAK1 overexpression, reduced glycerol formation), (HAP4 overexpression, reduced glycerol formation), (SAK1 overexpression, increased growth rates), (HAP4 overexpression, lesser impact on growth rates), (SAK1 overexpression, increased biomass yields), (HAP4 overexpression, no increase in biomass yields), (SAK1 overexpression, restored growth on ethanol in sdh2 deletion strain), (combined SAK1 and HAP4 overexpression, no restoration of growth on ethanol in sdh2 deletion strain), (HAP4 overexpression alone, no restoration of growth on ethanol in sdh2 deletion strain), (sdh2 deletion, production of succinic acid), (SAK1 and HAP4 overexpression, production of succinic acid), (SAK1 overexpression, alleviation of glucose repression), (SAK1 overexpression, shift in fermentative/oxidative balance)\n",
      "Progress: 91.35% completed\n",
      "Response for Row 1828:\n",
      "Answer to Question 2: (deletion of gene YHR087w, lower levels of yeast Hsp proteins)\n",
      "Progress: 91.40% completed\n",
      "Response for Row 1829:\n",
      "Answer to Question 2: (overexpression of gene EXG1, consequence β-glucosidase activity), (overexpression of gene SPR1, consequence β-glucosidase activity), (overexpression of gene YIR007W, consequence β-glucosidase activity), (deletion of gene EXG1, consequence loss of capability to hydrolyze flavonoid glucosides)\n",
      "Progress: 91.45% completed\n",
      "Response for Row 1830:\n",
      "Answer to Question 2: (expression of gene Asr1, consequence increased ethanol tolerance), (deletion of gene Asr1, consequence hypersensitivity to ethanol)\n",
      "Progress: 91.50% completed\n",
      "Response for Row 1831:\n",
      "Answer to Question 2: (deletion of YCA1, cell death response), (deletion of genes involved in mitochondrial electron transport chain, propolis sensitivity), (deletion of genes involved in vacuolar acidification, propolis sensitivity), (deletion of genes involved in negative regulation of transcription from RNA polymerase II promoter, propolis sensitivity), (deletion of genes involved in regulation of macroautophagy associated with protein targeting to vacuoles, propolis sensitivity), (deletion of genes involved in cellular response to starvation, propolis sensitivity)\n",
      "Progress: 91.55% completed\n",
      "Response for Row 1832:\n",
      "Answer to Question 2: (deletion of gene vps3, enhanced endoglucanase activity), (deletion of gene vps16, enhanced β-glucosidase activity), (deletion of gene rav1, increased endoglucanase activity when combined with vps3 deletion), (deletion of gene vps3, increased endoglucanase activity when combined with rav1 deletion)\n",
      "Progress: 91.60% completed\n",
      "Response for Row 1833:\n",
      "Answer to Question 2: (expression of gene OrdA, HOMST is produced from OMST), (expression of gene OrdA, HOMST is a precursor of AFB(1)), (expression of gene OrdA, HOMST is a precursor of AFG(1)), (expression of gene CypA, involved in conversion of HOMST to AFG(1)), (expression of gene NadA, involved in conversion of HOMST to AFG(1))\n",
      "Progress: 91.65% completed\n",
      "Response for Row 1834:\n",
      "Answer to Question 2: (deletion of gene GPD1, lower glycerol yield), (deletion of gene GPD1, higher ethanol yield), (deletion of gene GPD1, sensitivity to osmotic stress), (deletion of gene GPD1, failure to ferment on 25% glucose), (over-expression of gene TPS1, restored osmotic stress tolerance), (over-expression of gene TPS2, improved osmotic stress tolerance), (over-expression of gene TPS1, further reduced glycerol yield), (over-expression of gene TPS2, further reduced glycerol yield), (expression of gene GAPN, lower glycerol yield), (expression of gene GAPN, higher ethanol yield)\n",
      "Progress: 91.70% completed\n",
      "Response for Row 1835:\n",
      "Answer to Question 2:  (deletion of gene myo1, consequence viable and dividing strains), (deletion of gene myo1, consequence absence of cytokinetic ring), (deletion of gene myo1, consequence activation of PKC1-dependent cell wall integrity pathway), (deletion of gene myo1, consequence 1,301 differentially regulated ribosome-bound mRNAs), (deletion of gene myo1, consequence 204 co-regulated by transcription and translation), (deletion of gene myo1, consequence down and up regulation of ribosomal protein and cell wall biosynthesis mRNAs), (deletion of gene myo1, consequence accumulation of phosphorylated form of eukaryotic translation initiation factor 2 (eIF2α-P)), (deletion of gene myo1, consequence reduction in steady state levels of translation initiation factor eIF4Gp), (deletion of gene gcn2 in myo1Δ, consequence abolished eIF2α-P phosphorylation), (deletion of gene gcn2 in myo1Δ, consequence severe growth defect), (deletion of gene myo1, consequence presence of P-bodies), (deletion of gene myo1, consequence selective co-precipitation of RP mRNAs with eIF2α-P), (deletion of gene myo1, consequence selective regulation of mRNA translation efficiency), (deletion of gene myo1, consequence increase in levels of eIF2α-P), (deletion of gene myo1, consequence genetic interaction between GCN2 and MYO1), (deletion of gene myo1, consequence other signaling pathways important for survival).\n",
      "Progress: 91.75% completed\n",
      "Response for Row 1836:\n",
      "Answer to Question 2: (deletion of gene MER1, delay in expression of Mer1p-responsive genes), (overexpression of gene UME6, induction of MER1 mRNA), (overexpression of gene UME6, induction of Mer1p substrate pre-mRNAs), (deletion of gene MER1, reduction in splicing of Mer1p substrate pre-mRNAs), (overexpression of gene MER1, activation of Ndt80p), (deletion of gene NAM8, disruption of splicing activation by Mer1p)\n",
      "Progress: 91.80% completed\n",
      "Response for Row 1837:\n",
      "Answer to Question 2: (deletion of gene CRK1, consequence 1: increased mating efficiency), (deletion of gene CRK1, consequence 2: earlier production of dikaryotic filaments), (deletion of gene CRK1, consequence 3: earlier production of basidia), (deletion of gene CRK1, consequence 4: earlier production of basidiospores), (overexpression of gene CRK1, consequence 1: inhibited mating), (overexpression of gene CRK1, consequence 2: defective monokaryotic fruiting)\n",
      "Progress: 91.85% completed\n",
      "Response for Row 1838:\n",
      "Answer to Question 2: (deletion of growth signaling pathways, reduced superoxide anions), (deletion of growth signaling pathways, enhanced longevity), (deletion of growth signaling pathways, more efficient G0/G1 arrest), (downregulation of growth signaling pathways, reduced superoxide anions), (caloric restriction, elevated levels of H₂O₂), (caloric restriction, activates superoxide dismutases), (high glucose, increased superoxide anions), (mutations activating growth signaling, increased superoxide anions), (high glucose, activates DNA damage responses), (high glucose, preferentially kills stationary phase cells failing to arrest growth in G0/G1)\n",
      "Progress: 91.90% completed\n",
      "Response for Row 1839:\n",
      "Answer to Question 2: (deletion of gene MGA2, elevated relative amounts of squalene), (deletion of gene MGA2, reduced basal expression of ERG1), (deletion of gene SPT23, no impact on squalene levels), (deletion of gene MGA2, no effect on response to miconazole treatment), (deletion of gene MGA2, reduced ERG1 expression comparable to mga2Δ/spt23Δ strain)\n",
      "Progress: 91.95% completed\n",
      "Response for Row 1840:\n",
      "Answer to Question 2: (deletion of gene UME6, failure to germinate), (deletion of gene UME6, reduced germination efficiency), (deletion of gene UME6, inviable cells in microcolonies), (deletion of gene UME6, transient impact on cell viability)\n",
      "Progress: 92.00% completed\n",
      "Response for Row 1841:\n",
      "Answer to Question 2: (deletion of gene OLE1, reduced growth without unsaturated fatty acids), (deletion of gene OLE1, reduced production of unsaturated fatty acids), (deletion of gene OLE1, impaired pseudohyphal formation), (deletion of gene OLE1, increased sensitivity to salts), (deletion of gene OLE1, increased sensitivity to SDS), (deletion of gene OLE1, increased sensitivity to H2O2), (deletion of gene OLE1, increased sensitivity to human serum), (deletion of gene OLE1, increased sensitivity to fetal bovine serum), (deletion of gene OLE1, enhanced killing by macrophages), (deletion of gene OLE1, increased production of nitric oxide by macrophages), (deletion of gene OLE1, reduced fungal virulence in murine infection)\n",
      "Progress: 92.05% completed\n",
      "Response for Row 1842:\n",
      "Answer to Question 2: (deletion of gene Ste50, consequence 1), (deletion of gene Ste50, consequence 2), (deletion of gene Ste50, consequence 3), (deletion of gene Ste50, consequence 4), (deletion of gene CRG1 and Ste50, consequence 5)\n",
      "Progress: 92.10% completed\n",
      "Response for Row 1843:\n",
      "Answer to Question 2: (deletion of gene xxx, reduced survival in drying process), (activation of over 450 genes, increased survival in drying process), (down-regulation of 330 genes, increased survival in drying process), (overexpression of autophagy processes, clearance of protein aggregates/damaged organelles), (overexpression of autophagy processes, recycling of nutrients), (activation of functional carbon source sensing networks, increased survival of desiccation), (activation of PKA, Tor and Snf1 protein kinase complexes, increased survival of desiccation), (changes in nitrogen metabolism, coping with drying process), (activation of environmental stress response, repression of ribosome production), (activation of environmental stress response, induction of genes for coping with oxidative stress), (activation of environmental stress response, induction of genes for coping with osmotic stress)\n",
      "Progress: 92.15% completed\n",
      "Response for Row 1844:\n",
      "Answer to Question 2: (deletion of gene iron accumulation, consequence drop in energy production), (deletion of gene iron accumulation, consequence decrease in activity of enzymes involved in protein protection and regeneration), (overexpression of oxidative stress, consequence protein carbonylation), (overexpression of oxidative stress, consequence damage of glycolytic enzymes), (oxidation of peroxiredoxin (TSA1), consequence impact on enzymatic activities), (oxidation of thioredoxin II (TRX2), consequence impact on enzymatic activities)\n",
      "Progress: 92.20% completed\n",
      "Response for Row 1845:\n",
      "Answer to Question 2: (deletion of gene mid2, transcriptional response to Congo Red reduced), (deletion of gene wsc1, transcriptional response to Caspofungin reduced), (deletion of gene mid2, little effect on transcriptional response to Zymolyase), (deletion of gene wsc1, little effect on transcriptional response to Zymolyase), (activation of gene sho1, transcriptional response to Zymolyase increased)\n",
      "Progress: 92.25% completed\n",
      "Response for Row 1846:\n",
      "Answer to Question 2: (deletion of gene xxx, synthetic lethal with gene yyy), (deletion of gene xxx, synthetic growth defect with gene yyy), (deletion of essential gene xxx, buffered by gene yyy), (deletion of gene xxx, within pathway interaction with gene yyy), (deletion of gene xxx, suppressive interaction with gene yyy)\n",
      "Progress: 92.30% completed\n",
      "Response for Row 1847:\n",
      "Answer to Question 2: (deletion of gene MIG1, reduction in acetate production), (deletion of gene MIG1, decrease in ethanol production), (deletion of gene MIG2, increase in pyruvate concentration), (deletion of gene MIG2, increase in glycerol concentration)\n",
      "Progress: 92.35% completed\n",
      "Response for Row 1848:\n",
      "Answer to Question 2: (deletion of gene GAP1, consequence formation of extrachromosomal DNA circles containing GAP1), (deletion of gene GAP1, consequence production of a single hybrid LTR at the GAP1 chromosomal locus), (overexpression of gene GAP1, consequence increased frequency of GAP1(circle) generation under nitrogen-limiting conditions), (deletion of gene GAP1, consequence selection under various non-amino acid nitrogen limitations including ammonium, urea, and allantoin limitation)\n",
      "Progress: 92.40% completed\n",
      "Response for Row 1849:\n",
      "Answer to Question 2: (deletion of gene UBI4, increased H(2)S productivity), (deletion of gene SKP2, increased H(2)S productivity), (deletion of gene SKP2, increased SO(2) productivity), (deletion of gene SKP2, Met14p more stable), (deletion of gene SKP2, slower growth under nutrient-limited conditions), (deletion of gene SKP2, lower concentration of intracellular cysteine), (deletion of gene SKP2, Str2p more stable), (deletion of gene SKP2, Str3p more stable), (deletion of gene SKP2, Str4p more stable), (overexpression of gene STR1, severe growth defect in skp2 mutant), (overexpression of gene STR4, severe growth defect in skp2 mutant)\n",
      "Progress: 92.45% completed\n",
      "Response for Row 1850:\n",
      "Answer to Question 2: (deletion of gene YDL080C, no noticeable effect on decreasing the production of higher alcohols), (deletion of gene YDL080C, increase in ethanol production)\n",
      "Progress: 92.50% completed\n",
      "Response for Row 1851:\n",
      "Answer to Question 2: (deletion of gene FgStuA, reduced pathogenicity on wheat heads), (deletion of gene FgStuA, impaired spore production), (deletion of gene FgStuA, lack of perithecia and sexual ascospores), (deletion of gene FgStuA, lack of conidiophores and phialides), (deletion of gene FgStuA, delayed production of aberrant macroconidia), (deletion of gene FgStuA, reduced production of secondary metabolites), (deletion of gene FgStuA, differential expression of thousands of genes during asexual sporulation and infection of wheat heads), (deletion of gene FgStuA, differential expression of genes under conditions inducing secondary metabolites)\n",
      "Progress: 92.55% completed\n",
      "Response for Row 1852:\n",
      "Answer to Question 2: (deletion of gene SDH1, interruption of TCA cycle after succinate), (deletion of gene SDH2, interruption of TCA cycle after succinate), (deletion of gene IDH1, interruption of TCA cycle after isocitrate), (deletion of gene IDP1, interruption of TCA cycle after isocitrate), (deletion of genes SDH1, SDH2, IDH1, IDP1, production of succinic acid at titer 3.62 g L(-1)), (deletion of genes SDH1, SDH2, IDH1, IDP1, succinic acid yield of 0.11 mol (mol glucose)(-1)), (deletion of genes SDH1, SDH2, IDH1, IDP1, succinic acid not accumulated intracellularly)\n",
      "Progress: 92.60% completed\n",
      "Response for Row 1853:\n",
      "Answer to Question 2: (deletion of gene FPS1, reduced glycerol production), (deletion of gene GPD2, reduced glycerol production), (overexpression of gene pGcyaDak, increased ethanol production), (overexpression of gene pGup1Cas, provided osmotic protection), (deletion of gene FPS1, increased sensitivity to osmotic stress), (deletion of gene GPD2, increased sensitivity to osmotic stress)\n",
      "Progress: 92.65% completed\n",
      "Response for Row 1854:\n",
      "Answer to Question 2: (deletion of individual genes, change in sporulation efficiency), (deletion of individual genes, discovery of robustness and hysteresis in the yeast sporulation network), (double gene deletion, patterns of sporulation efficiency change)\n",
      "Progress: 92.70% completed\n",
      "Response for Row 1855:\n",
      "Answer to Question 2: (deletion of gene GPD2, decrease in glycerol production), (deletion of gene GPD2, increase in ethanol yield), (expression of gene GAPN, decrease in glycerol production), (expression of gene GAPN, increase in ethanol yield), (expression of gene GAPDH, decrease in glycerol production), (expression of gene GAPDH, increase in ethanol yield), (deletion of gene GPD2, decrease in maximum specific growth rates), (expression of gene GAPN, increase in maximum specific growth rates), (expression of gene GAPDH, increase in maximum specific growth rates)\n",
      "Progress: 92.75% completed\n",
      "Response for Row 1856:\n",
      "Answer to Question 2: (deletion of gene HMT1, deregulates the recruitment of U1 snRNP), (deletion of gene HMT1, reduces binding of splicing factors to intron-containing genes), (deletion of gene HMT1, increases splicing efficiency of developmentally regulated mRNAs), (deletion of gene HMT1, increases association between Snp1 and Npl3)\n",
      "Progress: 92.80% completed\n",
      "Response for Row 1857:\n",
      "Answer to Question 2: (deletion of gene Cdc13, consequence 1), (deletion of gene Sgs1, consequence 1), (deletion of gene Exo1, consequence 1), (deletion of gene Rad9, consequence 1), (deletion of gene SGS1, consequence 2), (deletion of gene EXO1, consequence 2), (deletion of gene RAD9, consequence 2), (deletion of gene CDC13, consequence 3)\n",
      "Progress: 92.85% completed\n",
      "Response for Row 1858:\n",
      "Answer to Question 2:  (deletion of gene ALS3, consequence 1), (deletion of gene ALS3, consequence 2), (expression of gene Als3p, consequence 3), (deletion of gene sspA sspB, consequence 4), (expression of gene SspB, consequence 5)\n",
      "Progress: 92.90% completed\n",
      "Response for Row 1859:\n",
      "Answer to Question 2: (deletion of gene sod1, increased sensitivity to salicylaldehyde), (deletion of gene sod2, increased sensitivity to salicylaldehyde), (deletion of gene glr1, increased sensitivity to salicylaldehyde), (deletion of gene vph2, increased sensitivity to salicylaldehyde)\n",
      "Progress: 92.95% completed\n",
      "Response for Row 1860:\n",
      "Answer to Question 2: (deletion of gene stb5, consequence auxotrophy for methionine), (deletion of gene stb5, consequence auxotrophy for pyrimidine nucleotides), (deletion of gene stb5, consequence impaired aerobic NADPH status), (deletion of gene stb5, consequence reduced acetate level), (deletion of gene stb5, consequence increased α-ketoglutarate level), (deletion of gene stb5, consequence reduced flux through the pentose phosphate pathway), (overexpression of gene stb5, consequence growth defect on glucose), (overexpression of gene stb5, consequence excess NADPH in the oxidative pentose phosphate pathway)\n",
      "Progress: 93.00% completed\n",
      "Response for Row 1861:\n",
      "Answer to Question 2: (deletion of gene IXR1, increase in cell resistance to UV light), (deletion of gene IXR1, increase in cell resistance to gamma-rays), (deletion of gene IXR1, increase in cell resistance to MMS), (deletion of gene IXR1, increase in spontaneous mutagenesis), (deletion of gene IXR1, decrease in UV-induced mutations), (mutation of gene HMO1, increase in sensitivity to cisplatin), (mutation of gene HMO1, no effect on sensitivity to UV light), (mutation of gene HMO1, increase in spontaneous mutagenesis), (mutation of gene HMO1, increase in UV-induced mutagenesis), (double mutation hmo1 ixr1, additive interaction in lethal effect of cisplatin), (double mutation hmo1 ixr1, additive interaction in lethal effect of UV light), (double mutation hmo1 ixr1, additive interaction in UV-induced mutagenesis), (mutation of gene IXR1, increase in spontaneous mutagenesis mediated by replication errors), (mutation of gene HMO1, increase in repair mutagenesis), (combined activity of Hmo1 and Ixr1 proteins, efficient correction of repair errors), (combined activity of Hmo1 and Ixr1 proteins, efficient correction of replication errors)\n",
      "Progress: 93.05% completed\n",
      "Response for Row 1862:\n",
      "Answer to Question 2: (deletion of gene BCS1, arrest of bc(1) assembly), (deletion of gene BCS1, formation of functionally inactive bc(1) core structure), (expression of gene BCS1, rescue of bc(1) core structure), (expression of gene BCS1, formation of functional homodimeric bc(1) complex), (expression of gene BCS1, conversion into supercomplexes with cytochrome c oxidase complex), (mutation R81C in gene BCS1, restoration of bc(1) complex and supercomplexes), (mutation R45C in human ortholog BCS1L, severe bc(1) complex deficiency), (mutation K192P in gene BCS1, inability to recover bc(1) core structure), (mutation F401I in gene BCS1, inability to recover bc(1) core structure)\n",
      "Progress: 93.10% completed\n",
      "Response for Row 1863:\n",
      "Answer to Question 2: (introduction of gene xyd1, D-xylonate production), (introduction of gene xyd1, D-xylonate export to supernatant), (deletion of gene GRE3, reduction in xylitol production), (deletion of gene GRE3, increase in D-xylonate yield from D-xylose), (deletion of gene GRE3, reduction in D-xylose uptake), (co-expression of gene udhA with xyd1, enhancement in NADPH consumption), (co-expression of gene gapB with xyd1, enhancement in NADPH consumption), (co-expression of gene GDH2 with xyd1, enhancement in NADPH consumption)\n",
      "Progress: 93.15% completed\n",
      "Response for Row 1864:\n",
      "Answer to Question 2: (deletion of gene GAS1, growth defects), (overexpression of gene GAS3, reduced growth rate), (overexpression of gene GAS3, hyperaccumulation of Gas3p in membranes), (overexpression of gene GAS3, hyperaccumulation of Gas3p in cell wall), (overexpression of gene GAS3 in gas1Delta mutant, exacerbated defects), (mutation of gene GAS3 to Gas3p-E283Q, deleterious to gas1Delta cells)\n",
      "Progress: 93.20% completed\n",
      "Response for Row 1865:\n",
      "Answer to Question 2: (overexpression of carotenogenic genes, increased carotenoid levels), (overexpression of carotenogenic genes, induction of genes involved in pleiotropic drug resistance), (deletion of pdr10, decreased transformation efficiency), (deletion of pdr10, decreased growth rates), (deletion of pdr10, lower carotenoid production levels)\n",
      "Progress: 93.25% completed\n",
      "Response for Row 1866:\n",
      "Answer to Question 2: (deletion of gene TPA1, decrease of translation termination efficacy), (deletion of gene TPA1, increase in mRNAs half-lives), (deletion of gene TPA1, longer mRNA poly(A) tails), (integrity of Tpa1 active site, essential for correct translation termination), (presence of Yor051c/Ett1, essential for correct translation termination), (Tpa1, represses expression of genes regulated by Hap1), (Hap1, regulates genes essential for cell response to hypoxia)\n",
      "Progress: 93.30% completed\n",
      "Response for Row 1867:\n",
      "Answer to Question 2: (deletion of gene CLB1, consequence minor effects on mitosis), (deletion of gene CLB3, consequence minor effects on mitosis), (deletion of gene CLB4, consequence minor effects on mitosis), (overexpression of gene CLB2, consequence sufficient for essential CDK functions in mitosis), (modification of gene CLB2 with Myc12 epitope, consequence loss of spindle assembly function), (modification of gene CLB2 with Myc12 epitope, consequence no defects in late mitotic functions), (modification of gene CLB2 with Myc12 epitope, consequence induction of switch from polarized to isotropic growth similar to endogenous Clb2)\n",
      "Progress: 93.35% completed\n",
      "Response for Row 1868:\n",
      "Answer to Question 2: (deletion of gene PYC, inability to grow on glucose without C(4) compounds supplementation), (overexpression of gene PCK1, no growth on glucose as sole carbon source), (mutation in gene PYK, enables growth on glucose using PEPCK as sole anaplerotic enzyme), (reaction catalyzed by PEPCK, production of ATP), (original route through pyruvate kinase and pyruvate carboxylase, ATP neutral)\n",
      "Progress: 93.40% completed\n",
      "Response for Row 1869:\n",
      "Answer to Question 2: (expression of gene HMG2, increase in squalene pool), (expression of gene HMG2, increase in lanosterol pool), (deletion of gene ERG6, elevated transfer of surplus squalene into C27 sterols)\n",
      "Progress: 93.45% completed\n",
      "Response for Row 1870:\n",
      "Answer to Question 2: (deletion of gene tesB, decreased 3-hydroxybutyric acid), (deletion of gene tesB, decreased 3-hydroxyhexanoic acid), (deletion of gene tesB, decreased 3-hydroxyoctanoic acid), (deletion of gene tesB, decreased hexanoic acid), (replacement of gene tesB with PTE1, increased proportion of 3-hydroxybutyric acid), (replacement of gene tesB with tesB, increased proportion of 3-hydroxyhexanoic acid), (replacement of gene tesB with tesB, increased proportion of 3-hydroxyoctanoic acid)\n",
      "Progress: 93.50% completed\n",
      "Response for Row 1871:\n",
      "Answer to Question 2: (deletion of gene prb1, increase of MGF yield), (expression of gene MGF, low yield due to rapid degradation), (deletion of gene prb1, fivefold increase of MGF yield)\n",
      "Progress: 93.55% completed\n",
      "Response for Row 1872:\n",
      "Answer to Question 2: (deletion of genes affecting mitochondrial function, increased sensitivity to BiQs), (deletion of genes affecting cellular respiration, increased sensitivity to BiQs), (over-expression of NDE1 gene, increased ROS production), (over-expression of NDE1 gene, increased sensitivity of wild type yeast to BiQ)\n",
      "Progress: 93.60% completed\n",
      "Response for Row 1873:\n",
      "Answer to Question 2: (deletion of gene multidrug resistance genes, consequence multidrug resistance), (deletion of gene multidrug resistance genes, consequence buffers against phenotypic variation), (deletion of gene Rcy1, consequence boosts chemical resistance), (deletion of gene Wss1, consequence boosts chemical resistance), (deletion of gene Ubp16, consequence boosts chemical resistance), (deletion of gene Irs4, consequence boosts peroxisome recycling), (deletion of gene NPR1, consequence boosts phosphorylation cascades), (deletion of gene MCK1, consequence boosts phosphorylation cascades), (deletion of gene HOG, consequence boosts phosphorylation cascades), (overexpression of gene mitochondrial protein synthesis sub-network, consequence explores alternative survival strategies), (overexpression of gene acetyltransferases, consequence supports secondary metabolism), (overexpression of gene NTN hydrolase, consequence supports secondary metabolism)\n",
      "Progress: 93.65% completed\n",
      "Response for Row 1874:\n",
      "Answer to Question 2: (deletion of gene aclA, growth is greatly diminished on glucose), (deletion of gene aclA, growth is greatly diminished on proline), (deletion of gene aclB, growth is greatly diminished on glucose), (deletion of gene aclB, growth is greatly diminished on proline), (deletion of gene aclA, growth is not affected on acetate), (deletion of gene aclA, growth is not affected on ethanol), (deletion of gene aclB, growth is not affected on acetate), (deletion of gene aclB, growth is not affected on ethanol), (deletion of gene aclA, addition of acetate restores growth on glucose), (deletion of gene aclB, addition of acetate restores growth on glucose), (deletion of gene aclA, addition of acetate restores growth on proline), (deletion of gene aclB, addition of acetate restores growth on proline), (deletion of gene aclA, transcription of aclA and aclB is repressed by growth on acetate), (deletion of gene aclB, transcription of aclA and aclB is repressed by growth on acetate), (deletion of gene aclA, production of asexual spores is greatly reduced), (deletion of gene aclB, production of asexual spores is greatly reduced), (deletion of gene aclA, complete absence of sexual development), (deletion of gene aclB, complete absence of sexual development)\n",
      "Progress: 93.70% completed\n",
      "Response for Row 1875:\n",
      "Answer to Question 2: (deletion of gene p66(Shc), extends lifespan), (deletion of gene p66(Shc), displays protective effects in cardiovascular disease), (activation of gene SIRT1, deacetylates p53), (activation of gene SIRT1, deacetylates endothelial nitric oxide synthase), (activation of gene SIRT1, deacetylates forkhead box protein)\n",
      "Progress: 93.75% completed\n",
      "Response for Row 1876:\n",
      "Answer to Question 2: (deletion of gene involved in cell wall integrity/structure, increase in sensitivity to Echinacea), (deletion of gene involved in cell wall integrity/structure, increase in sonication-associated cell death), (deletion of gene involved in cell wall integrity/structure, increase in cell wall damage)\n",
      "Progress: 93.80% completed\n",
      "Response for Row 1877:\n",
      "Answer to Question 2: (deletion of TOR, consequence degradation of Rrn3p), (deletion of TOR, consequence reduction in initiation competent Pol I-Rrn3p complexes), (overexpression of non-degradable Rrn3p, consequence increase in Pol I-Rrn3p complexes), (adjustment of Rrn3p levels, consequence influence on rRNA gene transcription)\n",
      "Progress: 93.85% completed\n",
      "Response for Row 1878:\n",
      "Answer to Question 2: (overexpression of gene Stb3, slow growth phenotype), (overexpression of gene Stb3, reduced expression of non-RP genes), (overexpression of gene Stb3, drop in the rate of amino acid incorporation), (loss of gene Hos2, reversal of Stb3 overexpression slow growth phenotype), (overexpression of gene SCH9, reversal of Stb3 overexpression slow growth phenotype), (deletion of gene PPH22, reversal of Stb3 overexpression slow growth phenotype), (inhibition of Tor signaling, increased severity of Stb3 overexpression phenotype), (inhibition of Tor signaling, restriction of Stb3 to the nucleus)\n",
      "Progress: 93.90% completed\n",
      "Response for Row 1879:\n",
      "Answer to Question 2: (deletion of Sir2, increase in NER efficiency), (presence of Sir2, inability to stimulate histone H4 acetylation), (presence of Sir2, inability to stimulate histone H3 acetylation), (UV radiation, stimulates histones H3 acetylation), (UV radiation, stimulates histones H4 acetylation)\n",
      "Progress: 93.95% completed\n",
      "Response for Row 1880:\n",
      "Answer to Question 2: (deletion of TorC1, consequence NCR-sensitive transcription is derepressed), (overexpression of Sit4, consequence nuclear Gln3 localization), (overexpression of Pph21/22-Tpd3-Cdc55/Rts1, consequence nuclear Gat1 localization), (deletion of TorC1, consequence nuclear Gat1 localization), (overexpression of Sit4, consequence weak response to Rap treatment), (overexpression of Pph21/22-Tpd3-Cdc55/Rts1, consequence strong response to Rap treatment), (overexpression of Msx, consequence strong response of Gln3 localization), (overexpression of Msx, consequence immunity of Gat1 localization)\n",
      "Progress: 94.00% completed\n",
      "Response for Row 1881:\n",
      "Answer to Question 2: (deletion of gene sed1, consequence 1: higher BGL-activity), (deletion of gene sed1, consequence 2: maintains wild type growth), (deletion of gene sed1, consequence 3: influences the activity of BGL on the cell surface), (deletion of gene tos6, consequence 1: higher BGL-activity), (deletion of gene tos6, consequence 2: maintains wild type growth), (deletion of SED1 ORF, consequence 1: enhances BGL activity on the cell surface), (deletion of SED1 promoter, consequence 1: increases further BGL activity on the cell surface), (homozygous deletion of SED1 gene and promoter, consequence 1: most efficient display of BGL), (homozygous deletion of SED1 gene and promoter, consequence 2: 1.6-fold increase of BGL-activity compared to wild type)\n",
      "Progress: 94.05% completed\n",
      "Response for Row 1882:\n",
      "Answer to Question 2: (deletion of SEC59 ortholog, supports normal growth), (deletion of SEC59 ortholog, supports N-glycosylation), (deletion of SEC59 ortholog, supports GPI anchor synthesis), (expression of Gl-UPPS, makes dehydrodolichol), (expression of Gl-UPPS, supports normal growth), (expression of Gl-UPPS, supports N-glycosylation), (expression of Gl-UPPS, supports GPI anchor synthesis)\n",
      "Progress: 94.10% completed\n",
      "Response for Row 1883:\n",
      "Answer to Question 2: (deletion of gene MSS116, affects splicing of mitochondrial catalytic introns), (expression of mutated MSS116, affects ATP hydrolysis efficiency), (expression of mutated MSS116, affects helix unwinding efficiency), (inefficient ATP hydrolysis by MSS116, impairs splicing function), (efficient helix unwinding by MSS116, does not ensure splicing function)\n",
      "Progress: 94.15% completed\n",
      "Response for Row 1884:\n",
      "Answer to Question 2: (deletion of gene Gre2p, lacks activity towards 2,5-hexanedione reduction), (expression of gene Gre2p, affords (2S,5S)-hexanediol with >99% conversion yield and >99.9% de and ee), (expression of gene Gre2p, results in high volumetric productivity of 70 g L(-1) d(-1) (2S,5S)-hexanediol)\n",
      "Progress: 94.20% completed\n",
      "Response for Row 1885:\n",
      "Answer to Question 2: (deletion of bacterial genome, high background of 5-FOA resistant clones), (expression of I-SceI, generation of double-strand break), (generation of double-strand break, promotion of intra-molecular homologous recombination), (intra-molecular homologous recombination, excision of CORE), (excision of CORE, seamless modification of bacterial genome)\n",
      "Progress: 94.25% completed\n",
      "Response for Row 1886:\n",
      "Answer to Question 2: (deletion of gene ANP1, increased susceptibility to high-glucose stress), (deletion of gene GCN4, increased susceptibility to high-glucose stress), (deletion of gene GCR1, increased susceptibility to high-glucose stress), (deletion of gene involved in vacuolar function, increased susceptibility to high-glucose stress), (deletion of gene involved in cell wall biogenesis, increased susceptibility to high-glucose stress), (deletion of gene involved in regulation of transcription, increased susceptibility to high-glucose stress)\n",
      "Progress: 94.30% completed\n",
      "Response for Row 1887:\n",
      "Answer to Question 2: (deletion of genes involved in mitochondrial functions, hypersensitivity to CTBT), (deletion of genes involved in DNA repair, hypersensitivity to CTBT), (deletion of genes involved in transcription, hypersensitivity to CTBT), (deletion of genes involved in chromatin remodeling, hypersensitivity to CTBT), (deletion of genes involved in oxidative stress response, hypersensitivity to CTBT), (CTBT treatment, rapid induction of oxidant and stress response defense genes), (CTBT treatment, Yap1 accumulation in the nucleus), (CTBT treatment, acute oxidative stress), (CTBT treatment, superoxide production associated with mitochondria)\n",
      "Progress: 94.35% completed\n",
      "Response for Row 1888:\n",
      "Answer to Question 2: (deletion of gene subset, optimization of downstream gene activity), (deletion of gene subset, optimization of metabolite activity)\n",
      "Progress: 94.40% completed\n",
      "Response for Row 1889:\n",
      "Answer to Question 2: (deletion of gene Cat8p, consequence 1), (deletion of gene Cat8p, reduced respiration), (deletion of gene Rtg-proteins, consequence 1), (deletion of gene Rtg-proteins, reduced respiration under partially respiratory conditions), (deletion of gene Hap-complex, consequence 1), (deletion of gene Hap-complex, reduced respiration under partially respiratory conditions), (deletion of gene Hap-complex, reduced respiration under fully respiratory conditions)\n",
      "Progress: 94.45% completed\n",
      "Response for Row 1890:\n",
      "Answer to Question 2: (deletion of gene eIF4G1, consequence impaired growth), (deletion of gene eIF4G1, consequence reduced global translation initiation rates), (deletion of gene eIF4G1, consequence lower levels of total eIF4G), (deletion of gene eIF4G1, consequence preferential effect on mRNAs with longer poly(A) tails)\n",
      "Progress: 94.50% completed\n",
      "Response for Row 1891:\n",
      "Answer to Question 2: (deletion of gene ERG4, impaired shmoo formation), (deletion of gene ERG4, low cell fusion efficiency), (supplementation with ergosterol, partially suppressed shmooing defect), (removal of ergosta-5,7,22,24(28)-tetraenol, restored shmooing to wild-type levels), (removal of ergosta-5,7,22,24(28)-tetraenol, restored cell fusion to wild-type levels)\n",
      "Progress: 94.55% completed\n",
      "Response for Row 1892:\n",
      "Answer to Question 2: (deletion of gene cafA, growth defect at low CO2), (deletion of gene cafB, growth defect at low CO2), (deletion of gene canB, growth defect at low CO2), (overexpression of gene cafB, rescue of high CO2-requiring phenotype), (overexpression of gene canA, rescue of high CO2-requiring phenotype), (overexpression of gene canB, rescue of high CO2-requiring phenotype), (deletion of gene cafA, no effect on virulence), (deletion of gene cafB, no effect on virulence), (deletion of gene cafC, no effect on virulence), (deletion of gene cafD, no effect on virulence)\n",
      "Progress: 94.60% completed\n",
      "Response for Row 1893:\n",
      "Answer to Question 2: (deletion of gene GNA1, requirement for GlcNAc supplement), (expression of gene NGT1, rescue of gna1Delta strain), (expression of gene NAGK, rescue of gna1Delta strain), (metabolic replacement of GlcNAc by GlcNAz, incorporation into cell-surface glycans), (metabolic replacement of GlcNAc by GlcNAz, incorporation into secreted glycoproteins), (metabolic replacement of GlcNAc by GlcNAl, incorporation into cell-surface glycans), (metabolic replacement of GlcNAc by GlcNAl, incorporation into secreted glycoproteins)\n",
      "Progress: 94.65% completed\n",
      "Response for Row 1894:\n",
      "Answer to Question 2: (deletion of gene Aac2p, consequence growth defect), (deletion of gene Aac2p, consequence dependence on SAL1 gene product), (deletion of gene SAL1, consequence impact on mitochondrial ATP-related reactions), (expression of gene SAL1, consequence rescue of growth in Deltaaac2 mutants), (expression of gene SAL1, consequence maintenance of intramitochondrial ATP pool)\n",
      "Progress: 94.70% completed\n",
      "Response for Row 1895:\n",
      "Answer to Question 2: (deletion of gene rrm3, increased HTA2-HTB2 amplification), (deletion of gene dpb3, increased HTA2-HTB2 amplification), (deletion of gene dpb4, increased HTA2-HTB2 amplification), (deletion of gene clb5, increased HTA2-HTB2 amplification), (deletion of gene hht2-hhf2, increased HTA2-HTB2 amplification), (reduced histone H3-H4 levels, increased replication fork pauses), (impaired replication forks, reduced histone levels), (introduction of DNA replication stress by hydroxyurea, induced HTA2-HTB2 amplification)\n",
      "Progress: 94.75% completed\n",
      "Response for Row 1896:\n",
      "Answer to Question 2: (deletion of gene ZbGAS1, increased protein secretion), (deletion of gene ZbGAS1, higher release of Candida rugosa lipase CRL1), (deletion of gene ZbGAS1, higher release of Yarrowia lipolytica protease XPR2), (deletion of gene ZbGAS1, no significant effect on human IL-1beta secretion)\n",
      "Progress: 94.80% completed\n",
      "Response for Row 1897:\n",
      "Answer to Question 2: (knock out of cytochrome respiratory pathway, induction of alternative oxidase), (knock out of cytochrome respiratory pathway, reduction in free radical production), (disruption of PaNdk1, severe somatic and sexual defects), (disruption of PaNdk1, hypersensitivity to hydrogen peroxide), (disruption of PaNdk1, hypersensitivity to paraquat), (disruption of PaNdk1, increase of mtDNA stability), (disruption of PaNdk1, increase of lifespan)\n",
      "Progress: 94.85% completed\n",
      "Response for Row 1898:\n",
      "Answer to Question 2: (EXO5 deletion, amplifying either the ori3 or ori5 region of the mitochondrial genome), (EXO5 deletion, yield petites), (EXO5 deletion, mitochondrial genome instability), (EXO5 deletion, continuous rearrangement of mitochondrial genome)\n",
      "Progress: 94.90% completed\n",
      "Response for Row 1899:\n",
      "Answer to Question 2: (deletion of gene sco1, decreased Ty1 retrotransposition), (overproduction of reactive oxygen species, increased Ty1 retrotransposition), (treatment with methylmethane sulphonate, increased superoxide levels), (increase of superoxide anions, induction of Ty1 mobility), (disruption of gene yap1, increased intracellular levels of hydrogen peroxide), (exogenous addition of hydrogen peroxide, higher frequencies of retrotransposition)\n",
      "Progress: 94.95% completed\n",
      "Response for Row 1900:\n",
      "Answer to Question 2: (deletion of gene SPT6, severe impairment of growth), (deletion of gene SPT6, impaired capacity to undergo filamentous growth), (overexpression of gene SPT6, partial complementation in S. cerevisiae spt6 mutant)\n",
      "Progress: 95.00% completed\n",
      "Response for Row 1901:\n",
      "Answer to Question 2: (deletion of gene YCA1, increased viability), (deletion of gene YCA1, no improvement in overall strain performance), (deletion of gene YCA1, higher energy demand for maintenance)\n",
      "Progress: 95.05% completed\n",
      "Response for Row 1902:\n",
      "Answer to Question 2: (deletion of gene XRS2, consequence BER deficiency), (deletion of gene XRS2, consequence reduced gap-filling activity), (deletion of gene XRS2, consequence affected long-patch BER), (deletion of gene XRS2, consequence affected short-patch BER), (deletion of gene XRS2, consequence involvement downstream of AP-lyases Ntg1 and Ntg2), (deletion of gene XRS2, consequence missing effect of aphidicolin on BER efficiency)\n",
      "Progress: 95.10% completed\n",
      "Response for Row 1903:\n",
      "Answer to Question 2: (deletion of gene H2B residues 30-37, consequence reduced NER efficiency at RPB2 locus), (deletion of gene H2B residues 30-37, consequence reduced NER efficiency at HML locus), (deletion of gene H2B residues 30-37, consequence reduced NER efficiency at GAL10 locus), (deletion of gene H2B residues 30-37, consequence increased nucleosome accessibility), (deletion of gene H2B residues 30-37, consequence increased nucleosome mobility), (deletion of gene H2B residues 30-37, consequence increased DNA damage acquisition), (deletion of gene H2B residues 30-37, consequence reduced Snf5 recruitment to HML locus), (deletion of gene H2B residues 30-37, consequence lower binding affinity of SWI/SNF complex to nucleosomes)\n",
      "Progress: 95.15% completed\n",
      "Response for Row 1904:\n",
      "Answer to Question 2: (deletion of N-terminal region of rpS5, moderately reduced growth rates), (deletion of 46 N-terminal amino acids of rpS5, doubling time of about 3 h), (deletion of N-terminal region of rpS5, impact on the ability of 40S subunits to function properly in translation), (deletion of N-terminal region of rpS5, affect the efficiency of initiation), (deletion of N-terminal region of rpS5, impact on the recruitment of initiation factors eIF3 and eIF2)\n",
      "Progress: 95.20% completed\n",
      "Response for Row 1905:\n",
      "Answer to Question 2: (deletion of gene TGL2, reduction in cellular TAG hydrolysis), (overproduction of gene TGL2, increase in TAG degradation), (deletion of gene TGL2, absence of lipolytic activity in mitochondrial fraction), (overexpression of gene TGL2, strong lipolytic activity in mitochondrial fraction), (deletion of gene TGL2, increased sensitivity to antimitotic drug), (presence of oleic acid, complementation of antimitotic drug sensitivity in tgl2 null mutation)\n",
      "Progress: 95.25% completed\n",
      "Response for Row 1906:\n",
      "Answer to Question 2: (deletion of gene product functions in chromatin regulation, consequence 1), (deletion of gene product functions in chromatin regulation, reduced fitness pleiotropy), (deletion of gene regulated by chromatin state, consequence 1), (deletion of gene regulated by chromatin state, reduced fitness pleiotropy), (deletion of gene product functions as a transcription factor, consequence 1), (deletion of gene product functions as a transcription factor, reduced fitness pleiotropy), (deletion of gene containing TATA-box, consequence 1), (deletion of gene containing TATA-box, no significant effect on fitness pleiotropy when CRE is controlled), (deletion of gene product central in protein interaction network, consequence 1), (deletion of gene product central in protein interaction network, reduced fitness pleiotropy)\n",
      "Progress: 95.30% completed\n",
      "Response for Row 1907:\n",
      "Answer to Question 2: (deletion of gene GGC1, enhanced growth fitness in presence of rapamycin), (overexpression of gene GGC1, stabilization of GGC1 protein in presence of rapamycin), (deletion of gene GGC1, disruption of ribosome biogenesis regulation), (deletion of gene GGC1, disruption of cell cycle progression regulation)\n",
      "Progress: 95.35% completed\n",
      "Response for Row 1908:\n",
      "Answer to Question 2: (deletion of gene involved in homeostasis of protons, sensitivity to NiSO(4)), (deletion of gene involved in cation transport, sensitivity to NiSO(4)), (deletion of gene involved in transport ATPases, sensitivity to NiSO(4)), (deletion of gene involved in endocytosis, sensitivity to NiSO(4)), (deletion of gene involved in siderophore-iron transport, sensitivity to NiSO(4)), (deletion of gene involved in homeostasis of metal ions, sensitivity to NiSO(4)), (deletion of gene involved in diphthamide biosynthesis pathway, sensitivity to NiSO(4)), (deletion of gene involved in functioning and transport of the vacuole and lysosome, resistance to NiSO(4)), (deletion of gene involved in protein targeting, resistance to NiSO(4)), (deletion of gene involved in sorting, resistance to NiSO(4)), (deletion of gene involved in translocation, resistance to NiSO(4)), (deletion of gene involved in intra-Golgi transport, resistance to NiSO(4)), (deletion of gene involved in regulation of C-compound and carbohydrate metabolism, resistance to NiSO(4)), (deletion of gene involved in transcriptional repression, resistance to NiSO(4)), (deletion of gene involved in chromosome segregation/division, resistance to NiSO(4))\n",
      "Progress: 95.40% completed\n",
      "Response for Row 1909:\n",
      "Answer to Question 2: (deletion of gene GPD1, elimination of glycerol production), (deletion of gene GPD2, elimination of glycerol production), (deletion of gene GPD1, inability to grow anaerobically), (deletion of gene GPD2, inability to grow anaerobically), (expression of gene mhpF, restoration of anaerobic growth), (expression of gene mhpF, conversion of acetate to ethanol), (expression of gene mhpF, reduction of NADH by acetate), (deletion of gene GPD1, deletion of gene GPD2, elimination of glycerol production), (deletion of gene GPD1, deletion of gene GPD2, inability to grow anaerobically)\n",
      "Progress: 95.45% completed\n",
      "Response for Row 1910:\n",
      "Answer to Question 2: (deletion of gene xxx, increased acclimation), (deletion of gene xxx, increased growth rates), (deletion of gene xxx, improved survivability), (overexpression of gene xxx, increased glucose utilization), (overexpression of gene xxx, increased glycerol production), (overexpression of gene xxx, increased acetate production)\n",
      "Progress: 95.50% completed\n",
      "Response for Row 1911:\n",
      "Answer to Question 2: (expression of gene CLM1, production of longiborneol), (disruption of gene CLM1, inability to produce culmorin), (addition of longiborneol in CLM1 disruptants, conversion to culmorin), (expression of gene CLM1, requirement for culmorin biosynthesis)\n",
      "Progress: 95.55% completed\n",
      "Response for Row 1912:\n",
      "Answer to Question 2: (deletion of gene cre1, no significant change in flux distribution), (induction of cellulase genes expression, no alteration in flux distribution)\n",
      "Progress: 95.60% completed\n",
      "Response for Row 1913:\n",
      "Answer to Question 2: (deletion of gene amyR, consequence poor growth on starch medium), (deletion of gene malR, consequence loss of malP and malT expression), (deletion of gene malR, consequence restricted growth on maltose medium), (overexpression of gene malT, consequence increase in intracellular alpha-glucosidase activity), (deletion of gene malP, consequence reduced expression of malT and malR), (deletion of gene malP, consequence growth defect on maltose)\n",
      "Progress: 95.65% completed\n",
      "Response for Row 1914:\n",
      "Answer to Question 2: (deletion of gene yap1, decreased rate of reduction of coniferyl aldehyde to coniferyl alcohol), (deletion of gene atr1, decreased rate of reduction of coniferyl aldehyde to coniferyl alcohol), (deletion of gene flr1, decreased rate of reduction of coniferyl aldehyde to coniferyl alcohol), (overexpression of gene atr1, improved resistance against inhibitors in lignocellulose hydrolysates), (overexpression of gene flr1, improved resistance against inhibitors in lignocellulose hydrolysates), (overexpression of gene yap1, improved resistance against inhibitors in lignocellulose hydrolysates)\n",
      "Progress: 95.70% completed\n",
      "Response for Row 1915:\n",
      "Answer to Question 2: (deletion of gene SSX2 NLS, consequence 1: improved secretion of SSX2), (overexpression of gene SSX2, consequence 1: enhanced intracellular SSX2 levels), (expression of gene SSX2 with NLS, consequence 1: accumulation in the endoplasmic reticulum), (expression of gene SSX2 with NLS, consequence 2: misfolding in the endoplasmic reticulum), (deletion of gene SSX2 NLS, consequence 2: correct folding of SSX2 inside the ER)\n",
      "Progress: 95.75% completed\n",
      "Response for Row 1916:\n",
      "Answer to Question 2: (deletion of gene yca1, suppression of TBT-induced cell death), (exposure to TBT, production of reactive oxygen species), (TBT exposure, induction of apoptosis-like cell death), (deletion of gene pdr5, increased sensitivity to TBT)\n",
      "Progress: 95.80% completed\n",
      "Response for Row 1917:\n",
      "Answer to Question 2: (deletion of gene RRG1, respiratory deficiency), (deletion of gene RRG2, respiratory deficiency), (deletion of gene RRG3, respiratory deficiency), (deletion of gene RRG4, respiratory deficiency), (deletion of gene RRG5, respiratory deficiency), (deletion of gene RRG6, respiratory deficiency), (deletion of gene RRG7, respiratory deficiency), (deletion of gene RRG8, respiratory deficiency), (deletion of gene RRG9, respiratory deficiency), (deletion of gene RRG10, respiratory deficiency), (deletion of 16 genes, compromised maintenance of mitochondrial genome), (deletion of 88 genes, impaired mitochondrial protein translation), (deletion of 10 genes, impaired expression of specific mitochondrial gene products), (deletion of genes involved in cytochrome c oxidase assembly, increased sensitivity to aging)\n",
      "Progress: 95.85% completed\n",
      "Response for Row 1918:\n",
      "Answer to Question 2: (deletion of gene PDC1, increased L-lactate productivity), (overexpression of gene human LDH, increased L-lactate productivity), (overexpression of gene bovine LDH, increased L-lactate productivity), (deletion of 289 genes, altered L-lactate productivity)\n",
      "Progress: 95.90% completed\n",
      "Response for Row 1919:\n",
      "Answer to Question 2: (deletion of gene Gpx3, consequence 1: increased DHA-mediated stress), (overexpression of gene Gpx3, consequence 1: upregulation of Dak1 activity), (overexpression of gene Gpx3, consequence 2: suppression of AGE formation)\n",
      "Progress: 95.95% completed\n",
      "Response for Row 1920:\n",
      "Answer to Question 2: (deletion of gene encoding heme oxygenase, protracted metabolic cycles), (expression of heme oxygenase, generation of carbon monoxide), (administration of CO, phase advancement into the oxidative, respiratory phase)\n",
      "Progress: 96.00% completed\n",
      "Response for Row 1921:\n",
      "Answer to Question 2: (deletion of gene MSN2, consequence 1), (deletion of gene MSN4, consequence 1), (deletion of gene MSN2, consequence 2), (deletion of gene MSN4, consequence 2)\n",
      "Progress: 96.05% completed\n",
      "Response for Row 1922:\n",
      "Answer to Question 2: (deletion of acyltransferases, lack of detectable neutral lipids), (deletion of acyltransferases, lack of cytoplasmic lipid droplets), (deletion of acyltransferases, sensitivity to unsaturated fatty acids), (expression of human diacylglycerol acyltransferase 2, reversal of phenotypes), (fatty acid-mediated death, apoptotic cell death), (fatty acid-mediated death, transcriptional induction of stress-response pathways), (fatty acid-mediated death, elevated oxidative stress), (fatty acid-mediated death, activation of unfolded protein response), (diacylglycerol kinase-mediated phosphatidic acid biosynthesis, modifier of lipotoxicity), (production of phosphatidylcholine via methylation of phosphatidylethanolamine, modifier of lipotoxicity), (ablation of phospholipid and triglyceride biosynthesis, increased sensitivity to saturated fatty acids), (normal sphingolipid biosynthesis, required for optimal growth), (vesicular transport, required for optimal growth), (denudation of triglyceride biosynthesis, mediated resistance to exogenous fatty acids)\n",
      "Progress: 96.10% completed\n",
      "Response for Row 1923:\n",
      "Answer to Question 2: (deletion of gene hxk2, increase in TCA cycle flux), (deletion of gene grr1, increase in TCA cycle flux), (deletion of gene hxk2, decrease in glucose uptake rate), (deletion of gene grr1, decrease in glucose uptake rate), (low glucose uptake rate, increase in TCA cycle activity), (unfavorable pH values, increase in TCA cycle activity), (sodium chloride stress, increase in TCA cycle activity)\n",
      "Progress: 96.15% completed\n",
      "Response for Row 1924:\n",
      "Answer to Question 2: (overexpression of gene RPI1, consequence 1: prolong survival of yeast cells during fermentation), (overexpression of gene RPI1, consequence 2: retain viability by over 50-fold better than wild type after 36 hours of fermentation at 38 degrees C), (deletion of gene RPI1, consequence 1: drastic reduction in viability during fermentation), (overexpression of gene RPI1, consequence 3: fortify yeast cell wall), (overexpression of gene RPI1, consequence 4: increase resistance to cell lytic enzyme zymolyase), (overexpression of gene RPI1, consequence 5: facilitate economical production of ethanol)\n",
      "Progress: 96.20% completed\n",
      "Response for Row 1925:\n",
      "Answer to Question 2: (deletion of gene PSD1, loss of PSD activity in mitochondria), (deletion of gene PSD1, severe growth defect on minimal media), (deletion of gene PSD1, depletion of cellular phosphatidylethanolamine levels), (deletion of gene PSD1, depletion of mitochondrial phosphatidylethanolamine levels), (supplementation with ethanolamine, compensation for PSD1 deletion), (deletion of gene PSD1, inability of Psd2p to compensate), (expression of gene Psd1p, synthesis of unsaturated phosphatidylethanolamine species), (expression of gene Psd2p, synthesis of unsaturated phosphatidylethanolamine species), (phosphatidylethanolamine species containing palmitic acid, preferential assembly into mitochondria)\n",
      "Progress: 96.25% completed\n",
      "Response for Row 1926:\n",
      "Answer to Question 2: (deletion of gene Asf1, increased GCR rates), (deletion of gene Asf1, sensitivity to hydroxyurea), (deletion of gene Asf1, sensitivity to methyl methanesulfonate), (deletion of gene Rad6, suppression of increased GCR rates in asf1 mutants), (deletion of gene Siz1, suppression of increased GCR rates in asf1 mutants), (deletion of gene Srs2, suppression of increased GCR rates in asf1 mutants), (deletion of gene Asf1, increased dependency on Dun1), (deletion of gene Asf1, cooperation with Rad6 in ubiquitination of PCNA)\n",
      "Progress: 96.30% completed\n",
      "Response for Row 1927:\n",
      "Answer to Question 2: (deletion of gene Aovps5, higher activity of tripeptidyl peptidase), (deletion of gene Aovps5, higher amount of nitrogenous constituent in wheat gluten degradation)\n",
      "Progress: 96.35% completed\n",
      "Response for Row 1928:\n",
      "Answer to Question 2: (deletion of gene GDH1, increase in available NADPH), (deletion of gene GDH1, increase in cubebol titer), (deletion of gene GDH1, decrease in maximum specific growth rate), (over-expression of gene GDH2, improvement in growth rate)\n",
      "Progress: 96.40% completed\n",
      "Response for Row 1929:\n",
      "Answer to Question 2: (Lsm1 over-expression, inhibits growth), (Lsm1 over-expression, depletes U6 snRNA), (Lsm1 over-expression, alters pre-mRNA splicing), (Lsm1 over-expression, decreases U6 snRNA levels), (Lsm1 over-expression, reduces availability of Lsm2-7 proteins), (Lsm1 over-expression, displays minor alterations in mRNA decay), (Lsm1 over-expression, increases susceptibility to mutations inhibiting cytoplasmic deadenylation)\n",
      "Progress: 96.45% completed\n",
      "Response for Row 1930:\n",
      "Answer to Question 2: (overexpression of gene p53, consequence yeast cell death on minimal medium), (overexpression of gene R282W p53 mutant, consequence inhibition of yeast growth on minimal medium), (overexpression of gene R282W p53 mutant, consequence induction of ROS production), (mutation of gene p53, consequence inactivation of p53 through gross deletion), (mutation of gene p53, consequence inactivation of p53 through nonsense mutations), (mutation of gene p53, consequence inactivation of p53 through deletion of I255), (mutation of gene p53, consequence inactivation of p53 through substitution of R337G)\n",
      "Progress: 96.50% completed\n",
      "Response for Row 1931:\n",
      "Answer to Question 2: (deletion of gene OCH1, defect in hyperglycosylation), (deletion of gene OCH1, inability in adding mannose to the core oligosaccharide), (deletion of gene OCH1, reduction in outer-chain mannose residues), (deletion of gene MNN1, reduction in outer-chain mannose residues), (deletion of gene OCH1, presence of oligosaccharide Man(13-14)GlcNAc(2)), (deletion of gene OCH1 and MNN1, presence of oligosaccharide Man(9-11)GlcNAc(2))\n",
      "Progress: 96.55% completed\n",
      "Response for Row 1932:\n",
      "Answer to Question 2: (deletion of gene xxx, occurrence of tri-partite repair product), (overexpression of Rad51, recovery from checkpoint signal), (increasing distance between repeated sequences, enhanced dependence on Rad51 for colony formation)\n",
      "Progress: 96.60% completed\n",
      "Response for Row 1933:\n",
      "Answer to Question 2: (constitutive expression of Hac1, increase in overall process specific productivity), (deletion of GAS1, increase in overall process specific productivity), (constitutive expression of Hac1 and deletion of GAS1, about a 7-fold increase in overall process specific productivity)\n",
      "Progress: 96.65% completed\n",
      "Response for Row 1934:\n",
      "Answer to Question 2: (deletion of gene RAM component, consequence cell lysis), (deletion of gene CBK1, consequence reduced fertility), (deletion of gene CBK1, consequence reduced expression of mating type-specific genes), (mutation in gene BRR1, consequence restoration of fertility), (mutation in gene MPT5, consequence restoration of fertility)\n",
      "Progress: 96.70% completed\n",
      "Response for Row 1935:\n",
      "Answer to Question 2: (expression of gene McPHGPx, GSH-dependent peroxidase activity towards phospholipid hydroperoxide), (expression of gene McPHGPx, GSH-dependent peroxidase activity towards H2O2), (expression of gene McPHGPx, GSH-dependent peroxidase activity towards tert-butyl hydroperoxide), (deletion of gene PHGPx in Saccharomyces cerevisiae, susceptibility to linolenic acid), (expression of gene McPHGPx, rescue of susceptibilities to linolenic acid in PHGPx-deletion mutant yeast)\n",
      "Progress: 96.75% completed\n",
      "Response for Row 1936:\n",
      "Answer to Question 2: (deletion of gene SPO14, affects PEA production), (deletion of gene YJU3, affects PEA inactivation), (heterologous expression of huntingtin, affects PEA metabolism), (heterologous expression of alpha-synuclein, affects PEA metabolism)\n",
      "Progress: 96.80% completed\n",
      "Response for Row 1937:\n",
      "Answer to Question 2: (deletion of gene xxx, identification of suppressor mutations), (deletion of gene xxx, construction of genetic interaction network), (deletion of gene xxx, revelation of new components of specific pathways)\n",
      "Progress: 96.85% completed\n",
      "Response for Row 1938:\n",
      "Answer to Question 2: (deletion of gene UMP1, enhanced mitochondrial point mutagenesis), (deletion of gene UMP1, increased frequency of respiration-defective mutants), (deletion of gene UMP1, decreased stability of mtDNA), (deletion of gene UMP1, increased ROS production), (deletion of gene UMP1, reduced level of Msh1p)\n",
      "Progress: 96.90% completed\n",
      "Response for Row 1939:\n",
      "Answer to Question 2: (deletion of gene CYS4, deficiency in cysteine), (deletion of gene CYS4, deficiency in glutathione), (deletion of gene CYS4, rust-colored colonies), (deletion of gene CYS4, drug-dependent growth defects), (deletion of genes in sulfur assimilation pathway, enhancement or suppression of rust coloration)\n",
      "Progress: 96.95% completed\n",
      "Response for Row 1940:\n",
      "Answer to Question 2: (deletion of gene not specified, consequence similar wild-type predictions for biomass and ethanol yields), (deletion of gene not specified, consequence smaller ethanol yield enhancements in seven compartment model), (simplification of seven compartment model to two intracellular compartments, consequence increased ethanol yields), (zeroing of metabolic reactions associated with unexpressed genes, consequence negligible changes in steady-state predictions), (dynamic extensions of models, consequence similar predictions of optimal batch ethanol productivity for wild type), (implementable gene insertions, consequence significantly different predictions in single compartment model), (combined deletion/overexpression/insertion mutant, consequence improved ethanol productivity capabilities)\n",
      "Progress: 97.00% completed\n",
      "Response for Row 1941:\n",
      "Answer to Question 2: (deletion of gene SCH9, depletion of extracellular ethanol), (deletion of gene SCH9, reduction of stored lipids), (deletion of gene SCH9, synthesis and release of glycerol), (deletion of gene GPD1, reversal of chronological life span extension), (deletion of gene GPD2, reversal of chronological life span extension), (deletion of gene RHR2, reversal of chronological life span extension), (deletion of gene TOR1, up-regulation of glycerol biosynthesis genes), (deletion of gene RAS2, up-regulation of glycerol biosynthesis genes), (production of glycerol, optimization of life span extension)\n",
      "Progress: 97.05% completed\n",
      "Response for Row 1942:\n",
      "Answer to Question 2: (deletion of Rpd3 function, higher levels of histone H3 acetylation), (deletion of Rpd3 function, higher levels of histone H4 acetylation), (regulation by Rpd3L, silencing transcription), (regulation by Rpd3S, recompacting chromatin), (loss of Rpd3 function, significant association with Rpd3 chromatin binding), (loss of Rpd3 function, control of gene expression)\n",
      "Progress: 97.10% completed\n",
      "Response for Row 1943:\n",
      "Answer to Question 2: (deletion of gene YTA7, affected enzymatic activity of cis-prenyltransferase), (deletion of gene YTA7, altered cellular levels of isoprenoid compounds), (deletion of gene YTA7, rendered cells hypersensitive to lovastatin), (overexpression of gene HMG2, restored growth of yta7Delta cells in presence of lovastatin), (impairment of isoprenoid pathway by lovastatin, altered expression level of YTA7 gene), (impairment of isoprenoid pathway by zaragozic acid, altered expression level of YTA7 gene)\n",
      "Progress: 97.15% completed\n",
      "Response for Row 1944:\n",
      "Answer to Question 2: (depletion of gene Cit2, consequence 1), (overexpression of gene Cit2, consequence 1), (depletion of gene Cit2, consequence 2), (overexpression of gene Cit2, consequence 2), (deletion of gene mls1, consequence 1), (deletion of gene ac1, consequence 1), (deletion of gene icl1, consequence 1), (addition of succinic acid, consequence 1)\n",
      "Progress: 97.20% completed\n",
      "Response for Row 1945:\n",
      "Answer to Question 2: (deletion of gene npt1, decrease in lifespan under calorie restriction), (deletion of gene qpt1, no change in lifespan under calorie restriction), (overexpression of salvage pathway, increase in lifespan under calorie restriction), (deletion of gene qpt1, exclusive utilization of extracellular nicotinic acid for NAD+ and NADH biosynthesis), (deletion of gene npt1, no lifespan increase under calorie restriction), (calorie restriction, decrease in NAD+ concentrations), (deletion of gene npt1, no decrease in NADH concentrations under calorie restriction), (deletion of gene qpt1, no decrease in NADH concentrations under calorie restriction)\n",
      "Progress: 97.25% completed\n",
      "Response for Row 1946:\n",
      "Answer to Question 2: (deletion of gene sidI, affects activation of HSF1), (expression of gene sidI, inhibition of host protein synthesis), (expression of gene sidI, interaction with eEF1A), (expression of gene sidI, interaction with eEF1Bgamma), (expression of gene sidI, activation of stress response)\n",
      "Progress: 97.30% completed\n",
      "Response for Row 1947:\n",
      "Answer to Question 2: (deletion of gene STL1, reduced glycerol uptake), (deletion of gene STL1, poor growth on minimal glycerol medium), (expression of gene STL1, active glycerol uptake), (expression of gene STL1, response to salt stress), (deletion of gene STL2, no effect on glycerol uptake), (deletion of gene GUP1, no effect on glycerol uptake)\n",
      "Progress: 97.35% completed\n",
      "Response for Row 1948:\n",
      "Answer to Question 2: (Deletion of gene FTL1, consequence 1: Reduced conidiation), (Deletion of gene FTL1, consequence 2: Failure to cause typical disease symptoms), (Deletion of gene FTL1, consequence 3: Failure to colonize vascular tissues of rachis), (Deletion of gene FTL1, consequence 4: Failure to cause necrosis on rachis), (Deletion of gene FTL1, consequence 5: Defective in spreading from infected anthers to ovaries), (Deletion of gene FTL1, consequence 6: Increased sensitivity to plant defensins MsDef1 and osmotin), (Deletion of gene FTL1, consequence 7: Significant reduction in histone deacetylation)\n",
      "Progress: 97.40% completed\n",
      "Response for Row 1949:\n",
      "Answer to Question 2: (deletion of gene yidC2, stress-sensitive phenotype), (deletion of gene yidC1, less severe phenotype), (deletion of gene SRP, viable in S. mutans), (deletion of gene SRP and yidC2, severely compromised phenotype), (deletion of gene oxa1, complemented by YidC2), (expression of gene YidC2, mediates cotranslational insertion of mitochondrial translation products into the inner membrane)\n",
      "Progress: 97.45% completed\n",
      "Response for Row 1950:\n",
      "Answer to Question 2: (deletion of gene CuURA3, consequence uracil auxotroph), (deletion of gene CuURA3, consequence null mutant), (deletion of gene CuURA3, consequence supports possibility of tetraploid state)\n",
      "Progress: 97.50% completed\n",
      "Response for Row 1951:\n",
      "Answer to Question 2: (deletion of gene Car1, accumulation of CGP up to 4% (wt/wt)), (deletion of gene Car2, accumulation of CGP up to 4% (wt/wt)), (deletion of gene Arg1, accumulation of CGP up to 15.3% (wt/wt)), (deletion of gene Arg3, accumulation of CGP up to 15.3% (wt/wt)), (deletion of gene Arg4, accumulation of CGP up to 15.3% (wt/wt)), (deletion of gene Arg1, CGP consists of up to 20 mol% citrulline), (deletion of gene Arg3, CGP consists of up to 8 mol% ornithine), (deletion of gene Arg4, CGP consists of up to 16 mol% lysine)\n",
      "Progress: 97.55% completed\n",
      "Response for Row 1952:\n",
      "Answer to Question 2: (deletion of gene Abf1 binding site in SUP35 promoter, no effect on yeast viability), (mutation of ATG1 in SUP35, lethal effect in glucose medium), (mutation of ATG1 in SUP35, partial restoration of viability with galactose), (mutation of ATG1 in SUP35, partial restoration of viability under histidine starvation), (mutation of ATG124 in SUP35, no effect on yeast viability), (mutation of ATG254 in SUP35, no effect on yeast viability)\n",
      "Progress: 97.60% completed\n",
      "Response for Row 1953:\n",
      "Answer to Question 2: (deletion of gene GAP1, impaired fermentation capacity), (deletion of gene GAP1, modified nitrogen consumption rate), (deletion of gene GAP1, altered gene expression of other permeases), (deletion of gene GAP1, reduced arginine consumption)\n",
      "Progress: 97.65% completed\n",
      "Response for Row 1954:\n",
      "Answer to Question 2: (deletion of gene ADH6, prevention of reduction of vanillin to vanillyl alcohol), (expression of gene 3-dehydroshikimate dehydratase, production of intermediate in vanillin biosynthesis), (expression of gene ACAR, reduction of aromatic carboxylic acid to vanillin precursor), (expression of gene O-methyltransferase, methylation of vanillin precursor), (coexpression of gene phosphopantetheinyl transferase, activation of ACAR enzyme), (expression of gene UDP-glycosyltransferase, conversion of vanillin into vanillin beta-D-glucoside)\n",
      "Progress: 97.70% completed\n",
      "Response for Row 1955:\n",
      "Answer to Question 2: (deletion of larger chromosomal regions, adaptation to brewing), (massive unilateral decrease of the ribosomal DNA cluster, post-hybridization evolution process), (bilateral truncations of over 60 genes, post-hybridization evolution process), (truncations and deletions of less efficient maltose uptake genes, adaptation to brewing), (truncations and deletions of less efficient maltotriose uptake genes, adaptation to brewing)\n",
      "Progress: 97.75% completed\n",
      "Response for Row 1956:\n",
      "Answer to Question 2: (deletion of gene MgCRZ1, hypersensitivity to Ca(2+)), (deletion of gene MgCRZ1, reduced number of conidia), (deletion of gene MgCRZ1, abnormal appressoria formation), (deletion of gene MgCRZ1, little or no melanin production), (deletion of gene MgCRZ1, delayed lipid metabolism), (deletion of gene MgCRZ1, decline in turgor pressure), (deletion of gene MgCRZ1, attenuation of pathogenicity)\n",
      "Progress: 97.80% completed\n",
      "Response for Row 1957:\n",
      "Answer to Question 2: (deletion of Yaf9, counteracts efficiency of HOG pathway), (overexpression of Yaf9, activates chromatin remodeling and transcription regulation), (activation of CWI pathway, due to low concentration of cesium chloride), (mutation in CWI pathway, shows sensitivity to cesium chloride)\n",
      "Progress: 97.85% completed\n",
      "Response for Row 1958:\n",
      "Answer to Question 2: (imeB deletion, retarded growth), (imeB deletion, overproduction of fertile sexual fruiting bodies in the presence of light), (imeB deletion, abnormal differentiation of sexual Hülle cells in submerged cultures), (imeB deletion, increased sexual development dependent on VeA), (combined deletion of imeB and fphA, complete loss of light response), (imeB deletion, failure to produce sterigmatocystin), (ImeB, needed for expression of the sterigmatocystin gene cluster)\n",
      "Progress: 97.90% completed\n",
      "Response for Row 1959:\n",
      "Answer to Question 2: (deletion of Not4p, abrogated degradation of arrest products), (insertion of 12 consecutive basic amino acids, degradation of truncated GFP products), (deletion of Not4p, disruption of interaction with Ubc4/Ubc5 E2 enzyme), (deletion of Not4p, disruption of ubiquitin-protein ligase activity)\n",
      "Progress: 97.95% completed\n",
      "Response for Row 1960:\n",
      "Answer to Question 2: (deletion of gene YIL064w, negatively affecting secretory production of reporter alpha-amylase), (deletion of gene YIL064w, necessary for efficient secretory production of murine alpha-amylase), (deletion of gene YIL064w, necessary for efficient secretory production of fungal polygalacturonase), (deletion of gene YIL064w, defects at very early stages of endocytic transport), (deletion of gene YIL064w, defects at later stages of endocytic transport), (deletion of gene YIL064w, low transfection efficiency)\n",
      "Progress: 98.00% completed\n",
      "Response for Row 1961:\n",
      "Answer to Question 2: (deletion of gene H3K4, decreased UV survival), (deletion of gene H3K4, impaired NER at HML locus), (deletion of gene H3K79, decreased UV survival), (deletion of gene H3K79, impaired NER at HML locus), (enhanced recruitment of Sir complex at HML locus, reduced MNase accessibility), (enhanced recruitment of Sir complex at HML locus, impaired DNA repair efficiency), (deletion of gene SIR2, restored MNase accessibility), (deletion of gene SIR2, restored DNA repair efficiency), (deletion of gene SIR3, restored MNase accessibility), (deletion of gene SIR3, restored DNA repair efficiency), (UV irradiation, decreased expression of RAD16)\n",
      "Progress: 98.05% completed\n",
      "Response for Row 1962:\n",
      "Answer to Question 2: (no gene expression/deletion mentioned, method for efficient mutagenesis reported)\n",
      "Progress: 98.10% completed\n",
      "Response for Row 1963:\n",
      "Answer to Question 2: (deletion of SAC1, increased levels of most sphingolipid species), (deletion of SAC1, reduced inositol phosphosphingolipids), (deletion of SAC1, decreased PtdIns), (acute attenuation of PtdIns(4)P production, increased PtdIns), (acute attenuation of PtdIns(4)P production, reduced sphingoid bases), (inhibition of Aur1p, overcome reduction of sphingoid bases), (modulation of PtdIns(4)P metabolism, initiated sphingolipid-dependent biological effects)\n",
      "Progress: 98.15% completed\n",
      "Response for Row 1964:\n",
      "Answer to Question 2: (deletion of gene och1, growth defect), (deletion of gene och1, decreased protein productivity), (deletion of gene mnn1, growth defect), (deletion of gene mnn1, decreased protein productivity), (deletion of gene mnn4, growth defect), (deletion of gene mnn4, decreased protein productivity), (mutation of gene pol3-01, alleviation of growth defect), (mutation of gene cdc6-1, alleviation of growth defect), (alleviation of growth defect, improved protein productivity)\n",
      "Progress: 98.20% completed\n",
      "Response for Row 1965:\n",
      "Answer to Question 2: (deletion of gene xxx, survival increased by duplicates), (deletion of gene xxx, survival increase up to 13%), (deletion of gene xxx, buffering capacity higher for highly expressed genes), (deletion of gene xxx, sequence similarity weakly predicts buffering capacity), (deletion of gene xxx, number of duplicates per gene weakly predicts buffering capacity), (deletion of gene xxx, increased transcriptional and translational activity in buffering duplicates)\n",
      "Progress: 98.25% completed\n",
      "Response for Row 1966:\n",
      "Answer to Question 2: (deletion of gene VDAC-1, severely compromised in DNA import), (deletion of gene VDAC-2, severely compromised in DNA import)\n",
      "Progress: 98.30% completed\n",
      "Response for Row 1967:\n",
      "Answer to Question 2: (deletion of gene TAE1, alters ribosomal profile), (deletion of gene TAE1, defects in translation efficiency), (deletion of gene TAE1, defects in translation fidelity), (overexpression of gene TAE1, suppresses phenotypes related to translation)\n",
      "Progress: 98.35% completed\n",
      "Response for Row 1968:\n",
      "Answer to Question 2: (deletion of gene Gas1p, 4-ethylphenol sorption capacity decreased by 75%)\n",
      "Progress: 98.40% completed\n",
      "Response for Row 1969:\n",
      "Answer to Question 2: (deletion of gene tpi1, increased carbon flux to DHAP), (deletion of gene tpi1, increased glycerol production), (introduction of gene mgs, increased methylglyoxal), (introduction of genes mgs and gldA, increased 1,2-propanediol)\n",
      "Progress: 98.45% completed\n",
      "Response for Row 1970:\n",
      "Answer to Question 2: (deletion of gene xxx, increased succinic acid production), (deletion of gene yyy, increased lactic acid production)\n",
      "Progress: 98.50% completed\n",
      "Response for Row 1971:\n",
      "Answer to Question 2: (deletion of Hsp90, increased phenotypic variation), (deletion of chromosome organization and DNA integrity gene products, increased morphological variation), (deletion of RNA elongation gene products, increased morphological variation), (deletion of protein modification gene products, increased morphological variation), (deletion of cell cycle gene products, increased morphological variation), (deletion of response to stimuli such as stress gene products, increased morphological variation)\n",
      "Progress: 98.55% completed\n",
      "Response for Row 1972:\n",
      "Answer to Question 2: (deletion of gene Ybr030w, unmethylated Rpl42ab), (deletion of gene SET7, unmethylated Rpl42ab), (deletion of gene SET7, slow growth phenotype), (deletion of gene Ybr030w and SET7, slow growth phenotype), (expression of gene Ybr030w, methylation of Lys-40 in Rpl42ab), (expression of gene SET7, methylation of Lys-55 in Rpl42ab)\n",
      "Progress: 98.60% completed\n",
      "Response for Row 1973:\n",
      "Answer to Question 2: (deletion of gene Ynm3, consequence cell survival at higher temperature is impaired), (deletion of gene Ynm3, consequence increased heat sensitivity), (deletion of gene Ynm3, consequence reduced chaperone activity), (presence of gene Ynm3, consequence improved efficiency of proteolysis of aberrant proteins), (presence of gene Ynm3, consequence prevention of formation of nonproductive toxic aggregates)\n",
      "Progress: 98.65% completed\n",
      "Response for Row 1974:\n",
      "Answer to Question 2: (deletion of gene Rrp6p, accumulation of poly(A)+ rRNA degradation intermediates), (deletion of gene Dis3p, increase in the levels of degradation substrates common to the core exosome and Rrp6p), (deletion of gene Rrp43p, accumulation of degradation intermediates), (mutation in gene Rrp6p, loss of co-purification with the core exosome), (disruption of Rrp6p-core exosome interaction, inability to efficiently degrade certain poly(A)+ rRNA processing products)\n",
      "Progress: 98.70% completed\n",
      "Response for Row 1975:\n",
      "Answer to Question 2: (deletion of gene At1g27980, decreased fresh weight decrease rate), (deletion of gene At1g27980, altered leaf temperature), (deletion of gene At1g27980, increased dehydration stress tolerance)\n",
      "Progress: 98.75% completed\n",
      "Response for Row 1976:\n",
      "Answer to Question 2: (deletion of gene mik-1, reduced growth of basal hyphae), (deletion of gene mek-1, reduced growth of basal hyphae), (deletion of gene mak-1, reduced growth of basal hyphae), (deletion of gene mik-1, reduced formation of asexual macroconidia), (deletion of gene mek-1, almost abolished formation of asexual macroconidia), (deletion of gene mak-1, almost abolished formation of asexual macroconidia), (deletion of gene mik-1, increased abundance of arthroconidia), (deletion of gene mek-1, increased abundance of arthroconidia), (deletion of gene mak-1, increased abundance of arthroconidia), (deletion of gene mik-1, inability to form protoperithecia), (deletion of gene mek-1, inability to form protoperithecia), (deletion of gene mak-1, inability to form protoperithecia), (deletion of gene mik-1, increased levels of tyrosinase mRNA), (deletion of gene mek-1, increased levels of tyrosinase mRNA), (deletion of gene mak-1, increased levels of tyrosinase mRNA), (deletion of gene mik-1, increased levels of tyrosinase protein), (deletion of gene mek-1, increased levels of tyrosinase protein), (deletion of gene mak-1, increased levels of tyrosinase protein), (deletion of gene mik-1, increased production of l-DOPA melanin), (deletion of gene mek-1, increased production of l-DOPA melanin), (deletion of gene mak-1, increased production of l-DOPA melanin)\n",
      "Progress: 98.80% completed\n",
      "Response for Row 1977:\n",
      "Answer to Question 2: (deletion of gene PEX5, relocation of Icl1 to cytosol), (deletion of gene PEX5, relocation of Mls1 to cytosol), (deletion of gene FOX2, reduced growth on non-fermentable carbon sources), (deletion of gene FOX2, presence of giant peroxisomes), (deletion of gene PEX5 in fox2Delta/Delta strain, restoration of growth on acetate), (deletion of gene PEX5 in fox2Delta/Delta strain, restoration of growth on ethanol)\n",
      "Progress: 98.85% completed\n",
      "Response for Row 1978:\n",
      "Answer to Question 2: (deletion of gene ydjL, abolished 2,3-butanediol production), (deletion of gene ydjL, no detectable AR activity), (deletion of gene ydjL, no detectable BDH activity), (deletion of gene ydjL, accumulation of acetoin), (deletion of gene ydjL, affected lactate production kinetics)\n",
      "Progress: 98.90% completed\n",
      "Response for Row 1979:\n",
      "Answer to Question 2: (deletion of gene GAL4, prevents induced galactose toxicity), (deletion of gene GAL1, prevents induced galactose toxicity), (overexpression of gene GRE3, cells more tolerant to lithium), (overexpression of gene GRE3, accumulates more galactitol), (overexpression of gene GRE3, accumulates less galactose-1-phosphate), (overexpression of gene GRE3, suppresses galactose growth defect in gal7-deleted strain), (overexpression of gene GRE3, suppresses galactose growth defect in gal10-deleted strain)\n",
      "Progress: 98.95% completed\n",
      "Response for Row 1980:\n",
      "Answer to Question 2: (deletion of a single candidate gene, no effect on yeast growth in the presence of inhibitors)\n",
      "Progress: 99.00% completed\n",
      "Response for Row 1981:\n",
      "Answer to Question 2: (deletion of gene FAS II, inefficient 5' processing of mitochondrial precursor tRNAs), (deletion of gene FAS II, accumulation of precursor containing RPM1 and tRNA(Pro)), (deletion of gene FAS II, blockage of Pet127-driven 5' processing of RPM1), (deletion of gene FAS II, loss of lipoic acid attachment to subunits of three key mitochondrial enzymes), (deletion of gene FAS II, requirement for octanoic acid as a precursor for lipoic acid synthesis and attachment), (deletion of gene FAS II, requirement for a product of the FAS II pathway for RNase P RNA maturation), (deletion of gene FAS II, requirement for a product of the FAS II pathway for lipoic acid production needed for pyruvate dehydrogenase activity)\n",
      "Progress: 99.05% completed\n",
      "Response for Row 1982:\n",
      "Answer to Question 2: (deletion of specialized translesion DNA polymerases, increased mutation frequency upon 4-NQO treatment), (increase in dNTP concentration, improved tolerance to 4-NQO induced damage), (increase in dNTP concentration, increased bypass efficiency of 8-oxoG lesion by DNA polymerase epsilon), (insertion of dATP opposite 8-oxoG lesion)\n",
      "Progress: 99.10% completed\n",
      "Response for Row 1983:\n",
      "Answer to Question 2: (deletion of gene eIF3a, impairment of translation initiation), (deletion of gene eIF3a, reduction of binding of eIF3 and associated eIFs to native preinitiation complexes), (deletion of gene eIF3a, severe blockage of the induction of GCN4 translation via reinitiation), (deletion of gene eIF3a, inability of 40S ribosomes to resume scanning after terminating at the first upstream ORF (uORF1)), (interaction between eIF3a-NTD and sequences 5' of uORF1, enhancement of reinitiation), (increasing length of uORF1, decline in reinitiation efficiency), (eIF3 retention on ribosomes throughout uORF1 translation, stabilization of mRNA association with post-termination 40S subunits), (eIF3 retention on ribosomes throughout uORF1 translation, enable resumption of scanning for reinitiation downstream)\n",
      "Progress: 99.15% completed\n",
      "Response for Row 1984:\n",
      "Answer to Question 2: (deletion of gene KlPCL1, no effect on sporulation), (deletion of gene KlPCL1, no effect on growth abilities under various physiological conditions)\n",
      "Progress: 99.20% completed\n",
      "Response for Row 1985:\n",
      "Answer to Question 2:  (deletion of gene tag, consequence filamentation phenotype after MMS exposure), (deletion of gene dinG, consequence filamentation phenotype after MMS exposure), (deletion of gene dinG, consequence decoupling of filamentation from SulA levels), (deletion of gene dinG, consequence associated with SulA-independent filamentation pathway)\n",
      "Progress: 99.25% completed\n",
      "Response for Row 1986:\n",
      "Answer to Question 2: (deletion of multiple nonessential genes, normal growth), (deletion of multiple nonessential genes, possible synthetic growth defects)\n",
      "Progress: 99.30% completed\n",
      "Response for Row 1987:\n",
      "Answer to Question 2: (deletion of gene xxx, consequence 1), (deletion of gene yyy, consequence 2)\n",
      "Progress: 99.35% completed\n",
      "Response for Row 1988:\n",
      "Answer to Question 2: (deletion of gene ACS1, consequence growth on acetate), (deletion of gene ACS1, consequence full virulence), (deletion of gene SNF1, consequence alternative carbon source utilization), (deletion of gene SNF1, consequence response to nitrosative stress), (deletion of gene SNF1, consequence melanin production), (deletion of gene SNF1, consequence virulence)\n",
      "Progress: 99.40% completed\n",
      "Response for Row 1989:\n",
      "Answer to Question 2: (deletion of gene KTI11, consequence antisuppression of nonsense and missense suppressor tRNAs), (deletion of gene KTI11, consequence zymocin resistance), (deletion of gene KTI11, consequence protection against tRNase attack of zymocin), (deletion of gene KTI11, consequence resistance against diphtheria toxin), (deletion of gene KTI11, consequence protection against ADP-ribosylation of eEF2 by DT), (deletion of gene KTI11, consequence induction of resistance against sordarin), (mutation in gene KTI11, consequence dissociation of zymocin resistance from DT sensitivity)\n",
      "Progress: 99.45% completed\n",
      "Response for Row 1990:\n",
      "Answer to Question 2: (deletion of gene hap4, altered growth characteristics), (deletion of gene mig1, altered growth characteristics), (deletion of gene qdr3, increased ethanol production), (deletion of gene pdr3, altered growth characteristics), (deletion of gene qcr7, compensated effect on respiratory function), (deletion of gene cyt1, avoided NAD(+)/NADH imbalance), (deletion of gene rip1, avoided NAD(+)/NADH imbalance), (deletion of gene gdh1, overexpression of glt1 improves ethanol production), (overexpression of gene glt1, improved ethanol production), (deletion of gene fps1, no effect on glycerol efflux)\n",
      "Progress: 99.50% completed\n",
      "Response for Row 1991:\n",
      "Answer to Question 2: (deletion of gene BTN2, decreases flor yeast resistance to high ethanol concentrations), (deletion of gene BTN2, increases fermentative capacity of flor yeast), (deletion of gene BTN2, increases flor yeast sliding motility), (deletion of gene BTN2, increases transcription of the FLO11 gene), (overexpression of gene BTN2, prevents growth on nonfermentable carbon sources), (BTN2 transcription induced by acetaldehyde, increases BTN2 expression), (BTN2 repression by ethanol, decreases BTN2 expression)\n",
      "Progress: 99.55% completed\n",
      "Response for Row 1992:\n",
      "Answer to Question 2: (deletion of gene GAP1, consequence 1: lower production of 3MH on synthetic medium), (overexpression of gene GAP1, consequence 1: higher production of 3MH on synthetic medium), (addition of DAP, consequence 1: decrease of 3MH production on grape must), (substitution of urea for DAP, consequence 1: increase of 3MH production), (relief of nitrogen catabolite repression (NCR), consequence 1: higher production of 3MH on synthetic medium)\n",
      "Progress: 99.60% completed\n",
      "Response for Row 1993:\n",
      "Answer to Question 2: (deletion of gene LYS20, reduced homocitrate synthesis under fermentative metabolism), (deletion of gene LYS21, reduced homocitrate synthesis during growth on ethanol), (overexpression of gene LYS20, increased homocitrate synthesis under fermentative metabolism), (overexpression of gene LYS21, increased homocitrate synthesis during growth on ethanol), (lysine inhibition of Lys21p, reduced activity of Lys21p), (lysine inhibition of Lys20p, less pronounced effect compared to Lys21p), (positive cooperativity of Lys21p for alpha-KG, enhanced alpha-KG binding in presence of lysine)\n",
      "Progress: 99.65% completed\n",
      "Response for Row 1994:\n",
      "Answer to Question 2: (deletion of gene HU, consequence DNA looping defect), (expression of gene Nhp6A, consequence rescue of DNA looping), (expression of gene Nhp6A, consequence alteration of DNA twist), (deletion of gene HMGB2-box A, consequence failure to rescue looping)\n",
      "Progress: 99.70% completed\n",
      "Response for Row 1995:\n",
      "Answer to Question 2: (deletion of gene RAD9, reduced growth in sodium selenite), (deletion of gene RAD6, reduced growth in sodium selenite), (deletion of gene RAD18, reduced growth in sodium selenite), (deletion of gene in oxidative stress pathway, reduced growth in selenomethionine), (deletion of gene in oxidative stress pathway, reduced growth in sodium selenite)\n",
      "Progress: 99.75% completed\n",
      "Response for Row 1996:\n",
      "Answer to Question 2: (deletion of gene transporter, consequence reduced filament formation), (overexpression of gene transporter, consequence enhanced filament formation), (presence of fusel alcohols, consequence induction of filament formation), (accumulation of 3-methyl-1-butanol, consequence increased mitochondrial mass), (accumulation of 3-methyl-1-butanol, consequence increased chitin content)\n",
      "Progress: 99.80% completed\n",
      "Response for Row 1997:\n",
      "Answer to Question 2: (deletion of gene SAW1, consequence abolished association of Rad1 at SSA intermediates in vivo), (deletion of gene SAW1, consequence cells accumulate recombination intermediates blocked at the Rad1/Rad10-dependent 3' flap cleavage step), (deletion of gene SLX4, consequence cells accumulate recombination intermediates blocked at the Rad1/Rad10-dependent 3' flap cleavage step)\n",
      "Progress: 99.85% completed\n",
      "Response for Row 1998:\n",
      "Answer to Question 2: (deletion of gene dss-1, consequence embryonic lethality), (deletion of gene dss-1, consequence larval arrest), (deletion of gene dss-1, consequence impaired oogenesis), (expression of gene dss-1, consequence rescued temperature-sensitive growth phenotype in yeast), (expression of gene dss-1, consequence partial rescue of accumulation of polyubiquitinated proteins in yeast)\n",
      "Progress: 99.90% completed\n",
      "Response for Row 1999:\n",
      "Answer to Question 2: (deletion of gene Kti13, consequence zymocin resistance), (deletion of gene Kti13, consequence rescue of thermosensitive growth defect of secretory mutants), (deletion of gene Kti13, consequence synthetic sickness or lethality when combined with kti11Delta), (deletion of gene Elongator, consequence rescue of thermosensitive growth defect of secretory mutants), (deletion of gene Elongator, consequence synthetic sickness or lethality when combined with kti13Delta)\n",
      "Progress: 99.95% completed\n",
      "Response for Row 2000:\n",
      "Answer to Question 2: (deletion of gene Irc7, decrease in 4MSP release), (deletion of gene Irc7, decrease in 3SH release), (deletion of gene Ure2p, altered transcriptional regulation of IRC7), (deletion of gene Gln3p, altered transcriptional regulation of IRC7)\n",
      "Progress: 100.00% completed\n"
     ]
    }
   ],
   "source": [
    "# Qwen reads abstract and identify knowledge\n",
    "\n",
    "import pandas as pd\n",
    "import os\n",
    "import torch\n",
    "import gc\n",
    "\n",
    "# Assuming `tokenizer`, `model`, and `device` are already defined and initialized\n",
    "# Example device initialization: device = torch.device(\"cuda\" if torch.cuda.is_available() else \"cpu\")\n",
    "\n",
    "# Function to interact with the LLM API using the new method, now with customizable system prompts\n",
    "def ask_questions(abstract, questions, system_prompts):\n",
    "    responses = []\n",
    "    for question, system_prompt in zip(questions, system_prompts):\n",
    "        # Combine the question and abstract to form the prompt\n",
    "        prompt_text = question + \" \" + str(abstract)\n",
    "        \n",
    "        # Prepare the messages for the new API, using a customizable system prompt\n",
    "        messages = [\n",
    "            {\"role\": \"system\", \"content\": system_prompt},\n",
    "            {\"role\": \"user\", \"content\": prompt_text}\n",
    "        ]\n",
    "        text = tokenizer.apply_chat_template(\n",
    "            messages,\n",
    "            tokenize=False,\n",
    "            add_generation_prompt=True\n",
    "        )\n",
    "        model_inputs = tokenizer([text], return_tensors=\"pt\").to(device)\n",
    "        \n",
    "        # Generate response with the new API\n",
    "        generated_ids = model.generate(\n",
    "            model_inputs.input_ids,\n",
    "            max_new_tokens=5000\n",
    "        )\n",
    "        generated_ids = [\n",
    "            output_ids[len(input_ids):] for input_ids, output_ids in zip(model_inputs.input_ids, generated_ids)\n",
    "        ]\n",
    "        \n",
    "        # Decode the generated response\n",
    "        response = tokenizer.batch_decode(generated_ids, skip_special_tokens=True)[0]\n",
    "        responses.append(response)\n",
    "    return responses\n",
    "\n",
    "# Read the Excel file\n",
    "file_path = excel_filename  # Replace with your file path\n",
    "df = pd.read_excel(file_path)\n",
    "\n",
    "questions = [\" \"]  # Placeholder question that won't be used\n",
    "\n",
    "system_prompts = [\n",
    "    \"You are specialized for analyzing scientific paper abstracts, Extract entities and causal relationships from scientific paper abstracts. Focus on genes overexpression/deletion, rationale for expression/deletion, products, and metabolites related to expression/deletion. Output in (expression/deletion of gene xxx, consequence 1), (expression/deletion of gene xxx, consequence 2)... format with no additional text.\"\n",
    "]\n",
    "\n",
    "# Process each abstract and store the response\n",
    "total_rows = len(df)\n",
    "for i, row in df.iterrows():\n",
    "    # Clear the console at the beginning of each iteration\n",
    "    os.system('cls' if os.name == 'nt' else 'clear')\n",
    "\n",
    "    # Since we're only asking one question now, directly get the response for the second (index 0) system prompt\n",
    "    response = ask_questions(row['Abstract'], [questions[0]], [system_prompts[0]])[0]\n",
    "\n",
    "    # Store the response in the DataFrame\n",
    "    df.at[i, 'Answer to Question 2'] = response\n",
    "\n",
    "    # Show the response\n",
    "    print(f\"Response for Row {i+1}:\")\n",
    "    print(f\"Answer to Question 2: {response}\")\n",
    "\n",
    "    # Calculate and show the progress percentage\n",
    "    progress = ((i + 1) / total_rows) * 100\n",
    "    print(f\"Progress: {progress:.2f}% completed\")\n",
    "\n",
    "# Save the updated DataFrame back to an Excel file\n",
    "output_file_path = 'updated(Qwen2.5 32b)_'+keyword+'_causal.xlsx'  # Replace with your desired output file path\n",
    "df.to_excel(output_file_path, index=False)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "d57f9118-ba3c-4612-b2d1-c986b369a064",
   "metadata": {
    "scrolled": true,
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "deletion of gene tri18, absence of type D trichothecene toxin epiroridin E, reduction in pathogenicity of P. roridum towards pumpkin, heterologous expression of tri3, toxin resistance in Saccharomyces cerevisiae, heterologous expression of tri17KR, expression of genes related to the uptake of phosphate, improved wastewater treatment performance, expression of genes related to the uptake of ammonium ions, overexpression of metabolism-related genes, altered metabolic pathways, overexpression of stress response genes, enhanced stress tolerance, overexpression of cell adhesion genes, increased cell attachment, overexpression of gene C3H15, lower chlorophyll content, reduced photosynthetic efficiency, accelerated senescence, altered mobilization of copper to seeds, altered mobilization of zinc to seeds, deletion of gene zwf1, change in protein content, change in enzymatic activity, deletion of gene gnd1, deletion of gene ald6, overexpression of gene POS5Δ17, enhanced reduced nicotinamide adenine dinucleotide phosphate pool, elevated titer of vitamin A by almost 46%, introduction of gene retinol dehydrogenase 12, achieved more than 90% retinol purity, combinational expression of genes Mbblh and Ssbco, increased flux from β-carotene to vitamin A metabolism, multi-copy integration of gene Mbblh, improved synthesis of vitamin A, deletion of oxygen, enhanced fermentation rate, biomass yield increase, overexpression of genes involved in lipid biosynthesis pathways, anaerobiosis, overexpression of genes associated with thiamine metabolism, presence of oxygen, overexpression of genes associated with methionine metabolism, overexpression of genes associated with sulfur metabolism, decline in total levels of NAD(H, total NADP(H, expression of gene invertase, consequence increases extracellular hexose concentrations, consequence suppresses metabolic efficiency of competitors, consequence enhances hexose capture rate, consequence induces transporter expression, overexpression of gene tHMGR, increased FPP production, overexpression of gene ERG20, overexpression of gene GGPPS, overexpression of gene TS, increased taxadiene production, overexpression of all mevalonate pathway genes, increased FPP availability, enhanced taxadiene production, Deletion of gene FgMDM33, consequence 1: defects in mycelial growth, consequence 2: reduction in conidia production, consequence 3: decreased virulence, consequence 4: elongated mitochondria, consequence 5: respiratory-deficient growth phenotype, consequence 6: reduction in proteolysis of GFP-FgAtg8, consequence 7: upregulation of FgAIF1 gene expression, expression of gene YJR107W, consequence specific activity of 0.397 U/mg, consequence optimum temperature of 37 °C, consequence pH 6, expression of gene LIP3, consequence no lipase activity, consequence specific activity of 1.36 U/mg in bioreactor, consequence specific activity of 1.25 U/mg in bioreactor, co-expression of α-amylase gene, enzyme production, SPR hydrolysis, glucose fermentation, co-expression of glucoamylase gene, expression of α-amylase gene, production of 33.03 g/L ethanol, expression of glucoamylase gene, production of 28.37 g/L ethanol, co-expression of α-amylase and glucoamylase genes, production of 42.22 g/L ethanol, increased ethanol concentration to 50.55 g/L, deletion of gene SSK1, consequence high antioxidant activity, deletion of gene HIP1R, overexpression of gene PAS13, consequence balanced profile, overexpression of gene FRI751, consequence high polyphenol content, yeast mixture administration, increased faecal consistency, increased abundance of Bifidobacterium, increased abundance of Coprococcus 2, decreased abundance of Clostridium Sensu Stricto 1, increased villous height, increased villous width, increased crypt depth, higher presence of PAS-positive mucins, asymmetric bulge at 3' end of miR-5p, reduced precision of the second cleavage, asymmetric bulge at other sites of miR-5p, mainly affected the accumulation level of miR-5p, many asymmetric bulges in miR-3p, strong negative impact on processing precision and accumulation level of miR-5p, asymmetric bulges on middle of miR-5p, less impact on DCL1 cleavage efficiency and precision, asymmetric bulges on miR-3p or near ends of miR-5p, strongly reduce DCL1 cleavage activity, precision or both, interaction between 2-nt 3' overhang of pre-miRNA and 3'BP, important for precision of the second cleavage reaction, deletion of gene PRS5, increase in RNA content, upregulation of genes sharing similar functions, deletion of gene DBP8, upregulation of gene TAL1, upregulation of gene PGM2, activation of pentose phosphate pathway, upregulation of CDK genes, increase in intracellular nucleotide levels, polygenic recombinant strain G03-△PRS5+△DBP8, increase in RNA content by 71.8%, polygenic recombinant strain G03-△PRS5+△DBP8+PGM2, increase in RNA content by 80.1%, increase in maximum specific growth rate, overexpression of gene MGD1, consequence increased monogalactosyldiacylglycerol production, overexpression of gene GAT1, overexpression of gene ICT1, overexpression of gene PAH1, multi-copy integration of MGD1 expression cassette, deletion of gene tcb1, upregulation of glycolysis, upregulation of fermentation, upregulation of amino acid synthesis, upregulation of low-affinity glucose uptake, downregulation of tricarboxylic acid cycle, downregulation of respiration, downregulation of high-affinity glucose uptake, downregulation of amino acid uptake, increased growth, increased glucose consumption, increased CO2 production, increased ethanol generation, expression of gene MET15, growth induction, expression of gene FAR1, growth inhibition, expression of gene RgCuAO1, production of salidroside, expression of gene RgPAR1, expression of gene RgUGT2, expression of gene RgTyDC2, deletion of gene med3, impaired S. cerevisiae growth under hyperosmotic stress, decreased intracellular NADPH content, increased oxidative stress, elevated levels of intracellular reactive oxygen species, impacted bud formation, deletion of chaperone genes, sensitivity to canavanine, canavanine exposure, attenuation of RTG-target-gene expression, deletion of idh1, deletion of idh2, synthetic growth inhibition with chaperone mutants, overexpression of gene ZWF1, increased xylitol production, deletion of gene ALD6, reduced expression of gene PGI1, improved xylitol production on glucose, heterologous expression of gene GDP1, expression of gene POS5, improved l-galactonate production, deletion of natural promoters, limited transcriptional capacity, overexpression using synthetic promoters, enhanced metabolic engineering requirements, expression of Frs2v, cell growth inhibition, sensitivity to 2 mM 4-CP, removal of Frs2v-expressing plasmid, after culturing in 4-CP-containing medium, scarless gene replacement, expression of gene EhN3, high desaturation activity toward C18-ω6 PUFAs, high desaturation activity toward C20-ω6 PUFAs, utilization of galactolipids as substrates, decreased growth temperature, increased expression of gene EhN3, increased production of ω3-PUFAs, increased production of OPA, deletion of gene ulp2, increase in the levels of intron-containing pre-mRNA of ribosomal protein transcripts, improper splicing of ribosomal protein transcripts, deletion of gene Esc2, consequence Ty1 transcription受影响, deletion of gene Rad57, overexpression of gene GAGsi, consequence gene silencing, consequence silent chromatin formation, upregulation of genes involved in maintaining cell wall integrity, consequence 1, upregulation of genes involved in cell wall structural components, consequence 2, upregulation of genes involved in the production of H₂S, consequence 3, upregulation of HSP stress response genes, consequence 4, downregulation of genes belonging to the FLO family, consequence 5, expression of gene mleS, reduction in L-malic acid content, expression of gene MAE1, expression of gene mleA, expression of gene MAE2, expression of gene melP, expression of gene MAE1 and mleS, improvement in flavor quality, expression of gene MAE1 and MAE2, deletion of gene HMX1, consequence: downregulation of 14 genes related to the cell wall and membrane system, consequence: decrease in acetyl-CoA levels by 1.02-fold, consequence: decrease in riboflavin levels by 1.44-fold, consequence: increase in pantothenic acid content, no specific gene expression/deletion mentioned, general discussion on splicing regulation in yeast, deletion of gene involved in nitrogen metabolism, deletion of gene involved in pentose phosphate pathway, deletion of gene involved in ethanol conversion, deletion of gene involved in amino acid metabolism, deletion of gene involved in fatty acid β-oxidation, overexpression of gene involved in fatty acid precursor biosynthesis, consequence 6, overexpression of gene involved in boosting carbon fluxes of acetyl-CoA, consequence 7, overexpression of gene involved in boosting carbon fluxes of NADPH, consequence 8, TORC1 inhibition, increased proteasome assembly chaperone translation, activation of MAPK Mpk1, relocalisation of assembly chaperone mRNA to patches of dense actin, activation of cell wall integrity pathway, activation through Wsc1, Wsc3 and Wsc4 sensor proteins, Sch9 inhibition, necessary for driving protein expression, deletion of gene transmembrane transport, consequence generalized downregulation, overexpression of gene sulfur compounds, consequence upregulation, overexpression of gene pentose phosphate pathway, overexpression of key genes tHMG1-ERG20, increased valencene yield by 93.49%, iterative integration of the valencene synthase gene VSm, increased valencene yield to 254.67 mg/L, deletion of gene IRA2, increased glycolytic flux, deletion of gene IMA1, overexpression of gene MALX, increased sugar consumption, overexpression of maltose-related genes, increased maltotriose consumption, deletion of gene tre49304, improved cellulase production, deletion of gene tre120391, deletion of gene tre123541, overexpression of regulatory factors, enhanced cellulase production, RNA interference of specific genes, increased endoglucanase activity, sub-genomic engineering, improved microbial performance, deletion of gene not mentioned, sanguinarine cytotoxicity, expression of reticuline oxidases, increased sanguinarine biosynthesis, expression of flavoproteins, enhanced enzyme function, expression of cytochrome P450 enzymes, SIMTeGES implementation, decoupling cell growth from product synthesis, engineered yeast strains, production of sanguinarine at 448.64 mg L-1, biosynthesis of fluorinated sanguinarine, deletion of gene ade2, off-target effects, heteroclones presence, deletion of gene trp1, successful knockout, expression of gene ade2, red color phenotype, HMO1 overexpression, enhanced ER stress tolerance, improved protein folding efficiency, BIO5 deletion, HMO1 overexpression and BIO5 deletion, 2.14-fold increase in α-amylase production, 2.04-fold increase in cell density, deletion of gene related to amino acid metabolism, consequence 1: decrease in amino acid metabolites, overexpression of gene related to nucleotide metabolism, consequence 1: increase in nucleotide metabolites, deletion of gene related to fructose and mannose metabolism, consequence 1: decreased fructose and mannose metabolism, deletion of gene related to galactose metabolism, consequence 1: decreased galactose metabolism, deletion of gene related to carboxylic acid metabolism, consequence 1: decrease in carboxylic acids, deletion of gene related to protein synthesis and metabolism, consequence 1: decreased protein synthesis and metabolism, deletion of gene ScAlaX, induced Ala- and Thr-to-Ser misincorporation, deletion of gene hAlaX, overexpression of gene hAlaX, impeded the decoding efficiency of consecutive Ser codons, gain in geneticin resistance, sensitivity to cycloheximide, expression of rate-limited enzymes, enhancement of precursor flux, suppression of competitive pathway flux, mitigation of cytotoxicity, optimization of substrate utilization, refinement of fermentation process, co-expression of gene ARO10, yield of 2-phenylethanol increased, co-expression of gene yahK, introduction of amino acid substitution ARO10 I544W, 2-phenylethanol yield increased 4.1-fold, expression of gene ARO10 I544W, 2-phenylethanol yield of 2.5 g/L achieved, yield from glucose of 0.16 g/g achieved, gene expression of multiple genes in glycolysis, increased β-carotene production, gene expression of multiple genes in mevalonate, gene expression of multiple genes in β-carotene synthesis, decreased expression of ERG9, deletion of gene candidate genes, consequence total carotenoid production improvement, deletion of gene eight out of ten candidate genes, consequence total carotenoid production 1.7 times compared with the parental strain, overexpression of gene PDR5, latent-stationary phase post-diauxic shift, mitochondrial regulation, lipidomic remodeling, overexpression of gene CIS1, overexpression of gene YGR035C, knockout of gene PDR5, altered post-diauxic shift phenotype, rescue of gene PDR5, restored post-diauxic shift phenotype, CBD treatment, fatty acid production, glycerophospholipid production, phosphate-salvage indicators, deletion of gene ortholog, consequence 1: functional redundancy in gene-protein-reaction rule, overexpression of gene ortholog, consequence 1: enhanced metabolic capabilities, consequence 2: altered oxidative and non-oxidative parts of the pentose phosphate pathway, deletion of gene GAL80, delayed response to low-temperature triggers, weak response at 30°C, overexpression of gene Gal4p, suboptimal induction, overexpression of gene Gal3Cp, rapid stringent high-level induction, deletion of gene MKS1, increase in glycerol production, reduction in ethanol production, decrease in acetic acid levels, deletion of gene RTG2, defect in wine fermentation speed, expression of gene Cppks1, detrimental effect on growth of transformed yeast cells, production of phleichrome, expression of gene encoding PPTase from A. nidulans, enzymatic activation of Cppks1 protein product, cultivation at 20 °C and 25 °C, changes in pigmentation, cultivation at 30 °C, no changes in pigmentation, presence of phleichrome in cell extracts, verified by LC/MS/MS, deletion of gene PRS1-PRS5, consequence reduced PRPP production, consequence affects cell wall integrity signalling pathway, expression of gene Prs5, consequence increased expression of FKS2, consequence increased expression of Rlm1, no gene expression/deletion mentioned, consequence: detection of long disome footprints outside regions of non-random elongation stalls, consequence: identification of footprints linked to translation initiation and protein biosynthesis rates, consequence: application of STE to investigate rapid changes to mRNA translation in yeast undergoing glucose depletion, deletion of genes involved in the glyoxylate cycle, overexpression of genes involved in gluconeogenesis, overexpression of genes involved in the pentose phosphate pathway, overexpression of genes involved in protein folding, overexpression of genes involved in meiosis, deletion of genes related to the biosynthesis of vitamin B6, deletion of genes related to the biosynthesis of thiamin, deletion of genes related to the biosynthesis of purine, overexpression of genes related to protein folding, overexpression of genes related to toxin efflux, overexpression of genes related to cell wall remodeling, overexpression of genes connected to the pentose phosphate pathway, overexpression of genes connected to gluconeogenesis, overexpression of genes connected to fatty acid utilization, overexpression of HSP42, overexpression of HSP26, no gene overexpression/deletion mentioned, consequence 1: accurate prediction of translation initiation rate in Saccharomyces cerevisiae, consequence 2: optimization of nucleotide sequence of any gene to achieve a specific target initiation rate, deletion of gene prp45(1-169), consequence chromatin remodellers and modifiers as enriched category, consequence strong growth defect with htz1, consequence disproportionate effect on low copy number transcripts of intron-containing genes, deletion of gene srb2 intron, consequence partial repair of growth phenotypes, deletion of gene prp45(1-330), consequence synthetic growth defect with htz1Δ at 16°C, consequence enhancement of pre-mRNA hyper-accumulation with htz1Δ, znf1 deletion, hypersensitivity to isobutanol, hypersensitivity to acid inhibitors, overexpression of gene ZNF1, enhanced cell survival, isobutanol induction, activation of ATP1-3 expression, activation of COX6 expression, Znf1 activation, PPP genes upregulation, valine biosynthetic genes upregulation, maintenance of NADP/NADPH redox balance, overexpression of gene EcCAX3, consequence tolerance to different metal stresses in yeast, consequence better tolerance to higher concentration of different metal ions in Arabidopsis plants, consequence abundance of metal transporters transcripts under metal stress conditions, consequence abundance of cation exchanger transporter transcripts under metal stress conditions, consequence higher accumulation of macro-elements under different metal stress, consequence higher accumulation of micro-elements under different metal stress, overexpression of HAC1, enhanced stress signal transduction, expanded signal output range of UPRE2m, use of UPRE2m, increased recombinant protein production, increased natural product synthesis, overexpression of gene AoErg24A, increased ergosterol content, overexpression of gene AoErg24B, decreased ergosterol content, expression of gene TDH3, mutation in promoter of gene TDH3, mutation disrupting binding sites for transcription factors, mutation disrupting TATA box, overexpression of gene AcFMO, catalyzes the synthesis of alliin by S-allyl-l-cysteine, deletion of gene fol3, increased OA titer, deletion of gene abz1, deletion of gene abz2, regulation of gene fol3 by malonyl-CoA, improved sustainable OA production, regulation of gene abz1 by malonyl-CoA, regulation of gene abz2 by malonyl-CoA, regulation of gene acc1 by malonyl-CoA, expression of gene ARO8, highest normalized gene expression in SC1, SC4, SC5 samples, expression of gene ARO9, expression of gene ADH2, no significant difference among Mp strains, expression of gene ADH5, no expression in Mp strains except MP6, higher in Sc strains compared to Mp strains, expression of tRNA-Leu variant, defects in fluorescent protein production, confirmed mistranslation with Leu, expression of tRNA-Leu anticodon variant, dramatic reductions in growth, expression of tRNA-Ala anticodon variant, more tolerable than Leu mis-incorporation, overexpression of gene Cox20, improved respiratory growth of COX2W56R strain, overexpression of gene Oxa1, overexpression of gene Pse1, deletion of gene MGR2, altered Cox2W56R import, overexpression of gene MGR2, overexpression of gene NtWRKY70b, consequence increased levels of reactive oxygen species (ROS, consequence decreased H2S content, disruption of gene NtWRKY70b, consequence decreased accumulation of ROS, consequence elevated concentration of H2S, consequence upregulated expression of senescence-associated genes, consequence upregulated expression of ROS biosynthesis-related genes NtRbohD and NtRbohE, consequence inhibited expression of H2S biosynthesis-related genes NtDCD and NtCYSC1, NtWRKY70b, consequence activated transcript levels of NtRbohD, consequence activated transcript levels of NtPPH, consequence repressed expression of NtDCD, consequence repressed expression of NtCYCS1, expression of gene EcOSC, production of skeleton products 1-3, expression of gene CYP716s, production of minor oxidation products 4-9, expression of gene CYP716E60, production of 6β-OH derivatives, presence of compound 9, inhibition of NO production, reduction of IL-1β production, reduction of IL-6 production, presence of compounds 1, 2, 8, 9, increase in HepG2 cell survival rate, expression of gene ycf1, consequence oxidative stress, consequence mutagenic stress resistance, overexpression of gene involved in oxidative stress, consequence changes in gene expression patterns, overexpression of gene involved in mutagenic stress, expression of sugar transporter-encoding genes, higher expression in LBCM31, expression of genes involved in iron metabolism, higher in LBCM109, expression of genes promoting accumulation of sphingolipids, expression of genes promoting accumulation of phospholipids, expression of genes promoting accumulation of ergosterol, heterologous expression of fsr1, bostrycoidin biosynthesis, heterologous expression of fsr2, heterologous expression of fsr3, heterologous expression of 6MSAS, 6-methylsalicylic acid biosynthesis, overexpression of TGL4, no effect on polyketide production, overexpression of AOX2, expression of genes responsible for ammonium transport in Chlamydomonas reinhardtii, increased inorganic nitrogen availability, expression of genes responsible for CO2 enrichment mechanism in Chlamydomonas reinhardtii, not the main reason for growth enhancement, expression of genes responsible for glycolysis in Saccharomyces cerevisiae, increased energy availability for Chlamydomonas reinhardtii growth, expression of genes responsible for stress responses in Saccharomyces cerevisiae, improved growth conditions for Chlamydomonas reinhardtii, levan treatment, increased yeast viability, prevented accumulation of reactive oxygen species, prevented accumulation of malondialdehyde, prevented accumulation of protein carbonyl content, ameliorated reduced mitochondrial membrane potential, ameliorated loss of plasma membrane integrity, ameliorated externalization of phosphatidylserine, modulated expression of genes in TORC1/Sch9 pathway, extended CLS of sod2Δ mutant, extended CLS of pep4Δ mutant, extended CLS of pkh2Δ mutant, extended CLS of rim15Δ mutant, extended CLS of atg1 mutant, extended CLS of ras2Δ mutant, did not affect CLS of tor1Δ mutant, did not affect CLS of sch9Δ mutant, overexpression of gene SmMYB113, increased flower drop rate, reduced fruit yield, activation of gene SmACS1 by SmMYB113, increased ethylene biosynthesis, activation of gene SmACS8 by SmMYB113, activation of gene SmACO4 by SmMYB113, interaction between SmMYB113 and SmERF38, enhanced activation of SmACS1 promoter, no direct consequences provided, deletion of gene PFK1, higher oxidative tolerance, slower growth in lag and exponential growth stages, higher growth in stationary stage, decelerated glucose consumption rate, decreased expression of HXT2, decreased expression of HXT5, decreased expression of HXT6, increased expression of ZWF1, higher residual glucose in medium, lower ethanol yield in early stage, decreased expression of PDC5, decreased expression of PDC6, lower interspecies competitiveness, increased ethanol production efficiency in stationary stage, expression of BmXyn11A xylanase, improved GX hydrolysis, expression of XylA β-xylosidase, expression of Agu115 α-methyl-glucuronidase, no additional growth benefit, co-expression of complementary enzymes, boosted yeast growth, increased ethanol fermentation, external enzyme supplementation, overexpression of shikimic acid pathway genes, increased precursor supply, overexpression of chorismic acid pathway genes, enhanced expression of CIT, increased intracellular acetyl-CoA content, enhanced expression of MAC1/3, enhanced expression of CTP1, enhanced expression of YHM2, enhanced expression of RtME, enhanced expression of MDH, synthesis of trans-cinnamic acid by ZmPAL, enhanced chrysin production, synthesis of chrysin through ScCPR-EbFNSI-1 and PcFNSI, increased chrysin concentration, deletion of gene rgt1, consequence ght3 transcriptional change, consequence ght4 transcriptional change, consequence ght6 transcriptional change, consequence ght7 transcriptional change, overexpression of gene rgt1, deletion of gene ZWF1, reduced [NADPH] level, disturbed growth on glucose, disturbed growth on xylose, replacement of gene TDH3 with GDP1, increased xylose consumption rate after glucose depletion, increased ethanol yield, modulation of expression of ZWF1, optimized competitive equilibrium between glucose-xylose co-metabolism, use of EMP instead of PPP, reduced wasteful metabolic cycle, reduced CO2 release, yeast supplementation, higher final body weights, weight gains, specific growth rates, garlic supplementation, yeast + garlic supplementation, elevated levels of reduced glutathione, activities of catalase, activities of superoxide dismutase, structural enhancements in liver, structural enhancements in intestine, increased expression of igf-1, increased expression of ghr1, increased expression of il-10, increased expression of lyz, increased expression of hep, improved resistance to A. sobria infection, improved survival rates, lower mortality, deletion of gene tsa1, growth defect, mutation in gene cdc19, reduced activity of Pyk1, depletion of glycolytic intermediates, restriction of amino acid synthesis pathways, decreased activity of fructose-bisphosphate aldolase 1, deletion of gene expression by limonene, growth inhibition of Candida spp., potentiation of Fluconazole effect, reduction in morphological transition from yeast to hyphae, deletion of gene HcSE1-2, low yield of cucurbitacin IIa, deletion of gene HcOSC1-6, deletion of gene HcCYP87D20, deletion of gene HcCYP81Q59, deletion of gene HcAT1, expression of gene HcSE1-2, increased production of 11-carbonyl-20β-hydroxy-Cuol, expression of gene HcOSC1-6, expression of gene HcCYP87D20, expression of gene HcCYP81Q59, expression of gene HcAT1, silencing of gene GaKAN2, decrease in single-cell trichomes, reduction in internodal distance, expression of gene GaKAN2, regulation of trichomes development through GaERS, enrichment in biosynthesis of secondary metabolites, enrichment in plant hormone signal transduction pathways, deletion of gene ino2, increase in lycopene yield, deletion of gene ino4, overexpression of gene ino2, no promotion of lycopene accumulation, decrease in intracellular free fatty acids, glycerol addition, enhanced CK production, up-regulation of MVA pathway genes, up-regulation of UDPG synthesis pathway genes, glycerol utilization, increased terpene synthesis pathway flux, facilitated production of glycosyl donors, deletion of gene atpD, consequence reduced acid production, deletion of gene eno, expression of gene PwNAC31, consequence upregulation of drought-responsive genes DREB2A and ERD1, consequence enhancement of seed vigor and germination rates in atnac072 Arabidopsis mutants, consequence restoration of hypersensitive phenotype under drought stress in atnac072 Arabidopsis mutants, interaction of protein PwNAC31 with DREB2A, consequence regulation of ERD1 expression, interaction of protein PwNAC31 with ABF3, consequence not specified, deletion of Ca2+, higher expression of SOD1, GSH1, GPX2 genes, availability of Ca2+, higher ethanol output, higher glycerol output, lower trehalose synthesis capacity, higher intracellular reactive oxygen species content, lower intracellular malondialdehyde content, higher superoxide dismutase activity, higher catalase activity, higher glutathione content, higher expression of antioxidant-related genes except SOD1 and CTT1, deletion of sterol 22-desaturase gene, increased campesterol production to 6.6 mg/L, expression of sterol Δ7-reductase, increased campesterol production to 88.3 mg/L, enhanced expression of sterol Δ7-reductase, increased campesterol titer to 438.28 mg/L in shake flask, increased campesterol titer to 1.44 g/L in 5 L bioreactor, ZNF598 overexpression, aborts stalled translation of mitochondrial outer membrane-associated mRNAs, removes faulty translation products, improves mitochondrial health, improves tissue health, deletion of Gal4 DNA binding, decrease in transcription, expression of Gal4 DNA binding, increase in transcription, deletion/mutation of gene SLC12A5, reduced cell surface expression of KCC2, decreased activity of KCC2, expression of gene SLC12A5, increased stability of KCC2 in primary rat neurons, increased glycosylation of KCC2 in ER and Golgi, overexpression of gene xylA, consequence xylose utilization, overexpression of gene Xltr1p, consequence enhance xylose utilization, overexpression of gene ACS, consequence promote xylose utilization and ethanol yield, overexpression of gene ADH, overexpression of gene TKL, over-expression of gene TaSnRK1α, increase in starch content, enlargement of starch granules, restoration of starch accumulation, increase in AGPase activity, deletion of genes involved in protein biosynthesis, moderate downregulation, expression of secretory pathway genes, regulation seemingly detrimental to protein secretion, overexpression of storage lipids, preparation for long-term survival, overexpression of storage carbohydrates, expression of pathway genes, improved pathway flux, doubled production titers of astaxanthin, expression of XYL1 from Scheffersomyces xylosifermentans, improved anaerobic growth on xylose, expression of XYL genes from Scheffersomyces stipitis, no improved anaerobic growth on xylose, high aeration, decrease in enzyme activity related to xylose metabolism, decrease in gene expression related to xylose metabolism, oxygen levels, affect xylose metabolism balance, over-expression of TaHXK7-1A, increased O₂·⁻ content, increased malondialdehyde content, up-regulated RBOHs genes, inhibited growth of Arabidopsis seedlings, increased ROS accumulation, gene silencing of TaHXK7-1, decreased O₂·⁻ content, decreased malondialdehyde content, down-regulated RBOHs genes, improved drought resistance of wheat, knockdown of gene Eo-eIF5A, inhibited +1 PRF of the η-tubulin gene, expression of gene Eo-eIF5A<sup>H</sup>, promoted +1 PRF at the canonical AAA_UAA frameshifting site of Euplotes, deletion of gene OCA6, increased secreted laccase activity, deletion of gene PMT1, deletion of gene PMT2, deletion of gene SKI3, deletion of gene SCJ1, decreased secreted laccase activity, overexpression of gene SCJ1, detrimental effect on laccase secretion, deletion of gene CGB, hyper-susceptibility to Pseudomonas syringae, loss of tRNA thiolation, deletion of gene CTU2, compromised transcriptome reprogramming during immune responses, compromised proteome reprogramming during immune responses, reduced translation of gene NPR1, compromised salicylic acid signaling, methylation of histone H3 on lysine-4 residue, facilitation of transcriptional output of the genome, establishment of cellular identity, deletion of gene RPG, consequence higher oxidant sensitivity, consequence increased H2O2 concentrations, consequence elevated catalase activity, consequence higher SOD activities, consequence increased transcription of CTT1, consequence increased transcription of CTA1, consequence enhanced ribosomes' ability to translate mRNAs associated with oxidative stress-related genes, consequence cleavage of 25S rRNA, consequence altered translation efficiency of CTT1 mRNA, consequence altered translation efficiency of CTA1 mRNA, expression of gene xylA, increased xylose utilization, expression of homologous genes related to xylose utilization, increase in expression level of pathways associated with sugar metabolism, improved sugar utilization, initial increase followed by decrease in genes related to glycerol lipid metabolism, optimized metabolic pathway, deletion of genes, genome restructuring, overexpression of BnaGPAT9-C1, increased erucic acid content, depletion of gene ASF1, decrease in replicative histones at protein level, decrease in replicative histones at RNA level, increase in nascent/newly synthesized RNAs, 3' end processing defect of replicative histone RNAs, increase in pre-mRNAs/unprocessed transcripts, production defect of replicative histone RNAs, general discussion on RNA production and adaptation in yeast Saccharomyces cerevisiae, expression of AnanC4H, increase in lignin content, increase in number of lignified cells in stems, deletion of gene related to lactate metabolism, consequence differential gene expression, deletion of gene related to iron uptake, deletion of gene related to cell wall structure, deletion of gene TCF25, mild negative effects on cell growth, post-translational protein degradation by the proteasome, aging leads to reduced mRNA levels of RQC genes, reduced mRNA levels in specific tissues, deletion of translation initiation, consequence NAC and Zuotin can still cross-link to each other, presence of NAC, consequence central globular domain of NAC is modestly shifted, presence of Hsp70, consequence Hsp70 can situate with peptide binding site at the tunnel exit, presence of Zuotin, consequence J-domain of Zuotin is appropriately positioned to drive stabilization of nascent chain binding, deletion of gene Zn2+, consequence oxidation of thioredoxins in vivo, consequence induction of oxidative stress, consequence interaction with multiple metals including Mn2+ and Cu2+, expression of gene thiolutin, consequence direct inhibition of RNA polymerase II transcription initiation in vitro, consequence pause prone, defective elongation in vitro, consequence widespread Pol II occupancy changes in vivo, consequence Tor pathway inhibition in vivo, consequence stress induction in vivo, deletion of gene Cncat, impact on H2O2-induced oxidative stress, over-expression of gene CnCat, improvement in oxidative resistance, reduction in intracellular ROS accumulation, deletion of gene CRS1, reduced synthesis of CysSSH, reduced chronological aging, increased endoplasmic reticulum stress response, impaired mitochondrial bioenergetics, deletion of gene Mex67, consequence Dbp5 recruitment to mRNA is abolished, overexpression of gene Dbp5, consequence dominant-negative effect on tRNA export, binding of gene Dbp5 to Gle1, consequence required for tRNA export, interaction of gene Dbp5 with tRNA, consequence does not activate Dbp5 ATPase activity alone, interaction of gene Dbp5 with Gle1 and tRNA, consequence fully activates Dbp5 ATPase activity, expression of gene green fluorescent protein, consequence carbon-source dependent expression, consequence inulin favorable carbon source, consequence high correlation between in vivo expression and sucrose condition, deletion of gene SAM1, altered chromosome stability, decreased AdoMet concentration, deletion of gene SAM2, increased AdoMet concentration, increased sensitivity or resistance to azoles, increased sensitivity or resistance to cisplatin, increased sensitivity or resistance to oxidative stress, perturbations in arginine biosynthesis, altered sensitivity to DNA synthesis inhibitors, altered response to tamoxifen, deletion of RNA polymerase II activity, altered initiation efficiency, selective modulation of preference for initiating nucleotide, overexpression of guanosine-5'-triphosphate, increased initiation efficiency at first five transcript positions, overexpression of cytosine-5'-triphosphate, increased initiation efficiency at second position, overexpression of uridine-5'-triphosphate, general review of meiotic gene expression control, expression of gene PgfB, consequence heterologous production of nerol, expression of gene homologous protein of PgfB, expression of gene HAP2, activation of hybrid promoter, overexpression of promoter phy47, 2.93-fold higher GFP expression levels than GAP, application of UAS<sub>E</sub> and URS<sub>D</sub> to promoter GAP, 34% increase in expression level, application of UAS<sub>E</sub> and URS<sub>D</sub> to promoter ENO1, 18% decrease in expression level, construction of artificial promoters with regulatory sequence, 61% expansion of expression range, use of hybrid promoter variants, higher titers of pinene and rHuPH20 biosynthesis, expression of gene Lyz-1, production of N-acetylmuramoyl-L-alanine amidase, expression of gene Lyz-2, production of D-alanyl-D-alanine carboxypeptidase, expression of gene Lyz1, antimicrobial activity against Micrococcus luteus, antimicrobial activity against Salmonella typhimurium, higher concentration required to control Saccharomyces cerevisiae compared to Aspergillus oryzae, lower concentration required to disintegrate Gram-positive bacteria cells compared to Gram-negative bacteria cells, cloning of polypeptide-p gene, production of recombinant secretion protein, transformation into Saccharomyces boulardii, reduction in blood glucose level, expression of gene P450_cur, consequence increased 7β-hydroxylation of androstadienedione, consequence increased 7β-hydroxylation of dehydroepiandrosterone, consequence increased 7β-hydroxylation of androstenediol, consequence increased 7α-hydroxylation of 1-dehydrotestosterone, expression of gene CPR, consequence enhanced functional activity of P450_cur, deletion of gene CCL5, consequence downregulation in monocytes, deletion of gene IL6, deletion of gene NFKB, consequence downregulation in neutrophils, deletion of gene Rds2, expression of gene Rds2, deletion of gene Gsm1, expression of gene Gsm1, binding of protein Rds2, binding of protein Gsm1, cooperative binding of proteins Gsm1 and Rds2, binding of protein Ert1, deletion of gene GPA1, consequence slower cell-cycle progression, overexpression of gene GPA1, consequence increased mating efficiency,  deletion of gene THI3, decreased isoamyl alcohol production, lowered decarboxylase activity, reduced thiamine levels, reduced thiamine diphosphate (TPP, decreased isoamyl alcohol production under low-thiamine conditions, CRISPR-edited defects in PtrSWRA-PtrAREB1-2 pentamer, impaired drought tolerance, elevated expression of PtrHox2, improved drought tolerance, elevated expression of PtrHox46, elevated expression of PtrHox52, improved growth, expression of gene sHSP-HB8, consequence increased cell viability at 42°C, consequence 17.2% increase in cell density, expression of gene HSP12, consequence 19.8% increase in cell density, expression of gene HSP26, expression of gene HSP30, expression of gene HSP42, expression of gene ibpa-MB4, expression of gene PK, consequence enhanced protein quality at higher temperatures, expression of gene MDH, integration of 2μ sequence-based yeast episomal plasmid, increased nerolidol production, integration of yeast integrative plasmid, increased diversity in sesquiterpene productivities, increased gene copy number, increased valencene production, sesquiterpene synthase expression level increase, gene upregulation associated with glycolysis, elevated levels of fermentation products, gene upregulation associated with energy metabolism, reduction in glycolytic activity, diminished ATP supply, formation of flocs, protection against toxic chemicals, variations in the nucleotide sequences of the flocculin genes and their regulators, distinct flocculation traits, expression of gene udh, consequence increased mucic acid production, consequence improved productivity up to 0.83 g/L-h, consequence theoretical yield of 0.18 g/g using pectin-containing citrus peel waste, utilization of xylose as co-substrate, consequence overcoming NAD+ dependency for pectin to mucic acid conversion, deletion of TORC1, consequence 1: reduced cellular anabolism, consequence 2: reduced cellular proliferation, overexpression of Sfp1, consequence 1: suppression of temperature-sensitive TORC1 mutant phenotype, phosphorylation of Sfp1 by TORC1, consequence 1: protection of Sfp1 from proteasomal degradation, positive regulation of genes involved in ribosome production by Sfp1, consequence 1: increased ribosome biosynthesis, interaction of Ifh1 with Fhl1, consequence 1: regulation of Ifh1 nuclear localization in response to nutrient levels, regulation of Ifh1 nuclear localization by TORC1 and Sfp1, consequence 1: modulation of ribosome biosynthesis genes expression, deletion of mRNA by nuclear exosome-NNS pathway, consequence 1: suppression of mRNAs involved in protein production, consequence 2: suppression of mRNAs involved in cellular organization, consequence 3: suppression of mRNAs involved in metabolism, overexpression of CALCOCO2, promotion of BVDV propagation, knockdown of CALCOCO2, inhibition of BVDV replication, expression of CALCOCO2, inhibition of IFN-I production, inhibition of IFN-stimulated gene production, reduction of IRF3 protein levels, interaction of CALCOCO2 with IRF3, promotion of Npro-mediated ubiquitination degradation of IRF3, deletion of gene not specified, accumulation of toxic pyrophosphate intermediates, expression of MVA pathway enzymes in mitochondria, increased terpenoid production, accumulation of mono- and pyrophosphate compounds, mitigation of mono- and pyrophosphate compound accumulation, improved squalene titer, upregulation of gene SDS3, improved biocontrol efficacy, upregulation of gene DAL81, upregulation of gene DSE1, upregulation of gene SNF5, upregulation of gene SUN41, upregulation of gene FLO8, upregulation of gene HOP1, deletion of gene eGFP, repression efficiency from 62% to 177%, overexpression of gene eGFP, activation efficiency from 154% to 320%, expression of Wolbachia effector proteins in S. cerevisiae, expression of CI-related proteins in S. cerevisiae, expression of genes involved in pullulan synthesis, increased pullulan production, expression of genes involved in melanin synthesis, increased melanin production, expression of genes involved in polymalic acid synthesis, increased polymalic acid production, overexpression of RAS2S225A, mislocalisation, constitutive activation, slow growth, loss of respiration, reduced stress response, quiescence, overexpression of RAS2S225E, deletion of CUP9, rescue of quiescence, addition of peptone, deletion of gene Fpt1, reduction of RNAPIII eviction, impairment of shutdown of ribosome biogenesis genes upon nutrient perturbation, deletion of gene involved in Cd tolerance, deletion of gene involved in Zn tolerance, deletion of gene involved in Fe tolerance, deletion of gene involved in Mn tolerance, deletion of gene involved in B tolerance, deletion of gene involved in Cu tolerance, expression of GPCR signaling pathway, reshaped yeast galactose regulon, expression of synthetic transcription factor, coupled GAL system with GPCR signaling pathway, modulation of key components of GPCR signaling cascade, refabricated dynamic range, sensitivity, and response time of the GAL system, expression of α factor, achieved quorum-sensing behaviors, repurposing of GAL system, made responsive to heat shock, S-adenosyl-l-methionine usage by S. cerevisiae, counteracts acid production of L. plantarum, S. cerevisiae impacts gene expression, upregulates protein synthesis pathways, upregulates cell growth pathways, activates ribosome pathway, activates SNARE interactions pathway, activates basal transcription factors pathway, activates MAPK signaling pathway, deletion of subcellular localization signal of gene T16H, no effect on gene expression levels, expression of peroxisome-localized T16H, highest transformation efficiency, expression of T16H in different organelles, varying conversion rates from tabersonine to 16-hydroxy tabersonine, expression of gene CENP-A, consequence reduced histone humanization efficiency, consequence toxic to yeast, consequence disrupted cell cycle progression, consequence disrupted kinetochore function in wild-type cells, deletion of gene subunits of 3 conserved chromatin remodeling complexes, consequence suppressed CENP-A toxicity, amplification of gene GAP1, higher RNA abundance, reduced effect at proteome level, involvement of post-transcriptional dosage compensation, genes with different translational efficiency, enrichment for upstream open reading frames, enrichment for RNA binding sites for SSD1, genes with lower protein expression efficiency, enrichment for genes encoding proteins in protein complexes, deletion of interleukin-1β transcription, effect on distal gut inflammation, deletion of interleukin-6 mRNA, effect on proximal gut inflammation, no change in TNF-α mRNA, no effect on inflammation, deletion of gene tHMG1, decrease in carotenoid productivity, deletion of gene ERG8, overexpression of gene tHMG1, increase in carotenoid productivity, overexpression of gene ERG8, expression of gene BRCA2, consequence increased radiation resistance, consequence increased DSB repair frequency, deletion of gene Rad52, consequence reduced DSB repair efficiency, consequence improved protection against ionizing radiation by at least 1.93-fold, consequence increased repair frequency by 6.1-fold, deletion of gene involved in cell membrane permeability, consequence 1: reduction in permeability and integrity of cell membrane, overexpression of gene involved in ergosterol synthesis, consequence 1: increase in ergosterol proportion in cell membrane, overexpression of gene involved in unsaturated fatty acid synthesis, consequence 1: increase in unsaturated fatty acids proportion in cell membrane, overexpression of DEGs in ergosterol synthesis pathway, consequence 1: increase in ergosterol synthesis, overexpression of DEGs in linoleic acid synthesis pathway, consequence 1: increase in linoleic acid synthesis, activation of cell wall integrity (CWI) pathway, consequence 1: enhanced cell wall integrity, activation of high osmolarity glycerol mitogen-activated protein kinase (HOG-MAPK) pathway, consequence 1: enhanced osmotic stress response, deletion of gene rer2, growth defect at higher temperature, expression of DHDDS-like protein, production of medium-chain C55 product, expression of DHDDS-like protein and NgBR homologue, production of longer-chain products, hypomodification of tRNA core, increased translational errors, modifications of tRNA anticodon loop, modulate accuracy, tRNA core modifications, stabilize tRNA structure, stabilizing tRNA structure, avoid degradation, degradation of tRNA, increased at high growth temperatures, introduction of mysA, introduction of mysB, introduction of mysC, introduction of mysD, introduction of mysE, overexpression of GND1, overexpression of ZWF1, introduction of xylose utilization pathway, strengthening promoters, increasing gene copies, CRISPR-Cpf1 genome-editing system in Saccharomyces cerevisiae, plasmid curing efficiency 96 ± 4%, transforming only one crRNA into S. cerevisiae, curing efficiency about 70%, CRISPR-Cas editing technology, produces traceless products without foreign genes, deletion of gene intron splicing efficiency, consequence reduced splicing, overexpression of gene intron splicing efficiency, consequence increased use of cryptic splice sites, mutation in gene intron splicing efficiency, consequence formation of RNA structures that pair N50 bases, deletion of gene xxx, disruption at high efficiency, expression of heterologous protein, integration from a safe chromosomal harbour site, expression of foreign genes, impaired by endogenous factors, facilitated by promoter engineering, improved by codon optimization, enhanced by signal peptide fusion, gene disruption, performed by CRISPR/Cas, foreign gene insertion, site-directed mutation, expression of biosynthetic genes, production of Orsellinic acid, production of Colletochlorins, production of Colletorins, overexpression of gene OsGSTZ4, conferred cellular Cd tolerance, improved plant growth, attenuated Cd-induced toxicity, accumulated more Cd in roots, induction of gene OsGSTZ4, CHH hypermethylation, detection of 24-nt siRNAs, loss of function of gene osdrm2, compromised CHH methylation, defective in gene osrdr2i, promoter deletion of gene OsGSTZ4, essential for proper expression, deletion of gene GCN4, heightened mitochondrial activity, increase in reactive oxygen species (ROS, reduced ethanol production, reduction in trehalose, disparities in free amino acids, disrupted cellular homeostasis, shortened lifespan, deletion of inflammatory response genes, increased plasminogen expression, increased glutathione metabolism genes expression, increase in cembratriene-ol titer by 1.84-fold, overexpression of gene cbts*Δp, increase in cembratriene-ol titer by 2.22-fold, overexpression of gene ERG9, increase in cembratriene-ol titer by 1.80-fold, increase in cembratriene-ol titer by 26.1-fold, overexpression of gene ggpps, mitochondrial compartmentalized expression of gene cbts*Δp, increase in cembratriene-ol titer by 76.3-fold, mitochondrial compartmentalized expression of gene ggpps, deletion of gene ADE2, expression of gene ErCas12a, decreased maximum specific growth rate on glucose, deletion of transcription factors, rapid destabilization of existing ribosomal protein transcripts, gradual slowing of new transcript production, enhanced raw starch conversion, 36% higher ethanol concentration, allotopic expression of mitochondrial genes, adaptation of codon usage, inclusion of sequences encoding mitochondrial targeting signals, consideration of average apparent free energy of membrane insertion, consideration of final functional topology, consideration of protein translocator TIM23 sorting mechanism, increase in expression of genes related to key components of membrane microdomains, improvement in acid tolerance, knockout of gene ERG6, reduction in acid tolerance, disruption of membrane microdomains, inhibition of the formation of the H+ ATPase complex, deletion of gene Not4, disruption of ribosomal protein homeostasis, overexpression of gene Not4, inhibition of 40S ribosomal autophagy, interaction of gene Not4 with endolysosomal pathway, control of RP expression, expression of gene TUB1, consequence cell fitness during microtubule stress, consequence promotes α-tubulin expression, consequence requires chromatin remodeling genes, consequence requires genes involved in α/β-tubulin heterodimer assembly, consequence requires genes involved in spindle assembly checkpoint, expression of gene AG1IA_06082, decreased sensitivity to fungicides, expression of gene AG1IA_08645, expression of ABC transporter-encoding genes, increased MDR, expression of MFS transporter-encoding genes, overexpression of gene AG1IA_06082, high accumulation of SYP-14288, overexpression of gene AG1IA_08645, deletion of gene in glycolysis pathway, lower expression in W303a, overexpression of gene ARO9, increased phenylalanine production in CEN.PK2-1C, overexpression of gene ARO7, overexpression of gene PHA2, overexpression of seven DEGs associated with phospholipid biosynthesis, increased phospholipids production in W303a, expression of gene 4CL, consequence increased coumaroyl-CoA production, expression of gene BAS, consequence increased raspberry ketone synthesis, expression of gene 4CL and BAS, consequence production of over 30 μg/g raspberry ketone in N. benthamiana, consequence production of up to 60 mg/L raspberry ketone in S. cerevisiae, deletion of gene TOLLIP, consequence decrease in doxorubicin resistance, consequence increase in sensitivity to doxorubicin, overexpression of gene TOLLIP, consequence increase in doxorubicin resistance, deletion of nitrogen, consequence increased total carbohydrate percentage, consequence increased lipid percentage, consequence increased lipid productivity, overexpression of delta-9 fatty acid desaturase gene, consequence increased oleic acid level, consequence increased palmitoleic acid level, deletion of TNFα gene, reduction of inflammation in non-stimulated colon cells, reduction of inflammation in lipopolysaccharide-stimulated colon cells, deletion of gene FYV6, decreases splicing of reporter genes, accumulates first-step products, defect in exon ligation, changes 3' splice site selection in SUS1 transcript, deletion of gene RAS/PKA, consequence diminished stress tolerance, deletion of gene Snf1, deletion of gene HOG, overexpression of metabolic enzymes, consequence enhanced xylose fermentation, deletion of gene GCNT3, downregulation of mucin type O-glycan biosynthesis, deletion of gene GALNT5, deletion of gene B3GNT3, deletion of gene GALNT18, deletion of gene GALNT14, deletion of gene COL6A1, downregulation of cell and tissue structure genes, deletion of gene COL1A1, deletion of gene COL4A1, deletion of gene COL1A2, deletion of gene COL6A2, deletion of gene LAMB2, deletion of gene ITGA8, deletion of gene ITGA2, deletion of gene ITGA5, deletion of gene CCL16, downregulation of chemokines, deletion of gene CCL21, deletion of gene CCL14, deletion of gene CXCL12, deletion of gene CXCL14, overexpression of gene GSTA1, upregulation of glutathione metabolism, overexpression of gene RRM2B, deletion of gene GSTM1, downregulation of glutathione metabolism, deletion of gene GPX8, no specific consequences related to gene expression/deletion, decrease in expression of ATF1 gene, decrease in enzyme synthesis, transcriptional repression of ATF1 gene, decrease in acetate production, expression of ATF1 gene, regulated by ROX1, regulated by RAP1, expression of Pho89, control by Crz1, deletion of TRPCa7-like, impaired expression of Pho89, expression of SOD, consequence 1: increased antioxidant enzyme activity, expression of CAT, expression of GR, decrease of ROS, consequence 1: reduced oxidative stress, decrease of MDA, reduction of oxidative stress, consequence 1: elevated ethanol fermentation efficiency, deletion of introns, introduction of introns, enhancement of gene expression, increased efficacy of recombinant expression cassettes, deletion of gene variants, inactive genes, expression of gene variants, unique pathway configuration, supply of gRNAs, activation of gene expression, activation of chromosomal repair pathway, self-assembly of pathway, generation of pathway configurations, distinct color phenotypes, carotenoid product profiles, deletion of gene Mec1p, consequence downregulates efficiency of transcription termination, consequence reduces efficiency of pre-mRNAs cleavage at the pA sites, recombinant rhfFSH expression, stimulated E2 levels, upregulated aromatase gene expression, recombinant rhfLH expression, deletion of RQC components, activation of stress response, neurodegenerative diseases, regulation of protein synthesis, prevention of aberrant or unnecessary proteins production, overexpression of gene involved in trafficking of vesicles, increased α-amylase secretion, deletion of gene involved in trafficking of vesicles, overexpression of gene involved in endosome to Golgi transport, deletion of gene involved in endosome to Golgi transport, overexpression of gene at phagophore assembly site, deletion of gene at phagophore assembly site, overexpression of gene involved in cell cycle, deletion of gene involved in cell cycle, overexpression of gene involved in energy supply, deletion of gene involved in energy supply, no expression/deletion of gene mentioned, improvement in prediction of protein abundance, improved correlation with CAI, expression of multiple genes, efficient polycistronic gene expression, expression of 13 biosynthetic genes, production of 1,090.41 ± 80.92 mg L<sup>-1</sup> squalene, production of 1.04 ± 0.02 mg L<sup>-1</sup> mogrol, deletion of ribosomal proteins, consequence high-level cotranslational protein degradation, overexpression of ribosomal proteins, consequence increased cotranslational folding efficiency, fusion of ubiquitin to ribosomal proteins, consequence protection from cotranslational protein degradation, expression of GmPHT2;2, higher phosphorus content in chloroplasts, higher whole-plant phosphorus content, expression of GmPHT2;1, lower phosphorus content compared to GmPHT2;2, overexpression of GmPHT2, increased phosphorus utilization in yeast, up-regulation of GmPHT2 by low-temperature stress, implication in low-temperature response, expression of GmPHT2;1 and GmPHT2;2, cyclic pattern in response to light, localization of GmPHT2 protein, chloroplast, overexpression of gene ClNAC100, induced by phosphate deficiency, positively regulated by ABA, negatively regulated by JA, highly expressed in roots, highly expressed in leaves, deletion of gene Opi1, hypersensitivity to genotoxins, delayed G1-to-S-phase transition, decreased gamma-H2A levels, increased mitochondrial DNA instability upon MMS treatment, activation of transcription factor Ino2-Ino4, responsible for genotoxin sensitivity in Opi1-deficient cells, production of inositol pyrophosphates by Kcs1, counteracts Opi1 function during MMS-induced stress, overexpression of gene IME4, elevation of mRNA levels of methylated genes in glycolysis, elevation of mRNA levels of methylated genes in acetyl-CoA synthesis, elevation of mRNA levels of methylated genes in shikimate/aromatic amino acid synthesis, induction of gene ACS1, induction of gene ADH2, increase in titers of isoprenoids, increase in titers of aromatic compounds, high gene expression difference, abiotic stress tolerance enhancement, cell wall/carbohydrate synthesis enhancement, biostimulant coating seed treatment, similar pathway enhancement, common genes activation, plant protection from abiotic stresses, higher level of sugars in plant, LPS injection, upregulation of some tissue inflammation-related genes, downregulation of intestinal tight junction-related genes, elevation of liver phospho-nuclear factor kappa B (p-NF-κB, elevation of phospho-inhibitory subunit of NF-κB (p-IκBα, elevation of phospho-c-Jun N-terminal kinase (p-JNK, elevation of Nuclear factor kappa-B (NF-κB, elevation of the inhibitory subunit of NF-κB (IκBα, maternal dietary supplementation with XPC, reduction of inflammatory factors in plasma and thymus, downregulation of IL-6 gene expression in thymus, downregulation of IL-10 gene expression in thymus, decrease of JNK protein expression in liver, deletion of gene Saccharomyces cerevisiae terminator, consequence promoter strength increase, consequence terminator strength increase, overexpression of gene TATA box-like sequence, consequence adequate execution of both functions, regulation of gene UAS, consequence promoter-like strength increase, regulation of gene spacer sequence, consequence terminator-like strength increase, application of bifunctional elements, consequence improved pathway assembly efficiency, consequence higher lycopene yield, deletion of nitrogen source, consequence 1: activation of transcription by GATA family transcriptional activators, consequence 2: recruitment of SAGA chromatin remodeling complex, inhibition of glutamine production, deletion of pyrimidine dimers by TC-NER, removal from transcribed strand of active genes, deletion of pyrimidine dimers by GG-NER, removal from inactive genome, UV irradiation, increase in RNA polymerase-I density at 5'-end of rRNA genes, elongation and pausing/release of RNA polymerase-I at PDs, formation of discrete chromatin structures, replacement of RNA polymerase-I by nucleosomes, participation of GG-NER in PD removal from TS, density of RNA polymerase-I, faster removal of PDs in 5'-end than 3'-end of gene, GG-NER, sole responsible for PD removal from non-transcribed strand of rRNA gene coding regions, chromosomal deletion of gene STB5, enhances autophagy, loss of gene Stb5, upregulation of ATG transcripts, Stb5-mediated impact, effect on genes involved in NADPH production, effect on genes involved in pentose phosphate pathway, osFAR1 overexpression, increased production of primary alcohol, enhanced amounts of C22:0 primary alcohol, enhanced amounts of C24:0 primary alcohol, enhanced amounts of C24:0 to C32:0 primary alcohols, increased content of C24:0 to C30:0 primary alcohols, reduced cuticle permeability, enhanced drought tolerance, deletion of gene gRNAs, consequence increased guide RNA cutting efficiency, expression of gene gRNAs, consequence integration efficiency of a gene cassette, consequence improved cellular fitness, consequence increased GFP expression level, expression of five genes, consequence improved production of 5-aminolevulinic acid, expression of five gene cassettes, consequence production of 9 g/L succinic acid, deletion of gene PDR1, reduced yeast resistance to tyrosol, deletion of gene PDR3, expression of gene PDR5, contributed to yeast resistance to tyrosol, tyrosol, inhibited efflux of rhodamine 6G, induced MDR, Pdr5-GFP levels, increased, reduced yeast ability to accumulate Nile red, inhibited cytostatic effect of clotrimazole, change in AdoMet concentrations, sensitivity or resistance to azoles, sensitivity or resistance to cisplatin, sensitivity or resistance to oxidative stress, sensitivity or resistance to DNA synthesis inhibitors, sensitivity or resistance to tamoxifen, overexpression of gene PAA1, acetylation of different aralkylamines, overexpression of gene HPA2, AANAT activity, exogenous expression of long-chain human insulin growth factor-1, promotion of cell proliferation, exogenous expression of platelet-derived growth factor-BB, exogenous expression of basic fibroblast growth factor, exogenous expression of epidermal growth factor, endogenous protease knockout, enhancement of cytokine stability, genomic co-expression, increased yield of cytokines, expression frame gene order optimization, improved cytokine production efficiency, overexpression of gene YPR015C, increased resistance to furfural, improved enzyme reductase activity, accumulation of oxygen reactive species, involvement in amino acid biosynthesis, oxidative stress response, cell wall response, heat shock protein response, mitochondrial-associated protein response, deletion of gene CgMCUR1, reduced acetate tolerance, reduced H2O2 tolerance, reduced high temperature tolerance, expression of gene CgMCUR1, improved acetic acid tolerance, improved H2O2 tolerance, improved high temperature tolerance, enhanced intracellular proline accumulation, overexpression of gene CgMCUR1, affected proline metabolism, reduced cellular lipid peroxidation, altered SFA to UFA ratio, increased ethanol production, increased conversion rate, improved ethanol yield in undetoxified cellulose hydrolysate, improved conversion rate in undetoxified cellulose hydrolysate, codon pair usage patterns, modulation of translation efficiency, protein folding, mRNA decay, dicodon-based measure, greater correlations with gene expression level, dicodons associated with low adaptiveness, mediate strong translational inhibition in yeast, overexpression of gene LcMT3, resistance to salt in E. coli cells, resistance to salt in yeast, higher germination rates in transgenic plants, longer roots in transgenic plants, reduced accumulation of malondialdehyde in transgenic plants, reduced relative conductivity in transgenic plants, reduced reactive oxygen species in transgenic plants, increased concentration of proline in transgenic plants, increased relative water content in transgenic plants, increased chlorophyll content in transgenic plants, increased activity of superoxide dismutase in transgenic plants, increased activity of peroxidase in transgenic plants, increased activity of catalase in transgenic plants, reduced Na+ accumulation in transgenic plants, lower Na+/K+ ratio in transgenic plants, regulatory effect on transporter proteins SOS, regulatory effect on transporter proteins NHX1, expression of gene PfCOQ2, complementation of growth defect in coq2Δ mutant strain, production of ubiquinone UQ in coq2Δ mutant, detection of UQ when labeled with [ring-14C(U, deletion of gene x, increased NPA secretion, deletion of gene y, deletion of gene z, deletion of gene w, increased copy number of NPA expression cassette, increased NPA concentration in supernatant, use of native and synthetic secretion signals, tuned NPA secretion, deletion of gene GAL1p, consequence reduced toxic effect on host cell, expression of gene IdeS, consequence production of protease IdeS, expression of fused protein IdeS-GFP, consequence measurable fluorescence with flow cytometry, loss of gene DPH, increased resistance to sordarin, increased -1 ribosomal frameshifting, increased ribosomal drop-off during elongation, premature termination at out-of-frame stop codons, depletion of glucose, trigger environmental stress response, recurring glucose shortages, increase ribosome portfolio, expression of MCT1 gene, higher volatile fatty acid absorption by the rumen epithelium, decrease in expression of Protozoa, due to antimicrobial effects of turmeric and thymol, decrease in expression of Anaerobic fungi, decrease in expression of Prevotellaceae_UCG-004, due to antimicrobial modulation, decrease in expression of Bacteroidetes_BD2-2, decrease in expression of Papillibacter, decrease in expression of Schwartzia, decrease in expression of Absconditabacteriales_SR1, decrease in expression of Prevotellaceae_NK3B31_group, decrease in expression of Clostridia_UCG-014, decrease in expression of Fibrobacter succinogenes, increase in expression of Selenomonas ruminantium, increased activity of native PGAL1 by 131.20%, increased activity of heterologous PSkGAL2 by 72.45%, expression of gene KlLac9p, increased activity of PScGAL1 by 41.56%, increased activity of PSkGAL2 by 100.63%, optimized GAL expression system, increased production of β-carotene by 9.02-fold, upregulation of gene expression of key enzymes in fatty acid synthesis pathway, increase in POA titer, deletion of gene RAD6, consequence H2BK123 ubiquitylation impairment, mutation in gene RAD6, consequence similar transcriptome profile to H2B ubiquitylation site mutant, interaction between Rtf1 subunit HMD and Rad6, consequence stimulation of H2BK123ub, overexpression of gene OsDHN2, consequence improved Cd tolerance, consequence reduced Cd concentration in yeast, consequence increased expression levels of SOD1, consequence increased expression levels of CTA1, consequence increased expression levels of GSH1, consequence increased expression levels of CTT1, consequence increased antioxidant enzymatic activities, overexpression of key genes, removal of limitations in protein secretion, inappropriate gene expression, negative effects, overexpression of ERO1, increased α-amylase production, overexpression of SLY1, overexpression of gene CYP709C69, consequence diclofop-methyl resistance in plants, consequence production of hydroxylated diclofop acid in yeast, overexpression of gene CYP81A12/21, consequence production of hydroxylated diclofop acid, consequence presumed clomazone-activating function, consequence part of transcriptionally linked overexpression with CYP709C69, overexpression of gene GmC4H2, increased p-coumaric acid production, overexpression of gene GmC4H14, overexpression of gene GmC4H20, deletion of gene GmC4H10, no change in p-coumaric acid production, deletion of gene OsSMC5, deletion of gene OsSMC6, deletion of gene OsMMS21, introduction of PRK, increased acetaldehyde production, increased acetate production, reduction of RuBisCO copy number, decreased acetaldehyde production, decreased acetate production, C-terminal fusion of 19-amino-acid tag to PRK, use of ANB1 promoter for PRK synthesis, overexpression of PRK and RuBisCO, expression of plaA gene, increased production of plantaricin 423, expression of munST4SA gene, increased production of mundticin ST4SA, codon optimization of plaA gene, no significant increase in PlaX activity, codon optimization of munST4SA gene, eight-fold increase in MunX activity against Listeria monocytogenes, use of MFα1 secretion signal, superior secretion of bacteriocins compared to XYNSEC, HPLC-purification of codon-optimized genes, yield of 20.9 mg/L MunX, yield of 18.4 mg/L PlaX, production of PlaX, MIC of 1.18 µM against L. monocytogenes, production of MunX, MIC of 108.52 nM against L. monocytogenes, monitoring mitochondrial translation by labeling newly synthesized mitochondrial polypeptides with [S<sup>35</sup>]-methionine, knocking out the endogenous bypass gene, increased d-pantothenic acid production, adjusting the copy number of the pathway modules, balancing NADPH utilization, regulating the GAL inducible system, loss-of-function mutation in OsSPT4, severe reproductive growth defects, loss-of-function mutation in OsSPT5-1, deletion of OsSPT5-2, no phenotypes, mutation in OsSPT5-1, gibberellin-related dwarfed defects, weak root system, short life cycle, interaction between OsSPT5-1 and APO2, regulation of rice shoot growth, OsSPT5-1 overexpression, involvement in gibberellin pathway, involvement in auxin pathway, involvement in cytokinin pathway, None, identification of motifs bound by BpERF2, verification of motifs by yeast one hybrid, verification of motifs by electrophoretic mobility shift assay, verification of motifs by ChIP study, heterologous expression of gene CYP51, demethylates obtusifoliol, demethylates 4-desmethyl-24,25-dihydrolanosterol, identification of sterol side-chain reductase, reduces Δ24,28 double bonds, reduces Δ24,25 double bonds, reduction of cycloartenol to cycloartanol, initiates cholesterogenesis, expression of super-clogger ANT1 variant, neurodegeneration, age-dependent dominant myopathy, missense mutations in ADP/ATP translocase 1 (ANT1), protein accumulation along the protein import pathway, mutations in ADP/ATP carrier 2 (Aac2), obstruction of general protein translocation into mitochondria, impairment of mitochondrial respiration, impairment of cytosolic proteostasis, reduction in cell viability, double mutant Aac2/ANT1, severe clogging at the translocase of the outer membrane (TOM, severe clogging at the translocase of the outer membrane (TOM) complex, extreme toxicity in yeast, overexpression of gene SsWRKY1, higher antioxidant enzyme activity, higher proline content, lower malondialdehyde content, lower ROS accumulation, upregulation of six stress-related genes, deletion of gene AQPF, reduced hydrogen peroxide transport, impaired phenotypic growth, expression of gene ScFLO1, improved tolerance to d-lactic acid, increased specific growth rate, expression of gene LpDLDH, increased d-lactic acid production, upregulation of lactate transport genes, increased tolerance to d-lactic acid, upregulation of iron metabolism genes, expression of BfPMHA, increased tolerance to hypo-salinity, higher expression level with greater low salinity stress, overexpression of BfPMHA, enhanced yeast tolerance to NaCl stress, overexpression of BfFBA, overexpression of BfGAPDH, overexpression of BfMnSOD, deletion of gene T7RNAP in yeast, failure to create a functional T7RNAP-based expression system, overexpression of gene T7RNAP in yeast, potential increase in efficiency and processivity of yeast as an expression system, expression of gene KatA, increased catalase activity, maximum yield 338.8 ± 9.6 U/mL, specific activity 14826.58 U/mg, optimal activity at 25 °C and pH 11.0, Km for hydrogen peroxide 10.9 ± 0.5 mM, kcat/Km 5788.1 ± 25.6 s^-1 mM^-1, deletion of gene ssu1, increased intracellular selenomethionine content, expression of gene ssu1, selenium efflux process, mutation in gene fzf1, selenium tolerance, selenite, induces expression of ssu1, decarboxylase production by yeast, conversion of p-coumaric acid to vinylphenols, decarboxylase production by lactic acid bacteria, conversion of ferulic acid to vinylphenols, vinylphenol reductase action, production of 4-ethylphenol, production of 4-ethylguaiacol, deletion of gene sky1, pathways involved in the regulation of gene expression are downregulated, mitochondrial processes are upregulated, deletion of gene sky2, metabolic processes are affected, arginine-induced hyphae formation is impaired, upregulation of genes encoding hexose monophosphate biosynthesis pathway, RNA accumulation, upregulation of genes encoding sulfur-containing amino acid biosynthesis pathway, overexpression of gene Xks1, decreased ATP levels, decreased expression of gene Xks1, restored growth at high D-xylose concentrations, increased ATP levels, high rates of xylose metabolism, deletion of gene ERG5, thermotolerant capability improvement, deletion of gene ERG4, deletion of gene ERG3, increase in ethanol production, up-regulation of gene ERG1, up-regulation of gene ERG11, down-regulation of gene ERG9, down-regulation of gene ERG26, improvement in cell viability, expression of CHS, increase in anthocyanidin production, expression of F3H, expression of VmF3Hv1, best performance in anthocyanidin production, expression of VuCHS, deletion of gene FgIDOA, abnormal conidial morphology, reduced mycelial growth, decreased virulence in wheat heads, reduced deoxynivalenol accumulation, deletion of gene FgIDOA/B, NAD<sup>+</sup> auxotrophy, deletion of gene FgIDOB, shifts toward alternative TRP degradation pathways to melatonin and indole derivatives, deletion of gene BcTaf14, abnormal colony morphology, reduced conidiation, reduced virulence, altered responses to various stresses, mutation of gene BcTaf14 G80 and W81, affected regulatory effect on mycelial growth, affected regulatory effect on virulence, did not affect production and morphology of conidia, absence of ET domain in BcTaf14, unable to localize to nucleus, expression of BcTaf14 without ET domain, defects not recovered to wild-type levels, deletion of gene ipac2, viable virus formation, plaque formation similar to intact ipac2, expression of gene GFP, recombinant replication-competent ILTV generation, deletion of pro-growth mRNAs, controlled by Gcn2, controlled by Ccr4, phosphorylation of eIF2α, leads to induction of ISR, induction of ISR, results in translation of Gcn4, translation of Gcn4, does not necessarily result in canonical Gcn4-dependent transcription, overexpression of gene GhNRT2.1e, increased biomass, increased nitrogen accumulation, increased nitrate accumulation, increased nitrogen uptake, increased nitrogen utilization efficiency, increased nitrogen-metabolizing enzyme activity, increased amino acid content, silencing of gene GhNRT2.1e, suppressed nitrate uptake, suppressed nitrate accumulation, hampered plant growth, affected nitrogen metabolism processes, reduced tolerance to low nitrate, expression of gene Lg-FLO1, increased flocculation, expression of gene RIM15, regulates flocculation, nitrogen starvation, promotes flocculation, amino acid starvation, consequence: formation of the 3' end of a eukaryotic mRNA, consequence: cleavage and polyadenylation specificity factor (CPSF, consequence: regulation of cleavage site selection, consequence: control of poly(A, impact on mRNA stability and translational efficiency discussed, ribosomes control mRNA stability, impact of codon optimality, transfer RNAs affect mRNA stability, role as master gene regulators, translation elongation rate triggers mRNA decay, consequence of codon optimality, exogenous addition of 2-PE, promotion of ethanol production, reduction of negative impact of L. plantarum on S. cerevisiae, decrease in relative expression levels of ADH1, decrease in relative expression levels of MLS1, decrease in relative expression levels of CIT2, decrease in relative expression levels of IDH1, decrease in relative expression levels of CIT1, inhibition of glyoxylic and tricarboxylic acid cycles, promotion of ethanol accumulation, heterologous expression of mlrA, reduction of microcystins by 83%, expression of mlrA from plasmid, reduction in toxin content, linearization of microcystin structure by mlrA, reduction in toxicity and stability, deletion of gene DDGS, consequence no mycosporines production, deletion of gene OMT, deletion of gene ATPG, consequence accumulation of 4-deoxygadusol, expression of gene DDGS, consequence production of 4-deoxygadusol, expression of gene OMT, consequence production of MG, expression of gene ATPG, integration of cluster DDGS, OMT, ATPG, consequence production of MG, OMT, ATPG, consequence production of mycosporine glutaminol glucoside, deletion of gene MIG1, consequence upregulation of mycosporinogenesis, deletion of gene CYC8, deletion of gene OPI1, deletion of gene ROX1, consequence downregulation of mycosporinogenesis, deletion of gene SKN7, deletion of gene TUP6, consequence no effect on mycosporinogenesis, deletion of gene YAP6, RhPLATZ9-silenced, higher ROS contents, RhWRKY33a-silenced, overexpression of RhWRKY33a, delayed flower senescence, overexpression of RhPLATZ9, upregulated RhRbohs, RhPLATZ9, represses RhRbohD, deletion of gene eEF3, increase in product from downstream ORF, enhancement of release of luciferase from polysome by eRF1 and eRF3, expression of gene sensor array, consequence low background, consequence 40- to 5000-fold dynamic range, expression of gene heterologous pathway, consequence identification of bottlenecks in metabolic pathway, deletion of ppe-miR393a, increased PpTIR1, deletion of ppe-miR393b, overexpression of PpTIR1, increased interaction with PpIAA13, overexpression of PpIAA13, increased binding to PpACS1 promoter, increased transcription of PpACS1, overexpression of PpACS1, suppression of ppe-miR393a, decreased ethylene production, suppression of ppe-miR393b, overexpression of ppe-miR393a, overexpression of ppe-miR393b, increased fruit softening, suppression of PpIAA13, decreased fruit softening, deletion of gene Cdc73, consequence alteration of Paf1C-dependent histone modifications genome-wide, depletion of Spt6, consequence dissociation of Paf1 from chromatin, deletion of gene daf-2, consequence decreased ROS level, deletion of gene daf-16, consequence increased GSH content, consequence reduced intestinal damage, no specific gene product or metabolite consequences mentioned, deletion of gene luxS, deletion of gene pfs, expression of α-amylase, improved product titers, diminished production yields without control, reduced efficiency without control, deletion of gene ScNRK1, consequence enzyme activity in fermentation broth 14.75 IU/mL, expression of gene ScNRK1, consequence specific enzyme activity 2252.59 IU/mg, immobilization of ScNRK1, consequence increase in optimum temperature by 10°C, consequence improvement in temperature stability, consequence activity remains above 80% after four cycles, expression of gene Cas9-Brex27, increased efficiency of quadruple integration, attracted Rad51 recombinase to double-strand breaks, integration of four genes in mogrol biosynthetic pathway, achieved through CMI system, deletion of gene MPC1, no significant impact on lactic acid production, negatively affected cell growth, expression of gene VHb, detected in aerobic cultivation, blocking cell respiration using hypoxia, suitable for producing lactic acid, overexpression of surveillance systems, consequence: disposal of incomplete nascent chains, deletion of proper mRNA processing, consequence: stalled translating ribosomes, overexpression of co-translational quality control systems, consequence: elimination of harmful products from aberrant translation, expression of gene MrBBS, increased (-, deletion of gene ERG9, reduced competitive pathway, fusion of genes ERG20 and MrBBS, overexpression of gene PDR15, increased extracellular (-, deletion of H2A.Z C terminus, consequence: reduced occupation level of H2A.Z, consequence: improper positioning of targeted nucleosomes, overexpression of ZA protein, consequence: altered nucleosome positioning depending on the locus, overexpression of ZA-rII', consequence: enhanced transcriptional activation through interaction with chromatin-remodeling entities and/or transcription modulators, consequence: disruption of gene regulation primarily in genes without H2A.Z nucleosome in the promoter, interaction with SWR1-C, consequence: localization of H2A.Z derivatives at specific loci, deletion of gene TrLYS9, lysine auxotrophy, impaired vegetative growth, impaired conidiation, impaired conidial germination, impaired lysine biosynthesis, impaired cellulase production, depletion of Not1, mRNAs become soluble, mitochondrial mRNAs become soluble, depletion of Not4, mRNAs with lower non-optimal codon content become soluble, mRNAs with higher expression become soluble, impacts ribosome dwelling according to codon optimality for soluble mRNAs, overexpression of gene TpCHS1, consequence abnormal cell morphology, consequence reduced growth rates, consequence inhibition of the G2/M phase, deletion of gene CDS6, decreased mitochondrial PG biosynthesis capacity, decreased mitochondrial CL biosynthesis capacity, severe growth deficiency, plant death, abnormal mitochondrial ultrastructure, missing cristae structures, impaired TAG degradation in lipid droplets, impaired starch degradation in chloroplasts, upregulation of genes involved in mitochondrial electron transport chain, altered polar glycerolipids content, loss of cell proliferation capacity, higher oxidase activity, expression of CmPT2, recovery of yeast strain MB192 growth under LP conditions, mutation of G84A, inability to grow normally under LP conditions, mutation of G222A, mutation of T239A, mutation of Y242A, mutation of N422A, overexpression of CmPT2, increase in plant height, increase in effective panicle numbers, increase in branch numbers, increase in yield, overexpression of Cmphts, decrease in plant height, decrease in effective panicle numbers, higher Pi content in roots under HP conditions, higher Pi content in roots under LP conditions, overexpression of N422A, lower Pi content in leaves under HP conditions, lower Pi content in leaves under LP conditions, lower Pi content in roots under HP conditions, lower Pi content in roots under LP conditions, inhibition of PSR genes transcription under LP conditions, overexpression of T239A, different expression pattern of phosphorus-related genes under LP conditions, overexpression of Y242A, deletion of gene Huura3, deletion of gene Huhis3, deletion of gene Huleu2, deletion of gene Huade2, deletion of APS2, delayed endocytosis, overexpression of DID2, cell cycle arrest, deletion of profilin, increased F-actin, overexpression of coronin, favor assembly of actin filaments, expression of gene LmMTP1, increased Zn tolerance, increased Co tolerance, increased Cd tolerance, increased Zn content, increased Co content, decreased Cd content, consequence 1: decreased fermentation efficiency, consequence 2: altered aroma production, overexpression of nitrogen, consequence 1: increased transcript levels of genes involved in ethyl and acetate esters production, consequence 2: up-regulation of BGL2 gene, consequence 3: up-regulation of EXG1 gene, consequence 4: increased terpenol concentration, expression of eponemycin BGC, production of TMC-86A, biosynthesis of 4,5-dehydro-l-leucine, expression of TMC-86A BGC, deletion of gene SUT125, down-regulation of PIL1, deletion of gene PIL1, increased secretion of invertase, deletion of gene LSP1, deletion of gene SUR7, increased secretion of GFP, increased secretion of Pexelizumab scFv, up-regulation of sphingolipid biosynthesis, up-regulation of serine biosynthesis, deletion of gene FoRSR1, reduction in mycelial dry weight, blunted and swollen hyphae, fewer and morphologically abnormal conidia, defective chlamydospore formation, limited necrosis at wounding sites, affected pigmentation, affected autophagy, reduced production of fusaric acid, downregulation of genes involved in secondary metabolism, deletion of gene NOD-like receptor signalling pathway, consequence decreased immune response, deletion of gene influenza A, deletion of gene cytokine-cytokine receptor interaction, deletion of gene haematopoietic cell lineage, deletion of gene African trypanosomiasis, overexpression of gene selenoprotein W, consequence increased antioxidant activity, overexpression of gene glutathione peroxidase 1, overexpression of gene glutathione S-transferase A1, overexpression of gene tumour necrosis factor, consequence increased inflammatory response, overexpression of gene tumour necrosis factor superfamily member 10, overexpression of gene tumour necrosis factor superfamily member 8, overexpression of gene tumour necrosis factor superfamily member 13b, precision editing, large DNA region replacement, transcription termination of gene, production of mature mRNA, export of mRNA from nucleus, premature transcription termination of gene, altered gene expression, transcription readthrough of gene, deletion of ubiquitination sites on eS10, failure to perform NGD, deletion of ubiquitination sites on uS10, action of nuclease NONU-1, occurs after ZNF-598 activity, involvement of ribosome rescue factors HBS-1/PELO-1, facilitates mRNA decay via NONU-1, deletion of gene HXT2.4, improved co-assimilation of glucose and xylose, expression of mutant HXT2.4, overexpression of gene PvAvh77, non-specific cell death in Vitis riparia, non-specific cell death in Nicotiana benthamiana, non-specific cell death in Nicotiana tabacum, neutralization of host immunity in Vitis vinifera, enhancement of P. viticola colonization in Vitis vinifera, truncation of gene PvAvh77 to PvAvh77-M2, increased induction of cell death in Nicotiana benthamiana, increased induction of cell death in Vitis riparia, application of exogenous PvAvh77-M2, increased accumulation of salicylic acid in grapevine, increased accumulation of H2O2 in grapevine, reduced infection of P. viticola in grapevine, expression of D-lactate dehydrogenase, increased D-lactic acid productivity, expression of glycolytic genes, consequence improved targeted chromosomal integration, expression of foreign gene in Saccharomyces cerevisiae, consequence in-frame expression of foreign protein, use of CRISPR-Cas9, consequence 74% efficiency in targeted chromosomal integration, use of 15 bp microhomology flanked arms, consequence 50% targeted knock-in, optimization of CRISPR-Cas9 method, consequence no off-target effects, expression of gene interleukin 15, consequence increased expression due to RAC feeding, deletion of gene interleukin 8, consequence decreased expression due to RAC feeding, expression of gene interleukin 6, consequence tended to decrease in SCFP29 and SCFP57 on day -13, heterologous expression of gene EgrGH64, production of recombinant protein EgrGH64, hydrolysis of laminarin, hydrolysis of paramylon, hydrolysis of ramified β-glucan, production of laminaripentaose, inhibition of fungal growth, deletion of gene sod1, improvement in ATP supply, 22.3% improvement in SAM content, positive regulation of SAM synthesis, enhancement of substrate assimilation leading to increased SAM production, deletion of gene STSs, consequence reduced sesquiterpenoid production, expression of gene PoSTS-06, consequence production of pleostene, expression of gene CsAMT2.2, consequence NH4+ uptake, expression of gene CsAMT2.3, consequence upregulation under high NH4+ treatment, consequence high expression under low NH4+ treatment, consequence higher transport efficiency than CsAMT2.2, overexpression of MdTCP17, inhibited AR formation, overexpression of MdWOX11, promoted AR primordium formation, interference of MdWOX11, inhibited AR primordium production, reduced binding of MdWOX11 and MdLBD29 promoters, coexpression of MdTCP17 and MdWOX11, reduced MdLBD29 expression, deletion of gene not applicable, cell communication modulated by light-dependent production of α-factor pheromone, expression of gene luciferase, activated upon constant blue light, response level varies with dark/blue-light transitions, amplitude of response tuned by modifying strain ratio, higher fold inductions in two-population system compared to autonomous strains, deletion of gene mismatch, consequence fatty alcohol production compromised, expression of gene MaFAR1, consequence improved fatty alcohol biosynthesis 1.62-fold, enhancement of peroxisomal supply of acyl-CoA, consequence improved fatty alcohol production, enhancement of peroxisomal supply of NADPH, consequence improved fatty alcohol production 2.52 times higher than the starting strain, deletion of endogenous galactose metabolic pathway genes, improved production of 2'-FL, improved production of 3-FL, expression of Kα, enhanced expression of pathway genes, expression of Gal4p, increased production of 2'-FL, increased production of 3-FL, overexpression of gene TOMM40, consequence late onset Alzheimer's Disease, overexpression of gene Tom40, consequence caspase-dependent cell death, consequence degeneration of neuronal eye tissue, consequence upregulation of DmHsp22, consequence activation of regulated death pathways, deletion of gene Fir1, impaired immune responses, reduced resistance to Pseudomonas syringae pv. tomato DC3000, decreased production of reactive oxygen species, reduced PATHOGENESIS-RELATED 1b (PR1b, increased Pst DC3000 population growth, failure to close stomata upon infection, altered expression of jasmonic acid-related genes, deregulation of JA response, increased susceptibility to Pst DC3000 when inoculated by dipping, increased resistance to Botrytis cinerea following PTI activation, deletion of gene Achog1, delayed conidia production, reduced conidia number, increased sensitivity to hyperosmolarity, increased sensitivity to oxidative stress, inhibited growth under alkaline stress, decreased pigmentation, suppressed expression of genes related to asexual development, suppressed expression of genes related to osmotic stress resistance, suppressed expression of genes related to oxidative stress resistance, deletion of gene DBP1, reduced expression of MRP51, function reduction of MRP51, insertion of selection cassette, repression of MRP51, bidirectional promoter activity, production of divergent transcript, divergent transcript, transcriptional interference of MRP51, translational repression of MRP51, production of LUTI, deletion of glucose, consequence reduced growth rate, overexpression of PKA, consequence increased internal growth capacities, overexpression of Snf1, consequence down-regulation of stress response, deletion of gene hap4, improved growth rate in glucose-maltose shift, deletion of gene hap5, deletion of gene cin5, lower transcriptional activity of maltose metabolism genes, lower fermentation efficiency in QC18 strain, higher expression levels of glucose transporters, longer lag phase in QC18 strain, allelic variants of CIN5, activation of repressor of diauxic shift in QC18 strain, expression of gene ScAMT2;1, consequence increased ammonium uptake by roots, consequence increased nitrogen translocation to shoots, consequence increased biomass production, consequence K<sub>m</sub> of 90.17 µM, consequence V<sub>max</sub> of 338.99 µmoles h<sup>-1</sup> g<sup>-1</sup> root DW, deletion of TdGASA genes, impaired growth and development, impaired flower induction, impaired seed development, overexpression of TdGASA1, enhanced salt tolerance, overexpression of TdGASA4, enhanced osmotic tolerance, overexpression of TdGASA14, enhanced oxidative tolerance, overexpression of TdGASA19, enhanced heat tolerance, expression of IL-1β, immunomodulation, expression of IL-6, expression of INF-γ, deletion of gene FaSmi1, consequence mycelial growth defects, consequence asexual reproduction defects, consequence virulence reduction, consequence increased sensitivity to osmotic stresses, consequence increased tolerance to oxidative stresses, consequence increased tolerance to cell wall integrity inhibitors, consequence lower expression of antioxidant system-associated genes, consequence decreased DON content, deletion of gene VdERG2, reduced ergosterol content, decreased conidial germination rate, decreased conidial yield, abnormal conidia production, decreased melanin production, reduced microsclerotia production, decreased expression of genes VaflM, Vayg1, VDH1, VdLAC, VdSCD, VT4HR, increased sensitivity to congo red, increased sensitivity to sodium dodecyl sulfate, increased sensitivity to hydrogen peroxide, weakened mycelium penetration ability, impaired pathogenicity, deletion of gene N/A, consequence Colocalization of enzymes increases pathway flux and synthesis of nonnative products, expression of gene Pex15, consequence Enables display of pathway enzymes on the surface of the peroxisome, expression of gene Cat2, consequence Converts acetyl-carnitine to acetyl-CoA upon export, increasing acetyl-CoA levels 3.7-fold, expression of gene Acc1, consequence Converts acetyl-CoA to malonyl-CoA, expression of gene 2-pyrone synthase, consequence Increases synthesis of the polyketide triacetic acid lactone by 21% over cytosolic expression, overexpression of gene Pex11, consequence Increases peroxisomal membrane area, enhancing access to acetyl-CoA pool and synthesis of acetyl-CoA-based products, deletion of gene cyclooxygenase-2, consequence decreased mRNA expression in dogs fed SCFP, deletion of gene malondialdehyde, overexpression of gene cyclooxygenase-2, consequence increased mRNA expression in control dogs, overexpression of gene malondialdehyde, deletion of MS2-MCP system, activation of nonsense-mediated mRNA decay pathway, expression of improved MS2-MCP system, increase in efficiency of translation termination of tagged mRNAs, deletion of gene Dot6, slower growth acclimation, activation of gene Dot6, faster production of Ctt1 protein, deletion of gene yil014c-a, improved ncAA incorporation efficiency, deletion of gene alo1Δ, deletion of gene atg8, higher sensitivity to ER stress, attenuation of UPR gene expression, deletion of gene ire1, defect in ER-phagy, attenuated vacuolar fusion, overexpression of gene CYP79A2, consequence 2-fold increase in BGLS production, overexpression of gene CYP83B1, overexpression of gene SOT16, consequence 1.7-fold increase in BGLS production, introduction of gene APK1, overexpression of gene MET3, consequence 2.4-fold increase in BGLS production, overexpression of gene MET14, consequence 12.81 μmol/L BGLS production, deletion of gene GmIFN-γ, consequence strong signal not detected until 40 days post hatching, expression of gene GmIFN-γ, consequence strong antivirus effect, consequence high transcript levels of MHC class I chain-related protein A (MICA, consequence upregulation of zebrafish IFN-γ transcript, overexpressing artemisinin biosynthesis pathway genes, improving artemisinin content, blocking key genes in competitive pathways, regulating the expression of transcription factors related to artemisinin biosynthesis, overexpression of gene BUD1, increased surface-displayed α-amylase activity, overexpression of gene CDC42, overexpression of gene AXL1, overexpression of gene BUD10, enhanced secreted α-amylase activity, overexpression of gene BUD3, overexpression of gene BUD4, overexpression of gene BUD7, increased surface-displayed α-amylase by 56%, increased secreted α-amylase by 49%, co-overexpression of gene BUD1 and CDC42, increased α-amylase surface display by 100%, two genomic copies of gene BUD1, improved α-amylase secretion by 92%, expression of Y. lipolytica ACC, higher accumulation of malonyl-CoA, decreased growth rate, concomitant expression of homologous and heterologous ACC1 genes, elimination of growth defect, marginal reduction of malonyl-CoA accumulation, expression of Hog1, activation of autophagy-related genes, accumulation of lipid droplets, trigger vacuole calcium release, deletion of Hog1, shrinkage in the cell wall, condensation of the cytoplasmic part, high-level ROS production, leptospiral LPS, Hog1 phosphorylation, decreased expression of gene IL6, decreased expression of gene TNF, decreased expression of gene TGFB, decreased expression of gene CLDN3, decreased expression of gene MUC2, deletion of gene Gal1, constitutive expression of HAC1i, growth retardation, frequent yield of fast-growing and low-UPR progeny, weak ER stress induction, rescued growth of HAC1i cells, expression of ER-located GFP, stable strong UPR activity, highly expanded ER, abundant production of triglycerides, abundant production of heterogenous carotenoids, deletion of gene ARS, consequence integration efficiency increase, consequence cell fitness impact, overexpression of gene β-carotene pathway, consequence increased β-carotene production, expression of gene nrps33, accumulation of anthranilic acid, affected transcription of tryptophan biosynthesis genes, inhibition of Trp4 activity, down-regulation of downstream tryptophan biosynthesis genes, deletion of gene nhp6AΔBΔ, consequence SNR6 transcription restored, consequence fitness restored, consequence heat sensitivity reversed, deletion of gene hmbAΔ, consequence ascospore formation affected, consequence utilization of various carbon- and nitrogen-sources affected, consequence radial growth rate affected, consequence hypha elongation by polarized growth affected, consequence sterigmatocystin production not complemented, expression of gene HmbA, consequence utilization of various carbon- and nitrogen-sources influenced, consequence stress tolerance influenced, consequence secondary metabolism influenced, consequence hyphae elongation influenced, consequence maintenance of polarized growth influenced, consequence normal expression of endochitinase chiA influenced, introduction of t26AaLS1, increase in linalool production, introduction of IUP, increase in IPP/DMAPP production, integration of IDI1, increase in metabolic flux of IPP, integration of ERG20<sup>F96W-N127W</sup>, copy number variation of t26AaLS1, increase in linalool titer, copy number variation of ERG20<sup>F96W-N127W</sup>, optimization of isoprenol and prenol concentrations, addition of Mg<sup>2+</sup>, expression of gene Ura1, supports synthesis of pyrimidines, expression of gene from Alternaria alternata, supported growth of S. cerevisiae ura1Δ mutant, expression of gene from Schizophyllum commune, expression of gene encoding dihydrouracil oxidases, shows uracil- and thymine-dependent oxygen consumption, shows hydrogen-peroxide production, catalyses reaction dihydrouracil + O₂ → uracil + H₂O₂, expression of DHO, strains were highly sensitive to 5-fluorodihydrouracil, plasmids were readily lost during growth on 5F-dhu-containing media, deletion of cytokines, consequence under-regulation in the lung, overexpression of Saccharomyces cerevisiae boulardii CNCM I-1079, consequence improvement in piglet performance during lactation, consequence reduction in inflammatory status of the lungs, deletion of IgA producing cells, consequence fewer cells in the lymph node, deletion of gene UBR1, consequence alters abundance of 36 proteins, overexpression of gene UBR1, consequence increases UPS activity, deletion of gene DOA10, consequence reduces recognition of cellular proteins, overexpression of gene NTA1, consequence increases processing of cellular proteins, deletion of gene UBC6, consequence reduces ubiquitination of cellular proteins, deletion of gene ERG28, reduced total cholesterol levels, 60-75% reduction in the rate of cholesterol synthesis, expression of gene ERG28, rescued reduction in the rate of cholesterol synthesis, impaired activation of SREBP-2, deletion of gene SD38, reduced cell length, affected lipid metabolism, reduced content of ACC, overexpression of VLCFA C24:0, increased expression of OsACS3, increased expression of OsACS4, increased expression of OsACO7, increased plant height, exogenous application of ACC, exogenous application of ethephon, deletion of gene Acpbs2, reduced conidia number in hypertonic media, increased sensitivity to 3 M NaCl, increased sensitivity to 3 M sucrose, increased sensitivity to 3 M sorbitol, increased sensitivity to high oxidative stress, inhibited growth under alkaline-pH stress, lighter colony color, altered expression of genes related to conidial development, altered expression of genes related to osmotic stress, altered expression of genes related to oxidative stress, altered expression of genes related to carbon metabolism, disrupted interaction with Achog1, recombinant expression of TV-AFB<sub>1</sub>D, AFB<sub>1</sub> degradation efficiency increased, AFB<sub>1</sub> concentration decreased to 32.6 μg/ml, expression of AOX1 in S. cerevisiae, AFB<sub>1</sub> content decreased to 3.4 μg/g, expression of CaMV 35S promoter in S. cerevisiae, AFB<sub>1</sub> content decreased to 2.9 μg/g, expression of TV-AFB<sub>1</sub>D in S. cerevisiae, ethanol production increased, expression of gene Kluyveromyces marxianus TPI, constitutive expression in Pichia pastoris, expression of gene Hansenula polymorpha PMA, expression of gene Candida tropicalis ICL, inducible expression in Pichia pastoris, expression of gene Saccharomyces cerevisiae CUP, expression of gene ScCUP, responsive to Cu<sup>2+</sup> in Pichia pastoris, recombinant expression of egtA, EGT biosynthetic capacity, EGT production, recombinant expression of egtA and egt2a, decrease in EGT levels, expression of gene TOR1, increased growth signaling, expression of gene MSN2, increased stress response, expression of gene MSN4, addition of morin, reduced gene expression of TOR1, reduced gene expression of MSN2, reduced gene expression of MSN4, heterologous expression of BFTS, production of diterpene variediene, production of neovariediene, production of neoflexibilene, heterologous expression of BFTS and VndE, production of seven norditerpenoids, modification by VndE, multi-site hydroxylation by A. oryzae endogenous enzymes, transcriptional device library, expression strength of 16-520%, expression strength of 30-500%, PAOX1-driven pathway, 5.5-fold production of dihydromonacolin L, selected devices from cTRDL, 3.0-fold production of monacolin J, upregulation of gene expression, improvement in mucosal barrier function, increase in goblet cells, enhancement of tight junction, increase in sIgA level, decrease in pathogenic bacteria, stabilization of microbiome, increase in beneficial bacteria, promotion of anti-inflammatory response, suppression of inflammatory response, no specific gene overexpression/deletion mentioned, no specific consequences mentioned, expression of gene Cas9, deletion of gene catalase, consequence lower mRNA expression, overexpression of gene superoxide dismutase, consequence higher serum concentrations, deletion of gene clp1, increased fungal background autophagy level, reduced hyphal growth, hyperbranching, abnormal fungal morphology, hindered appressorial glycogen metabolism, hindered turgor production, weakened plant infection, decreased virulence, focus on discussion of bacterial hosts and gene expression systems, deletion of gene REG1, improved SAM production, enhanced glucose utilization, elevated glycolytic intermediates, extended lifespan, overexpression of gene SNF1, increased expression of genes involved in glucose transport and glycolysis, enhanced expression of ACS1, enhanced expression of ALD6, enhanced activity of ADH2, enhanced gene expressions involved in sulfur-containing amino acids, overexpression of gene PgCBF3, consequence increased survival rate after cold treatment, overexpression of gene PgCBF7, consequence increased soluble sugar content, consequence increased proline content, consequence decreased electrolyte leakage, consequence decreased malondialdehyde content, consequence decreased reactive oxygen species production, consequence elevated catalase enzyme activity, consequence elevated peroxidase enzyme activity, consequence elevated superoxide dismutase enzyme activity, consequence upregulated expression of AtCOR15A, consequence upregulated expression of AtCOR47, consequence upregulated expression of AtRD29A, consequence upregulated expression of AtKIN1, deletion of gene MoDHX35, reduced tolerance to doxorubicin, reduced appressorium formation rate, attenuated virulence, deletion of D14, activation of D14 to bind to downstream signaling partners, overexpression of SL, induction of D14 interaction with SMXL7, induction of D14 interaction with MAX2, presence of MAX2, strengthening of SL-induced D14 interaction with SMXL7, presence of SMXL7, weakening of SL-induced D14 interaction with MAX2, deletion of gene RPD3, decrease in de novo pathway metabolites production, deletion of gene HST1, increase in BNA gene expression, reduction in acetylation of H4K5 and H4K12, increase in acetylation of H4K5 and H4K12, opposition to spreading of Hst1-dependent heterochromatin, overexpression of gene Mit1, upregulates pAOX1, increases VEGF production by 1.9 fold, increases eGFP production by 2.2 fold, duplicates AOX1 enzyme activity, enhances yeast cells' capacity for methanol detoxification, intracellular overexpression of gene Mit1, upregulates heterologous target gene eGFP production, deletion of gene L7/uL30 major form, consequence 1: favoring translation of genes with short open reading frames, overexpression of gene L7/uL30 minor form, consequence 1: increasing translation of long open reading frames, consequence 2: promoting tolerance to drugs through increased translation of cell wall proteins, deletion of gene Ltn1, consequence 1: impaired ubiquitination of substrates, overexpression of gene Ago, consequence 1: enhanced RQC activity, consequence 2: increased elimination of defective nascent polypeptides, consequence 3: removal of unwanted protein products, overexpression of eGFP, varied GFP signal, overexpression of MaFAR1, fatty alcohol production, upregulation of lamBDCA, increased adhesion capability, downregulation of ari1, decreased growth, downregulation of skg6, downregulation of kex2/gas1, downregulation of DNA to RNA to protein pathways, downregulation of cell cycle, decreased proliferation, downregulation of meiosis, downregulation of MAPK signaling pathways, expression of genes responsible for growth and ethanol stress, increased ethanol fermentation efficiency, expression of genes responsible for oxidative stress, expression of genes responsible for acetic acid stress, expression of genes responsible for DNA repair, expression of genes responsible for pyruvate-to-tricarboxylic acid (TCA) pathway, expression of genes responsible for pyruvate-to-ethanol pathway, deletion of glycosylation site N15, lower secretion efficiency, deletion of glycosylation site N70, deletion of glycosylation sites N15 and N70, preservation of glycosylation sites, higher thermal stability, slightly higher proliferation efficacy, deletion of gene GPD2, increase in ethanol content, decrease in glycerol content, deletion of gene FPS1, deletion of gene ADH2, deletion of gene DLD3, decrease in acetic acid content, decrease in lactic acid content, integration of MCR-C, stable gene expression, higher 3-HP production 4.4 g/L under batch fermentation, higher 3-HP production 56.5 g/L under fed-batch fermentation, plasmid loss, instability of the engineered strain, toxic intermediate malonate semialdehyde, cause of plasmid loss, consequence: large intermolecular interfaces promote cotranslational assembly, decrease in succinic acid content, deletion of non-phytate phosphorus, consequence final body weight and total gain decrease, overexpression of phytase, consequence final body weight and total gain increase, overexpression of multi-strain probiotic, overexpression of Saccharomyces cerevisiae, overexpression of fumaric acid, consequence feed conversion ratio equivalent to positive control, consequence improved feed conversion ratio, consequence greater dressing percentage, consequence higher relative weights of spleen and bursa of Fabricius, consequence higher relative weights of spleen and bursen of Fabricius, consequence improved crude protein digestibility, consequence improved crude fiber digestibility, consequence improved calcium digestibility, consequence improved phosphorus digestibility, consequence greater tibia breaking strength, consequence increased tibia ash content, consequence increased tibia calcium concentration, consequence increased tibia phosphorus concentration, consequence increased tibia cortical area thickness, consequence reduced serum total cholesterol, consequence reduced serum triglycerides, consequence reduced serum triiodothyronine, consequence reduced serum thyroxine, consequence reduced serum glucose, consequence reduced serum alkaline phosphatase, consequence increased serum calcium, consequence increased serum phosphorus, consequence enhanced villi height, consequence enhanced villi width, consequence enhanced villi absorptive surface area, consequence up-regulated NaP-IIb mRNA expression, deletion of gene OLI1, consequence 1: reduced ethanol production, consequence 2: altered porphyrin biosynthesis, overexpression of gene CYC3, consequence 1: enhanced ethanol production, consequence 2: increased oxidative phosphorylation, overexpression of gene ADH5, overexpression of gene PDA1, regulation by Tup1, consequence 1: affects expression of CYC3, consequence 2: affects expression of ADH5, eIF4A overexpression, accelerates eIF4E-mRNA association, modulates eIF4F-mRNA interactions, catalyzes ATP hydrolysis, ATP hydrolysis, ejects eIF4E, ejects eIF4E•eIF4G, deletion of gene SOD, deletion of gene GPx, deletion of gene COX II, deletion of gene TNF-α, deletion of gene Casp-3, deletion of gene Claudin-3, overexpression of gene SsSWEET13c, consequence plant root elongation, consequence increase fresh weight of Arabidopsis thaliana, positive regulation of gene SsSWEET13c by SsbHLH34, consequence increased activity in morning, negative regulation of gene SsSWEET13c by SsbZIP44, consequence decreased activity in morning, positive regulation of gene SsSWEET13c by SsTFIIIA-a, positive regulation of gene SsSWEET13c by SsMYR2, positive regulation of gene SsSWEET13c by SsRAP2.4, positive regulation of gene SsSWEET13c by SsbHLH035, negative regulation of gene SsSWEET13c by SsABS5, negative regulation of gene SsSWEET13c by SsTFIIIA-b, negative regulation of gene SsSWEET13c by SsERF4, positive regulation of gene SsSWEET13c by SsbZIP44, consequence increased sugar accumulation in sink tissues, positive regulation of gene SsSWEET13c by SsTFIIIA-b, consequence decreased sugar accumulation in sink tissues, negative regulation of gene SsSWEET13c by SsHB36, negative regulation of gene SsSWEET13c by SsDEL1, overexpression of gene ZmWRKY20, decrease in salt tolerance, reprogramming of genes involved in defense response, reprogramming of genes involved in oxidation-reduction response, interaction of ZmWRKY20 and ZmWRKY115, repression of ZmbZIP111 expression, increase in sensitivity to salt stress, deletion of gene gapA, decrease in glucose consumption by competing pathways, deletion of gene budA, decrease in by-products pathways, expression of gene gpd1, production of glycerol, expression of gene gpp2, expression of CRISPRi system, inhibition of gapA, inhibition of budA, overexpression of glycerol pathway genes, use of glucose as carbon source, production of 3-HP, deletion of gene rimO, failure to activate starvation-induced BGCs, down-regulation of genes involved in basic metabolism, down-regulation of genes involved in membrane biogenesis, down-regulation of genes involved in growth, overexpression of gene rimO, increased resistance to oxidative stress, increased resistance to osmotic stress, insensitivity to glucose repression, overproduction of several SMs, expression of gene rimO, activation of sterigmatocystin gene cluster, induction of aflR transcription, required for nuclear accumulation of AflR protein, deletion/mutation of gene ZNHIT3, decrease in steady-state Hit1 protein levels, significant reduction of box C/D snoRNA levels, defects in rRNA processing, altered cellular translation, site-specific changes in the rRNA modification pattern, deletion of gene XdWC2, similar carotenoid profile to ARTP mutant, regulation of CrtI expression, regulation of CrtS expression, overexpression of gene CrtI, increase in 3-hydroxy-3', 4'-didehydro-β, ψ-carotene-4-one production, interaction of XdWC2 with XdWC1, formation of WCC complex, transcriptional increase of XdWC2, no significant change in carotenoid profile, transcriptional increase of CrtS, deletion of protease gene, reduced protease activity, deletion of sporulation gene, reduced sporulation, deletion of autolysis-related gene, reduced autolysis, overproduction of Sox, increased secretion of Sox, overproduction of interleukin-1β, increased secretion of interleukin-1β, expression of ACC1, increased MCFAEE levels, expression of FAS1, expression of FAA2, expression of EEB1, deletion of POX1, decreased MCFAEE levels, overexpression of gene OsNACL35, increased resistance to salinity stress, deletion of gene OsNACL35, hypersensitivity to salt stress, upregulation of gene OsDCD1, increased H2S content, consequence reduced cell growth, consequence increased cell growth, consequence decreased protein synthesis, consequence increased protein synthesis, deletion of rRNA, consequence impaired ribosome assembly, overexpression of rRNA, consequence enhanced ribosome assembly, consequence altered protein expression profile, expression of Cas9, high-efficiency genomic integrations, markerless transformations, overexpression of gene of interest, supports localization studies, supports overexpression studies, expression of green fluorescent protein, colocalization experiments, expression of red fluorescent protein, expression of ADH genes, increased alcoholic fermentation, deletion of ACO2 gene expression, altered malate metabolism, inhibition of tricarboxylic acid cycle enzymes, increased malate metabolism, altered metabolism to overcome nitrosative stress, deletion of gene expression levels, enhanced predictive capability of metabolic models, overexpression of chromatin accessibility levels, deletion of epigenetic information, limited metabolic modeling accuracy, overexpression of RNA-seq data, improved condition-specific metabolic simulations, overexpression of ATAC-seq data, enhanced metabolic predictions, no direct consequence related to gene overexpression/deletion, deletion of gene CLN3, predominance of G2/M phase cells, increased biofilm formation, increased extracellular polysaccharide formation, increased expression of genes in the FLO gene family, deletion of gene SIC1, deletion of gene ACE2, contrasting performance compared to CLN3 and SIC1 deletion, deletion of gene SPT10, increase in total phospholipids, insufficient accumulation of key metabolites, massive endoplasmic reticulum degradation, overexpression of gene DGK1, stable massive ER structure, decrease in autophagy, increase in MEV/ERG pathway activity, heterologous expression of β-amyrin synthase, production of triterpene β-amyrin, deletion of gene CYP2E1, consequence 1: reduced inflammation, consequence 2: improved fatty liver condition, inhibition of gene MBOAT7, consequence 1: exacerbation of fatty liver, inhibition of gene TMC4, expression of gene CYP7A1, consequence 1: inhibition of lipid production, fermentation by Lactobacillus plantarum, consequence 1: reduction in liver cholesterol, consequence 2: reduction in fat accumulation, consequence 3: promotion of short-chain fatty acid production, consequence 4: regulation and restoration of intestinal flora, consequence 5: decrease in endotoxin-bearing Proteobacteria levels, expression of gene VDS, consequence valerena-4,7(11, consequence increased valerena-4,7(11, overexpression of gene ERG12, overexpression of gene ERG19, expression of gene LsGAO2, consequence valerenic acid production, overexpression of gene HAA1, enhancement of carbohydrate metabolism pathways, enhancement of amino acid metabolism pathways, enhancement of transcription pathways, enhancement of translation pathways, enhancement of cofactors metabolism pathways, enhancement of vitamins metabolism pathways, involvement in oxidative phosphorylation, involvement in purine metabolism, involvement in vitamin B6 metabolism, involvement in spliceosome, overexpression of gene TYE7, involvement in riboflavin metabolism, preference for enhancing oxidative phosphorylation, tendency to upregulate glycolysis/gluconeogenesis, rewiring global gene expressions, positive perturbations of energy metabolism, positive perturbations of amino acid metabolism, strengthening of key cofactor metabolism, improvement in transcriptional regulation, improvement in translational regulation, deletion of gene EHD3, consequence malonyl-CoA hydrolase activity, mutation of gene EHD3, consequence F121 to I121, consequence E124 to S124, integration of mutant EHD3, consequence increased expression, integration of ACC1, regulation of key enzyme expression in malonyl-CoA pathway, consequence improved precursor supply, inhibition of competing pathways, consequence increased malonate production, deletion of gene n/a, consequence increased gene activation efficiency 34.9-fold, consequence increased gene repression efficiency 95%, expression of gene n/a, consequence improved metabolic pathway rewiring for 3-hydroxypropanoic acid production, decrease of toxin production by lentil extract, reduced pro-inflammatory interleukin-8 production, modulated mucus-related gene induction, reduction of ETEC adhesion by yeast product, reduced adhesion to mucin, reduced adhesion to Caco-2/HT29-MTX cells, modulation of toxin-related gene expression by both fiber-containing products, strengthened intestinal barrier function, deletion of gene XsGAO1, consequence Germacrene A not converted to Germacrene A acid, deletion of gene XsGAO2, expression of gene XsGAO1, consequence Germacrene A converted to Germacrene A acid, expression of gene XsGAO2, overexpression of gene ari1, higher tolerance to furfural, higher tolerance to HMF, increased growth rate in aldehyde-containing environment, increased ethanol production capacity in aldehyde-containing environment, consequence constitutive promoter activity maintained at high temperature, consequence constitutive promoter activity maintained in presence of ethanol, consequence responds to high temperature, expression of gene PCK1, consequence responds to gluconeogenic sources, expression of gene XYL1, consequence responds to presence of xylose oligomers, thi4 gene overexpression, TOL production increase, involvement in TOL production, Trp overexpression, Phe overexpression, reduction in Trp's stimulatory effect on TOL production, stimulation of Ehrlich pathway activity, allantoin catabolic genes dal1, dal2, dal4 overexpression, role in TOL production, transcriptional activator gene aro80 overexpression, role in TOL production, amplification of heterologous gene copies, improvement in production of nerolidol, improvement in production of limonene, improvement in production of lycopene, increasing gene dosage, overcoming metabolic bottlenecks, expression of gene Erg1p, consequence squalene loss reduction, expression of gene ERG1 under P<sub>THI2</sub>, consequence 20 times higher squalene production, overexpression of rate-limiting enzyme, consequence increased squalene production, deletion of non-essential competing pathway gene, deletion of gene CiSHT1, reduced synthesis of tetracoumaroyl spermine, deletion of gene CiSHT2, reduced initial N-acylation steps, expression of gene CiSHT2, accumulation of partially substituted spermine, coexpression of gene CiSHT2 and CiSHT1, synthesis and accumulation of fully substituted spermine, availability of spermine, increased accumulation of spermine derivatives over spermidine derivatives, deletion of gene EpSDR-5, reduction in jatrophanes, increase in ingenanes, deletion of gene Casbene Synthase, reduction in ingenanes, deletion of gene CYP71D365, expression of gene EpSDR-5, conversion of jolkinol C to jolkinol E, decreased expression of proinflammatory cytokines, enhanced gut barrier, increased expression of TUNEL-positive tumor epithelial cells, increased apoptosis, decreased expression of oncocyte proliferation indicators, decreased tumor multiplicity, expression of gene ldhA, D-lactic acid production, PDLA accumulation, D-lactic acid content in P(LA-3HB, overexpression of gene OPI3, overexpression of gene EFM6, overexpression of gene YOL014W, expression of LacZ gene, activation of β-galactosidase, hydrolysis of X-gal, production of 5,5'-dibromo-4,4'-dichloro-indigo, measurement of protein-protein interaction strength, use of ONPG, detection of β-galactosidase activity, no consequence related to expression/deletion, cloning and expression of gene CaHMGS, increased mevalonate biosynthesis in yeast, overexpression of gene CaHMGS, higher expression level in roots of Centella asiatica, deletion of HMGS in Saccharomyces cerevisiae strain YSC1021, reduced mevalonate biosynthesis, expression of gene CaHMGS, enhanced accumulation of isoprenoids and triterpenoid saponins, deletion of gene sorB, consequence sorbicillinoid production affected, deletion of gene axl1, consequence no impact on yield, deletion of gene axl2, consequence no impact on dry weight, deletion of gene bud3, consequence no impact on pH, deletion of gene bud4, consequence no impact on stress tolerance, consequence arthrospores development differs from wild-type, reconstitution of brassinolide biosynthesis pathway, production of brassinolide intermediates, heterologous expression of fungal biosynthetic gene cluster, increase yields of natural products, investigation of biosynthetic pathway, deletion of gene Ccr4, increased CLB1 mRNA levels, increased CLB2 mRNA levels, increased CLB3 mRNA levels, increased CLB4 mRNA levels, increased CLB5 mRNA levels, increased CLB6 mRNA levels, increased Clb1 protein levels, increased Clb4 protein levels, increased Clb6 protein levels, Ccr4-mediated regulation, destabilization of CLB6 mRNA through coding sequence, destabilization of CLB6 mRNA through 3' UTR, deletion of multiple genes, overexpression of multiple genes, increased S-adenosyl-L-methionine production, expression of BGCs, yield improvement, discovery of new-to-nature compounds, host engineering of E. coli, production of secondary metabolites, host engineering of S. cerevisiae, engineering of non-model microbes, biosynthesis of secondary metabolites, genetic manipulation in non-model microorganisms, optimization of phenotypes, improvement of yields of secondary metabolites, deletion of gene T4-endonuclease V, consequence CPD formation and repair monitored, overexpression of ncRNA transcription, consequence enhancement of NER in subtelomeric elements Y' and X, deletion of gene activity in silenced heterochromatin, consequence slow repair of UV photoproducts,  deletion of gene EHT1, consequence lower amounts of ethyl esters,  deletion of gene EEB1, consequence increased expression of ATF2, consequence increased content of acetate esters, consequence affected production of higher alcohols, overexpression of gene hchA, improved tolerance against salinity stress, improved tolerance against dicarbonyl stress, improved tolerance against oxidative stress, overexpression of gene EcGLY-III, tolerance against salinity at seedling stage, tolerance against salinity at reproductive stage, increased glyoxalase activity, increased antioxidant enzyme activity, resistance to accumulation of methylglyoxal, resistance to accumulation of reactive oxygen species, increased anti-glycation activity, prevention of stress-induced senescence, attenuation of OTC gene, reduced ornithine production, overexpression of SPE1-S gene, increased ornithine production, increased putrescine production, overexpression of ArgB/C gene, increased ornithine accumulation, overexpression of BbgCYP11A1, increased cholesterol processing, overexpression of plant CPR from Anemarrhena asphodeloides, higher bufadienolides production, overexpression of gene spt23, lower thermal tolerance, deletion of gene spt23, higher survival rate, higher biomass, higher ethanol production, lower total unsaturated lipid acid contents, lower LTR richness, higher transposon activities, deletion of gene POT1, consequence 1: increased genetic stability, consequence 2: increased p-coumaric acid production, overexpression of p-CA synthesis cassette, consequence 1: increased p-coumaric acid production, consequence 2: stable p-coumaric acid production in nonselective medium, deletion of AAT gene in S. cerevisiae, decreased antifungal activity, expression of GcAAT gene in S. cerevisiae AAT knockout strain, 32% decrease in dry biomass weight of Rhizoctonia solani, supplementation with L-leucine in native host, induced expression of GcAAT gene, supplementation with α-ketoisocaproate in native host, increased oxidative stress in R. solani, distorted mycelial morphology in R. solani, increased hyphal branching in R. solani, feeding with L-leucine in S. cerevisiae AAT knockout strain expressing GcAAT, 21% increase in isoamyl acetate ester, feeding with α-ketoisocaproate in S. cerevisiae AAT knockout strain expressing GcAAT, 48% increase in isoamyl acetate ester, 58% higher antifungal activity against R. solani, codon-optimized gene sequences of Ulp1 and Ulp2, cloned and expressed in E. coli, expression of SpUlp1, higher affinity towards SUMO compared to ScUlp1, purification of SpUlp1 catalytic domain, 83.33% recovery yield, 90% cleavage of fusion proteins within 1 hour, fed-batch cultivation, 3.5 g/L of SpUlp1SD production, Y_P/X of 77.26 mg/g DCW, volumetric productivity of 205.88 mg/L/h, expression of gene IaAO2, C-28 oxidation of α-amyrin, C-28 oxidation of β-amyrin, expression of gene IaAO4, C-23 oxidation of ursolic acid, C-23 oxidation of oleanolic acid, expression of gene IaAO5, C-24 oxidation of β-amyrin, expression of genes IaAO1, IaAO4, IaCPR, production of C-23 hydroxyl ursane-type triterpenoid derivatives, overexpression of gene ENA5, improved multiple stress-tolerance, regulation of expression of gene ASP3, regulation of expression of gene YOL162W, regulation of expression of gene YOR012W, regulation of expression of gene Crz1p, regulation of expression of gene Tos8p, deletion of prephenate dehydratase, alleviation of byproducts formation, tuning down expression of OPc4CL2, inducible expression of enzymes, balance cellular growth and production, overexpression of genes using optogenetics, improved fermentation control, improved productivity, overexpression of gene LpCPC, increased tolerance to NaHCO3 stress, higher chlorophyll content, higher soluble phenol content, higher lignin content, lower relative conductivity, higher expression of NHX1, higher expression of SOS1, higher expression of CCR1, higher expression of CAD, expression of eEmRFP, gene expression in yeast, gene expression in mammalian cells, expression of GFP-fused human synaptobrevin VAMP1, proof-of-concept in yeast, proof-of-concept in E. coli, proof-of-concept in HEK293 cells, expression of wild-type CLuc luciferase, expression of codon-changed CLuc luciferase, improved expression of ARV1, decreased membrane fluidity, enhanced acid tolerance, improved expression of GIT1, increased membrane integrity, improved oxidative tolerance, improved expression of NFT1, reduced membrane permeability, engineered Med15 V76R/R84K, increased pyruvate production, expression of gene CsupYPAQ, production of (Z, expression of gene CsupFAR2, expression of gene Atr∆11, expression of gene ATF1, deletion of gene RPL3, consequence degradation of RPL3 mRNA, deletion of gene RPL4, consequence degradation of RPL4 mRNA, overexpression of gene RPL3, consequence massive aggregation of Rpl3, overexpression of gene RPL4, consequence massive aggregation of Rpl4, deletion of gene TOM1, consequence perturbation of overall proteostasis, deletion of gene SAE2, increased CRISPR/SpCas9-induced mutagenesis efficiency, increased CRISPR/AsCpf1-induced mutagenesis efficiency, deletion of gene EXO1, deletion of gene FUN30, overexpression of gene MRE11-H125N, diversified CRISPR/SpCas9-induced mutations at URA3 locus, diversified CRISPR/SpCas9-induced mutations at ADE2 locus, modulation of gene FPS1 expression, improved ethanol/glycerol ratio, modulation of gene GPD1 expression, deletion of gene for high concentration of vinblastine and vincristine, consequence low production in planta, expression of gene through metabolic engineering in heterologous systems, consequence improved MIA biosynthesis, expression of gene through yeast engineering, consequence alternative production source of phytomolecules, consequence discovery of missing plant pathway enzymes and genes, deletion of AtGTR1, reduced glucosinolate transport, reduced phytohormone transport, mutation of EXXEK motif in AtGTR1, impaired glucosinolate transport function, phosphorylation of Thr105 in AtGTR1, regulation of protein expression, T105D mutation in AtGTR1, abrogation of glucosinolate transport function, constitutive expression of gene GPH1, partial suppression of filamentation, repression of gene CaCDC4, deletion of gene GPH1, reduction in germ tube formation, reduction in cell surface hydrophobicity, increase in fibronectin binding, increase in adhesion, similar capacity to form mature biofilm, expression of gene McBBE, consequence 1.12 mg/L yield of (S, deletion of N-terminal amino acids of gene McBBE, consequence decrease in (S, integration of gene McBBE using CRISPR-Cas9, consequence no significant increase in (S, integrated expression of coTAL, improved caffeic acid production, episomal expression of coC3H, episomal expression of coCPR1, delta integration of coC3H, increased caffeic acid production, delta integration of coCPR1, delta integration method, 50-fold increased caffeic acid production, deletion of ERG20 promoter, increased GPP production, replacement of ERG20 promoter with ERG1 promoter, increased GPP-derived linalool production, increased NPP-derived nerol production, deletion of gene VdAda1, consequence severe defects in the formation of conidia, consequence severe defects in the formation of microsclerotia, consequence defects in melanin biosynthesis, consequence reduction in virulence, consequence genome-wide alteration of the V. dahliae transcriptome, consequence altered expression of genes encoding transcription factors, consequence altered expression of genes encoding secreted proteins, deletion of gene FvMed1, increased FB1 production, expression of gene NDI1, resisted rotenone-induced changes in cell morphology, resisted rotenone-induced loss of cell viability, resisted rotenone-induced accumulation of α-synuclein, resisted rotenone-induced accumulation of pS129 α-synuclein, resisted rotenone-induced mitochondrial ROS production, resisted rotenone-induced mitochondria-mediated apoptosis, increased basal oxygen consumption, increased maximal oxygen consumption, increased mitochondrial coupling efficiency, increased basal ATP content, increased oligomycin-sensitive ATP content, increased complex I activity, increased number of viable dopaminergic neurons, increased expression of tyrosine hydroxylase, increased content of dopamine in right striatum, deletion of gene Cbf1, consequence 1: accumulation of cenRNAs at all phases of the cell cycle, consequence 2: increased chromosome mis-segregation, expression of ZNF1, activation of xylose metabolism, activation of xylose reductase genes, activation of pentose phosphate shunt, activation of related pathways for xylose utilisation, deletion of Bud21, overexpression of ZNF1, increased xylitol production to 12.14 g/L, increased tolerance to lignocellulosic inhibitors, improved cell dry weight, overexpression of gene XySC1, consequence: efficient consumption and metabolism of xylose under aerobic conditions, overexpression of gene Itr1, increase in glucaric acid yield by 26%, fusional expression of MIOX4 and Udh, increase in glucaric acid yield by 40%, down-expression of gene ZWF1, improvement in glucaric acid production efficiency, deletion of gene MoOPY2, consequence hyphal growth defects, consequence conidiation defects, consequence germ tube extension defects, consequence appressorium formation defects, consequence appressorium turgor generation defects, consequence invasive growth defects, consequence attenuated pathogenicity, consequence cumulative MoAtg8 lipidation, consequence rapid GFP-MoAtg8 degradation, TOR signaling overexpression, regulates ribosome production, represses autophagy, represses cellular stress response, PKA signaling overexpression, Sch9 overexpression, shares kinase consensus motif with PKA, TORC1 overexpression, directly regulates PKA, TOR and PKA signaling overexpression, targets shared transcription factors, activates PP2A<sup>Sit4</sup>/PP2A branch, mutation of gene TDH3 UAS, enhanced promoter strength, increased transcriptional regulation dynamic range, improved β-carotene production, expression of GDP-bound Gtr1p, resistance to hydrogen peroxide, expression of GTP-bound Gtr1p, sensitivity to hydrogen peroxide, deletion of Iml1p, induction of autophagy, decreased SNQ2 expression, overexpression of SNQ2, rescue of oxidative stress sensitivity, deletion of gene CAP1, reduced utilization of non-preferred amino acids, deletion of gene END3, overexpression of gene CAP1, increased utilization of non-preferred amino acids, overexpression of gene END3, reduced accumulation of urea, deletion of gene ATP12, deletion of gene MRPL22, deletion of gene MRP1, deletion of gene NAM9, overexpression of gene ATP12, overexpression of gene MRPL22, overexpression of gene MRP1, overexpression of gene NAM9, deletion of gene Dxo1, consequence 1: impaired final 5' end processing of 25S rRNA, overexpression of gene Dxo1, consequence 1: enhanced distributive exonuclease activity on endonuclease or decapping products, overexpression of gene AoErg10s, consequence growth speed change, consequence sporulation change, consequence fatty acid content change, consequence ergosterol content change, expression of gene AoErg10A, consequence recovery of S. cerevisiae erg10 weak mutant phenotype, expression of gene AoErg10D, expression of gene AoErg10E, expression of gene AoErg10F, expression of gene AoErg10B ΔMTS, consequence recovery of S. cerevisiae pot1 mutant phenotype, expression of gene amyA, increased amylolytic activity, expression of gene glaA, production of more than 4 g/L ethanol, deletion of gene TUP1/CYC8, compromised tolerance to heavy metals exposure, derepression of GSH1 expression, altered endoplasmic reticulum homeostasis, failure to activate unfolded protein response pathway, impaired activation of KAR2 transcription, deletion of gene whi2, suppression of survivin gene expression, apoptosis in breast cancer cells, deletion of ERG6 gene, increase in DSG synthesis, reduction in by-product formation, overexpression of cholesterol 22-hydroxylase, construction of DSG biosynthesis pathway, optimization of CYP to CPR ratio, increase in DSG production, overexpression of PDR5, contribution to DSG production, overexpression of PRY1, overexpression of gene PK-w, consequence functionally active proteinase K, consequence secretion into the culture medium, consequence productivity of 25 μg/mL, consequence high specific proteolytic activity of ~5000 U/mg, expression of cinB, nearly complete loss of embryo viability, expression of cinA, rescue of embryo viability, expression of CidB, inhibition of growth, expression of CidA, rescue of growth, expression of CinB, dependence on nuclease active site for toxicity, expression of gene D-limonene, consequence 1.48-fold production improvement in S. cerevisiae, expression of gene ovalbumin, consequence 1.71-fold production improvement in B. subtilis, overexpression of gene RAD52, enhanced HR efficiency, overexpression of gene RAD59, overexpression of gene MRE11, overexpression of gene SAE2, enabled single-locus integration of heterologous gene expression cassettes with ∼40 bp homology arms with efficiencies as high as 100%, enabled two-loci integration of heterologous gene expression cassettes with ∼40 bp homology arms with efficiencies as high as ∼98%, enabled three-loci integration of heterologous gene expression cassettes with ∼40 bp homology arms with efficiencies as high as ∼81%, construction of β-carotene producing strain, demonstrated in a single step, optimization of betaxanthin producing strain, CBG exposure, reduces reactive oxygen species levels, inhibits Interleukin-1β release, inhibits Interleukin-6 release, inhibits Interleukin-8 release, inhibits tumor necrosis factor α release, CBG application, improves transepidermal water loss, reduces redness appearance, discussion on conserved transcriptional regulators across species, altered translational efficiency during ageing, decline in cellular proteostasis, ageing, altered kinetics of translation elongation, ribosome pausing at specific positions in aged yeast and worms, increased ribosome collisions, trigger ribosome-associated quality control (RQC, aged yeast cells, impaired clearance and increased aggregation of RQC substrates, long-lived yeast mutants, reduced age-dependent ribosome pausing, greater flux through the RQC pathway, nascent polypeptides exhibiting age-dependent ribosome pausing in C. elegans, strongly enriched among age-dependent protein aggregates, increased pausing and aggregation of many components of proteostasis, proteostasis collapse, systemic decline during ageing, translational control, shapes the proteome, alternative 5' UTR isoforms, increased translation, C-rich silencers, repress translation, phosphorylation of Sen1, reduces efficiency of Sen1-mediated termination, impairs termination at unannotated non-coding gene, impairment of termination at unannotated non-coding gene, represses expression of Zap1, repression of Zap1, causes expression pattern mimicking conditions of Zn excess, deletion of gene ILV1, intracellular accumulation of isoleucine, extracellular overproduction of 2-methyl-1-butanol, increased production of 2-methylbutyl acetate, general discussion on 2,3-BDO production using genetic engineering and metabolic engineering in Saccharomyces cerevisiae, induction of anticancer immunity, improved abundance and functional orientation of effector components in tumor microenvironment, improved abundance and functional orientation of effector components in TDLN, treatment with small size YCW NPs, anticancer efficiency in melanoma-challenged mice, combination with PD-L1 blockade, enhanced anticancer efficiency, SptF catalyzes several continuous oxidation reactions, SptF exhibits extremely broad substrate specificity, SptF efficiently produced unique cyclopropane-ring-fused 5/3/5/5/6/6 and 5/3/6/6/6 scaffolds from terretonins, SptF hydroxylates steroids with different regiospecificities, crystallographic and structure-based mutagenesis studies revealed the molecular basis of the enzyme reactions, malleability of the loop region contributes to the remarkable substrate promiscuity, deletion of gene PDC, consequence glucose uptake decreased, expression of gene dCas9, consequence ethanol synthesis reduced, consequence 2,3-butanediol production increased, deletion of activation domains from dSpCas9, improved interaction with AcrIIs, expression of AcrIIA4, high inhibition at low concentrations, limited control of gene transcription regulation hindered, expression of bare dSpCas9, best solution for building synthetic genetic networks in yeast, no direct causal relationship extracted, deletion of gene related to protein biosynthesis, consequence inhibition of protein biosynthesis, overexpression of gene related to glycolysis, consequence activation of intracellular metabolic pathways of energy production, overexpression of gene related to glycogen synthesis, consequence increase of glycogen, deletion of gene related to energy consumption metabolism, consequence decrease of energy consumption metabolism, overexpression of gene related to autophagy, consequence cells undergoing autophagy, overexpression of gene related to MAPK signaling pathway, consequence activation of intracellular metabolic pathways, overexpression of gene related to redox regulation, deletion of gene related to amino acid synthesis, consequence suppression of amino acid synthesis, overexpression of gene related to sexual reproduction and spore formation, consequence induction of sexual reproduction and spore formation, deletion of gene RIM21, improved flocculation capability, up-regulated expression of FLO5, up-regulated expression of Lg-FLO1, enhanced resistance to cell wall inhibitors, overexpression of gene IZH1, down-regulation of ERG9 expression, diversion of metabolic flux to β-carotene pathway, addition of oleic acid, improvement in LDs-triacylglycerol metabolism, increase in β-carotene content, induction of extra constitutive β-carotene synthesis pathway, conversion of precursor farnesol to β-carotene, engineering of LDs related genes, minor promotion on β-carotene accumulation, deletion of gene SCW4, consequence increased secretion of hGH, consequence increased secretion of exendin-4, consequence increased secretion of hPTH, modification of gene SCW4, consequence improved function as secretion fusion partner, presence of O-glycosylation sites in Scw4p, consequence enhanced protein secretion, presence of hydrophilic sequences in Scw4p, deletion of gene Spt6-tSH2:Rpb1 interface, changes in transcription start site selection, changes in Spt6 accumulation pattern at specific sites, diminished splicing efficiency, impaired chromatin-mediated repression, role in maintaining +1 nucleosomes at ribosomal protein genes, deletion of CRE1/CreA, release from carbon catabolite repression, overexpression of CRE1/CreA, repression of cellulolytic gene expression, interaction of CRE1/CreA with Tup1-Cyc8, transcriptional repression, interaction of PoCyc8 with PoCreA, interaction of PoCyc8 with PoSet2, methylation of H3K36 by PoSet2, expression of TF PoCreA, positive correlation with di-methylation level of H3K36 in cellulolytic gene promoters, deletion of gene ncRNA, consequence 1: more abundant reverse-transcribed DNA (RT-DNA, expression of gene retron operon, consequence 1: efficient reverse transcription, consequence 2: more efficient genome editing, expression of gene retron RT-DNA, consequence 1: precise editing of cultured human cells, expression of gene RmproB, high protease activity of 1242.2 U/mL, optimal activity at pH 2.5, optimal activity at 40 °C, stability within pH 1.5-6.5, stability up to 45 °C, broad substrate specificity, milk-clotting activity of 3894.1 SU/mg, cleavage of specific peptide bonds in κ-casein, preparation of ACE-inhibitory peptides, high ACE-inhibitory activity IC50 of 0.195 mg/mL, expression of heterologous terpene synthases, increased isoprenoid production, metabolic engineering of the mevalonate pathway, metabolic engineering of the MEP pathway, expression tuning of heterologous genes, improved isoprenoid production, deletion of nucleosomes, increased GG-NER efficiency, presence of nucleosomes, obstructed GG-NER, overexpression of RNA polymerase-I, enhanced TC-NER, dislodgement of RNA polymerase-I, partial replacement by nucleosomes, expression of gene CYP5150L8, formation of HLDOA, expression of gene CYP5139G1, C-28 oxidation of HLDOA, formation of DHLDOA, adjustment of hygromycin dosage, manipulation of plasmid copy number pRS425-Hyg-CYP5150L8-iGLCPR, adjustment of geneticin G418 dosage, manipulation of plasmid copy number pRS426-KanMx-CYP5139G1, balanced expression of CYP5150L8 and CYP5139G1, improvement of DHLDOA production, deletion of nonoptimal codons, decrease in transcript's half-life, insertion of second nonoptimal cluster near 5' end, increase in mRNA's half-life, 5' end cluster suppression, reduction in ribosome queues, introduction of two nonoptimal clusters, increase in mRNA half-life up to fourfold, presence of two clusters, 5' end cluster determines mRNA half-life, expression of gene lgdA, consequence LGDH activity, consequence facilitates bacterial catabolism of levoglucosan, expression of gene sugar isomerase, expression of gene transport proteins, expression of L-A, provides proteins for expression, provides replication for both viruses, provides capsids for both viruses, expression of M, encodes the toxin, expression of L-A-1, associates with M1, expression of L-A-2, associates with M2, expression of L-A-28, associates with M28, expression of L-A-lus, associates with Mlus, deletion of gene Gcn5, consequence 1: decrease in histone acetylation, consequence 2: affects gene expression and development, deletion of gene chiffon, consequence 1: decrease in histone acetylation exceeding Ada2b mutation, expression of chiffon gene product CHAT, consequence 1: regulates expression of a subset of developmental genes, expression of chiffon gene product DDK, consequence 1: necessary for maternally driven DNA replication in the embryo, expression of genes related to central carbon metabolism flux, consequence 1: minimizes proteome cost in Crabtree-positive yeasts, expression of genes related to translation efficiency, consequence 1: lower ribosomal content and/or efficiency in Crabtree-positive yeasts, expression of genes related to ATP yield, consequence 1: maximized ATP yield in Crabtree-negative yeasts, expression of genes related to protein translation parameters, consequence 1: higher protein translation parameters in Crabtree-negative yeasts, deletion of gene FDC1, avoid ferulic acid deviation, expression of gene 4CL1, produce curcumin, expression of gene FerA, expression of gene PKSs from Oryza sativa, expression of gene PKSs from C. longa, expression of gene BbGT, consequence improved water solubility of natural products, consequence conversion of quercetin to five monoglucosylated products, consequence conversion of quercetin to one diglucosylated product, consequence production of quercetin-7-O-β-D-glucoside in E. coli, consequence production of quercetin-7-O-β-D-glucoside in P. putida, consequence production of quercetin-7-O-β-D-glucoside in P. pastoris, consequence production of quercetin-3-O-β-D-glucoside in S. cerevisiae, consequence stimulation by Ca<sup>2+</sup>, consequence stimulation by Mg<sup>2+</sup>, consequence stimulation by Mn<sup>2+</sup>, consequence inhibition by Zn<sup>2+</sup>, deletion of gene Gal80, consequence 1: stringent regulation of gene expression, expression of gene Gal4, consequence 1: copper-induced transcriptional activation, expression of gene Gal80, consequence 1: copper-inhibited repression, expression of gene CUP1, consequence 1: native copper-inducible promoter activity, expression of gene EGFP, consequence 1: 72-fold induction with CuIGR4 system, expression of gene responsible for lycopene production, consequence 1: 33-fold increase in lycopene production, gene conversion-like process of VLR genes, generation of distinct antigen receptors, expression of anti-carbohydrate VLRBs, isolation using yeast surface display platform, binding of anti-glycan VLRBs, enrichment by glycan microarrays, sequencing of VLR antibody clone, expression as VLR-IgG-Fc fusion protein, gene expression-based node addition, preserves resilience of PPI network, increases resilience of PPI network, mutation of gene MpFADS6[Q409R], consequence abolished desaturation, mutation of gene MpFADS6[M242P], mutation of gene MpFADS6[F419V], consequence significantly reduced catalytic activities, mutation of gene MpFADS6[A374Q], deletion of gene involved in glucose sensing, reduced glucose preference, overexpression of gene involved in xylose sensing, improved xylose utilization, modification of signaling pathway, enhanced xylose consumption rate, depression of whole-genome duplicates, growth adaptation to acidic stress, reduction of growth rate, increase in biomass production, overexpression of ROS response genes, adaptation to acidic stress, deletion of gene Fot1, reduced oligopeptide utilization, deletion of gene Fot2, overexpression of gene Fot1, increased oligopeptide utilization, overexpression of gene Fot2, improved fermentation efficiency, increased production of desirable organoleptic compounds, deletion of gene lac3, deletion of gene CpHsp24, overexpression of gene CpHsp24, increased laccase activity, interaction of protein CpHSP24, enhanced function of protein LAC3, deletion of gene associated with ribosome function, consequence 1: reduced protein synthesis, overexpression of gene related to energy metabolism, consequence 1: enhanced energy production, overexpression of Hsp70/Hsp90 and related molecular chaperones, consequence 1: increased recognition and degradation of misfolded proteins, overexpression of genes in ergosterol biosynthesis, consequence 1: increased tolerance to acetic acid stress, deletion of gene MoRPD3, lethal effect, depletion of gene MoRPD3, reduction in pathogenicity, delay in appressorium-mediated penetration, impaired invasive growth, smaller appressoria size, association of gene MoRPD3, regulation through TOR kinase pathway, overexpression of gene xxx, gene deletion simulations, gene expression data integration, deletion of gene cat, expression of gene cat, deletion of gene Set1, higher cellular mitochondrial reactive oxygen species (ROS, increased H2O2 sensitivity, prevents proper colony formation, defective expression of mitochondrial protein genes, disrupted mitochondrial membrane integrity, inducible expression of CePAH1, up-regulation of diacylglycerol, increase in oil content, supplementation with Saccharomyces cerevisiae fermentation products, reduced thoracic ultrasonography scores, less severe lung pathology scores, higher serum IL-6 concentrations, lower concentrations of serum nonesterified fatty acids, lower concentrations of beta-hydroxybutyric acid, overexpression of genes involved in pyruvate metabolism, overexpression of genes involved in glycolysis, overexpression of genes involved in pentose phosphate pathway, overexpression of genes involved in glucose transport, overexpression of genes involved in reducing power production, increase in intracellular glucose, increase in NADH content, decrease in acetate and its derivatives, deletion of gene for testosterone, consequence decreased sperm density and viability, deletion of gene for FSH, deletion of gene for LH, overexpression of gene for anti-apoptotic proteins, consequence improved sperm parameters, overexpression of gene for steroid synthetase, deletion of 3' UTR sequence elements, consequence 1: reduced mRNA stability, consequence 2: decreased translation, overexpression of consensus 3' UTR efficiency element, consequence 1: increased protein expression, binding of Puf proteins, consequence 1: increased expression, mutation within efficiency element motifs, consequence 1: altered binding affinity for Hrp1 protein, presence of consensus positioning element, consequence 1: minor effect on expression, deletion of GAL1, deletion of GAL80, overexpression of gene with synthetic GAL promoters, manipulation of UASs in GAL promoters, replacement of core promoter in GAL promoters, deletion of general stress response genes, improved robustness, overexpression of oxidative stress response genes, enhanced tolerance to inhibitors, overexpression of furan-aldehyde reductases, reduced toxicity of furans, deletion of certain transporter genes, altered transport capabilities, overexpression of specific drug:H+ antiporters, increased efflux of toxic compounds, overexpression of thiamin biosynthesis genes, improved thiamin levels, overexpression of biotin biosynthesis genes, improved biotin levels, deletion of gene ssl2, consequence: upstream TSS shifts at the model promoter ADH1, consequence: downstream TSS shifts at the model promoter ADH1, consequence: altering TSS selection by increasing the distribution of TSSs used at individual promoters, consequence: altering TSS selection by narrowing the distribution of TSSs used at individual promoters, deletion of gene BgtCDA1, inhibition of Bgt invasion, overexpression of gene TiAP1, up-regulation of cell wall xylan metabolism, up-regulation of lignin biosynthesis-related genes, up-regulation of defence genes involved in signal transduction, interaction of TiAP1, inactivation of BgtCDA1 deacetylation function, exposure of chitin oligomers, triggering of wheat immune response, inhibition of Bgt growth and penetration, enhancement of wheat resistance to pathogens, expression of gene HvHorcH, increased salt tolerance in yeast, increased protein levels in response to salinity treatment, increased salt tolerance in Arabidopsis thaliana, deletion of eCIS-related genes, no effect on morphological differentiation, no effect on antibiotic production, increased susceptibility to fungal invasion, SLP expression elevation, response to co-culture with fungi, co-culture with Bacillus subtilis, decreased SLP expression, increased antibiotic production, deletion of Cbp7, affects cytb mRNA stability, impairs detection of Cytb protein, deletion of Cbp8, interaction of Cbp7/Cbp8, modulates Cytb synthesis, modulates Cox1 synthesis, expression of gene ScCNS1, increased cordycepin production, expression of gene ScCNS2, up-regulation of gene ZWF1, enhanced pentose phosphate pathway activity, up-regulation of gene PRS4, enhanced purine metabolism, up-regulation of gene ADE4, addition of Cu²⁺, increased cordycepin yield, addition of adenine, deletion of gene for specific caffeoyl-CoA production, consequence MbA not enhanced in N. benthamiana, expression of gene Cc4CL1, consequence activation of coumaric and caffeic acid towards flavanone biosynthesis in yeast, expression of gene Cc4CL2, deletion of gene KanMX, alteration in transcription efficiency of neighboring genes, impairment of mRNA translation of neighboring genes, changes in protein abundance of neighboring genes, shift of transcription start site in 'head-to-head' orientation, introduction of new uAUG codons in expanded 5' UTR in 'head-to-head' orientation, activation of alternative polyadenylation signals in 'tail-to-tail' arrangement, alteration of 3' UTR in 'tail-to-tail' arrangement, expression of gene ZFN, deletion of gene LYS2, insertion at gene LYS2, optogenetic control of gene expression, precise control of cellular functions, optogenetic switches in yeasts, protein production, metabolite production, expression of gene product, consequence 1: fitness defect in specific mutant background, consequence 2: little effect on fitness in wild-type background, overexpression of wild-type gene, consequence 3: sensitization of yeast to increased dosage, overexpression of gene carrying dominant-negative missense mutation, consequence 4: model of protein-inhibitor interactions, deletion of gene akr1, increased intracellular ROS levels, cell death, deletion of gene erg3, calcium, inhibited increased intracellular ROS levels, inhibited cell death, induced expression of gene hxt1, induced expression of gene agp1, decreased cytosolic calcium concentration, calcium/calcineurin signalling pathway, induced expression of gene akr1, induced expression of gene erg3, deletion of U2AF1, alternative splicing, overexpression of two introns, alternative RNA isoforms, deletion of gene Pus4/TruB, consequence 1: greater proliferation rates, consequence 2: shortened lifespan, consequence 3: altered protein synthesis, consequence 4: differential synthesis of dozens of proteins, cloning and heterologous expression of gene Aa7DR1, Δ7-reduction activity in yeast, high expression of candidate genes in rhizome, accumulation of timosaponins in rhizomes, deletion of gene expression for key xylose-metabolism enzymes, reduced D-xylose consumption, deletion of gene expression for xylose reductase (XR), decreased specific activity of XR, deletion of gene expression for xylitol dehydrogenase (XDH), decreased specific activity of XDH, deletion of gene expression for xylulokinase (XK), decreased specific activity of XK, presence of D-glucose, inhibition of D-xylose consumption, prevention of D-xylose-mediated induction of XR gene, prevention of D-xylose-mediated induction of XDH gene, prevention of D-xylose-mediated induction of XK gene, deletion of BDF2, increased nuclear retention of BDF2 mRNA, mediated by RNase III-mediated decay (RMD, expression of Rnt1p, cleavage of BDF2 mRNA, inhibition of mRNA export, hyperactivation of RMD, deletion of gene NRRL3_09595, increased chitin deposition, sensitivity to SDS, production of unknown yellow pigment, deletion of gene cwcA, constitutive activation of cell wall stress response pathway, increased chitin content in cell wall, expression of gene Haa1, consequence relocation to the nucleus, expression of gene BM3R1, consequence control expression of a reporter gene, expression of gene reporter gene, consequence detection of acetic acid production, heterologous expression of VuNAC1, enhanced fermentation efficiency, altered cellular composition of biomolecules, conferred tolerance to high salinity, conferred tolerance to osmotic stress, conferred tolerance to freezing, conferred tolerance to aluminum toxicity, reprogrammed synthesis of nucleotides, reprogrammed synthesis of vitamin B complex, reprogrammed synthesis of amino acids, reprogrammed synthesis of antioxidants, reprogrammed synthesis of flavonoids, reprogrammed synthesis of energy currencies, reprogrammed synthesis of cofactors, improved survival during starvation, improved survival during stress recovery, deletion of gene TRM1, decreased induction of AOX1 promoter, deletion of gene TRM2, deletion of gene MPP1, overexpression of gene TRM1, increased induction of AOX1 promoter, overexpression of gene MPP1, presence of glycerol, catabolite repression of AOX1 expression, presence of glucose, deletion of S. uvarum chromosome I, loss of genetic material, duplication of S. cerevisiae chromosome III, increased gene dosage, duplication of S. uvarum chromosome VII-XVI, loss of heterozygosity on S. cerevisiae chromosome I, reduced genetic diversity, overexpression of unknown genes due to adaptation, increased sulfite resistance, slight overexpression of unknown genes due to adaptation, slightly improved ethanol tolerance, underexpression of unknown genes due to adaptation, decreased glucose and fructose fermentation capacity, deletion of lanosterol synthase, production of euphol, expression of EtOSC5, production of tirucallol, expression of EtOSC6, production of taraxasterol, production of β-amyrin, chromosomal integration of EtOSC5 and three other precursor genes, deletion of Mat*a*1, overexpression of transcripts for mating type specific genes, deletion of translation of pheromones and receptors, expression of gene ATF, consequence β-carotene production, consequence ∼4.8-fold higher β-carotene productivity compared to S. cerevisiae, consequence ∼59% of β-carotene productivity from farnesyl diphosphate in S. cerevisiae, consequence wide spectrum of inducible transcriptional outputs, consequence small growth defects, expression of gene PFUR17_02294, production of lecanoric acid, overexpression of gene patK, consequence 1: increased biosynthesis of 6-methyl salicylic acid, overexpression of gene patG, consequence 1: increased conversion of 6-methyl salicylic acid to m-cresol, deletion of gene ST, consequence 1: enhanced metabolic flux towards m-cresol production, deletion of gene EM, consequence 1: improved metabolic flux towards m-cresol production, promoter engineering, consequence 1: increased expression levels of patK and patG, gene multiplication, consequence 1: increased copy number of patK and patG leading to higher enzyme activity, fed-batch fermentation, consequence 1: increased m-cresol production to 2.03 g/L, deletion of ALT1, alanine accumulation, repression of GAD1, repression of UGA1, repression of UGA2, growth impairment, expression of ALT1, induction by alanine, repression of GABA shunt, deletion of UGA1, heat sensitivity, reduced NADPH pools, null expression of COX2, null expression of COX3, null expression of ATP6, decreased mitochondrial/nuclear DNA ratio, petite phenotype, deletion of gene TE repression mechanism, consequence 1: increased TE propagation, overexpression of gene Ty1_cer, consequence 1: higher translation efficiency, deletion of A at position -132 nt upstream of Spt15, increased isobutanol tolerance, insertion of G at position -65 nt upstream of Spt15, overexpression of ILV3 gene, increased isobutanol production, overexpression of ILV2 gene, overexpression of ARO10 gene, mutations of Spt15, increased expression of SPT15, downregulated expression of genes involved in TCA cycle, downregulated expression of genes involved in glyoxylic acid cycle, increased expression of genes related to cell stability, expression of gene Bs3, consequence hypersensitive response, consequence inhibition of yeast proliferation, consequence increase of intracellular oxidation state, mutation S211A in gene Bs3, consequence twofold increase in NADPH oxidase activity, consequence production of H2O2, heterologous expression of orotidine 5'-monophosphate decarboxylases, weak cross-species expression, pyrG promoters drive URA3 expression, efficient expression in yeast, inefficient cross-species splicing of Penicillium introns, deletion of stabilizing H-bonds, partially functional variants, elimination of conserved H-bond at N-latch position, non-functional Cin8 variant, additional H-bond stabilization of N-latch position, nonfunctional variant, replacement of N-latch asparagine with glycine, rescued functional defects, overexpression of Gas1, inhibition of glucan-elongating activity, overexpression of Gas2, overexpression of Crh1, inhibition of chitin-glucan transglycosylase activity, activation of cell-wall integrity pathway, transcriptional co-regulation, activation of high-osmolarity glycerol signaling pathway, up-regulation of genes involved in sugar fermentation, up-regulation of genes involved in ethanol metabolization, enhanced growth in adverse environments, up-regulation of genes involved in protective responses against abiotic stresses, production of high abundant proteins, expression of AHP1, production of peptides with health-beneficial activities, expression of TSA1, expression of HSP26, expression of SOD1, expression of HSP10, expression of UTR2, expression of ENO1/2, expression of TDH1/2/3, expression of ADH1/2, expression of FBA1, expression of PDC1, development of yETFL model for Saccharomyces cerevisiae, prediction of maximum growth rate by yETFL, prediction of essential genes by yETFL, prediction of overflow metabolism phenotype by yETFL, consequence 1: rapidly folding domains allow for efficient transmembrane integration with low-hydrophobicity thresholds, consequence 2: sequences that are intrinsically disordered allow for efficient transmembrane integration with low-hydrophobicity thresholds, consequence 3: very short sequences allow for efficient transmembrane integration with low-hydrophobicity thresholds, consequence 4: sequences capable of binding chaperones with high affinity allow for efficient transmembrane integration with low-hydrophobicity thresholds, consequence 5: long protein fragments interfere with membrane integration, requiring higher hydrophobicity, consequence 6: folding-deficient mutant domains interfere with membrane integration, requiring higher hydrophobicity, consequence 7: artificial sequences not binding chaperones interfere with membrane integration, requiring higher hydrophobicity, deletion of gene CWI pathway, consequence elevated ROS production, consequence cell death, deletion of gene VPS, deletion of gene calcium/calcineurin pathway, deletion of gene PHO pathways, deletion of gene hsm3, increase in frequency of mutations induced by different mutagens, suppression of UV-induced mutagenesis by inactivation of Polη polymerase, significant role of Polη for mutagenesis at NBP sites, lower efficiency of RNR gene expression after UV irradiation compared to wild-type cells, deletion of gene hif1, deletion of gene hat1, not specified in abstract, increase in dNTP levels, suppression of hsm3-dependent mutagenesis, suppression of hif1-dependent mutagenesis, Polη, responsible for hsm3-dependent mutagenesis, responsible for hif1-dependent mutagenesis, deletion of gene eIF4G, increased retention of eIF4A, increased efficiency of reinitiation on downstream initiation codons, mutation affecting eIF4A-binding site, elimination of specificity with mutation in cis-acting feature, mutation affecting integrity of eIF4G1-RNA2-binding domain, deletion of gene Gal4p, deletion of gene Gal80p, expression of gene Q System, expression of gene OptoQ-INVRT, expression of gene OptoQ-AMP, expression of gene tHMG1, consequence squalene production increase, consequence fluorescence intensity increase, deletion of gene rpc128-1007, upregulation of mRNA expression, derepression of general transcription factor Gcn4, increase in expression of ribosome biogenesis genes, downregulation of proteasome complex genes, deletion of gene Gas1, increased lysozyme activity, increased Ig proportion, increased mcsfra expression, increased hepcidin expression, overexpression of gene Gas1, enhanced disease resistance, overexpression of gene ARE2, increased ergosterol content to 10 mg/g DCW, overexpression of gene UPC2-1, increased ergosterol content to 16.7 mg/g DCW, overexpression of gene ACC1, increased ergosterol content from 16.7 to 20.7 mg/g DCW, deletion of gene related to glycosylation, consequence: improved functional expression of antibody fragments in S. cerevisiae, expression of gene related to α-amylase secretion, consequence: enhanced secretion capacity in HA strain, expression of gene related to amino acid metabolism, consequence: increased secretion of antibody fragments, expression of gene related to protein synthesis, expression of gene related to cell cycle regulation, consequence: altered secretion efficiency of antibody fragments, deletion of gene Ist2, overexpression of gene Osh6, overexpression of gene Osh7, expression of gene Rap1, consequence RPGs expression depends upon it, expression of gene Fhl1, consequence binds to RPG promoters, expression of gene Ifh1, consequence binds to RPG promoters and its binding correlates with RPG expression, expression of gene Sfp1, consequence activates RiBi and other growth-related genes, expression of gene Hmo1, consequence stabilizes or re-configures Fhl1-Ifh1 binding at RPG promoters, expression of gene Fpr1, consequence affects Fhl1-Ifh1 binding at RPG promoters, down regulation of gene Ifh1, consequence RPG expression is reduced under stress conditions, activation of RASTR, consequence controls RPG transcription through reversible condensation of Ifh1, deletion of gene IMS, decrease in isomultiflorenol production, deletion of gene CBS, decrease in cucurbitadienol production, overexpression of gene IMS, increase in isomultiflorenol production, overexpression of gene CBS, increase in cucurbitadienol production, deletion of gene alsgt-2, decrease in total SG, almost complete loss of GL-B, expression of gene ScARO10, increase in isoamyl alcohol titer, expression of gene ScARO10 with weak promoter P_UAS1B4-EXPm, highest titer increase, deletion of gene Gpa2, consequence regulating carbohydrate metabolism, deletion of gene Asc1, consequence regulating amino acid metabolism, consequence regulating purine metabolism, consequence determining amplitude of response to glucose addition, consequence affecting cell growth, deletion of cardiolipin synthesis, precluded stimulation of protein import, compromised cellular fitness, deletion of HAC1, promote nuclear accumulation of Hog1, deletion of IRE1, increase cytosolic and bud neck localization of Slt2, activation of UPR pathway, required for cadmium-induced phosphorylation of Hog1, Slt2, required for activation of UPR pathway through Swi6 and Rlm1, Hog1, regulate cellular process in absence of UPR pathway, deletion of gene RAD52, enhanced homologous recombination efficiency, increased simultaneous assembly of multiple fragments, disruption of multiple genes, efficient genome integration, assembly of multiple genes, regulation of gene expression, regulation of metabolic pathways, increased fatty alcohol production, deletion of SNORA48, change in translational efficiency, alteration in stoichiometric translation rates of sarcomere proteins, mutation in translation machinery, protein length-dependent shift in translational efficiency, deletion of N-linked glycosylation, consequence 1: improper protein folding, consequence 2: premature degradation, consequence 3: increased high-level expression yields, consequence 4: reduced immunological consequences, overexpression of OsFEX, enhanced fluoride tolerance in yeast, enhanced fluoride tolerance in Nicotiana benthamiana, suppression of OsFEX, fluoride-hyperaccumulative nature of rice cultivar, upregulation of OsFEX by gibberellic acid, increased OsFEX expression, upregulation of OsFEX by melatonin, suppression of OsFEX by abscisic acid, decreased OsFEX expression, deletion of β-isopropylmalate dehydrogenase gene, affects leucine biosynthesis, deletion of BCAA aminotransferase gene, affects BCAA biosynthesis, overexpression of BCAA aminotransferase gene, increases BCAA biosynthesis, regulation of leucine biosynthesis genes by LeuB, modulates response to leucine availability, deletion of gene PLN1, consequence 1: increased efficiency of converting DD to PPD by 394%, consequence 2: increased conversion rate of DD from 17.4% to 86.0%, overexpression of gene Pn3-29, consequence 1: reduced accumulation of PPD, consequence 2: increased production of CK to 41.3 mg/L/OD, depletion of gene Bdf1, consequence hypersensitivity of genes to depletion, depletion of gene Bdf2, depletion of gene Bdf1/2, consequence defects in transcription initiation, consequence defects in early elongation, consequence affect preinitiation complex formation, consequence reduced recruitment of TFIID, consequence reduced recruitment of Mediator, consequence reduced recruitment of basal transcription factors, deletion of gene DER1, deletion of gene ERO1, overexpression of heterologous proteins, consequence increased ER stress, consequence reduced protein secretion, deletion of gene CLEC7A, decrease in IL-8 production, decrease in TNF expression, overexpression of gene CLEC7A, increase in CD80 surface expression, increase in CD86 surface expression, increase in IL-8 production, increase in TNF mRNA expression, increase in IL1B mRNA expression, increase in IL6 mRNA expression, codon choice of genes, regulation of translation process timing, impact on protein folding classes, impact on functional classes involved in transcription process, low impact on proteins associated with energy production and metabolism, low average score k-mers, occurrence in coil or close to coil regions, high average score k-mers, occurrence in alpha helices and close to coil regions, significant codon patterns, enrichment at the 5' end of protein-coding sequences, deletion of leptin, higher in control group, deletion of growth hormone, deletion of MMP3 gene expression, lower in treatment group, overexpression of hormone receptors, higher in brain, overexpression of plasma cholesterol, overexpression of MDA, overexpression of AMP kinase, expression of gene SSU1, haploinsufficiency of gene SSU1, deletion of gene znf1, increased sensitivity to ethanol, increased sensitivity to β-mercaptoethanol, increased sensitivity to calcofluor white, increased glycerol levels, deletion of gene smi1, overexpression of gene znf1, increased ethanol tolerance, activation of gene gup1 by znf1, increased glycerol production, activation of gene gpp1 by znf1, activation of gene gpp2 by znf1, activation of gene gpd1 by znf1, activation of gene gat1 by znf1, activation of gene ole1 by znf1, increased fatty acid production, activation of gene fks1 by znf1, increased cell wall biosynthesis, activation of gene sed1 by znf1, activation of gene smi1 by znf1, activation of gene hsp30 by znf1, increased unfolded protein response, activation of gene hsp104 by znf1, activation of gene kar1 by znf1, activation of gene lhs1 by znf1, deletion of PDC genes, elimination of ethanol production, overexpression of PCAD, minimized PCA production, insertion of DHS dehydratase from P. anserina, biosynthesis of muconic acid, insertion of PCA decarboxylase from K. pneumoniae, increased yeast pitching rate, lowered biomass formation, use of PPG, in situ product removal, expression of recombinant proteins, increased production speed, ease of handling, expression of insulin analogues using PURE IVTT system, correct insulin expression and folding, similar insulin receptor affinity to yeast-produced insulin, deletion of gene AoTgla, increase of total lipid contents, deletion of gene AoTglb, increase in TAG fraction, prolonged lipid-accumulating stage, reduction in specific rate of lipid turnover, achievement of maximal lipid levels, achievement of maximal TAG levels, deletion of RTG pathway genes, activation of amino acid biosynthesis in U cells, repression of mitochondrial ribosome components expression in U cells, reduced mitochondrial translation in U cells, deletion of gene laccase, consequence reduced lignin-degrading ability, expression of gene laccase from Agrocybe pediades, consequence increased ability to oxidize aromatic compounds, expression of mutated laccase variant, consequence improved activity at neutral pH, consequence enhanced tolerance to inhibitors, consequence ability to oxidize high-redox potential mediators and recalcitrant organic dyes, expression of laccase with N-glycosylation sites, consequence increased secretion of enzyme, consequence altered catalytic activity, deletion of functional mitochondria, cell stress adaptation, expression of RTG-target genes, changes in nuclear gene expression, expression of CIT2 gene, increased Cit2p-GFP protein product, phosphorylation of Rtg1/3p, controls RTG-target gene transcription, consequence: translation efficiency varies across different organisms due to differences in sequence features, consequence: codon adaptation index shows high correlation in all analyzed organisms, deletion of gene Gcn2, improved α-amylase production, reduced intracellular H₂O₂ levels, reduced Hac1 splicing ratio, increased expression of antioxidants, increased expression of PDI1, deletion of gene function, tolerated in one genetic background, lethal in another genetic background, gain-of-function of modifier gene, suppression of fitness defects, deletion of gene MoWHI2, multiple appressoria formed by one conidium, pathogenicity significantly reduced, deletion of gene Mopsr1, cAMP levels increasing, inappropriate activation of MoTor signalling pathway, increased expression of MoTOR, activation of MoTor signalling pathway, increased expression of MoRS3, deletion of polyasparagine domain in Azf1p, genes upregulated related to carbon metabolism, genes upregulated related to storage, translation of uORFs, inhibition of main coding sequence translation, premature dissociation of ribosomal units, ribosome stalling, translational arrest of primary coding sequences, during stress, increased relative abundance of uORF-encoded peptides, decreased translation of uORFs, facilitating CDS translation during stress, up-regulation of translation, in a subset of genes during stress, uORFs, act as translational switches in genes encoding proteins needed rapidly upon stress, deletion of gene los1, increased extracellular level of anti-CD22 scFv, altered susceptibility to protein synthesis inhibitors, increased total proteins secreted, deletion of 5caC, consequence increased Pol II translocation, overexpression of 5caC, consequence Pol II pausing, interaction of 5caC with fork loop 3, consequence compromised NTP binding, interaction of 5caC with fork loop 2, presence of 5caC, consequence suppressed bridge helix kinking, deletion of gene Pbp1, consequence slower growth on medium containing glycerol, consequence slower growth on medium containing lactate, consequence decreased expression of gene PCK1, consequence decreased expression of gene FBP1, consequence decreased expression of gene COX10, consequence decreased expression of gene COX11, deletion of gene Dcp1, consequence recovery of COX10 expression in pbp1Δ mutant, deletion of gene Xrn1, consequence recovery of COX11 expression in pbp1Δ mutant, expression of UPO genes, production of four active UPOs, one-step purification, application in enantioselective conversion, yield of 24 mg/L secreted UPO enzyme, preparative scale conversion of a phenethylamine derivative reaching 98.6 % ee, overexpression of gene KCS1, increased octanoic acid titers, overexpression of gene FSH2, consequence: impact on yeast life span, consequence: involvement of TOR and cAMP/PKA pathways in response to stress factors and nutrients, consequence: reductive metabolism of living cells clears undesirable carbonyl compounds, consequence: improvement of product's shelf life through bottle conditioning, deletion of gene YGP1, consequence 20-fold increase of fluorescence upon addition of 30 mM acetic acid, overexpression of reporter gene, consequence significant drop of the fluorescence, application of yeast pheromone-based signal amplification system, consequence compensation for drop of fluorescence, deletion of intron in HAC1 mRNA, translational derepression of HAC1 mRNA, production of Hac1 protein, activation of expression of enzymes and chaperones, increase in protein-folding capacity of the cell, regulation by Vps34, regulation of HAC1 mRNA translation, regulation by TOR, regulation by GCN2, deletion of gene HXK1, deletion of gene HXT1, deletion of gene HXT2, deletion of gene HXT4, deletion of gene associated with mitochondria, mitochondrial damage, ROS accumulation, ATP content changes, deletion of oxidative phosphorylation-related gene, higher ROS production, reduced ATP levels, expression of FRE5, up-regulated during iron limitation, expression of FTR1, expression of FET3, overexpression of LcFTR1 I, high expression of intracellular transport of Fe3+, overexpression of LcFTR1 II, moderate expression of intracellular transport of Fe3+, overexpression of LcFTR1 IV, no direct consequences related to gene expression/deletion, expression of gene Talaromyces minioluteus dextranase, consequence increased extracellular yield of r-TmDEX, consequence no inhibition of cell growth, consequence volumetric productivity of 1706 U/L/h, consequence optimal activity at pH 4.5-5.5, consequence optimal activity at temperature 50-60 °C, addition of sucrose, consequence retention of r-TmDEX activity after 1-h incubation at 50-60 °C and pH 5.5, application of crude preparation of secreted r-TmDEX, consequence complete removal of bacterial dextran in deteriorated sugarcane juice, overexpression of gene not specified, increased protein synthesis in mitochondria, decreased protein synthesis in mitochondria, deletion of AHLs, consequence 1: major QS disruption, consequence 2: significant molecular changes, consequence 3: significant phenotypic changes, consequence 4: drastic changes in social interactions between C. violaceum and B. cereus, consequence 5: drastic changes in social interactions between C. violaceum and S. cerevisiae, consequence 6: drastic changes in social interactions between C. violaceum and murine macrophages, consequence 7: drastic changes in social interactions between C. violaceum and planarian Schmidtea mediterranea, deletion of gene pdc1, ethanol conversion decreased, glycerol conversion increased, acetic acid concentration decreased, acetoin concentration increased, pyruvate decarboxylase activity decreased, deletion of gene pdc5, deletion of gene mbkZ, reduced production of red pigments metacycloprodigiosin and undecylprodigiosin, expression of gene mbkZ, increased production of red pigments metacycloprodigiosin and undecylprodigiosin, consequence Gal4p expression without interruption, stop codon recognition by eRF1, termination of translation, near-cognate tRNA outcompetes eRF1, stop codon readthrough, readthrough, translation into mRNA 3'-UTR, cis-acting elements modulation, termination efficiency, readthrough efficiency, wild-type or temperature-sensitive eRF1 expression, ribosome profiling analysis, stop codon identity, influence on readthrough efficiency, nucleotide after stop codon, codon in the P site, 3'-UTR length, nts +5 to +9, mild effect on readthrough efficiency, low readthrough genes, shorter 3'-UTRs in thermally inactivated eRF1 cells, high readthrough genes, longer 3'-UTRs in thermally inactivated eRF1 cells, new mRNA or peptide features identification, affecting translation termination and readthrough efficiency, deletion of gene ARO8, reduced 2-PE content, reduced ethanol yield, decreased biofilm formation, reduced extracellular polysaccharides, reduced FLOp expression, deletion of gene ARO9, expression of S. cerevisiae gene, linked to nitrogen metabolism, linked to sulfur metabolism, linked to thiamine metabolism, deletion of gene, efficient CRISPR-mediated genome editing, expression of gene pathway, β-carotene synthesis, violacein production, R-loop accumulation, DNA double-strand breaks, R-loop presence, genome instability, deletion of gene involved in l-proline metabolism, deletion of gene involved in l-arginine metabolism, overexpression of gene in ubiquitin system, overexpression of transcription factors, optogenetic activation of gene expression, increased transcriptional response to blue light, 41-fold induction between dark and light conditions, efficient activation at low light duty cycles, strong homogeneous light-induction in bioreactors, enzyme expression controlled by optogenetic circuits, improved production of lactic acid, improved production of isobutanol, improved production of naringenin, deletion of asparaginase genes, deletion of maltose utilization loci, deletion of several genes from the tandem array of the DUP240 family, loss of a stop codon in the FLO8 gene, overexpression of gene CvHSF30-2, consequence 1: confers tolerance to heat stress in Escherichia coli, consequence 2: confers tolerance to heat stress in Saccharomyces cerevisiae, deletion of gene HSF30-2, consequence 1: leads to heat sensitivity in tobacco, consequence 2: leads to heat sensitivity in Clematis, consequence 3: elevates heat shock protein (HSP, negative regulation by gene CvHSFB2a, consequence 1: reduces heat shock protein (HSP, overexpression of gene RtFAD2, consequence increased LA contents, consequence increased LA titers, consequence increased lipid titer, consequence increased lipid content, overexpression of gene RtFAD1, consequence down-regulated in co-overexpression with RtFAD2, expression of gene Lbo_PPTQ, expression of gene Lbo_31670, chain shortening of (Z, Z)-9-dodecenoic acid), (expression of gene Lbo_49602, chain shortening of (Z, deletion of gene GSP2, deletion of gene CTT1, overexpression of gene HXT1, deletion of gene BdFAR4, reduced content of C20:0 fatty alcohols, reduced content of C22:0 fatty alcohols, overexpression of gene BdFAR4, accumulation of C20:0 fatty alcohols, accumulation of C22:0 fatty alcohols, transcriptional regulation by BdMYB41, increased BdFAR4 transcript levels, no specific consequence related to gene expression/deletion, overexpression of gene XYL1, faster xylose consumption, overexpression of gene XYL2, overexpression of gene XKS, overexpression of gene XUT4, overexpression of gene CeDGAT2-2, consequence TAG biosynthesis restoration in yeast mutant H1246, consequence significant increase in oleic acid level in transgenic yeast, consequence dramatic enhancement of leafy oil by 7.15-fold in tobacco compared to wild-type, consequence dramatic enhancement of leafy oil by 1.7-fold in tobacco compared to plants expressing AtDGAT1, consequence increase in oleic acid from 5.1% to 31.33% in tobacco leaves, consequence reduction of saturated fatty acids in tobacco leaves, consequence small decline in starch content in tobacco, consequence no significant negative impact on photosynthesis in tobacco, consequence no significant negative impact on growth rates in tobacco, consequence no significant negative impact on seed germination in tobacco, shuttle expression of GAL1/10 bidirectional promoter, catabolite repression in E. coli, shuttle expression of multigene pathway, cinnamyl alcohol production, expression of gene ads, enhanced artemisinin production, expression of gene fps, gene expression of metabolic specialization genes, consequence 1: cellular adaptation to grape must environment, consequence 2: ethanol tolerance, gene expression of unknown key genes, consequence 1: impact on industrial wine fermentation outcomes, gene expression of unknown pathways, consequence 2: need for further research, expression of gene psulω3, consequence eicosapentaenoic acid production, expression of gene pmd17, up-regulation of genes associated with one-carbon metabolism, improved cellular redox homeostasis, down-regulation of genes related with DNA repair, reduced DNA repair activity, down-regulation of genes related with stress responses, down-regulation of genes related with ion homeostasis, altered ion homeostasis, down-regulation of genes associated with protein quality control systems, lower protein turnover, expression of gene HXT7, suppression in presence of glucose, derepression by xylose, massive accumulation of xylitol, expression of gene XYL2, expression of gene XYL1-K270R, balanced xylose metabolism, expression of pentose phosphate pathway genes, expression of XYL1-K270R and XYL2, ethanol production, overexpression of vtc2, increased VC accumulation, fusion and expression of L-GalDH and L-GLDH, balanced gene expression and cell growth, highest VC production from glucose, expression of gene OVCA2, increased production of reactive oxygen species, decreased cell growth, disturbed mitochondrial morphology, reduced membrane potential, increased chromatin condensation, externalization of phosphatidylserine, complementation of gene OVCA2 in fsh3Δ cells, reduced cell growth, increased apoptotic phenotypes, overexpression of Haa1p, improved xylose consumption rates, enhanced ethanol yield, overexpression of Tye7p, co-overexpression of Haa1p and Tye7p, increased xylose consumption rate, deletion of gene PTH3, impaired respiratory growth, reduced protein expression of mtDNA-encoded genes, presence of antibiotics, stalled mitoribosomes, deletion of gene PTH4, less impairment in respiratory growth compared to PTH3 deletion, overexpression of gene ChbZIP1, increased alkali resistance in Saccharomyces cerevisiae, increased alkali resistance in Arabidopsis, upregulation of gene GPX1, upregulation of gene DOX1, upregulation of gene CAT2, upregulation of gene EMB, enhanced antioxidant ability in Arabidopsis, expression of gene TPI1, increase in plasmid viability, increase in plasmid copy number, overexpression of gene RAF1, higher copy number, overexpression of gene REP2, overexpression of gene TPI1, smaller cell-to-cell variation, increased DHAA production, biosensor development for small molecule detection, biosensor application in regulating gene expression in yeast, biosensor application in detecting metabolite production from multi-enzyme biosynthetic pathway, expression of hexose transporters, emergence of glucose gradients, expression of other genes regulated by glucose availability, quantified with intra-colony growth rates, deletion of gene PpHCT, consequence 1: reduced production of caffeate derivatives, consequence 2: impaired formation of an intact cuticle, expression of gene D-xylose isomerase, consequence faster aerobic growth on D-xylose media, consequence 72% higher D-xylose isomerization rate, consequence 2.6 times higher specific activity, consequence 37% lower K<sub>M</sub> for D-xylose, consequence higher activity over a broader temperature range, deletion of gene dugB, increase in GSH content, deletion of gene dugC, reduced conidium production, disturbed sexual development, downregulation of steC, downregulation of sskB, downregulation of hogA, downregulation of mkkA, downregulation of flbA, downregulation of flbC, downregulation of flbE, downregulation of nosA, downregulation of rosA, downregulation of nsdC, downregulation of nsdD, slowed depletion of GSH pools, reduced accumulation of reactive oxygen species, decreased autolytic cell wall degradation, decreased enzyme secretion, increased sterigmatocystin formation, over-expression of ILV2, increased isobutanol titers, over-expression of ILV3, over-expression of ARO10, mutation in multiple genes, improved isobutanol tolerance, over-expression of CWP2, over-expression of SRP4039, stalled mRNA translation, production of incompletely synthesized proteins, stalled translation of nonstop mRNA, nucleolar sequestration of defective proteins, polylysine tracts production by translation of poly(A) tails, nucleolar sequestration, increase in solubility of invading proteins, nucleolar disruption, alteration of nucleoli dynamics, alteration of nucleoli morphology, alteration of resistance to stress in cell culture, alteration of resistance to stress in intact flies, deletion of DDK, reduced phosphorylation of Mcm2-7 tails, overexpression of DDK, increased phosphorylation of Mcm2-7 tails, phosphorylation of Mcm2-7 tails, increased number of CtGs formed, formation of CtGs, increased frequency of CMG formation, knockdown of Trcyc8, impaired cellulase gene expression, knockdown of Trtup1, repression of Trtup1, defect in XYR1 occupancy, repression of Trcyc8, loss of histone H4 at cellulase gene promoters, XYR1 overexpression, cannot overcome defects in Trtup1 repressed cells, cannot overcome defects in Trcyc8 repressed cells, overexpression of gene RPL40B, increase in octanoic acid titers by 40%, deletion of survivin gene expression, decreased cell viability, induced apoptosis, deletion of gene CCP1, consequence increased pyruvate productivity, overexpression of gene CCP1, consequence decreased pyruvate productivity, consequence activated respiratory metabolism, consequence increased reactive oxygen species, focus on Pichia pastoris as host for heterologous protein expression, high-cell density fermentation method for P. pastoris discussed, improvements in gene-editing tools, strain improvement for P. pastoris, expression of gene CaNES1, consequence differential transcription in different tissues of C. angulatus, expression of gene CaNES2, consequence production of trans-nerolidol 7.01 g/L in S. cerevisiae, upregulation of SWEET gene, sucrose accumulation in M. truncatula root tissue, upregulation of GspInv gene, increased sucrose utilization by w5, upregulation of GspSUT1 gene, increased sucrose transport by w5, heterologous expression of GspInv, invertase function confirmed, heterologous expression of GspSUT1, sugar transporter function confirmed, deletion of Hsf1p binding site, reduced copper-dependent gelatin induction, overexpression of CUP1 promoter, increased gelatin expression, induction by copper, dose-dependent gelatin expression, deletion of gene MoMca1, delay in conidial germination, attenuation of disease severity, deletion of gene MoMca2, deletion of gene MoMca1 and MoMca2, increased radial growth in presence of oxidative stress, increased accumulation of insoluble aggregates, heterologous expression of fungal BGCs, possibility investigation, effectiveness evaluation, deletion of gene GFP, consequence assessed by levels of GFP fluorescence, consequence assessing growth rates of GFP integrated strains, expression of gene synthetic gRNA-binding sites, consequence resulted in five different yeast strains harbouring 1-5 synthetic gRNA-binding sites in their genomes, expression of gene 2-, 3-, 4-, or 5-genes, consequence demonstrated with efficiencies from 85% to 98%, expression of gene 6-gene, consequence generating a yeast strain producing a sesquiterpene lactone, kauniolide, deletion of gene AcOTAbZIP, loss of OTA, loss of OTB, loss of OTβ, down-regulation of AcOTApks, down-regulation of AcOTAnrps, down-regulation of AcOTAp450, down-regulation of AcOTAhal, anthranilate accumulation, expression of anthranilic acid methyltransferase 1, methyl anthranilate production, deletion of gene NRG1, consequence production of mannosyl-erythritol lipids, consequence inhibition of expression of siderophore cluster genes, consequence filamentous growth, consequence virulence, consequence alteration of 368 genes expression, consequence up-regulation of 205 genes, consequence down-regulation of 163 genes, consequence affects HOG1 for osmotic stress, consequence affects RIM101 for alkaline pH, consequence affects WCO1 encoding PAS domain, consequence glucose repression, consequence metabolic adaptation to adverse conditions, consequence cellular transport, consequence cell rescue, consequence defense, consequence interaction with acidic pH environment, gene expression changes, inhibition of cell proliferation, cell wall damage, alteration of function and morphology of organelles, presence of oxidative stress bio-indicators, genotoxicity, expression of gene β-carotene pathway, expression of gene zeaxanthin pathway, consequence increased zeaxanthin production, expression of gene astaxanthin pathway, consequence increased astaxanthin production, overexpression of polytryptophan sequences, induction of Rqc2-independent RQC, requirement of Hel2-mediated ribosome ubiquitination, requirement of the RQT complex, inhibition of CAT-tailing by tandem CGA codons, induction of Hel2-mediated canonical RQC-coupled NGD, induction of RQC-uncoupled NGD outside the stalled ribosomes, deletion of cap in mRNA, consequence 1: reduced translation efficiency, expression of BBV-seq, consequence 1: increased translation efficiency on cap-free mRNA, consequence 2: enhanced cap-dependent translation initiation, interaction of BBV-seq with eIF4G1, consequence 1: promotion of cap-independent translation initiation, expression of gene AnfH, active in electron donation to NifDK, expression of gene AnfDK, exists as apo-AnfDK complex, addition of FeMo-co, activates apo-AnfDK for acetylene reduction, deletion of essential subunits of 26S proteasome, sensitivity to genotoxic stress, sensitivity to proteotoxic stress, sensitivity to high salinity, sensitivity to osmolarity, deletion of gene ADY2, reduced carotenoid accumulation, deletion of gene HES1, deletion of gene CUP1, overexpression of gene ACE1, increased carotenoid yield, addition of copper ions, consequence 1: improvement of biofuels yield, consequence 2: improvement of chemicals yield, consequence 3: enabling real-time control through optogenetic circuits, deletion of gene Atg16, fails to rescue autophagy-defective phenotype, mutation in amphipathic α-helix of Atg16, mutation in coiled-coil region of Atg16, deletion of gene PMA1, reduction in ribosome sequestration at the ER membrane, overexpression of cell-wall component genes, increased biosynthetic demands in the early secretory pathway, deletion of gene Dbr1p, accumulation of intron RNA lariats, equivalent production of minus strand strong stop DNA, accumulation of branched Ty1 RNAs, differential expression of ribosomal proteins, affecting relative proportion between ohnologs, differential expression of cytosolic ribosomal proteins, reprogramming of translation apparatus, differential expression of mitochondrial ribosomal proteins, differential expression of genes coding for products related to translation processes, affecting translation accuracy, differential expression of genes coding for products related to rRNA post-transcriptional maturation, differential expression of genes coding for products related to mRNA decapping, differential expression of genes coding for products related to snoRNA synthesis, deletion of kinase module in S. cerevisiae core-TFIIH, reduced dsDNA-stimulated ATPase activity, upstream shift in start-site observed in vitro, deletion of butyrate-producing commensals, increased Enterobacteriaceae, overexpression of inflammatory gene signatures, overexpression of extracellular matrix gene signatures, reduced levels of butyrate-producing commensals, deletion of respiratory electron transport gene signature, increased inflammatory gene signatures, overexpression of respiratory gene signatures, decreased inflammatory gene signatures, deletion of extracellular matrix gene expression, presence of Aggregatibacter and Blautia bacteria, adriamycin interaction with phosphofructokinase, impaired enzyme activity, adriamycin interaction with enolase, adriamycin causes dysregulation of hexokinase, impaired glycolysis, adriamycin causes dysregulation of phosphoglycerate mutase, adriamycin causes dysregulation of enolase, adriamycin causes downregulation of pyruvate kinase, adriamycin causes downregulation of sod, adriamycin causes upregulation of bax, promotes apoptosis, adriamycin causes upregulation of caspase, deletion of gene CYP719A37, consequence 1: no methylenedioxy bridge formation with ferulic acid, consequence 2: no methylenedioxy bridge formation with feruperine, expression of gene CYP719A37, consequence 1: methylenedioxy bridge formation with feruperic acid, consequence 2: production of piperic acid, expression of tlr2, consequence elevation, expression of il1r1, expression of irak4, expression of tollip2, expression of tnfα, expression of il10, expression of tgfβ, expression of il1β, expression of ifnγ, expression of il12, expression of gata3, overexpression of gene PDR3, improvement of lactic acid tolerance, contribution to yeast stress tolerance, deletion of gene MSS2, impaired growth upon chitosan treatment, slow growth on glycerol as sole carbon source, impaired ATP production, impaired invasive ability in vitro, impaired invasive ability ex vivo, impaired biofilm formation, repression of gene MSS2, inhibited by chitosan, leads to repression of mitochondrial function, SNF1 overexpression, enhanced cell tolerance in freezing, enhanced fermentation capacity in freezing, upregulated proteasome, altered carbon source metabolism, altered protectant molecule metabolism, changed cell membrane components, increased leucine level before freezing, increased proline level before freezing, increased serine level before freezing, increased isoleucine level before freezing, increased arginine level before freezing, increased homocitrulline level before freezing, increased glycerol level before freezing, increased palmitic acid level before freezing, increased lysophosphatidylcholine level before freezing, increased lysophosphatidylethanolamine level before freezing, maintained high proline level after freezing, maintained high lysine level after freezing, maintained high glycerol level after freezing, increased lysophosphatidylcholine level after freezing, increased lysophosphatidylethanolamine level after freezing, expression of gene chimeric cellobiohydrolase I, consequence glycosylation differences, consequence surface charge differences, consequence thermal stability differences, consequence proteolytic stability differences, consequence biomass digestion efficacy differences, expression of gene chimeric cellobiohydrolase I in L. starkeyi, consequence inferior cellulase production, expression of gene chimeric cellobiohydrolase I in S. cerevisiae, consequence lowest activity against dilute-acid-pretreated corn stover, expression of gene chimeric cellobiohydrolase I in Y. lipolytica, consequence lower glycosylation extent, consequence better protease stability, consequence higher activity against dilute-acid-pretreated corn stover, deletion of gene Gbp2, consequence 1: promotion of degradation of faulty pre-mRNAs, deletion of gene Hrb1, expression of gene Gbp2, consequence 1: function in nonsense mediated decay of spliced premature termination codon-containing mRNAs, expression of gene Hrb1, consequence 2: help to transmit the Upf1-mediated PTC recognition to the transcripts ends, consequence 3: promote translation repression of spliced transcripts that contain a PTC, consequence 4: facilitate degradation of RNA, consequence: regulation of translation elongation ensures correct folding of proteins, consequence: detection of widespread ribosome collisions, consequence: collisions occur more frequently in gap regions between α-helices, consequence: paused or collided ribosomes are associated with specific chaperones, deletion of gene AGP1, reduced higher alcohol formation, deletion of gene GDH1, deletion of gene THR6, deletion of gene HTZ1, consequence increased tyrosine biosynthesis, consequence increased p-coumaric acid titer, random mutagenesis, consequence increased tyrosine production, deletion of gene Rpn4, overexpression of gene RPN4, deletion of gene MSN2, decrease in liamocin yield by 92.28%, complementation of gene MSN2, restoration of liamocin synthesis, knockout of gene BCY1, decrease in liamocin yield by 96.73%, deletion of gene HOG1, decrease in liamocin biosynthesis by 98.09%, reduction in Msn2 and liamocin synthesis, high glucose and low nitrogen conditions, positive regulation of genes related to liamocin biosynthesis by Msn2, deletion of gene expression, consequence 1: inability to balance the glycolytic pathway through the glycogen/trehalose shunt, expression of new isoforms of glycolytic enzymes, consequence 1: increase in the amount of adenosine triphosphate, consequence 2: increase in efficiency of product formation, consequence 3: maintenance of glycolytic balance, expression of gene speD, increased spermidine titer, expression of gene speE, co-expression of gene speD and speE, spermidine titer of 105.2 mg/L, fermentation medium optimization, spermidine titer increased to 227.4 mg/L, consumption of S-adenosylmethionine, accumulation of spermidine, expression of SLC28A3, increased liver mRNA, expression of SLC29A1, expression of SLC29A2, expression of PC, expression of PCK1, expression of FBP1, expression of SREBP1c, expression of HSL, expression of CYP7a1, deletion of EAAT2, decreased placental gene expression, deletion of EAAT3, deletion of LAT1, deletion of PAT1, deletion of PPARγ, decreased placental protein expression, deletion of AKT, deletion of phosphorylated-AKT, deletion of phosphorylated-mTOR, deletion of Raptor, deletion of gene Yas1p, deletion of gene Yas2p, deletion of gene Yas3p, deletion of gene BDH1, consequence 1: significant enhancement of acetoin synthesis, overexpression of gene BDH2, consequence 2: 2.6-fold increase of TTMP production, overexpression of gene ALS, consequence 1: enhancement of diacetyl synthesis, replacement of promoter ILV2 with PGK1p, consequence 1: no further increase in diacetyl, acetoin, and TTMP production, mutations in splicing factors, carcinogenic mutations, faithful selection of key intronic regions, splicing modulators, molecular mechanism of splicing, pre-mRNA cleavage, chemical details by quantum-classical molecular dynamics, deletion of gene ERAD components, overexpression of gene ERAD components, deletion of gene fab1, affects growth, affects conidia development, affects sclerotial development, affects cellular acidification, affects metal ion homeostasis, affects aflatoxin production, affects pathogenicity, leads to vacuolar dysregulation, impairs aflatoxin synthesis, deletion of gene Spt4/5, consequence reduced association with elongation complexes, overexpression of gene Gal4-VP16, consequence increased binding of RNApII to template DNA, binding of gene Spt4/5, consequence association of mRNA binding protein Hek2, expression of gene TaCslA12, production of wheat-like linear mannan, lack of glucose residues in mannan, moderate acetylation of mannan, consequence suppressed expression, deletion of gene CXCL16, deletion of gene IL8, deletion of gene IL1B, consequence tended to decrease expression, overexpression of gene mtRNAP, consequence promoter melting, consequence template alignment, consequence DNA scrunching, consequence abortive synthesis, consequence transition into elongation, interaction with transcription factor MTF1, consequence base-specific interactions with flipped non-template nucleotides \"AAGT\", formation of initiation complex IC, consequence positioning of RNA and NTP analog UTPαS, consequence NT scrunching into an NT loop, stabilization by MTF1 C-tail, consequence stabilization of the scrunched NT loop, dynamic equilibrium between IC and PmIC states, consequence frequent scrunching/unscrunching transitions, steric clashes of the inflating NT loop and growing RNA:DNA with the C-tail, deletion of gene RME1, abolishes chlamydosporulation, overexpression of gene RME1, bypasses the requirement for chlamydosporulation cues and regulators, activation of gene expression by Rme1, leads to upregulation of genes during chlamydosporulation, expression of genes related to amino acid metabolic processes, increased 2,3-BDO tolerance, introduction of synonymous mutations in genes, improved allocation of ribosomes, decreased traffic jams during translation, increased resources availability in the cell, improved growth-rate, expression of gene CYP716A249, consequence production of oleanolic acid, consequence oxidation of C-28 position of β-amyrin, expression of gene S. cerevisiae terminator, consequence tunable range of 17-fold in P. pastoris, consequence maintenance of function in P. pastoris, consequence successful tuning of protein expression levels, expression of gene betaxanthin pathway, expression of gene mRNA abundance, consequence increase in protein expression levels, stability of mRNA 3'-UTR secondary structure, consequence reflection of protein expression levels, consequence mitochondrial function modulation, consequence cellular homeostasis maintenance, consequence bioenergetic production modulation, general discussion on ribosome heterogeneity and its role in translation specificity, failure in ribosome-mediated regulation of protein synthesis can play a crucial role in disease development, expression of gene FgSULT1, increased solubility of unnatural natural product sulfates, modification of benzenediol lactones, modification of anthraquinone, expression of gene GAL1, consequence 132.9 ± 22.6-fold induction in darkness, expression of metabolic enzymes, consequence boosts production of lactic acid by more than 50%, consequence boosts production of isobutanol by more than 15%, deletion of gene coq2, inhibition of CoQ biosynthesis by Bz, overexpression of gene ppt1, overcoming inhibition of CoQ biosynthesis by Bz, expression of gene, deletion of gene Gal4, insufficient inducibility, overexpression of gene Gal4 mutants, improved transcriptional regulation, overexpression of gene Gal4<sup>T406A/V413A</sup>, increased transcription of P<sub>GAL</sub>-driven lycopene pathway genes, 48% higher lycopene accumulation, gene overexpression in brewer's yeast strains, adaptation to brewing environment, gene overexpression in baker's yeast, adaptation to osmotic stress, gene overexpression in yeast used for ethanol production, adaptation to toxic stress, deletion of gene Cldn2, consequence increased gene expression in PH mice, deletion of gene Cldn3, deletion of gene Cldn8, deletion of gene Ocln, development of RiboDiPA tool for analyzing ribosome occupancy patterns, RiboDiPA identifies genes with significant differences in ribosome occupancy patterns, deletion of gene mpkB, reduction in conidia production, vesicular dome-immaturity in the conidiophore head, decreased phialide numbers, very short stalks, inability to produce sclerotia, normal production of aflatoxin B1, deletion of gene pab1, decrease in translation termination efficiency, proximity of nonsense codon to mRNA 3' end, increase in premature termination products, decrease in readthrough efficiencies, lengthening of 3' UTR, increase in readthrough efficiencies for PTC, inefficient termination, trigger for nonsense-mediated mRNA decay, total higher alcohol concentration decreased by 27.31%, deletion of gene CAN1, total content of higher alcohols increased by 211.44%, deletion of gene GAT1, total content of higher alcohols increased by 28.36%, deletion of gene SOR1, increased xylitol yield, deletion of gene SOR2, deletion of gene XKS1, weakened SOR1 expression, weakened SOR2 expression, reduced xylose to xylitol, ectopic expression of CSyGTs, complement soyasaponin biosynthesis, CSyGT mutants, do not accumulate soyasaponin, expression of CSyGTs, produce glycyrrhizin de novo, mutated XR, higher transcript levels of XYL1, higher transcript levels of XYL2, higher transcript levels of XKS1, improved xylose consumption rate, improved ethanol yield, upregulated genes related to protein biosynthesis in diploid strains, downregulated genes involved in glyoxylate shunt in diploid strains, improved biomass yield in diploid strains, haploid vs diploid, downregulated genes involved in glycolysis in diploid strains, downregulated genes involved in MAPK signaling pathway in diploid strains, upregulated oxidative stress related TFs in diploid strains, deletion of tyrosine feedback inhibition, increased naringenin production, downregulation of competitive pathway, multi-copy integration of flavonoid pathway genes, increased copy numbers of (2S)-naringenin metabolic pathway genes, increased 4CL transcription, increased CHS transcription, enhanced naringenin production, expression of nuclear receptors, regulation of cytokine genes, overexpression of TFs, modulation of cytokine production, deletion of specific TFs, alteration in cytokine gene promoters interaction, up-regulation of Sho1, activation of MAPK pathway, filamentous growth, up-regulation of fatty acid synthesis genes, lipid droplets accumulation, up-regulation of fatty acid degradation genes, lipid mobilization, overexpression of gene Cwh41p, increase in protein production, upregulation of glycogen metabolism, increase in α-amylase productivity, upregulation of trehalose metabolism, improved folding precision, reduced energy expenditure, regulation of N-glycans, deletion of gene MTQ2, severe growth impairment, impaired nucleolar release of pre-60S, reduced production of mature 5.8S rRNA, reduced production of mature 25S rRNA, overexpression of gene SSA1, increased vector yields, overexpression of gene SSE1, overexpression of gene SSE2, overexpression of gene GTT1, overexpression of gene RVB2, overexpression of gene HSP40, overexpression of gene HSC70, overexpression of heat shock proteins, activation of unfolded protein response, overexpression of stress protein markers, overexpression of proteins involved in oxidative stress response, reduction in reactive oxygen species effects, deletion of gene Nrc, discussion on method for identifying translation initiation sites, highlights challenges in studying non-canonical protein isoforms, supports expanded use of empirical protein-coding region identification, deletion of gene HIM1, consequence reduces efficiency of induction of RNR genes after UV irradiation, deletion of gene RAD30, consequence suppresses him1-dependent UV mutagenesis, overexpression of gene Polη, consequence increases him1-dependent mutagenesis at NBP sites, consequence suppresses him1-dependent mutagenesis, expression of gene SlCHS, nitrogen-carbon bond formation, production of hydroxycinnamic acid amide, expression of gene SlMT2, methylation of 3-hydroxyanthranilic acid, condensation of 3-HAA methyl ester and p-coumaroyl-CoA, formation of amide bond, deletion of gene Matα:57-70, consequence highest G-CSF titre of 39.4 ± 1.4 mg/L, deletion of gene Matα:30-43, consequence completely diminished G-CSF secretion, deletion of gene Matα:47-49, consequence dampened the secretion of WT-GCSF, addition of deletion Matα:57-70 to Matα:30-43, consequence partially restored G-CSF secretion by 60%, addition of deletion Matα:57-70 to Matα:47-49, deletion of gene Matα:57-70 and Matα:47-49, consequence stimulatory effect on G-CSF secretion with CO-GCSF, activation of target gene, optogenetic control of protein subcellular localization, reconstitution of protein activity, optogenetic control of protein sequestration, protein oligomerization, FUN-LOV system, precise activation of target gene, deletion of HMGR1, increase in total carotenoids to 32 mg/gDCW, expression of carotenogenic genes CrtE, CrtYB, CrtI, increase in total carotenoids to 16 mg/gDCW, expression of PhCCD1, production of β-ionone up to 33 mg/L, deletion of gene Apn1, deletion of gene Apn2, deletion of gene Rad27, deletion of gene Ntg1, consequence DNA replication stress, deletion of gene Ntg2, co-expression of gene Tfu_0875, AA production, co-expression of gene Tfu_2399, co-expression of gene Tfu_0067, co-expression of gene Tfu_1647, co-expression of gene Tfu_2576, heterologous expression of HaWS, 57 kDa protein confirmed by immunoblotting, accumulation of C-32 to C-36 WE, synthesis of specific WE of desired lengths, ectopic expression of gene OsDGAT1-1, restoration of TAG synthesis, restoration of lipid body formation, ectopic expression of gene OsDGAT1-2, deletion/mutation in N-terminal region of OsDGAT1-1, reduced lipid accumulation, alteration in N-terminal region of OsDGAT1-1, reduced functionality, expression of gene OsDGAT1-1, accumulation of triacylglycerol, expression of gene OsDGAT1-2, no expression/deletion of gene specified, complexity of optimal control problems, prediction of temporal profiles of enzyme activation, prediction of metabolite concentrations, heterologous expression of FvPPT, production of core scaffold compound of bostrycoidin, heterologous genes from photosynthetic organisms, enhancement of precursors supply, proper truncation of the N-terminal transit peptide, improvement in tocotrienols production, cold-shock-triggered temperature control system, efficient control of two-stage fermentation, engineered yeast, production of 320 mg/L tocotrienols, overexpression of gene ScHac1p, improved GOX secretion capacity, increased overall enzyme production, overexpression of gene PpHac1p, overexpression of gene TrHac1p, overexpression of gene HsXbp1, loss of GOX gene, disruption of gene MTI1, consequence haploid and stably heterothallic, consequence inability to self-diploidize, mutation in gene MTI1, consequence formation of truncated product, consequence heterothallism from original homothallism, expression of norcoclaurine synthase, toxicity in Saccharomyces cerevisiae, cytosolic expression of norcoclaurine synthase, restriction of (S, targeting of norcoclaurine synthase to peroxisome, alleviation of toxicity, promotion of increased (S, expression of engineered transcription factors, mimicking oleate response, creation of larger peroxisomes, larger peroxisomes, increase in benzylisoquinoline alkaloid titer, upregulation of mevalonate biosynthetic gene, enhancement of astragaloside production, addition of yeast extract, stimulation of astragaloside content, pretreatment with yeast extract, increase in astragaloside levels, deletion of gene DCL4, no 22nt TBSV-derived small RNA detected, deletion of key RNAi factors, TBSV-BS3Ng undergoes non-canonical RNAi defensive reaction, deletion of I-SceI site, cells survive cleavage, integration of antibiotic marker-free cassette, no need for plasmid curing, expression of λ-red proteins, facilitates genome integration, cleavage by I-SceI, kills cells with intact I-SceI site, up-regulation of genes involved in ROS production, promotion of ROS accumulation, down-regulation of catalase, deletion of templating base, conformation change of DNA substrate, incorporation of nucleotide, conversion of pyrophosphate to monophosphates, breaking of hydrogen bonds between nucleotide and arginine side chain, prevention of potentially mutagenic processive synthesis, facilitation of dissociation of DNA product from enzyme, consequence not applicable, consequence increase in specific TAL titer, consequence decrease in expression with increasing temperature, expression of gene SSA3, consequence increase in expression above 40 °C, expression of gene ADH1, deletion of gene YALI0D20196g, abolished citrate production, expression of gene YALI0D20196p, enhanced citrate production 5.2-fold, facilitated citrate export in S. cerevisiae, complemented ability to secrete citrate in A. niger, deletion of gene AOX, consequence strong inhibition of growth, consequence respiratory capacity decrease, expression of gene AOX, consequence no effect on growth, consequence minor effect on growth, deletion of gene PsSOD2, reduced Pst virulence, restricted hyphal development, expression of gene PsSOD2, suppressed flg22-elicited ROS production, deletion of gene Put6, consequence 1: perturbs mitochondrial proline homeostasis, consequence 2: defects in proline utilization, deletion of gene Put7, cohesin binding, formation of loops of up to 50 kb, inter-sister connections every ~35 kb, increased D-LA production, expression of gene HXK2, expression of gene PGI1, expression of gene PFK1, expression of gene PFK2, expression of gene FBA1, expression of gene PGK1, expression of gene GPM1, expression of gene ENO2, expression of gene PYK2, expression of gene D-LDH, consequence 1: redirection of gene expression towards genes involved in central carbon metabolism, consequence 2: redirection of gene expression towards genes involved in lipid metabolism, consequence 3: redirection of gene expression towards genes involved in remodeling of the cell wall, consequence 4: redirection of gene expression towards genes involved in autophagy, induction of genes involved in lactate generation, consequence 5: redirection of metabolic flux towards lactate, overexpression of GAL1, enhanced galactose utilization rate, overexpression of GAL7, overexpression of GAL10, overexpression of PGM2, twofold increase in galactose utilization rate, 1.4-fold increase in ethanol production, increased glucose-6-phosphate levels, increased fructose-6-phosphate levels, enhanced glycolytic flux, deletion of adhesion ability, deletion of auto-aggregation, deletion of exopolysaccharide production, alteration of biofilm-related gene expression, deletion of TBP, repression of interaction with spurious DNA, overexpression of SAGA, stimulation of transcription, presence of SAGA, efficient TBP delivery onto gene promoters, overrepresentation of GCN codons, depress translation, disrupting conformance to (GCN)n, neutral or positive effects on translation, interaction surface CAR, modulates mRNA-ribosome interaction, modulation of mRNA-ribosome interaction, regulates protein translation, expression of crt1 gene, high-affinity cellobiose transporter function, lactose transport capability, expression of RUT-C30 variant of crt1 gene, full restoration of cellulase induction, expression of cellobiose transporters from other fungal species, expression of QM6a variant of crt1 gene, partial complementation of cellulase induction, deletion of gene TPS2, decline in stress response capacity, decline in translational rate, gene overexpression, reduced gene-space, improved sample classification accuracy, gene deletion, deletion of gene PIP1, consequence interaction with PCNA and Rad6-Rad18减弱, deletion of gene PIP2, consequence maintaining architecture and dynamics of multi-protein complex减弱, deletion of gene SMO4, consequence nucleolar hypertrophy, consequence nucleolar disorganization, consequence overaccumulation of 5.8S rRNA precursors, consequence overaccumulation of 18S rRNA precursors, consequence imbalanced 40S:60S ribosome subunit ratio, deletion of nuclear genes, respiratory growth impairment, deletion of twelve genes, viability loss in absence of mtDNA, overexpression of ARG8^m, selection for maintenance of mtDNA, deletion of up to 176 nuclear genes, impairment in expression of mitochondria-encoded proteins, deletion of gene CIT2, no effect on β-amyrin production, deletion of gene MLS1, increase in β-amyrin production by 1.85 times, weakened expression of gene PGI1, increase in β-amyrin yield by 3.75 times, deletion of gene product, transcriptional repression by Tet-OFF promoter, proteolytic elimination of target protein by AID system, expression of dCas9, alteration of gene expression up to ~2.5-fold, expression of dCas9-VP64 fusion, expression of dCas9-VPR fusion, expression of dCas9-Mxi1 fusion, cellular damage accumulation, decline in protein quality control, maintenance of cellular homeostasis in germline, elimination of age-induced damage in budding yeast, expression of TLR2b, upregulated by Y, upregulated by YCW, upregulated by YCW + E, upregulated by PY/N, expression of TLR4, downregulated by E, downregulated by YCW, downregulated by YCW + E, upregulated by DDGS, expression of IL-4, expression of IL-10, expression of IFN-γ, expression of IL-12, overexpression of gene PMA1, decrease in CPC production, shift in beta-lactam intermediates content, decrease in cef genes expression, decrease in ATP level, deletion of IL-1β expression, reduced cytokine response, deletion of TNF-α expression, overexpression of IL-1β, enhanced cytokine response, overexpression of TNF-α, deletion of anticodon loop modifications, synthetic growth defects, translational inefficiency of certain mRNAs, disturbed protein homeostasis, accumulation of protein aggregates, transcriptional changes resembling cellular response to nutrient starvation, consequence elevated reactive oxygen species (ROS, activation of gene not specified, consequence ROS defense mechanisms, consequence: regulation of mitochondrial transcription through dynamic conformational changes, consequence: stalled initiation complexes remain in dynamic scrunching and unscrunching states, consequence: abrupt transition to an unbent conformation at position +8, heterologous expression of gene PaFAD2, conversion of oleic acid to linoleic acid, conversion of palmitoleic acid to 9,12-hexadecadienoic acid, activation of gene mCherry, repression of gene mVenus, overexpression of gene CRISPR-ARE, method to measure pairing of DNA loci during homologous recombination, visualization of DNA at break site and homologous DNA repair template, amplification of gene(s) of interest, high gene-expression levels, accurate gene dosage, stable propagation in selection-free media, overexpression of gene PdGLX1, enhanced growth in E. coli, enhanced MG detoxification ability in E. coli, complemented loss-of-function MG hypersensitive GLO1 in yeast, promoted growth in yeast, reduced accumulation of ROS in yeast, consequence: study of eukaryotic termination mechanisms, consequence: identification of potential therapeutics that modulate termination, cloning and expression of EGT biosynthetic genes from Grifola frondosa, increased EGT production in Saccharomyces cerevisiae EC1118, daily addition of 1% glycerol to culture medium, increased EGT production to 20.61 mg/L, deletion of metabolic flux imbalances, improvement in industry-scale production of high-value chemicals, expression of multiple genes using COMPASS, balanced expression in biochemical pathways, expression of pathway genes controlled by inducible artificial transcription factors, enhanced gene expression balance, integration of genes using CRISPR/Cas9, one-step multilocus integration, development of AssemblX toolkit for designing and building large DNA constructs, allows assembly of up to 25 functional units, uses overlap-based, scar-free, and sequence-independent cloning methods, relies on rare cutting homing endonucleases and computationally optimized overlap sequences, consequence n-butyl acetate production, consequence direct production from glucose, consequence elimination of lipase addition, deletion of gene Cpl1, increased adhesive and invasive growth, accumulation of intracellular labile zinc ion, increased glucose uptake, higher level of mitochondrial membrane potential, increased ATP production, increased ROS production, activation of Snf1, increased invasive growth, Deletion of gene FgLsb6, consequence 1: defects in vegetative growth, consequence 2: decreased deoxynivalenol (DON, consequence 3: reduced pathogenicity, consequence 4: increased resistance to osmotic stress, consequence 5: increased resistance to oxidative stress, consequence 6: increased resistance to cell wall stress, consequence 7: defective generation of PI4P on endosomes, consequence 8: defective endocytosis, expression of gene GH from Caulobacter crescentus, increased cellobiose conversion to glucose, expression of gene GH, enhanced growth on cellobiose, upregulation of secretion-related proteins, aided release of GHs, upregulation of transport-related proteins, facilitated cellobiose entry into the periplasm, upregulation of outer membrane-related proteins, promoted surface display of GHs, improved transcript levels, increased protein concentrations, higher amylase activities, expression of α-amylase gene and glucoamylase gene, improved fermentation vigour, 97% of theoretical ethanol yield, 100% conversion of available carbon to products, deletion of A503 from Bp BteA, enhanced cytotoxic activity on mammalian HeLa cells, increased toxicity to Saccharomyces cerevisiae cells, insertion of A503 into Bb BteA, decreased cytotoxicity to yeast, decreased cytotoxicity to HeLa cells, expression of Bp BteAΔA503, increased virulence of B. pertussis B1917 in mouse model, reduced LD50 in mouse model, less inflammatory pathology in infected mouse lungs at sublethal infectious doses, deletion of gene PGI1, accumulation of intracellular NADPH, co-utilization of glucose and xylose more effectively, release of glucose repression on xylose utilization, decreased ethanol productivity, activity of PPP, regulation of NADPH/NADP+ ratio, NADPH/NADP+ ratio, regulation of expression of NDE1, regulation of expression of NDE2, regulation of expression of GLR1, regulation of expression of GDP1, deletion of gene expression in Streptococcus thermophilus, consequence: differential gene expression in amino acid metabolism, consequence: differential gene expression in citrate metabolism, consequence: differential gene expression in urease metabolism, consequence: differential gene expression in purine metabolism, consequence: differential gene expression in pyrimidine metabolism, overexpression of peptides in growth media, consequence: modulation of quorum sensing system expression, deletion of gene FLO400, consequence prevents pseudohyphae formation, deletion of gene FLO5-1, consequence hampers FLO11 expression, expression of gene FLO400, consequence leads to induction of FLO11, expression of gene FLO5-1, chromatin modulation, consequence regulates expression of FLO400 and FLO5-1, mitochondrial volume fraction increase, increase in mRNA localization to mitochondria, mitochondrial mRNA localization increase, enhance protein synthesis by increasing mRNA localization to mitochondria, failure in gene editing with previous methods, expression of sgRNA, high efficiency CRISPR/Cas9 gene editing, successful gene editing with new tRNA-sgRNA fusions, integration of gene xxx, successful with new tRNA-sgRNA fusions, deletion of miR365, improvement of PTOA symptoms, engulfment of NPs-YCWP by macrophages, protection of miR365 antagomir/AAT by YCWP, prevention of degradation in gastrointestinal tract, deletion of gene ZIP3, tolerance to toxic concentrations of manganese, imbalanced expression of intracellular metal transporters, higher sensitivity to reactive oxygen species, alteration in the expression of ROS-detoxifying enzyme-coding genes, decreased melanization, decreased secretion of glucuronoxylomannan, altered extracellular vesicle dimensions profile, deletion of gene atsf1, consequence leaf variegation, consequence reduced accumulation of mature plastid ribosomal RNA, mutation in gene atsf1, consequence hypersensitivity to coldness, consequence low splicing activity for introns with suboptimal BPS sequences, consequence upregulation of many genes, consequence downregulation of many genes, consequence insufficient intron splicing, increased fitness burden, decreased fitness burden, increased vanillin-β-glucoside production, increased total vanillin-β-glucoside pathway metabolite accumulation, overexpression of gene Calb, increased secretory production of Candida antarctica lipase B, use of DHAS TT terminator, inducible CALB expression levels similar to AOX1 terminator, threefold higher constitutive expression of CALB under GAP promoter, enhanced transcript stability by correct choice of transcription terminators, increased transcription rate, overexpression of gene FHL1, consequence RNA content increased by 14.94%, overexpression of gene SSF2, consequence RNA content increased by 24.4%, deletion of gene HRP1, consequence RNA content increased by 19.36%, overexpression of gene IFH1, consequence no significant change in RNA production, overexpression of gene SSF1, consequence part of complex key to RP gene transcription, consequence important role in ribosome biosynthesis, consequence removes negative regulator of cell growth, overexpression of gene IFH1, overexpression of gene SSF2, deletion of gene HRP1, consequence RNA content increased by 38.8%, overexpression of gene IFH1, overexpression of gene SSF2, deletion of gene HRP1, consequence increased cell size, overexpression of gene IFH1, overexpression of gene SSF2, deletion of gene HRP1, consequence increased growth rate, deletion of canonical start codon usage, production of alternative N-terminally extended protein isoforms, overexpression of non-AUG start codon usage, increased during meiosis, decrease in eIF5A, induction of non-AUG initiation, deletion of gene PIGR, no difference in expression, deletion of gene CD79A, overexpression of gene PIGR, higher secretory IgA concentrations in ileum, overexpression of gene CD79A, higher secretory IgA concentrations in colon, greater number of IgA-producing plasma cells in ileum, greater number of IgA-producing plasma cells in colon, expression of sidD gene, increase under iron limitation and presence of 600-800 μM heavy metal, expression of gene btrC, production of 2-DOI, reduction of 2-DOI into scyllo-quercitol, reduction of 2-DOI into (-, deletion of gene pgi1, inactive Pgi1, increased production of carbocyclic compounds, deletion of gene zwf1, trace production of compounds, assimilation of 2-DOI by S. cerevisiae, expression of gene SRS pathway, consequence H2S production, expression of gene Sulfur metabolism, expression of gene Glycine, serine and threonine metabolism, consequence H2S production, expression of gene Biosynthesis of amino acids, overexpression of gene sulfate reduction sequence (SRS) pathway, consequence increased H2S production, deletion of gene sulfate reduction sequence (SRS) pathway, consequence decreased H2S production, overexpression of gene HAC1, improvement in xylanase production, integration of gene xynB, constitutive secretory expression of xylanase, increase in yield of xylanase to 381 U/mL, deletion of gene RIM15, enhanced biofilm formation, improved fermentation ability, increased expression of adhesion protein FLO11, synthesis of trehalose, deletion of gene GT1, impaired zinc uptake activity, deletion of gene ZRT1, zinc deficiency, upregulation of gene ZRT1, block of Gt1 function, deletion of transaminase gene, consequence: potential loss of transamination activity, overexpression of transgene, consequence: increased biomass up to 60% wet cell weight by volume, overexpression of a key protein, increase in alcoholic fermentation rate, modification of a specific domain or amino acid, increase in ethanol tolerance, heterologous expression of NDH, increased cellular sensitivity to furfural, overexpression of GLR1, enhanced furfural tolerance, overexpression of OYE2, overexpression of IDP1, expression of PNTB, little impact on furfural tolerance, expression of POS5, expression of Rad51-NTD, high-level production of native Rad51, increased half-life of Rad51, phosphorylation by Mec1/Tel1, antagonizes proteasomal degradation pathway, upregulation of acid-responsive genes involved in ergosterol synthesis, increased acid tolerance, upregulation of acid-responsive genes involved in ferrous iron uptake, upregulation of genes for fatty acid ethyl ester synthesis, enhanced aroma compound formation, upregulation of genes for aromatic amino acid biosynthesis, deletion of gene ARO10, reduction in benzyl alcohol production, deletion of gene RPH1, consequence alleviates cell slow growth, consequence derepression of rRNA transcription, consequence derepression of RPG transcription, phosphorylation of gene RPH1, consequence more resistance to rapamycin treatment, consequence decreased association with ribosome-related genes, consequence faster cell growth, deletion of DH1 domain, complete loss of activity in the biosynthesis of VLCPUFAs, mutation of DH1 domain, deletion of DH2 domain, production of a small amount of DPA, mutation of DH2 domain, substitution of DH1 with DH2, expression of mutated DH2, expression of DH1, essential for the biosynthesis of VLCPUFAs, expression of DH2, required for the biosynthesis of DHA, overexpression of genes from the NA strain, higher growth efficiency under nitrogen-limited conditions, overexpression of ECM38 allele, expression of gene PvFAD2, consequence increase in LA production, expression of gene PvFAD3, consequence increase in ALA production, deletion of SSU1, confirmation of role in sulphite tolerance, deletion of gene ChrR1, increase in carotenoid production, overexpression of gene crtE1, no production of geranyl pyrophosphate, overexpression of gene crtE2, expression of gene GPPS, production of geraniol, expression of gene GES, production of amorphadiene, deletion of gene segments, alteration of chemical product formation, overexpression of mutated genes, identification of important determinants for successful mutagenesis, no specific consequences related to gene overexpression/deletion mentioned, overexpression of gene SUT2, promotes expression of AOX1, increases ergosterol content, expression of gene SUT2, induced by methanol, repressed by glycerol, P_SUT2-driven expression of gene MXR1, elevates expression level of Mxr1, high activity of P_AOX1, P_AOX1-driven expression of gene enhanced green fluorescent protein, improved by 18%, genetic encoding of biosensors, dynamic response, sensitivity, operational range, regulatory architecture modification, dynamic assembly, protein interaction alteration, regulatory function, genetic element modification, interaction efficiency, deletion of gene TRF5, increases RNA abundance for mRNAs specifically showing high Trf5 binding, deletion of gene CgHHF1, MMS resistance, deletion of gene CgHHF2, higher rate of homologous recombination, reduction in histone H4 protein levels, stress resistance, fusion of FAS subunits, increased FAS activity, improved fatty acid production, consequence competitive superiority, deletion of gene psr1psr2, consequence production of diffusible compound, consequence resistance to diffusible compound, focus on expression systems for membrane proteins, deletion of gene YJL175W, reduction in transcription of genes encoding highly expressed, secreted proteins, overall reduction in translation, up-regulation of genes required for ergosterol biosynthesis, increased levels of ergosteryl esters, aeration with microbubbles, increased O2 transfer to the propagation cultures, expression of gene HXK2S14A, increased ethanol production rate, decreased xylose utilization, eliminated positive effect of Hxk2p, expression of MET5, decrease in CFU values, expression of MET10, overexpression of sulfite reductase, decrease in selenium concentration, negative effect on cell population, increase in sulfite concentration, positive effect on cell population, production of selenium sulfide nanoparticles, drop in S. cerevisiae cell population, deletion of gene Bud27, alteration of phosphorylation kinetics of ribosomal protein S6, impact on homeostasis of ribosome biogenesis process, modulation of transcription mediated by RNA polymerase I, influence on transcription of RNA polymerase III, regulation of RNA polymerase II-dependent transcription of ribosomal proteins, regulation of ribosome biogenesis regulon genes, deletion of gene XRN1, impaired gene silencing, affecting the composition of siRNA species, influencing the efficiency of siRNA loading into Argonaute, degradation of cleaved passenger strand, degradation of sliced target RNA, osw2 Δ, enhanced catalytic activity, excellent reusability, thermostability, tolerance to various harsh conditions, deletion of uORF3, consequence 1: up-regulation of HNT1 translation, overexpression of uORF3, consequence 1: down-regulation of HNT1 translation, expression of SbDGAT1-1, restored TAG accumulation capability, expression of SbDGAT1-2, deletion of N-terminal region in SbDGAT1-1, decreased TAG accumulation, decreased FA accumulation, expression of gene TDH1, increased promoter strength, expression of gene INO1, expression of gene SED1, expression of gene CCW12, expression of gene TEF1, weaker promoter strength compared to 15 other promoters, optimized biosynthesis pathway of (2S, consequence central metabolites drastically deplete, consequence intracellular AMP-to-ATP ratio strongly increases, activation of bulk autophagy, consequence frees amino acids, activation of β-oxidation, consequence lipid degradation, consequence ensures long-term survival during starvation, deletion of gene isc10, reduced efficiency of spore differentiation, overexpression of gene ssp2 before MII, deletion of gene ama1, Isc10 not degraded, activation of gene smk1 by ssp2, triggered by degradation of Isc10, expression of gene CYP505D13, enhanced ST-3 production, increased ST-3 titer to 78.61 mg/L, improved bioproduction efficiency 6.43 times, deletion of gene Aim23p, quantitative imbalance of mitochondrial translation products, increased amount of Pet111p, interaction of Aim23p with Pet111p, ensures proper COX2 mRNA translation, deletion of gene Vta1, consequence colorless microsclerotia, consequence no melanization, activation of gene Vta1, consequence melanin production, consequence transcription of PKS1, consequence transcription of LAC1, deletion of DNA polymerase alpha/primase, disruption of Okazaki fragment initiation, disruption of replication origin firing, deletion of Ctf4, impairment of lagging-strand initiation, severe depletion of DNA polymerase alpha/primase, activation of the checkpoint essential for viability, deletion of gene products, consequence old daughters grow more slowly, consequence old daughters age, consequence old daughters have lower fluorescence levels, expression of envelope domain III of dengue virus serotype-1, increased secretory expression, improved screening using immunoassay method, deletion of Ded1p function, represses translation of housekeeping mRNAs, promotes translation of stress mRNAs, expression of piggyBac transposase, long-term gene expression, mutation of piggyBac transposase, higher transpose activity, expression of bz-hyPBase, high-efficiency editing in CHO cells, high-efficiency editing in T cells, deletion of gene SET1, increase in serine-2 phosphorylation within the C-terminal domain of RNA polymerase II, increase in recruitment of the cleavage/polyadenylation complex, deletion of gene SET2, chemical inhibition of TOR signaling, Set1-dependent alternative polyadenylation, Set2-dependent alternative polyadenylation, expression of gene SET1, decrease in efficiency of using single pA sites, expression of gene SET2, control nucleosome occupancy around pA sites, expression of gene involved in oxidative stress, consequence: enhanced oxidative stress response in blastospores, expression of gene involved in amino acid metabolism, consequence: increased amino acid metabolism in blastospores, expression of gene involved in respiration processes, consequence: enhanced respiration in blastospores, expression of gene involved in transmembrane transport, consequence: improved transmembrane transport in blastospores, expression of gene involved in production of secondary metabolites, consequence: increased production of secondary metabolites in blastospores, expression of gene involved in growth, consequence: enhanced growth in hyphae, expression of gene involved in metabolism, consequence: increased metabolism in hyphae, expression of gene involved in cell wall re-organization, consequence: altered cell wall structure in hyphae, deletion of Fld1, LD biogenesis impaired, deletion of Nem1, overexpression of Lro1, increased LD formation, overexpression of Dga1, reduced recruitment of TAG-synthases, presence of Fld1, increased DAG production at ER subdomains, presence of Nem1, gene expression of stress tolerance genes, gene expression of growth genes, gene expression of metabolic genes, expression of gene Cys3, upregulation, expression of gene Cys4, expression of gene MET17, expression of gene IRC7, increased 4-MMP release, deletion of gene AtSFC1, reduced seed germination, impaired radicle growth, reduced respiration rate, expression of gene SNZ2, increased intracellular vitamin B6 and thiamine, expression of gene THI6, expression of gene THI21, expression of gene THI80, proanthocyanidins treatment, increased cell viability, increased unsaturated fatty acids content, decreased saturated fatty acids content, upregulation of 17 ribosomal proteins, activation of reserve translational capacity, heterologous expression of sugar transporters, utilization of cellodextrins, intracellular expressed β-glucosidase, expression of cellodextrin transporters from Aspergillus niger, transport of cellobiose with low affinity, expression of Ls120451 from Lipomyces starkeyi, efficient growth on cellobiose, expression of Ls120451, growth on glucose, growth on lactose, no growth on cellotriose, no growth on cellotetraose, expression of CdtG from Penicillium oxalicum, consumption of cellobiose at concentrations below 1.0 g/L more efficiently than Ls120451, consumption of cellobiose at higher concentrations more efficiently than CdtG, production of up to 30% more ethanol than CdtG, deletion of gene glycolysis, consequence repression of glycolytic pathway, overexpression of gene FLO11, consequence enhanced cell adhesion, deletion of gene ion transporters, consequence reduced ion transportation, overexpression of gene oxidative carbon metabolism, consequence increased metabolic activity, overexpression of gene high affinity transport of sugars, consequence enhanced sugar uptake, overexpression of gene glycerol utilization, consequence increased glycerol breakdown, overexpression of gene sulfur metabolism, consequence enhanced sulfur compound processing, overexpression of gene protein quality control and recycling, consequence improved protein maintenance, overexpression of gene cell wall biogenesis, consequence strengthened cell wall, overexpression of gene apoptosis, consequence regulated cell death, conformational fluctuation of host soluble prion protein, regulates efficiency of cross-species prion transmission, specific conformation of short disordered segment, governs interspecies prion transmissibility, critical residues within segment, alter dynamics and conformation of soluble prion proteins, methylene group difference between asparagine and glutamine, changes short segment structure, short segment structure change, modulates cross-seeding activity, overexpression of synapsin-1, improved viability of PC12 cells by up to 71.17%, downregulation of septin-5, improved viability of PC12 cells by up to 67.00%, overexpression of alcohol dehydrogenase-6, led to 22.38- and 12.33-fold decreases in the production of MDA and H₂O₂, respectively, led to 7.15-, 13.93-, 12.08-, and 8.11-fold improvements in the activity of catalase, glutathione reductase, superoxide dismutase, and glutathione peroxidase, respectively, addition of xylose, transcriptional activation of target genes, activation of mcherry, activation of xylose utilization gene, deletion of gene GID4, accumulation of Mdh2, high malate production, deletion of gene GID1, deletion of gene GID2, deletion of gene GID3, deletion of gene GID5, deletion of gene GID8, deletion of gene GID9, co-adaptation index increase, positive correlation with protein abundance, positive correlation with translation rate, high co-AI, positive correlation with mRNA secondary structure stability, positive correlation with mRNA half-life, highly expressed proteins encoded by high co-AI genes, assisted by molecular chaperones to attain proper functional conformation, prevention of protein accumulation by molecular chaperones, expression of gene ADH4, increased response to MMP, overexpression of gene ADH4, increased methionol production in yeast strain S4, induced expression of gene ADH4 in E. coli, preparation of 42 kDa Adh4p, catalysis by Adh4p, conversion of MMP to methionol, overexpression of genes associated with potassium synthesis, increased tolerance to acetic acid, overexpression of genes associated with iron synthesis, overexpression of genes associated with zinc synthesis, overexpression of genes associated with glutathione synthesis, provision of nicotinamide adenine dinucleotide, upregulation of TCA cycle, upregulation of chaperones, enhanced protein stability, upregulation of coenzyme synthesis, improved metabolic function, upregulation of energy production pathways, increased cellular energy, upregulation of energy transformation pathways, enhanced stress response, expression of Braz in S. cerevisiae, higher solubility, expression of Braz in S. cerevisiae, increased safety (GRAS, expression of Braz in E. coli, higher yield of GST-Braz, expression of Braz in E. coli, lower solubility compared to S. cerevisiae, review of NR biosynthesis mechanisms, expression of gene rimA, increase in rimocidin production, expression of gene rimB, expression of gene rimC, expression of gene rimD, expression of gene rimE, expression of gene rimF, expression of gene rimG, expression of gene rimH, expression of gene rimI, expression of gene rimJ, expression of gene rimK, over-expression of gene DUR3, increased utilization of urea, reduced urea concentration by 92.0%, reduced EC concentration by 58.5%, synchronization of protein synthesis with cell division, synchronization of glycolysis with cell division, synchronization of amino acid metabolism with cell division, increase in mitochondrial volume, overexpression of xylose reductase gene from Neurospora crassa controlled by PCgcwp1, xylitol production increased by 204%, overexpression of xylose reductase gene from Neurospora crassa controlled by PCgaac, medium-high xylitol production at 42°C, overexpression of xylose reductase gene from Neurospora crassa controlled by PCgpot1, overexpression of xylose reductase gene from Neurospora crassa controlled by PCghsp12, modest xylitol production at 42°C, overexpression of xylose reductase gene from Neurospora crassa controlled by PCgatp1, overexpression of xylose reductase gene from Neurospora crassa controlled by PCgino1, weaker xylitol production at 42°C, overexpression of xylose reductase gene from Neurospora crassa controlled by PCgscl, mutations in gene KARS, affect cytosolic LysRS level, affect cytosolic tRNA aminoacylation in vitro, mutation c.517T>C in gene KARS, deficient in charging mitochondrial tRNA<sup>Lys</sup> in vitro, deficient in charging mitochondrial tRNA<sup>Lys</sup> in vivo, down-regulation of nuclear factor-kappa B, reduction in neutrophil infiltration, reduction in production of proinflammatory cytokines, up-regulation of Zonna occludens-1, increase in intestinal barrier integrity, up-regulation of Zonna occludens-2, up-regulation of Claudin-1, up-regulation of Occludin, up-regulation of Mucin-1, up-regulation of Mucin-2, gene expression regulation by promoters, improved activity of displayed proteins, improved specificity of displayed proteins, improved stability of displayed proteins, secretion signals affecting secretion efficiency, length of anchor proteins affecting accessibility, recombinant expression of CGA-N12 in Pichia pastoris GS115, direct secretion into culture media, enhanced expression effect, expression of CGA-N12, recognition and excision by endogenous protease Kex2, recognition and excision by endogenous protease Ste13, release of short CGA-N12 peptides from tandem repeat sequences, yield of up to 30 mg/L of yeast culture, loss of specificity between ncRNA-mRNA interaction profiles, repression of ncRNA affinities with mRNAs, heterologous gene expression, improving ginsenoside production, deletion of gene RPN4, no increase in autophagy activity upon ethanol stress, deregulation of PRE1 leading to Rpn4 accumulation, increased PRB1 expression upon ethanol stress, stabilization of Rpn4, contribution to PRB1 upregulation, repression of Rpn4 binding sites in PRB1 promoter, inhibition of PRB1 induction and hypersensitivity to ethanol, heterologous expression of Clostridium pathway, increased butanol production, modification of amino acid uptake pathways, expression of COX1, restricted synthesis by nuclear-encoded translational activator, expression of ATP synthase subunits, regulated by F1 module, expression of Cox6, part of high-molecular-weight complexes affecting ratio of COX and ATP synthase, expression of Atp9, expression of ShmA, production of preshimalactone, expression of ShmA and ShmB, production of shimalactones A and B, expression of gene lipoxygenase, consequence (R, -)-1-octen-3-ol biosynthesis), (expression of gene peroxide lyase, consequence (R, deletion of gene Sen1, consequence 1: increased pervasive transcription, expression of gene Sen1, consequence 1: termination of non-coding transcription, interaction between Sen1 and CTD, consequence 1: enhanced release of paused RNAPII from DNA, deletion of gene AaBRE1, loss of H2B monoubiquitination, decrease in H3K4me3 enrichment, suppression of genes encoding C2H2 zinc-finger family transcription factors, suppression of genes encoding cell wall-degrading enzymes, suppression of genes encoding chitin-binding proteins, reduction in hyphal growth, reduction in conidial formation, idle design of gene regulation, minimized crosstalk, excess use of regulation, high crosstalk levels, positive regulation of frequently expressed genes, increased crosstalk, negative regulation of infrequently expressed genes, default unregulated state of genes in idle design, frequently required activity state, expression of CD gene, enhanced cytotoxicity, expression of NTR gene, suppression of tumor growth, prolonged survival, administration of 5-fluorocytosine, administration of CB1954, deletion of gene Aoglo3, growth defects, increase in hyphal septum, decrease in sporulation capacity, change in spore shape, decrease in spore germination rate, transcriptional repression of sporulation-related genes, sensitivity to chemical stressors, involvement in endocytosis, increase in myelin figures, increase in autophagosomes, affect production of mycelial traps, affect production of serine proteases, expression of gene TBRT, consequence enhanced activities of Bap2 and Tat1, consequence improved growth, deletion of gene TBRT, consequence decreased transcription of TAT1, activity of TOR signaling pathway, consequence repressed expression of BAP2 and TAT1, consequence induced expression of TBRT, co-expression of gene SchTPS1, production of (-, co-expression of gene VvSTO2, overexpression of gene TtASR1, increase in E. coli viability under heat stress, increase in E. coli viability under cold stress, increase in Saccharomyces cerevisiae tolerance under salt stress, increase in Saccharomyces cerevisiae tolerance under osmotic stress, upregulation of gene TtASR1 by ABA, response to abiotic stress, upregulation of gene TtASR1 by PEG, upregulation of gene TtASR1 by salt stress in Mahmoudi roots, associated with salt tolerance, downregulation of gene TtASR1 by salt stress in Azizi roots, associated with salt sensitivity, deletion of sirtuins, affected relevant phenotypes in K. lactis, no effect on phenotypes in S. cerevisiae, deletion of gene Maf1, consequence 1: sequesters Pol III elements involved in transcription initiation, consequence 2: binds the mobile C34 winged helix 2 domain, sealing off the active site, consequence 3: Maf1 binding site overlaps with that of TFIIIB in the preinitiation complex, deletion of gene involved in metabolism, increased sensitivity to SDS, deletion of gene involved in DNA processing, deletion of gene involved in cellular transport, deletion of gene involved in transport facilities, deletion of gene involved in transport routes, deletion of gene involved in transcription, deletion of gene involved in protein with binding function or cofactor requirement, up-regulation of antioxidant defense genes, response to oxidative damage, expression of gene DES1, production of oleic acid, expression of gene DES2, production of 16:1<sup>Δ11cis</sup>, production of vaccenic acid, deletion of gene PWP2, increased CCM titer, deletion of gene EST2, deletion of gene ATG1, deletion of gene DIT1, deletion of gene CDC15, deletion of gene CTS2, deletion of gene MNE1, duplication of gene encoding dehydroshikimate dehydratase, overexpression of gene encoding PCA decarboxylase, overexpression of gene encoding AROM protein without shikimate dehydrogenase domain, restoration of URA3 prototrophy, deletion of gene tg, consequence high levels of squalene, consequence high levels of ergosterol, consequence upregulation of MEV pathway genes, consequence elevation in squalene levels, consequence elevation in ergosterol levels, consequence increase in transcript levels of MEV pathway genes, overexpression of gene β-Amyrin synthase, consequence production of β-Amyrin, appropriate folding, secretion to external environment, easy purification, expression of subunit vaccines, affects medical biotechnology market, expression of gene Xltr1p, consequence increased d-xylose uptake, expression of gene N326F^Xltr1p, consequence high efficiency d-xylose transport, consequence reduced glucose utilization, expression of gene F300A^Xltr1p, consequence increased glucose utilization, consequence loss of d-xylose transport activity, deletion of gene NUP170, lower amount of SAGA complex bound to target genes, levels of SAGA complex decreased, levels of SLIK complexes increased, increased nuclear activity of Pep4, production of the SLIK complex, increased nuclear entry of Pep4, increase in the sieving limits of the NPC diffusion channel, overexpression of gene LEU4, increase in isopentanol to isobutanol product ratio, overexpression of gene LEU1, overexpression of gene LEU2, mitochondrial localization of LEU4, reduction in intermediate loss, mitochondrial localization of LEU1, mitochondrial localization of LEU2, boost in isopentanol production, increase in isopentanol production, isopentanol accounts for 86% of total BCHAs produced, highest isopentanol titer reported for Saccharomyces cerevisiae, expression of gene TMEM165, abrogates Ca2+-induced growth defects, abrogates Mn2+-induced growth defects, prevents excessive Mn2+ accumulation, corrects glycosylation defects, mediates Ca2+ influx, mediates Mn2+ influx, mutation E108G in gene TMEM165, reduces TMEM165 activity, HAC1 overexpression, enhanced transcriptional activity of GAL1 promoter, increased nerolidol titer, increased cell vitality, metabolic regulation of monocarboxylic acid metabolic processes, metabolic regulation of cellular amino acid biosynthetic process, deletion of gene ERG20, lethal phenotype, successful construction with high efficiency, improvement in mutation efficiency, simplification of genetic manipulation procedure, regulation of mutant gene expression, improvement in genome integration efficiency, optimization of integration modules, introduction of ScADH1, increase in ethanol production from glucose by 57%, overexpression of GLT-1, increase in ethanol production from glucose by 131%, overexpression of CDT-1/CDT-2, increase in ethanol production from cellobiose by 200%, turning down expression of pyc gene, increase in ethanol production by 23%, gene knockout data, observe gene expression levels at steady-state, determine non-existent regulation, reduce number of regulation required for inference, deletion of gene nif in non-diazotrophs, inability to fix nitrogen, expression of gene nif from Klebsiella oxytoca in Escherichia coli, enhanced nitrogen fixation, expression of gene nif from Azotobacter vinelandii in Saccharomyces cerevisiae, expression of gene nif from Paenibacillus polymyxa in tobacco plant, deletion of gene Morgs, consequence sRNA production alteration, overexpression of gene milR236, consequence delayed appressorium formation, consequence virulence attenuation, deletion of gene Mohat1, binding of transcription factor MoMsn2 to promoter of milR236, consequence suppression of MoHAT1 transcription, suppression of MoHAT1 transcription, consequence reduced appressorium formation, consequence reduced virulence, expression of CrtZ from Pantoea ananatis, increased violaxanthin yield, expression of truncated ZEP from Haematococcus lacustris, co-expression of truncated ferredoxin-3 and truncated root ferredoxin oxidoreductase-1, 2.2-fold increase in violaxanthin yield, increasing gene copy number of carotenogenic genes, reached 7.3 mg/g DCW violaxanthin yield, overexpression of gene Candida tropicalis XR, increased xylose consumption, overexpression of gene S. cerevisiae Gre3p aldose reductase, overexpression of gene NADH-preferring C. tropicalis XR mutant, decreased xylose consumption, lower xylitol yield, replacement of ZWF1 gene promoter with GPD promoter, deletion of gene XYL2, no increase in xylitol yield, expression of gene ORF131, consequence activation of immune-related genes CXCa, consequence activation of immune-related genes CXCR1, consequence activation of immune-related genes IL-1β, consequence activation of immune-related genes TNF-α, consequence activation of immune-related genes INF-a1, consequence activation of immune-related genes Mx-1, consequence activation of immune-related genes IgM, consequence activation of immune-related genes IgT1, consequence production of specific serum IgM, consequence relative percent of survival ranging from 53.33% to 66.67%, deletion of gene YHB1, consequence down-regulation, deletion of gene PRB1, consequence clipping of Histone H3 N-terminal tails, deletion of SAGA, reduced TBP delivery to gene promoters, impaired transcription factor IIA efficiency, decreased gene expression regulation, co-translational interaction of β-subunit with nascent α-subunit, initiates assembly, exchange of two amino acids in emerging interface, prevents assembly, co-translational interaction of subunits at other sites, formation of dimeric subunit, orchestrates complex formation, dimeric subunit, acts as platform for post-translational phosphopantetheinylation, kinetic proofreading of activator-DNA recognition, higher dissociation energies imply longer DNA dwell times of the activator, lower activator concentrations entail longer pauses between binding events, activated transcription occurs in stochastic bursts, deletion of gene gsy1, consequence sporulation inefficiency, deletion of gene gsy2, deletion of gene tps1, deletion of gene cln3, consequence increased cell size, consequence increased mitochondria stores, consequence increased lipid stores, consequence partial rescue of meiosis progression, consequence partial rescue of spore ascus formation, expression of gene elovl4, consequence elongation of C18-22 polyunsaturated fatty acids, consequence production of C24 products, consequence production of C26 products, consequence elongation of saturated fatty acids, consequence production of 28:0 product, consequence production of 30:0 product, no specific consequences related to gene overexpression/deletion, expression of zeaxanthin epoxidase, increased integration efficiency, minimal experimental effort, deletion of ISC machinery components, impaired Fe/S cluster synthesis, disrupted Fe/S cluster trafficking, inhibited Fe/S cluster insertion into target apoproteins, deletion of Hsp70 chaperone, impaired Fe/S cluster trafficking, disrupted Fe/S cluster insertion into target apoproteins, deletion of gene REV1, no significant increase in meiotic mutagenicity, deletion of gene REV3, deletion of gene POL4, expression of gene CAN1, increased mutation frequency in meiosis compared to mitosis, meiotic recombination, introduction of new mutations at DNA sequence level, deletion of gene PP_2426, decrease in vanillyl alcohol yield, expression of gene FEZ21_09870, conversion of vanillin to vanillyl alcohol, expression of gene PP_2426, expression of gene PP_3839, oxidation of coniferyl alcohol to coniferyl aldehyde, expression of gene RNAPII, consequence RNA polymerase II elongation rates measured in vivo, transient inhibition of gene CDK9, consequence early elongation inhibited, introduction of 4-thiouridine (4SU) labeling, consequence specific capture of new transcripts, use of TT<sub>chem</sub>-seq, consequence high-resolution profiles of transcribed regions achieved, deletion of gene eIF1, consequence increased translation of numerous upstream open reading frames (uORFs, NCCs)), (deletion of gene eIF1, consequence increased translation of numerous upstream open reading frames (uORFs, consequence reduced translation of the downstream main coding sequences (CDS, consequence elevated initiation at certain NCCs initiating amino-terminal extensions of GRS1 product, consequence elevated initiation at certain NCCs initiating amino-terminal extensions of ALA1 product, consequence elevated initiation at main CDS AUG codons with especially poor context, consequence altered ratios of protein isoforms translated from near-cognate versus AUG start codons, deletion of gene warA, sensitivity to sorbic acid, sensitivity to propionic acid, sensitivity to benzoic acid, overexpression of gene An14g03570, resistance to weak acids, deletion of gene pdrA, reduced resistance to sorbic acid, reduced expression of duplicated genes, reduced protein-to-mRNA abundance ratio, reduced expression of dosage sensitive genes, reduced translational efficiency, translationally repressed duplicated genes, inhibited at the level of transcription, increased cobalt sensitivity, increased intracellular cobalt accumulation, decreased expression of oxidative stress scavenging genes, sensitivity to tunicamycin, induction of unfolded protein response (UPR, activation of Ire1, deletion of gene PSK1, unstable EMO, disappearance of fluctuations in EMO, reduced amplitude of EMO, downregulation of glycogen synthesis, downregulation of glycogen breakdown, decrease in glucose concentrations, decrease in NADH levels, deletion of gene atg22, delays acetic acid-induced programmed cell death, maintains cell wall integrity, protects cytomembrane integrity, changes cytomembrane phospholipids, changes sterols, changes fatty acids, increases intracellular amino acids, overexpression of gene atg22, enhances acetic acid-induced programmed cell death, upregulates heat shock protein family expression, upregulates cell wall integrity genes expression, upregulates autophagy genes expression, deletion of gene PbPAT14, consequence dwarf yellowing phenotype, consequence shorter, thinner stems, consequence reduction in cell number, consequence higher level of endogenous abscisic acid (ABA, consequence higher transcript level of ABA pathway genes, deletion of Cbs1, permits cytochrome b synthesis, engagement of Cbp3-Cbp6 in assembly of cytochrome b, represses translation of cytochrome b-encoding mRNA by Cbs1, deletion of gene HbSUT5, decrease in Suc uptake efficiency, decrease in Suc level in latex cytosol, overexpression of gene HbSUT5, increase in Suc sink strength in laticifers, decrease in transcript levels in Hevea latex and bark under ethephon treatment, deletion of gene SRS domain superfamily, consequence reduced parasite interaction with host, overexpression of gene GPI biosynthetic enzymes, consequence increased parasite survival and pathogenicity, no specific gene overexpression/deletion or consequences described, overexpression of gene rsp5<sup>A401E</sup>, consequence growth restoration by RA, inhibition of enzyme calcineurin, consequence suppressed β-hexosaminidase release, inhibition of enzyme GSK-3β, consequence suppressed glucose production, deletion of gene major paralog, consequence increased expression of minor paralog under stress, overexpression of gene major paralog, consequence dominant ribosome production under normal conditions, deletion of a 23.8 kb genomic region, increased p-CA production, upregulation of ENO2, upregulation of TKL1, upregulation of ARO1, upregulation of ARO2, upregulation of ARO4, upregulation of ARO7, upregulation of TYR1, change in AVT6 transcription, change in VBA5 transcription, change in THI72 transcription, change in ARO80 transcription, over-expression of human wtp53, induction of apoptosis, targeting mitochondria, over-expression of mitochondrial HXK-2, accumulation of G6PDH, accumulation of 6PGDH, increased NADPH production, scavenging of ROS, resistance to apoptosis, deletion of hxk2, defective growth, lack of G6PDH accumulation, lack of 6PGDH accumulation, no direct consequences related to gene expression/deletion mentioned, deregulation of phospholipid metabolism, production of oleoylethanolamide, engineering of subcellular trafficking, increase in oleoylethanolamide production, screening of biosynthetic enzymes, enhancement in oleoylethanolamide yield, deletion of gene AT-rich downstream DNA, consequence Pol I pausing, consequence inhibition of Pol I nucleolytic cleavage activity, overexpression of gene GC-rich downstream DNA, consequence decrease in Pol I occupancy, overexpression of gene AT-rich downstream DNA, consequence increase in Pol I occupancy, expression of gene BaFAD2, desaturation of oleic acid into linoleic acid, desaturation of palmitoleic acid into hexadecadienoic acid, expression of gene BaFAD3, conversion of linoleic acid to α-linolenic acid, expression of gene BaD6D-1, conversion of linoleic acid to γ-linolenic acid, conversion of α-linolenic acid to stearidonic acid, deletion of gene erg1, functional complementation by TwSQE6, functional complementation by TwSQE7, expression of gene TwOSC4, product cycloartenol, expression of gene TwOSC6, expression of gene TwOSC8, product β-amyrin, deletion of gene HEM25, lower ethanol production, expression of gene HEM25, expression of gene MdSTP13a, uptake of hexose, uptake of Suc, essential for pollen tube growth on Glc, essential for pollen tube growth on Suc, activation of gene MdSTP13a, by MdMYB39L, repression of gene MdSTP13a, blocks sorbitol-modulated pollen tube growth, deletion of orgA, abolishment of diffusible pigment biosynthesis, deletion of orgB, deletion of orgC, enhanced expression of org, production of thailandene A-C, insertion upstream of σ54-dependent transcription factor, deletion of gene uORFs, consequence increased translational efficiency of main ORF at 37 °C, consequence decreased translational efficiency of main ORF at 20 °C, overexpression of gene uORFs starting with AUG, consequence increased translational efficiency at 37 °C, consequence decreased translational efficiency at 20 °C, overexpression of gene uORFs starting with NCCs, consequence variable translational efficiency based on mRNA-specific features, alteration in translation of specific uORFs by temperature, consequence regulation of gene expression, expression of gene gRNA, consequence rapid and efficient multiplexed refactoring of natural product BGCs in yeast, consequence simultaneous replacement of up to four promoter sites with nearly 100% efficiency, consequence simultaneous replacement of a total of six promoter sites with 68% efficiency, consequence simultaneous replacement of a total of eight promoter sites with 32% efficiency, expression of gene mtg, consequence extracellular activity of 16.1 U/mg, replacement of native signal peptide with SP_sacB, consequence increased extracellular activity, replacement of pro-peptide with S. hygroscopicus pro-peptide, introduction of modified VMA intein, consequence simplified activation process, placement of cleavage site between pro-peptide and core domain, consequence activation process at 18 °C, promoter replacement with PyvyD, consequence extracellular enzymatic activity reached 2.6 U/mg, consequence: resistance to copper, consequence: resistance to SO₂, general discussion on potential for CRISPR/Cas9 system use, stable integration of tabgl1 gene, secretion of TaBgl1, hydrolysis of cellobiose, production of ethanol, high osmotolerance, resistance to various concentrations of ethanol, resistance to furfural, resistance to high temperatures, deletion of gRNA plasmid, increased efficiency of sequential gene targeting, increased production of patchoulol, disruption of gene xxx, replacement of promoter, deletion of gene a1S, effect on mating-type gene expression at MATa locus, effect on mating-type gene expression at MATα locus, overexpression of gene POS5, lower biomass-substrate yields, 2-fold increase in product specific productivity, no significant effect on growth physiology when targeted to mitochondria, no significant effect on recombinant protein production when targeted to mitochondria, overproduction of recombinant protein, increased energy demand, endoplasmic reticulum stress, oxidative stress, deviation from optimal NAD(P, expression of gene COX20, reduced levels of reactive oxygen species, reduction in apoptotic cells, reduction in necrotic cells, overexpression of gene COX20, induced respiratory growth in Δimp1, induced respiratory growth in Δcox18, presence of acetic acid, induces oxidative stress, induces programmed cell death, depletion of gene Pol5, affects processing of pre-rRNAs for large subunit, affects processing of pre-rRNAs for small subunit, impairs recruitment of ribosomal proteins forming polypeptide exit tunnel in LSU, impairs release of 5' ETS fragments from early pre-40S particles, deletion of gene FAF1, deletion of gene SDA1, deletion of gene DAN1, deletion of gene TIR1, overexpression of genes involved in energy conservation and production pathways, caffeine hyperresistance, overexpression of genes involved in pleiotropic drug resistance, single nucleotide polymorphism in PDR1, potential role in caffeine hyperresistance, single nucleotide polymorphism in PDR5, single nucleotide polymorphism in RIM8, PEX34 overexpression, stress to endoplasmic reticulum, stress to mitochondria, enhanced acetate utilization in agc1∆ yeast, redirection of acetyl-CoA from synthetic reactions for amino acids, Yhm2p requirement, PEX34 stimulated growth on acetate medium, increased activity of redox shuttle involving mitochondrial citrate export, increased formation of products from the glyoxylate shunt, enhanced utilization of acetyl-CoA, promotion of alternative mitochondrial redox shuttle activity, deletion of gene erg9, increase in valencene yield, deletion of gene rox1, overexpression of FPP upstream genes in MVA pathway, introduction of valencene synthase, biosynthesis of valencene, use of P<sub>HXT7</sub>-VS-T<sub>TPI1</sub> cassette, improvement in valencene production, deletion of gene HNT1, decreased protein production, overexpression of gene Zap1, shift from SLT HNT1 RNA to LLT HNT1 RNA, increased RNA of HNT1, expression of gene LLT HNT1, dependent on Zap1, decrease in aromatic amino acid gene expression, decrease in aromatic amino acid production, up-regulation of the glyoxylate pathway, increase in p-coumaric acid production, integration of tHMG1, integration of UPC2-1, knockdown of ERG9, reduced squalene synthesis, mutation in ERG20 to mERG20, increased geranylgeranyl diphosphate synthesis, expression of BTS1-ERG20, increased GGOH production, expression of ERG20-mERG20, highest GGOH production, expression of mERG20-ERG20, integration of nor-CPS, produced nor-copalyl diphosphate, integration of ent-CPS, produced ent-copalyl diphosphate, integration of syn-CPS, produced syn-copalyl diphosphate, deletion of gene Cln3, consequence 1: prevents irreversible commitment to the cell division cycle, overexpression of gene Cln3, consequence 1: increases Cln3 concentration leading to passage through Start, consequence 2: observed in both daughter and mother cells across growth conditions, deletion of gene mkt1, consequence defective dikaryotic hyphal production, consequence reduced pheromone gene (MFα1, consequence attenuated virulence, deletion of gene pbp1, deletion of nuclear and mitochondrial gene products, deficits in energy production, defects in mitochondrial functions, deletion of C20-hydroxylation by-products, increase in hydrocortisone productivity, overexpression of ClCDR4 gene, protein engineering of CYP5311B2 to R126D/Y398F variant, deletion of gene VdSho1, reduced penetration, defective melanin biosynthesis, deletion of gene Vst50, defective penetration, defective melanin production, deletion of gene Vst7, deletion of gene Vst11, overexpression of gene Vst50, restored penetration, restored melanin production, overexpression of gene Hub1, enhances splicing efficiency, relaxes constraints on splice-site usage, relaxes constraints on branch-site usage, promotes efficient splicing of introns with non-canonical splice-sites, impairment of metallothionein-mediated defense system, increases importance of Hub1 for cadmium tolerance, discussion on CRISPR technology improvements, exploration of gene editing and transcriptional regulation at genome scale, enabling programmable transcriptional regulation using endonuclease-null Cas enzymes, co-expression of cellulase and xylanase, increased cellulase activity, increased xylanase activity, enhanced ethanol production, improved efficiency in using partly delignified corn stover, low pH, increased maintenance-energy requirements, increased death rates, high CO<sub>2</sub>, affected genome-wide transcriptional responses to low pH, low pH and high CO<sub>2</sub>, interference with low-pH signaling, common MAPK signaling pathways, involved in low-pH and high-CO<sub>2</sub> signaling, cell wall integrity pathway, activated by low-pH and high-CO<sub>2</sub>, high-osmolarity glycerol pathway, calcineurin pathway, deletion of actin crosslinkers, softer Arp2/3-branched actin networks, decreased efficiency of endocytosis, loss of actin crosslinking, limiting the transmission of the force, failure of endocytosis, increased lifetime of endocytic patches, increased number of patches, reduced pool of polymerizable actin, slowing down actin assembly, further impairing endocytosis, deletion of gene uS3/Rps3, consequence 1: reduced stop codon readthrough, consequence 2: increased stop codon readthrough, mutation of residue K108E in uS3/Rps3, mutation of residue R116D in uS3/Rps3, consequence 1: increased stop codon readthrough, combination of R116D mutation with readthrough-reducing mutations in eIF3, consequence 1: suppressed readthrough phenotypes, combination of K108E mutation with readthrough-reducing mutations in eIF3, consequence 1: exacerbated readthrough phenotypes, deletion of gene CYP706A3, increase in sesquiterpene emissions, increase in monoterpene emissions, overexpression of gene CYP706A3, decrease in sesquiterpene emissions, decrease in monoterpene emissions, generation of terpene oxides, combined expression of genes TPS11 and CYP706A3, suppression of volatile emissions, generation of soluble sesquiterpene oxides, expression of gene CYP706A3, avoidance of feeding by Plutella xylostella larvae, modulation of floral microbiome, consequence: development of platform for simultaneous soluble secretion and cell surface display of protein in yeast Saccharomyces cerevisiae using F2A peptide, deletion of gene transcription factors, consequence ethanol yields on biomass change, consequence involvement in cell cycling, consequence involvement in chromatin remodeling, consequence involvement in transcription, consequence involvement in stress response, consequence involvement in protein synthesis, consequence involvement in lipid synthesis, no specific consequence mentioned, deletion of gene Pop2, increased ERCs formation, accumulation of transcripts from E-pro, reduced cohesin amounts, reduced condensin amounts, decreased rRNA amount, similar phenotypes to Pop2 deletion, deletion of FgBMH1, consequence 1: effect on gene expression, consequence 2: effect on DON production, consequence 3: effect on mycelial growth, deletion of FgBMH2, consequence 4: regulation of reproduction, consequence 5: regulation of virulence, consequence 6: regulation of glutamine-induced AreA repression, higher ethanol yield, expression of gene XKS1, expression of gene TAL1, expression of gene PYK1, expression of gene MGT05196, constitutive promoters use, increased copies of six-gene cluster, deletion of gene BGS13, elevated levels of secretion for a variety of reporter proteins, reduced protein kinase C activity, different location in the cell compared to the wt protein, inherent structural problems in the cell wall, increased release of normally secreted, endogenous proteins, increased release of endoplasmic reticulum-resident chaperone proteins, deletion of gene HEM1, consequence 5-aminolevulinic acid auxotrophy, expression of gene α-ketoglutarate dehydrogenase complex subunits, consequence increased αKGDH activity in cytosolic fraction, expression of gene lipoic-acid scavenging enzyme, consequence enabled cytosolic lipoylation of αKGDH complex, expression of gene bacterial ALA synthase, consequence synthesis of 5-aminolevulinic acid from cytosolic succinyl-CoA, deletion of gene involved in chromatin remodeling, increased 3-HP production, overexpression of gene involved in malonyl-CoA precursor supply, increased malonyl-CoA levels, overexpression of gene involved in cofactor supply, enhanced endogenous metabolic fluxes towards acetyl-CoA, enhanced endogenous metabolic fluxes towards malonyl-CoA, deletion of gene IL-8, consequence upregulation in beef steers fed CON diet on day 21, expression of gene TLR1, consequence increased expression with PROB supplementation, expression of gene TLR2, expression of gene TLR6, expression of gene STAT6, expression of gene ICAM1, expression of gene RORC, expression of gene TBX21, expression of gene CXCR3, expression of gene TNF, expression of gene CASP1, expression of metabolite 5-methylcytosine, consequence increased concentration with PROB supplementation, expression of metabolite indoleacrylic acid, expression of gene CaOSC, consequence lupeol biosynthesis, consequence synthesis of lupeol as a single product, consequence increase in lupeol biosynthesis after induction through fermentation process, overexpression of gene QDR3, improved tolerance to glutaric acid, improved tolerance to adipic acid, improved tolerance to pimelic acid, improved tolerance to muconic acid, improved tolerance to glutaconic acid, improved production of muconic acid, deletion of gene LAC9, consequence growth impairment on lactose, consequence growth impairment on galactose, overexpression of gene LAC9, consequence enhanced lactose metabolism, consequence enhanced galactose metabolism, upregulation of genes encoding CDK, observed in G03-△PRS5 strain, influence through pentose phosphate pathway, polygenic recombinant strains G03-△PRS5+△DBP8, polygenic recombinant strains G03-△PRS5+△DBP8+PGM2, polygenic recombinant strains, deletion of gene Tcb1, decrease in intracellular NADPH content, impact on bud formation, synergistic effect with ZWF1 overexpression on glucose, improved xylitol production on glucose but not with enhanced ZWF1 activity, deletion of gene irc10, increase in GC frequencies, deletion of gene irc22, deletion of gene irc23, deletion of gene irc24, deletion of gene irc19, reduction in GC frequencies, deletion of gene irc21, reduction in SSA frequency, reduction in deletion products, reduction in gene conversion products, synergistic interaction with rad52Δ in SSA, efficient β-carotene synthesis from sucrose, achieved β-carotene titers up to 727.8 ± 68.0 mg/L, use of agricultural by-products, yielded β-carotene titer of 354.9 ± 8.2 mg/L, enhanced sensitivity to isobutanol, enhanced sensitivity to acid inhibitors, ATP1-3 expression, COX6 expression, PPP genes expression, ATP1-3 activation, COX6 activation, deletion of gene FAA1, increased free fatty acid concentrations, deletion of gene FAA4, deletion of gene POX1, deletion of gene ADO1, enhanced intracellular S-adenosylmethionine levels, overexpression of gene SAM2, expression of gene DmJHAMT, enhanced FAME production, higher OA titer, dynamic regulation of gene fol3, dynamic regulation of gene abz1, dynamic regulation of gene abz2, dynamic regulation of gene acc1, slower growth in lag and exponential growth stages under 2% and 5% glucose, higher growth in stationary stage under 2% and 5% glucose, decelerated glucose consumption rate during lag growth phase, higher amount of residual glucose in medium, reduced interspecies competitiveness, increased growth of co-cultured E. coli, increased ethanol production efficiency at stationary stage to antagonize E. coli, expression of gene SCJ1, restored laccase secretion, overexpression of six target genes, knockout of seven target genes, knockout of SDH subunit gene, dual-gene deletion of GPD and SDH, significant enhancements in ethanol production, consequence efficient electron transport chain, consequence key growth factor synthesis, expression of gene not specified, consequence high lactate production, significant impact on H2O2-induced oxidative stress, improved oxidative resistance, changes in AdoMet concentrations, changes in response to DNA synthesis inhibitors, changes in response to tamoxifen, deletion of oxidoreductases, improvement in vanillin accumulation, overexpression of coenzyme-A free pathways, enhancement in vanillin production, optimization of NADPH supply, increase in vanillin productivity, optimization of S-adenosylmethionine supply, metabolic reconfiguration of yeast central metabolism, enhancement in vanillin productivity, deletion of gene met17, inability to grow on sulfate as sole sulfur source, ability to grow on sulfate at high cell densities, accumulation of hydrogen sulfide, enables growth of met17Δ on sulfate, induction of gene hsu1, response to sulfur starvation, response to high sulfide concentrations, gene hsu1, enables assimilation of hydrogen sulfide, low efficiency of sulfide assimilation in met17Δ, explains density-dependent growth on sulfate, deletion of gene SIW14, increase in cells with abnormal endoplasmic reticulum, increase in accumulation of reactive oxygen species, involvement of gene OCA1, protein phosphorylation pathway, involvement of gene SIW14, deletion of gene PfVP1, delayed ring stage development, blockage of ring-to-trophozoite transition, onset of parasite death, state of quiescence, rescue of quiescent cell fate, loss of function for CHH methylation in osdrm2, compromised CHH methylation at the promoter and transcript level of OsGSTZ4, defective in RNA-dependent RNA polymerase 2 in osrdr2i, deletion of gene CAR1, reduces accumulation of ethyl carbamate, causes strong growth inhibition on medium with arginine as only nitrogen source, decreases splicing in vivo of reporter genes, accumulates first-step products in vitro, deletion of gene pdc, subdue ethanol formation, introduction of gene l-lactate dehydrogenase from Weizmannia coagulans 2-6, redirect ethanol fermentation pathway to l-LA production, introduction of gene L-ldhA from Bos taurus, deletion of gene dld(chr3), increase optical purity of l-LA, deletion of gene BUD21, improvement in growth, enhancement in substrate utilization, increase in ethanol productivity, no confirmation of positive effect on aerobic growth in non-engineered strains, no confirmation of positive effect on xylose utilization in non-engineered strains, upregulation of select ATG transcripts under nutrient-replete conditions, increased Pdr5-GFP levels, changes in sensitivity or resistance to azoles, changes in sensitivity or resistance to cisplatin, changes in sensitivity or resistance to oxidative stress, changes in sensitivity or resistance to DNA synthesis inhibitors, changes in sensitivity or resistance to tamoxifen, deletion of gene MoCBP7, increased protein secretion, deletion of gene involved in calcium metabolism, resistance to aluminium, resistance to gallium, resistance to indium, deletion of gene involved in Ire1/Hac1-mediated protection, deletion of gene involved in vesicle-mediated transport, deletion of gene involved in autophagy, deletion of gene involved in protein folding, deletion of gene involved in phospholipid metabolism, deletion of gene involved in chorismate metabolic processes, deletion of gene Hsp26, decreased production of ethyl acetate, deletion of gene Hsp78, deletion of gene Hsp82, overexpression of gene HSP30, increased transcriptional abundance of ATF1 gene, increased production of ethyl acetate, overexpression of gene Hsf1, alleviated unfolded protein response, reprogramed pattern of ATF1 gene transcript levels, overexpression of gene Hsf1 and Hsps, strong promoter TDH3p, decreased transcription level of ATF1, activated unfolded protein reaction, deletion of gene DFG5, improved BGL1 secretion, deletion of gene YPK1, deletion of gene FYV5, deletion of gene CCW12, deletion of gene KRE1, double deletion of genes FYV5 and CCW12, increased secreted BGL1 activity by 6.13-fold, increased surface-displayed BGL1 activity by 7.99-fold, selenite induction, improvement of thermotolerance, deletion of ET domain in BcTaf14, deletion of gene DDGS;OMT;ATPG, consequence not producing mycosporines, consequence accumulating intermediate 4-deoxygadusol, expression of gene DDGS;OMT, consequence producing 4-deoxygadusol, expression of gene DDGS;OMT;ATPG, consequence producing MG, integration of gene DDGS;OMT;ATPG, consequence producing MG and mycosporine glutaminol glucoside, deletion of gene QDR3, reduced support for Lactobacillus fermentum growth, deletion of gene QDR1, reduced lactic acid production, deletion of gene QDR2, no significant reduction in ethanol production, increased secretion of Pexelizumab scFv peptide, redistribution of sphingolipid in the plasma membrane, deletion of gene SSE1, consequence severe synthetic growth defects, deletion of gene PRE9, deletion of gene PBA3, deletion of gene UMP1, consequence high levels of ubiquitin-conjugates, consequence accumulation of distinct small complexes with α subunits, increased IA titer, deletion of gene HXK2, deletion of gene GSF2, overexpression of gene cadA, overexpression of gene Dz.ada, overexpression of gene Mt.acs, overexpression of gene aac2, overexpression of gene related to glucose derepression, overexpression of acetyl-CoA synthesis pathway, improved ATP supply, deletion of gene pho84, increased vanillin tolerance, deletion of gene lam3, deletion of genes involved in phosphate metabolism, deletion of genes involved in inositol polyphosphate metabolism, deletion of genes involved in sterol transport, activation of phosphate signaling pathway, decreased vanillin tolerance, reduction of sterol transport from plasma membrane, vanillin, inhibits growth in Saccharomyces cerevisiae, pho84Δ, promotes rapid ethanol production, lam3Δ, expression of Kα in mating type \"a\" yeast, reduced growth interference, enhanced expression of pathway genes by Gal4p, delay in conidia production, reduction in conidia number, suppression of genes related to asexual development, suppression of genes related to osmotic stress resistance, suppression of genes related to oxidative stress resistance, deletion of gene Met17p, consequence organosulfur auxotrophy, expression of gene HSU1, consequence substitution for Met17p function, consequence reassimilation of hydrosulfide ions, consequence formation of homocysteine, decreased microsclerotia production, decreased expression of VaflM, decreased expression of Vayg1, decreased expression of VDH1, decreased expression of VdLAC, decreased expression of VdSCD, decreased expression of VT4HR, weakened penetration ability of mycelium, impaired growth of mycelium, deletion of gene MoGap1, sporulation reduction, increased sensitivity to cell wall stress, increased sensitivity to carbon source stress, decreased TOR activity, improved level of autophagy, increased secreted α-amylase activity, increased surface-displayed α-amylase expression by 56%, increased secreted α-amylase expression by 49%, upregulation of genes associated with the pentose phosphate pathway, upregulation of genes associated with cell wall organization or biogenesis, upregulation of genes associated with glutathione metabolism, upregulation of genes associated with protein refolding, upregulation of genes associated with phosphatidylcholine biosynthesis, upregulation of genes associated with AdoMet synthesis, inactivation or structure alteration of YNL194Cp, constitutive expression, inducible expression, expression of gene HpPMA, highest promoter activity, elevating Cu<sup>2+</sup>, increased expression of reporter gene with ScCUP, deletion of gene FgELO2, serious defects in vegetative growth, disrupted conidiation, disrupted ER-localization of 14-α-demethylase FgCyp51s, elevated sensitivity to tebuconazole, decreased phosphorylation of high osmolarity glycerol (HOG, increased sensitivity to osmotic stress, reduced fungal virulence, inhibited deoxynivalenol production, inhibited formation of infection structures, elevated levels of glycolytic intermediates, increased expression of ACS1, increased expression of ALD6, enhanced ADH2 activity, enhanced sulfur-containing amino acids biosynthesis, extended yeast lifespan, deletion of gene gpd1, improved D-LA titer, deletion of gene gpd2, deletion of gene adh1, expression of gene DLDH, decreases in production of de novo pathway metabolites, reduced acetylation of H4K5 and H4K12, increases acetylation of H4K5 and H4K12, increases BNA gene expression, decreases repression of de novo NAD<sup>+</sup> synthesis, deletion of gene MoSCS2, reduction in growth, reduction in conidiation, abnormal septum formation, delayed appressorium formation, inability to form huge turgor pressure, inhibition of invasive hyphae expansion, decline in pathogenicity, deletion of gene HXS1, slower rate of glucose consumption, slower rate of xylose consumption, 4 times less ethanol production, overexpression of gene HXS1, 10% increase of ethanol production from glucose, more than 2 times increase of ethanol production from xylose, overexpression of gene AZF1, 10% more ethanol production in glucose medium, 2.4 times more ethanol production in xylose medium, deletion of gene AZF1, 1.5 times drop in ethanol accumulation in xylose medium, 1.8 times drop in ethanol accumulation in glucose medium, deletion of gene fet3, inability to grow in low-iron conditions, deletion of gene arn1-4, expression of gene sib1+, production of ferrichrome, expression of gene sib2+, deletion of gene sib1+, defect in ferrichrome production, deletion of gene sib2+, deletion of gene sib3+, severe growth defect in iron-poor media, inability to promote ferrichrome-dependent growth of S. cerevisiae, deletion of gene las21, impaired yeast growth on medium with cell wall stressors, deletion of gene ecm33, overexpression of gene ECM33, improved specific yield of p-coumaric acid, deletion of gene ECM33, improved specific yield of β-carotene, decreased chitin deposition, deletion of gene BAT2, decline in yields of 2-methyl-butyraldehyde, decline in yields of 3-methyl-butyraldehyde, decline in yields of 2-methyl-1-butanol, decline in yields of 3-methyl-1-butanol, increase in yields of 2-methylbutyl-acetate, increase in yields of 3-methylbutyl-acetate, overexpression of Ile, increase in yields of 2-methyl-butyraldehyde, increase in yields of 2-methyl-1-butanol, overexpression of Leu, increase in yields of 3-methyl-butyraldehyde, increase in yields of 3-methyl-1-butanol, beneficial for surviving some toxins, detrimental for others, deletion of gene Gem1, consequence 1: decreased viability, consequence 2: increased mitochondrial H2O2 production, consequence 3: increased ER stress, consequence 4: increased growth in presence of mutant alpha-synuclein, consequence 5: reduced ER stress when expressing A53T alpha-synuclein, consequence 6: increased ability to deal with oxidative stress, overexpression of Sox, production of reactive hydrogen peroxide, undesired cross-linking of functional proteins, overexpression of interleukin-1β, pro-inflammatory cytokine production, gene deletion of unknown gene, improved α-amylase secretion, point mutation in unknown gene, deletion of gene Aqr1, no impact at mild acetic acid stress, deletion of gene Tpo2, decreased growth at severe acetic acid stress, deletion of gene Tpo3, crucial role in acetic acid tolerance, deletion of gene AQR1, least effect on growth and acetate efflux, deletion of gene Tpo2 and Tpo3, enhanced severe growth defects at 20 mM acetic acid, reduced rate of acetate secretion, overexpression of gene TPO2, restored tolerance in ∆tpo2∆tpo3∆ strain, overexpression of gene TPO3, accumulation of acetate intracellularly, activation of gene ethanol metabolism, under acetic acid stress conditions, reduction of gene acetic acid production, deletion of gene ATF1, increase of 2-PE production by 114% in PK-2C strain, decrease of 2-PE yield from 3.50 to 0.83 g/L in CWY-132 strain, deletion of gene ALD2, increase of 2-PE yield from 0.21 to 1.20 g/L in PK-2C strain, decrease of 2-PE yield from 3.50 to 3.02 g/L in CWY-132 strain, deletion of gene ALD3, decrease of 2-PE yield from 3.50 to 2.93 g/L in CWY-132 strain, deletion of genes ALD2 and ALD3, decrease of 2-PE yield to 1.65 g/L in CWY-132 strain, supply of NADH, increase of 2-PE production in S. cerevisiae, supply of GSH, increased hydroxytyrosol yield, overexpression of gene paHD, improved catalytic performance for hydroxytyrosol production, overexpression of gene aro4K229L, enhanced hydroxytyrosol precursor supply, overexpression of gene aro7G141S, overexpression of gene AAS, enhanced tyrosine metabolic pathway for hydroxytyrosol production, overexpression of gene Bbxfpkopt, redistributed metabolic flux for increased hydroxytyrosol production, expression of gene ERG1, partially substituted spermine accumulation, CiSHT2 catalyzes first N-acylation steps, CiSHT2 catalyzes, N5,N10-dicoumaroyl spermine formation, CiSHT1 uses, N5,N10-dicoumaroyl spermine to synthesize tetracoumaroyl spermine, deletion of gene GAL7, involvement in HR repair pathway, involvement in NHEJ repair pathway, deletion of gene YMR130W, deletion of gene YHI9, enhanced sensitivity to DNA damage-inducing drugs, interaction with SIR2, interaction with SIR3, interaction with SIR4, deletion of gene SNF1, affects exponential growth, affects mitochondrial respiration, affects transcript levels of hexose transporters, correlates positively with Vmax of hexose transporters, correlates negatively with mitochondrial respiration, deletion of gene SNF1/HXK2/MIG1, most affects HXT2 gene transcript levels,  deletion of gene EHT1 and EEB1, consequence distinct ethyl ester profiles, lower unsaturated lipid acid content, higher transposon activity, loss of exons in DDS-encoding gene, lack of dammarane-type saponin production, overexpression of PgDDS gene, recovery of dammarane-type saponins accumulation, tandem duplication of AeCYP72As, synthesis of oleanane-type saponins, tandem duplication of AeCSLMs, tandem duplication of AeUGT73s, synthesis of oleanane-type saponins and aralosides, consequence regulation of transcription factors expression, consequence regulation of secreted proteins expression, consequence increased chromosome mis-segregation, consequence accumulation of cenRNAs at all phases of the cell cycle, elimination of galactose supplementation requirement, comparable artemisinic acid production, mutation of gene URA3, affected growth, reduced artemisinic acid production, restoration of gene ura3, normal growth, comparable artemisinic acid yield, deletion of gene ATG32, increased fermentation performance, no significant differences in CO2 production, no significant differences in final ethanol titers, no significant differences in cell viability, deletion of gene involved in endocytosis, disruption of vesicle-mediated transport, disruption of glycosylation, deletion of gene encoding transferases, induction of cytotoxicity, deletion of gene encoding proteases, deletion of gene encoding chaperones, disruption of metabolic processes, deletion of gene FgPEX22-like, consequence 1: impaired sexual and asexual reproduction, consequence 2: reduced fatty acid utilization, consequence 3: decreased pathogenicity, consequence 4: reduced production of deoxynivalenol, consequence 5: increased production of lipid droplets, consequence 6: decreased elimination of reactive oxygen species, consequence 7: disrupted biogenesis of Woronin bodies, overexpression of gene VlAbcG1a, increased tolerance to β-pinene, deletion of gene VlAbcG1a homologous gene, increased sensitivity to certain monoterpenes, increased virulence in tomato plants, deletion of gene ADA2, increase in efficiency of AMT, increase in double-strand DNA breaks, association of gene MoRPD3 with TOR kinase pathway, dependency of appressorium formation on hydrophilic surfaces, dependency of appressorium formation on TOR's inhibition, metabolic network models enable analysis, integration of gene expression data, metabolic network models require GPR rules, reconstruction of GPRs, manual and time-consuming process, automating GPR reconstruction, GPRuler tool proposed, GPRuler, reconstructs GPRs from organism name or existing model, GPRuler performance, evaluated with small-scale and genome-scale metabolic models, reproduces original GPR rules with high accuracy, reveals more accurate than original models in some cases, enables study of context-specific metabolic networks, deletion of open reading frame, consequence complete deletion, insertion of gene fragments to essential genes, consequence median efficiency 52.1%, introduction of single amino acid substitutions, consequence median efficiency 79.2%, reduced virulence of C. albicans, increased cellular mitochondrial reactive oxygen species (ROS, prevents proper colony formation when interacting with macrophage, defect in expression of mitochondrial protein genes, attenuated virulence in vivo, SLP expression, elevated upon co-culture with fungi, change in protein abundance of neighboring genes, shift of transcription start site of neighboring genes in head-to-head orientation, introduction of new uAUG codons in 5' UTR of neighboring genes in head-to-head orientation, activation of alternative polyadenylation signals in neighboring genes in tail-to-tail arrangement, alteration in 3' UTR of neighboring genes in tail-to-tail arrangement, increased rates of aerobic and anaerobic xylose fermentation, deletion of gene ISU1, deletion of gene GRE3, duplications of genes encoding engineered xylose metabolism enzymes, duplications of gene TKL1, deletion of gene Akr1, increased cell death, deletion of gene Erg3, addition of calcium, calcium induced expression of gene HXT1, maintained intracellular ROS levels, calcium induced expression of gene AGP1, calcium/calcineurin signalling pathway induced expression of gene AKR1, helped yeast cells survive under high calcium concentrations, calcium/calcineurin signalling pathway induced expression of gene ERG3, increased cell wall chitin, increased chitin content, overexpression of gene CAB1 W331R, level of CoA nucleotides increased 15-fold, overexpression of wild-type CAB1, level of CoA nucleotides increased fourfold, supplementation with pantothenate, level of CoA nucleotides increased 2.3-fold, overexpression of gene FEN2, minor increase of CoA nucleotides, deletion of gene PCD1, synonymous mutation at position 315 nt downstream of Spt15, downregulation of genes involved in TCA cycle, downregulation of genes involved in glyoxylic acid cycle, reduced production of acetaldehyde, deletion of gene THI3, reduced production of fusel alcohols, deletion of gene ADH2 and THI3, reduced yields of acetaldehyde, reduced yields of isoamyl alcohol, reduced yields of iso-butanol, deletion of gene FpHsp104, severe defects in heat shock resistance, defects in conidia production, defects in conidia morphology, defects in conidia germination, reduced transcript levels of FpFluG, reduced transcript levels of FpVosA, reduced transcript levels of FpWetA, reduced transcript levels of FpAbaA, deletion of gene MBP1, decrease in final ethanol concentration under oxygen-limited conditions, decrease in respiration intensity under aerobic conditions, decrease in respiration intensity under oxygen-limited conditions, pseudohyphal growth, increase in cell volume, decrease in the fraction of budded cells, overexpression of gene Dga1p, increased TAG accumulation, deletion of gene ACC1, deletion of genes involved in glycogen synthesis, deletion of genes involved in steryl ester synthesis, deletion of genes involved in TAG hydrolysis, deletion of gene CKB1, promotion of nuclear accumulation of Hog1, increase in cytosolic localization of Slt2, increase in bud neck localization of Slt2, overexpression of gene ERG, increased squalene accumulation, overexpression of gene HMG, overexpression of gene DGA1, increased lipid production, deletion of gene PEX10, deletion of gene URE2, increased lipid accumulation, increased squalene content, deletion of gene URE2, positive synergistic effect on squalene production, insertion of CDO from C. albicans, reduced biomass formation, consequence lower availability of FAN, consequence lower yield of higher alcohols, consequence downregulation of GAP1, consequence downregulation of ARO9, consequence downregulation of ARO10, deletion of a single pivotal gene, extensive effects induced, genes upregulated related to carbon storage, extends replicative lifespan, increases extracellular level of anti-CD22 scFv, alters susceptibility to protein synthesis inhibitors, no significant influence on ethanol production from xylose, no significant influence on ethanol production from glucose, deletion of both genes MIG1 and MIG2, reduced ethanol production from xylose, reduced ethanol production from glucose, deletion of gene HAP4-A, increased ethanol production from xylose, deletion of gene TUP1, overexpression of gene HAP4-A, overexpression of gene TUP1, deletion of gene IRC7, deletion of gene KGD1, improved tolerance to catechol, disruption of ergosterol biosynthesis, adipic acid hypersensitivity, disrupting protein trafficking, disrupting vacuolar transport, adipic acid, disrupts Membrane Compartment of Can1 (MCC, impacts endocytosis, rapid internalization of Can1, vacuolar degradation, ethanol conversion decreased by 30.19%, glycerol conversion increased by 40.005%, acetic acid concentration decreased by 43.54%, acetoin concentration increased by 12.79 times, pyruvate decarboxylase activity decreased by 40.91%, deletion of gene pdc1 and pdc5, production of 2,3-BD increased, elevated citronellol titer, overexpression of gene Erg20<sup>F96W</sup>, increased GPP, overexpression of gene CrIS, produced S-citronellol, deletion of gene GPD1, minimizes glycerol production, deletion of gene DLD1, prevents D-lactic acid conversion to pyruvate, overexpression of gene LpDLDH, increases D-lactic acid production, downregulation of gene ADH1, minimizes ethanol production, accumulation of intracellular manganese, manganese sensitivity, addition of CaCl2, inhibition of manganese sensitivity, regulation by calcium/calcineurin signaling pathway, interaction between manganese and calcium, function of Pmr1, transport of Ca2+ and Mn2+, deletion of gene PaPRO1, sufficient production of PaE without degradation fragments, deletion of gene PaPRO2, overexpression of gene PaE, increased production of PaE, maintenance of PaE activity after cultivation, consumption of geraniol, production of geranic acid, production of citronellic acid, production of citronellol, deletion of gene adh3, deletion of gene adh4, deletion of gene sfa1, less impairment of respiratory growth compared to PTH3, tetracycline presence, reduced protein expression due to PTH3 deletion, increased GSH content, deletion of gene dugB and dugC, downregulated steC, downregulated sskB, downregulated hogA, downregulated mkkA, downregulated flbA, downregulated flbC, downregulated flbE, downregulated nosA, downregulated rosA, downregulated nsdC, downregulated nsdD, slowed GSH depletion during carbon starvation, reduced reactive oxygen species accumulation, deletion of competing pathway genes, increased carbon flux towards desired product, overexpression of pathway enzymes, increased pathway efficiency, colocalization of enzymes, increased reaction likelihood along pathway, mutations in 59 genes, improved cell growth, enhanced basal transcription factors, improved cell integrity signaling, enhanced translation initiation and elongation, improved ribosome assembly and function, enhanced oxidative stress response, improved tolerance toward isobutanol, deletion of gene CLB5, reduced efficiency of replication initiation in rDNA, arrested forks more stably at the RFB, led to homologous recombination-dependent rDNA instability, deletion of gene FOB1, did not completely suppress rDNA instability in clb5Δ, degradation of Erg20p, redirection of metabolic flux toward monoterpene production, depletion of hexokinase-2, lifting of glucose repression, boosting of sesquiterpene nerolidol production to 3.5 g L^-1, depletion of Acc1p, growth arrest, no diminishment of production capacity in nerolidol-producing yeast, increased radial growth in the presence of oxidative stress, sensitivity to high osmolarity, deletion of gene kcs1, increased SAM by 83.3%, deletion of gene arg82, increased SAM by 31.8%, improved transcription levels of various glycolytic genes, enhanced activities of relative enzymes, enhanced levels of glycolytic intermediates, enhanced levels of ATP, deletion of gene kcs1 in Ymls1ΔGAPmK, increased SAM to 2.89 g L^-1 in a 250-mL flask, increased SAM to 8.86 g L^-1 in a 10-L fermentation tank, slow growth on glycerol, impaired invasive ability, leads to impaired aflatoxin synthesis, brown colony phenotype, expression of gene ScRad52, improved HR efficiency, gene targeting efficiencies up to 95%, deletion of gene ku70, traditionally used strategy for improving HR efficiency, consequence abolishes chlamydosporulation, consequence bypasses requirement for chlamydosporulation cues and regulators, overcome inhibition of CoQ biosynthesis by Bz, deletion of gene ppt1, inhibited CoQ biosynthesis by Bz, addition of benzoic acid, inhibited CoQ biosynthesis, addition of p-aminobenzoate, counteracted inhibition of CoQ biosynthesis by Bz, addition of p-hydroxybenzoate, increased xylitol accumulation, deletion of HXT gene, chimeric HXT sequences, interstitial deletions, cleavage of HXT gene, gene conversions, deletion products, impairment of SSA pathway, reduced survival frequency, disruption of Rad51-dependent homologous recombination, modest reduction in survival frequency, deletion of gene Matα:Δ57-70, consequence increased G-CSF secretion, consequence highest G-CSF titre of 39.4 ± 1.4 mg/L, deletion of gene Matα:Δ30-43, consequence diminished G-CSF secretion, deletion of gene Matα:Δ47-49, consequence dampened G-CSF secretion, deletion of gene Matα:Δ57-70;47-49, consequence stimulatory effect on G-CSF secretion, codon optimization of gene GCSF, consequence overcoming control by signal peptide segments, deletion of gene Matα:Δ57-70;30-43, consequence partial restoration of G-CSF secretion, overexpression of gene MAL62, increase in freezing tolerance, increase in UDPG-dependent trehalose synthesis, abolition of UDPG-dependent trehalose synthase activity, failure to promote intracellular trehalose synthesis upon MAL62 overexpression, lack of trehalose synthesis, slight reduction in cell viability during prefermentation freezing with MAL62 overexpression, MAL62 overexpression, increase in intracellular glycerol concentration, elevated transcription of glycerol synthesis pathway members, deletion of gene TSA1, diminishes growth in medium with glucose, diminishes growth in medium with sucrose, diminishes growth in molasses, lowers trehalose accumulation in early growth stages, increases trehalose accumulation at end of batch growth, increases glycogen accumulation at entry of stationary phase, reduces fermentative capacity in grape juice, decreases acetic acid production, decreases acetaldehyde production, deletion of gene TPK1, reduces efficiency of NHEJ repair of breaks with overhangs, reduces efficiency of NHEJ repair of blunt ends, defects in the repair of chromosomal breaks induced by HO-site specific endonuclease, consequence high sensitivity to high osmolarity, consequence reduction in trap formation, consequence reduction in predation efficiency, consequence required for conidiation, deletion of gene MSB2, consequence respiratory capacity reduced, presence of gene AOX, consequence no effect of inhibitors on growth, consequence minor effect of inhibitors on growth, consequence respiratory growth impairment, consequence viability loss in absence of mtDNA, upregulation of 176 nuclear genes, consequence expression of mitochondria-encoded proteins, increased β-amyrin production by 1.85 times, increased β-amyrin yield by 3.75 times, deletion of gene TPO1, MCFOH production improvement, expression of modified MmCAR, increased MCFOH production, deletion of gene MCK1, enrichment in all processes tested, deletion of gene RIM11, deletion of gene MRK1, deletion of gene YGK3, reflects presence of significant stresses, affected by feeding scheme, invasive growth, ΔFglsb6 deletion, defects in vegetative growth, decreased deoxynivalenol (DON, reduced pathogenicity, increased resistance to cell wall stresses, defective generation of PI4P on endosomes, defective endocytosis, deletion of gene UvATG8, consequence vegetative growth defect, consequence conidiation defect, consequence oxidative stress adaptation defect, consequence hyperosmotic stress adaptation defect, consequence cell wall stress adaptation defect, consequence production of toxic compounds defect, consequence reduced conidiation, consequence reduced secondary spore formation, decreased cytotoxicity to yeast cells, yielded less inflammatory pathology in infected mouse lungs at sublethal infectious doses, improved gene editing efficiency, enhanced chromosomal gene integration, successful gene deletion, consequence forms complex with Ifh1p to play a key role in transcription of ribosomal protein gene, consequence forms complex with Fhl1p to play a key role in transcription of ribosomal protein gene, consequence plays an important role in ribosome biosynthesis, consequence eliminates negative regulator of cell growth, overexpression of gene IFH1, overexpression of gene SSF2, deletion of gene HRP1, consequence cell size increased, overexpression of gene IFH1, overexpression of gene SSF2, deletion of gene HRP1, consequence growth rate increased, hampered cell cycle arrest, increased expression level of adhesion protein FLO11, increased synthesis of trehalose, reduced production of benzyl alcohol, deletion of gene FUM1, overexpression of gene PYC2, alleviates cell slow growth, derepression of rRNA transcription, derepression of RPG transcription, phosphorylation of Rph1, resistance to rapamycin treatment, decreased association with ribosome-related genes, faster cell growth, DH1 domain, DH2 domain, reduced sulphite tolerance, expression of gene MXR1, elevated expression level of Mxr1, high activity of P<sub>AOX1</sub>, P<sub>AOX1</sub> expression, improved production of heterologous proteins by 18%, TRF5 deletion, increase in cell wall chitin, expression of HXK2S14A, deletion of MIG1, impaired gene silencing in N. castellii, affecting the composition of siRNA species in the cell, decreased FA 16:0 accumulation, decreased FA 18:0 accumulation, deletion of gene thiamine transporter, consequence growth rate decrease, consequence growth efficiency decrease, consequence CO2 production decrease, activation by Vta1, requirement of Vta1, consequence survival of microsclerotia, deletion of gene YPS1-5, consequence highly sensitive to α-factor pheromone, consequence mates with reduced efficiency, overexpression of gene SET1, overexpression of gene SET2, elimination of yeast ability to grow on glucose as the sole carbon source in Saccharomyces cerevisiae, no growth failure in glucose, galactose, or fructose in Kluyveromyces lactis, no effect on yeast growth rate in Kluyveromyces lactis, absence of trehalose in the metabolism of Kluyveromyces lactis, consequence suppressed cell lysis, consequence improved stress tolerance, activation of CWI pathway, consequence enhanced squalene production efficiency, consequence restored membrane rigidity, deletion of gene MHY1, deletion of gene pex3, decrease of ethanol production from xylose, deletion of gene cta1, impairment of ethanol production from xylose, overexpression of gene pex34, activation of xylose alcoholic fermentation, integration of gene ARO10*, consequence YMGA* produced 0.14 mM tyrosol from 2% glucose without expensive media supplements, deletion of gene feaB, consequence increased tyrosol production, deletion of gene pheA, deletion of gene tyrB, deletion of gene tyrR, increasing copy number of gene ARO10* to five copies, consequence increased tyrosol production to 10.92 mM, optimizing fermentation conditions, consequence increased tyrosol production to 28 mM, deregulation of gene PRE1 leading to Rpn4 accumulation, inhibition of PRB1 induction and YPL hypersensitivity to ethanol, suppression of transcription of genes encoding C2H2 zinc-finger family transcription factors, suppression of transcription of genes encoding cell wall-degrading enzymes, suppression of transcription of genes encoding chitin-binding proteins, reduced conidial formation, transcriptional repression of fluG, transcriptional repression of rodA, transcriptional repression of abaA, transcriptional repression of medA, transcriptional repression of lreA, sensitivity to Congo red, sensitivity to NaCl, sensitivity to sorbitol, elevated intracellular ROS levels, overexpression of antioxidant defense genes, up-regulation due to SDS stress, overexpression of gene C. tropicalis XR mutant, replacement of ZWF1 promoter with GPD promoter, consequence clipping Histone H3 N-terminal tails, inefficient sporulation, partial compensation by trehalose, partial compensation by glycogen, increase in cell size, increase in mitochondria, increase in lipid stores, rescue of meiosis progression, rescue of spore ascus formation, increased dependency on mitochondrial activity, increased dependency on lipid stores, induction of unfolded protein response, activation of serine/threonine kinase Ire1, activation of endoribonuclease Ire1, high efficiency of gene alteration, introduction of site-specific polymorphism, insertion of tag, cleavage within GFP, replacing GFP tag with other sequences, upregulates cell wall integrity related genes expression, upregulates autophagy related genes expression, upregulation of AVT6, upregulation of VBA5, upregulation of THI72, upregulation of ARO80, over-expression of p53, targeting of mitochondria, accumulation of PPP enzymes, enhanced mitochondrial NADPH production, over-expression of HXK-2, enhanced association with mitochondria, increased NADPH, deletion of gene GND1, deletion of gene GLN3, increased tolerance to branched-chain alcohols, upregulation of gene GLN3, induces nitrogen starvation response, upregulation of gene GCN4, disruption of nitrogen starvation response by deletion of GLN3, enhances tolerance to isobutanol, boosts isobutanol production 4.9-fold, deletion of gene rpe1, deletion of gene pda1, increased l-PAC formation, deletion of gene FpDEP1, retarded growth on PDA plates, reduced aerial hyphae formation, no typical leaf lesions, limited expansion capability of coleoptiles, massive reactive oxygen species (ROS, limited expansion to neighboring cells, sensitivity to H2O2, expression of gene FpDEP1-GFP, complementation of mutant phenotypes, localization to the nucleus in spores and mycelia, restores penetration, restores melanin production, prime editing of HBB, correction of sickle cell disease cause, prime editing of HEXA, correction of Tay-Sachs disease cause, prime editing of PRNP, installation of protective transversion, consequence reduced ethanol yields on biomass, similar unstable rDNA phenotype, deletion of chromosomal regions, enhanced by CRISPR/Cas9, replacement of chromosomal regions, integration of CRISPR/Cas9, activates homologous recombination, multiple chromosomal regions manipulation, enabled by CRISPR-PCD, enabled by CRISPR-PCRep, deletion of gene sucC, increase in SAM titer, expression of gene SAM2, expression of gene metA, improvement in SAM production, expression of gene metB, expression of gene mccA, production of lactic acid, deletion of gene JEN1 function, shift in product formation towards ethanol, deletion of gene ADY2 function, upregulation of gene JEN1, upregulation of gene ADY2, deletion of gene PHO13, increased arabinose consumption rate, increased specific ethanol productivity, activation of gene TAL1, depletion of sedoheptulose, upregulation of pentose phosphate pathway, trehalose accumulation, downregulation of TCA cycle, high efficiency gene knockout, simultaneous deletion of two different genes, mediated by inducible promoter, expression of gRNA, mediated by constitutive promoter, redox imbalance resolved, overexpression of NADH oxidase, introduction of Candida tropicalis PDC1, growth defect resolved, minimized ethanol production, deletion of GPD1 and GPD2, glycerol production eliminated, 108.6 g/L 2,3-BD produced, negligible glycerol production, deletion of gene gal80, consequence artemisinic acid production without galactose induction, deletion of gene TPK2, increased cAMP levels, decreased PKA activity, deletion of gene BCY1, deletion of gene TPK3, overexpression of cAMP, improved heat resistance, no influence on growth, no influence on fermentation properties, enhanced maltose metabolism, trehalose synthesis before fermentation, similar effect on cell viability after freezing as TPS1 overexpression, NTH1 deletion, strengthened freezing tolerance with MAL62 overexpression, strengthened freezing tolerance with TPS1 overexpression, improved leavening ability with MAL62 overexpression, improved leavening ability with TPS1 overexpression, overexpression of gene Ilv2, increase of isobutanol production, overexpression of gene Ilv5, overexpression of gene Ilv3, deletion of gene ILV2, deletion of gene BDH2, deletion of gene LEU4, deletion of gene LEU9, deletion of gene ECM31, deletion of gene ADH1, increase of glycerol formation, overexpression of gene New1, rescues inviability of yeast strain lacking eEF3, loss of gene New1, leads to ribosome queuing upstream of 3'-terminal lysine and arginine codons, deletion of gene Ded1, reduction in translation efficiency of many mRNAs, deletion of gene Dbp1, overexpression of gene Dbp1, improvement in translation of Ded1-hyperdependent mRNAs, increased dependence on Dbp1 for translation, increased dependence on Ded1 for translation, expression of gene Dbp1, acceleration of 48S PIC assembly on structured 5'-UTRs, decreased PIC attachment and scanning on many mRNAs, deletion of gene FvSec4, no impairment in expression of key conidiation-related genes, no defect in sexual development, defect in corn stalk rot virulence, alteration of fumonisin B1 production, higher sensitivity to oxidative stress, higher sensitivity to cell wall stress agents, defects in carbon utilization, expression of gene LI1035, consequence inhibits yeast growth, consequence interacts with components of the MAPK pathway, consequence inhibits MAPK signaling cascades in yeast, consequence inhibits MAPK signaling cascades in mammalian cells, consequence regulates actin organization in yeast, consequence regulates actin organization in mammalian cells, deletion of gene ATG2, failure to increase chronological lifespan, overexpression of gene ATG32, increase in autophagy, overexpression of gene SOD1, participation in antiaging effect of CuB, overexpression of gene SOD2, overexpression of gene UTH1, overexpression of gene SKN7, increase in Sod activity, increased survival rate under oxidative stress, decrease in ROS, reduced oxidative damage, decrease in MDA production, reduced lipid peroxidation, expression of mfsG, consequence 1: increased efflux ability of fungitoxic compounds, deletion of mfsG, consequence 1: decreased efflux ability, consequence 2: increased accumulation of isothiocyanates, consequence 3: increased sensitivity to isothiocyanates, consequence 4: reduced virulence towards plants containing glucosinolates, deletion of various gene targets, greater than 97% gene disruption efficiency, deletion of double gene, disruption efficiency of 90%, deletion of triple gene, disruption efficiency of 47%, deletion of gene TPS1, consequence decreased trehalose, consequence decreased Val, consequence decreased Thr, consequence decreased Lys, consequence decreased Asp, consequence decreased His, consequence decreased Trp, consequence decreased malonate, consequence decreased citrate, consequence decreased uridine, consequence decreased guanosine, consequence decreased inosine, consequence decreased AMP, consequence decreased C10:0, consequence decreased C16:1, consequence inhibited glycolysis, consequence inhibited tricarboxylic acid cycle, consequence promoted phosphopentose pathway, consequence promoted nucleotide synthesis, consequence reduced acetyl-CoA-mediated mycotoxin biosynthesis, consequence increased fatty acids, consequence impaired trehalose biosynthesis, consequence trehalose 6-phosphate accumulation, higher alcohol reduction, overexpression of gene ATF1, ester improvement, deletion of pCS, change in fluorescence intensity, expression of pCS, drives EGFP expression, heat stress, increases fluorescence intensity, deletion of pCS between -500 bp and -400 bp, alters heat response, mutation of pCS between -490 bp and -500 bp, affects heat-inducible element, deletion of gene FRA2, upregulation of iron-sulfur metabolism, expression of gene xylB, part of Weimberg pathway, expression of gene xylD, expression of gene xylX, not functional in S. cerevisiae, replacement of gene xylA with ksaD, higher in vitro activity observed, loss of gene XylB, minimizing xylonate accumulation, introduction of gene xylD, part of lower Weimberg pathway, introduction of gene xylX, introduction of gene ksaD, expression of Weimberg pathway, detection of intermediates and biomass generation from xylose, deletion of gene pcr4401, consequence albino phenotype, heterologous expression of gene pcr4401, consequence melanin precursor YWA1 synthase, heterologous expression of uncharacterized PKS gene from Aspergillus, consequence biosynthetic products elucidated, heterologous expression of two uncharacterized PKS genes from Penicillium strains, deletion of gene ira2, consequence specific xylose consumption rate increase, consequence ethanol production rate increase, consequence biomass titre decrease, deletion of gene isu1, deletion of gene hog1, consequence xylose uptake improvement, consequence signalling of high glucose, consequence signalling of low glucose, deletion of gene FgPEX1, production of perithecia reduced, shortage of acetyl-CoA, reduced deoxynivalenol production, inhibited virulence, increased accumulation of lipid droplets, increased endogenous reactive oxygen species, involvement in cell wall integrity maintenance, involvement in Woronin bodies maintenance, deletion of gene FgPEX10, deletion of gene SIR2, increased rDNA origin efficiency, relocalization of Mcm2-7 helicases, SNF1 deletion, decreased glycolytic flux, increased NAD(P, enhanced HXK2 gene transcription, decreased mitochondrial respiration, increased mitochondrial respiration, deletion of gene tir1, reduction in higher alcohol production by 23.5%, deletion of gene gap1, reduction in higher alcohol production by 19.66%, overexpression of LmSAP, increased tolerance to salt and osmotic stresses, increased SOD activity, increased CAT activity, increased POD activity, decreased MDA levels, overexpression of LmSAPΔAN1, deletion of A20 domain in LmSAP, reduced tolerance to salt and osmotic stresses, reduced SOD activity, reduced CAT activity, reduced POD activity, increased MDA levels, deletion of both A20 and AN1 domains in LmSAP, deletion of gene OSW2, improved activity of encapsulated AI, no damage to thermostability of AI, no damage to pH tolerance of AI, no damage to resistance toward SDS of AI, no damage to resistance toward proteinase treatments of AI, expression of gene AI, production of l-ribulose from l-arabinose, 25% conversion rate of l-arabinose to l-ribulose, deletion of linker 2, increase in gene expression of weak and medium-strength terminators, decrease in gene expression of strong terminators, reduction of stem length, deletion of gene pmr1, consequence less susceptibility to growth inhibition from artemisinin and its derivatives, consequence elevated basal ROS levels, consequence no enhancement of ROS accumulation upon artemisinin exposure, consequence accumulation of excess manganese ions, gene up-regulation, increased metabolic reaction activity, gene down-regulation, decreased metabolic reaction activity, introduction of heterologous genes, overexpression of gene STB5, increased free fatty acid production, wide-ranging effects on the transcriptome, reduced flux through the oxidative PPP, induction of target genes GND1/2, TAL1, ALD6, YEF1, increase of DHN-melanin production, deletion of gene mpkA, normalization of DHN-melanin production, deletion of gene gpaA, deletion of gene gprM, deletion of gene URA3, successful targeted disruption, deletion of gene CAR2, editing efficiencies reached roughly 50%, deletion of gene tptA, decreased resistance to iron chelator BPS, severe growth defects, decreased expression of hapX, overexpression of gene hapX, rescued growth defect, increased siderophore production, attenuation of virulence in murine model, deletion of gene YER161C (SPT2), increased alkali tolerance, promotes xylose growth and metabolism, decouples growth from metabolism to enable robust fermentation without division, 87% efficiency, 60% efficiency, 23% efficiency, deletion of eight genes, 30-fold increase in free fatty acid production, deletion of gene YPL062W, upregulation of pyruvate dehydrogenase bypass, upregulation of mevalonate pathway, upregulation of heterologous expression of GAL promoter-regulated genes, upregulation of energy metabolism, upregulation of membrane composition synthesis, increased production of carotenoids, increased production of all major terpenoid classes, increased expression of ALD6 negatively correlated to terpenoid productivity, deletion of gene TRX1, reduced growth, impaired sugar consumption, reduction in ribosomal proteins, reduction in translation elongation factors, reduction in glycolysis enzymes, reduction in amino acid biosynthesis enzymes, increase in proteogenic amino acids, increase in phospholipids, increase in sphingolipids, higher fatty acid desaturase Ole1p content, low glycolytic activity, downregulation of all hexokinases, early aggregation of pyruvate kinase Cdc19p, deletion of gene TRX2, deletion of gene VPS21, consequence severe defect in the AP-3 pathway, deletion of gene YPT7, consequence defect in the CPY pathway, activation of gene YPT7, consequence mediates vacuolar fusion, consequence cargos trafficked via the VPS or endocytic pathway accumulate beside nucleus, consequence cargo trafficked via the AP-3 pathway disperse in the cytosol, reduced thiol release, mutation of gene IRC7, reduced cysteine-S-conjugate β-lyase activity, variation of gene IRC7, modulated formation of volatile sulfur compounds, deletion of gene HapB, tagging of gene HapB, deletion of gene rtfA, consequence 9, consequence 10, deletion of gene ptn1, reduction in mating efficiency, reduction in virulence, reduction in tumor formation, reduction in mature teliospores, reduction in spore germination, consequence increase in ethanol, consequence reduction in glycerol, deletion of gene ssk1, consequence improved ethanol yields, deletion of gene smp1, deletion of gene hot1, consequence tested but no osmosensitivity, consequence decrease in glycerol yield, consequence decrease in acetic acid yields, overexpression of key genes associated with fatty acid synthesis, increased lycopene accumulation, overexpression of OLE1, modulation of TAG fatty acyl composition, deletion of FLD1, regulation of lipid-droplet size, deletion of gene products functioning in gene expression, deviation from gamma distribution of cell size, deletion of gene products functioning in transcription or transcription-linked processes, altered activity substitution in Rpo21p/Rpb1, increased cell size, extreme altered activity Pol II substitutions, deviation from expected gamma distribution of cell size, deletion of gene PFK-2, reduced glycolytic flux, eliminated pathway to produce fructose-2,6-bisphosphate, expression of gene MTH1-ΔT, lowered glucose uptake rate, down-regulation of gene PFK-1, reduced glucose metabolism, down-regulation of gene Pyk1, combined strategies, improved N-acetylglucosamine production, deletion of gene STL1, decreased growth in high salt concentrations, disturbed intracellular pH homeostasis, expression of gene STL1, improved growth under osmotic stress, expression of gene SkSTL1 in Saccharomyces cerevisiae, complemented osmosensitivity of hog1Δ stl1Δ mutant, deletion of gene FET3, sensitivity to EGCG, deletion of gene FTR1, presence of EGCG, induction of Cu(I, surplus of Cu(II) and EGCG, synergistic deleterious effect on wild type cells, deletion of gene aro7, consequence increased mandelic acid production, expression of gene hmaS, consequence production of hydroxymandelic acid, expression of gene pheAfbr, expression of gene aro7 and pha2, consequence compartmentalization of mandelic acid biosynthesis pathway, deletion of gene FLO5, re-organization of the cell wall, finding different adhesion ways, expression of gene FLO5, up-regulation of genes involved in stress response, higher alcohol production, improved alcoholic fermentation, deletion of gene CDC55, abolished high fermentation performance, enhanced TORC1 signaling, increased initial fermentation rate, elevated PP2A^B55δ activity, high fermentation performance, genome modification, integration of gene TAL, biosynthesis of resveratrol, integration of gene 4CL, integration of gene STS, fusion expression of gene TAL, increased resveratrol production, fusion expression of gene 4CL, fusion expression of gene STS, integration of gene HSA, biosynthesis of human serum albumin, integration of gene cadA, biosynthesis of cadaverine, deletion of gene GcnE, abrogated conidiation, impeded vegetative growth of hyphae, increase of glutamine synthetase activity, deletion of gene gbpA, affected conidiogenesis, deletion of gene glnA, affected glutamine biosynthesis, deletion of gene FZO1, reduces mitophagy, activates retrograde response, activation of retrograde response, accumulates Atg32-Atg11-Dnm1 complexes, accumulation of Atg32-Atg11-Dnm1 complexes, engages mitophagy receptor Atg32, deletion of gene YALI0E25135g, decrease in resistance to hyperosmotic stress, negative effect on erythritol production, morphological defects, resistance to Congo red, resistance to calcofluor white, Deletion of gene aro8, decrease in methionol production by 44% in BY4743, decrease in methionol production by 92% in F15, Deletion of gene aro9, increase in methionol production by 46% in F15, Deletion of gene aro10, no effect on methionol production, Deletion of genes aro8 and aro9, increase in methionol production in F15 compared to single Δaro8, Deletion of genes aro8, aro9, and aro10, increase in methionol production in F15 compared to single Δaro8, improved ethanol yield by up to 74.7%, deletion of gene IAH1, consequence 1: decrease in esterase activity, consequence 2: increase in relative abundance of acetate and ethyl esters, deletion of gene TIP1, consequence 1: increase in ethyl esters, deletion of gene IAH1 and TIP1, consequence 1: aroma profile comparable to wild type levels, deletion of gene Abp1p, consequence resistance to wBm0076 expression, expression of gene wBm0076, consequence disruption of actin cytoskeleton dynamics, consequence colocalization with yeast cortical actin patches, expression of gene wBm0152, consequence disruption of endosome:vacuole cargo trafficking, deletion of 10 amino acids in HPV 33 L1 protein, soluble expression, deletion of 15 amino acids in HPV 52 L1 protein, deletion of 10 amino acids in HPV 58 L1 protein, deletion of gene YNL010W, consequence 1: inability to hydrolyze sorbitol-6-phosphate, consequence 2: inability to hydrolyze ribitol-5-phosphate, consequence 3: inability to hydrolyze (d, expression of gene PYP1, consequence 1: increased yeast growth rate, consequence 2: maintenance of Pgi activity in presence of environmental sugar alcohols, consequence 3: prevention of inhibition of glycolysis by sugar alcohol phosphates, deletion of gene YER182W, improvement of PDV biosynthesis, deletion of gene FLO11, overexpression of gene LsFAD2, accumulation of linoleic acid, reduction in oleic acid levels, overexpression of gene LsFAD3, production of α-linolenic acid, deletion of gene LsFAD2, accumulation of oleic acid, deletion of gene LsFAD3, deletion of gene spermidine synthase, severe growth defect, hypersensitivity to furans, hypersensitivity to acetic acid, overexpression of gene spermidine synthase, increased tolerance to furans, deletion of gene CSS1, increase in astaxanthin yield, expression of gene CrtZ from Agrobacterium aurantiacum, increase in astaxanthin ratio, ARTP mutagenesis, lower ethyl acetate production, overexpression of gene EAT1, increased ethyl acetate production, mutation eat1K179fs, mutation snf8E148*, decreased ethyl acetate production, presence of eat1K179fs and ATF1, higher acetate esters production, introduction of snf8E148* with ATF1, decreased ethyl acetate levels, snf8E148* mutation, affects other aroma compounds, affects fermentation, deletion of gene Mrx15, consequence 1: disruption of cotranslational protein insertion, consequence 2: altered interaction with the large ribosomal subunit, overexpression of gene Mrx15, consequence 1: enhanced membrane protein biogenesis, consequence 2: increased contact with mitochondrial translation products, overexpression of gene ERG pathway genes, increased doubling time, respiratory deficiencies on glycerol media, cell wall insufficiencies on Congo red media, disrupted ion homeostasis under iron starvation conditions, disrupted ion homeostasis under calcium starvation conditions, deletion of gene ERG pathway genes, altered susceptibility to fluconazole, altered susceptibility to fenpropimorph, altered susceptibility to lovastatin, altered susceptibility to nystatin, altered susceptibility to amphotericin B, altered susceptibility to terbinafine, gene transformation cassettes, increased transformation efficiency, TDH1p natMX, five times greater efficiency, transformation cassettes encoding fluorescent proteins, discrimination between subpopulations of transformed B. bruxellensis cells, deletion of gene ADH3, deletion of gene ADH4, deletion of gene ADH5, deletion of gene ADH6, overexpression of gene ERG13, enhanced metabolic flux through MVA pathway, enhancement of metabolic flux through MVA pathway, overexpression of lupeol synthase, increased yield of pentacyclic triterpenes, production of lupeol, overexpression of ATF1, deletion of BAT2, increased AATase activity, PGK1p library use, precise control of ATF1 expression, regulation of acetate esters production, expression of pATOM36, rescues growth defects in yeast cells lacking Mim1 and/or Mim2, rescues mitochondrial biogenesis defects in yeast cells lacking Mim1 and/or Mim2, rescues morphology defects in yeast cells lacking Mim1 and/or Mim2, co-expression of Mim1 and Mim2, restores assembly defects of MOM proteins in trypanosomes ablated for pATOM36, restores insertion defects of MOM proteins in trypanosomes ablated for pATOM36, overexpression of gene ACC1**, consequence 129 mg∙gCDW^-1 of TAGs, deletion of gene TGL3, consequence 218 mg∙gCDW^-1 of TAGs, deletion of gene TGL4, deletion of gene TGL5, deletion of gene ARE1, consequence no change in TAGs levels, deletion of gene GUT2, deletion of gene PXA1, consequence 254 mg∙gCDW^-1 of TAGs, expression of gene MAL73L, enhanced maltose fermentation, increased growth rate on maltose, deletion of gene MAL73, decreased maltose fermentation ability, deletion of gene Mig1p, improved tolerance towards acetic acid, improved tolerance towards formic acid, improved tolerance towards levulinic acid, reduced acetic acid production, reduced glycerol production, increased acetate consumption, increased formate consumption, deletion of gene YEL013W, increase in carotenoid yield, deletion of gene SGS1, deletion of smORFs, deletion of gene involved in ergosterol biosynthesis, increased MEHP sensitivity, deletion of gene involved in tryptophan biosynthesis, deletion of gene regulating ion homeostasis, deletion of gene involved in protein sorting, deletion of gene involved in cell wall integrity, deletion of gene involved in intracellular pH homeostasis, deletion of gene ALE1, reduction in efficiency of palmitoleic acid introduction to HHPC, reduction in molecular species containing acyl residues in HHPC, deletion of gene ZRT2, reduced kidney fungal burden, growth inhibition by neutrophil extracellular traps, overexpression of gene ZRT1, supports growth at pH 7 and above, deletion of gene Zrc1, affects zinc tolerance, impacts virulence, impacts liver colonization, deletion of gene RTS1, no synthetic rescue of temperature-sensitivity in ORC2 mutants, additive phenotype indicating separate biological processes from ORC2, increased temperature-sensitivity in ts-replication factor strains, deletion of gene(s) in BGC, increased pathway productivity, production of new bottromycin-related metabolites, deletion of gene fragment, change in gene structure, mutation of gene fragment, recombination of gene fragment, deletion of RPN4, impaired model dough fermentation, deletion of PDR1, deletion of PDR3, overexpression of genes involved in oxidative phosphorylation, decrease in glycolytic capacity, low mRNA levels of proteasome complex component genes, accumulation of ubiquitinated proteins, deletion of gene OSH6, growth defects on n-alkanes of 10-16 carbons, production of functional cytochrome P450 not observed, transcription of ALK1 observed, translation of ALK1 transcript observed, phospholipid composition altered, ratio of phosphatidylserine reduced, deletion of gene Tr69957, affected fungal growth, affected biomass accumulation, affected sugar uptake, affected gene expression of cellobiohydrolases, affected gene expression of β-glucosidases, affected gene expression of xylanases, expression of gene SOD2, consequence high tolerance to oxidative stress, expression of gene CTT1, removal of gene Msn2, consequence significant decrease in CTT1-expression, removal of gene Yap1, abundance of Msn2, consequence high expression of ROS-degrading enzymes, deletion of gene hst3, active glycolysis, deletion of gene hst4, deletion of gene sir2, deletion of gene fbp1, high glucose consumption, active ethanol productivity, accumulation of glycolytic metabolites, accumulation of secondary metabolites, synthesis of nucleotides through PP pathway, consequence 20-fold higher squalene accumulation, consequence enhanced lipid biosynthesis, consequence reduced lipid β-oxidation, deletion of gene PXA2, simultaneous overexpression of gene tHMG1 and DGA1, consequence over 250-fold higher squalene accumulation, deletion of genes involved in transmembrane and membrane transport, sensitivity to ZnONPs, deletion of genes involved in cellular ion homeostasis, deletion of genes involved in cell wall organization or biogenesis, deletion of genes involved in transcription and RNA processing, sensitivity to AgNPs, deletion of genes involved in cellular respiration, deletion of genes involved in endocytosis and vesicular transport, deletion of STR3, divert accumulated thiols to GSH biosynthesis, overexpression of CYS3, overexpression of CYS4, up-regulation of PDI, reduce unfolded protein response, accumulation of homocysteine, promote cysteine synthesis, accumulation of cystathionine, higher concentration of glutamic acid, contribute to GSH synthesis, higher concentration of glycine, higher concentration of cysteine, deletion of gene GAP1, reduction in extracellular urea, overexpression of gene Gln3p<sub>1-653</sub>, deletion of gene ALP1, overexpression of tdcB, increase in 1-propanol production, enhancement of threonine biosynthesis from aspartate by thrA, thrB and thrC, increase in 1-propanol production, deletion of GLY1 gene, construction of an artificial 2 KB biosynthetic pathway from pyruvate via citramalate (cimA), deletion of gene involved in signaling cascades, affects lipid accumulation, deletion of gene involved in gene expression, deletion of gene involved in protein modification, deletion of gene involved in vesicular trafficking, deletion of gene involved in amino acid synthesis, deletion of gene involved in tRNA modification, deletion of Nup mutants, reduction in Ty1 mobility, impairment in Ty1 insertion upstream of tRNA genes, deletion of components of the NPC nuclear basket, mis-targeting of Ty1 elements to the ends of chromosomes, deletion of gene MFA2, reduction in fermentation duration by 55%, deletion of gene UBC13, reduction in fermentation duration, deletion of gene MMS2, deletion of gene UBP7, deletion of gene UBI4, deletion of gene BRO1, deletion of gene EAR1, deletion of gene MRP17, deletion of gene MVB12, deletion of gene AAT2, overexpression of gene MIG1, consequence enhanced growth, consequence decreased xylitol accumulation, consequence increased biomass accumulation, consequence improved activities of XR, consequence improved activities of XDH, consequence improved activities of XK, deletion of fatty acid activation genes, redirected fatty acid metabolism, reduced feedback regulation, deletion of main storage lipid formation genes, constrained lipid metabolic network, deletion of phosphatidate phosphatase genes, fatty acid flux directed toward phospholipids, increases of phospholipids by up to 2.8-fold, increases of FFAs by up to 40-fold, deletion of phospholipase genes PLB1 and PLB2, 46% decrease in FFA levels, 105% increase in phospholipid levels, deletion of gene NYn43, faster fermentation rate, deletion of gene MRI1, reduction in DMTS-P1, suppression of DMTS formation, no change in sake brewing properties, deletion of gene TcGS, increased NH4+ toxicity, overexpression of gene TcGS, reduced NH4+ toxicity, impaired escape from parasitophorous vacuole, enhanced escape from parasitophorous vacuole, inhibition of GS activity by Methionine sulfoximine, reduced parasite survival under excess ammonium conditions, consequence n-propanol concentration decreased, consequence active amyl alcohol concentration decreased, consequence 2-phenylethanol concentration decreased, consequence isobutanol concentration increased, consequence isoamyl alcohol concentration increased, deletion of gene LEU1, consequence n-propanol concentration increased, consequence isoamyl alcohol concentration decreased, deletion of gene LEU2, deletion of gene Ndi1p, consequence 1: abolishment of sporulation, consequence 2: failure of premeiotic DNA replication, consequence 3: insufficient IME1 expression, overexpression of gene RIM101, consequence 1: inhibition of Smp1p, consequence 2: promotion of IME1 expression, consequence 3: promotion of meiosis initiation, deletion of gene recD, consequence 1: drastically decreased the length of resection, consequence 2: allowed for recombination with short ectopic plasmid homologies, consequence 3: significantly increased the efficiency of horizontal gene transfer between strains, deletion of shear stress genes, consequence 1: cellular redox responses, consequence 2: apoptosis, shortest fermentation, up-regulation of SLT2, up-regulation of HOG1, up-regulation of CHS3, alteration in transcript abundance of GLN1, alteration in transcript abundance of GLT1, alteration in transcript abundance of GDH1, alteration in transcript abundance of GDH2, deletion of gene FUS3, consequence increased yeast lifespan, deletion of gene KSS1, deletion of gene MAPK and TOR1, consequence greater lifespan increase than tor1Δ alone, endogenous expression of non-activatable form of Fus3, consequence similar effects as deletion of MAPK and TOR1, deletion of gene FUS3 under nitrogen starvation, consequence decreased CLS, deletion of gene Hxt, consequence increased D-xylose affinity, expression of gene Hxt2C505P, mutation of gene Hxt2, deletion of secondary metabolite gene clusters, improved mithramycin A production, overexpression of gene, improved endoglucanase display, increased production of medium-chain α, ω-DCAs, enabled production of C14 α, ω-DCAs, enabled production of C16 α, ω-DCAs, no significant impact on production of α, ω-dicarboxylic acids, overexpression of thioesterase genes, deletion of gene TES1, regulation of gene OLE1, regulation of gene HMG1, regulation of gene ACS1, deletion of 25 genes encoding transport proteins, lower specific growth rate, lower biomass-specific lactate production rate, expression of Lactobacillus casei lactate dehydrogenase (LcLDH), increased lactate production, deletion of 25 transporter genes, unaffected lactate export rates at fixed specific growth rate, heterologous overexpression of mcr gene, 3-HP production, increase in 3-HP titer, increase in promoter strength and mcr gene copy number, improvement in 3-HP titer, overexpression of mcr gene, decrease in acetyl-CoA pool size, rapid degradation of 3-HP in stationary phase, accumulation of intracellular 3-hydroxypropionyl-CoA, deletion of META1_4251 gene, reduction in 3-HP degradation rate, deletion of endogenous gene products, repair of existing mutations, expression of multiple sgRNAs, multiplexing with a single sgRNA, installation of artificial Cas9 target sequences, reduction/elimination in possible off-target effects, deletion of gene FgPLD1, consequence hyphal growth affected, consequence conidiation affected, consequence perithecium formation affected, consequence reduced deoxynivalenol production, consequence reduced virulence in flowering wheat heads, consequence reduced virulence in corn silks, consequence D-xylose dehydrogenase activity, consequence D-xylonate dehydratase activity, expression of gene yagE, consequence glycolaldehyde production, expression of gene yjhH, expression of gene aldA, consequence glycolic acid production, expression of gene ldhL, consequence lactic acid production, expression of gene xylB and xylD, consequence 29 mg/L 2K3DXA production, consequence 9 g/L D-xylonic acid accumulation, FRA2 deletion, consequence improved dehydratase activity, targeting xylD expression to mitochondria, expression of gene aldA and ldhL, consequence 1 g/L glycolic acid accumulation, endogenous aldo-keto reductase activity, consequence ethylene glycol production, deletion of gene BAT1, increase in isobutanol production by 14.2-fold, additional 2.1-fold increase in isobutanol production when biosynthetic pathway is compartmentalized in mitochondria, addition of valine, inhibition of isobutanol production, removal of valine, increase in isobutanol production, interaction between valine and Ilv6p, affects isobutanol production, overexpression of gene BAT1, influences isobutanol production, overexpression of gene BAT2, boost in 2-methyl-1-butanol production, deletion of gene HUR1-A, reduced efficiency of NHEJ in overhang end plasmid repair assays, reduced efficiency of NHEJ in blunt end plasmid repair assays, reduced efficiency of chromosomal repair, reduced efficiency of HR repair, deletion of gene ECM22, mating defects, deletion of gene UPC2, deletion of gene PRM4, reduced mating efficiency, overexpression of gene STE12, increased PRM1 levels, deletion of gene STE12, lowered PRM1 expression, deletion of gene ECM22 and UPC2, abolished PRM1 expression, mutation of Ste12-binding sites in PRM1 promoter, ectopic expression of Saccharomyces cerevisiae MATα2, suppresses sporulation, inhibits riboflavin overproduction, downregulates AgSOK2, deletion of AgSOK2, reduces riboflavin production, blocks sporulation, AgSok2, promotes sporulation, does not control AgIME1 transcription, targets AgIME2, targets AgNDT80, deletion of CAN1 gene, effectiveness in Saccharomyces cerevisiae, introduction of single base mutation A128C into rpsL gene, effectiveness in Escherichia coli, deletion of gene using RMGE-bceSIIM, higher counter-selection efficiency in Escherichia coli, replacement of gene using RMGE-bceSIIM, deletion of gene using RMGE-mcrA, higher counter-selection efficiency in Bacillus subtilis, replacement of gene using RMGE-mcrA, deletion of gene ctpxa1, weakened expression of ctpxa1, increased dodecanedioic acid production, gene expression of biomass hydrolyzing enzymes, efficient biomass utilization, gene expression of transporters, enhanced sugar input, improved product output, deletion of gene PEX22, increased malate production, DMS sensitivity, deletion of genes encoding peroxins, mislocalization of Mdh3p, deletion of gene FgOXP1, decreased virulence on wheat, decreased deoxynivalenol production, deletion of gene FgOXP2, defective conidiation, defective sexual reproduction, deletion of gene FgOXP1 and FgOXP2, severe defects in conidiation, severe defects in DON production, severe defects in virulence on plants, deletion of gene PYK1, deletion of gene PFK2, expression of gene HuPYK1, expression of gene HuPFK1, expression of gene HuPFK2, expression of G-protein-coupled receptor, detection of serotonin, overproduction of serotonin, increased production of hydroxystrictosidine, deletion of gene LIG4, enhanced probability of obtaining correctly modified transformants, replacement of ORF in gene ADE2 by SAT1 selective marker, adenine auxotrophy, introduction of premature stop codon in gene ADE2, replacement of ORF in gene MET15 by SAT1 selective marker, methionine auxotrophy, introduction of premature stop codon in gene MET15, insertion of triple HA-tag sequence in gene SOK2, verified in western blot, increased CRISPR-Cas9 mediated targeting efficiency, Deletion of gene lst4, reduced H2S production, Deletion of gene lst7, Deletion of gene agp1, Deletion of gene gnp1, Deletion of gene mup1, Deletion of gene stp1, Deletion of gene dal81, deletion of gene nuo1, down-regulation of nuclear-encoded CI subunit proteins, decreased phospholipid synthesis, decreased ergosterol synthesis, decreased cell wall mannan synthesis, increased CIV proteins, increased AOX2, participation in nucleotide synthesis, participation in ribosomal biogenesis, deletion of gene nuo2, involvement in vesicle trafficking, deletion of gene goa1, regulation of membrane transporter proteins, regulation of ROS removal, regulation of substrates trafficking between peroxisomes and mitochondria, nuo2, goa1, avirulence phenotype in murine model of invasive disease, expression of gene Lm. ldhA, increased d-LA production, reduced d-LA degradation, mutation of gene SUR1 I245S, increased LA tolerance, loss-of-function mutation of gene ERF2, deletion of gene PDC1, additional integration of gene Lm. ldhA, deletion of gene nox-1, failure to induce hepatic cell death, overexpression of gene TG2, enhancement of cellular TG activity, induction of hepatic cell death, production of metabolite ROS, enhancement of nuclear TG activity, deletion of gene TG2, inhibition of cell death, administration of product N-acetyl cysteine, inhibition of enhanced TG activity, administration of product corn peptide fraction, inhibition of hepatic ROS and TG activities, deletion of gene bgl2, consequence 15% reduction in β-(1,6, deletion of gene gas1, consequence 70% reduction in β-(1,6, deletion of gene bgl2, consequence drastically sick phenotype, expression of gene ScGas1p, consequence introduction of β-(1,6, expression of gene Gel4p, consequence dual β-(1,3, presence of CBM43, consequence required for dual β-(1,3, deletion of gene Ylsdh5, accumulation of by-product acetate, deletion of gene Ylach, elimination of acetic acid formation, improvement of succinic acid production, improvement of cell growth, overexpression of gene ScPCK, improvement of succinic acid titer, overexpression of gene YlSCS2, improved growth under acetic acid stress, improved growth under ethanol stress, improved growth under hydrogen peroxide stress, increased ethanol production by 14.7%, decreased intracellular acetic acid, decreased ROS accumulation, decreased plasma membrane permeability, deletion of gene DGA1, deletion of gene HFD1, overexpression of gene FAR, overexpression of mutant acetyl-CoA carboxylase, limiting gene GDH1 expression, overexpression of gene OLE1, overexpression of gene BIO1, growth in biotin-free medium, deletion of gene PDR12, deletion of gene yro2, consequence reduced DIV secretion, deletion of gene mrh1, deletion of gene hxt5, consequence increased isobutanol production, deletion of 18 genes, consequence slightly increased intermediates secretion, consequence decreased isobutanol production, deletion of gene Pub1, decreased efficiency of nonsense readthrough at a majority but not all tetranucleotide stop signals, deletion of gene Dhh1, decrease in expression of Ste12, severe mating defects, mutation in ATPase A motif of Dhh1, mutation in ATPase B motif of Dhh1, C-terminus deletion of Dhh1, reduced Ste12 protein expression, dispensable for growth at 37°C, role in regulating Ste12 protein expression, role in mating processes, P-body accumulation induced by α-mating factor, requires ATPase domain of Dhh1, requires RNA-binding domain of Dhh1, requires Q/P-rich C-terminal domains of Dhh1, deletion of gene YAP1, narrowed response to LoaOOH, decreased expression of Tsa1p, increased expression of Ahp1p, increased expression of Hsp31p, deletion of gene OYE3, altered expression of Sod1p, altered expression of Hsp60p, altered expression of Ssa1p, altered expression of Sse1p, altered expression of Pst2p, altered expression of Tpm1p, deletion of gene OYE2, removal of gene Mep1, growth dependent on extracellular NH3 concentrations, removal of gene Mep2, removal of gene Mep3, higher intracellular NHX concentration, up-regulation of vacuolar proteases, increased stress response genes, higher turnover rates of biomass components, gene deletion and replacement, increased flexibility for refactoring pathways, deletion of gene Crz1, intermediate phenotypes between wild-type and calcineurin mutants, deletion of calcineurin docking domain, inability of Crz1 to translocate into the nucleus under thermal stress, deletion of gene cna1, down-regulation of 102 genes regulated in a calcineurin-Crz1-dependent manner at 37°C, deletion of gene crz1, regulation of 59 genes independently of calcineurin, activation of calcineurin, increased Crz1 binding to target gene promoters upon thermal stress, deletion of gene calcineurin, impact on mRNA expression/stability through factors beyond Crz1, deletion of gene hexokinase, consequence increased d-xylose utilization in high d-glucose, mutation in gene CYC8, consequence altered expression of 95 genes, consequence upregulation of HXT10, consequence upregulation of HXT13, consequence upregulation of HXT15, consequence upregulation of HXT16, consequence increased d-xylose uptake in presence of d-glucose, consequence elevated Vmax for d-glucose transport, consequence elevated Vmax for d-xylose transport, identification of genes involved in biological processes, probing genetic networks, identification of multiple naturally occurring allelic variants, contribution to polygenic phenotypes, limitation on use of modern genetic techniques, production of volatile aroma compounds, deletion of gene Hmo1p, reduced repair efficiency, reduced repair accuracy, inhibition of DNA end resection, restored repair efficiency, deletion of gene Ku, inhibition of resection, excessive DNA resection, inhibitory for A-EJ, deletion of gene in O1A O-antigen gene cluster, consequence: altered substrate preferences of Wzx flippases, replacement of gene in O1A O-antigen gene cluster, increase in FFA levels, dynamic expression of gene FAA1, regulation of FFA and acyl-CoA pools, expression of gene MmCAR, production of fatty alcohols from FFAs, expression of gene Adh5, expression of fatty acyl-CoA reductase from Marinobacter aquaeolei, production of fatty alcohols from acyl-CoA, expression of gene FAA1 under HXT1 promoter, increased accumulation of fatty alcohols, decrease in FFA levels, deletion of gene YjgB, increase in protoilludene production, overexpression of gene IDI, enhancement of protoilludene production to 67.4mg/L, overexpression of gene Fpr, increase in protoilludene production to 318.8mg/L, overexpression of gene FldA, introduction of gene tPos5p, increase in protoilludene production to 512.7mg/L, deletion of gene spt3, increased expression of Fps1, deletion of gene srb5, deletion of gene gcn5, expression of gene GCN4, deletion of uORF1, deletion of uORF1 and uORF2, no specific consequences related to expression/deletion, overexpression of gene Saccharomyces cerevisiae, consequence increased protein phosphorylation, deletion of gene Saccharomyces cerevisiae, consequence decreased protein phosphorylation, overexpression of gene Candida albicans, deletion of gene Candida albicans, deletion of gene CAT8, diminished growth on glycerol, diminished growth on ethanol, diminished respiration on xylose, activated transcription of gene XYL3, activated transcription of gene DAS1, activated transcription of gene RPE1, increase in activity of enzymes involved in xylose metabolism, increase in ethanol production from xylose, overexpression of gene CAT8, drop in ethanol production from xylose, increased translation of proteins that facilitate cellular response to stress, deletion of gene UPF1, deletion of gene UPF2, deletion of gene UPF3, atf1 overexpression, higher production of acetate esters, lower production of higher alcohols, iah1 deletion, increased production of isobutyl acetate, increased production of isoamyl acetate, bat2 deletion, decline in production of higher alcohols, reduced hydrolysis of acetate esters, atf1 overexpression and bat2 deletion interaction, reduction in diacetyl content, overexpression of gene ILV5, deletion of gene SCH9, deregulated NCE103 expression on RNA level, deregulated NCE103 expression on protein level, expression of gene NCE103, acceleration of CO2 conversion to bicarbonate, activation of gene ScCst6, regulation of NCE103 expression, activation of gene ScSch9, control of ScCst6/Rca1 activity, deletion of gene SCH9 homologue, impaired CO2-dependent regulation of NCE103 expression, deletion of gene mel1, elimination of α-galactosidase reactions, elimination of melibiose degradation, elimination of galactose by-product formation, integration of gene fsy1, contribution to fructose transport, integration of gene ffzi1, overexpression of gene gcr1, enhancement of fructose by-product biopurification rate, deletion of gene MAL11, no growth on sucrose, deletion of gene MAL21, deletion of gene MAL31, deletion of gene MPH2, deletion of gene MPH3, deletion of gene SUC2, growth on sucrose, deletion of gene MAL12, deletion of gene MAL22, deletion of gene MAL32, upregulation of gene IMA1, increased sucrose-hydrolysis activity, upregulation of gene IMA5, no sucrose hydrolysis, deletion of gene IMA2, deletion of gene IMA3, deletion of gene IMA4, deletion of gene IMA5, increased SAM accumulation, mutation of gene, deletion of gene hrdA, increased levels of mutant MsdS in intracellular fraction, overexpression of mutant MsdS, secreted to medium, underwent hyperglycosylation, induced UPR, deletion of gene α-amylase, alleviated UPR induction by mutant MsdS overexpression, SCH9 deletion, shortens chronological life span, increases glycerol production, upregulates amino acid biosynthesis pathways, increases glycerol-3-P dehydrogenase Gpd1p protein level, downregulates proteinogenic amino acids, increases lipids such as ergosterol, GCN4 overexpression, downregulates glycolysis, deletion of gene VPS30, consequence 1: increased production of recombinant antibody 2-fold, consequence 2: increased expression of endogenous yeast acid phosphatase 1.7-fold, deletion of gene TAR1, consequence 1: mildly affected secretion levels, consequence 2: increased expression of endogenous yeast acid phosphatase 2.4-fold, deletion of gene glo1, consequence significant production of H2S, consequence inefficient sulfate assimilation, consequence normal growth on cysteine, overexpression of gene glo1, consequence dominant negative phenotype, consequence reduced expression of Met17p, deletion of NC domain, abrogates RNA-based inhibition of Gag-membrane binding, deletion of gene Set2, transcription initiation of antisense RNAs, production of Set2-repressed antisense transcripts (SRATs, addition of H3K36 methyl mark by Set2, suppression of SRATs, deletion of gene VPS10, consequence Vps10p is the dominant sorting receptor for PrA Golgi-to-vacuolar transportation, overexpression of gene VPS10, consequence PrA activity in the broth lowered by 76.96%, consequence increased growth rate, consequence higher biomass, consequence faster fermentation speed, consequence thorough wort fermentation, consequence 1.41-fold higher ethanol production, deletion of gene SEC5, consequence PrA activity in the broth lowered by 32.39%, deletion of 28 amino acids in the hydrophilic loop connecting TM8 and TM9 of Acr3, no effect on Acr3 folding, no effect on Acr3 trafficking, no effect on substrate binding, no effect on transport activity, deletion of transcription factor, improved ethanol production, increased xylose import rates, improved xylose integration into central carbon metabolism, deletion of gene UBP3, exacerbated karyotype-specific phenotypes, increased global stresses of aneuploid cells, enhanced oxidative stress, increased proteotoxic stress, impaired proteasome function, deletion of gene USP10, detrimental to the fitness of human cells upon chromosome missegregation, autophagy inhibition, deletion of gene Sef1, consequence full growth under iron limitation conditions, overexpression of gene Aft1, consequence activation under iron starvation conditions, overexpression of gene Cth2, consequence degradation of mRNAs encoding iron-requiring enzymes, deletion of GPD2, no effect on acetate consumption, overexpression of AADH, overexpression of ACS, expression of XR and XDH, redox imbalance, coupling of NADH-dependent acetate reduction pathway, efficient xylose fermentation, in situ detoxification of acetate, deletion of gene PRE3, impaired growth on glucose, deletion of gene IDI1, deletion of gene MRPL37, deletion of gene NOC3, deletion of gene NOP56, deletion of gene RPL17, deletion of gene OCH1, severe defects in cell growth, defects in morphogenesis, response to environmental challenges, overexpression of gene RHO1, partial recovery of growth defect, enhanced glucan layer, recovered mannoprotein layer, overexpression of gene FKS2, complementation of cell wall structure, improved temperature tolerance, improved osmotic potential, improved drug sensitivity, increased N-glycan site occupancy, increased amount of secreted glycoproteins, deletion of introns from RPS25A, RPS26A, RPS26B, and RPS24A promoters, decreased luciferase expression, expression of intron-containing promoters RPS25A, RPS26A, RPS26B, and RPS24A, higher expression than TDH3 promoter, insertion of RPS25A intron into TDH3 promoter, 50-fold higher expression than TDH3p alone, insertion of RPS25A intron into GAL10 promoter, enhanced expression, insertion of RPS25A intron into CUP1 promoter, RPS25A intron in chimeric promoter, enhances expression of some recombinant proteins, deletion of gene involved in mitochondrial functions, resistance to valinomycin, resistance to nigericin, deletion of gene involved in ribosome biogenesis, hypersensitivity to valinomycin, hypersensitivity to nigericin, deletion of gene involved in telomere maintenance, deletion of gene involved in vacuolar biogenesis, deletion of gene involved in stress response, improved LA tolerance, improved LA production, improved glucose uptake, derepression of glucose-repressed genes, increased respiratory pathway activity, more efficient ATP generation, more efficient NAD+ generation, improved D-LA production, deletion of gene KU70, enhanced double-crossover homologous recombination rate, increased precise homologous recombination rate, deletion of 11 target genes encoding alcohol dehydrogenase and alcohol oxidase, facilitated by enhanced gene deletion efficiency, expression of Cre recombinase, removal of selection marker gene, deletion of gene PDE2, altered glycerol production, altered pyruvate derived metabolites production, deletion of gene CYP61, decreased ergosterol biosynthesis, increased astaxanthin production, expression/deletion of gene xxx, deletion of gene Tom1, consequence 1: reduced ubiquitination of multiple RPs, consequence 2: exceptional accumulation of detergent-insoluble proteins including multiple RPs, consequence 3: hypersensitivity to imbalances in production of RPs and rRNA, overproduction of gene Rpl26, consequence 1: fails to assemble into ribosomes, consequence 2: degraded in the nucleus/nucleolus by a ubiquitin-proteasome system, deletion of gene GLC3, improvement of SAM accumulation, deletion of gene SPE2, no obvious effect on SAM production, deletion of gene ERG6, 12.47g/L SAM production, 81.0% increase in SAM production compared to parent strain, deletion of nuclease activity in CRISPR/Cas9, reduction in toxicity, expression of Target-AID complex, induction of specific point mutation at cytidines, use of uracil DNA glycosylase inhibitor, suppression of indel formation, improvement in mutagenesis efficiency, deletion of gene EXG1, enhancement of flavonoid glucoside production, over-expression of phosphoglucomutase, increase in uridine diphosphate glucose synthesis, over-expression of UTP-glucose-1-phosphate uridylyltransferase, expression of gene SbGT34, synthesis of flavonoid glucosides, deletion of gene SPR1, reduction in glucosidic bond hydrolysis, deletion of gene YIR007W, deletion of gene TORC1, consequence decreases inhibition of SNF1 by TORC1, deletion of gene PKA, consequence mitigates PKA pathway, no specific gene deletion, consequence coordinates unknown processes, consequence diminishes PKA inhibition on SNF1, overexpression of gene Rim15, consequence intensifies Rim15 activity, deletion of gene Sch9, consequence inhibits Sch9 activated by PKH1/2, deletion of gene PRO1, defect for ribophagy, sensitivity to various stresses, expression of inactive γ-Glutamyl kinase, re-routing carbon flux towards desired product, growth-product coupling, expression of gene pfldh, l-lactic acid formation from glucose, l-lactic acid formation from xylose, reduced ethanol formation, reduced xylitol formation, deletion of gene MoYcp4, consequence growth rate reduced, consequence conidial production increased, consequence appressorial turgor abnormal, consequence ability to infect rice and barley reduced, consequence pathogenicity decreased, deletion of gene BcMtg2, defect in spore production, defect in conidial germination, defect in sclerotial formation, increased sensitivity to environmental stresses, increased sensitivity to Congo red, decreased virulence on host plant tissues, minor glycerol response to osmosis stress, increase in ethyl caproate production, overexpression of gene FAS1, increase in acetyl-CoA content, increase in ethyl acetate content, overexpression of gene FAS2, increase in FAEEs content, high gene deletion efficiency, deletion of WHEP domain, reduced catalytic efficiency, increased thermal stability, decreased structural flexibility, elimination of glycerol production, overexpression of gene AlsS, increase in acetoin production, overexpression of gene AlsD, reduction in conversion of acetoin to 2,3-butanediol, overexpression of gene NoxE, relief of cofactor imbalance, deletion of gene end3, consequence internalization of AgNPs not observed, deletion of genes related to oxidative stress defense, consequence no change in toxicity pattern, deletion of genes related to cell wall/membrane integrity, deletion of gene Irc3, loss of mitochondrial genome integrity, accumulation of irregular branched molecules, overexpression of gene RecG, complementation of Irc3 deletion, deletion of gene F508, consequence functional expression defect of CFTR at apical plasma membrane, consequence degradation of misfolded CFTR at ER, deletion of gene F670, consequence folding and trafficking defects of Yor1, silencing of gene RPL12, consequence increased folding efficiency of ΔF508-CFTR, consequence increased conformational stability of ΔF508-CFTR, consequence augmented PM density and function of ΔF508-CFTR, combination of RPL12 knockdown and VX-809, consequence restoration of mutant function to ~50% of wild-type channel, improved extracellular activities of heterologous proteins, deletion of gene MNN9, up-regulated genes in the secretory pathway, affected cell wall integrity, redirection of metabolic fluxes, chromosomal integration of expression cassettes, expression of beta-carotene biosynthetic pathway, increased beta-carotene production, deletion of ALD6, increase in glycerol yield, enhancement of acetate consumption, expression of NAD(+)-dependent 6-phosphogluconate dehydrogenase, increase in cytosolic NADH formation, replacement of glycerol formation by acetate reduction pathway, increase in acetate consumption, combined expression of NAD(+)-dependent 6-phosphogluconate dehydrogenase and deletion of ALD6, increase in ethanol yield on glucose, deletion of gene ASC1, compromised translation of specific functional groups, cellular phenotypes consistent with gene-specific translation defects, depletion of eIF4G, mimics the translational defects of ASC1 mutants, deletion of gene MoYVH1, consequence 1: significant reductions in vegetative growth, consequence 2: conidial production reduction, consequence 3: virulence reduction, consequence 4: defects in cell-wall integrity, consequence 5: hyposensitivity to exogenous osmotic stress, consequence 6: defects in appressorium function, consequence 7: invasive hyphae growth defects, overexpression of gene MoPDEH, consequence 1: suppression of conidia formation defects, consequence 2: suppression of pathogenicity defects, overexpression of gene pbcA, consequence PD production, deletion of gene pbcA, consequence abolished PD production, overexpression of gene cps/ks, consequence ent-kaurene production, overexpression of gene gamma-terpinene synthase, consequence gamma-terpinene production, deletion of gene Sgs1, deletion of gene Mph1, overexpression of gene Rad51, deletion of gene metJ, increase in SAM availability, overexpression of gene metA, overexpression of gene cysE, overexpression of gene mtn, increase in SAM regeneration, overexpression of gene luxS, supplementation with methionine, increase in vanillate titers, deletion of gene JJJ1, shorter lag phase, increased specific ethanol production rate, higher long chain fatty acid content, higher trehalose content, higher catalase content, deletion of gene CgTRK1, consequence affects growth at low potassium concentrations, consequence reduces tolerance to toxic alkali-metal cations, consequence reduces tolerance to cationic drugs, consequence alters membrane potential, consequence alters intracellular pH, increased trehalose content, increased cell viability after freezing, deletion of gene NTH1, strengthened freezing tolerance, improved leavening ability after freezing-thaw stress, reduced conidial production, decreased sclerotial production, reduced production of aflatoxin B1, altered production of secondary metabolites, controls expression of veA, controls expression of laeA, deletion of gene horA, reduced coenzyme Q content, impaired response to oxidative stress, impaired response to reductive stress, delay in germination, accumulation of NADH, increased resistance against antifungal drugs, attenuated virulence in murine infection models, deletion of gene MNN1, elimination of interfering structures, expression of UDP-N-acetylglucosamine transporter from Kluyveromyces lactis, increase in complex-type glycan abundance, improvement in glycan pattern homogeneity, improvement in complex-type glycosylation efficiency, deletion of gene NIK1, insensitivity to dicarboximide and phenylpyrrole fungicides, increased sensitivity to hyperosmotic stress, increased tolerance to 6-methoxy-2-benzoxazolinone, reduced production of pycnidia, overexpression of gene ARO10, increased 2-PE production, overexpression of gene ADH1, deletion of gene aro8, deletion of gene aat2, increased ethanol yields, deletion of gene SWI5, overexpression of gene ACE2, reduced ethanol yields, overexpression of gene SWI5, deletion of gene CTS1, deletion of gene RPS4a, deletion of gene EGT2, deletion of gene DSE2, deletion of gene SCP160, overexpression of gene CTS1, overexpression of gene RPS4a, deletion of gene MCR1, decreased ethanol yields, deletion of gene HO, deletion of gene PEF1, delay in penetration, delay in invasive growth, expression of gene PEF1, peroxisomal fission, formation of stringy linked immature peroxisomes, efficient utilization of fatty acids, increased expression of PEF1, reduced intracellular production of ROS, induced ROS accumulation in host cells, consequence increased fermentation rate, consequence utilization of available glucose instead of its storage, consequence decreased fungicide resistance, consequence reduced synthesis and transport of complex sugars into the membrane, overexpression of gene coaA, increased CoA synthesis, deletion of genes ADH1-5, expression of gene adhE(A267T/E568K), converted acetaldehyde into acetyl-CoA, decreased ethanol formation, decreased glycerol formation, increased n-butanol production, deletion of gene PDH, consequence 1: predominant role in linking glycolysis and TCA cycle can be replaced by cytosolic PDH bypass and Cit2, deletion of gene Ach1, consequence 1: strongly impaired growth, consequence 2: high incidence of respiratory deficiency, overexpression of gene AGP2, consequence 1: enables carnitine shuttle to efficiently link glycolysis and TCA cycle, overexpression of gene HNM1, overexpression of gene YAT2, overexpression of gene YAT1, overexpression of gene CRC1, overexpression of gene CAT2, increase in growth rate, alteration in cell membrane composition, alteration in cell wall composition, reduction in protein aggregation, exposure to lactic acid, decrease in phosphatidylcholine, rearrangement in lipid hydrocarbon tails, rearrangement in cell wall carbohydrates, transient protein aggregation, elevated sensitivity to oxidants, failure to detoxify H2O2 effectively, minor necrosis on susceptible citrus leaves, resistance to dicarboximide fungicides, resistance to phenylpyrrole fungicides, impaired germ tube production from conidia, involvement in multi-drug resistance, deletion of gene XsTPS2, consequence none of the XsTPS2 product, guaia-4,6-diene, can be detected, deletion of gene RTT109, enhanced resistance to acetic acid, improved growth compared to wild-type, shortened lag phase, advanced glucose consumption, elevated transcription levels of HSP12, elevated transcription levels of CTT1, elevated transcription levels of GSH1, increased activities of antioxidant enzymes, improved resistance to oxidative stress, deletion of gene PUB1, increase in Gpd1p activity, higher intracellular glycerol concentration, greater tolerance to osmotic stress, enhancement of nicotinamidase activity, deletion of gene CA_C1502, consequence C. acetobutylicum Δcac1502Δupp strain is marker-less, deletion of gene upp, deletion of gene CA_C3535, consequence improves efficiency of method, deletion of gene ctfAB, consequence strains no longer produce acetone, deletion of gene ldhA, consequence strains no longer produce l-lactate,  deletion of gene CAR1, reduced arginase activity, decreased urea production, decreased ethyl carbamate production, deletion of upstream components of CWI pathway, activation of Slt2 by heat stress, interaction of Ste11 and Mkk1 mediated by Nst1, connection of high-osmolarity glycerol and pheromone pathways to CWI pathway in response to heat and pheromone, activation of Ser(407) and Thr(411) residues of Mkk1, activation by MAPK pathways, maf1 deletion, increased pyruvate kinase activity, elevated lipid content, accumulation of smaller lipid droplets, deletion of gene lys5, expression of gene EppA, enhancement of ethanol production from xylose, overexpression of gene XK, increase in ethanol yield, upregulation of GND1 expression, upregulation of SOL3 expression, upregulation of TAL1 expression, upregulation of RKI1 expression, upregulation of TKL1 expression, upregulation of NDE1 expression, upregulation of ACO1 expression, upregulation of ACO2 expression, upregulation of SDH2 expression, upregulation of IDH1 expression, upregulation of IDH2 expression, upregulation of ATP7 expression, upregulation of ATP19 expression, upregulation of SDH4 expression, upregulation of SDH3 expression, upregulation of CMC2 expression, upregulation of ATP15 expression, change in expression levels of 125 cell cycle genes, deletion of gene lag1, consequence elevated free fatty acids, consequence elevated sphingoid bases, deletion of gene lac1, deletion of gene ypc1, consequence small amounts of complex sphingolipids, deletion of gene ydc1, deletion of gene yjr116w, consequence oxygen-dependent hypersensitivity to Cu2+, consequence increased mitochondrial production of reactive oxygen species, consequence mitochondrially orchestrated programmed cell death in presence of copper, deletion of gene ypr114w, consequence general copper hypersensitivity, consequence fragmented vacuoles, consequence produces less ROS than wild type, consequence increased chronological life span, stabilization of numerous mRNAs, induction of selected lncRNAs, activation of HOG signaling pathway, increase in titer of SBCFAs, improve production of SBCFAs, overexpression of PDR12, increase secretion of SBCFAs, deletion of gene maf1, general activation of RNAP III, decrease in N2,N2-dimethyl G26 (m(2, antisuppression phenotype, over-expression of gene Trm1, reversal of maf1-antisuppression, mutation in RNAP III, decrease in tRNA transcription, increase in m(2, treatment with rapamycin, decrease in tRNA synthesis, deletion of gene Zap1p, reduced expression of ETR1, reduced mitochondrial fatty acid biosynthesis, reduction in lipoic acid content, deletion of gene ETR1, accumulation of TAG, introduction of ETR1 in zap1Δ strain, rescue of TAG level, compromised mitochondrial fatty acid biosynthesis, reduction in lipoic acid, loss of mitochondrial function, defect in mitochondria, slowed ATP production, carbon channeled towards lipid biosynthesis, stored as TAG, increased sensitivity to 5-fluorouracil, increased sensitivity to nystatin, altered transcription profile similar to human HepG2 cells, expression of gene GYP8, increased acetic acid tolerance, expression of gene WSC4, expression of gene PMT1, expression of gene KTR7, expression of gene RKR1, expression of gene TIF3, expression of gene ILV3, expression of gene ZBAI_02295, expression of gene ZbMSN4, expression of gene ZbTIF3, consequence 1: mimics the addition of glucose, consequence 2: prevents glucose-induced mRNA decay, deletion of gene VTS1, consequence 1: does not slow the rate of glucose-induced mRNA decay, expression of gene SNF1, consequence 1: influences mRNA stability by altering the recruitment activity of the transcription factor Adr1, knockdown of gene SSB1, increased lactic acid tolerance, increased cell growth, increased L-lactic acid production, deletion of mitochondrial gene, enhanced sensitivity to sampangine, expression of sampangine, increased oxygen consumption rates, induction of reactive oxygen species, alteration in heme biosynthesis, redox cycling leading to superoxide production, high concentration of sampangine, neurotoxicity, overexpression of genes of alkaline phosphatase, reduced biomass accumulation, overexpression of genes of apyrase, overexpression of enzymes involved in futile cycles, overexpression of XYL1, improved xylose fermentation, overexpression of XYL2, overexpression of XYL3, deletion of CAT8, overexpression of DAS1, increased ethanol yield from xylose, overexpression of TAL2, deletion of ATG13, deletion of gene sdh1, consequence shorter life span, deletion of gene sdh2, deletion of gene sdh4, deletion of gene cor1, deletion of gene cyt1, deletion of gene qcr7, deletion of gene qcr8, deletion of gene rip1, deletion of gene cox6, deletion of gene cox7, deletion of gene cox9, deletion of gene atp4, deletion of gene atp7, deletion of gene atp17, deletion of gene complex II, consequence higher superoxide levels, deletion of gene complex IV, deletion of gene complex V, consequence lower ATP levels, D-LA production, inhibition of D-LA consumption, deletion of gene JEN1, reduction of ethanol production, enhancement of LA tolerance, increased D-LA production up to 48.9 g/L, increased D-LA production up to 112.0 g/L, deletion of gene glpO, affected production of H2O2, deletion of gene PYC1, deletion of gene PYC2, deletion of gene PYC3, mutation of gene TUP1, enabled mannitol assimilation, mutation of gene CYC8, deletion of Puf4, decrease in gene expression of mitochondrial complex-related mRNAs, deletion of Puf3, increase in gene expression of mitochondrial complex-related mRNAs, expansion of Puf4 targets, inclusion of TCA cycle genes, expansion of Puf3 targets, inclusion of ETC complex I genes, deletion of gene involved in mitochondrial translation, altered extracellular ATP levels, deletion of gene involved in vesicle sorting in the late endosome, deletion of gene involved in amino acid signaling, altered ATP efflux, deletion of gene involved in mitochondrial retrograde signaling, deletion of gene protein kinase, consequence growth rate alteration, deletion of gene phosphatase, consequence lag time duration change, consequence growth efficiency change, consequence stress-induced diauxic shift delay, deletion of gene JAC1, consequence lethal in S. pombe, consequence lethal in S. cerevisiae, mutation in gene JAC1, consequence non-lethal in S. cerevisiae, coevolution of gene Ssq1, consequence increased efficiency of interaction with Jac1, sequence divergence of gene Jac1, consequence higher in species encoding Ssq1, fitness loss due to mistranslation mitigated, overexpression of hexose transporter genes, acceleration of ubiquitin-proteasome-mediated protein degradation, reduction of toxic protein aggregation, increase in protein synthesis, mitigation of deleterious effects of mistranslation, duplication of genes, large phenotypic leaps enabling rapid adaptation, deletion of gene flo8, consequence adhesion or flocculation in haploids, consequence pseudohyphal growth in diploids, deletion of gene somA, consequence slow growth, consequence block in asexual development, consequence inability to form further differentiated structures beyond aerial hyphae, deletion of gene ptaB, consequence similar phenotype to somA deletion, expression of gene somA, consequence biofilm formation, consequence regulation of transcription factors controlling conidiation or adhesion, consequence reduced pathogenicity, deletion of gene GLT1, negative effect on scavenging activity of glutamate against ROS accumulation, overexpression of gene GLT1, lower viability in glutamine medium, deletion of gene AQP8, impaired glucose-anabolic pathways, impaired synthesis of UDP-glucose, impaired synthesis of trehalose, impaired synthesis of glycogen, impaired accumulation of cell wall β-glucans, increased fermentation rate, impaired transcriptional induction of PGM2, impaired transcriptional induction of UGP1, downregulation of UDPG synthesis pathway, accelerated alcoholic fermentation, deletion of gene essential genes, consequence reduce genome size, expression of gene significant pathway genes, consequence improve vanillin production, consequence improve acetate production, overexpression of gene dhaB1B2B3, overexpression of gene dhaR1R2, overexpression of gene aldhH, overexpression of gene GPD1, overexpression of gene GPP2, deletion of gene ptsG, relief of carbon catabolite repression, overexpression of gene xylR, simultaneous utilization of glucose and xylose, production of glycerol at 0.48 g/g yield, production of glycerol at 0.35 g/L-h productivity, expression of genes dhaB1B2B3, dhaR1R2, aldhH, GPD1, GPP2, ptsG deletion, xylR overexpression, production of 3-HP at 29.4 g/L, dhaR1R2, aldhH, GPD1, GPP2, ptsG deletion, xylR overexpression, production of 3-HP at 0.54 g/L-h productivity, dhaR1R2, aldhH, GPD1, GPP2, ptsG deletion, xylR overexpression, production of 3-HP at 0.36 g/g yield, deletion of gene HAP4, increased maximum concentration of ethanol, increased production rate of ethanol, increased yield of ethanol, capability to produce ethanol from xylose under aerobic conditions, improved ethanol yield from detoxified hydrolysate of wood chips, depletion of gene Aur1, accumulation of ceramides, overexpression of gene YPC1, rescue of cells under Aur1 depletion, occurrence of ceramide hydrolysis, requirement for vesicle-mediated transport between Golgi apparatus, endosomes, and vacuole, acute repression of gene Aur1 by AbA, activation of vacuolar acidification, induction of cell wall integrity pathway, overexpression of gene Hxt7, increased glucose uptake rate, overexpression of gene Hxt2, overexpression of gene Hxt4, deletion of gene STD1, differential effect on glucose uptake rate, deletion of gene MTH1, overexpression of gene GCR1, increased transcription levels of HXT1, increased transcription levels of ribosomal protein genes, improved lactic acid production, increased productivity of lactic acid, increased titers of lactic acid, deletion of gene Slt2, differential effect of Rohitukine on cell viability, deletion of gene Hog1, overexpression of gene p53, triggered apoptosis in A549 cells, overexpression of gene caspase9, down regulation of gene Bcl-2, Rohitukine exposure, generation of reactive oxygen species, deletion of FAA2, increase in hexanoic acid levels, increase in octanoic acid levels, deletion of PXA1, deletion of ANT1, deletion of PEX11, optimization of hFAS, increase in decanoic acid levels, increase in total short chain fatty acid levels, consequence hypersensitivity to xenobiotics, consequence unaltered oxidative stress response, consequence unaltered general stress response, consequence unaltered unfolded stress response, consequence unaltered calcium signaling pathways, consequence increased bioactivity of compounds, deletion of genes involved in proline synthesis, weakened strain tolerance against FAP stress, deletion of genes involved in myo-inositol synthesis, addition of proline in medium, protective effect on cell growth under FAP stress, addition of myo-inositol in medium, overexpression of key gene PRO1, significantly increased FAP tolerance, overexpression of key gene INO1, increase of proline content, decrease in intracellular level of reactive oxygen species under FAP condition, deletion of gene GntA, complete absence of zwitterionic GIPC, expression of gene GntA, production of N-acetylhexosamine-IPC, generation of glycolipid product resistant to saponification and digestion by phosphatidylinositol-phospholipase C, deletion of genes encoding proteins homologous to those implicated in the Saccharomyces cerevisiae secretion apparatus, altered cellulolytic activity, deletion of genes homologous to those known to differ between the Trichoderma reesei hyper-secreting strain RUT-C30 and its ancestral wild-type strain, deletion of genes encoding proteins identified in the secretome of N. crassa when cultured on plant biomass, deletion of genes encoding proteins predicted to traverse the secretory pathway, deletion of genes encoding proteins that act in transcription, deletion of genes encoding proteins that act in protein synthesis, deletion of genes encoding proteins that act in intracellular trafficking, deletion of genes encoding fungal-specific proteins of undetermined function, deletion of genes peripherally linked to protein processing or secretion, deletion of genes involved in the SREBP pathway, hyper-production phenotype, deletion of SREBP pathway components in T. reesei, deletion of a predicted E3 ligase, altered protein production under cellulolytic conditions, deletion of SREBP transcription factor, reducing glycerol export/formation, expression of gene gapN, improving redox balance, overexpression of gene UTR1, combined strategy, increasing ethanol yield by 4.6%, expression of gene CC1225, production of D-gluconate, production of D-sorbitol, conversion of D-xylose to xylitol, production of D-xylonate, deletion of gene Gre3p, no effect on xylitol production in Cc aaor expressing strain, decrease in xylitol production in Zm gfor expressing strain, expression of gene CC1225 and xylC, increase in D-xylonate production, increase in D-xylitol production, deletion of gene filC, accumulation of filipin III derivatives, deletion of gene filD, disruption of genes in the genomic region, identification of biosynthetic cluster for the 28-membered ring of the polyene macrolide filipin, cytochrome P450 monooxygenase encoded by filC, catalyses specific hydroxylations at C26, cytochrome P450 monooxygenase encoded by filD, catalyses specific hydroxylations at C1', formation of filipin III from filipin I through filipin II, alternative route, formation of filipin III from filipin I through 1'-hydroxyfilipin I, 1'-hydroxyfilipin and filipin II, enhanced antifungal bioactivity, over-expression of glutathione synthases, increased glutathione production, deletion of glutathione reductase, increased GSSG content, over-expression of ERV1, increased GSSG content by 2.9-fold, decreased GSH content, over-expression of ERO1, increased GSSG content by 2.0-fold, did not decrease GSH content, over-expression of Pdi1, enhanced increase in GSSG content due to ERO1 over-expression, increase in recombinant fungal α-amylase yield, increase in productivity of recombinant proteins, improvement in human insulin precursor production, improvement in yeast endogenous enzyme invertase production, upregulation of gene UPC2, effect on protein production, deletion of RMT genes, decrease in ηMMA, decrease in δMMA, increase in sDMA, decrease in aDMA, deletion of gene HTM1, increase in antibody production, deletion of gene HRD1, decrease in antibody production, deletion of gene HRD3, deletion of gene UBC7, deletion of gene YOS9, stronger decrease in antibody production, deletion of gene IRE1, decreased production in wild-type cells, reversed effects in combination with Δhtm1, reversed effects in combination with Δyos9, deletion of genes YOS9 and IRE1, delayed IgG clearance from cells, deletion of genes HTM1 and IRE1, KlGCR1 deletion, diminishes glucose consumption, increases resistance to oxidative stress, increases glucose 6P dehydrogenase activity, increases NADPH/NADP(+, increases GSH/GSSG ratio, decreases ethanol production, transcriptional regulation of KlZWF1, increased flux of glucose through the pentose phosphate pathway, decreases mitochondrial respiration, deletion of gene SUB1, increased sporulation efficiency, increased middle sporulation gene transcript levels, SUB1 downregulation, induction of middle sporulation gene expression, SUB1 deletion, genetic interaction with HOS2, deletion of gene erg5, higher ratio of cytosolic free NADH/NAD+, more glycerol accumulation, more ethanol accumulation, expression of NOX, alleviated redox imbalance, expression of AOX1, alleviation of redox imbalance, increased production of 7-dehydrocholesterol by 74.4%, reduced ratio of free NADH/NAD+ by 78.0%, reduced glycerol concentration by 50.7%, reduced ethanol concentration by 7.9%, delay in vacuolar rupture, alternative form of PCD, catastrophic consequences for underdeveloped spores, deletion of gene FAS2, consequence Native FAS activity eliminated, expression of gene acpS + acpP, consequence Enzyme activity confirmed, expression of gene fabB, expression of gene fabD, expression of gene fabG, expression of gene fabH, expression of gene fabI, expression of gene fabZ, expression of gene tesA, consequence Increased total fatty acid titer, expression of gene fatB, consequence Increased total fatty acid titer 1.7-fold, consequence Shifted fatty acid profile towards C14 production, consequence Increased C14 production from <1% to more than 30%, consequence Reduced C18 production from 39% to 8%, deletion of gene dga1, improvement in lipid accumulation, deletion of gene dga2, deletion of gene lro1, deletion of gene are1, overexpression of gene dga2, deletion of gene mfe1, further increase in lipid accumulation, expression of gene suc2, establishment of sucrose utilization, increase in lipid storage, increase in biomass yield, increased GPD2 promoter activity, overexpression of gene gpd2, hyperactive GPD2 promoter, excess oxygen, inactive GPD2 promoter, continuous cultivation at 0.3 h^-1, expression of gene FLO1, consequence strong flocculation, consequence better cell growth under acetic acid stress, consequence higher ethanol production rate under acetic acid stress, expression of gene FLO1c, deletion of PTR2 gene in Saccharomyces cerevisiae, inability to uptake dipeptides, expression of BghPTR2 gene, high during appressorial germ tube formation, low in haustoria, high during conidiation and secondary infection, BghPTR2 gene expression, importance for initial infection establishment, suggested role in internal transport of nutrient reserves, suggested role in uptake of breakdown products from plant surface, increase in correct gene targeting, absence of clear side effects, overexpression of gene ldhA, high lactate dehydrogenase activity, production of lactic acid from glucose, production of lactic acid from xylose, substrate-dependent product formation, ethanol as major product from glucose, lactic acid as minor product from glucose, lactic acid as exclusive product from xylose under oxygen-limited conditions, impact on organic acid profiles, deletion of gene AAD6, deletion of gene SER33, deletion of gene ICL1, deletion of gene GLY1, deletion of gene SFC1, deletion of gene SER1, deletion of gene AGX1, deletion of gene OSM1, deletion of gene LOA1, deletion of functional domain in CrtYB, lycopene cyclase function inactivated, coexpression of CrtYB11M, increased lycopene production, coexpression of CrtE, coexpression of CrtI, coexpression of tHMG1, enhanced catalytic performance of CrtE, increased FPP competitiveness, increased copy number of Crt genes, reduced squalene production, deletion of gene pyruvate dehydrogenase complex, improved isobutanol production, deletion of gene isobutyrate biosynthetic pathway, reduced carbon outflow competing with isobutanol biosynthesis, deletion of gene pantothenate biosynthetic pathway, deletion of gene isoleucine biosynthetic pathway, increased isobutanol productivity, combination of ILV1 deletion and activation of two steps of the Ehrlich pathway, combination of ILV1 deletion and transhydrogenase-like shunt, increase in sclareol production, deletion of gene dos2, deletion of gene yer134c, deletion of gene vba5, deletion of gene ynr063w, deletion of gene ygr259c, deletion of gene ε-COP, no detectable changes in vegetative growth, no detectable changes in asexual development, decrease in production of cleistothecium, over-expression of gene ε-COP, no rescue of thermo-sensitive growth defect in α-COP mutant, deletion of gene FaTBCA, abnormal conidiation, increased sensitivity to low temperatures, loss of growth ability at 4°C, blebbing phenotypes in hyphae at 4°C, enlarged nodes containing several nuclei at 4°C, decreased virulence on wheat head, decreased virulence on non-host tomato, deletion of gene FgVAM7, reduces vegetative growth, reduces conidiation, reduces conidial germination, reduces sexual reproduction, reduces virulence, defect in vacuolar maintenance, insensitive to salt stress, insensitive to osmotic stress, hypersensitive to cell wall stressors, reduces expression of gene TRI5, reduces expression of gene TRI6, reduces expression of gene TRI101, reduces DON production, deletion of target mRNA sequences, impact on detection and interpretation of biological functions, modification of target mRNA sequences, improvement in signal and reproducibility in single-RNA detection, synonymous modification to FRET biosensors, elimination of aberrant truncation products, correct expression of full-length biosensors, significant difference in biological interpretations, deletion of gene ACO1, decrease in succinic acid levels, deletion of gene ICL1, decrease in succinic acid levels, deletion of gene IDH1, deletion of gene IDP1, decrease in succinic acid production, deletion of gene IDP1, affect fermentation rate, deletion of gene SDH1, deletion of gene SDH2, increase in succinic acid accumulation, deletion of gene FRD1, deletion of gene OSM1, no effect on fermentation rate, deletion of gene OSM1, no effect on succinic acid production, deletion of gene GAL3, consequence PARP-1 protein level unchanged, deletion of gene HHO1, consequence increased nuclear localization of PARP-1 after UV radiation, deletion of gene HUL4, deletion of gene OTU1, deletion of gene POM152, deletion of gene SNT1, consequence increased nuclear localization of PARP-1, expression of gene PARP-1, consequence inhibited yeast growth, consequence reduced UV-induced homologous recombination, consequence produced PAR products, deletion of gene Fg06297, deletion of gene Fg03333, deletion of gene Fg03826, deletion of gene Fg07932, deletion of gene Fg10516, negative regulation of MAPK signaling, deletion of gene Fg12867, deletion of gene FgCdc14, involved in ribosome biogenesis, deletion of gene PdMit1, slow radial growth, decreased conidia production, delayed conidial germination, hypersensitivity to Ca(2+, inhibition of vacuole fusion upon Ca(2+, significant inhibition of Ca(2+, deletion of gene SIN3, weak UV-sensitivity, decrease in spontaneous levels, decrease in UV-induced levels, malfunction of ribonucleotide reductase activity regulation, decrease in dNTP pool, inaccurate error-prone damage bypass postreplication repair pathway, reduction in the incidence of mutations, deletion of gene IKS1, increased glucose consumption rate, increased ethanol productivity, no expression/deletion of gene, protein synthesis impacted by iterated AAA codons, ribosome pausing on consecutive AAA-lysines, ribosome sliding on homopolymeric A sequence, premature translation termination on out of frame stop codons, mRNAs targeted for Nonsense-Mediated-Decay (NMD, deletion of gene SSA1, increase in intracellular protein content, deletion of gene SSB1, deletion of gene RPO41, increase in synthesis of abortive transcripts, increase in probability of polymerase disengagement from template, increase in amount of slippage products from homopolymeric template, decrease in binding affinity for mitochondrial transcription factor Mtf1, decrease in higher alcohols production, decrease in isobutanol production, decrease in isoamyl alcohol production, deletion of gene Os3378, reduced transposition activity, fusion of fluorescent protein to gene Os3378, enhanced transposition activity, substitution in gene Os3378, altered transposition activity, induction of Aro80, activation of ARO9, activation of ARO10, expression of alsS, expression of alsD, deletion of gene RPE1, simultaneous utilization of xylose and glucose, xylose consumption at the beginning of glucose consumption, xylose consumption ratio of about 1:10 with glucose, expression of gene redox system, production of tridecane, production of pentadecane, production of heptadecane, deletion of ACA tRNA, modest efficiency decrease, deletion of gene V1 sub-complex of yeast V-ATPase, consequence cysteine degradation impairment, deletion of gene associated with vacuole acidification, deletion of gene involved in translation, consequence cysteine catabolism impairment, deletion of gene involved in RNA processing, deletion of gene involved in folate-derived one-carbon metabolism, deletion of gene involved in mitochondrial iron-sulfur homeostasis, improved vanillin production, increased growth rate, expression of GhPAS2, enhanced VLCFA biosynthesis, restored viability to yeast psh1-deletion strain, application of acetochlor, suppressed fiber cell protrusion, overexpression of proteins involved in carboxylic acid metabolism, enhanced in wild-type ovules, overexpression of proteins involved in small-molecule metabolic processes, overexpression of proteins involved in hormone regulation, overexpression of proteins involved in lipid metabolism, elevation of Hsp104, increased under heat stress in stationary growth phase, loss of mitochondrial DNA, inhibited induction of Hsp104 synthesis under heat stress, petite mutation, suppressed activity of antioxidant enzymes, lower thermotolerance, inhibited production of reactive oxygen species, disruption of mitochondrial functional state, suppressed expression level of yeast nuclear genes upon transitioning to stationary growth phase, deletion of gene COX9, higher ethanol production rates by 37%, slight growth disadvantages, deletion of gene QCR9, higher ethanol production rates by 27%, deletion of gene QCR6, improved ethanol production rate by 24%, no growth disadvantage, deletion of gene cluster for red-pigmented undecylprodiginine complex, demonstration of efficacy and efficiency of system, expression of I-SceI gene, production of yeast I-SceI homing endonuclease, creation of double strand breaks at unique 18 base pair recognition sequence, homologous recombination, survival of genomes, increased production of amorphadiene, yield of amorphadiene produced 8-10-fold greater compared to the wild type, reduction in cell viability upon MMS treatment, reduction in cell viability upon UV treatment, decrease in gene-targeting efficiency, decrease in endogenous levels of Rad51, decrease in endogenous levels of Rad52, inhibition of gene Hsp82 by 17-AAG, proteasomal degradation of Rad51, mutation in ATPase domain of Hsp82, decrease in Rad51 protein levels, decrease in Rad52 protein levels, charged linker deletion in Hsp82, abrogation of HR activity, decrease in Rad51 focus formation, deletion of gene vsl1, no obvious defect in vacuolar protein transport, extreme defects in vacuolar protein transport when combined with fsv1 deletion, vacuolar morphology defect when combined with fsv1 deletion, overexpression of gene fsv1, mislocalization of GFP-Vsl1p, overexpression of gene VAM7, suppression of ZnCl2 sensitivity in vsl1Δ cells, suppression of vacuolar morphology defect in vsl1Δfsv1Δ cells, deletion of gene Ran GTPase, SG defects, enhanced heat-induced mutation rate, increased stress-induced mRNA accumulation, decreased stress-repressed mRNA levels, consequence increased L-lactic acid production, deletion of gene CYB2, expression of gene LDH, deletion of gene NDE1, consequence improved intracellular redox balance, deletion of gene NDE2, acetaldehyde accumulation, increased HEMF production, higher resistance to lactic acid, higher lactic acid production, detrimental effect, deletion of gene PpATT1, improved cellular fitness levels, improved thermal tolerance levels, reduced cell lysis defects, improved fermentation robustness, increased product yields, deletion of gene BAR1, higher degree of synchronization with α-factor, ability to synchronize at high cell density, presence of gene BAR1, need for higher concentration of α-factor, transient synchrony, lowered level of reactive oxygen species, decreased membrane permeability, activation of mitogen-activated protein kinases Hog1, activation of mitogen-activated protein kinases Slt2, deletion of gene involved in MAPK-signaling pathways, sensitive growth response to HMF, deletion of gene involved in PI-signaling pathways, overexpression of genes involved in MAPK-signaling pathways, increased tolerance to HMF, overexpression of genes involved in PI-signaling pathways, mutation in genes involved in purine metabolism, unknown consequence, mutation in genes involved in pyrimidine metabolism, improved ethanol production from xylose, further improvement in yeast growth, improved ability to ferment xylose, elimination of competing pathways, overexpression of gene Leu3Δ601, increased transcription of endogenous genes in valine and leucine biosynthetic pathways, overexpression of genes ILV2, ILV3, ILV5, ARO10, and ADH2, 376.9 mg/L isobutanol production from 100 g/L glucose, increased 3-methyl-1-butanol synthesis, overexpression of gene LEU4 (D578Y), 765.7 mg/L 3-methyl-1-butanol production, overexpression of gene Dga1∆Np, increased POA content, increased dry cell weight, high concentration of methionine, increased POA production, low temperatures (20-25 °C), overexpression of gene Ole1p, increased POA content in wild-type strain, genomic Ole1p in dga1 mutant, sufficient for optimal POA production, deletion of gene ypk1, increased PC deacylation, increased glycerophosphocholine production, deletion of gene plb1, curtails increased PC deacylation in ypk1Δ mutant, elevated Plb1 protein levels, elevated PLB1 mRNA levels, deletion of gene plb1 in ypk1Δ mutant, exacerbated slowed growth, exacerbated sensitivity to cell wall perturbation, inhibition of sphingolipid biosynthesis, increased PLB1 expression, control of Crz1, regulation of PLB1 expression upon inhibition of sphingolipid synthesis, regulation of PLB1 expression upon deletion of gene ypk1, deletion of gene EFM3, increases sensitivity to antibiotics that target translation, decreases translational fidelity, deletion of gene EFM2, deletion of gene CgVPS15, impairment of phagosomal maturation, hyperadherence to epithelial cells, susceptibility to antimicrobial arsenal, susceptibility to diverse stresses, virulence attenuation in mice, deletion of gene CgVPS34, deletion of gene MrKu70, increased gene disruption efficiency, 93% frequency of disrupting gene Cag8, no difference in development compared to wild-type, no difference in pathogenicity compared to wild-type, no difference in tolerance to various abiotic stresses compared to wild-type, accurate targeting into yeast genome, mutation of gene xxx, integration into yeast genome, fusion of gene xxx, construction at N-terminus, construction at C-terminus, deletion of gene Mrlig4, no apparent defects in vegetative growth, no apparent defects in colony phenotype, no apparent defects in microscopic morphology, no apparent defects in spore yield, no apparent defects in production of Monascus pigments, no apparent defects in production of citrinin, increased gene targeting of ku70, increased gene targeting of triA, increased sensitivity to methyl methanesulfonate, deletion of histone deacetylase, higher RIND-EDSB levels, deletion of endonucleases, deletion of topoisomerase, deletion of DNA repair regulators, deletion of high-mobility group box proteins, lower RIND-EDSB levels, deletion of Sir2, deletion of gene FMP21, decreased respiratory growth rate, increased thermosensitivity, overexpression of gene FMP21, improved thermotolerance, deletion of gene involved in mitochondrial protein biosynthesis process, complete mtDNA loss, deletion of gene involved in nucleic acid metabolism, deletion of gene involved in oxidative phosphorylation, deletion of gene involved in bud-site selection, deletion of gene involved in mitochondrial intermembrane space protein import, deletion of gene involved in assembly of cytochrome-c oxidase, deletion of gene involved in vacuolar protein sorting, deletion of gene involved in protein-nucleus import, deletion of gene involved in calcium-mediated signaling, deletion of gene involved in heme biosynthesis, deletion of gene involved in iron homeostasis, deletion of gene FgILV1, consequence inability to grow on minimal medium, consequence inability to grow on fructose gelatin agar lacking Ile, consequence exogenous supplementation of Ile rescues growth defect, consequence exogenous supplementation of 2-KB rescues growth defect, consequence exogenous supplementation of Met rescues growth defect, consequence defects in conidial formation, consequence defects in conidial germination, consequence decreased virulence on wheat heads, consequence low level of deoxynivalenol (DON, introduction of ldhL gene, increased lactic acid production, deletion of xylose reductase gene, deletion of xylitol dehydrogenase gene, overexpression of xylulokinase gene, introduction of xylose isomerase gene, deletion of gene YDR307W, improved tolerance to ethanol, deletion of gene YHL042W, transcriptional re-programming of metabolism, mis-perception of nutritional environment, excess amount of glucose perception, deficiency of methionine or sulfur perception, mutation leading to increase in ribosome biogenesis, improvement of ethanol tolerance, modulation of chronological life span, contribution to ethanol tolerance, deletion of hexose transporter in Saccharomyces cerevisiae, rescued by ApST3, rescued by ApST4, overexpression of ApST4 in yeast, transports glucose, transports fructose, increased sensitivity to trichothecin, higher reactive oxygen species levels, reduced cytotoxicity with antioxidant treatment, reduced cytotoxicity with rapamycin treatment, impaired mitophagy leading to increased sensitivity, deletion of trehalase gene, accumulation of trehalose, stress tolerance to acetic acid, deletion of gene involved in cell wall remodeling, deletion of gene involved in membrane transport, deletion of gene involved in energy metabolism, deletion of gene involved in amino acid biosynthesis and transport, deletion of gene involved in global transcription regulation, promotes maltose metabolism, enhances leavening ability, deletion of gene SSN6, negative for glucose derepression, hinders maltose metabolism, deletion of gene in phenylalanine biosynthesis pathway, synthesis of phenylalanine and phenylpyruvate as by-products, non-enzymatic rearrangement of chorismate to phenylpyruvate, enzymatic transamination of phenylpyruvate to form phenylalanine, SOD2 deletion, increased lifespan, cryptotanshinone treatment, lifespan extension, activation of Tor1, activation of Sch9, cryptotranshinone treatment, activation of Gcn2, improved tolerance to formic acid, improved tolerance to acetic acid, improved tolerance to levulinic acid, overexpression of gene TAL1, deletion of gene PHO13 and overexpression of gene TAL1, reduced xylitol accumulation, reduces FLO11 expression, reduces adhesion of cells to themselves, reduces adhesion of cells to agar, deletion of gene GCN5, overexpression of gene FLO5, increases flocculation, deletion of gene SAS2, enhances flocculation, enhances maltose fermentation, deletion of gene ARO8a, no change in aroma production, deletion of gene ARO8b, impaired aroma production, deletion of gene ARO80, overexpression of gene ARO80, increased VOC production, deletion of gene dps1, sensitivity to oxidative stress, production of sulfide, requirement of antioxidant for growth on minimal medium, failure to survive at stationary phase, deletion of gene dlp1, deletion of gene coq3, deletion of gene coq4, deletion of gene coq5, deletion of gene coq6, deletion of gene coq7, deletion of gene coq8, deletion of gene coq9, expression of human COQ3, recovery of CoQ10 production, expression of human COQ7, expression of Arabidopsis thaliana COQ6, deletion of endogenous prenyltransferase genes, insufficient (+, heterologous expression of FPP synthase gene ispA, (+, heterologous expression of ERG20, heterologous expression of valencene synthase from Nootka cypress, improved (+, deletion of gene SIZ1, higher furfural tolerance, rapid furfural reduction to furfuryl alcohol, higher ethanol productivity in presence of furfural, tolerance towards oxidative stress, deletion of gene FLX1, deletion of insert in Δ12 desaturase, abolished activity, reintroduction of two histidine residues in Δ12 desaturase, partially restored activity, ruled out role in sex pheromone synthesis for Δ5 and Δ9/C16:0 desaturases, deletion of gene MoAND1, impaired nuclear distribution along the hyphae, ∼65% decrease in conidia production, abnormalities in septation pattern, abnormalities in nuclear distribution, inability to germinate, defective programmed nuclear migration, defective programmed nuclear positioning, delay in penetration of host plants, indicates essentiality under experimental conditions, reveals function not observable in wild-type gene in vivo, reveals function not observable in purified gene product in vitro, deletion of Snf1 regulation on Acc1, enhanced activity of Acc1, mutation of Ser659 in Acc1, increased total fatty acid content, mutation of Ser1157 in Acc1, increased supply of malonyl-CoA, increased production of fatty acid ethyl esters, increased production of 3-hydroxypropionic acid, accumulation of free fatty acids, expression of gene Acot5s, higher unsaturated fatty acid ratio, restored growth, increased extracellular total fatty acids, deletion of gene Cdh1, decreased origin activity, deletion of gene Sic1, deletion of gene Cdh1 and Sic1, compromised origin redundancy, CDK deregulation, decreased number of active origins, decreased number of dormant origins, origin shortage, increased chromosome instability, deletion of beta-oxidation pathway, deletion of acyl-CoA synthetases, overexpression of thioesterases, overexpression of ACC1, no significant increase in fatty acid production, deletion of ADH1, redirected metabolic flux to fatty acid synthesis, identification of ACC1 phosphorylation sites, potential regulation of ACC1 activity, stable maintenance of integrated cassette, overexpression of gene ACB1, 40% increase in FAEE production, overexpression of gene gapN, increased vanillin production, overexpression of alsS, conversion of pyruvate into 2,3-BDO, overexpression of alsD, deletion of PDC1, elimination of pyruvate decarboxylation reaction, deletion of PDC5, introduction of cellobiose utilization pathway, production of 2,3-BDO from cellobiose, upregulation of ARO10, production of phenylethanol, changes in intracellular concentrations of amino acids, combination of aro8Δ mutation with other mutations, stimulation of phenylethanol production, deletion of neutral lipid biosynthesis genes in S. cerevisiae, rescue of cell growth by CtDGAT2b, overexpression of CtDGAT2b in S. cerevisiae, increased triacylglycerol production, presence of CtDGAT2b in S. cerevisiae, incorporation of free fatty acids into triacylglycerol, mutation in CtDGAT2a/b, altered efficiency in triacylglycerol production, overexpression of CtDGAT2b in C. tropicalis, enhanced production of storage lipid enriched with saturated fatty acids, no effect on growth with aromatic amino acids, overexpression of gene ARO8, initialization of histidine degradation, supports growth on histidine as sole nitrogen source, deletion of gene tpsA, lower internal trehalose contents, deletion of gene tppA, deletion of gene tppB, abnormal conidiophores, accumulated levels of trehalose-6-phosphate, conidia more susceptible to thermal stress, increased splicing efficiency, overexpression of gene PDR12, increased tolerance to sorbic acid, increased tolerance to propionic acid, increased tolerance to levulinic acid, increased tolerance to formic acid, rapid recovery in acetic acid, deletion of gene CMK1, different response to weak organic acids when combined with deletion of PDR12, deletion of gene TEL1, increased sensitivity to camptothecin, synthetic cytotoxic interactions with 14 other gene deletions, telomere defects, conserved synthetic cytotoxic interaction in Caenorhabditis elegans, deletion of gene eco1, reduced ribosomal DNA (rDNA, transcriptional signature of starvation, deletion of gene fob1, rescues rRNA production, partially rescues transcription genome-wide, corrects genome-wide replication defects, corrects nucleolar structure, corrects rDNA segregation, gdh-null mutant, viable on all nitrogen sources tested, GS/GOGAT as primary alternative pathway on ammonia and glutamine, growth on proline requires both PUT and GS/GOGAT pathways, growth on glucose-urea media elicited decrease in GOGAT activity, growth on glucose-urea media elicited increase in P5CDH activity, deletion of gene fumarate reductase, consequence increased succinate yield, deletion of gene pyruvate formate lyase, deletion of gene citrate lyase, deletion of gene β-galactosidase, deletion of gene aconitase, deletion of gene ACA13, reduced Ca(2+, defects in compatible pollen germination, defects in seed production, evolution of orthogonal ribosomes, increased efficiency of unnatural amino acid incorporation, manipulating the genome, enhanced efficiency of unnatural amino acid incorporation, evolution of orthogonal quadruplet decoding ribosomes, increased number of distinct amino acids that can be site-specifically encoded, discovery of mutually orthogonal synthetase/tRNA pairs, development and application of the pyrrolysyl-tRNA synthetase/tRNA pair, aided incorporation of unnatural amino acids in eukaryotic cells and animals, combining chemoselective reactions with encoded amino acids, facilitated installation of posttranslational modifications, facilitated rapid derivatization with diverse fluorophores for imaging, deletion of gene involved in nitrate utilization, consequence inability to utilize nitrate, deletion of gene TPA1, modulates termination efficiency up to ∼10-fold, affects translational accuracy in a stop codon context-dependent manner, hydroxylation of gene RPS23, increases translational accuracy, decreases translational accuracy, determines viability as a consequence of nonsense codon suppression, deletion of gene OLA1, affects stop codon bypass, deletion of gene BSC2, deletion of gene YNL040W, deletion of PKMT gene, altered stoichiometry of lysine methylation, processive methylation on lysine residues, selective methylation by specific PKMTs, broad range of stoichiometries of lysine methylation, specialized roles in regulation of non-histone protein activity, new relationships between SEE1 and EFM1 in EF1α methylation, reinterpretation of EFM1's direct PKMT activity, succinate titer improved to 8.09±0.28g/L, deletion of gene FgIlv5, failure to grow under BCAA-limited conditions, decrease in aerial hyphae formation, decrease in red pigmentation, blockage of conidia formation, reduced virulence on wheat heads, low deoxynivalenol production in wheat kernels, deletion of gene gpi10, consequence Man-III addition defect, deletion of gene smp3, consequence Man-IV addition defect, expression of gene Gpi10/PigB from T. cruzi, consequence complements gpi10Δ mutation, consequence complements smp3Δ mutation, expression of gene Gpi10/PigB from T. congolense, expression of gene Gpi10/PigB from P. falciparum, expression of gene PIGB from T. vivax, expression of gene PIGB from T. brucei, expression of gene T. cruzi PIGB, consequence complements gpi10Δ/smp3Δ double mutant, deletion of gene HLJ1, deletion of gene TEF2, consequence 1: impaired cellular protein solubility, consequence 2: disruption of co-translational quality control, consequence 3: impaired proteasome assembly, consequence 1: enhanced clearance of aberrant proteins via the proteasome, deletion of gene pox1, expression of gene pox2, increased ratio of MCFAs, expression of gene crot, unblocked β-oxidation pathway, normal growth rate, expression of gene pox2, increased fatty acid accumulation, expression of gene pox2, increased total fatty acid production, expression of gene pox2, increased intracellular MCFAs, expression of gene pox2, increased MCFAs in culture medium, expression of gene pox2, expression of gene crot, increased total fatty acid production, expression of gene pox2, expression of gene crot, increased intracellular MCFAs, expression of gene pox2, expression of gene crot, increased MCFAs in culture medium, deletion of tRNA gene, no appreciable phenotype in rich medium, additional phenotypes under challenging conditions, robustness facilitated by backup compensation, deletion of tRNA gene with identical anticodon, different contribution to cellular fitness, deletion of single-copy tRNA family gene, stress response, deletion of multi-copy tRNA family gene, increased expression of translation machinery, deletion of gene AIM44, defects in contractile ring closure, delay in Hof1p phosphorylation, altered Hof1p localization, overexpression of gene DBF2, rescues cytokinesis defect, marked decline in the rate of growth, efficiency of growth remains nearly unaffected, affecting membrane integrity, decrease in efficiency of growth, affecting ion homeostasis, adh1Δ, accumulation of 120 mg/L 1-butanol, highest reported 1-butanol titer of 242.8 mg/L, hypersensitivity to H2O2, impaired oxidative stress tolerance, deletion of essential gene xxx, enrichment in nucleus localization, enrichment in nucleolus localization, over-representation in rRNA synthesis, over-representation in tRNA synthesis, deletion of COQ genes, respiratory deficiency, decreased levels of Coq4, decreased levels of Coq6, decreased levels of Coq7, decreased levels of Coq9, over-expression of Coq8, restoration of steady-state levels of sensitive Coq polypeptides, promotion of synthesis of late-stage Q-intermediates, affects association of several Coq polypeptides in high molecular mass complexes, supplementation with exogenous Q6, increased steady-state levels of Coq4, increased steady-state levels of Coq7, increased steady-state levels of Coq9, promotion of formation of late-stage Q-intermediates, disruption of gene MoMSN2, defects in aerial hyphal growth, reduction in conidial production, impaired infection of host plants, reduced expression of genes involved in conidiophore formation, impaired appressorium formation, impaired penetration, sensitivity to common osmotic stressors, modulation of genes participating in conidiation, modulation of genes participating in infection, modulation of genes participating in cell-wall integrity, modulation of genes participating in stress response, deletion of gene figA, consequence retardant hyphal growth, consequence sharp reduction of conidial production, consequence impairment of homothallic mating, consequence requirement of HACS midA for heterothallic mating, consequence decreased expression of brlA, consequence decreased expression of nsdD, deletion of gene involved in cell wall biosynthesis, compromised cell surface integrity, deletion of gene involved in calcium homeostasis, increased susceptibility to stress conditions, deletion of gene involved in nutritional response, impaired growth under nutrient-limited conditions, increased sensitivity to harsh phagosomal environment, deletion of gene involved in protein glycosylation, altered recognition by macrophages, reduced iron uptake, deletion of gene Sir2p, overexpression of gene Sirt1, restoration of transcriptional silencing at rDNA, repression of extrachromosomal rDNA circles formation, decrease in histone acetylation at H3K9, decrease in histone acetylation at H4K16, deletion of gene Gln3, activation of NCR-sensitive transcription by Gat1, activation of Gln3 and Gat1, activation of GAT1 transcription, production of Gat1 isoforms IsoA and IsoB, low level production in rich and poor nitrogen conditions, placement of Myc(13) tag after Gat1 Ser-233, increased production of Gat1 isoforms IsoC-F in proline medium compared to glutamine medium, requirement of Gln3 and UASGATA promoter elements, production of Gat1 isoforms IsoA, IsoC, and IsoD, regulated production of wild type versus truncated Gat1 proteins, failure to be regulated in parallel, sequence(s) downstream of Gat1 leucine 61, responsible for nonparallel production, deletion of gene Ppq1, consequence 1: up-regulation of mating signaling, overexpression of gene Ppq1, consequence 1: down-regulation of mating signaling, dephosphorylation by Ppq1, consequence 1: targeting at or upstream of the terminal MAP kinase Fus3, deletion of gene SGR810-815, deletion of gene LPD1, overexpression of gene pyruvate carboxylase, overexpression of gene malate dehydrogenase, overexpression of gene malic enzyme, increased pyruvate supply for isobutanol biosynthesis, activation of transhydrogenase-like shunt, increased isobutanol levels, deletion of gene FgFim, reduced rate of mycelial growth, delayed conidium germination, irregularly shaped hyphae, lack of sexual reproduction, decreased resistance to JS399-19, increased sensitivity to metal cations, increased sensitivity to cell membrane damage, increased sensitivity to cell wall damage, impaired virulence on flowering wheat heads, reduced production of deoxynivalenol, lower expression of Cyp51 genes, comK overexpression, fast preparation of competent cell, exogenous endonuclease I-SceI overexpression, double-strand break repair, upp gene deletion, counter-selection marker function, foreign dsDNA integration, enhancement by double-crossover homologous recombination, DSB repair, stimulation of second intramolecular homologous recombination, second intramolecular homologous recombination, enhancement of resolution frequency, deletion of gene ARV1, decreased neutral lipid synthesis, increased fatty acid sensitivity, lipoapoptosis, overexpression of gene ARV1, increased triglyceride mass, increased lipid droplet number, up-regulation of DGAT1, up-regulation of CD36, transcriptional regulation of PPARα by ARV1, deletion of methionine, yeast chronological lifespan extension, increase of glutamic acid, restriction of glucose, low acetic acid production, distinct mechanisms of yeast longevity, expression of gene CmINV, complementation of growth on sucrose, production of short-chain fructooligosaccharides, deletion of gene suc2, inability to grow on sucrose, increased activity by 50 mM LiCl, deletion of gene Tat1p, interference with aromatic and branched-chain amino acids transport, deletion of gene Gap1p, deletion of genes involved in biosynthesis and transport of aromatic and branched-chain amino acids, increased susceptibility to eugenol, eugenol exposure, leakage of cell contents, reduced statistical power without biological replicates, 96% power to detect a two-fold change with 6 million reads per condition, deletion of gene FUI1, consequence uridine transport unaffected, deletion of gene DUR3, consequence normal growth on urea, deletion of gene FUR4, consequence normal growth on uracil, deletion of gene hemF, impaired growth, cellular porphyrin accumulation, deletion of gene hemH, overexpression of gene hemH, increased cellular heme content, overexpression of gene hemA, no increase in cellular heme content, overexpression of gene hemF, increased cellular porphyrin intermediates, deletion of gene HpFAD2, absence of C18:2 and αC18:3, deletion of gene HpFAD3, absence of αC18:3, heterologous expression of gene HpFAD2, presence of C18:2 and αC18:3, heterologous expression of gene HpFAD3, low oxygen conditions, up-regulation of HpFAD2, up-regulation of HpFAD3, excess of C18:1, repression of HpFAD3 transcription, excess of C18:2, excess of C18:3, deletion of gene Mcm1, consequence competitive interference between paralogs, overexpression of gene Mcm1, consequence increased molecular complexity of gene regulatory network, deletion of gene hxk2, consequence mitochondrial dysfunction, consequence accumulation of ROS, consequence increase of apoptotic cells, consequence DNA fragmentation, consequence externalization of phosphatidylserine, deletion of gene PEX3, increased production of HBsAg, production of HBsAg nearly 8-9% of the total cell protein, deletion of gene ole1, expression of gene fD9desA, expression of gene fD9desB, expression of gene fD9desC, expression of gene fD9desD, deletion of gene Prodh, hypovirulence, lower level of sporulation, deletion of gene P5Cdh, supplementation of exogenous proline, increase in intracellular level of proline, increase in activity of Prodh, increase in activity of P5Cdh, cells infected with hypovirus, down-regulation of Prodh, down-regulation of P5Cdh, disruption of genes involved in conversion of arginine to ornithine, no effect on virulence, disruption of genes involved in conversion of ornithine and glutamate to P5C, disruption of genes involved in conversion of P5C to proline, deletion of gene pro1, reduction in asexual sporulation, deletion of gene pro2, deletion of gene Med18, consequence activation of transcription at the CYC1 promoter is impaired, deletion of gene Med20, deletion of gene Med19, deletion of Mediator components, consequence normal levels of recruitment and activity of TFIIH at the CYC1 promoter are reduced, consequence activation by the same activator at a different gene is unaffected, overexpression of Mediator components, consequence triggering of inactive polymerase at CYC1 into a productively elongating form is enhanced, deletion of gene Aoso, effects on formation and localization of stress granules in response to heat stress, formation of mutagenesis cassette, self-excision of mutagenesis cassette, promotion of homologous recombination, seamless deletion via single transformation, dependency on length of inverted repeat, improvement of deletion efficiency with increase of inverted repeat length, deletion of entire paralog set, severe reduction of dityrosine fluorescence, deletion of genes in network, reduced dityrosine assembly, presence of gene products, involvement of lipid droplets in outer spore wall formation, deletion of gene Pdc, consequence 1: rapid glucose consumption, consequence 2: no ethanol production, mutation in gene MTH1, overexpression of gene alsS, consequence 1: efficient 2,3-BD production, overexpression of gene alsD, overexpression of gene BDH1, consequence 2: faster glucose consumption, deletion of RTG2, AA-PCD, deletion of RTG3, up-regulation of peroxisomal isoform of citrate synthase, resistance to AA-PCD, up-regulation of isocitrate dehydrogenase isoform 1, deletion of MKS1, no effect on AA-PCD, deletion of HAP4, activation of RTG pathway, decrease in ROS production, decrease in cytochrome c release, rad52 deletion, decreased efficiency of GRC, dnl4 deletion, increased frequency of GRC in C. glabrata, minimal increase in GRC frequency in S. cerevisiae, expression of gene um06350.1, consequence xylan degradation, consequence production of xylotriose, increased phenylpyruvate formation, increased phenylacetaldehyde formation, increased transcription of ARO9 and ARO10, use of two-phase fermentation method, increased 2-PE extraction, deletion of gene YJR142W, lowered oxythiamin resistance, overexpression of gene YJR142W, raised oxythiamin resistance, expression of plant counterpart gene, restored oxythiamin resistance to YJR142W deletant, expression of bacterial counterpart gene, deletion of gene KCS1, consequence inositol auxotrophy, consequence decreased intracellular inositol, consequence decreased phosphatidylinositol, consequence decreased transcription of INO1, consequence does not fully rescue INO1 expression in kcs1Δ, expression of gene KCS1, consequence increased Kcs1 protein levels in response to inositol depletion, Kcs1-catalyzed production of inositol pyrophosphates from inositol pentakisphosphate, consequence optimal INO1 transcription, over-expression of the glutathione peroxidase 3 (GPX3) gene, over-expression of the GSH1/GSH2 genes, increased total glutathione content, deleting the glutathione reductase (GLR1) gene, yielded 7.3-fold more GSSG, enhanced total glutathione content up to 1.6-fold, deletion of insertion peptide in GlyRS1, impaired catalytic efficiency in vitro, 2- to 3-fold increase in Km, 5- to 8-fold decrease in kcat, mutation of insertion peptide in GlyRS1, GRS2 codon optimization, functional gene conversion, dispensable for protein stability in vivo, dispensable for rescue activity of GRS1 at 30°C in vivo, deletion of gene SFP1, decrease in Sup35p amount, overexpression of gene SFP1, increase in termination efficiency, increase in Sup35p amount, deletion of gene KRE6, facilitation of mogroside III E production, consequence lower ethanol yields, deletion of gene tdh3, upregulation of gene tps1, consequence decrease in ethanol production, consequence increase in trehalose biosynthesis, deletion of gene meu1, decrease in DMTS-P1 production, deletion of gene mri1, deletion of gene mde1, decrease in DMTS producing potential, deletion of gene spe2, increase in DMTS-P1 content, deletion of gene mri1, further decrease in DMTS producing potential, impaired ethanol production, diminished plasma membrane H⁺-ATPase activation, depletion of nearly all adenine nucleotides, conversion of adenine nucleotides into inosine, deletion of gene AMD1, prevention of inosine formation, expression of Tps1 protein from Yarrowia lipolytica, 90%-reduced T6P content, mutation in gene TPS2, hyperaccumulation of T6P, increase in cytosolic pH, reduction in growth rates on non-fermentable carbon sources, deletion of gene Pot1p, consequence 1: impaired fatty acid utilisation, deletion of gene Fox3p, consequence 1: partial impairment in fatty acid utilisation at elevated temperatures, deletion of gene Pot13p, consequence 1: no detectable phenotype, deletion of multiple genes Pot1p, Fox3p, Pot13p, consequence 1: no significant attenuation in virulence, deletion of gene UGA1, GABA growth defect, thermosensitivity, heat-induced production of reactive oxygen species, expression of gene AtGABA-T, GABA growth, thermotolerance, limiting production of ROS, expression of gene ScGABA-TKG, deletion of gene involved in electron transport chain, attenuation of oxidative stress, partial amelioration of cell death, increased DNA content, defect in mitosis, defect in cytokinesis, defect in cell separation, overexpression of gene SPT15-M2, enhanced ethanol tolerance, tolerance to hyperosmotic stress, improved ethanol production from VHG fermentations, constitutive and sustained activation of Hog1, reduced accumulation of ROS, overexpression of gene SPT15-M3, sensitivity to 50% glucose, deletion of gene HSP12, deletion of gene AIM17, deletion of gene SSA4, deletion of gene USV1, deletion of gene IGD1, deletion of gene AAF1, reduced acetic acid levels in wine by up to 39.2%, did not increase acetaldehyde levels, reduced acetic acid levels, increased acetaldehyde levels by 41.4%, overexpression of gene AAF1, increased acetic acid levels in wine in an Ald6p-dependent manner, produced significantly more acetic acid in wine in an ald4Δald6Δ strain, deletion of gene SIT4, decreased pyruvate decarboxylase activity, reduced fermentative capacity, hyperphosphorylation of Pdc1p in logarithmic phase, phosphorylation of Pdc1p, increased K(m, decreased V(max, deletion of gene pdc6, deletion of gene aro10, deletion of gene thi3, expression of gene LgScAro10, expression of gene LgSeubAro10, induction of gene LgScAro10 by phenylalanine, induction of gene LgSeubARO10 by phenylalanine, higher basal expression of gene LgSeubARO10 during growth with ammonia, deletion of gene LgSeubARO80, deletion of enzymes essential for sporulation, impact on metabolic pathways, overexpression of metabolic genes required for sporulation, enhancement of ATP production in early phase, increase in carbohydrate breakdown in middle and late stages, deletion of gene RRN10, reduced RNA content, overexpression of gene RPL40A, restored growth defects, increased RNA content, deletion of ribosome biogenesis-related genes, increased L-lactate production, detrimental to cell growth, introduction of gene LDH, efficient production of lactate, exclusive expression of gene PDC, metabolic flow towards ethanol fermentation, exclusive expression of gene LDH, metabolic flow towards lactate fermentation, overexpression of gene SPBC16A3.08c, extended chronological lifespan, decreased mating efficiency, high temperature-sensitive growth, low temperature-sensitive growth, deletion of gene oga1 (+), caffeine-sensitive phenotype, canavanine-resistant phenotype, deletion of gene xxx and gene yyy, synthetic lethality, disruption of metabolic pathways, few examples of synthetic lethality, formation of protein-protein interactions, genes involved in essential pairs, recent duplicates, functional similarity, preservation of essential protein complexes, improved prediction of intra-cellular fluxes, coupling cellular biological objective with product flux, increased lipid content, dependent on glucose concentration, dependent on nitrogen limitation, dependent on active leucine biosynthesis, deletion of 3' terminal region of dga1 ORF, overexpression of gene Esa1p, reverted lipid content to wild-type level, decreased ESA1 mRNA expression, deletion of FAD-dependent monooxygenase, altered oxidation state of intermediates, deletion of cytochrome P450 oxygenase 1, deletion of cytochrome P450 oxygenase 2, heterologous expression of FAD-dependent monooxygenase, formation of intermediate compounds, heterologous expression of cytochrome P450 oxygenase 1, heterologous expression of cytochrome P450 oxygenase 2, deletion of gene mdh1, failure to produce succinate, deletion of gene oac1, deletion of gene dic1, production of 0.02 C-mol/C-mol glucose succinate, reductive TCA cycle activity, overproduction of gene DIN7, enhanced Mhr1-dependent mtDNA replication, increased number of residual DSBs at ori5 in HS-ρ(-, increased deletion mutagenesis at the ori5 region in ρ(+, simultaneous overproduction of gene Mhr1, suppressed enhanced mtDNA replication, suppressed increased residual DSBs at ori5, suppressed increased deletion mutagenesis at the ori5 region, relative activity levels of Din7 and Mhr1, modulated preference for replication versus homologous recombination to repair DSBs at ori5, deletion of gene RAD50, deletion of gene RAD51, deletion of gene RAD54, deletion of gene RAD55, deletion of gene RAD57, deletion of gene MRE11, deletion of gene XRS2, deletion of 73 additional genes, expression of HIV-1 Gag protein, release of Gag virus-like particles, expression of HIV-1 Tat, no support for transcription, replacement of HIV-1 promoter U3 with yeast glyceraldehyde-3-phosphate dehydrogenase gene promoter, restoration of gag mRNA transcription, expression of HIV-1 Rev, no support for protein synthesis, deletion of R-U5 and SL region, restoration of Gag protein expression, co-expression of HIV-1 genome with R-U5-SL region and Gag protein, transcription and incorporation of HIV-1 genomic RNA into Gag VLPs, deletion of R-U5 to gag gene, failure of HIV-1 RNA packaging into Gag VLPs, deletion of gene Dga1p, reduced TG synthesis, deletion of gene Lro1p, major reduction in TG synthesis, deletion of gene Are2p, absence of SE synthesis, preference for polyunsaturated fatty acids, low fatty acid substrate specificity, preference for certain fatty acids in SE, Lro1p, insufficient SE for lipid droplet formation, Lro1p, Dga1p, insufficient TG for lipid droplet biogenesis, expression of gene ssk1(E330N…K356N), reduced glycerol/ethanol ratio, compromised volumetric productivity, compromised osmotolerance, deletion of mannosyltransferases, consequence 1: obtaining more than 90% homogeneity in glycan structures, consequence decreased xylitol yield, consequence increased ethanol yield, consequence increased intracellular xylitol concentration, consequence decreased xylose consumption rate under anaerobic conditions, deletion of PHA synthase gene, increased 3HA production, deletion of β-oxidation pathway gene, expression of tesB gene, increased 3HTD production, increased 3HDD production, expression of PTE1 gene, increased 3HD production, deletion of amino acids 57-70 in α-mating factor secretion signal, increased secretion of horseradish peroxidase, increased secretion of lipase, deletion of gene PPR1, drastic growth defects, increased microconidia production, deletion of gene PPR2, elevated fumonisin production, abnormally swollen cells formation, early hyphal branching during conidia germination, impact on small RNAs, temperature-sensitive mutation of gene xxx, deletion of gene PilA, consequence viability unaffected, consequence germination unaffected, deletion of gene PilB, deletion of gene SurG, deletion of genes PilA, PilB, SurG, consequence viability unaffected, PilB, SurG, consequence germination unaffected, presence of myriocin, consequence reduced number of PilA-GFP foci, consequence altered distribution of PilA-GFP foci, deletion of gene PHO3, consequence cAPase-negative phenotype, consequence genetic instability of cAPase activity, consequence generation of chimeric PHO5/3 fusion genes, deletion of gene CnPlc1, reduced IP(3, impaired thermotolerance, compromised cell walls, reduced capsule production, reduced melanization, defective cell separation, inability to form mating filaments, deletion of gene CnArg1, increased IP(3, enlarged vacuoles, overexpression of gene CnPlc1, increased PIP(2, overexpression of gene CnArg1, increased complex inositol polyphosphates synthesis, essential for cellular homeostasis, essential for virulence, overexpression of gene ERR2, complement growth defect in double-null strain, overexpression of gene ERR3, deletion of gene Sdh3p, increase in succinate production, deletion of gene Ser3p/Ser33p, reduction in serine formation from glycolysis, decrease in succinate depletion by succinate dehydrogenase complex, redirection of carbon flux towards succinate production, expression of gene LAC4, abolished activity of acid/neutral trehalase, expression of gene LAC12, relieved glucose repression, no physiological burden in cells, increased galactose consumption, specific lactose uptake rate 0.98 g lactose g CDW(-1, -1)), (expression of gene LAC12, specific lactose uptake rate 2.50 g lactose g CDW(-1, (expression of gene LAC4, ethanol production 24.3 g lactose g CDW(-1, (expression of gene LAC12, ethanol production 23.4 g lactose g CDW(-1, deletion of gene VPS34, reduced mRNA production from long or G+C-rich coding sequences, reduced rate of RNA polymerase II progression through lacZ CDS, deletion of gene VPS15, reduced NuA4 occupancy in certain transcribed CDS, impaired localization of the induced GAL1 gene to the nuclear periphery, deletion of gene FgMETB, inability to grow on fructose gelatin agar medium containing SO(4, 2-) as sole sulfur source), (deletion of gene FgMETB, more than 90% conidia unable to germinate in 2% sucrose solution within 6 or 12 h of incubation, rescue of mycelial growth and spore germination by supplementation of 1 mM Met or 0.5 mg/ml homocysteine, decreased virulence on wheat heads, low level of deoxynivalenol (DON, increased sensitivity to the sterol demethylation inhibitor tebuconazole, deletion of gene SLI1, almost complete loss of tri- and tetraacetyl phytosphingosines, deletion of gene ATF2, 15-fold increase in triacetyl phytosphingosine, FA production, slightly lower growth of S. cerevisiae, introduction of gene RoPYC, increased FA production to 1134±48 mg L(-1, introduction of gene SFC1, increased FA production to 1675±52 mg L(-1, Deletion of gene dse2, enhanced resistance to lactic acid, Deletion of gene scw11, Deletion of gene eaf3, Deletion of gene sed1, shortened lag phase in 6% lactic acid, Introduction of gene LDH, increased lactic acid productivity, Quadruple deletion of genes dse2, scw11, eaf3, sed1, 27% increase in lactic acid productivity, overexpression of gene SIR2, extends CLS, reduces acetate production, overexpression of gene HST3, shortens CLS, increases ethanol level, reduces acetic acid production, moderately extends CLS, deletion of gene NGR1, decreases CLS, deletion of gene UTH4, deletion of gene PUF3, increases CLS, blocks stress granule formation, deletion of gene YCA1, deletion of gene AIF1, overexpression of optimized, cytosolically localized valine biosynthesis pathway, complemented valine auxotrophy, overexpression of xylose isomerase XylA, enabled growth on D-xylose, overexpression of transaldolase Tal1, overexpression of xylulokinase Xks1, overexpression of ketoacid decarboxylase Aro10, enabled fermentation to isobutanol, overexpression of alcohol dehydrogenase Adh2, deletion of gene ubc7, increased sesquiterpene production, deletion of gene ssm4, deletion of gene doa10, deletion of gene pho86, Hmg1p stabilization, Hmg2p stabilization, deletion of gene SsSOD1, increased sensitivity to heavy metal toxicity, reduced ability to detoxify superoxide in infected leaves, reduced expression levels of sspg1, reduced expression levels of sspg3, increase in mutation rates, reduction in efficiency of MMR of replication errors generated by Pol α, implication that 5' ends serve as entry points for 5' excision of replication errors made by Pol α, mutation rates in Pol α mutator strain are 5- to 25-fold lower compared to an msh2Δ strain completely lacking MMR, deletion of gene isc1, higher resistance to acetic acid, lower levels of some phytoceramide species, lower levels of ROS production, reduced mitochondrial alterations, deletion of gene alg3, deletion of gene alg11, expression of artificial flippase, expression of protozoan oligosaccharyltransferase, expression of human N-acetylglucosaminyltransferases I, expression of human N-acetylglucosaminyltransferases II, deletion of gene RAD1, loss of LLR activity, deletion of gene CSM3, exacerbation of loop repair failure when combined with CSM3 deletion, exacerbation of loop repair failure when combined with RAD1 deletion, expression of gene CSM3, control of minisatellite length expansions, expression of gene TOF1, expression of gene MRC1, control of minisatellite contractions, deletion of gene GAL1, graded response, overexpression of gene GAL1, inactivation of remaining feedback loop, overexpression of gene GAL3, altered [PSI(+, changed Rnq1p aggregate morphology, modified Rnq1p aggregate size, variant dominance alteration, deletion/insertion changes, effect on multiple genes, over 180 SNPs, more than one-quarter nonsynonymous mutations, altered expression of approximately 900 genes, similar alterations in all three evolved strains, genes with a role in oxidative metabolism, altered expression, decreased ethanol production, switch from fermentation to respiration, deletion of gene pfh1, sensitive to growth under aerobic conditions, increased levels of total iron, hallmarks of oxidative stress such as protein carbonylation, decreased aconitase activity, lower levels of oxygen consumption, gene products up-regulated upon iron depletion in wild-type cells are constitutively up-regulated, gene products down-regulated upon iron depletion in wild-type cells are constitutively down-regulated, decrease of cytosolic available iron, activation of Grx4, decrease in higher metabolite formation, decrease in higher alcohol production, deletion of gene HOM2, deletion of gene PAD1, deletion of gene PRO2, deletion of gene SPE1, double-deletion of genes BAT2 and HOM2, greater decrease in isobutanol production, greater decrease in isoamyl alcohol production, greater decrease in isobutyric acid production, greater decrease in isovaleric acid production, triple-deletion of genes BAT2, THI3, and AAD6, greatest impact on end metabolite production, THI3, and HOM2, greatest impact on end metabolite production, triple-deletion of genes HOM2, PRO2, and AAD6, greater effect on isoamyl alcohol levels, PRO2, and AAD6, greater effect on isovaleric acid levels, deletion of gene PDR8, consequence nerolidol production, deletion of gene ABZ1, consequence 2-phenyl ethanol production, deletion of gene PLB2, consequence ethyl esters production, generation of product derivatives, creation of artificial gene clusters, gene expression, heterologous expression, introduction of single or multiple point mutations, expression of gene Pi-xylA, overexpression of gene XKS1, overexpression of gene RPE1, overexpression of gene RKI1, overexpression of gene TKL1, poor xylose utilization initially, deletion of gene COX4, decreased xylose growth in respiratory deficiency strain, increased activity of xylose isomerase, upregulation of enzymes involved in glycolysis, upregulation of enzymes involved in glutamate synthesis, downregulation of trehalose synthesis, maximizing productivity, minimizing raw material costs, altering product rates, altering product costs, altering product yields, altering product purity, production of predominantly Man(3, deletion of gene Hrp3, nucleosome perturbation across the chromosome, production of antisense transcripts, destabilization of the DNA-histone interaction, cell cycle-dependent alleviation of heterochromatin silencing, deletion of gene otg1(+), complete loss of galactosylation, deletion of gene otg2(+), deletion of gene otg3(+), expression of gene otg2(+), Gal transfer activity toward pyridylaminated Man(9, 2) oligosaccharide), (expression of gene otg2(+, expression of gene otg3(+), generation of α1,3-linkage, deletion of segment from rat CCT, increase in lipid-independent V(max, weakening of membrane binding affinity, deletion of gene MFG1, shift in half-cell glutathione redox potential by 75-10 mV, deletion of gene ZrTRK1, growth at low K(+, decreased potassium content in cells, plasma membrane hyperpolarization, increased plasma membrane depolarization during potassium starvation, reduced effect on ZrPma1 activity, expression of gene ZrTRK1, complementation of S. cerevisiae trk1Δ trk2Δ cell phenotypes, deletion of gene tpi, overexpression of gene tpi2, deletion of gene tRNALeu(GAG), no phenotypic consequence, deletion of gene tRNALeu(UAG), deletion of three genes for tRNALeu(UAG), inability to compensate decoding capacity by another tRNALeu isoacceptor, deletion of gene NEO1, consequence haploinsufficiency in presence of tripelennamine, overexpression of autophagy-related genes, consequence hypersensitivity to tripelennamine, expression of MsrB, catalytic efficiency towards free Met-O >2000-fold lower than yeast fRMsr, expression of MsrA, k(cat, expression of yeast MsrA, ratio of free to peptide Msr activity 1:0.8, expression of MsrBs and MsrAs, apparent in vivo free Msr activities in yeast cells, deletion of gene CSF1, unable to complete fermentation, overexpression of gene CSF1, decreased lag phase, reached higher populations than control strain, deletion of gene HSP104, overexpression of gene HSP104, deletion of gene TIR2, overexpression of gene TIR2, increased gap-repair efficiency, deletion of gene veA, consequence biosynthesis of sterigmatocystin, consequence inability to produce toxin, mutation in gene rtfA, consequence regain capability to produce toxin, consequence restore mycotoxin production, consequence decrease in penicillin production, consequence decrease in growth rate, consequence reduction in conidiation, consequence loss of sexual development, deletion of gene SsMNT1, inability to synthesize specific oligosaccharides, expression of gene SsMNT1, addition of up to two mannose residues to acceptor molecules, involvement in N-linked glycosylation pathway, involvement in O-linked glycosylation pathway, disruption in glycosylation pathways, deletion of CCR4-NOT complex, compromised aneuploid viability, deletion of poly(A)-specific nuclease, deletion of genes required for homologous recombination repair, improved xylose assimilation, enhanced butanol production, optimized metabolic pathways, Deletion of gene AMT1, reduced vegetative growth, increased sensitivity to membrane stress, reduced DON mycotoxin production, reduced virulence on flowering wheat heads, impaired nuclear export of FgHrp1, normal nuclear export of FgNab2, Δamt1 Δamt2 double mutants show similar defects, Deletion of gene AMT2, similar defects to Δamt1 mutant when deleted together, deletion of gene related to sulfur metabolism, increase in production of 3MH and 3MHA, decrease in production of 3MH and 3MHA, deletion of gene related to nitrogen metabolism, addition of urea, addition of di-ammonium phosphate, addition of cysteine, addition of S-ethyl-L-cysteine, deletion of gene DBP2, accumulation of noncoding transcripts, inefficient 3' end formation, appearance of aberrant transcriptional initiation products, synthetic lethal with deletion of RRP6, deletion of gene THI2, enhanced pyruvate yield, decreased ethanol titre, addition of thiamine, increased pyruvate concentration, deletion of gene ady2, consequence spore formation regulation affected, deletion of gene nce103, overexpression of gene ras, consequence increased PKA activation, overexpression of gene Cyr1, consequence reduced PKA activation, deletion of gene Gpr1, overexpression of gene Ady1, consequence proper localization of Mpc70 and Spo74, consequence initiation of spore formation, increased 2,3-butanediol production, overproduction of glycerol, reduced 2,3-butanediol production, reduced acetate production, deletion of gene SPCC1450.16c, increased TG content by 1.7-fold, deletion of gene SPAC1786.01c, deletion of gene SPAC1A6.05c, triple deletion of genes SPCC1450.16c, SPAC1786.01c, SPAC1A6.05c, increased TG content to 2.7-fold, triple deletion mutant, growth sensitive to cerulenin, restored growth defect by cerulenin, introduction of CpFAH12 gene, increased ricinoleic acid composition by 1.1-fold, increased total fatty acid content by 1.6-fold, increased ricinoleic acid production by 1.8-fold, deletion of gene FgOS-2, reduced DON production in planta, reduced ZEA production in planta, increased DON production in vitro, similar ZEA production in vitro as wild type, reduced pathogenicity towards maize, reduced pathogenicity towards wheat, inability to reach rachis node in wheat, increased resistance to fludioxonil, reduced growth on NaCl-supplemented agar, impaired conidial germination on osmotic media, swollen germ tubes containing multiple nuclei, failure to produce perithecia, failure to produce ascospores, modulation of catalase transcription and activity, regulation of noxC gene, regulation of atf1 gene, increase in flux from pentose phosphate pathway into glycolytic pathway, deletion of gene TAL1, decrease in growth on xylose, decrease in xylose-fermenting ability, deletion of gene TKL1, requirement for glucose utilization, deletion of gene NQM1, slight influence on growth on xylose, slight influence on xylose-fermenting ability, deletion of gene TKL2, impairment in rate of xylose consumption, impairment in ethanol production, overexpression of gene TKL2, deletion of gene yar1, accumulation of 20S pre-rRNA, 40S export defect, deletion of gene rps3, overexpression of gene rps3, suppression of yar1 deletion defects, enhancement of rps3 mutation phenotypes, deletion of gene Sc-HO, deletion of HDS1 domain, increased cytoplasmic localization of Sec7, deletion of Arf1, interaction of Sec7 with activated Arf1, localization of Sec7 to TGN, quicker process of mitochondrial degeneration, altered intracellular amino acid profile, laxer nitrogen catabolite repression regulation, improved fermentation activity in the first hours of fermentation, defective stress response, decreased synthesis of trehalose, decreased synthesis of glycogen, impaired G1 arrest, suppression of defective stress response, loss of function of gene RIM15, improved fermentation rate, loss of function of gene IGO1, loss of function of gene IGO2, deletion of gene STF2, promotion of reactive oxygen species production, induction of apoptotic cell death, overexpression of gene STF2, reduction in ROS production, deletion of gene Fus3, deletion of gene Sey1, reduced frequency of Chr4 disomy, deletion of gene Glo3, deletion of gene Gcs2, disruption of Sey1, elongated cytoplasmic ER, disruption of Glo3, reduced reticulation in ER, disruption of Gcs2, deletion of Sey1, expanded ER cisternae, deletion of gene MHO1, consequence double-deleted cells lacking MHO1 and PLC1 proliferate for up to ten generations, consequence no effect on invasion induced by nutrient deprivation, overexpression of gene MHO1, consequence blocks the invasive ability of yeast cells, combinatorial loss of mini-chromosomes, deletion of 12 genes, loss of mini-chromosomes, genome-wide amplifications, amplification of mitochondrial genome, amplification of genomic regions not in mini-chromosomes, deletion of gene ribosomal protein, consequence resistance to ER stress, consequence enhanced tunicamycin resistance, consequence independent of Hac1, consequence reduced translation, loss of gene YKL225W, loss of gene PAU16, loss of gene YKL223W, loss of gene YKL222C, loss of gene MCH2, loss of gene FRE2, loss of gene COS9, loss of gene SRY1, loss of gene JEN1, loss of gene URA1, loss of gene YHL050C, loss of gene YKL050W-A, loss of gene YHL049C, loss of gene YHL048C-A, loss of gene COS8, loss of gene YHLComega1, loss of gene ARN2, loss of gene YHL046W-A, loss of gene PAU13, loss of gene YHL045W, loss of gene YHL044W, loss of gene ECM34, loss of gene YHL042W, loss of gene YHL041W, loss of gene ARN1, deletion of gene pre9, decrease in boundary function, deletion of gene ump1, increased ethanol yield in medium with 30 mM acetic acid, fourfold higher specific ethanol productivity in presence of 90 mM furfural, greater than fourfold higher xylose consumption rate from rice straw hydrolysate, deletion of SIZ2, consequence increased gene silencing, overexpression of SIZ2, consequence reduced binding of Sir2 to telomeres, mutation of ESC1, consequence enhanced antagonism to gene silencing by SIZ2 overexpression, deletion of gene DPP1, increase in α-santalene titer, overexpression of gene HMGR, increase in α-santalene productivity, deletion of gene LPP1, modulation of expression of gene ERG9, diversion of carbon flux from sterol synthesis towards α-santalene, use of HXT1 promoter to control ERG9 expression, improvement in α-santalene productivity by 3.4 fold, overexpression of gene RDL2, increased tolerance to oxidative stress, deletion of gene RDL2, increased reactive oxygen species production, loss of viability at end of fermentation, upregulation of adenylate cyclase gene, higher PKA activity, removal of δ element in promoter, increased life span, higher expression of glycolytic genes, faster growth, higher level of protein degradation by autophagy, fast growth, deletion of essential gene, affects large range of other essential genes, deletion of non-essential gene, interrupts other non-essential genes, deletion of essential genes, halts essential metabolites production, deletion of gene GTR1, affected acetate production, deletion of gene GTR2, deletion of gene LIP5, deletion of gene LSM1, deletion of gene PHO85, deletion of gene PLM2, deletion of gene RTG1, deletion of gene EMI5, affected succinate production, deletion of gene SDH2, deletion of gene SDH4, deletion of gene TCM62, deletion of gene YDR379C-A, overexpression of gene LIP5, affected acetate production in S. cerevisiae, overexpression of gene EMI5, affected acetate production in S. pastorianus, overexpression of gene RTG2, overexpression of gene UBP3, affected citrate production in S. pastorianus, affected lactate production in S. pastorianus, affected succinate production in S. pastorianus, deletion of gene sulE, loss of de-esterification ability, increased sensitivity to herbicides, expression of gene sulE, ability to de-esterify sulfonylurea herbicides, increased resistance to herbicides, deletion of gene YBR246W, accumulation of diphthine, deletion of gene HpYPS1, minor influence on sensitivity to cell wall stress, deletion of gene HpYPS7, expression of gene HpYPS1, restoration of growth defects in S. cerevisiae yps1-deletion strains, expression of gene HpYPS7, limited complementation effect on S. cerevisiae yps7-deletion strain, induction by sodium dodecyl sulphate, induction by tunicamycin, elimination of aberrant proteolytic cleavage of recombinant proteins, deletion of gene sla1(+), irregularities in tRNA processing, altered distribution of pre-tRNA intermediates, increased mRNAs from amino acid metabolism genes, slow growth in Edinburgh minimal medium, sensitivity to rapamycin, deficiency of leucine uptake, hypersensitivity to NH4, genetic links to the TOR pathway, overexpression of gene los1(+), suppression of reduction in pre-tRNA levels, suppression of AAM gene up-regulation, suppression of slow growth of sla1-Δ cells, deletion of ladB, growth arrest on galactitol, significantly slower growth on D-galactose with D-xylose, deletion of xyrA, reduced growth on D-galactose, overexpression of ladB in Saccharomyces cerevisiae, production of L-xylo-3-hexulose from galactitol,  deletion of gene nde1,  deletion of gene tcm62,  deletion of gene rip1,  deletion of gene cyt1,  deletion of gene qrc8,  deletion of gene pet117,  deletion of gene cox11,  deletion of gene atp11,  deletion of gene fmc1, consequence lower respiration rate, consequence higher intrinsic superoxide radicals, increased resistance to PAF26, disrupted induction of NO, deletion of gene ARG1, reduced basal NO levels, presence of PAF26, induced intracellular production of NO, blocked NO production in Δarg1 mutant, presence of l-NAME, restored yeast growth in presence of PAF26, cloning and transformation of 4CL1 gene, resveratrol production, cloning and transformation of STS gene, deletion of PAD1 gene, no enhancement of resveratrol production, deletion of gene Cin8 loop 8 insert, consequence reduced midzone-directed motility, consequence affected ionic strength-dependent directional switching, consequence reduced efficiency to support spindle elongation, production of acetaldehyde, efficient reduction of acetaldehyde to ethanol, oxidation of produced or added ethanol, production of ethanol from glucose, expression of gene ADH3, poor growth on glucose, Deletion of gene msnA, Retarded colony growth, Increased conidiation, Slightly higher aflatoxin production, Elevated kojic acid production, Up-regulation of superoxide dismutase, Up-regulation of catalase, Up-regulation of cytochrome c peroxidase, Higher levels of reactive oxygen species (ROS, deletion of gene KNR4, increased release of mannoproteins during fermentation, autolysis continues after aging for 78 days, moderate flocculence, changes in expression of Flo1p-regulated genes, overexpression of FKS1, deletion of gene PPT1, consequence constitutive hyperphosphorylation of Hsf1p, consequence enhanced fermentation ability, overexpression of gene PPT1, consequence recovery of HSE-mediated stress response in sake yeast, decrease in ethanol production, upregulation of other isozyme genes, deletion of all known ADH isozyme genes, disruption of ethanol production, insertional mutagenesis of gene CaGIT1, abolished ability to use GroPIns as phosphate source at acidic pH, abolished transport of [(3, reintegration of gene GIT1, restored ability to use GroPIns as phosphate source, restored transport of [(3, growth under low-phosphate conditions, increased label uptake from [(3, deletion of gene CaPHO4, prevented utilization of GroPIns as phosphate source, did not affect utilization of GroPCho as phosphate source, deletion of gene MNL2, diminished degradation efficiency of misfolded CPY(*, deletion of AGT1, reversion of growth characteristics, expression of AGT1, improved sucrose-transport capacity, expression of SUC2 with modified promoter and 5' coding sequences, predominant cytosolic localization of invertase, overexpression of AGT1, elevated sucrose-transport capacity, deletion of gene yfh1, consequence Iron homoeostasis was improved, consequence Soluble/usable mitochondrial iron was increased, consequence Accumulation of insoluble/non-usable iron within mitochondria was prevented, consequence Cytochromes were returned to normal, consequence Haem synthesis was restored, consequence Fe-S cluster enzyme activities were improved, consequence Efficiency of new Fe-S cluster synthesis was markedly increased, deletion of gene RNT1, increased expression of Snf3p-Rgt2p-associated transcription factors, promoter-specific transcription of glucose sensing genes, consequence reduced glycerol production, deletion of gene GPD1 and GDH1, consequence lowest glycerol concentration of 2.31 g/L, consequence 46.4% lower glycerol concentration than wild-type strain, consequence slight improvement in ethanol yield to 0.414 g/g, introduction of E. coli genes, improved higher alcohol productivity, deletion of gene Pdr1p, overexpression of gene Pdr5p, overexpression of gene Snq2p, consequence 1: endocytic patches carrying Abp1 and Las17 persisted much longer, consequence 2: scission efficiency was greatly affected, consequence 3: frequency of formation of endocytic sites carrying Rvs161- and Rvs167-GFP was affected, consequence 4: mistargeting of synaptojanins Sjl1 and Sjl2 to the cytoplasm, consequence 5: severe motility defect of Abp1-GFP during its internalization in a codeletion with SJL2, deletion of gene Sc-ILV6, reduction of diacetyl concentration in green beer, protein degradation, expression of gene Flp recombinase, deletion efficiency of 96%, basal activity of GAL1 promoter, pre-excision frequency of 2%, spontaneous recombination between FRT sites, induction of gene GTS1, cell aggregation, protein aggregation, ROS production, apoptosis, deletion of gene POR1, altered ROS production profile, altered apoptosis profile, deletion of gene efflux pumps, consequence hypersensitivity against several compounds, mutation of gene RME1, decreased expression of gene frataxin, oxidative damage of mitochondria, iron accumulation in mitochondria, iron-sulfur cluster defects, high sensitivity to oxidative stress, treatment with idebenone, improvement in neurological functions, reduction of ROS production, decrease in mitochondrial mass, deletion of gene STB3, partial bypass of growth defects, deletion of gene DOT6, deletion of gene TOD6, phosphorylation of Stb3 by Sch9, inhibition of Stb3, phosphorylation of Dot6 by Sch9, inhibition of Dot6, phosphorylation of Tod6 by Sch9, inhibition of Tod6, dephosphorylation of Stb3, enabling recruitment of RPD3L, dephosphorylation of Dot6, dephosphorylation of Tod6, recruitment of RPD3L, repression of Ribi/RP gene promoters, Sch9 regulation, promotion of ribosome biogenesis, promotion of ribosomal protein gene expression, upregulation of Pdr5p, diclofenac tolerance, deletion of PKC pathway components, increased diclofenac toxicity, incubation with calcofluor white, decreased diclofenac toxicity, downregulation of ribosome biogenesis genes, effect on diclofenac tolerance, downregulation of zinc-responsive genes, deletion of Zap1p, decreased fermentative ability, decreased succinate production, expression of gene SOR1, increased isoamyl alcohol production, expression of gene srlD, increased 2-phenyl alcohol production, reduced ethyl acetate concentration, increased isobutyric acid, improved product yield by 2~4 folds, gene knockout, deletion of gene SHB17, decrease in sedoheptulose-1,7-bisphosphate hydrolysis, disruption in riboneogenesis pathway, overexpression of gene SHB17, increase in flux through SHB17, increase in ribose-5-phosphate production, deletion of gene CHD1, abolition of PHO5 gene expression, deletion of gene bug1, reduced efficiency of [PSI+] induction, deletion of gene bem1, deletion of gene arf1, deletion of gene las17, inhibited [PSI+] induction and ring formation, deletion of gene vps5, deletion of gene sac6, deletion of gene Hub1, consequence cannot use certain non-canonical 5' splice sites, consequence defective in alternative SRC1 splicing, deletion of gene chc1, higher levels of intracellular glutathione, deletion of gene cst6, deletion of gene ddc1, deletion of gene def1, deletion of gene pep12, deletion of gene rts1, deletion of gene ubp6, deletion of gene yih1, overexpression of gene DEF1, higher production of glutathione, overexpression of gene CYS4, overexpression of gene GSH1, combination of overexpression of GSH1 and deletion of one of the eight genes, multiplier effect on activation of glutathione synthesis, deletion of def1, higher levels of intracellular methionine, deletion of pep12, deletion of ubp6, overexpression of DEF1, higher levels of oxidized glutathione, overexpression of GSH1, increased glutathione production in Candida utilis, deletion of gene YFR017C, reduction in ability to accumulate glycogen during aerobic growth, reduction in ability to accumulate glycogen during fermentation, expression of gene YFR017C, inhibition of Gdb1p activity, enhancement of glucose storage as glycogen, expression of gene mvaK1, synthesis of β-carotene, complementation of Erg12 function, expression of carotenogenic genes of Xanthophyllomyces dendrorhous, deletion of gene erg12, requirement for MvaK1 function, high plasmid stability, stable carotenoid production, reduced 4MMP production, overexpression of gene IRC7, increased 4MMP production, increased 3MH production, increased 3MHA production,  deletion of gene BAR1, failure to display a distinct sterol-rich domain at the hyphal tip, non-pathogenic when inoculated onto wheat heads, severe perturbation in in vitro growth, incapability of producing perithecia, production of macroconidia only in the presence of NaCl, necessity for the production of glucosylceramides,  deletion of glucosylceramide synthase gene, resistance to heat stable antifungal factor (HSAF, expression of gene Pc22g24830, production of oxalic acid, deletion of gene Pc22g28430, elimination of oxalate production, improvement of ad-6-APA yields, impact on higher alcohol production, deletion of gene yyy, deletion of gene zzz, concerted evolution, rate and dynamics of rDNA repeats studied, mutation in spacer of rDNA unit, monitored fate without selectable marker, duplication and deletion events, occur frequently, large deletions, increase homogenization efficiency, deletion of gene YKU70, affects telomere capping, mutation in gene CDC13, deletion of gene in NMD pathway, suppresses cdc13-1 mutation, enhances yku70Δ mutation, increased levels of Stn1, affects growth of cells with cdc13-1 mutation, affects growth of cells with yku70Δ mutation, deletion of gene RPL12A, slow-growth phenotype, deletion of gene RPL12B, suppression of PHO84 transcription, decrease in phosphate transport activity, decrease in acidic phosphatase activity, addition of gene RPL12A, lessening of suppression of PHO84 transcription, addition of gene RPL12B, increase in phosphate transport activity, increase in acidic phosphatase activity, deletion of gene MATE2, disappearance of leaf anthocyanin pigmentation, pale flower color, drastic decreases in the levels of various flavonoids, increase in seed proanthocyanidin biosynthesis, malonylation of flavonoid glucosides, increased affinity for uptake by MATE2, increased transport efficiency for uptake by MATE2, deletion of gene CnSEC14-1, consequence CnPlb1 secretion is dependent on CnSec14-1p, consequence hypovirulent in mice, consequence did not disseminate to the CNS by day 14 post infection, consequence reduced macrophage expulsion of live cells, deletion of gene CnSEC14-2, consequence did not abrogate CnPlb1 secretion or virulence, deletion of gene CnSFH5, reconstitution of gene CnSEC14-1 with CnSEC14-1, consequence restored CnPlb1 secretion and virulence, reconstitution of gene CnSEC14-1 with CnSEC14-2, expression of gene CnPlb1, consequence essential for dissemination to the CNS, consequence essential for vomocytosis, overexpression of gene YCL056c/Pex34p, increased numbers of peroxisomes, deletion of gene PEX34, reduced peroxisome division, interaction of Pex34p, Pex11p, Pex25p, Pex27p, deletion of gene SPT15, improved survival upon ethanol shock, deletion of gene FPG1, consequence foamless strain, overexpression of gene FPG1, consequence foam hyper-producing strain, deletion of Pho8, increase in internal concentration of NmR, activation of PHO pathway, increase in release rate of NmR, deletion of Fun26, change in size of vacuolar and cytosolic NmR pools, depletion of NaMN, mediation of NMN catabolism, contribution to NmR salvage, contribution to phosphate acquisition, consequence heat tolerance, co-expression of gene SSA1, consequence inhibition of Gts1p nuclear import, co-expression of gene SSA2, deletion of gene SSA1 and SSA2, consequence enhancement of Gts1p-mediated heat tolerance, consequence increased production of Ssa1p and Ssa2p, deletion of gene hacA, consequence UPR-deficient strain, consequence reduced virulence in Alternaria brassicicola, overexpression of gene hacA, consequence constitutively activated UPR strain, activation of gene HAC1, consequence increased translation of Hac1p, splicing of HAC1 mRNA, consequence relief of translational block in HAC1 mRNA, stress-induced splicing of unconventional intron, consequence activation of UPR pathway, expression of UPR target genes, consequence regulation of folding, yield, and delivery of secretory proteins, deletion of gene RPA49, disappearance of nucleolar structure, decrease in Pol I loading rate per gene, decreased contact between adjacent Pol I complexes, restoration of nucleolar assembly by decreasing ribosomal gene copy number, deletion of gene VMA1, five- to sevenfold elevated riboflavin production, twofold decreased iron cell content, deletion of gene FRA1, 1.8-2.2-fold more iron accumulation, five- to sevenfold more riboflavin production, inability to grow at 37°C, deletion of gene FES1, SAK1 overexpression, upregulation of glucose-repressed genes, HAP4 overexpression, reduced glycerol formation, increased growth rates, lesser impact on growth rates, increased biomass yields, no increase in biomass yields, restored growth on ethanol in sdh2 deletion strain, combined SAK1 and HAP4 overexpression, no restoration of growth on ethanol in sdh2 deletion strain, HAP4 overexpression alone, sdh2 deletion, production of succinic acid, SAK1 and HAP4 overexpression, alleviation of glucose repression, shift in fermentative/oxidative balance, deletion of gene YHR087w, lower levels of yeast Hsp proteins, overexpression of gene EXG1, consequence β-glucosidase activity, overexpression of gene SPR1, overexpression of gene YIR007W, consequence loss of capability to hydrolyze flavonoid glucosides, expression of gene Asr1, consequence increased ethanol tolerance, deletion of gene Asr1, consequence hypersensitivity to ethanol, deletion of YCA1, cell death response, deletion of genes involved in mitochondrial electron transport chain, propolis sensitivity, deletion of genes involved in vacuolar acidification, deletion of genes involved in negative regulation of transcription from RNA polymerase II promoter, deletion of genes involved in regulation of macroautophagy associated with protein targeting to vacuoles, deletion of genes involved in cellular response to starvation, deletion of gene vps3, enhanced endoglucanase activity, deletion of gene vps16, enhanced β-glucosidase activity, deletion of gene rav1, increased endoglucanase activity when combined with vps3 deletion, increased endoglucanase activity when combined with rav1 deletion, expression of gene OrdA, HOMST is produced from OMST, HOMST is a precursor of AFB(1, HOMST is a precursor of AFG(1, expression of gene CypA, involved in conversion of HOMST to AFG(1, expression of gene NadA, lower glycerol yield, sensitivity to osmotic stress, failure to ferment on 25% glucose, over-expression of gene TPS1, restored osmotic stress tolerance, over-expression of gene TPS2, improved osmotic stress tolerance, further reduced glycerol yield, expression of gene GAPN, deletion of gene myo1, consequence viable and dividing strains, consequence absence of cytokinetic ring, consequence activation of PKC1-dependent cell wall integrity pathway, consequence 1,301 differentially regulated ribosome-bound mRNAs, consequence 204 co-regulated by transcription and translation, consequence down and up regulation of ribosomal protein and cell wall biosynthesis mRNAs, consequence accumulation of phosphorylated form of eukaryotic translation initiation factor 2 (eIF2α-P, consequence reduction in steady state levels of translation initiation factor eIF4Gp, deletion of gene gcn2 in myo1Δ, consequence abolished eIF2α-P phosphorylation, consequence severe growth defect, consequence presence of P-bodies, consequence selective co-precipitation of RP mRNAs with eIF2α-P, consequence selective regulation of mRNA translation efficiency, consequence increase in levels of eIF2α-P, consequence genetic interaction between GCN2 and MYO1, consequence other signaling pathways important for survival, deletion of gene MER1, delay in expression of Mer1p-responsive genes, overexpression of gene UME6, induction of MER1 mRNA, induction of Mer1p substrate pre-mRNAs, reduction in splicing of Mer1p substrate pre-mRNAs, overexpression of gene MER1, activation of Ndt80p, deletion of gene NAM8, disruption of splicing activation by Mer1p, deletion of gene CRK1, consequence 1: increased mating efficiency, consequence 2: earlier production of dikaryotic filaments, consequence 3: earlier production of basidia, consequence 4: earlier production of basidiospores, overexpression of gene CRK1, consequence 1: inhibited mating, consequence 2: defective monokaryotic fruiting, deletion of growth signaling pathways, reduced superoxide anions, enhanced longevity, more efficient G0/G1 arrest, downregulation of growth signaling pathways, caloric restriction, elevated levels of H₂O₂, activates superoxide dismutases, high glucose, increased superoxide anions, mutations activating growth signaling, activates DNA damage responses, preferentially kills stationary phase cells failing to arrest growth in G0/G1, deletion of gene MGA2, elevated relative amounts of squalene, reduced basal expression of ERG1, deletion of gene SPT23, no impact on squalene levels, no effect on response to miconazole treatment, reduced ERG1 expression comparable to mga2Δ/spt23Δ strain, deletion of gene UME6, failure to germinate, reduced germination efficiency, inviable cells in microcolonies, transient impact on cell viability, deletion of gene OLE1, reduced growth without unsaturated fatty acids, reduced production of unsaturated fatty acids, impaired pseudohyphal formation, increased sensitivity to salts, increased sensitivity to H2O2, increased sensitivity to human serum, increased sensitivity to fetal bovine serum, enhanced killing by macrophages, increased production of nitric oxide by macrophages, reduced fungal virulence in murine infection, deletion of gene Ste50, deletion of gene CRG1 and Ste50, reduced survival in drying process, activation of over 450 genes, increased survival in drying process, down-regulation of 330 genes, overexpression of autophagy processes, clearance of protein aggregates/damaged organelles, recycling of nutrients, activation of functional carbon source sensing networks, increased survival of desiccation, activation of PKA, Tor and Snf1 protein kinase complexes, increased survival of desiccation, changes in nitrogen metabolism, coping with drying process, activation of environmental stress response, repression of ribosome production, induction of genes for coping with oxidative stress, induction of genes for coping with osmotic stress, deletion of gene iron accumulation, consequence drop in energy production, consequence decrease in activity of enzymes involved in protein protection and regeneration, overexpression of oxidative stress, consequence protein carbonylation, consequence damage of glycolytic enzymes, oxidation of peroxiredoxin (TSA1), consequence impact on enzymatic activities, oxidation of thioredoxin II (TRX2), deletion of gene mid2, transcriptional response to Congo Red reduced, deletion of gene wsc1, transcriptional response to Caspofungin reduced, little effect on transcriptional response to Zymolyase, activation of gene sho1, transcriptional response to Zymolyase increased, synthetic lethal with gene yyy, synthetic growth defect with gene yyy, buffered by gene yyy, within pathway interaction with gene yyy, suppressive interaction with gene yyy, reduction in acetate production, deletion of gene MIG2, increase in pyruvate concentration, increase in glycerol concentration, consequence formation of extrachromosomal DNA circles containing GAP1, consequence production of a single hybrid LTR at the GAP1 chromosomal locus, overexpression of gene GAP1, consequence increased frequency of GAP1(circle, consequence selection under various non-amino acid nitrogen limitations including ammonium, urea, and allantoin limitation, increased H(2, deletion of gene SKP2, increased SO(2, Met14p more stable, slower growth under nutrient-limited conditions, lower concentration of intracellular cysteine, Str2p more stable, Str3p more stable, Str4p more stable, overexpression of gene STR1, severe growth defect in skp2 mutant, overexpression of gene STR4, deletion of gene YDL080C, no noticeable effect on decreasing the production of higher alcohols, deletion of gene FgStuA, reduced pathogenicity on wheat heads, impaired spore production, lack of perithecia and sexual ascospores, lack of conidiophores and phialides, delayed production of aberrant macroconidia, reduced production of secondary metabolites, differential expression of thousands of genes during asexual sporulation and infection of wheat heads, differential expression of genes under conditions inducing secondary metabolites, interruption of TCA cycle after succinate, interruption of TCA cycle after isocitrate, deletion of gene IDP1, deletion of genes SDH1, SDH2, IDH1, IDP1, production of succinic acid at titer 3.62 g L(-1, SDH2, IDH1, IDP1, succinic acid yield of 0.11 mol (mol glucose, (deletion of genes SDH1, SDH2, IDH1, IDP1, succinic acid not accumulated intracellularly, overexpression of gene pGcyaDak, overexpression of gene pGup1Cas, provided osmotic protection, deletion of individual genes, change in sporulation efficiency, discovery of robustness and hysteresis in the yeast sporulation network, double gene deletion, patterns of sporulation efficiency change, decrease in glycerol production, expression of gene GAPDH, decrease in maximum specific growth rates, increase in maximum specific growth rates, deletion of gene HMT1, deregulates the recruitment of U1 snRNP, reduces binding of splicing factors to intron-containing genes, increases splicing efficiency of developmentally regulated mRNAs, increases association between Snp1 and Npl3, deletion of gene Cdc13, deletion of gene Exo1, deletion of gene Rad9, deletion of gene RAD9, deletion of gene CDC13, deletion of gene ALS3, expression of gene Als3p, deletion of gene sspA sspB, expression of gene SspB, increased sensitivity to salicylaldehyde, deletion of gene sod2, deletion of gene glr1, deletion of gene vph2, deletion of gene stb5, consequence auxotrophy for methionine, consequence auxotrophy for pyrimidine nucleotides, consequence impaired aerobic NADPH status, consequence reduced acetate level, consequence increased α-ketoglutarate level, consequence reduced flux through the pentose phosphate pathway, overexpression of gene stb5, consequence growth defect on glucose, consequence excess NADPH in the oxidative pentose phosphate pathway, deletion of gene IXR1, increase in cell resistance to UV light, increase in cell resistance to gamma-rays, increase in cell resistance to MMS, increase in spontaneous mutagenesis, decrease in UV-induced mutations, mutation of gene HMO1, increase in sensitivity to cisplatin, no effect on sensitivity to UV light, increase in UV-induced mutagenesis, double mutation hmo1 ixr1, additive interaction in lethal effect of cisplatin, additive interaction in lethal effect of UV light, additive interaction in UV-induced mutagenesis, mutation of gene IXR1, increase in spontaneous mutagenesis mediated by replication errors, increase in repair mutagenesis, combined activity of Hmo1 and Ixr1 proteins, efficient correction of repair errors, efficient correction of replication errors, deletion of gene BCS1, arrest of bc(1, formation of functionally inactive bc(1, expression of gene BCS1, rescue of bc(1, formation of functional homodimeric bc(1, conversion into supercomplexes with cytochrome c oxidase complex, mutation R81C in gene BCS1, restoration of bc(1, mutation R45C in human ortholog BCS1L, severe bc(1, mutation K192P in gene BCS1, inability to recover bc(1, mutation F401I in gene BCS1, introduction of gene xyd1, D-xylonate production, D-xylonate export to supernatant, reduction in xylitol production, increase in D-xylonate yield from D-xylose, reduction in D-xylose uptake, co-expression of gene udhA with xyd1, enhancement in NADPH consumption, co-expression of gene gapB with xyd1, co-expression of gene GDH2 with xyd1, deletion of gene GAS1, overexpression of gene GAS3, reduced growth rate, hyperaccumulation of Gas3p in membranes, hyperaccumulation of Gas3p in cell wall, overexpression of gene GAS3 in gas1Delta mutant, exacerbated defects, mutation of gene GAS3 to Gas3p-E283Q, deleterious to gas1Delta cells, overexpression of carotenogenic genes, increased carotenoid levels, induction of genes involved in pleiotropic drug resistance, deletion of pdr10, decreased transformation efficiency, decreased growth rates, lower carotenoid production levels, decrease of translation termination efficacy, increase in mRNAs half-lives, longer mRNA poly(A, integrity of Tpa1 active site, essential for correct translation termination, presence of Yor051c/Ett1, Tpa1, represses expression of genes regulated by Hap1, Hap1, regulates genes essential for cell response to hypoxia, deletion of gene CLB1, consequence minor effects on mitosis, deletion of gene CLB3, deletion of gene CLB4, overexpression of gene CLB2, consequence sufficient for essential CDK functions in mitosis, modification of gene CLB2 with Myc12 epitope, consequence loss of spindle assembly function, consequence no defects in late mitotic functions, consequence induction of switch from polarized to isotropic growth similar to endogenous Clb2, deletion of gene PYC, inability to grow on glucose without C(4, overexpression of gene PCK1, no growth on glucose as sole carbon source, mutation in gene PYK, enables growth on glucose using PEPCK as sole anaplerotic enzyme, reaction catalyzed by PEPCK, production of ATP, original route through pyruvate kinase and pyruvate carboxylase, ATP neutral, expression of gene HMG2, increase in squalene pool, increase in lanosterol pool, elevated transfer of surplus squalene into C27 sterols, deletion of gene tesB, decreased 3-hydroxybutyric acid, decreased 3-hydroxyhexanoic acid, decreased 3-hydroxyoctanoic acid, decreased hexanoic acid, replacement of gene tesB with PTE1, increased proportion of 3-hydroxybutyric acid, replacement of gene tesB with tesB, increased proportion of 3-hydroxyhexanoic acid, increased proportion of 3-hydroxyoctanoic acid, deletion of gene prb1, increase of MGF yield, expression of gene MGF, low yield due to rapid degradation, fivefold increase of MGF yield, deletion of genes affecting mitochondrial function, increased sensitivity to BiQs, deletion of genes affecting cellular respiration, over-expression of NDE1 gene, increased sensitivity of wild type yeast to BiQ, deletion of gene multidrug resistance genes, consequence multidrug resistance, consequence buffers against phenotypic variation, deletion of gene Rcy1, consequence boosts chemical resistance, deletion of gene Wss1, deletion of gene Ubp16, deletion of gene Irs4, consequence boosts peroxisome recycling, deletion of gene NPR1, consequence boosts phosphorylation cascades, overexpression of gene mitochondrial protein synthesis sub-network, consequence explores alternative survival strategies, overexpression of gene acetyltransferases, consequence supports secondary metabolism, overexpression of gene NTN hydrolase, deletion of gene aclA, growth is greatly diminished on glucose, growth is greatly diminished on proline, deletion of gene aclB, growth is not affected on acetate, growth is not affected on ethanol, addition of acetate restores growth on glucose, addition of acetate restores growth on proline, transcription of aclA and aclB is repressed by growth on acetate, production of asexual spores is greatly reduced, complete absence of sexual development, deletion of gene p66(Shc), extends lifespan, displays protective effects in cardiovascular disease, activation of gene SIRT1, deacetylates p53, deacetylates endothelial nitric oxide synthase, deacetylates forkhead box protein, deletion of gene involved in cell wall integrity/structure, increase in sensitivity to Echinacea, increase in sonication-associated cell death, increase in cell wall damage, deletion of TOR, consequence degradation of Rrn3p, consequence reduction in initiation competent Pol I-Rrn3p complexes, overexpression of non-degradable Rrn3p, consequence increase in Pol I-Rrn3p complexes, adjustment of Rrn3p levels, consequence influence on rRNA gene transcription, overexpression of gene Stb3, slow growth phenotype, reduced expression of non-RP genes, drop in the rate of amino acid incorporation, loss of gene Hos2, reversal of Stb3 overexpression slow growth phenotype, overexpression of gene SCH9, deletion of gene PPH22, inhibition of Tor signaling, increased severity of Stb3 overexpression phenotype, restriction of Stb3 to the nucleus, increase in NER efficiency, presence of Sir2, inability to stimulate histone H4 acetylation, inability to stimulate histone H3 acetylation, UV radiation, stimulates histones H3 acetylation, stimulates histones H4 acetylation, deletion of TorC1, consequence NCR-sensitive transcription is derepressed, overexpression of Sit4, consequence nuclear Gln3 localization, overexpression of Pph21/22-Tpd3-Cdc55/Rts1, consequence nuclear Gat1 localization, consequence weak response to Rap treatment, consequence strong response to Rap treatment, overexpression of Msx, consequence strong response of Gln3 localization, consequence immunity of Gat1 localization, deletion of gene sed1, consequence 1: higher BGL-activity, consequence 2: maintains wild type growth, consequence 3: influences the activity of BGL on the cell surface, deletion of gene tos6, deletion of SED1 ORF, consequence 1: enhances BGL activity on the cell surface, deletion of SED1 promoter, consequence 1: increases further BGL activity on the cell surface, homozygous deletion of SED1 gene and promoter, consequence 1: most efficient display of BGL, consequence 2: 1.6-fold increase of BGL-activity compared to wild type, deletion of SEC59 ortholog, supports normal growth, supports N-glycosylation, supports GPI anchor synthesis, expression of Gl-UPPS, makes dehydrodolichol, deletion of gene MSS116, affects splicing of mitochondrial catalytic introns, expression of mutated MSS116, affects ATP hydrolysis efficiency, affects helix unwinding efficiency, inefficient ATP hydrolysis by MSS116, impairs splicing function, efficient helix unwinding by MSS116, does not ensure splicing function, deletion of gene Gre2p, lacks activity towards 2,5-hexanedione reduction, expression of gene Gre2p, affords (2S,5S, results in high volumetric productivity of 70 g L(-1, deletion of bacterial genome, high background of 5-FOA resistant clones, expression of I-SceI, generation of double-strand break, promotion of intra-molecular homologous recombination, intra-molecular homologous recombination, excision of CORE, seamless modification of bacterial genome, deletion of gene ANP1, increased susceptibility to high-glucose stress, deletion of gene GCR1, deletion of gene involved in vacuolar function, deletion of gene involved in cell wall biogenesis, deletion of gene involved in regulation of transcription, deletion of genes involved in mitochondrial functions, hypersensitivity to CTBT, deletion of genes involved in DNA repair, deletion of genes involved in transcription, deletion of genes involved in chromatin remodeling, deletion of genes involved in oxidative stress response, CTBT treatment, rapid induction of oxidant and stress response defense genes, Yap1 accumulation in the nucleus, acute oxidative stress, superoxide production associated with mitochondria, deletion of gene subset, optimization of downstream gene activity, optimization of metabolite activity, deletion of gene Cat8p, reduced respiration, deletion of gene Rtg-proteins, reduced respiration under partially respiratory conditions, deletion of gene Hap-complex, reduced respiration under fully respiratory conditions, deletion of gene eIF4G1, consequence impaired growth, consequence reduced global translation initiation rates, consequence lower levels of total eIF4G, consequence preferential effect on mRNAs with longer poly(A, impaired shmoo formation, low cell fusion efficiency, supplementation with ergosterol, partially suppressed shmooing defect, 28)-tetraenol, restored shmooing to wild-type levels, restored cell fusion to wild-type levels, deletion of gene cafA, growth defect at low CO2, deletion of gene cafB, deletion of gene canB, overexpression of gene cafB, rescue of high CO2-requiring phenotype, overexpression of gene canA, overexpression of gene canB, deletion of gene cafC, deletion of gene cafD, deletion of gene GNA1, requirement for GlcNAc supplement, expression of gene NGT1, rescue of gna1Delta strain, expression of gene NAGK, metabolic replacement of GlcNAc by GlcNAz, incorporation into cell-surface glycans, incorporation into secreted glycoproteins, metabolic replacement of GlcNAc by GlcNAl, deletion of gene Aac2p, consequence growth defect, consequence dependence on SAL1 gene product, deletion of gene SAL1, consequence impact on mitochondrial ATP-related reactions, expression of gene SAL1, consequence rescue of growth in Deltaaac2 mutants, consequence maintenance of intramitochondrial ATP pool, deletion of gene rrm3, increased HTA2-HTB2 amplification, deletion of gene dpb3, deletion of gene dpb4, deletion of gene clb5, deletion of gene hht2-hhf2, reduced histone H3-H4 levels, increased replication fork pauses, impaired replication forks, reduced histone levels, introduction of DNA replication stress by hydroxyurea, induced HTA2-HTB2 amplification, deletion of gene ZbGAS1, higher release of Candida rugosa lipase CRL1, higher release of Yarrowia lipolytica protease XPR2, no significant effect on human IL-1beta secretion, knock out of cytochrome respiratory pathway, induction of alternative oxidase, reduction in free radical production, disruption of PaNdk1, severe somatic and sexual defects, hypersensitivity to hydrogen peroxide, hypersensitivity to paraquat, increase of mtDNA stability, increase of lifespan, EXO5 deletion, amplifying either the ori3 or ori5 region of the mitochondrial genome, yield petites, mitochondrial genome instability, continuous rearrangement of mitochondrial genome, deletion of gene sco1, decreased Ty1 retrotransposition, overproduction of reactive oxygen species, increased Ty1 retrotransposition, treatment with methylmethane sulphonate, increased superoxide levels, increase of superoxide anions, induction of Ty1 mobility, disruption of gene yap1, increased intracellular levels of hydrogen peroxide, exogenous addition of hydrogen peroxide, higher frequencies of retrotransposition, deletion of gene SPT6, severe impairment of growth, impaired capacity to undergo filamentous growth, overexpression of gene SPT6, partial complementation in S. cerevisiae spt6 mutant, increased viability, no improvement in overall strain performance, higher energy demand for maintenance, consequence BER deficiency, consequence reduced gap-filling activity, consequence affected long-patch BER, consequence affected short-patch BER, consequence involvement downstream of AP-lyases Ntg1 and Ntg2, consequence missing effect of aphidicolin on BER efficiency, deletion of gene H2B residues 30-37, consequence reduced NER efficiency at RPB2 locus, consequence reduced NER efficiency at HML locus, consequence reduced NER efficiency at GAL10 locus, consequence increased nucleosome accessibility, consequence increased nucleosome mobility, consequence increased DNA damage acquisition, consequence reduced Snf5 recruitment to HML locus, consequence lower binding affinity of SWI/SNF complex to nucleosomes, deletion of N-terminal region of rpS5, moderately reduced growth rates, deletion of 46 N-terminal amino acids of rpS5, doubling time of about 3 h, impact on the ability of 40S subunits to function properly in translation, affect the efficiency of initiation, impact on the recruitment of initiation factors eIF3 and eIF2, deletion of gene TGL2, reduction in cellular TAG hydrolysis, overproduction of gene TGL2, increase in TAG degradation, absence of lipolytic activity in mitochondrial fraction, overexpression of gene TGL2, strong lipolytic activity in mitochondrial fraction, increased sensitivity to antimitotic drug, presence of oleic acid, complementation of antimitotic drug sensitivity in tgl2 null mutation, deletion of gene product functions in chromatin regulation, reduced fitness pleiotropy, deletion of gene regulated by chromatin state, deletion of gene product functions as a transcription factor, deletion of gene containing TATA-box, no significant effect on fitness pleiotropy when CRE is controlled, deletion of gene product central in protein interaction network, deletion of gene GGC1, enhanced growth fitness in presence of rapamycin, overexpression of gene GGC1, stabilization of GGC1 protein in presence of rapamycin, disruption of ribosome biogenesis regulation, disruption of cell cycle progression regulation, deletion of gene involved in homeostasis of protons, sensitivity to NiSO(4, deletion of gene involved in cation transport, deletion of gene involved in transport ATPases, deletion of gene involved in siderophore-iron transport, deletion of gene involved in homeostasis of metal ions, deletion of gene involved in diphthamide biosynthesis pathway, deletion of gene involved in functioning and transport of the vacuole and lysosome, resistance to NiSO(4, deletion of gene involved in protein targeting, deletion of gene involved in sorting, deletion of gene involved in translocation, deletion of gene involved in intra-Golgi transport, deletion of gene involved in regulation of C-compound and carbohydrate metabolism, deletion of gene involved in transcriptional repression, deletion of gene involved in chromosome segregation/division, inability to grow anaerobically, expression of gene mhpF, restoration of anaerobic growth, conversion of acetate to ethanol, reduction of NADH by acetate, deletion of gene GPD2, elimination of glycerol production, deletion of gene GPD2, inability to grow anaerobically, increased acclimation, improved survivability, increased glucose utilization, expression of gene CLM1, production of longiborneol, disruption of gene CLM1, inability to produce culmorin, addition of longiborneol in CLM1 disruptants, conversion to culmorin, requirement for culmorin biosynthesis, deletion of gene cre1, no significant change in flux distribution, induction of cellulase genes expression, no alteration in flux distribution, deletion of gene amyR, consequence poor growth on starch medium, deletion of gene malR, consequence loss of malP and malT expression, consequence restricted growth on maltose medium, overexpression of gene malT, consequence increase in intracellular alpha-glucosidase activity, deletion of gene malP, consequence reduced expression of malT and malR, consequence growth defect on maltose, deletion of gene yap1, decreased rate of reduction of coniferyl aldehyde to coniferyl alcohol, deletion of gene atr1, deletion of gene flr1, overexpression of gene atr1, improved resistance against inhibitors in lignocellulose hydrolysates, overexpression of gene flr1, overexpression of gene yap1, deletion of gene SSX2 NLS, consequence 1: improved secretion of SSX2, overexpression of gene SSX2, consequence 1: enhanced intracellular SSX2 levels, expression of gene SSX2 with NLS, consequence 1: accumulation in the endoplasmic reticulum, consequence 2: misfolding in the endoplasmic reticulum, consequence 2: correct folding of SSX2 inside the ER, deletion of gene yca1, suppression of TBT-induced cell death, exposure to TBT, production of reactive oxygen species, TBT exposure, induction of apoptosis-like cell death, deletion of gene pdr5, increased sensitivity to TBT, deletion of gene RRG1, deletion of gene RRG2, deletion of gene RRG3, deletion of gene RRG4, deletion of gene RRG5, deletion of gene RRG6, deletion of gene RRG7, deletion of gene RRG8, deletion of gene RRG9, deletion of gene RRG10, deletion of 16 genes, compromised maintenance of mitochondrial genome, deletion of 88 genes, impaired mitochondrial protein translation, deletion of 10 genes, impaired expression of specific mitochondrial gene products, deletion of genes involved in cytochrome c oxidase assembly, increased sensitivity to aging, increased L-lactate productivity, overexpression of gene human LDH, overexpression of gene bovine LDH, deletion of 289 genes, altered L-lactate productivity, deletion of gene Gpx3, consequence 1: increased DHA-mediated stress, overexpression of gene Gpx3, consequence 1: upregulation of Dak1 activity, consequence 2: suppression of AGE formation, deletion of gene encoding heme oxygenase, protracted metabolic cycles, expression of heme oxygenase, generation of carbon monoxide, administration of CO, phase advancement into the oxidative, respiratory phase, deletion of gene MSN4, deletion of acyltransferases, lack of detectable neutral lipids, lack of cytoplasmic lipid droplets, sensitivity to unsaturated fatty acids, expression of human diacylglycerol acyltransferase 2, reversal of phenotypes, fatty acid-mediated death, apoptotic cell death, transcriptional induction of stress-response pathways, elevated oxidative stress, diacylglycerol kinase-mediated phosphatidic acid biosynthesis, modifier of lipotoxicity, production of phosphatidylcholine via methylation of phosphatidylethanolamine, ablation of phospholipid and triglyceride biosynthesis, increased sensitivity to saturated fatty acids, normal sphingolipid biosynthesis, required for optimal growth, vesicular transport, denudation of triglyceride biosynthesis, mediated resistance to exogenous fatty acids, increase in TCA cycle flux, deletion of gene grr1, decrease in glucose uptake rate, low glucose uptake rate, increase in TCA cycle activity, unfavorable pH values, sodium chloride stress, overexpression of gene RPI1, consequence 1: prolong survival of yeast cells during fermentation, consequence 2: retain viability by over 50-fold better than wild type after 36 hours of fermentation at 38 degrees C, deletion of gene RPI1, consequence 1: drastic reduction in viability during fermentation, consequence 3: fortify yeast cell wall, consequence 4: increase resistance to cell lytic enzyme zymolyase, consequence 5: facilitate economical production of ethanol, deletion of gene PSD1, loss of PSD activity in mitochondria, severe growth defect on minimal media, depletion of cellular phosphatidylethanolamine levels, depletion of mitochondrial phosphatidylethanolamine levels, supplementation with ethanolamine, compensation for PSD1 deletion, inability of Psd2p to compensate, expression of gene Psd1p, synthesis of unsaturated phosphatidylethanolamine species, expression of gene Psd2p, phosphatidylethanolamine species containing palmitic acid, preferential assembly into mitochondria, deletion of gene Asf1, increased GCR rates, sensitivity to hydroxyurea, sensitivity to methyl methanesulfonate, deletion of gene Rad6, suppression of increased GCR rates in asf1 mutants, deletion of gene Siz1, deletion of gene Srs2, increased dependency on Dun1, cooperation with Rad6 in ubiquitination of PCNA, deletion of gene Aovps5, higher activity of tripeptidyl peptidase, higher amount of nitrogenous constituent in wheat gluten degradation, increase in available NADPH, increase in cubebol titer, decrease in maximum specific growth rate, over-expression of gene GDH2, improvement in growth rate, Lsm1 over-expression, inhibits growth, depletes U6 snRNA, alters pre-mRNA splicing, decreases U6 snRNA levels, reduces availability of Lsm2-7 proteins, displays minor alterations in mRNA decay, increases susceptibility to mutations inhibiting cytoplasmic deadenylation, consequence yeast cell death on minimal medium, overexpression of gene R282W p53 mutant, consequence inhibition of yeast growth on minimal medium, consequence induction of ROS production, mutation of gene p53, consequence inactivation of p53 through gross deletion, consequence inactivation of p53 through nonsense mutations, consequence inactivation of p53 through deletion of I255, consequence inactivation of p53 through substitution of R337G, defect in hyperglycosylation, inability in adding mannose to the core oligosaccharide, reduction in outer-chain mannose residues, presence of oligosaccharide Man(13-14, 2)), (deletion of gene OCH1 and MNN1, presence of oligosaccharide Man(9-11, occurrence of tri-partite repair product, overexpression of Rad51, recovery from checkpoint signal, increasing distance between repeated sequences, enhanced dependence on Rad51 for colony formation, constitutive expression of Hac1, increase in overall process specific productivity, deletion of GAS1, constitutive expression of Hac1 and deletion of GAS1, about a 7-fold increase in overall process specific productivity, deletion of gene RAM component, consequence cell lysis, deletion of gene CBK1, consequence reduced fertility, consequence reduced expression of mating type-specific genes, mutation in gene BRR1, consequence restoration of fertility, mutation in gene MPT5, expression of gene McPHGPx, GSH-dependent peroxidase activity towards phospholipid hydroperoxide, GSH-dependent peroxidase activity towards H2O2, GSH-dependent peroxidase activity towards tert-butyl hydroperoxide, deletion of gene PHGPx in Saccharomyces cerevisiae, susceptibility to linolenic acid, rescue of susceptibilities to linolenic acid in PHGPx-deletion mutant yeast, deletion of gene SPO14, affects PEA production, deletion of gene YJU3, affects PEA inactivation, heterologous expression of huntingtin, affects PEA metabolism, heterologous expression of alpha-synuclein, identification of suppressor mutations, construction of genetic interaction network, revelation of new components of specific pathways, enhanced mitochondrial point mutagenesis, increased frequency of respiration-defective mutants, decreased stability of mtDNA, reduced level of Msh1p, deletion of gene CYS4, deficiency in cysteine, deficiency in glutathione, rust-colored colonies, drug-dependent growth defects, deletion of genes in sulfur assimilation pathway, enhancement or suppression of rust coloration, consequence similar wild-type predictions for biomass and ethanol yields, consequence smaller ethanol yield enhancements in seven compartment model, simplification of seven compartment model to two intracellular compartments, consequence increased ethanol yields, zeroing of metabolic reactions associated with unexpressed genes, consequence negligible changes in steady-state predictions, dynamic extensions of models, consequence similar predictions of optimal batch ethanol productivity for wild type, implementable gene insertions, consequence significantly different predictions in single compartment model, combined deletion/overexpression/insertion mutant, consequence improved ethanol productivity capabilities, depletion of extracellular ethanol, reduction of stored lipids, synthesis and release of glycerol, reversal of chronological life span extension, deletion of gene RHR2, deletion of gene TOR1, up-regulation of glycerol biosynthesis genes, deletion of gene RAS2, optimization of life span extension, deletion of Rpd3 function, higher levels of histone H3 acetylation, higher levels of histone H4 acetylation, regulation by Rpd3L, silencing transcription, regulation by Rpd3S, recompacting chromatin, loss of Rpd3 function, significant association with Rpd3 chromatin binding, control of gene expression, deletion of gene YTA7, affected enzymatic activity of cis-prenyltransferase, altered cellular levels of isoprenoid compounds, rendered cells hypersensitive to lovastatin, overexpression of gene HMG2, restored growth of yta7Delta cells in presence of lovastatin, impairment of isoprenoid pathway by lovastatin, altered expression level of YTA7 gene, impairment of isoprenoid pathway by zaragozic acid, depletion of gene Cit2, overexpression of gene Cit2, deletion of gene mls1, deletion of gene ac1, deletion of gene icl1, addition of succinic acid, deletion of gene npt1, decrease in lifespan under calorie restriction, deletion of gene qpt1, no change in lifespan under calorie restriction, overexpression of salvage pathway, increase in lifespan under calorie restriction, exclusive utilization of extracellular nicotinic acid for NAD+ and NADH biosynthesis, no lifespan increase under calorie restriction, calorie restriction, decrease in NAD+ concentrations, no decrease in NADH concentrations under calorie restriction, deletion of gene sidI, affects activation of HSF1, expression of gene sidI, inhibition of host protein synthesis, interaction with eEF1A, interaction with eEF1Bgamma, reduced glycerol uptake, poor growth on minimal glycerol medium, active glycerol uptake, response to salt stress, deletion of gene STL2, no effect on glycerol uptake, deletion of gene GUP1, Deletion of gene FTL1, consequence 1: Reduced conidiation, consequence 2: Failure to cause typical disease symptoms, consequence 3: Failure to colonize vascular tissues of rachis, consequence 4: Failure to cause necrosis on rachis, consequence 5: Defective in spreading from infected anthers to ovaries, consequence 6: Increased sensitivity to plant defensins MsDef1 and osmotin, consequence 7: Significant reduction in histone deacetylation, deletion of gene yidC2, stress-sensitive phenotype, deletion of gene yidC1, less severe phenotype, deletion of gene SRP, viable in S. mutans, deletion of gene SRP and yidC2, severely compromised phenotype, deletion of gene oxa1, complemented by YidC2, expression of gene YidC2, mediates cotranslational insertion of mitochondrial translation products into the inner membrane, deletion of gene CuURA3, consequence uracil auxotroph, consequence null mutant, consequence supports possibility of tetraploid state, deletion of gene Car1, accumulation of CGP up to 4% (wt/wt, deletion of gene Car2, deletion of gene Arg1, accumulation of CGP up to 15.3% (wt/wt, deletion of gene Arg3, deletion of gene Arg4, CGP consists of up to 20 mol% citrulline, CGP consists of up to 8 mol% ornithine, CGP consists of up to 16 mol% lysine, deletion of gene Abf1 binding site in SUP35 promoter, no effect on yeast viability, mutation of ATG1 in SUP35, lethal effect in glucose medium, partial restoration of viability with galactose, partial restoration of viability under histidine starvation, mutation of ATG124 in SUP35, mutation of ATG254 in SUP35, impaired fermentation capacity, modified nitrogen consumption rate, altered gene expression of other permeases, reduced arginine consumption, prevention of reduction of vanillin to vanillyl alcohol, expression of gene 3-dehydroshikimate dehydratase, production of intermediate in vanillin biosynthesis, expression of gene ACAR, reduction of aromatic carboxylic acid to vanillin precursor, expression of gene O-methyltransferase, methylation of vanillin precursor, coexpression of gene phosphopantetheinyl transferase, activation of ACAR enzyme, expression of gene UDP-glycosyltransferase, conversion of vanillin into vanillin beta-D-glucoside, deletion of larger chromosomal regions, adaptation to brewing, massive unilateral decrease of the ribosomal DNA cluster, post-hybridization evolution process, bilateral truncations of over 60 genes, truncations and deletions of less efficient maltose uptake genes, truncations and deletions of less efficient maltotriose uptake genes, deletion of gene MgCRZ1, reduced number of conidia, abnormal appressoria formation, little or no melanin production, delayed lipid metabolism, decline in turgor pressure, attenuation of pathogenicity, deletion of Yaf9, counteracts efficiency of HOG pathway, overexpression of Yaf9, activates chromatin remodeling and transcription regulation, due to low concentration of cesium chloride, mutation in CWI pathway, shows sensitivity to cesium chloride, imeB deletion, retarded growth, overproduction of fertile sexual fruiting bodies in the presence of light, abnormal differentiation of sexual Hülle cells in submerged cultures, increased sexual development dependent on VeA, combined deletion of imeB and fphA, complete loss of light response, failure to produce sterigmatocystin, ImeB, needed for expression of the sterigmatocystin gene cluster, deletion of Not4p, abrogated degradation of arrest products, insertion of 12 consecutive basic amino acids, degradation of truncated GFP products, disruption of interaction with Ubc4/Ubc5 E2 enzyme, disruption of ubiquitin-protein ligase activity, deletion of gene YIL064w, negatively affecting secretory production of reporter alpha-amylase, necessary for efficient secretory production of murine alpha-amylase, necessary for efficient secretory production of fungal polygalacturonase, defects at very early stages of endocytic transport, defects at later stages of endocytic transport, low transfection efficiency, deletion of gene H3K4, decreased UV survival, impaired NER at HML locus, deletion of gene H3K79, enhanced recruitment of Sir complex at HML locus, reduced MNase accessibility, impaired DNA repair efficiency, restored MNase accessibility, restored DNA repair efficiency, deletion of gene SIR3, decreased expression of RAD16, method for efficient mutagenesis reported, deletion of SAC1, increased levels of most sphingolipid species, reduced inositol phosphosphingolipids, decreased PtdIns, acute attenuation of PtdIns(4)P production, increased PtdIns, reduced sphingoid bases, inhibition of Aur1p, overcome reduction of sphingoid bases, modulation of PtdIns(4)P metabolism, initiated sphingolipid-dependent biological effects, deletion of gene och1, decreased protein productivity, deletion of gene mnn1, deletion of gene mnn4, mutation of gene pol3-01, alleviation of growth defect, mutation of gene cdc6-1, improved protein productivity, survival increased by duplicates, survival increase up to 13%, buffering capacity higher for highly expressed genes, sequence similarity weakly predicts buffering capacity, number of duplicates per gene weakly predicts buffering capacity, increased transcriptional and translational activity in buffering duplicates, deletion of gene VDAC-1, severely compromised in DNA import, deletion of gene VDAC-2, deletion of gene TAE1, alters ribosomal profile, defects in translation efficiency, defects in translation fidelity, overexpression of gene TAE1, suppresses phenotypes related to translation, deletion of gene Gas1p, 4-ethylphenol sorption capacity decreased by 75%, deletion of gene tpi1, increased carbon flux to DHAP, introduction of gene mgs, increased methylglyoxal, introduction of genes mgs and gldA, increased 1,2-propanediol, increased succinic acid production, deletion of Hsp90, increased phenotypic variation, deletion of chromosome organization and DNA integrity gene products, increased morphological variation, deletion of RNA elongation gene products, deletion of protein modification gene products, deletion of cell cycle gene products, deletion of response to stimuli such as stress gene products, deletion of gene Ybr030w, unmethylated Rpl42ab, deletion of gene SET7, deletion of gene Ybr030w and SET7, expression of gene Ybr030w, methylation of Lys-40 in Rpl42ab, expression of gene SET7, methylation of Lys-55 in Rpl42ab, deletion of gene Ynm3, consequence cell survival at higher temperature is impaired, consequence increased heat sensitivity, consequence reduced chaperone activity, presence of gene Ynm3, consequence improved efficiency of proteolysis of aberrant proteins, consequence prevention of formation of nonproductive toxic aggregates, deletion of gene Rrp6p, accumulation of poly(A, deletion of gene Dis3p, increase in the levels of degradation substrates common to the core exosome and Rrp6p, deletion of gene Rrp43p, accumulation of degradation intermediates, mutation in gene Rrp6p, loss of co-purification with the core exosome, disruption of Rrp6p-core exosome interaction, inability to efficiently degrade certain poly(A, deletion of gene At1g27980, decreased fresh weight decrease rate, altered leaf temperature, increased dehydration stress tolerance, deletion of gene mik-1, reduced growth of basal hyphae, deletion of gene mek-1, deletion of gene mak-1, reduced formation of asexual macroconidia, almost abolished formation of asexual macroconidia, increased abundance of arthroconidia, inability to form protoperithecia, increased levels of tyrosinase mRNA, increased levels of tyrosinase protein, increased production of l-DOPA melanin, deletion of gene PEX5, relocation of Icl1 to cytosol, relocation of Mls1 to cytosol, deletion of gene FOX2, reduced growth on non-fermentable carbon sources, presence of giant peroxisomes, deletion of gene PEX5 in fox2Delta/Delta strain, restoration of growth on acetate, restoration of growth on ethanol, deletion of gene ydjL, abolished 2,3-butanediol production, no detectable AR activity, no detectable BDH activity, accumulation of acetoin, affected lactate production kinetics, deletion of gene GAL4, prevents induced galactose toxicity, overexpression of gene GRE3, cells more tolerant to lithium, accumulates more galactitol, accumulates less galactose-1-phosphate, suppresses galactose growth defect in gal7-deleted strain, suppresses galactose growth defect in gal10-deleted strain, deletion of a single candidate gene, no effect on yeast growth in the presence of inhibitors, deletion of gene FAS II, inefficient 5' processing of mitochondrial precursor tRNAs, accumulation of precursor containing RPM1 and tRNA(Pro, blockage of Pet127-driven 5' processing of RPM1, loss of lipoic acid attachment to subunits of three key mitochondrial enzymes, requirement for octanoic acid as a precursor for lipoic acid synthesis and attachment, requirement for a product of the FAS II pathway for RNase P RNA maturation, requirement for a product of the FAS II pathway for lipoic acid production needed for pyruvate dehydrogenase activity, deletion of specialized translesion DNA polymerases, increased mutation frequency upon 4-NQO treatment, increase in dNTP concentration, improved tolerance to 4-NQO induced damage, increased bypass efficiency of 8-oxoG lesion by DNA polymerase epsilon, deletion of gene eIF3a, impairment of translation initiation, reduction of binding of eIF3 and associated eIFs to native preinitiation complexes, severe blockage of the induction of GCN4 translation via reinitiation, inability of 40S ribosomes to resume scanning after terminating at the first upstream ORF (uORF1, interaction between eIF3a-NTD and sequences 5' of uORF1, enhancement of reinitiation, increasing length of uORF1, decline in reinitiation efficiency, eIF3 retention on ribosomes throughout uORF1 translation, stabilization of mRNA association with post-termination 40S subunits, enable resumption of scanning for reinitiation downstream, deletion of gene KlPCL1, no effect on sporulation, no effect on growth abilities under various physiological conditions, deletion of gene tag, consequence filamentation phenotype after MMS exposure, deletion of gene dinG, consequence decoupling of filamentation from SulA levels, consequence associated with SulA-independent filamentation pathway, deletion of multiple nonessential genes, possible synthetic growth defects, deletion of gene ACS1, consequence growth on acetate, consequence full virulence, consequence alternative carbon source utilization, consequence response to nitrosative stress, deletion of gene KTI11, consequence antisuppression of nonsense and missense suppressor tRNAs, consequence zymocin resistance, consequence protection against tRNase attack of zymocin, consequence resistance against diphtheria toxin, consequence protection against ADP-ribosylation of eEF2 by DT, consequence induction of resistance against sordarin, mutation in gene KTI11, consequence dissociation of zymocin resistance from DT sensitivity, altered growth characteristics, deletion of gene mig1, deletion of gene qdr3, deletion of gene pdr3, compensated effect on respiratory function, avoided NAD(+, deletion of gene gdh1, overexpression of glt1 improves ethanol production, overexpression of gene glt1, deletion of gene fps1, no effect on glycerol efflux, deletion of gene BTN2, decreases flor yeast resistance to high ethanol concentrations, increases fermentative capacity of flor yeast, increases flor yeast sliding motility, increases transcription of the FLO11 gene, overexpression of gene BTN2, prevents growth on nonfermentable carbon sources, BTN2 transcription induced by acetaldehyde, increases BTN2 expression, BTN2 repression by ethanol, decreases BTN2 expression, consequence 1: lower production of 3MH on synthetic medium, consequence 1: higher production of 3MH on synthetic medium, addition of DAP, consequence 1: decrease of 3MH production on grape must, substitution of urea for DAP, consequence 1: increase of 3MH production, relief of nitrogen catabolite repression (NCR), deletion of gene LYS20, reduced homocitrate synthesis under fermentative metabolism, deletion of gene LYS21, reduced homocitrate synthesis during growth on ethanol, overexpression of gene LYS20, increased homocitrate synthesis under fermentative metabolism, overexpression of gene LYS21, increased homocitrate synthesis during growth on ethanol, lysine inhibition of Lys21p, reduced activity of Lys21p, lysine inhibition of Lys20p, less pronounced effect compared to Lys21p, positive cooperativity of Lys21p for alpha-KG, enhanced alpha-KG binding in presence of lysine, deletion of gene HU, consequence DNA looping defect, expression of gene Nhp6A, consequence rescue of DNA looping, consequence alteration of DNA twist, deletion of gene HMGB2-box A, consequence failure to rescue looping, reduced growth in sodium selenite, deletion of gene RAD18, deletion of gene in oxidative stress pathway, reduced growth in selenomethionine, deletion of gene transporter, consequence reduced filament formation, overexpression of gene transporter, consequence enhanced filament formation, presence of fusel alcohols, consequence induction of filament formation, accumulation of 3-methyl-1-butanol, consequence increased mitochondrial mass, consequence increased chitin content, deletion of gene SAW1, consequence abolished association of Rad1 at SSA intermediates in vivo, consequence cells accumulate recombination intermediates blocked at the Rad1/Rad10-dependent 3' flap cleavage step, deletion of gene SLX4, deletion of gene dss-1, consequence embryonic lethality, consequence larval arrest, consequence impaired oogenesis, expression of gene dss-1, consequence rescued temperature-sensitive growth phenotype in yeast, consequence partial rescue of accumulation of polyubiquitinated proteins in yeast, deletion of gene Kti13, consequence rescue of thermosensitive growth defect of secretory mutants, consequence synthetic sickness or lethality when combined with kti11Delta, deletion of gene Elongator, consequence synthetic sickness or lethality when combined with kti13Delta, deletion of gene Irc7, decrease in 4MSP release, decrease in 3SH release, deletion of gene Ure2p, altered transcriptional regulation of IRC7, deletion of gene Gln3p\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\Users\\shawn\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\transformers\\tokenization_utils_base.py:1601: FutureWarning: `clean_up_tokenization_spaces` was not set. It will be set to `True` by default. This behavior will be depracted in transformers v4.45, and will be then set to `False` by default. For more details check this issue: https://github.com/huggingface/transformers/issues/31884\n",
      "  warnings.warn(\n",
      "C:\\Users\\shawn\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\transformers\\models\\bert\\modeling_bert.py:439: UserWarning: 1Torch was not compiled with flash attention. (Triggered internally at C:\\actions-runner\\_work\\pytorch\\pytorch\\builder\\windows\\pytorch\\aten\\src\\ATen\\native\\transformers\\cuda\\sdp_utils.cpp:555.)\n",
      "  attn_output = torch.nn.functional.scaled_dot_product_attention(\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Progress: 0.01%\n",
      "Progress: 0.03%\n",
      "Progress: 0.04%\n",
      "Progress: 0.06%\n",
      "Progress: 0.07%\n",
      "Progress: 0.09%\n",
      "Progress: 0.10%\n",
      "Progress: 0.11%\n",
      "Progress: 0.13%\n",
      "Progress: 0.14%\n",
      "Progress: 0.16%\n",
      "Progress: 0.17%\n",
      "Progress: 0.18%\n",
      "Progress: 0.20%\n",
      "Progress: 0.21%\n",
      "Progress: 0.23%\n",
      "Progress: 0.24%\n",
      "Progress: 0.26%\n",
      "Progress: 0.27%\n",
      "Progress: 0.28%\n",
      "Progress: 0.30%\n",
      "Progress: 0.31%\n",
      "Progress: 0.33%\n",
      "Progress: 0.34%\n",
      "Progress: 0.36%\n",
      "Progress: 0.37%\n",
      "Progress: 0.38%\n",
      "Progress: 0.40%\n",
      "Progress: 0.41%\n",
      "Progress: 0.43%\n",
      "Progress: 0.44%\n",
      "Progress: 0.45%\n",
      "Progress: 0.47%\n",
      "Progress: 0.48%\n",
      "Progress: 0.50%\n",
      "Progress: 0.51%\n",
      "Progress: 0.53%\n",
      "Progress: 0.54%\n",
      "Progress: 0.55%\n",
      "Progress: 0.57%\n",
      "Progress: 0.58%\n",
      "Progress: 0.60%\n",
      "Progress: 0.61%\n",
      "Progress: 0.63%\n",
      "Progress: 0.64%\n",
      "Progress: 0.65%\n",
      "Progress: 0.67%\n",
      "Progress: 0.68%\n",
      "Progress: 0.70%\n",
      "Progress: 0.71%\n",
      "Progress: 0.72%\n",
      "Progress: 0.74%\n",
      "Progress: 0.75%\n",
      "Progress: 0.77%\n",
      "Progress: 0.78%\n",
      "Progress: 0.80%\n",
      "Progress: 0.81%\n",
      "Progress: 0.82%\n",
      "Progress: 0.84%\n",
      "Progress: 0.85%\n",
      "Progress: 0.87%\n",
      "Progress: 0.88%\n",
      "Progress: 0.89%\n",
      "Progress: 0.91%\n",
      "Progress: 0.92%\n",
      "Progress: 0.94%\n",
      "Progress: 0.95%\n",
      "Progress: 0.97%\n",
      "Progress: 0.98%\n",
      "Progress: 0.99%\n",
      "Progress: 1.01%\n",
      "Progress: 1.02%\n",
      "Progress: 1.04%\n",
      "Progress: 1.05%\n",
      "Progress: 1.06%\n",
      "Progress: 1.08%\n",
      "Progress: 1.09%\n",
      "Progress: 1.11%\n",
      "Progress: 1.12%\n",
      "Progress: 1.14%\n",
      "Progress: 1.15%\n",
      "Progress: 1.16%\n",
      "Progress: 1.18%\n",
      "Progress: 1.19%\n",
      "Progress: 1.21%\n",
      "Progress: 1.22%\n",
      "Progress: 1.23%\n",
      "Progress: 1.25%\n",
      "Progress: 1.26%\n",
      "Progress: 1.28%\n",
      "Progress: 1.29%\n",
      "Progress: 1.30%\n",
      "Progress: 1.32%\n",
      "Progress: 1.33%\n",
      "Progress: 1.35%\n",
      "Progress: 1.36%\n",
      "Progress: 1.38%\n",
      "Progress: 1.39%\n",
      "Progress: 1.40%\n",
      "Progress: 1.42%\n",
      "Progress: 1.43%\n",
      "Progress: 1.45%\n",
      "Progress: 1.46%\n",
      "Progress: 1.47%\n",
      "Progress: 1.49%\n",
      "Progress: 1.50%\n",
      "Progress: 1.52%\n",
      "Progress: 1.53%\n",
      "Progress: 1.55%\n",
      "Progress: 1.56%\n",
      "Progress: 1.57%\n",
      "Progress: 1.59%\n",
      "Progress: 1.60%\n",
      "Progress: 1.62%\n",
      "Progress: 1.63%\n",
      "Progress: 1.64%\n",
      "Progress: 1.66%\n",
      "Progress: 1.67%\n",
      "Progress: 1.69%\n",
      "Progress: 1.70%\n",
      "Progress: 1.71%\n",
      "Progress: 1.73%\n",
      "Progress: 1.74%\n",
      "Progress: 1.76%\n",
      "Progress: 1.77%\n",
      "Progress: 1.79%\n",
      "Progress: 1.80%\n",
      "Progress: 1.81%\n",
      "Progress: 1.83%\n",
      "Progress: 1.84%\n",
      "Progress: 1.86%\n",
      "Progress: 1.87%\n",
      "Progress: 1.88%\n",
      "Progress: 1.90%\n",
      "Progress: 1.91%\n",
      "Progress: 1.93%\n",
      "Progress: 1.94%\n",
      "Progress: 1.95%\n",
      "Progress: 1.97%\n",
      "Progress: 1.98%\n",
      "Progress: 2.00%\n",
      "Progress: 2.01%\n",
      "Progress: 2.02%\n",
      "Progress: 2.04%\n",
      "Progress: 2.05%\n",
      "Progress: 2.07%\n",
      "Progress: 2.08%\n",
      "Progress: 2.10%\n",
      "Progress: 2.11%\n",
      "Progress: 2.12%\n",
      "Progress: 2.14%\n",
      "Progress: 2.15%\n",
      "Progress: 2.17%\n",
      "Progress: 2.18%\n",
      "Progress: 2.19%\n",
      "Progress: 2.21%\n",
      "Progress: 2.22%\n",
      "Progress: 2.24%\n",
      "Progress: 2.25%\n",
      "Progress: 2.26%\n",
      "Progress: 2.28%\n",
      "Progress: 2.29%\n",
      "Progress: 2.31%\n",
      "Progress: 2.32%\n",
      "Progress: 2.33%\n",
      "Progress: 2.35%\n",
      "Progress: 2.36%\n",
      "Progress: 2.38%\n",
      "Progress: 2.39%\n",
      "Progress: 2.40%\n",
      "Progress: 2.42%\n",
      "Progress: 2.43%\n",
      "Progress: 2.45%\n",
      "Progress: 2.46%\n",
      "Progress: 2.47%\n",
      "Progress: 2.49%\n",
      "Progress: 2.50%\n",
      "Progress: 2.52%\n",
      "Progress: 2.53%\n",
      "Progress: 2.55%\n",
      "Progress: 2.56%\n",
      "Progress: 2.57%\n",
      "Progress: 2.59%\n",
      "Progress: 2.60%\n",
      "Progress: 2.62%\n",
      "Progress: 2.63%\n",
      "Progress: 2.64%\n",
      "Progress: 2.66%\n",
      "Progress: 2.67%\n",
      "Progress: 2.69%\n",
      "Progress: 2.70%\n",
      "Progress: 2.71%\n",
      "Progress: 2.73%\n",
      "Progress: 2.74%\n",
      "Progress: 2.76%\n",
      "Progress: 2.77%\n",
      "Progress: 2.78%\n",
      "Progress: 2.80%\n",
      "Progress: 2.81%\n",
      "Progress: 2.83%\n",
      "Progress: 2.84%\n",
      "Progress: 2.85%\n",
      "Progress: 2.87%\n",
      "Progress: 2.88%\n",
      "Progress: 2.90%\n",
      "Progress: 2.91%\n",
      "Progress: 2.92%\n",
      "Progress: 2.94%\n",
      "Progress: 2.95%\n",
      "Progress: 2.97%\n",
      "Progress: 2.98%\n",
      "Progress: 2.99%\n",
      "Progress: 3.01%\n",
      "Progress: 3.02%\n",
      "Progress: 3.04%\n",
      "Progress: 3.05%\n",
      "Progress: 3.06%\n",
      "Progress: 3.08%\n",
      "Progress: 3.09%\n",
      "Progress: 3.11%\n",
      "Progress: 3.12%\n",
      "Progress: 3.13%\n",
      "Progress: 3.15%\n",
      "Progress: 3.16%\n",
      "Progress: 3.18%\n",
      "Progress: 3.19%\n",
      "Progress: 3.20%\n",
      "Progress: 3.22%\n",
      "Progress: 3.23%\n",
      "Progress: 3.25%\n",
      "Progress: 3.26%\n",
      "Progress: 3.27%\n",
      "Progress: 3.29%\n",
      "Progress: 3.30%\n",
      "Progress: 3.32%\n",
      "Progress: 3.33%\n",
      "Progress: 3.34%\n",
      "Progress: 3.36%\n",
      "Progress: 3.37%\n",
      "Progress: 3.39%\n",
      "Progress: 3.40%\n",
      "Progress: 3.41%\n",
      "Progress: 3.43%\n",
      "Progress: 3.44%\n",
      "Progress: 3.46%\n",
      "Progress: 3.47%\n",
      "Progress: 3.48%\n",
      "Progress: 3.50%\n",
      "Progress: 3.51%\n",
      "Progress: 3.53%\n",
      "Progress: 3.54%\n",
      "Progress: 3.55%\n",
      "Progress: 3.57%\n",
      "Progress: 3.58%\n",
      "Progress: 3.60%\n",
      "Progress: 3.61%\n",
      "Progress: 3.62%\n",
      "Progress: 3.64%\n",
      "Progress: 3.65%\n",
      "Progress: 3.67%\n",
      "Progress: 3.68%\n",
      "Progress: 3.69%\n",
      "Progress: 3.71%\n",
      "Progress: 3.72%\n",
      "Progress: 3.74%\n",
      "Progress: 3.75%\n",
      "Progress: 3.76%\n",
      "Progress: 3.78%\n",
      "Progress: 3.79%\n",
      "Progress: 3.81%\n",
      "Progress: 3.82%\n",
      "Progress: 3.83%\n",
      "Progress: 3.85%\n",
      "Progress: 3.86%\n",
      "Progress: 3.88%\n",
      "Progress: 3.89%\n",
      "Progress: 3.90%\n",
      "Progress: 3.92%\n",
      "Progress: 3.93%\n",
      "Progress: 3.95%\n",
      "Progress: 3.96%\n",
      "Progress: 3.97%\n",
      "Progress: 3.99%\n",
      "Progress: 4.00%\n",
      "Progress: 4.01%\n",
      "Progress: 4.03%\n",
      "Progress: 4.04%\n",
      "Progress: 4.06%\n",
      "Progress: 4.07%\n",
      "Progress: 4.08%\n",
      "Progress: 4.10%\n",
      "Progress: 4.11%\n",
      "Progress: 4.13%\n",
      "Progress: 4.14%\n",
      "Progress: 4.15%\n",
      "Progress: 4.17%\n",
      "Progress: 4.18%\n",
      "Progress: 4.20%\n",
      "Progress: 4.21%\n",
      "Progress: 4.22%\n",
      "Progress: 4.24%\n",
      "Progress: 4.25%\n",
      "Progress: 4.27%\n",
      "Progress: 4.28%\n",
      "Progress: 4.29%\n",
      "Progress: 4.31%\n",
      "Progress: 4.32%\n",
      "Progress: 4.34%\n",
      "Progress: 4.35%\n",
      "Progress: 4.36%\n",
      "Progress: 4.38%\n",
      "Progress: 4.39%\n",
      "Progress: 4.40%\n",
      "Progress: 4.42%\n",
      "Progress: 4.43%\n",
      "Progress: 4.45%\n",
      "Progress: 4.46%\n",
      "Progress: 4.47%\n",
      "Progress: 4.49%\n",
      "Progress: 4.50%\n",
      "Progress: 4.52%\n",
      "Progress: 4.53%\n",
      "Progress: 4.54%\n",
      "Progress: 4.56%\n",
      "Progress: 4.57%\n",
      "Progress: 4.59%\n",
      "Progress: 4.60%\n",
      "Progress: 4.61%\n",
      "Progress: 4.63%\n",
      "Progress: 4.64%\n",
      "Progress: 4.66%\n",
      "Progress: 4.67%\n",
      "Progress: 4.68%\n",
      "Progress: 4.70%\n",
      "Progress: 4.71%\n",
      "Progress: 4.72%\n",
      "Progress: 4.74%\n",
      "Progress: 4.75%\n",
      "Progress: 4.77%\n",
      "Progress: 4.78%\n",
      "Progress: 4.79%\n",
      "Progress: 4.81%\n",
      "Progress: 4.82%\n",
      "Progress: 4.84%\n",
      "Progress: 4.85%\n",
      "Progress: 4.86%\n",
      "Progress: 4.88%\n",
      "Progress: 4.89%\n",
      "Progress: 4.91%\n",
      "Progress: 4.92%\n",
      "Progress: 4.93%\n",
      "Progress: 4.95%\n",
      "Progress: 4.96%\n",
      "Progress: 4.97%\n",
      "Progress: 4.99%\n",
      "Progress: 5.00%\n",
      "Progress: 5.02%\n",
      "Progress: 5.03%\n",
      "Progress: 5.04%\n",
      "Progress: 5.06%\n",
      "Progress: 5.07%\n",
      "Progress: 5.09%\n",
      "Progress: 5.10%\n",
      "Progress: 5.11%\n",
      "Progress: 5.13%\n",
      "Progress: 5.14%\n",
      "Progress: 5.15%\n",
      "Progress: 5.17%\n",
      "Progress: 5.18%\n",
      "Progress: 5.20%\n",
      "Progress: 5.21%\n",
      "Progress: 5.22%\n",
      "Progress: 5.24%\n",
      "Progress: 5.25%\n",
      "Progress: 5.27%\n",
      "Progress: 5.28%\n",
      "Progress: 5.29%\n",
      "Progress: 5.31%\n",
      "Progress: 5.32%\n",
      "Progress: 5.33%\n",
      "Progress: 5.35%\n",
      "Progress: 5.36%\n",
      "Progress: 5.38%\n",
      "Progress: 5.39%\n",
      "Progress: 5.40%\n",
      "Progress: 5.42%\n",
      "Progress: 5.43%\n",
      "Progress: 5.45%\n",
      "Progress: 5.46%\n",
      "Progress: 5.47%\n",
      "Progress: 5.49%\n",
      "Progress: 5.50%\n",
      "Progress: 5.51%\n",
      "Progress: 5.53%\n",
      "Progress: 5.54%\n",
      "Progress: 5.56%\n",
      "Progress: 5.57%\n",
      "Progress: 5.58%\n",
      "Progress: 5.60%\n",
      "Progress: 5.61%\n",
      "Progress: 5.63%\n",
      "Progress: 5.64%\n",
      "Progress: 5.65%\n",
      "Progress: 5.67%\n",
      "Progress: 5.68%\n",
      "Progress: 5.69%\n",
      "Progress: 5.71%\n",
      "Progress: 5.72%\n",
      "Progress: 5.74%\n",
      "Progress: 5.75%\n",
      "Progress: 5.76%\n",
      "Progress: 5.78%\n",
      "Progress: 5.79%\n",
      "Progress: 5.80%\n",
      "Progress: 5.82%\n",
      "Progress: 5.83%\n",
      "Progress: 5.85%\n",
      "Progress: 5.86%\n",
      "Progress: 5.87%\n",
      "Progress: 5.89%\n",
      "Progress: 5.90%\n",
      "Progress: 5.92%\n",
      "Progress: 5.93%\n",
      "Progress: 5.94%\n",
      "Progress: 5.96%\n",
      "Progress: 5.97%\n",
      "Progress: 5.98%\n",
      "Progress: 6.00%\n",
      "Progress: 6.01%\n",
      "Progress: 6.03%\n",
      "Progress: 6.04%\n",
      "Progress: 6.05%\n",
      "Progress: 6.07%\n",
      "Progress: 6.08%\n",
      "Progress: 6.09%\n",
      "Progress: 6.11%\n",
      "Progress: 6.12%\n",
      "Progress: 6.14%\n",
      "Progress: 6.15%\n",
      "Progress: 6.16%\n",
      "Progress: 6.18%\n",
      "Progress: 6.19%\n",
      "Progress: 6.21%\n",
      "Progress: 6.22%\n",
      "Progress: 6.23%\n",
      "Progress: 6.25%\n",
      "Progress: 6.26%\n",
      "Progress: 6.27%\n",
      "Progress: 6.29%\n",
      "Progress: 6.30%\n",
      "Progress: 6.32%\n",
      "Progress: 6.33%\n",
      "Progress: 6.34%\n",
      "Progress: 6.36%\n",
      "Progress: 6.37%\n",
      "Progress: 6.38%\n",
      "Progress: 6.40%\n",
      "Progress: 6.41%\n",
      "Progress: 6.43%\n",
      "Progress: 6.44%\n",
      "Progress: 6.45%\n",
      "Progress: 6.47%\n",
      "Progress: 6.48%\n",
      "Progress: 6.49%\n",
      "Progress: 6.51%\n",
      "Progress: 6.52%\n",
      "Progress: 6.54%\n",
      "Progress: 6.55%\n",
      "Progress: 6.56%\n",
      "Progress: 6.58%\n",
      "Progress: 6.59%\n",
      "Progress: 6.60%\n",
      "Progress: 6.62%\n",
      "Progress: 6.63%\n",
      "Progress: 6.65%\n",
      "Progress: 6.66%\n",
      "Progress: 6.67%\n",
      "Progress: 6.69%\n",
      "Progress: 6.70%\n",
      "Progress: 6.71%\n",
      "Progress: 6.73%\n",
      "Progress: 6.74%\n",
      "Progress: 6.76%\n",
      "Progress: 6.77%\n",
      "Progress: 6.78%\n",
      "Progress: 6.80%\n",
      "Progress: 6.81%\n",
      "Progress: 6.82%\n",
      "Progress: 6.84%\n",
      "Progress: 6.85%\n",
      "Progress: 6.87%\n",
      "Progress: 6.88%\n",
      "Progress: 6.89%\n",
      "Progress: 6.91%\n",
      "Progress: 6.92%\n",
      "Progress: 6.93%\n",
      "Progress: 6.95%\n",
      "Progress: 6.96%\n",
      "Progress: 6.98%\n",
      "Progress: 6.99%\n",
      "Progress: 7.00%\n",
      "Progress: 7.02%\n",
      "Progress: 7.03%\n",
      "Progress: 7.04%\n",
      "Progress: 7.06%\n",
      "Progress: 7.07%\n",
      "Progress: 7.09%\n",
      "Progress: 7.10%\n",
      "Progress: 7.11%\n",
      "Progress: 7.13%\n",
      "Progress: 7.14%\n",
      "Progress: 7.15%\n",
      "Progress: 7.17%\n",
      "Progress: 7.18%\n",
      "Progress: 7.19%\n",
      "Progress: 7.21%\n",
      "Progress: 7.22%\n",
      "Progress: 7.24%\n",
      "Progress: 7.25%\n",
      "Progress: 7.26%\n",
      "Progress: 7.28%\n",
      "Progress: 7.29%\n",
      "Progress: 7.30%\n",
      "Progress: 7.32%\n",
      "Progress: 7.33%\n",
      "Progress: 7.35%\n",
      "Progress: 7.36%\n",
      "Progress: 7.37%\n",
      "Progress: 7.39%\n",
      "Progress: 7.40%\n",
      "Progress: 7.41%\n",
      "Progress: 7.43%\n",
      "Progress: 7.44%\n",
      "Progress: 7.46%\n",
      "Progress: 7.47%\n",
      "Progress: 7.48%\n",
      "Progress: 7.50%\n",
      "Progress: 7.51%\n",
      "Progress: 7.52%\n",
      "Progress: 7.54%\n",
      "Progress: 7.55%\n",
      "Progress: 7.56%\n",
      "Progress: 7.58%\n",
      "Progress: 7.59%\n",
      "Progress: 7.61%\n",
      "Progress: 7.62%\n",
      "Progress: 7.63%\n",
      "Progress: 7.65%\n",
      "Progress: 7.66%\n",
      "Progress: 7.67%\n",
      "Progress: 7.69%\n",
      "Progress: 7.70%\n",
      "Progress: 7.72%\n",
      "Progress: 7.73%\n",
      "Progress: 7.74%\n",
      "Progress: 7.76%\n",
      "Progress: 7.77%\n",
      "Progress: 7.78%\n",
      "Progress: 7.80%\n",
      "Progress: 7.81%\n",
      "Progress: 7.82%\n",
      "Progress: 7.84%\n",
      "Progress: 7.85%\n",
      "Progress: 7.87%\n",
      "Progress: 7.88%\n",
      "Progress: 7.89%\n",
      "Progress: 7.91%\n",
      "Progress: 7.92%\n",
      "Progress: 7.93%\n",
      "Progress: 7.95%\n",
      "Progress: 7.96%\n",
      "Progress: 7.97%\n",
      "Progress: 7.99%\n",
      "Progress: 8.00%\n",
      "Progress: 8.02%\n",
      "Progress: 8.03%\n",
      "Progress: 8.04%\n",
      "Progress: 8.06%\n",
      "Progress: 8.07%\n",
      "Progress: 8.08%\n",
      "Progress: 8.10%\n",
      "Progress: 8.11%\n",
      "Progress: 8.12%\n",
      "Progress: 8.14%\n",
      "Progress: 8.15%\n",
      "Progress: 8.17%\n",
      "Progress: 8.18%\n",
      "Progress: 8.19%\n",
      "Progress: 8.21%\n",
      "Progress: 8.22%\n",
      "Progress: 8.23%\n",
      "Progress: 8.25%\n",
      "Progress: 8.26%\n",
      "Progress: 8.27%\n",
      "Progress: 8.29%\n",
      "Progress: 8.30%\n",
      "Progress: 8.32%\n",
      "Progress: 8.33%\n",
      "Progress: 8.34%\n",
      "Progress: 8.36%\n",
      "Progress: 8.37%\n",
      "Progress: 8.38%\n",
      "Progress: 8.40%\n",
      "Progress: 8.41%\n",
      "Progress: 8.42%\n",
      "Progress: 8.44%\n",
      "Progress: 8.45%\n",
      "Progress: 8.47%\n",
      "Progress: 8.48%\n",
      "Progress: 8.49%\n",
      "Progress: 8.51%\n",
      "Progress: 8.52%\n",
      "Progress: 8.53%\n",
      "Progress: 8.55%\n",
      "Progress: 8.56%\n",
      "Progress: 8.57%\n",
      "Progress: 8.59%\n",
      "Progress: 8.60%\n",
      "Progress: 8.62%\n",
      "Progress: 8.63%\n",
      "Progress: 8.64%\n",
      "Progress: 8.66%\n",
      "Progress: 8.67%\n",
      "Progress: 8.68%\n",
      "Progress: 8.70%\n",
      "Progress: 8.71%\n",
      "Progress: 8.72%\n",
      "Progress: 8.74%\n",
      "Progress: 8.75%\n",
      "Progress: 8.76%\n",
      "Progress: 8.78%\n",
      "Progress: 8.79%\n",
      "Progress: 8.81%\n",
      "Progress: 8.82%\n",
      "Progress: 8.83%\n",
      "Progress: 8.85%\n",
      "Progress: 8.86%\n",
      "Progress: 8.87%\n",
      "Progress: 8.89%\n",
      "Progress: 8.90%\n",
      "Progress: 8.91%\n",
      "Progress: 8.93%\n",
      "Progress: 8.94%\n",
      "Progress: 8.95%\n",
      "Progress: 8.97%\n",
      "Progress: 8.98%\n",
      "Progress: 9.00%\n",
      "Progress: 9.01%\n",
      "Progress: 9.02%\n",
      "Progress: 9.04%\n",
      "Progress: 9.05%\n",
      "Progress: 9.06%\n",
      "Progress: 9.08%\n",
      "Progress: 9.09%\n",
      "Progress: 9.10%\n",
      "Progress: 9.12%\n",
      "Progress: 9.13%\n",
      "Progress: 9.14%\n",
      "Progress: 9.16%\n",
      "Progress: 9.17%\n",
      "Progress: 9.19%\n",
      "Progress: 9.20%\n",
      "Progress: 9.21%\n",
      "Progress: 9.23%\n",
      "Progress: 9.24%\n",
      "Progress: 9.25%\n",
      "Progress: 9.27%\n",
      "Progress: 9.28%\n",
      "Progress: 9.29%\n",
      "Progress: 9.31%\n",
      "Progress: 9.32%\n",
      "Progress: 9.33%\n",
      "Progress: 9.35%\n",
      "Progress: 9.36%\n",
      "Progress: 9.38%\n",
      "Progress: 9.39%\n",
      "Progress: 9.40%\n",
      "Progress: 9.42%\n",
      "Progress: 9.43%\n",
      "Progress: 9.44%\n",
      "Progress: 9.46%\n",
      "Progress: 9.47%\n",
      "Progress: 9.48%\n",
      "Progress: 9.50%\n",
      "Progress: 9.51%\n",
      "Progress: 9.52%\n",
      "Progress: 9.54%\n",
      "Progress: 9.55%\n",
      "Progress: 9.57%\n",
      "Progress: 9.58%\n",
      "Progress: 9.59%\n",
      "Progress: 9.61%\n",
      "Progress: 9.62%\n",
      "Progress: 9.63%\n",
      "Progress: 9.65%\n",
      "Progress: 9.66%\n",
      "Progress: 9.67%\n",
      "Progress: 9.69%\n",
      "Progress: 9.70%\n",
      "Progress: 9.71%\n",
      "Progress: 9.73%\n",
      "Progress: 9.74%\n",
      "Progress: 9.75%\n",
      "Progress: 9.77%\n",
      "Progress: 9.78%\n",
      "Progress: 9.79%\n",
      "Progress: 9.81%\n",
      "Progress: 9.82%\n",
      "Progress: 9.84%\n",
      "Progress: 9.85%\n",
      "Progress: 9.86%\n",
      "Progress: 9.88%\n",
      "Progress: 9.89%\n",
      "Progress: 9.90%\n",
      "Progress: 9.92%\n",
      "Progress: 9.93%\n",
      "Progress: 9.94%\n",
      "Progress: 9.96%\n",
      "Progress: 9.97%\n",
      "Progress: 9.98%\n",
      "Progress: 10.00%\n",
      "Progress: 10.01%\n",
      "Progress: 10.02%\n",
      "Progress: 10.04%\n",
      "Progress: 10.05%\n",
      "Progress: 10.07%\n",
      "Progress: 10.08%\n",
      "Progress: 10.09%\n",
      "Progress: 10.11%\n",
      "Progress: 10.12%\n",
      "Progress: 10.13%\n",
      "Progress: 10.15%\n",
      "Progress: 10.16%\n",
      "Progress: 10.17%\n",
      "Progress: 10.19%\n",
      "Progress: 10.20%\n",
      "Progress: 10.21%\n",
      "Progress: 10.23%\n",
      "Progress: 10.24%\n",
      "Progress: 10.25%\n",
      "Progress: 10.27%\n",
      "Progress: 10.28%\n",
      "Progress: 10.29%\n",
      "Progress: 10.31%\n",
      "Progress: 10.32%\n",
      "Progress: 10.33%\n",
      "Progress: 10.35%\n",
      "Progress: 10.36%\n",
      "Progress: 10.38%\n",
      "Progress: 10.39%\n",
      "Progress: 10.40%\n",
      "Progress: 10.42%\n",
      "Progress: 10.43%\n",
      "Progress: 10.44%\n",
      "Progress: 10.46%\n",
      "Progress: 10.47%\n",
      "Progress: 10.48%\n",
      "Progress: 10.50%\n",
      "Progress: 10.51%\n",
      "Progress: 10.52%\n",
      "Progress: 10.54%\n",
      "Progress: 10.55%\n",
      "Progress: 10.56%\n",
      "Progress: 10.58%\n",
      "Progress: 10.59%\n",
      "Progress: 10.60%\n",
      "Progress: 10.62%\n",
      "Progress: 10.63%\n",
      "Progress: 10.64%\n",
      "Progress: 10.66%\n",
      "Progress: 10.67%\n",
      "Progress: 10.68%\n",
      "Progress: 10.70%\n",
      "Progress: 10.71%\n",
      "Progress: 10.73%\n",
      "Progress: 10.74%\n",
      "Progress: 10.75%\n",
      "Progress: 10.77%\n",
      "Progress: 10.78%\n",
      "Progress: 10.79%\n",
      "Progress: 10.81%\n",
      "Progress: 10.82%\n",
      "Progress: 10.83%\n",
      "Progress: 10.85%\n",
      "Progress: 10.86%\n",
      "Progress: 10.87%\n",
      "Progress: 10.89%\n",
      "Progress: 10.90%\n",
      "Progress: 10.91%\n",
      "Progress: 10.93%\n",
      "Progress: 10.94%\n",
      "Progress: 10.95%\n",
      "Progress: 10.97%\n",
      "Progress: 10.98%\n",
      "Progress: 10.99%\n",
      "Progress: 11.01%\n",
      "Progress: 11.02%\n",
      "Progress: 11.03%\n",
      "Progress: 11.05%\n",
      "Progress: 11.06%\n",
      "Progress: 11.07%\n",
      "Progress: 11.09%\n",
      "Progress: 11.10%\n",
      "Progress: 11.11%\n",
      "Progress: 11.13%\n",
      "Progress: 11.14%\n",
      "Progress: 11.15%\n",
      "Progress: 11.17%\n",
      "Progress: 11.18%\n",
      "Progress: 11.20%\n",
      "Progress: 11.21%\n",
      "Progress: 11.22%\n",
      "Progress: 11.24%\n",
      "Progress: 11.25%\n",
      "Progress: 11.26%\n",
      "Progress: 11.28%\n",
      "Progress: 11.29%\n",
      "Progress: 11.30%\n",
      "Progress: 11.32%\n",
      "Progress: 11.33%\n",
      "Progress: 11.34%\n",
      "Progress: 11.36%\n",
      "Progress: 11.37%\n",
      "Progress: 11.38%\n",
      "Progress: 11.40%\n",
      "Progress: 11.41%\n",
      "Progress: 11.42%\n",
      "Progress: 11.44%\n",
      "Progress: 11.45%\n",
      "Progress: 11.46%\n",
      "Progress: 11.48%\n",
      "Progress: 11.49%\n",
      "Progress: 11.50%\n",
      "Progress: 11.52%\n",
      "Progress: 11.53%\n",
      "Progress: 11.54%\n",
      "Progress: 11.56%\n",
      "Progress: 11.57%\n",
      "Progress: 11.58%\n",
      "Progress: 11.60%\n",
      "Progress: 11.61%\n",
      "Progress: 11.62%\n",
      "Progress: 11.64%\n",
      "Progress: 11.65%\n",
      "Progress: 11.66%\n",
      "Progress: 11.68%\n",
      "Progress: 11.69%\n",
      "Progress: 11.70%\n",
      "Progress: 11.72%\n",
      "Progress: 11.73%\n",
      "Progress: 11.74%\n",
      "Progress: 11.76%\n",
      "Progress: 11.77%\n",
      "Progress: 11.78%\n",
      "Progress: 11.80%\n",
      "Progress: 11.81%\n",
      "Progress: 11.82%\n",
      "Progress: 11.84%\n",
      "Progress: 11.85%\n",
      "Progress: 11.86%\n",
      "Progress: 11.88%\n",
      "Progress: 11.89%\n",
      "Progress: 11.90%\n",
      "Progress: 11.92%\n",
      "Progress: 11.93%\n",
      "Progress: 11.94%\n",
      "Progress: 11.96%\n",
      "Progress: 11.97%\n",
      "Progress: 11.98%\n",
      "Progress: 12.00%\n",
      "Progress: 12.01%\n",
      "Progress: 12.02%\n",
      "Progress: 12.04%\n",
      "Progress: 12.05%\n",
      "Progress: 12.06%\n",
      "Progress: 12.08%\n",
      "Progress: 12.09%\n",
      "Progress: 12.10%\n",
      "Progress: 12.12%\n",
      "Progress: 12.13%\n",
      "Progress: 12.14%\n",
      "Progress: 12.16%\n",
      "Progress: 12.17%\n",
      "Progress: 12.18%\n",
      "Progress: 12.20%\n",
      "Progress: 12.21%\n",
      "Progress: 12.22%\n",
      "Progress: 12.24%\n",
      "Progress: 12.25%\n",
      "Progress: 12.26%\n",
      "Progress: 12.28%\n",
      "Progress: 12.29%\n",
      "Progress: 12.30%\n",
      "Progress: 12.32%\n",
      "Progress: 12.33%\n",
      "Progress: 12.34%\n",
      "Progress: 12.36%\n",
      "Progress: 12.37%\n",
      "Progress: 12.38%\n",
      "Progress: 12.40%\n",
      "Progress: 12.41%\n",
      "Progress: 12.42%\n",
      "Progress: 12.44%\n",
      "Progress: 12.45%\n",
      "Progress: 12.46%\n",
      "Progress: 12.48%\n",
      "Progress: 12.49%\n",
      "Progress: 12.50%\n",
      "Progress: 12.52%\n",
      "Progress: 12.53%\n",
      "Progress: 12.54%\n",
      "Progress: 12.56%\n",
      "Progress: 12.57%\n",
      "Progress: 12.58%\n",
      "Progress: 12.60%\n",
      "Progress: 12.61%\n",
      "Progress: 12.62%\n",
      "Progress: 12.64%\n",
      "Progress: 12.65%\n",
      "Progress: 12.66%\n",
      "Progress: 12.68%\n",
      "Progress: 12.69%\n",
      "Progress: 12.70%\n",
      "Progress: 12.72%\n",
      "Progress: 12.73%\n",
      "Progress: 12.74%\n",
      "Progress: 12.76%\n",
      "Progress: 12.77%\n",
      "Progress: 12.78%\n",
      "Progress: 12.80%\n",
      "Progress: 12.81%\n",
      "Progress: 12.82%\n",
      "Progress: 12.84%\n",
      "Progress: 12.85%\n",
      "Progress: 12.86%\n",
      "Progress: 12.88%\n",
      "Progress: 12.89%\n",
      "Progress: 12.90%\n",
      "Progress: 12.92%\n",
      "Progress: 12.93%\n",
      "Progress: 12.94%\n",
      "Progress: 12.96%\n",
      "Progress: 12.97%\n",
      "Progress: 12.98%\n",
      "Progress: 13.00%\n",
      "Progress: 13.01%\n",
      "Progress: 13.02%\n",
      "Progress: 13.04%\n",
      "Progress: 13.05%\n",
      "Progress: 13.06%\n",
      "Progress: 13.08%\n",
      "Progress: 13.09%\n",
      "Progress: 13.10%\n",
      "Progress: 13.12%\n",
      "Progress: 13.13%\n",
      "Progress: 13.14%\n",
      "Progress: 13.16%\n",
      "Progress: 13.17%\n",
      "Progress: 13.18%\n",
      "Progress: 13.20%\n",
      "Progress: 13.21%\n",
      "Progress: 13.22%\n",
      "Progress: 13.24%\n",
      "Progress: 13.25%\n",
      "Progress: 13.26%\n",
      "Progress: 13.27%\n",
      "Progress: 13.29%\n",
      "Progress: 13.30%\n",
      "Progress: 13.31%\n",
      "Progress: 13.33%\n",
      "Progress: 13.34%\n",
      "Progress: 13.35%\n",
      "Progress: 13.37%\n",
      "Progress: 13.38%\n",
      "Progress: 13.39%\n",
      "Progress: 13.41%\n",
      "Progress: 13.42%\n",
      "Progress: 13.43%\n",
      "Progress: 13.45%\n",
      "Progress: 13.46%\n",
      "Progress: 13.47%\n",
      "Progress: 13.49%\n",
      "Progress: 13.50%\n",
      "Progress: 13.51%\n",
      "Progress: 13.53%\n",
      "Progress: 13.54%\n",
      "Progress: 13.55%\n",
      "Progress: 13.57%\n",
      "Progress: 13.58%\n",
      "Progress: 13.59%\n",
      "Progress: 13.61%\n",
      "Progress: 13.62%\n",
      "Progress: 13.63%\n",
      "Progress: 13.65%\n",
      "Progress: 13.66%\n",
      "Progress: 13.67%\n",
      "Progress: 13.69%\n",
      "Progress: 13.70%\n",
      "Progress: 13.71%\n",
      "Progress: 13.73%\n",
      "Progress: 13.74%\n",
      "Progress: 13.75%\n",
      "Progress: 13.76%\n",
      "Progress: 13.78%\n",
      "Progress: 13.79%\n",
      "Progress: 13.80%\n",
      "Progress: 13.82%\n",
      "Progress: 13.83%\n",
      "Progress: 13.84%\n",
      "Progress: 13.86%\n",
      "Progress: 13.87%\n",
      "Progress: 13.88%\n",
      "Progress: 13.90%\n",
      "Progress: 13.91%\n",
      "Progress: 13.92%\n",
      "Progress: 13.94%\n",
      "Progress: 13.95%\n",
      "Progress: 13.96%\n",
      "Progress: 13.98%\n",
      "Progress: 13.99%\n",
      "Progress: 14.00%\n",
      "Progress: 14.02%\n",
      "Progress: 14.03%\n",
      "Progress: 14.04%\n",
      "Progress: 14.06%\n",
      "Progress: 14.07%\n",
      "Progress: 14.08%\n",
      "Progress: 14.09%\n",
      "Progress: 14.11%\n",
      "Progress: 14.12%\n",
      "Progress: 14.13%\n",
      "Progress: 14.15%\n",
      "Progress: 14.16%\n",
      "Progress: 14.17%\n",
      "Progress: 14.19%\n",
      "Progress: 14.20%\n",
      "Progress: 14.21%\n",
      "Progress: 14.23%\n",
      "Progress: 14.24%\n",
      "Progress: 14.25%\n",
      "Progress: 14.27%\n",
      "Progress: 14.28%\n",
      "Progress: 14.29%\n",
      "Progress: 14.31%\n",
      "Progress: 14.32%\n",
      "Progress: 14.33%\n",
      "Progress: 14.35%\n",
      "Progress: 14.36%\n",
      "Progress: 14.37%\n",
      "Progress: 14.38%\n",
      "Progress: 14.40%\n",
      "Progress: 14.41%\n",
      "Progress: 14.42%\n",
      "Progress: 14.44%\n",
      "Progress: 14.45%\n",
      "Progress: 14.46%\n",
      "Progress: 14.48%\n",
      "Progress: 14.49%\n",
      "Progress: 14.50%\n",
      "Progress: 14.52%\n",
      "Progress: 14.53%\n",
      "Progress: 14.54%\n",
      "Progress: 14.56%\n",
      "Progress: 14.57%\n",
      "Progress: 14.58%\n",
      "Progress: 14.60%\n",
      "Progress: 14.61%\n",
      "Progress: 14.62%\n",
      "Progress: 14.63%\n",
      "Progress: 14.65%\n",
      "Progress: 14.66%\n",
      "Progress: 14.67%\n",
      "Progress: 14.69%\n",
      "Progress: 14.70%\n",
      "Progress: 14.71%\n",
      "Progress: 14.73%\n",
      "Progress: 14.74%\n",
      "Progress: 14.75%\n",
      "Progress: 14.77%\n",
      "Progress: 14.78%\n",
      "Progress: 14.79%\n",
      "Progress: 14.81%\n",
      "Progress: 14.82%\n",
      "Progress: 14.83%\n",
      "Progress: 14.84%\n",
      "Progress: 14.86%\n",
      "Progress: 14.87%\n",
      "Progress: 14.88%\n",
      "Progress: 14.90%\n",
      "Progress: 14.91%\n",
      "Progress: 14.92%\n",
      "Progress: 14.94%\n",
      "Progress: 14.95%\n",
      "Progress: 14.96%\n",
      "Progress: 14.98%\n",
      "Progress: 14.99%\n",
      "Progress: 15.00%\n",
      "Progress: 15.02%\n",
      "Progress: 15.03%\n",
      "Progress: 15.04%\n",
      "Progress: 15.05%\n",
      "Progress: 15.07%\n",
      "Progress: 15.08%\n",
      "Progress: 15.09%\n",
      "Progress: 15.11%\n",
      "Progress: 15.12%\n",
      "Progress: 15.13%\n",
      "Progress: 15.15%\n",
      "Progress: 15.16%\n",
      "Progress: 15.17%\n",
      "Progress: 15.19%\n",
      "Progress: 15.20%\n",
      "Progress: 15.21%\n",
      "Progress: 15.23%\n",
      "Progress: 15.24%\n",
      "Progress: 15.25%\n",
      "Progress: 15.26%\n",
      "Progress: 15.28%\n",
      "Progress: 15.29%\n",
      "Progress: 15.30%\n",
      "Progress: 15.32%\n",
      "Progress: 15.33%\n",
      "Progress: 15.34%\n",
      "Progress: 15.36%\n",
      "Progress: 15.37%\n",
      "Progress: 15.38%\n",
      "Progress: 15.40%\n",
      "Progress: 15.41%\n",
      "Progress: 15.42%\n",
      "Progress: 15.43%\n",
      "Progress: 15.45%\n",
      "Progress: 15.46%\n",
      "Progress: 15.47%\n",
      "Progress: 15.49%\n",
      "Progress: 15.50%\n",
      "Progress: 15.51%\n",
      "Progress: 15.53%\n",
      "Progress: 15.54%\n",
      "Progress: 15.55%\n",
      "Progress: 15.57%\n",
      "Progress: 15.58%\n",
      "Progress: 15.59%\n",
      "Progress: 15.60%\n",
      "Progress: 15.62%\n",
      "Progress: 15.63%\n",
      "Progress: 15.64%\n",
      "Progress: 15.66%\n",
      "Progress: 15.67%\n",
      "Progress: 15.68%\n",
      "Progress: 15.70%\n",
      "Progress: 15.71%\n",
      "Progress: 15.72%\n",
      "Progress: 15.74%\n",
      "Progress: 15.75%\n",
      "Progress: 15.76%\n",
      "Progress: 15.77%\n",
      "Progress: 15.79%\n",
      "Progress: 15.80%\n",
      "Progress: 15.81%\n",
      "Progress: 15.83%\n",
      "Progress: 15.84%\n",
      "Progress: 15.85%\n",
      "Progress: 15.87%\n",
      "Progress: 15.88%\n",
      "Progress: 15.89%\n",
      "Progress: 15.91%\n",
      "Progress: 15.92%\n",
      "Progress: 15.93%\n",
      "Progress: 15.94%\n",
      "Progress: 15.96%\n",
      "Progress: 15.97%\n",
      "Progress: 15.98%\n",
      "Progress: 16.00%\n",
      "Progress: 16.01%\n",
      "Progress: 16.02%\n",
      "Progress: 16.04%\n",
      "Progress: 16.05%\n",
      "Progress: 16.06%\n",
      "Progress: 16.07%\n",
      "Progress: 16.09%\n",
      "Progress: 16.10%\n",
      "Progress: 16.11%\n",
      "Progress: 16.13%\n",
      "Progress: 16.14%\n",
      "Progress: 16.15%\n",
      "Progress: 16.17%\n",
      "Progress: 16.18%\n",
      "Progress: 16.19%\n",
      "Progress: 16.21%\n",
      "Progress: 16.22%\n",
      "Progress: 16.23%\n",
      "Progress: 16.24%\n",
      "Progress: 16.26%\n",
      "Progress: 16.27%\n",
      "Progress: 16.28%\n",
      "Progress: 16.30%\n",
      "Progress: 16.31%\n",
      "Progress: 16.32%\n",
      "Progress: 16.34%\n",
      "Progress: 16.35%\n",
      "Progress: 16.36%\n",
      "Progress: 16.37%\n",
      "Progress: 16.39%\n",
      "Progress: 16.40%\n",
      "Progress: 16.41%\n",
      "Progress: 16.43%\n",
      "Progress: 16.44%\n",
      "Progress: 16.45%\n",
      "Progress: 16.47%\n",
      "Progress: 16.48%\n",
      "Progress: 16.49%\n",
      "Progress: 16.50%\n",
      "Progress: 16.52%\n",
      "Progress: 16.53%\n",
      "Progress: 16.54%\n",
      "Progress: 16.56%\n",
      "Progress: 16.57%\n",
      "Progress: 16.58%\n",
      "Progress: 16.60%\n",
      "Progress: 16.61%\n",
      "Progress: 16.62%\n",
      "Progress: 16.63%\n",
      "Progress: 16.65%\n",
      "Progress: 16.66%\n",
      "Progress: 16.67%\n",
      "Progress: 16.69%\n",
      "Progress: 16.70%\n",
      "Progress: 16.71%\n",
      "Progress: 16.73%\n",
      "Progress: 16.74%\n",
      "Progress: 16.75%\n",
      "Progress: 16.76%\n",
      "Progress: 16.78%\n",
      "Progress: 16.79%\n",
      "Progress: 16.80%\n",
      "Progress: 16.82%\n",
      "Progress: 16.83%\n",
      "Progress: 16.84%\n",
      "Progress: 16.86%\n",
      "Progress: 16.87%\n",
      "Progress: 16.88%\n",
      "Progress: 16.89%\n",
      "Progress: 16.91%\n",
      "Progress: 16.92%\n",
      "Progress: 16.93%\n",
      "Progress: 16.95%\n",
      "Progress: 16.96%\n",
      "Progress: 16.97%\n",
      "Progress: 16.99%\n",
      "Progress: 17.00%\n",
      "Progress: 17.01%\n",
      "Progress: 17.02%\n",
      "Progress: 17.04%\n",
      "Progress: 17.05%\n",
      "Progress: 17.06%\n",
      "Progress: 17.08%\n",
      "Progress: 17.09%\n",
      "Progress: 17.10%\n",
      "Progress: 17.11%\n",
      "Progress: 17.13%\n",
      "Progress: 17.14%\n",
      "Progress: 17.15%\n",
      "Progress: 17.17%\n",
      "Progress: 17.18%\n",
      "Progress: 17.19%\n",
      "Progress: 17.21%\n",
      "Progress: 17.22%\n",
      "Progress: 17.23%\n",
      "Progress: 17.24%\n",
      "Progress: 17.26%\n",
      "Progress: 17.27%\n",
      "Progress: 17.28%\n",
      "Progress: 17.30%\n",
      "Progress: 17.31%\n",
      "Progress: 17.32%\n",
      "Progress: 17.33%\n",
      "Progress: 17.35%\n",
      "Progress: 17.36%\n",
      "Progress: 17.37%\n",
      "Progress: 17.39%\n",
      "Progress: 17.40%\n",
      "Progress: 17.41%\n",
      "Progress: 17.43%\n",
      "Progress: 17.44%\n",
      "Progress: 17.45%\n",
      "Progress: 17.46%\n",
      "Progress: 17.48%\n",
      "Progress: 17.49%\n",
      "Progress: 17.50%\n",
      "Progress: 17.52%\n",
      "Progress: 17.53%\n",
      "Progress: 17.54%\n",
      "Progress: 17.55%\n",
      "Progress: 17.57%\n",
      "Progress: 17.58%\n",
      "Progress: 17.59%\n",
      "Progress: 17.61%\n",
      "Progress: 17.62%\n",
      "Progress: 17.63%\n",
      "Progress: 17.65%\n",
      "Progress: 17.66%\n",
      "Progress: 17.67%\n",
      "Progress: 17.68%\n",
      "Progress: 17.70%\n",
      "Progress: 17.71%\n",
      "Progress: 17.72%\n",
      "Progress: 17.74%\n",
      "Progress: 17.75%\n",
      "Progress: 17.76%\n",
      "Progress: 17.77%\n",
      "Progress: 17.79%\n",
      "Progress: 17.80%\n",
      "Progress: 17.81%\n",
      "Progress: 17.83%\n",
      "Progress: 17.84%\n",
      "Progress: 17.85%\n",
      "Progress: 17.86%\n",
      "Progress: 17.88%\n",
      "Progress: 17.89%\n",
      "Progress: 17.90%\n",
      "Progress: 17.92%\n",
      "Progress: 17.93%\n",
      "Progress: 17.94%\n",
      "Progress: 17.95%\n",
      "Progress: 17.97%\n",
      "Progress: 17.98%\n",
      "Progress: 17.99%\n",
      "Progress: 18.01%\n",
      "Progress: 18.02%\n",
      "Progress: 18.03%\n",
      "Progress: 18.04%\n",
      "Progress: 18.06%\n",
      "Progress: 18.07%\n",
      "Progress: 18.08%\n",
      "Progress: 18.10%\n",
      "Progress: 18.11%\n",
      "Progress: 18.12%\n",
      "Progress: 18.14%\n",
      "Progress: 18.15%\n",
      "Progress: 18.16%\n",
      "Progress: 18.17%\n",
      "Progress: 18.19%\n",
      "Progress: 18.20%\n",
      "Progress: 18.21%\n",
      "Progress: 18.23%\n",
      "Progress: 18.24%\n",
      "Progress: 18.25%\n",
      "Progress: 18.26%\n",
      "Progress: 18.28%\n",
      "Progress: 18.29%\n",
      "Progress: 18.30%\n",
      "Progress: 18.32%\n",
      "Progress: 18.33%\n",
      "Progress: 18.34%\n",
      "Progress: 18.35%\n",
      "Progress: 18.37%\n",
      "Progress: 18.38%\n",
      "Progress: 18.39%\n",
      "Progress: 18.41%\n",
      "Progress: 18.42%\n",
      "Progress: 18.43%\n",
      "Progress: 18.44%\n",
      "Progress: 18.46%\n",
      "Progress: 18.47%\n",
      "Progress: 18.48%\n",
      "Progress: 18.50%\n",
      "Progress: 18.51%\n",
      "Progress: 18.52%\n",
      "Progress: 18.53%\n",
      "Progress: 18.55%\n",
      "Progress: 18.56%\n",
      "Progress: 18.57%\n",
      "Progress: 18.59%\n",
      "Progress: 18.60%\n",
      "Progress: 18.61%\n",
      "Progress: 18.62%\n",
      "Progress: 18.64%\n",
      "Progress: 18.65%\n",
      "Progress: 18.66%\n",
      "Progress: 18.67%\n",
      "Progress: 18.69%\n",
      "Progress: 18.70%\n",
      "Progress: 18.71%\n",
      "Progress: 18.73%\n",
      "Progress: 18.74%\n",
      "Progress: 18.75%\n",
      "Progress: 18.76%\n",
      "Progress: 18.78%\n",
      "Progress: 18.79%\n",
      "Progress: 18.80%\n",
      "Progress: 18.82%\n",
      "Progress: 18.83%\n",
      "Progress: 18.84%\n",
      "Progress: 18.85%\n",
      "Progress: 18.87%\n",
      "Progress: 18.88%\n",
      "Progress: 18.89%\n",
      "Progress: 18.91%\n",
      "Progress: 18.92%\n",
      "Progress: 18.93%\n",
      "Progress: 18.94%\n",
      "Progress: 18.96%\n",
      "Progress: 18.97%\n",
      "Progress: 18.98%\n",
      "Progress: 19.00%\n",
      "Progress: 19.01%\n",
      "Progress: 19.02%\n",
      "Progress: 19.03%\n",
      "Progress: 19.05%\n",
      "Progress: 19.06%\n",
      "Progress: 19.07%\n",
      "Progress: 19.09%\n",
      "Progress: 19.10%\n",
      "Progress: 19.11%\n",
      "Progress: 19.12%\n",
      "Progress: 19.14%\n",
      "Progress: 19.15%\n",
      "Progress: 19.16%\n",
      "Progress: 19.17%\n",
      "Progress: 19.19%\n",
      "Progress: 19.20%\n",
      "Progress: 19.21%\n",
      "Progress: 19.23%\n",
      "Progress: 19.24%\n",
      "Progress: 19.25%\n",
      "Progress: 19.26%\n",
      "Progress: 19.28%\n",
      "Progress: 19.29%\n",
      "Progress: 19.30%\n",
      "Progress: 19.32%\n",
      "Progress: 19.33%\n",
      "Progress: 19.34%\n",
      "Progress: 19.35%\n",
      "Progress: 19.37%\n",
      "Progress: 19.38%\n",
      "Progress: 19.39%\n",
      "Progress: 19.40%\n",
      "Progress: 19.42%\n",
      "Progress: 19.43%\n",
      "Progress: 19.44%\n",
      "Progress: 19.46%\n",
      "Progress: 19.47%\n",
      "Progress: 19.48%\n",
      "Progress: 19.49%\n",
      "Progress: 19.51%\n",
      "Progress: 19.52%\n",
      "Progress: 19.53%\n",
      "Progress: 19.55%\n",
      "Progress: 19.56%\n",
      "Progress: 19.57%\n",
      "Progress: 19.58%\n",
      "Progress: 19.60%\n",
      "Progress: 19.61%\n",
      "Progress: 19.62%\n",
      "Progress: 19.63%\n",
      "Progress: 19.65%\n",
      "Progress: 19.66%\n",
      "Progress: 19.67%\n",
      "Progress: 19.69%\n",
      "Progress: 19.70%\n",
      "Progress: 19.71%\n",
      "Progress: 19.72%\n",
      "Progress: 19.74%\n",
      "Progress: 19.75%\n",
      "Progress: 19.76%\n",
      "Progress: 19.77%\n",
      "Progress: 19.79%\n",
      "Progress: 19.80%\n",
      "Progress: 19.81%\n",
      "Progress: 19.83%\n",
      "Progress: 19.84%\n",
      "Progress: 19.85%\n",
      "Progress: 19.86%\n",
      "Progress: 19.88%\n",
      "Progress: 19.89%\n",
      "Progress: 19.90%\n",
      "Progress: 19.92%\n",
      "Progress: 19.93%\n",
      "Progress: 19.94%\n",
      "Progress: 19.95%\n",
      "Progress: 19.97%\n",
      "Progress: 19.98%\n",
      "Progress: 19.99%\n",
      "Progress: 20.00%\n",
      "Progress: 20.02%\n",
      "Progress: 20.03%\n",
      "Progress: 20.04%\n",
      "Progress: 20.06%\n",
      "Progress: 20.07%\n",
      "Progress: 20.08%\n",
      "Progress: 20.09%\n",
      "Progress: 20.11%\n",
      "Progress: 20.12%\n",
      "Progress: 20.13%\n",
      "Progress: 20.14%\n",
      "Progress: 20.16%\n",
      "Progress: 20.17%\n",
      "Progress: 20.18%\n",
      "Progress: 20.20%\n",
      "Progress: 20.21%\n",
      "Progress: 20.22%\n",
      "Progress: 20.23%\n",
      "Progress: 20.25%\n",
      "Progress: 20.26%\n",
      "Progress: 20.27%\n",
      "Progress: 20.28%\n",
      "Progress: 20.30%\n",
      "Progress: 20.31%\n",
      "Progress: 20.32%\n",
      "Progress: 20.33%\n",
      "Progress: 20.35%\n",
      "Progress: 20.36%\n",
      "Progress: 20.37%\n",
      "Progress: 20.39%\n",
      "Progress: 20.40%\n",
      "Progress: 20.41%\n",
      "Progress: 20.42%\n",
      "Progress: 20.44%\n",
      "Progress: 20.45%\n",
      "Progress: 20.46%\n",
      "Progress: 20.47%\n",
      "Progress: 20.49%\n",
      "Progress: 20.50%\n",
      "Progress: 20.51%\n",
      "Progress: 20.53%\n",
      "Progress: 20.54%\n",
      "Progress: 20.55%\n",
      "Progress: 20.56%\n",
      "Progress: 20.58%\n",
      "Progress: 20.59%\n",
      "Progress: 20.60%\n",
      "Progress: 20.61%\n",
      "Progress: 20.63%\n",
      "Progress: 20.64%\n",
      "Progress: 20.65%\n",
      "Progress: 20.66%\n",
      "Progress: 20.68%\n",
      "Progress: 20.69%\n",
      "Progress: 20.70%\n",
      "Progress: 20.72%\n",
      "Progress: 20.73%\n",
      "Progress: 20.74%\n",
      "Progress: 20.75%\n",
      "Progress: 20.77%\n",
      "Progress: 20.78%\n",
      "Progress: 20.79%\n",
      "Progress: 20.80%\n",
      "Progress: 20.82%\n",
      "Progress: 20.83%\n",
      "Progress: 20.84%\n",
      "Progress: 20.85%\n",
      "Progress: 20.87%\n",
      "Progress: 20.88%\n",
      "Progress: 20.89%\n",
      "Progress: 20.91%\n",
      "Progress: 20.92%\n",
      "Progress: 20.93%\n",
      "Progress: 20.94%\n",
      "Progress: 20.96%\n",
      "Progress: 20.97%\n",
      "Progress: 20.98%\n",
      "Progress: 20.99%\n",
      "Progress: 21.01%\n",
      "Progress: 21.02%\n",
      "Progress: 21.03%\n",
      "Progress: 21.04%\n",
      "Progress: 21.06%\n",
      "Progress: 21.07%\n",
      "Progress: 21.08%\n",
      "Progress: 21.10%\n",
      "Progress: 21.11%\n",
      "Progress: 21.12%\n",
      "Progress: 21.13%\n",
      "Progress: 21.15%\n",
      "Progress: 21.16%\n",
      "Progress: 21.17%\n",
      "Progress: 21.18%\n",
      "Progress: 21.20%\n",
      "Progress: 21.21%\n",
      "Progress: 21.22%\n",
      "Progress: 21.23%\n",
      "Progress: 21.25%\n",
      "Progress: 21.26%\n",
      "Progress: 21.27%\n",
      "Progress: 21.28%\n",
      "Progress: 21.30%\n",
      "Progress: 21.31%\n",
      "Progress: 21.32%\n",
      "Progress: 21.34%\n",
      "Progress: 21.35%\n",
      "Progress: 21.36%\n",
      "Progress: 21.37%\n",
      "Progress: 21.39%\n",
      "Progress: 21.40%\n",
      "Progress: 21.41%\n",
      "Progress: 21.42%\n",
      "Progress: 21.44%\n",
      "Progress: 21.45%\n",
      "Progress: 21.46%\n",
      "Progress: 21.47%\n",
      "Progress: 21.49%\n",
      "Progress: 21.50%\n",
      "Progress: 21.51%\n",
      "Progress: 21.52%\n",
      "Progress: 21.54%\n",
      "Progress: 21.55%\n",
      "Progress: 21.56%\n",
      "Progress: 21.57%\n",
      "Progress: 21.59%\n",
      "Progress: 21.60%\n",
      "Progress: 21.61%\n",
      "Progress: 21.63%\n",
      "Progress: 21.64%\n",
      "Progress: 21.65%\n",
      "Progress: 21.66%\n",
      "Progress: 21.68%\n",
      "Progress: 21.69%\n",
      "Progress: 21.70%\n",
      "Progress: 21.71%\n",
      "Progress: 21.73%\n",
      "Progress: 21.74%\n",
      "Progress: 21.75%\n",
      "Progress: 21.76%\n",
      "Progress: 21.78%\n",
      "Progress: 21.79%\n",
      "Progress: 21.80%\n",
      "Progress: 21.81%\n",
      "Progress: 21.83%\n",
      "Progress: 21.84%\n",
      "Progress: 21.85%\n",
      "Progress: 21.86%\n",
      "Progress: 21.88%\n",
      "Progress: 21.89%\n",
      "Progress: 21.90%\n",
      "Progress: 21.91%\n",
      "Progress: 21.93%\n",
      "Progress: 21.94%\n",
      "Progress: 21.95%\n",
      "Progress: 21.96%\n",
      "Progress: 21.98%\n",
      "Progress: 21.99%\n",
      "Progress: 22.00%\n",
      "Progress: 22.02%\n",
      "Progress: 22.03%\n",
      "Progress: 22.04%\n",
      "Progress: 22.05%\n",
      "Progress: 22.07%\n",
      "Progress: 22.08%\n",
      "Progress: 22.09%\n",
      "Progress: 22.10%\n",
      "Progress: 22.12%\n",
      "Progress: 22.13%\n",
      "Progress: 22.14%\n",
      "Progress: 22.15%\n",
      "Progress: 22.17%\n",
      "Progress: 22.18%\n",
      "Progress: 22.19%\n",
      "Progress: 22.20%\n",
      "Progress: 22.22%\n",
      "Progress: 22.23%\n",
      "Progress: 22.24%\n",
      "Progress: 22.25%\n",
      "Progress: 22.27%\n",
      "Progress: 22.28%\n",
      "Progress: 22.29%\n",
      "Progress: 22.30%\n",
      "Progress: 22.32%\n",
      "Progress: 22.33%\n",
      "Progress: 22.34%\n",
      "Progress: 22.35%\n",
      "Progress: 22.37%\n",
      "Progress: 22.38%\n",
      "Progress: 22.39%\n",
      "Progress: 22.40%\n",
      "Progress: 22.42%\n",
      "Progress: 22.43%\n",
      "Progress: 22.44%\n",
      "Progress: 22.45%\n",
      "Progress: 22.47%\n",
      "Progress: 22.48%\n",
      "Progress: 22.49%\n",
      "Progress: 22.50%\n",
      "Progress: 22.52%\n",
      "Progress: 22.53%\n",
      "Progress: 22.54%\n",
      "Progress: 22.55%\n",
      "Progress: 22.57%\n",
      "Progress: 22.58%\n",
      "Progress: 22.59%\n",
      "Progress: 22.60%\n",
      "Progress: 22.62%\n",
      "Progress: 22.63%\n",
      "Progress: 22.64%\n",
      "Progress: 22.65%\n",
      "Progress: 22.67%\n",
      "Progress: 22.68%\n",
      "Progress: 22.69%\n",
      "Progress: 22.70%\n",
      "Progress: 22.72%\n",
      "Progress: 22.73%\n",
      "Progress: 22.74%\n",
      "Progress: 22.75%\n",
      "Progress: 22.77%\n",
      "Progress: 22.78%\n",
      "Progress: 22.79%\n",
      "Progress: 22.81%\n",
      "Progress: 22.82%\n",
      "Progress: 22.83%\n",
      "Progress: 22.84%\n",
      "Progress: 22.86%\n",
      "Progress: 22.87%\n",
      "Progress: 22.88%\n",
      "Progress: 22.89%\n",
      "Progress: 22.91%\n",
      "Progress: 22.92%\n",
      "Progress: 22.93%\n",
      "Progress: 22.94%\n",
      "Progress: 22.95%\n",
      "Progress: 22.97%\n",
      "Progress: 22.98%\n",
      "Progress: 22.99%\n",
      "Progress: 23.00%\n",
      "Progress: 23.02%\n",
      "Progress: 23.03%\n",
      "Progress: 23.04%\n",
      "Progress: 23.05%\n",
      "Progress: 23.07%\n",
      "Progress: 23.08%\n",
      "Progress: 23.09%\n",
      "Progress: 23.10%\n",
      "Progress: 23.12%\n",
      "Progress: 23.13%\n",
      "Progress: 23.14%\n",
      "Progress: 23.15%\n",
      "Progress: 23.17%\n",
      "Progress: 23.18%\n",
      "Progress: 23.19%\n",
      "Progress: 23.20%\n",
      "Progress: 23.22%\n",
      "Progress: 23.23%\n",
      "Progress: 23.24%\n",
      "Progress: 23.25%\n",
      "Progress: 23.27%\n",
      "Progress: 23.28%\n",
      "Progress: 23.29%\n",
      "Progress: 23.30%\n",
      "Progress: 23.32%\n",
      "Progress: 23.33%\n",
      "Progress: 23.34%\n",
      "Progress: 23.35%\n",
      "Progress: 23.37%\n",
      "Progress: 23.38%\n",
      "Progress: 23.39%\n",
      "Progress: 23.40%\n",
      "Progress: 23.42%\n",
      "Progress: 23.43%\n",
      "Progress: 23.44%\n",
      "Progress: 23.45%\n",
      "Progress: 23.47%\n",
      "Progress: 23.48%\n",
      "Progress: 23.49%\n",
      "Progress: 23.50%\n",
      "Progress: 23.52%\n",
      "Progress: 23.53%\n",
      "Progress: 23.54%\n",
      "Progress: 23.55%\n",
      "Progress: 23.57%\n",
      "Progress: 23.58%\n",
      "Progress: 23.59%\n",
      "Progress: 23.60%\n",
      "Progress: 23.62%\n",
      "Progress: 23.63%\n",
      "Progress: 23.64%\n",
      "Progress: 23.65%\n",
      "Progress: 23.67%\n",
      "Progress: 23.68%\n",
      "Progress: 23.69%\n",
      "Progress: 23.70%\n",
      "Progress: 23.72%\n",
      "Progress: 23.73%\n",
      "Progress: 23.74%\n",
      "Progress: 23.75%\n",
      "Progress: 23.76%\n",
      "Progress: 23.78%\n",
      "Progress: 23.79%\n",
      "Progress: 23.80%\n",
      "Progress: 23.81%\n",
      "Progress: 23.83%\n",
      "Progress: 23.84%\n",
      "Progress: 23.85%\n",
      "Progress: 23.86%\n",
      "Progress: 23.88%\n",
      "Progress: 23.89%\n",
      "Progress: 23.90%\n",
      "Progress: 23.91%\n",
      "Progress: 23.93%\n",
      "Progress: 23.94%\n",
      "Progress: 23.95%\n",
      "Progress: 23.96%\n",
      "Progress: 23.98%\n",
      "Progress: 23.99%\n",
      "Progress: 24.00%\n",
      "Progress: 24.01%\n",
      "Progress: 24.03%\n",
      "Progress: 24.04%\n",
      "Progress: 24.05%\n",
      "Progress: 24.06%\n",
      "Progress: 24.08%\n",
      "Progress: 24.09%\n",
      "Progress: 24.10%\n",
      "Progress: 24.11%\n",
      "Progress: 24.12%\n",
      "Progress: 24.14%\n",
      "Progress: 24.15%\n",
      "Progress: 24.16%\n",
      "Progress: 24.17%\n",
      "Progress: 24.19%\n",
      "Progress: 24.20%\n",
      "Progress: 24.21%\n",
      "Progress: 24.22%\n",
      "Progress: 24.24%\n",
      "Progress: 24.25%\n",
      "Progress: 24.26%\n",
      "Progress: 24.27%\n",
      "Progress: 24.29%\n",
      "Progress: 24.30%\n",
      "Progress: 24.31%\n",
      "Progress: 24.32%\n",
      "Progress: 24.34%\n",
      "Progress: 24.35%\n",
      "Progress: 24.36%\n",
      "Progress: 24.37%\n",
      "Progress: 24.38%\n",
      "Progress: 24.40%\n",
      "Progress: 24.41%\n",
      "Progress: 24.42%\n",
      "Progress: 24.43%\n",
      "Progress: 24.45%\n",
      "Progress: 24.46%\n",
      "Progress: 24.47%\n",
      "Progress: 24.48%\n",
      "Progress: 24.50%\n",
      "Progress: 24.51%\n",
      "Progress: 24.52%\n",
      "Progress: 24.53%\n",
      "Progress: 24.55%\n",
      "Progress: 24.56%\n",
      "Progress: 24.57%\n",
      "Progress: 24.58%\n",
      "Progress: 24.60%\n",
      "Progress: 24.61%\n",
      "Progress: 24.62%\n",
      "Progress: 24.63%\n",
      "Progress: 24.64%\n",
      "Progress: 24.66%\n",
      "Progress: 24.67%\n",
      "Progress: 24.68%\n",
      "Progress: 24.69%\n",
      "Progress: 24.71%\n",
      "Progress: 24.72%\n",
      "Progress: 24.73%\n",
      "Progress: 24.74%\n",
      "Progress: 24.76%\n",
      "Progress: 24.77%\n",
      "Progress: 24.78%\n",
      "Progress: 24.79%\n",
      "Progress: 24.80%\n",
      "Progress: 24.82%\n",
      "Progress: 24.83%\n",
      "Progress: 24.84%\n",
      "Progress: 24.85%\n",
      "Progress: 24.87%\n",
      "Progress: 24.88%\n",
      "Progress: 24.89%\n",
      "Progress: 24.90%\n",
      "Progress: 24.92%\n",
      "Progress: 24.93%\n",
      "Progress: 24.94%\n",
      "Progress: 24.95%\n",
      "Progress: 24.97%\n",
      "Progress: 24.98%\n",
      "Progress: 24.99%\n",
      "Progress: 25.00%\n",
      "Progress: 25.01%\n",
      "Progress: 25.03%\n",
      "Progress: 25.04%\n",
      "Progress: 25.05%\n",
      "Progress: 25.06%\n",
      "Progress: 25.08%\n",
      "Progress: 25.09%\n",
      "Progress: 25.10%\n",
      "Progress: 25.11%\n",
      "Progress: 25.13%\n",
      "Progress: 25.14%\n",
      "Progress: 25.15%\n",
      "Progress: 25.16%\n",
      "Progress: 25.17%\n",
      "Progress: 25.19%\n",
      "Progress: 25.20%\n",
      "Progress: 25.21%\n",
      "Progress: 25.22%\n",
      "Progress: 25.24%\n",
      "Progress: 25.25%\n",
      "Progress: 25.26%\n",
      "Progress: 25.27%\n",
      "Progress: 25.29%\n",
      "Progress: 25.30%\n",
      "Progress: 25.31%\n",
      "Progress: 25.32%\n",
      "Progress: 25.33%\n",
      "Progress: 25.35%\n",
      "Progress: 25.36%\n",
      "Progress: 25.37%\n",
      "Progress: 25.38%\n",
      "Progress: 25.40%\n",
      "Progress: 25.41%\n",
      "Progress: 25.42%\n",
      "Progress: 25.43%\n",
      "Progress: 25.45%\n",
      "Progress: 25.46%\n",
      "Progress: 25.47%\n",
      "Progress: 25.48%\n",
      "Progress: 25.49%\n",
      "Progress: 25.51%\n",
      "Progress: 25.52%\n",
      "Progress: 25.53%\n",
      "Progress: 25.54%\n",
      "Progress: 25.56%\n",
      "Progress: 25.57%\n",
      "Progress: 25.58%\n",
      "Progress: 25.59%\n",
      "Progress: 25.61%\n",
      "Progress: 25.62%\n",
      "Progress: 25.63%\n",
      "Progress: 25.64%\n",
      "Progress: 25.65%\n",
      "Progress: 25.67%\n",
      "Progress: 25.68%\n",
      "Progress: 25.69%\n",
      "Progress: 25.70%\n",
      "Progress: 25.72%\n",
      "Progress: 25.73%\n",
      "Progress: 25.74%\n",
      "Progress: 25.75%\n",
      "Progress: 25.76%\n",
      "Progress: 25.78%\n",
      "Progress: 25.79%\n",
      "Progress: 25.80%\n",
      "Progress: 25.81%\n",
      "Progress: 25.83%\n",
      "Progress: 25.84%\n",
      "Progress: 25.85%\n",
      "Progress: 25.86%\n",
      "Progress: 25.87%\n",
      "Progress: 25.89%\n",
      "Progress: 25.90%\n",
      "Progress: 25.91%\n",
      "Progress: 25.92%\n",
      "Progress: 25.94%\n",
      "Progress: 25.95%\n",
      "Progress: 25.96%\n",
      "Progress: 25.97%\n",
      "Progress: 25.99%\n",
      "Progress: 26.00%\n",
      "Progress: 26.01%\n",
      "Progress: 26.02%\n",
      "Progress: 26.03%\n",
      "Progress: 26.05%\n",
      "Progress: 26.06%\n",
      "Progress: 26.07%\n",
      "Progress: 26.08%\n",
      "Progress: 26.10%\n",
      "Progress: 26.11%\n",
      "Progress: 26.12%\n",
      "Progress: 26.13%\n",
      "Progress: 26.14%\n",
      "Progress: 26.16%\n",
      "Progress: 26.17%\n",
      "Progress: 26.18%\n",
      "Progress: 26.19%\n",
      "Progress: 26.21%\n",
      "Progress: 26.22%\n",
      "Progress: 26.23%\n",
      "Progress: 26.24%\n",
      "Progress: 26.25%\n",
      "Progress: 26.27%\n",
      "Progress: 26.28%\n",
      "Progress: 26.29%\n",
      "Progress: 26.30%\n",
      "Progress: 26.32%\n",
      "Progress: 26.33%\n",
      "Progress: 26.34%\n",
      "Progress: 26.35%\n",
      "Progress: 26.36%\n",
      "Progress: 26.38%\n",
      "Progress: 26.39%\n",
      "Progress: 26.40%\n",
      "Progress: 26.41%\n",
      "Progress: 26.43%\n",
      "Progress: 26.44%\n",
      "Progress: 26.45%\n",
      "Progress: 26.46%\n",
      "Progress: 26.47%\n",
      "Progress: 26.49%\n",
      "Progress: 26.50%\n",
      "Progress: 26.51%\n",
      "Progress: 26.52%\n",
      "Progress: 26.53%\n",
      "Progress: 26.55%\n",
      "Progress: 26.56%\n",
      "Progress: 26.57%\n",
      "Progress: 26.58%\n",
      "Progress: 26.60%\n",
      "Progress: 26.61%\n",
      "Progress: 26.62%\n",
      "Progress: 26.63%\n",
      "Progress: 26.64%\n",
      "Progress: 26.66%\n",
      "Progress: 26.67%\n",
      "Progress: 26.68%\n",
      "Progress: 26.69%\n",
      "Progress: 26.71%\n",
      "Progress: 26.72%\n",
      "Progress: 26.73%\n",
      "Progress: 26.74%\n",
      "Progress: 26.75%\n",
      "Progress: 26.77%\n",
      "Progress: 26.78%\n",
      "Progress: 26.79%\n",
      "Progress: 26.80%\n",
      "Progress: 26.82%\n",
      "Progress: 26.83%\n",
      "Progress: 26.84%\n",
      "Progress: 26.85%\n",
      "Progress: 26.86%\n",
      "Progress: 26.88%\n",
      "Progress: 26.89%\n",
      "Progress: 26.90%\n",
      "Progress: 26.91%\n",
      "Progress: 26.92%\n",
      "Progress: 26.94%\n",
      "Progress: 26.95%\n",
      "Progress: 26.96%\n",
      "Progress: 26.97%\n",
      "Progress: 26.99%\n",
      "Progress: 27.00%\n",
      "Progress: 27.01%\n",
      "Progress: 27.02%\n",
      "Progress: 27.03%\n",
      "Progress: 27.05%\n",
      "Progress: 27.06%\n",
      "Progress: 27.07%\n",
      "Progress: 27.08%\n",
      "Progress: 27.10%\n",
      "Progress: 27.11%\n",
      "Progress: 27.12%\n",
      "Progress: 27.13%\n",
      "Progress: 27.14%\n",
      "Progress: 27.16%\n",
      "Progress: 27.17%\n",
      "Progress: 27.18%\n",
      "Progress: 27.19%\n",
      "Progress: 27.20%\n",
      "Progress: 27.22%\n",
      "Progress: 27.23%\n",
      "Progress: 27.24%\n",
      "Progress: 27.25%\n",
      "Progress: 27.27%\n",
      "Progress: 27.28%\n",
      "Progress: 27.29%\n",
      "Progress: 27.30%\n",
      "Progress: 27.31%\n",
      "Progress: 27.33%\n",
      "Progress: 27.34%\n",
      "Progress: 27.35%\n",
      "Progress: 27.36%\n",
      "Progress: 27.37%\n",
      "Progress: 27.39%\n",
      "Progress: 27.40%\n",
      "Progress: 27.41%\n",
      "Progress: 27.42%\n",
      "Progress: 27.43%\n",
      "Progress: 27.45%\n",
      "Progress: 27.46%\n",
      "Progress: 27.47%\n",
      "Progress: 27.48%\n",
      "Progress: 27.50%\n",
      "Progress: 27.51%\n",
      "Progress: 27.52%\n",
      "Progress: 27.53%\n",
      "Progress: 27.54%\n",
      "Progress: 27.56%\n",
      "Progress: 27.57%\n",
      "Progress: 27.58%\n",
      "Progress: 27.59%\n",
      "Progress: 27.60%\n",
      "Progress: 27.62%\n",
      "Progress: 27.63%\n",
      "Progress: 27.64%\n",
      "Progress: 27.65%\n",
      "Progress: 27.66%\n",
      "Progress: 27.68%\n",
      "Progress: 27.69%\n",
      "Progress: 27.70%\n",
      "Progress: 27.71%\n",
      "Progress: 27.73%\n",
      "Progress: 27.74%\n",
      "Progress: 27.75%\n",
      "Progress: 27.76%\n",
      "Progress: 27.77%\n",
      "Progress: 27.79%\n",
      "Progress: 27.80%\n",
      "Progress: 27.81%\n",
      "Progress: 27.82%\n",
      "Progress: 27.83%\n",
      "Progress: 27.85%\n",
      "Progress: 27.86%\n",
      "Progress: 27.87%\n",
      "Progress: 27.88%\n",
      "Progress: 27.89%\n",
      "Progress: 27.91%\n",
      "Progress: 27.92%\n",
      "Progress: 27.93%\n",
      "Progress: 27.94%\n",
      "Progress: 27.96%\n",
      "Progress: 27.97%\n",
      "Progress: 27.98%\n",
      "Progress: 27.99%\n",
      "Progress: 28.00%\n",
      "Progress: 28.02%\n",
      "Progress: 28.03%\n",
      "Progress: 28.04%\n",
      "Progress: 28.05%\n",
      "Progress: 28.06%\n",
      "Progress: 28.08%\n",
      "Progress: 28.09%\n",
      "Progress: 28.10%\n",
      "Progress: 28.11%\n",
      "Progress: 28.12%\n",
      "Progress: 28.14%\n",
      "Progress: 28.15%\n",
      "Progress: 28.16%\n",
      "Progress: 28.17%\n",
      "Progress: 28.18%\n",
      "Progress: 28.20%\n",
      "Progress: 28.21%\n",
      "Progress: 28.22%\n",
      "Progress: 28.23%\n",
      "Progress: 28.24%\n",
      "Progress: 28.26%\n",
      "Progress: 28.27%\n",
      "Progress: 28.28%\n",
      "Progress: 28.29%\n",
      "Progress: 28.31%\n",
      "Progress: 28.32%\n",
      "Progress: 28.33%\n",
      "Progress: 28.34%\n",
      "Progress: 28.35%\n",
      "Progress: 28.37%\n",
      "Progress: 28.38%\n",
      "Progress: 28.39%\n",
      "Progress: 28.40%\n",
      "Progress: 28.41%\n",
      "Progress: 28.43%\n",
      "Progress: 28.44%\n",
      "Progress: 28.45%\n",
      "Progress: 28.46%\n",
      "Progress: 28.47%\n",
      "Progress: 28.49%\n",
      "Progress: 28.50%\n",
      "Progress: 28.51%\n",
      "Progress: 28.52%\n",
      "Progress: 28.53%\n",
      "Progress: 28.55%\n",
      "Progress: 28.56%\n",
      "Progress: 28.57%\n",
      "Progress: 28.58%\n",
      "Progress: 28.59%\n",
      "Progress: 28.61%\n",
      "Progress: 28.62%\n",
      "Progress: 28.63%\n",
      "Progress: 28.64%\n",
      "Progress: 28.65%\n",
      "Progress: 28.67%\n",
      "Progress: 28.68%\n",
      "Progress: 28.69%\n",
      "Progress: 28.70%\n",
      "Progress: 28.71%\n",
      "Progress: 28.73%\n",
      "Progress: 28.74%\n",
      "Progress: 28.75%\n",
      "Progress: 28.76%\n",
      "Progress: 28.77%\n",
      "Progress: 28.79%\n",
      "Progress: 28.80%\n",
      "Progress: 28.81%\n",
      "Progress: 28.82%\n",
      "Progress: 28.83%\n",
      "Progress: 28.85%\n",
      "Progress: 28.86%\n",
      "Progress: 28.87%\n",
      "Progress: 28.88%\n",
      "Progress: 28.89%\n",
      "Progress: 28.91%\n",
      "Progress: 28.92%\n",
      "Progress: 28.93%\n",
      "Progress: 28.94%\n",
      "Progress: 28.95%\n",
      "Progress: 28.97%\n",
      "Progress: 28.98%\n",
      "Progress: 28.99%\n",
      "Progress: 29.00%\n",
      "Progress: 29.01%\n",
      "Progress: 29.03%\n",
      "Progress: 29.04%\n",
      "Progress: 29.05%\n",
      "Progress: 29.06%\n",
      "Progress: 29.07%\n",
      "Progress: 29.09%\n",
      "Progress: 29.10%\n",
      "Progress: 29.11%\n",
      "Progress: 29.12%\n",
      "Progress: 29.13%\n",
      "Progress: 29.15%\n",
      "Progress: 29.16%\n",
      "Progress: 29.17%\n",
      "Progress: 29.18%\n",
      "Progress: 29.19%\n",
      "Progress: 29.21%\n",
      "Progress: 29.22%\n",
      "Progress: 29.23%\n",
      "Progress: 29.24%\n",
      "Progress: 29.25%\n",
      "Progress: 29.27%\n",
      "Progress: 29.28%\n",
      "Progress: 29.29%\n",
      "Progress: 29.30%\n",
      "Progress: 29.31%\n",
      "Progress: 29.33%\n",
      "Progress: 29.34%\n",
      "Progress: 29.35%\n",
      "Progress: 29.36%\n",
      "Progress: 29.37%\n",
      "Progress: 29.39%\n",
      "Progress: 29.40%\n",
      "Progress: 29.41%\n",
      "Progress: 29.42%\n",
      "Progress: 29.43%\n",
      "Progress: 29.45%\n",
      "Progress: 29.46%\n",
      "Progress: 29.47%\n",
      "Progress: 29.48%\n",
      "Progress: 29.49%\n",
      "Progress: 29.51%\n",
      "Progress: 29.52%\n",
      "Progress: 29.53%\n",
      "Progress: 29.54%\n",
      "Progress: 29.55%\n",
      "Progress: 29.56%\n",
      "Progress: 29.58%\n",
      "Progress: 29.59%\n",
      "Progress: 29.60%\n",
      "Progress: 29.61%\n",
      "Progress: 29.62%\n",
      "Progress: 29.64%\n",
      "Progress: 29.65%\n",
      "Progress: 29.66%\n",
      "Progress: 29.67%\n",
      "Progress: 29.68%\n",
      "Progress: 29.70%\n",
      "Progress: 29.71%\n",
      "Progress: 29.72%\n",
      "Progress: 29.73%\n",
      "Progress: 29.74%\n",
      "Progress: 29.76%\n",
      "Progress: 29.77%\n",
      "Progress: 29.78%\n",
      "Progress: 29.79%\n",
      "Progress: 29.80%\n",
      "Progress: 29.82%\n",
      "Progress: 29.83%\n",
      "Progress: 29.84%\n",
      "Progress: 29.85%\n",
      "Progress: 29.86%\n",
      "Progress: 29.87%\n",
      "Progress: 29.89%\n",
      "Progress: 29.90%\n",
      "Progress: 29.91%\n",
      "Progress: 29.92%\n",
      "Progress: 29.93%\n",
      "Progress: 29.95%\n",
      "Progress: 29.96%\n",
      "Progress: 29.97%\n",
      "Progress: 29.98%\n",
      "Progress: 29.99%\n",
      "Progress: 30.01%\n",
      "Progress: 30.02%\n",
      "Progress: 30.03%\n",
      "Progress: 30.04%\n",
      "Progress: 30.05%\n",
      "Progress: 30.07%\n",
      "Progress: 30.08%\n",
      "Progress: 30.09%\n",
      "Progress: 30.10%\n",
      "Progress: 30.11%\n",
      "Progress: 30.12%\n",
      "Progress: 30.14%\n",
      "Progress: 30.15%\n",
      "Progress: 30.16%\n",
      "Progress: 30.17%\n",
      "Progress: 30.18%\n",
      "Progress: 30.20%\n",
      "Progress: 30.21%\n",
      "Progress: 30.22%\n",
      "Progress: 30.23%\n",
      "Progress: 30.24%\n",
      "Progress: 30.26%\n",
      "Progress: 30.27%\n",
      "Progress: 30.28%\n",
      "Progress: 30.29%\n",
      "Progress: 30.30%\n",
      "Progress: 30.32%\n",
      "Progress: 30.33%\n",
      "Progress: 30.34%\n",
      "Progress: 30.35%\n",
      "Progress: 30.36%\n",
      "Progress: 30.37%\n",
      "Progress: 30.39%\n",
      "Progress: 30.40%\n",
      "Progress: 30.41%\n",
      "Progress: 30.42%\n",
      "Progress: 30.43%\n",
      "Progress: 30.45%\n",
      "Progress: 30.46%\n",
      "Progress: 30.47%\n",
      "Progress: 30.48%\n",
      "Progress: 30.49%\n",
      "Progress: 30.51%\n",
      "Progress: 30.52%\n",
      "Progress: 30.53%\n",
      "Progress: 30.54%\n",
      "Progress: 30.55%\n",
      "Progress: 30.56%\n",
      "Progress: 30.58%\n",
      "Progress: 30.59%\n",
      "Progress: 30.60%\n",
      "Progress: 30.61%\n",
      "Progress: 30.62%\n",
      "Progress: 30.64%\n",
      "Progress: 30.65%\n",
      "Progress: 30.66%\n",
      "Progress: 30.67%\n",
      "Progress: 30.68%\n",
      "Progress: 30.69%\n",
      "Progress: 30.71%\n",
      "Progress: 30.72%\n",
      "Progress: 30.73%\n",
      "Progress: 30.74%\n",
      "Progress: 30.75%\n",
      "Progress: 30.77%\n",
      "Progress: 30.78%\n",
      "Progress: 30.79%\n",
      "Progress: 30.80%\n",
      "Progress: 30.81%\n",
      "Progress: 30.83%\n",
      "Progress: 30.84%\n",
      "Progress: 30.85%\n",
      "Progress: 30.86%\n",
      "Progress: 30.87%\n",
      "Progress: 30.88%\n",
      "Progress: 30.90%\n",
      "Progress: 30.91%\n",
      "Progress: 30.92%\n",
      "Progress: 30.93%\n",
      "Progress: 30.94%\n",
      "Progress: 30.96%\n",
      "Progress: 30.97%\n",
      "Progress: 30.98%\n",
      "Progress: 30.99%\n",
      "Progress: 31.00%\n",
      "Progress: 31.01%\n",
      "Progress: 31.03%\n",
      "Progress: 31.04%\n",
      "Progress: 31.05%\n",
      "Progress: 31.06%\n",
      "Progress: 31.07%\n",
      "Progress: 31.09%\n",
      "Progress: 31.10%\n",
      "Progress: 31.11%\n",
      "Progress: 31.12%\n",
      "Progress: 31.13%\n",
      "Progress: 31.14%\n",
      "Progress: 31.16%\n",
      "Progress: 31.17%\n",
      "Progress: 31.18%\n",
      "Progress: 31.19%\n",
      "Progress: 31.20%\n",
      "Progress: 31.22%\n",
      "Progress: 31.23%\n",
      "Progress: 31.24%\n",
      "Progress: 31.25%\n",
      "Progress: 31.26%\n",
      "Progress: 31.27%\n",
      "Progress: 31.29%\n",
      "Progress: 31.30%\n",
      "Progress: 31.31%\n",
      "Progress: 31.32%\n",
      "Progress: 31.33%\n",
      "Progress: 31.34%\n",
      "Progress: 31.36%\n",
      "Progress: 31.37%\n",
      "Progress: 31.38%\n",
      "Progress: 31.39%\n",
      "Progress: 31.40%\n",
      "Progress: 31.42%\n",
      "Progress: 31.43%\n",
      "Progress: 31.44%\n",
      "Progress: 31.45%\n",
      "Progress: 31.46%\n",
      "Progress: 31.47%\n",
      "Progress: 31.49%\n",
      "Progress: 31.50%\n",
      "Progress: 31.51%\n",
      "Progress: 31.52%\n",
      "Progress: 31.53%\n",
      "Progress: 31.55%\n",
      "Progress: 31.56%\n",
      "Progress: 31.57%\n",
      "Progress: 31.58%\n",
      "Progress: 31.59%\n",
      "Progress: 31.60%\n",
      "Progress: 31.62%\n",
      "Progress: 31.63%\n",
      "Progress: 31.64%\n",
      "Progress: 31.65%\n",
      "Progress: 31.66%\n",
      "Progress: 31.67%\n",
      "Progress: 31.69%\n",
      "Progress: 31.70%\n",
      "Progress: 31.71%\n",
      "Progress: 31.72%\n",
      "Progress: 31.73%\n",
      "Progress: 31.75%\n",
      "Progress: 31.76%\n",
      "Progress: 31.77%\n",
      "Progress: 31.78%\n",
      "Progress: 31.79%\n",
      "Progress: 31.80%\n",
      "Progress: 31.82%\n",
      "Progress: 31.83%\n",
      "Progress: 31.84%\n",
      "Progress: 31.85%\n",
      "Progress: 31.86%\n",
      "Progress: 31.87%\n",
      "Progress: 31.89%\n",
      "Progress: 31.90%\n",
      "Progress: 31.91%\n",
      "Progress: 31.92%\n",
      "Progress: 31.93%\n",
      "Progress: 31.94%\n",
      "Progress: 31.96%\n",
      "Progress: 31.97%\n",
      "Progress: 31.98%\n",
      "Progress: 31.99%\n",
      "Progress: 32.00%\n",
      "Progress: 32.02%\n",
      "Progress: 32.03%\n",
      "Progress: 32.04%\n",
      "Progress: 32.05%\n",
      "Progress: 32.06%\n",
      "Progress: 32.07%\n",
      "Progress: 32.09%\n",
      "Progress: 32.10%\n",
      "Progress: 32.11%\n",
      "Progress: 32.12%\n",
      "Progress: 32.13%\n",
      "Progress: 32.14%\n",
      "Progress: 32.16%\n",
      "Progress: 32.17%\n",
      "Progress: 32.18%\n",
      "Progress: 32.19%\n",
      "Progress: 32.20%\n",
      "Progress: 32.21%\n",
      "Progress: 32.23%\n",
      "Progress: 32.24%\n",
      "Progress: 32.25%\n",
      "Progress: 32.26%\n",
      "Progress: 32.27%\n",
      "Progress: 32.29%\n",
      "Progress: 32.30%\n",
      "Progress: 32.31%\n",
      "Progress: 32.32%\n",
      "Progress: 32.33%\n",
      "Progress: 32.34%\n",
      "Progress: 32.36%\n",
      "Progress: 32.37%\n",
      "Progress: 32.38%\n",
      "Progress: 32.39%\n",
      "Progress: 32.40%\n",
      "Progress: 32.41%\n",
      "Progress: 32.43%\n",
      "Progress: 32.44%\n",
      "Progress: 32.45%\n",
      "Progress: 32.46%\n",
      "Progress: 32.47%\n",
      "Progress: 32.48%\n",
      "Progress: 32.50%\n",
      "Progress: 32.51%\n",
      "Progress: 32.52%\n",
      "Progress: 32.53%\n",
      "Progress: 32.54%\n",
      "Progress: 32.55%\n",
      "Progress: 32.57%\n",
      "Progress: 32.58%\n",
      "Progress: 32.59%\n",
      "Progress: 32.60%\n",
      "Progress: 32.61%\n",
      "Progress: 32.62%\n",
      "Progress: 32.64%\n",
      "Progress: 32.65%\n",
      "Progress: 32.66%\n",
      "Progress: 32.67%\n",
      "Progress: 32.68%\n",
      "Progress: 32.69%\n",
      "Progress: 32.71%\n",
      "Progress: 32.72%\n",
      "Progress: 32.73%\n",
      "Progress: 32.74%\n",
      "Progress: 32.75%\n",
      "Progress: 32.76%\n",
      "Progress: 32.78%\n",
      "Progress: 32.79%\n",
      "Progress: 32.80%\n",
      "Progress: 32.81%\n",
      "Progress: 32.82%\n",
      "Progress: 32.83%\n",
      "Progress: 32.85%\n",
      "Progress: 32.86%\n",
      "Progress: 32.87%\n",
      "Progress: 32.88%\n",
      "Progress: 32.89%\n",
      "Progress: 32.90%\n",
      "Progress: 32.92%\n",
      "Progress: 32.93%\n",
      "Progress: 32.94%\n",
      "Progress: 32.95%\n",
      "Progress: 32.96%\n",
      "Progress: 32.97%\n",
      "Progress: 32.99%\n",
      "Progress: 33.00%\n",
      "Progress: 33.01%\n",
      "Progress: 33.02%\n",
      "Progress: 33.03%\n",
      "Progress: 33.04%\n",
      "Progress: 33.06%\n",
      "Progress: 33.07%\n",
      "Progress: 33.08%\n",
      "Progress: 33.09%\n",
      "Progress: 33.10%\n",
      "Progress: 33.11%\n",
      "Progress: 33.13%\n",
      "Progress: 33.14%\n",
      "Progress: 33.15%\n",
      "Progress: 33.16%\n",
      "Progress: 33.17%\n",
      "Progress: 33.18%\n",
      "Progress: 33.20%\n",
      "Progress: 33.21%\n",
      "Progress: 33.22%\n",
      "Progress: 33.23%\n",
      "Progress: 33.24%\n",
      "Progress: 33.25%\n",
      "Progress: 33.27%\n",
      "Progress: 33.28%\n",
      "Progress: 33.29%\n",
      "Progress: 33.30%\n",
      "Progress: 33.31%\n",
      "Progress: 33.32%\n",
      "Progress: 33.33%\n",
      "Progress: 33.35%\n",
      "Progress: 33.36%\n",
      "Progress: 33.37%\n",
      "Progress: 33.38%\n",
      "Progress: 33.39%\n",
      "Progress: 33.40%\n",
      "Progress: 33.42%\n",
      "Progress: 33.43%\n",
      "Progress: 33.44%\n",
      "Progress: 33.45%\n",
      "Progress: 33.46%\n",
      "Progress: 33.47%\n",
      "Progress: 33.49%\n",
      "Progress: 33.50%\n",
      "Progress: 33.51%\n",
      "Progress: 33.52%\n",
      "Progress: 33.53%\n",
      "Progress: 33.54%\n",
      "Progress: 33.56%\n",
      "Progress: 33.57%\n",
      "Progress: 33.58%\n",
      "Progress: 33.59%\n",
      "Progress: 33.60%\n",
      "Progress: 33.61%\n",
      "Progress: 33.63%\n",
      "Progress: 33.64%\n",
      "Progress: 33.65%\n",
      "Progress: 33.66%\n",
      "Progress: 33.67%\n",
      "Progress: 33.68%\n",
      "Progress: 33.69%\n",
      "Progress: 33.71%\n",
      "Progress: 33.72%\n",
      "Progress: 33.73%\n",
      "Progress: 33.74%\n",
      "Progress: 33.75%\n",
      "Progress: 33.76%\n",
      "Progress: 33.78%\n",
      "Progress: 33.79%\n",
      "Progress: 33.80%\n",
      "Progress: 33.81%\n",
      "Progress: 33.82%\n",
      "Progress: 33.83%\n",
      "Progress: 33.85%\n",
      "Progress: 33.86%\n",
      "Progress: 33.87%\n",
      "Progress: 33.88%\n",
      "Progress: 33.89%\n",
      "Progress: 33.90%\n",
      "Progress: 33.91%\n",
      "Progress: 33.93%\n",
      "Progress: 33.94%\n",
      "Progress: 33.95%\n",
      "Progress: 33.96%\n",
      "Progress: 33.97%\n",
      "Progress: 33.98%\n",
      "Progress: 34.00%\n",
      "Progress: 34.01%\n",
      "Progress: 34.02%\n",
      "Progress: 34.03%\n",
      "Progress: 34.04%\n",
      "Progress: 34.05%\n",
      "Progress: 34.06%\n",
      "Progress: 34.08%\n",
      "Progress: 34.09%\n",
      "Progress: 34.10%\n",
      "Progress: 34.11%\n",
      "Progress: 34.12%\n",
      "Progress: 34.13%\n",
      "Progress: 34.15%\n",
      "Progress: 34.16%\n",
      "Progress: 34.17%\n",
      "Progress: 34.18%\n",
      "Progress: 34.19%\n",
      "Progress: 34.20%\n",
      "Progress: 34.22%\n",
      "Progress: 34.23%\n",
      "Progress: 34.24%\n",
      "Progress: 34.25%\n",
      "Progress: 34.26%\n",
      "Progress: 34.27%\n",
      "Progress: 34.28%\n",
      "Progress: 34.30%\n",
      "Progress: 34.31%\n",
      "Progress: 34.32%\n",
      "Progress: 34.33%\n",
      "Progress: 34.34%\n",
      "Progress: 34.35%\n",
      "Progress: 34.37%\n",
      "Progress: 34.38%\n",
      "Progress: 34.39%\n",
      "Progress: 34.40%\n",
      "Progress: 34.41%\n",
      "Progress: 34.42%\n",
      "Progress: 34.43%\n",
      "Progress: 34.45%\n",
      "Progress: 34.46%\n",
      "Progress: 34.47%\n",
      "Progress: 34.48%\n",
      "Progress: 34.49%\n",
      "Progress: 34.50%\n",
      "Progress: 34.51%\n",
      "Progress: 34.53%\n",
      "Progress: 34.54%\n",
      "Progress: 34.55%\n",
      "Progress: 34.56%\n",
      "Progress: 34.57%\n",
      "Progress: 34.58%\n",
      "Progress: 34.60%\n",
      "Progress: 34.61%\n",
      "Progress: 34.62%\n",
      "Progress: 34.63%\n",
      "Progress: 34.64%\n",
      "Progress: 34.65%\n",
      "Progress: 34.66%\n",
      "Progress: 34.68%\n",
      "Progress: 34.69%\n",
      "Progress: 34.70%\n",
      "Progress: 34.71%\n",
      "Progress: 34.72%\n",
      "Progress: 34.73%\n",
      "Progress: 34.74%\n",
      "Progress: 34.76%\n",
      "Progress: 34.77%\n",
      "Progress: 34.78%\n",
      "Progress: 34.79%\n",
      "Progress: 34.80%\n",
      "Progress: 34.81%\n",
      "Progress: 34.83%\n",
      "Progress: 34.84%\n",
      "Progress: 34.85%\n",
      "Progress: 34.86%\n",
      "Progress: 34.87%\n",
      "Progress: 34.88%\n",
      "Progress: 34.89%\n",
      "Progress: 34.91%\n",
      "Progress: 34.92%\n",
      "Progress: 34.93%\n",
      "Progress: 34.94%\n",
      "Progress: 34.95%\n",
      "Progress: 34.96%\n",
      "Progress: 34.97%\n",
      "Progress: 34.99%\n",
      "Progress: 35.00%\n",
      "Progress: 35.01%\n",
      "Progress: 35.02%\n",
      "Progress: 35.03%\n",
      "Progress: 35.04%\n",
      "Progress: 35.05%\n",
      "Progress: 35.07%\n",
      "Progress: 35.08%\n",
      "Progress: 35.09%\n",
      "Progress: 35.10%\n",
      "Progress: 35.11%\n",
      "Progress: 35.12%\n",
      "Progress: 35.14%\n",
      "Progress: 35.15%\n",
      "Progress: 35.16%\n",
      "Progress: 35.17%\n",
      "Progress: 35.18%\n",
      "Progress: 35.19%\n",
      "Progress: 35.20%\n",
      "Progress: 35.22%\n",
      "Progress: 35.23%\n",
      "Progress: 35.24%\n",
      "Progress: 35.25%\n",
      "Progress: 35.26%\n",
      "Progress: 35.27%\n",
      "Progress: 35.28%\n",
      "Progress: 35.30%\n",
      "Progress: 35.31%\n",
      "Progress: 35.32%\n",
      "Progress: 35.33%\n",
      "Progress: 35.34%\n",
      "Progress: 35.35%\n",
      "Progress: 35.36%\n",
      "Progress: 35.38%\n",
      "Progress: 35.39%\n",
      "Progress: 35.40%\n",
      "Progress: 35.41%\n",
      "Progress: 35.42%\n",
      "Progress: 35.43%\n",
      "Progress: 35.44%\n",
      "Progress: 35.46%\n",
      "Progress: 35.47%\n",
      "Progress: 35.48%\n",
      "Progress: 35.49%\n",
      "Progress: 35.50%\n",
      "Progress: 35.51%\n",
      "Progress: 35.52%\n",
      "Progress: 35.54%\n",
      "Progress: 35.55%\n",
      "Progress: 35.56%\n",
      "Progress: 35.57%\n",
      "Progress: 35.58%\n",
      "Progress: 35.59%\n",
      "Progress: 35.60%\n",
      "Progress: 35.62%\n",
      "Progress: 35.63%\n",
      "Progress: 35.64%\n",
      "Progress: 35.65%\n",
      "Progress: 35.66%\n",
      "Progress: 35.67%\n",
      "Progress: 35.68%\n",
      "Progress: 35.70%\n",
      "Progress: 35.71%\n",
      "Progress: 35.72%\n",
      "Progress: 35.73%\n",
      "Progress: 35.74%\n",
      "Progress: 35.75%\n",
      "Progress: 35.76%\n",
      "Progress: 35.78%\n",
      "Progress: 35.79%\n",
      "Progress: 35.80%\n",
      "Progress: 35.81%\n",
      "Progress: 35.82%\n",
      "Progress: 35.83%\n",
      "Progress: 35.84%\n",
      "Progress: 35.86%\n",
      "Progress: 35.87%\n",
      "Progress: 35.88%\n",
      "Progress: 35.89%\n",
      "Progress: 35.90%\n",
      "Progress: 35.91%\n",
      "Progress: 35.92%\n",
      "Progress: 35.94%\n",
      "Progress: 35.95%\n",
      "Progress: 35.96%\n",
      "Progress: 35.97%\n",
      "Progress: 35.98%\n",
      "Progress: 35.99%\n",
      "Progress: 36.00%\n",
      "Progress: 36.01%\n",
      "Progress: 36.03%\n",
      "Progress: 36.04%\n",
      "Progress: 36.05%\n",
      "Progress: 36.06%\n",
      "Progress: 36.07%\n",
      "Progress: 36.08%\n",
      "Progress: 36.09%\n",
      "Progress: 36.11%\n",
      "Progress: 36.12%\n",
      "Progress: 36.13%\n",
      "Progress: 36.14%\n",
      "Progress: 36.15%\n",
      "Progress: 36.16%\n",
      "Progress: 36.17%\n",
      "Progress: 36.19%\n",
      "Progress: 36.20%\n",
      "Progress: 36.21%\n",
      "Progress: 36.22%\n",
      "Progress: 36.23%\n",
      "Progress: 36.24%\n",
      "Progress: 36.25%\n",
      "Progress: 36.26%\n",
      "Progress: 36.28%\n",
      "Progress: 36.29%\n",
      "Progress: 36.30%\n",
      "Progress: 36.31%\n",
      "Progress: 36.32%\n",
      "Progress: 36.33%\n",
      "Progress: 36.34%\n",
      "Progress: 36.36%\n",
      "Progress: 36.37%\n",
      "Progress: 36.38%\n",
      "Progress: 36.39%\n",
      "Progress: 36.40%\n",
      "Progress: 36.41%\n",
      "Progress: 36.42%\n",
      "Progress: 36.44%\n",
      "Progress: 36.45%\n",
      "Progress: 36.46%\n",
      "Progress: 36.47%\n",
      "Progress: 36.48%\n",
      "Progress: 36.49%\n",
      "Progress: 36.50%\n",
      "Progress: 36.51%\n",
      "Progress: 36.53%\n",
      "Progress: 36.54%\n",
      "Progress: 36.55%\n",
      "Progress: 36.56%\n",
      "Progress: 36.57%\n",
      "Progress: 36.58%\n",
      "Progress: 36.59%\n",
      "Progress: 36.61%\n",
      "Progress: 36.62%\n",
      "Progress: 36.63%\n",
      "Progress: 36.64%\n",
      "Progress: 36.65%\n",
      "Progress: 36.66%\n",
      "Progress: 36.67%\n",
      "Progress: 36.68%\n",
      "Progress: 36.70%\n",
      "Progress: 36.71%\n",
      "Progress: 36.72%\n",
      "Progress: 36.73%\n",
      "Progress: 36.74%\n",
      "Progress: 36.75%\n",
      "Progress: 36.76%\n",
      "Progress: 36.78%\n",
      "Progress: 36.79%\n",
      "Progress: 36.80%\n",
      "Progress: 36.81%\n",
      "Progress: 36.82%\n",
      "Progress: 36.83%\n",
      "Progress: 36.84%\n",
      "Progress: 36.85%\n",
      "Progress: 36.87%\n",
      "Progress: 36.88%\n",
      "Progress: 36.89%\n",
      "Progress: 36.90%\n",
      "Progress: 36.91%\n",
      "Progress: 36.92%\n",
      "Progress: 36.93%\n",
      "Progress: 36.94%\n",
      "Progress: 36.96%\n",
      "Progress: 36.97%\n",
      "Progress: 36.98%\n",
      "Progress: 36.99%\n",
      "Progress: 37.00%\n",
      "Progress: 37.01%\n",
      "Progress: 37.02%\n",
      "Progress: 37.04%\n",
      "Progress: 37.05%\n",
      "Progress: 37.06%\n",
      "Progress: 37.07%\n",
      "Progress: 37.08%\n",
      "Progress: 37.09%\n",
      "Progress: 37.10%\n",
      "Progress: 37.11%\n",
      "Progress: 37.13%\n",
      "Progress: 37.14%\n",
      "Progress: 37.15%\n",
      "Progress: 37.16%\n",
      "Progress: 37.17%\n",
      "Progress: 37.18%\n",
      "Progress: 37.19%\n",
      "Progress: 37.20%\n",
      "Progress: 37.22%\n",
      "Progress: 37.23%\n",
      "Progress: 37.24%\n",
      "Progress: 37.25%\n",
      "Progress: 37.26%\n",
      "Progress: 37.27%\n",
      "Progress: 37.28%\n",
      "Progress: 37.29%\n",
      "Progress: 37.31%\n",
      "Progress: 37.32%\n",
      "Progress: 37.33%\n",
      "Progress: 37.34%\n",
      "Progress: 37.35%\n",
      "Progress: 37.36%\n",
      "Progress: 37.37%\n",
      "Progress: 37.38%\n",
      "Progress: 37.40%\n",
      "Progress: 37.41%\n",
      "Progress: 37.42%\n",
      "Progress: 37.43%\n",
      "Progress: 37.44%\n",
      "Progress: 37.45%\n",
      "Progress: 37.46%\n",
      "Progress: 37.47%\n",
      "Progress: 37.49%\n",
      "Progress: 37.50%\n",
      "Progress: 37.51%\n",
      "Progress: 37.52%\n",
      "Progress: 37.53%\n",
      "Progress: 37.54%\n",
      "Progress: 37.55%\n",
      "Progress: 37.56%\n",
      "Progress: 37.58%\n",
      "Progress: 37.59%\n",
      "Progress: 37.60%\n",
      "Progress: 37.61%\n",
      "Progress: 37.62%\n",
      "Progress: 37.63%\n",
      "Progress: 37.64%\n",
      "Progress: 37.65%\n",
      "Progress: 37.67%\n",
      "Progress: 37.68%\n",
      "Progress: 37.69%\n",
      "Progress: 37.70%\n",
      "Progress: 37.71%\n",
      "Progress: 37.72%\n",
      "Progress: 37.73%\n",
      "Progress: 37.74%\n",
      "Progress: 37.76%\n",
      "Progress: 37.77%\n",
      "Progress: 37.78%\n",
      "Progress: 37.79%\n",
      "Progress: 37.80%\n",
      "Progress: 37.81%\n",
      "Progress: 37.82%\n",
      "Progress: 37.83%\n",
      "Progress: 37.85%\n",
      "Progress: 37.86%\n",
      "Progress: 37.87%\n",
      "Progress: 37.88%\n",
      "Progress: 37.89%\n",
      "Progress: 37.90%\n",
      "Progress: 37.91%\n",
      "Progress: 37.92%\n",
      "Progress: 37.94%\n",
      "Progress: 37.95%\n",
      "Progress: 37.96%\n",
      "Progress: 37.97%\n",
      "Progress: 37.98%\n",
      "Progress: 37.99%\n",
      "Progress: 38.00%\n",
      "Progress: 38.01%\n",
      "Progress: 38.03%\n",
      "Progress: 38.04%\n",
      "Progress: 38.05%\n",
      "Progress: 38.06%\n",
      "Progress: 38.07%\n",
      "Progress: 38.08%\n",
      "Progress: 38.09%\n",
      "Progress: 38.10%\n",
      "Progress: 38.11%\n",
      "Progress: 38.13%\n",
      "Progress: 38.14%\n",
      "Progress: 38.15%\n",
      "Progress: 38.16%\n",
      "Progress: 38.17%\n",
      "Progress: 38.18%\n",
      "Progress: 38.19%\n",
      "Progress: 38.20%\n",
      "Progress: 38.22%\n",
      "Progress: 38.23%\n",
      "Progress: 38.24%\n",
      "Progress: 38.25%\n",
      "Progress: 38.26%\n",
      "Progress: 38.27%\n",
      "Progress: 38.28%\n",
      "Progress: 38.29%\n",
      "Progress: 38.30%\n",
      "Progress: 38.32%\n",
      "Progress: 38.33%\n",
      "Progress: 38.34%\n",
      "Progress: 38.35%\n",
      "Progress: 38.36%\n",
      "Progress: 38.37%\n",
      "Progress: 38.38%\n",
      "Progress: 38.39%\n",
      "Progress: 38.41%\n",
      "Progress: 38.42%\n",
      "Progress: 38.43%\n",
      "Progress: 38.44%\n",
      "Progress: 38.45%\n",
      "Progress: 38.46%\n",
      "Progress: 38.47%\n",
      "Progress: 38.48%\n",
      "Progress: 38.49%\n",
      "Progress: 38.51%\n",
      "Progress: 38.52%\n",
      "Progress: 38.53%\n",
      "Progress: 38.54%\n",
      "Progress: 38.55%\n",
      "Progress: 38.56%\n",
      "Progress: 38.57%\n",
      "Progress: 38.58%\n",
      "Progress: 38.60%\n",
      "Progress: 38.61%\n",
      "Progress: 38.62%\n",
      "Progress: 38.63%\n",
      "Progress: 38.64%\n",
      "Progress: 38.65%\n",
      "Progress: 38.66%\n",
      "Progress: 38.67%\n",
      "Progress: 38.68%\n",
      "Progress: 38.70%\n",
      "Progress: 38.71%\n",
      "Progress: 38.72%\n",
      "Progress: 38.73%\n",
      "Progress: 38.74%\n",
      "Progress: 38.75%\n",
      "Progress: 38.76%\n",
      "Progress: 38.77%\n",
      "Progress: 38.78%\n",
      "Progress: 38.80%\n",
      "Progress: 38.81%\n",
      "Progress: 38.82%\n",
      "Progress: 38.83%\n",
      "Progress: 38.84%\n",
      "Progress: 38.85%\n",
      "Progress: 38.86%\n",
      "Progress: 38.87%\n",
      "Progress: 38.88%\n",
      "Progress: 38.90%\n",
      "Progress: 38.91%\n",
      "Progress: 38.92%\n",
      "Progress: 38.93%\n",
      "Progress: 38.94%\n",
      "Progress: 38.95%\n",
      "Progress: 38.96%\n",
      "Progress: 38.97%\n",
      "Progress: 38.98%\n",
      "Progress: 39.00%\n",
      "Progress: 39.01%\n",
      "Progress: 39.02%\n",
      "Progress: 39.03%\n",
      "Progress: 39.04%\n",
      "Progress: 39.05%\n",
      "Progress: 39.06%\n",
      "Progress: 39.07%\n",
      "Progress: 39.08%\n",
      "Progress: 39.10%\n",
      "Progress: 39.11%\n",
      "Progress: 39.12%\n",
      "Progress: 39.13%\n",
      "Progress: 39.14%\n",
      "Progress: 39.15%\n",
      "Progress: 39.16%\n",
      "Progress: 39.17%\n",
      "Progress: 39.18%\n",
      "Progress: 39.20%\n",
      "Progress: 39.21%\n",
      "Progress: 39.22%\n",
      "Progress: 39.23%\n",
      "Progress: 39.24%\n",
      "Progress: 39.25%\n",
      "Progress: 39.26%\n",
      "Progress: 39.27%\n",
      "Progress: 39.28%\n",
      "Progress: 39.30%\n",
      "Progress: 39.31%\n",
      "Progress: 39.32%\n",
      "Progress: 39.33%\n",
      "Progress: 39.34%\n",
      "Progress: 39.35%\n",
      "Progress: 39.36%\n",
      "Progress: 39.37%\n",
      "Progress: 39.38%\n",
      "Progress: 39.40%\n",
      "Progress: 39.41%\n",
      "Progress: 39.42%\n",
      "Progress: 39.43%\n",
      "Progress: 39.44%\n",
      "Progress: 39.45%\n",
      "Progress: 39.46%\n",
      "Progress: 39.47%\n",
      "Progress: 39.48%\n",
      "Progress: 39.50%\n",
      "Progress: 39.51%\n",
      "Progress: 39.52%\n",
      "Progress: 39.53%\n",
      "Progress: 39.54%\n",
      "Progress: 39.55%\n",
      "Progress: 39.56%\n",
      "Progress: 39.57%\n",
      "Progress: 39.58%\n",
      "Progress: 39.59%\n",
      "Progress: 39.61%\n",
      "Progress: 39.62%\n",
      "Progress: 39.63%\n",
      "Progress: 39.64%\n",
      "Progress: 39.65%\n",
      "Progress: 39.66%\n",
      "Progress: 39.67%\n",
      "Progress: 39.68%\n",
      "Progress: 39.69%\n",
      "Progress: 39.71%\n",
      "Progress: 39.72%\n",
      "Progress: 39.73%\n",
      "Progress: 39.74%\n",
      "Progress: 39.75%\n",
      "Progress: 39.76%\n",
      "Progress: 39.77%\n",
      "Progress: 39.78%\n",
      "Progress: 39.79%\n",
      "Progress: 39.80%\n",
      "Progress: 39.82%\n",
      "Progress: 39.83%\n",
      "Progress: 39.84%\n",
      "Progress: 39.85%\n",
      "Progress: 39.86%\n",
      "Progress: 39.87%\n",
      "Progress: 39.88%\n",
      "Progress: 39.89%\n",
      "Progress: 39.90%\n",
      "Progress: 39.92%\n",
      "Progress: 39.93%\n",
      "Progress: 39.94%\n",
      "Progress: 39.95%\n",
      "Progress: 39.96%\n",
      "Progress: 39.97%\n",
      "Progress: 39.98%\n",
      "Progress: 39.99%\n",
      "Progress: 40.00%\n",
      "Progress: 40.01%\n",
      "Progress: 40.03%\n",
      "Progress: 40.04%\n",
      "Progress: 40.05%\n",
      "Progress: 40.06%\n",
      "Progress: 40.07%\n",
      "Progress: 40.08%\n",
      "Progress: 40.09%\n",
      "Progress: 40.10%\n",
      "Progress: 40.11%\n",
      "Progress: 40.12%\n",
      "Progress: 40.14%\n",
      "Progress: 40.15%\n",
      "Progress: 40.16%\n",
      "Progress: 40.17%\n",
      "Progress: 40.18%\n",
      "Progress: 40.19%\n",
      "Progress: 40.20%\n",
      "Progress: 40.21%\n",
      "Progress: 40.22%\n",
      "Progress: 40.23%\n",
      "Progress: 40.25%\n",
      "Progress: 40.26%\n",
      "Progress: 40.27%\n",
      "Progress: 40.28%\n",
      "Progress: 40.29%\n",
      "Progress: 40.30%\n",
      "Progress: 40.31%\n",
      "Progress: 40.32%\n",
      "Progress: 40.33%\n",
      "Progress: 40.34%\n",
      "Progress: 40.36%\n",
      "Progress: 40.37%\n",
      "Progress: 40.38%\n",
      "Progress: 40.39%\n",
      "Progress: 40.40%\n",
      "Progress: 40.41%\n",
      "Progress: 40.42%\n",
      "Progress: 40.43%\n",
      "Progress: 40.44%\n",
      "Progress: 40.45%\n",
      "Progress: 40.47%\n",
      "Progress: 40.48%\n",
      "Progress: 40.49%\n",
      "Progress: 40.50%\n",
      "Progress: 40.51%\n",
      "Progress: 40.52%\n",
      "Progress: 40.53%\n",
      "Progress: 40.54%\n",
      "Progress: 40.55%\n",
      "Progress: 40.56%\n",
      "Progress: 40.58%\n",
      "Progress: 40.59%\n",
      "Progress: 40.60%\n",
      "Progress: 40.61%\n",
      "Progress: 40.62%\n",
      "Progress: 40.63%\n",
      "Progress: 40.64%\n",
      "Progress: 40.65%\n",
      "Progress: 40.66%\n",
      "Progress: 40.67%\n",
      "Progress: 40.68%\n",
      "Progress: 40.70%\n",
      "Progress: 40.71%\n",
      "Progress: 40.72%\n",
      "Progress: 40.73%\n",
      "Progress: 40.74%\n",
      "Progress: 40.75%\n",
      "Progress: 40.76%\n",
      "Progress: 40.77%\n",
      "Progress: 40.78%\n",
      "Progress: 40.79%\n",
      "Progress: 40.81%\n",
      "Progress: 40.82%\n",
      "Progress: 40.83%\n",
      "Progress: 40.84%\n",
      "Progress: 40.85%\n",
      "Progress: 40.86%\n",
      "Progress: 40.87%\n",
      "Progress: 40.88%\n",
      "Progress: 40.89%\n",
      "Progress: 40.90%\n",
      "Progress: 40.91%\n",
      "Progress: 40.93%\n",
      "Progress: 40.94%\n",
      "Progress: 40.95%\n",
      "Progress: 40.96%\n",
      "Progress: 40.97%\n",
      "Progress: 40.98%\n",
      "Progress: 40.99%\n",
      "Progress: 41.00%\n",
      "Progress: 41.01%\n",
      "Progress: 41.02%\n",
      "Progress: 41.03%\n",
      "Progress: 41.05%\n",
      "Progress: 41.06%\n",
      "Progress: 41.07%\n",
      "Progress: 41.08%\n",
      "Progress: 41.09%\n",
      "Progress: 41.10%\n",
      "Progress: 41.11%\n",
      "Progress: 41.12%\n",
      "Progress: 41.13%\n",
      "Progress: 41.14%\n",
      "Progress: 41.16%\n",
      "Progress: 41.17%\n",
      "Progress: 41.18%\n",
      "Progress: 41.19%\n",
      "Progress: 41.20%\n",
      "Progress: 41.21%\n",
      "Progress: 41.22%\n",
      "Progress: 41.23%\n",
      "Progress: 41.24%\n",
      "Progress: 41.25%\n",
      "Progress: 41.26%\n",
      "Progress: 41.28%\n",
      "Progress: 41.29%\n",
      "Progress: 41.30%\n",
      "Progress: 41.31%\n",
      "Progress: 41.32%\n",
      "Progress: 41.33%\n",
      "Progress: 41.34%\n",
      "Progress: 41.35%\n",
      "Progress: 41.36%\n",
      "Progress: 41.37%\n",
      "Progress: 41.38%\n",
      "Progress: 41.40%\n",
      "Progress: 41.41%\n",
      "Progress: 41.42%\n",
      "Progress: 41.43%\n",
      "Progress: 41.44%\n",
      "Progress: 41.45%\n",
      "Progress: 41.46%\n",
      "Progress: 41.47%\n",
      "Progress: 41.48%\n",
      "Progress: 41.49%\n",
      "Progress: 41.50%\n",
      "Progress: 41.51%\n",
      "Progress: 41.53%\n",
      "Progress: 41.54%\n",
      "Progress: 41.55%\n",
      "Progress: 41.56%\n",
      "Progress: 41.57%\n",
      "Progress: 41.58%\n",
      "Progress: 41.59%\n",
      "Progress: 41.60%\n",
      "Progress: 41.61%\n",
      "Progress: 41.62%\n",
      "Progress: 41.63%\n",
      "Progress: 41.65%\n",
      "Progress: 41.66%\n",
      "Progress: 41.67%\n",
      "Progress: 41.68%\n",
      "Progress: 41.69%\n",
      "Progress: 41.70%\n",
      "Progress: 41.71%\n",
      "Progress: 41.72%\n",
      "Progress: 41.73%\n",
      "Progress: 41.74%\n",
      "Progress: 41.75%\n",
      "Progress: 41.76%\n",
      "Progress: 41.78%\n",
      "Progress: 41.79%\n",
      "Progress: 41.80%\n",
      "Progress: 41.81%\n",
      "Progress: 41.82%\n",
      "Progress: 41.83%\n",
      "Progress: 41.84%\n",
      "Progress: 41.85%\n",
      "Progress: 41.86%\n",
      "Progress: 41.87%\n",
      "Progress: 41.88%\n",
      "Progress: 41.90%\n",
      "Progress: 41.91%\n",
      "Progress: 41.92%\n",
      "Progress: 41.93%\n",
      "Progress: 41.94%\n",
      "Progress: 41.95%\n",
      "Progress: 41.96%\n",
      "Progress: 41.97%\n",
      "Progress: 41.98%\n",
      "Progress: 41.99%\n",
      "Progress: 42.00%\n",
      "Progress: 42.01%\n",
      "Progress: 42.03%\n",
      "Progress: 42.04%\n",
      "Progress: 42.05%\n",
      "Progress: 42.06%\n",
      "Progress: 42.07%\n",
      "Progress: 42.08%\n",
      "Progress: 42.09%\n",
      "Progress: 42.10%\n",
      "Progress: 42.11%\n",
      "Progress: 42.12%\n",
      "Progress: 42.13%\n",
      "Progress: 42.14%\n",
      "Progress: 42.16%\n",
      "Progress: 42.17%\n",
      "Progress: 42.18%\n",
      "Progress: 42.19%\n",
      "Progress: 42.20%\n",
      "Progress: 42.21%\n",
      "Progress: 42.22%\n",
      "Progress: 42.23%\n",
      "Progress: 42.24%\n",
      "Progress: 42.25%\n",
      "Progress: 42.26%\n",
      "Progress: 42.27%\n",
      "Progress: 42.28%\n",
      "Progress: 42.30%\n",
      "Progress: 42.31%\n",
      "Progress: 42.32%\n",
      "Progress: 42.33%\n",
      "Progress: 42.34%\n",
      "Progress: 42.35%\n",
      "Progress: 42.36%\n",
      "Progress: 42.37%\n",
      "Progress: 42.38%\n",
      "Progress: 42.39%\n",
      "Progress: 42.40%\n",
      "Progress: 42.41%\n",
      "Progress: 42.43%\n",
      "Progress: 42.44%\n",
      "Progress: 42.45%\n",
      "Progress: 42.46%\n",
      "Progress: 42.47%\n",
      "Progress: 42.48%\n",
      "Progress: 42.49%\n",
      "Progress: 42.50%\n",
      "Progress: 42.51%\n",
      "Progress: 42.52%\n",
      "Progress: 42.53%\n",
      "Progress: 42.54%\n",
      "Progress: 42.55%\n",
      "Progress: 42.57%\n",
      "Progress: 42.58%\n",
      "Progress: 42.59%\n",
      "Progress: 42.60%\n",
      "Progress: 42.61%\n",
      "Progress: 42.62%\n",
      "Progress: 42.63%\n",
      "Progress: 42.64%\n",
      "Progress: 42.65%\n",
      "Progress: 42.66%\n",
      "Progress: 42.67%\n",
      "Progress: 42.68%\n",
      "Progress: 42.70%\n",
      "Progress: 42.71%\n",
      "Progress: 42.72%\n",
      "Progress: 42.73%\n",
      "Progress: 42.74%\n",
      "Progress: 42.75%\n",
      "Progress: 42.76%\n",
      "Progress: 42.77%\n",
      "Progress: 42.78%\n",
      "Progress: 42.79%\n",
      "Progress: 42.80%\n",
      "Progress: 42.81%\n",
      "Progress: 42.82%\n",
      "Progress: 42.84%\n",
      "Progress: 42.85%\n",
      "Progress: 42.86%\n",
      "Progress: 42.87%\n",
      "Progress: 42.88%\n",
      "Progress: 42.89%\n",
      "Progress: 42.90%\n",
      "Progress: 42.91%\n",
      "Progress: 42.92%\n",
      "Progress: 42.93%\n",
      "Progress: 42.94%\n",
      "Progress: 42.95%\n",
      "Progress: 42.96%\n",
      "Progress: 42.97%\n",
      "Progress: 42.99%\n",
      "Progress: 43.00%\n",
      "Progress: 43.01%\n",
      "Progress: 43.02%\n",
      "Progress: 43.03%\n",
      "Progress: 43.04%\n",
      "Progress: 43.05%\n",
      "Progress: 43.06%\n",
      "Progress: 43.07%\n",
      "Progress: 43.08%\n",
      "Progress: 43.09%\n",
      "Progress: 43.10%\n",
      "Progress: 43.11%\n",
      "Progress: 43.13%\n",
      "Progress: 43.14%\n",
      "Progress: 43.15%\n",
      "Progress: 43.16%\n",
      "Progress: 43.17%\n",
      "Progress: 43.18%\n",
      "Progress: 43.19%\n",
      "Progress: 43.20%\n",
      "Progress: 43.21%\n",
      "Progress: 43.22%\n",
      "Progress: 43.23%\n",
      "Progress: 43.24%\n",
      "Progress: 43.25%\n",
      "Progress: 43.26%\n",
      "Progress: 43.28%\n",
      "Progress: 43.29%\n",
      "Progress: 43.30%\n",
      "Progress: 43.31%\n",
      "Progress: 43.32%\n",
      "Progress: 43.33%\n",
      "Progress: 43.34%\n",
      "Progress: 43.35%\n",
      "Progress: 43.36%\n",
      "Progress: 43.37%\n",
      "Progress: 43.38%\n",
      "Progress: 43.39%\n",
      "Progress: 43.40%\n",
      "Progress: 43.41%\n",
      "Progress: 43.43%\n",
      "Progress: 43.44%\n",
      "Progress: 43.45%\n",
      "Progress: 43.46%\n",
      "Progress: 43.47%\n",
      "Progress: 43.48%\n",
      "Progress: 43.49%\n",
      "Progress: 43.50%\n",
      "Progress: 43.51%\n",
      "Progress: 43.52%\n",
      "Progress: 43.53%\n",
      "Progress: 43.54%\n",
      "Progress: 43.55%\n",
      "Progress: 43.56%\n",
      "Progress: 43.58%\n",
      "Progress: 43.59%\n",
      "Progress: 43.60%\n",
      "Progress: 43.61%\n",
      "Progress: 43.62%\n",
      "Progress: 43.63%\n",
      "Progress: 43.64%\n",
      "Progress: 43.65%\n",
      "Progress: 43.66%\n",
      "Progress: 43.67%\n",
      "Progress: 43.68%\n",
      "Progress: 43.69%\n",
      "Progress: 43.70%\n",
      "Progress: 43.71%\n",
      "Progress: 43.72%\n",
      "Progress: 43.74%\n",
      "Progress: 43.75%\n",
      "Progress: 43.76%\n",
      "Progress: 43.77%\n",
      "Progress: 43.78%\n",
      "Progress: 43.79%\n",
      "Progress: 43.80%\n",
      "Progress: 43.81%\n",
      "Progress: 43.82%\n",
      "Progress: 43.83%\n",
      "Progress: 43.84%\n",
      "Progress: 43.85%\n",
      "Progress: 43.86%\n",
      "Progress: 43.87%\n",
      "Progress: 43.88%\n",
      "Progress: 43.90%\n",
      "Progress: 43.91%\n",
      "Progress: 43.92%\n",
      "Progress: 43.93%\n",
      "Progress: 43.94%\n",
      "Progress: 43.95%\n",
      "Progress: 43.96%\n",
      "Progress: 43.97%\n",
      "Progress: 43.98%\n",
      "Progress: 43.99%\n",
      "Progress: 44.00%\n",
      "Progress: 44.01%\n",
      "Progress: 44.02%\n",
      "Progress: 44.03%\n",
      "Progress: 44.04%\n",
      "Progress: 44.06%\n",
      "Progress: 44.07%\n",
      "Progress: 44.08%\n",
      "Progress: 44.09%\n",
      "Progress: 44.10%\n",
      "Progress: 44.11%\n",
      "Progress: 44.12%\n",
      "Progress: 44.13%\n",
      "Progress: 44.14%\n",
      "Progress: 44.15%\n",
      "Progress: 44.16%\n",
      "Progress: 44.17%\n",
      "Progress: 44.18%\n",
      "Progress: 44.19%\n",
      "Progress: 44.20%\n",
      "Progress: 44.21%\n",
      "Progress: 44.23%\n",
      "Progress: 44.24%\n",
      "Progress: 44.25%\n",
      "Progress: 44.26%\n",
      "Progress: 44.27%\n",
      "Progress: 44.28%\n",
      "Progress: 44.29%\n",
      "Progress: 44.30%\n",
      "Progress: 44.31%\n",
      "Progress: 44.32%\n",
      "Progress: 44.33%\n",
      "Progress: 44.34%\n",
      "Progress: 44.35%\n",
      "Progress: 44.36%\n",
      "Progress: 44.37%\n",
      "Progress: 44.38%\n",
      "Progress: 44.40%\n",
      "Progress: 44.41%\n",
      "Progress: 44.42%\n",
      "Progress: 44.43%\n",
      "Progress: 44.44%\n",
      "Progress: 44.45%\n",
      "Progress: 44.46%\n",
      "Progress: 44.47%\n",
      "Progress: 44.48%\n",
      "Progress: 44.49%\n",
      "Progress: 44.50%\n",
      "Progress: 44.51%\n",
      "Progress: 44.52%\n",
      "Progress: 44.53%\n",
      "Progress: 44.54%\n",
      "Progress: 44.55%\n",
      "Progress: 44.56%\n",
      "Progress: 44.58%\n",
      "Progress: 44.59%\n",
      "Progress: 44.60%\n",
      "Progress: 44.61%\n",
      "Progress: 44.62%\n",
      "Progress: 44.63%\n",
      "Progress: 44.64%\n",
      "Progress: 44.65%\n",
      "Progress: 44.66%\n",
      "Progress: 44.67%\n",
      "Progress: 44.68%\n",
      "Progress: 44.69%\n",
      "Progress: 44.70%\n",
      "Progress: 44.71%\n",
      "Progress: 44.72%\n",
      "Progress: 44.73%\n",
      "Progress: 44.74%\n",
      "Progress: 44.76%\n",
      "Progress: 44.77%\n",
      "Progress: 44.78%\n",
      "Progress: 44.79%\n",
      "Progress: 44.80%\n",
      "Progress: 44.81%\n",
      "Progress: 44.82%\n",
      "Progress: 44.83%\n",
      "Progress: 44.84%\n",
      "Progress: 44.85%\n",
      "Progress: 44.86%\n",
      "Progress: 44.87%\n",
      "Progress: 44.88%\n",
      "Progress: 44.89%\n",
      "Progress: 44.90%\n",
      "Progress: 44.91%\n",
      "Progress: 44.92%\n",
      "Progress: 44.94%\n",
      "Progress: 44.95%\n",
      "Progress: 44.96%\n",
      "Progress: 44.97%\n",
      "Progress: 44.98%\n",
      "Progress: 44.99%\n",
      "Progress: 45.00%\n",
      "Progress: 45.01%\n",
      "Progress: 45.02%\n",
      "Progress: 45.03%\n",
      "Progress: 45.04%\n",
      "Progress: 45.05%\n",
      "Progress: 45.06%\n",
      "Progress: 45.07%\n",
      "Progress: 45.08%\n",
      "Progress: 45.09%\n",
      "Progress: 45.10%\n",
      "Progress: 45.11%\n",
      "Progress: 45.13%\n",
      "Progress: 45.14%\n",
      "Progress: 45.15%\n",
      "Progress: 45.16%\n",
      "Progress: 45.17%\n",
      "Progress: 45.18%\n",
      "Progress: 45.19%\n",
      "Progress: 45.20%\n",
      "Progress: 45.21%\n",
      "Progress: 45.22%\n",
      "Progress: 45.23%\n",
      "Progress: 45.24%\n",
      "Progress: 45.25%\n",
      "Progress: 45.26%\n",
      "Progress: 45.27%\n",
      "Progress: 45.28%\n",
      "Progress: 45.29%\n",
      "Progress: 45.30%\n",
      "Progress: 45.31%\n",
      "Progress: 45.33%\n",
      "Progress: 45.34%\n",
      "Progress: 45.35%\n",
      "Progress: 45.36%\n",
      "Progress: 45.37%\n",
      "Progress: 45.38%\n",
      "Progress: 45.39%\n",
      "Progress: 45.40%\n",
      "Progress: 45.41%\n",
      "Progress: 45.42%\n",
      "Progress: 45.43%\n",
      "Progress: 45.44%\n",
      "Progress: 45.45%\n",
      "Progress: 45.46%\n",
      "Progress: 45.47%\n",
      "Progress: 45.48%\n",
      "Progress: 45.49%\n",
      "Progress: 45.50%\n",
      "Progress: 45.51%\n",
      "Progress: 45.52%\n",
      "Progress: 45.54%\n",
      "Progress: 45.55%\n",
      "Progress: 45.56%\n",
      "Progress: 45.57%\n",
      "Progress: 45.58%\n",
      "Progress: 45.59%\n",
      "Progress: 45.60%\n",
      "Progress: 45.61%\n",
      "Progress: 45.62%\n",
      "Progress: 45.63%\n",
      "Progress: 45.64%\n",
      "Progress: 45.65%\n",
      "Progress: 45.66%\n",
      "Progress: 45.67%\n",
      "Progress: 45.68%\n",
      "Progress: 45.69%\n",
      "Progress: 45.70%\n",
      "Progress: 45.71%\n",
      "Progress: 45.72%\n",
      "Progress: 45.73%\n",
      "Progress: 45.75%\n",
      "Progress: 45.76%\n",
      "Progress: 45.77%\n",
      "Progress: 45.78%\n",
      "Progress: 45.79%\n",
      "Progress: 45.80%\n",
      "Progress: 45.81%\n",
      "Progress: 45.82%\n",
      "Progress: 45.83%\n",
      "Progress: 45.84%\n",
      "Progress: 45.85%\n",
      "Progress: 45.86%\n",
      "Progress: 45.87%\n",
      "Progress: 45.88%\n",
      "Progress: 45.89%\n",
      "Progress: 45.90%\n",
      "Progress: 45.91%\n",
      "Progress: 45.92%\n",
      "Progress: 45.93%\n",
      "Progress: 45.94%\n",
      "Progress: 45.95%\n",
      "Progress: 45.97%\n",
      "Progress: 45.98%\n",
      "Progress: 45.99%\n",
      "Progress: 46.00%\n",
      "Progress: 46.01%\n",
      "Progress: 46.02%\n",
      "Progress: 46.03%\n",
      "Progress: 46.04%\n",
      "Progress: 46.05%\n",
      "Progress: 46.06%\n",
      "Progress: 46.07%\n",
      "Progress: 46.08%\n",
      "Progress: 46.09%\n",
      "Progress: 46.10%\n",
      "Progress: 46.11%\n",
      "Progress: 46.12%\n",
      "Progress: 46.13%\n",
      "Progress: 46.14%\n",
      "Progress: 46.15%\n",
      "Progress: 46.16%\n",
      "Progress: 46.17%\n",
      "Progress: 46.18%\n",
      "Progress: 46.20%\n",
      "Progress: 46.21%\n",
      "Progress: 46.22%\n",
      "Progress: 46.23%\n",
      "Progress: 46.24%\n",
      "Progress: 46.25%\n",
      "Progress: 46.26%\n",
      "Progress: 46.27%\n",
      "Progress: 46.28%\n",
      "Progress: 46.29%\n",
      "Progress: 46.30%\n",
      "Progress: 46.31%\n",
      "Progress: 46.32%\n",
      "Progress: 46.33%\n",
      "Progress: 46.34%\n",
      "Progress: 46.35%\n",
      "Progress: 46.36%\n",
      "Progress: 46.37%\n",
      "Progress: 46.38%\n",
      "Progress: 46.39%\n",
      "Progress: 46.40%\n",
      "Progress: 46.41%\n",
      "Progress: 46.42%\n",
      "Progress: 46.43%\n",
      "Progress: 46.45%\n",
      "Progress: 46.46%\n",
      "Progress: 46.47%\n",
      "Progress: 46.48%\n",
      "Progress: 46.49%\n",
      "Progress: 46.50%\n",
      "Progress: 46.51%\n",
      "Progress: 46.52%\n",
      "Progress: 46.53%\n",
      "Progress: 46.54%\n",
      "Progress: 46.55%\n",
      "Progress: 46.56%\n",
      "Progress: 46.57%\n",
      "Progress: 46.58%\n",
      "Progress: 46.59%\n",
      "Progress: 46.60%\n",
      "Progress: 46.61%\n",
      "Progress: 46.62%\n",
      "Progress: 46.63%\n",
      "Progress: 46.64%\n",
      "Progress: 46.65%\n",
      "Progress: 46.66%\n",
      "Progress: 46.67%\n",
      "Progress: 46.68%\n",
      "Progress: 46.69%\n",
      "Progress: 46.71%\n",
      "Progress: 46.72%\n",
      "Progress: 46.73%\n",
      "Progress: 46.74%\n",
      "Progress: 46.75%\n",
      "Progress: 46.76%\n",
      "Progress: 46.77%\n",
      "Progress: 46.78%\n",
      "Progress: 46.79%\n",
      "Progress: 46.80%\n",
      "Progress: 46.81%\n",
      "Progress: 46.82%\n",
      "Progress: 46.83%\n",
      "Progress: 46.84%\n",
      "Progress: 46.85%\n",
      "Progress: 46.86%\n",
      "Progress: 46.87%\n",
      "Progress: 46.88%\n",
      "Progress: 46.89%\n",
      "Progress: 46.90%\n",
      "Progress: 46.91%\n",
      "Progress: 46.92%\n",
      "Progress: 46.93%\n",
      "Progress: 46.94%\n",
      "Progress: 46.95%\n",
      "Progress: 46.96%\n",
      "Progress: 46.98%\n",
      "Progress: 46.99%\n",
      "Progress: 47.00%\n",
      "Progress: 47.01%\n",
      "Progress: 47.02%\n",
      "Progress: 47.03%\n",
      "Progress: 47.04%\n",
      "Progress: 47.05%\n",
      "Progress: 47.06%\n",
      "Progress: 47.07%\n",
      "Progress: 47.08%\n",
      "Progress: 47.09%\n",
      "Progress: 47.10%\n",
      "Progress: 47.11%\n",
      "Progress: 47.12%\n",
      "Progress: 47.13%\n",
      "Progress: 47.14%\n",
      "Progress: 47.15%\n",
      "Progress: 47.16%\n",
      "Progress: 47.17%\n",
      "Progress: 47.18%\n",
      "Progress: 47.19%\n",
      "Progress: 47.20%\n",
      "Progress: 47.21%\n",
      "Progress: 47.22%\n",
      "Progress: 47.23%\n",
      "Progress: 47.24%\n",
      "Progress: 47.25%\n",
      "Progress: 47.26%\n",
      "Progress: 47.28%\n",
      "Progress: 47.29%\n",
      "Progress: 47.30%\n",
      "Progress: 47.31%\n",
      "Progress: 47.32%\n",
      "Progress: 47.33%\n",
      "Progress: 47.34%\n",
      "Progress: 47.35%\n",
      "Progress: 47.36%\n",
      "Progress: 47.37%\n",
      "Progress: 47.38%\n",
      "Progress: 47.39%\n",
      "Progress: 47.40%\n",
      "Progress: 47.41%\n",
      "Progress: 47.42%\n",
      "Progress: 47.43%\n",
      "Progress: 47.44%\n",
      "Progress: 47.45%\n",
      "Progress: 47.46%\n",
      "Progress: 47.47%\n",
      "Progress: 47.48%\n",
      "Progress: 47.49%\n",
      "Progress: 47.50%\n",
      "Progress: 47.51%\n",
      "Progress: 47.52%\n",
      "Progress: 47.53%\n",
      "Progress: 47.54%\n",
      "Progress: 47.55%\n",
      "Progress: 47.56%\n",
      "Progress: 47.57%\n",
      "Progress: 47.58%\n",
      "Progress: 47.60%\n",
      "Progress: 47.61%\n",
      "Progress: 47.62%\n",
      "Progress: 47.63%\n",
      "Progress: 47.64%\n",
      "Progress: 47.65%\n",
      "Progress: 47.66%\n",
      "Progress: 47.67%\n",
      "Progress: 47.68%\n",
      "Progress: 47.69%\n",
      "Progress: 47.70%\n",
      "Progress: 47.71%\n",
      "Progress: 47.72%\n",
      "Progress: 47.73%\n",
      "Progress: 47.74%\n",
      "Progress: 47.75%\n",
      "Progress: 47.76%\n",
      "Progress: 47.77%\n",
      "Progress: 47.78%\n",
      "Progress: 47.79%\n",
      "Progress: 47.80%\n",
      "Progress: 47.81%\n",
      "Progress: 47.82%\n",
      "Progress: 47.83%\n",
      "Progress: 47.84%\n",
      "Progress: 47.85%\n",
      "Progress: 47.86%\n",
      "Progress: 47.87%\n",
      "Progress: 47.88%\n",
      "Progress: 47.89%\n",
      "Progress: 47.90%\n",
      "Progress: 47.91%\n",
      "Progress: 47.92%\n",
      "Progress: 47.93%\n",
      "Progress: 47.94%\n",
      "Progress: 47.96%\n",
      "Progress: 47.97%\n",
      "Progress: 47.98%\n",
      "Progress: 47.99%\n",
      "Progress: 48.00%\n",
      "Progress: 48.01%\n",
      "Progress: 48.02%\n",
      "Progress: 48.03%\n",
      "Progress: 48.04%\n",
      "Progress: 48.05%\n",
      "Progress: 48.06%\n",
      "Progress: 48.07%\n",
      "Progress: 48.08%\n",
      "Progress: 48.09%\n",
      "Progress: 48.10%\n",
      "Progress: 48.11%\n",
      "Progress: 48.12%\n",
      "Progress: 48.13%\n",
      "Progress: 48.14%\n",
      "Progress: 48.15%\n",
      "Progress: 48.16%\n",
      "Progress: 48.17%\n",
      "Progress: 48.18%\n",
      "Progress: 48.19%\n",
      "Progress: 48.20%\n",
      "Progress: 48.21%\n",
      "Progress: 48.22%\n",
      "Progress: 48.23%\n",
      "Progress: 48.24%\n",
      "Progress: 48.25%\n",
      "Progress: 48.26%\n",
      "Progress: 48.27%\n",
      "Progress: 48.28%\n",
      "Progress: 48.29%\n",
      "Progress: 48.30%\n",
      "Progress: 48.31%\n",
      "Progress: 48.32%\n",
      "Progress: 48.33%\n",
      "Progress: 48.34%\n",
      "Progress: 48.35%\n",
      "Progress: 48.36%\n",
      "Progress: 48.38%\n",
      "Progress: 48.39%\n",
      "Progress: 48.40%\n",
      "Progress: 48.41%\n",
      "Progress: 48.42%\n",
      "Progress: 48.43%\n",
      "Progress: 48.44%\n",
      "Progress: 48.45%\n",
      "Progress: 48.46%\n",
      "Progress: 48.47%\n",
      "Progress: 48.48%\n",
      "Progress: 48.49%\n",
      "Progress: 48.50%\n",
      "Progress: 48.51%\n",
      "Progress: 48.52%\n",
      "Progress: 48.53%\n",
      "Progress: 48.54%\n",
      "Progress: 48.55%\n",
      "Progress: 48.56%\n",
      "Progress: 48.57%\n",
      "Progress: 48.58%\n",
      "Progress: 48.59%\n",
      "Progress: 48.60%\n",
      "Progress: 48.61%\n",
      "Progress: 48.62%\n",
      "Progress: 48.63%\n",
      "Progress: 48.64%\n",
      "Progress: 48.65%\n",
      "Progress: 48.66%\n",
      "Progress: 48.67%\n",
      "Progress: 48.68%\n",
      "Progress: 48.69%\n",
      "Progress: 48.70%\n",
      "Progress: 48.71%\n",
      "Progress: 48.72%\n",
      "Progress: 48.73%\n",
      "Progress: 48.74%\n",
      "Progress: 48.75%\n",
      "Progress: 48.76%\n",
      "Progress: 48.77%\n",
      "Progress: 48.78%\n",
      "Progress: 48.79%\n",
      "Progress: 48.80%\n",
      "Progress: 48.81%\n",
      "Progress: 48.82%\n",
      "Progress: 48.83%\n",
      "Progress: 48.84%\n",
      "Progress: 48.85%\n",
      "Progress: 48.86%\n",
      "Progress: 48.87%\n",
      "Progress: 48.89%\n",
      "Progress: 48.90%\n",
      "Progress: 48.91%\n",
      "Progress: 48.92%\n",
      "Progress: 48.93%\n",
      "Progress: 48.94%\n",
      "Progress: 48.95%\n",
      "Progress: 48.96%\n",
      "Progress: 48.97%\n",
      "Progress: 48.98%\n",
      "Progress: 48.99%\n",
      "Progress: 49.00%\n",
      "Progress: 49.01%\n",
      "Progress: 49.02%\n",
      "Progress: 49.03%\n",
      "Progress: 49.04%\n",
      "Progress: 49.05%\n",
      "Progress: 49.06%\n",
      "Progress: 49.07%\n",
      "Progress: 49.08%\n",
      "Progress: 49.09%\n",
      "Progress: 49.10%\n",
      "Progress: 49.11%\n",
      "Progress: 49.12%\n",
      "Progress: 49.13%\n",
      "Progress: 49.14%\n",
      "Progress: 49.15%\n",
      "Progress: 49.16%\n",
      "Progress: 49.17%\n",
      "Progress: 49.18%\n",
      "Progress: 49.19%\n",
      "Progress: 49.20%\n",
      "Progress: 49.21%\n",
      "Progress: 49.22%\n",
      "Progress: 49.23%\n",
      "Progress: 49.24%\n",
      "Progress: 49.25%\n",
      "Progress: 49.26%\n",
      "Progress: 49.27%\n",
      "Progress: 49.28%\n",
      "Progress: 49.29%\n",
      "Progress: 49.30%\n",
      "Progress: 49.31%\n",
      "Progress: 49.32%\n",
      "Progress: 49.33%\n",
      "Progress: 49.34%\n",
      "Progress: 49.35%\n",
      "Progress: 49.36%\n",
      "Progress: 49.37%\n",
      "Progress: 49.38%\n",
      "Progress: 49.39%\n",
      "Progress: 49.40%\n",
      "Progress: 49.41%\n",
      "Progress: 49.42%\n",
      "Progress: 49.43%\n",
      "Progress: 49.44%\n",
      "Progress: 49.45%\n",
      "Progress: 49.46%\n",
      "Progress: 49.47%\n",
      "Progress: 49.48%\n",
      "Progress: 49.49%\n",
      "Progress: 49.50%\n",
      "Progress: 49.51%\n",
      "Progress: 49.52%\n",
      "Progress: 49.53%\n",
      "Progress: 49.54%\n",
      "Progress: 49.55%\n",
      "Progress: 49.56%\n",
      "Progress: 49.57%\n",
      "Progress: 49.58%\n",
      "Progress: 49.59%\n",
      "Progress: 49.61%\n",
      "Progress: 49.62%\n",
      "Progress: 49.63%\n",
      "Progress: 49.64%\n",
      "Progress: 49.65%\n",
      "Progress: 49.66%\n",
      "Progress: 49.67%\n",
      "Progress: 49.68%\n",
      "Progress: 49.69%\n",
      "Progress: 49.70%\n",
      "Progress: 49.71%\n",
      "Progress: 49.72%\n",
      "Progress: 49.73%\n",
      "Progress: 49.74%\n",
      "Progress: 49.75%\n",
      "Progress: 49.76%\n",
      "Progress: 49.77%\n",
      "Progress: 49.78%\n",
      "Progress: 49.79%\n",
      "Progress: 49.80%\n",
      "Progress: 49.81%\n",
      "Progress: 49.82%\n",
      "Progress: 49.83%\n",
      "Progress: 49.84%\n",
      "Progress: 49.85%\n",
      "Progress: 49.86%\n",
      "Progress: 49.87%\n",
      "Progress: 49.88%\n",
      "Progress: 49.89%\n",
      "Progress: 49.90%\n",
      "Progress: 49.91%\n",
      "Progress: 49.92%\n",
      "Progress: 49.93%\n",
      "Progress: 49.94%\n",
      "Progress: 49.95%\n",
      "Progress: 49.96%\n",
      "Progress: 49.97%\n",
      "Progress: 49.98%\n",
      "Progress: 49.99%\n",
      "Progress: 50.00%\n",
      "Progress: 50.01%\n",
      "Progress: 50.02%\n",
      "Progress: 50.03%\n",
      "Progress: 50.04%\n",
      "Progress: 50.05%\n",
      "Progress: 50.06%\n",
      "Progress: 50.07%\n",
      "Progress: 50.08%\n",
      "Progress: 50.09%\n",
      "Progress: 50.10%\n",
      "Progress: 50.11%\n",
      "Progress: 50.12%\n",
      "Progress: 50.13%\n",
      "Progress: 50.14%\n",
      "Progress: 50.15%\n",
      "Progress: 50.16%\n",
      "Progress: 50.17%\n",
      "Progress: 50.18%\n",
      "Progress: 50.19%\n",
      "Progress: 50.20%\n",
      "Progress: 50.21%\n",
      "Progress: 50.22%\n",
      "Progress: 50.23%\n",
      "Progress: 50.24%\n",
      "Progress: 50.25%\n",
      "Progress: 50.26%\n",
      "Progress: 50.27%\n",
      "Progress: 50.28%\n",
      "Progress: 50.29%\n",
      "Progress: 50.30%\n",
      "Progress: 50.31%\n",
      "Progress: 50.32%\n",
      "Progress: 50.33%\n",
      "Progress: 50.34%\n",
      "Progress: 50.35%\n",
      "Progress: 50.36%\n",
      "Progress: 50.37%\n",
      "Progress: 50.38%\n",
      "Progress: 50.39%\n",
      "Progress: 50.40%\n",
      "Progress: 50.41%\n",
      "Progress: 50.42%\n",
      "Progress: 50.43%\n",
      "Progress: 50.44%\n",
      "Progress: 50.45%\n",
      "Progress: 50.46%\n",
      "Progress: 50.47%\n",
      "Progress: 50.48%\n",
      "Progress: 50.49%\n",
      "Progress: 50.50%\n",
      "Progress: 50.51%\n",
      "Progress: 50.52%\n",
      "Progress: 50.53%\n",
      "Progress: 50.54%\n",
      "Progress: 50.55%\n",
      "Progress: 50.56%\n",
      "Progress: 50.57%\n",
      "Progress: 50.58%\n",
      "Progress: 50.59%\n",
      "Progress: 50.60%\n",
      "Progress: 50.61%\n",
      "Progress: 50.62%\n",
      "Progress: 50.63%\n",
      "Progress: 50.64%\n",
      "Progress: 50.65%\n",
      "Progress: 50.66%\n",
      "Progress: 50.67%\n",
      "Progress: 50.68%\n",
      "Progress: 50.69%\n",
      "Progress: 50.70%\n",
      "Progress: 50.71%\n",
      "Progress: 50.72%\n",
      "Progress: 50.73%\n",
      "Progress: 50.74%\n",
      "Progress: 50.75%\n",
      "Progress: 50.76%\n",
      "Progress: 50.77%\n",
      "Progress: 50.78%\n",
      "Progress: 50.79%\n",
      "Progress: 50.80%\n",
      "Progress: 50.81%\n",
      "Progress: 50.82%\n",
      "Progress: 50.83%\n",
      "Progress: 50.84%\n",
      "Progress: 50.85%\n",
      "Progress: 50.86%\n",
      "Progress: 50.87%\n",
      "Progress: 50.88%\n",
      "Progress: 50.89%\n",
      "Progress: 50.90%\n",
      "Progress: 50.91%\n",
      "Progress: 50.92%\n",
      "Progress: 50.93%\n",
      "Progress: 50.94%\n",
      "Progress: 50.95%\n",
      "Progress: 50.96%\n",
      "Progress: 50.97%\n",
      "Progress: 50.98%\n",
      "Progress: 50.99%\n",
      "Progress: 51.00%\n",
      "Progress: 51.01%\n",
      "Progress: 51.02%\n",
      "Progress: 51.03%\n",
      "Progress: 51.04%\n",
      "Progress: 51.05%\n",
      "Progress: 51.06%\n",
      "Progress: 51.07%\n",
      "Progress: 51.08%\n",
      "Progress: 51.09%\n",
      "Progress: 51.10%\n",
      "Progress: 51.11%\n",
      "Progress: 51.12%\n",
      "Progress: 51.13%\n",
      "Progress: 51.14%\n",
      "Progress: 51.15%\n",
      "Progress: 51.16%\n",
      "Progress: 51.17%\n",
      "Progress: 51.18%\n",
      "Progress: 51.19%\n",
      "Progress: 51.20%\n",
      "Progress: 51.21%\n",
      "Progress: 51.22%\n",
      "Progress: 51.23%\n",
      "Progress: 51.24%\n",
      "Progress: 51.25%\n",
      "Progress: 51.26%\n",
      "Progress: 51.27%\n",
      "Progress: 51.28%\n",
      "Progress: 51.29%\n",
      "Progress: 51.30%\n",
      "Progress: 51.31%\n",
      "Progress: 51.32%\n",
      "Progress: 51.33%\n",
      "Progress: 51.34%\n",
      "Progress: 51.35%\n",
      "Progress: 51.36%\n",
      "Progress: 51.37%\n",
      "Progress: 51.38%\n",
      "Progress: 51.39%\n",
      "Progress: 51.40%\n",
      "Progress: 51.41%\n",
      "Progress: 51.42%\n",
      "Progress: 51.43%\n",
      "Progress: 51.44%\n",
      "Progress: 51.45%\n",
      "Progress: 51.46%\n",
      "Progress: 51.47%\n",
      "Progress: 51.48%\n",
      "Progress: 51.49%\n",
      "Progress: 51.50%\n",
      "Progress: 51.51%\n",
      "Progress: 51.52%\n",
      "Progress: 51.53%\n",
      "Progress: 51.54%\n",
      "Progress: 51.55%\n",
      "Progress: 51.56%\n",
      "Progress: 51.57%\n",
      "Progress: 51.58%\n",
      "Progress: 51.59%\n",
      "Progress: 51.60%\n",
      "Progress: 51.61%\n",
      "Progress: 51.62%\n",
      "Progress: 51.63%\n",
      "Progress: 51.63%\n",
      "Progress: 51.64%\n",
      "Progress: 51.65%\n",
      "Progress: 51.66%\n",
      "Progress: 51.67%\n",
      "Progress: 51.68%\n",
      "Progress: 51.69%\n",
      "Progress: 51.70%\n",
      "Progress: 51.71%\n",
      "Progress: 51.72%\n",
      "Progress: 51.73%\n",
      "Progress: 51.74%\n",
      "Progress: 51.75%\n",
      "Progress: 51.76%\n",
      "Progress: 51.77%\n",
      "Progress: 51.78%\n",
      "Progress: 51.79%\n",
      "Progress: 51.80%\n",
      "Progress: 51.81%\n",
      "Progress: 51.82%\n",
      "Progress: 51.83%\n",
      "Progress: 51.84%\n",
      "Progress: 51.85%\n",
      "Progress: 51.86%\n",
      "Progress: 51.87%\n",
      "Progress: 51.88%\n",
      "Progress: 51.89%\n",
      "Progress: 51.90%\n",
      "Progress: 51.91%\n",
      "Progress: 51.92%\n",
      "Progress: 51.93%\n",
      "Progress: 51.94%\n",
      "Progress: 51.95%\n",
      "Progress: 51.96%\n",
      "Progress: 51.97%\n",
      "Progress: 51.98%\n",
      "Progress: 51.99%\n",
      "Progress: 52.00%\n",
      "Progress: 52.01%\n",
      "Progress: 52.02%\n",
      "Progress: 52.03%\n",
      "Progress: 52.04%\n",
      "Progress: 52.05%\n",
      "Progress: 52.06%\n",
      "Progress: 52.07%\n",
      "Progress: 52.08%\n",
      "Progress: 52.09%\n",
      "Progress: 52.10%\n",
      "Progress: 52.11%\n",
      "Progress: 52.12%\n",
      "Progress: 52.13%\n",
      "Progress: 52.14%\n",
      "Progress: 52.15%\n",
      "Progress: 52.16%\n",
      "Progress: 52.17%\n",
      "Progress: 52.18%\n",
      "Progress: 52.19%\n",
      "Progress: 52.20%\n",
      "Progress: 52.21%\n",
      "Progress: 52.22%\n",
      "Progress: 52.23%\n",
      "Progress: 52.24%\n",
      "Progress: 52.25%\n",
      "Progress: 52.26%\n",
      "Progress: 52.27%\n",
      "Progress: 52.28%\n",
      "Progress: 52.29%\n",
      "Progress: 52.30%\n",
      "Progress: 52.31%\n",
      "Progress: 52.32%\n",
      "Progress: 52.33%\n",
      "Progress: 52.34%\n",
      "Progress: 52.34%\n",
      "Progress: 52.35%\n",
      "Progress: 52.36%\n",
      "Progress: 52.37%\n",
      "Progress: 52.38%\n",
      "Progress: 52.39%\n",
      "Progress: 52.40%\n",
      "Progress: 52.41%\n",
      "Progress: 52.42%\n",
      "Progress: 52.43%\n",
      "Progress: 52.44%\n",
      "Progress: 52.45%\n",
      "Progress: 52.46%\n",
      "Progress: 52.47%\n",
      "Progress: 52.48%\n",
      "Progress: 52.49%\n",
      "Progress: 52.50%\n",
      "Progress: 52.51%\n",
      "Progress: 52.52%\n",
      "Progress: 52.53%\n",
      "Progress: 52.54%\n",
      "Progress: 52.55%\n",
      "Progress: 52.56%\n",
      "Progress: 52.57%\n",
      "Progress: 52.58%\n",
      "Progress: 52.59%\n",
      "Progress: 52.60%\n",
      "Progress: 52.61%\n",
      "Progress: 52.62%\n",
      "Progress: 52.63%\n",
      "Progress: 52.64%\n",
      "Progress: 52.65%\n",
      "Progress: 52.66%\n",
      "Progress: 52.67%\n",
      "Progress: 52.68%\n",
      "Progress: 52.69%\n",
      "Progress: 52.70%\n",
      "Progress: 52.71%\n",
      "Progress: 52.72%\n",
      "Progress: 52.73%\n",
      "Progress: 52.74%\n",
      "Progress: 52.75%\n",
      "Progress: 52.76%\n",
      "Progress: 52.77%\n",
      "Progress: 52.78%\n",
      "Progress: 52.79%\n",
      "Progress: 52.80%\n",
      "Progress: 52.81%\n",
      "Progress: 52.82%\n",
      "Progress: 52.83%\n",
      "Progress: 52.83%\n",
      "Progress: 52.84%\n",
      "Progress: 52.85%\n",
      "Progress: 52.86%\n",
      "Progress: 52.87%\n",
      "Progress: 52.88%\n",
      "Progress: 52.89%\n",
      "Progress: 52.90%\n",
      "Progress: 52.91%\n",
      "Progress: 52.92%\n",
      "Progress: 52.93%\n",
      "Progress: 52.94%\n",
      "Progress: 52.95%\n",
      "Progress: 52.96%\n",
      "Progress: 52.97%\n",
      "Progress: 52.98%\n",
      "Progress: 52.99%\n",
      "Progress: 53.00%\n",
      "Progress: 53.01%\n",
      "Progress: 53.02%\n",
      "Progress: 53.03%\n",
      "Progress: 53.04%\n",
      "Progress: 53.05%\n",
      "Progress: 53.06%\n",
      "Progress: 53.07%\n",
      "Progress: 53.08%\n",
      "Progress: 53.09%\n",
      "Progress: 53.10%\n",
      "Progress: 53.11%\n",
      "Progress: 53.12%\n",
      "Progress: 53.13%\n",
      "Progress: 53.14%\n",
      "Progress: 53.15%\n",
      "Progress: 53.16%\n",
      "Progress: 53.17%\n",
      "Progress: 53.18%\n",
      "Progress: 53.19%\n",
      "Progress: 53.20%\n",
      "Progress: 53.21%\n",
      "Progress: 53.22%\n",
      "Progress: 53.23%\n",
      "Progress: 53.23%\n",
      "Progress: 53.24%\n",
      "Progress: 53.25%\n",
      "Progress: 53.26%\n",
      "Progress: 53.27%\n",
      "Progress: 53.28%\n",
      "Progress: 53.29%\n",
      "Progress: 53.30%\n",
      "Progress: 53.31%\n",
      "Progress: 53.32%\n",
      "Progress: 53.33%\n",
      "Progress: 53.34%\n",
      "Progress: 53.35%\n",
      "Progress: 53.36%\n",
      "Progress: 53.37%\n",
      "Progress: 53.38%\n",
      "Progress: 53.39%\n",
      "Progress: 53.40%\n",
      "Progress: 53.41%\n",
      "Progress: 53.42%\n",
      "Progress: 53.43%\n",
      "Progress: 53.44%\n",
      "Progress: 53.45%\n",
      "Progress: 53.46%\n",
      "Progress: 53.47%\n",
      "Progress: 53.48%\n",
      "Progress: 53.49%\n",
      "Progress: 53.50%\n",
      "Progress: 53.51%\n",
      "Progress: 53.52%\n",
      "Progress: 53.53%\n",
      "Progress: 53.54%\n",
      "Progress: 53.55%\n",
      "Progress: 53.56%\n",
      "Progress: 53.57%\n",
      "Progress: 53.57%\n",
      "Progress: 53.58%\n",
      "Progress: 53.59%\n",
      "Progress: 53.60%\n",
      "Progress: 53.61%\n",
      "Progress: 53.62%\n",
      "Progress: 53.63%\n",
      "Progress: 53.64%\n",
      "Progress: 53.65%\n",
      "Progress: 53.66%\n",
      "Progress: 53.67%\n",
      "Progress: 53.68%\n",
      "Progress: 53.69%\n",
      "Progress: 53.70%\n",
      "Progress: 53.71%\n",
      "Progress: 53.72%\n",
      "Progress: 53.73%\n",
      "Progress: 53.74%\n",
      "Progress: 53.75%\n",
      "Progress: 53.76%\n",
      "Progress: 53.77%\n",
      "Progress: 53.78%\n",
      "Progress: 53.79%\n",
      "Progress: 53.80%\n",
      "Progress: 53.81%\n",
      "Progress: 53.82%\n",
      "Progress: 53.83%\n",
      "Progress: 53.84%\n",
      "Progress: 53.85%\n",
      "Progress: 53.86%\n",
      "Progress: 53.87%\n",
      "Progress: 53.87%\n",
      "Progress: 53.88%\n",
      "Progress: 53.89%\n",
      "Progress: 53.90%\n",
      "Progress: 53.91%\n",
      "Progress: 53.92%\n",
      "Progress: 53.93%\n",
      "Progress: 53.94%\n",
      "Progress: 53.95%\n",
      "Progress: 53.96%\n",
      "Progress: 53.97%\n",
      "Progress: 53.98%\n",
      "Progress: 53.99%\n",
      "Progress: 54.00%\n",
      "Progress: 54.01%\n",
      "Progress: 54.02%\n",
      "Progress: 54.03%\n",
      "Progress: 54.04%\n",
      "Progress: 54.05%\n",
      "Progress: 54.06%\n",
      "Progress: 54.07%\n",
      "Progress: 54.08%\n",
      "Progress: 54.09%\n",
      "Progress: 54.10%\n",
      "Progress: 54.11%\n",
      "Progress: 54.12%\n",
      "Progress: 54.13%\n",
      "Progress: 54.14%\n",
      "Progress: 54.15%\n",
      "Progress: 54.15%\n",
      "Progress: 54.16%\n",
      "Progress: 54.17%\n",
      "Progress: 54.18%\n",
      "Progress: 54.19%\n",
      "Progress: 54.20%\n",
      "Progress: 54.21%\n",
      "Progress: 54.22%\n",
      "Progress: 54.23%\n",
      "Progress: 54.24%\n",
      "Progress: 54.25%\n",
      "Progress: 54.26%\n",
      "Progress: 54.27%\n",
      "Progress: 54.28%\n",
      "Progress: 54.29%\n",
      "Progress: 54.30%\n",
      "Progress: 54.31%\n",
      "Progress: 54.32%\n",
      "Progress: 54.33%\n",
      "Progress: 54.34%\n",
      "Progress: 54.35%\n",
      "Progress: 54.36%\n",
      "Progress: 54.37%\n",
      "Progress: 54.38%\n",
      "Progress: 54.39%\n",
      "Progress: 54.40%\n",
      "Progress: 54.40%\n",
      "Progress: 54.41%\n",
      "Progress: 54.42%\n",
      "Progress: 54.43%\n",
      "Progress: 54.44%\n",
      "Progress: 54.45%\n",
      "Progress: 54.46%\n",
      "Progress: 54.47%\n",
      "Progress: 54.48%\n",
      "Progress: 54.49%\n",
      "Progress: 54.50%\n",
      "Progress: 54.51%\n",
      "Progress: 54.52%\n",
      "Progress: 54.53%\n",
      "Progress: 54.54%\n",
      "Progress: 54.55%\n",
      "Progress: 54.56%\n",
      "Progress: 54.57%\n",
      "Progress: 54.58%\n",
      "Progress: 54.59%\n",
      "Progress: 54.60%\n",
      "Progress: 54.61%\n",
      "Progress: 54.62%\n",
      "Progress: 54.63%\n",
      "Progress: 54.64%\n",
      "Progress: 54.64%\n",
      "Progress: 54.65%\n",
      "Progress: 54.66%\n",
      "Progress: 54.67%\n",
      "Progress: 54.68%\n",
      "Progress: 54.69%\n",
      "Progress: 54.70%\n",
      "Progress: 54.71%\n",
      "Progress: 54.72%\n",
      "Progress: 54.73%\n",
      "Progress: 54.74%\n",
      "Progress: 54.75%\n",
      "Progress: 54.76%\n",
      "Progress: 54.77%\n",
      "Progress: 54.78%\n",
      "Progress: 54.79%\n",
      "Progress: 54.80%\n",
      "Progress: 54.81%\n",
      "Progress: 54.82%\n",
      "Progress: 54.83%\n",
      "Progress: 54.84%\n",
      "Progress: 54.85%\n",
      "Progress: 54.86%\n",
      "Progress: 54.87%\n",
      "Progress: 54.87%\n",
      "Progress: 54.88%\n",
      "Progress: 54.89%\n",
      "Progress: 54.90%\n",
      "Progress: 54.91%\n",
      "Progress: 54.92%\n",
      "Progress: 54.93%\n",
      "Progress: 54.94%\n",
      "Progress: 54.95%\n",
      "Progress: 54.96%\n",
      "Progress: 54.97%\n",
      "Progress: 54.98%\n",
      "Progress: 54.99%\n",
      "Progress: 55.00%\n",
      "Progress: 55.01%\n",
      "Progress: 55.02%\n",
      "Progress: 55.03%\n",
      "Progress: 55.04%\n",
      "Progress: 55.05%\n",
      "Progress: 55.06%\n",
      "Progress: 55.07%\n",
      "Progress: 55.08%\n",
      "Progress: 55.08%\n",
      "Progress: 55.09%\n",
      "Progress: 55.10%\n",
      "Progress: 55.11%\n",
      "Progress: 55.12%\n",
      "Progress: 55.13%\n",
      "Progress: 55.14%\n",
      "Progress: 55.15%\n",
      "Progress: 55.16%\n",
      "Progress: 55.17%\n",
      "Progress: 55.18%\n",
      "Progress: 55.19%\n",
      "Progress: 55.20%\n",
      "Progress: 55.21%\n",
      "Progress: 55.22%\n",
      "Progress: 55.23%\n",
      "Progress: 55.24%\n",
      "Progress: 55.25%\n",
      "Progress: 55.26%\n",
      "Progress: 55.27%\n",
      "Progress: 55.28%\n",
      "Progress: 55.28%\n",
      "Progress: 55.29%\n",
      "Progress: 55.30%\n",
      "Progress: 55.31%\n",
      "Progress: 55.32%\n",
      "Progress: 55.33%\n",
      "Progress: 55.34%\n",
      "Progress: 55.35%\n",
      "Progress: 55.36%\n",
      "Progress: 55.37%\n",
      "Progress: 55.38%\n",
      "Progress: 55.39%\n",
      "Progress: 55.40%\n",
      "Progress: 55.41%\n",
      "Progress: 55.42%\n",
      "Progress: 55.43%\n",
      "Progress: 55.44%\n",
      "Progress: 55.45%\n",
      "Progress: 55.46%\n",
      "Progress: 55.47%\n",
      "Progress: 55.47%\n",
      "Progress: 55.48%\n",
      "Progress: 55.49%\n",
      "Progress: 55.50%\n",
      "Progress: 55.51%\n",
      "Progress: 55.52%\n",
      "Progress: 55.53%\n",
      "Progress: 55.54%\n",
      "Progress: 55.55%\n",
      "Progress: 55.56%\n",
      "Progress: 55.57%\n",
      "Progress: 55.58%\n",
      "Progress: 55.59%\n",
      "Progress: 55.60%\n",
      "Progress: 55.61%\n",
      "Progress: 55.62%\n",
      "Progress: 55.63%\n",
      "Progress: 55.64%\n",
      "Progress: 55.65%\n",
      "Progress: 55.66%\n",
      "Progress: 55.66%\n",
      "Progress: 55.67%\n",
      "Progress: 55.68%\n",
      "Progress: 55.69%\n",
      "Progress: 55.70%\n",
      "Progress: 55.71%\n",
      "Progress: 55.72%\n",
      "Progress: 55.73%\n",
      "Progress: 55.74%\n",
      "Progress: 55.75%\n",
      "Progress: 55.76%\n",
      "Progress: 55.77%\n",
      "Progress: 55.78%\n",
      "Progress: 55.79%\n",
      "Progress: 55.80%\n",
      "Progress: 55.81%\n",
      "Progress: 55.82%\n",
      "Progress: 55.83%\n",
      "Progress: 55.83%\n",
      "Progress: 55.84%\n",
      "Progress: 55.85%\n",
      "Progress: 55.86%\n",
      "Progress: 55.87%\n",
      "Progress: 55.88%\n",
      "Progress: 55.89%\n",
      "Progress: 55.90%\n",
      "Progress: 55.91%\n",
      "Progress: 55.92%\n",
      "Progress: 55.93%\n",
      "Progress: 55.94%\n",
      "Progress: 55.95%\n",
      "Progress: 55.96%\n",
      "Progress: 55.97%\n",
      "Progress: 55.98%\n",
      "Progress: 55.99%\n",
      "Progress: 56.00%\n",
      "Progress: 56.01%\n",
      "Progress: 56.01%\n",
      "Progress: 56.02%\n",
      "Progress: 56.03%\n",
      "Progress: 56.04%\n",
      "Progress: 56.05%\n",
      "Progress: 56.06%\n",
      "Progress: 56.07%\n",
      "Progress: 56.08%\n",
      "Progress: 56.09%\n",
      "Progress: 56.10%\n",
      "Progress: 56.11%\n",
      "Progress: 56.12%\n",
      "Progress: 56.13%\n",
      "Progress: 56.14%\n",
      "Progress: 56.15%\n",
      "Progress: 56.16%\n",
      "Progress: 56.17%\n",
      "Progress: 56.17%\n",
      "Progress: 56.18%\n",
      "Progress: 56.19%\n",
      "Progress: 56.20%\n",
      "Progress: 56.21%\n",
      "Progress: 56.22%\n",
      "Progress: 56.23%\n",
      "Progress: 56.24%\n",
      "Progress: 56.25%\n",
      "Progress: 56.26%\n",
      "Progress: 56.27%\n",
      "Progress: 56.28%\n",
      "Progress: 56.29%\n",
      "Progress: 56.30%\n",
      "Progress: 56.31%\n",
      "Progress: 56.32%\n",
      "Progress: 56.33%\n",
      "Progress: 56.33%\n",
      "Progress: 56.34%\n",
      "Progress: 56.35%\n",
      "Progress: 56.36%\n",
      "Progress: 56.37%\n",
      "Progress: 56.38%\n",
      "Progress: 56.39%\n",
      "Progress: 56.40%\n",
      "Progress: 56.41%\n",
      "Progress: 56.42%\n",
      "Progress: 56.43%\n",
      "Progress: 56.44%\n",
      "Progress: 56.45%\n",
      "Progress: 56.46%\n",
      "Progress: 56.47%\n",
      "Progress: 56.48%\n",
      "Progress: 56.49%\n",
      "Progress: 56.49%\n",
      "Progress: 56.50%\n",
      "Progress: 56.51%\n",
      "Progress: 56.52%\n",
      "Progress: 56.53%\n",
      "Progress: 56.54%\n",
      "Progress: 56.55%\n",
      "Progress: 56.56%\n",
      "Progress: 56.57%\n",
      "Progress: 56.58%\n",
      "Progress: 56.59%\n",
      "Progress: 56.60%\n",
      "Progress: 56.61%\n",
      "Progress: 56.62%\n",
      "Progress: 56.63%\n",
      "Progress: 56.64%\n",
      "Progress: 56.64%\n",
      "Progress: 56.65%\n",
      "Progress: 56.66%\n",
      "Progress: 56.67%\n",
      "Progress: 56.68%\n",
      "Progress: 56.69%\n",
      "Progress: 56.70%\n",
      "Progress: 56.71%\n",
      "Progress: 56.72%\n",
      "Progress: 56.73%\n",
      "Progress: 56.74%\n",
      "Progress: 56.75%\n",
      "Progress: 56.76%\n",
      "Progress: 56.77%\n",
      "Progress: 56.78%\n",
      "Progress: 56.79%\n",
      "Progress: 56.79%\n",
      "Progress: 56.80%\n",
      "Progress: 56.81%\n",
      "Progress: 56.82%\n",
      "Progress: 56.83%\n",
      "Progress: 56.84%\n",
      "Progress: 56.85%\n",
      "Progress: 56.86%\n",
      "Progress: 56.87%\n",
      "Progress: 56.88%\n",
      "Progress: 56.89%\n",
      "Progress: 56.90%\n",
      "Progress: 56.91%\n",
      "Progress: 56.92%\n",
      "Progress: 56.93%\n",
      "Progress: 56.93%\n",
      "Progress: 56.94%\n",
      "Progress: 56.95%\n",
      "Progress: 56.96%\n",
      "Progress: 56.97%\n",
      "Progress: 56.98%\n",
      "Progress: 56.99%\n",
      "Progress: 57.00%\n",
      "Progress: 57.01%\n",
      "Progress: 57.02%\n",
      "Progress: 57.03%\n",
      "Progress: 57.04%\n",
      "Progress: 57.05%\n",
      "Progress: 57.06%\n",
      "Progress: 57.07%\n",
      "Progress: 57.07%\n",
      "Progress: 57.08%\n",
      "Progress: 57.09%\n",
      "Progress: 57.10%\n",
      "Progress: 57.11%\n",
      "Progress: 57.12%\n",
      "Progress: 57.13%\n",
      "Progress: 57.14%\n",
      "Progress: 57.15%\n",
      "Progress: 57.16%\n",
      "Progress: 57.17%\n",
      "Progress: 57.18%\n",
      "Progress: 57.19%\n",
      "Progress: 57.20%\n",
      "Progress: 57.21%\n",
      "Progress: 57.21%\n",
      "Progress: 57.22%\n",
      "Progress: 57.23%\n",
      "Progress: 57.24%\n",
      "Progress: 57.25%\n",
      "Progress: 57.26%\n",
      "Progress: 57.27%\n",
      "Progress: 57.28%\n",
      "Progress: 57.29%\n",
      "Progress: 57.30%\n",
      "Progress: 57.31%\n",
      "Progress: 57.32%\n",
      "Progress: 57.33%\n",
      "Progress: 57.34%\n",
      "Progress: 57.34%\n",
      "Progress: 57.35%\n",
      "Progress: 57.36%\n",
      "Progress: 57.37%\n",
      "Progress: 57.38%\n",
      "Progress: 57.39%\n",
      "Progress: 57.40%\n",
      "Progress: 57.41%\n",
      "Progress: 57.42%\n",
      "Progress: 57.43%\n",
      "Progress: 57.44%\n",
      "Progress: 57.45%\n",
      "Progress: 57.46%\n",
      "Progress: 57.47%\n",
      "Progress: 57.47%\n",
      "Progress: 57.48%\n",
      "Progress: 57.49%\n",
      "Progress: 57.50%\n",
      "Progress: 57.51%\n",
      "Progress: 57.52%\n",
      "Progress: 57.53%\n",
      "Progress: 57.54%\n",
      "Progress: 57.55%\n",
      "Progress: 57.56%\n",
      "Progress: 57.57%\n",
      "Progress: 57.58%\n",
      "Progress: 57.59%\n",
      "Progress: 57.60%\n",
      "Progress: 57.60%\n",
      "Progress: 57.61%\n",
      "Progress: 57.62%\n",
      "Progress: 57.63%\n",
      "Progress: 57.64%\n",
      "Progress: 57.65%\n",
      "Progress: 57.66%\n",
      "Progress: 57.67%\n",
      "Progress: 57.68%\n",
      "Progress: 57.69%\n",
      "Progress: 57.70%\n",
      "Progress: 57.71%\n",
      "Progress: 57.72%\n",
      "Progress: 57.72%\n",
      "Progress: 57.73%\n",
      "Progress: 57.74%\n",
      "Progress: 57.75%\n",
      "Progress: 57.76%\n",
      "Progress: 57.77%\n",
      "Progress: 57.78%\n",
      "Progress: 57.79%\n",
      "Progress: 57.80%\n",
      "Progress: 57.81%\n",
      "Progress: 57.82%\n",
      "Progress: 57.83%\n",
      "Progress: 57.84%\n",
      "Progress: 57.84%\n",
      "Progress: 57.85%\n",
      "Progress: 57.86%\n",
      "Progress: 57.87%\n",
      "Progress: 57.88%\n",
      "Progress: 57.89%\n",
      "Progress: 57.90%\n",
      "Progress: 57.91%\n",
      "Progress: 57.92%\n",
      "Progress: 57.93%\n",
      "Progress: 57.94%\n",
      "Progress: 57.95%\n",
      "Progress: 57.96%\n",
      "Progress: 57.97%\n",
      "Progress: 57.97%\n",
      "Progress: 57.98%\n",
      "Progress: 57.99%\n",
      "Progress: 58.00%\n",
      "Progress: 58.01%\n",
      "Progress: 58.02%\n",
      "Progress: 58.03%\n",
      "Progress: 58.04%\n",
      "Progress: 58.05%\n",
      "Progress: 58.06%\n",
      "Progress: 58.07%\n",
      "Progress: 58.08%\n",
      "Progress: 58.08%\n",
      "Progress: 58.09%\n",
      "Progress: 58.10%\n",
      "Progress: 58.11%\n",
      "Progress: 58.12%\n",
      "Progress: 58.13%\n",
      "Progress: 58.14%\n",
      "Progress: 58.15%\n",
      "Progress: 58.16%\n",
      "Progress: 58.17%\n",
      "Progress: 58.18%\n",
      "Progress: 58.19%\n",
      "Progress: 58.20%\n",
      "Progress: 58.20%\n",
      "Progress: 58.21%\n",
      "Progress: 58.22%\n",
      "Progress: 58.23%\n",
      "Progress: 58.24%\n",
      "Progress: 58.25%\n",
      "Progress: 58.26%\n",
      "Progress: 58.27%\n",
      "Progress: 58.28%\n",
      "Progress: 58.29%\n",
      "Progress: 58.30%\n",
      "Progress: 58.31%\n",
      "Progress: 58.31%\n",
      "Progress: 58.32%\n",
      "Progress: 58.33%\n",
      "Progress: 58.34%\n",
      "Progress: 58.35%\n",
      "Progress: 58.36%\n",
      "Progress: 58.37%\n",
      "Progress: 58.38%\n",
      "Progress: 58.39%\n",
      "Progress: 58.40%\n",
      "Progress: 58.41%\n",
      "Progress: 58.42%\n",
      "Progress: 58.43%\n",
      "Progress: 58.43%\n",
      "Progress: 58.44%\n",
      "Progress: 58.45%\n",
      "Progress: 58.46%\n",
      "Progress: 58.47%\n",
      "Progress: 58.48%\n",
      "Progress: 58.49%\n",
      "Progress: 58.50%\n",
      "Progress: 58.51%\n",
      "Progress: 58.52%\n",
      "Progress: 58.53%\n",
      "Progress: 58.54%\n",
      "Progress: 58.54%\n",
      "Progress: 58.55%\n",
      "Progress: 58.56%\n",
      "Progress: 58.57%\n",
      "Progress: 58.58%\n",
      "Progress: 58.59%\n",
      "Progress: 58.60%\n",
      "Progress: 58.61%\n",
      "Progress: 58.62%\n",
      "Progress: 58.63%\n",
      "Progress: 58.64%\n",
      "Progress: 58.65%\n",
      "Progress: 58.65%\n",
      "Progress: 58.66%\n",
      "Progress: 58.67%\n",
      "Progress: 58.68%\n",
      "Progress: 58.69%\n",
      "Progress: 58.70%\n",
      "Progress: 58.71%\n",
      "Progress: 58.72%\n",
      "Progress: 58.73%\n",
      "Progress: 58.74%\n",
      "Progress: 58.75%\n",
      "Progress: 58.75%\n",
      "Progress: 58.76%\n",
      "Progress: 58.77%\n",
      "Progress: 58.78%\n",
      "Progress: 58.79%\n",
      "Progress: 58.80%\n",
      "Progress: 58.81%\n",
      "Progress: 58.82%\n",
      "Progress: 58.83%\n",
      "Progress: 58.84%\n",
      "Progress: 58.85%\n",
      "Progress: 58.86%\n",
      "Progress: 58.86%\n",
      "Progress: 58.87%\n",
      "Progress: 58.88%\n",
      "Progress: 58.89%\n",
      "Progress: 58.90%\n",
      "Progress: 58.91%\n",
      "Progress: 58.92%\n",
      "Progress: 58.93%\n",
      "Progress: 58.94%\n",
      "Progress: 58.95%\n",
      "Progress: 58.96%\n",
      "Progress: 58.96%\n",
      "Progress: 58.97%\n",
      "Progress: 58.98%\n",
      "Progress: 58.99%\n",
      "Progress: 59.00%\n",
      "Progress: 59.01%\n",
      "Progress: 59.02%\n",
      "Progress: 59.03%\n",
      "Progress: 59.04%\n",
      "Progress: 59.05%\n",
      "Progress: 59.06%\n",
      "Progress: 59.06%\n",
      "Progress: 59.07%\n",
      "Progress: 59.08%\n",
      "Progress: 59.09%\n",
      "Progress: 59.10%\n",
      "Progress: 59.11%\n",
      "Progress: 59.12%\n",
      "Progress: 59.13%\n",
      "Progress: 59.14%\n",
      "Progress: 59.15%\n",
      "Progress: 59.16%\n",
      "Progress: 59.17%\n",
      "Progress: 59.17%\n",
      "Progress: 59.18%\n",
      "Progress: 59.19%\n",
      "Progress: 59.20%\n",
      "Progress: 59.21%\n",
      "Progress: 59.22%\n",
      "Progress: 59.23%\n",
      "Progress: 59.24%\n",
      "Progress: 59.25%\n",
      "Progress: 59.26%\n",
      "Progress: 59.27%\n",
      "Progress: 59.27%\n",
      "Progress: 59.28%\n",
      "Progress: 59.29%\n",
      "Progress: 59.30%\n",
      "Progress: 59.31%\n",
      "Progress: 59.32%\n",
      "Progress: 59.33%\n",
      "Progress: 59.34%\n",
      "Progress: 59.35%\n",
      "Progress: 59.36%\n",
      "Progress: 59.36%\n",
      "Progress: 59.37%\n",
      "Progress: 59.38%\n",
      "Progress: 59.39%\n",
      "Progress: 59.40%\n",
      "Progress: 59.41%\n",
      "Progress: 59.42%\n",
      "Progress: 59.43%\n",
      "Progress: 59.44%\n",
      "Progress: 59.45%\n",
      "Progress: 59.46%\n",
      "Progress: 59.46%\n",
      "Progress: 59.47%\n",
      "Progress: 59.48%\n",
      "Progress: 59.49%\n",
      "Progress: 59.50%\n",
      "Progress: 59.51%\n",
      "Progress: 59.52%\n",
      "Progress: 59.53%\n",
      "Progress: 59.54%\n",
      "Progress: 59.55%\n",
      "Progress: 59.56%\n",
      "Progress: 59.56%\n",
      "Progress: 59.57%\n",
      "Progress: 59.58%\n",
      "Progress: 59.59%\n",
      "Progress: 59.60%\n",
      "Progress: 59.61%\n",
      "Progress: 59.62%\n",
      "Progress: 59.63%\n",
      "Progress: 59.64%\n",
      "Progress: 59.65%\n",
      "Progress: 59.65%\n",
      "Progress: 59.66%\n",
      "Progress: 59.67%\n",
      "Progress: 59.68%\n",
      "Progress: 59.69%\n",
      "Progress: 59.70%\n",
      "Progress: 59.71%\n",
      "Progress: 59.72%\n",
      "Progress: 59.73%\n",
      "Progress: 59.74%\n",
      "Progress: 59.74%\n",
      "Progress: 59.75%\n",
      "Progress: 59.76%\n",
      "Progress: 59.77%\n",
      "Progress: 59.78%\n",
      "Progress: 59.79%\n",
      "Progress: 59.80%\n",
      "Progress: 59.81%\n",
      "Progress: 59.82%\n",
      "Progress: 59.83%\n",
      "Progress: 59.84%\n",
      "Progress: 59.84%\n",
      "Progress: 59.85%\n",
      "Progress: 59.86%\n",
      "Progress: 59.87%\n",
      "Progress: 59.88%\n",
      "Progress: 59.89%\n",
      "Progress: 59.90%\n",
      "Progress: 59.91%\n",
      "Progress: 59.92%\n",
      "Progress: 59.93%\n",
      "Progress: 59.93%\n",
      "Progress: 59.94%\n",
      "Progress: 59.95%\n",
      "Progress: 59.96%\n",
      "Progress: 59.97%\n",
      "Progress: 59.98%\n",
      "Progress: 59.99%\n",
      "Progress: 60.00%\n",
      "Progress: 60.01%\n",
      "Progress: 60.02%\n",
      "Progress: 60.02%\n",
      "Progress: 60.03%\n",
      "Progress: 60.04%\n",
      "Progress: 60.05%\n",
      "Progress: 60.06%\n",
      "Progress: 60.07%\n",
      "Progress: 60.08%\n",
      "Progress: 60.09%\n",
      "Progress: 60.10%\n",
      "Progress: 60.11%\n",
      "Progress: 60.11%\n",
      "Progress: 60.12%\n",
      "Progress: 60.13%\n",
      "Progress: 60.14%\n",
      "Progress: 60.15%\n",
      "Progress: 60.16%\n",
      "Progress: 60.17%\n",
      "Progress: 60.18%\n",
      "Progress: 60.19%\n",
      "Progress: 60.20%\n",
      "Progress: 60.20%\n",
      "Progress: 60.21%\n",
      "Progress: 60.22%\n",
      "Progress: 60.23%\n",
      "Progress: 60.24%\n",
      "Progress: 60.25%\n",
      "Progress: 60.26%\n",
      "Progress: 60.27%\n",
      "Progress: 60.28%\n",
      "Progress: 60.28%\n",
      "Progress: 60.29%\n",
      "Progress: 60.30%\n",
      "Progress: 60.31%\n",
      "Progress: 60.32%\n",
      "Progress: 60.33%\n",
      "Progress: 60.34%\n",
      "Progress: 60.35%\n",
      "Progress: 60.36%\n",
      "Progress: 60.37%\n",
      "Progress: 60.37%\n",
      "Progress: 60.38%\n",
      "Progress: 60.39%\n",
      "Progress: 60.40%\n",
      "Progress: 60.41%\n",
      "Progress: 60.42%\n",
      "Progress: 60.43%\n",
      "Progress: 60.44%\n",
      "Progress: 60.45%\n",
      "Progress: 60.46%\n",
      "Progress: 60.46%\n",
      "Progress: 60.47%\n",
      "Progress: 60.48%\n",
      "Progress: 60.49%\n",
      "Progress: 60.50%\n",
      "Progress: 60.51%\n",
      "Progress: 60.52%\n",
      "Progress: 60.53%\n",
      "Progress: 60.54%\n",
      "Progress: 60.54%\n",
      "Progress: 60.55%\n",
      "Progress: 60.56%\n",
      "Progress: 60.57%\n",
      "Progress: 60.58%\n",
      "Progress: 60.59%\n",
      "Progress: 60.60%\n",
      "Progress: 60.61%\n",
      "Progress: 60.62%\n",
      "Progress: 60.63%\n",
      "Progress: 60.63%\n",
      "Progress: 60.64%\n",
      "Progress: 60.65%\n",
      "Progress: 60.66%\n",
      "Progress: 60.67%\n",
      "Progress: 60.68%\n",
      "Progress: 60.69%\n",
      "Progress: 60.70%\n",
      "Progress: 60.71%\n",
      "Progress: 60.71%\n",
      "Progress: 60.72%\n",
      "Progress: 60.73%\n",
      "Progress: 60.74%\n",
      "Progress: 60.75%\n",
      "Progress: 60.76%\n",
      "Progress: 60.77%\n",
      "Progress: 60.78%\n",
      "Progress: 60.79%\n",
      "Progress: 60.79%\n",
      "Progress: 60.80%\n",
      "Progress: 60.81%\n",
      "Progress: 60.82%\n",
      "Progress: 60.83%\n",
      "Progress: 60.84%\n",
      "Progress: 60.85%\n",
      "Progress: 60.86%\n",
      "Progress: 60.87%\n",
      "Progress: 60.87%\n",
      "Progress: 60.88%\n",
      "Progress: 60.89%\n",
      "Progress: 60.90%\n",
      "Progress: 60.91%\n",
      "Progress: 60.92%\n",
      "Progress: 60.93%\n",
      "Progress: 60.94%\n",
      "Progress: 60.95%\n",
      "Progress: 60.95%\n",
      "Progress: 60.96%\n",
      "Progress: 60.97%\n",
      "Progress: 60.98%\n",
      "Progress: 60.99%\n",
      "Progress: 61.00%\n",
      "Progress: 61.01%\n",
      "Progress: 61.02%\n",
      "Progress: 61.03%\n",
      "Progress: 61.03%\n",
      "Progress: 61.04%\n",
      "Progress: 61.05%\n",
      "Progress: 61.06%\n",
      "Progress: 61.07%\n",
      "Progress: 61.08%\n",
      "Progress: 61.09%\n",
      "Progress: 61.10%\n",
      "Progress: 61.11%\n",
      "Progress: 61.11%\n",
      "Progress: 61.12%\n",
      "Progress: 61.13%\n",
      "Progress: 61.14%\n",
      "Progress: 61.15%\n",
      "Progress: 61.16%\n",
      "Progress: 61.17%\n",
      "Progress: 61.18%\n",
      "Progress: 61.19%\n",
      "Progress: 61.19%\n",
      "Progress: 61.20%\n",
      "Progress: 61.21%\n",
      "Progress: 61.22%\n",
      "Progress: 61.23%\n",
      "Progress: 61.24%\n",
      "Progress: 61.25%\n",
      "Progress: 61.26%\n",
      "Progress: 61.27%\n",
      "Progress: 61.27%\n",
      "Progress: 61.28%\n",
      "Progress: 61.29%\n",
      "Progress: 61.30%\n",
      "Progress: 61.31%\n",
      "Progress: 61.32%\n",
      "Progress: 61.33%\n",
      "Progress: 61.34%\n",
      "Progress: 61.35%\n",
      "Progress: 61.35%\n",
      "Progress: 61.36%\n",
      "Progress: 61.37%\n",
      "Progress: 61.38%\n",
      "Progress: 61.39%\n",
      "Progress: 61.40%\n",
      "Progress: 61.41%\n",
      "Progress: 61.42%\n",
      "Progress: 61.42%\n",
      "Progress: 61.43%\n",
      "Progress: 61.44%\n",
      "Progress: 61.45%\n",
      "Progress: 61.46%\n",
      "Progress: 61.47%\n",
      "Progress: 61.48%\n",
      "Progress: 61.49%\n",
      "Progress: 61.50%\n",
      "Progress: 61.50%\n",
      "Progress: 61.51%\n",
      "Progress: 61.52%\n",
      "Progress: 61.53%\n",
      "Progress: 61.54%\n",
      "Progress: 61.55%\n",
      "Progress: 61.56%\n",
      "Progress: 61.57%\n",
      "Progress: 61.57%\n",
      "Progress: 61.58%\n",
      "Progress: 61.59%\n",
      "Progress: 61.60%\n",
      "Progress: 61.61%\n",
      "Progress: 61.62%\n",
      "Progress: 61.63%\n",
      "Progress: 61.64%\n",
      "Progress: 61.65%\n",
      "Progress: 61.65%\n",
      "Progress: 61.66%\n",
      "Progress: 61.67%\n",
      "Progress: 61.68%\n",
      "Progress: 61.69%\n",
      "Progress: 61.70%\n",
      "Progress: 61.71%\n",
      "Progress: 61.72%\n",
      "Progress: 61.72%\n",
      "Progress: 61.73%\n",
      "Progress: 61.74%\n",
      "Progress: 61.75%\n",
      "Progress: 61.76%\n",
      "Progress: 61.77%\n",
      "Progress: 61.78%\n",
      "Progress: 61.79%\n",
      "Progress: 61.79%\n",
      "Progress: 61.80%\n",
      "Progress: 61.81%\n",
      "Progress: 61.82%\n",
      "Progress: 61.83%\n",
      "Progress: 61.84%\n",
      "Progress: 61.85%\n",
      "Progress: 61.86%\n",
      "Progress: 61.87%\n",
      "Progress: 61.87%\n",
      "Progress: 61.88%\n",
      "Progress: 61.89%\n",
      "Progress: 61.90%\n",
      "Progress: 61.91%\n",
      "Progress: 61.92%\n",
      "Progress: 61.93%\n",
      "Progress: 61.94%\n",
      "Progress: 61.94%\n",
      "Progress: 61.95%\n",
      "Progress: 61.96%\n",
      "Progress: 61.97%\n",
      "Progress: 61.98%\n",
      "Progress: 61.99%\n",
      "Progress: 62.00%\n",
      "Progress: 62.01%\n",
      "Progress: 62.01%\n",
      "Progress: 62.02%\n",
      "Progress: 62.03%\n",
      "Progress: 62.04%\n",
      "Progress: 62.05%\n",
      "Progress: 62.06%\n",
      "Progress: 62.07%\n",
      "Progress: 62.08%\n",
      "Progress: 62.08%\n",
      "Progress: 62.09%\n",
      "Progress: 62.10%\n",
      "Progress: 62.11%\n",
      "Progress: 62.12%\n",
      "Progress: 62.13%\n",
      "Progress: 62.14%\n",
      "Progress: 62.15%\n",
      "Progress: 62.15%\n",
      "Progress: 62.16%\n",
      "Progress: 62.17%\n",
      "Progress: 62.18%\n",
      "Progress: 62.19%\n",
      "Progress: 62.20%\n",
      "Progress: 62.21%\n",
      "Progress: 62.22%\n",
      "Progress: 62.22%\n",
      "Progress: 62.23%\n",
      "Progress: 62.24%\n",
      "Progress: 62.25%\n",
      "Progress: 62.26%\n",
      "Progress: 62.27%\n",
      "Progress: 62.28%\n",
      "Progress: 62.29%\n",
      "Progress: 62.29%\n",
      "Progress: 62.30%\n",
      "Progress: 62.31%\n",
      "Progress: 62.32%\n",
      "Progress: 62.33%\n",
      "Progress: 62.34%\n",
      "Progress: 62.35%\n",
      "Progress: 62.36%\n",
      "Progress: 62.36%\n",
      "Progress: 62.37%\n",
      "Progress: 62.38%\n",
      "Progress: 62.39%\n",
      "Progress: 62.40%\n",
      "Progress: 62.41%\n",
      "Progress: 62.42%\n",
      "Progress: 62.43%\n",
      "Progress: 62.43%\n",
      "Progress: 62.44%\n",
      "Progress: 62.45%\n",
      "Progress: 62.46%\n",
      "Progress: 62.47%\n",
      "Progress: 62.48%\n",
      "Progress: 62.49%\n",
      "Progress: 62.50%\n",
      "Progress: 62.50%\n",
      "Progress: 62.51%\n",
      "Progress: 62.52%\n",
      "Progress: 62.53%\n",
      "Progress: 62.54%\n",
      "Progress: 62.55%\n",
      "Progress: 62.56%\n",
      "Progress: 62.57%\n",
      "Progress: 62.57%\n",
      "Progress: 62.58%\n",
      "Progress: 62.59%\n",
      "Progress: 62.60%\n",
      "Progress: 62.61%\n",
      "Progress: 62.62%\n",
      "Progress: 62.63%\n",
      "Progress: 62.63%\n",
      "Progress: 62.64%\n",
      "Progress: 62.65%\n",
      "Progress: 62.66%\n",
      "Progress: 62.67%\n",
      "Progress: 62.68%\n",
      "Progress: 62.69%\n",
      "Progress: 62.70%\n",
      "Progress: 62.70%\n",
      "Progress: 62.71%\n",
      "Progress: 62.72%\n",
      "Progress: 62.73%\n",
      "Progress: 62.74%\n",
      "Progress: 62.75%\n",
      "Progress: 62.76%\n",
      "Progress: 62.77%\n",
      "Progress: 62.77%\n",
      "Progress: 62.78%\n",
      "Progress: 62.79%\n",
      "Progress: 62.80%\n",
      "Progress: 62.81%\n",
      "Progress: 62.82%\n",
      "Progress: 62.83%\n",
      "Progress: 62.83%\n",
      "Progress: 62.84%\n",
      "Progress: 62.85%\n",
      "Progress: 62.86%\n",
      "Progress: 62.87%\n",
      "Progress: 62.88%\n",
      "Progress: 62.89%\n",
      "Progress: 62.90%\n",
      "Progress: 62.90%\n",
      "Progress: 62.91%\n",
      "Progress: 62.92%\n",
      "Progress: 62.93%\n",
      "Progress: 62.94%\n",
      "Progress: 62.95%\n",
      "Progress: 62.96%\n",
      "Progress: 62.96%\n",
      "Progress: 62.97%\n",
      "Progress: 62.98%\n",
      "Progress: 62.99%\n",
      "Progress: 63.00%\n",
      "Progress: 63.01%\n",
      "Progress: 63.02%\n",
      "Progress: 63.03%\n",
      "Progress: 63.03%\n",
      "Progress: 63.04%\n",
      "Progress: 63.05%\n",
      "Progress: 63.06%\n",
      "Progress: 63.07%\n",
      "Progress: 63.08%\n",
      "Progress: 63.09%\n",
      "Progress: 63.09%\n",
      "Progress: 63.10%\n",
      "Progress: 63.11%\n",
      "Progress: 63.12%\n",
      "Progress: 63.13%\n",
      "Progress: 63.14%\n",
      "Progress: 63.15%\n",
      "Progress: 63.15%\n",
      "Progress: 63.16%\n",
      "Progress: 63.17%\n",
      "Progress: 63.18%\n",
      "Progress: 63.19%\n",
      "Progress: 63.20%\n",
      "Progress: 63.21%\n",
      "Progress: 63.22%\n",
      "Progress: 63.22%\n",
      "Progress: 63.23%\n",
      "Progress: 63.24%\n",
      "Progress: 63.25%\n",
      "Progress: 63.26%\n",
      "Progress: 63.27%\n",
      "Progress: 63.28%\n",
      "Progress: 63.28%\n",
      "Progress: 63.29%\n",
      "Progress: 63.30%\n",
      "Progress: 63.31%\n",
      "Progress: 63.32%\n",
      "Progress: 63.33%\n",
      "Progress: 63.34%\n",
      "Progress: 63.34%\n",
      "Progress: 63.35%\n",
      "Progress: 63.36%\n",
      "Progress: 63.37%\n",
      "Progress: 63.38%\n",
      "Progress: 63.39%\n",
      "Progress: 63.40%\n",
      "Progress: 63.40%\n",
      "Progress: 63.41%\n",
      "Progress: 63.42%\n",
      "Progress: 63.43%\n",
      "Progress: 63.44%\n",
      "Progress: 63.45%\n",
      "Progress: 63.46%\n",
      "Progress: 63.47%\n",
      "Progress: 63.47%\n",
      "Progress: 63.48%\n",
      "Progress: 63.49%\n",
      "Progress: 63.50%\n",
      "Progress: 63.51%\n",
      "Progress: 63.52%\n",
      "Progress: 63.53%\n",
      "Progress: 63.53%\n",
      "Progress: 63.54%\n",
      "Progress: 63.55%\n",
      "Progress: 63.56%\n",
      "Progress: 63.57%\n",
      "Progress: 63.58%\n",
      "Progress: 63.59%\n",
      "Progress: 63.59%\n",
      "Progress: 63.60%\n",
      "Progress: 63.61%\n",
      "Progress: 63.62%\n",
      "Progress: 63.63%\n",
      "Progress: 63.64%\n",
      "Progress: 63.65%\n",
      "Progress: 63.65%\n",
      "Progress: 63.66%\n",
      "Progress: 63.67%\n",
      "Progress: 63.68%\n",
      "Progress: 63.69%\n",
      "Progress: 63.70%\n",
      "Progress: 63.71%\n",
      "Progress: 63.71%\n",
      "Progress: 63.72%\n",
      "Progress: 63.73%\n",
      "Progress: 63.74%\n",
      "Progress: 63.75%\n",
      "Progress: 63.76%\n",
      "Progress: 63.77%\n",
      "Progress: 63.77%\n",
      "Progress: 63.78%\n",
      "Progress: 63.79%\n",
      "Progress: 63.80%\n",
      "Progress: 63.81%\n",
      "Progress: 63.82%\n",
      "Progress: 63.83%\n",
      "Progress: 63.83%\n",
      "Progress: 63.84%\n",
      "Progress: 63.85%\n",
      "Progress: 63.86%\n",
      "Progress: 63.87%\n",
      "Progress: 63.88%\n",
      "Progress: 63.89%\n",
      "Progress: 63.89%\n",
      "Progress: 63.90%\n",
      "Progress: 63.91%\n",
      "Progress: 63.92%\n",
      "Progress: 63.93%\n",
      "Progress: 63.94%\n",
      "Progress: 63.95%\n",
      "Progress: 63.95%\n",
      "Progress: 63.96%\n",
      "Progress: 63.97%\n",
      "Progress: 63.98%\n",
      "Progress: 63.99%\n",
      "Progress: 64.00%\n",
      "Progress: 64.01%\n",
      "Progress: 64.01%\n",
      "Progress: 64.02%\n",
      "Progress: 64.03%\n",
      "Progress: 64.04%\n",
      "Progress: 64.05%\n",
      "Progress: 64.06%\n",
      "Progress: 64.06%\n",
      "Progress: 64.07%\n",
      "Progress: 64.08%\n",
      "Progress: 64.09%\n",
      "Progress: 64.10%\n",
      "Progress: 64.11%\n",
      "Progress: 64.12%\n",
      "Progress: 64.12%\n",
      "Progress: 64.13%\n",
      "Progress: 64.14%\n",
      "Progress: 64.15%\n",
      "Progress: 64.16%\n",
      "Progress: 64.17%\n",
      "Progress: 64.18%\n",
      "Progress: 64.18%\n",
      "Progress: 64.19%\n",
      "Progress: 64.20%\n",
      "Progress: 64.21%\n",
      "Progress: 64.22%\n",
      "Progress: 64.23%\n",
      "Progress: 64.24%\n",
      "Progress: 64.24%\n",
      "Progress: 64.25%\n",
      "Progress: 64.26%\n",
      "Progress: 64.27%\n",
      "Progress: 64.28%\n",
      "Progress: 64.29%\n",
      "Progress: 64.29%\n",
      "Progress: 64.30%\n",
      "Progress: 64.31%\n",
      "Progress: 64.32%\n",
      "Progress: 64.33%\n",
      "Progress: 64.34%\n",
      "Progress: 64.35%\n",
      "Progress: 64.35%\n",
      "Progress: 64.36%\n",
      "Progress: 64.37%\n",
      "Progress: 64.38%\n",
      "Progress: 64.39%\n",
      "Progress: 64.40%\n",
      "Progress: 64.41%\n",
      "Progress: 64.41%\n",
      "Progress: 64.42%\n",
      "Progress: 64.43%\n",
      "Progress: 64.44%\n",
      "Progress: 64.45%\n",
      "Progress: 64.46%\n",
      "Progress: 64.46%\n",
      "Progress: 64.47%\n",
      "Progress: 64.48%\n",
      "Progress: 64.49%\n",
      "Progress: 64.50%\n",
      "Progress: 64.51%\n",
      "Progress: 64.52%\n",
      "Progress: 64.52%\n",
      "Progress: 64.53%\n",
      "Progress: 64.54%\n",
      "Progress: 64.55%\n",
      "Progress: 64.56%\n",
      "Progress: 64.57%\n",
      "Progress: 64.57%\n",
      "Progress: 64.58%\n",
      "Progress: 64.59%\n",
      "Progress: 64.60%\n",
      "Progress: 64.61%\n",
      "Progress: 64.62%\n",
      "Progress: 64.63%\n",
      "Progress: 64.63%\n",
      "Progress: 64.64%\n",
      "Progress: 64.65%\n",
      "Progress: 64.66%\n",
      "Progress: 64.67%\n",
      "Progress: 64.68%\n",
      "Progress: 64.68%\n",
      "Progress: 64.69%\n",
      "Progress: 64.70%\n",
      "Progress: 64.71%\n",
      "Progress: 64.72%\n",
      "Progress: 64.73%\n",
      "Progress: 64.74%\n",
      "Progress: 64.74%\n",
      "Progress: 64.75%\n",
      "Progress: 64.76%\n",
      "Progress: 64.77%\n",
      "Progress: 64.78%\n",
      "Progress: 64.79%\n",
      "Progress: 64.79%\n",
      "Progress: 64.80%\n",
      "Progress: 64.81%\n",
      "Progress: 64.82%\n",
      "Progress: 64.83%\n",
      "Progress: 64.84%\n",
      "Progress: 64.85%\n",
      "Progress: 64.85%\n",
      "Progress: 64.86%\n",
      "Progress: 64.87%\n",
      "Progress: 64.88%\n",
      "Progress: 64.89%\n",
      "Progress: 64.90%\n",
      "Progress: 64.90%\n",
      "Progress: 64.91%\n",
      "Progress: 64.92%\n",
      "Progress: 64.93%\n",
      "Progress: 64.94%\n",
      "Progress: 64.95%\n",
      "Progress: 64.96%\n",
      "Progress: 64.96%\n",
      "Progress: 64.97%\n",
      "Progress: 64.98%\n",
      "Progress: 64.99%\n",
      "Progress: 65.00%\n",
      "Progress: 65.01%\n",
      "Progress: 65.01%\n",
      "Progress: 65.02%\n",
      "Progress: 65.03%\n",
      "Progress: 65.04%\n",
      "Progress: 65.05%\n",
      "Progress: 65.06%\n",
      "Progress: 65.06%\n",
      "Progress: 65.07%\n",
      "Progress: 65.08%\n",
      "Progress: 65.09%\n",
      "Progress: 65.10%\n",
      "Progress: 65.11%\n",
      "Progress: 65.11%\n",
      "Progress: 65.12%\n",
      "Progress: 65.13%\n",
      "Progress: 65.14%\n",
      "Progress: 65.15%\n",
      "Progress: 65.16%\n",
      "Progress: 65.17%\n",
      "Progress: 65.17%\n",
      "Progress: 65.18%\n",
      "Progress: 65.19%\n",
      "Progress: 65.20%\n",
      "Progress: 65.21%\n",
      "Progress: 65.22%\n",
      "Progress: 65.22%\n",
      "Progress: 65.23%\n",
      "Progress: 65.24%\n",
      "Progress: 65.25%\n",
      "Progress: 65.26%\n",
      "Progress: 65.27%\n",
      "Progress: 65.27%\n",
      "Progress: 65.28%\n",
      "Progress: 65.29%\n",
      "Progress: 65.30%\n",
      "Progress: 65.31%\n",
      "Progress: 65.32%\n",
      "Progress: 65.32%\n",
      "Progress: 65.33%\n",
      "Progress: 65.34%\n",
      "Progress: 65.35%\n",
      "Progress: 65.36%\n",
      "Progress: 65.37%\n",
      "Progress: 65.38%\n",
      "Progress: 65.38%\n",
      "Progress: 65.39%\n",
      "Progress: 65.40%\n",
      "Progress: 65.41%\n",
      "Progress: 65.42%\n",
      "Progress: 65.43%\n",
      "Progress: 65.43%\n",
      "Progress: 65.44%\n",
      "Progress: 65.45%\n",
      "Progress: 65.46%\n",
      "Progress: 65.47%\n",
      "Progress: 65.48%\n",
      "Progress: 65.48%\n",
      "Progress: 65.49%\n",
      "Progress: 65.50%\n",
      "Progress: 65.51%\n",
      "Progress: 65.52%\n",
      "Progress: 65.53%\n",
      "Progress: 65.53%\n",
      "Progress: 65.54%\n",
      "Progress: 65.55%\n",
      "Progress: 65.56%\n",
      "Progress: 65.57%\n",
      "Progress: 65.58%\n",
      "Progress: 65.58%\n",
      "Progress: 65.59%\n",
      "Progress: 65.60%\n",
      "Progress: 65.61%\n",
      "Progress: 65.62%\n",
      "Progress: 65.63%\n",
      "Progress: 65.63%\n",
      "Progress: 65.64%\n",
      "Progress: 65.65%\n",
      "Progress: 65.66%\n",
      "Progress: 65.67%\n",
      "Progress: 65.68%\n",
      "Progress: 65.68%\n",
      "Progress: 65.69%\n",
      "Progress: 65.70%\n",
      "Progress: 65.71%\n",
      "Progress: 65.72%\n",
      "Progress: 65.73%\n",
      "Progress: 65.73%\n",
      "Progress: 65.74%\n",
      "Progress: 65.75%\n",
      "Progress: 65.76%\n",
      "Progress: 65.77%\n",
      "Progress: 65.78%\n",
      "Progress: 65.78%\n",
      "Progress: 65.79%\n",
      "Progress: 65.80%\n",
      "Progress: 65.81%\n",
      "Progress: 65.82%\n",
      "Progress: 65.83%\n",
      "Progress: 65.83%\n",
      "Progress: 65.84%\n",
      "Progress: 65.85%\n",
      "Progress: 65.86%\n",
      "Progress: 65.87%\n",
      "Progress: 65.88%\n",
      "Progress: 65.88%\n",
      "Progress: 65.89%\n",
      "Progress: 65.90%\n",
      "Progress: 65.91%\n",
      "Progress: 65.92%\n",
      "Progress: 65.93%\n",
      "Progress: 65.93%\n",
      "Progress: 65.94%\n",
      "Progress: 65.95%\n",
      "Progress: 65.96%\n",
      "Progress: 65.97%\n",
      "Progress: 65.98%\n",
      "Progress: 65.98%\n",
      "Progress: 65.99%\n",
      "Progress: 66.00%\n",
      "Progress: 66.01%\n",
      "Progress: 66.02%\n",
      "Progress: 66.03%\n",
      "Progress: 66.03%\n",
      "Progress: 66.04%\n",
      "Progress: 66.05%\n",
      "Progress: 66.06%\n",
      "Progress: 66.07%\n",
      "Progress: 66.07%\n",
      "Progress: 66.08%\n",
      "Progress: 66.09%\n",
      "Progress: 66.10%\n",
      "Progress: 66.11%\n",
      "Progress: 66.12%\n",
      "Progress: 66.12%\n",
      "Progress: 66.13%\n",
      "Progress: 66.14%\n",
      "Progress: 66.15%\n",
      "Progress: 66.16%\n",
      "Progress: 66.17%\n",
      "Progress: 66.17%\n",
      "Progress: 66.18%\n",
      "Progress: 66.19%\n",
      "Progress: 66.20%\n",
      "Progress: 66.21%\n",
      "Progress: 66.22%\n",
      "Progress: 66.22%\n",
      "Progress: 66.23%\n",
      "Progress: 66.24%\n",
      "Progress: 66.25%\n",
      "Progress: 66.26%\n",
      "Progress: 66.27%\n",
      "Progress: 66.27%\n",
      "Progress: 66.28%\n",
      "Progress: 66.29%\n",
      "Progress: 66.30%\n",
      "Progress: 66.31%\n",
      "Progress: 66.31%\n",
      "Progress: 66.32%\n",
      "Progress: 66.33%\n",
      "Progress: 66.34%\n",
      "Progress: 66.35%\n",
      "Progress: 66.36%\n",
      "Progress: 66.36%\n",
      "Progress: 66.37%\n",
      "Progress: 66.38%\n",
      "Progress: 66.39%\n",
      "Progress: 66.40%\n",
      "Progress: 66.41%\n",
      "Progress: 66.41%\n",
      "Progress: 66.42%\n",
      "Progress: 66.43%\n",
      "Progress: 66.44%\n",
      "Progress: 66.45%\n",
      "Progress: 66.46%\n",
      "Progress: 66.46%\n",
      "Progress: 66.47%\n",
      "Progress: 66.48%\n",
      "Progress: 66.49%\n",
      "Progress: 66.50%\n",
      "Progress: 66.50%\n",
      "Progress: 66.51%\n",
      "Progress: 66.52%\n",
      "Progress: 66.53%\n",
      "Progress: 66.54%\n",
      "Progress: 66.55%\n",
      "Progress: 66.55%\n",
      "Progress: 66.56%\n",
      "Progress: 66.57%\n",
      "Progress: 66.58%\n",
      "Progress: 66.59%\n",
      "Progress: 66.60%\n",
      "Progress: 66.60%\n",
      "Progress: 66.61%\n",
      "Progress: 66.62%\n",
      "Progress: 66.63%\n",
      "Progress: 66.64%\n",
      "Progress: 66.64%\n",
      "Progress: 66.65%\n",
      "Progress: 66.66%\n",
      "Progress: 66.67%\n",
      "Progress: 66.68%\n",
      "Progress: 66.69%\n",
      "Progress: 66.69%\n",
      "Progress: 66.70%\n",
      "Progress: 66.71%\n",
      "Progress: 66.72%\n",
      "Progress: 66.73%\n",
      "Progress: 66.73%\n",
      "Progress: 66.74%\n",
      "Progress: 66.75%\n",
      "Progress: 66.76%\n",
      "Progress: 66.77%\n",
      "Progress: 66.78%\n",
      "Progress: 66.78%\n",
      "Progress: 66.79%\n",
      "Progress: 66.80%\n",
      "Progress: 66.81%\n",
      "Progress: 66.82%\n",
      "Progress: 66.82%\n",
      "Progress: 66.83%\n",
      "Progress: 66.84%\n",
      "Progress: 66.85%\n",
      "Progress: 66.86%\n",
      "Progress: 66.87%\n",
      "Progress: 66.87%\n",
      "Progress: 66.88%\n",
      "Progress: 66.89%\n",
      "Progress: 66.90%\n",
      "Progress: 66.91%\n",
      "Progress: 66.92%\n",
      "Progress: 66.92%\n",
      "Progress: 66.93%\n",
      "Progress: 66.94%\n",
      "Progress: 66.95%\n",
      "Progress: 66.96%\n",
      "Progress: 66.96%\n",
      "Progress: 66.97%\n",
      "Progress: 66.98%\n",
      "Progress: 66.99%\n",
      "Progress: 67.00%\n",
      "Progress: 67.01%\n",
      "Progress: 67.01%\n",
      "Progress: 67.02%\n",
      "Progress: 67.03%\n",
      "Progress: 67.04%\n",
      "Progress: 67.05%\n",
      "Progress: 67.05%\n",
      "Progress: 67.06%\n",
      "Progress: 67.07%\n",
      "Progress: 67.08%\n",
      "Progress: 67.09%\n",
      "Progress: 67.09%\n",
      "Progress: 67.10%\n",
      "Progress: 67.11%\n",
      "Progress: 67.12%\n",
      "Progress: 67.13%\n",
      "Progress: 67.14%\n",
      "Progress: 67.14%\n",
      "Progress: 67.15%\n",
      "Progress: 67.16%\n",
      "Progress: 67.17%\n",
      "Progress: 67.18%\n",
      "Progress: 67.18%\n",
      "Progress: 67.19%\n",
      "Progress: 67.20%\n",
      "Progress: 67.21%\n",
      "Progress: 67.22%\n",
      "Progress: 67.23%\n",
      "Progress: 67.23%\n",
      "Progress: 67.24%\n",
      "Progress: 67.25%\n",
      "Progress: 67.26%\n",
      "Progress: 67.27%\n",
      "Progress: 67.27%\n",
      "Progress: 67.28%\n",
      "Progress: 67.29%\n",
      "Progress: 67.30%\n",
      "Progress: 67.31%\n",
      "Progress: 67.31%\n",
      "Progress: 67.32%\n",
      "Progress: 67.33%\n",
      "Progress: 67.34%\n",
      "Progress: 67.35%\n",
      "Progress: 67.36%\n",
      "Progress: 67.36%\n",
      "Progress: 67.37%\n",
      "Progress: 67.38%\n",
      "Progress: 67.39%\n",
      "Progress: 67.40%\n",
      "Progress: 67.40%\n",
      "Progress: 67.41%\n",
      "Progress: 67.42%\n",
      "Progress: 67.43%\n",
      "Progress: 67.44%\n",
      "Progress: 67.45%\n",
      "Progress: 67.45%\n",
      "Progress: 67.46%\n",
      "Progress: 67.47%\n",
      "Progress: 67.48%\n",
      "Progress: 67.49%\n",
      "Progress: 67.49%\n",
      "Progress: 67.50%\n",
      "Progress: 67.51%\n",
      "Progress: 67.52%\n",
      "Progress: 67.53%\n",
      "Progress: 67.53%\n",
      "Progress: 67.54%\n",
      "Progress: 67.55%\n",
      "Progress: 67.56%\n",
      "Progress: 67.57%\n",
      "Progress: 67.57%\n",
      "Progress: 67.58%\n",
      "Progress: 67.59%\n",
      "Progress: 67.60%\n",
      "Progress: 67.61%\n",
      "Progress: 67.62%\n",
      "Progress: 67.62%\n",
      "Progress: 67.63%\n",
      "Progress: 67.64%\n",
      "Progress: 67.65%\n",
      "Progress: 67.66%\n",
      "Progress: 67.66%\n",
      "Progress: 67.67%\n",
      "Progress: 67.68%\n",
      "Progress: 67.69%\n",
      "Progress: 67.70%\n",
      "Progress: 67.70%\n",
      "Progress: 67.71%\n",
      "Progress: 67.72%\n",
      "Progress: 67.73%\n",
      "Progress: 67.74%\n",
      "Progress: 67.74%\n",
      "Progress: 67.75%\n",
      "Progress: 67.76%\n",
      "Progress: 67.77%\n",
      "Progress: 67.78%\n",
      "Progress: 67.79%\n",
      "Progress: 67.79%\n",
      "Progress: 67.80%\n",
      "Progress: 67.81%\n",
      "Progress: 67.82%\n",
      "Progress: 67.83%\n",
      "Progress: 67.83%\n",
      "Progress: 67.84%\n",
      "Progress: 67.85%\n",
      "Progress: 67.86%\n",
      "Progress: 67.87%\n",
      "Progress: 67.87%\n",
      "Progress: 67.88%\n",
      "Progress: 67.89%\n",
      "Progress: 67.90%\n",
      "Progress: 67.91%\n",
      "Progress: 67.91%\n",
      "Progress: 67.92%\n",
      "Progress: 67.93%\n",
      "Progress: 67.94%\n",
      "Progress: 67.95%\n",
      "Progress: 67.95%\n",
      "Progress: 67.96%\n",
      "Progress: 67.97%\n",
      "Progress: 67.98%\n",
      "Progress: 67.99%\n",
      "Progress: 67.99%\n",
      "Progress: 68.00%\n",
      "Progress: 68.01%\n",
      "Progress: 68.02%\n",
      "Progress: 68.03%\n",
      "Progress: 68.04%\n",
      "Progress: 68.04%\n",
      "Progress: 68.05%\n",
      "Progress: 68.06%\n",
      "Progress: 68.07%\n",
      "Progress: 68.08%\n",
      "Progress: 68.08%\n",
      "Progress: 68.09%\n",
      "Progress: 68.10%\n",
      "Progress: 68.11%\n",
      "Progress: 68.12%\n",
      "Progress: 68.12%\n",
      "Progress: 68.13%\n",
      "Progress: 68.14%\n",
      "Progress: 68.15%\n",
      "Progress: 68.16%\n",
      "Progress: 68.16%\n",
      "Progress: 68.17%\n",
      "Progress: 68.18%\n",
      "Progress: 68.19%\n",
      "Progress: 68.20%\n",
      "Progress: 68.20%\n",
      "Progress: 68.21%\n",
      "Progress: 68.22%\n",
      "Progress: 68.23%\n",
      "Progress: 68.24%\n",
      "Progress: 68.24%\n",
      "Progress: 68.25%\n",
      "Progress: 68.26%\n",
      "Progress: 68.27%\n",
      "Progress: 68.28%\n",
      "Progress: 68.28%\n",
      "Progress: 68.29%\n",
      "Progress: 68.30%\n",
      "Progress: 68.31%\n",
      "Progress: 68.32%\n",
      "Progress: 68.32%\n",
      "Progress: 68.33%\n",
      "Progress: 68.34%\n",
      "Progress: 68.35%\n",
      "Progress: 68.36%\n",
      "Progress: 68.36%\n",
      "Progress: 68.37%\n",
      "Progress: 68.38%\n",
      "Progress: 68.39%\n",
      "Progress: 68.40%\n",
      "Progress: 68.40%\n",
      "Progress: 68.41%\n",
      "Progress: 68.42%\n",
      "Progress: 68.43%\n",
      "Progress: 68.44%\n",
      "Progress: 68.44%\n",
      "Progress: 68.45%\n",
      "Progress: 68.46%\n",
      "Progress: 68.47%\n",
      "Progress: 68.48%\n",
      "Progress: 68.48%\n",
      "Progress: 68.49%\n",
      "Progress: 68.50%\n",
      "Progress: 68.51%\n",
      "Progress: 68.52%\n",
      "Progress: 68.52%\n",
      "Progress: 68.53%\n",
      "Progress: 68.54%\n",
      "Progress: 68.55%\n",
      "Progress: 68.56%\n",
      "Progress: 68.56%\n",
      "Progress: 68.57%\n",
      "Progress: 68.58%\n",
      "Progress: 68.59%\n",
      "Progress: 68.60%\n",
      "Progress: 68.60%\n",
      "Progress: 68.61%\n",
      "Progress: 68.62%\n",
      "Progress: 68.63%\n",
      "Progress: 68.64%\n",
      "Progress: 68.64%\n",
      "Progress: 68.65%\n",
      "Progress: 68.66%\n",
      "Progress: 68.67%\n",
      "Progress: 68.68%\n",
      "Progress: 68.68%\n",
      "Progress: 68.69%\n",
      "Progress: 68.70%\n",
      "Progress: 68.71%\n",
      "Progress: 68.72%\n",
      "Progress: 68.72%\n",
      "Progress: 68.73%\n",
      "Progress: 68.74%\n",
      "Progress: 68.75%\n",
      "Progress: 68.76%\n",
      "Progress: 68.76%\n",
      "Progress: 68.77%\n",
      "Progress: 68.78%\n",
      "Progress: 68.79%\n",
      "Progress: 68.79%\n",
      "Progress: 68.80%\n",
      "Progress: 68.81%\n",
      "Progress: 68.82%\n",
      "Progress: 68.83%\n",
      "Progress: 68.83%\n",
      "Progress: 68.84%\n",
      "Progress: 68.85%\n",
      "Progress: 68.86%\n",
      "Progress: 68.87%\n",
      "Progress: 68.87%\n",
      "Progress: 68.88%\n",
      "Progress: 68.89%\n",
      "Progress: 68.90%\n",
      "Progress: 68.91%\n",
      "Progress: 68.91%\n",
      "Progress: 68.92%\n",
      "Progress: 68.93%\n",
      "Progress: 68.94%\n",
      "Progress: 68.95%\n",
      "Progress: 68.95%\n",
      "Progress: 68.96%\n",
      "Progress: 68.97%\n",
      "Progress: 68.98%\n",
      "Progress: 68.99%\n",
      "Progress: 68.99%\n",
      "Progress: 69.00%\n",
      "Progress: 69.01%\n",
      "Progress: 69.02%\n",
      "Progress: 69.03%\n",
      "Progress: 69.03%\n",
      "Progress: 69.04%\n",
      "Progress: 69.05%\n",
      "Progress: 69.06%\n",
      "Progress: 69.06%\n",
      "Progress: 69.07%\n",
      "Progress: 69.08%\n",
      "Progress: 69.09%\n",
      "Progress: 69.10%\n",
      "Progress: 69.10%\n",
      "Progress: 69.11%\n",
      "Progress: 69.12%\n",
      "Progress: 69.13%\n",
      "Progress: 69.14%\n",
      "Progress: 69.14%\n",
      "Progress: 69.15%\n",
      "Progress: 69.16%\n",
      "Progress: 69.17%\n",
      "Progress: 69.18%\n",
      "Progress: 69.18%\n",
      "Progress: 69.19%\n",
      "Progress: 69.20%\n",
      "Progress: 69.21%\n",
      "Progress: 69.21%\n",
      "Progress: 69.22%\n",
      "Progress: 69.23%\n",
      "Progress: 69.24%\n",
      "Progress: 69.25%\n",
      "Progress: 69.25%\n",
      "Progress: 69.26%\n",
      "Progress: 69.27%\n",
      "Progress: 69.28%\n",
      "Progress: 69.29%\n",
      "Progress: 69.29%\n",
      "Progress: 69.30%\n",
      "Progress: 69.31%\n",
      "Progress: 69.32%\n",
      "Progress: 69.33%\n",
      "Progress: 69.33%\n",
      "Progress: 69.34%\n",
      "Progress: 69.35%\n",
      "Progress: 69.36%\n",
      "Progress: 69.36%\n",
      "Progress: 69.37%\n",
      "Progress: 69.38%\n",
      "Progress: 69.39%\n",
      "Progress: 69.40%\n",
      "Progress: 69.40%\n",
      "Progress: 69.41%\n",
      "Progress: 69.42%\n",
      "Progress: 69.43%\n",
      "Progress: 69.44%\n",
      "Progress: 69.44%\n",
      "Progress: 69.45%\n",
      "Progress: 69.46%\n",
      "Progress: 69.47%\n",
      "Progress: 69.47%\n",
      "Progress: 69.48%\n",
      "Progress: 69.49%\n",
      "Progress: 69.50%\n",
      "Progress: 69.51%\n",
      "Progress: 69.51%\n",
      "Progress: 69.52%\n",
      "Progress: 69.53%\n",
      "Progress: 69.54%\n",
      "Progress: 69.55%\n",
      "Progress: 69.55%\n",
      "Progress: 69.56%\n",
      "Progress: 69.57%\n",
      "Progress: 69.58%\n",
      "Progress: 69.58%\n",
      "Progress: 69.59%\n",
      "Progress: 69.60%\n",
      "Progress: 69.61%\n",
      "Progress: 69.62%\n",
      "Progress: 69.62%\n",
      "Progress: 69.63%\n",
      "Progress: 69.64%\n",
      "Progress: 69.65%\n",
      "Progress: 69.66%\n",
      "Progress: 69.66%\n",
      "Progress: 69.67%\n",
      "Progress: 69.68%\n",
      "Progress: 69.69%\n",
      "Progress: 69.69%\n",
      "Progress: 69.70%\n",
      "Progress: 69.71%\n",
      "Progress: 69.72%\n",
      "Progress: 69.73%\n",
      "Progress: 69.73%\n",
      "Progress: 69.74%\n",
      "Progress: 69.75%\n",
      "Progress: 69.76%\n",
      "Progress: 69.77%\n",
      "Progress: 69.77%\n",
      "Progress: 69.78%\n",
      "Progress: 69.79%\n",
      "Progress: 69.80%\n",
      "Progress: 69.80%\n",
      "Progress: 69.81%\n",
      "Progress: 69.82%\n",
      "Progress: 69.83%\n",
      "Progress: 69.84%\n",
      "Progress: 69.84%\n",
      "Progress: 69.85%\n",
      "Progress: 69.86%\n",
      "Progress: 69.87%\n",
      "Progress: 69.87%\n",
      "Progress: 69.88%\n",
      "Progress: 69.89%\n",
      "Progress: 69.90%\n",
      "Progress: 69.91%\n",
      "Progress: 69.91%\n",
      "Progress: 69.92%\n",
      "Progress: 69.93%\n",
      "Progress: 69.94%\n",
      "Progress: 69.94%\n",
      "Progress: 69.95%\n",
      "Progress: 69.96%\n",
      "Progress: 69.97%\n",
      "Progress: 69.98%\n",
      "Progress: 69.98%\n",
      "Progress: 69.99%\n",
      "Progress: 70.00%\n",
      "Progress: 70.01%\n",
      "Progress: 70.01%\n",
      "Progress: 70.02%\n",
      "Progress: 70.03%\n",
      "Progress: 70.04%\n",
      "Progress: 70.05%\n",
      "Progress: 70.05%\n",
      "Progress: 70.06%\n",
      "Progress: 70.07%\n",
      "Progress: 70.08%\n",
      "Progress: 70.09%\n",
      "Progress: 70.09%\n",
      "Progress: 70.10%\n",
      "Progress: 70.11%\n",
      "Progress: 70.12%\n",
      "Progress: 70.12%\n",
      "Progress: 70.13%\n",
      "Progress: 70.14%\n",
      "Progress: 70.15%\n",
      "Progress: 70.16%\n",
      "Progress: 70.16%\n",
      "Progress: 70.17%\n",
      "Progress: 70.18%\n",
      "Progress: 70.19%\n",
      "Progress: 70.19%\n",
      "Progress: 70.20%\n",
      "Progress: 70.21%\n",
      "Progress: 70.22%\n",
      "Progress: 70.22%\n",
      "Progress: 70.23%\n",
      "Progress: 70.24%\n",
      "Progress: 70.25%\n",
      "Progress: 70.26%\n",
      "Progress: 70.26%\n",
      "Progress: 70.27%\n",
      "Progress: 70.28%\n",
      "Progress: 70.29%\n",
      "Progress: 70.29%\n",
      "Progress: 70.30%\n",
      "Progress: 70.31%\n",
      "Progress: 70.32%\n",
      "Progress: 70.33%\n",
      "Progress: 70.33%\n",
      "Progress: 70.34%\n",
      "Progress: 70.35%\n",
      "Progress: 70.36%\n",
      "Progress: 70.36%\n",
      "Progress: 70.37%\n",
      "Progress: 70.38%\n",
      "Progress: 70.39%\n",
      "Progress: 70.40%\n",
      "Progress: 70.40%\n",
      "Progress: 70.41%\n",
      "Progress: 70.42%\n",
      "Progress: 70.43%\n",
      "Progress: 70.43%\n",
      "Progress: 70.44%\n",
      "Progress: 70.45%\n",
      "Progress: 70.46%\n",
      "Progress: 70.47%\n",
      "Progress: 70.47%\n",
      "Progress: 70.48%\n",
      "Progress: 70.49%\n",
      "Progress: 70.50%\n",
      "Progress: 70.50%\n",
      "Progress: 70.51%\n",
      "Progress: 70.52%\n",
      "Progress: 70.53%\n",
      "Progress: 70.53%\n",
      "Progress: 70.54%\n",
      "Progress: 70.55%\n",
      "Progress: 70.56%\n",
      "Progress: 70.57%\n",
      "Progress: 70.57%\n",
      "Progress: 70.58%\n",
      "Progress: 70.59%\n",
      "Progress: 70.60%\n",
      "Progress: 70.60%\n",
      "Progress: 70.61%\n",
      "Progress: 70.62%\n",
      "Progress: 70.63%\n",
      "Progress: 70.64%\n",
      "Progress: 70.64%\n",
      "Progress: 70.65%\n",
      "Progress: 70.66%\n",
      "Progress: 70.67%\n",
      "Progress: 70.67%\n",
      "Progress: 70.68%\n",
      "Progress: 70.69%\n",
      "Progress: 70.70%\n",
      "Progress: 70.70%\n",
      "Progress: 70.71%\n",
      "Progress: 70.72%\n",
      "Progress: 70.73%\n",
      "Progress: 70.74%\n",
      "Progress: 70.74%\n",
      "Progress: 70.75%\n",
      "Progress: 70.76%\n",
      "Progress: 70.77%\n",
      "Progress: 70.77%\n",
      "Progress: 70.78%\n",
      "Progress: 70.79%\n",
      "Progress: 70.80%\n",
      "Progress: 70.80%\n",
      "Progress: 70.81%\n",
      "Progress: 70.82%\n",
      "Progress: 70.83%\n",
      "Progress: 70.84%\n",
      "Progress: 70.84%\n",
      "Progress: 70.85%\n",
      "Progress: 70.86%\n",
      "Progress: 70.87%\n",
      "Progress: 70.87%\n",
      "Progress: 70.88%\n",
      "Progress: 70.89%\n",
      "Progress: 70.90%\n",
      "Progress: 70.90%\n",
      "Progress: 70.91%\n",
      "Progress: 70.92%\n",
      "Progress: 70.93%\n",
      "Progress: 70.94%\n",
      "Progress: 70.94%\n",
      "Progress: 70.95%\n",
      "Progress: 70.96%\n",
      "Progress: 70.97%\n",
      "Progress: 70.97%\n",
      "Progress: 70.98%\n",
      "Progress: 70.99%\n",
      "Progress: 71.00%\n",
      "Progress: 71.00%\n",
      "Progress: 71.01%\n",
      "Progress: 71.02%\n",
      "Progress: 71.03%\n",
      "Progress: 71.03%\n",
      "Progress: 71.04%\n",
      "Progress: 71.05%\n",
      "Progress: 71.06%\n",
      "Progress: 71.07%\n",
      "Progress: 71.07%\n",
      "Progress: 71.08%\n",
      "Progress: 71.09%\n",
      "Progress: 71.10%\n",
      "Progress: 71.10%\n",
      "Progress: 71.11%\n",
      "Progress: 71.12%\n",
      "Progress: 71.13%\n",
      "Progress: 71.13%\n",
      "Progress: 71.14%\n",
      "Progress: 71.15%\n",
      "Progress: 71.16%\n",
      "Progress: 71.16%\n",
      "Progress: 71.17%\n",
      "Progress: 71.18%\n",
      "Progress: 71.19%\n",
      "Progress: 71.20%\n",
      "Progress: 71.20%\n",
      "Progress: 71.21%\n",
      "Progress: 71.22%\n",
      "Progress: 71.23%\n",
      "Progress: 71.23%\n",
      "Progress: 71.24%\n",
      "Progress: 71.25%\n",
      "Progress: 71.26%\n",
      "Progress: 71.26%\n",
      "Progress: 71.27%\n",
      "Progress: 71.28%\n",
      "Progress: 71.29%\n",
      "Progress: 71.29%\n",
      "Progress: 71.30%\n",
      "Progress: 71.31%\n",
      "Progress: 71.32%\n",
      "Progress: 71.33%\n",
      "Progress: 71.33%\n",
      "Progress: 71.34%\n",
      "Progress: 71.35%\n",
      "Progress: 71.36%\n",
      "Progress: 71.36%\n",
      "Progress: 71.37%\n",
      "Progress: 71.38%\n",
      "Progress: 71.39%\n",
      "Progress: 71.39%\n",
      "Progress: 71.40%\n",
      "Progress: 71.41%\n",
      "Progress: 71.42%\n",
      "Progress: 71.42%\n",
      "Progress: 71.43%\n",
      "Progress: 71.44%\n",
      "Progress: 71.45%\n",
      "Progress: 71.45%\n",
      "Progress: 71.46%\n",
      "Progress: 71.47%\n",
      "Progress: 71.48%\n",
      "Progress: 71.48%\n",
      "Progress: 71.49%\n",
      "Progress: 71.50%\n",
      "Progress: 71.51%\n",
      "Progress: 71.52%\n",
      "Progress: 71.52%\n",
      "Progress: 71.53%\n",
      "Progress: 71.54%\n",
      "Progress: 71.55%\n",
      "Progress: 71.55%\n",
      "Progress: 71.56%\n",
      "Progress: 71.57%\n",
      "Progress: 71.58%\n",
      "Progress: 71.58%\n",
      "Progress: 71.59%\n",
      "Progress: 71.60%\n",
      "Progress: 71.61%\n",
      "Progress: 71.61%\n",
      "Progress: 71.62%\n",
      "Progress: 71.63%\n",
      "Progress: 71.64%\n",
      "Progress: 71.64%\n",
      "Progress: 71.65%\n",
      "Progress: 71.66%\n",
      "Progress: 71.67%\n",
      "Progress: 71.67%\n",
      "Progress: 71.68%\n",
      "Progress: 71.69%\n",
      "Progress: 71.70%\n",
      "Progress: 71.70%\n",
      "Progress: 71.71%\n",
      "Progress: 71.72%\n",
      "Progress: 71.73%\n",
      "Progress: 71.74%\n",
      "Progress: 71.74%\n",
      "Progress: 71.75%\n",
      "Progress: 71.76%\n",
      "Progress: 71.77%\n",
      "Progress: 71.77%\n",
      "Progress: 71.78%\n",
      "Progress: 71.79%\n",
      "Progress: 71.80%\n",
      "Progress: 71.80%\n",
      "Progress: 71.81%\n",
      "Progress: 71.82%\n",
      "Progress: 71.83%\n",
      "Progress: 71.83%\n",
      "Progress: 71.84%\n",
      "Progress: 71.85%\n",
      "Progress: 71.86%\n",
      "Progress: 71.86%\n",
      "Progress: 71.87%\n",
      "Progress: 71.88%\n",
      "Progress: 71.89%\n",
      "Progress: 71.89%\n",
      "Progress: 71.90%\n",
      "Progress: 71.91%\n",
      "Progress: 71.92%\n",
      "Progress: 71.92%\n",
      "Progress: 71.93%\n",
      "Progress: 71.94%\n",
      "Progress: 71.95%\n",
      "Progress: 71.95%\n",
      "Progress: 71.96%\n",
      "Progress: 71.97%\n",
      "Progress: 71.98%\n",
      "Progress: 71.98%\n",
      "Progress: 71.99%\n",
      "Progress: 72.00%\n",
      "Progress: 72.01%\n",
      "Progress: 72.01%\n",
      "Progress: 72.02%\n",
      "Progress: 72.03%\n",
      "Progress: 72.04%\n",
      "Progress: 72.04%\n",
      "Progress: 72.05%\n",
      "Progress: 72.06%\n",
      "Progress: 72.07%\n",
      "Progress: 72.07%\n",
      "Progress: 72.08%\n",
      "Progress: 72.09%\n",
      "Progress: 72.10%\n",
      "Progress: 72.10%\n",
      "Progress: 72.11%\n",
      "Progress: 72.12%\n",
      "Progress: 72.13%\n",
      "Progress: 72.13%\n",
      "Progress: 72.14%\n",
      "Progress: 72.15%\n",
      "Progress: 72.16%\n",
      "Progress: 72.16%\n",
      "Progress: 72.17%\n",
      "Progress: 72.18%\n",
      "Progress: 72.19%\n",
      "Progress: 72.19%\n",
      "Progress: 72.20%\n",
      "Progress: 72.21%\n",
      "Progress: 72.22%\n",
      "Progress: 72.22%\n",
      "Progress: 72.23%\n",
      "Progress: 72.24%\n",
      "Progress: 72.25%\n",
      "Progress: 72.25%\n",
      "Progress: 72.26%\n",
      "Progress: 72.27%\n",
      "Progress: 72.28%\n",
      "Progress: 72.28%\n",
      "Progress: 72.29%\n",
      "Progress: 72.30%\n",
      "Progress: 72.31%\n",
      "Progress: 72.31%\n",
      "Progress: 72.32%\n",
      "Progress: 72.33%\n",
      "Progress: 72.34%\n",
      "Progress: 72.34%\n",
      "Progress: 72.35%\n",
      "Progress: 72.36%\n",
      "Progress: 72.37%\n",
      "Progress: 72.37%\n",
      "Progress: 72.38%\n",
      "Progress: 72.39%\n",
      "Progress: 72.40%\n",
      "Progress: 72.40%\n",
      "Progress: 72.41%\n",
      "Progress: 72.42%\n",
      "Progress: 72.43%\n",
      "Progress: 72.43%\n",
      "Progress: 72.44%\n",
      "Progress: 72.45%\n",
      "Progress: 72.46%\n",
      "Progress: 72.46%\n",
      "Progress: 72.47%\n",
      "Progress: 72.48%\n",
      "Progress: 72.49%\n",
      "Progress: 72.49%\n",
      "Progress: 72.50%\n",
      "Progress: 72.51%\n",
      "Progress: 72.52%\n",
      "Progress: 72.52%\n",
      "Progress: 72.53%\n",
      "Progress: 72.54%\n",
      "Progress: 72.55%\n",
      "Progress: 72.55%\n",
      "Progress: 72.56%\n",
      "Progress: 72.57%\n",
      "Progress: 72.58%\n",
      "Progress: 72.58%\n",
      "Progress: 72.59%\n",
      "Progress: 72.60%\n",
      "Progress: 72.61%\n",
      "Progress: 72.61%\n",
      "Progress: 72.62%\n",
      "Progress: 72.63%\n",
      "Progress: 72.64%\n",
      "Progress: 72.64%\n",
      "Progress: 72.65%\n",
      "Progress: 72.66%\n",
      "Progress: 72.67%\n",
      "Progress: 72.67%\n",
      "Progress: 72.68%\n",
      "Progress: 72.69%\n",
      "Progress: 72.70%\n",
      "Progress: 72.70%\n",
      "Progress: 72.71%\n",
      "Progress: 72.72%\n",
      "Progress: 72.73%\n",
      "Progress: 72.73%\n",
      "Progress: 72.74%\n",
      "Progress: 72.75%\n",
      "Progress: 72.75%\n",
      "Progress: 72.76%\n",
      "Progress: 72.77%\n",
      "Progress: 72.78%\n",
      "Progress: 72.78%\n",
      "Progress: 72.79%\n",
      "Progress: 72.80%\n",
      "Progress: 72.81%\n",
      "Progress: 72.81%\n",
      "Progress: 72.82%\n",
      "Progress: 72.83%\n",
      "Progress: 72.84%\n",
      "Progress: 72.84%\n",
      "Progress: 72.85%\n",
      "Progress: 72.86%\n",
      "Progress: 72.87%\n",
      "Progress: 72.87%\n",
      "Progress: 72.88%\n",
      "Progress: 72.89%\n",
      "Progress: 72.90%\n",
      "Progress: 72.90%\n",
      "Progress: 72.91%\n",
      "Progress: 72.92%\n",
      "Progress: 72.93%\n",
      "Progress: 72.93%\n",
      "Progress: 72.94%\n",
      "Progress: 72.95%\n",
      "Progress: 72.95%\n",
      "Progress: 72.96%\n",
      "Progress: 72.97%\n",
      "Progress: 72.98%\n",
      "Progress: 72.98%\n",
      "Progress: 72.99%\n",
      "Progress: 73.00%\n",
      "Progress: 73.01%\n",
      "Progress: 73.01%\n",
      "Progress: 73.02%\n",
      "Progress: 73.03%\n",
      "Progress: 73.04%\n",
      "Progress: 73.04%\n",
      "Progress: 73.05%\n",
      "Progress: 73.06%\n",
      "Progress: 73.07%\n",
      "Progress: 73.07%\n",
      "Progress: 73.08%\n",
      "Progress: 73.09%\n",
      "Progress: 73.10%\n",
      "Progress: 73.10%\n",
      "Progress: 73.11%\n",
      "Progress: 73.12%\n",
      "Progress: 73.12%\n",
      "Progress: 73.13%\n",
      "Progress: 73.14%\n",
      "Progress: 73.15%\n",
      "Progress: 73.15%\n",
      "Progress: 73.16%\n",
      "Progress: 73.17%\n",
      "Progress: 73.18%\n",
      "Progress: 73.18%\n",
      "Progress: 73.19%\n",
      "Progress: 73.20%\n",
      "Progress: 73.21%\n",
      "Progress: 73.21%\n",
      "Progress: 73.22%\n",
      "Progress: 73.23%\n",
      "Progress: 73.24%\n",
      "Progress: 73.24%\n",
      "Progress: 73.25%\n",
      "Progress: 73.26%\n",
      "Progress: 73.26%\n",
      "Progress: 73.27%\n",
      "Progress: 73.28%\n",
      "Progress: 73.29%\n",
      "Progress: 73.29%\n",
      "Progress: 73.30%\n",
      "Progress: 73.31%\n",
      "Progress: 73.32%\n",
      "Progress: 73.32%\n",
      "Progress: 73.33%\n",
      "Progress: 73.34%\n",
      "Progress: 73.35%\n",
      "Progress: 73.35%\n",
      "Progress: 73.36%\n",
      "Progress: 73.37%\n",
      "Progress: 73.38%\n",
      "Progress: 73.38%\n",
      "Progress: 73.39%\n",
      "Progress: 73.40%\n",
      "Progress: 73.40%\n",
      "Progress: 73.41%\n",
      "Progress: 73.42%\n",
      "Progress: 73.43%\n",
      "Progress: 73.43%\n",
      "Progress: 73.44%\n",
      "Progress: 73.45%\n",
      "Progress: 73.46%\n",
      "Progress: 73.46%\n",
      "Progress: 73.47%\n",
      "Progress: 73.48%\n",
      "Progress: 73.49%\n",
      "Progress: 73.49%\n",
      "Progress: 73.50%\n",
      "Progress: 73.51%\n",
      "Progress: 73.51%\n",
      "Progress: 73.52%\n",
      "Progress: 73.53%\n",
      "Progress: 73.54%\n",
      "Progress: 73.54%\n",
      "Progress: 73.55%\n",
      "Progress: 73.56%\n",
      "Progress: 73.57%\n",
      "Progress: 73.57%\n",
      "Progress: 73.58%\n",
      "Progress: 73.59%\n",
      "Progress: 73.60%\n",
      "Progress: 73.60%\n",
      "Progress: 73.61%\n",
      "Progress: 73.62%\n",
      "Progress: 73.62%\n",
      "Progress: 73.63%\n",
      "Progress: 73.64%\n",
      "Progress: 73.65%\n",
      "Progress: 73.65%\n",
      "Progress: 73.66%\n",
      "Progress: 73.67%\n",
      "Progress: 73.68%\n",
      "Progress: 73.68%\n",
      "Progress: 73.69%\n",
      "Progress: 73.70%\n",
      "Progress: 73.70%\n",
      "Progress: 73.71%\n",
      "Progress: 73.72%\n",
      "Progress: 73.73%\n",
      "Progress: 73.73%\n",
      "Progress: 73.74%\n",
      "Progress: 73.75%\n",
      "Progress: 73.76%\n",
      "Progress: 73.76%\n",
      "Progress: 73.77%\n",
      "Progress: 73.78%\n",
      "Progress: 73.78%\n",
      "Progress: 73.79%\n",
      "Progress: 73.80%\n",
      "Progress: 73.81%\n",
      "Progress: 73.81%\n",
      "Progress: 73.82%\n",
      "Progress: 73.83%\n",
      "Progress: 73.84%\n",
      "Progress: 73.84%\n",
      "Progress: 73.85%\n",
      "Progress: 73.86%\n",
      "Progress: 73.86%\n",
      "Progress: 73.87%\n",
      "Progress: 73.88%\n",
      "Progress: 73.89%\n",
      "Progress: 73.89%\n",
      "Progress: 73.90%\n",
      "Progress: 73.91%\n",
      "Progress: 73.92%\n",
      "Progress: 73.92%\n",
      "Progress: 73.93%\n",
      "Progress: 73.94%\n",
      "Progress: 73.94%\n",
      "Progress: 73.95%\n",
      "Progress: 73.96%\n",
      "Progress: 73.97%\n",
      "Progress: 73.97%\n",
      "Progress: 73.98%\n",
      "Progress: 73.99%\n",
      "Progress: 74.00%\n",
      "Progress: 74.00%\n",
      "Progress: 74.01%\n",
      "Progress: 74.02%\n",
      "Progress: 74.02%\n",
      "Progress: 74.03%\n",
      "Progress: 74.04%\n",
      "Progress: 74.05%\n",
      "Progress: 74.05%\n",
      "Progress: 74.06%\n",
      "Progress: 74.07%\n",
      "Progress: 74.08%\n",
      "Progress: 74.08%\n",
      "Progress: 74.09%\n",
      "Progress: 74.10%\n",
      "Progress: 74.10%\n",
      "Progress: 74.11%\n",
      "Progress: 74.12%\n",
      "Progress: 74.13%\n",
      "Progress: 74.13%\n",
      "Progress: 74.14%\n",
      "Progress: 74.15%\n",
      "Progress: 74.16%\n",
      "Progress: 74.16%\n",
      "Progress: 74.17%\n",
      "Progress: 74.18%\n",
      "Progress: 74.18%\n",
      "Progress: 74.19%\n",
      "Progress: 74.20%\n",
      "Progress: 74.21%\n",
      "Progress: 74.21%\n",
      "Progress: 74.22%\n",
      "Progress: 74.23%\n",
      "Progress: 74.23%\n",
      "Progress: 74.24%\n",
      "Progress: 74.25%\n",
      "Progress: 74.26%\n",
      "Progress: 74.26%\n",
      "Progress: 74.27%\n",
      "Progress: 74.28%\n",
      "Progress: 74.29%\n",
      "Progress: 74.29%\n",
      "Progress: 74.30%\n",
      "Progress: 74.31%\n",
      "Progress: 74.31%\n",
      "Progress: 74.32%\n",
      "Progress: 74.33%\n",
      "Progress: 74.34%\n",
      "Progress: 74.34%\n",
      "Progress: 74.35%\n",
      "Progress: 74.36%\n",
      "Progress: 74.36%\n",
      "Progress: 74.37%\n",
      "Progress: 74.38%\n",
      "Progress: 74.39%\n",
      "Progress: 74.39%\n",
      "Progress: 74.40%\n",
      "Progress: 74.41%\n",
      "Progress: 74.41%\n",
      "Progress: 74.42%\n",
      "Progress: 74.43%\n",
      "Progress: 74.44%\n",
      "Progress: 74.44%\n",
      "Progress: 74.45%\n",
      "Progress: 74.46%\n",
      "Progress: 74.47%\n",
      "Progress: 74.47%\n",
      "Progress: 74.48%\n",
      "Progress: 74.49%\n",
      "Progress: 74.49%\n",
      "Progress: 74.50%\n",
      "Progress: 74.51%\n",
      "Progress: 74.52%\n",
      "Progress: 74.52%\n",
      "Progress: 74.53%\n",
      "Progress: 74.54%\n",
      "Progress: 74.54%\n",
      "Progress: 74.55%\n",
      "Progress: 74.56%\n",
      "Progress: 74.57%\n",
      "Progress: 74.57%\n",
      "Progress: 74.58%\n",
      "Progress: 74.59%\n",
      "Progress: 74.59%\n",
      "Progress: 74.60%\n",
      "Progress: 74.61%\n",
      "Progress: 74.62%\n",
      "Progress: 74.62%\n",
      "Progress: 74.63%\n",
      "Progress: 74.64%\n",
      "Progress: 74.64%\n",
      "Progress: 74.65%\n",
      "Progress: 74.66%\n",
      "Progress: 74.67%\n",
      "Progress: 74.67%\n",
      "Progress: 74.68%\n",
      "Progress: 74.69%\n",
      "Progress: 74.69%\n",
      "Progress: 74.70%\n",
      "Progress: 74.71%\n",
      "Progress: 74.72%\n",
      "Progress: 74.72%\n",
      "Progress: 74.73%\n",
      "Progress: 74.74%\n",
      "Progress: 74.74%\n",
      "Progress: 74.75%\n",
      "Progress: 74.76%\n",
      "Progress: 74.77%\n",
      "Progress: 74.77%\n",
      "Progress: 74.78%\n",
      "Progress: 74.79%\n",
      "Progress: 74.80%\n",
      "Progress: 74.80%\n",
      "Progress: 74.81%\n",
      "Progress: 74.82%\n",
      "Progress: 74.82%\n",
      "Progress: 74.83%\n",
      "Progress: 74.84%\n",
      "Progress: 74.84%\n",
      "Progress: 74.85%\n",
      "Progress: 74.86%\n",
      "Progress: 74.87%\n",
      "Progress: 74.87%\n",
      "Progress: 74.88%\n",
      "Progress: 74.89%\n",
      "Progress: 74.89%\n",
      "Progress: 74.90%\n",
      "Progress: 74.91%\n",
      "Progress: 74.92%\n",
      "Progress: 74.92%\n",
      "Progress: 74.93%\n",
      "Progress: 74.94%\n",
      "Progress: 74.94%\n",
      "Progress: 74.95%\n",
      "Progress: 74.96%\n",
      "Progress: 74.97%\n",
      "Progress: 74.97%\n",
      "Progress: 74.98%\n",
      "Progress: 74.99%\n",
      "Progress: 74.99%\n",
      "Progress: 75.00%\n",
      "Progress: 75.01%\n",
      "Progress: 75.02%\n",
      "Progress: 75.02%\n",
      "Progress: 75.03%\n",
      "Progress: 75.04%\n",
      "Progress: 75.04%\n",
      "Progress: 75.05%\n",
      "Progress: 75.06%\n",
      "Progress: 75.07%\n",
      "Progress: 75.07%\n",
      "Progress: 75.08%\n",
      "Progress: 75.09%\n",
      "Progress: 75.09%\n",
      "Progress: 75.10%\n",
      "Progress: 75.11%\n",
      "Progress: 75.12%\n",
      "Progress: 75.12%\n",
      "Progress: 75.13%\n",
      "Progress: 75.14%\n",
      "Progress: 75.14%\n",
      "Progress: 75.15%\n",
      "Progress: 75.16%\n",
      "Progress: 75.17%\n",
      "Progress: 75.17%\n",
      "Progress: 75.18%\n",
      "Progress: 75.19%\n",
      "Progress: 75.19%\n",
      "Progress: 75.20%\n",
      "Progress: 75.21%\n",
      "Progress: 75.21%\n",
      "Progress: 75.22%\n",
      "Progress: 75.23%\n",
      "Progress: 75.24%\n",
      "Progress: 75.24%\n",
      "Progress: 75.25%\n",
      "Progress: 75.26%\n",
      "Progress: 75.26%\n",
      "Progress: 75.27%\n",
      "Progress: 75.28%\n",
      "Progress: 75.29%\n",
      "Progress: 75.29%\n",
      "Progress: 75.30%\n",
      "Progress: 75.31%\n",
      "Progress: 75.31%\n",
      "Progress: 75.32%\n",
      "Progress: 75.33%\n",
      "Progress: 75.34%\n",
      "Progress: 75.34%\n",
      "Progress: 75.35%\n",
      "Progress: 75.36%\n",
      "Progress: 75.36%\n",
      "Progress: 75.37%\n",
      "Progress: 75.38%\n",
      "Progress: 75.38%\n",
      "Progress: 75.39%\n",
      "Progress: 75.40%\n",
      "Progress: 75.41%\n",
      "Progress: 75.41%\n",
      "Progress: 75.42%\n",
      "Progress: 75.43%\n",
      "Progress: 75.43%\n",
      "Progress: 75.44%\n",
      "Progress: 75.45%\n",
      "Progress: 75.46%\n",
      "Progress: 75.46%\n",
      "Progress: 75.47%\n",
      "Progress: 75.48%\n",
      "Progress: 75.48%\n",
      "Progress: 75.49%\n",
      "Progress: 75.50%\n",
      "Progress: 75.50%\n",
      "Progress: 75.51%\n",
      "Progress: 75.52%\n",
      "Progress: 75.53%\n",
      "Progress: 75.53%\n",
      "Progress: 75.54%\n",
      "Progress: 75.55%\n",
      "Progress: 75.55%\n",
      "Progress: 75.56%\n",
      "Progress: 75.57%\n",
      "Progress: 75.57%\n",
      "Progress: 75.58%\n",
      "Progress: 75.59%\n",
      "Progress: 75.60%\n",
      "Progress: 75.60%\n",
      "Progress: 75.61%\n",
      "Progress: 75.62%\n",
      "Progress: 75.62%\n",
      "Progress: 75.63%\n",
      "Progress: 75.64%\n",
      "Progress: 75.65%\n",
      "Progress: 75.65%\n",
      "Progress: 75.66%\n",
      "Progress: 75.67%\n",
      "Progress: 75.67%\n",
      "Progress: 75.68%\n",
      "Progress: 75.69%\n",
      "Progress: 75.69%\n",
      "Progress: 75.70%\n",
      "Progress: 75.71%\n",
      "Progress: 75.72%\n",
      "Progress: 75.72%\n",
      "Progress: 75.73%\n",
      "Progress: 75.74%\n",
      "Progress: 75.74%\n",
      "Progress: 75.75%\n",
      "Progress: 75.76%\n",
      "Progress: 75.76%\n",
      "Progress: 75.77%\n",
      "Progress: 75.78%\n",
      "Progress: 75.79%\n",
      "Progress: 75.79%\n",
      "Progress: 75.80%\n",
      "Progress: 75.81%\n",
      "Progress: 75.81%\n",
      "Progress: 75.82%\n",
      "Progress: 75.83%\n",
      "Progress: 75.83%\n",
      "Progress: 75.84%\n",
      "Progress: 75.85%\n",
      "Progress: 75.86%\n",
      "Progress: 75.86%\n",
      "Progress: 75.87%\n",
      "Progress: 75.88%\n",
      "Progress: 75.88%\n",
      "Progress: 75.89%\n",
      "Progress: 75.90%\n",
      "Progress: 75.90%\n",
      "Progress: 75.91%\n",
      "Progress: 75.92%\n",
      "Progress: 75.93%\n",
      "Progress: 75.93%\n",
      "Progress: 75.94%\n",
      "Progress: 75.95%\n",
      "Progress: 75.95%\n",
      "Progress: 75.96%\n",
      "Progress: 75.97%\n",
      "Progress: 75.97%\n",
      "Progress: 75.98%\n",
      "Progress: 75.99%\n",
      "Progress: 75.99%\n",
      "Progress: 76.00%\n",
      "Progress: 76.01%\n",
      "Progress: 76.02%\n",
      "Progress: 76.02%\n",
      "Progress: 76.03%\n",
      "Progress: 76.04%\n",
      "Progress: 76.04%\n",
      "Progress: 76.05%\n",
      "Progress: 76.06%\n",
      "Progress: 76.06%\n",
      "Progress: 76.07%\n",
      "Progress: 76.08%\n",
      "Progress: 76.09%\n",
      "Progress: 76.09%\n",
      "Progress: 76.10%\n",
      "Progress: 76.11%\n",
      "Progress: 76.11%\n",
      "Progress: 76.12%\n",
      "Progress: 76.13%\n",
      "Progress: 76.13%\n",
      "Progress: 76.14%\n",
      "Progress: 76.15%\n",
      "Progress: 76.16%\n",
      "Progress: 76.16%\n",
      "Progress: 76.17%\n",
      "Progress: 76.18%\n",
      "Progress: 76.18%\n",
      "Progress: 76.19%\n",
      "Progress: 76.20%\n",
      "Progress: 76.20%\n",
      "Progress: 76.21%\n",
      "Progress: 76.22%\n",
      "Progress: 76.22%\n",
      "Progress: 76.23%\n",
      "Progress: 76.24%\n",
      "Progress: 76.25%\n",
      "Progress: 76.25%\n",
      "Progress: 76.26%\n",
      "Progress: 76.27%\n",
      "Progress: 76.27%\n",
      "Progress: 76.28%\n",
      "Progress: 76.29%\n",
      "Progress: 76.29%\n",
      "Progress: 76.30%\n",
      "Progress: 76.31%\n",
      "Progress: 76.31%\n",
      "Progress: 76.32%\n",
      "Progress: 76.33%\n",
      "Progress: 76.34%\n",
      "Progress: 76.34%\n",
      "Progress: 76.35%\n",
      "Progress: 76.36%\n",
      "Progress: 76.36%\n",
      "Progress: 76.37%\n",
      "Progress: 76.38%\n",
      "Progress: 76.38%\n",
      "Progress: 76.39%\n",
      "Progress: 76.40%\n",
      "Progress: 76.40%\n",
      "Progress: 76.41%\n",
      "Progress: 76.42%\n",
      "Progress: 76.43%\n",
      "Progress: 76.43%\n",
      "Progress: 76.44%\n",
      "Progress: 76.45%\n",
      "Progress: 76.45%\n",
      "Progress: 76.46%\n",
      "Progress: 76.47%\n",
      "Progress: 76.47%\n",
      "Progress: 76.48%\n",
      "Progress: 76.49%\n",
      "Progress: 76.49%\n",
      "Progress: 76.50%\n",
      "Progress: 76.51%\n",
      "Progress: 76.52%\n",
      "Progress: 76.52%\n",
      "Progress: 76.53%\n",
      "Progress: 76.54%\n",
      "Progress: 76.54%\n",
      "Progress: 76.55%\n",
      "Progress: 76.56%\n",
      "Progress: 76.56%\n",
      "Progress: 76.57%\n",
      "Progress: 76.58%\n",
      "Progress: 76.58%\n",
      "Progress: 76.59%\n",
      "Progress: 76.60%\n",
      "Progress: 76.60%\n",
      "Progress: 76.61%\n",
      "Progress: 76.62%\n",
      "Progress: 76.63%\n",
      "Progress: 76.63%\n",
      "Progress: 76.64%\n",
      "Progress: 76.65%\n",
      "Progress: 76.65%\n",
      "Progress: 76.66%\n",
      "Progress: 76.67%\n",
      "Progress: 76.67%\n",
      "Progress: 76.68%\n",
      "Progress: 76.69%\n",
      "Progress: 76.69%\n",
      "Progress: 76.70%\n",
      "Progress: 76.71%\n",
      "Progress: 76.71%\n",
      "Progress: 76.72%\n",
      "Progress: 76.73%\n",
      "Progress: 76.74%\n",
      "Progress: 76.74%\n",
      "Progress: 76.75%\n",
      "Progress: 76.76%\n",
      "Progress: 76.76%\n",
      "Progress: 76.77%\n",
      "Progress: 76.78%\n",
      "Progress: 76.78%\n",
      "Progress: 76.79%\n",
      "Progress: 76.80%\n",
      "Progress: 76.80%\n",
      "Progress: 76.81%\n",
      "Progress: 76.82%\n",
      "Progress: 76.82%\n",
      "Progress: 76.83%\n",
      "Progress: 76.84%\n",
      "Progress: 76.84%\n",
      "Progress: 76.85%\n",
      "Progress: 76.86%\n",
      "Progress: 76.87%\n",
      "Progress: 76.87%\n",
      "Progress: 76.88%\n",
      "Progress: 76.89%\n",
      "Progress: 76.89%\n",
      "Progress: 76.90%\n",
      "Progress: 76.91%\n",
      "Progress: 76.91%\n",
      "Progress: 76.92%\n",
      "Progress: 76.93%\n",
      "Progress: 76.93%\n",
      "Progress: 76.94%\n",
      "Progress: 76.95%\n",
      "Progress: 76.95%\n",
      "Progress: 76.96%\n",
      "Progress: 76.97%\n",
      "Progress: 76.97%\n",
      "Progress: 76.98%\n",
      "Progress: 76.99%\n",
      "Progress: 77.00%\n",
      "Progress: 77.00%\n",
      "Progress: 77.01%\n",
      "Progress: 77.02%\n",
      "Progress: 77.02%\n",
      "Progress: 77.03%\n",
      "Progress: 77.04%\n",
      "Progress: 77.04%\n",
      "Progress: 77.05%\n",
      "Progress: 77.06%\n",
      "Progress: 77.06%\n",
      "Progress: 77.07%\n",
      "Progress: 77.08%\n",
      "Progress: 77.08%\n",
      "Progress: 77.09%\n",
      "Progress: 77.10%\n",
      "Progress: 77.10%\n",
      "Progress: 77.11%\n",
      "Progress: 77.12%\n",
      "Progress: 77.12%\n",
      "Progress: 77.13%\n",
      "Progress: 77.14%\n",
      "Progress: 77.15%\n",
      "Progress: 77.15%\n",
      "Progress: 77.16%\n",
      "Progress: 77.17%\n",
      "Progress: 77.17%\n",
      "Progress: 77.18%\n",
      "Progress: 77.19%\n",
      "Progress: 77.19%\n",
      "Progress: 77.20%\n",
      "Progress: 77.21%\n",
      "Progress: 77.21%\n",
      "Progress: 77.22%\n",
      "Progress: 77.23%\n",
      "Progress: 77.23%\n",
      "Progress: 77.24%\n",
      "Progress: 77.25%\n",
      "Progress: 77.25%\n",
      "Progress: 77.26%\n",
      "Progress: 77.27%\n",
      "Progress: 77.27%\n",
      "Progress: 77.28%\n",
      "Progress: 77.29%\n",
      "Progress: 77.29%\n",
      "Progress: 77.30%\n",
      "Progress: 77.31%\n",
      "Progress: 77.32%\n",
      "Progress: 77.32%\n",
      "Progress: 77.33%\n",
      "Progress: 77.34%\n",
      "Progress: 77.34%\n",
      "Progress: 77.35%\n",
      "Progress: 77.36%\n",
      "Progress: 77.36%\n",
      "Progress: 77.37%\n",
      "Progress: 77.38%\n",
      "Progress: 77.38%\n",
      "Progress: 77.39%\n",
      "Progress: 77.40%\n",
      "Progress: 77.40%\n",
      "Progress: 77.41%\n",
      "Progress: 77.42%\n",
      "Progress: 77.42%\n",
      "Progress: 77.43%\n",
      "Progress: 77.44%\n",
      "Progress: 77.44%\n",
      "Progress: 77.45%\n",
      "Progress: 77.46%\n",
      "Progress: 77.46%\n",
      "Progress: 77.47%\n",
      "Progress: 77.48%\n",
      "Progress: 77.48%\n",
      "Progress: 77.49%\n",
      "Progress: 77.50%\n",
      "Progress: 77.50%\n",
      "Progress: 77.51%\n",
      "Progress: 77.52%\n",
      "Progress: 77.52%\n",
      "Progress: 77.53%\n",
      "Progress: 77.54%\n",
      "Progress: 77.54%\n",
      "Progress: 77.55%\n",
      "Progress: 77.56%\n",
      "Progress: 77.57%\n",
      "Progress: 77.57%\n",
      "Progress: 77.58%\n",
      "Progress: 77.59%\n",
      "Progress: 77.59%\n",
      "Progress: 77.60%\n",
      "Progress: 77.61%\n",
      "Progress: 77.61%\n",
      "Progress: 77.62%\n",
      "Progress: 77.63%\n",
      "Progress: 77.63%\n",
      "Progress: 77.64%\n",
      "Progress: 77.65%\n",
      "Progress: 77.65%\n",
      "Progress: 77.66%\n",
      "Progress: 77.67%\n",
      "Progress: 77.67%\n",
      "Progress: 77.68%\n",
      "Progress: 77.69%\n",
      "Progress: 77.69%\n",
      "Progress: 77.70%\n",
      "Progress: 77.71%\n",
      "Progress: 77.71%\n",
      "Progress: 77.72%\n",
      "Progress: 77.73%\n",
      "Progress: 77.73%\n",
      "Progress: 77.74%\n",
      "Progress: 77.75%\n",
      "Progress: 77.75%\n",
      "Progress: 77.76%\n",
      "Progress: 77.77%\n",
      "Progress: 77.77%\n",
      "Progress: 77.78%\n",
      "Progress: 77.79%\n",
      "Progress: 77.79%\n",
      "Progress: 77.80%\n",
      "Progress: 77.81%\n",
      "Progress: 77.81%\n",
      "Progress: 77.82%\n",
      "Progress: 77.83%\n",
      "Progress: 77.83%\n",
      "Progress: 77.84%\n",
      "Progress: 77.85%\n",
      "Progress: 77.85%\n",
      "Progress: 77.86%\n",
      "Progress: 77.87%\n",
      "Progress: 77.87%\n",
      "Progress: 77.88%\n",
      "Progress: 77.89%\n",
      "Progress: 77.89%\n",
      "Progress: 77.90%\n",
      "Progress: 77.91%\n",
      "Progress: 77.91%\n",
      "Progress: 77.92%\n",
      "Progress: 77.93%\n",
      "Progress: 77.93%\n",
      "Progress: 77.94%\n",
      "Progress: 77.95%\n",
      "Progress: 77.95%\n",
      "Progress: 77.96%\n",
      "Progress: 77.97%\n",
      "Progress: 77.97%\n",
      "Progress: 77.98%\n",
      "Progress: 77.99%\n",
      "Progress: 77.99%\n",
      "Progress: 78.00%\n",
      "Progress: 78.01%\n",
      "Progress: 78.01%\n",
      "Progress: 78.02%\n",
      "Progress: 78.03%\n",
      "Progress: 78.03%\n",
      "Progress: 78.04%\n",
      "Progress: 78.05%\n",
      "Progress: 78.05%\n",
      "Progress: 78.06%\n",
      "Progress: 78.07%\n",
      "Progress: 78.07%\n",
      "Progress: 78.08%\n",
      "Progress: 78.09%\n",
      "Progress: 78.09%\n",
      "Progress: 78.10%\n",
      "Progress: 78.11%\n",
      "Progress: 78.11%\n",
      "Progress: 78.12%\n",
      "Progress: 78.13%\n",
      "Progress: 78.13%\n",
      "Progress: 78.14%\n",
      "Progress: 78.15%\n",
      "Progress: 78.15%\n",
      "Progress: 78.16%\n",
      "Progress: 78.17%\n",
      "Progress: 78.17%\n",
      "Progress: 78.18%\n",
      "Progress: 78.19%\n",
      "Progress: 78.19%\n",
      "Progress: 78.20%\n",
      "Progress: 78.21%\n",
      "Progress: 78.21%\n",
      "Progress: 78.22%\n",
      "Progress: 78.23%\n",
      "Progress: 78.23%\n",
      "Progress: 78.24%\n",
      "Progress: 78.25%\n",
      "Progress: 78.25%\n",
      "Progress: 78.26%\n",
      "Progress: 78.27%\n",
      "Progress: 78.27%\n",
      "Progress: 78.28%\n",
      "Progress: 78.29%\n",
      "Progress: 78.29%\n",
      "Progress: 78.30%\n",
      "Progress: 78.31%\n",
      "Progress: 78.31%\n",
      "Progress: 78.32%\n",
      "Progress: 78.33%\n",
      "Progress: 78.33%\n",
      "Progress: 78.34%\n",
      "Progress: 78.35%\n",
      "Progress: 78.35%\n",
      "Progress: 78.36%\n",
      "Progress: 78.37%\n",
      "Progress: 78.37%\n",
      "Progress: 78.38%\n",
      "Progress: 78.39%\n",
      "Progress: 78.39%\n",
      "Progress: 78.40%\n",
      "Progress: 78.41%\n",
      "Progress: 78.41%\n",
      "Progress: 78.42%\n",
      "Progress: 78.43%\n",
      "Progress: 78.43%\n",
      "Progress: 78.44%\n",
      "Progress: 78.45%\n",
      "Progress: 78.45%\n",
      "Progress: 78.46%\n",
      "Progress: 78.47%\n",
      "Progress: 78.47%\n",
      "Progress: 78.48%\n",
      "Progress: 78.49%\n",
      "Progress: 78.49%\n",
      "Progress: 78.50%\n",
      "Progress: 78.51%\n",
      "Progress: 78.51%\n",
      "Progress: 78.52%\n",
      "Progress: 78.53%\n",
      "Progress: 78.53%\n",
      "Progress: 78.54%\n",
      "Progress: 78.55%\n",
      "Progress: 78.55%\n",
      "Progress: 78.56%\n",
      "Progress: 78.56%\n",
      "Progress: 78.57%\n",
      "Progress: 78.58%\n",
      "Progress: 78.58%\n",
      "Progress: 78.59%\n",
      "Progress: 78.60%\n",
      "Progress: 78.60%\n",
      "Progress: 78.61%\n",
      "Progress: 78.62%\n",
      "Progress: 78.62%\n",
      "Progress: 78.63%\n",
      "Progress: 78.64%\n",
      "Progress: 78.64%\n",
      "Progress: 78.65%\n",
      "Progress: 78.66%\n",
      "Progress: 78.66%\n",
      "Progress: 78.67%\n",
      "Progress: 78.68%\n",
      "Progress: 78.68%\n",
      "Progress: 78.69%\n",
      "Progress: 78.70%\n",
      "Progress: 78.70%\n",
      "Progress: 78.71%\n",
      "Progress: 78.72%\n",
      "Progress: 78.72%\n",
      "Progress: 78.73%\n",
      "Progress: 78.74%\n",
      "Progress: 78.74%\n",
      "Progress: 78.75%\n",
      "Progress: 78.76%\n",
      "Progress: 78.76%\n",
      "Progress: 78.77%\n",
      "Progress: 78.78%\n",
      "Progress: 78.78%\n",
      "Progress: 78.79%\n",
      "Progress: 78.79%\n",
      "Progress: 78.80%\n",
      "Progress: 78.81%\n",
      "Progress: 78.81%\n",
      "Progress: 78.82%\n",
      "Progress: 78.83%\n",
      "Progress: 78.83%\n",
      "Progress: 78.84%\n",
      "Progress: 78.85%\n",
      "Progress: 78.85%\n",
      "Progress: 78.86%\n",
      "Progress: 78.87%\n",
      "Progress: 78.87%\n",
      "Progress: 78.88%\n",
      "Progress: 78.89%\n",
      "Progress: 78.89%\n",
      "Progress: 78.90%\n",
      "Progress: 78.91%\n",
      "Progress: 78.91%\n",
      "Progress: 78.92%\n",
      "Progress: 78.93%\n",
      "Progress: 78.93%\n",
      "Progress: 78.94%\n",
      "Progress: 78.95%\n",
      "Progress: 78.95%\n",
      "Progress: 78.96%\n",
      "Progress: 78.96%\n",
      "Progress: 78.97%\n",
      "Progress: 78.98%\n",
      "Progress: 78.98%\n",
      "Progress: 78.99%\n",
      "Progress: 79.00%\n",
      "Progress: 79.00%\n",
      "Progress: 79.01%\n",
      "Progress: 79.02%\n",
      "Progress: 79.02%\n",
      "Progress: 79.03%\n",
      "Progress: 79.04%\n",
      "Progress: 79.04%\n",
      "Progress: 79.05%\n",
      "Progress: 79.06%\n",
      "Progress: 79.06%\n",
      "Progress: 79.07%\n",
      "Progress: 79.08%\n",
      "Progress: 79.08%\n",
      "Progress: 79.09%\n",
      "Progress: 79.10%\n",
      "Progress: 79.10%\n",
      "Progress: 79.11%\n",
      "Progress: 79.11%\n",
      "Progress: 79.12%\n",
      "Progress: 79.13%\n",
      "Progress: 79.13%\n",
      "Progress: 79.14%\n",
      "Progress: 79.15%\n",
      "Progress: 79.15%\n",
      "Progress: 79.16%\n",
      "Progress: 79.17%\n",
      "Progress: 79.17%\n",
      "Progress: 79.18%\n",
      "Progress: 79.19%\n",
      "Progress: 79.19%\n",
      "Progress: 79.20%\n",
      "Progress: 79.21%\n",
      "Progress: 79.21%\n",
      "Progress: 79.22%\n",
      "Progress: 79.23%\n",
      "Progress: 79.23%\n",
      "Progress: 79.24%\n",
      "Progress: 79.24%\n",
      "Progress: 79.25%\n",
      "Progress: 79.26%\n",
      "Progress: 79.26%\n",
      "Progress: 79.27%\n",
      "Progress: 79.28%\n",
      "Progress: 79.28%\n",
      "Progress: 79.29%\n",
      "Progress: 79.30%\n",
      "Progress: 79.30%\n",
      "Progress: 79.31%\n",
      "Progress: 79.32%\n",
      "Progress: 79.32%\n",
      "Progress: 79.33%\n",
      "Progress: 79.34%\n",
      "Progress: 79.34%\n",
      "Progress: 79.35%\n",
      "Progress: 79.35%\n",
      "Progress: 79.36%\n",
      "Progress: 79.37%\n",
      "Progress: 79.37%\n",
      "Progress: 79.38%\n",
      "Progress: 79.39%\n",
      "Progress: 79.39%\n",
      "Progress: 79.40%\n",
      "Progress: 79.41%\n",
      "Progress: 79.41%\n",
      "Progress: 79.42%\n",
      "Progress: 79.43%\n",
      "Progress: 79.43%\n",
      "Progress: 79.44%\n",
      "Progress: 79.45%\n",
      "Progress: 79.45%\n",
      "Progress: 79.46%\n",
      "Progress: 79.46%\n",
      "Progress: 79.47%\n",
      "Progress: 79.48%\n",
      "Progress: 79.48%\n",
      "Progress: 79.49%\n",
      "Progress: 79.50%\n",
      "Progress: 79.50%\n",
      "Progress: 79.51%\n",
      "Progress: 79.52%\n",
      "Progress: 79.52%\n",
      "Progress: 79.53%\n",
      "Progress: 79.54%\n",
      "Progress: 79.54%\n",
      "Progress: 79.55%\n",
      "Progress: 79.55%\n",
      "Progress: 79.56%\n",
      "Progress: 79.57%\n",
      "Progress: 79.57%\n",
      "Progress: 79.58%\n",
      "Progress: 79.59%\n",
      "Progress: 79.59%\n",
      "Progress: 79.60%\n",
      "Progress: 79.61%\n",
      "Progress: 79.61%\n",
      "Progress: 79.62%\n",
      "Progress: 79.63%\n",
      "Progress: 79.63%\n",
      "Progress: 79.64%\n",
      "Progress: 79.64%\n",
      "Progress: 79.65%\n",
      "Progress: 79.66%\n",
      "Progress: 79.66%\n",
      "Progress: 79.67%\n",
      "Progress: 79.68%\n",
      "Progress: 79.68%\n",
      "Progress: 79.69%\n",
      "Progress: 79.70%\n",
      "Progress: 79.70%\n",
      "Progress: 79.71%\n",
      "Progress: 79.72%\n",
      "Progress: 79.72%\n",
      "Progress: 79.73%\n",
      "Progress: 79.73%\n",
      "Progress: 79.74%\n",
      "Progress: 79.75%\n",
      "Progress: 79.75%\n",
      "Progress: 79.76%\n",
      "Progress: 79.77%\n",
      "Progress: 79.77%\n",
      "Progress: 79.78%\n",
      "Progress: 79.79%\n",
      "Progress: 79.79%\n",
      "Progress: 79.80%\n",
      "Progress: 79.80%\n",
      "Progress: 79.81%\n",
      "Progress: 79.82%\n",
      "Progress: 79.82%\n",
      "Progress: 79.83%\n",
      "Progress: 79.84%\n",
      "Progress: 79.84%\n",
      "Progress: 79.85%\n",
      "Progress: 79.86%\n",
      "Progress: 79.86%\n",
      "Progress: 79.87%\n",
      "Progress: 79.88%\n",
      "Progress: 79.88%\n",
      "Progress: 79.89%\n",
      "Progress: 79.89%\n",
      "Progress: 79.90%\n",
      "Progress: 79.91%\n",
      "Progress: 79.91%\n",
      "Progress: 79.92%\n",
      "Progress: 79.93%\n",
      "Progress: 79.93%\n",
      "Progress: 79.94%\n",
      "Progress: 79.95%\n",
      "Progress: 79.95%\n",
      "Progress: 79.96%\n",
      "Progress: 79.96%\n",
      "Progress: 79.97%\n",
      "Progress: 79.98%\n",
      "Progress: 79.98%\n",
      "Progress: 79.99%\n",
      "Progress: 80.00%\n",
      "Progress: 80.00%\n",
      "Progress: 80.01%\n",
      "Progress: 80.02%\n",
      "Progress: 80.02%\n",
      "Progress: 80.03%\n",
      "Progress: 80.03%\n",
      "Progress: 80.04%\n",
      "Progress: 80.05%\n",
      "Progress: 80.05%\n",
      "Progress: 80.06%\n",
      "Progress: 80.07%\n",
      "Progress: 80.07%\n",
      "Progress: 80.08%\n",
      "Progress: 80.09%\n",
      "Progress: 80.09%\n",
      "Progress: 80.10%\n",
      "Progress: 80.10%\n",
      "Progress: 80.11%\n",
      "Progress: 80.12%\n",
      "Progress: 80.12%\n",
      "Progress: 80.13%\n",
      "Progress: 80.14%\n",
      "Progress: 80.14%\n",
      "Progress: 80.15%\n",
      "Progress: 80.16%\n",
      "Progress: 80.16%\n",
      "Progress: 80.17%\n",
      "Progress: 80.17%\n",
      "Progress: 80.18%\n",
      "Progress: 80.19%\n",
      "Progress: 80.19%\n",
      "Progress: 80.20%\n",
      "Progress: 80.21%\n",
      "Progress: 80.21%\n",
      "Progress: 80.22%\n",
      "Progress: 80.22%\n",
      "Progress: 80.23%\n",
      "Progress: 80.24%\n",
      "Progress: 80.24%\n",
      "Progress: 80.25%\n",
      "Progress: 80.26%\n",
      "Progress: 80.26%\n",
      "Progress: 80.27%\n",
      "Progress: 80.28%\n",
      "Progress: 80.28%\n",
      "Progress: 80.29%\n",
      "Progress: 80.29%\n",
      "Progress: 80.30%\n",
      "Progress: 80.31%\n",
      "Progress: 80.31%\n",
      "Progress: 80.32%\n",
      "Progress: 80.33%\n",
      "Progress: 80.33%\n",
      "Progress: 80.34%\n",
      "Progress: 80.34%\n",
      "Progress: 80.35%\n",
      "Progress: 80.36%\n",
      "Progress: 80.36%\n",
      "Progress: 80.37%\n",
      "Progress: 80.38%\n",
      "Progress: 80.38%\n",
      "Progress: 80.39%\n",
      "Progress: 80.40%\n",
      "Progress: 80.40%\n",
      "Progress: 80.41%\n",
      "Progress: 80.41%\n",
      "Progress: 80.42%\n",
      "Progress: 80.43%\n",
      "Progress: 80.43%\n",
      "Progress: 80.44%\n",
      "Progress: 80.45%\n",
      "Progress: 80.45%\n",
      "Progress: 80.46%\n",
      "Progress: 80.46%\n",
      "Progress: 80.47%\n",
      "Progress: 80.48%\n",
      "Progress: 80.48%\n",
      "Progress: 80.49%\n",
      "Progress: 80.50%\n",
      "Progress: 80.50%\n",
      "Progress: 80.51%\n",
      "Progress: 80.51%\n",
      "Progress: 80.52%\n",
      "Progress: 80.53%\n",
      "Progress: 80.53%\n",
      "Progress: 80.54%\n",
      "Progress: 80.55%\n",
      "Progress: 80.55%\n",
      "Progress: 80.56%\n",
      "Progress: 80.57%\n",
      "Progress: 80.57%\n",
      "Progress: 80.58%\n",
      "Progress: 80.58%\n",
      "Progress: 80.59%\n",
      "Progress: 80.60%\n",
      "Progress: 80.60%\n",
      "Progress: 80.61%\n",
      "Progress: 80.62%\n",
      "Progress: 80.62%\n",
      "Progress: 80.63%\n",
      "Progress: 80.63%\n",
      "Progress: 80.64%\n",
      "Progress: 80.65%\n",
      "Progress: 80.65%\n",
      "Progress: 80.66%\n",
      "Progress: 80.67%\n",
      "Progress: 80.67%\n",
      "Progress: 80.68%\n",
      "Progress: 80.68%\n",
      "Progress: 80.69%\n",
      "Progress: 80.70%\n",
      "Progress: 80.70%\n",
      "Progress: 80.71%\n",
      "Progress: 80.72%\n",
      "Progress: 80.72%\n",
      "Progress: 80.73%\n",
      "Progress: 80.73%\n",
      "Progress: 80.74%\n",
      "Progress: 80.75%\n",
      "Progress: 80.75%\n",
      "Progress: 80.76%\n",
      "Progress: 80.77%\n",
      "Progress: 80.77%\n",
      "Progress: 80.78%\n",
      "Progress: 80.78%\n",
      "Progress: 80.79%\n",
      "Progress: 80.80%\n",
      "Progress: 80.80%\n",
      "Progress: 80.81%\n",
      "Progress: 80.82%\n",
      "Progress: 80.82%\n",
      "Progress: 80.83%\n",
      "Progress: 80.83%\n",
      "Progress: 80.84%\n",
      "Progress: 80.85%\n",
      "Progress: 80.85%\n",
      "Progress: 80.86%\n",
      "Progress: 80.87%\n",
      "Progress: 80.87%\n",
      "Progress: 80.88%\n",
      "Progress: 80.88%\n",
      "Progress: 80.89%\n",
      "Progress: 80.90%\n",
      "Progress: 80.90%\n",
      "Progress: 80.91%\n",
      "Progress: 80.91%\n",
      "Progress: 80.92%\n",
      "Progress: 80.93%\n",
      "Progress: 80.93%\n",
      "Progress: 80.94%\n",
      "Progress: 80.95%\n",
      "Progress: 80.95%\n",
      "Progress: 80.96%\n",
      "Progress: 80.96%\n",
      "Progress: 80.97%\n",
      "Progress: 80.98%\n",
      "Progress: 80.98%\n",
      "Progress: 80.99%\n",
      "Progress: 81.00%\n",
      "Progress: 81.00%\n",
      "Progress: 81.01%\n",
      "Progress: 81.01%\n",
      "Progress: 81.02%\n",
      "Progress: 81.03%\n",
      "Progress: 81.03%\n",
      "Progress: 81.04%\n",
      "Progress: 81.05%\n",
      "Progress: 81.05%\n",
      "Progress: 81.06%\n",
      "Progress: 81.06%\n",
      "Progress: 81.07%\n",
      "Progress: 81.08%\n",
      "Progress: 81.08%\n",
      "Progress: 81.09%\n",
      "Progress: 81.09%\n",
      "Progress: 81.10%\n",
      "Progress: 81.11%\n",
      "Progress: 81.11%\n",
      "Progress: 81.12%\n",
      "Progress: 81.13%\n",
      "Progress: 81.13%\n",
      "Progress: 81.14%\n",
      "Progress: 81.14%\n",
      "Progress: 81.15%\n",
      "Progress: 81.16%\n",
      "Progress: 81.16%\n",
      "Progress: 81.17%\n",
      "Progress: 81.18%\n",
      "Progress: 81.18%\n",
      "Progress: 81.19%\n",
      "Progress: 81.19%\n",
      "Progress: 81.20%\n",
      "Progress: 81.21%\n",
      "Progress: 81.21%\n",
      "Progress: 81.22%\n",
      "Progress: 81.22%\n",
      "Progress: 81.23%\n",
      "Progress: 81.24%\n",
      "Progress: 81.24%\n",
      "Progress: 81.25%\n",
      "Progress: 81.26%\n",
      "Progress: 81.26%\n",
      "Progress: 81.27%\n",
      "Progress: 81.27%\n",
      "Progress: 81.28%\n",
      "Progress: 81.29%\n",
      "Progress: 81.29%\n",
      "Progress: 81.30%\n",
      "Progress: 81.30%\n",
      "Progress: 81.31%\n",
      "Progress: 81.32%\n",
      "Progress: 81.32%\n",
      "Progress: 81.33%\n",
      "Progress: 81.34%\n",
      "Progress: 81.34%\n",
      "Progress: 81.35%\n",
      "Progress: 81.35%\n",
      "Progress: 81.36%\n",
      "Progress: 81.37%\n",
      "Progress: 81.37%\n",
      "Progress: 81.38%\n",
      "Progress: 81.38%\n",
      "Progress: 81.39%\n",
      "Progress: 81.40%\n",
      "Progress: 81.40%\n",
      "Progress: 81.41%\n",
      "Progress: 81.42%\n",
      "Progress: 81.42%\n",
      "Progress: 81.43%\n",
      "Progress: 81.43%\n",
      "Progress: 81.44%\n",
      "Progress: 81.45%\n",
      "Progress: 81.45%\n",
      "Progress: 81.46%\n",
      "Progress: 81.46%\n",
      "Progress: 81.47%\n",
      "Progress: 81.48%\n",
      "Progress: 81.48%\n",
      "Progress: 81.49%\n",
      "Progress: 81.49%\n",
      "Progress: 81.50%\n",
      "Progress: 81.51%\n",
      "Progress: 81.51%\n",
      "Progress: 81.52%\n",
      "Progress: 81.53%\n",
      "Progress: 81.53%\n",
      "Progress: 81.54%\n",
      "Progress: 81.54%\n",
      "Progress: 81.55%\n",
      "Progress: 81.56%\n",
      "Progress: 81.56%\n",
      "Progress: 81.57%\n",
      "Progress: 81.57%\n",
      "Progress: 81.58%\n",
      "Progress: 81.59%\n",
      "Progress: 81.59%\n",
      "Progress: 81.60%\n",
      "Progress: 81.60%\n",
      "Progress: 81.61%\n",
      "Progress: 81.62%\n",
      "Progress: 81.62%\n",
      "Progress: 81.63%\n",
      "Progress: 81.64%\n",
      "Progress: 81.64%\n",
      "Progress: 81.65%\n",
      "Progress: 81.65%\n",
      "Progress: 81.66%\n",
      "Progress: 81.67%\n",
      "Progress: 81.67%\n",
      "Progress: 81.68%\n",
      "Progress: 81.68%\n",
      "Progress: 81.69%\n",
      "Progress: 81.70%\n",
      "Progress: 81.70%\n",
      "Progress: 81.71%\n",
      "Progress: 81.71%\n",
      "Progress: 81.72%\n",
      "Progress: 81.73%\n",
      "Progress: 81.73%\n",
      "Progress: 81.74%\n",
      "Progress: 81.74%\n",
      "Progress: 81.75%\n",
      "Progress: 81.76%\n",
      "Progress: 81.76%\n",
      "Progress: 81.77%\n",
      "Progress: 81.78%\n",
      "Progress: 81.78%\n",
      "Progress: 81.79%\n",
      "Progress: 81.79%\n",
      "Progress: 81.80%\n",
      "Progress: 81.81%\n",
      "Progress: 81.81%\n",
      "Progress: 81.82%\n",
      "Progress: 81.82%\n",
      "Progress: 81.83%\n",
      "Progress: 81.84%\n",
      "Progress: 81.84%\n",
      "Progress: 81.85%\n",
      "Progress: 81.85%\n",
      "Progress: 81.86%\n",
      "Progress: 81.87%\n",
      "Progress: 81.87%\n",
      "Progress: 81.88%\n",
      "Progress: 81.88%\n",
      "Progress: 81.89%\n",
      "Progress: 81.90%\n",
      "Progress: 81.90%\n",
      "Progress: 81.91%\n",
      "Progress: 81.91%\n",
      "Progress: 81.92%\n",
      "Progress: 81.93%\n",
      "Progress: 81.93%\n",
      "Progress: 81.94%\n",
      "Progress: 81.94%\n",
      "Progress: 81.95%\n",
      "Progress: 81.96%\n",
      "Progress: 81.96%\n",
      "Progress: 81.97%\n",
      "Progress: 81.98%\n",
      "Progress: 81.98%\n",
      "Progress: 81.99%\n",
      "Progress: 81.99%\n",
      "Progress: 82.00%\n",
      "Progress: 82.01%\n",
      "Progress: 82.01%\n",
      "Progress: 82.02%\n",
      "Progress: 82.02%\n",
      "Progress: 82.03%\n",
      "Progress: 82.04%\n",
      "Progress: 82.04%\n",
      "Progress: 82.05%\n",
      "Progress: 82.05%\n",
      "Progress: 82.06%\n",
      "Progress: 82.07%\n",
      "Progress: 82.07%\n",
      "Progress: 82.08%\n",
      "Progress: 82.08%\n",
      "Progress: 82.09%\n",
      "Progress: 82.10%\n",
      "Progress: 82.10%\n",
      "Progress: 82.11%\n",
      "Progress: 82.11%\n",
      "Progress: 82.12%\n",
      "Progress: 82.13%\n",
      "Progress: 82.13%\n",
      "Progress: 82.14%\n",
      "Progress: 82.14%\n",
      "Progress: 82.15%\n",
      "Progress: 82.16%\n",
      "Progress: 82.16%\n",
      "Progress: 82.17%\n",
      "Progress: 82.17%\n",
      "Progress: 82.18%\n",
      "Progress: 82.19%\n",
      "Progress: 82.19%\n",
      "Progress: 82.20%\n",
      "Progress: 82.20%\n",
      "Progress: 82.21%\n",
      "Progress: 82.22%\n",
      "Progress: 82.22%\n",
      "Progress: 82.23%\n",
      "Progress: 82.23%\n",
      "Progress: 82.24%\n",
      "Progress: 82.25%\n",
      "Progress: 82.25%\n",
      "Progress: 82.26%\n",
      "Progress: 82.26%\n",
      "Progress: 82.27%\n",
      "Progress: 82.28%\n",
      "Progress: 82.28%\n",
      "Progress: 82.29%\n",
      "Progress: 82.29%\n",
      "Progress: 82.30%\n",
      "Progress: 82.31%\n",
      "Progress: 82.31%\n",
      "Progress: 82.32%\n",
      "Progress: 82.32%\n",
      "Progress: 82.33%\n",
      "Progress: 82.34%\n",
      "Progress: 82.34%\n",
      "Progress: 82.35%\n",
      "Progress: 82.35%\n",
      "Progress: 82.36%\n",
      "Progress: 82.37%\n",
      "Progress: 82.37%\n",
      "Progress: 82.38%\n",
      "Progress: 82.38%\n",
      "Progress: 82.39%\n",
      "Progress: 82.40%\n",
      "Progress: 82.40%\n",
      "Progress: 82.41%\n",
      "Progress: 82.41%\n",
      "Progress: 82.42%\n",
      "Progress: 82.43%\n",
      "Progress: 82.43%\n",
      "Progress: 82.44%\n",
      "Progress: 82.44%\n",
      "Progress: 82.45%\n",
      "Progress: 82.46%\n",
      "Progress: 82.46%\n",
      "Progress: 82.47%\n",
      "Progress: 82.47%\n",
      "Progress: 82.48%\n",
      "Progress: 82.49%\n",
      "Progress: 82.49%\n",
      "Progress: 82.50%\n",
      "Progress: 82.50%\n",
      "Progress: 82.51%\n",
      "Progress: 82.51%\n",
      "Progress: 82.52%\n",
      "Progress: 82.53%\n",
      "Progress: 82.53%\n",
      "Progress: 82.54%\n",
      "Progress: 82.54%\n",
      "Progress: 82.55%\n",
      "Progress: 82.56%\n",
      "Progress: 82.56%\n",
      "Progress: 82.57%\n",
      "Progress: 82.57%\n",
      "Progress: 82.58%\n",
      "Progress: 82.59%\n",
      "Progress: 82.59%\n",
      "Progress: 82.60%\n",
      "Progress: 82.60%\n",
      "Progress: 82.61%\n",
      "Progress: 82.62%\n",
      "Progress: 82.62%\n",
      "Progress: 82.63%\n",
      "Progress: 82.63%\n",
      "Progress: 82.64%\n",
      "Progress: 82.65%\n",
      "Progress: 82.65%\n",
      "Progress: 82.66%\n",
      "Progress: 82.66%\n",
      "Progress: 82.67%\n",
      "Progress: 82.68%\n",
      "Progress: 82.68%\n",
      "Progress: 82.69%\n",
      "Progress: 82.69%\n",
      "Progress: 82.70%\n",
      "Progress: 82.70%\n",
      "Progress: 82.71%\n",
      "Progress: 82.72%\n",
      "Progress: 82.72%\n",
      "Progress: 82.73%\n",
      "Progress: 82.73%\n",
      "Progress: 82.74%\n",
      "Progress: 82.75%\n",
      "Progress: 82.75%\n",
      "Progress: 82.76%\n",
      "Progress: 82.76%\n",
      "Progress: 82.77%\n",
      "Progress: 82.78%\n",
      "Progress: 82.78%\n",
      "Progress: 82.79%\n",
      "Progress: 82.79%\n",
      "Progress: 82.80%\n",
      "Progress: 82.81%\n",
      "Progress: 82.81%\n",
      "Progress: 82.82%\n",
      "Progress: 82.82%\n",
      "Progress: 82.83%\n",
      "Progress: 82.83%\n",
      "Progress: 82.84%\n",
      "Progress: 82.85%\n",
      "Progress: 82.85%\n",
      "Progress: 82.86%\n",
      "Progress: 82.86%\n",
      "Progress: 82.87%\n",
      "Progress: 82.88%\n",
      "Progress: 82.88%\n",
      "Progress: 82.89%\n",
      "Progress: 82.89%\n",
      "Progress: 82.90%\n",
      "Progress: 82.91%\n",
      "Progress: 82.91%\n",
      "Progress: 82.92%\n",
      "Progress: 82.92%\n",
      "Progress: 82.93%\n",
      "Progress: 82.93%\n",
      "Progress: 82.94%\n",
      "Progress: 82.95%\n",
      "Progress: 82.95%\n",
      "Progress: 82.96%\n",
      "Progress: 82.96%\n",
      "Progress: 82.97%\n",
      "Progress: 82.98%\n",
      "Progress: 82.98%\n",
      "Progress: 82.99%\n",
      "Progress: 82.99%\n",
      "Progress: 83.00%\n",
      "Progress: 83.01%\n",
      "Progress: 83.01%\n",
      "Progress: 83.02%\n",
      "Progress: 83.02%\n",
      "Progress: 83.03%\n",
      "Progress: 83.03%\n",
      "Progress: 83.04%\n",
      "Progress: 83.05%\n",
      "Progress: 83.05%\n",
      "Progress: 83.06%\n",
      "Progress: 83.06%\n",
      "Progress: 83.07%\n",
      "Progress: 83.08%\n",
      "Progress: 83.08%\n",
      "Progress: 83.09%\n",
      "Progress: 83.09%\n",
      "Progress: 83.10%\n",
      "Progress: 83.10%\n",
      "Progress: 83.11%\n",
      "Progress: 83.12%\n",
      "Progress: 83.12%\n",
      "Progress: 83.13%\n",
      "Progress: 83.13%\n",
      "Progress: 83.14%\n",
      "Progress: 83.15%\n",
      "Progress: 83.15%\n",
      "Progress: 83.16%\n",
      "Progress: 83.16%\n",
      "Progress: 83.17%\n",
      "Progress: 83.18%\n",
      "Progress: 83.18%\n",
      "Progress: 83.19%\n",
      "Progress: 83.19%\n",
      "Progress: 83.20%\n",
      "Progress: 83.20%\n",
      "Progress: 83.21%\n",
      "Progress: 83.22%\n",
      "Progress: 83.22%\n",
      "Progress: 83.23%\n",
      "Progress: 83.23%\n",
      "Progress: 83.24%\n",
      "Progress: 83.25%\n",
      "Progress: 83.25%\n",
      "Progress: 83.26%\n",
      "Progress: 83.26%\n",
      "Progress: 83.27%\n",
      "Progress: 83.27%\n",
      "Progress: 83.28%\n",
      "Progress: 83.29%\n",
      "Progress: 83.29%\n",
      "Progress: 83.30%\n",
      "Progress: 83.30%\n",
      "Progress: 83.31%\n",
      "Progress: 83.31%\n",
      "Progress: 83.32%\n",
      "Progress: 83.33%\n",
      "Progress: 83.33%\n",
      "Progress: 83.34%\n",
      "Progress: 83.34%\n",
      "Progress: 83.35%\n",
      "Progress: 83.36%\n",
      "Progress: 83.36%\n",
      "Progress: 83.37%\n",
      "Progress: 83.37%\n",
      "Progress: 83.38%\n",
      "Progress: 83.38%\n",
      "Progress: 83.39%\n",
      "Progress: 83.40%\n",
      "Progress: 83.40%\n",
      "Progress: 83.41%\n",
      "Progress: 83.41%\n",
      "Progress: 83.42%\n",
      "Progress: 83.43%\n",
      "Progress: 83.43%\n",
      "Progress: 83.44%\n",
      "Progress: 83.44%\n",
      "Progress: 83.45%\n",
      "Progress: 83.45%\n",
      "Progress: 83.46%\n",
      "Progress: 83.47%\n",
      "Progress: 83.47%\n",
      "Progress: 83.48%\n",
      "Progress: 83.48%\n",
      "Progress: 83.49%\n",
      "Progress: 83.49%\n",
      "Progress: 83.50%\n",
      "Progress: 83.51%\n",
      "Progress: 83.51%\n",
      "Progress: 83.52%\n",
      "Progress: 83.52%\n",
      "Progress: 83.53%\n",
      "Progress: 83.54%\n",
      "Progress: 83.54%\n",
      "Progress: 83.55%\n",
      "Progress: 83.55%\n",
      "Progress: 83.56%\n",
      "Progress: 83.56%\n",
      "Progress: 83.57%\n",
      "Progress: 83.58%\n",
      "Progress: 83.58%\n",
      "Progress: 83.59%\n",
      "Progress: 83.59%\n",
      "Progress: 83.60%\n",
      "Progress: 83.60%\n",
      "Progress: 83.61%\n",
      "Progress: 83.62%\n",
      "Progress: 83.62%\n",
      "Progress: 83.63%\n",
      "Progress: 83.63%\n",
      "Progress: 83.64%\n",
      "Progress: 83.64%\n",
      "Progress: 83.65%\n",
      "Progress: 83.66%\n",
      "Progress: 83.66%\n",
      "Progress: 83.67%\n",
      "Progress: 83.67%\n",
      "Progress: 83.68%\n",
      "Progress: 83.68%\n",
      "Progress: 83.69%\n",
      "Progress: 83.70%\n",
      "Progress: 83.70%\n",
      "Progress: 83.71%\n",
      "Progress: 83.71%\n",
      "Progress: 83.72%\n",
      "Progress: 83.73%\n",
      "Progress: 83.73%\n",
      "Progress: 83.74%\n",
      "Progress: 83.74%\n",
      "Progress: 83.75%\n",
      "Progress: 83.75%\n",
      "Progress: 83.76%\n",
      "Progress: 83.77%\n",
      "Progress: 83.77%\n",
      "Progress: 83.78%\n",
      "Progress: 83.78%\n",
      "Progress: 83.79%\n",
      "Progress: 83.79%\n",
      "Progress: 83.80%\n",
      "Progress: 83.81%\n",
      "Progress: 83.81%\n",
      "Progress: 83.82%\n",
      "Progress: 83.82%\n",
      "Progress: 83.83%\n",
      "Progress: 83.83%\n",
      "Progress: 83.84%\n",
      "Progress: 83.85%\n",
      "Progress: 83.85%\n",
      "Progress: 83.86%\n",
      "Progress: 83.86%\n",
      "Progress: 83.87%\n",
      "Progress: 83.87%\n",
      "Progress: 83.88%\n",
      "Progress: 83.89%\n",
      "Progress: 83.89%\n",
      "Progress: 83.90%\n",
      "Progress: 83.90%\n",
      "Progress: 83.91%\n",
      "Progress: 83.91%\n",
      "Progress: 83.92%\n",
      "Progress: 83.93%\n",
      "Progress: 83.93%\n",
      "Progress: 83.94%\n",
      "Progress: 83.94%\n",
      "Progress: 83.95%\n",
      "Progress: 83.95%\n",
      "Progress: 83.96%\n",
      "Progress: 83.97%\n",
      "Progress: 83.97%\n",
      "Progress: 83.98%\n",
      "Progress: 83.98%\n",
      "Progress: 83.99%\n",
      "Progress: 83.99%\n",
      "Progress: 84.00%\n",
      "Progress: 84.01%\n",
      "Progress: 84.01%\n",
      "Progress: 84.02%\n",
      "Progress: 84.02%\n",
      "Progress: 84.03%\n",
      "Progress: 84.03%\n",
      "Progress: 84.04%\n",
      "Progress: 84.04%\n",
      "Progress: 84.05%\n",
      "Progress: 84.06%\n",
      "Progress: 84.06%\n",
      "Progress: 84.07%\n",
      "Progress: 84.07%\n",
      "Progress: 84.08%\n",
      "Progress: 84.08%\n",
      "Progress: 84.09%\n",
      "Progress: 84.10%\n",
      "Progress: 84.10%\n",
      "Progress: 84.11%\n",
      "Progress: 84.11%\n",
      "Progress: 84.12%\n",
      "Progress: 84.12%\n",
      "Progress: 84.13%\n",
      "Progress: 84.14%\n",
      "Progress: 84.14%\n",
      "Progress: 84.15%\n",
      "Progress: 84.15%\n",
      "Progress: 84.16%\n",
      "Progress: 84.16%\n",
      "Progress: 84.17%\n",
      "Progress: 84.18%\n",
      "Progress: 84.18%\n",
      "Progress: 84.19%\n",
      "Progress: 84.19%\n",
      "Progress: 84.20%\n",
      "Progress: 84.20%\n",
      "Progress: 84.21%\n",
      "Progress: 84.22%\n",
      "Progress: 84.22%\n",
      "Progress: 84.23%\n",
      "Progress: 84.23%\n",
      "Progress: 84.24%\n",
      "Progress: 84.24%\n",
      "Progress: 84.25%\n",
      "Progress: 84.25%\n",
      "Progress: 84.26%\n",
      "Progress: 84.27%\n",
      "Progress: 84.27%\n",
      "Progress: 84.28%\n",
      "Progress: 84.28%\n",
      "Progress: 84.29%\n",
      "Progress: 84.29%\n",
      "Progress: 84.30%\n",
      "Progress: 84.31%\n",
      "Progress: 84.31%\n",
      "Progress: 84.32%\n",
      "Progress: 84.32%\n",
      "Progress: 84.33%\n",
      "Progress: 84.33%\n",
      "Progress: 84.34%\n",
      "Progress: 84.34%\n",
      "Progress: 84.35%\n",
      "Progress: 84.36%\n",
      "Progress: 84.36%\n",
      "Progress: 84.37%\n",
      "Progress: 84.37%\n",
      "Progress: 84.38%\n",
      "Progress: 84.38%\n",
      "Progress: 84.39%\n",
      "Progress: 84.40%\n",
      "Progress: 84.40%\n",
      "Progress: 84.41%\n",
      "Progress: 84.41%\n",
      "Progress: 84.42%\n",
      "Progress: 84.42%\n",
      "Progress: 84.43%\n",
      "Progress: 84.43%\n",
      "Progress: 84.44%\n",
      "Progress: 84.45%\n",
      "Progress: 84.45%\n",
      "Progress: 84.46%\n",
      "Progress: 84.46%\n",
      "Progress: 84.47%\n",
      "Progress: 84.47%\n",
      "Progress: 84.48%\n",
      "Progress: 84.49%\n",
      "Progress: 84.49%\n",
      "Progress: 84.50%\n",
      "Progress: 84.50%\n",
      "Progress: 84.51%\n",
      "Progress: 84.51%\n",
      "Progress: 84.52%\n",
      "Progress: 84.52%\n",
      "Progress: 84.53%\n",
      "Progress: 84.54%\n",
      "Progress: 84.54%\n",
      "Progress: 84.55%\n",
      "Progress: 84.55%\n",
      "Progress: 84.56%\n",
      "Progress: 84.56%\n",
      "Progress: 84.57%\n",
      "Progress: 84.57%\n",
      "Progress: 84.58%\n",
      "Progress: 84.59%\n",
      "Progress: 84.59%\n",
      "Progress: 84.60%\n",
      "Progress: 84.60%\n",
      "Progress: 84.61%\n",
      "Progress: 84.61%\n",
      "Progress: 84.62%\n",
      "Progress: 84.63%\n",
      "Progress: 84.63%\n",
      "Progress: 84.64%\n",
      "Progress: 84.64%\n",
      "Progress: 84.65%\n",
      "Progress: 84.65%\n",
      "Progress: 84.66%\n",
      "Progress: 84.66%\n",
      "Progress: 84.67%\n",
      "Progress: 84.68%\n",
      "Progress: 84.68%\n",
      "Progress: 84.69%\n",
      "Progress: 84.69%\n",
      "Progress: 84.70%\n",
      "Progress: 84.70%\n",
      "Progress: 84.71%\n",
      "Progress: 84.71%\n",
      "Progress: 84.72%\n",
      "Progress: 84.73%\n",
      "Progress: 84.73%\n",
      "Progress: 84.74%\n",
      "Progress: 84.74%\n",
      "Progress: 84.75%\n",
      "Progress: 84.75%\n",
      "Progress: 84.76%\n",
      "Progress: 84.76%\n",
      "Progress: 84.77%\n",
      "Progress: 84.78%\n",
      "Progress: 84.78%\n",
      "Progress: 84.79%\n",
      "Progress: 84.79%\n",
      "Progress: 84.80%\n",
      "Progress: 84.80%\n",
      "Progress: 84.81%\n",
      "Progress: 84.81%\n",
      "Progress: 84.82%\n",
      "Progress: 84.83%\n",
      "Progress: 84.83%\n",
      "Progress: 84.84%\n",
      "Progress: 84.84%\n",
      "Progress: 84.85%\n",
      "Progress: 84.85%\n",
      "Progress: 84.86%\n",
      "Progress: 84.86%\n",
      "Progress: 84.87%\n",
      "Progress: 84.88%\n",
      "Progress: 84.88%\n",
      "Progress: 84.89%\n",
      "Progress: 84.89%\n",
      "Progress: 84.90%\n",
      "Progress: 84.90%\n",
      "Progress: 84.91%\n",
      "Progress: 84.91%\n",
      "Progress: 84.92%\n",
      "Progress: 84.92%\n",
      "Progress: 84.93%\n",
      "Progress: 84.94%\n",
      "Progress: 84.94%\n",
      "Progress: 84.95%\n",
      "Progress: 84.95%\n",
      "Progress: 84.96%\n",
      "Progress: 84.96%\n",
      "Progress: 84.97%\n",
      "Progress: 84.97%\n",
      "Progress: 84.98%\n",
      "Progress: 84.99%\n",
      "Progress: 84.99%\n",
      "Progress: 85.00%\n",
      "Progress: 85.00%\n",
      "Progress: 85.01%\n",
      "Progress: 85.01%\n",
      "Progress: 85.02%\n",
      "Progress: 85.02%\n",
      "Progress: 85.03%\n",
      "Progress: 85.04%\n",
      "Progress: 85.04%\n",
      "Progress: 85.05%\n",
      "Progress: 85.05%\n",
      "Progress: 85.06%\n",
      "Progress: 85.06%\n",
      "Progress: 85.07%\n",
      "Progress: 85.07%\n",
      "Progress: 85.08%\n",
      "Progress: 85.08%\n",
      "Progress: 85.09%\n",
      "Progress: 85.10%\n",
      "Progress: 85.10%\n",
      "Progress: 85.11%\n",
      "Progress: 85.11%\n",
      "Progress: 85.12%\n",
      "Progress: 85.12%\n",
      "Progress: 85.13%\n",
      "Progress: 85.13%\n",
      "Progress: 85.14%\n",
      "Progress: 85.15%\n",
      "Progress: 85.15%\n",
      "Progress: 85.16%\n",
      "Progress: 85.16%\n",
      "Progress: 85.17%\n",
      "Progress: 85.17%\n",
      "Progress: 85.18%\n",
      "Progress: 85.18%\n",
      "Progress: 85.19%\n",
      "Progress: 85.19%\n",
      "Progress: 85.20%\n",
      "Progress: 85.21%\n",
      "Progress: 85.21%\n",
      "Progress: 85.22%\n",
      "Progress: 85.22%\n",
      "Progress: 85.23%\n",
      "Progress: 85.23%\n",
      "Progress: 85.24%\n",
      "Progress: 85.24%\n",
      "Progress: 85.25%\n",
      "Progress: 85.25%\n",
      "Progress: 85.26%\n",
      "Progress: 85.27%\n",
      "Progress: 85.27%\n",
      "Progress: 85.28%\n",
      "Progress: 85.28%\n",
      "Progress: 85.29%\n",
      "Progress: 85.29%\n",
      "Progress: 85.30%\n",
      "Progress: 85.30%\n",
      "Progress: 85.31%\n",
      "Progress: 85.31%\n",
      "Progress: 85.32%\n",
      "Progress: 85.33%\n",
      "Progress: 85.33%\n",
      "Progress: 85.34%\n",
      "Progress: 85.34%\n",
      "Progress: 85.35%\n",
      "Progress: 85.35%\n",
      "Progress: 85.36%\n",
      "Progress: 85.36%\n",
      "Progress: 85.37%\n",
      "Progress: 85.37%\n",
      "Progress: 85.38%\n",
      "Progress: 85.39%\n",
      "Progress: 85.39%\n",
      "Progress: 85.40%\n",
      "Progress: 85.40%\n",
      "Progress: 85.41%\n",
      "Progress: 85.41%\n",
      "Progress: 85.42%\n",
      "Progress: 85.42%\n",
      "Progress: 85.43%\n",
      "Progress: 85.43%\n",
      "Progress: 85.44%\n",
      "Progress: 85.45%\n",
      "Progress: 85.45%\n",
      "Progress: 85.46%\n",
      "Progress: 85.46%\n",
      "Progress: 85.47%\n",
      "Progress: 85.47%\n",
      "Progress: 85.48%\n",
      "Progress: 85.48%\n",
      "Progress: 85.49%\n",
      "Progress: 85.49%\n",
      "Progress: 85.50%\n",
      "Progress: 85.50%\n",
      "Progress: 85.51%\n",
      "Progress: 85.52%\n",
      "Progress: 85.52%\n",
      "Progress: 85.53%\n",
      "Progress: 85.53%\n",
      "Progress: 85.54%\n",
      "Progress: 85.54%\n",
      "Progress: 85.55%\n",
      "Progress: 85.55%\n",
      "Progress: 85.56%\n",
      "Progress: 85.56%\n",
      "Progress: 85.57%\n",
      "Progress: 85.58%\n",
      "Progress: 85.58%\n",
      "Progress: 85.59%\n",
      "Progress: 85.59%\n",
      "Progress: 85.60%\n",
      "Progress: 85.60%\n",
      "Progress: 85.61%\n",
      "Progress: 85.61%\n",
      "Progress: 85.62%\n",
      "Progress: 85.62%\n",
      "Progress: 85.63%\n",
      "Progress: 85.63%\n",
      "Progress: 85.64%\n",
      "Progress: 85.65%\n",
      "Progress: 85.65%\n",
      "Progress: 85.66%\n",
      "Progress: 85.66%\n",
      "Progress: 85.67%\n",
      "Progress: 85.67%\n",
      "Progress: 85.68%\n",
      "Progress: 85.68%\n",
      "Progress: 85.69%\n",
      "Progress: 85.69%\n",
      "Progress: 85.70%\n",
      "Progress: 85.70%\n",
      "Progress: 85.71%\n",
      "Progress: 85.72%\n",
      "Progress: 85.72%\n",
      "Progress: 85.73%\n",
      "Progress: 85.73%\n",
      "Progress: 85.74%\n",
      "Progress: 85.74%\n",
      "Progress: 85.75%\n",
      "Progress: 85.75%\n",
      "Progress: 85.76%\n",
      "Progress: 85.76%\n",
      "Progress: 85.77%\n",
      "Progress: 85.77%\n",
      "Progress: 85.78%\n",
      "Progress: 85.79%\n",
      "Progress: 85.79%\n",
      "Progress: 85.80%\n",
      "Progress: 85.80%\n",
      "Progress: 85.81%\n",
      "Progress: 85.81%\n",
      "Progress: 85.82%\n",
      "Progress: 85.82%\n",
      "Progress: 85.83%\n",
      "Progress: 85.83%\n",
      "Progress: 85.84%\n",
      "Progress: 85.84%\n",
      "Progress: 85.85%\n",
      "Progress: 85.85%\n",
      "Progress: 85.86%\n",
      "Progress: 85.87%\n",
      "Progress: 85.87%\n",
      "Progress: 85.88%\n",
      "Progress: 85.88%\n",
      "Progress: 85.89%\n",
      "Progress: 85.89%\n",
      "Progress: 85.90%\n",
      "Progress: 85.90%\n",
      "Progress: 85.91%\n",
      "Progress: 85.91%\n",
      "Progress: 85.92%\n",
      "Progress: 85.92%\n",
      "Progress: 85.93%\n",
      "Progress: 85.94%\n",
      "Progress: 85.94%\n",
      "Progress: 85.95%\n",
      "Progress: 85.95%\n",
      "Progress: 85.96%\n",
      "Progress: 85.96%\n",
      "Progress: 85.97%\n",
      "Progress: 85.97%\n",
      "Progress: 85.98%\n",
      "Progress: 85.98%\n",
      "Progress: 85.99%\n",
      "Progress: 85.99%\n",
      "Progress: 86.00%\n",
      "Progress: 86.00%\n",
      "Progress: 86.01%\n",
      "Progress: 86.02%\n",
      "Progress: 86.02%\n",
      "Progress: 86.03%\n",
      "Progress: 86.03%\n",
      "Progress: 86.04%\n",
      "Progress: 86.04%\n",
      "Progress: 86.05%\n",
      "Progress: 86.05%\n",
      "Progress: 86.06%\n",
      "Progress: 86.06%\n",
      "Progress: 86.07%\n",
      "Progress: 86.07%\n",
      "Progress: 86.08%\n",
      "Progress: 86.08%\n",
      "Progress: 86.09%\n",
      "Progress: 86.09%\n",
      "Progress: 86.10%\n",
      "Progress: 86.11%\n",
      "Progress: 86.11%\n",
      "Progress: 86.12%\n",
      "Progress: 86.12%\n",
      "Progress: 86.13%\n",
      "Progress: 86.13%\n",
      "Progress: 86.14%\n",
      "Progress: 86.14%\n",
      "Progress: 86.15%\n",
      "Progress: 86.15%\n",
      "Progress: 86.16%\n",
      "Progress: 86.16%\n",
      "Progress: 86.17%\n",
      "Progress: 86.17%\n",
      "Progress: 86.18%\n",
      "Progress: 86.18%\n",
      "Progress: 86.19%\n",
      "Progress: 86.20%\n",
      "Progress: 86.20%\n",
      "Progress: 86.21%\n",
      "Progress: 86.21%\n",
      "Progress: 86.22%\n",
      "Progress: 86.22%\n",
      "Progress: 86.23%\n",
      "Progress: 86.23%\n",
      "Progress: 86.24%\n",
      "Progress: 86.24%\n",
      "Progress: 86.25%\n",
      "Progress: 86.25%\n",
      "Progress: 86.26%\n",
      "Progress: 86.26%\n",
      "Progress: 86.27%\n",
      "Progress: 86.27%\n",
      "Progress: 86.28%\n",
      "Progress: 86.29%\n",
      "Progress: 86.29%\n",
      "Progress: 86.30%\n",
      "Progress: 86.30%\n",
      "Progress: 86.31%\n",
      "Progress: 86.31%\n",
      "Progress: 86.32%\n",
      "Progress: 86.32%\n",
      "Progress: 86.33%\n",
      "Progress: 86.33%\n",
      "Progress: 86.34%\n",
      "Progress: 86.34%\n",
      "Progress: 86.35%\n",
      "Progress: 86.35%\n",
      "Progress: 86.36%\n",
      "Progress: 86.36%\n",
      "Progress: 86.37%\n",
      "Progress: 86.37%\n",
      "Progress: 86.38%\n",
      "Progress: 86.39%\n",
      "Progress: 86.39%\n",
      "Progress: 86.40%\n",
      "Progress: 86.40%\n",
      "Progress: 86.41%\n",
      "Progress: 86.41%\n",
      "Progress: 86.42%\n",
      "Progress: 86.42%\n",
      "Progress: 86.43%\n",
      "Progress: 86.43%\n",
      "Progress: 86.44%\n",
      "Progress: 86.44%\n",
      "Progress: 86.45%\n",
      "Progress: 86.45%\n",
      "Progress: 86.46%\n",
      "Progress: 86.46%\n",
      "Progress: 86.47%\n",
      "Progress: 86.47%\n",
      "Progress: 86.48%\n",
      "Progress: 86.48%\n",
      "Progress: 86.49%\n",
      "Progress: 86.50%\n",
      "Progress: 86.50%\n",
      "Progress: 86.51%\n",
      "Progress: 86.51%\n",
      "Progress: 86.52%\n",
      "Progress: 86.52%\n",
      "Progress: 86.53%\n",
      "Progress: 86.53%\n",
      "Progress: 86.54%\n",
      "Progress: 86.54%\n",
      "Progress: 86.55%\n",
      "Progress: 86.55%\n",
      "Progress: 86.56%\n",
      "Progress: 86.56%\n",
      "Progress: 86.57%\n",
      "Progress: 86.57%\n",
      "Progress: 86.58%\n",
      "Progress: 86.58%\n",
      "Progress: 86.59%\n",
      "Progress: 86.59%\n",
      "Progress: 86.60%\n",
      "Progress: 86.60%\n",
      "Progress: 86.61%\n",
      "Progress: 86.62%\n",
      "Progress: 86.62%\n",
      "Progress: 86.63%\n",
      "Progress: 86.63%\n",
      "Progress: 86.64%\n",
      "Progress: 86.64%\n",
      "Progress: 86.65%\n",
      "Progress: 86.65%\n",
      "Progress: 86.66%\n",
      "Progress: 86.66%\n",
      "Progress: 86.67%\n",
      "Progress: 86.67%\n",
      "Progress: 86.68%\n",
      "Progress: 86.68%\n",
      "Progress: 86.69%\n",
      "Progress: 86.69%\n",
      "Progress: 86.70%\n",
      "Progress: 86.70%\n",
      "Progress: 86.71%\n",
      "Progress: 86.71%\n",
      "Progress: 86.72%\n",
      "Progress: 86.72%\n",
      "Progress: 86.73%\n",
      "Progress: 86.73%\n",
      "Progress: 86.74%\n",
      "Progress: 86.75%\n",
      "Progress: 86.75%\n",
      "Progress: 86.76%\n",
      "Progress: 86.76%\n",
      "Progress: 86.77%\n",
      "Progress: 86.77%\n",
      "Progress: 86.78%\n",
      "Progress: 86.78%\n",
      "Progress: 86.79%\n",
      "Progress: 86.79%\n",
      "Progress: 86.80%\n",
      "Progress: 86.80%\n",
      "Progress: 86.81%\n",
      "Progress: 86.81%\n",
      "Progress: 86.82%\n",
      "Progress: 86.82%\n",
      "Progress: 86.83%\n",
      "Progress: 86.83%\n",
      "Progress: 86.84%\n",
      "Progress: 86.84%\n",
      "Progress: 86.85%\n",
      "Progress: 86.85%\n",
      "Progress: 86.86%\n",
      "Progress: 86.86%\n",
      "Progress: 86.87%\n",
      "Progress: 86.87%\n",
      "Progress: 86.88%\n",
      "Progress: 86.88%\n",
      "Progress: 86.89%\n",
      "Progress: 86.89%\n",
      "Progress: 86.90%\n",
      "Progress: 86.91%\n",
      "Progress: 86.91%\n",
      "Progress: 86.92%\n",
      "Progress: 86.92%\n",
      "Progress: 86.93%\n",
      "Progress: 86.93%\n",
      "Progress: 86.94%\n",
      "Progress: 86.94%\n",
      "Progress: 86.95%\n",
      "Progress: 86.95%\n",
      "Progress: 86.96%\n",
      "Progress: 86.96%\n",
      "Progress: 86.97%\n",
      "Progress: 86.97%\n",
      "Progress: 86.98%\n",
      "Progress: 86.98%\n",
      "Progress: 86.99%\n",
      "Progress: 86.99%\n",
      "Progress: 87.00%\n",
      "Progress: 87.00%\n",
      "Progress: 87.01%\n",
      "Progress: 87.01%\n",
      "Progress: 87.02%\n",
      "Progress: 87.02%\n",
      "Progress: 87.03%\n",
      "Progress: 87.03%\n",
      "Progress: 87.04%\n",
      "Progress: 87.04%\n",
      "Progress: 87.05%\n",
      "Progress: 87.05%\n",
      "Progress: 87.06%\n",
      "Progress: 87.06%\n",
      "Progress: 87.07%\n",
      "Progress: 87.07%\n",
      "Progress: 87.08%\n",
      "Progress: 87.08%\n",
      "Progress: 87.09%\n",
      "Progress: 87.10%\n",
      "Progress: 87.10%\n",
      "Progress: 87.11%\n",
      "Progress: 87.11%\n",
      "Progress: 87.12%\n",
      "Progress: 87.12%\n",
      "Progress: 87.13%\n",
      "Progress: 87.13%\n",
      "Progress: 87.14%\n",
      "Progress: 87.14%\n",
      "Progress: 87.15%\n",
      "Progress: 87.15%\n",
      "Progress: 87.16%\n",
      "Progress: 87.16%\n",
      "Progress: 87.17%\n",
      "Progress: 87.17%\n",
      "Progress: 87.18%\n",
      "Progress: 87.18%\n",
      "Progress: 87.19%\n",
      "Progress: 87.19%\n",
      "Progress: 87.20%\n",
      "Progress: 87.20%\n",
      "Progress: 87.21%\n",
      "Progress: 87.21%\n",
      "Progress: 87.22%\n",
      "Progress: 87.22%\n",
      "Progress: 87.23%\n",
      "Progress: 87.23%\n",
      "Progress: 87.24%\n",
      "Progress: 87.24%\n",
      "Progress: 87.25%\n",
      "Progress: 87.25%\n",
      "Progress: 87.26%\n",
      "Progress: 87.26%\n",
      "Progress: 87.27%\n",
      "Progress: 87.27%\n",
      "Progress: 87.28%\n",
      "Progress: 87.28%\n",
      "Progress: 87.29%\n",
      "Progress: 87.29%\n",
      "Progress: 87.30%\n",
      "Progress: 87.30%\n",
      "Progress: 87.31%\n",
      "Progress: 87.31%\n",
      "Progress: 87.32%\n",
      "Progress: 87.32%\n",
      "Progress: 87.33%\n",
      "Progress: 87.33%\n",
      "Progress: 87.34%\n",
      "Progress: 87.34%\n",
      "Progress: 87.35%\n",
      "Progress: 87.35%\n",
      "Progress: 87.36%\n",
      "Progress: 87.36%\n",
      "Progress: 87.37%\n",
      "Progress: 87.37%\n",
      "Progress: 87.38%\n",
      "Progress: 87.38%\n",
      "Progress: 87.39%\n",
      "Progress: 87.39%\n",
      "Progress: 87.40%\n",
      "Progress: 87.41%\n",
      "Progress: 87.41%\n",
      "Progress: 87.42%\n",
      "Progress: 87.42%\n",
      "Progress: 87.43%\n",
      "Progress: 87.43%\n",
      "Progress: 87.44%\n",
      "Progress: 87.44%\n",
      "Progress: 87.45%\n",
      "Progress: 87.45%\n",
      "Progress: 87.46%\n",
      "Progress: 87.46%\n",
      "Progress: 87.47%\n",
      "Progress: 87.47%\n",
      "Progress: 87.48%\n",
      "Progress: 87.48%\n",
      "Progress: 87.49%\n",
      "Progress: 87.49%\n",
      "Progress: 87.50%\n",
      "Progress: 87.50%\n",
      "Progress: 87.51%\n",
      "Progress: 87.51%\n",
      "Progress: 87.52%\n",
      "Progress: 87.52%\n",
      "Progress: 87.53%\n",
      "Progress: 87.53%\n",
      "Progress: 87.54%\n",
      "Progress: 87.54%\n",
      "Progress: 87.55%\n",
      "Progress: 87.55%\n",
      "Progress: 87.56%\n",
      "Progress: 87.56%\n",
      "Progress: 87.57%\n",
      "Progress: 87.57%\n",
      "Progress: 87.58%\n",
      "Progress: 87.58%\n",
      "Progress: 87.59%\n",
      "Progress: 87.59%\n",
      "Progress: 87.60%\n",
      "Progress: 87.60%\n",
      "Progress: 87.61%\n",
      "Progress: 87.61%\n",
      "Progress: 87.62%\n",
      "Progress: 87.62%\n",
      "Progress: 87.63%\n",
      "Progress: 87.63%\n",
      "Progress: 87.64%\n",
      "Progress: 87.64%\n",
      "Progress: 87.65%\n",
      "Progress: 87.65%\n",
      "Progress: 87.66%\n",
      "Progress: 87.66%\n",
      "Progress: 87.67%\n",
      "Progress: 87.67%\n",
      "Progress: 87.68%\n",
      "Progress: 87.68%\n",
      "Progress: 87.69%\n",
      "Progress: 87.69%\n",
      "Progress: 87.70%\n",
      "Progress: 87.70%\n",
      "Progress: 87.71%\n",
      "Progress: 87.71%\n",
      "Progress: 87.72%\n",
      "Progress: 87.72%\n",
      "Progress: 87.73%\n",
      "Progress: 87.73%\n",
      "Progress: 87.74%\n",
      "Progress: 87.74%\n",
      "Progress: 87.75%\n",
      "Progress: 87.75%\n",
      "Progress: 87.76%\n",
      "Progress: 87.76%\n",
      "Progress: 87.77%\n",
      "Progress: 87.77%\n",
      "Progress: 87.78%\n",
      "Progress: 87.78%\n",
      "Progress: 87.79%\n",
      "Progress: 87.79%\n",
      "Progress: 87.80%\n",
      "Progress: 87.80%\n",
      "Progress: 87.81%\n",
      "Progress: 87.81%\n",
      "Progress: 87.82%\n",
      "Progress: 87.82%\n",
      "Progress: 87.83%\n",
      "Progress: 87.83%\n",
      "Progress: 87.84%\n",
      "Progress: 87.84%\n",
      "Progress: 87.85%\n",
      "Progress: 87.85%\n",
      "Progress: 87.86%\n",
      "Progress: 87.86%\n",
      "Progress: 87.87%\n",
      "Progress: 87.87%\n",
      "Progress: 87.88%\n",
      "Progress: 87.88%\n",
      "Progress: 87.89%\n",
      "Progress: 87.89%\n",
      "Progress: 87.90%\n",
      "Progress: 87.90%\n",
      "Progress: 87.91%\n",
      "Progress: 87.91%\n",
      "Progress: 87.91%\n",
      "Progress: 87.92%\n",
      "Progress: 87.92%\n",
      "Progress: 87.93%\n",
      "Progress: 87.93%\n",
      "Progress: 87.94%\n",
      "Progress: 87.94%\n",
      "Progress: 87.95%\n",
      "Progress: 87.95%\n",
      "Progress: 87.96%\n",
      "Progress: 87.96%\n",
      "Progress: 87.97%\n",
      "Progress: 87.97%\n",
      "Progress: 87.98%\n",
      "Progress: 87.98%\n",
      "Progress: 87.99%\n",
      "Progress: 87.99%\n",
      "Progress: 88.00%\n",
      "Progress: 88.00%\n",
      "Progress: 88.01%\n",
      "Progress: 88.01%\n",
      "Progress: 88.02%\n",
      "Progress: 88.02%\n",
      "Progress: 88.03%\n",
      "Progress: 88.03%\n",
      "Progress: 88.04%\n",
      "Progress: 88.04%\n",
      "Progress: 88.05%\n",
      "Progress: 88.05%\n",
      "Progress: 88.06%\n",
      "Progress: 88.06%\n",
      "Progress: 88.07%\n",
      "Progress: 88.07%\n",
      "Progress: 88.08%\n",
      "Progress: 88.08%\n",
      "Progress: 88.09%\n",
      "Progress: 88.09%\n",
      "Progress: 88.10%\n",
      "Progress: 88.10%\n",
      "Progress: 88.11%\n",
      "Progress: 88.11%\n",
      "Progress: 88.12%\n",
      "Progress: 88.12%\n",
      "Progress: 88.13%\n",
      "Progress: 88.13%\n",
      "Progress: 88.14%\n",
      "Progress: 88.14%\n",
      "Progress: 88.15%\n",
      "Progress: 88.15%\n",
      "Progress: 88.16%\n",
      "Progress: 88.16%\n",
      "Progress: 88.17%\n",
      "Progress: 88.17%\n",
      "Progress: 88.18%\n",
      "Progress: 88.18%\n",
      "Progress: 88.19%\n",
      "Progress: 88.19%\n",
      "Progress: 88.20%\n",
      "Progress: 88.20%\n",
      "Progress: 88.21%\n",
      "Progress: 88.21%\n",
      "Progress: 88.21%\n",
      "Progress: 88.22%\n",
      "Progress: 88.22%\n",
      "Progress: 88.23%\n",
      "Progress: 88.23%\n",
      "Progress: 88.24%\n",
      "Progress: 88.24%\n",
      "Progress: 88.25%\n",
      "Progress: 88.25%\n",
      "Progress: 88.26%\n",
      "Progress: 88.26%\n",
      "Progress: 88.27%\n",
      "Progress: 88.27%\n",
      "Progress: 88.28%\n",
      "Progress: 88.28%\n",
      "Progress: 88.29%\n",
      "Progress: 88.29%\n",
      "Progress: 88.30%\n",
      "Progress: 88.30%\n",
      "Progress: 88.31%\n",
      "Progress: 88.31%\n",
      "Progress: 88.32%\n",
      "Progress: 88.32%\n",
      "Progress: 88.33%\n",
      "Progress: 88.33%\n",
      "Progress: 88.34%\n",
      "Progress: 88.34%\n",
      "Progress: 88.35%\n",
      "Progress: 88.35%\n",
      "Progress: 88.36%\n",
      "Progress: 88.36%\n",
      "Progress: 88.37%\n",
      "Progress: 88.37%\n",
      "Progress: 88.38%\n",
      "Progress: 88.38%\n",
      "Progress: 88.39%\n",
      "Progress: 88.39%\n",
      "Progress: 88.39%\n",
      "Progress: 88.40%\n",
      "Progress: 88.40%\n",
      "Progress: 88.41%\n",
      "Progress: 88.41%\n",
      "Progress: 88.42%\n",
      "Progress: 88.42%\n",
      "Progress: 88.43%\n",
      "Progress: 88.43%\n",
      "Progress: 88.44%\n",
      "Progress: 88.44%\n",
      "Progress: 88.45%\n",
      "Progress: 88.45%\n",
      "Progress: 88.46%\n",
      "Progress: 88.46%\n",
      "Progress: 88.47%\n",
      "Progress: 88.47%\n",
      "Progress: 88.48%\n",
      "Progress: 88.48%\n",
      "Progress: 88.49%\n",
      "Progress: 88.49%\n",
      "Progress: 88.50%\n",
      "Progress: 88.50%\n",
      "Progress: 88.51%\n",
      "Progress: 88.51%\n",
      "Progress: 88.52%\n",
      "Progress: 88.52%\n",
      "Progress: 88.53%\n",
      "Progress: 88.53%\n",
      "Progress: 88.54%\n",
      "Progress: 88.54%\n",
      "Progress: 88.54%\n",
      "Progress: 88.55%\n",
      "Progress: 88.55%\n",
      "Progress: 88.56%\n",
      "Progress: 88.56%\n",
      "Progress: 88.57%\n",
      "Progress: 88.57%\n",
      "Progress: 88.58%\n",
      "Progress: 88.58%\n",
      "Progress: 88.59%\n",
      "Progress: 88.59%\n",
      "Progress: 88.60%\n",
      "Progress: 88.60%\n",
      "Progress: 88.61%\n",
      "Progress: 88.61%\n",
      "Progress: 88.62%\n",
      "Progress: 88.62%\n",
      "Progress: 88.63%\n",
      "Progress: 88.63%\n",
      "Progress: 88.64%\n",
      "Progress: 88.64%\n",
      "Progress: 88.65%\n",
      "Progress: 88.65%\n",
      "Progress: 88.66%\n",
      "Progress: 88.66%\n",
      "Progress: 88.66%\n",
      "Progress: 88.67%\n",
      "Progress: 88.67%\n",
      "Progress: 88.68%\n",
      "Progress: 88.68%\n",
      "Progress: 88.69%\n",
      "Progress: 88.69%\n",
      "Progress: 88.70%\n",
      "Progress: 88.70%\n",
      "Progress: 88.71%\n",
      "Progress: 88.71%\n",
      "Progress: 88.72%\n",
      "Progress: 88.72%\n",
      "Progress: 88.73%\n",
      "Progress: 88.73%\n",
      "Progress: 88.74%\n",
      "Progress: 88.74%\n",
      "Progress: 88.75%\n",
      "Progress: 88.75%\n",
      "Progress: 88.76%\n",
      "Progress: 88.76%\n",
      "Progress: 88.77%\n",
      "Progress: 88.77%\n",
      "Progress: 88.77%\n",
      "Progress: 88.78%\n",
      "Progress: 88.78%\n",
      "Progress: 88.79%\n",
      "Progress: 88.79%\n",
      "Progress: 88.80%\n",
      "Progress: 88.80%\n",
      "Progress: 88.81%\n",
      "Progress: 88.81%\n",
      "Progress: 88.82%\n",
      "Progress: 88.82%\n",
      "Progress: 88.83%\n",
      "Progress: 88.83%\n",
      "Progress: 88.84%\n",
      "Progress: 88.84%\n",
      "Progress: 88.85%\n",
      "Progress: 88.85%\n",
      "Progress: 88.86%\n",
      "Progress: 88.86%\n",
      "Progress: 88.87%\n",
      "Progress: 88.87%\n",
      "Progress: 88.87%\n",
      "Progress: 88.88%\n",
      "Progress: 88.88%\n",
      "Progress: 88.89%\n",
      "Progress: 88.89%\n",
      "Progress: 88.90%\n",
      "Progress: 88.90%\n",
      "Progress: 88.91%\n",
      "Progress: 88.91%\n",
      "Progress: 88.92%\n",
      "Progress: 88.92%\n",
      "Progress: 88.93%\n",
      "Progress: 88.93%\n",
      "Progress: 88.94%\n",
      "Progress: 88.94%\n",
      "Progress: 88.95%\n",
      "Progress: 88.95%\n",
      "Progress: 88.96%\n",
      "Progress: 88.96%\n",
      "Progress: 88.96%\n",
      "Progress: 88.97%\n",
      "Progress: 88.97%\n",
      "Progress: 88.98%\n",
      "Progress: 88.98%\n",
      "Progress: 88.99%\n",
      "Progress: 88.99%\n",
      "Progress: 89.00%\n",
      "Progress: 89.00%\n",
      "Progress: 89.01%\n",
      "Progress: 89.01%\n",
      "Progress: 89.02%\n",
      "Progress: 89.02%\n",
      "Progress: 89.03%\n",
      "Progress: 89.03%\n",
      "Progress: 89.04%\n",
      "Progress: 89.04%\n",
      "Progress: 89.04%\n",
      "Progress: 89.05%\n",
      "Progress: 89.05%\n",
      "Progress: 89.06%\n",
      "Progress: 89.06%\n",
      "Progress: 89.07%\n",
      "Progress: 89.07%\n",
      "Progress: 89.08%\n",
      "Progress: 89.08%\n",
      "Progress: 89.09%\n",
      "Progress: 89.09%\n",
      "Progress: 89.10%\n",
      "Progress: 89.10%\n",
      "Progress: 89.11%\n",
      "Progress: 89.11%\n",
      "Progress: 89.12%\n",
      "Progress: 89.12%\n",
      "Progress: 89.12%\n",
      "Progress: 89.13%\n",
      "Progress: 89.13%\n",
      "Progress: 89.14%\n",
      "Progress: 89.14%\n",
      "Progress: 89.15%\n",
      "Progress: 89.15%\n",
      "Progress: 89.16%\n",
      "Progress: 89.16%\n",
      "Progress: 89.17%\n",
      "Progress: 89.17%\n",
      "Progress: 89.18%\n",
      "Progress: 89.18%\n",
      "Progress: 89.19%\n",
      "Progress: 89.19%\n",
      "Progress: 89.20%\n",
      "Progress: 89.20%\n",
      "Progress: 89.20%\n",
      "Progress: 89.21%\n",
      "Progress: 89.21%\n",
      "Progress: 89.22%\n",
      "Progress: 89.22%\n",
      "Progress: 89.23%\n",
      "Progress: 89.23%\n",
      "Progress: 89.24%\n",
      "Progress: 89.24%\n",
      "Progress: 89.25%\n",
      "Progress: 89.25%\n",
      "Progress: 89.26%\n",
      "Progress: 89.26%\n",
      "Progress: 89.27%\n",
      "Progress: 89.27%\n",
      "Progress: 89.27%\n",
      "Progress: 89.28%\n",
      "Progress: 89.28%\n",
      "Progress: 89.29%\n",
      "Progress: 89.29%\n",
      "Progress: 89.30%\n",
      "Progress: 89.30%\n",
      "Progress: 89.31%\n",
      "Progress: 89.31%\n",
      "Progress: 89.32%\n",
      "Progress: 89.32%\n",
      "Progress: 89.33%\n",
      "Progress: 89.33%\n",
      "Progress: 89.33%\n",
      "Progress: 89.34%\n",
      "Progress: 89.34%\n",
      "Progress: 89.35%\n",
      "Progress: 89.35%\n",
      "Progress: 89.36%\n",
      "Progress: 89.36%\n",
      "Progress: 89.37%\n",
      "Progress: 89.37%\n",
      "Progress: 89.38%\n",
      "Progress: 89.38%\n",
      "Progress: 89.39%\n",
      "Progress: 89.39%\n",
      "Progress: 89.40%\n",
      "Progress: 89.40%\n",
      "Progress: 89.40%\n",
      "Progress: 89.41%\n",
      "Progress: 89.41%\n",
      "Progress: 89.42%\n",
      "Progress: 89.42%\n",
      "Progress: 89.43%\n",
      "Progress: 89.43%\n",
      "Progress: 89.44%\n",
      "Progress: 89.44%\n",
      "Progress: 89.45%\n",
      "Progress: 89.45%\n",
      "Progress: 89.46%\n",
      "Progress: 89.46%\n",
      "Progress: 89.46%\n",
      "Progress: 89.47%\n",
      "Progress: 89.47%\n",
      "Progress: 89.48%\n",
      "Progress: 89.48%\n",
      "Progress: 89.49%\n",
      "Progress: 89.49%\n",
      "Progress: 89.50%\n",
      "Progress: 89.50%\n",
      "Progress: 89.51%\n",
      "Progress: 89.51%\n",
      "Progress: 89.52%\n",
      "Progress: 89.52%\n",
      "Progress: 89.52%\n",
      "Progress: 89.53%\n",
      "Progress: 89.53%\n",
      "Progress: 89.54%\n",
      "Progress: 89.54%\n",
      "Progress: 89.55%\n",
      "Progress: 89.55%\n",
      "Progress: 89.56%\n",
      "Progress: 89.56%\n",
      "Progress: 89.57%\n",
      "Progress: 89.57%\n",
      "Progress: 89.58%\n",
      "Progress: 89.58%\n",
      "Progress: 89.58%\n",
      "Progress: 89.59%\n",
      "Progress: 89.59%\n",
      "Progress: 89.60%\n",
      "Progress: 89.60%\n",
      "Progress: 89.61%\n",
      "Progress: 89.61%\n",
      "Progress: 89.62%\n",
      "Progress: 89.62%\n",
      "Progress: 89.63%\n",
      "Progress: 89.63%\n",
      "Progress: 89.63%\n",
      "Progress: 89.64%\n",
      "Progress: 89.64%\n",
      "Progress: 89.65%\n",
      "Progress: 89.65%\n",
      "Progress: 89.66%\n",
      "Progress: 89.66%\n",
      "Progress: 89.67%\n",
      "Progress: 89.67%\n",
      "Progress: 89.68%\n",
      "Progress: 89.68%\n",
      "Progress: 89.69%\n",
      "Progress: 89.69%\n",
      "Progress: 89.69%\n",
      "Progress: 89.70%\n",
      "Progress: 89.70%\n",
      "Progress: 89.71%\n",
      "Progress: 89.71%\n",
      "Progress: 89.72%\n",
      "Progress: 89.72%\n",
      "Progress: 89.73%\n",
      "Progress: 89.73%\n",
      "Progress: 89.74%\n",
      "Progress: 89.74%\n",
      "Progress: 89.74%\n",
      "Progress: 89.75%\n",
      "Progress: 89.75%\n",
      "Progress: 89.76%\n",
      "Progress: 89.76%\n",
      "Progress: 89.77%\n",
      "Progress: 89.77%\n",
      "Progress: 89.78%\n",
      "Progress: 89.78%\n",
      "Progress: 89.79%\n",
      "Progress: 89.79%\n",
      "Progress: 89.79%\n",
      "Progress: 89.80%\n",
      "Progress: 89.80%\n",
      "Progress: 89.81%\n",
      "Progress: 89.81%\n",
      "Progress: 89.82%\n",
      "Progress: 89.82%\n",
      "Progress: 89.83%\n",
      "Progress: 89.83%\n",
      "Progress: 89.84%\n",
      "Progress: 89.84%\n",
      "Progress: 89.84%\n",
      "Progress: 89.85%\n",
      "Progress: 89.85%\n",
      "Progress: 89.86%\n",
      "Progress: 89.86%\n",
      "Progress: 89.87%\n",
      "Progress: 89.87%\n",
      "Progress: 89.88%\n",
      "Progress: 89.88%\n",
      "Progress: 89.89%\n",
      "Progress: 89.89%\n",
      "Progress: 89.89%\n",
      "Progress: 89.90%\n",
      "Progress: 89.90%\n",
      "Progress: 89.91%\n",
      "Progress: 89.91%\n",
      "Progress: 89.92%\n",
      "Progress: 89.92%\n",
      "Progress: 89.93%\n",
      "Progress: 89.93%\n",
      "Progress: 89.94%\n",
      "Progress: 89.94%\n",
      "Progress: 89.94%\n",
      "Progress: 89.95%\n",
      "Progress: 89.95%\n",
      "Progress: 89.96%\n",
      "Progress: 89.96%\n",
      "Progress: 89.97%\n",
      "Progress: 89.97%\n",
      "Progress: 89.98%\n",
      "Progress: 89.98%\n",
      "Progress: 89.98%\n",
      "Progress: 89.99%\n",
      "Progress: 89.99%\n",
      "Progress: 90.00%\n",
      "Progress: 90.00%\n",
      "Progress: 90.01%\n",
      "Progress: 90.01%\n",
      "Progress: 90.02%\n",
      "Progress: 90.02%\n",
      "Progress: 90.03%\n",
      "Progress: 90.03%\n",
      "Progress: 90.03%\n",
      "Progress: 90.04%\n",
      "Progress: 90.04%\n",
      "Progress: 90.05%\n",
      "Progress: 90.05%\n",
      "Progress: 90.06%\n",
      "Progress: 90.06%\n",
      "Progress: 90.07%\n",
      "Progress: 90.07%\n",
      "Progress: 90.07%\n",
      "Progress: 90.08%\n",
      "Progress: 90.08%\n",
      "Progress: 90.09%\n",
      "Progress: 90.09%\n",
      "Progress: 90.10%\n",
      "Progress: 90.10%\n",
      "Progress: 90.11%\n",
      "Progress: 90.11%\n",
      "Progress: 90.11%\n",
      "Progress: 90.12%\n",
      "Progress: 90.12%\n",
      "Progress: 90.13%\n",
      "Progress: 90.13%\n",
      "Progress: 90.14%\n",
      "Progress: 90.14%\n",
      "Progress: 90.15%\n",
      "Progress: 90.15%\n",
      "Progress: 90.16%\n",
      "Progress: 90.16%\n",
      "Progress: 90.16%\n",
      "Progress: 90.17%\n",
      "Progress: 90.17%\n",
      "Progress: 90.18%\n",
      "Progress: 90.18%\n",
      "Progress: 90.19%\n",
      "Progress: 90.19%\n",
      "Progress: 90.20%\n",
      "Progress: 90.20%\n",
      "Progress: 90.20%\n",
      "Progress: 90.21%\n",
      "Progress: 90.21%\n",
      "Progress: 90.22%\n",
      "Progress: 90.22%\n",
      "Progress: 90.23%\n",
      "Progress: 90.23%\n",
      "Progress: 90.24%\n",
      "Progress: 90.24%\n",
      "Progress: 90.24%\n",
      "Progress: 90.25%\n",
      "Progress: 90.25%\n",
      "Progress: 90.26%\n",
      "Progress: 90.26%\n",
      "Progress: 90.27%\n",
      "Progress: 90.27%\n",
      "Progress: 90.28%\n",
      "Progress: 90.28%\n",
      "Progress: 90.28%\n",
      "Progress: 90.29%\n",
      "Progress: 90.29%\n",
      "Progress: 90.30%\n",
      "Progress: 90.30%\n",
      "Progress: 90.31%\n",
      "Progress: 90.31%\n",
      "Progress: 90.32%\n",
      "Progress: 90.32%\n",
      "Progress: 90.32%\n",
      "Progress: 90.33%\n",
      "Progress: 90.33%\n",
      "Progress: 90.34%\n",
      "Progress: 90.34%\n",
      "Progress: 90.35%\n",
      "Progress: 90.35%\n",
      "Progress: 90.36%\n",
      "Progress: 90.36%\n",
      "Progress: 90.36%\n",
      "Progress: 90.37%\n",
      "Progress: 90.37%\n",
      "Progress: 90.38%\n",
      "Progress: 90.38%\n",
      "Progress: 90.39%\n",
      "Progress: 90.39%\n",
      "Progress: 90.39%\n",
      "Progress: 90.40%\n",
      "Progress: 90.40%\n",
      "Progress: 90.41%\n",
      "Progress: 90.41%\n",
      "Progress: 90.42%\n",
      "Progress: 90.42%\n",
      "Progress: 90.43%\n",
      "Progress: 90.43%\n",
      "Progress: 90.43%\n",
      "Progress: 90.44%\n",
      "Progress: 90.44%\n",
      "Progress: 90.45%\n",
      "Progress: 90.45%\n",
      "Progress: 90.46%\n",
      "Progress: 90.46%\n",
      "Progress: 90.47%\n",
      "Progress: 90.47%\n",
      "Progress: 90.47%\n",
      "Progress: 90.48%\n",
      "Progress: 90.48%\n",
      "Progress: 90.49%\n",
      "Progress: 90.49%\n",
      "Progress: 90.50%\n",
      "Progress: 90.50%\n",
      "Progress: 90.50%\n",
      "Progress: 90.51%\n",
      "Progress: 90.51%\n",
      "Progress: 90.52%\n",
      "Progress: 90.52%\n",
      "Progress: 90.53%\n",
      "Progress: 90.53%\n",
      "Progress: 90.54%\n",
      "Progress: 90.54%\n",
      "Progress: 90.54%\n",
      "Progress: 90.55%\n",
      "Progress: 90.55%\n",
      "Progress: 90.56%\n",
      "Progress: 90.56%\n",
      "Progress: 90.57%\n",
      "Progress: 90.57%\n",
      "Progress: 90.57%\n",
      "Progress: 90.58%\n",
      "Progress: 90.58%\n",
      "Progress: 90.59%\n",
      "Progress: 90.59%\n",
      "Progress: 90.60%\n",
      "Progress: 90.60%\n",
      "Progress: 90.61%\n",
      "Progress: 90.61%\n",
      "Progress: 90.61%\n",
      "Progress: 90.62%\n",
      "Progress: 90.62%\n",
      "Progress: 90.63%\n",
      "Progress: 90.63%\n",
      "Progress: 90.64%\n",
      "Progress: 90.64%\n",
      "Progress: 90.64%\n",
      "Progress: 90.65%\n",
      "Progress: 90.65%\n",
      "Progress: 90.66%\n",
      "Progress: 90.66%\n",
      "Progress: 90.67%\n",
      "Progress: 90.67%\n",
      "Progress: 90.67%\n",
      "Progress: 90.68%\n",
      "Progress: 90.68%\n",
      "Progress: 90.69%\n",
      "Progress: 90.69%\n",
      "Progress: 90.70%\n",
      "Progress: 90.70%\n",
      "Progress: 90.71%\n",
      "Progress: 90.71%\n",
      "Progress: 90.71%\n",
      "Progress: 90.72%\n",
      "Progress: 90.72%\n",
      "Progress: 90.73%\n",
      "Progress: 90.73%\n",
      "Progress: 90.74%\n",
      "Progress: 90.74%\n",
      "Progress: 90.74%\n",
      "Progress: 90.75%\n",
      "Progress: 90.75%\n",
      "Progress: 90.76%\n",
      "Progress: 90.76%\n",
      "Progress: 90.77%\n",
      "Progress: 90.77%\n",
      "Progress: 90.77%\n",
      "Progress: 90.78%\n",
      "Progress: 90.78%\n",
      "Progress: 90.79%\n",
      "Progress: 90.79%\n",
      "Progress: 90.80%\n",
      "Progress: 90.80%\n",
      "Progress: 90.80%\n",
      "Progress: 90.81%\n",
      "Progress: 90.81%\n",
      "Progress: 90.82%\n",
      "Progress: 90.82%\n",
      "Progress: 90.83%\n",
      "Progress: 90.83%\n",
      "Progress: 90.84%\n",
      "Progress: 90.84%\n",
      "Progress: 90.84%\n",
      "Progress: 90.85%\n",
      "Progress: 90.85%\n",
      "Progress: 90.86%\n",
      "Progress: 90.86%\n",
      "Progress: 90.87%\n",
      "Progress: 90.87%\n",
      "Progress: 90.87%\n",
      "Progress: 90.88%\n",
      "Progress: 90.88%\n",
      "Progress: 90.89%\n",
      "Progress: 90.89%\n",
      "Progress: 90.90%\n",
      "Progress: 90.90%\n",
      "Progress: 90.90%\n",
      "Progress: 90.91%\n",
      "Progress: 90.91%\n",
      "Progress: 90.92%\n",
      "Progress: 90.92%\n",
      "Progress: 90.93%\n",
      "Progress: 90.93%\n",
      "Progress: 90.93%\n",
      "Progress: 90.94%\n",
      "Progress: 90.94%\n",
      "Progress: 90.95%\n",
      "Progress: 90.95%\n",
      "Progress: 90.96%\n",
      "Progress: 90.96%\n",
      "Progress: 90.96%\n",
      "Progress: 90.97%\n",
      "Progress: 90.97%\n",
      "Progress: 90.98%\n",
      "Progress: 90.98%\n",
      "Progress: 90.99%\n",
      "Progress: 90.99%\n",
      "Progress: 90.99%\n",
      "Progress: 91.00%\n",
      "Progress: 91.00%\n",
      "Progress: 91.01%\n",
      "Progress: 91.01%\n",
      "Progress: 91.02%\n",
      "Progress: 91.02%\n",
      "Progress: 91.02%\n",
      "Progress: 91.03%\n",
      "Progress: 91.03%\n",
      "Progress: 91.04%\n",
      "Progress: 91.04%\n",
      "Progress: 91.04%\n",
      "Progress: 91.05%\n",
      "Progress: 91.05%\n",
      "Progress: 91.06%\n",
      "Progress: 91.06%\n",
      "Progress: 91.07%\n",
      "Progress: 91.07%\n",
      "Progress: 91.07%\n",
      "Progress: 91.08%\n",
      "Progress: 91.08%\n",
      "Progress: 91.09%\n",
      "Progress: 91.09%\n",
      "Progress: 91.10%\n",
      "Progress: 91.10%\n",
      "Progress: 91.10%\n",
      "Progress: 91.11%\n",
      "Progress: 91.11%\n",
      "Progress: 91.12%\n",
      "Progress: 91.12%\n",
      "Progress: 91.13%\n",
      "Progress: 91.13%\n",
      "Progress: 91.13%\n",
      "Progress: 91.14%\n",
      "Progress: 91.14%\n",
      "Progress: 91.15%\n",
      "Progress: 91.15%\n",
      "Progress: 91.16%\n",
      "Progress: 91.16%\n",
      "Progress: 91.16%\n",
      "Progress: 91.17%\n",
      "Progress: 91.17%\n",
      "Progress: 91.18%\n",
      "Progress: 91.18%\n",
      "Progress: 91.18%\n",
      "Progress: 91.19%\n",
      "Progress: 91.19%\n",
      "Progress: 91.20%\n",
      "Progress: 91.20%\n",
      "Progress: 91.21%\n",
      "Progress: 91.21%\n",
      "Progress: 91.21%\n",
      "Progress: 91.22%\n",
      "Progress: 91.22%\n",
      "Progress: 91.23%\n",
      "Progress: 91.23%\n",
      "Progress: 91.24%\n",
      "Progress: 91.24%\n",
      "Progress: 91.24%\n",
      "Progress: 91.25%\n",
      "Progress: 91.25%\n",
      "Progress: 91.26%\n",
      "Progress: 91.26%\n",
      "Progress: 91.27%\n",
      "Progress: 91.27%\n",
      "Progress: 91.27%\n",
      "Progress: 91.28%\n",
      "Progress: 91.28%\n",
      "Progress: 91.29%\n",
      "Progress: 91.29%\n",
      "Progress: 91.29%\n",
      "Progress: 91.30%\n",
      "Progress: 91.30%\n",
      "Progress: 91.31%\n",
      "Progress: 91.31%\n",
      "Progress: 91.32%\n",
      "Progress: 91.32%\n",
      "Progress: 91.32%\n",
      "Progress: 91.33%\n",
      "Progress: 91.33%\n",
      "Progress: 91.34%\n",
      "Progress: 91.34%\n",
      "Progress: 91.34%\n",
      "Progress: 91.35%\n",
      "Progress: 91.35%\n",
      "Progress: 91.36%\n",
      "Progress: 91.36%\n",
      "Progress: 91.37%\n",
      "Progress: 91.37%\n",
      "Progress: 91.37%\n",
      "Progress: 91.38%\n",
      "Progress: 91.38%\n",
      "Progress: 91.39%\n",
      "Progress: 91.39%\n",
      "Progress: 91.39%\n",
      "Progress: 91.40%\n",
      "Progress: 91.40%\n",
      "Progress: 91.41%\n",
      "Progress: 91.41%\n",
      "Progress: 91.42%\n",
      "Progress: 91.42%\n",
      "Progress: 91.42%\n",
      "Progress: 91.43%\n",
      "Progress: 91.43%\n",
      "Progress: 91.44%\n",
      "Progress: 91.44%\n",
      "Progress: 91.44%\n",
      "Progress: 91.45%\n",
      "Progress: 91.45%\n",
      "Progress: 91.46%\n",
      "Progress: 91.46%\n",
      "Progress: 91.47%\n",
      "Progress: 91.47%\n",
      "Progress: 91.47%\n",
      "Progress: 91.48%\n",
      "Progress: 91.48%\n",
      "Progress: 91.49%\n",
      "Progress: 91.49%\n",
      "Progress: 91.49%\n",
      "Progress: 91.50%\n",
      "Progress: 91.50%\n",
      "Progress: 91.51%\n",
      "Progress: 91.51%\n",
      "Progress: 91.52%\n",
      "Progress: 91.52%\n",
      "Progress: 91.52%\n",
      "Progress: 91.53%\n",
      "Progress: 91.53%\n",
      "Progress: 91.54%\n",
      "Progress: 91.54%\n",
      "Progress: 91.54%\n",
      "Progress: 91.55%\n",
      "Progress: 91.55%\n",
      "Progress: 91.56%\n",
      "Progress: 91.56%\n",
      "Progress: 91.57%\n",
      "Progress: 91.57%\n",
      "Progress: 91.57%\n",
      "Progress: 91.58%\n",
      "Progress: 91.58%\n",
      "Progress: 91.59%\n",
      "Progress: 91.59%\n",
      "Progress: 91.59%\n",
      "Progress: 91.60%\n",
      "Progress: 91.60%\n",
      "Progress: 91.61%\n",
      "Progress: 91.61%\n",
      "Progress: 91.61%\n",
      "Progress: 91.62%\n",
      "Progress: 91.62%\n",
      "Progress: 91.63%\n",
      "Progress: 91.63%\n",
      "Progress: 91.64%\n",
      "Progress: 91.64%\n",
      "Progress: 91.64%\n",
      "Progress: 91.65%\n",
      "Progress: 91.65%\n",
      "Progress: 91.66%\n",
      "Progress: 91.66%\n",
      "Progress: 91.66%\n",
      "Progress: 91.67%\n",
      "Progress: 91.67%\n",
      "Progress: 91.68%\n",
      "Progress: 91.68%\n",
      "Progress: 91.68%\n",
      "Progress: 91.69%\n",
      "Progress: 91.69%\n",
      "Progress: 91.70%\n",
      "Progress: 91.70%\n",
      "Progress: 91.71%\n",
      "Progress: 91.71%\n",
      "Progress: 91.71%\n",
      "Progress: 91.72%\n",
      "Progress: 91.72%\n",
      "Progress: 91.73%\n",
      "Progress: 91.73%\n",
      "Progress: 91.73%\n",
      "Progress: 91.74%\n",
      "Progress: 91.74%\n",
      "Progress: 91.75%\n",
      "Progress: 91.75%\n",
      "Progress: 91.75%\n",
      "Progress: 91.76%\n",
      "Progress: 91.76%\n",
      "Progress: 91.77%\n",
      "Progress: 91.77%\n",
      "Progress: 91.77%\n",
      "Progress: 91.78%\n",
      "Progress: 91.78%\n",
      "Progress: 91.79%\n",
      "Progress: 91.79%\n",
      "Progress: 91.80%\n",
      "Progress: 91.80%\n",
      "Progress: 91.80%\n",
      "Progress: 91.81%\n",
      "Progress: 91.81%\n",
      "Progress: 91.82%\n",
      "Progress: 91.82%\n",
      "Progress: 91.82%\n",
      "Progress: 91.83%\n",
      "Progress: 91.83%\n",
      "Progress: 91.84%\n",
      "Progress: 91.84%\n",
      "Progress: 91.84%\n",
      "Progress: 91.85%\n",
      "Progress: 91.85%\n",
      "Progress: 91.86%\n",
      "Progress: 91.86%\n",
      "Progress: 91.86%\n",
      "Progress: 91.87%\n",
      "Progress: 91.87%\n",
      "Progress: 91.88%\n",
      "Progress: 91.88%\n",
      "Progress: 91.88%\n",
      "Progress: 91.89%\n",
      "Progress: 91.89%\n",
      "Progress: 91.90%\n",
      "Progress: 91.90%\n",
      "Progress: 91.90%\n",
      "Progress: 91.91%\n",
      "Progress: 91.91%\n",
      "Progress: 91.92%\n",
      "Progress: 91.92%\n",
      "Progress: 91.93%\n",
      "Progress: 91.93%\n",
      "Progress: 91.93%\n",
      "Progress: 91.94%\n",
      "Progress: 91.94%\n",
      "Progress: 91.95%\n",
      "Progress: 91.95%\n",
      "Progress: 91.95%\n",
      "Progress: 91.96%\n",
      "Progress: 91.96%\n",
      "Progress: 91.97%\n",
      "Progress: 91.97%\n",
      "Progress: 91.97%\n",
      "Progress: 91.98%\n",
      "Progress: 91.98%\n",
      "Progress: 91.99%\n",
      "Progress: 91.99%\n",
      "Progress: 91.99%\n",
      "Progress: 92.00%\n",
      "Progress: 92.00%\n",
      "Progress: 92.01%\n",
      "Progress: 92.01%\n",
      "Progress: 92.01%\n",
      "Progress: 92.02%\n",
      "Progress: 92.02%\n",
      "Progress: 92.03%\n",
      "Progress: 92.03%\n",
      "Progress: 92.03%\n",
      "Progress: 92.04%\n",
      "Progress: 92.04%\n",
      "Progress: 92.05%\n",
      "Progress: 92.05%\n",
      "Progress: 92.05%\n",
      "Progress: 92.06%\n",
      "Progress: 92.06%\n",
      "Progress: 92.07%\n",
      "Progress: 92.07%\n",
      "Progress: 92.07%\n",
      "Progress: 92.08%\n",
      "Progress: 92.08%\n",
      "Progress: 92.09%\n",
      "Progress: 92.09%\n",
      "Progress: 92.09%\n",
      "Progress: 92.10%\n",
      "Progress: 92.10%\n",
      "Progress: 92.11%\n",
      "Progress: 92.11%\n",
      "Progress: 92.11%\n",
      "Progress: 92.12%\n",
      "Progress: 92.12%\n",
      "Progress: 92.13%\n",
      "Progress: 92.13%\n",
      "Progress: 92.13%\n",
      "Progress: 92.14%\n",
      "Progress: 92.14%\n",
      "Progress: 92.15%\n",
      "Progress: 92.15%\n",
      "Progress: 92.15%\n",
      "Progress: 92.16%\n",
      "Progress: 92.16%\n",
      "Progress: 92.17%\n",
      "Progress: 92.17%\n",
      "Progress: 92.17%\n",
      "Progress: 92.18%\n",
      "Progress: 92.18%\n",
      "Progress: 92.19%\n",
      "Progress: 92.19%\n",
      "Progress: 92.19%\n",
      "Progress: 92.20%\n",
      "Progress: 92.20%\n",
      "Progress: 92.21%\n",
      "Progress: 92.21%\n",
      "Progress: 92.21%\n",
      "Progress: 92.22%\n",
      "Progress: 92.22%\n",
      "Progress: 92.23%\n",
      "Progress: 92.23%\n",
      "Progress: 92.23%\n",
      "Progress: 92.24%\n",
      "Progress: 92.24%\n",
      "Progress: 92.25%\n",
      "Progress: 92.25%\n",
      "Progress: 92.25%\n",
      "Progress: 92.26%\n",
      "Progress: 92.26%\n",
      "Progress: 92.27%\n",
      "Progress: 92.27%\n",
      "Progress: 92.27%\n",
      "Progress: 92.28%\n",
      "Progress: 92.28%\n",
      "Progress: 92.28%\n",
      "Progress: 92.29%\n",
      "Progress: 92.29%\n",
      "Progress: 92.30%\n",
      "Progress: 92.30%\n",
      "Progress: 92.30%\n",
      "Progress: 92.31%\n",
      "Progress: 92.31%\n",
      "Progress: 92.32%\n",
      "Progress: 92.32%\n",
      "Progress: 92.32%\n",
      "Progress: 92.33%\n",
      "Progress: 92.33%\n",
      "Progress: 92.34%\n",
      "Progress: 92.34%\n",
      "Progress: 92.34%\n",
      "Progress: 92.35%\n",
      "Progress: 92.35%\n",
      "Progress: 92.36%\n",
      "Progress: 92.36%\n",
      "Progress: 92.36%\n",
      "Progress: 92.37%\n",
      "Progress: 92.37%\n",
      "Progress: 92.38%\n",
      "Progress: 92.38%\n",
      "Progress: 92.38%\n",
      "Progress: 92.39%\n",
      "Progress: 92.39%\n",
      "Progress: 92.40%\n",
      "Progress: 92.40%\n",
      "Progress: 92.40%\n",
      "Progress: 92.41%\n",
      "Progress: 92.41%\n",
      "Progress: 92.41%\n",
      "Progress: 92.42%\n",
      "Progress: 92.42%\n",
      "Progress: 92.43%\n",
      "Progress: 92.43%\n",
      "Progress: 92.43%\n",
      "Progress: 92.44%\n",
      "Progress: 92.44%\n",
      "Progress: 92.45%\n",
      "Progress: 92.45%\n",
      "Progress: 92.45%\n",
      "Progress: 92.46%\n",
      "Progress: 92.46%\n",
      "Progress: 92.47%\n",
      "Progress: 92.47%\n",
      "Progress: 92.47%\n",
      "Progress: 92.48%\n",
      "Progress: 92.48%\n",
      "Progress: 92.49%\n",
      "Progress: 92.49%\n",
      "Progress: 92.49%\n",
      "Progress: 92.50%\n",
      "Progress: 92.50%\n",
      "Progress: 92.50%\n",
      "Progress: 92.51%\n",
      "Progress: 92.51%\n",
      "Progress: 92.52%\n",
      "Progress: 92.52%\n",
      "Progress: 92.52%\n",
      "Progress: 92.53%\n",
      "Progress: 92.53%\n",
      "Progress: 92.54%\n",
      "Progress: 92.54%\n",
      "Progress: 92.54%\n",
      "Progress: 92.55%\n",
      "Progress: 92.55%\n",
      "Progress: 92.56%\n",
      "Progress: 92.56%\n",
      "Progress: 92.56%\n",
      "Progress: 92.57%\n",
      "Progress: 92.57%\n",
      "Progress: 92.57%\n",
      "Progress: 92.58%\n",
      "Progress: 92.58%\n",
      "Progress: 92.59%\n",
      "Progress: 92.59%\n",
      "Progress: 92.59%\n",
      "Progress: 92.60%\n",
      "Progress: 92.60%\n",
      "Progress: 92.61%\n",
      "Progress: 92.61%\n",
      "Progress: 92.61%\n",
      "Progress: 92.62%\n",
      "Progress: 92.62%\n",
      "Progress: 92.63%\n",
      "Progress: 92.63%\n",
      "Progress: 92.63%\n",
      "Progress: 92.64%\n",
      "Progress: 92.64%\n",
      "Progress: 92.64%\n",
      "Progress: 92.65%\n",
      "Progress: 92.65%\n",
      "Progress: 92.66%\n",
      "Progress: 92.66%\n",
      "Progress: 92.66%\n",
      "Progress: 92.67%\n",
      "Progress: 92.67%\n",
      "Progress: 92.68%\n",
      "Progress: 92.68%\n",
      "Progress: 92.68%\n",
      "Progress: 92.69%\n",
      "Progress: 92.69%\n",
      "Progress: 92.69%\n",
      "Progress: 92.70%\n",
      "Progress: 92.70%\n",
      "Progress: 92.71%\n",
      "Progress: 92.71%\n",
      "Progress: 92.71%\n",
      "Progress: 92.72%\n",
      "Progress: 92.72%\n",
      "Progress: 92.73%\n",
      "Progress: 92.73%\n",
      "Progress: 92.73%\n",
      "Progress: 92.74%\n",
      "Progress: 92.74%\n",
      "Progress: 92.74%\n",
      "Progress: 92.75%\n",
      "Progress: 92.75%\n",
      "Progress: 92.76%\n",
      "Progress: 92.76%\n",
      "Progress: 92.76%\n",
      "Progress: 92.77%\n",
      "Progress: 92.77%\n",
      "Progress: 92.77%\n",
      "Progress: 92.78%\n",
      "Progress: 92.78%\n",
      "Progress: 92.79%\n",
      "Progress: 92.79%\n",
      "Progress: 92.79%\n",
      "Progress: 92.80%\n",
      "Progress: 92.80%\n",
      "Progress: 92.81%\n",
      "Progress: 92.81%\n",
      "Progress: 92.81%\n",
      "Progress: 92.82%\n",
      "Progress: 92.82%\n",
      "Progress: 92.82%\n",
      "Progress: 92.83%\n",
      "Progress: 92.83%\n",
      "Progress: 92.84%\n",
      "Progress: 92.84%\n",
      "Progress: 92.84%\n",
      "Progress: 92.85%\n",
      "Progress: 92.85%\n",
      "Progress: 92.86%\n",
      "Progress: 92.86%\n",
      "Progress: 92.86%\n",
      "Progress: 92.87%\n",
      "Progress: 92.87%\n",
      "Progress: 92.87%\n",
      "Progress: 92.88%\n",
      "Progress: 92.88%\n",
      "Progress: 92.89%\n",
      "Progress: 92.89%\n",
      "Progress: 92.89%\n",
      "Progress: 92.90%\n",
      "Progress: 92.90%\n",
      "Progress: 92.90%\n",
      "Progress: 92.91%\n",
      "Progress: 92.91%\n",
      "Progress: 92.92%\n",
      "Progress: 92.92%\n",
      "Progress: 92.92%\n",
      "Progress: 92.93%\n",
      "Progress: 92.93%\n",
      "Progress: 92.93%\n",
      "Progress: 92.94%\n",
      "Progress: 92.94%\n",
      "Progress: 92.95%\n",
      "Progress: 92.95%\n",
      "Progress: 92.95%\n",
      "Progress: 92.96%\n",
      "Progress: 92.96%\n",
      "Progress: 92.96%\n",
      "Progress: 92.97%\n",
      "Progress: 92.97%\n",
      "Progress: 92.98%\n",
      "Progress: 92.98%\n",
      "Progress: 92.98%\n",
      "Progress: 92.99%\n",
      "Progress: 92.99%\n",
      "Progress: 93.00%\n",
      "Progress: 93.00%\n",
      "Progress: 93.00%\n",
      "Progress: 93.01%\n",
      "Progress: 93.01%\n",
      "Progress: 93.01%\n",
      "Progress: 93.02%\n",
      "Progress: 93.02%\n",
      "Progress: 93.03%\n",
      "Progress: 93.03%\n",
      "Progress: 93.03%\n",
      "Progress: 93.04%\n",
      "Progress: 93.04%\n",
      "Progress: 93.04%\n",
      "Progress: 93.05%\n",
      "Progress: 93.05%\n",
      "Progress: 93.06%\n",
      "Progress: 93.06%\n",
      "Progress: 93.06%\n",
      "Progress: 93.07%\n",
      "Progress: 93.07%\n",
      "Progress: 93.07%\n",
      "Progress: 93.08%\n",
      "Progress: 93.08%\n",
      "Progress: 93.09%\n",
      "Progress: 93.09%\n",
      "Progress: 93.09%\n",
      "Progress: 93.10%\n",
      "Progress: 93.10%\n",
      "Progress: 93.10%\n",
      "Progress: 93.11%\n",
      "Progress: 93.11%\n",
      "Progress: 93.12%\n",
      "Progress: 93.12%\n",
      "Progress: 93.12%\n",
      "Progress: 93.13%\n",
      "Progress: 93.13%\n",
      "Progress: 93.13%\n",
      "Progress: 93.14%\n",
      "Progress: 93.14%\n",
      "Progress: 93.14%\n",
      "Progress: 93.15%\n",
      "Progress: 93.15%\n",
      "Progress: 93.16%\n",
      "Progress: 93.16%\n",
      "Progress: 93.16%\n",
      "Progress: 93.17%\n",
      "Progress: 93.17%\n",
      "Progress: 93.17%\n",
      "Progress: 93.18%\n",
      "Progress: 93.18%\n",
      "Progress: 93.19%\n",
      "Progress: 93.19%\n",
      "Progress: 93.19%\n",
      "Progress: 93.20%\n",
      "Progress: 93.20%\n",
      "Progress: 93.20%\n",
      "Progress: 93.21%\n",
      "Progress: 93.21%\n",
      "Progress: 93.22%\n",
      "Progress: 93.22%\n",
      "Progress: 93.22%\n",
      "Progress: 93.23%\n",
      "Progress: 93.23%\n",
      "Progress: 93.23%\n",
      "Progress: 93.24%\n",
      "Progress: 93.24%\n",
      "Progress: 93.25%\n",
      "Progress: 93.25%\n",
      "Progress: 93.25%\n",
      "Progress: 93.26%\n",
      "Progress: 93.26%\n",
      "Progress: 93.26%\n",
      "Progress: 93.27%\n",
      "Progress: 93.27%\n",
      "Progress: 93.27%\n",
      "Progress: 93.28%\n",
      "Progress: 93.28%\n",
      "Progress: 93.29%\n",
      "Progress: 93.29%\n",
      "Progress: 93.29%\n",
      "Progress: 93.30%\n",
      "Progress: 93.30%\n",
      "Progress: 93.30%\n",
      "Progress: 93.31%\n",
      "Progress: 93.31%\n",
      "Progress: 93.32%\n",
      "Progress: 93.32%\n",
      "Progress: 93.32%\n",
      "Progress: 93.33%\n",
      "Progress: 93.33%\n",
      "Progress: 93.33%\n",
      "Progress: 93.34%\n",
      "Progress: 93.34%\n",
      "Progress: 93.34%\n",
      "Progress: 93.35%\n",
      "Progress: 93.35%\n",
      "Progress: 93.36%\n",
      "Progress: 93.36%\n",
      "Progress: 93.36%\n",
      "Progress: 93.37%\n",
      "Progress: 93.37%\n",
      "Progress: 93.37%\n",
      "Progress: 93.38%\n",
      "Progress: 93.38%\n",
      "Progress: 93.39%\n",
      "Progress: 93.39%\n",
      "Progress: 93.39%\n",
      "Progress: 93.40%\n",
      "Progress: 93.40%\n",
      "Progress: 93.40%\n",
      "Progress: 93.41%\n",
      "Progress: 93.41%\n",
      "Progress: 93.41%\n",
      "Progress: 93.42%\n",
      "Progress: 93.42%\n",
      "Progress: 93.43%\n",
      "Progress: 93.43%\n",
      "Progress: 93.43%\n",
      "Progress: 93.44%\n",
      "Progress: 93.44%\n",
      "Progress: 93.44%\n",
      "Progress: 93.45%\n",
      "Progress: 93.45%\n",
      "Progress: 93.45%\n",
      "Progress: 93.46%\n",
      "Progress: 93.46%\n",
      "Progress: 93.47%\n",
      "Progress: 93.47%\n",
      "Progress: 93.47%\n",
      "Progress: 93.48%\n",
      "Progress: 93.48%\n",
      "Progress: 93.48%\n",
      "Progress: 93.49%\n",
      "Progress: 93.49%\n",
      "Progress: 93.49%\n",
      "Progress: 93.50%\n",
      "Progress: 93.50%\n",
      "Progress: 93.51%\n",
      "Progress: 93.51%\n",
      "Progress: 93.51%\n",
      "Progress: 93.52%\n",
      "Progress: 93.52%\n",
      "Progress: 93.52%\n",
      "Progress: 93.53%\n",
      "Progress: 93.53%\n",
      "Progress: 93.53%\n",
      "Progress: 93.54%\n",
      "Progress: 93.54%\n",
      "Progress: 93.55%\n",
      "Progress: 93.55%\n",
      "Progress: 93.55%\n",
      "Progress: 93.56%\n",
      "Progress: 93.56%\n",
      "Progress: 93.56%\n",
      "Progress: 93.57%\n",
      "Progress: 93.57%\n",
      "Progress: 93.57%\n",
      "Progress: 93.58%\n",
      "Progress: 93.58%\n",
      "Progress: 93.58%\n",
      "Progress: 93.59%\n",
      "Progress: 93.59%\n",
      "Progress: 93.60%\n",
      "Progress: 93.60%\n",
      "Progress: 93.60%\n",
      "Progress: 93.61%\n",
      "Progress: 93.61%\n",
      "Progress: 93.61%\n",
      "Progress: 93.62%\n",
      "Progress: 93.62%\n",
      "Progress: 93.62%\n",
      "Progress: 93.63%\n",
      "Progress: 93.63%\n",
      "Progress: 93.64%\n",
      "Progress: 93.64%\n",
      "Progress: 93.64%\n",
      "Progress: 93.65%\n",
      "Progress: 93.65%\n",
      "Progress: 93.65%\n",
      "Progress: 93.66%\n",
      "Progress: 93.66%\n",
      "Progress: 93.66%\n",
      "Progress: 93.67%\n",
      "Progress: 93.67%\n",
      "Progress: 93.67%\n",
      "Progress: 93.68%\n",
      "Progress: 93.68%\n",
      "Progress: 93.69%\n",
      "Progress: 93.69%\n",
      "Progress: 93.69%\n",
      "Progress: 93.70%\n",
      "Progress: 93.70%\n",
      "Progress: 93.70%\n",
      "Progress: 93.71%\n",
      "Progress: 93.71%\n",
      "Progress: 93.71%\n",
      "Progress: 93.72%\n",
      "Progress: 93.72%\n",
      "Progress: 93.72%\n",
      "Progress: 93.73%\n",
      "Progress: 93.73%\n",
      "Progress: 93.74%\n",
      "Progress: 93.74%\n",
      "Progress: 93.74%\n",
      "Progress: 93.75%\n",
      "Progress: 93.75%\n",
      "Progress: 93.75%\n",
      "Progress: 93.76%\n",
      "Progress: 93.76%\n",
      "Progress: 93.76%\n",
      "Progress: 93.77%\n",
      "Progress: 93.77%\n",
      "Progress: 93.77%\n",
      "Progress: 93.78%\n",
      "Progress: 93.78%\n",
      "Progress: 93.79%\n",
      "Progress: 93.79%\n",
      "Progress: 93.79%\n",
      "Progress: 93.80%\n",
      "Progress: 93.80%\n",
      "Progress: 93.80%\n",
      "Progress: 93.81%\n",
      "Progress: 93.81%\n",
      "Progress: 93.81%\n",
      "Progress: 93.82%\n",
      "Progress: 93.82%\n",
      "Progress: 93.82%\n",
      "Progress: 93.83%\n",
      "Progress: 93.83%\n",
      "Progress: 93.83%\n",
      "Progress: 93.84%\n",
      "Progress: 93.84%\n",
      "Progress: 93.85%\n",
      "Progress: 93.85%\n",
      "Progress: 93.85%\n",
      "Progress: 93.86%\n",
      "Progress: 93.86%\n",
      "Progress: 93.86%\n",
      "Progress: 93.87%\n",
      "Progress: 93.87%\n",
      "Progress: 93.87%\n",
      "Progress: 93.88%\n",
      "Progress: 93.88%\n",
      "Progress: 93.88%\n",
      "Progress: 93.89%\n",
      "Progress: 93.89%\n",
      "Progress: 93.89%\n",
      "Progress: 93.90%\n",
      "Progress: 93.90%\n",
      "Progress: 93.91%\n",
      "Progress: 93.91%\n",
      "Progress: 93.91%\n",
      "Progress: 93.92%\n",
      "Progress: 93.92%\n",
      "Progress: 93.92%\n",
      "Progress: 93.93%\n",
      "Progress: 93.93%\n",
      "Progress: 93.93%\n",
      "Progress: 93.94%\n",
      "Progress: 93.94%\n",
      "Progress: 93.94%\n",
      "Progress: 93.95%\n",
      "Progress: 93.95%\n",
      "Progress: 93.95%\n",
      "Progress: 93.96%\n",
      "Progress: 93.96%\n",
      "Progress: 93.96%\n",
      "Progress: 93.97%\n",
      "Progress: 93.97%\n",
      "Progress: 93.98%\n",
      "Progress: 93.98%\n",
      "Progress: 93.98%\n",
      "Progress: 93.99%\n",
      "Progress: 93.99%\n",
      "Progress: 93.99%\n",
      "Progress: 94.00%\n",
      "Progress: 94.00%\n",
      "Progress: 94.00%\n",
      "Progress: 94.01%\n",
      "Progress: 94.01%\n",
      "Progress: 94.01%\n",
      "Progress: 94.02%\n",
      "Progress: 94.02%\n",
      "Progress: 94.02%\n",
      "Progress: 94.03%\n",
      "Progress: 94.03%\n",
      "Progress: 94.03%\n",
      "Progress: 94.04%\n",
      "Progress: 94.04%\n",
      "Progress: 94.04%\n",
      "Progress: 94.05%\n",
      "Progress: 94.05%\n",
      "Progress: 94.06%\n",
      "Progress: 94.06%\n",
      "Progress: 94.06%\n",
      "Progress: 94.07%\n",
      "Progress: 94.07%\n",
      "Progress: 94.07%\n",
      "Progress: 94.08%\n",
      "Progress: 94.08%\n",
      "Progress: 94.08%\n",
      "Progress: 94.09%\n",
      "Progress: 94.09%\n",
      "Progress: 94.09%\n",
      "Progress: 94.10%\n",
      "Progress: 94.10%\n",
      "Progress: 94.10%\n",
      "Progress: 94.11%\n",
      "Progress: 94.11%\n",
      "Progress: 94.11%\n",
      "Progress: 94.12%\n",
      "Progress: 94.12%\n",
      "Progress: 94.12%\n",
      "Progress: 94.13%\n",
      "Progress: 94.13%\n",
      "Progress: 94.13%\n",
      "Progress: 94.14%\n",
      "Progress: 94.14%\n",
      "Progress: 94.15%\n",
      "Progress: 94.15%\n",
      "Progress: 94.15%\n",
      "Progress: 94.16%\n",
      "Progress: 94.16%\n",
      "Progress: 94.16%\n",
      "Progress: 94.17%\n",
      "Progress: 94.17%\n",
      "Progress: 94.17%\n",
      "Progress: 94.18%\n",
      "Progress: 94.18%\n",
      "Progress: 94.18%\n",
      "Progress: 94.19%\n",
      "Progress: 94.19%\n",
      "Progress: 94.19%\n",
      "Progress: 94.20%\n",
      "Progress: 94.20%\n",
      "Progress: 94.20%\n",
      "Progress: 94.21%\n",
      "Progress: 94.21%\n",
      "Progress: 94.21%\n",
      "Progress: 94.22%\n",
      "Progress: 94.22%\n",
      "Progress: 94.22%\n",
      "Progress: 94.23%\n",
      "Progress: 94.23%\n",
      "Progress: 94.23%\n",
      "Progress: 94.24%\n",
      "Progress: 94.24%\n",
      "Progress: 94.24%\n",
      "Progress: 94.25%\n",
      "Progress: 94.25%\n",
      "Progress: 94.25%\n",
      "Progress: 94.26%\n",
      "Progress: 94.26%\n",
      "Progress: 94.27%\n",
      "Progress: 94.27%\n",
      "Progress: 94.27%\n",
      "Progress: 94.28%\n",
      "Progress: 94.28%\n",
      "Progress: 94.28%\n",
      "Progress: 94.29%\n",
      "Progress: 94.29%\n",
      "Progress: 94.29%\n",
      "Progress: 94.30%\n",
      "Progress: 94.30%\n",
      "Progress: 94.30%\n",
      "Progress: 94.31%\n",
      "Progress: 94.31%\n",
      "Progress: 94.31%\n",
      "Progress: 94.32%\n",
      "Progress: 94.32%\n",
      "Progress: 94.32%\n",
      "Progress: 94.33%\n",
      "Progress: 94.33%\n",
      "Progress: 94.33%\n",
      "Progress: 94.34%\n",
      "Progress: 94.34%\n",
      "Progress: 94.34%\n",
      "Progress: 94.35%\n",
      "Progress: 94.35%\n",
      "Progress: 94.35%\n",
      "Progress: 94.36%\n",
      "Progress: 94.36%\n",
      "Progress: 94.36%\n",
      "Progress: 94.37%\n",
      "Progress: 94.37%\n",
      "Progress: 94.37%\n",
      "Progress: 94.38%\n",
      "Progress: 94.38%\n",
      "Progress: 94.38%\n",
      "Progress: 94.39%\n",
      "Progress: 94.39%\n",
      "Progress: 94.39%\n",
      "Progress: 94.40%\n",
      "Progress: 94.40%\n",
      "Progress: 94.40%\n",
      "Progress: 94.41%\n",
      "Progress: 94.41%\n",
      "Progress: 94.41%\n",
      "Progress: 94.42%\n",
      "Progress: 94.42%\n",
      "Progress: 94.42%\n",
      "Progress: 94.43%\n",
      "Progress: 94.43%\n",
      "Progress: 94.43%\n",
      "Progress: 94.44%\n",
      "Progress: 94.44%\n",
      "Progress: 94.44%\n",
      "Progress: 94.45%\n",
      "Progress: 94.45%\n",
      "Progress: 94.45%\n",
      "Progress: 94.46%\n",
      "Progress: 94.46%\n",
      "Progress: 94.46%\n",
      "Progress: 94.47%\n",
      "Progress: 94.47%\n",
      "Progress: 94.47%\n",
      "Progress: 94.48%\n",
      "Progress: 94.48%\n",
      "Progress: 94.48%\n",
      "Progress: 94.49%\n",
      "Progress: 94.49%\n",
      "Progress: 94.49%\n",
      "Progress: 94.50%\n",
      "Progress: 94.50%\n",
      "Progress: 94.50%\n",
      "Progress: 94.51%\n",
      "Progress: 94.51%\n",
      "Progress: 94.51%\n",
      "Progress: 94.52%\n",
      "Progress: 94.52%\n",
      "Progress: 94.52%\n",
      "Progress: 94.53%\n",
      "Progress: 94.53%\n",
      "Progress: 94.53%\n",
      "Progress: 94.54%\n",
      "Progress: 94.54%\n",
      "Progress: 94.54%\n",
      "Progress: 94.55%\n",
      "Progress: 94.55%\n",
      "Progress: 94.55%\n",
      "Progress: 94.56%\n",
      "Progress: 94.56%\n",
      "Progress: 94.56%\n",
      "Progress: 94.57%\n",
      "Progress: 94.57%\n",
      "Progress: 94.57%\n",
      "Progress: 94.58%\n",
      "Progress: 94.58%\n",
      "Progress: 94.58%\n",
      "Progress: 94.59%\n",
      "Progress: 94.59%\n",
      "Progress: 94.59%\n",
      "Progress: 94.60%\n",
      "Progress: 94.60%\n",
      "Progress: 94.60%\n",
      "Progress: 94.61%\n",
      "Progress: 94.61%\n",
      "Progress: 94.61%\n",
      "Progress: 94.62%\n",
      "Progress: 94.62%\n",
      "Progress: 94.62%\n",
      "Progress: 94.63%\n",
      "Progress: 94.63%\n",
      "Progress: 94.63%\n",
      "Progress: 94.64%\n",
      "Progress: 94.64%\n",
      "Progress: 94.64%\n",
      "Progress: 94.65%\n",
      "Progress: 94.65%\n",
      "Progress: 94.65%\n",
      "Progress: 94.66%\n",
      "Progress: 94.66%\n",
      "Progress: 94.66%\n",
      "Progress: 94.67%\n",
      "Progress: 94.67%\n",
      "Progress: 94.67%\n",
      "Progress: 94.68%\n",
      "Progress: 94.68%\n",
      "Progress: 94.68%\n",
      "Progress: 94.69%\n",
      "Progress: 94.69%\n",
      "Progress: 94.69%\n",
      "Progress: 94.70%\n",
      "Progress: 94.70%\n",
      "Progress: 94.70%\n",
      "Progress: 94.71%\n",
      "Progress: 94.71%\n",
      "Progress: 94.71%\n",
      "Progress: 94.72%\n",
      "Progress: 94.72%\n",
      "Progress: 94.72%\n",
      "Progress: 94.73%\n",
      "Progress: 94.73%\n",
      "Progress: 94.73%\n",
      "Progress: 94.74%\n",
      "Progress: 94.74%\n",
      "Progress: 94.74%\n",
      "Progress: 94.75%\n",
      "Progress: 94.75%\n",
      "Progress: 94.75%\n",
      "Progress: 94.76%\n",
      "Progress: 94.76%\n",
      "Progress: 94.76%\n",
      "Progress: 94.76%\n",
      "Progress: 94.77%\n",
      "Progress: 94.77%\n",
      "Progress: 94.77%\n",
      "Progress: 94.78%\n",
      "Progress: 94.78%\n",
      "Progress: 94.78%\n",
      "Progress: 94.79%\n",
      "Progress: 94.79%\n",
      "Progress: 94.79%\n",
      "Progress: 94.80%\n",
      "Progress: 94.80%\n",
      "Progress: 94.80%\n",
      "Progress: 94.81%\n",
      "Progress: 94.81%\n",
      "Progress: 94.81%\n",
      "Progress: 94.82%\n",
      "Progress: 94.82%\n",
      "Progress: 94.82%\n",
      "Progress: 94.83%\n",
      "Progress: 94.83%\n",
      "Progress: 94.83%\n",
      "Progress: 94.84%\n",
      "Progress: 94.84%\n",
      "Progress: 94.84%\n",
      "Progress: 94.85%\n",
      "Progress: 94.85%\n",
      "Progress: 94.85%\n",
      "Progress: 94.86%\n",
      "Progress: 94.86%\n",
      "Progress: 94.86%\n",
      "Progress: 94.87%\n",
      "Progress: 94.87%\n",
      "Progress: 94.87%\n",
      "Progress: 94.87%\n",
      "Progress: 94.88%\n",
      "Progress: 94.88%\n",
      "Progress: 94.88%\n",
      "Progress: 94.89%\n",
      "Progress: 94.89%\n",
      "Progress: 94.89%\n",
      "Progress: 94.90%\n",
      "Progress: 94.90%\n",
      "Progress: 94.90%\n",
      "Progress: 94.91%\n",
      "Progress: 94.91%\n",
      "Progress: 94.91%\n",
      "Progress: 94.92%\n",
      "Progress: 94.92%\n",
      "Progress: 94.92%\n",
      "Progress: 94.93%\n",
      "Progress: 94.93%\n",
      "Progress: 94.93%\n",
      "Progress: 94.94%\n",
      "Progress: 94.94%\n",
      "Progress: 94.94%\n",
      "Progress: 94.95%\n",
      "Progress: 94.95%\n",
      "Progress: 94.95%\n",
      "Progress: 94.96%\n",
      "Progress: 94.96%\n",
      "Progress: 94.96%\n",
      "Progress: 94.96%\n",
      "Progress: 94.97%\n",
      "Progress: 94.97%\n",
      "Progress: 94.97%\n",
      "Progress: 94.98%\n",
      "Progress: 94.98%\n",
      "Progress: 94.98%\n",
      "Progress: 94.99%\n",
      "Progress: 94.99%\n",
      "Progress: 94.99%\n",
      "Progress: 95.00%\n",
      "Progress: 95.00%\n",
      "Progress: 95.00%\n",
      "Progress: 95.01%\n",
      "Progress: 95.01%\n",
      "Progress: 95.01%\n",
      "Progress: 95.02%\n",
      "Progress: 95.02%\n",
      "Progress: 95.02%\n",
      "Progress: 95.03%\n",
      "Progress: 95.03%\n",
      "Progress: 95.03%\n",
      "Progress: 95.03%\n",
      "Progress: 95.04%\n",
      "Progress: 95.04%\n",
      "Progress: 95.04%\n",
      "Progress: 95.05%\n",
      "Progress: 95.05%\n",
      "Progress: 95.05%\n",
      "Progress: 95.06%\n",
      "Progress: 95.06%\n",
      "Progress: 95.06%\n",
      "Progress: 95.07%\n",
      "Progress: 95.07%\n",
      "Progress: 95.07%\n",
      "Progress: 95.08%\n",
      "Progress: 95.08%\n",
      "Progress: 95.08%\n",
      "Progress: 95.09%\n",
      "Progress: 95.09%\n",
      "Progress: 95.09%\n",
      "Progress: 95.09%\n",
      "Progress: 95.10%\n",
      "Progress: 95.10%\n",
      "Progress: 95.10%\n",
      "Progress: 95.11%\n",
      "Progress: 95.11%\n",
      "Progress: 95.11%\n",
      "Progress: 95.12%\n",
      "Progress: 95.12%\n",
      "Progress: 95.12%\n",
      "Progress: 95.13%\n",
      "Progress: 95.13%\n",
      "Progress: 95.13%\n",
      "Progress: 95.14%\n",
      "Progress: 95.14%\n",
      "Progress: 95.14%\n",
      "Progress: 95.15%\n",
      "Progress: 95.15%\n",
      "Progress: 95.15%\n",
      "Progress: 95.15%\n",
      "Progress: 95.16%\n",
      "Progress: 95.16%\n",
      "Progress: 95.16%\n",
      "Progress: 95.17%\n",
      "Progress: 95.17%\n",
      "Progress: 95.17%\n",
      "Progress: 95.18%\n",
      "Progress: 95.18%\n",
      "Progress: 95.18%\n",
      "Progress: 95.19%\n",
      "Progress: 95.19%\n",
      "Progress: 95.19%\n",
      "Progress: 95.20%\n",
      "Progress: 95.20%\n",
      "Progress: 95.20%\n",
      "Progress: 95.20%\n",
      "Progress: 95.21%\n",
      "Progress: 95.21%\n",
      "Progress: 95.21%\n",
      "Progress: 95.22%\n",
      "Progress: 95.22%\n",
      "Progress: 95.22%\n",
      "Progress: 95.23%\n",
      "Progress: 95.23%\n",
      "Progress: 95.23%\n",
      "Progress: 95.24%\n",
      "Progress: 95.24%\n",
      "Progress: 95.24%\n",
      "Progress: 95.25%\n",
      "Progress: 95.25%\n",
      "Progress: 95.25%\n",
      "Progress: 95.25%\n",
      "Progress: 95.26%\n",
      "Progress: 95.26%\n",
      "Progress: 95.26%\n",
      "Progress: 95.27%\n",
      "Progress: 95.27%\n",
      "Progress: 95.27%\n",
      "Progress: 95.28%\n",
      "Progress: 95.28%\n",
      "Progress: 95.28%\n",
      "Progress: 95.29%\n",
      "Progress: 95.29%\n",
      "Progress: 95.29%\n",
      "Progress: 95.29%\n",
      "Progress: 95.30%\n",
      "Progress: 95.30%\n",
      "Progress: 95.30%\n",
      "Progress: 95.31%\n",
      "Progress: 95.31%\n",
      "Progress: 95.31%\n",
      "Progress: 95.32%\n",
      "Progress: 95.32%\n",
      "Progress: 95.32%\n",
      "Progress: 95.33%\n",
      "Progress: 95.33%\n",
      "Progress: 95.33%\n",
      "Progress: 95.33%\n",
      "Progress: 95.34%\n",
      "Progress: 95.34%\n",
      "Progress: 95.34%\n",
      "Progress: 95.35%\n",
      "Progress: 95.35%\n",
      "Progress: 95.35%\n",
      "Progress: 95.36%\n",
      "Progress: 95.36%\n",
      "Progress: 95.36%\n",
      "Progress: 95.37%\n",
      "Progress: 95.37%\n",
      "Progress: 95.37%\n",
      "Progress: 95.37%\n",
      "Progress: 95.38%\n",
      "Progress: 95.38%\n",
      "Progress: 95.38%\n",
      "Progress: 95.39%\n",
      "Progress: 95.39%\n",
      "Progress: 95.39%\n",
      "Progress: 95.40%\n",
      "Progress: 95.40%\n",
      "Progress: 95.40%\n",
      "Progress: 95.41%\n",
      "Progress: 95.41%\n",
      "Progress: 95.41%\n",
      "Progress: 95.41%\n",
      "Progress: 95.42%\n",
      "Progress: 95.42%\n",
      "Progress: 95.42%\n",
      "Progress: 95.43%\n",
      "Progress: 95.43%\n",
      "Progress: 95.43%\n",
      "Progress: 95.44%\n",
      "Progress: 95.44%\n",
      "Progress: 95.44%\n",
      "Progress: 95.44%\n",
      "Progress: 95.45%\n",
      "Progress: 95.45%\n",
      "Progress: 95.45%\n",
      "Progress: 95.46%\n",
      "Progress: 95.46%\n",
      "Progress: 95.46%\n",
      "Progress: 95.47%\n",
      "Progress: 95.47%\n",
      "Progress: 95.47%\n",
      "Progress: 95.47%\n",
      "Progress: 95.48%\n",
      "Progress: 95.48%\n",
      "Progress: 95.48%\n",
      "Progress: 95.49%\n",
      "Progress: 95.49%\n",
      "Progress: 95.49%\n",
      "Progress: 95.50%\n",
      "Progress: 95.50%\n",
      "Progress: 95.50%\n",
      "Progress: 95.51%\n",
      "Progress: 95.51%\n",
      "Progress: 95.51%\n",
      "Progress: 95.51%\n",
      "Progress: 95.52%\n",
      "Progress: 95.52%\n",
      "Progress: 95.52%\n",
      "Progress: 95.53%\n",
      "Progress: 95.53%\n",
      "Progress: 95.53%\n",
      "Progress: 95.54%\n",
      "Progress: 95.54%\n",
      "Progress: 95.54%\n",
      "Progress: 95.54%\n",
      "Progress: 95.55%\n",
      "Progress: 95.55%\n",
      "Progress: 95.55%\n",
      "Progress: 95.56%\n",
      "Progress: 95.56%\n",
      "Progress: 95.56%\n",
      "Progress: 95.57%\n",
      "Progress: 95.57%\n",
      "Progress: 95.57%\n",
      "Progress: 95.57%\n",
      "Progress: 95.58%\n",
      "Progress: 95.58%\n",
      "Progress: 95.58%\n",
      "Progress: 95.59%\n",
      "Progress: 95.59%\n",
      "Progress: 95.59%\n",
      "Progress: 95.60%\n",
      "Progress: 95.60%\n",
      "Progress: 95.60%\n",
      "Progress: 95.60%\n",
      "Progress: 95.61%\n",
      "Progress: 95.61%\n",
      "Progress: 95.61%\n",
      "Progress: 95.62%\n",
      "Progress: 95.62%\n",
      "Progress: 95.62%\n",
      "Progress: 95.63%\n",
      "Progress: 95.63%\n",
      "Progress: 95.63%\n",
      "Progress: 95.63%\n",
      "Progress: 95.64%\n",
      "Progress: 95.64%\n",
      "Progress: 95.64%\n",
      "Progress: 95.65%\n",
      "Progress: 95.65%\n",
      "Progress: 95.65%\n",
      "Progress: 95.65%\n",
      "Progress: 95.66%\n",
      "Progress: 95.66%\n",
      "Progress: 95.66%\n",
      "Progress: 95.67%\n",
      "Progress: 95.67%\n",
      "Progress: 95.67%\n",
      "Progress: 95.68%\n",
      "Progress: 95.68%\n",
      "Progress: 95.68%\n",
      "Progress: 95.68%\n",
      "Progress: 95.69%\n",
      "Progress: 95.69%\n",
      "Progress: 95.69%\n",
      "Progress: 95.70%\n",
      "Progress: 95.70%\n",
      "Progress: 95.70%\n",
      "Progress: 95.71%\n",
      "Progress: 95.71%\n",
      "Progress: 95.71%\n",
      "Progress: 95.71%\n",
      "Progress: 95.72%\n",
      "Progress: 95.72%\n",
      "Progress: 95.72%\n",
      "Progress: 95.73%\n",
      "Progress: 95.73%\n",
      "Progress: 95.73%\n",
      "Progress: 95.73%\n",
      "Progress: 95.74%\n",
      "Progress: 95.74%\n",
      "Progress: 95.74%\n",
      "Progress: 95.75%\n",
      "Progress: 95.75%\n",
      "Progress: 95.75%\n",
      "Progress: 95.75%\n",
      "Progress: 95.76%\n",
      "Progress: 95.76%\n",
      "Progress: 95.76%\n",
      "Progress: 95.77%\n",
      "Progress: 95.77%\n",
      "Progress: 95.77%\n",
      "Progress: 95.78%\n",
      "Progress: 95.78%\n",
      "Progress: 95.78%\n",
      "Progress: 95.78%\n",
      "Progress: 95.79%\n",
      "Progress: 95.79%\n",
      "Progress: 95.79%\n",
      "Progress: 95.80%\n",
      "Progress: 95.80%\n",
      "Progress: 95.80%\n",
      "Progress: 95.80%\n",
      "Progress: 95.81%\n",
      "Progress: 95.81%\n",
      "Progress: 95.81%\n",
      "Progress: 95.82%\n",
      "Progress: 95.82%\n",
      "Progress: 95.82%\n",
      "Progress: 95.83%\n",
      "Progress: 95.83%\n",
      "Progress: 95.83%\n",
      "Progress: 95.83%\n",
      "Progress: 95.84%\n",
      "Progress: 95.84%\n",
      "Progress: 95.84%\n",
      "Progress: 95.85%\n",
      "Progress: 95.85%\n",
      "Progress: 95.85%\n",
      "Progress: 95.85%\n",
      "Progress: 95.86%\n",
      "Progress: 95.86%\n",
      "Progress: 95.86%\n",
      "Progress: 95.87%\n",
      "Progress: 95.87%\n",
      "Progress: 95.87%\n",
      "Progress: 95.87%\n",
      "Progress: 95.88%\n",
      "Progress: 95.88%\n",
      "Progress: 95.88%\n",
      "Progress: 95.89%\n",
      "Progress: 95.89%\n",
      "Progress: 95.89%\n",
      "Progress: 95.89%\n",
      "Progress: 95.90%\n",
      "Progress: 95.90%\n",
      "Progress: 95.90%\n",
      "Progress: 95.91%\n",
      "Progress: 95.91%\n",
      "Progress: 95.91%\n",
      "Progress: 95.91%\n",
      "Progress: 95.92%\n",
      "Progress: 95.92%\n",
      "Progress: 95.92%\n",
      "Progress: 95.93%\n",
      "Progress: 95.93%\n",
      "Progress: 95.93%\n",
      "Progress: 95.93%\n",
      "Progress: 95.94%\n",
      "Progress: 95.94%\n",
      "Progress: 95.94%\n",
      "Progress: 95.95%\n",
      "Progress: 95.95%\n",
      "Progress: 95.95%\n",
      "Progress: 95.95%\n",
      "Progress: 95.96%\n",
      "Progress: 95.96%\n",
      "Progress: 95.96%\n",
      "Progress: 95.97%\n",
      "Progress: 95.97%\n",
      "Progress: 95.97%\n",
      "Progress: 95.97%\n",
      "Progress: 95.98%\n",
      "Progress: 95.98%\n",
      "Progress: 95.98%\n",
      "Progress: 95.99%\n",
      "Progress: 95.99%\n",
      "Progress: 95.99%\n",
      "Progress: 95.99%\n",
      "Progress: 96.00%\n",
      "Progress: 96.00%\n",
      "Progress: 96.00%\n",
      "Progress: 96.01%\n",
      "Progress: 96.01%\n",
      "Progress: 96.01%\n",
      "Progress: 96.01%\n",
      "Progress: 96.02%\n",
      "Progress: 96.02%\n",
      "Progress: 96.02%\n",
      "Progress: 96.03%\n",
      "Progress: 96.03%\n",
      "Progress: 96.03%\n",
      "Progress: 96.03%\n",
      "Progress: 96.04%\n",
      "Progress: 96.04%\n",
      "Progress: 96.04%\n",
      "Progress: 96.05%\n",
      "Progress: 96.05%\n",
      "Progress: 96.05%\n",
      "Progress: 96.05%\n",
      "Progress: 96.06%\n",
      "Progress: 96.06%\n",
      "Progress: 96.06%\n",
      "Progress: 96.07%\n",
      "Progress: 96.07%\n",
      "Progress: 96.07%\n",
      "Progress: 96.07%\n",
      "Progress: 96.08%\n",
      "Progress: 96.08%\n",
      "Progress: 96.08%\n",
      "Progress: 96.09%\n",
      "Progress: 96.09%\n",
      "Progress: 96.09%\n",
      "Progress: 96.09%\n",
      "Progress: 96.10%\n",
      "Progress: 96.10%\n",
      "Progress: 96.10%\n",
      "Progress: 96.11%\n",
      "Progress: 96.11%\n",
      "Progress: 96.11%\n",
      "Progress: 96.11%\n",
      "Progress: 96.12%\n",
      "Progress: 96.12%\n",
      "Progress: 96.12%\n",
      "Progress: 96.12%\n",
      "Progress: 96.13%\n",
      "Progress: 96.13%\n",
      "Progress: 96.13%\n",
      "Progress: 96.14%\n",
      "Progress: 96.14%\n",
      "Progress: 96.14%\n",
      "Progress: 96.14%\n",
      "Progress: 96.15%\n",
      "Progress: 96.15%\n",
      "Progress: 96.15%\n",
      "Progress: 96.16%\n",
      "Progress: 96.16%\n",
      "Progress: 96.16%\n",
      "Progress: 96.16%\n",
      "Progress: 96.17%\n",
      "Progress: 96.17%\n",
      "Progress: 96.17%\n",
      "Progress: 96.18%\n",
      "Progress: 96.18%\n",
      "Progress: 96.18%\n",
      "Progress: 96.18%\n",
      "Progress: 96.19%\n",
      "Progress: 96.19%\n",
      "Progress: 96.19%\n",
      "Progress: 96.19%\n",
      "Progress: 96.20%\n",
      "Progress: 96.20%\n",
      "Progress: 96.20%\n",
      "Progress: 96.21%\n",
      "Progress: 96.21%\n",
      "Progress: 96.21%\n",
      "Progress: 96.21%\n",
      "Progress: 96.22%\n",
      "Progress: 96.22%\n",
      "Progress: 96.22%\n",
      "Progress: 96.23%\n",
      "Progress: 96.23%\n",
      "Progress: 96.23%\n",
      "Progress: 96.23%\n",
      "Progress: 96.24%\n",
      "Progress: 96.24%\n",
      "Progress: 96.24%\n",
      "Progress: 96.24%\n",
      "Progress: 96.25%\n",
      "Progress: 96.25%\n",
      "Progress: 96.25%\n",
      "Progress: 96.26%\n",
      "Progress: 96.26%\n",
      "Progress: 96.26%\n",
      "Progress: 96.26%\n",
      "Progress: 96.27%\n",
      "Progress: 96.27%\n",
      "Progress: 96.27%\n",
      "Progress: 96.27%\n",
      "Progress: 96.28%\n",
      "Progress: 96.28%\n",
      "Progress: 96.28%\n",
      "Progress: 96.29%\n",
      "Progress: 96.29%\n",
      "Progress: 96.29%\n",
      "Progress: 96.29%\n",
      "Progress: 96.30%\n",
      "Progress: 96.30%\n",
      "Progress: 96.30%\n",
      "Progress: 96.30%\n",
      "Progress: 96.31%\n",
      "Progress: 96.31%\n",
      "Progress: 96.31%\n",
      "Progress: 96.32%\n",
      "Progress: 96.32%\n",
      "Progress: 96.32%\n",
      "Progress: 96.32%\n",
      "Progress: 96.33%\n",
      "Progress: 96.33%\n",
      "Progress: 96.33%\n",
      "Progress: 96.33%\n",
      "Progress: 96.34%\n",
      "Progress: 96.34%\n",
      "Progress: 96.34%\n",
      "Progress: 96.35%\n",
      "Progress: 96.35%\n",
      "Progress: 96.35%\n",
      "Progress: 96.35%\n",
      "Progress: 96.36%\n",
      "Progress: 96.36%\n",
      "Progress: 96.36%\n",
      "Progress: 96.36%\n",
      "Progress: 96.37%\n",
      "Progress: 96.37%\n",
      "Progress: 96.37%\n",
      "Progress: 96.38%\n",
      "Progress: 96.38%\n",
      "Progress: 96.38%\n",
      "Progress: 96.38%\n",
      "Progress: 96.39%\n",
      "Progress: 96.39%\n",
      "Progress: 96.39%\n",
      "Progress: 96.39%\n",
      "Progress: 96.40%\n",
      "Progress: 96.40%\n",
      "Progress: 96.40%\n",
      "Progress: 96.41%\n",
      "Progress: 96.41%\n",
      "Progress: 96.41%\n",
      "Progress: 96.41%\n",
      "Progress: 96.42%\n",
      "Progress: 96.42%\n",
      "Progress: 96.42%\n",
      "Progress: 96.42%\n",
      "Progress: 96.43%\n",
      "Progress: 96.43%\n",
      "Progress: 96.43%\n",
      "Progress: 96.43%\n",
      "Progress: 96.44%\n",
      "Progress: 96.44%\n",
      "Progress: 96.44%\n",
      "Progress: 96.45%\n",
      "Progress: 96.45%\n",
      "Progress: 96.45%\n",
      "Progress: 96.45%\n",
      "Progress: 96.46%\n",
      "Progress: 96.46%\n",
      "Progress: 96.46%\n",
      "Progress: 96.46%\n",
      "Progress: 96.47%\n",
      "Progress: 96.47%\n",
      "Progress: 96.47%\n",
      "Progress: 96.48%\n",
      "Progress: 96.48%\n",
      "Progress: 96.48%\n",
      "Progress: 96.48%\n",
      "Progress: 96.49%\n",
      "Progress: 96.49%\n",
      "Progress: 96.49%\n",
      "Progress: 96.49%\n",
      "Progress: 96.50%\n",
      "Progress: 96.50%\n",
      "Progress: 96.50%\n",
      "Progress: 96.50%\n",
      "Progress: 96.51%\n",
      "Progress: 96.51%\n",
      "Progress: 96.51%\n",
      "Progress: 96.52%\n",
      "Progress: 96.52%\n",
      "Progress: 96.52%\n",
      "Progress: 96.52%\n",
      "Progress: 96.53%\n",
      "Progress: 96.53%\n",
      "Progress: 96.53%\n",
      "Progress: 96.53%\n",
      "Progress: 96.54%\n",
      "Progress: 96.54%\n",
      "Progress: 96.54%\n",
      "Progress: 96.54%\n",
      "Progress: 96.55%\n",
      "Progress: 96.55%\n",
      "Progress: 96.55%\n",
      "Progress: 96.55%\n",
      "Progress: 96.56%\n",
      "Progress: 96.56%\n",
      "Progress: 96.56%\n",
      "Progress: 96.57%\n",
      "Progress: 96.57%\n",
      "Progress: 96.57%\n",
      "Progress: 96.57%\n",
      "Progress: 96.58%\n",
      "Progress: 96.58%\n",
      "Progress: 96.58%\n",
      "Progress: 96.58%\n",
      "Progress: 96.59%\n",
      "Progress: 96.59%\n",
      "Progress: 96.59%\n",
      "Progress: 96.59%\n",
      "Progress: 96.60%\n",
      "Progress: 96.60%\n",
      "Progress: 96.60%\n",
      "Progress: 96.60%\n",
      "Progress: 96.61%\n",
      "Progress: 96.61%\n",
      "Progress: 96.61%\n",
      "Progress: 96.62%\n",
      "Progress: 96.62%\n",
      "Progress: 96.62%\n",
      "Progress: 96.62%\n",
      "Progress: 96.63%\n",
      "Progress: 96.63%\n",
      "Progress: 96.63%\n",
      "Progress: 96.63%\n",
      "Progress: 96.64%\n",
      "Progress: 96.64%\n",
      "Progress: 96.64%\n",
      "Progress: 96.64%\n",
      "Progress: 96.65%\n",
      "Progress: 96.65%\n",
      "Progress: 96.65%\n",
      "Progress: 96.65%\n",
      "Progress: 96.66%\n",
      "Progress: 96.66%\n",
      "Progress: 96.66%\n",
      "Progress: 96.66%\n",
      "Progress: 96.67%\n",
      "Progress: 96.67%\n",
      "Progress: 96.67%\n",
      "Progress: 96.68%\n",
      "Progress: 96.68%\n",
      "Progress: 96.68%\n",
      "Progress: 96.68%\n",
      "Progress: 96.69%\n",
      "Progress: 96.69%\n",
      "Progress: 96.69%\n",
      "Progress: 96.69%\n",
      "Progress: 96.70%\n",
      "Progress: 96.70%\n",
      "Progress: 96.70%\n",
      "Progress: 96.70%\n",
      "Progress: 96.71%\n",
      "Progress: 96.71%\n",
      "Progress: 96.71%\n",
      "Progress: 96.71%\n",
      "Progress: 96.72%\n",
      "Progress: 96.72%\n",
      "Progress: 96.72%\n",
      "Progress: 96.72%\n",
      "Progress: 96.73%\n",
      "Progress: 96.73%\n",
      "Progress: 96.73%\n",
      "Progress: 96.73%\n",
      "Progress: 96.74%\n",
      "Progress: 96.74%\n",
      "Progress: 96.74%\n",
      "Progress: 96.74%\n",
      "Progress: 96.75%\n",
      "Progress: 96.75%\n",
      "Progress: 96.75%\n",
      "Progress: 96.76%\n",
      "Progress: 96.76%\n",
      "Progress: 96.76%\n",
      "Progress: 96.76%\n",
      "Progress: 96.77%\n",
      "Progress: 96.77%\n",
      "Progress: 96.77%\n",
      "Progress: 96.77%\n",
      "Progress: 96.78%\n",
      "Progress: 96.78%\n",
      "Progress: 96.78%\n",
      "Progress: 96.78%\n",
      "Progress: 96.79%\n",
      "Progress: 96.79%\n",
      "Progress: 96.79%\n",
      "Progress: 96.79%\n",
      "Progress: 96.80%\n",
      "Progress: 96.80%\n",
      "Progress: 96.80%\n",
      "Progress: 96.80%\n",
      "Progress: 96.81%\n",
      "Progress: 96.81%\n",
      "Progress: 96.81%\n",
      "Progress: 96.81%\n",
      "Progress: 96.82%\n",
      "Progress: 96.82%\n",
      "Progress: 96.82%\n",
      "Progress: 96.82%\n",
      "Progress: 96.83%\n",
      "Progress: 96.83%\n",
      "Progress: 96.83%\n",
      "Progress: 96.83%\n",
      "Progress: 96.84%\n",
      "Progress: 96.84%\n",
      "Progress: 96.84%\n",
      "Progress: 96.84%\n",
      "Progress: 96.85%\n",
      "Progress: 96.85%\n",
      "Progress: 96.85%\n",
      "Progress: 96.85%\n",
      "Progress: 96.86%\n",
      "Progress: 96.86%\n",
      "Progress: 96.86%\n",
      "Progress: 96.86%\n",
      "Progress: 96.87%\n",
      "Progress: 96.87%\n",
      "Progress: 96.87%\n",
      "Progress: 96.87%\n",
      "Progress: 96.88%\n",
      "Progress: 96.88%\n",
      "Progress: 96.88%\n",
      "Progress: 96.88%\n",
      "Progress: 96.89%\n",
      "Progress: 96.89%\n",
      "Progress: 96.89%\n",
      "Progress: 96.89%\n",
      "Progress: 96.90%\n",
      "Progress: 96.90%\n",
      "Progress: 96.90%\n",
      "Progress: 96.90%\n",
      "Progress: 96.91%\n",
      "Progress: 96.91%\n",
      "Progress: 96.91%\n",
      "Progress: 96.91%\n",
      "Progress: 96.92%\n",
      "Progress: 96.92%\n",
      "Progress: 96.92%\n",
      "Progress: 96.92%\n",
      "Progress: 96.93%\n",
      "Progress: 96.93%\n",
      "Progress: 96.93%\n",
      "Progress: 96.93%\n",
      "Progress: 96.94%\n",
      "Progress: 96.94%\n",
      "Progress: 96.94%\n",
      "Progress: 96.94%\n",
      "Progress: 96.95%\n",
      "Progress: 96.95%\n",
      "Progress: 96.95%\n",
      "Progress: 96.95%\n",
      "Progress: 96.96%\n",
      "Progress: 96.96%\n",
      "Progress: 96.96%\n",
      "Progress: 96.96%\n",
      "Progress: 96.97%\n",
      "Progress: 96.97%\n",
      "Progress: 96.97%\n",
      "Progress: 96.97%\n",
      "Progress: 96.98%\n",
      "Progress: 96.98%\n",
      "Progress: 96.98%\n",
      "Progress: 96.98%\n",
      "Progress: 96.99%\n",
      "Progress: 96.99%\n",
      "Progress: 96.99%\n",
      "Progress: 96.99%\n",
      "Progress: 97.00%\n",
      "Progress: 97.00%\n",
      "Progress: 97.00%\n",
      "Progress: 97.00%\n",
      "Progress: 97.01%\n",
      "Progress: 97.01%\n",
      "Progress: 97.01%\n",
      "Progress: 97.01%\n",
      "Progress: 97.02%\n",
      "Progress: 97.02%\n",
      "Progress: 97.02%\n",
      "Progress: 97.02%\n",
      "Progress: 97.03%\n",
      "Progress: 97.03%\n",
      "Progress: 97.03%\n",
      "Progress: 97.03%\n",
      "Progress: 97.04%\n",
      "Progress: 97.04%\n",
      "Progress: 97.04%\n",
      "Progress: 97.04%\n",
      "Progress: 97.05%\n",
      "Progress: 97.05%\n",
      "Progress: 97.05%\n",
      "Progress: 97.05%\n",
      "Progress: 97.06%\n",
      "Progress: 97.06%\n",
      "Progress: 97.06%\n",
      "Progress: 97.06%\n",
      "Progress: 97.07%\n",
      "Progress: 97.07%\n",
      "Progress: 97.07%\n",
      "Progress: 97.07%\n",
      "Progress: 97.08%\n",
      "Progress: 97.08%\n",
      "Progress: 97.08%\n",
      "Progress: 97.08%\n",
      "Progress: 97.08%\n",
      "Progress: 97.09%\n",
      "Progress: 97.09%\n",
      "Progress: 97.09%\n",
      "Progress: 97.09%\n",
      "Progress: 97.10%\n",
      "Progress: 97.10%\n",
      "Progress: 97.10%\n",
      "Progress: 97.10%\n",
      "Progress: 97.11%\n",
      "Progress: 97.11%\n",
      "Progress: 97.11%\n",
      "Progress: 97.11%\n",
      "Progress: 97.12%\n",
      "Progress: 97.12%\n",
      "Progress: 97.12%\n",
      "Progress: 97.12%\n",
      "Progress: 97.13%\n",
      "Progress: 97.13%\n",
      "Progress: 97.13%\n",
      "Progress: 97.13%\n",
      "Progress: 97.14%\n",
      "Progress: 97.14%\n",
      "Progress: 97.14%\n",
      "Progress: 97.14%\n",
      "Progress: 97.15%\n",
      "Progress: 97.15%\n",
      "Progress: 97.15%\n",
      "Progress: 97.15%\n",
      "Progress: 97.15%\n",
      "Progress: 97.16%\n",
      "Progress: 97.16%\n",
      "Progress: 97.16%\n",
      "Progress: 97.16%\n",
      "Progress: 97.17%\n",
      "Progress: 97.17%\n",
      "Progress: 97.17%\n",
      "Progress: 97.17%\n",
      "Progress: 97.18%\n",
      "Progress: 97.18%\n",
      "Progress: 97.18%\n",
      "Progress: 97.18%\n",
      "Progress: 97.19%\n",
      "Progress: 97.19%\n",
      "Progress: 97.19%\n",
      "Progress: 97.19%\n",
      "Progress: 97.20%\n",
      "Progress: 97.20%\n",
      "Progress: 97.20%\n",
      "Progress: 97.20%\n",
      "Progress: 97.20%\n",
      "Progress: 97.21%\n",
      "Progress: 97.21%\n",
      "Progress: 97.21%\n",
      "Progress: 97.21%\n",
      "Progress: 97.22%\n",
      "Progress: 97.22%\n",
      "Progress: 97.22%\n",
      "Progress: 97.22%\n",
      "Progress: 97.23%\n",
      "Progress: 97.23%\n",
      "Progress: 97.23%\n",
      "Progress: 97.23%\n",
      "Progress: 97.24%\n",
      "Progress: 97.24%\n",
      "Progress: 97.24%\n",
      "Progress: 97.24%\n",
      "Progress: 97.25%\n",
      "Progress: 97.25%\n",
      "Progress: 97.25%\n",
      "Progress: 97.25%\n",
      "Progress: 97.25%\n",
      "Progress: 97.26%\n",
      "Progress: 97.26%\n",
      "Progress: 97.26%\n",
      "Progress: 97.26%\n",
      "Progress: 97.27%\n",
      "Progress: 97.27%\n",
      "Progress: 97.27%\n",
      "Progress: 97.27%\n",
      "Progress: 97.28%\n",
      "Progress: 97.28%\n",
      "Progress: 97.28%\n",
      "Progress: 97.28%\n",
      "Progress: 97.29%\n",
      "Progress: 97.29%\n",
      "Progress: 97.29%\n",
      "Progress: 97.29%\n",
      "Progress: 97.29%\n",
      "Progress: 97.30%\n",
      "Progress: 97.30%\n",
      "Progress: 97.30%\n",
      "Progress: 97.30%\n",
      "Progress: 97.31%\n",
      "Progress: 97.31%\n",
      "Progress: 97.31%\n",
      "Progress: 97.31%\n",
      "Progress: 97.32%\n",
      "Progress: 97.32%\n",
      "Progress: 97.32%\n",
      "Progress: 97.32%\n",
      "Progress: 97.33%\n",
      "Progress: 97.33%\n",
      "Progress: 97.33%\n",
      "Progress: 97.33%\n",
      "Progress: 97.33%\n",
      "Progress: 97.34%\n",
      "Progress: 97.34%\n",
      "Progress: 97.34%\n",
      "Progress: 97.34%\n",
      "Progress: 97.35%\n",
      "Progress: 97.35%\n",
      "Progress: 97.35%\n",
      "Progress: 97.35%\n",
      "Progress: 97.36%\n",
      "Progress: 97.36%\n",
      "Progress: 97.36%\n",
      "Progress: 97.36%\n",
      "Progress: 97.36%\n",
      "Progress: 97.37%\n",
      "Progress: 97.37%\n",
      "Progress: 97.37%\n",
      "Progress: 97.37%\n",
      "Progress: 97.38%\n",
      "Progress: 97.38%\n",
      "Progress: 97.38%\n",
      "Progress: 97.38%\n",
      "Progress: 97.39%\n",
      "Progress: 97.39%\n",
      "Progress: 97.39%\n",
      "Progress: 97.39%\n",
      "Progress: 97.39%\n",
      "Progress: 97.40%\n",
      "Progress: 97.40%\n",
      "Progress: 97.40%\n",
      "Progress: 97.40%\n",
      "Progress: 97.41%\n",
      "Progress: 97.41%\n",
      "Progress: 97.41%\n",
      "Progress: 97.41%\n",
      "Progress: 97.42%\n",
      "Progress: 97.42%\n",
      "Progress: 97.42%\n",
      "Progress: 97.42%\n",
      "Progress: 97.42%\n",
      "Progress: 97.43%\n",
      "Progress: 97.43%\n",
      "Progress: 97.43%\n",
      "Progress: 97.43%\n",
      "Progress: 97.44%\n",
      "Progress: 97.44%\n",
      "Progress: 97.44%\n",
      "Progress: 97.44%\n",
      "Progress: 97.44%\n",
      "Progress: 97.45%\n",
      "Progress: 97.45%\n",
      "Progress: 97.45%\n",
      "Progress: 97.45%\n",
      "Progress: 97.46%\n",
      "Progress: 97.46%\n",
      "Progress: 97.46%\n",
      "Progress: 97.46%\n",
      "Progress: 97.47%\n",
      "Progress: 97.47%\n",
      "Progress: 97.47%\n",
      "Progress: 97.47%\n",
      "Progress: 97.47%\n",
      "Progress: 97.48%\n",
      "Progress: 97.48%\n",
      "Progress: 97.48%\n",
      "Progress: 97.48%\n",
      "Progress: 97.49%\n",
      "Progress: 97.49%\n",
      "Progress: 97.49%\n",
      "Progress: 97.49%\n",
      "Progress: 97.49%\n",
      "Progress: 97.50%\n",
      "Progress: 97.50%\n",
      "Progress: 97.50%\n",
      "Progress: 97.50%\n",
      "Progress: 97.51%\n",
      "Progress: 97.51%\n",
      "Progress: 97.51%\n",
      "Progress: 97.51%\n",
      "Progress: 97.51%\n",
      "Progress: 97.52%\n",
      "Progress: 97.52%\n",
      "Progress: 97.52%\n",
      "Progress: 97.52%\n",
      "Progress: 97.53%\n",
      "Progress: 97.53%\n",
      "Progress: 97.53%\n",
      "Progress: 97.53%\n",
      "Progress: 97.53%\n",
      "Progress: 97.54%\n",
      "Progress: 97.54%\n",
      "Progress: 97.54%\n",
      "Progress: 97.54%\n",
      "Progress: 97.55%\n",
      "Progress: 97.55%\n",
      "Progress: 97.55%\n",
      "Progress: 97.55%\n",
      "Progress: 97.55%\n",
      "Progress: 97.56%\n",
      "Progress: 97.56%\n",
      "Progress: 97.56%\n",
      "Progress: 97.56%\n",
      "Progress: 97.57%\n",
      "Progress: 97.57%\n",
      "Progress: 97.57%\n",
      "Progress: 97.57%\n",
      "Progress: 97.57%\n",
      "Progress: 97.58%\n",
      "Progress: 97.58%\n",
      "Progress: 97.58%\n",
      "Progress: 97.58%\n",
      "Progress: 97.59%\n",
      "Progress: 97.59%\n",
      "Progress: 97.59%\n",
      "Progress: 97.59%\n",
      "Progress: 97.59%\n",
      "Progress: 97.60%\n",
      "Progress: 97.60%\n",
      "Progress: 97.60%\n",
      "Progress: 97.60%\n",
      "Progress: 97.61%\n",
      "Progress: 97.61%\n",
      "Progress: 97.61%\n",
      "Progress: 97.61%\n",
      "Progress: 97.61%\n",
      "Progress: 97.62%\n",
      "Progress: 97.62%\n",
      "Progress: 97.62%\n",
      "Progress: 97.62%\n",
      "Progress: 97.63%\n",
      "Progress: 97.63%\n",
      "Progress: 97.63%\n",
      "Progress: 97.63%\n",
      "Progress: 97.63%\n",
      "Progress: 97.64%\n",
      "Progress: 97.64%\n",
      "Progress: 97.64%\n",
      "Progress: 97.64%\n",
      "Progress: 97.65%\n",
      "Progress: 97.65%\n",
      "Progress: 97.65%\n",
      "Progress: 97.65%\n",
      "Progress: 97.65%\n",
      "Progress: 97.66%\n",
      "Progress: 97.66%\n",
      "Progress: 97.66%\n",
      "Progress: 97.66%\n",
      "Progress: 97.66%\n",
      "Progress: 97.67%\n",
      "Progress: 97.67%\n",
      "Progress: 97.67%\n",
      "Progress: 97.67%\n",
      "Progress: 97.68%\n",
      "Progress: 97.68%\n",
      "Progress: 97.68%\n",
      "Progress: 97.68%\n",
      "Progress: 97.68%\n",
      "Progress: 97.69%\n",
      "Progress: 97.69%\n",
      "Progress: 97.69%\n",
      "Progress: 97.69%\n",
      "Progress: 97.70%\n",
      "Progress: 97.70%\n",
      "Progress: 97.70%\n",
      "Progress: 97.70%\n",
      "Progress: 97.70%\n",
      "Progress: 97.71%\n",
      "Progress: 97.71%\n",
      "Progress: 97.71%\n",
      "Progress: 97.71%\n",
      "Progress: 97.71%\n",
      "Progress: 97.72%\n",
      "Progress: 97.72%\n",
      "Progress: 97.72%\n",
      "Progress: 97.72%\n",
      "Progress: 97.73%\n",
      "Progress: 97.73%\n",
      "Progress: 97.73%\n",
      "Progress: 97.73%\n",
      "Progress: 97.73%\n",
      "Progress: 97.74%\n",
      "Progress: 97.74%\n",
      "Progress: 97.74%\n",
      "Progress: 97.74%\n",
      "Progress: 97.74%\n",
      "Progress: 97.75%\n",
      "Progress: 97.75%\n",
      "Progress: 97.75%\n",
      "Progress: 97.75%\n",
      "Progress: 97.76%\n",
      "Progress: 97.76%\n",
      "Progress: 97.76%\n",
      "Progress: 97.76%\n",
      "Progress: 97.76%\n",
      "Progress: 97.77%\n",
      "Progress: 97.77%\n",
      "Progress: 97.77%\n",
      "Progress: 97.77%\n",
      "Progress: 97.77%\n",
      "Progress: 97.78%\n",
      "Progress: 97.78%\n",
      "Progress: 97.78%\n",
      "Progress: 97.78%\n",
      "Progress: 97.79%\n",
      "Progress: 97.79%\n",
      "Progress: 97.79%\n",
      "Progress: 97.79%\n",
      "Progress: 97.79%\n",
      "Progress: 97.80%\n",
      "Progress: 97.80%\n",
      "Progress: 97.80%\n",
      "Progress: 97.80%\n",
      "Progress: 97.80%\n",
      "Progress: 97.81%\n",
      "Progress: 97.81%\n",
      "Progress: 97.81%\n",
      "Progress: 97.81%\n",
      "Progress: 97.81%\n",
      "Progress: 97.82%\n",
      "Progress: 97.82%\n",
      "Progress: 97.82%\n",
      "Progress: 97.82%\n",
      "Progress: 97.83%\n",
      "Progress: 97.83%\n",
      "Progress: 97.83%\n",
      "Progress: 97.83%\n",
      "Progress: 97.83%\n",
      "Progress: 97.84%\n",
      "Progress: 97.84%\n",
      "Progress: 97.84%\n",
      "Progress: 97.84%\n",
      "Progress: 97.84%\n",
      "Progress: 97.85%\n",
      "Progress: 97.85%\n",
      "Progress: 97.85%\n",
      "Progress: 97.85%\n",
      "Progress: 97.85%\n",
      "Progress: 97.86%\n",
      "Progress: 97.86%\n",
      "Progress: 97.86%\n",
      "Progress: 97.86%\n",
      "Progress: 97.86%\n",
      "Progress: 97.87%\n",
      "Progress: 97.87%\n",
      "Progress: 97.87%\n",
      "Progress: 97.87%\n",
      "Progress: 97.88%\n",
      "Progress: 97.88%\n",
      "Progress: 97.88%\n",
      "Progress: 97.88%\n",
      "Progress: 97.88%\n",
      "Progress: 97.89%\n",
      "Progress: 97.89%\n",
      "Progress: 97.89%\n",
      "Progress: 97.89%\n",
      "Progress: 97.89%\n",
      "Progress: 97.90%\n",
      "Progress: 97.90%\n",
      "Progress: 97.90%\n",
      "Progress: 97.90%\n",
      "Progress: 97.90%\n",
      "Progress: 97.91%\n",
      "Progress: 97.91%\n",
      "Progress: 97.91%\n",
      "Progress: 97.91%\n",
      "Progress: 97.91%\n",
      "Progress: 97.92%\n",
      "Progress: 97.92%\n",
      "Progress: 97.92%\n",
      "Progress: 97.92%\n",
      "Progress: 97.92%\n",
      "Progress: 97.93%\n",
      "Progress: 97.93%\n",
      "Progress: 97.93%\n",
      "Progress: 97.93%\n",
      "Progress: 97.94%\n",
      "Progress: 97.94%\n",
      "Progress: 97.94%\n",
      "Progress: 97.94%\n",
      "Progress: 97.94%\n",
      "Progress: 97.95%\n",
      "Progress: 97.95%\n",
      "Progress: 97.95%\n",
      "Progress: 97.95%\n",
      "Progress: 97.95%\n",
      "Progress: 97.96%\n",
      "Progress: 97.96%\n",
      "Progress: 97.96%\n",
      "Progress: 97.96%\n",
      "Progress: 97.96%\n",
      "Progress: 97.97%\n",
      "Progress: 97.97%\n",
      "Progress: 97.97%\n",
      "Progress: 97.97%\n",
      "Progress: 97.97%\n",
      "Progress: 97.98%\n",
      "Progress: 97.98%\n",
      "Progress: 97.98%\n",
      "Progress: 97.98%\n",
      "Progress: 97.98%\n",
      "Progress: 97.99%\n",
      "Progress: 97.99%\n",
      "Progress: 97.99%\n",
      "Progress: 97.99%\n",
      "Progress: 97.99%\n",
      "Progress: 98.00%\n",
      "Progress: 98.00%\n",
      "Progress: 98.00%\n",
      "Progress: 98.00%\n",
      "Progress: 98.00%\n",
      "Progress: 98.01%\n",
      "Progress: 98.01%\n",
      "Progress: 98.01%\n",
      "Progress: 98.01%\n",
      "Progress: 98.01%\n",
      "Progress: 98.02%\n",
      "Progress: 98.02%\n",
      "Progress: 98.02%\n",
      "Progress: 98.02%\n",
      "Progress: 98.02%\n",
      "Progress: 98.03%\n",
      "Progress: 98.03%\n",
      "Progress: 98.03%\n",
      "Progress: 98.03%\n",
      "Progress: 98.03%\n",
      "Progress: 98.04%\n",
      "Progress: 98.04%\n",
      "Progress: 98.04%\n",
      "Progress: 98.04%\n",
      "Progress: 98.04%\n",
      "Progress: 98.05%\n",
      "Progress: 98.05%\n",
      "Progress: 98.05%\n",
      "Progress: 98.05%\n",
      "Progress: 98.05%\n",
      "Progress: 98.06%\n",
      "Progress: 98.06%\n",
      "Progress: 98.06%\n",
      "Progress: 98.06%\n",
      "Progress: 98.06%\n",
      "Progress: 98.07%\n",
      "Progress: 98.07%\n",
      "Progress: 98.07%\n",
      "Progress: 98.07%\n",
      "Progress: 98.07%\n",
      "Progress: 98.08%\n",
      "Progress: 98.08%\n",
      "Progress: 98.08%\n",
      "Progress: 98.08%\n",
      "Progress: 98.08%\n",
      "Progress: 98.09%\n",
      "Progress: 98.09%\n",
      "Progress: 98.09%\n",
      "Progress: 98.09%\n",
      "Progress: 98.09%\n",
      "Progress: 98.10%\n",
      "Progress: 98.10%\n",
      "Progress: 98.10%\n",
      "Progress: 98.10%\n",
      "Progress: 98.10%\n",
      "Progress: 98.11%\n",
      "Progress: 98.11%\n",
      "Progress: 98.11%\n",
      "Progress: 98.11%\n",
      "Progress: 98.11%\n",
      "Progress: 98.11%\n",
      "Progress: 98.12%\n",
      "Progress: 98.12%\n",
      "Progress: 98.12%\n",
      "Progress: 98.12%\n",
      "Progress: 98.12%\n",
      "Progress: 98.13%\n",
      "Progress: 98.13%\n",
      "Progress: 98.13%\n",
      "Progress: 98.13%\n",
      "Progress: 98.13%\n",
      "Progress: 98.14%\n",
      "Progress: 98.14%\n",
      "Progress: 98.14%\n",
      "Progress: 98.14%\n",
      "Progress: 98.14%\n",
      "Progress: 98.15%\n",
      "Progress: 98.15%\n",
      "Progress: 98.15%\n",
      "Progress: 98.15%\n",
      "Progress: 98.15%\n",
      "Progress: 98.16%\n",
      "Progress: 98.16%\n",
      "Progress: 98.16%\n",
      "Progress: 98.16%\n",
      "Progress: 98.16%\n",
      "Progress: 98.17%\n",
      "Progress: 98.17%\n",
      "Progress: 98.17%\n",
      "Progress: 98.17%\n",
      "Progress: 98.17%\n",
      "Progress: 98.18%\n",
      "Progress: 98.18%\n",
      "Progress: 98.18%\n",
      "Progress: 98.18%\n",
      "Progress: 98.18%\n",
      "Progress: 98.18%\n",
      "Progress: 98.19%\n",
      "Progress: 98.19%\n",
      "Progress: 98.19%\n",
      "Progress: 98.19%\n",
      "Progress: 98.19%\n",
      "Progress: 98.20%\n",
      "Progress: 98.20%\n",
      "Progress: 98.20%\n",
      "Progress: 98.20%\n",
      "Progress: 98.20%\n",
      "Progress: 98.21%\n",
      "Progress: 98.21%\n",
      "Progress: 98.21%\n",
      "Progress: 98.21%\n",
      "Progress: 98.21%\n",
      "Progress: 98.22%\n",
      "Progress: 98.22%\n",
      "Progress: 98.22%\n",
      "Progress: 98.22%\n",
      "Progress: 98.22%\n",
      "Progress: 98.22%\n",
      "Progress: 98.23%\n",
      "Progress: 98.23%\n",
      "Progress: 98.23%\n",
      "Progress: 98.23%\n",
      "Progress: 98.23%\n",
      "Progress: 98.24%\n",
      "Progress: 98.24%\n",
      "Progress: 98.24%\n",
      "Progress: 98.24%\n",
      "Progress: 98.24%\n",
      "Progress: 98.25%\n",
      "Progress: 98.25%\n",
      "Progress: 98.25%\n",
      "Progress: 98.25%\n",
      "Progress: 98.25%\n",
      "Progress: 98.25%\n",
      "Progress: 98.26%\n",
      "Progress: 98.26%\n",
      "Progress: 98.26%\n",
      "Progress: 98.26%\n",
      "Progress: 98.26%\n",
      "Progress: 98.27%\n",
      "Progress: 98.27%\n",
      "Progress: 98.27%\n",
      "Progress: 98.27%\n",
      "Progress: 98.27%\n",
      "Progress: 98.28%\n",
      "Progress: 98.28%\n",
      "Progress: 98.28%\n",
      "Progress: 98.28%\n",
      "Progress: 98.28%\n",
      "Progress: 98.28%\n",
      "Progress: 98.29%\n",
      "Progress: 98.29%\n",
      "Progress: 98.29%\n",
      "Progress: 98.29%\n",
      "Progress: 98.29%\n",
      "Progress: 98.30%\n",
      "Progress: 98.30%\n",
      "Progress: 98.30%\n",
      "Progress: 98.30%\n",
      "Progress: 98.30%\n",
      "Progress: 98.31%\n",
      "Progress: 98.31%\n",
      "Progress: 98.31%\n",
      "Progress: 98.31%\n",
      "Progress: 98.31%\n",
      "Progress: 98.31%\n",
      "Progress: 98.32%\n",
      "Progress: 98.32%\n",
      "Progress: 98.32%\n",
      "Progress: 98.32%\n",
      "Progress: 98.32%\n",
      "Progress: 98.33%\n",
      "Progress: 98.33%\n",
      "Progress: 98.33%\n",
      "Progress: 98.33%\n",
      "Progress: 98.33%\n",
      "Progress: 98.33%\n",
      "Progress: 98.34%\n",
      "Progress: 98.34%\n",
      "Progress: 98.34%\n",
      "Progress: 98.34%\n",
      "Progress: 98.34%\n",
      "Progress: 98.35%\n",
      "Progress: 98.35%\n",
      "Progress: 98.35%\n",
      "Progress: 98.35%\n",
      "Progress: 98.35%\n",
      "Progress: 98.35%\n",
      "Progress: 98.36%\n",
      "Progress: 98.36%\n",
      "Progress: 98.36%\n",
      "Progress: 98.36%\n",
      "Progress: 98.36%\n",
      "Progress: 98.37%\n",
      "Progress: 98.37%\n",
      "Progress: 98.37%\n",
      "Progress: 98.37%\n",
      "Progress: 98.37%\n",
      "Progress: 98.37%\n",
      "Progress: 98.38%\n",
      "Progress: 98.38%\n",
      "Progress: 98.38%\n",
      "Progress: 98.38%\n",
      "Progress: 98.38%\n",
      "Progress: 98.39%\n",
      "Progress: 98.39%\n",
      "Progress: 98.39%\n",
      "Progress: 98.39%\n",
      "Progress: 98.39%\n",
      "Progress: 98.39%\n",
      "Progress: 98.40%\n",
      "Progress: 98.40%\n",
      "Progress: 98.40%\n",
      "Progress: 98.40%\n",
      "Progress: 98.40%\n",
      "Progress: 98.41%\n",
      "Progress: 98.41%\n",
      "Progress: 98.41%\n",
      "Progress: 98.41%\n",
      "Progress: 98.41%\n",
      "Progress: 98.41%\n",
      "Progress: 98.42%\n",
      "Progress: 98.42%\n",
      "Progress: 98.42%\n",
      "Progress: 98.42%\n",
      "Progress: 98.42%\n",
      "Progress: 98.43%\n",
      "Progress: 98.43%\n",
      "Progress: 98.43%\n",
      "Progress: 98.43%\n",
      "Progress: 98.43%\n",
      "Progress: 98.43%\n",
      "Progress: 98.44%\n",
      "Progress: 98.44%\n",
      "Progress: 98.44%\n",
      "Progress: 98.44%\n",
      "Progress: 98.44%\n",
      "Progress: 98.44%\n",
      "Progress: 98.45%\n",
      "Progress: 98.45%\n",
      "Progress: 98.45%\n",
      "Progress: 98.45%\n",
      "Progress: 98.45%\n",
      "Progress: 98.46%\n",
      "Progress: 98.46%\n",
      "Progress: 98.46%\n",
      "Progress: 98.46%\n",
      "Progress: 98.46%\n",
      "Progress: 98.46%\n",
      "Progress: 98.47%\n",
      "Progress: 98.47%\n",
      "Progress: 98.47%\n",
      "Progress: 98.47%\n",
      "Progress: 98.47%\n",
      "Progress: 98.47%\n",
      "Progress: 98.48%\n",
      "Progress: 98.48%\n",
      "Progress: 98.48%\n",
      "Progress: 98.48%\n",
      "Progress: 98.48%\n",
      "Progress: 98.49%\n",
      "Progress: 98.49%\n",
      "Progress: 98.49%\n",
      "Progress: 98.49%\n",
      "Progress: 98.49%\n",
      "Progress: 98.49%\n",
      "Progress: 98.50%\n",
      "Progress: 98.50%\n",
      "Progress: 98.50%\n",
      "Progress: 98.50%\n",
      "Progress: 98.50%\n",
      "Progress: 98.50%\n",
      "Progress: 98.51%\n",
      "Progress: 98.51%\n",
      "Progress: 98.51%\n",
      "Progress: 98.51%\n",
      "Progress: 98.51%\n",
      "Progress: 98.52%\n",
      "Progress: 98.52%\n",
      "Progress: 98.52%\n",
      "Progress: 98.52%\n",
      "Progress: 98.52%\n",
      "Progress: 98.52%\n",
      "Progress: 98.53%\n",
      "Progress: 98.53%\n",
      "Progress: 98.53%\n",
      "Progress: 98.53%\n",
      "Progress: 98.53%\n",
      "Progress: 98.53%\n",
      "Progress: 98.54%\n",
      "Progress: 98.54%\n",
      "Progress: 98.54%\n",
      "Progress: 98.54%\n",
      "Progress: 98.54%\n",
      "Progress: 98.54%\n",
      "Progress: 98.55%\n",
      "Progress: 98.55%\n",
      "Progress: 98.55%\n",
      "Progress: 98.55%\n",
      "Progress: 98.55%\n",
      "Progress: 98.55%\n",
      "Progress: 98.56%\n",
      "Progress: 98.56%\n",
      "Progress: 98.56%\n",
      "Progress: 98.56%\n",
      "Progress: 98.56%\n",
      "Progress: 98.56%\n",
      "Progress: 98.57%\n",
      "Progress: 98.57%\n",
      "Progress: 98.57%\n",
      "Progress: 98.57%\n",
      "Progress: 98.57%\n",
      "Progress: 98.58%\n",
      "Progress: 98.58%\n",
      "Progress: 98.58%\n",
      "Progress: 98.58%\n",
      "Progress: 98.58%\n",
      "Progress: 98.58%\n",
      "Progress: 98.59%\n",
      "Progress: 98.59%\n",
      "Progress: 98.59%\n",
      "Progress: 98.59%\n",
      "Progress: 98.59%\n",
      "Progress: 98.59%\n",
      "Progress: 98.60%\n",
      "Progress: 98.60%\n",
      "Progress: 98.60%\n",
      "Progress: 98.60%\n",
      "Progress: 98.60%\n",
      "Progress: 98.60%\n",
      "Progress: 98.61%\n",
      "Progress: 98.61%\n",
      "Progress: 98.61%\n",
      "Progress: 98.61%\n",
      "Progress: 98.61%\n",
      "Progress: 98.61%\n",
      "Progress: 98.62%\n",
      "Progress: 98.62%\n",
      "Progress: 98.62%\n",
      "Progress: 98.62%\n",
      "Progress: 98.62%\n",
      "Progress: 98.62%\n",
      "Progress: 98.63%\n",
      "Progress: 98.63%\n",
      "Progress: 98.63%\n",
      "Progress: 98.63%\n",
      "Progress: 98.63%\n",
      "Progress: 98.63%\n",
      "Progress: 98.64%\n",
      "Progress: 98.64%\n",
      "Progress: 98.64%\n",
      "Progress: 98.64%\n",
      "Progress: 98.64%\n",
      "Progress: 98.64%\n",
      "Progress: 98.65%\n",
      "Progress: 98.65%\n",
      "Progress: 98.65%\n",
      "Progress: 98.65%\n",
      "Progress: 98.65%\n",
      "Progress: 98.65%\n",
      "Progress: 98.66%\n",
      "Progress: 98.66%\n",
      "Progress: 98.66%\n",
      "Progress: 98.66%\n",
      "Progress: 98.66%\n",
      "Progress: 98.66%\n",
      "Progress: 98.67%\n",
      "Progress: 98.67%\n",
      "Progress: 98.67%\n",
      "Progress: 98.67%\n",
      "Progress: 98.67%\n",
      "Progress: 98.67%\n",
      "Progress: 98.68%\n",
      "Progress: 98.68%\n",
      "Progress: 98.68%\n",
      "Progress: 98.68%\n",
      "Progress: 98.68%\n",
      "Progress: 98.68%\n",
      "Progress: 98.69%\n",
      "Progress: 98.69%\n",
      "Progress: 98.69%\n",
      "Progress: 98.69%\n",
      "Progress: 98.69%\n",
      "Progress: 98.69%\n",
      "Progress: 98.69%\n",
      "Progress: 98.70%\n",
      "Progress: 98.70%\n",
      "Progress: 98.70%\n",
      "Progress: 98.70%\n",
      "Progress: 98.70%\n",
      "Progress: 98.70%\n",
      "Progress: 98.71%\n",
      "Progress: 98.71%\n",
      "Progress: 98.71%\n",
      "Progress: 98.71%\n",
      "Progress: 98.71%\n",
      "Progress: 98.71%\n",
      "Progress: 98.72%\n",
      "Progress: 98.72%\n",
      "Progress: 98.72%\n",
      "Progress: 98.72%\n",
      "Progress: 98.72%\n",
      "Progress: 98.72%\n",
      "Progress: 98.73%\n",
      "Progress: 98.73%\n",
      "Progress: 98.73%\n",
      "Progress: 98.73%\n",
      "Progress: 98.73%\n",
      "Progress: 98.73%\n",
      "Progress: 98.74%\n",
      "Progress: 98.74%\n",
      "Progress: 98.74%\n",
      "Progress: 98.74%\n",
      "Progress: 98.74%\n",
      "Progress: 98.74%\n",
      "Progress: 98.74%\n",
      "Progress: 98.75%\n",
      "Progress: 98.75%\n",
      "Progress: 98.75%\n",
      "Progress: 98.75%\n",
      "Progress: 98.75%\n",
      "Progress: 98.75%\n",
      "Progress: 98.76%\n",
      "Progress: 98.76%\n",
      "Progress: 98.76%\n",
      "Progress: 98.76%\n",
      "Progress: 98.76%\n",
      "Progress: 98.76%\n",
      "Progress: 98.77%\n",
      "Progress: 98.77%\n",
      "Progress: 98.77%\n",
      "Progress: 98.77%\n",
      "Progress: 98.77%\n",
      "Progress: 98.77%\n",
      "Progress: 98.77%\n",
      "Progress: 98.78%\n",
      "Progress: 98.78%\n",
      "Progress: 98.78%\n",
      "Progress: 98.78%\n",
      "Progress: 98.78%\n",
      "Progress: 98.78%\n",
      "Progress: 98.79%\n",
      "Progress: 98.79%\n",
      "Progress: 98.79%\n",
      "Progress: 98.79%\n",
      "Progress: 98.79%\n",
      "Progress: 98.79%\n",
      "Progress: 98.80%\n",
      "Progress: 98.80%\n",
      "Progress: 98.80%\n",
      "Progress: 98.80%\n",
      "Progress: 98.80%\n",
      "Progress: 98.80%\n",
      "Progress: 98.80%\n",
      "Progress: 98.81%\n",
      "Progress: 98.81%\n",
      "Progress: 98.81%\n",
      "Progress: 98.81%\n",
      "Progress: 98.81%\n",
      "Progress: 98.81%\n",
      "Progress: 98.82%\n",
      "Progress: 98.82%\n",
      "Progress: 98.82%\n",
      "Progress: 98.82%\n",
      "Progress: 98.82%\n",
      "Progress: 98.82%\n",
      "Progress: 98.82%\n",
      "Progress: 98.83%\n",
      "Progress: 98.83%\n",
      "Progress: 98.83%\n",
      "Progress: 98.83%\n",
      "Progress: 98.83%\n",
      "Progress: 98.83%\n",
      "Progress: 98.84%\n",
      "Progress: 98.84%\n",
      "Progress: 98.84%\n",
      "Progress: 98.84%\n",
      "Progress: 98.84%\n",
      "Progress: 98.84%\n",
      "Progress: 98.84%\n",
      "Progress: 98.85%\n",
      "Progress: 98.85%\n",
      "Progress: 98.85%\n",
      "Progress: 98.85%\n",
      "Progress: 98.85%\n",
      "Progress: 98.85%\n",
      "Progress: 98.86%\n",
      "Progress: 98.86%\n",
      "Progress: 98.86%\n",
      "Progress: 98.86%\n",
      "Progress: 98.86%\n",
      "Progress: 98.86%\n",
      "Progress: 98.86%\n",
      "Progress: 98.87%\n",
      "Progress: 98.87%\n",
      "Progress: 98.87%\n",
      "Progress: 98.87%\n",
      "Progress: 98.87%\n",
      "Progress: 98.87%\n",
      "Progress: 98.88%\n",
      "Progress: 98.88%\n",
      "Progress: 98.88%\n",
      "Progress: 98.88%\n",
      "Progress: 98.88%\n",
      "Progress: 98.88%\n",
      "Progress: 98.88%\n",
      "Progress: 98.89%\n",
      "Progress: 98.89%\n",
      "Progress: 98.89%\n",
      "Progress: 98.89%\n",
      "Progress: 98.89%\n",
      "Progress: 98.89%\n",
      "Progress: 98.89%\n",
      "Progress: 98.90%\n",
      "Progress: 98.90%\n",
      "Progress: 98.90%\n",
      "Progress: 98.90%\n",
      "Progress: 98.90%\n",
      "Progress: 98.90%\n",
      "Progress: 98.91%\n",
      "Progress: 98.91%\n",
      "Progress: 98.91%\n",
      "Progress: 98.91%\n",
      "Progress: 98.91%\n",
      "Progress: 98.91%\n",
      "Progress: 98.91%\n",
      "Progress: 98.92%\n",
      "Progress: 98.92%\n",
      "Progress: 98.92%\n",
      "Progress: 98.92%\n",
      "Progress: 98.92%\n",
      "Progress: 98.92%\n",
      "Progress: 98.92%\n",
      "Progress: 98.93%\n",
      "Progress: 98.93%\n",
      "Progress: 98.93%\n",
      "Progress: 98.93%\n",
      "Progress: 98.93%\n",
      "Progress: 98.93%\n",
      "Progress: 98.93%\n",
      "Progress: 98.94%\n",
      "Progress: 98.94%\n",
      "Progress: 98.94%\n",
      "Progress: 98.94%\n",
      "Progress: 98.94%\n",
      "Progress: 98.94%\n",
      "Progress: 98.94%\n",
      "Progress: 98.95%\n",
      "Progress: 98.95%\n",
      "Progress: 98.95%\n",
      "Progress: 98.95%\n",
      "Progress: 98.95%\n",
      "Progress: 98.95%\n",
      "Progress: 98.96%\n",
      "Progress: 98.96%\n",
      "Progress: 98.96%\n",
      "Progress: 98.96%\n",
      "Progress: 98.96%\n",
      "Progress: 98.96%\n",
      "Progress: 98.96%\n",
      "Progress: 98.97%\n",
      "Progress: 98.97%\n",
      "Progress: 98.97%\n",
      "Progress: 98.97%\n",
      "Progress: 98.97%\n",
      "Progress: 98.97%\n",
      "Progress: 98.97%\n",
      "Progress: 98.98%\n",
      "Progress: 98.98%\n",
      "Progress: 98.98%\n",
      "Progress: 98.98%\n",
      "Progress: 98.98%\n",
      "Progress: 98.98%\n",
      "Progress: 98.98%\n",
      "Progress: 98.99%\n",
      "Progress: 98.99%\n",
      "Progress: 98.99%\n",
      "Progress: 98.99%\n",
      "Progress: 98.99%\n",
      "Progress: 98.99%\n",
      "Progress: 98.99%\n",
      "Progress: 99.00%\n",
      "Progress: 99.00%\n",
      "Progress: 99.00%\n",
      "Progress: 99.00%\n",
      "Progress: 99.00%\n",
      "Progress: 99.00%\n",
      "Progress: 99.00%\n",
      "Progress: 99.01%\n",
      "Progress: 99.01%\n",
      "Progress: 99.01%\n",
      "Progress: 99.01%\n",
      "Progress: 99.01%\n",
      "Progress: 99.01%\n",
      "Progress: 99.01%\n",
      "Progress: 99.02%\n",
      "Progress: 99.02%\n",
      "Progress: 99.02%\n",
      "Progress: 99.02%\n",
      "Progress: 99.02%\n",
      "Progress: 99.02%\n",
      "Progress: 99.02%\n",
      "Progress: 99.03%\n",
      "Progress: 99.03%\n",
      "Progress: 99.03%\n",
      "Progress: 99.03%\n",
      "Progress: 99.03%\n",
      "Progress: 99.03%\n",
      "Progress: 99.03%\n",
      "Progress: 99.04%\n",
      "Progress: 99.04%\n",
      "Progress: 99.04%\n",
      "Progress: 99.04%\n",
      "Progress: 99.04%\n",
      "Progress: 99.04%\n",
      "Progress: 99.04%\n",
      "Progress: 99.04%\n",
      "Progress: 99.05%\n",
      "Progress: 99.05%\n",
      "Progress: 99.05%\n",
      "Progress: 99.05%\n",
      "Progress: 99.05%\n",
      "Progress: 99.05%\n",
      "Progress: 99.05%\n",
      "Progress: 99.06%\n",
      "Progress: 99.06%\n",
      "Progress: 99.06%\n",
      "Progress: 99.06%\n",
      "Progress: 99.06%\n",
      "Progress: 99.06%\n",
      "Progress: 99.06%\n",
      "Progress: 99.07%\n",
      "Progress: 99.07%\n",
      "Progress: 99.07%\n",
      "Progress: 99.07%\n",
      "Progress: 99.07%\n",
      "Progress: 99.07%\n",
      "Progress: 99.07%\n",
      "Progress: 99.08%\n",
      "Progress: 99.08%\n",
      "Progress: 99.08%\n",
      "Progress: 99.08%\n",
      "Progress: 99.08%\n",
      "Progress: 99.08%\n",
      "Progress: 99.08%\n",
      "Progress: 99.08%\n",
      "Progress: 99.09%\n",
      "Progress: 99.09%\n",
      "Progress: 99.09%\n",
      "Progress: 99.09%\n",
      "Progress: 99.09%\n",
      "Progress: 99.09%\n",
      "Progress: 99.09%\n",
      "Progress: 99.10%\n",
      "Progress: 99.10%\n",
      "Progress: 99.10%\n",
      "Progress: 99.10%\n",
      "Progress: 99.10%\n",
      "Progress: 99.10%\n",
      "Progress: 99.10%\n",
      "Progress: 99.11%\n",
      "Progress: 99.11%\n",
      "Progress: 99.11%\n",
      "Progress: 99.11%\n",
      "Progress: 99.11%\n",
      "Progress: 99.11%\n",
      "Progress: 99.11%\n",
      "Progress: 99.11%\n",
      "Progress: 99.12%\n",
      "Progress: 99.12%\n",
      "Progress: 99.12%\n",
      "Progress: 99.12%\n",
      "Progress: 99.12%\n",
      "Progress: 99.12%\n",
      "Progress: 99.12%\n",
      "Progress: 99.13%\n",
      "Progress: 99.13%\n",
      "Progress: 99.13%\n",
      "Progress: 99.13%\n",
      "Progress: 99.13%\n",
      "Progress: 99.13%\n",
      "Progress: 99.13%\n",
      "Progress: 99.13%\n",
      "Progress: 99.14%\n",
      "Progress: 99.14%\n",
      "Progress: 99.14%\n",
      "Progress: 99.14%\n",
      "Progress: 99.14%\n",
      "Progress: 99.14%\n",
      "Progress: 99.14%\n",
      "Progress: 99.14%\n",
      "Progress: 99.15%\n",
      "Progress: 99.15%\n",
      "Progress: 99.15%\n",
      "Progress: 99.15%\n",
      "Progress: 99.15%\n",
      "Progress: 99.15%\n",
      "Progress: 99.15%\n",
      "Progress: 99.16%\n",
      "Progress: 99.16%\n",
      "Progress: 99.16%\n",
      "Progress: 99.16%\n",
      "Progress: 99.16%\n",
      "Progress: 99.16%\n",
      "Progress: 99.16%\n",
      "Progress: 99.16%\n",
      "Progress: 99.17%\n",
      "Progress: 99.17%\n",
      "Progress: 99.17%\n",
      "Progress: 99.17%\n",
      "Progress: 99.17%\n",
      "Progress: 99.17%\n",
      "Progress: 99.17%\n",
      "Progress: 99.17%\n",
      "Progress: 99.18%\n",
      "Progress: 99.18%\n",
      "Progress: 99.18%\n",
      "Progress: 99.18%\n",
      "Progress: 99.18%\n",
      "Progress: 99.18%\n",
      "Progress: 99.18%\n",
      "Progress: 99.19%\n",
      "Progress: 99.19%\n",
      "Progress: 99.19%\n",
      "Progress: 99.19%\n",
      "Progress: 99.19%\n",
      "Progress: 99.19%\n",
      "Progress: 99.19%\n",
      "Progress: 99.19%\n",
      "Progress: 99.20%\n",
      "Progress: 99.20%\n",
      "Progress: 99.20%\n",
      "Progress: 99.20%\n",
      "Progress: 99.20%\n",
      "Progress: 99.20%\n",
      "Progress: 99.20%\n",
      "Progress: 99.20%\n",
      "Progress: 99.21%\n",
      "Progress: 99.21%\n",
      "Progress: 99.21%\n",
      "Progress: 99.21%\n",
      "Progress: 99.21%\n",
      "Progress: 99.21%\n",
      "Progress: 99.21%\n",
      "Progress: 99.21%\n",
      "Progress: 99.22%\n",
      "Progress: 99.22%\n",
      "Progress: 99.22%\n",
      "Progress: 99.22%\n",
      "Progress: 99.22%\n",
      "Progress: 99.22%\n",
      "Progress: 99.22%\n",
      "Progress: 99.22%\n",
      "Progress: 99.23%\n",
      "Progress: 99.23%\n",
      "Progress: 99.23%\n",
      "Progress: 99.23%\n",
      "Progress: 99.23%\n",
      "Progress: 99.23%\n",
      "Progress: 99.23%\n",
      "Progress: 99.23%\n",
      "Progress: 99.24%\n",
      "Progress: 99.24%\n",
      "Progress: 99.24%\n",
      "Progress: 99.24%\n",
      "Progress: 99.24%\n",
      "Progress: 99.24%\n",
      "Progress: 99.24%\n",
      "Progress: 99.24%\n",
      "Progress: 99.25%\n",
      "Progress: 99.25%\n",
      "Progress: 99.25%\n",
      "Progress: 99.25%\n",
      "Progress: 99.25%\n",
      "Progress: 99.25%\n",
      "Progress: 99.25%\n",
      "Progress: 99.25%\n",
      "Progress: 99.26%\n",
      "Progress: 99.26%\n",
      "Progress: 99.26%\n",
      "Progress: 99.26%\n",
      "Progress: 99.26%\n",
      "Progress: 99.26%\n",
      "Progress: 99.26%\n",
      "Progress: 99.26%\n",
      "Progress: 99.27%\n",
      "Progress: 99.27%\n",
      "Progress: 99.27%\n",
      "Progress: 99.27%\n",
      "Progress: 99.27%\n",
      "Progress: 99.27%\n",
      "Progress: 99.27%\n",
      "Progress: 99.27%\n",
      "Progress: 99.28%\n",
      "Progress: 99.28%\n",
      "Progress: 99.28%\n",
      "Progress: 99.28%\n",
      "Progress: 99.28%\n",
      "Progress: 99.28%\n",
      "Progress: 99.28%\n",
      "Progress: 99.28%\n",
      "Progress: 99.28%\n",
      "Progress: 99.29%\n",
      "Progress: 99.29%\n",
      "Progress: 99.29%\n",
      "Progress: 99.29%\n",
      "Progress: 99.29%\n",
      "Progress: 99.29%\n",
      "Progress: 99.29%\n",
      "Progress: 99.29%\n",
      "Progress: 99.30%\n",
      "Progress: 99.30%\n",
      "Progress: 99.30%\n",
      "Progress: 99.30%\n",
      "Progress: 99.30%\n",
      "Progress: 99.30%\n",
      "Progress: 99.30%\n",
      "Progress: 99.30%\n",
      "Progress: 99.31%\n",
      "Progress: 99.31%\n",
      "Progress: 99.31%\n",
      "Progress: 99.31%\n",
      "Progress: 99.31%\n",
      "Progress: 99.31%\n",
      "Progress: 99.31%\n",
      "Progress: 99.31%\n",
      "Progress: 99.31%\n",
      "Progress: 99.32%\n",
      "Progress: 99.32%\n",
      "Progress: 99.32%\n",
      "Progress: 99.32%\n",
      "Progress: 99.32%\n",
      "Progress: 99.32%\n",
      "Progress: 99.32%\n",
      "Progress: 99.32%\n",
      "Progress: 99.33%\n",
      "Progress: 99.33%\n",
      "Progress: 99.33%\n",
      "Progress: 99.33%\n",
      "Progress: 99.33%\n",
      "Progress: 99.33%\n",
      "Progress: 99.33%\n",
      "Progress: 99.33%\n",
      "Progress: 99.33%\n",
      "Progress: 99.34%\n",
      "Progress: 99.34%\n",
      "Progress: 99.34%\n",
      "Progress: 99.34%\n",
      "Progress: 99.34%\n",
      "Progress: 99.34%\n",
      "Progress: 99.34%\n",
      "Progress: 99.34%\n",
      "Progress: 99.34%\n",
      "Progress: 99.35%\n",
      "Progress: 99.35%\n",
      "Progress: 99.35%\n",
      "Progress: 99.35%\n",
      "Progress: 99.35%\n",
      "Progress: 99.35%\n",
      "Progress: 99.35%\n",
      "Progress: 99.35%\n",
      "Progress: 99.36%\n",
      "Progress: 99.36%\n",
      "Progress: 99.36%\n",
      "Progress: 99.36%\n",
      "Progress: 99.36%\n",
      "Progress: 99.36%\n",
      "Progress: 99.36%\n",
      "Progress: 99.36%\n",
      "Progress: 99.36%\n",
      "Progress: 99.37%\n",
      "Progress: 99.37%\n",
      "Progress: 99.37%\n",
      "Progress: 99.37%\n",
      "Progress: 99.37%\n",
      "Progress: 99.37%\n",
      "Progress: 99.37%\n",
      "Progress: 99.37%\n",
      "Progress: 99.37%\n",
      "Progress: 99.38%\n",
      "Progress: 99.38%\n",
      "Progress: 99.38%\n",
      "Progress: 99.38%\n",
      "Progress: 99.38%\n",
      "Progress: 99.38%\n",
      "Progress: 99.38%\n",
      "Progress: 99.38%\n",
      "Progress: 99.38%\n",
      "Progress: 99.39%\n",
      "Progress: 99.39%\n",
      "Progress: 99.39%\n",
      "Progress: 99.39%\n",
      "Progress: 99.39%\n",
      "Progress: 99.39%\n",
      "Progress: 99.39%\n",
      "Progress: 99.39%\n",
      "Progress: 99.39%\n",
      "Progress: 99.40%\n",
      "Progress: 99.40%\n",
      "Progress: 99.40%\n",
      "Progress: 99.40%\n",
      "Progress: 99.40%\n",
      "Progress: 99.40%\n",
      "Progress: 99.40%\n",
      "Progress: 99.40%\n",
      "Progress: 99.40%\n",
      "Progress: 99.41%\n",
      "Progress: 99.41%\n",
      "Progress: 99.41%\n",
      "Progress: 99.41%\n",
      "Progress: 99.41%\n",
      "Progress: 99.41%\n",
      "Progress: 99.41%\n",
      "Progress: 99.41%\n",
      "Progress: 99.41%\n",
      "Progress: 99.42%\n",
      "Progress: 99.42%\n",
      "Progress: 99.42%\n",
      "Progress: 99.42%\n",
      "Progress: 99.42%\n",
      "Progress: 99.42%\n",
      "Progress: 99.42%\n",
      "Progress: 99.42%\n",
      "Progress: 99.42%\n",
      "Progress: 99.43%\n",
      "Progress: 99.43%\n",
      "Progress: 99.43%\n",
      "Progress: 99.43%\n",
      "Progress: 99.43%\n",
      "Progress: 99.43%\n",
      "Progress: 99.43%\n",
      "Progress: 99.43%\n",
      "Progress: 99.43%\n",
      "Progress: 99.43%\n",
      "Progress: 99.44%\n",
      "Progress: 99.44%\n",
      "Progress: 99.44%\n",
      "Progress: 99.44%\n",
      "Progress: 99.44%\n",
      "Progress: 99.44%\n",
      "Progress: 99.44%\n",
      "Progress: 99.44%\n",
      "Progress: 99.44%\n",
      "Progress: 99.45%\n",
      "Progress: 99.45%\n",
      "Progress: 99.45%\n",
      "Progress: 99.45%\n",
      "Progress: 99.45%\n",
      "Progress: 99.45%\n",
      "Progress: 99.45%\n",
      "Progress: 99.45%\n",
      "Progress: 99.45%\n",
      "Progress: 99.45%\n",
      "Progress: 99.46%\n",
      "Progress: 99.46%\n",
      "Progress: 99.46%\n",
      "Progress: 99.46%\n",
      "Progress: 99.46%\n",
      "Progress: 99.46%\n",
      "Progress: 99.46%\n",
      "Progress: 99.46%\n",
      "Progress: 99.46%\n",
      "Progress: 99.47%\n",
      "Progress: 99.47%\n",
      "Progress: 99.47%\n",
      "Progress: 99.47%\n",
      "Progress: 99.47%\n",
      "Progress: 99.47%\n",
      "Progress: 99.47%\n",
      "Progress: 99.47%\n",
      "Progress: 99.47%\n",
      "Progress: 99.47%\n",
      "Progress: 99.48%\n",
      "Progress: 99.48%\n",
      "Progress: 99.48%\n",
      "Progress: 99.48%\n",
      "Progress: 99.48%\n",
      "Progress: 99.48%\n",
      "Progress: 99.48%\n",
      "Progress: 99.48%\n",
      "Progress: 99.48%\n",
      "Progress: 99.49%\n",
      "Progress: 99.49%\n",
      "Progress: 99.49%\n",
      "Progress: 99.49%\n",
      "Progress: 99.49%\n",
      "Progress: 99.49%\n",
      "Progress: 99.49%\n",
      "Progress: 99.49%\n",
      "Progress: 99.49%\n",
      "Progress: 99.49%\n",
      "Progress: 99.50%\n",
      "Progress: 99.50%\n",
      "Progress: 99.50%\n",
      "Progress: 99.50%\n",
      "Progress: 99.50%\n",
      "Progress: 99.50%\n",
      "Progress: 99.50%\n",
      "Progress: 99.50%\n",
      "Progress: 99.50%\n",
      "Progress: 99.50%\n",
      "Progress: 99.51%\n",
      "Progress: 99.51%\n",
      "Progress: 99.51%\n",
      "Progress: 99.51%\n",
      "Progress: 99.51%\n",
      "Progress: 99.51%\n",
      "Progress: 99.51%\n",
      "Progress: 99.51%\n",
      "Progress: 99.51%\n",
      "Progress: 99.51%\n",
      "Progress: 99.52%\n",
      "Progress: 99.52%\n",
      "Progress: 99.52%\n",
      "Progress: 99.52%\n",
      "Progress: 99.52%\n",
      "Progress: 99.52%\n",
      "Progress: 99.52%\n",
      "Progress: 99.52%\n",
      "Progress: 99.52%\n",
      "Progress: 99.52%\n",
      "Progress: 99.53%\n",
      "Progress: 99.53%\n",
      "Progress: 99.53%\n",
      "Progress: 99.53%\n",
      "Progress: 99.53%\n",
      "Progress: 99.53%\n",
      "Progress: 99.53%\n",
      "Progress: 99.53%\n",
      "Progress: 99.53%\n",
      "Progress: 99.53%\n",
      "Progress: 99.53%\n",
      "Progress: 99.54%\n",
      "Progress: 99.54%\n",
      "Progress: 99.54%\n",
      "Progress: 99.54%\n",
      "Progress: 99.54%\n",
      "Progress: 99.54%\n",
      "Progress: 99.54%\n",
      "Progress: 99.54%\n",
      "Progress: 99.54%\n",
      "Progress: 99.54%\n",
      "Progress: 99.55%\n",
      "Progress: 99.55%\n",
      "Progress: 99.55%\n",
      "Progress: 99.55%\n",
      "Progress: 99.55%\n",
      "Progress: 99.55%\n",
      "Progress: 99.55%\n",
      "Progress: 99.55%\n",
      "Progress: 99.55%\n",
      "Progress: 99.55%\n",
      "Progress: 99.55%\n",
      "Progress: 99.56%\n",
      "Progress: 99.56%\n",
      "Progress: 99.56%\n",
      "Progress: 99.56%\n",
      "Progress: 99.56%\n",
      "Progress: 99.56%\n",
      "Progress: 99.56%\n",
      "Progress: 99.56%\n",
      "Progress: 99.56%\n",
      "Progress: 99.56%\n",
      "Progress: 99.57%\n",
      "Progress: 99.57%\n",
      "Progress: 99.57%\n",
      "Progress: 99.57%\n",
      "Progress: 99.57%\n",
      "Progress: 99.57%\n",
      "Progress: 99.57%\n",
      "Progress: 99.57%\n",
      "Progress: 99.57%\n",
      "Progress: 99.57%\n",
      "Progress: 99.57%\n",
      "Progress: 99.58%\n",
      "Progress: 99.58%\n",
      "Progress: 99.58%\n",
      "Progress: 99.58%\n",
      "Progress: 99.58%\n",
      "Progress: 99.58%\n",
      "Progress: 99.58%\n",
      "Progress: 99.58%\n",
      "Progress: 99.58%\n",
      "Progress: 99.58%\n",
      "Progress: 99.58%\n",
      "Progress: 99.59%\n",
      "Progress: 99.59%\n",
      "Progress: 99.59%\n",
      "Progress: 99.59%\n",
      "Progress: 99.59%\n",
      "Progress: 99.59%\n",
      "Progress: 99.59%\n",
      "Progress: 99.59%\n",
      "Progress: 99.59%\n",
      "Progress: 99.59%\n",
      "Progress: 99.59%\n",
      "Progress: 99.60%\n",
      "Progress: 99.60%\n",
      "Progress: 99.60%\n",
      "Progress: 99.60%\n",
      "Progress: 99.60%\n",
      "Progress: 99.60%\n",
      "Progress: 99.60%\n",
      "Progress: 99.60%\n",
      "Progress: 99.60%\n",
      "Progress: 99.60%\n",
      "Progress: 99.60%\n",
      "Progress: 99.61%\n",
      "Progress: 99.61%\n",
      "Progress: 99.61%\n",
      "Progress: 99.61%\n",
      "Progress: 99.61%\n",
      "Progress: 99.61%\n",
      "Progress: 99.61%\n",
      "Progress: 99.61%\n",
      "Progress: 99.61%\n",
      "Progress: 99.61%\n",
      "Progress: 99.61%\n",
      "Progress: 99.62%\n",
      "Progress: 99.62%\n",
      "Progress: 99.62%\n",
      "Progress: 99.62%\n",
      "Progress: 99.62%\n",
      "Progress: 99.62%\n",
      "Progress: 99.62%\n",
      "Progress: 99.62%\n",
      "Progress: 99.62%\n",
      "Progress: 99.62%\n",
      "Progress: 99.62%\n",
      "Progress: 99.63%\n",
      "Progress: 99.63%\n",
      "Progress: 99.63%\n",
      "Progress: 99.63%\n",
      "Progress: 99.63%\n",
      "Progress: 99.63%\n",
      "Progress: 99.63%\n",
      "Progress: 99.63%\n",
      "Progress: 99.63%\n",
      "Progress: 99.63%\n",
      "Progress: 99.63%\n",
      "Progress: 99.63%\n",
      "Progress: 99.64%\n",
      "Progress: 99.64%\n",
      "Progress: 99.64%\n",
      "Progress: 99.64%\n",
      "Progress: 99.64%\n",
      "Progress: 99.64%\n",
      "Progress: 99.64%\n",
      "Progress: 99.64%\n",
      "Progress: 99.64%\n",
      "Progress: 99.64%\n",
      "Progress: 99.64%\n",
      "Progress: 99.64%\n",
      "Progress: 99.65%\n",
      "Progress: 99.65%\n",
      "Progress: 99.65%\n",
      "Progress: 99.65%\n",
      "Progress: 99.65%\n",
      "Progress: 99.65%\n",
      "Progress: 99.65%\n",
      "Progress: 99.65%\n",
      "Progress: 99.65%\n",
      "Progress: 99.65%\n",
      "Progress: 99.65%\n",
      "Progress: 99.65%\n",
      "Progress: 99.66%\n",
      "Progress: 99.66%\n",
      "Progress: 99.66%\n",
      "Progress: 99.66%\n",
      "Progress: 99.66%\n",
      "Progress: 99.66%\n",
      "Progress: 99.66%\n",
      "Progress: 99.66%\n",
      "Progress: 99.66%\n",
      "Progress: 99.66%\n",
      "Progress: 99.66%\n",
      "Progress: 99.66%\n",
      "Progress: 99.67%\n",
      "Progress: 99.67%\n",
      "Progress: 99.67%\n",
      "Progress: 99.67%\n",
      "Progress: 99.67%\n",
      "Progress: 99.67%\n",
      "Progress: 99.67%\n",
      "Progress: 99.67%\n",
      "Progress: 99.67%\n",
      "Progress: 99.67%\n",
      "Progress: 99.67%\n",
      "Progress: 99.67%\n",
      "Progress: 99.68%\n",
      "Progress: 99.68%\n",
      "Progress: 99.68%\n",
      "Progress: 99.68%\n",
      "Progress: 99.68%\n",
      "Progress: 99.68%\n",
      "Progress: 99.68%\n",
      "Progress: 99.68%\n",
      "Progress: 99.68%\n",
      "Progress: 99.68%\n",
      "Progress: 99.68%\n",
      "Progress: 99.68%\n",
      "Progress: 99.69%\n",
      "Progress: 99.69%\n",
      "Progress: 99.69%\n",
      "Progress: 99.69%\n",
      "Progress: 99.69%\n",
      "Progress: 99.69%\n",
      "Progress: 99.69%\n",
      "Progress: 99.69%\n",
      "Progress: 99.69%\n",
      "Progress: 99.69%\n",
      "Progress: 99.69%\n",
      "Progress: 99.69%\n",
      "Progress: 99.69%\n",
      "Progress: 99.70%\n",
      "Progress: 99.70%\n",
      "Progress: 99.70%\n",
      "Progress: 99.70%\n",
      "Progress: 99.70%\n",
      "Progress: 99.70%\n",
      "Progress: 99.70%\n",
      "Progress: 99.70%\n",
      "Progress: 99.70%\n",
      "Progress: 99.70%\n",
      "Progress: 99.70%\n",
      "Progress: 99.70%\n",
      "Progress: 99.70%\n",
      "Progress: 99.71%\n",
      "Progress: 99.71%\n",
      "Progress: 99.71%\n",
      "Progress: 99.71%\n",
      "Progress: 99.71%\n",
      "Progress: 99.71%\n",
      "Progress: 99.71%\n",
      "Progress: 99.71%\n",
      "Progress: 99.71%\n",
      "Progress: 99.71%\n",
      "Progress: 99.71%\n",
      "Progress: 99.71%\n",
      "Progress: 99.71%\n",
      "Progress: 99.72%\n",
      "Progress: 99.72%\n",
      "Progress: 99.72%\n",
      "Progress: 99.72%\n",
      "Progress: 99.72%\n",
      "Progress: 99.72%\n",
      "Progress: 99.72%\n",
      "Progress: 99.72%\n",
      "Progress: 99.72%\n",
      "Progress: 99.72%\n",
      "Progress: 99.72%\n",
      "Progress: 99.72%\n",
      "Progress: 99.72%\n",
      "Progress: 99.73%\n",
      "Progress: 99.73%\n",
      "Progress: 99.73%\n",
      "Progress: 99.73%\n",
      "Progress: 99.73%\n",
      "Progress: 99.73%\n",
      "Progress: 99.73%\n",
      "Progress: 99.73%\n",
      "Progress: 99.73%\n",
      "Progress: 99.73%\n",
      "Progress: 99.73%\n",
      "Progress: 99.73%\n",
      "Progress: 99.73%\n",
      "Progress: 99.73%\n",
      "Progress: 99.74%\n",
      "Progress: 99.74%\n",
      "Progress: 99.74%\n",
      "Progress: 99.74%\n",
      "Progress: 99.74%\n",
      "Progress: 99.74%\n",
      "Progress: 99.74%\n",
      "Progress: 99.74%\n",
      "Progress: 99.74%\n",
      "Progress: 99.74%\n",
      "Progress: 99.74%\n",
      "Progress: 99.74%\n",
      "Progress: 99.74%\n",
      "Progress: 99.75%\n",
      "Progress: 99.75%\n",
      "Progress: 99.75%\n",
      "Progress: 99.75%\n",
      "Progress: 99.75%\n",
      "Progress: 99.75%\n",
      "Progress: 99.75%\n",
      "Progress: 99.75%\n",
      "Progress: 99.75%\n",
      "Progress: 99.75%\n",
      "Progress: 99.75%\n",
      "Progress: 99.75%\n",
      "Progress: 99.75%\n",
      "Progress: 99.75%\n",
      "Progress: 99.76%\n",
      "Progress: 99.76%\n",
      "Progress: 99.76%\n",
      "Progress: 99.76%\n",
      "Progress: 99.76%\n",
      "Progress: 99.76%\n",
      "Progress: 99.76%\n",
      "Progress: 99.76%\n",
      "Progress: 99.76%\n",
      "Progress: 99.76%\n",
      "Progress: 99.76%\n",
      "Progress: 99.76%\n",
      "Progress: 99.76%\n",
      "Progress: 99.76%\n",
      "Progress: 99.76%\n",
      "Progress: 99.77%\n",
      "Progress: 99.77%\n",
      "Progress: 99.77%\n",
      "Progress: 99.77%\n",
      "Progress: 99.77%\n",
      "Progress: 99.77%\n",
      "Progress: 99.77%\n",
      "Progress: 99.77%\n",
      "Progress: 99.77%\n",
      "Progress: 99.77%\n",
      "Progress: 99.77%\n",
      "Progress: 99.77%\n",
      "Progress: 99.77%\n",
      "Progress: 99.77%\n",
      "Progress: 99.78%\n",
      "Progress: 99.78%\n",
      "Progress: 99.78%\n",
      "Progress: 99.78%\n",
      "Progress: 99.78%\n",
      "Progress: 99.78%\n",
      "Progress: 99.78%\n",
      "Progress: 99.78%\n",
      "Progress: 99.78%\n",
      "Progress: 99.78%\n",
      "Progress: 99.78%\n",
      "Progress: 99.78%\n",
      "Progress: 99.78%\n",
      "Progress: 99.78%\n",
      "Progress: 99.78%\n",
      "Progress: 99.79%\n",
      "Progress: 99.79%\n",
      "Progress: 99.79%\n",
      "Progress: 99.79%\n",
      "Progress: 99.79%\n",
      "Progress: 99.79%\n",
      "Progress: 99.79%\n",
      "Progress: 99.79%\n",
      "Progress: 99.79%\n",
      "Progress: 99.79%\n",
      "Progress: 99.79%\n",
      "Progress: 99.79%\n",
      "Progress: 99.79%\n",
      "Progress: 99.79%\n",
      "Progress: 99.79%\n",
      "Progress: 99.79%\n",
      "Progress: 99.80%\n",
      "Progress: 99.80%\n",
      "Progress: 99.80%\n",
      "Progress: 99.80%\n",
      "Progress: 99.80%\n",
      "Progress: 99.80%\n",
      "Progress: 99.80%\n",
      "Progress: 99.80%\n",
      "Progress: 99.80%\n",
      "Progress: 99.80%\n",
      "Progress: 99.80%\n",
      "Progress: 99.80%\n",
      "Progress: 99.80%\n",
      "Progress: 99.80%\n",
      "Progress: 99.80%\n",
      "Progress: 99.81%\n",
      "Progress: 99.81%\n",
      "Progress: 99.81%\n",
      "Progress: 99.81%\n",
      "Progress: 99.81%\n",
      "Progress: 99.81%\n",
      "Progress: 99.81%\n",
      "Progress: 99.81%\n",
      "Progress: 99.81%\n",
      "Progress: 99.81%\n",
      "Progress: 99.81%\n",
      "Progress: 99.81%\n",
      "Progress: 99.81%\n",
      "Progress: 99.81%\n",
      "Progress: 99.81%\n",
      "Progress: 99.81%\n",
      "Progress: 99.81%\n",
      "Progress: 99.82%\n",
      "Progress: 99.82%\n",
      "Progress: 99.82%\n",
      "Progress: 99.82%\n",
      "Progress: 99.82%\n",
      "Progress: 99.82%\n",
      "Progress: 99.82%\n",
      "Progress: 99.82%\n",
      "Progress: 99.82%\n",
      "Progress: 99.82%\n",
      "Progress: 99.82%\n",
      "Progress: 99.82%\n",
      "Progress: 99.82%\n",
      "Progress: 99.82%\n",
      "Progress: 99.82%\n",
      "Progress: 99.82%\n",
      "Progress: 99.83%\n",
      "Progress: 99.83%\n",
      "Progress: 99.83%\n",
      "Progress: 99.83%\n",
      "Progress: 99.83%\n",
      "Progress: 99.83%\n",
      "Progress: 99.83%\n",
      "Progress: 99.83%\n",
      "Progress: 99.83%\n",
      "Progress: 99.83%\n",
      "Progress: 99.83%\n",
      "Progress: 99.83%\n",
      "Progress: 99.83%\n",
      "Progress: 99.83%\n",
      "Progress: 99.83%\n",
      "Progress: 99.83%\n",
      "Progress: 99.83%\n",
      "Progress: 99.84%\n",
      "Progress: 99.84%\n",
      "Progress: 99.84%\n",
      "Progress: 99.84%\n",
      "Progress: 99.84%\n",
      "Progress: 99.84%\n",
      "Progress: 99.84%\n",
      "Progress: 99.84%\n",
      "Progress: 99.84%\n",
      "Progress: 99.84%\n",
      "Progress: 99.84%\n",
      "Progress: 99.84%\n",
      "Progress: 99.84%\n",
      "Progress: 99.84%\n",
      "Progress: 99.84%\n",
      "Progress: 99.84%\n",
      "Progress: 99.84%\n",
      "Progress: 99.84%\n",
      "Progress: 99.85%\n",
      "Progress: 99.85%\n",
      "Progress: 99.85%\n",
      "Progress: 99.85%\n",
      "Progress: 99.85%\n",
      "Progress: 99.85%\n",
      "Progress: 99.85%\n",
      "Progress: 99.85%\n",
      "Progress: 99.85%\n",
      "Progress: 99.85%\n",
      "Progress: 99.85%\n",
      "Progress: 99.85%\n",
      "Progress: 99.85%\n",
      "Progress: 99.85%\n",
      "Progress: 99.85%\n",
      "Progress: 99.85%\n",
      "Progress: 99.85%\n",
      "Progress: 99.85%\n",
      "Progress: 99.86%\n",
      "Progress: 99.86%\n",
      "Progress: 99.86%\n",
      "Progress: 99.86%\n",
      "Progress: 99.86%\n",
      "Progress: 99.86%\n",
      "Progress: 99.86%\n",
      "Progress: 99.86%\n",
      "Progress: 99.86%\n",
      "Progress: 99.86%\n",
      "Progress: 99.86%\n",
      "Progress: 99.86%\n",
      "Progress: 99.86%\n",
      "Progress: 99.86%\n",
      "Progress: 99.86%\n",
      "Progress: 99.86%\n",
      "Progress: 99.86%\n",
      "Progress: 99.86%\n",
      "Progress: 99.86%\n",
      "Progress: 99.87%\n",
      "Progress: 99.87%\n",
      "Progress: 99.87%\n",
      "Progress: 99.87%\n",
      "Progress: 99.87%\n",
      "Progress: 99.87%\n",
      "Progress: 99.87%\n",
      "Progress: 99.87%\n",
      "Progress: 99.87%\n",
      "Progress: 99.87%\n",
      "Progress: 99.87%\n",
      "Progress: 99.87%\n",
      "Progress: 99.87%\n",
      "Progress: 99.87%\n",
      "Progress: 99.87%\n",
      "Progress: 99.87%\n",
      "Progress: 99.87%\n",
      "Progress: 99.87%\n",
      "Progress: 99.87%\n",
      "Progress: 99.88%\n",
      "Progress: 99.88%\n",
      "Progress: 99.88%\n",
      "Progress: 99.88%\n",
      "Progress: 99.88%\n",
      "Progress: 99.88%\n",
      "Progress: 99.88%\n",
      "Progress: 99.88%\n",
      "Progress: 99.88%\n",
      "Progress: 99.88%\n",
      "Progress: 99.88%\n",
      "Progress: 99.88%\n",
      "Progress: 99.88%\n",
      "Progress: 99.88%\n",
      "Progress: 99.88%\n",
      "Progress: 99.88%\n",
      "Progress: 99.88%\n",
      "Progress: 99.88%\n",
      "Progress: 99.88%\n",
      "Progress: 99.88%\n",
      "Progress: 99.89%\n",
      "Progress: 99.89%\n",
      "Progress: 99.89%\n",
      "Progress: 99.89%\n",
      "Progress: 99.89%\n",
      "Progress: 99.89%\n",
      "Progress: 99.89%\n",
      "Progress: 99.89%\n",
      "Progress: 99.89%\n",
      "Progress: 99.89%\n",
      "Progress: 99.89%\n",
      "Progress: 99.89%\n",
      "Progress: 99.89%\n",
      "Progress: 99.89%\n",
      "Progress: 99.89%\n",
      "Progress: 99.89%\n",
      "Progress: 99.89%\n",
      "Progress: 99.89%\n",
      "Progress: 99.89%\n",
      "Progress: 99.89%\n",
      "Progress: 99.89%\n",
      "Progress: 99.89%\n",
      "Progress: 99.90%\n",
      "Progress: 99.90%\n",
      "Progress: 99.90%\n",
      "Progress: 99.90%\n",
      "Progress: 99.90%\n",
      "Progress: 99.90%\n",
      "Progress: 99.90%\n",
      "Progress: 99.90%\n",
      "Progress: 99.90%\n",
      "Progress: 99.90%\n",
      "Progress: 99.90%\n",
      "Progress: 99.90%\n",
      "Progress: 99.90%\n",
      "Progress: 99.90%\n",
      "Progress: 99.90%\n",
      "Progress: 99.90%\n",
      "Progress: 99.90%\n",
      "Progress: 99.90%\n",
      "Progress: 99.90%\n",
      "Progress: 99.90%\n",
      "Progress: 99.90%\n",
      "Progress: 99.90%\n",
      "Progress: 99.91%\n",
      "Progress: 99.91%\n",
      "Progress: 99.91%\n",
      "Progress: 99.91%\n",
      "Progress: 99.91%\n",
      "Progress: 99.91%\n",
      "Progress: 99.91%\n",
      "Progress: 99.91%\n",
      "Progress: 99.91%\n",
      "Progress: 99.91%\n",
      "Progress: 99.91%\n",
      "Progress: 99.91%\n",
      "Progress: 99.91%\n",
      "Progress: 99.91%\n",
      "Progress: 99.91%\n",
      "Progress: 99.91%\n",
      "Progress: 99.91%\n",
      "Progress: 99.91%\n",
      "Progress: 99.91%\n",
      "Progress: 99.91%\n",
      "Progress: 99.91%\n",
      "Progress: 99.91%\n",
      "Progress: 99.91%\n",
      "Progress: 99.92%\n",
      "Progress: 99.92%\n",
      "Progress: 99.92%\n",
      "Progress: 99.92%\n",
      "Progress: 99.92%\n",
      "Progress: 99.92%\n",
      "Progress: 99.92%\n",
      "Progress: 99.92%\n",
      "Progress: 99.92%\n",
      "Progress: 99.92%\n",
      "Progress: 99.92%\n",
      "Progress: 99.92%\n",
      "Progress: 99.92%\n",
      "Progress: 99.92%\n",
      "Progress: 99.92%\n",
      "Progress: 99.92%\n",
      "Progress: 99.92%\n",
      "Progress: 99.92%\n",
      "Progress: 99.92%\n",
      "Progress: 99.92%\n",
      "Progress: 99.92%\n",
      "Progress: 99.92%\n",
      "Progress: 99.92%\n",
      "Progress: 99.92%\n",
      "Progress: 99.92%\n",
      "Progress: 99.93%\n",
      "Progress: 99.93%\n",
      "Progress: 99.93%\n",
      "Progress: 99.93%\n",
      "Progress: 99.93%\n",
      "Progress: 99.93%\n",
      "Progress: 99.93%\n",
      "Progress: 99.93%\n",
      "Progress: 99.93%\n",
      "Progress: 99.93%\n",
      "Progress: 99.93%\n",
      "Progress: 99.93%\n",
      "Progress: 99.93%\n",
      "Progress: 99.93%\n",
      "Progress: 99.93%\n",
      "Progress: 99.93%\n",
      "Progress: 99.93%\n",
      "Progress: 99.93%\n",
      "Progress: 99.93%\n",
      "Progress: 99.93%\n",
      "Progress: 99.93%\n",
      "Progress: 99.93%\n",
      "Progress: 99.93%\n",
      "Progress: 99.93%\n",
      "Progress: 99.93%\n",
      "Progress: 99.93%\n",
      "Progress: 99.93%\n",
      "Progress: 99.94%\n",
      "Progress: 99.94%\n",
      "Progress: 99.94%\n",
      "Progress: 99.94%\n",
      "Progress: 99.94%\n",
      "Progress: 99.94%\n",
      "Progress: 99.94%\n",
      "Progress: 99.94%\n",
      "Progress: 99.94%\n",
      "Progress: 99.94%\n",
      "Progress: 99.94%\n",
      "Progress: 99.94%\n",
      "Progress: 99.94%\n",
      "Progress: 99.94%\n",
      "Progress: 99.94%\n",
      "Progress: 99.94%\n",
      "Progress: 99.94%\n",
      "Progress: 99.94%\n",
      "Progress: 99.94%\n",
      "Progress: 99.94%\n",
      "Progress: 99.94%\n",
      "Progress: 99.94%\n",
      "Progress: 99.94%\n",
      "Progress: 99.94%\n",
      "Progress: 99.94%\n",
      "Progress: 99.94%\n",
      "Progress: 99.94%\n",
      "Progress: 99.94%\n",
      "Progress: 99.95%\n",
      "Progress: 99.95%\n",
      "Progress: 99.95%\n",
      "Progress: 99.95%\n",
      "Progress: 99.95%\n",
      "Progress: 99.95%\n",
      "Progress: 99.95%\n",
      "Progress: 99.95%\n",
      "Progress: 99.95%\n",
      "Progress: 99.95%\n",
      "Progress: 99.95%\n",
      "Progress: 99.95%\n",
      "Progress: 99.95%\n",
      "Progress: 99.95%\n",
      "Progress: 99.95%\n",
      "Progress: 99.95%\n",
      "Progress: 99.95%\n",
      "Progress: 99.95%\n",
      "Progress: 99.95%\n",
      "Progress: 99.95%\n",
      "Progress: 99.95%\n",
      "Progress: 99.95%\n",
      "Progress: 99.95%\n",
      "Progress: 99.95%\n",
      "Progress: 99.95%\n",
      "Progress: 99.95%\n",
      "Progress: 99.95%\n",
      "Progress: 99.95%\n",
      "Progress: 99.95%\n",
      "Progress: 99.95%\n",
      "Progress: 99.95%\n",
      "Progress: 99.95%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.96%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.97%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.98%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 99.99%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 100.00%\n",
      "Progress: 5.00% complete.\n",
      "Progress: 10.00% complete.\n",
      "Progress: 15.00% complete.\n",
      "Progress: 20.00% complete.\n",
      "Progress: 25.00% complete.\n",
      "Progress: 30.00% complete.\n",
      "Progress: 35.00% complete.\n",
      "Progress: 40.00% complete.\n",
      "Progress: 45.00% complete.\n",
      "Progress: 50.00% complete.\n",
      "Progress: 55.00% complete.\n",
      "Progress: 60.00% complete.\n",
      "Progress: 65.00% complete.\n",
      "Progress: 70.00% complete.\n",
      "Progress: 75.00% complete.\n",
      "Progress: 80.00% complete.\n",
      "Progress: 85.00% complete.\n",
      "Progress: 90.00% complete.\n",
      "Progress: 95.00% complete.\n",
      "Progress: 100.00% complete.\n",
      "Excel file has been modified and saved as: modified_updated(Qwen2.5 32b)_saccharomyces cerevisiae production gene expressionsaccharomyces cerevisiae production gene deletion_causal.xlsx\n"
     ]
    }
   ],
   "source": [
    "# remove repeat words\n",
    "import pandas as pd\n",
    "import re\n",
    "\n",
    "# Load an Excel file\n",
    "df = pd.read_excel(output_file_path, engine='openpyxl')\n",
    "\n",
    "# Fill NaN values in 'Response to New Question' column with zero\n",
    "df['Answer to Question 2'] = df['Answer to Question 2'].fillna(0)\n",
    "\n",
    "# Convert the column values to strings (to ensure compatibility with re.findall)\n",
    "column_values = df['Answer to Question 2'].astype(str).tolist()\n",
    "\n",
    "# Initialize an empty list to hold entities\n",
    "entities = []\n",
    "\n",
    "# Regular expression to match the pattern (entity A, entity B)\n",
    "pattern = r'\\(([^,]+), ([^\\)]+)\\)'\n",
    "\n",
    "# Iterate over each cell in the column\n",
    "for value in column_values:\n",
    "    # Find all matches of the pattern in the cell\n",
    "    matches = re.findall(pattern, value)\n",
    "    # For each match, extend the entities list with the extracted entities\n",
    "    for match in matches:\n",
    "        entities.extend(match)  # This adds both entity A and entity B to the list\n",
    "\n",
    "# Remove duplicates if necessary\n",
    "entities = list(dict.fromkeys(entities))\n",
    "\n",
    "# Join the entities with commas\n",
    "entities_string = ', '.join(entities)\n",
    "\n",
    "print(entities_string)\n",
    "\n",
    "import pandas as pd\n",
    "from sentence_transformers import SentenceTransformer, util\n",
    "\n",
    "# Load your Excel file\n",
    "file_path = output_file_path\n",
    "df = pd.read_excel(file_path, engine='openpyxl')\n",
    "\n",
    "# Assuming you have a list of entities and their embeddings already\n",
    "entities = [entity.strip() for entity in entities_string.split(',')]\n",
    "t2vmodel = SentenceTransformer('all-MiniLM-L6-v2')\n",
    "embeddings = t2vmodel.encode(entities)\n",
    "\n",
    "# Identify similar phrases and store them in a dictionary\n",
    "similar_phrases = {}\n",
    "\n",
    "# Calculate total iterations for progress tracking\n",
    "total_iterations = sum(range(len(entities)))\n",
    "\n",
    "# Initialize a counter to track progress\n",
    "current_iteration = 0\n",
    "\n",
    "for i in range(len(entities)):\n",
    "    for j in range(i + 1, len(entities)):\n",
    "        similarity = util.pytorch_cos_sim(embeddings[i], embeddings[j])\n",
    "        if similarity.item() > 0.8:\n",
    "            # Assuming entities[i] is the first phrase and entities[j] is the similar one\n",
    "            similar_phrases[entities[j]] = entities[i]\n",
    "\n",
    "        # Update the current iteration counter after each inner loop iteration\n",
    "        current_iteration += 1\n",
    "\n",
    "    # Print the percentage completed\n",
    "    percentage_completed = (current_iteration / total_iterations) * 100\n",
    "    print(f\"Progress: {percentage_completed:.2f}%\")\n",
    "\n",
    "# Note: Printing progress in the inner loop might slow down your code execution,\n",
    "# especially if 'entities' is very large. You might want to update the progress\n",
    "# less frequently, for example, only after each completion of the outer loop.\n",
    "\n",
    "\n",
    "# Specify the column you want to modify\n",
    "specific_column = 'Answer to Question 2'\n",
    "\n",
    "# Calculate total iterations for progress tracking (only for the specific column)\n",
    "total_rows = len(df)\n",
    "current_iteration = 0\n",
    "\n",
    "# Iterate through the specific column to substitute similar phrases\n",
    "for index, row in df.iterrows():\n",
    "    current_iteration += 1\n",
    "    # Print progress every 100 rows to avoid performance degradation\n",
    "    if current_iteration % 100 == 0 or current_iteration == total_rows:\n",
    "        progress_percentage = (current_iteration / total_rows) * 100\n",
    "        print(f\"Progress: {progress_percentage:.2f}% complete.\")\n",
    "    \n",
    "    cell_value = str(row[specific_column])\n",
    "    for similar, original in similar_phrases.items():\n",
    "        # Check if the phrase contains 'Yarrowia', if so, skip substitution\n",
    "        if 'Yarrowia' in cell_value or 'Yarrowia' in similar:\n",
    "            continue\n",
    "        if similar in cell_value:\n",
    "            # Substitute similar phrase with the original phrase, ignoring errors if not found\n",
    "            try:\n",
    "                df.at[index, specific_column] = cell_value.replace(similar, original)\n",
    "            except Exception as e:\n",
    "                print(f\"Error substituting phrase: {e}\")\n",
    "                continue\n",
    "\n",
    "# Save the modified Excel file\n",
    "modified_file_path = 'modified_' + file_path\n",
    "df.to_excel(modified_file_path, index=False, engine='openpyxl')\n",
    "\n",
    "print(\"Excel file has been modified and saved as:\", modified_file_path)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "e9f5ece9-0a9a-45af-af4d-5583a119824a",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "4311922f-a6ec-4a92-bd85-bd7540ad0291",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
